---
title: "Intro to ggplot2"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2020**

## Objective for this notebook analysis: 

We'll use a real gene expression dataset to get comfortable making visualizations using ggplot2. 
We've [performed differential expression analyses](./scripts/00-setup-intro-to-R.R) on a pre-processed [astrocytoma microarray dataset](https://www.refine.bio/experiments/GSE44971/gene-expression-data-from-pilocytic-astrocytoma-tumour-samples-and-normal-cerebellum-controls).
We'll start by making a volcano plot of differential gene expression results from this experiment.
We performed three sets of contrasts:  

1) `sex` category contrasting: `Male` vs `Female`  
2) `tissue` category contrasting : `Pilocytic astrocytoma tumor` samples vs `normal cerebellum` samples  
3) An interaction of both `sex` and `tissue`. 

**More ggplot2 resources:**  
+ [handy cheatsheet for ggplot2](https://rstudio.com/wp-content/uploads/2015/03/ggplot2-cheatsheet.pdf)  
+ [ggplot2 courses and books from Tidyverse](https://ggplot2.tidyverse.org/)  
+ [ggplot2 data viz chapter of R for Data Science](https://r4ds.had.co.nz/data-visualisation.html)  
+ [ggplot2 online tutorial](http://r-statistics.co/Complete-Ggplot2-Tutorial-Part1-With-R-Code.html)  

## Set Up

We saved these results to a tab separated values (TSV) file called `gene_results_GSE44971.tsv`.
It's been saved to the `data` folder. 
File paths are relative to where this notebook file (.Rmd) is saved.
So we can reference it later, let's make a variable with our data directory name. 

```{r}
data_dir <- "data"
```

Let's declare our output folder name as its own variable. 

```{r}
plots_dir <- "plots"
```

We can also create a directory if it doesn't already exist. 

```{r}
# The if statement here tests whether the plot directory exists and
# only executes the  expressions between the braces if it does not.
if (!dir.exists(plots_dir)) {
  dir.create(plots_dir)
}
```

In this notebook we will be using functions from the Tidyverse set of packages, so we need to load in those functions using `library()`.
We could load the individual packages we need one at a time, but it is convenient for now to load them all with the `tidyverse` package, which groups many of them together.
Keep a look out for where we tell you which individual package different functions come from.

```{r}
library(tidyverse)
```

## Read in the differential expression analysis results file

Here we are using a `tidyverse` function `read_tsv()` from the `readr` package.
We are also using `file.path()`, which allows us to specify file locations that will work regardless of the operating system being used. 
The resulting data.frame will be stored in a variable named `stats_df`.

```{r}
stats_df <- read_tsv(file.path(
  data_dir,
  "gene_results_GSE44971.tsv"
))
```

Let's take a look at what this data.frame looks like: 

```{r}
stats_df
```

We can take a look at a column individually by using a `$`. 
Note we are using `head()` so the whole thing doesn't print out. 

```{r}
head(stats_df$contrast)
```

If we want to see a specific set of values, we can use brackets with the indices of the values we'd like returned.

```{r}
stats_df$avg_expression[6:10]
```

Let's checkout how many rows of data there are per each contrast.

```{r}
summary(stats_df)
```

## Set up the dataset

Before we make our plot, we want to calculate a set of new values for each row; transformations of the raw statistics in our table.
To do this we will use a function from the `dplyr` package called `mutate()` to make a new column of -log10 p values.

```{r}
stats_df <- mutate(stats_df, # data.frame we'd like to add a variable to
  neg_log_p_val = -log10(p_value)
)
```

Let's filter to only `male_female` contrast data. 
First let's try out a logical expression: 

```{r}
stats_df$contrast == "male_female"
```

Notice that we are not assigning the results to a variable, so this filtered dataset will not be saved to the environment

```{r}
filter(stats_df, contrast == "male_female")
```

Now we can assign the results to a new data.frame: `male_female_df`. 

```{r}
male_female_df <- filter(stats_df, contrast == "male_female")
```

## Plotting this data

Let's make a volcano plot with this data. 
First let's take a look at only the tumor vs. normal comparison. 
Let's save this as a separate data.frame by assigning it a new name. 

```{r}
tumor_normal_df <- filter(stats_df, contrast == "astrocytoma_normal")
```

To make this plot we will be using functions from the `ggplot2` package, the main plotting package of the tidyverse.
We use the first function, `ggplot()` to define the data that will be plotted.
`ggplot()` takes two main arguments:  

1. `data`, which is the data frame that contains the data we want to plot.  
2. `mapping`, which is a special list made with the `aes()` function to describe which values will be used for each **aes**thetic component of the plot, such as the x and y coordinates of each point. 
(If you find calling things like the x and y coordinates "aesthetics" confusing, don't worry, you are not alone.)  

```{r}
ggplot(
  tumor_normal_df, # This first argument is the data.frame with the data we want to plot
  aes(
    x = log_fold_change, # This is the column name of the values we want to use
    # for the x coordinates
    y = neg_log_p_val
  ) # This is the column name of the data we want for the y-axis
)
```

You'll notice this plot doesn't have anything on it because we haven't 
specified a plot type yet. 
To do that, we will add another ggplot layer with `+` which will specify exactly what we want to plot.
A volcano plot is a special kind of scatter plot, so to make that we will want to plot individual points, which we can do with `geom_point()`.

```{r}
# This first part is the same as before
ggplot(
  tumor_normal_df,
  aes(
    x = log_fold_change,
    y = neg_log_p_val
  )
) +
  # Now we are adding on a layer to specify what kind of plot we want
  geom_point()
```

Here's a brief summary of ggplot2 structure. 
[ggplot2 structure](diagrams/ggplot_structure.png)

### Adjust our ggplot

Now that we have a base plot that shows our data, we can add layers on to it and adjust it.
We can adjust the color points using `color`. 

```{r}
ggplot(
  tumor_normal_df,
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  ) # We added this argument to color code the points!
) +
  geom_point()
```

Because we have so many points overlapping one another, we will want to adjust 
the transparency, which we can do with an `alpha` argument. 

```{r}
ggplot(
  tumor_normal_df,
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  )
) +
  geom_point(alpha = 0.2) # We are using the `alpha` argument to make our points transparent
```

Notice that we added the alpha within the `geom_point()` function, not to the `aes()`. 
We did this because we want all of the points to have the same level of transparency, and it will not vary depending on any variable in the data.
We can also change the background and appearance of the plot as a whole by adding a `theme`.

```{r}
ggplot(
  tumor_normal_df,
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  )
) +
  geom_point(alpha = 0.2) +
  theme_bw() # Add on this set of appearance presets to make it pretty
```

We are not limited to a single plotting layer. 
For example, if we want to add a horizontal line to indicate a significance cutoff, we can do that with `geom_hline()`.
For now, we will choose the value of 5.5 (that is close to a Bonferonni correction) and add that to the plot.

```{r}
ggplot(
  tumor_normal_df,
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  )
) +
  geom_point(alpha = 0.2) + 
  geom_hline(yintercept = 5.5, color = "darkgreen") # we can specify colors by names here
```

We can also change the background and appearance of the plot as a whole by adding a `theme`.
We can change the x and y labels by using `ylab` and `xlab` functions and add a
title using `ggtitle`.

```{r}
ggplot(
  tumor_normal_df,
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  )
) +
  geom_point(alpha = 0.2) +
  geom_hline(yintercept = 5.5, color = "darkgreen") +
  theme_bw() +
  xlab("log2 Fold Change Tumor/Normal") + # Add an x label
  ylab("-log10 p value") # Add a y label
ggtitle("Astrocytoma Tumor vs Normal Cerebellum") # Add main title
```

Use this chunk to make the same kind of plot as the previous chunk but instead plot the male female contrast data, that is stored in `male_female_df`. 

```{r}
# Use this chunk to make the same kind of volcano plot, but with the male-female contrast data.
```

Turns out, we don't have to plot each contrast separately, instead, we can use the original data frame that contains all three contrasts' data, `stats_df`, and add a `facet_wrap` to make each contrast its own plot. 

```{r}
ggplot(
  stats_df, # Switch to the bigger data.frame with all three contrasts' data
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  )
) +
  geom_point(alpha = 0.2) +
  geom_hline(yintercept = 5.5, color = "darkgreen") +
  theme_bw() +
  facet_wrap(~contrast) +
  xlab("log2 Fold Change") + # Now that this includes the other contrasts,
                             # we'll make this label more general
  ylab("-log10 p value")
```

We can store the plot as an object in the global environment by using `<-` operator. 
Here we will call it `volcano_plot`. 

```{r}
volcano_plot <- ggplot(
  stats_df, # We are calling this plot `volcano_plot`
  aes(
    x = log_fold_change,
    y = neg_log_p_val,
    color = avg_expression
  )
) +
  geom_point(alpha = 0.2) +
  geom_hline(yintercept = 5.5, color = "darkgreen") +
  theme_bw() +
  facet_wrap(~contrast) +
  xlab("log2 Fold Change") +
  ylab("-log10 p value")
```

When we are happy with our plot, we can save the plot using `ggsave`. 

```{r}
ggsave(
  plot = volcano_plot,
  filename = file.path(plots_dir, "volcano_plot.png")
)
```

### Session Info

```{r}
# Print out the versions and packages we are using in this session
sessionInfo()
```
