---
title: "Pathway analysis: Over-representation analysis (ORA)"
output:
  html_notebook:
    toc: true
    toc_float: true
author: CCDL for ALSF
date: 2020
---

## Objectives

This notebook will demonstrate how to:

- Perform gene identifier conversion with [`AnnotationDBI` annotation packages](https://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf)
- Access [Molecular Signatures Database gene set collections](https://www.gsea-msigdb.org/gsea/msigdb/collections.jsp) via the `msigdbr` package
- Prepare gene sets for over-representation analysis, including an appropriate background set
- Perform over-representation analysis with the `clusterProfiler` package

---

In this notebook, we'll cover a type of pathway or gene set analysis called over-representation analysis (ORA).
The idea behind ORA is relatively straightforward: given a set of genes, do these genes overlap with a pathway more than we expect by chance?
The simplicity of only requiring an input gene set (sort of, more on that below) can be attractive.

ORA has some limitations, outlined nicely (and more extensively!) in [Khatri _et al._ (2012)]( https://doi.org/10.1371/journal.pcbi.1002375).
One of the main issues with ORA is that typically all genes are treated as equal -- the context of the magnitude of a change we may be measuring is removed and each gene is treated as independent, which can sometimes result in an incorrect estimate of significance.

We will use the [`clusterProfiler` package](https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html) ([Yu *et al.* 2012](https://doi.org/10.1089/omi.2011.0118.)) to perform ORA.
`clusterProfiler` has many built-in functions that will run a specific type of analysis using a specific source of pathways/gene sets automatically, but for our purposes we're going to keep things as general as possible.
See the [`clusterProfiler` book](https://yulab-smu.github.io/clusterProfiler-book/index.html) for more information about the package's full suite of functionality.

Because different bioinformatics tools often require different types of gene identifiers, we'll also cover how to convert between gene identifiers using [`AnnotationDbi`](https://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html) Bioconductor packages in this notebook.
Check out the [_AnnotationDbi: Introduction To Bioconductor Annotation Packages_ (Carlson 2020.) vignette](https://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf) for more information.

#### Other resources

* For another example using `clusterProfiler`, see [_Intro to DGE: Functional Analysis._ from Harvard Chan Bioinformatics Core Training.](https://hbctraining.github.io/DGE_workshop/lessons/09_functional_analysis.html)
* [`WebGestaltR`](https://cran.r-project.org/web/packages/WebGestaltR/) is another R package that can be used for ORA.

## Set up

### Libraries

```{r libraries}
# Package we'll use for ORA
library(clusterProfiler)
# Package that contains MSigDB gene sets in tidy format
library(msigdbr)
# Mus musculus annotation package we'll use for gene identifier conversion
library(org.Mm.eg.db)
```

### Directories and files

#### Directories

```{r create_ora_directory, live = TRUE}
# We'll create a directory to specifically hold the ORA results if it doesn't
# exist yet
results_dir <- file.path("results", "leukemia")
if (!dir.exists(results_dir)) {
  dir.create(results_dir, recursive = TRUE)
}
```

#### Input files

For our ORA example, we're going to use two tables of differential gene expression (DGE) analysis results.

```{r input_directory}
input_dir <- file.path("data", "leukemia")

# This file contains the DGE results for a cell population with high stem cell
# capacity as compared to a cell population
vs_low_file <- file.path(input_dir,
                         "high_capacity_vs_low_capacity_results.tsv")

# This file contains the DGE results for the high stem cell capacity population
# vs. unsorted cells (e.g., a mixture of capacities)
vs_unsorted_file <- file.path(input_dir,
                              "high_capacity_vs_unsorted_results.tsv")
```

#### Output files

We'll save the table of ORA results (e.g., p-values).

```{r output_file, live = TRUE}
kegg_results_file <- file.path(results_dir, "leukemia_kegg_ora_results.tsv")
```

## Gene sets

We will use gene sets from the [Molecular Signatures Database (MSigDB)](https://www.gsea-msigdb.org/gsea/msigdb/index.jsp) from the Broad Institute ([Subramanian, Tamayo *et al.* 2005](https://doi.org/10.1073/pnas.0506580102)).
The [`msigdbr`](https://cran.r-project.org/web/packages/msigdbr/index.html) package contains MSigDB datasets already in the tidy format required by `clusterProfiler` and supports multiple organisms.

Let's take a look at what organisms the package supports.

```{r show_species}
msigdbr_species()
```

The results we're interested in here come from mouse samples, so we can obtain just the gene sets relevant to _M. musculus_ with the `species` argument to `msigdbr()`.

```{r mm_df, live = TRUE}
mm_msigdb_df <- msigdbr(species = "Mus musculus")
```

MSigDB contains 8 different gene set collections.

    H: hallmark gene sets
    C1: positional gene sets
    C2: curated gene sets
    C3: motif gene sets
    C4: computational gene sets
    C5: GO gene sets
    C6: oncogenic signatures
    C7: immunologic signatures

In this example, we will use canonical pathways which are ([ref](https://www.gsea-msigdb.org/gsea/msigdb/collections.jsp)):

> Gene sets from pathway databases. Usually, these gene sets are canonical representations of a biological process compiled by domain experts.

And are a subset of `C2: curated gene sets`.
Specifically, we will use the [KEGG (Kyoto Encyclopedia of Genes and Genomes)](https://www.genome.jp/kegg/) pathways.

```{r filter_to_kegg}
# Filter the mouse data frame to the KEGG pathways that are included in the
# curated gene sets
mm_kegg_df <- mm_msigdb_df |>
  dplyr::filter(gs_cat == "C2",  # curated gene sets
                gs_subcat == "CP:KEGG")  # KEGG pathways
```

*Note: We could specified that we wanted the KEGG gene sets using the `category` and `subcategory` arguments of `msigdbr()`, but we're going for general steps!*

```{r mm_kegg_columns}
colnames(mm_kegg_df)
```

The `clusterProfiler` function we will use requires a data frame with two columns, where one column contains the term identifier or name and one column contains gene identifiers that match our gene lists we want to check for enrichment.
Our data frame with KEGG terms contains Entrez IDs and gene symbols.

## Read in DGE results and prep

```{r read_in_dge_results, live = TRUE}
vs_low_df <- readr::read_tsv(vs_low_file)
```

Let's take a peek at the top of the DGE results data frame.

```{r peek_at_dge_results, live = TRUE}
head(vs_low_df)
```

### Gene identifier conversion

Our data frame of DGE results contains Ensembl gene identifiers.
So we will need to convert from these identifiers into either the gene symbols or Entrez IDs that are present in the data we extracted with `msigdbr()`.

We're going to convert our identifiers to gene symbols because they are a bit more human readable, but you can, with the change of a single argument, use the same code to convert to many other types of identifiers!

The annotation package `org.Mm.eg.db` contains information for different identifiers.
`org.Mm.eg.db` is specific to _Mus musculus_ -- this is what the `Mm` in the package name is referencing.
To perform gene identifier conversion in human (_Homo sapiens_) we could use `org.Hs.eg.db`;
we would use `org.Dr.eg.db` for zebrafish (_Danio rerio_).

We can see what types of IDs are available to us in an annotation package with `keytypes()`.

```{r keytypes, live = TRUE}
keytypes(org.Mm.eg.db)
```

Even though we'll use this package to convert from Ensembl gene IDs (`ENSEMBL`) to gene symbols (`SYMBOL`), we could just as easily use it to convert from an Ensembl transcript ID (`ENSEMBLTRANS`) to Entrez IDs (`ENTREZID`).

The function we will use to map from Ensembl gene IDs to gene symbols is called `mapIds()`.

```{r map_to_symbol}
# This returns a named vector which we can convert to a data frame, where
# the keys (Ensembl IDs) are the names
symbols_vector <- mapIds(org.Mm.eg.db,  # Specify the annotation package
                         # The vector of gene identifiers we want to
                         # map
                         keys = vs_low_df$Gene,
                         # What type of gene identifiers we're starting
                         # with
                         keytype = "ENSEMBL",
                         # The type of gene identifier we want returned
                         column = "SYMBOL",
                         # In the case of 1:many mappings, return the
                         # first one. This is default behavior!
                         multiVals = "first")

# We would like a data frame we can join to the DGE results
symbols_df <- data.frame(
  ensembl_id = names(symbols_vector),
  gene_symbol = symbols_vector
)
```

This message is letting us know that sometimes Ensembl gene identifiers will map to multiple gene symbols.
In this case, it's also possible that a gene symbol will map to multiple Ensembl IDs.

Now we are ready to add the gene symbols to our data frame with the DGE results.
We can use a _join_ function from the `dplyr` package to do this, which will use the Ensembl gene IDs in both data frames to determine how to join the rows.

Let's do this first for the comparison to the low stem cell capacity population.

```{r add_symbols, live = TRUE}
vs_low_df <- symbols_df |>
  # An *inner* join will only return rows that are in both data frames
  dplyr::inner_join(vs_low_df,
                    # The name of the column that contains the Ensembl gene IDs
                    # in the left data frame and right data frame
                    by = c("ensembl_id" = "Gene"))
```

### Drop `NA` values

Some of these rows have `NA` values in `padj`, which [can happen for a number of reasons](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#pvaluesNA) including when all samples have zero counts or a gene has low mean expression.

Let's filter to rows that _do not have any `NA`_ using a function `tidyr::drop_na()`.
This will also drop genes that have an Ensembl gene identifier but no gene symbol!

```{r complete_cases}
# Remove rows that are not complete (e.g., contain NAs) by filtering to only
# complete rows
vs_low_df <- vs_low_df |>
  tidyr::drop_na()
```

**Now we'll read in our data frame of DGE results from another comparison.**
To save us some time during instruction, we've already done the gene identifier conversion and filtering to remove `NA` values in [this notebook](https://github.com/AlexsLemonade/training-modules/tree/master/pathway-analysis/setup/01-leukemia_DGE.Rmd).
We took a different series of steps to achieve the same thing, which is often possible in R!


```{r read_in_unsorted}
vs_unsorted_df <- readr::read_tsv(vs_unsorted_file)
```

## Over-representation Analysis (ORA)

To test for over-representation, we can calculate a p-value with a hypergeometric test ([ref](https://yulab-smu.github.io/clusterProfiler-book/chapter2.html#over-representation-analysis)).

\(p = 1 - \displaystyle\sum_{i = 0}^{k-1}\frac{ {M \choose i}{ {N-M} \choose {n-i} } } { {N \choose n} }\)

Where `N` is the number of genes in the background distribution, `M` is the number of genes in a pathway, `n` is the number of genes we are interested in (our marker genes), and `k` is the number of genes that overlap between the pathway and our marker genes.

Borrowing an example from [_clusterProfiler: universal enrichment tool for functional and comparative study_ (Yu )](http://yulab-smu.top/clusterProfiler-book/chapter2.html#over-representation-analysis):

> **Example**: Suppose we have 17,980 genes detected in a Microarray study and 57 genes were differentially expressed. Among the differential expressed genes, 28 are annotated to a gene set.

We'll call genes that are differentially expressed `gene_in_interest` and genes that are in the gene set `in_gene_set`.

```{r gene_table}
gene_table <- data.frame(
  gene_not_interest = c(2613, 15310),
  gene_in_interest = c(28, 29)
)
rownames(gene_table) <- c("in_gene_set", "not_in_gene_set")

gene_table
```

We can assess if the 28 overlapping genes mean that the differentially expressed genes are over-represented in the gene set with the hypergeometric distribution.
This corresponds to a one-sided Fisher's exact test.

```{r fisher_test}
fisher.test(gene_table, alternative = "greater")
```

When we test **multiple pathways or gene sets**, the p-values then need to be **adjusted** for multiple hypothesis testing.

### High stem cell capacity ORA

Our DGE results are from data published as part of [Sachs _et al._ (2014)](https://dx.doi.org/10.1182%2Fblood-2013-08-521708).
The authors sorted populations of primary leukemia cells and examined the stem cell capacity of these cell populations.
(This study may sound familiar if you've worked on one of our bulk RNA-seq exercise notebooks in the past!)

We compared the population that the authors identified as having high stem cell capacity to a low stem cell capacity population.
We also compared the high stem cell capacity cells to a mix of populations (e.g., unsorted cells).
You can see the code in [here](https://github.com/AlexsLemonade/training-modules/tree/master/pathway-analysis/setup/01-leukemia_DGE.Rmd).

We're interested in what pathways are over-represented in genes that specifically distinguish the high capacity population from the low capacity population.

Let's generate a list of genes that have higher expression in the high stem cell capacity population compared to the low stem cell capacity population, but we'll also want to exclude genes that show up in our _other comparison_ to unsorted cells.

We'll start with the high stem cell capacity vs. low stem cell capacity population comparison.
Genes with positive log2 fold-changes (LFC) will be more highly expressed in the high stem cell capacity cells based on how we set up the analysis.

```{r high_capacity_genes}
vs_low_genes <- vs_low_df |>
  # Filter to the positive LFC and filter based on significance too (padj)
  dplyr::filter(log2FoldChange > 0,
                padj < 0.05) |>
  # Return a vector of gene symbols
  dplyr::pull(gene_symbol)
```

Although we're picking a _commonly used_ cutoff (FDR < 0.05), it's still arbitrary and we could just as easily pick a different threshold for our LFC values.
When we generate lists of genes of interest for ORA, we typically pick an arbitrary cutoff.
This is one of the approach's weaknesses -- we've removed all other context.

Now, we'll take the same steps for our other results.

```{r unsorted_genes_to_remove, live = TRUE}
vs_unsorted_genes <- vs_unsorted_df |>
  dplyr::filter(log2FoldChange > 0,
                padj < 0.05) |>
  dplyr::pull(gene_symbol)
```

We want genes that are in the first comparison but not in the second!
We can use `setdiff()`, a base R function for set operations, to get the list that we want.

```{r setdiff}
# What genes are in the first set but *not* in the second set
genes_for_ora <- setdiff(vs_low_genes, vs_unsorted_genes)
```

#### Background set

As we saw above, calculating the p-value relies on the number of genes in the background distribution.
Sometimes folks consider genes from the entire genome to comprise the background, but in the example borrowed from the `clusterProfiler` authors, they state:

> 17,980 genes detected in a Microarray study

Where the key phrase is **genes detected**.

If we were unable to include a gene in one of our differential expression comparisons because, for example, it had low mean expression in our experiment and therefore was filtered out in our `tidyr::drop_na()` step, we shouldn't include in our background set.

We can use another function for set operations, `intersect()`, to get our background set of genes that were included in both comparisons.

```{r get_background_set}
# intersect() will return the genes in both sets - we are using the entire data
# frame here (complete cases), not just the significant genes
background_set <- intersect(vs_low_df$gene_symbol,
                            vs_unsorted_df$gene_symbol)

# Remove anything that couldn't reliably be measured/assessed in both from the
# genes of interest list - using intersect() will drop anything in the first set
# that isn't also in the second set
genes_for_ora <- intersect(genes_for_ora, background_set)
```

#### Run `enricher()`

Now that we have our background set, our genes of interest, and our pathway information, we're ready to run ORA using the `enricher()` function.

```{r kegg_ora}
kegg_ora_results <- enricher(
  gene = genes_for_ora,  # Genes of interest
  pvalueCutoff = 0.05,
  pAdjustMethod = "BH",  # FDR
  universe = background_set,  # Background set
  # The pathway information should be a data frame with a term name or
  # identifier and the gene identifiers
  TERM2GENE = dplyr::select(mm_kegg_df,
                            gs_name,
                            gene_symbol)
)
```

*Note: using `enrichKEGG()` is a shortcut for doing ORA using KEGG, but the approach we covered here can be used with any gene sets you'd like!*

What is returned by `enricher()`?

```{r view_kegg_ora, eval = FALSE}
View(kegg_ora_results)
```

The information we're most likely interested in is in the `results` slot.
Let's convert this into a data frame that we can write to file.

```{r kegg_df}
kegg_result_df <- data.frame(kegg_ora_results@result)
```

#### Visualizing results

We can use a dot plot to visualize our significant enrichment results.

```{r dotplot, live = TRUE}
enrichplot::dotplot(kegg_ora_results)
```

We can use an [UpSet plot](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4720993/) to visualize the **overlap** between the gene sets that were returned as significant.

```{r upsetplot, live = TRUE}
enrichplot::upsetplot(kegg_ora_results)
```

We can see that some of the DNA repair pathways share genes.
Gene sets or pathways aren't independent, either!
Sometimes multiple pathways that show up in our results as significant are indicative of only a handful of genes in our gene list.

We can look at the `geneID` column of our results to see what genes overlap; it's a good idea to take a look.

```{r look_at_gene_ids, live = TRUE, rownames.print = FALSE}
kegg_result_df |>
  # Use dplyr::select() - the name of the pathway is in the ID column
  dplyr::select(ID, geneID)
```

#### Write results to file

```{r write_results, live = TRUE}
readr::write_tsv(kegg_result_df, file = kegg_results_file)
```

## Session Info

```{r session_info}
sessionInfo()
```
