---
title: "Pathway analysis: Gene Set Enrichment Analysis (GSEA)"
output: 
  html_notebook:
    toc: true
    toc_float: true
author: CCDL for ALSF
date: 2020
---

In this notebook, we will perform Gene Set Enrichment Analysis (GSEA) on the neuroblastoma cell line differential gene expression (DGE) results we generated during the RNA-seq module.

To refresh our memory, we analyzed data from [Harenza *et al.* (2017)](https://doi.org/10.1038/sdata.2017.33) and specifically tested for DGE between with _MYCN_ amplified cell lines and non-amplified cell lines using `DESeq2`.
We have a table of results that contains our log2 fold changes and adjusted p-values.

GSEA is a functional class scoring (FCS) approach to pathway analysis that was first introduced in [Subramanian _et al._ (2005)](https://doi.org/10.1073/pnas.0506580102).
The rationale behind FCS approaches is that small changes in individual genes that participate in the same biological process or pathway can be significant and of biological interest.
FCS methods are better suited for identifying these pathways that show coordinated changes than ORA.
In ORA, we pick a cutoff that _typically_ only captures genes with large individual changes.

There are 3 general steps in FCS methods ([Khatri _et al._ 2012]( https://doi.org/10.1371/journal.pcbi.1002375)):
 
1. Calculate a gene-level statistic (we'll use log2 fold change from DESeq2 here)
2. Gene-level statistics are aggregated into a pathway-level statistic
3. Assess the statistical significance of the pathway-level statistic

#### Other resources

* For another example using `clusterProfiler` for GSEA, see [_Intro to DGE: Functional Analysis._ from Harvard Chan Bioinformatics Core Training.](https://hbctraining.github.io/DGE_workshop/lessons/09_functional_analysis.html)
* The way we'll use `clusterProfiler` here uses `fgsea` (Fast Gene Set Enrichment Analysis) under the hood.
You can read more about fgsea in [Korotkevich _et al._ (2021)](https://doi.org/10.1101/060012).
* [refine.bio examples Gene set enrichment analysis - RNA-seq](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/pathway-analysis_rnaseq_02_gsea.html) from which this material has been adapted.

## Set up

### Libraries

```{r libraries}
# Package to run GSEA
library(clusterProfiler)
# Package that contains the MSigDB gene sets in tidy format
library(msigdbr)
```

### Directories and Files

#### Directories

```{r}
# Where the DGE results are stored
input_dir <- file.path("..", "RNA-seq", "results", "NB-cell")

# We will create a directory to specifically hold our GSEA results if it does 
# not yet exist
output_dir <- file.path("results", "NB-cell")
if (!dir.exists(output_dir)) {
  dir.create(output_dir, recursive = TRUE)
}
```

#### Input files

```{r input_files}
# DGE results
dge_results_file <- file.path(input_dir,
                             "NB-cell_DESeq_amplified_v_nonamplified_results.tsv")
```

#### Output files

```{r output_files}
# GSEA pathway-level scores and statistics
gsea_results_file <- file.path(output_dir,
                               "NB-cell_gsea_results.tsv")
```

## Gene Set Enrichment Analysis

_Adapted from [refine.bio examples](https://github.com/AlexsLemonade/refinebio-examples/blob/33cdeff66d57f9fe8ee4fcb5156aea4ac2dce07f/03-rnaseq/pathway-analysis_rnaseq_02_gsea.Rmd)_

![](diagrams/subramanian_fig1.jpg)

**Figure 1. [Subramanian _et al._ (2005)](https://doi.org/10.1073/pnas.0506580102).**

GSEA calculates a pathway-level metric, called an enrichment score (sometimes abbreviated as ES), by ranking genes by a gene-level statistic.
This score reflects whether or not a gene set or pathway is overrepresented at the top or bottom of the gene rankings ([Subramanian _et al._ 2005](https://doi.org/10.1073/pnas.0506580102); [Yu](http://yulab-smu.top/clusterProfiler-book/chapter2.html#gene-set-enrichment-analysis)) 

Specifically, genes are ranked from most positive to most negative based on their statistic and a running sum is calculated by starting with the most highly ranked genes and increasing the score when a gene is in the pathway and decreasing the score when a gene is not.
In this example, the enrichment score for a pathway is the running sum's maximum deviation from zero.
GSEA also assesses statistical significance of the scores for each pathway through permutation testing. 
As a result, each input pathway will have a p-value associated with it that is then corrected for multiple hypothesis testing ([Subramanian _et al._ 2005](https://doi.org/10.1073/pnas.0506580102); [Yu](http://yulab-smu.top/clusterProfiler-book/chapter2.html#gene-set-enrichment-analysis)).

The implementation of GSEA we use in here examples requires a gene list ordered by some statistic and input gene sets. 
When you use previously computed gene-level statistics with GSEA, it is called GSEA pre-ranked.

### Gene sets

In the previous notebook, we used KEGG pathways for over-representation analysis.
We identified pathways that were significantly over-represented and found that the significant pathways shared genes.
FCS methods analyze each pathway independently and essentially ignore this overlap between gene sets.

MSigDB includes a collection called Hallmark gene sets ([Liberzon *et al.* 2015](https://dx.doi.org/10.1016%2Fj.cels.2015.12.004))
Here's an excerpt of the [collection description](https://www.gsea-msigdb.org/gsea/msigdb/collection_details.jsp#H):

> Hallmark gene sets summarize and represent specific well-defined biological states or processes and display coherent expression. These gene sets were generated by a computational methodology based on identifying gene set overlaps and retaining genes that display coordinate expression. The hallmarks reduce noise and redundancy and provide a better delineated biological space for GSEA.

We'll use the Hallmark collection for GSEA.
Notably, there are only 50 gene sets included in this collection.
The fewer gene sets we test, the lower our multiple hypothesis testing burden.

We can retrieve only the Hallmark gene sets by specifying `category = "H"` to the `msigdbr()` function.

```{r hallmark_sets, live = TRUE}
hs_hallmark_df <- msigdbr(species = "Homo sapiens",
                          category = "H")
```

## Differential gene expression results

```{r read_in_dge, live = TRUE}
# Read in the DGE results
dge_results_df <- readr::read_tsv(dge_results_file)
```

```{r}
head(dge_results_df)
```

Since this data frame of DGE results includes gene symbols, we do not need to perform any kind of gene identifier conversion.
We do, however, need to check for duplicate gene symbols. 
We can accomplish this with `duplicated()`, which returns a logical vector (e.g., `TRUE` or `FALSE`).
The function `sum()` will count `TRUE` values as 1s and `FALSE` as 0s, so using it with `duplicated()` will count the number of duplicate values.

```{r any_duplicated, live = TRUE}
sum(duplicated(dge_results_df$gene_symbol))
```

This will cause a problem when we go to run GSEA.

### Removing duplicates

The GSEA approach requires on discriminating between genes that are in a gene set and those that are not. 
Practically speaking, gene sets are just collections of gene identifiers!
When the function we use for GSEA pre-ranked gets a list with duplicated gene identifiers, it can produce unexpected results.

Compared to the total number of genes that are in our results, there are not a lot of duplicates but we'll still need to make a decision about how to handle them.

Let's get a vector of the duplicated gene symbols so we can use it to explore our filtering steps.

```{r gene_dups, live = TRUE}
duplicated_gene_symbols <- dge_results_df %>%
  dplyr::filter(duplicated(gene_symbol)) %>%
  dplyr::pull(gene_symbol)
```

Now we'll look at the values for the the duplicated gene symbols.

```{r show_gene_dups}
dge_results_df %>%
  dplyr::filter(gene_symbol %in% duplicated_gene_symbols) %>%
  dplyr::arrange(gene_symbol)
```

We can see that the associated values vary for each row.

Let's keep the gene symbols associated with the higher absolute value of the log2 fold change.
GSEA relies on genes' rankings on the basis of a gene-level statistic and the enrichment score that is calculated reflects the degree to which genes in a gene set are overrepresented in the top or bottom of the rankings ([Subramanian _et al._ 2005](https://doi.org/10.1073/pnas.0506580102); [Yu](http://yulab-smu.top/clusterProfiler-book/chapter2.html#gene-set-enrichment-analysis)).

Retaining the instance of the gene symbols with the higher absolute value of a gene-level statistic means that we will retain the value that is likely to be more highly- or lowly-ranked or, put another way, the values less likely to be towards the middle of the ranked gene list.
We should keep this decision in mind when interpreting our results.
For example, if all the duplicate identifiers happened to be in a particular gene set, we may get an overly optimistic view of how perturbed that gene set is because we preferentially selected instances of the identifier that have a higher absolute value of the statistic used for ranking.

In the next chunk, we are going to filter out the duplicated rows using the `dplyr::distinct()` function
This will keep the first row with the duplicated value thus keeping the row with the highest absolute value of the log2 fold change.

```{r filter_dge}
filtered_dge_df <- dge_results_df %>%
  # Sort so that the highest absolute values of the log2 fold change are at the
  # top
  dplyr::arrange(dplyr::desc(abs(log2FoldChange))) %>%
  # Filter out the duplicated rows using `dplyr::distinct()`
  dplyr::distinct(gene_symbol, .keep_all = TRUE)
```

Let's see what happened to our duplicate identifiers.

```{r show_filtered_dge, live = TRUE}
# Subset to & arrange by gene symbols that were duplicated in the original 
# data frame of results
filtered_dge_df %>%
  dplyr::filter(gene_symbol %in% duplicated_gene_symbols) %>%
  dplyr::arrange(gene_symbol)
```

Now we're ready to prep our pre-ranked list for GSEA.

### Pre-ranked list

The `GSEA()` function takes a pre-ranked (sorted) named vector of statistics, where the names in the vector are gene identifiers.
This is step 1 -- gene-level statistics.

```{r lfc_vector}
lfc_vector <- filtered_dge_df %>%
  dplyr::pull(log2FoldChange, name = gene_symbol)
lfc_vector <- sort(lfc_vector, decreasing = TRUE)
```

Let's look at the top ranked values.

```{r head_lfc, live = TRUE}
# Look at first entries of the log2 fold change vector
head(lfc_vector)
```

And the bottom of the list.

```{r tail_lfc, live = TRUE}
# Look at the last entries of the log2 fold change vector
tail(lfc_vector)
```

## Run GSEA

Now for the analysis!

We can use the `GSEA()` function to perform GSEA with any generic set of gene sets, but there are several functions for using specific, commonly used gene sets (e.g., `gseKEGG()`).

```{r run_gsea}
gsea_results <- GSEA(geneList = lfc_vector,  # ordered ranked gene list
                     minGSSize = 25,  # minimum gene set size
                     maxGSSize = 500,  # maximum gene set set
                     pvalueCutoff = 0.05,
                     pAdjustMethod = "BH",  # correction for multiple hypothesis testing
                     TERM2GENE = dplyr::select(hs_hallmark_df,
                                               gs_name,
                                               gene_symbol))
```

The warning about ties means that there are multiple genes that have the same log2 fold change value. 
This percentage is small and unlikely to impact our results.
A large number of ties might tell us there's something wrong with our DGE results ([Ballereau _et al._ 2018](https://bioinformatics-core-shared-training.github.io/cruk-summer-school-2018/RNASeq2018/html/06_Gene_set_testing.nb.html)).

Let's take a look at the GSEA results.

```{r view_gsea, live = TRUE, eval = FALSE}
View(gsea_results@result %>%
       dplyr::arrange(dplyr::desc(NES))
)
```

Normalized enrichment scores (NES) are enrichment scores that are scaled to make gene sets that contain different number of genes comparable.

Let's write these results to file.

```{r write_gsea}
gsea_results@result %>% readr::write_tsv(gsea_results_file)
```

### Visualizing GSEA results

We can visualize GSEA results for individual pathways or gene sets using `enrichplot::gseaplot()`.
Let's take a look at 3 different pathways -- one with a highly positive NES, one with a highly negative NES, and one that was not a significant result -- to get more insight into how ES are calculated.

#### Highly Positive NES

The gene set `HALLMARK_MYC_TARGETS_V1` had high positive log2 fold changes.
Recall a positive log2 fold change means a it had a higher expression value in _MYCN_ amplified cell lines.

```{r myc_v1}
enrichplot::gseaplot(gsea_results,
                     geneSetID = "HALLMARK_MYC_TARGETS_V1",
                     title = "HALLMARK_MYC_TARGETS_V1",
                     color.line = "#0066FF")
```

Notice how the genes that are in the gene set, indicated by the black bars, tend to be on the left side of the graph indicating that they have positive gene-level scores.

#### Highly Negative NES

The gene set `HALLMARK_INTERFERON_ALPHA_RESPONSE` had a highly negative NES.

```{r inflammatory}
enrichplot::gseaplot(gsea_results,
                     geneSetID = "HALLMARK_INTERFERON_ALPHA_RESPONSE",
                     title = "HALLMARK_INTERFERON_ALPHA_RESPONSE",
                     color.line = "#0066FF")
```

This gene set shows the opposite pattern -- genes in the pathway tend to be on the right side of the graph.

#### A non-significant result

The `@results` slot will only show gene sets that pass the `pvalueCutoff` threshold we supplied to `GSEA()`, but we can plot any gene set so long as we know its name.
Let's look at `HALLMARK_P53_PATHWAY`, which was not in the results we viewed earlier.

```{r p53, live = TRUE}
enrichplot::gseaplot(gsea_results,
                     geneSetID = "HALLMARK_P53_PATHWAY",
                     title = "HALLMARK_P53_PATHWAY",
                     color.line = "#0066FF")
```

Genes in the pathway are distributed more evenly throughout the ranked list, resulting in a more "middling" score.

*Note: The plots returned by `enrichplot::gseaplot` are ggplots, so we could use `ggplot2::ggsave()` to save them to file if we wanted to.*

## Session Info

```{r session_info}
sessionInfo()
```
