---
title: "Pathway analysis: Gene Set Enrichment Analysis (GSEA)"
output: 
  html_notebook:
    toc: true
    toc_float: true
author: CCDL for ALSF
date: 2020
---

In this notebook, we will perform Gene Set Enrichment Analysis (GSEA) on the neuroblastoma cell line differential gene expression (DGE) results we generated during the RNA-seq module.

To refresh our memory, we analyzed data from [Harenza *et al.* (2017)](https://doi.org/10.1038/sdata.2017.33) and specifically tested for DGE between with _MYCN_ amplified cell lines and non-amplified cell lines using `DESeq2`.
We have a table of results that contains our log2 fold changes and adjusted p-values.

*Note: We've taken an additional step to get more accurate log2 fold change estimates. 
You can see the code for that in `setup/01-prepare_NB_cell_line.Rmd`.*

GSEA is a functional class scoring (FCS) approach to pathway analysis that was first introduced in [Subramanian, Tamayo *et al.* (2005)](https://doi.org/10.1073/pnas.0506580102).
The rationale behind FCS approaches is that small changes in individual genes that participate in the same biological process or pathway can be significant and of biological interest.
FCS methods are better suited for identifying these pathways that show coordinated changes than ORA.
In ORA, we pick a cutoff that _typically_ only captures genes with large individual changes.

There are 3 general steps in FCS methods ([Khatri, Sirota, and Butte. (2012)]( https://doi.org/10.1371/journal.pcbi.1002375)):
 
1. Calculate a gene-level statistic (we'll use log2 fold change from DESeq2 here)
2. Gene-level statistics are aggregated into a pathway-level statistic
3. Assess the statistical significance of the pathway-level statistic

#### Other resources

* For another example using `clusterProfiler` for GSEA, see [_Intro to DGE: Functional Analysis._ from Harvard Chan Bioinformatics Core Training.](https://hbctraining.github.io/DGE_workshop/lessons/09_functional_analysis.html)
* The way we'll use `clusterProfiler` here uses `fgsea` (Fast Gene Set Enrichment Analysis) under the hood.
You can read more about fgsea in [Korotkevich, Sukhov, Sergushichev. (2019)](https://doi.org/10.1101/060012).

## Set up

### Libraries

```{r libraries}
# Package to run GSEA
library(clusterProfiler)
# Package that contains the MSigDB gene sets in tidy format
library(msigdbr)
# Annotation package that we will use for human gene identifier conversion
library(org.Hs.eg.db)
```

### Directories and Files

#### Directories

```{r}
# Where the DGE results are stored
input_dir <- file.path("results", "gene-metrics")

# We will create a directory to specifically hold our GSEA results if it does 
# not yet exist
output_dir <- file.path("results", "gsea")
if (!dir.exists(output_dir)) {
  dir.create(output_dir, recursive = TRUE)
}
```

#### Input files

```{r input_files}
# DGE results
dge_results_file <- file.path(input_dir,
                             "nb_cell_line_mycn_amplified_v_nonamplified.tsv")
```

#### Output files

```{r output_files}
# GSEA pathway-level scores and statistics
gsea_results_file <- file.path(output_dir,
                               "nb_cell_line_gsea_results.tsv")
```

## Gene sets

In the previous notebook, we used KEGG pathways for over-representation analysis.
We identified pathways that were significantly over-represented and found that the significant pathways shared genes.
FCS methods analyze each pathway independently and essentially ignore this overlap between gene sets.

MSigDB includes a collection called Hallmark gene sets ([Liberzon *et al.* (2015).](https://dx.doi.org/10.1016%2Fj.cels.2015.12.004))
Here's an excerpt of the [collection description](https://www.gsea-msigdb.org/gsea/msigdb/collection_details.jsp#H):

> Hallmark gene sets summarize and represent specific well-defined biological states or processes and display coherent expression. These gene sets were generated by a computational methodology based on identifying gene set overlaps and retaining genes that display coordinate expression. The hallmarks reduce noise and redundancy and provide a better delineated biological space for GSEA.

We'll use the Hallmark collection for GSEA.
Notably, there are only 50 gene sets included in this collection.
The fewer gene sets we test, the lower our multiple hypothesis testing burden.

We can retreive only the Hallmark gene sets by specifying `category = "H"` to the `msigdbr()` function.

```{r hallmark_sets, live = TRUE}
hs_hallmark_df <- msigdbr(species = "Homo sapiens",
                          category = "H")
```

## Differential gene expression results

```{r read_in_dge, live = TRUE}
# Read in the DGE results
dge_results_df <- readr::read_tsv(dge_results_file)
```

```{r}
head(dge_results_df)
```

We can take the same steps we did earlier to convert these Ensembl gene IDs to Entrez IDs. 
We have to change the `column` argument to `mapIds()` (and the downstream steps).

When `mapIds()` is run with `multiValues = "first"` and it encounters multiple matches, only the first identifier is returned.
This is the default behavior.
It will also return a named vector of the IDs you queried for (`column`), where the names are the input identifiers (`keys`), in the same order as the keys which means you can use it directly with `dplyr::mutate()`.

```{r convert_entrez, live = TRUE}
dge_results_df <- dge_results_df %>%
  # Create a new column 'entrez_id' that contains the Entrez IDs returned by
  # mapIds()
  dplyr::mutate(entrez_id = mapIds(org.Hs.eg.db, 
                                   keys = dge_results_df$Gene, 
                                   column = "ENTREZID", 
                                   keytype = "ENSEMBL", 
                                   multiVals = "first")
  ) %>%
  # We need these gene identifiers to perform our pathway analysis, so remove
  # any genes where we don't have Entrez IDs
  dplyr::filter(!is.na(entrez_id))
```

### Preranked list

The `GSEA()` function takes a preranked (sorted) named vector of statistics, where the names in the vector are gene identifiers.
This is step 1 -- gene-level statistics.

```{r lfc_vector}
lfc_vector <- dge_results_df$log2FoldChange
names(lfc_vector) <- dge_results_df$entrez_id
lfc_vector <- sort(lfc_vector, decreasing = TRUE)
```

```{r head_lfc, live = TRUE}
# Look at first entries of the log2 fold change vector
head(lfc_vector)
```

```{r tail_lfc, live = TRUE}
# Look at the last entries of the log2 fold change vector
tail(lfc_vector)
```

## GSEA

![](diagrams/subramanian_fig1.jpg)

**Figure 1. [Subramanian, Tamayo *et al.* (2005)](https://doi.org/10.1073/pnas.0506580102).**

The enrichment score (ES) for a pathway, a pathway-level statistic, is calculated using our gene-level statistics.
Genes are ranked from most highly positive to most highly negative and weighting them according to their gene-level statistic.
A running score is calculated by starting with the most highly ranked genes and increasing the score when a gene is in the pathway and decreasing the score when a gene is not in the pathway.
The ES is the maximum deviation from zero.
Significance is assessed by generating a null distribution by sampling random gene sets of the same size and an FDR (false discovery rate) value is calculated to account for multiple hypothesis testing.
([Subramanian, Tamayo *et al.* (2005)](https://doi.org/10.1073/pnas.0506580102); [Korotkevich, Sukhov, Sergushichev. (2019)](https://doi.org/10.1101/060012)).

We can use the `GSEA()` function to perform GSEA with any generic set of gene sets, but there are several functions for using specific, commonly used gene sets (e.g., `gseKEGG()`).

```{r run_gsea}
gsea_results <- GSEA(geneList = lfc_vector,  # ordered ranked gene list
                     nPerm = 1000,  # number of permutations
                     minGSSize = 25,  # minimum gene set size
                     maxGSSize = 500,  # maximum gene set set
                     pvalueCutoff = 0.05,
                     pAdjustMethod = "BH",  # Benjamini-Hochberg correction
                     TERM2GENE = dplyr::select(hs_hallmark_df,
                                               gs_name,
                                               entrez_gene))
```

Let's take a look at the results.

```{r view_gsea, live = TRUE, eval = FALSE}
View(gsea_results@result %>%
       dplyr::arrange(dplyr::desc(NES))
)
```

Normalized enrichment scores (NES) are enrichment scores that are scaled to make gene sets that contain different number of genes comparable.

### Visualizing GSEA results

We can visualize GSEA results for individual pathways or gene sets using `enrichplot::gseaplot()`.
Let's take a look at 3 different pathways -- one with a highly positive NES, one with a highly negative NES, and one somewhere in the middle -- to get more insight into how ES are calculated.

#### Highly Positive NES

The gene set `HALLMARK_MYC_TARGETS_V2` had high positive log2 fold changes.
Recall a positive log2 fold change means a it had a higher expression value in _MYCN_ amplified cell lines.

```{r myc_v2}
enrichplot::gseaplot(gsea_results,
                     geneSetID = "HALLMARK_MYC_TARGETS_V2",
                     title = "HALLMARK_MYC_TARGETS_V2")
```

Notice how the genes that are in the gene set, indicated by the black bars, tend to be on the left side of the graph indicating that they have positive gene-level scores.

#### Highly Negative NES

The gene set `HALLMARK_INFLAMMATORY_RESPONSE` had a highly negative NES.

```{r inflammatory}
enrichplot::gseaplot(gsea_results,
                     geneSetID = "HALLMARK_INFLAMMATORY_RESPONSE",
                     title = "HALLMARK_INFLAMMATORY_RESPONSE")
```

This gene set shows the opposite pattern -- genes in the pathway tend to be on the right side of the graph.

#### Somewhere in the middle

A moderately negative NES is somewhere in the middle in this particular experiment.
Let's look at `HALLMARK_P53_PATHWAY`.

```{r p53}
enrichplot::gseaplot(gsea_results,
                     geneSetID = "HALLMARK_P53_PATHWAY",
                     title = "HALLMARK_P53_PATHWAY")
```

Genes in the pathway are distributed more evenly throughout the ranked list, resulting in a more "middling" score.

*Note: The plots returned by `enrichplot::gseaplot` are ggplots, so we can use `ggplot2::ggsave()` to save them to file.*

## Session Info

```{r session_info}
sessionInfo()
```
