---
title: "Pathway analysis: Gene Set Variation Analysis (GSVA)"
output: 
  html_notebook:
    toc: true
    toc_float: true
author: CCDL for ALSF
date: 2020
---

So far every pathway analysis method we've covered relies on some information about groups in our data.
For over-representation analysis (ORA), we took the top 100 genes that distinguished neurons from other cell types in a scRNA-seq experiment -- it relied on cell type labels.
In the Gene Set Enrichment Analysis (GSEA) example, we used statistics from a differential gene expression (DGE) analysis where we compared _MYCN_ amplified cell lines to non-amplified cell lines; we needed that amplification status information.

What if we're less sure about groups in our data or we want to analyze our data in a more unsupervised manner?

In this notebook we will cover a method called Gene Set Variation Analysis (GSVA) ([Hänzelmann, Castelo, and Guinney. (2013)](https://doi.org/10.1186/1471-2105-14-7)) that allows us to calculate gene set or pathway scores on a per-sample basis.

We'll use the 

#### Other resources

* [Malhotra. _Decoding Gene Set Variation Analysis_. 2018.](https://towardsdatascience.com/decoding-gene-set-variation-analysis-8193a0cfda3)

## Set up

### Libraries

```{r}
# Pipes
library(magrittr)
# Gene Set Variation Analysis
library(GSVA)
```

### Directories and files

#### Directories

```{r}
# We have some medulloblastoma data from the OpenPBTA project that we've 
# prepared ahead of time
input_dir <- file.path("data", "medulloblastoma")

# Create a directory specifically for our GSVA results if it does not yet exist
output_dir <- file.path("results", "gsva") 
if (!dir.exists(output_dir)) {
  dir.create(output_dir, recursive = TRUE)
}
```

#### Input

We have VST transformed RNA-seq data, annotated with gene symbols, that has been collapsed such that there are no duplicated gene identifiers (see `setup`).

```{r}
rnaseq_file <- file.path(input_dir, "medulloblastoma_vst_collapsed.tsv")
```

#### Output

```{r}
gsva_results_file <- file.path(output_dir, "medulloblastoma_gsva_results.tsv")
```

## Gene sets

The function that we will use to run GSVA wants the gene sets to be in a list, rather than a tidy data frame that we used with `clusterProfiler` (although it does accept other formats).

We're going to take this opportunity to introduce a different format that gene sets are often distributed in called [GMT (Gene Matrix Transposed)](https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats#GMT:_Gene_Matrix_Transposed_file_format_.28.2A.gmt.29).

We're going to read in the Hallmark collection file from [MSigDB](https://www.gsea-msigdb.org/gsea/msigdb/index.jsp), rather than using `msigdbr` like we did in earlier notebooks.

The RNA-seq data uses gene symbols, so we need gene sets that use gene symbols, too.

```{r}
# R can often read in data from a URL
hallmarks_url <- "https://data.broadinstitute.org/gsea-msigdb/msigdb/release/7.1/h.all.v7.1.symbols.gmt"

# QuSAGE is another pathway analysis method, the qusage package has a function
# for reading GMT files and turning them into a list
hallmarks_list <- qusage::read.gmt(hallmarks_url)
```

What does this list look like?

```{r}
head(hallmarks_list)
```

## RNA-seq data

We have VST transformed RNA-seq data, which is on a log2-like scale.
These data are from the Open Pediatric Brain Tumor Atlas (OpenPBTA)
OpenPBTA is a collaborative project organized by the CCDL and the Center for Data-Driven Discovery in Biomedicine (D3b) at the Children's Hospital of Philadelphia conducted openly on GitHub.

You can read more about the project [here](https://github.com/alexslemonade/openpbta-analysis/#openpbta-analysis).

We're only working with the medulloblastoma samples in this example.

```{r}
rnaseq_df <- readr::read_tsv(rnaseq_file)
```

```{r}
rnaseq_df[1:5, 1:5]
```

For GSVA, we need a matrix.

```{r}
rnaseq_mat <- rnaseq_df %>%
  tibble::column_to_rownames("gene_symbol") %>%
  as.matrix()
```

*Note: If we had duplicate gene symbols here, we couldn't set them as rownames.*

## GSVA

![](diagrams/hanzelmann_fig1.jpg)

**Figure 1 from [Hänzelmann, Castelo, and Guinney. (2013)](https://doi.org/10.1186/1471-2105-14-7).**

You may notice that GSVA has some commonalities with GSEA.
Rather than ranking genes based on some statistic, GSVA ranks genes based on their expression level relative to the sample distribution and calculates pathway-level scores.
The inituition here is that we will get a pathway-level score that indicates if 
genes in a pathway that act concordantly in one direction (overexpressed or underexpressed relative to the overall population) ([Hänzelmann, Castelo, and Guinney. (2013)](https://doi.org/10.1186/1471-2105-14-7)).

The output is a gene set by sample matrix of GSVA scores.

### Perform GSVA

```{r}
gsva_results <- gsva(rnaseq_mat, hallmarks_list,
                     method = "gsva",
                     # Appropriate for our transformed data on log2-like scale
                     kcdf = "Gaussian",  
                     # Minimum gene set size
                     min.sz = 15, 
                     # Maximum gene set size
                     max.sz = 500,
                     # Compute Gaussian-distributed scores
                     mx.diff = TRUE,
                     # Use 4 cores
                     parallel.sz = 4)
```

**Note: the `gsva()` documentation says we can use `kcdf = "Gaussian"` if we had RNA-seq log-CPMs, log-RPKMs or log-TPMs, but we would use `kcdf = "Poisson"` on integer counts.**

```{r}
gsva_results[1:5, 1:5]
```

### How can you use these scores?

If you did have groups information for your samples, you could test for pathway scores differences between groups.
Here's [an example](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/9b44bf1c186b3126b16dbe5b87756b3eae3feec2/analyses/gene-set-enrichment-analysis/02-model-gsea.nb.html) of that from the OpenPBTA project itself! 

You can also visualize this matrix in a heatmap.
Here's a figure from the OpenPBTA project, where the middle panel is a heatmap of GSVA scores that were significantly different between histologies.

![](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/9b44bf1c186b3126b16dbe5b87756b3eae3feec2/figures/pngs/transcriptomic-overview.png)

[*Code here.*](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/9b44bf1c186b3126b16dbe5b87756b3eae3feec2/figures/scripts/transcriptomic-overview.R#L106)

### Write results to file

```{r}
gsva_results %>%
  as.data.frame() %>%
  tibble::rownames_to_column("pathway") %>%
  readr::write_tsv(path = gsva_results_file)
```

## Session Info

```{r}
sessionInfo()
```

