---
title: "Gastric cancer: gene-level summarization with `tximport`"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2021**

In this notebook, we'll import transcript level output from `salmon quant` and
summarize it to the gene level using 'transcript expression import' - [`tximport`](https://bioconductor.org/packages/release/bioc/html/tximport.html) and 'tximport meta' - [`tximeta`](https://bioconductor.org/packages/release/bioc/html/tximeta.html). 


![](diagrams/rna-seq_5.png)

For more information about `tximeta`, see [this excellent vignette](https://www.bioconductor.org/packages/devel/bioc/vignettes/tximeta/inst/doc/tximeta.html) from Love, Soneson, Hickey, and Patro.

We'll need the `quant.sf` files for all the samples in an experiment and a file
that maps between Ensembl transcript ids and Ensembl gene ids.
In refine.bio, we generate these along side the transcriptome indices for an
organism.

## Libraries and functions

```{r library, live = TRUE}
# Load the tximeta package
library(tximeta)

# Load the SummarizedExperiment package
library(SummarizedExperiment)
```

## Directories and files

```{r directories, live = TRUE}
# directory where the quant files are located, each sample is its own
# directory
quant_dir <- file.path("data", "gastric-cancer", "salmon_quant")

# create a directory to hold the tximport results if it doesn't exist yet
txi_dir <- file.path("data", "gastric-cancer", "tximport")
if (!dir.exists(txi_dir)) {
  dir.create(txi_dir, recursive = TRUE)
}
```

```{r input-names}
# the quant files themselves
sf_files <- list.files(quant_dir, recursive = TRUE, full.names = TRUE,
                       pattern = "quant.sf")
```

**Output**

```{r output-names, live = TRUE}
# Name the output gastric-cancer_tximport.RDS and use the directory created 
# above as the rest of the path
txi_out_file <- file.path(txi_dir, "gastric-cancer_tximport.RDS")
```

### Naming

All output files from `salmon quant` we'll use with `tximport` are named 
`quant.sf`.
The file names do not have any information about the sample name.

```{r sf_files, live = TRUE}
# Let's look at the full path for the quant.sf files
sf_files
```

Let's extract the _sample_ names from the **file paths** using the `stringr` 
package.

Notice how the file path is separated by `/`.
If we were to split up this character by `/`, the fourth item is the sample 
name.
This is exactly what `stringr::word` allows us to do: split up the file paths
by `/` and extract the sample names.

```{r sample_names}
sample_names <- stringr::word(sf_files, 4, sep = "/")
sample_names
```

`tximeta` needs a data.frame with these columns: 
- a `files` column  with the file paths to the quant.sf files
- a `names` column with the sample names

```{r names_sf_files}
coldata <- data.frame(files = sf_files, 
                      names = sample_names, 
                      stringsAsFactors = FALSE)
```

## Set up our tximeta object

Using our `coldata` data frame we set up, we can run the `tximeta()` which will 
extract species specific information.

```{r read_tsv, live = TRUE}
txi_meta <- tximeta(coldata)
```

*There are extra steps needed if you are using any species other than human and mouse.
See our other species examples in `tximeta_for_other_species.Rmd`. 

### Summarize to gene

We'll summarize to the gene level using the `summarizeToGene()` function.

```{r make-txi}
# Summarize to the gene-level
gene_summarized <- summarizeToGene(txi_meta) 
```

We can use the `class` function to see what type of object `gene_summarized` is.

```{r class-txi, live = TRUE}
# Check what type of object gene_summarized is
class(gene_summarized)
```

This tells us that `gene_summarized` is an object called a [`SummarizedExperiment`](https://www.rdocumentation.org/packages/SummarizedExperiment/versions/1.2.3/topics/SummarizedExperiment-class) which can be handled by functions from the package of the same name. 
We more specifically have a `RangedSummarizedExperiment` which is a more specific type of `SummarizedExperiment`.

SummarizedExperiment objects have this general structure:  

![SummarizedExperiment](diagrams/SummarizeExperiment-structure.png)

This figure is from this handy vignette about [`SummarizedExperiment` objects](https://www.bioconductor.org/packages/devel/bioc/vignettes/SummarizedExperiment/inst/doc/SummarizedExperiment.html).

The `assay` slot in `SummarizedExperiment`s hold the experiment data, a gene x sample matrix. 
In this case, it has our gene-level abundance information. 

Multiple `assays` can be stored in an `SummarizedExperiment` and we can use the `assayNames()` function to see what assays are included in `gene_summarized`. 

```{r assay-names, live = TRUE}
assayNames(gene_summarized)
```

If we want to extract an `assay`'s data, we can use `assay()` function and 
specify the name of the assay we want to extract.

```{r assay, live = TRUE}
counts_mat <- assay(gene_summarized, "counts")
```

We can use the `class` function to see what type of object the `tximport`
function returns.

```{r counts, live = TRUE}
# Check what type of object counts_mat is
class(counts_mat)
```

### Save to file

We could use `readr::write_tsv` to save `counts_mat` only but `gene_summarized` 
has a lot of information stored here beyond the counts, so we may want to save 
all of this to a RDS object. 

```{r write-txi, live = TRUE}
# Write gene_summarized to RDS object
readr::write_rds(gene_summarized, file = txi_out_file)
```

We'll import this with the `DESeq2` package in the next notebook.

## Session Info

Record session info for reproducibility & provenance purposes.

```{r sessioninfo}
sessionInfo()
```
