---
title: "Gastric cancer: gene-level summarization with `tximport`"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2018**

In this notebook, we'll import transcript level output from `salmon quant` and
summarize it to the gene level using [`tximport`](https://bioconductor.org/packages/release/bioc/html/tximport.html).

![](diagrams/rna-seq_5.png)

For more information about `tximport`, see [this excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/tximport/inst/doc/tximport.html) from Love, Soneson, and Robinson.

We'll need the `quant.sf` files for all the samples in an experiment and a file
that maps between Ensembl transcript ids and Ensembl gene ids.
In refine.bio, we generate these along side the transcriptome indices for an
organism.

## Libraries and functions

```{r library, live = TRUE}
# Load the tximport package
library(tximport)
```

## Directories and files

```{r directories, live = TRUE}
# directory where the quant files are located, each sample is its own
# directory
quant_dir <- file.path("data", "gastric-cancer", "salmon_quant")

# create a directory to hold the tximport results if it doesn't exist yet
txi_dir <- file.path("data", "gastric-cancer", "tximport")
if (!dir.exists(txi_dir)) {
  dir.create(txi_dir, recursive = TRUE)
}
```

```{r input-names}
# the quant files themselves
sf_files <- list.files(quant_dir, recursive = TRUE, full.names = TRUE,
                       pattern = "quant.sf")
```

**Output**

```{r output-names, live = TRUE}
# Name the output gastric-cancer_tximport.RDS and use the directory created 
# above as the rest of the path
txi_out_file <- file.path(txi_dir, "gastric-cancer_tximport.RDS")
```

### Naming

All output files from `salmon quant` we'll use with `tximport` are named 
`quant.sf`.
The file names do not have any information about the sample name.

```{r sf_files, live = TRUE}
# Let's look at the full path for the quant.sf files
sf_files
```

Let's extract the _sample_ names from the **file paths** using the `stringr` 
package.

Notice how the file path is separated by `/`.
If we were to split up this character by `/`, the fourth item is the sample 
name.
This is exactly what `stringr::word` allows us to do: split up the file paths
by `/` and extract the sample names.

```{r sample_names}
sample_names <- stringr::word(sf_files, 4, sep = "/")
sample_names
```

We'll add these names to the file paths vector, that way they are reflected in
what we do next with `tximport`.

```{r names_sf_files}
coldata <- data.frame(files = sf_files, 
                      names = sample_names, 
                      stringsAsFactors = FALSE)
```

## tximport

```{r read_tsv, live = TRUE}
# BiocManager::install("tximeta")
txi_meta <- tximeta::tximeta(coldata)
```

### Summarize to gene

We'll summarize to the gene level.
Notice that `tx2gene_df` uses identifiers that don't include [Ensembl version information](https://useast.ensembl.org/Help/Faq?id=488).
Version numbers follow a period (`.`) at the end of the identifier. 
For example, the second version of a human Ensembl transcript ID follows this pattern: `ENSTXXXXXXXXXX.2`.

We need to use the `ignoreTxVersion = TRUE` argument to `tximport` because we don't care about transcript versions.

```{r make-txi}
gene_summarized <- summarizeToGene(txi_meta, 
                                   countsFromAbundance = "no", 
                                   ignoreTxVersion = TRUE) 
```

We can use the `class` function to see what type of object `gene_summarized` is.

```{r class-txi, live = TRUE}
# Check what type of object gene_summarized is
class(gene_summarized)
```

This tells us that `gene_summarized` is a special type of object called a `SummarizedExperiment` which can be handled by the package of the same name. 

Let's take a look at what data we have from this. 

```{r assay-names, live = TRUE}
SummarizedExperiment::assayNames(gene_summarized)
```

![tximport assay](diagrams/tximport.png)

If we want to extract the data itself, we can use `SummarizedExperiment::assay()`.

```{r assay, live = TRUE}
counts_mat <- SummarizedExperiment::assay(gene_summarized, "counts")
```

We can use the `class` function to see what type of object the `tximport`
function returns.

```{r counts, live = TRUE}
# Check what type of object txi is
class(counts_mat)
```

### Save to file

We'll import this with the `DESeq2` package.

```{r write-txi, live = TRUE}
# Write txi to RDS object
readr::write_rds(txi_meta, file = txi_out_file)
```

## Session Info

Record session info for reproducibility & provenance purposes.

```{r sessioninfo}
sessionInfo()
```
