---
title: "Gastric cancer: exploratory analysis"
author: CCDL for ALSF
date: 2021
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

In this notebook, we'll import the gastric cancer data and do some exploratory
analyses and visual inspection.
We'll use the [`DESeq2`](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) package for this.

![](diagrams/rna-seq_6.png)

`DESeq2` also has an [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) 
from Love, Anders, and Huber from which this is adapted (see also: [Love, Anders, and Huber. _Genome Biology_. 2014.](https://doi.org/10.1186/s13059-014-0550-8)).

## Libraries and functions

```{r library, live = TRUE}
# Load the DESeq2 library
library(DESeq2)
```

```{r magrittr}
# magrittr for the pipe
library(magrittr)
```

## Directories and files

```{r input-files}
# Main data directory
data_dir <- file.path("data", "gastric-cancer")

# directory with the tximeta processed data
txi_dir <- file.path(data_dir, "txi")
txi_file <- file.path(txi_dir, "gastric-cancer_tximeta.RDS")
```

```{r metadata-file}
# sample metadata file
meta_file <- file.path(data_dir, "gastric-cancer_metadata.tsv")
```

We'll create a directory to hold our plots.

```{r plots-dir, live = TRUE}
# Create a plots directory if it does not exist yet
plots_dir <- file.path("plots", "gastric-cancer")
if (!dir.exists(plots_dir)) {
  dir.create(plots_dir, recursive = TRUE)
}
```

**Output**

```{r output-files, live = TRUE}
# We will save a PDF copy of the PCA plot to the plots directory
# and name the file "gastric-cancer_PC_scatter.pdf"
pca_plot_file <- file.path(plots_dir, "gastric-cancer_PC_scatter.pdf")
```

## DESeq2

### Creating a DESeq2 dataset from a tximeta object

First, let's read in the data we processed with `tximeta`.

```{r read-rds, live = TRUE}
# Read in the RDS file we created in the last notebook
gene_summarized <- readr::read_rds(txi_file)
```

### Set up DESeq2 object

We use the tissue of origin in the design formula because that will allow us to model this variable of interest.

```{r ddset}
ddset <- DESeqDataSet(gene_summarized,
                      design = ~ tissue)
```

### Variance stabilizing transformation

Raw count data is not usually suitable for the algorithms we use for dimensionality reduction, clustering, or heatmaps. 
To improve this, we will transform the count data to create an expression measure that is better suited for these analyses. 
The core transformation will map the expression to a log2 scale, while accounting for some of the expected variation among samples and genes.

Since different samples are usually sequenced to different depths, we want to transform our RNA-seq count data to make different samples more directly comparable. 
We also want to deal with the fact that genes with low counts are also likely to have higher variance (on the log2 scale), as that could bias our clustering.
To handle both of these considerations, we can calculate a Variance Stabilizing Transformation of the count data, and work with that transformed data for our analysis.

See [this section of the `DESeq2` vignette](http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#data-transformations-and-visualization) for more on this topic. 

```{r vst}
vst_data <- vst(ddset)
```

### Principal component analysis

Principal component analysis (PCA) is a dimensionality reduction technique that allows us to identify the largest components of variation in a complex dataset.
Our expression data can be thought of as mapping each sample in a multidimensional space defined by the expression level of each gene.
The expression of many of those genes are correlated, so we can often get a better, simpler picture of the data by combining the information from those correlated genes.

PCA rotates and transforms this space so that each axis is now a combination of multiple correlated genes, ordered so the first axes capture the most variation from the data. 
These new axes are the "principal components."
If we look at the first few components, we can often get a nice overview of relationships among the samples in the data.

The `plotPCA()` function we will use from the `DESeq2` package calculates and plots the first two principal components (PC1 and PC2).
Visualizing PC1 and PC2 can give us insight into how different variables (e.g., tissue source) affect our dataset and help us spot any technical effects (more on that below).


```{r plotPCA, live = TRUE}
# DESeq2 built in function is called plotPCA and we want to color points by
# tissue
plotPCA(vst_data, intgroup = "tissue")
```

Save the most recent plot to file with `ggsave` from `ggplot2`

```{r save-pdf}
# Save the PDF file 
ggplot2::ggsave(pca_plot_file, plot = ggplot2::last_plot())
```

## A note on technical effects

We don't have batch information (i.e., when the samples were run) for this particular experiment, but let's imagine that `SRR585574` and `SRR585576` were run separately from all other samples.
We'll add this as a new "toy" column in the sample data (`colData`).

```{r extract-sample, live = TRUE}
# Extract colData
sample_info <- colData(vst_data)

# Print out preview
sample_info
```

Now we can add a new column with toy batch information and re-store the `colData()`. 

```{r add-batch}
# Add batch information
sample_info$batch <- c("batch1", "batch1", "batch1", "batch1", "batch2", 
                       "batch1", "batch2", "batch1")
```
                            
If this batch information were real we would have included it with the sample metadata when we made the original `SummarizedExperiment` object with `tximeta`.
We would then include it in the model stored in our DESeq2 object using the `design` argument (`design = ~ tissue + batch`) and we would re-run the `DESeqDataSet()` and `vst()` steps we did above. 
Here we will take a bit of a shortcut and add it directly to the `colData()` for our `vst()`-transformed data.

```{r coldata-vst, live = TRUE}
# Add coldata() with batch info to vst_data
colData(vst_data) <- sample_info
```

```{r plotPCA-2, live = TRUE}
# PCA plot - tissue *and* batch
# We want plotPCA to return the data so we can have more control about the plot
pca_data <- plotPCA(vst_data, 
                    intgroup = c("tissue", "batch"), 
                    returnData = TRUE)
```

```{r percent_var}
# Here we are setting up the percent variance that we are extracting from the `pca_data` object
percent_var <- round(100 * attr(pca_data, "percentVar"))
```

Let's use ggplot to visualize the first two principal components. 

```{r color-by-batch, live = TRUE}
# Color points by "batch" and use shape to indicate the tissue of origin
ggplot2::ggplot(pca_data, ggplot2::aes(PC1, PC2, 
                                       color = batch, 
                                       shape = tissue)) +
  ggplot2::geom_point(size = 3) +
  ggplot2::xlab(paste0("PC1: ", percent_var[1],"% variance")) +
  ggplot2::ylab(paste0("PC2: ", percent_var[2],"% variance")) + 
  ggplot2::coord_fixed()
```

## Session Info

Record session info for reproducibility & provenance purposes.

```{r sessioninfo}
sessionInfo()
```
