---
title: "Gastric cancer: exploratory analysis"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2018**

In this notebook, we'll import the gastric cancer data and do some exploratory
analyses and visual inspection.
We'll use the [`DESeq2`](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) package for this.

`DESeq2` also has an 
[excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) 
from Love, Anders, and Huber from which this is adapted 
(see also: [Love, Anders, and Huber. _Genome Biology_. 2014.](https://doi.org/10.1186/s13059-014-0550-8)).

## Libraries and functions

```{r}
library(DESeq2)
```

```{r}
# magrittr pipe
`%>%` <- dplyr::`%>%`
```

## Directories and files

```{r}
# directory with the tximport processed data
txi_dir <- file.path("data", "tximport", "gastric_cancer")
txi_file <- file.path(txi_dir, "gastric_cancer_tximport.RDS")
```

We'll create a directory to hold our plots.

```{r}
plots_dir <- "plots"
if (!dir.exists(plots_dir)) {
  dir.create(plots_dir, recursive = TRUE)
}
```

```{r}
# sample metadata file
meta_dir <- file.path("data", "sample_metadata")
meta_file <- file.path(meta_dir, "gastric_cancer",
                       "gastric_cancer_sample_metadata.tsv")
```

**Output**

```{r}
# PCA plot
pca_plot_file <- file.path(plots_dir, "gastric_cancer_PC_scatter.pdf")
```

## DESeq2

### Creating a DESeq2 dataset from tximport object

First, let's read in the data we processed with `tximport`.

```{r}
txi <- readr::read_rds(txi_file)
```

```{r}
head(txi$counts)  # gene-level counts
```

```{r}
head(txi$abundance)  # gene-level TPM
```

And now we'll read in the sample metadata.

```{r}
sample_meta_df <- readr::read_tsv(meta_file)
sample_meta_df
```

```{r}
colnames(txi$counts)
```

We'll reorder the sample metadata to match the order in `txi`

```{r}
sample_meta_df <- sample_meta_df %>%
  dplyr::arrange(accession)
sample_meta_df
```

```{r}
all.equal(colnames(txi$counts), sample_meta_df$accession)
```

We use the tissue of origin in the design formula because that will allow us
to model this variable of interest.

```{r}
ddset <- DESeqDataSetFromTximport(txi = txi,
                                  colData = sample_meta_df,
                                  design = ~ tissue)
```

### Variance-stablizing transformation

Before visualizing the data, we'll transform it such that it is on a `log2` 
scale and library size is taken into account with the `DESeq2` function
for variance-stablizing transformations.

```{r}
vst_data <- vst(ddset)
```

### Principal components analysis

Principal components analysis (PCA) is a dimensionality reduction technique
that captures the main sources of variation in our data in the first two 
principal components (PC1 and PC2).
Visualizing PC1 and PC2 can give us insight into how different variables (e.g.,
tissue source) affect our dataset and help us spot any technical effects 
(more on that below).

`DESeq2` has built-in functionality for performing PCA.

```{r}
DESeq2::plotPCA(vst_data, intgroup = "tissue")
```

Save the most recent plot to file with `ggsave` from `ggplot2`

```{r}
ggplot2::ggsave(pca_plot_file, plot = ggplot2::last_plot())
```


## A note on technical effects

We don't have batch information (i.e., when the samples were run) for this 
particular experiment, but let's imagine that `SRR585574` and `SRR585576` were
run separately from all other samples.
We'll put this in new "toy" column in `sample_meta_df`.

```{r}
sample_meta_df$batch <- c("batch1", "batch1", "batch1", "batch1", "batch2", 
                          "batch1", "batch2", "batch1")
```

Let's rerun the `DESeq2` steps.

```{r}
# Note: if we were using this for differential expression AND we truly knew
# the batch, we'd likely use design = ~ batch + tissue
ddset_batch <- DESeqDataSetFromTximport(txi = txi,
                                        colData = sample_meta_df,
                                        design = ~ tissue)
vst_batch <- vst(ddset_batch)
```

```{r}
pca_data <- DESeq2::plotPCA(vst_batch, intgroup = c("tissue", "batch"), 
                            returnData = TRUE)
percent_var <- round(100 * attr(pca_data, "percentVar"))
ggplot2::ggplot(pca_data, ggplot2::aes(PC1, PC2, color = batch, 
                                       shape = tissue)) +
  ggplot2::geom_point(size = 3) +
  ggplot2::xlab(paste0("PC1: ", percent_var[1],"% variance")) +
  ggplot2::ylab(paste0("PC2: ", percent_var[2],"% variance")) + 
  ggplot2::coord_fixed()
```

## Session Info

Record session info for reproducibility & provenence purposes.

```{r}
sessionInfo()
```