---
title: "Neuroblastoma Cell Line: Differential expression analysis with DESeq2"
author: CCDL for ALSF
date: 2021
output:
  html_notebook:
    toc: true
    toc_float: true
---


## Objectives

This notebook will demonstrate how to:

- Perform differential expression analysis with `DESeq2`
- Apply a shrinkage algorithm to improve estimates of expression changes
- Draw a volcano plot with the `EnhancedVolcano` package

---

In this notebook, we'll perform an analysis to identify the genes that are differentially expressed in _MYCN_ amplified vs. nonamplified neuroblastoma cell lines.

These RNA-seq data are from [Harenza, _et al._ (2017)](https://doi.org/10.1038/sdata.2017.33).

More information about DESeq2 can be found in the [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) from Love, Anders, and Huber from which this is adapted (see also: [Love, _et al._ (2014)](https://doi.org/10.1186/s13059-014-0550-8)).

DESeq2 takes unnormalized counts or estimated counts and does the following:

* [Estimates size factors](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/estimateSizeFactors)
* [Estimates dispersion](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/estimateDispersions)
* Negative binomial generalized linear model fitting and [Wald statistics](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/nbinomWaldTest)

![](diagrams/rna-seq_6.png)

## Libraries and functions

```{r library}
# Load the DESeq2 library
library(DESeq2)

# We will be making fancy volcano plots
library(EnhancedVolcano)
```

## Directories and files

**Input**

```{r input-files}
# directory with the tximeta processed data
txi_dir <- file.path("data", "NB-cell", "txi")
txi_file <- file.path(txi_dir, "NB-cell_tximeta.rds")
```


**Output**

We'll create a results directory to hold our results.

```{r results-dir}
# Create a results directory if it doesn't already exist
results_dir <- file.path("results", "NB-cell")
fs::dir_create(results_dir)
```

We will also need a directory to store our plots.

```{r plots-dir, live = TRUE}
# Create a plots directory if it doesn't already exist

```


```{r output-files}
# RDS for the output of DESeq analysis
deseq_file <- file.path(results_dir,
                        "NB-cell_DESeq_amplified_v_nonamplified.rds")

# DESeq2 results table
deseq_df_file <- file.path(results_dir,
                           "NB-cell_DESeq_amplified_v_nonamplified_results.tsv")

# PNG of the volcano plot
volcano_file <- file.path(plots_dir, "NB-cell_volcano.png")
```

## DESeq2

### Creating a DESeq2 dataset from tximeta object

First, let's read in the data we processed with `tximeta`.

#### Preparation

```{r read_rds, live = TRUE}
# Read in the RDS file we created in the last notebook

```

We're most interested in _MYCN_ amplification, which we had stored in the `status` column of the sample metadata of `gene_summarized`.
While the sample metadata is stored internally in the `colData` slot, the `SummarizedExperiment` object makes it easy for us to access it as if it were just a column of a data frame, using the familiar `$` syntax.


```{r Status, live = TRUE}

```

This is stored as a `character` type, but to give a bit more information to `DESeq`, we will convert this to a `factor`.

```{r status_factor, live = TRUE}

```

We'll want to use the "Nonamplified" samples as our _reference_.
Let's look at the `levels` of `status`.

```{r levels}
levels(gene_summarized$status)
```

We can see that these are in alphabetical order, so "Amplified" samples would be the reference.
We can use the `relevel()` function to remedy this.

```{r relevel}
gene_summarized$status <- relevel(gene_summarized$status, ref = "Nonamplified")
```

```{r check-levels, live = TRUE}
# Check what the levels are now

```



#### DESeq Dataset creation

```{r ddset, live = TRUE}
# Create a DESeq2 dataset from `gene_summarized`
# remember that `status` is the variable of interest here

```

## Differential expression analysis

### Filtering low-expressed genes

Genes that have very low counts are not likely to yield reliable differential expression results, so we will do some light [pre-filtering](http://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#pre-filtering).
We will keep only genes with total counts of at least 10 across all samples.

```{r filter_ddset}
# create a vector of TRUE and FALSE values where
# TRUE corresponds to genes with counts of at least 10 
genes_to_keep <- rowSums(counts(ddset)) >= 10
# use which() to prevent any NAs sneaking through
ddset <- ddset[which(genes_to_keep), ]
```


### The `DESeq()` function

We'll now use the wrapper function `DESeq()` to perform our differential expression analysis.
As mentioned earlier, this performs a number of steps, including an [outlier removal procedure](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#approach-to-count-outliers).
For this particular dataset, there is a pretty large number of outliers, which can be a bit of a red flag, but we will proceed for now.

```{r DESeq}
deseq_object <- DESeq(ddset)
```

Let's save this to our results file.

```{r write_rds, live = TRUE}
# Save the results as an RDS

```

Now we will have a look at the results table.

```{r deseq_results}
deseq_results <- results(deseq_object)
deseq_results
```

How many genes were differentially expressed (FDR < 0.05)?

```{r results_summary}
summary(deseq_results, alpha = 0.05)
```


### Shrinking log2 fold change estimates

The estimates of log2 fold change calculated by `DESeq()` are not corrected for expression level.
This means that when counts are small, we are likely to end up with some large fold change values that overestimate the true extent of the change between conditions.

We can correct this by applying a "shrinkage" procedure, which will adjust large values with small counts downward, while preserving values with larger counts, which are likely to be more accurate.

To do this, we will use the `lfcShrink()` function, but first we need to know the name and/or position of the "coefficient" that was calculated by `DESeq()`, which we can do with the `resultsNames()` function

```{r deseq_coef}
# get the deseq coefficient names:
resultsNames(deseq_object)
```

We are interested in the `status` coefficient, which is in position 2.

There are a few options for the shrinkage estimation.
The default is `apeglm` ([Zhu _et al._ 2018](https://doi.org/10.1093/bioinformatics/bty895)), but we have found that this can be sensitive to extreme outliers, which are definitely a factor in this data set.
So for this data set we will be using `ashr` ([Stephens 2017](https://doi.org/10.1093/biostatistics/kxw041))

```{r lfc_shrink}
# calculate shrunken log2 fold change estimates
deseq_shrunken <- lfcShrink(deseq_object,
  coef = 2, # the coefficient we want to reestimate
  type = "ashr" # We will use `ashr` for estimation
)
```

Let's compare the log2 fold change estimates from the two results tables by creating a plot.

First we will combine the results into a new data frame.

```{r compare_shrink}
comparison_df <- data.frame(
  lfc_original = deseq_results$log2FoldChange,
  lfc_shrunken = deseq_shrunken$log2FoldChange,
  logmean = log10(deseq_results$baseMean)
)
```

Now we can plot the original and shrunken log2 fold change values to see what happened after shrinkage.

```{r plot_comparison}
ggplot(comparison_df,
  aes(
    x = lfc_original,
    y = lfc_shrunken,
    color = logmean
  )
) +
  geom_point(alpha = 0.1) +
  theme_bw() +
  scale_color_viridis_c() +
  coord_cartesian(xlim = c(-10, 10), ylim = c(-10, 10)) # zoom in on the middle
```

We will now do a bit of manipulation to store the results in a data frame and add the gene symbols.

```{r results_dataframe}
# this is of class DESeqResults -- we want a data frame
deseq_df <- deseq_shrunken |>
  # convert to a data frame
  as.data.frame() |>
  # the gene ids were stored as row names -- let's them a column for easy display
  tibble::rownames_to_column(var = "gene_id") |>
  # add on the gene symbols from the original deseq object
  dplyr::mutate(gene_symbol = rowData(deseq_object)$gene_name)
```

Let's print out the results table, sorted by log2 fold change.
The highest values should be genes more expressed in the _MYCN_ amplified cell lines.

```{r sorted_table, live = TRUE}
# Print the table sorted by log2FoldChange

```

Now let's write the full results table to a file.

```{r write_tsv}
readr::write_tsv(deseq_df, file = deseq_df_file)
```


## Making a Volcano Plot

With these shrunken effect sizes, we will draw a volcano plot, using the [`EnhancedVolcano` package](https://github.com/kevinblighe/EnhancedVolcano) to make it a bit easier.
This package automatically color codes the points by cutoffs for both significance and fold change and labels many of the significant genes (subject to spacing).
`EnhancedVolcano` has many, many options, which is a good thing if you don't like all of it's default settings.
Even better, it outputs a `ggplot2` object, so if we want to customize it further, we can do that with the same `ggplot2` commands we have used before.

```{r volcano}
EnhancedVolcano(deseq_df,
  x = "log2FoldChange", # fold change statistic to plot
  y = "pvalue", # significance values
  lab = deseq_df$gene_symbol, # labels for points
  pCutoff = 1e-05, # The p value cutoff we will use (default)
  FCcutoff = 1, # The fold change cutoff (default)
  title = NULL, # no title
  subtitle = NULL, # or subtitle
  caption = NULL, # or caption
  labSize = 3 # smaller labels
) +
  # change the overall theme
  theme_classic() +
  # move the legend to the bottom
  theme(legend.position = "bottom")
```

We will save this plot to a file as well:

```{r save_plot}
ggsave(volcano_file, plot = last_plot())
```


## Session Info

Record session info for reproducibility & provenance purposes.

```{r sessioninfo}
sessionInfo()
```
