---
title: "Neuroblastoma Cell Line: Differential expression analysis with DESeq2"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2018**

In this notebook, we'll perform an analysis to identify the genes that are 
differentially expressed in _MYCN_ amplified vs. nonamplified neuroblastoma 
cell lines. 

These RNA-seq data are from 
[Harenza, et al. _Scientific Data._ 2017.](https://doi.org/10.1038/sdata.2017.33).

More information about DESeq2 can be found in the [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) 
from Love, Anders, and Huber from which this is adapted 
(see also: [Love, Anders, and Huber. _Genome Biology_. 2014.](https://doi.org/10.1186/s13059-014-0550-8))

DESeq2 takes unnormalized counts or estimated counts and does the following:

* [Estimates size factors](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/estimateSizeFactors)
* [Estimates dispersion](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/estimateDispersions) 
* Negative binomial generalized linear model fitting and [Wald statistics](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/nbinomWaldTest)

![](diagrams/rna-seq_6.png)

## Libraries and functions

```{r library}
# Load the DESeq2 library
library(DESeq2)
```

```{r magrittr}
# magrittr pipe
`%>%` <- dplyr::`%>%`
```

## Directories and files

```{r input-files}
# directory with the tximport processed data
txi_dir <- file.path("data", "tximport", "NB_cell_line")
txi_file <- file.path(txi_dir, "NB_cell_line_tximport.RDS")
```

We'll create a results directory to hold our results.

```{r results-dir}
# Create a results directory if it doesn't already exist
results_dir <- "results"
if (!dir.exists(results_dir)) {
  dir.create(results_dir, recursive = TRUE)
}
```

```{r plots-dir, live = TRUE}
# Create a plots directory if it doesn't already exist
plots_dir <- "plots"
if (!dir.exists(plots_dir)) {
  dir.create(plots_dir, recursive = TRUE)
}
```

```{r meta-files}
# sample metadata path and  file
meta_dir <- file.path("data", "sample_metadata")
meta_file <- file.path(meta_dir, "NB_cell_line",
                       "NB_cell_line_metadata_cleaned.tsv")
```

**Output**

```{r output-files}
# RDS for the output of DESeq function
deseq_file <- file.path(results_dir, 
                        "NB_cell_line_DESeq_amplified_v_nonamplified.RDS")
# DESeq2 results data.frame
deseq_df_file <- 
  file.path(results_dir,
            "NB_cell_line_DESeq_amplified_v_nonamplified_results.tsv")

# PNG of the heatmap of DEGs
heatmap_file <- file.path(plots_dir, "NB_cell_line_heatmap.png")
```

## DESeq2

### Creating a DESeq2 dataset from tximport object

First, let's read in the data we processed with `tximport`.

```{r read-rds, live = TRUE}
# Read the RDS file for the NB cell line data
txi <- readr::read_rds(txi_file)
```

Now, the sample metadata.

```{r sample_meta_df, live = TRUE}
# Read in the sample metadata TSV file and check it out
sample_meta_df <- readr::read_tsv(meta_file)
sample_meta_df
```

We're most interested in _MYCN_ amplification, which is in the `Status` column
of `sample_meta_df`

```{r Status, live = TRUE}
# Look at the first few entries in the Status column
head(sample_meta_df$Status)
```

Let's check that the ordering of samples is the same.

```{r all.equal}
tximport_sample_names <- colnames(txi$counts)
all.equal(tximport_sample_names, sample_meta_df$Sample_SRR_accession)
```

Let's reorder by `Sample_SRR_accession` and see if that remedies the ordering.

```{r arrange-sample_meta_df}
sample_meta_df <- sample_meta_df %>%
  dplyr::arrange(Sample_SRR_accession)
```

```{r all.equal-2}
all.equal(tximport_sample_names, sample_meta_df$Sample_SRR_accession)
```

It worked!

#### Dataset creation

```{r ddset, live = TRUE}
# Create a DESeq2 dataset - remember Status is the variable of interest here
ddset <- DESeqDataSetFromTximport(txi = txi,
                                  colData = sample_meta_df,
                                  design = ~ Status)
```

### Differential expression

#### Preparation

Let's do some [pre-filtering](http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#pre-filtering) 
to keep rows with 10 or more reads total.

```{r rowSums}
genes_to_keep <- rowSums(counts(ddset)) >= 10
ddset <- ddset[genes_to_keep, ]
```

We'll want to use the "Nonamplified" samples as our _reference_.
Let's look at the `levels` of `Status`.

```{r levels}
levels(ddset$Status)
```

We can see that these are in alphabetical order, so "Amplified" samples would
be the reference. 
We can use the `relevel` function to remedy this.

```{r relevel}
ddset$Status <- relevel(ddset$Status, ref = "Nonamplified")
```

```{r check-levels, live = TRUE}
# Check what the levels are now
levels(ddset$Status)
```

#### Differential expression analysis

We'll use the wrapper function `DESeq` to do our differential expression 
analysis.

```{r DESeq}
deseq_object <- DESeq(ddset)
```

Let's save this to our results file.

```{r write_rds, live = TRUE}
# Save the results as an RDS
readr::write_rds(deseq_object, path = deseq_file)
```

Let's take a look at the results table.

```{r deseq_results}
deseq_results <- results(deseq_object)
deseq_results
```

```{r arrange-deseq_results}
# this is of class DESeqResults -- we want a data.frame
deseq_df <- deseq_results %>%
  # make into data.frame
  as.data.frame() %>%
  # the gene names are rownames -- let's make this it's own column for easy 
  # display
  tibble::rownames_to_column(var = "Gene")

deseq_df %>%
  # let's sort by statistic -- the highest values should be what is up in the
  # MYCN amplified cell lines
  dplyr::arrange(dplyr::desc(stat))
```

Let's take a look at what the two top genes are:

* [`ENSG00000233718`](https://useast.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000233718;r=2:15921037-15942249)
* [`ENSG00000134323`](http://useast.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000134323;r=2:15940564-15947007)

How many genes were differentially expressed (FDR < 0.05)?

```{r summary}
summary(results(deseq_object), alpha = 0.05)
```

**Write full data.frame to file**

```{r write_tsv}
readr::write_tsv(deseq_df, path = deseq_df_file)
```

#### Visualize differentially expressed genes with a heatmap

Let's get an overall picture of our results using a heatmap. 
Before we can do this, we will need to transform our data so that when we make
the heatmap the genes are on a more similar scale and the heatmap is 
interpretable. 
We will use variance stabilizing transformation from DESeq2 to do this.
Refer to the help page or the DESeq2 docs for more information. 

```{r vst, live = TRUE}
# Use the vst function to transform the data
vst_data <- vst(ddset)
```

We need to extract the normalized gene matrix from the `vst_data` object. 

```{r vst-assay}
vst_data <- assay(vst_data)
```

We will make a heatmap of only the significant genes, so let's obtain the gene 
IDs that have an adjusted p-value less than 0.05.

```{r signif_genes, live = TRUE}
# Filter to padj < 0.05 and then only extract the Genes
signif_genes <- deseq_df %>%
  dplyr::filter(padj < 0.05) %>%
  dplyr::pull(Gene)
```

Now we will use `signif_genes` to filter `vst_data` to only have significant 
genes' data. 

```{r signif_vst_data}
signif_vst_data <- vst_data %>%
  as.data.frame() %>%
  dplyr::filter(rownames(vst_data) %in% signif_genes)
```

Let's make the results into a heatmap using the `pheatmap` package. 
We will use the `scale = "row"` argument so that the data is standardized by
gene (i.e., will be made into [z-scores](https://www.statisticshowto.datasciencecentral.com/probability-and-statistics/z-score/)). 
Standardizing the data by gene makes it so our heatmap is more easily 
interpretable despite how different genes' distributions may be. 

```{r pheatmap}
pheatmap::pheatmap(signif_vst_data, 
                   show_rownames = FALSE,
                   scale = "row",
                   colorRampPalette(c("deepskyblue",
                                      "black",
                                      "yellow"))(25))
```

If we want to save this heatmap as a PNG, we have to use the `png` function that 
is part of base R, run `pheatmap::pheatmap` again, and then use `dev.off()`.

```{r png, live = TRUE}
png(filename = heatmap_file)
# Use the same command to create the heatmap
pheatmap::pheatmap(signif_vst_data, 
                   show_rownames = FALSE,
                   scale = "row",
                   colorRampPalette(c("deepskyblue",
                                      "black",
                                      "yellow"))(25))
# This shuts down the current graphics device (the PNG we are saving)
dev.off()
```

We can specify the dimensions and resolution of the PNG that is saved by 
supplying different arguments to the `png` function.

```{r png-adjust}
png(filename = heatmap_file, 
    width = 7, 
    height = 7, 
    units = "in", 
    res = 300)
# Heatmap time!
pheatmap::pheatmap(signif_vst_data, 
                   show_rownames = FALSE,
                   scale = "row",
                   colorRampPalette(c("deepskyblue",
                                      "black",
                                      "yellow"))(25))
# Shut down the current graphics device
dev.off()
```

## Session Info

Record session info for reproducibility & provenance purposes.

```{r sessioninfo}
sessionInfo()
```
