---
title: "scRNA-seq Normalization"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2019**

In this notebook, we'll perform quality control analyses and normalization of 
scRNA-seq count data. 
    
For this tutorial, we will be using a pair of single-cell analysis specific 
R packages: `scater` and `scran` to work with our data. 
This tutorial is in part based on the [scran
tutorial.](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html)
  
## Set Up 

For these analyses, we will need `scater` and `scran` packages, which have been 
installed already on your Docker container. 

```{r Set Up}
# Read in a function for filtering the gene matrix from a source script
source(file.path("scripts", "GeneMatrixFilter.R"))

# Magrittr pipe
`%>%` <- dplyr::`%>%`

# Load the already installed packages
library(scater)
library(scran)
```

## Import single-cell RNA-seq counts matrix

This [data set](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) 
we are using is glioblastoma data that was Fluorescence-Activated Cell sorted 
and then was processed by paired end sequencing using Smart-seq2 protocol 
[(Darmanis et al. _Cell Reports._ 2017.).](https://www.ncbi.nlm.nih.gov/pubmed/29091775).
We're using a subset of this dataset (n = 1854 cells).
You have been provided with a counts matrix from single-cell RNA-seq data that 
has already been processed using `Salmon` and `tximport` and saved the counts to 
a tsv file, just as we did in the bulk RNA-seq modules.
As far as *tag-based* scRNA-seq data, we will explain how to process raw data, 
like fastq files, in the next section, so you can also get it to a count matrix
and could use these same steps to filter and normalize the data.
  
```{r Import Data}
# Read in the data
sc_data <- readr::read_tsv(file.path("data", "unfiltered_darmanis_counts.tsv"), 
                           progress = FALSE)

# Read in the metadata
sc_metadata <- readr::read_tsv(file.path("data", "unfiltered_darmanis_metadata.tsv"), 
                               progress = FALSE)
```

Now that we have loaded our data, let's take a look at what it looks like 
overall. 
Use a command we have used previously to look at the first few rows of `sc_data`.

```{r}
# Put a command here that would allow you to see the first few rows of `sc_data`
```

You'll notice the first column contains the gene information and the rest are samples. 
For the analyses in this notebook, the gene column (which is a `character` type), will get in 
the way, so turn this column into the rownames instead (so it is out of our way but we still
have the information).

```{r Remove the gene column}
# Set the column as the gene names
sc_data <- sc_data %>% tibble::column_to_rownames("genes")
```

Let's look at how the gene means look in this dataset. 
We will use `apply` in order to calculate things across our data.frame. 
The second argument in `apply` specifies whether we are calculating by rows or columns. 
(1 = rows, 2 = columns).

```{r}
# Let's calculate the gene means and make a density plot
gene_means <- apply(sc_data, 1, mean)
# What do you think the dimensions of `gene_means` are? 
```

Let's make our first density plot with these data.

```{r}
# Plot the density of the means using ggplot2's `qplot`
qplot(gene_means, geom = "density", xlab = "Gene Mean Counts") 
```

The counts data has so many zeroes, which we should expect. 
83.3% of the data points are zeroes. 

## Filter the counts matrix
  
The small amount of RNA in a single cell results in higher chances of errors or 
biases in amplification steps, so we don't necessarily want to keep the data for 
all the cells and genes

#### Total counts as a quality measure

Let's take a look at this data from how many counts are there per sample. 
We will use `apply` again here, but what will we use for the second argument?

```{r}
# Make a vector of the total number of counts per sample
total_counts <- apply(sc_data, 2, sum)
```

We want to get the `sum` of each column (ie for each cell), so we put `2`
as our argument in `apply` this time.

```{r}
# Take a look at the summary statistics for the total counts
summary(total_counts)
```

Yikes, one of the samples has only 48 counts. 
It's highly likely that this 'cell' is either an empty well or did not get 
sequenced properly. 

Let's visualize the distribution of total counts to see if the 48 count sample 
is the only one we should get rid of.

In following graphs, we will use vertical red lines to indicate possible 
cutoffs. 

```{r Explore total counts distribution}
# Let's use the same kind of plot as above but add more layers
qplot(total_counts, geom = "density", xlab = "Total Counts Per Sample") + # ggplot2 uses `+` to add layers
  geom_density(fill = "lightblue") + # This will add a color fill to shape
  geom_vline(xintercept = 50000, color = "red") # This will add our line for a cutoff
```

We will learn more about `ggplot2` later on, but note that layers and extra 
features of a plot are added with `+` like above.
Let's filter out some of the samples that have lower amount total counts. 
We also will need to remove these samples from our metadata.

```{r Filter by total counts}
# Filter our data based on this cutoff 
sc_data_filtered <- sc_data %>% 
  dplyr::select(which(total_counts > 50000))
```

Remember that now that we've filtered our data, we need to filter our metadata
accordingly.

```{r Filter metadata accordingly}
# So we can be sure that our data matches or metadata, we will use a filter
sc_metadata_filtered <- sc_metadata %>% 
  dplyr::filter(geo_accession %in% colnames(sc_data_filtered))

# Let's double check that the samples in the data and metadata are the same.
all.equal(colnames(sc_data_filtered), sc_metadata_filtered$geo_accession)
```

### Number of genes a cell expresses as a quality measure

What if a single gene accounted for all counts in a particular cell?
This cell would not have helpful data for us, so we should look to remove any 
cells we suspect might not have a useful amount of its transcriptome measured.
But before we can determine how many genes we consider a particular cell to be 
expressing we need to determine a numeric cutoff for what we consider to be a 
detected gene.
How many counts must there be for you to consider a gene expressed? 
Here let's go for a simple detection cutoff of > 0. 

```{r Create detection matrix}
detection_mat <- as.matrix(sc_data_filtered) > 0 
```

Now that we have turned our data into a matrix of `TRUE/FALSE` for detection, we
can sum this data by column to effectively get a vector of how many genes were 
measured in each cell. 

```{r}
# Make a vector that contains the number of genes expressed by a particular cell
num_genes_exp <- apply(detection_mat, 2, sum)
```

Let's plot this using the same style and type of graph as above. 

```{r}
# Let's take a look at what this looks like:
qplot(num_genes_exp, geom = "density", xlab = "Number of Genes Expressed by Each Sample") +
  geom_density(fill = "lightblue") + 
  theme_classic() # Adding this to make it a prettier style aesthetically
```

But as you may have noticed in our `sc_metadata` we have cell-type information 
for these data. 
Let's use the power of our single-cell resolution data to our advantage and look 
at this data with cell-type labels. 
In order to do this we need to prepare a dataframe that has our `num_genes_exp`
numbers along side our cell-type labels.

```{r}
# Let's make a dataframe with this information
num_genes_exp_df <- data.frame(num_genes_exp, 
                               'cell_type' = sc_metadata_filtered$cell.type.ch1) 
```

The next graph we are using the same steps as before but to make individual graphs for each 
cell type, we are using a fancy option in ggplot2 called `facet_wrap`.

```{r}
# Plot these data
ggplot(num_genes_exp_df, aes(x = num_genes_exp)) + 
  geom_density(fill = "lightblue") + 
  geom_vline(xintercept = 750, color = "red") + 
  facet_wrap(~ cell_type) +
  theme_classic()
  # Facet wrap takes a label you give it and makes individual smaller graphs from it. 
```

Now we are aware of how our filtering may be affecting different cell types differently. 

### Number of samples that express a gene as a quality measure

Now we have an idea of what samples we probably want to get rid of.
But what if our data contains genes that we can't reliably measure in these cells?

Let's use our `detection_mat` to add up how many samples express each of 
these genes so we can try to filter out unreliably measured genes.  

```{r}
# Make a vector that contains the number of cells that express a particular gene
num_samples_that_exp <- apply(detection_mat, 1, sum)
```

Let's make another density plot with the number of samples that express each gene:

```{r}
# Let's take a look at what this looks like:
qplot(num_samples_that_exp, geom = "density", xlab = "Number of Samples Expressing Each Gene") + 
  geom_density(fill = "lightblue") +
  geom_vline(xintercept = 10, col = "red") +
  theme_classic()
```

### Apply a filter based on our criteria

Now that we've explored our data, let's used the information we've learned to 
actually filter our data.

```{r Apply filter}
sc_data_filtered <- GeneMatrixFilter(sc_data_filtered, 
                                     min_counts = 0, 
                                     num_samples = 10, 
                                     num_genes = 750)
```

Need to filter our metadata accordingly, again:

```{r}
# Filter metadata accordingly again
sc_metadata_filtered <- sc_metadata_filtered %>% 
  dplyr::filter(geo_accession %in% colnames(sc_data_filtered))
```

## Set up a SingleCellExperiment object from count data

For the rest of this module and some of the upcoming modules, we are going to 
use some single-cell RNA-seq R packages called `scater` and `scran` to do some 
processing for us.
The functions in these packages require the data to be in their own special 
object type (not an uncommon thing for R packages to do) called `SingleCellExperiment`.
So first, we are going to set up our data in this format.

```{r Set up SCE object}
# Prep a matrix with just the rounded counts so we can create an sce object
counts_mat <- round(as.matrix(sc_data_filtered))
```

Put our counts matrix in this special `SingleCellExperiment` function: 

```{r}
# Set up the Single Cell Experiment object 
sce_raw <- SingleCellExperiment::SingleCellExperiment(list(counts = counts_mat))
```

Special object type are difficult to use at times if we don't know what they 
look like. 
Based on things we've learned in the previous modules, use this space to explore
the `SingleCellExperiment` object you just made.

```{r Explore SCE structure}
# The SingleCellExperiment is a special type of object used by scater and scran
# R packages, explore its properties here. 
# Also check it out in the environment tab.
```

## Normalization of count data 

In whatever data we are working with, we are always looking to maximize biological 
variance and minimize technical variance. 
This is where normalization methods usually come into the workflow.
With this high amount of zeroes in our data, this makes normalization particularly
tricky.
So in order to account for these zeroes, we normalize cells in groups with other 
cells like them; a method introduced in [Lun et al 2016](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0947-7).

*These steps may take a minute or so.*

```{r Normalize Data}
# Set seed for reproducibility
set.seed(1234)

# Step 1) Group cells with other like cells by clustering.  
qclust <- scran::quickCluster(sce_raw)

# Step 2) Compute sum factors for each cell cluster grouping.  
sce_raw <- scran::computeSumFactors(sce_raw, clusters = qclust)

# Step 3) Normalize using these pooled sum factors and log transform. 
sce_norm <- scater::normalize(sce_raw)
```

## Compare normalized data to count data

One way to determine whether our normalization yields biologically relevant 
results, is to plot it and see if similarly labeled samples and cells end up 
together.
Because plotting 1000's of genes together isn't practical, we will reduce the 
dimensions of our data using Principal Components Analysis. 
Note that the `t()` function is being used here to transpose the `matrix`, so 
it is samples x genes. 
We could however, obtain PCA scores for each gene if we didn't transpose our
data.frame first. 

```{r}
# Use PCA for dimension reduction of cells' raw data
count_pca <- prcomp(t(counts(sce_raw)))

# Use PCA for dimension reduction of cells' scran normalized data
norm_pca <- prcomp(t(logcounts(sce_norm)))
```

Now we will retrieve the PCA scores, by using `$x` at the end of our `prcomp`
objects.

```{r Graph PCA}
# Make the counts PC scores into a dataframe with cell type labels for easy graphing 
count_pca_scores <- data.frame(count_pca$x[, 1:2],
                               'cell_type' = sc_metadata_filtered$cell.type.ch1)

# Set up the normalized PC scores in the same way
norm_pca_scores <- data.frame(norm_pca$x[, 1:2],
                              'cell_type' = sc_metadata_filtered$cell.type.ch1)
```

Let's plot the counts pca scores with their cell labels:

```{r Plotting counts pca}
# Now plot counts pca
ggplot(count_pca_scores, aes(x = PC1, y = PC2, color = cell_type)) +
  geom_point() + # We use this so it is a scatterplot
  ggtitle("Counts PCA Scores") + # Put a title for this graph
  colorblindr::scale_color_OkabeIto() # We are adding this so it is colorblind friendly
```

We've plotted the counts data for you. Knowing that we want the same graph,
but different data, use the above template to plot the normalized data.

Let's plot the `norm_pca_scores` data:

```{r Plotting norms pca}
# Copy and paste the above template but plot the normalized data instead
```

The normalization has appeared to help separate the rest of the brain cells 
from the immune cells. 

## Save the normalized data to tsv file

In case we wanted to return to this data later, let's save the normalized data
to a tsv file. 
`readr` requires that it be a `data.frame` so we will convert our `matrix` first.

```{r Save Data to .tsv}
# Save this gene matrix to a tsv file
logcounts(sce_norm) %>% as.data.frame() %>%
  readr::write_tsv(file.path("data", "scran_norm_gbm_gene_matrix.tsv"))
```

### Print session info

```{r}
sessionInfo()
```
