<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>scRNA-seq Normalization</title>

<script>/*! jQuery v1.11.3 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
!function(a,b){"object"==typeof module&&"object"==typeof module.exports?module.exports=a.document?b(a,!0):function(a){if(!a.document)throw new Error("jQuery requires a window with a document");return b(a)}:b(a)}("undefined"!=typeof window?window:this,function(a,b){var c=[],d=c.slice,e=c.concat,f=c.push,g=c.indexOf,h={},i=h.toString,j=h.hasOwnProperty,k={},l="1.11.3",m=function(a,b){return new m.fn.init(a,b)},n=/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g,o=/^-ms-/,p=/-([\da-z])/gi,q=function(a,b){return b.toUpperCase()};m.fn=m.prototype={jquery:l,constructor:m,selector:"",length:0,toArray:function(){return d.call(this)},get:function(a){return null!=a?0>a?this[a+this.length]:this[a]:d.call(this)},pushStack:function(a){var b=m.merge(this.constructor(),a);return b.prevObject=this,b.context=this.context,b},each:function(a,b){return m.each(this,a,b)},map:function(a){return this.pushStack(m.map(this,function(b,c){return a.call(b,c,b)}))},slice:function(){return this.pushStack(d.apply(this,arguments))},first:function(){return this.eq(0)},last:function(){return this.eq(-1)},eq:function(a){var b=this.length,c=+a+(0>a?b:0);return this.pushStack(c>=0&&b>c?[this[c]]:[])},end:function(){return this.prevObject||this.constructor(null)},push:f,sort:c.sort,splice:c.splice},m.extend=m.fn.extend=function(){var a,b,c,d,e,f,g=arguments[0]||{},h=1,i=arguments.length,j=!1;for("boolean"==typeof g&&(j=g,g=arguments[h]||{},h++),"object"==typeof g||m.isFunction(g)||(g={}),h===i&&(g=this,h--);i>h;h++)if(null!=(e=arguments[h]))for(d in e)a=g[d],c=e[d],g!==c&&(j&&c&&(m.isPlainObject(c)||(b=m.isArray(c)))?(b?(b=!1,f=a&&m.isArray(a)?a:[]):f=a&&m.isPlainObject(a)?a:{},g[d]=m.extend(j,f,c)):void 0!==c&&(g[d]=c));return g},m.extend({expando:"jQuery"+(l+Math.random()).replace(/\D/g,""),isReady:!0,error:function(a){throw new Error(a)},noop:function(){},isFunction:function(a){return"function"===m.type(a)},isArray:Array.isArray||function(a){return"array"===m.type(a)},isWindow:function(a){return null!=a&&a==a.window},isNumeric:function(a){return!m.isArray(a)&&a-parseFloat(a)+1>=0},isEmptyObject:function(a){var b;for(b in a)return!1;return!0},isPlainObject:function(a){var b;if(!a||"object"!==m.type(a)||a.nodeType||m.isWindow(a))return!1;try{if(a.constructor&&!j.call(a,"constructor")&&!j.call(a.constructor.prototype,"isPrototypeOf"))return!1}catch(c){return!1}if(k.ownLast)for(b in a)return j.call(a,b);for(b in a);return void 0===b||j.call(a,b)},type:function(a){return null==a?a+"":"object"==typeof a||"function"==typeof a?h[i.call(a)]||"object":typeof a},globalEval:function(b){b&&m.trim(b)&&(a.execScript||function(b){a.eval.call(a,b)})(b)},camelCase:function(a){return a.replace(o,"ms-").replace(p,q)},nodeName:function(a,b){return a.nodeName&&a.nodeName.toLowerCase()===b.toLowerCase()},each:function(a,b,c){var d,e=0,f=a.length,g=r(a);if(c){if(g){for(;f>e;e++)if(d=b.apply(a[e],c),d===!1)break}else for(e in a)if(d=b.apply(a[e],c),d===!1)break}else if(g){for(;f>e;e++)if(d=b.call(a[e],e,a[e]),d===!1)break}else for(e in a)if(d=b.call(a[e],e,a[e]),d===!1)break;return a},trim:function(a){return null==a?"":(a+"").replace(n,"")},makeArray:function(a,b){var c=b||[];return null!=a&&(r(Object(a))?m.merge(c,"string"==typeof a?[a]:a):f.call(c,a)),c},inArray:function(a,b,c){var d;if(b){if(g)return g.call(b,a,c);for(d=b.length,c=c?0>c?Math.max(0,d+c):c:0;d>c;c++)if(c in b&&b[c]===a)return c}return-1},merge:function(a,b){var c=+b.length,d=0,e=a.length;while(c>d)a[e++]=b[d++];if(c!==c)while(void 0!==b[d])a[e++]=b[d++];return a.length=e,a},grep:function(a,b,c){for(var d,e=[],f=0,g=a.length,h=!c;g>f;f++)d=!b(a[f],f),d!==h&&e.push(a[f]);return e},map:function(a,b,c){var d,f=0,g=a.length,h=r(a),i=[];if(h)for(;g>f;f++)d=b(a[f],f,c),null!=d&&i.push(d);else for(f in a)d=b(a[f],f,c),null!=d&&i.push(d);return e.apply([],i)},guid:1,proxy:function(a,b){var c,e,f;return"string"==typeof b&&(f=a[b],b=a,a=f),m.isFunction(a)?(c=d.call(arguments,2),e=function(){return a.apply(b||this,c.concat(d.call(arguments)))},e.guid=a.guid=a.guid||m.guid++,e):void 0},now:function(){return+new Date},support:k}),m.each("Boolean Number String Function Array Date RegExp Object Error".split(" "),function(a,b){h["[object "+b+"]"]=b.toLowerCase()});function r(a){var b="length"in a&&a.length,c=m.type(a);return"function"===c||m.isWindow(a)?!1:1===a.nodeType&&b?!0:"array"===c||0===b||"number"==typeof b&&b>0&&b-1 in a}var s=function(a){var b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u="sizzle"+1*new Date,v=a.document,w=0,x=0,y=ha(),z=ha(),A=ha(),B=function(a,b){return a===b&&(l=!0),0},C=1<<31,D={}.hasOwnProperty,E=[],F=E.pop,G=E.push,H=E.push,I=E.slice,J=function(a,b){for(var c=0,d=a.length;d>c;c++)if(a[c]===b)return c;return-1},K="checked|selected|async|autofocus|autoplay|controls|defer|disabled|hidden|ismap|loop|multiple|open|readonly|required|scoped",L="[\\x20\\t\\r\\n\\f]",M="(?:\\\\.|[\\w-]|[^\\x00-\\xa0])+",N=M.replace("w","w#"),O="\\["+L+"*("+M+")(?:"+L+"*([*^$|!~]?=)"+L+"*(?:'((?:\\\\.|[^\\\\'])*)'|\"((?:\\\\.|[^\\\\\"])*)\"|("+N+"))|)"+L+"*\\]",P=":("+M+")(?:\\((('((?:\\\\.|[^\\\\'])*)'|\"((?:\\\\.|[^\\\\\"])*)\")|((?:\\\\.|[^\\\\()[\\]]|"+O+")*)|.*)\\)|)",Q=new RegExp(L+"+","g"),R=new RegExp("^"+L+"+|((?:^|[^\\\\])(?:\\\\.)*)"+L+"+$","g"),S=new RegExp("^"+L+"*,"+L+"*"),T=new RegExp("^"+L+"*([>+~]|"+L+")"+L+"*"),U=new RegExp("="+L+"*([^\\]'\"]*?)"+L+"*\\]","g"),V=new RegExp(P),W=new RegExp("^"+N+"$"),X={ID:new RegExp("^#("+M+")"),CLASS:new RegExp("^\\.("+M+")"),TAG:new RegExp("^("+M.replace("w","w*")+")"),ATTR:new RegExp("^"+O),PSEUDO:new RegExp("^"+P),CHILD:new RegExp("^:(only|first|last|nth|nth-last)-(child|of-type)(?:\\("+L+"*(even|odd|(([+-]|)(\\d*)n|)"+L+"*(?:([+-]|)"+L+"*(\\d+)|))"+L+"*\\)|)","i"),bool:new RegExp("^(?:"+K+")$","i"),needsContext:new RegExp("^"+L+"*[>+~]|:(even|odd|eq|gt|lt|nth|first|last)(?:\\("+L+"*((?:-\\d)?\\d*)"+L+"*\\)|)(?=[^-]|$)","i")},Y=/^(?:input|select|textarea|button)$/i,Z=/^h\d$/i,$=/^[^{]+\{\s*\[native \w/,_=/^(?:#([\w-]+)|(\w+)|\.([\w-]+))$/,aa=/[+~]/,ba=/'|\\/g,ca=new RegExp("\\\\([\\da-f]{1,6}"+L+"?|("+L+")|.)","ig"),da=function(a,b,c){var d="0x"+b-65536;return d!==d||c?b:0>d?String.fromCharCode(d+65536):String.fromCharCode(d>>10|55296,1023&d|56320)},ea=function(){m()};try{H.apply(E=I.call(v.childNodes),v.childNodes),E[v.childNodes.length].nodeType}catch(fa){H={apply:E.length?function(a,b){G.apply(a,I.call(b))}:function(a,b){var c=a.length,d=0;while(a[c++]=b[d++]);a.length=c-1}}}function ga(a,b,d,e){var f,h,j,k,l,o,r,s,w,x;if((b?b.ownerDocument||b:v)!==n&&m(b),b=b||n,d=d||[],k=b.nodeType,"string"!=typeof a||!a||1!==k&&9!==k&&11!==k)return d;if(!e&&p){if(11!==k&&(f=_.exec(a)))if(j=f[1]){if(9===k){if(h=b.getElementById(j),!h||!h.parentNode)return d;if(h.id===j)return d.push(h),d}else if(b.ownerDocument&&(h=b.ownerDocument.getElementById(j))&&t(b,h)&&h.id===j)return d.push(h),d}else{if(f[2])return H.apply(d,b.getElementsByTagName(a)),d;if((j=f[3])&&c.getElementsByClassName)return H.apply(d,b.getElementsByClassName(j)),d}if(c.qsa&&(!q||!q.test(a))){if(s=r=u,w=b,x=1!==k&&a,1===k&&"object"!==b.nodeName.toLowerCase()){o=g(a),(r=b.getAttribute("id"))?s=r.replace(ba,"\\$&"):b.setAttribute("id",s),s="[id='"+s+"'] ",l=o.length;while(l--)o[l]=s+ra(o[l]);w=aa.test(a)&&pa(b.parentNode)||b,x=o.join(",")}if(x)try{return H.apply(d,w.querySelectorAll(x)),d}catch(y){}finally{r||b.removeAttribute("id")}}}return i(a.replace(R,"$1"),b,d,e)}function ha(){var a=[];function b(c,e){return a.push(c+" ")>d.cacheLength&&delete b[a.shift()],b[c+" "]=e}return b}function ia(a){return a[u]=!0,a}function ja(a){var b=n.createElement("div");try{return!!a(b)}catch(c){return!1}finally{b.parentNode&&b.parentNode.removeChild(b),b=null}}function ka(a,b){var c=a.split("|"),e=a.length;while(e--)d.attrHandle[c[e]]=b}function la(a,b){var c=b&&a,d=c&&1===a.nodeType&&1===b.nodeType&&(~b.sourceIndex||C)-(~a.sourceIndex||C);if(d)return d;if(c)while(c=c.nextSibling)if(c===b)return-1;return a?1:-1}function ma(a){return function(b){var c=b.nodeName.toLowerCase();return"input"===c&&b.type===a}}function na(a){return function(b){var c=b.nodeName.toLowerCase();return("input"===c||"button"===c)&&b.type===a}}function oa(a){return ia(function(b){return b=+b,ia(function(c,d){var e,f=a([],c.length,b),g=f.length;while(g--)c[e=f[g]]&&(c[e]=!(d[e]=c[e]))})})}function pa(a){return a&&"undefined"!=typeof a.getElementsByTagName&&a}c=ga.support={},f=ga.isXML=function(a){var b=a&&(a.ownerDocument||a).documentElement;return b?"HTML"!==b.nodeName:!1},m=ga.setDocument=function(a){var b,e,g=a?a.ownerDocument||a:v;return g!==n&&9===g.nodeType&&g.documentElement?(n=g,o=g.documentElement,e=g.defaultView,e&&e!==e.top&&(e.addEventListener?e.addEventListener("unload",ea,!1):e.attachEvent&&e.attachEvent("onunload",ea)),p=!f(g),c.attributes=ja(function(a){return a.className="i",!a.getAttribute("className")}),c.getElementsByTagName=ja(function(a){return a.appendChild(g.createComment("")),!a.getElementsByTagName("*").length}),c.getElementsByClassName=$.test(g.getElementsByClassName),c.getById=ja(function(a){return o.appendChild(a).id=u,!g.getElementsByName||!g.getElementsByName(u).length}),c.getById?(d.find.ID=function(a,b){if("undefined"!=typeof b.getElementById&&p){var c=b.getElementById(a);return c&&c.parentNode?[c]:[]}},d.filter.ID=function(a){var b=a.replace(ca,da);return function(a){return a.getAttribute("id")===b}}):(delete d.find.ID,d.filter.ID=function(a){var b=a.replace(ca,da);return function(a){var c="undefined"!=typeof a.getAttributeNode&&a.getAttributeNode("id");return c&&c.value===b}}),d.find.TAG=c.getElementsByTagName?function(a,b){return"undefined"!=typeof b.getElementsByTagName?b.getElementsByTagName(a):c.qsa?b.querySelectorAll(a):void 0}:function(a,b){var c,d=[],e=0,f=b.getElementsByTagName(a);if("*"===a){while(c=f[e++])1===c.nodeType&&d.push(c);return d}return f},d.find.CLASS=c.getElementsByClassName&&function(a,b){return p?b.getElementsByClassName(a):void 0},r=[],q=[],(c.qsa=$.test(g.querySelectorAll))&&(ja(function(a){o.appendChild(a).innerHTML="<a id='"+u+"'></a><select id='"+u+"-\f]' msallowcapture=''><option selected=''></option></select>",a.querySelectorAll("[msallowcapture^='']").length&&q.push("[*^$]="+L+"*(?:''|\"\")"),a.querySelectorAll("[selected]").length||q.push("\\["+L+"*(?:value|"+K+")"),a.querySelectorAll("[id~="+u+"-]").length||q.push("~="),a.querySelectorAll(":checked").length||q.push(":checked"),a.querySelectorAll("a#"+u+"+*").length||q.push(".#.+[+~]")}),ja(function(a){var b=g.createElement("input");b.setAttribute("type","hidden"),a.appendChild(b).setAttribute("name","D"),a.querySelectorAll("[name=d]").length&&q.push("name"+L+"*[*^$|!~]?="),a.querySelectorAll(":enabled").length||q.push(":enabled",":disabled"),a.querySelectorAll("*,:x"),q.push(",.*:")})),(c.matchesSelector=$.test(s=o.matches||o.webkitMatchesSelector||o.mozMatchesSelector||o.oMatchesSelector||o.msMatchesSelector))&&ja(function(a){c.disconnectedMatch=s.call(a,"div"),s.call(a,"[s!='']:x"),r.push("!=",P)}),q=q.length&&new RegExp(q.join("|")),r=r.length&&new RegExp(r.join("|")),b=$.test(o.compareDocumentPosition),t=b||$.test(o.contains)?function(a,b){var c=9===a.nodeType?a.documentElement:a,d=b&&b.parentNode;return a===d||!(!d||1!==d.nodeType||!(c.contains?c.contains(d):a.compareDocumentPosition&&16&a.compareDocumentPosition(d)))}:function(a,b){if(b)while(b=b.parentNode)if(b===a)return!0;return!1},B=b?function(a,b){if(a===b)return l=!0,0;var d=!a.compareDocumentPosition-!b.compareDocumentPosition;return d?d:(d=(a.ownerDocument||a)===(b.ownerDocument||b)?a.compareDocumentPosition(b):1,1&d||!c.sortDetached&&b.compareDocumentPosition(a)===d?a===g||a.ownerDocument===v&&t(v,a)?-1:b===g||b.ownerDocument===v&&t(v,b)?1:k?J(k,a)-J(k,b):0:4&d?-1:1)}:function(a,b){if(a===b)return l=!0,0;var c,d=0,e=a.parentNode,f=b.parentNode,h=[a],i=[b];if(!e||!f)return a===g?-1:b===g?1:e?-1:f?1:k?J(k,a)-J(k,b):0;if(e===f)return la(a,b);c=a;while(c=c.parentNode)h.unshift(c);c=b;while(c=c.parentNode)i.unshift(c);while(h[d]===i[d])d++;return d?la(h[d],i[d]):h[d]===v?-1:i[d]===v?1:0},g):n},ga.matches=function(a,b){return ga(a,null,null,b)},ga.matchesSelector=function(a,b){if((a.ownerDocument||a)!==n&&m(a),b=b.replace(U,"='$1']"),!(!c.matchesSelector||!p||r&&r.test(b)||q&&q.test(b)))try{var d=s.call(a,b);if(d||c.disconnectedMatch||a.document&&11!==a.document.nodeType)return d}catch(e){}return ga(b,n,null,[a]).length>0},ga.contains=function(a,b){return(a.ownerDocument||a)!==n&&m(a),t(a,b)},ga.attr=function(a,b){(a.ownerDocument||a)!==n&&m(a);var e=d.attrHandle[b.toLowerCase()],f=e&&D.call(d.attrHandle,b.toLowerCase())?e(a,b,!p):void 0;return void 0!==f?f:c.attributes||!p?a.getAttribute(b):(f=a.getAttributeNode(b))&&f.specified?f.value:null},ga.error=function(a){throw new Error("Syntax error, unrecognized expression: "+a)},ga.uniqueSort=function(a){var b,d=[],e=0,f=0;if(l=!c.detectDuplicates,k=!c.sortStable&&a.slice(0),a.sort(B),l){while(b=a[f++])b===a[f]&&(e=d.push(f));while(e--)a.splice(d[e],1)}return k=null,a},e=ga.getText=function(a){var b,c="",d=0,f=a.nodeType;if(f){if(1===f||9===f||11===f){if("string"==typeof a.textContent)return a.textContent;for(a=a.firstChild;a;a=a.nextSibling)c+=e(a)}else if(3===f||4===f)return a.nodeValue}else while(b=a[d++])c+=e(b);return c},d=ga.selectors={cacheLength:50,createPseudo:ia,match:X,attrHandle:{},find:{},relative:{">":{dir:"parentNode",first:!0}," ":{dir:"parentNode"},"+":{dir:"previousSibling",first:!0},"~":{dir:"previousSibling"}},preFilter:{ATTR:function(a){return a[1]=a[1].replace(ca,da),a[3]=(a[3]||a[4]||a[5]||"").replace(ca,da),"~="===a[2]&&(a[3]=" "+a[3]+" "),a.slice(0,4)},CHILD:function(a){return a[1]=a[1].toLowerCase(),"nth"===a[1].slice(0,3)?(a[3]||ga.error(a[0]),a[4]=+(a[4]?a[5]+(a[6]||1):2*("even"===a[3]||"odd"===a[3])),a[5]=+(a[7]+a[8]||"odd"===a[3])):a[3]&&ga.error(a[0]),a},PSEUDO:function(a){var b,c=!a[6]&&a[2];return X.CHILD.test(a[0])?null:(a[3]?a[2]=a[4]||a[5]||"":c&&V.test(c)&&(b=g(c,!0))&&(b=c.indexOf(")",c.length-b)-c.length)&&(a[0]=a[0].slice(0,b),a[2]=c.slice(0,b)),a.slice(0,3))}},filter:{TAG:function(a){var b=a.replace(ca,da).toLowerCase();return"*"===a?function(){return!0}:function(a){return a.nodeName&&a.nodeName.toLowerCase()===b}},CLASS:function(a){var b=y[a+" "];return b||(b=new RegExp("(^|"+L+")"+a+"("+L+"|$)"))&&y(a,function(a){return b.test("string"==typeof a.className&&a.className||"undefined"!=typeof a.getAttribute&&a.getAttribute("class")||"")})},ATTR:function(a,b,c){return function(d){var e=ga.attr(d,a);return null==e?"!="===b:b?(e+="","="===b?e===c:"!="===b?e!==c:"^="===b?c&&0===e.indexOf(c):"*="===b?c&&e.indexOf(c)>-1:"$="===b?c&&e.slice(-c.length)===c:"~="===b?(" "+e.replace(Q," ")+" ").indexOf(c)>-1:"|="===b?e===c||e.slice(0,c.length+1)===c+"-":!1):!0}},CHILD:function(a,b,c,d,e){var f="nth"!==a.slice(0,3),g="last"!==a.slice(-4),h="of-type"===b;return 1===d&&0===e?function(a){return!!a.parentNode}:function(b,c,i){var j,k,l,m,n,o,p=f!==g?"nextSibling":"previousSibling",q=b.parentNode,r=h&&b.nodeName.toLowerCase(),s=!i&&!h;if(q){if(f){while(p){l=b;while(l=l[p])if(h?l.nodeName.toLowerCase()===r:1===l.nodeType)return!1;o=p="only"===a&&!o&&"nextSibling"}return!0}if(o=[g?q.firstChild:q.lastChild],g&&s){k=q[u]||(q[u]={}),j=k[a]||[],n=j[0]===w&&j[1],m=j[0]===w&&j[2],l=n&&q.childNodes[n];while(l=++n&&l&&l[p]||(m=n=0)||o.pop())if(1===l.nodeType&&++m&&l===b){k[a]=[w,n,m];break}}else if(s&&(j=(b[u]||(b[u]={}))[a])&&j[0]===w)m=j[1];else while(l=++n&&l&&l[p]||(m=n=0)||o.pop())if((h?l.nodeName.toLowerCase()===r:1===l.nodeType)&&++m&&(s&&((l[u]||(l[u]={}))[a]=[w,m]),l===b))break;return m-=e,m===d||m%d===0&&m/d>=0}}},PSEUDO:function(a,b){var c,e=d.pseudos[a]||d.setFilters[a.toLowerCase()]||ga.error("unsupported pseudo: "+a);return e[u]?e(b):e.length>1?(c=[a,a,"",b],d.setFilters.hasOwnProperty(a.toLowerCase())?ia(function(a,c){var d,f=e(a,b),g=f.length;while(g--)d=J(a,f[g]),a[d]=!(c[d]=f[g])}):function(a){return e(a,0,c)}):e}},pseudos:{not:ia(function(a){var b=[],c=[],d=h(a.replace(R,"$1"));return d[u]?ia(function(a,b,c,e){var f,g=d(a,null,e,[]),h=a.length;while(h--)(f=g[h])&&(a[h]=!(b[h]=f))}):function(a,e,f){return b[0]=a,d(b,null,f,c),b[0]=null,!c.pop()}}),has:ia(function(a){return function(b){return ga(a,b).length>0}}),contains:ia(function(a){return a=a.replace(ca,da),function(b){return(b.textContent||b.innerText||e(b)).indexOf(a)>-1}}),lang:ia(function(a){return W.test(a||"")||ga.error("unsupported lang: "+a),a=a.replace(ca,da).toLowerCase(),function(b){var c;do if(c=p?b.lang:b.getAttribute("xml:lang")||b.getAttribute("lang"))return c=c.toLowerCase(),c===a||0===c.indexOf(a+"-");while((b=b.parentNode)&&1===b.nodeType);return!1}}),target:function(b){var c=a.location&&a.location.hash;return c&&c.slice(1)===b.id},root:function(a){return a===o},focus:function(a){return a===n.activeElement&&(!n.hasFocus||n.hasFocus())&&!!(a.type||a.href||~a.tabIndex)},enabled:function(a){return a.disabled===!1},disabled:function(a){return a.disabled===!0},checked:function(a){var b=a.nodeName.toLowerCase();return"input"===b&&!!a.checked||"option"===b&&!!a.selected},selected:function(a){return a.parentNode&&a.parentNode.selectedIndex,a.selected===!0},empty:function(a){for(a=a.firstChild;a;a=a.nextSibling)if(a.nodeType<6)return!1;return!0},parent:function(a){return!d.pseudos.empty(a)},header:function(a){return Z.test(a.nodeName)},input:function(a){return Y.test(a.nodeName)},button:function(a){var b=a.nodeName.toLowerCase();return"input"===b&&"button"===a.type||"button"===b},text:function(a){var b;return"input"===a.nodeName.toLowerCase()&&"text"===a.type&&(null==(b=a.getAttribute("type"))||"text"===b.toLowerCase())},first:oa(function(){return[0]}),last:oa(function(a,b){return[b-1]}),eq:oa(function(a,b,c){return[0>c?c+b:c]}),even:oa(function(a,b){for(var c=0;b>c;c+=2)a.push(c);return a}),odd:oa(function(a,b){for(var c=1;b>c;c+=2)a.push(c);return a}),lt:oa(function(a,b,c){for(var d=0>c?c+b:c;--d>=0;)a.push(d);return a}),gt:oa(function(a,b,c){for(var d=0>c?c+b:c;++d<b;)a.push(d);return a})}},d.pseudos.nth=d.pseudos.eq;for(b in{radio:!0,checkbox:!0,file:!0,password:!0,image:!0})d.pseudos[b]=ma(b);for(b in{submit:!0,reset:!0})d.pseudos[b]=na(b);function qa(){}qa.prototype=d.filters=d.pseudos,d.setFilters=new qa,g=ga.tokenize=function(a,b){var c,e,f,g,h,i,j,k=z[a+" "];if(k)return b?0:k.slice(0);h=a,i=[],j=d.preFilter;while(h){(!c||(e=S.exec(h)))&&(e&&(h=h.slice(e[0].length)||h),i.push(f=[])),c=!1,(e=T.exec(h))&&(c=e.shift(),f.push({value:c,type:e[0].replace(R," ")}),h=h.slice(c.length));for(g in d.filter)!(e=X[g].exec(h))||j[g]&&!(e=j[g](e))||(c=e.shift(),f.push({value:c,type:g,matches:e}),h=h.slice(c.length));if(!c)break}return b?h.length:h?ga.error(a):z(a,i).slice(0)};function ra(a){for(var b=0,c=a.length,d="";c>b;b++)d+=a[b].value;return d}function sa(a,b,c){var d=b.dir,e=c&&"parentNode"===d,f=x++;return b.first?function(b,c,f){while(b=b[d])if(1===b.nodeType||e)return a(b,c,f)}:function(b,c,g){var h,i,j=[w,f];if(g){while(b=b[d])if((1===b.nodeType||e)&&a(b,c,g))return!0}else while(b=b[d])if(1===b.nodeType||e){if(i=b[u]||(b[u]={}),(h=i[d])&&h[0]===w&&h[1]===f)return j[2]=h[2];if(i[d]=j,j[2]=a(b,c,g))return!0}}}function ta(a){return a.length>1?function(b,c,d){var e=a.length;while(e--)if(!a[e](b,c,d))return!1;return!0}:a[0]}function ua(a,b,c){for(var d=0,e=b.length;e>d;d++)ga(a,b[d],c);return c}function va(a,b,c,d,e){for(var f,g=[],h=0,i=a.length,j=null!=b;i>h;h++)(f=a[h])&&(!c||c(f,d,e))&&(g.push(f),j&&b.push(h));return g}function wa(a,b,c,d,e,f){return d&&!d[u]&&(d=wa(d)),e&&!e[u]&&(e=wa(e,f)),ia(function(f,g,h,i){var j,k,l,m=[],n=[],o=g.length,p=f||ua(b||"*",h.nodeType?[h]:h,[]),q=!a||!f&&b?p:va(p,m,a,h,i),r=c?e||(f?a:o||d)?[]:g:q;if(c&&c(q,r,h,i),d){j=va(r,n),d(j,[],h,i),k=j.length;while(k--)(l=j[k])&&(r[n[k]]=!(q[n[k]]=l))}if(f){if(e||a){if(e){j=[],k=r.length;while(k--)(l=r[k])&&j.push(q[k]=l);e(null,r=[],j,i)}k=r.length;while(k--)(l=r[k])&&(j=e?J(f,l):m[k])>-1&&(f[j]=!(g[j]=l))}}else r=va(r===g?r.splice(o,r.length):r),e?e(null,g,r,i):H.apply(g,r)})}function xa(a){for(var b,c,e,f=a.length,g=d.relative[a[0].type],h=g||d.relative[" "],i=g?1:0,k=sa(function(a){return a===b},h,!0),l=sa(function(a){return J(b,a)>-1},h,!0),m=[function(a,c,d){var e=!g&&(d||c!==j)||((b=c).nodeType?k(a,c,d):l(a,c,d));return b=null,e}];f>i;i++)if(c=d.relative[a[i].type])m=[sa(ta(m),c)];else{if(c=d.filter[a[i].type].apply(null,a[i].matches),c[u]){for(e=++i;f>e;e++)if(d.relative[a[e].type])break;return wa(i>1&&ta(m),i>1&&ra(a.slice(0,i-1).concat({value:" "===a[i-2].type?"*":""})).replace(R,"$1"),c,e>i&&xa(a.slice(i,e)),f>e&&xa(a=a.slice(e)),f>e&&ra(a))}m.push(c)}return ta(m)}function ya(a,b){var c=b.length>0,e=a.length>0,f=function(f,g,h,i,k){var l,m,o,p=0,q="0",r=f&&[],s=[],t=j,u=f||e&&d.find.TAG("*",k),v=w+=null==t?1:Math.random()||.1,x=u.length;for(k&&(j=g!==n&&g);q!==x&&null!=(l=u[q]);q++){if(e&&l){m=0;while(o=a[m++])if(o(l,g,h)){i.push(l);break}k&&(w=v)}c&&((l=!o&&l)&&p--,f&&r.push(l))}if(p+=q,c&&q!==p){m=0;while(o=b[m++])o(r,s,g,h);if(f){if(p>0)while(q--)r[q]||s[q]||(s[q]=F.call(i));s=va(s)}H.apply(i,s),k&&!f&&s.length>0&&p+b.length>1&&ga.uniqueSort(i)}return k&&(w=v,j=t),r};return c?ia(f):f}return h=ga.compile=function(a,b){var c,d=[],e=[],f=A[a+" "];if(!f){b||(b=g(a)),c=b.length;while(c--)f=xa(b[c]),f[u]?d.push(f):e.push(f);f=A(a,ya(e,d)),f.selector=a}return f},i=ga.select=function(a,b,e,f){var i,j,k,l,m,n="function"==typeof a&&a,o=!f&&g(a=n.selector||a);if(e=e||[],1===o.length){if(j=o[0]=o[0].slice(0),j.length>2&&"ID"===(k=j[0]).type&&c.getById&&9===b.nodeType&&p&&d.relative[j[1].type]){if(b=(d.find.ID(k.matches[0].replace(ca,da),b)||[])[0],!b)return e;n&&(b=b.parentNode),a=a.slice(j.shift().value.length)}i=X.needsContext.test(a)?0:j.length;while(i--){if(k=j[i],d.relative[l=k.type])break;if((m=d.find[l])&&(f=m(k.matches[0].replace(ca,da),aa.test(j[0].type)&&pa(b.parentNode)||b))){if(j.splice(i,1),a=f.length&&ra(j),!a)return H.apply(e,f),e;break}}}return(n||h(a,o))(f,b,!p,e,aa.test(a)&&pa(b.parentNode)||b),e},c.sortStable=u.split("").sort(B).join("")===u,c.detectDuplicates=!!l,m(),c.sortDetached=ja(function(a){return 1&a.compareDocumentPosition(n.createElement("div"))}),ja(function(a){return a.innerHTML="<a href='#'></a>","#"===a.firstChild.getAttribute("href")})||ka("type|href|height|width",function(a,b,c){return c?void 0:a.getAttribute(b,"type"===b.toLowerCase()?1:2)}),c.attributes&&ja(function(a){return a.innerHTML="<input/>",a.firstChild.setAttribute("value",""),""===a.firstChild.getAttribute("value")})||ka("value",function(a,b,c){return c||"input"!==a.nodeName.toLowerCase()?void 0:a.defaultValue}),ja(function(a){return null==a.getAttribute("disabled")})||ka(K,function(a,b,c){var d;return c?void 0:a[b]===!0?b.toLowerCase():(d=a.getAttributeNode(b))&&d.specified?d.value:null}),ga}(a);m.find=s,m.expr=s.selectors,m.expr[":"]=m.expr.pseudos,m.unique=s.uniqueSort,m.text=s.getText,m.isXMLDoc=s.isXML,m.contains=s.contains;var t=m.expr.match.needsContext,u=/^<(\w+)\s*\/?>(?:<\/\1>|)$/,v=/^.[^:#\[\.,]*$/;function w(a,b,c){if(m.isFunction(b))return m.grep(a,function(a,d){return!!b.call(a,d,a)!==c});if(b.nodeType)return m.grep(a,function(a){return a===b!==c});if("string"==typeof b){if(v.test(b))return m.filter(b,a,c);b=m.filter(b,a)}return m.grep(a,function(a){return m.inArray(a,b)>=0!==c})}m.filter=function(a,b,c){var d=b[0];return c&&(a=":not("+a+")"),1===b.length&&1===d.nodeType?m.find.matchesSelector(d,a)?[d]:[]:m.find.matches(a,m.grep(b,function(a){return 1===a.nodeType}))},m.fn.extend({find:function(a){var b,c=[],d=this,e=d.length;if("string"!=typeof a)return this.pushStack(m(a).filter(function(){for(b=0;e>b;b++)if(m.contains(d[b],this))return!0}));for(b=0;e>b;b++)m.find(a,d[b],c);return c=this.pushStack(e>1?m.unique(c):c),c.selector=this.selector?this.selector+" "+a:a,c},filter:function(a){return this.pushStack(w(this,a||[],!1))},not:function(a){return this.pushStack(w(this,a||[],!0))},is:function(a){return!!w(this,"string"==typeof a&&t.test(a)?m(a):a||[],!1).length}});var x,y=a.document,z=/^(?:\s*(<[\w\W]+>)[^>]*|#([\w-]*))$/,A=m.fn.init=function(a,b){var c,d;if(!a)return this;if("string"==typeof a){if(c="<"===a.charAt(0)&&">"===a.charAt(a.length-1)&&a.length>=3?[null,a,null]:z.exec(a),!c||!c[1]&&b)return!b||b.jquery?(b||x).find(a):this.constructor(b).find(a);if(c[1]){if(b=b instanceof m?b[0]:b,m.merge(this,m.parseHTML(c[1],b&&b.nodeType?b.ownerDocument||b:y,!0)),u.test(c[1])&&m.isPlainObject(b))for(c in b)m.isFunction(this[c])?this[c](b[c]):this.attr(c,b[c]);return this}if(d=y.getElementById(c[2]),d&&d.parentNode){if(d.id!==c[2])return x.find(a);this.length=1,this[0]=d}return this.context=y,this.selector=a,this}return a.nodeType?(this.context=this[0]=a,this.length=1,this):m.isFunction(a)?"undefined"!=typeof x.ready?x.ready(a):a(m):(void 0!==a.selector&&(this.selector=a.selector,this.context=a.context),m.makeArray(a,this))};A.prototype=m.fn,x=m(y);var B=/^(?:parents|prev(?:Until|All))/,C={children:!0,contents:!0,next:!0,prev:!0};m.extend({dir:function(a,b,c){var d=[],e=a[b];while(e&&9!==e.nodeType&&(void 0===c||1!==e.nodeType||!m(e).is(c)))1===e.nodeType&&d.push(e),e=e[b];return d},sibling:function(a,b){for(var c=[];a;a=a.nextSibling)1===a.nodeType&&a!==b&&c.push(a);return c}}),m.fn.extend({has:function(a){var b,c=m(a,this),d=c.length;return this.filter(function(){for(b=0;d>b;b++)if(m.contains(this,c[b]))return!0})},closest:function(a,b){for(var c,d=0,e=this.length,f=[],g=t.test(a)||"string"!=typeof a?m(a,b||this.context):0;e>d;d++)for(c=this[d];c&&c!==b;c=c.parentNode)if(c.nodeType<11&&(g?g.index(c)>-1:1===c.nodeType&&m.find.matchesSelector(c,a))){f.push(c);break}return this.pushStack(f.length>1?m.unique(f):f)},index:function(a){return a?"string"==typeof a?m.inArray(this[0],m(a)):m.inArray(a.jquery?a[0]:a,this):this[0]&&this[0].parentNode?this.first().prevAll().length:-1},add:function(a,b){return this.pushStack(m.unique(m.merge(this.get(),m(a,b))))},addBack:function(a){return this.add(null==a?this.prevObject:this.prevObject.filter(a))}});function D(a,b){do a=a[b];while(a&&1!==a.nodeType);return a}m.each({parent:function(a){var b=a.parentNode;return b&&11!==b.nodeType?b:null},parents:function(a){return m.dir(a,"parentNode")},parentsUntil:function(a,b,c){return m.dir(a,"parentNode",c)},next:function(a){return D(a,"nextSibling")},prev:function(a){return D(a,"previousSibling")},nextAll:function(a){return m.dir(a,"nextSibling")},prevAll:function(a){return m.dir(a,"previousSibling")},nextUntil:function(a,b,c){return m.dir(a,"nextSibling",c)},prevUntil:function(a,b,c){return m.dir(a,"previousSibling",c)},siblings:function(a){return m.sibling((a.parentNode||{}).firstChild,a)},children:function(a){return m.sibling(a.firstChild)},contents:function(a){return m.nodeName(a,"iframe")?a.contentDocument||a.contentWindow.document:m.merge([],a.childNodes)}},function(a,b){m.fn[a]=function(c,d){var e=m.map(this,b,c);return"Until"!==a.slice(-5)&&(d=c),d&&"string"==typeof d&&(e=m.filter(d,e)),this.length>1&&(C[a]||(e=m.unique(e)),B.test(a)&&(e=e.reverse())),this.pushStack(e)}});var E=/\S+/g,F={};function G(a){var b=F[a]={};return m.each(a.match(E)||[],function(a,c){b[c]=!0}),b}m.Callbacks=function(a){a="string"==typeof a?F[a]||G(a):m.extend({},a);var b,c,d,e,f,g,h=[],i=!a.once&&[],j=function(l){for(c=a.memory&&l,d=!0,f=g||0,g=0,e=h.length,b=!0;h&&e>f;f++)if(h[f].apply(l[0],l[1])===!1&&a.stopOnFalse){c=!1;break}b=!1,h&&(i?i.length&&j(i.shift()):c?h=[]:k.disable())},k={add:function(){if(h){var d=h.length;!function f(b){m.each(b,function(b,c){var d=m.type(c);"function"===d?a.unique&&k.has(c)||h.push(c):c&&c.length&&"string"!==d&&f(c)})}(arguments),b?e=h.length:c&&(g=d,j(c))}return this},remove:function(){return h&&m.each(arguments,function(a,c){var d;while((d=m.inArray(c,h,d))>-1)h.splice(d,1),b&&(e>=d&&e--,f>=d&&f--)}),this},has:function(a){return a?m.inArray(a,h)>-1:!(!h||!h.length)},empty:function(){return h=[],e=0,this},disable:function(){return h=i=c=void 0,this},disabled:function(){return!h},lock:function(){return i=void 0,c||k.disable(),this},locked:function(){return!i},fireWith:function(a,c){return!h||d&&!i||(c=c||[],c=[a,c.slice?c.slice():c],b?i.push(c):j(c)),this},fire:function(){return k.fireWith(this,arguments),this},fired:function(){return!!d}};return k},m.extend({Deferred:function(a){var b=[["resolve","done",m.Callbacks("once memory"),"resolved"],["reject","fail",m.Callbacks("once memory"),"rejected"],["notify","progress",m.Callbacks("memory")]],c="pending",d={state:function(){return c},always:function(){return e.done(arguments).fail(arguments),this},then:function(){var a=arguments;return m.Deferred(function(c){m.each(b,function(b,f){var g=m.isFunction(a[b])&&a[b];e[f[1]](function(){var a=g&&g.apply(this,arguments);a&&m.isFunction(a.promise)?a.promise().done(c.resolve).fail(c.reject).progress(c.notify):c[f[0]+"With"](this===d?c.promise():this,g?[a]:arguments)})}),a=null}).promise()},promise:function(a){return null!=a?m.extend(a,d):d}},e={};return d.pipe=d.then,m.each(b,function(a,f){var g=f[2],h=f[3];d[f[1]]=g.add,h&&g.add(function(){c=h},b[1^a][2].disable,b[2][2].lock),e[f[0]]=function(){return e[f[0]+"With"](this===e?d:this,arguments),this},e[f[0]+"With"]=g.fireWith}),d.promise(e),a&&a.call(e,e),e},when:function(a){var b=0,c=d.call(arguments),e=c.length,f=1!==e||a&&m.isFunction(a.promise)?e:0,g=1===f?a:m.Deferred(),h=function(a,b,c){return function(e){b[a]=this,c[a]=arguments.length>1?d.call(arguments):e,c===i?g.notifyWith(b,c):--f||g.resolveWith(b,c)}},i,j,k;if(e>1)for(i=new Array(e),j=new Array(e),k=new Array(e);e>b;b++)c[b]&&m.isFunction(c[b].promise)?c[b].promise().done(h(b,k,c)).fail(g.reject).progress(h(b,j,i)):--f;return f||g.resolveWith(k,c),g.promise()}});var H;m.fn.ready=function(a){return m.ready.promise().done(a),this},m.extend({isReady:!1,readyWait:1,holdReady:function(a){a?m.readyWait++:m.ready(!0)},ready:function(a){if(a===!0?!--m.readyWait:!m.isReady){if(!y.body)return setTimeout(m.ready);m.isReady=!0,a!==!0&&--m.readyWait>0||(H.resolveWith(y,[m]),m.fn.triggerHandler&&(m(y).triggerHandler("ready"),m(y).off("ready")))}}});function I(){y.addEventListener?(y.removeEventListener("DOMContentLoaded",J,!1),a.removeEventListener("load",J,!1)):(y.detachEvent("onreadystatechange",J),a.detachEvent("onload",J))}function J(){(y.addEventListener||"load"===event.type||"complete"===y.readyState)&&(I(),m.ready())}m.ready.promise=function(b){if(!H)if(H=m.Deferred(),"complete"===y.readyState)setTimeout(m.ready);else if(y.addEventListener)y.addEventListener("DOMContentLoaded",J,!1),a.addEventListener("load",J,!1);else{y.attachEvent("onreadystatechange",J),a.attachEvent("onload",J);var c=!1;try{c=null==a.frameElement&&y.documentElement}catch(d){}c&&c.doScroll&&!function e(){if(!m.isReady){try{c.doScroll("left")}catch(a){return setTimeout(e,50)}I(),m.ready()}}()}return H.promise(b)};var K="undefined",L;for(L in m(k))break;k.ownLast="0"!==L,k.inlineBlockNeedsLayout=!1,m(function(){var a,b,c,d;c=y.getElementsByTagName("body")[0],c&&c.style&&(b=y.createElement("div"),d=y.createElement("div"),d.style.cssText="position:absolute;border:0;width:0;height:0;top:0;left:-9999px",c.appendChild(d).appendChild(b),typeof b.style.zoom!==K&&(b.style.cssText="display:inline;margin:0;border:0;padding:1px;width:1px;zoom:1",k.inlineBlockNeedsLayout=a=3===b.offsetWidth,a&&(c.style.zoom=1)),c.removeChild(d))}),function(){var a=y.createElement("div");if(null==k.deleteExpando){k.deleteExpando=!0;try{delete a.test}catch(b){k.deleteExpando=!1}}a=null}(),m.acceptData=function(a){var b=m.noData[(a.nodeName+" ").toLowerCase()],c=+a.nodeType||1;return 1!==c&&9!==c?!1:!b||b!==!0&&a.getAttribute("classid")===b};var M=/^(?:\{[\w\W]*\}|\[[\w\W]*\])$/,N=/([A-Z])/g;function O(a,b,c){if(void 0===c&&1===a.nodeType){var d="data-"+b.replace(N,"-$1").toLowerCase();if(c=a.getAttribute(d),"string"==typeof c){try{c="true"===c?!0:"false"===c?!1:"null"===c?null:+c+""===c?+c:M.test(c)?m.parseJSON(c):c}catch(e){}m.data(a,b,c)}else c=void 0}return c}function P(a){var b;for(b in a)if(("data"!==b||!m.isEmptyObject(a[b]))&&"toJSON"!==b)return!1;

return!0}function Q(a,b,d,e){if(m.acceptData(a)){var f,g,h=m.expando,i=a.nodeType,j=i?m.cache:a,k=i?a[h]:a[h]&&h;if(k&&j[k]&&(e||j[k].data)||void 0!==d||"string"!=typeof b)return k||(k=i?a[h]=c.pop()||m.guid++:h),j[k]||(j[k]=i?{}:{toJSON:m.noop}),("object"==typeof b||"function"==typeof b)&&(e?j[k]=m.extend(j[k],b):j[k].data=m.extend(j[k].data,b)),g=j[k],e||(g.data||(g.data={}),g=g.data),void 0!==d&&(g[m.camelCase(b)]=d),"string"==typeof b?(f=g[b],null==f&&(f=g[m.camelCase(b)])):f=g,f}}function R(a,b,c){if(m.acceptData(a)){var d,e,f=a.nodeType,g=f?m.cache:a,h=f?a[m.expando]:m.expando;if(g[h]){if(b&&(d=c?g[h]:g[h].data)){m.isArray(b)?b=b.concat(m.map(b,m.camelCase)):b in d?b=[b]:(b=m.camelCase(b),b=b in d?[b]:b.split(" ")),e=b.length;while(e--)delete d[b[e]];if(c?!P(d):!m.isEmptyObject(d))return}(c||(delete g[h].data,P(g[h])))&&(f?m.cleanData([a],!0):k.deleteExpando||g!=g.window?delete g[h]:g[h]=null)}}}m.extend({cache:{},noData:{"applet ":!0,"embed ":!0,"object ":"clsid:D27CDB6E-AE6D-11cf-96B8-444553540000"},hasData:function(a){return a=a.nodeType?m.cache[a[m.expando]]:a[m.expando],!!a&&!P(a)},data:function(a,b,c){return Q(a,b,c)},removeData:function(a,b){return R(a,b)},_data:function(a,b,c){return Q(a,b,c,!0)},_removeData:function(a,b){return R(a,b,!0)}}),m.fn.extend({data:function(a,b){var c,d,e,f=this[0],g=f&&f.attributes;if(void 0===a){if(this.length&&(e=m.data(f),1===f.nodeType&&!m._data(f,"parsedAttrs"))){c=g.length;while(c--)g[c]&&(d=g[c].name,0===d.indexOf("data-")&&(d=m.camelCase(d.slice(5)),O(f,d,e[d])));m._data(f,"parsedAttrs",!0)}return e}return"object"==typeof a?this.each(function(){m.data(this,a)}):arguments.length>1?this.each(function(){m.data(this,a,b)}):f?O(f,a,m.data(f,a)):void 0},removeData:function(a){return this.each(function(){m.removeData(this,a)})}}),m.extend({queue:function(a,b,c){var d;return a?(b=(b||"fx")+"queue",d=m._data(a,b),c&&(!d||m.isArray(c)?d=m._data(a,b,m.makeArray(c)):d.push(c)),d||[]):void 0},dequeue:function(a,b){b=b||"fx";var c=m.queue(a,b),d=c.length,e=c.shift(),f=m._queueHooks(a,b),g=function(){m.dequeue(a,b)};"inprogress"===e&&(e=c.shift(),d--),e&&("fx"===b&&c.unshift("inprogress"),delete f.stop,e.call(a,g,f)),!d&&f&&f.empty.fire()},_queueHooks:function(a,b){var c=b+"queueHooks";return m._data(a,c)||m._data(a,c,{empty:m.Callbacks("once memory").add(function(){m._removeData(a,b+"queue"),m._removeData(a,c)})})}}),m.fn.extend({queue:function(a,b){var c=2;return"string"!=typeof a&&(b=a,a="fx",c--),arguments.length<c?m.queue(this[0],a):void 0===b?this:this.each(function(){var c=m.queue(this,a,b);m._queueHooks(this,a),"fx"===a&&"inprogress"!==c[0]&&m.dequeue(this,a)})},dequeue:function(a){return this.each(function(){m.dequeue(this,a)})},clearQueue:function(a){return this.queue(a||"fx",[])},promise:function(a,b){var c,d=1,e=m.Deferred(),f=this,g=this.length,h=function(){--d||e.resolveWith(f,[f])};"string"!=typeof a&&(b=a,a=void 0),a=a||"fx";while(g--)c=m._data(f[g],a+"queueHooks"),c&&c.empty&&(d++,c.empty.add(h));return h(),e.promise(b)}});var S=/[+-]?(?:\d*\.|)\d+(?:[eE][+-]?\d+|)/.source,T=["Top","Right","Bottom","Left"],U=function(a,b){return a=b||a,"none"===m.css(a,"display")||!m.contains(a.ownerDocument,a)},V=m.access=function(a,b,c,d,e,f,g){var h=0,i=a.length,j=null==c;if("object"===m.type(c)){e=!0;for(h in c)m.access(a,b,h,c[h],!0,f,g)}else if(void 0!==d&&(e=!0,m.isFunction(d)||(g=!0),j&&(g?(b.call(a,d),b=null):(j=b,b=function(a,b,c){return j.call(m(a),c)})),b))for(;i>h;h++)b(a[h],c,g?d:d.call(a[h],h,b(a[h],c)));return e?a:j?b.call(a):i?b(a[0],c):f},W=/^(?:checkbox|radio)$/i;!function(){var a=y.createElement("input"),b=y.createElement("div"),c=y.createDocumentFragment();if(b.innerHTML="  <link/><table></table><a href='/a'>a</a><input type='checkbox'/>",k.leadingWhitespace=3===b.firstChild.nodeType,k.tbody=!b.getElementsByTagName("tbody").length,k.htmlSerialize=!!b.getElementsByTagName("link").length,k.html5Clone="<:nav></:nav>"!==y.createElement("nav").cloneNode(!0).outerHTML,a.type="checkbox",a.checked=!0,c.appendChild(a),k.appendChecked=a.checked,b.innerHTML="<textarea>x</textarea>",k.noCloneChecked=!!b.cloneNode(!0).lastChild.defaultValue,c.appendChild(b),b.innerHTML="<input type='radio' checked='checked' name='t'/>",k.checkClone=b.cloneNode(!0).cloneNode(!0).lastChild.checked,k.noCloneEvent=!0,b.attachEvent&&(b.attachEvent("onclick",function(){k.noCloneEvent=!1}),b.cloneNode(!0).click()),null==k.deleteExpando){k.deleteExpando=!0;try{delete b.test}catch(d){k.deleteExpando=!1}}}(),function(){var b,c,d=y.createElement("div");for(b in{submit:!0,change:!0,focusin:!0})c="on"+b,(k[b+"Bubbles"]=c in a)||(d.setAttribute(c,"t"),k[b+"Bubbles"]=d.attributes[c].expando===!1);d=null}();var X=/^(?:input|select|textarea)$/i,Y=/^key/,Z=/^(?:mouse|pointer|contextmenu)|click/,$=/^(?:focusinfocus|focusoutblur)$/,_=/^([^.]*)(?:\.(.+)|)$/;function aa(){return!0}function ba(){return!1}function ca(){try{return y.activeElement}catch(a){}}m.event={global:{},add:function(a,b,c,d,e){var f,g,h,i,j,k,l,n,o,p,q,r=m._data(a);if(r){c.handler&&(i=c,c=i.handler,e=i.selector),c.guid||(c.guid=m.guid++),(g=r.events)||(g=r.events={}),(k=r.handle)||(k=r.handle=function(a){return typeof m===K||a&&m.event.triggered===a.type?void 0:m.event.dispatch.apply(k.elem,arguments)},k.elem=a),b=(b||"").match(E)||[""],h=b.length;while(h--)f=_.exec(b[h])||[],o=q=f[1],p=(f[2]||"").split(".").sort(),o&&(j=m.event.special[o]||{},o=(e?j.delegateType:j.bindType)||o,j=m.event.special[o]||{},l=m.extend({type:o,origType:q,data:d,handler:c,guid:c.guid,selector:e,needsContext:e&&m.expr.match.needsContext.test(e),namespace:p.join(".")},i),(n=g[o])||(n=g[o]=[],n.delegateCount=0,j.setup&&j.setup.call(a,d,p,k)!==!1||(a.addEventListener?a.addEventListener(o,k,!1):a.attachEvent&&a.attachEvent("on"+o,k))),j.add&&(j.add.call(a,l),l.handler.guid||(l.handler.guid=c.guid)),e?n.splice(n.delegateCount++,0,l):n.push(l),m.event.global[o]=!0);a=null}},remove:function(a,b,c,d,e){var f,g,h,i,j,k,l,n,o,p,q,r=m.hasData(a)&&m._data(a);if(r&&(k=r.events)){b=(b||"").match(E)||[""],j=b.length;while(j--)if(h=_.exec(b[j])||[],o=q=h[1],p=(h[2]||"").split(".").sort(),o){l=m.event.special[o]||{},o=(d?l.delegateType:l.bindType)||o,n=k[o]||[],h=h[2]&&new RegExp("(^|\\.)"+p.join("\\.(?:.*\\.|)")+"(\\.|$)"),i=f=n.length;while(f--)g=n[f],!e&&q!==g.origType||c&&c.guid!==g.guid||h&&!h.test(g.namespace)||d&&d!==g.selector&&("**"!==d||!g.selector)||(n.splice(f,1),g.selector&&n.delegateCount--,l.remove&&l.remove.call(a,g));i&&!n.length&&(l.teardown&&l.teardown.call(a,p,r.handle)!==!1||m.removeEvent(a,o,r.handle),delete k[o])}else for(o in k)m.event.remove(a,o+b[j],c,d,!0);m.isEmptyObject(k)&&(delete r.handle,m._removeData(a,"events"))}},trigger:function(b,c,d,e){var f,g,h,i,k,l,n,o=[d||y],p=j.call(b,"type")?b.type:b,q=j.call(b,"namespace")?b.namespace.split("."):[];if(h=l=d=d||y,3!==d.nodeType&&8!==d.nodeType&&!$.test(p+m.event.triggered)&&(p.indexOf(".")>=0&&(q=p.split("."),p=q.shift(),q.sort()),g=p.indexOf(":")<0&&"on"+p,b=b[m.expando]?b:new m.Event(p,"object"==typeof b&&b),b.isTrigger=e?2:3,b.namespace=q.join("."),b.namespace_re=b.namespace?new RegExp("(^|\\.)"+q.join("\\.(?:.*\\.|)")+"(\\.|$)"):null,b.result=void 0,b.target||(b.target=d),c=null==c?[b]:m.makeArray(c,[b]),k=m.event.special[p]||{},e||!k.trigger||k.trigger.apply(d,c)!==!1)){if(!e&&!k.noBubble&&!m.isWindow(d)){for(i=k.delegateType||p,$.test(i+p)||(h=h.parentNode);h;h=h.parentNode)o.push(h),l=h;l===(d.ownerDocument||y)&&o.push(l.defaultView||l.parentWindow||a)}n=0;while((h=o[n++])&&!b.isPropagationStopped())b.type=n>1?i:k.bindType||p,f=(m._data(h,"events")||{})[b.type]&&m._data(h,"handle"),f&&f.apply(h,c),f=g&&h[g],f&&f.apply&&m.acceptData(h)&&(b.result=f.apply(h,c),b.result===!1&&b.preventDefault());if(b.type=p,!e&&!b.isDefaultPrevented()&&(!k._default||k._default.apply(o.pop(),c)===!1)&&m.acceptData(d)&&g&&d[p]&&!m.isWindow(d)){l=d[g],l&&(d[g]=null),m.event.triggered=p;try{d[p]()}catch(r){}m.event.triggered=void 0,l&&(d[g]=l)}return b.result}},dispatch:function(a){a=m.event.fix(a);var b,c,e,f,g,h=[],i=d.call(arguments),j=(m._data(this,"events")||{})[a.type]||[],k=m.event.special[a.type]||{};if(i[0]=a,a.delegateTarget=this,!k.preDispatch||k.preDispatch.call(this,a)!==!1){h=m.event.handlers.call(this,a,j),b=0;while((f=h[b++])&&!a.isPropagationStopped()){a.currentTarget=f.elem,g=0;while((e=f.handlers[g++])&&!a.isImmediatePropagationStopped())(!a.namespace_re||a.namespace_re.test(e.namespace))&&(a.handleObj=e,a.data=e.data,c=((m.event.special[e.origType]||{}).handle||e.handler).apply(f.elem,i),void 0!==c&&(a.result=c)===!1&&(a.preventDefault(),a.stopPropagation()))}return k.postDispatch&&k.postDispatch.call(this,a),a.result}},handlers:function(a,b){var c,d,e,f,g=[],h=b.delegateCount,i=a.target;if(h&&i.nodeType&&(!a.button||"click"!==a.type))for(;i!=this;i=i.parentNode||this)if(1===i.nodeType&&(i.disabled!==!0||"click"!==a.type)){for(e=[],f=0;h>f;f++)d=b[f],c=d.selector+" ",void 0===e[c]&&(e[c]=d.needsContext?m(c,this).index(i)>=0:m.find(c,this,null,[i]).length),e[c]&&e.push(d);e.length&&g.push({elem:i,handlers:e})}return h<b.length&&g.push({elem:this,handlers:b.slice(h)}),g},fix:function(a){if(a[m.expando])return a;var b,c,d,e=a.type,f=a,g=this.fixHooks[e];g||(this.fixHooks[e]=g=Z.test(e)?this.mouseHooks:Y.test(e)?this.keyHooks:{}),d=g.props?this.props.concat(g.props):this.props,a=new m.Event(f),b=d.length;while(b--)c=d[b],a[c]=f[c];return a.target||(a.target=f.srcElement||y),3===a.target.nodeType&&(a.target=a.target.parentNode),a.metaKey=!!a.metaKey,g.filter?g.filter(a,f):a},props:"altKey bubbles cancelable ctrlKey currentTarget eventPhase metaKey relatedTarget shiftKey target timeStamp view which".split(" "),fixHooks:{},keyHooks:{props:"char charCode key keyCode".split(" "),filter:function(a,b){return null==a.which&&(a.which=null!=b.charCode?b.charCode:b.keyCode),a}},mouseHooks:{props:"button buttons clientX clientY fromElement offsetX offsetY pageX pageY screenX screenY toElement".split(" "),filter:function(a,b){var c,d,e,f=b.button,g=b.fromElement;return null==a.pageX&&null!=b.clientX&&(d=a.target.ownerDocument||y,e=d.documentElement,c=d.body,a.pageX=b.clientX+(e&&e.scrollLeft||c&&c.scrollLeft||0)-(e&&e.clientLeft||c&&c.clientLeft||0),a.pageY=b.clientY+(e&&e.scrollTop||c&&c.scrollTop||0)-(e&&e.clientTop||c&&c.clientTop||0)),!a.relatedTarget&&g&&(a.relatedTarget=g===a.target?b.toElement:g),a.which||void 0===f||(a.which=1&f?1:2&f?3:4&f?2:0),a}},special:{load:{noBubble:!0},focus:{trigger:function(){if(this!==ca()&&this.focus)try{return this.focus(),!1}catch(a){}},delegateType:"focusin"},blur:{trigger:function(){return this===ca()&&this.blur?(this.blur(),!1):void 0},delegateType:"focusout"},click:{trigger:function(){return m.nodeName(this,"input")&&"checkbox"===this.type&&this.click?(this.click(),!1):void 0},_default:function(a){return m.nodeName(a.target,"a")}},beforeunload:{postDispatch:function(a){void 0!==a.result&&a.originalEvent&&(a.originalEvent.returnValue=a.result)}}},simulate:function(a,b,c,d){var e=m.extend(new m.Event,c,{type:a,isSimulated:!0,originalEvent:{}});d?m.event.trigger(e,null,b):m.event.dispatch.call(b,e),e.isDefaultPrevented()&&c.preventDefault()}},m.removeEvent=y.removeEventListener?function(a,b,c){a.removeEventListener&&a.removeEventListener(b,c,!1)}:function(a,b,c){var d="on"+b;a.detachEvent&&(typeof a[d]===K&&(a[d]=null),a.detachEvent(d,c))},m.Event=function(a,b){return this instanceof m.Event?(a&&a.type?(this.originalEvent=a,this.type=a.type,this.isDefaultPrevented=a.defaultPrevented||void 0===a.defaultPrevented&&a.returnValue===!1?aa:ba):this.type=a,b&&m.extend(this,b),this.timeStamp=a&&a.timeStamp||m.now(),void(this[m.expando]=!0)):new m.Event(a,b)},m.Event.prototype={isDefaultPrevented:ba,isPropagationStopped:ba,isImmediatePropagationStopped:ba,preventDefault:function(){var a=this.originalEvent;this.isDefaultPrevented=aa,a&&(a.preventDefault?a.preventDefault():a.returnValue=!1)},stopPropagation:function(){var a=this.originalEvent;this.isPropagationStopped=aa,a&&(a.stopPropagation&&a.stopPropagation(),a.cancelBubble=!0)},stopImmediatePropagation:function(){var a=this.originalEvent;this.isImmediatePropagationStopped=aa,a&&a.stopImmediatePropagation&&a.stopImmediatePropagation(),this.stopPropagation()}},m.each({mouseenter:"mouseover",mouseleave:"mouseout",pointerenter:"pointerover",pointerleave:"pointerout"},function(a,b){m.event.special[a]={delegateType:b,bindType:b,handle:function(a){var c,d=this,e=a.relatedTarget,f=a.handleObj;return(!e||e!==d&&!m.contains(d,e))&&(a.type=f.origType,c=f.handler.apply(this,arguments),a.type=b),c}}}),k.submitBubbles||(m.event.special.submit={setup:function(){return m.nodeName(this,"form")?!1:void m.event.add(this,"click._submit keypress._submit",function(a){var b=a.target,c=m.nodeName(b,"input")||m.nodeName(b,"button")?b.form:void 0;c&&!m._data(c,"submitBubbles")&&(m.event.add(c,"submit._submit",function(a){a._submit_bubble=!0}),m._data(c,"submitBubbles",!0))})},postDispatch:function(a){a._submit_bubble&&(delete a._submit_bubble,this.parentNode&&!a.isTrigger&&m.event.simulate("submit",this.parentNode,a,!0))},teardown:function(){return m.nodeName(this,"form")?!1:void m.event.remove(this,"._submit")}}),k.changeBubbles||(m.event.special.change={setup:function(){return X.test(this.nodeName)?(("checkbox"===this.type||"radio"===this.type)&&(m.event.add(this,"propertychange._change",function(a){"checked"===a.originalEvent.propertyName&&(this._just_changed=!0)}),m.event.add(this,"click._change",function(a){this._just_changed&&!a.isTrigger&&(this._just_changed=!1),m.event.simulate("change",this,a,!0)})),!1):void m.event.add(this,"beforeactivate._change",function(a){var b=a.target;X.test(b.nodeName)&&!m._data(b,"changeBubbles")&&(m.event.add(b,"change._change",function(a){!this.parentNode||a.isSimulated||a.isTrigger||m.event.simulate("change",this.parentNode,a,!0)}),m._data(b,"changeBubbles",!0))})},handle:function(a){var b=a.target;return this!==b||a.isSimulated||a.isTrigger||"radio"!==b.type&&"checkbox"!==b.type?a.handleObj.handler.apply(this,arguments):void 0},teardown:function(){return m.event.remove(this,"._change"),!X.test(this.nodeName)}}),k.focusinBubbles||m.each({focus:"focusin",blur:"focusout"},function(a,b){var c=function(a){m.event.simulate(b,a.target,m.event.fix(a),!0)};m.event.special[b]={setup:function(){var d=this.ownerDocument||this,e=m._data(d,b);e||d.addEventListener(a,c,!0),m._data(d,b,(e||0)+1)},teardown:function(){var d=this.ownerDocument||this,e=m._data(d,b)-1;e?m._data(d,b,e):(d.removeEventListener(a,c,!0),m._removeData(d,b))}}}),m.fn.extend({on:function(a,b,c,d,e){var f,g;if("object"==typeof a){"string"!=typeof b&&(c=c||b,b=void 0);for(f in a)this.on(f,b,c,a[f],e);return this}if(null==c&&null==d?(d=b,c=b=void 0):null==d&&("string"==typeof b?(d=c,c=void 0):(d=c,c=b,b=void 0)),d===!1)d=ba;else if(!d)return this;return 1===e&&(g=d,d=function(a){return m().off(a),g.apply(this,arguments)},d.guid=g.guid||(g.guid=m.guid++)),this.each(function(){m.event.add(this,a,d,c,b)})},one:function(a,b,c,d){return this.on(a,b,c,d,1)},off:function(a,b,c){var d,e;if(a&&a.preventDefault&&a.handleObj)return d=a.handleObj,m(a.delegateTarget).off(d.namespace?d.origType+"."+d.namespace:d.origType,d.selector,d.handler),this;if("object"==typeof a){for(e in a)this.off(e,b,a[e]);return this}return(b===!1||"function"==typeof b)&&(c=b,b=void 0),c===!1&&(c=ba),this.each(function(){m.event.remove(this,a,c,b)})},trigger:function(a,b){return this.each(function(){m.event.trigger(a,b,this)})},triggerHandler:function(a,b){var c=this[0];return c?m.event.trigger(a,b,c,!0):void 0}});function da(a){var b=ea.split("|"),c=a.createDocumentFragment();if(c.createElement)while(b.length)c.createElement(b.pop());return c}var ea="abbr|article|aside|audio|bdi|canvas|data|datalist|details|figcaption|figure|footer|header|hgroup|mark|meter|nav|output|progress|section|summary|time|video",fa=/ jQuery\d+="(?:null|\d+)"/g,ga=new RegExp("<(?:"+ea+")[\\s/>]","i"),ha=/^\s+/,ia=/<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,ja=/<([\w:]+)/,ka=/<tbody/i,la=/<|&#?\w+;/,ma=/<(?:script|style|link)/i,na=/checked\s*(?:[^=]|=\s*.checked.)/i,oa=/^$|\/(?:java|ecma)script/i,pa=/^true\/(.*)/,qa=/^\s*<!(?:\[CDATA\[|--)|(?:\]\]|--)>\s*$/g,ra={option:[1,"<select multiple='multiple'>","</select>"],legend:[1,"<fieldset>","</fieldset>"],area:[1,"<map>","</map>"],param:[1,"<object>","</object>"],thead:[1,"<table>","</table>"],tr:[2,"<table><tbody>","</tbody></table>"],col:[2,"<table><tbody></tbody><colgroup>","</colgroup></table>"],td:[3,"<table><tbody><tr>","</tr></tbody></table>"],_default:k.htmlSerialize?[0,"",""]:[1,"X<div>","</div>"]},sa=da(y),ta=sa.appendChild(y.createElement("div"));ra.optgroup=ra.option,ra.tbody=ra.tfoot=ra.colgroup=ra.caption=ra.thead,ra.th=ra.td;function ua(a,b){var c,d,e=0,f=typeof a.getElementsByTagName!==K?a.getElementsByTagName(b||"*"):typeof a.querySelectorAll!==K?a.querySelectorAll(b||"*"):void 0;if(!f)for(f=[],c=a.childNodes||a;null!=(d=c[e]);e++)!b||m.nodeName(d,b)?f.push(d):m.merge(f,ua(d,b));return void 0===b||b&&m.nodeName(a,b)?m.merge([a],f):f}function va(a){W.test(a.type)&&(a.defaultChecked=a.checked)}function wa(a,b){return m.nodeName(a,"table")&&m.nodeName(11!==b.nodeType?b:b.firstChild,"tr")?a.getElementsByTagName("tbody")[0]||a.appendChild(a.ownerDocument.createElement("tbody")):a}function xa(a){return a.type=(null!==m.find.attr(a,"type"))+"/"+a.type,a}function ya(a){var b=pa.exec(a.type);return b?a.type=b[1]:a.removeAttribute("type"),a}function za(a,b){for(var c,d=0;null!=(c=a[d]);d++)m._data(c,"globalEval",!b||m._data(b[d],"globalEval"))}function Aa(a,b){if(1===b.nodeType&&m.hasData(a)){var c,d,e,f=m._data(a),g=m._data(b,f),h=f.events;if(h){delete g.handle,g.events={};for(c in h)for(d=0,e=h[c].length;e>d;d++)m.event.add(b,c,h[c][d])}g.data&&(g.data=m.extend({},g.data))}}function Ba(a,b){var c,d,e;if(1===b.nodeType){if(c=b.nodeName.toLowerCase(),!k.noCloneEvent&&b[m.expando]){e=m._data(b);for(d in e.events)m.removeEvent(b,d,e.handle);b.removeAttribute(m.expando)}"script"===c&&b.text!==a.text?(xa(b).text=a.text,ya(b)):"object"===c?(b.parentNode&&(b.outerHTML=a.outerHTML),k.html5Clone&&a.innerHTML&&!m.trim(b.innerHTML)&&(b.innerHTML=a.innerHTML)):"input"===c&&W.test(a.type)?(b.defaultChecked=b.checked=a.checked,b.value!==a.value&&(b.value=a.value)):"option"===c?b.defaultSelected=b.selected=a.defaultSelected:("input"===c||"textarea"===c)&&(b.defaultValue=a.defaultValue)}}m.extend({clone:function(a,b,c){var d,e,f,g,h,i=m.contains(a.ownerDocument,a);if(k.html5Clone||m.isXMLDoc(a)||!ga.test("<"+a.nodeName+">")?f=a.cloneNode(!0):(ta.innerHTML=a.outerHTML,ta.removeChild(f=ta.firstChild)),!(k.noCloneEvent&&k.noCloneChecked||1!==a.nodeType&&11!==a.nodeType||m.isXMLDoc(a)))for(d=ua(f),h=ua(a),g=0;null!=(e=h[g]);++g)d[g]&&Ba(e,d[g]);if(b)if(c)for(h=h||ua(a),d=d||ua(f),g=0;null!=(e=h[g]);g++)Aa(e,d[g]);else Aa(a,f);return d=ua(f,"script"),d.length>0&&za(d,!i&&ua(a,"script")),d=h=e=null,f},buildFragment:function(a,b,c,d){for(var e,f,g,h,i,j,l,n=a.length,o=da(b),p=[],q=0;n>q;q++)if(f=a[q],f||0===f)if("object"===m.type(f))m.merge(p,f.nodeType?[f]:f);else if(la.test(f)){h=h||o.appendChild(b.createElement("div")),i=(ja.exec(f)||["",""])[1].toLowerCase(),l=ra[i]||ra._default,h.innerHTML=l[1]+f.replace(ia,"<$1></$2>")+l[2],e=l[0];while(e--)h=h.lastChild;if(!k.leadingWhitespace&&ha.test(f)&&p.push(b.createTextNode(ha.exec(f)[0])),!k.tbody){f="table"!==i||ka.test(f)?"<table>"!==l[1]||ka.test(f)?0:h:h.firstChild,e=f&&f.childNodes.length;while(e--)m.nodeName(j=f.childNodes[e],"tbody")&&!j.childNodes.length&&f.removeChild(j)}m.merge(p,h.childNodes),h.textContent="";while(h.firstChild)h.removeChild(h.firstChild);h=o.lastChild}else p.push(b.createTextNode(f));h&&o.removeChild(h),k.appendChecked||m.grep(ua(p,"input"),va),q=0;while(f=p[q++])if((!d||-1===m.inArray(f,d))&&(g=m.contains(f.ownerDocument,f),h=ua(o.appendChild(f),"script"),g&&za(h),c)){e=0;while(f=h[e++])oa.test(f.type||"")&&c.push(f)}return h=null,o},cleanData:function(a,b){for(var d,e,f,g,h=0,i=m.expando,j=m.cache,l=k.deleteExpando,n=m.event.special;null!=(d=a[h]);h++)if((b||m.acceptData(d))&&(f=d[i],g=f&&j[f])){if(g.events)for(e in g.events)n[e]?m.event.remove(d,e):m.removeEvent(d,e,g.handle);j[f]&&(delete j[f],l?delete d[i]:typeof d.removeAttribute!==K?d.removeAttribute(i):d[i]=null,c.push(f))}}}),m.fn.extend({text:function(a){return V(this,function(a){return void 0===a?m.text(this):this.empty().append((this[0]&&this[0].ownerDocument||y).createTextNode(a))},null,a,arguments.length)},append:function(){return this.domManip(arguments,function(a){if(1===this.nodeType||11===this.nodeType||9===this.nodeType){var b=wa(this,a);b.appendChild(a)}})},prepend:function(){return this.domManip(arguments,function(a){if(1===this.nodeType||11===this.nodeType||9===this.nodeType){var b=wa(this,a);b.insertBefore(a,b.firstChild)}})},before:function(){return this.domManip(arguments,function(a){this.parentNode&&this.parentNode.insertBefore(a,this)})},after:function(){return this.domManip(arguments,function(a){this.parentNode&&this.parentNode.insertBefore(a,this.nextSibling)})},remove:function(a,b){for(var c,d=a?m.filter(a,this):this,e=0;null!=(c=d[e]);e++)b||1!==c.nodeType||m.cleanData(ua(c)),c.parentNode&&(b&&m.contains(c.ownerDocument,c)&&za(ua(c,"script")),c.parentNode.removeChild(c));return this},empty:function(){for(var a,b=0;null!=(a=this[b]);b++){1===a.nodeType&&m.cleanData(ua(a,!1));while(a.firstChild)a.removeChild(a.firstChild);a.options&&m.nodeName(a,"select")&&(a.options.length=0)}return this},clone:function(a,b){return a=null==a?!1:a,b=null==b?a:b,this.map(function(){return m.clone(this,a,b)})},html:function(a){return V(this,function(a){var b=this[0]||{},c=0,d=this.length;if(void 0===a)return 1===b.nodeType?b.innerHTML.replace(fa,""):void 0;if(!("string"!=typeof a||ma.test(a)||!k.htmlSerialize&&ga.test(a)||!k.leadingWhitespace&&ha.test(a)||ra[(ja.exec(a)||["",""])[1].toLowerCase()])){a=a.replace(ia,"<$1></$2>");try{for(;d>c;c++)b=this[c]||{},1===b.nodeType&&(m.cleanData(ua(b,!1)),b.innerHTML=a);b=0}catch(e){}}b&&this.empty().append(a)},null,a,arguments.length)},replaceWith:function(){var a=arguments[0];return this.domManip(arguments,function(b){a=this.parentNode,m.cleanData(ua(this)),a&&a.replaceChild(b,this)}),a&&(a.length||a.nodeType)?this:this.remove()},detach:function(a){return this.remove(a,!0)},domManip:function(a,b){a=e.apply([],a);var c,d,f,g,h,i,j=0,l=this.length,n=this,o=l-1,p=a[0],q=m.isFunction(p);if(q||l>1&&"string"==typeof p&&!k.checkClone&&na.test(p))return this.each(function(c){var d=n.eq(c);q&&(a[0]=p.call(this,c,d.html())),d.domManip(a,b)});if(l&&(i=m.buildFragment(a,this[0].ownerDocument,!1,this),c=i.firstChild,1===i.childNodes.length&&(i=c),c)){for(g=m.map(ua(i,"script"),xa),f=g.length;l>j;j++)d=i,j!==o&&(d=m.clone(d,!0,!0),f&&m.merge(g,ua(d,"script"))),b.call(this[j],d,j);if(f)for(h=g[g.length-1].ownerDocument,m.map(g,ya),j=0;f>j;j++)d=g[j],oa.test(d.type||"")&&!m._data(d,"globalEval")&&m.contains(h,d)&&(d.src?m._evalUrl&&m._evalUrl(d.src):m.globalEval((d.text||d.textContent||d.innerHTML||"").replace(qa,"")));i=c=null}return this}}),m.each({appendTo:"append",prependTo:"prepend",insertBefore:"before",insertAfter:"after",replaceAll:"replaceWith"},function(a,b){m.fn[a]=function(a){for(var c,d=0,e=[],g=m(a),h=g.length-1;h>=d;d++)c=d===h?this:this.clone(!0),m(g[d])[b](c),f.apply(e,c.get());return this.pushStack(e)}});var Ca,Da={};function Ea(b,c){var d,e=m(c.createElement(b)).appendTo(c.body),f=a.getDefaultComputedStyle&&(d=a.getDefaultComputedStyle(e[0]))?d.display:m.css(e[0],"display");return e.detach(),f}function Fa(a){var b=y,c=Da[a];return c||(c=Ea(a,b),"none"!==c&&c||(Ca=(Ca||m("<iframe frameborder='0' width='0' height='0'/>")).appendTo(b.documentElement),b=(Ca[0].contentWindow||Ca[0].contentDocument).document,b.write(),b.close(),c=Ea(a,b),Ca.detach()),Da[a]=c),c}!function(){var a;k.shrinkWrapBlocks=function(){if(null!=a)return a;a=!1;var b,c,d;return c=y.getElementsByTagName("body")[0],c&&c.style?(b=y.createElement("div"),d=y.createElement("div"),d.style.cssText="position:absolute;border:0;width:0;height:0;top:0;left:-9999px",c.appendChild(d).appendChild(b),typeof b.style.zoom!==K&&(b.style.cssText="-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;display:block;margin:0;border:0;padding:1px;width:1px;zoom:1",b.appendChild(y.createElement("div")).style.width="5px",a=3!==b.offsetWidth),c.removeChild(d),a):void 0}}();var Ga=/^margin/,Ha=new RegExp("^("+S+")(?!px)[a-z%]+$","i"),Ia,Ja,Ka=/^(top|right|bottom|left)$/;a.getComputedStyle?(Ia=function(b){return b.ownerDocument.defaultView.opener?b.ownerDocument.defaultView.getComputedStyle(b,null):a.getComputedStyle(b,null)},Ja=function(a,b,c){var d,e,f,g,h=a.style;return c=c||Ia(a),g=c?c.getPropertyValue(b)||c[b]:void 0,c&&(""!==g||m.contains(a.ownerDocument,a)||(g=m.style(a,b)),Ha.test(g)&&Ga.test(b)&&(d=h.width,e=h.minWidth,f=h.maxWidth,h.minWidth=h.maxWidth=h.width=g,g=c.width,h.width=d,h.minWidth=e,h.maxWidth=f)),void 0===g?g:g+""}):y.documentElement.currentStyle&&(Ia=function(a){return a.currentStyle},Ja=function(a,b,c){var d,e,f,g,h=a.style;return c=c||Ia(a),g=c?c[b]:void 0,null==g&&h&&h[b]&&(g=h[b]),Ha.test(g)&&!Ka.test(b)&&(d=h.left,e=a.runtimeStyle,f=e&&e.left,f&&(e.left=a.currentStyle.left),h.left="fontSize"===b?"1em":g,g=h.pixelLeft+"px",h.left=d,f&&(e.left=f)),void 0===g?g:g+""||"auto"});function La(a,b){return{get:function(){var c=a();if(null!=c)return c?void delete this.get:(this.get=b).apply(this,arguments)}}}!function(){var b,c,d,e,f,g,h;if(b=y.createElement("div"),b.innerHTML="  <link/><table></table><a href='/a'>a</a><input type='checkbox'/>",d=b.getElementsByTagName("a")[0],c=d&&d.style){c.cssText="float:left;opacity:.5",k.opacity="0.5"===c.opacity,k.cssFloat=!!c.cssFloat,b.style.backgroundClip="content-box",b.cloneNode(!0).style.backgroundClip="",k.clearCloneStyle="content-box"===b.style.backgroundClip,k.boxSizing=""===c.boxSizing||""===c.MozBoxSizing||""===c.WebkitBoxSizing,m.extend(k,{reliableHiddenOffsets:function(){return null==g&&i(),g},boxSizingReliable:function(){return null==f&&i(),f},pixelPosition:function(){return null==e&&i(),e},reliableMarginRight:function(){return null==h&&i(),h}});function i(){var b,c,d,i;c=y.getElementsByTagName("body")[0],c&&c.style&&(b=y.createElement("div"),d=y.createElement("div"),d.style.cssText="position:absolute;border:0;width:0;height:0;top:0;left:-9999px",c.appendChild(d).appendChild(b),b.style.cssText="-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;display:block;margin-top:1%;top:1%;border:1px;padding:1px;width:4px;position:absolute",e=f=!1,h=!0,a.getComputedStyle&&(e="1%"!==(a.getComputedStyle(b,null)||{}).top,f="4px"===(a.getComputedStyle(b,null)||{width:"4px"}).width,i=b.appendChild(y.createElement("div")),i.style.cssText=b.style.cssText="-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;display:block;margin:0;border:0;padding:0",i.style.marginRight=i.style.width="0",b.style.width="1px",h=!parseFloat((a.getComputedStyle(i,null)||{}).marginRight),b.removeChild(i)),b.innerHTML="<table><tr><td></td><td>t</td></tr></table>",i=b.getElementsByTagName("td"),i[0].style.cssText="margin:0;border:0;padding:0;display:none",g=0===i[0].offsetHeight,g&&(i[0].style.display="",i[1].style.display="none",g=0===i[0].offsetHeight),c.removeChild(d))}}}(),m.swap=function(a,b,c,d){var e,f,g={};for(f in b)g[f]=a.style[f],a.style[f]=b[f];e=c.apply(a,d||[]);for(f in b)a.style[f]=g[f];return e};var Ma=/alpha\([^)]*\)/i,Na=/opacity\s*=\s*([^)]*)/,Oa=/^(none|table(?!-c[ea]).+)/,Pa=new RegExp("^("+S+")(.*)$","i"),Qa=new RegExp("^([+-])=("+S+")","i"),Ra={position:"absolute",visibility:"hidden",display:"block"},Sa={letterSpacing:"0",fontWeight:"400"},Ta=["Webkit","O","Moz","ms"];function Ua(a,b){if(b in a)return b;var c=b.charAt(0).toUpperCase()+b.slice(1),d=b,e=Ta.length;while(e--)if(b=Ta[e]+c,b in a)return b;return d}function Va(a,b){for(var c,d,e,f=[],g=0,h=a.length;h>g;g++)d=a[g],d.style&&(f[g]=m._data(d,"olddisplay"),c=d.style.display,b?(f[g]||"none"!==c||(d.style.display=""),""===d.style.display&&U(d)&&(f[g]=m._data(d,"olddisplay",Fa(d.nodeName)))):(e=U(d),(c&&"none"!==c||!e)&&m._data(d,"olddisplay",e?c:m.css(d,"display"))));for(g=0;h>g;g++)d=a[g],d.style&&(b&&"none"!==d.style.display&&""!==d.style.display||(d.style.display=b?f[g]||"":"none"));return a}function Wa(a,b,c){var d=Pa.exec(b);return d?Math.max(0,d[1]-(c||0))+(d[2]||"px"):b}function Xa(a,b,c,d,e){for(var f=c===(d?"border":"content")?4:"width"===b?1:0,g=0;4>f;f+=2)"margin"===c&&(g+=m.css(a,c+T[f],!0,e)),d?("content"===c&&(g-=m.css(a,"padding"+T[f],!0,e)),"margin"!==c&&(g-=m.css(a,"border"+T[f]+"Width",!0,e))):(g+=m.css(a,"padding"+T[f],!0,e),"padding"!==c&&(g+=m.css(a,"border"+T[f]+"Width",!0,e)));return g}function Ya(a,b,c){var d=!0,e="width"===b?a.offsetWidth:a.offsetHeight,f=Ia(a),g=k.boxSizing&&"border-box"===m.css(a,"boxSizing",!1,f);if(0>=e||null==e){if(e=Ja(a,b,f),(0>e||null==e)&&(e=a.style[b]),Ha.test(e))return e;d=g&&(k.boxSizingReliable()||e===a.style[b]),e=parseFloat(e)||0}return e+Xa(a,b,c||(g?"border":"content"),d,f)+"px"}m.extend({cssHooks:{opacity:{get:function(a,b){if(b){var c=Ja(a,"opacity");return""===c?"1":c}}}},cssNumber:{columnCount:!0,fillOpacity:!0,flexGrow:!0,flexShrink:!0,fontWeight:!0,lineHeight:!0,opacity:!0,order:!0,orphans:!0,widows:!0,zIndex:!0,zoom:!0},cssProps:{"float":k.cssFloat?"cssFloat":"styleFloat"},style:function(a,b,c,d){if(a&&3!==a.nodeType&&8!==a.nodeType&&a.style){var e,f,g,h=m.camelCase(b),i=a.style;if(b=m.cssProps[h]||(m.cssProps[h]=Ua(i,h)),g=m.cssHooks[b]||m.cssHooks[h],void 0===c)return g&&"get"in g&&void 0!==(e=g.get(a,!1,d))?e:i[b];if(f=typeof c,"string"===f&&(e=Qa.exec(c))&&(c=(e[1]+1)*e[2]+parseFloat(m.css(a,b)),f="number"),null!=c&&c===c&&("number"!==f||m.cssNumber[h]||(c+="px"),k.clearCloneStyle||""!==c||0!==b.indexOf("background")||(i[b]="inherit"),!(g&&"set"in g&&void 0===(c=g.set(a,c,d)))))try{i[b]=c}catch(j){}}},css:function(a,b,c,d){var e,f,g,h=m.camelCase(b);return b=m.cssProps[h]||(m.cssProps[h]=Ua(a.style,h)),g=m.cssHooks[b]||m.cssHooks[h],g&&"get"in g&&(f=g.get(a,!0,c)),void 0===f&&(f=Ja(a,b,d)),"normal"===f&&b in Sa&&(f=Sa[b]),""===c||c?(e=parseFloat(f),c===!0||m.isNumeric(e)?e||0:f):f}}),m.each(["height","width"],function(a,b){m.cssHooks[b]={get:function(a,c,d){return c?Oa.test(m.css(a,"display"))&&0===a.offsetWidth?m.swap(a,Ra,function(){return Ya(a,b,d)}):Ya(a,b,d):void 0},set:function(a,c,d){var e=d&&Ia(a);return Wa(a,c,d?Xa(a,b,d,k.boxSizing&&"border-box"===m.css(a,"boxSizing",!1,e),e):0)}}}),k.opacity||(m.cssHooks.opacity={get:function(a,b){return Na.test((b&&a.currentStyle?a.currentStyle.filter:a.style.filter)||"")?.01*parseFloat(RegExp.$1)+"":b?"1":""},set:function(a,b){var c=a.style,d=a.currentStyle,e=m.isNumeric(b)?"alpha(opacity="+100*b+")":"",f=d&&d.filter||c.filter||"";c.zoom=1,(b>=1||""===b)&&""===m.trim(f.replace(Ma,""))&&c.removeAttribute&&(c.removeAttribute("filter"),""===b||d&&!d.filter)||(c.filter=Ma.test(f)?f.replace(Ma,e):f+" "+e)}}),m.cssHooks.marginRight=La(k.reliableMarginRight,function(a,b){return b?m.swap(a,{display:"inline-block"},Ja,[a,"marginRight"]):void 0}),m.each({margin:"",padding:"",border:"Width"},function(a,b){m.cssHooks[a+b]={expand:function(c){for(var d=0,e={},f="string"==typeof c?c.split(" "):[c];4>d;d++)e[a+T[d]+b]=f[d]||f[d-2]||f[0];return e}},Ga.test(a)||(m.cssHooks[a+b].set=Wa)}),m.fn.extend({css:function(a,b){return V(this,function(a,b,c){var d,e,f={},g=0;if(m.isArray(b)){for(d=Ia(a),e=b.length;e>g;g++)f[b[g]]=m.css(a,b[g],!1,d);return f}return void 0!==c?m.style(a,b,c):m.css(a,b)},a,b,arguments.length>1)},show:function(){return Va(this,!0)},hide:function(){return Va(this)},toggle:function(a){return"boolean"==typeof a?a?this.show():this.hide():this.each(function(){U(this)?m(this).show():m(this).hide()})}});function Za(a,b,c,d,e){
return new Za.prototype.init(a,b,c,d,e)}m.Tween=Za,Za.prototype={constructor:Za,init:function(a,b,c,d,e,f){this.elem=a,this.prop=c,this.easing=e||"swing",this.options=b,this.start=this.now=this.cur(),this.end=d,this.unit=f||(m.cssNumber[c]?"":"px")},cur:function(){var a=Za.propHooks[this.prop];return a&&a.get?a.get(this):Za.propHooks._default.get(this)},run:function(a){var b,c=Za.propHooks[this.prop];return this.options.duration?this.pos=b=m.easing[this.easing](a,this.options.duration*a,0,1,this.options.duration):this.pos=b=a,this.now=(this.end-this.start)*b+this.start,this.options.step&&this.options.step.call(this.elem,this.now,this),c&&c.set?c.set(this):Za.propHooks._default.set(this),this}},Za.prototype.init.prototype=Za.prototype,Za.propHooks={_default:{get:function(a){var b;return null==a.elem[a.prop]||a.elem.style&&null!=a.elem.style[a.prop]?(b=m.css(a.elem,a.prop,""),b&&"auto"!==b?b:0):a.elem[a.prop]},set:function(a){m.fx.step[a.prop]?m.fx.step[a.prop](a):a.elem.style&&(null!=a.elem.style[m.cssProps[a.prop]]||m.cssHooks[a.prop])?m.style(a.elem,a.prop,a.now+a.unit):a.elem[a.prop]=a.now}}},Za.propHooks.scrollTop=Za.propHooks.scrollLeft={set:function(a){a.elem.nodeType&&a.elem.parentNode&&(a.elem[a.prop]=a.now)}},m.easing={linear:function(a){return a},swing:function(a){return.5-Math.cos(a*Math.PI)/2}},m.fx=Za.prototype.init,m.fx.step={};var $a,_a,ab=/^(?:toggle|show|hide)$/,bb=new RegExp("^(?:([+-])=|)("+S+")([a-z%]*)$","i"),cb=/queueHooks$/,db=[ib],eb={"*":[function(a,b){var c=this.createTween(a,b),d=c.cur(),e=bb.exec(b),f=e&&e[3]||(m.cssNumber[a]?"":"px"),g=(m.cssNumber[a]||"px"!==f&&+d)&&bb.exec(m.css(c.elem,a)),h=1,i=20;if(g&&g[3]!==f){f=f||g[3],e=e||[],g=+d||1;do h=h||".5",g/=h,m.style(c.elem,a,g+f);while(h!==(h=c.cur()/d)&&1!==h&&--i)}return e&&(g=c.start=+g||+d||0,c.unit=f,c.end=e[1]?g+(e[1]+1)*e[2]:+e[2]),c}]};function fb(){return setTimeout(function(){$a=void 0}),$a=m.now()}function gb(a,b){var c,d={height:a},e=0;for(b=b?1:0;4>e;e+=2-b)c=T[e],d["margin"+c]=d["padding"+c]=a;return b&&(d.opacity=d.width=a),d}function hb(a,b,c){for(var d,e=(eb[b]||[]).concat(eb["*"]),f=0,g=e.length;g>f;f++)if(d=e[f].call(c,b,a))return d}function ib(a,b,c){var d,e,f,g,h,i,j,l,n=this,o={},p=a.style,q=a.nodeType&&U(a),r=m._data(a,"fxshow");c.queue||(h=m._queueHooks(a,"fx"),null==h.unqueued&&(h.unqueued=0,i=h.empty.fire,h.empty.fire=function(){h.unqueued||i()}),h.unqueued++,n.always(function(){n.always(function(){h.unqueued--,m.queue(a,"fx").length||h.empty.fire()})})),1===a.nodeType&&("height"in b||"width"in b)&&(c.overflow=[p.overflow,p.overflowX,p.overflowY],j=m.css(a,"display"),l="none"===j?m._data(a,"olddisplay")||Fa(a.nodeName):j,"inline"===l&&"none"===m.css(a,"float")&&(k.inlineBlockNeedsLayout&&"inline"!==Fa(a.nodeName)?p.zoom=1:p.display="inline-block")),c.overflow&&(p.overflow="hidden",k.shrinkWrapBlocks()||n.always(function(){p.overflow=c.overflow[0],p.overflowX=c.overflow[1],p.overflowY=c.overflow[2]}));for(d in b)if(e=b[d],ab.exec(e)){if(delete b[d],f=f||"toggle"===e,e===(q?"hide":"show")){if("show"!==e||!r||void 0===r[d])continue;q=!0}o[d]=r&&r[d]||m.style(a,d)}else j=void 0;if(m.isEmptyObject(o))"inline"===("none"===j?Fa(a.nodeName):j)&&(p.display=j);else{r?"hidden"in r&&(q=r.hidden):r=m._data(a,"fxshow",{}),f&&(r.hidden=!q),q?m(a).show():n.done(function(){m(a).hide()}),n.done(function(){var b;m._removeData(a,"fxshow");for(b in o)m.style(a,b,o[b])});for(d in o)g=hb(q?r[d]:0,d,n),d in r||(r[d]=g.start,q&&(g.end=g.start,g.start="width"===d||"height"===d?1:0))}}function jb(a,b){var c,d,e,f,g;for(c in a)if(d=m.camelCase(c),e=b[d],f=a[c],m.isArray(f)&&(e=f[1],f=a[c]=f[0]),c!==d&&(a[d]=f,delete a[c]),g=m.cssHooks[d],g&&"expand"in g){f=g.expand(f),delete a[d];for(c in f)c in a||(a[c]=f[c],b[c]=e)}else b[d]=e}function kb(a,b,c){var d,e,f=0,g=db.length,h=m.Deferred().always(function(){delete i.elem}),i=function(){if(e)return!1;for(var b=$a||fb(),c=Math.max(0,j.startTime+j.duration-b),d=c/j.duration||0,f=1-d,g=0,i=j.tweens.length;i>g;g++)j.tweens[g].run(f);return h.notifyWith(a,[j,f,c]),1>f&&i?c:(h.resolveWith(a,[j]),!1)},j=h.promise({elem:a,props:m.extend({},b),opts:m.extend(!0,{specialEasing:{}},c),originalProperties:b,originalOptions:c,startTime:$a||fb(),duration:c.duration,tweens:[],createTween:function(b,c){var d=m.Tween(a,j.opts,b,c,j.opts.specialEasing[b]||j.opts.easing);return j.tweens.push(d),d},stop:function(b){var c=0,d=b?j.tweens.length:0;if(e)return this;for(e=!0;d>c;c++)j.tweens[c].run(1);return b?h.resolveWith(a,[j,b]):h.rejectWith(a,[j,b]),this}}),k=j.props;for(jb(k,j.opts.specialEasing);g>f;f++)if(d=db[f].call(j,a,k,j.opts))return d;return m.map(k,hb,j),m.isFunction(j.opts.start)&&j.opts.start.call(a,j),m.fx.timer(m.extend(i,{elem:a,anim:j,queue:j.opts.queue})),j.progress(j.opts.progress).done(j.opts.done,j.opts.complete).fail(j.opts.fail).always(j.opts.always)}m.Animation=m.extend(kb,{tweener:function(a,b){m.isFunction(a)?(b=a,a=["*"]):a=a.split(" ");for(var c,d=0,e=a.length;e>d;d++)c=a[d],eb[c]=eb[c]||[],eb[c].unshift(b)},prefilter:function(a,b){b?db.unshift(a):db.push(a)}}),m.speed=function(a,b,c){var d=a&&"object"==typeof a?m.extend({},a):{complete:c||!c&&b||m.isFunction(a)&&a,duration:a,easing:c&&b||b&&!m.isFunction(b)&&b};return d.duration=m.fx.off?0:"number"==typeof d.duration?d.duration:d.duration in m.fx.speeds?m.fx.speeds[d.duration]:m.fx.speeds._default,(null==d.queue||d.queue===!0)&&(d.queue="fx"),d.old=d.complete,d.complete=function(){m.isFunction(d.old)&&d.old.call(this),d.queue&&m.dequeue(this,d.queue)},d},m.fn.extend({fadeTo:function(a,b,c,d){return this.filter(U).css("opacity",0).show().end().animate({opacity:b},a,c,d)},animate:function(a,b,c,d){var e=m.isEmptyObject(a),f=m.speed(b,c,d),g=function(){var b=kb(this,m.extend({},a),f);(e||m._data(this,"finish"))&&b.stop(!0)};return g.finish=g,e||f.queue===!1?this.each(g):this.queue(f.queue,g)},stop:function(a,b,c){var d=function(a){var b=a.stop;delete a.stop,b(c)};return"string"!=typeof a&&(c=b,b=a,a=void 0),b&&a!==!1&&this.queue(a||"fx",[]),this.each(function(){var b=!0,e=null!=a&&a+"queueHooks",f=m.timers,g=m._data(this);if(e)g[e]&&g[e].stop&&d(g[e]);else for(e in g)g[e]&&g[e].stop&&cb.test(e)&&d(g[e]);for(e=f.length;e--;)f[e].elem!==this||null!=a&&f[e].queue!==a||(f[e].anim.stop(c),b=!1,f.splice(e,1));(b||!c)&&m.dequeue(this,a)})},finish:function(a){return a!==!1&&(a=a||"fx"),this.each(function(){var b,c=m._data(this),d=c[a+"queue"],e=c[a+"queueHooks"],f=m.timers,g=d?d.length:0;for(c.finish=!0,m.queue(this,a,[]),e&&e.stop&&e.stop.call(this,!0),b=f.length;b--;)f[b].elem===this&&f[b].queue===a&&(f[b].anim.stop(!0),f.splice(b,1));for(b=0;g>b;b++)d[b]&&d[b].finish&&d[b].finish.call(this);delete c.finish})}}),m.each(["toggle","show","hide"],function(a,b){var c=m.fn[b];m.fn[b]=function(a,d,e){return null==a||"boolean"==typeof a?c.apply(this,arguments):this.animate(gb(b,!0),a,d,e)}}),m.each({slideDown:gb("show"),slideUp:gb("hide"),slideToggle:gb("toggle"),fadeIn:{opacity:"show"},fadeOut:{opacity:"hide"},fadeToggle:{opacity:"toggle"}},function(a,b){m.fn[a]=function(a,c,d){return this.animate(b,a,c,d)}}),m.timers=[],m.fx.tick=function(){var a,b=m.timers,c=0;for($a=m.now();c<b.length;c++)a=b[c],a()||b[c]!==a||b.splice(c--,1);b.length||m.fx.stop(),$a=void 0},m.fx.timer=function(a){m.timers.push(a),a()?m.fx.start():m.timers.pop()},m.fx.interval=13,m.fx.start=function(){_a||(_a=setInterval(m.fx.tick,m.fx.interval))},m.fx.stop=function(){clearInterval(_a),_a=null},m.fx.speeds={slow:600,fast:200,_default:400},m.fn.delay=function(a,b){return a=m.fx?m.fx.speeds[a]||a:a,b=b||"fx",this.queue(b,function(b,c){var d=setTimeout(b,a);c.stop=function(){clearTimeout(d)}})},function(){var a,b,c,d,e;b=y.createElement("div"),b.setAttribute("className","t"),b.innerHTML="  <link/><table></table><a href='/a'>a</a><input type='checkbox'/>",d=b.getElementsByTagName("a")[0],c=y.createElement("select"),e=c.appendChild(y.createElement("option")),a=b.getElementsByTagName("input")[0],d.style.cssText="top:1px",k.getSetAttribute="t"!==b.className,k.style=/top/.test(d.getAttribute("style")),k.hrefNormalized="/a"===d.getAttribute("href"),k.checkOn=!!a.value,k.optSelected=e.selected,k.enctype=!!y.createElement("form").enctype,c.disabled=!0,k.optDisabled=!e.disabled,a=y.createElement("input"),a.setAttribute("value",""),k.input=""===a.getAttribute("value"),a.value="t",a.setAttribute("type","radio"),k.radioValue="t"===a.value}();var lb=/\r/g;m.fn.extend({val:function(a){var b,c,d,e=this[0];{if(arguments.length)return d=m.isFunction(a),this.each(function(c){var e;1===this.nodeType&&(e=d?a.call(this,c,m(this).val()):a,null==e?e="":"number"==typeof e?e+="":m.isArray(e)&&(e=m.map(e,function(a){return null==a?"":a+""})),b=m.valHooks[this.type]||m.valHooks[this.nodeName.toLowerCase()],b&&"set"in b&&void 0!==b.set(this,e,"value")||(this.value=e))});if(e)return b=m.valHooks[e.type]||m.valHooks[e.nodeName.toLowerCase()],b&&"get"in b&&void 0!==(c=b.get(e,"value"))?c:(c=e.value,"string"==typeof c?c.replace(lb,""):null==c?"":c)}}}),m.extend({valHooks:{option:{get:function(a){var b=m.find.attr(a,"value");return null!=b?b:m.trim(m.text(a))}},select:{get:function(a){for(var b,c,d=a.options,e=a.selectedIndex,f="select-one"===a.type||0>e,g=f?null:[],h=f?e+1:d.length,i=0>e?h:f?e:0;h>i;i++)if(c=d[i],!(!c.selected&&i!==e||(k.optDisabled?c.disabled:null!==c.getAttribute("disabled"))||c.parentNode.disabled&&m.nodeName(c.parentNode,"optgroup"))){if(b=m(c).val(),f)return b;g.push(b)}return g},set:function(a,b){var c,d,e=a.options,f=m.makeArray(b),g=e.length;while(g--)if(d=e[g],m.inArray(m.valHooks.option.get(d),f)>=0)try{d.selected=c=!0}catch(h){d.scrollHeight}else d.selected=!1;return c||(a.selectedIndex=-1),e}}}}),m.each(["radio","checkbox"],function(){m.valHooks[this]={set:function(a,b){return m.isArray(b)?a.checked=m.inArray(m(a).val(),b)>=0:void 0}},k.checkOn||(m.valHooks[this].get=function(a){return null===a.getAttribute("value")?"on":a.value})});var mb,nb,ob=m.expr.attrHandle,pb=/^(?:checked|selected)$/i,qb=k.getSetAttribute,rb=k.input;m.fn.extend({attr:function(a,b){return V(this,m.attr,a,b,arguments.length>1)},removeAttr:function(a){return this.each(function(){m.removeAttr(this,a)})}}),m.extend({attr:function(a,b,c){var d,e,f=a.nodeType;if(a&&3!==f&&8!==f&&2!==f)return typeof a.getAttribute===K?m.prop(a,b,c):(1===f&&m.isXMLDoc(a)||(b=b.toLowerCase(),d=m.attrHooks[b]||(m.expr.match.bool.test(b)?nb:mb)),void 0===c?d&&"get"in d&&null!==(e=d.get(a,b))?e:(e=m.find.attr(a,b),null==e?void 0:e):null!==c?d&&"set"in d&&void 0!==(e=d.set(a,c,b))?e:(a.setAttribute(b,c+""),c):void m.removeAttr(a,b))},removeAttr:function(a,b){var c,d,e=0,f=b&&b.match(E);if(f&&1===a.nodeType)while(c=f[e++])d=m.propFix[c]||c,m.expr.match.bool.test(c)?rb&&qb||!pb.test(c)?a[d]=!1:a[m.camelCase("default-"+c)]=a[d]=!1:m.attr(a,c,""),a.removeAttribute(qb?c:d)},attrHooks:{type:{set:function(a,b){if(!k.radioValue&&"radio"===b&&m.nodeName(a,"input")){var c=a.value;return a.setAttribute("type",b),c&&(a.value=c),b}}}}}),nb={set:function(a,b,c){return b===!1?m.removeAttr(a,c):rb&&qb||!pb.test(c)?a.setAttribute(!qb&&m.propFix[c]||c,c):a[m.camelCase("default-"+c)]=a[c]=!0,c}},m.each(m.expr.match.bool.source.match(/\w+/g),function(a,b){var c=ob[b]||m.find.attr;ob[b]=rb&&qb||!pb.test(b)?function(a,b,d){var e,f;return d||(f=ob[b],ob[b]=e,e=null!=c(a,b,d)?b.toLowerCase():null,ob[b]=f),e}:function(a,b,c){return c?void 0:a[m.camelCase("default-"+b)]?b.toLowerCase():null}}),rb&&qb||(m.attrHooks.value={set:function(a,b,c){return m.nodeName(a,"input")?void(a.defaultValue=b):mb&&mb.set(a,b,c)}}),qb||(mb={set:function(a,b,c){var d=a.getAttributeNode(c);return d||a.setAttributeNode(d=a.ownerDocument.createAttribute(c)),d.value=b+="","value"===c||b===a.getAttribute(c)?b:void 0}},ob.id=ob.name=ob.coords=function(a,b,c){var d;return c?void 0:(d=a.getAttributeNode(b))&&""!==d.value?d.value:null},m.valHooks.button={get:function(a,b){var c=a.getAttributeNode(b);return c&&c.specified?c.value:void 0},set:mb.set},m.attrHooks.contenteditable={set:function(a,b,c){mb.set(a,""===b?!1:b,c)}},m.each(["width","height"],function(a,b){m.attrHooks[b]={set:function(a,c){return""===c?(a.setAttribute(b,"auto"),c):void 0}}})),k.style||(m.attrHooks.style={get:function(a){return a.style.cssText||void 0},set:function(a,b){return a.style.cssText=b+""}});var sb=/^(?:input|select|textarea|button|object)$/i,tb=/^(?:a|area)$/i;m.fn.extend({prop:function(a,b){return V(this,m.prop,a,b,arguments.length>1)},removeProp:function(a){return a=m.propFix[a]||a,this.each(function(){try{this[a]=void 0,delete this[a]}catch(b){}})}}),m.extend({propFix:{"for":"htmlFor","class":"className"},prop:function(a,b,c){var d,e,f,g=a.nodeType;if(a&&3!==g&&8!==g&&2!==g)return f=1!==g||!m.isXMLDoc(a),f&&(b=m.propFix[b]||b,e=m.propHooks[b]),void 0!==c?e&&"set"in e&&void 0!==(d=e.set(a,c,b))?d:a[b]=c:e&&"get"in e&&null!==(d=e.get(a,b))?d:a[b]},propHooks:{tabIndex:{get:function(a){var b=m.find.attr(a,"tabindex");return b?parseInt(b,10):sb.test(a.nodeName)||tb.test(a.nodeName)&&a.href?0:-1}}}}),k.hrefNormalized||m.each(["href","src"],function(a,b){m.propHooks[b]={get:function(a){return a.getAttribute(b,4)}}}),k.optSelected||(m.propHooks.selected={get:function(a){var b=a.parentNode;return b&&(b.selectedIndex,b.parentNode&&b.parentNode.selectedIndex),null}}),m.each(["tabIndex","readOnly","maxLength","cellSpacing","cellPadding","rowSpan","colSpan","useMap","frameBorder","contentEditable"],function(){m.propFix[this.toLowerCase()]=this}),k.enctype||(m.propFix.enctype="encoding");var ub=/[\t\r\n\f]/g;m.fn.extend({addClass:function(a){var b,c,d,e,f,g,h=0,i=this.length,j="string"==typeof a&&a;if(m.isFunction(a))return this.each(function(b){m(this).addClass(a.call(this,b,this.className))});if(j)for(b=(a||"").match(E)||[];i>h;h++)if(c=this[h],d=1===c.nodeType&&(c.className?(" "+c.className+" ").replace(ub," "):" ")){f=0;while(e=b[f++])d.indexOf(" "+e+" ")<0&&(d+=e+" ");g=m.trim(d),c.className!==g&&(c.className=g)}return this},removeClass:function(a){var b,c,d,e,f,g,h=0,i=this.length,j=0===arguments.length||"string"==typeof a&&a;if(m.isFunction(a))return this.each(function(b){m(this).removeClass(a.call(this,b,this.className))});if(j)for(b=(a||"").match(E)||[];i>h;h++)if(c=this[h],d=1===c.nodeType&&(c.className?(" "+c.className+" ").replace(ub," "):"")){f=0;while(e=b[f++])while(d.indexOf(" "+e+" ")>=0)d=d.replace(" "+e+" "," ");g=a?m.trim(d):"",c.className!==g&&(c.className=g)}return this},toggleClass:function(a,b){var c=typeof a;return"boolean"==typeof b&&"string"===c?b?this.addClass(a):this.removeClass(a):this.each(m.isFunction(a)?function(c){m(this).toggleClass(a.call(this,c,this.className,b),b)}:function(){if("string"===c){var b,d=0,e=m(this),f=a.match(E)||[];while(b=f[d++])e.hasClass(b)?e.removeClass(b):e.addClass(b)}else(c===K||"boolean"===c)&&(this.className&&m._data(this,"__className__",this.className),this.className=this.className||a===!1?"":m._data(this,"__className__")||"")})},hasClass:function(a){for(var b=" "+a+" ",c=0,d=this.length;d>c;c++)if(1===this[c].nodeType&&(" "+this[c].className+" ").replace(ub," ").indexOf(b)>=0)return!0;return!1}}),m.each("blur focus focusin focusout load resize scroll unload click dblclick mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave change select submit keydown keypress keyup error contextmenu".split(" "),function(a,b){m.fn[b]=function(a,c){return arguments.length>0?this.on(b,null,a,c):this.trigger(b)}}),m.fn.extend({hover:function(a,b){return this.mouseenter(a).mouseleave(b||a)},bind:function(a,b,c){return this.on(a,null,b,c)},unbind:function(a,b){return this.off(a,null,b)},delegate:function(a,b,c,d){return this.on(b,a,c,d)},undelegate:function(a,b,c){return 1===arguments.length?this.off(a,"**"):this.off(b,a||"**",c)}});var vb=m.now(),wb=/\?/,xb=/(,)|(\[|{)|(}|])|"(?:[^"\\\r\n]|\\["\\\/bfnrt]|\\u[\da-fA-F]{4})*"\s*:?|true|false|null|-?(?!0\d)\d+(?:\.\d+|)(?:[eE][+-]?\d+|)/g;m.parseJSON=function(b){if(a.JSON&&a.JSON.parse)return a.JSON.parse(b+"");var c,d=null,e=m.trim(b+"");return e&&!m.trim(e.replace(xb,function(a,b,e,f){return c&&b&&(d=0),0===d?a:(c=e||b,d+=!f-!e,"")}))?Function("return "+e)():m.error("Invalid JSON: "+b)},m.parseXML=function(b){var c,d;if(!b||"string"!=typeof b)return null;try{a.DOMParser?(d=new DOMParser,c=d.parseFromString(b,"text/xml")):(c=new ActiveXObject("Microsoft.XMLDOM"),c.async="false",c.loadXML(b))}catch(e){c=void 0}return c&&c.documentElement&&!c.getElementsByTagName("parsererror").length||m.error("Invalid XML: "+b),c};var yb,zb,Ab=/#.*$/,Bb=/([?&])_=[^&]*/,Cb=/^(.*?):[ \t]*([^\r\n]*)\r?$/gm,Db=/^(?:about|app|app-storage|.+-extension|file|res|widget):$/,Eb=/^(?:GET|HEAD)$/,Fb=/^\/\//,Gb=/^([\w.+-]+:)(?:\/\/(?:[^\/?#]*@|)([^\/?#:]*)(?::(\d+)|)|)/,Hb={},Ib={},Jb="*/".concat("*");try{zb=location.href}catch(Kb){zb=y.createElement("a"),zb.href="",zb=zb.href}yb=Gb.exec(zb.toLowerCase())||[];function Lb(a){return function(b,c){"string"!=typeof b&&(c=b,b="*");var d,e=0,f=b.toLowerCase().match(E)||[];if(m.isFunction(c))while(d=f[e++])"+"===d.charAt(0)?(d=d.slice(1)||"*",(a[d]=a[d]||[]).unshift(c)):(a[d]=a[d]||[]).push(c)}}function Mb(a,b,c,d){var e={},f=a===Ib;function g(h){var i;return e[h]=!0,m.each(a[h]||[],function(a,h){var j=h(b,c,d);return"string"!=typeof j||f||e[j]?f?!(i=j):void 0:(b.dataTypes.unshift(j),g(j),!1)}),i}return g(b.dataTypes[0])||!e["*"]&&g("*")}function Nb(a,b){var c,d,e=m.ajaxSettings.flatOptions||{};for(d in b)void 0!==b[d]&&((e[d]?a:c||(c={}))[d]=b[d]);return c&&m.extend(!0,a,c),a}function Ob(a,b,c){var d,e,f,g,h=a.contents,i=a.dataTypes;while("*"===i[0])i.shift(),void 0===e&&(e=a.mimeType||b.getResponseHeader("Content-Type"));if(e)for(g in h)if(h[g]&&h[g].test(e)){i.unshift(g);break}if(i[0]in c)f=i[0];else{for(g in c){if(!i[0]||a.converters[g+" "+i[0]]){f=g;break}d||(d=g)}f=f||d}return f?(f!==i[0]&&i.unshift(f),c[f]):void 0}function Pb(a,b,c,d){var e,f,g,h,i,j={},k=a.dataTypes.slice();if(k[1])for(g in a.converters)j[g.toLowerCase()]=a.converters[g];f=k.shift();while(f)if(a.responseFields[f]&&(c[a.responseFields[f]]=b),!i&&d&&a.dataFilter&&(b=a.dataFilter(b,a.dataType)),i=f,f=k.shift())if("*"===f)f=i;else if("*"!==i&&i!==f){if(g=j[i+" "+f]||j["* "+f],!g)for(e in j)if(h=e.split(" "),h[1]===f&&(g=j[i+" "+h[0]]||j["* "+h[0]])){g===!0?g=j[e]:j[e]!==!0&&(f=h[0],k.unshift(h[1]));break}if(g!==!0)if(g&&a["throws"])b=g(b);else try{b=g(b)}catch(l){return{state:"parsererror",error:g?l:"No conversion from "+i+" to "+f}}}return{state:"success",data:b}}m.extend({active:0,lastModified:{},etag:{},ajaxSettings:{url:zb,type:"GET",isLocal:Db.test(yb[1]),global:!0,processData:!0,async:!0,contentType:"application/x-www-form-urlencoded; charset=UTF-8",accepts:{"*":Jb,text:"text/plain",html:"text/html",xml:"application/xml, text/xml",json:"application/json, text/javascript"},contents:{xml:/xml/,html:/html/,json:/json/},responseFields:{xml:"responseXML",text:"responseText",json:"responseJSON"},converters:{"* text":String,"text html":!0,"text json":m.parseJSON,"text xml":m.parseXML},flatOptions:{url:!0,context:!0}},ajaxSetup:function(a,b){return b?Nb(Nb(a,m.ajaxSettings),b):Nb(m.ajaxSettings,a)},ajaxPrefilter:Lb(Hb),ajaxTransport:Lb(Ib),ajax:function(a,b){"object"==typeof a&&(b=a,a=void 0),b=b||{};var c,d,e,f,g,h,i,j,k=m.ajaxSetup({},b),l=k.context||k,n=k.context&&(l.nodeType||l.jquery)?m(l):m.event,o=m.Deferred(),p=m.Callbacks("once memory"),q=k.statusCode||{},r={},s={},t=0,u="canceled",v={readyState:0,getResponseHeader:function(a){var b;if(2===t){if(!j){j={};while(b=Cb.exec(f))j[b[1].toLowerCase()]=b[2]}b=j[a.toLowerCase()]}return null==b?null:b},getAllResponseHeaders:function(){return 2===t?f:null},setRequestHeader:function(a,b){var c=a.toLowerCase();return t||(a=s[c]=s[c]||a,r[a]=b),this},overrideMimeType:function(a){return t||(k.mimeType=a),this},statusCode:function(a){var b;if(a)if(2>t)for(b in a)q[b]=[q[b],a[b]];else v.always(a[v.status]);return this},abort:function(a){var b=a||u;return i&&i.abort(b),x(0,b),this}};if(o.promise(v).complete=p.add,v.success=v.done,v.error=v.fail,k.url=((a||k.url||zb)+"").replace(Ab,"").replace(Fb,yb[1]+"//"),k.type=b.method||b.type||k.method||k.type,k.dataTypes=m.trim(k.dataType||"*").toLowerCase().match(E)||[""],null==k.crossDomain&&(c=Gb.exec(k.url.toLowerCase()),k.crossDomain=!(!c||c[1]===yb[1]&&c[2]===yb[2]&&(c[3]||("http:"===c[1]?"80":"443"))===(yb[3]||("http:"===yb[1]?"80":"443")))),k.data&&k.processData&&"string"!=typeof k.data&&(k.data=m.param(k.data,k.traditional)),Mb(Hb,k,b,v),2===t)return v;h=m.event&&k.global,h&&0===m.active++&&m.event.trigger("ajaxStart"),k.type=k.type.toUpperCase(),k.hasContent=!Eb.test(k.type),e=k.url,k.hasContent||(k.data&&(e=k.url+=(wb.test(e)?"&":"?")+k.data,delete k.data),k.cache===!1&&(k.url=Bb.test(e)?e.replace(Bb,"$1_="+vb++):e+(wb.test(e)?"&":"?")+"_="+vb++)),k.ifModified&&(m.lastModified[e]&&v.setRequestHeader("If-Modified-Since",m.lastModified[e]),m.etag[e]&&v.setRequestHeader("If-None-Match",m.etag[e])),(k.data&&k.hasContent&&k.contentType!==!1||b.contentType)&&v.setRequestHeader("Content-Type",k.contentType),v.setRequestHeader("Accept",k.dataTypes[0]&&k.accepts[k.dataTypes[0]]?k.accepts[k.dataTypes[0]]+("*"!==k.dataTypes[0]?", "+Jb+"; q=0.01":""):k.accepts["*"]);for(d in k.headers)v.setRequestHeader(d,k.headers[d]);if(k.beforeSend&&(k.beforeSend.call(l,v,k)===!1||2===t))return v.abort();u="abort";for(d in{success:1,error:1,complete:1})v[d](k[d]);if(i=Mb(Ib,k,b,v)){v.readyState=1,h&&n.trigger("ajaxSend",[v,k]),k.async&&k.timeout>0&&(g=setTimeout(function(){v.abort("timeout")},k.timeout));try{t=1,i.send(r,x)}catch(w){if(!(2>t))throw w;x(-1,w)}}else x(-1,"No Transport");function x(a,b,c,d){var j,r,s,u,w,x=b;2!==t&&(t=2,g&&clearTimeout(g),i=void 0,f=d||"",v.readyState=a>0?4:0,j=a>=200&&300>a||304===a,c&&(u=Ob(k,v,c)),u=Pb(k,u,v,j),j?(k.ifModified&&(w=v.getResponseHeader("Last-Modified"),w&&(m.lastModified[e]=w),w=v.getResponseHeader("etag"),w&&(m.etag[e]=w)),204===a||"HEAD"===k.type?x="nocontent":304===a?x="notmodified":(x=u.state,r=u.data,s=u.error,j=!s)):(s=x,(a||!x)&&(x="error",0>a&&(a=0))),v.status=a,v.statusText=(b||x)+"",j?o.resolveWith(l,[r,x,v]):o.rejectWith(l,[v,x,s]),v.statusCode(q),q=void 0,h&&n.trigger(j?"ajaxSuccess":"ajaxError",[v,k,j?r:s]),p.fireWith(l,[v,x]),h&&(n.trigger("ajaxComplete",[v,k]),--m.active||m.event.trigger("ajaxStop")))}return v},getJSON:function(a,b,c){return m.get(a,b,c,"json")},getScript:function(a,b){return m.get(a,void 0,b,"script")}}),m.each(["get","post"],function(a,b){m[b]=function(a,c,d,e){return m.isFunction(c)&&(e=e||d,d=c,c=void 0),m.ajax({url:a,type:b,dataType:e,data:c,success:d})}}),m._evalUrl=function(a){return m.ajax({url:a,type:"GET",dataType:"script",async:!1,global:!1,"throws":!0})},m.fn.extend({wrapAll:function(a){if(m.isFunction(a))return this.each(function(b){m(this).wrapAll(a.call(this,b))});if(this[0]){var b=m(a,this[0].ownerDocument).eq(0).clone(!0);this[0].parentNode&&b.insertBefore(this[0]),b.map(function(){var a=this;while(a.firstChild&&1===a.firstChild.nodeType)a=a.firstChild;return a}).append(this)}return this},wrapInner:function(a){return this.each(m.isFunction(a)?function(b){m(this).wrapInner(a.call(this,b))}:function(){var b=m(this),c=b.contents();c.length?c.wrapAll(a):b.append(a)})},wrap:function(a){var b=m.isFunction(a);return this.each(function(c){m(this).wrapAll(b?a.call(this,c):a)})},unwrap:function(){return this.parent().each(function(){m.nodeName(this,"body")||m(this).replaceWith(this.childNodes)}).end()}}),m.expr.filters.hidden=function(a){return a.offsetWidth<=0&&a.offsetHeight<=0||!k.reliableHiddenOffsets()&&"none"===(a.style&&a.style.display||m.css(a,"display"))},m.expr.filters.visible=function(a){return!m.expr.filters.hidden(a)};var Qb=/%20/g,Rb=/\[\]$/,Sb=/\r?\n/g,Tb=/^(?:submit|button|image|reset|file)$/i,Ub=/^(?:input|select|textarea|keygen)/i;function Vb(a,b,c,d){var e;if(m.isArray(b))m.each(b,function(b,e){c||Rb.test(a)?d(a,e):Vb(a+"["+("object"==typeof e?b:"")+"]",e,c,d)});else if(c||"object"!==m.type(b))d(a,b);else for(e in b)Vb(a+"["+e+"]",b[e],c,d)}m.param=function(a,b){var c,d=[],e=function(a,b){b=m.isFunction(b)?b():null==b?"":b,d[d.length]=encodeURIComponent(a)+"="+encodeURIComponent(b)};if(void 0===b&&(b=m.ajaxSettings&&m.ajaxSettings.traditional),m.isArray(a)||a.jquery&&!m.isPlainObject(a))m.each(a,function(){e(this.name,this.value)});else for(c in a)Vb(c,a[c],b,e);return d.join("&").replace(Qb,"+")},m.fn.extend({serialize:function(){return m.param(this.serializeArray())},serializeArray:function(){return this.map(function(){var a=m.prop(this,"elements");return a?m.makeArray(a):this}).filter(function(){var a=this.type;return this.name&&!m(this).is(":disabled")&&Ub.test(this.nodeName)&&!Tb.test(a)&&(this.checked||!W.test(a))}).map(function(a,b){var c=m(this).val();return null==c?null:m.isArray(c)?m.map(c,function(a){return{name:b.name,value:a.replace(Sb,"\r\n")}}):{name:b.name,value:c.replace(Sb,"\r\n")}}).get()}}),m.ajaxSettings.xhr=void 0!==a.ActiveXObject?function(){return!this.isLocal&&/^(get|post|head|put|delete|options)$/i.test(this.type)&&Zb()||$b()}:Zb;var Wb=0,Xb={},Yb=m.ajaxSettings.xhr();a.attachEvent&&a.attachEvent("onunload",function(){for(var a in Xb)Xb[a](void 0,!0)}),k.cors=!!Yb&&"withCredentials"in Yb,Yb=k.ajax=!!Yb,Yb&&m.ajaxTransport(function(a){if(!a.crossDomain||k.cors){var b;return{send:function(c,d){var e,f=a.xhr(),g=++Wb;if(f.open(a.type,a.url,a.async,a.username,a.password),a.xhrFields)for(e in a.xhrFields)f[e]=a.xhrFields[e];a.mimeType&&f.overrideMimeType&&f.overrideMimeType(a.mimeType),a.crossDomain||c["X-Requested-With"]||(c["X-Requested-With"]="XMLHttpRequest");for(e in c)void 0!==c[e]&&f.setRequestHeader(e,c[e]+"");f.send(a.hasContent&&a.data||null),b=function(c,e){var h,i,j;if(b&&(e||4===f.readyState))if(delete Xb[g],b=void 0,f.onreadystatechange=m.noop,e)4!==f.readyState&&f.abort();else{j={},h=f.status,"string"==typeof f.responseText&&(j.text=f.responseText);try{i=f.statusText}catch(k){i=""}h||!a.isLocal||a.crossDomain?1223===h&&(h=204):h=j.text?200:404}j&&d(h,i,j,f.getAllResponseHeaders())},a.async?4===f.readyState?setTimeout(b):f.onreadystatechange=Xb[g]=b:b()},abort:function(){b&&b(void 0,!0)}}}});function Zb(){try{return new a.XMLHttpRequest}catch(b){}}function $b(){try{return new a.ActiveXObject("Microsoft.XMLHTTP")}catch(b){}}m.ajaxSetup({accepts:{script:"text/javascript, application/javascript, application/ecmascript, application/x-ecmascript"},contents:{script:/(?:java|ecma)script/},converters:{"text script":function(a){return m.globalEval(a),a}}}),m.ajaxPrefilter("script",function(a){void 0===a.cache&&(a.cache=!1),a.crossDomain&&(a.type="GET",a.global=!1)}),m.ajaxTransport("script",function(a){if(a.crossDomain){var b,c=y.head||m("head")[0]||y.documentElement;return{send:function(d,e){b=y.createElement("script"),b.async=!0,a.scriptCharset&&(b.charset=a.scriptCharset),b.src=a.url,b.onload=b.onreadystatechange=function(a,c){(c||!b.readyState||/loaded|complete/.test(b.readyState))&&(b.onload=b.onreadystatechange=null,b.parentNode&&b.parentNode.removeChild(b),b=null,c||e(200,"success"))},c.insertBefore(b,c.firstChild)},abort:function(){b&&b.onload(void 0,!0)}}}});var _b=[],ac=/(=)\?(?=&|$)|\?\?/;m.ajaxSetup({jsonp:"callback",jsonpCallback:function(){var a=_b.pop()||m.expando+"_"+vb++;return this[a]=!0,a}}),m.ajaxPrefilter("json jsonp",function(b,c,d){var e,f,g,h=b.jsonp!==!1&&(ac.test(b.url)?"url":"string"==typeof b.data&&!(b.contentType||"").indexOf("application/x-www-form-urlencoded")&&ac.test(b.data)&&"data");return h||"jsonp"===b.dataTypes[0]?(e=b.jsonpCallback=m.isFunction(b.jsonpCallback)?b.jsonpCallback():b.jsonpCallback,h?b[h]=b[h].replace(ac,"$1"+e):b.jsonp!==!1&&(b.url+=(wb.test(b.url)?"&":"?")+b.jsonp+"="+e),b.converters["script json"]=function(){return g||m.error(e+" was not called"),g[0]},b.dataTypes[0]="json",f=a[e],a[e]=function(){g=arguments},d.always(function(){a[e]=f,b[e]&&(b.jsonpCallback=c.jsonpCallback,_b.push(e)),g&&m.isFunction(f)&&f(g[0]),g=f=void 0}),"script"):void 0}),m.parseHTML=function(a,b,c){if(!a||"string"!=typeof a)return null;"boolean"==typeof b&&(c=b,b=!1),b=b||y;var d=u.exec(a),e=!c&&[];return d?[b.createElement(d[1])]:(d=m.buildFragment([a],b,e),e&&e.length&&m(e).remove(),m.merge([],d.childNodes))};var bc=m.fn.load;m.fn.load=function(a,b,c){if("string"!=typeof a&&bc)return bc.apply(this,arguments);var d,e,f,g=this,h=a.indexOf(" ");return h>=0&&(d=m.trim(a.slice(h,a.length)),a=a.slice(0,h)),m.isFunction(b)?(c=b,b=void 0):b&&"object"==typeof b&&(f="POST"),g.length>0&&m.ajax({url:a,type:f,dataType:"html",data:b}).done(function(a){e=arguments,g.html(d?m("<div>").append(m.parseHTML(a)).find(d):a)}).complete(c&&function(a,b){g.each(c,e||[a.responseText,b,a])}),this},m.each(["ajaxStart","ajaxStop","ajaxComplete","ajaxError","ajaxSuccess","ajaxSend"],function(a,b){m.fn[b]=function(a){return this.on(b,a)}}),m.expr.filters.animated=function(a){return m.grep(m.timers,function(b){return a===b.elem}).length};var cc=a.document.documentElement;function dc(a){return m.isWindow(a)?a:9===a.nodeType?a.defaultView||a.parentWindow:!1}m.offset={setOffset:function(a,b,c){var d,e,f,g,h,i,j,k=m.css(a,"position"),l=m(a),n={};"static"===k&&(a.style.position="relative"),h=l.offset(),f=m.css(a,"top"),i=m.css(a,"left"),j=("absolute"===k||"fixed"===k)&&m.inArray("auto",[f,i])>-1,j?(d=l.position(),g=d.top,e=d.left):(g=parseFloat(f)||0,e=parseFloat(i)||0),m.isFunction(b)&&(b=b.call(a,c,h)),null!=b.top&&(n.top=b.top-h.top+g),null!=b.left&&(n.left=b.left-h.left+e),"using"in b?b.using.call(a,n):l.css(n)}},m.fn.extend({offset:function(a){if(arguments.length)return void 0===a?this:this.each(function(b){m.offset.setOffset(this,a,b)});var b,c,d={top:0,left:0},e=this[0],f=e&&e.ownerDocument;if(f)return b=f.documentElement,m.contains(b,e)?(typeof e.getBoundingClientRect!==K&&(d=e.getBoundingClientRect()),c=dc(f),{top:d.top+(c.pageYOffset||b.scrollTop)-(b.clientTop||0),left:d.left+(c.pageXOffset||b.scrollLeft)-(b.clientLeft||0)}):d},position:function(){if(this[0]){var a,b,c={top:0,left:0},d=this[0];return"fixed"===m.css(d,"position")?b=d.getBoundingClientRect():(a=this.offsetParent(),b=this.offset(),m.nodeName(a[0],"html")||(c=a.offset()),c.top+=m.css(a[0],"borderTopWidth",!0),c.left+=m.css(a[0],"borderLeftWidth",!0)),{top:b.top-c.top-m.css(d,"marginTop",!0),left:b.left-c.left-m.css(d,"marginLeft",!0)}}},offsetParent:function(){return this.map(function(){var a=this.offsetParent||cc;while(a&&!m.nodeName(a,"html")&&"static"===m.css(a,"position"))a=a.offsetParent;return a||cc})}}),m.each({scrollLeft:"pageXOffset",scrollTop:"pageYOffset"},function(a,b){var c=/Y/.test(b);m.fn[a]=function(d){return V(this,function(a,d,e){var f=dc(a);return void 0===e?f?b in f?f[b]:f.document.documentElement[d]:a[d]:void(f?f.scrollTo(c?m(f).scrollLeft():e,c?e:m(f).scrollTop()):a[d]=e)},a,d,arguments.length,null)}}),m.each(["top","left"],function(a,b){m.cssHooks[b]=La(k.pixelPosition,function(a,c){return c?(c=Ja(a,b),Ha.test(c)?m(a).position()[b]+"px":c):void 0})}),m.each({Height:"height",Width:"width"},function(a,b){m.each({padding:"inner"+a,content:b,"":"outer"+a},function(c,d){m.fn[d]=function(d,e){var f=arguments.length&&(c||"boolean"!=typeof d),g=c||(d===!0||e===!0?"margin":"border");return V(this,function(b,c,d){var e;return m.isWindow(b)?b.document.documentElement["client"+a]:9===b.nodeType?(e=b.documentElement,Math.max(b.body["scroll"+a],e["scroll"+a],b.body["offset"+a],e["offset"+a],e["client"+a])):void 0===d?m.css(b,c,g):m.style(b,c,d,g)},b,f?d:void 0,f,null)}})}),m.fn.size=function(){return this.length},m.fn.andSelf=m.fn.addBack,"function"==typeof define&&define.amd&&define("jquery",[],function(){return m});var ec=a.jQuery,fc=a.$;return m.noConflict=function(b){return a.$===m&&(a.$=fc),b&&a.jQuery===m&&(a.jQuery=ec),m},typeof b===K&&(a.jQuery=a.$=m),m});
</script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<style type="text/css">html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(data:application/vnd.ms-fontobject;base64,n04AAEFNAAACAAIABAAAAAAABQAAAAAAAAABAJABAAAEAExQAAAAAAAAAAIAAAAAAAAAAAEAAAAAAAAAJxJ/LAAAAAAAAAAAAAAAAAAAAAAAACgARwBMAFkAUABIAEkAQwBPAE4AUwAgAEgAYQBsAGYAbABpAG4AZwBzAAAADgBSAGUAZwB1AGwAYQByAAAAeABWAGUAcgBzAGkAbwBuACAAMQAuADAAMAA5ADsAUABTACAAMAAwADEALgAwADAAOQA7AGgAbwB0AGMAbwBuAHYAIAAxAC4AMAAuADcAMAA7AG0AYQBrAGUAbwB0AGYALgBsAGkAYgAyAC4ANQAuADUAOAAzADIAOQAAADgARwBMAFkAUABIAEkAQwBPAE4AUwAgAEgAYQBsAGYAbABpAG4AZwBzACAAUgBlAGcAdQBsAGEAcgAAAAAAQlNHUAAAAAAAAAAAAAAAAAAAAAADAKncAE0TAE0ZAEbuFM3pjM/SEdmjKHUbyow8ATBE40IvWA3vTu8LiABDQ+pexwUMcm1SMnNryctQSiI1K5ZnbOlXKmnVV5YvRe6RnNMFNCOs1KNVpn6yZhCJkRtVRNzEufeIq7HgSrcx4S8h/v4vnrrKc6oCNxmSk2uKlZQHBii6iKFoH0746ThvkO1kJHlxjrkxs+LWORaDQBEtiYJIR5IB9Bi1UyL4Rmr0BNigNkMzlKQmnofBHviqVzUxwdMb3NdCn69hy+pRYVKGVS/1tnsqv4LL7wCCPZZAZPT4aCShHjHJVNuXbmMrY5LeQaGnvAkXlVrJgKRAUdFjrWEah9XebPeQMj7KS7DIBAFt8ycgC5PLGUOHSE3ErGZCiViNLL5ZARfywnCoZaKQCu6NuFX42AEeKtKUGnr/Cm2Cy8tpFhBPMW5Fxi4Qm4TkDWh4IWFDClhU2hRWosUWqcKLlgyXB+lSHaWaHiWlBAR8SeSgSPCQxdVQgzUixWKSTrIQEbU94viDctkvX+VSjJuUmV8L4CXShI11esnp0pjWNZIyxKHS4wVQ2ime1P4RnhvGw0aDN1OLAXGERsB7buFpFGGBAre4QEQR0HOIO5oYH305G+KspT/FupEGGafCCwxSe6ZUa+073rXHnNdVXE6eWvibUS27XtRzkH838mYLMBmYysZTM0EM3A1fbpCBYFccN1B/EnCYu/TgCGmr7bMh8GfYL+BfcLvB0gRagC09w9elfldaIy/hNCBLRgBgtCC7jAF63wLSMAfbfAlEggYU0bUA7ACCJmTDpEmJtI78w4/BO7dN7JR7J7ZvbYaUbaILSQsRBiF3HGk5fEg6p9unwLvn98r+vnsV+372uf1xBLq4qU/45fTuqaAP+pssmCCCTF0mhEow8ZXZOS8D7Q85JsxZ+Azok7B7O/f6J8AzYBySZQB/QHYUSA+EeQhEWiS6AIQzgcsDiER4MjgMBAWDV4AgQ3g1eBgIdweCQmCjJEMkJ+PKRWyFHHmg1Wi/6xzUgA0LREoKJChwnQa9B+5RQZRB3IlBlkAnxyQNaANwHMowzlYSMCBgnbpzvqpl0iTJNCQidDI9ZrSYNIRBhHtUa5YHMHxyGEik9hDE0AKj72AbTCaxtHPUaKZdAZSnQTyjGqGLsmBStCejApUhg4uBMU6mATujEl+KdDPbI6Ag4vLr+hjY6lbjBeoLKnZl0UZgRX8gTySOeynZVz1wOq7e1hFGYIq+MhrGxDLak0PrwYzSXtcuyhXEhwOYofiW+EcI/jw8P6IY6ed+etAbuqKp5QIapT77LnAe505lMuqL79a0ut4rWexzFttsOsLDy7zvtQzcq3U1qabe7tB0wHWVXji+zDbo8x8HyIRUbXnwUcklFv51fvTymiV+MXLSmGH9d9+aXpD5X6lao41anWGig7IwIdnoBY2ht/pO9mClLo4NdXHAsefqWUKlXJkbqPOFhMoR4aiA1BXqhRNbB2Xwi+7u/jpAoOpKJ0UX24EsrzMfHXViakCNcKjBxuQX8BO0ZqjJ3xXzf+61t2VXOSgJ8xu65QKgtN6FibPmPYsXbJRHHqbgATcSZxBqGiDiU4NNNsYBsKD0MIP/OfKnlk/Lkaid/O2NbKeuQrwOB2Gq3YHyr6ALgzym5wIBnsdC1ZkoBFZSQXChZvlesPqvK2c5oHHT3Q65jYpNxnQcGF0EHbvYqoFw60WNlXIHQF2HQB7zD6lWjZ9rVqUKBXUT6hrkZOle0RFYII0V5ZYGl1JAP0Ud1fZZMvSomBzJ710j4Me8mjQDwEre5Uv2wQfk1ifDwb5ksuJQQ3xt423lbuQjvoIQByQrNDh1JxGFkOdlJvu/gFtuW0wR4cgd+ZKesSV7QkNE2kw6AV4hoIuC02LGmTomyf8PiO6CZzOTLTPQ+HW06H+tx+bQ8LmDYg1pTFrp2oJXgkZTyeRJZM0C8aE2LpFrNVDuhARsN543/FV6klQ6Tv1OoZGXLv0igKrl/CmJxRmX7JJbJ998VSIPQRyDBICzl4JJlYHbdql30NvYcOuZ7a10uWRrgoieOdgIm4rlq6vNOQBuqESLbXG5lzdJGHw2m0sDYmODXbYGTfSTGRKpssTO95fothJCjUGQgEL4yKoGAF/0SrpUDNn8CBgBcSDQByAeNkCXp4S4Ro2Xh4OeaGRgR66PVOsU8bc6TR5/xTcn4IVMLOkXSWiXxkZQCbvKfmoAvQaKjO3EDKwkwqHChCDEM5loQRPd5ACBki1TjF772oaQhQbQ5C0lcWXPFOzrfsDGUXGrpxasbG4iab6eByaQkQfm0VFlP0ZsDkvvqCL6QXMUwCjdMx1ZOyKhTJ7a1GWAdOUcJ8RSejxNVyGs31OKMyRyBVoZFjqIkmKlLQ5eHMeEL4MkUf23cQ/1SgRCJ1dk4UdBT7OoyuNgLs0oCd8RnrEIb6QdMxT2QjD4zMrJkfgx5aDMcA4orsTtKCqWb/Veyceqa5OGSmB28YwH4rFbkQaLoUN8OQQYnD3w2eXpI4ScQfbCUZiJ4yMOIKLyyTc7BQ4uXUw6Ee6/xM+4Y67ngNBknxIPwuppgIhFcwJyr6EIj+LzNj/mfR2vhhRlx0BILZoAYruF0caWQ7YxO66UmeguDREAFHYuC7HJviRgVO6ruJH59h/C/PkgSle8xNzZJULLWq9JMDTE2fjGE146a1Us6PZDGYle6ldWRqn/pdpgHKNGrGIdkRK+KPETT9nKT6kLyDI8xd9A1FgWmXWRAIHwZ37WyZHOVyCadJEmMVz0MadMjDrPho+EIochkVC2xgGiwwsQ6DMv2P7UXqT4x7CdcYGId2BJQQa85EQKmCmwcRejQ9Bm4oATENFPkxPXILHpMPUyWTI5rjNOsIlmEeMbcOCEqInpXACYQ9DDxmFo9vcmsDblcMtg4tqBerNngkIKaFJmrQAPnq1dEzsMXcwjcHdfdCibcAxxA+q/j9m3LM/O7WJka4tSidVCjsvo2lQ/2ewyoYyXwAYyr2PlRoR5MpgVmSUIrM3PQxXPbgjBOaDQFIyFMJvx3Pc5RSYj12ySVF9fwFPQu2e2KWVoL9q3Ayv3IzpGHUdvdPdrNUdicjsTQ2ISy7QU3DrEytIjvbzJnAkmANXjAFERA0MUoPF3/5KFmW14bBNOhwircYgMqoDpUMcDtCmBE82QM2YtdjVLB4kBuKho/bcwQdeboqfQartuU3CsCf+cXkgYAqp/0Ee3RorAZt0AvvOCSI4JICIlGlsV0bsSid/NIEALAAzb6HAgyWHBps6xAOwkJIGcB82CxRQq4sJf3FzA70A+TRqcqjEMETCoez3mkPcpnoALs0ugJY8kQwrC+JE5ik3w9rzrvDRjAQnqgEVvdGrNwlanR0SOKWzxOJOvLJhcd8Cl4AshACUkv9czdMkJCVQSQhp6kp7StAlpVRpK0t0SW6LHeBJnE2QchB5Ccu8kxRghZXGIgZIiSj7gEKMJDClcnX6hgoqJMwiQDigIXg3ioFLCgDgjPtYHYpsF5EiA4kcnN18MZtOrY866dEQAb0FB34OGKHGZQjwW/WDHA60cYFaI/PjpzquUqdaYGcIq+mLez3WLFFCtNBN2QJcrlcoELgiPku5R5dSlJFaCEqEZle1AQzAKC+1SotMcBNyQUFuRHRF6OlimSBgjZeTBCwLyc6A+P/oFRchXTz5ADknYJHxzrJ5pGuIKRQISU6WyKTBBjD8WozmVYWIsto1AS5rxzKlvJu4E/vwOiKxRtCWsDM+eTHUrmwrCK5BIfMzGkD+0Fk5LzBs0jMYXktNDblB06LMNJ09U8pzSLmo14MS0OMjcdrZ31pyQqxJJpRImlSvfYAK8inkYU52QY2FPEVsjoWewpwhRp5yAuNpkqhdb7ku9Seefl2D0B8SMTFD90xi4CSOwwZy9IKkpMtI3FmFUg3/kFutpQGNc3pCR7gvC4sgwbupDu3DyEN+W6YGLNM21jpB49irxy9BSlHrVDlnihGKHwPrbVFtc+h1rVQKZduxIyojccZIIcOCmhEnC7UkY68WXKQgLi2JCDQkQWJRQuk60hZp0D3rtCTINSeY9Ej2kIKYfGxwOs4j9qMM7fYZiipzgcf7TamnehqdhsiMiCawXnz4xAbyCkLAx5EGbo3Ax1u3dUIKnTxIaxwQTHehPl3V491H0+bC5zgpGz7Io+mjdhKlPJ01EeMpM7UsRJMi1nGjmJg35i6bQBAAxjO/ENJubU2mg3ONySEoWklCwdABETcs7ck3jgiuU9pcKKpbgn+3YlzV1FzIkB6pmEDOSSyDfPPlQskznctFji0kpgZjW5RZe6x9kYT4KJcXg0bNiCyif+pZACCyRMmYsfiKmN9tSO65F0R2OO6ytlEhY5Sj6uRKfFxw0ijJaAx/k3QgnAFSq27/2i4GEBA+UvTJKK/9eISNvG46Em5RZfjTYLdeD8kdXHyrwId/DQZUaMCY4gGbke2C8vfjgV/Y9kkRQOJIn/xM9INZSpiBnqX0Q9GlQPpPKAyO5y+W5NMPSRdBCUlmuxl40ZfMCnf2Cp044uI9WLFtCi4YVxKjuRCOBWIb4XbIsGdbo4qtMQnNOQz4XDSui7W/N6l54qOynCqD3DpWQ+mpD7C40D8BZEWGJX3tlAaZBMj1yjvDYKwCJBa201u6nBKE5UE+7QSEhCwrXfbRZylAaAkplhBWX50dumrElePyNMRYUrC99UmcSSNgImhFhDI4BXjMtiqkgizUGCrZ8iwFxU6fQ8GEHCFdLewwxYWxgScAYMdMLmcZR6b7rZl95eQVDGVoUKcRMM1ixXQtXNkBETZkVVPg8LoSrdetHzkuM7DjZRHP02tCxA1fmkXKF3VzfN1pc1cv/8lbTIkkYpqKM9VOhp65ktYk+Q46myFWBapDfyWUCnsnI00QTBQmuFjMZTcd0V2NQ768Fhpby04k2IzNR1wKabuGJqYWwSly6ocMFGTeeI+ejsWDYgEvr66QgqdcIbFYDNgsm0x9UHY6SCd5+7tpsLpKdvhahIDyYmEJQCqMqtCF6UlrE5GXRmbu+vtm3BFSxI6ND6UxIE7GsGMgWqghXxSnaRJuGFveTcK5ZVSPJyjUxe1dKgI6kNF7EZhIZs8y8FVqwEfbM0Xk2ltORVDKZZM40SD3qQoQe0orJEKwPfZwm3YPqwixhUMOndis6MhbmfvLBKjC8sKKIZKbJk8L11oNkCQzCgvjhyyEiQSuJcgCQSG4Mocfgc0Hkwcjal1UNgP0CBPikYqBIk9tONv4kLtBswH07vUCjEaHiFGlLf8MgXKzSgjp2HolRRccAOh0ILHz9qlGgIFkwAnzHJRjWFhlA7ROwINyB5HFj59PRZHFor6voq7l23EPNRwdWhgawqbivLSjRA4htEYUFkjESu67icTg5S0aW1sOkCiIysfJ9UnIWevOOLGpepcBxy1wEhd2WI3AZg7sr9WBmHWyasxMcvY/iOmsLtHSWNUWEGk9hScMPShasUA1AcHOtRZlqMeQ0OzYS9vQvYUjOLrzP07BUAFikcJNMi7gIxEw4pL1G54TcmmmoAQ5s7TGWErJZ2Io4yQ0ljRYhL8H5e62oDtLF8aDpnIvZ5R3GWJyAugdiiJW9hQAVTsnCBHhwu7rkBlBX6r3b7ejEY0k5GGeyKv66v+6dg7mcJTrWHbtMywbedYqCQ0FPwoytmSWsL8WTtChZCKKzEF7vP6De4x2BJkkniMgSdWhbeBSLtJZR9CTHetK1xb34AYIJ37OegYIoPVbXgJ/qDQK+bfCtxQRVKQu77WzOoM6SGL7MaZwCGJVk46aImai9fmam+WpHG+0BtQPWUgZ7RIAlPq6lkECUhZQ2gqWkMYKcYMYaIc4gYCDFHYa2d1nzp3+J1eCBay8IYZ0wQRKGAqvCuZ/UgbQPyllosq+XtfKIZOzmeJqRazpmmoP/76YfkjzV2NlXTDSBYB04SVlNQsFTbGPk1t/I4Jktu0XSgifO2ozFOiwd/0SssJDn0dn4xqk4GDTTKX73/wQyBLdqgJ+Wx6AQaba3BA9CKEzjtQYIfAsiYamapq80LAamYjinlKXUkxdpIDk0puXUEYzSalfRibAeDAKpNiqQ0FTwoxuGYzRnisyTotdVTclis1LHRQCy/qqL8oUaQzWRxilq5Mi0IJGtMY02cGLD69vGjkj3p6pGePKI8bkBv5evq8SjjyU04vJR2cQXQwSJyoinDsUJHCQ50jrFTT7yRdbdYQMB3MYCb6uBzJ9ewhXYPAIZSXfeEQBZZ3GPN3Nbhh/wkvAJLXnQMdi5NYYZ5GHE400GS5rXkOZSQsdZgIbzRnF9ueLnsfQ47wHAsirITnTlkCcuWWIUhJSbpM3wWhXNHvt2xUsKKMpdBSbJnBMcihkoDqAd1Zml/R4yrzow1Q2A5G+kzo/RhRxQS2lCSDRV8LlYLBOOoo1bF4jwJAwKMK1tWLHlu9i0j4Ig8qVm6wE1DxXwAwQwsaBWUg2pOOol2dHxyt6npwJEdLDDVYyRc2D0HbcbLUJQj8gPevQBUBOUHXPrsAPBERICpnYESeu2OHotpXQxRGlCCtLdIsu23MhZVEoJg8Qumj/UMMc34IBqTKLDTp76WzL/dMjCxK7MjhiGjeYAC/kj/jY/Rde7hpSM1xChrog6yZ7OWTuD56xBJnGFE+pT2ElSyCnJcwVzCjkqeNLfMEJqKW0G7OFIp0G+9mh50I9o8k1tpCY0xYqFNIALgIfc2me4n1bmJnRZ89oepgLPT0NTMLNZsvSCZAc3TXaNB07vail36/dBySis4m9/DR8izaLJW6bWCkVgm5T+ius3ZXq4xI+GnbveLbdRwF2mNtsrE0JjYc1AXknCOrLSu7Te/r4dPYMCl5qtiHNTn+TPbh1jCBHH+dMJNhwNgs3nT+OhQoQ0vYif56BMG6WowAcHR3DjQolxLzyVekHj00PBAaW7IIAF1EF+uRIWyXjQMAs2chdpaKPNaB+kSezYt0+CA04sOg5vx8Fr7Ofa9sUv87h7SLAUFSzbetCCZ9pmyLt6l6/TzoA1/ZBG9bIUVHLAbi/kdBFgYGyGwRQGBpkqCEg2ah9UD6EedEcEL3j4y0BQQCiExEnocA3SZboh+epgd3YsOkHskZwPuQ5OoyA0fTA5AXrHcUOQF+zkJHIA7PwCDk1gGVmGUZSSoPhNf+Tklauz98QofOlCIQ/tCD4dosHYPqtPCXB3agggQQIqQJsSkB+qn0rkQ1toJjON/OtCIB9RYv3PqRA4C4U68ZMlZn6BdgEvi2ziU+TQ6NIw3ej+AtDwMGEZk7e2IjxUWKdAxyaw9OCwSmeADTPPleyk6UhGDNXQb++W6Uk4q6F7/rg6WVTo82IoCxSIsFDrav4EPHphD3u4hR53WKVvYZUwNCCeM4PMBWzK+EfIthZOkuAwPo5C5jgoZgn6dUdvx5rIDmd58cXXdKNfw3l+wM2UjgrDJeQHhbD7HW2QDoZMCujgIUkk5Fg8VCsdyjOtnGRx8wgKRPZN5dR0zPUyfGZFVihbFRniXZFOZGKPnEQzU3AnD1KfR6weHW2XS6KbPJxUkOTZsAB9vTVp3Le1F8q5l+DMcLiIq78jxAImD2pGFw0VHfRatScGlK6SMu8leTmhUSMy8Uhdd6xBiH3Gdman4tjQGLboJfqz6fL2WKHTmrfsKZRYX6BTDjDldKMosaSTLdQS7oDisJNqAUhw1PfTlnacCO8vl8706Km1FROgLDmudzxg+EWTiArtHgLsRrAXYWdB0NmToNCJdKm0KWycZQqb+Mw76Qy29iQ5up/X7oyw8QZ75kP5F6iJAJz6KCmqxz8fEa/xnsMYcIO/vEkGRuMckhr4rIeLrKaXnmIzlNLxbFspOphkcnJdnz/Chp/Vlpj2P7jJQmQRwGnltkTV5dbF9fE3/fxoSqTROgq9wFUlbuYzYcasE0ouzBo+dDCDzxKAfhbAZYxQiHrLzV2iVexnDX/QnT1fsT/xuhu1ui5qIytgbGmRoQkeQooO8eJNNZsf0iALur8QxZFH0nCMnjerYQqG1pIfjyVZWxhVRznmmfLG00BcBWJE6hzQWRyFknuJnXuk8A5FRDCulwrWASSNoBtR+CtGdkPwYN2o7DOw/VGlCZPusRBFXODQdUM5zeHDIVuAJBLqbO/f9Qua+pDqEPk230Sob9lEZ8BHiCorjVghuI0lI4JDgHGRDD/prQ84B1pVGkIpVUAHCG+iz3Bn3qm2AVrYcYWhock4jso5+J7HfHVj4WMIQdGctq3psBCVVzupQOEioBGA2Bk+UILT7+VoX5mdxxA5fS42gISQVi/HTzrgMxu0fY6hE1ocUwwbsbWcezrY2n6S8/6cxXkOH4prpmPuFoikTzY7T85C4T2XYlbxLglSv2uLCgFv8Quk/wdesUdWPeHYIH0R729JIisN9Apdd4eB10aqwXrPt+Su9mA8k8n1sjMwnfsfF2j3jMUzXepSHmZ/BfqXvzgUNQQWOXO8YEuFBh4QTYCkOAPxywpYu1VxiDyJmKVcmJPGWk/gc3Pov02StyYDahwmzw3E1gYC9wkupyWfDqDSUMpCTH5e5N8B//lHiMuIkTNw4USHrJU67bjXGqNav6PBuQSoqTxc8avHoGmvqNtXzIaoyMIQIiiUHIM64cXieouplhNYln7qgc4wBVAYR104kO+CvKqsg4yIUlFNThVUAKZxZt1XA34h3TCUUiXVkZ0w8Hh2R0Z5L0b4LZvPd/p1gi/07h8qfwHrByuSxglc9cI4QIg2oqvC/qm0i7tjPLTgDhoWTAKDO2ONW5oe+/eKB9vZB8K6C25yCZ9RFVMnb6NRdRjyVK57CHHSkJBfnM2/j4ODUwRkqrtBBCrDsDpt8jhZdXoy/1BCqw3sSGhgGGy0a5Jw6BP/TExoCmNFYjZl248A0osgPyGEmRA+fAsqPVaNAfytu0vuQJ7rk3J4kTDTR2AlCHJ5cls26opZM4w3jMULh2YXKpcqGBtuleAlOZnaZGbD6DHzMd6i2oFeJ8z9XYmalg1Szd/ocZDc1C7Y6vcALJz2lYnTXiWEr2wawtoR4g3jvWUU2Ngjd1cewtFzEvM1NiHZPeLlIXFbBPawxNgMwwAlyNSuGF3zizVeOoC9bag1qRAQKQE/EZBWC2J8mnXAN2aTBboZ7HewnObE8CwROudZHmUM5oZ/Ugd/JZQK8lvAm43uDRAbyW8gZ+ZGq0EVerVGUKUSm/Idn8AQHdR4m7bue88WBwft9mSCeMOt1ncBwziOmJYI2ZR7ewNMPiCugmSsE4EyQ+QATJG6qORMGd4snEzc6B4shPIo4G1T7PgSm8PY5eUkPdF8JZ0VBtadbHXoJgnEhZQaODPj2gpODKJY5Yp4DOsLBFxWbvXN755KWylJm+oOd4zEL9Hpubuy2gyyfxh8oEfFutnYWdfB8PdESLWYvSqbElP9qo3u6KTmkhoacDauMNNjj0oy40DFV7Ql0aZj77xfGl7TJNHnIwgqOkenruYYNo6h724+zUQ7+vkCpZB+pGA562hYQiDxHVWOq0oDQl/QsoiY+cuI7iWq/ZIBtHcXJ7kks+h2fCNUPA82BzjnqktNts+RLdk1VSu+tqEn7QZCCsvEqk6FkfiOYkrsw092J8jsfIuEKypNjLxrKA9kiA19mxBD2suxQKCzwXGws7kEJvlhUiV9tArLIdZW0IORcxEzdzKmjtFhsjKy/44XYXdI5noQoRcvjZ1RMPACRqYg2V1+OwOepcOknRLLFdYgTkT5UApt/JhLM3jeFYprZV+Zow2g8fP+U68hkKFWJj2yBbKqsrp25xkZX1DAjUw52IMYWaOhab8Kp05VrdNftqwRrymWF4OQSjbdfzmRZirK8FMJELEgER2PHjEAN9pGfLhCUiTJFbd5LBkOBMaxLr/A1SY9dXFz4RjzoU9ExfJCmx/I9FKEGT3n2cmzl2X42L3Jh+AbQq6sA+Ss1kitoa4TAYgKHaoybHUDJ51oETdeI/9ThSmjWGkyLi5QAGWhL0BG1UsTyRGRJOldKBrYJeB8ljLJHfATWTEQBXBDnQexOHTB+Un44zExFE4vLytcu5NwpWrUxO/0ZICUGM7hGABXym0V6ZvDST0E370St9MIWQOTWngeoQHUTdCJUP04spMBMS8LSker9cReVQkULFDIZDFPrhTzBl6sed9wcZQTbL+BDqMyaN3RJPh/anbx+Iv+qgQdAa3M9Z5JmvYlh4qop+Ho1F1W5gbOE9YKLgAnWytXElU4G8GtW47lhgFE6gaSs+gs37sFvi0PPVvA5dnCBgILTwoKd/+DoL9F6inlM7H4rOTzD79KJgKlZO/Zgt22UsKhrAaXU5ZcLrAglTVKJEmNJvORGN1vqrcfSMizfpsgbIe9zno+gBoKVXgIL/VI8dB1O5o/R3Suez/gD7M781ShjKpIIORM/nxG+jjhhgPwsn2IoXsPGPqYHXA63zJ07M2GPEykQwJBYLK808qYxuIew4frk52nhCsnCYmXiR6CuapvE1IwRB4/QftDbEn+AucIr1oxrLabRj9q4ae0+fXkHnteAJwXRbVkR0mctVSwEbqhJiMSZUp9DNbEDMmjX22m3ABpkrPQQTP3S1sib5pD2VRKRd+eNAjLYyT0hGrdjWJZy24OYXRoWQAIhGBZRxuBFMjjZQhpgrWo8SiFYbojcHO8V5DyscJpLTHyx9Fimassyo5U6WNtquUMYgccaHY5amgR3PQzq3ToNM5ABnoB9kuxsebqmYZm0R9qxJbFXCQ1UPyFIbxoUraTJFDpCk0Wk9GaYJKz/6oHwEP0Q14lMtlddQsOAU9zlYdMVHiT7RQP3XCmWYDcHCGbVRHGnHuwzScA0BaSBOGkz3lM8CArjrBsyEoV6Ys4qgDK3ykQQPZ3hCRGNXQTNNXbEb6tDiTDLKOyMzRhCFT+mAUmiYbV3YQVqFVp9dorv+TsLeCykS2b5yyu8AV7IS9cxcL8z4Kfwp+xJyYLv1OsxQCZwTB4a8BZ/5EdxTBJthApqyfd9u3ifr/WILTqq5VqgwMT9SOxbSGWLQJUUWCVi4k9tho9nEsbUh7U6NUsLmkYFXOhZ0kmamaJLRNJzSj/qn4Mso6zb6iLLBXoaZ6AqeWCjHQm2lztnejYYM2eubnpBdKVLORZhudH3JF1waBJKA9+W8EhMj3Kzf0L4vi4k6RoHh3Z5YgmSZmk6ns4fjScjAoL8GoOECgqgYEBYUGFVO4FUv4/YtowhEmTs0vrvlD/CrisnoBNDAcUi/teY7OctFlmARQzjOItrrlKuPO6E2Ox93L4O/4DcgV/dZ7qR3VBwVQxP1GCieA4RIpweYJ5FoYrHxqRBdJjnqbsikA2Ictbb8vE1GYIo9dacK0REgDX4smy6GAkxlH1yCGGsk+tgiDhNKuKu3yNrMdxafmKTF632F8Vx4BNK57GvlFisrkjN9WDAtjsWA0ENT2e2nETUb/n7qwhvGnrHuf5bX6Vh/n3xffU3PeHdR+FA92i6ufT3AlyAREoNDh6chiMWTvjKjHDeRhOa9YkOQRq1vQXEMppAQVwHCuIcV2g5rBn6GmZZpTR7vnSD6ZmhdSl176gqKTXu5E+YbfL0adwNtHP7dT7t7b46DVZIkzaRJOM+S6KcrzYVg+T3wSRFRQashjfU18NutrKa/7PXbtuJvpIjbgPeqd+pjmRw6YKpnANFSQcpzTZgpSNJ6J7uiagAbir/8tNXJ/OsOnRh6iuIexxrmkIneAgz8QoLmiaJ8sLQrELVK2yn3wOHp57BAZJhDZjTBzyoRAuuZ4eoxHruY1pSb7qq79cIeAdOwin4GdgMeIMHeG+FZWYaiUQQyC5b50zKjYw97dFjAeY2I4Bnl105Iku1y0lMA1ZHolLx19uZnRdILcXKlZGQx/GdEqSsMRU1BIrFqRcV1qQOOHyxOLXEGcbRtAEsuAC2V4K3p5mFJ22IDWaEkk9ttf5Izb2LkD1MnrSwztXmmD/Qi/EmVEFBfiKGmftsPwVaIoZanlKndMZsIBOskFYpDOq3QUs9aSbAAtL5Dbokus2G4/asthNMK5UQKCOhU97oaOYNGsTah+jfCKsZnTRn5TbhFX8ghg8CBYt/BjeYYYUrtUZ5jVij/op7V5SsbA4mYTOwZ46hqdpbB6Qvq3AS2HHNkC15pTDIcDNGsMPXaBidXYPHc6PJAkRh29Vx8KcgX46LoUQBhRM+3SW6Opll/wgxxsPgKJKzr5QCmwkUxNbeg6Wj34SUnEzOemSuvS2OetRCO8Tyy+QbSKVJcqkia+GvDefFwMOmgnD7h81TUtMn+mRpyJJ349HhAnoWFTejhpYTL9G8N2nVg1qkXBeoS9Nw2fB27t7trm7d/QK7Cr4uoCeOQ7/8JfKT77KiDzLImESHw/0wf73QeHu74hxv7uihi4fTX+XEwAyQG3264dwv17aJ5N335Vt9sdrAXhPOAv8JFvzqyYXwfx8WYJaef1gMl98JRFyl5Mv5Uo/oVH5ww5OzLFsiTPDns7fS6EURSSWd/92BxMYQ8sBaH+j+wthQPdVgDGpTfi+JQIWMD8xKqULliRH01rTeyF8x8q/GBEEEBrAJMPf25UQwi0b8tmqRXY7kIvNkzrkvRWLnxoGYEJsz8u4oOyMp8cHyaybb1HdMCaLApUE+/7xLIZGP6H9xuSEXp1zLIdjk5nBaMuV/yTDRRP8Y2ww5RO6d2D94o+6ucWIqUAvgHIHXhZsmDhjVLczmZ3ca0Cb3PpKwt2UtHVQ0BgFJsqqTsnzZPlKahRUkEu4qmkJt+kqdae76ViWe3STan69yaF9+fESD2lcQshLHWVu4ovItXxO69bqC5p1nZLvI8NdQB9s9UNaJGlQ5mG947ipdDA0eTIw/A1zEdjWquIsQXXGIVEH0thC5M+W9pZe7IhAVnPJkYCCXN5a32HjN6nsvokEqRS44tGIs7s2LVTvcrHAF+RVmI8L4HUYk4x+67AxSMJKqCg8zrGOgvK9kNMdDrNiUtSWuHFpC8/p5qIQrEo/H+1l/0cAwQ2nKmpWxKcMIuHY44Y6DlkpO48tRuUGBWT0FyHwSKO72Ud+tJUfdaZ4CWNijzZtlRa8+CkmO/EwHYfPZFU/hzjFWH7vnzHRMo+aF9u8qHSAiEkA2HjoNQPEwHsDKOt6hOoK3Ce/+/9boMWDa44I6FrQhdgS7OnNaSzwxWKZMcyHi6LN4WC6sSj0qm2PSOGBTvDs/GWJS6SwEN/ULwpb4LQo9fYjUfSXRwZkynUazlSpvX9e+G2zor8l+YaMxSEomDdLHGcD6YVQPegTaA74H8+V4WvJkFUrjMLGLlvSZQWvi8/QA7yzQ8GPno//5SJHRP/OqKObPCo81s/+6WgLqykYpGAgQZhVDEBPXWgU/WzFZjKUhSFInufPRiMAUULC6T11yL45ZrRoB4DzOyJShKXaAJIBS9wzLYIoCEcJKQW8GVCx4fihqJ6mshBUXSw3wWVj3grrHQlGNGhIDNNzsxQ3M+GWn6ASobIWC+LbYOC6UpahVO13Zs2zOzZC8z7FmA05JhUGyBsF4tsG0drcggIFzgg/kpf3+CnAXKiMgIE8Jk/Mhpkc8DUJEUzDSnWlQFme3d0sHZDrg7LavtsEX3cHwjCYA17pMTfx8Ajw9hHscN67hyo+RJQ4458RmPywXykkVcW688oVUrQhahpPRvTWPnuI0B+SkQu7dCyvLRyFYlC1LG1gRCIvn3rwQeINzZQC2KXq31FaR9UmVV2QeGVqBHjmE+VMd3b1fhCynD0pQNhCG6/WCDbKPyE7NRQzL3BzQAJ0g09aUzcQA6mUp9iZFK6Sbp/YbHjo++7/Wj8S4YNa+ZdqAw1hDrKWFXv9+zaXpf8ZTDSbiqsxnwN/CzK5tPkOr4tRh2kY3Bn9JtalbIOI4b3F7F1vPQMfoDcdxMS8CW9m/NCW/HILTUVWQIPiD0j1A6bo8vsv6P1hCESl2abrSJWDrq5sSzUpwoxaCU9FtJyYH4QFMxDBpkkBR6kn0LMPO+5EJ7Z6bCiRoPedRZ/P0SSdii7ZnPAtVwwHUidcdyspwncz5uq6vvm4IEDbJVLUFCn/LvIHfooUBTkFO130FC7CmmcrKdgDJcid9mvVzsDSibOoXtIf9k6ABle3PmIxejodc4aob0QKS432srrCMndbfD454q52V01G4q913mC5HOsTzWF4h2No1av1VbcUgWAqyoZl+11PoFYnNv2HwAODeNRkHj+8SF1fcvVBu6MrehHAZK1Gm69ICcTKizykHgGFx7QdowTVAsYEF2tVc0Z6wLryz2FI1sc5By2znJAAmINndoJiB4sfPdPrTC8RnkW7KRCwxC6YvXg5ahMlQuMpoCSXjOlBy0Kij+bsCYPbGp8BdCBiLmLSAkEQRaieWo1SYvZIKJGj9Ur/eWHjiB7SOVdqMAVmpBvfRiebsFjger7DC+8kRFGtNrTrnnGD2GAJb8rQCWkUPYHhwXsjNBSkE6lGWUj5QNhK0DMNM2l+kXRZ0KLZaGsFSIdQz/HXDxf3/TE30+DgBKWGWdxElyLccJfEpjsnszECNoDGZpdwdRgCixeg9L4EPhH+RptvRMVRaahu4cySjS3P5wxAUCPkmn+rhyASpmiTaiDeggaIxYBmtLZDDhiWIJaBgzfCsAGUF1Q1SFZYyXDt9skCaxJsxK2Ms65dmdp5WAZyxik/zbrTQk5KmgxCg/f45L0jywebOWUYFJQAJia7XzCV0x89rpp/f3AVWhSPyTanqmik2SkD8A3Ml4NhIGLAjBXtPShwKYfi2eXtrDuKLk4QlSyTw1ftXgwqA2jUuopDl+5tfUWZNwBpEPXghzbBggYCw/dhy0ntds2yeHCDKkF/YxQjNIL/F/37jLPHCKBO9ibwYCmuxImIo0ijV2Wbg3kSN2psoe8IsABv3RNFaF9uMyCtCYtqcD+qNOhwMlfARQUdJ2tUX+MNJqOwIciWalZsmEjt07tfa8ma4cji9sqz+Q9hWfmMoKEbIHPOQORbhQRHIsrTYlnVTNvcq1imqmmPDdVDkJgRcTgB8Sb6epCQVmFZe+jGDiNJQLWnfx+drTKYjm0G8yH0ZAGMWzEJhUEQ4Maimgf/bkvo8PLVBsZl152y5S8+HRDfZIMCbYZ1WDp4yrdchOJw8k6R+/2pHmydK4NIK2PHdFPHtoLmHxRDwLFb7eB+M4zNZcB9NrAgjVyzLM7xyYSY13ykWfIEEd2n5/iYp3ZdrCf7fL+en+sIJu2W7E30MrAgZBD1rAAbZHPgeAMtKCg3NpSpYQUDWJu9bT3V7tOKv+NRiJc8JAKqqgCA/PNRBR7ChpiEulyQApMK1AyqcWnpSOmYh6yLiWkGJ2mklCSPIqN7UypWj3dGi5MvsHQ87MrB4VFgypJaFriaHivwcHIpmyi5LhNqtem4q0n8awM19Qk8BOS0EsqGscuuydYsIGsbT5GHnERUiMpKJl4ON7qjB4fEqlGN/hCky89232UQCiaeWpDYCJINXjT6xl4Gc7DxRCtgV0i1ma4RgWLsNtnEBRQFqZggCLiuyEydmFd7WlogpkCw5G1x4ft2psm3KAREwVwr1Gzl6RT7FDAqpVal34ewVm3VH4qn5mjGj+bYL1NgfLNeXDwtmYSpwzbruDKpTjOdgiIHDVQSb5/zBgSMbHLkxWWgghIh9QTFSDILixVwg0Eg1puooBiHAt7DzwJ7m8i8/i+jHvKf0QDnnHVkVTIqMvIQImOrzCJwhSR7qYB5gSwL6aWL9hERHCZc4G2+JrpgHNB8eCCmcIWIQ6rSdyPCyftXkDlErUkHafHRlkOIjxGbAktz75bnh50dU7YHk+Mz7wwstg6RFZb+TZuSOx1qqP5C66c0mptQmzIC2dlpte7vZrauAMm/7RfBYkGtXWGiaWTtwvAQiq2oD4YixPLXE2khB2FRaNRDTk+9sZ6K74Ia9VntCpN4BhJGJMT4Z5c5FhSepRCRWmBXqx+whVZC4me4saDs2iNqXMuCl6iAZflH8fscC1sTsy4PHeC+XYuqMBMUun5YezKbRKmEPwuK+CLzijPEQgfhahQswBBLfg/GBgBiI4QwAqzJkkyYAWtjzSg2ILgMAgqxYfwERRo3zruBL9WOryUArSD8sQOcD7fvIODJxKFS615KFPsb68USBEPPj1orNzFY2xoTtNBVTyzBhPbhFH0PI5AtlJBl2aSgNPYzxYLw7XTDBDinmVoENwiGzmngrMo8OmnRP0Z0i0Zrln9DDFcnmOoBZjABaQIbPOJYZGqX+RCMlDDbElcjaROLDoualmUIQ88Kekk3iM4OQrADcxi3rJguS4MOIBIgKgXrjd1WkbCdqxJk/4efRIFsavZA7KvvJQqp3Iid5Z0NFc5aiMRzGN3vrpBzaMy4JYde3wr96PjN90AYOIbyp6T4zj8LoE66OGcX1Ef4Z3KoWLAUF4BTg7ug/AbkG5UNQXAMkQezujSHeir2uTThgd3gpyzDrbnEdDRH2W7U6PeRvBX1ZFMP5RM+Zu6UUZZD8hDPHldVWntTCNk7To8IeOW9yn2wx0gmurwqC60AOde4r3ETi5pVMSDK8wxhoGAoEX9NLWHIR33VbrbMveii2jAJlrxwytTHbWNu8Y4N8vCCyZjAX/pcsfwXbLze2+D+u33OGBoJyAAL3jn3RuEcdp5If8O+a4NKWvxOTyDltG0IWoHhwVGe7dKkCWFT++tm+haBCikRUUMrMhYKZJKYoVuv/bsJzO8DwfVIInQq3g3BYypiz8baogH3r3GwqCwFtZnz4xMjAVOYnyOi5HWbFA8n0qz1OjSpHWFzpQOpvkNETZBGpxN8ybhtqV/DMUxd9uFZmBfKXMCn/SqkWJyKPnT6lq+4zBZni6fYRByJn6OK+OgPBGRAJluwGSk4wxjOOzyce/PKODwRlsgrVkdcsEiYrqYdXo0Er2GXi2GQZd0tNJT6c9pK1EEJG1zgDJBoTVuCXGAU8BKTvCO/cEQ1Wjk3Zzuy90JX4m3O5IlxVFhYkSUwuQB2up7jhvkm+bddRQu5F9s0XftGEJ9JSuSk+ZachCbdU45fEqbugzTIUokwoAKvpUQF/CvLbWW5BNQFqFkJg2f30E/48StNe5QwBg8zz3YAJ82FZoXBxXSv4QDooDo79NixyglO9AembuBcx5Re3CwOKTHebOPhkmFC7wNaWtoBhFuV4AkEuJ0J+1pT0tLkvFVZaNzfhs/Kd3+A9YsImlO4XK4vpCo/elHQi/9gkFg07xxnuXLt21unCIpDV+bbRxb7FC6nWYTsMFF8+1LUg4JFjVt3vqbuhHmDKbgQ4e+RGizRiO8ky05LQGMdL2IKLSNar0kNG7lHJMaXr5mLdG3nykgj6vB/KVijd1ARWkFEf3yiUw1v/WaQivVUpIDdSNrrKbjO5NPnxz6qTTGgYg03HgPhDrCFyYZTi3XQw3HXCva39mpLNFtz8AiEhxAJHpWX13gCTAwgm9YTvMeiqetdNQv6IU0hH0G+ZManTqDLPjyrOse7WiiwOJCG+J0pZYULhN8NILulmYYvmVcV2MjAfA39sGKqGdjpiPo86fecg65UPyXDIAOyOkCx5NQsLeD4gGVjTVDwOHWkbbBW0GeNjDkcSOn2Nq4cEssP54t9D749A7M1AIOBl0Fi0sSO5v3P7LCBrM6ZwFY6kp2FX6AcbGUdybnfChHPyu6WlRZ2Fwv9YM0RMI7kISRgR8HpQSJJOyTfXj/6gQKuihPtiUtlCQVPohUgzfezTg8o1b3n9pNZeco1QucaoXe40Fa5JYhqdTspFmxGtW9h5ezLFZs3j/N46f+S2rjYNC2JySXrnSAFhvAkz9a5L3pza8eYKHNoPrvBRESpxYPJdKVUxBE39nJ1chrAFpy4MMkf0qKgYALctGg1DQI1kIymyeS2AJNT4X240d3IFQb/0jQbaHJ2YRK8A+ls6WMhWmpCXYG5jqapGs5/eOJErxi2/2KWVHiPellTgh/fNl/2KYPKb7DUcAg+mCOPQFCiU9Mq/WLcU1xxC8aLePFZZlE+PCLzf7ey46INWRw2kcXySR9FDgByXzfxiNKwDFbUSMMhALPFSedyjEVM5442GZ4hTrsAEvZxIieSHGSgkwFh/nFNdrrFD4tBH4Il7fW6ur4J8Xaz7RW9jgtuPEXQsYk7gcMs2neu3zJwTyUerHKSh1iTBkj2YJh1SSOZL5pLuQbFFAvyO4k1Hxg2h99MTC6cTUkbONQIAnEfGsGkNFWRbuRyyaEZInM5pij73EA9rPIUfU4XoqQpHT9THZkW+oKFLvpyvTBMM69tN1Ydwv1LIEhHsC+ueVG+w+kyCPsvV3erRikcscHjZCkccx6VrBkBRusTDDd8847GA7p2Ucy0y0HdSRN6YIBciYa4vuXcAZbQAuSEmzw+H/AuOx+aH+tBL88H57D0MsqyiZxhOEQkF/8DR1d2hSPMj/sNOa5rxcUnBgH8ictv2J+cb4BA4v3MCShdZ2vtK30vAwkobnEWh7rsSyhmos3WC93Gn9C4nnAd/PjMMtQfyDNZsOPd6XcAsnBE/mRHtHEyJMzJfZFLE9OvQa0i9kUmToJ0ZxknTgdl/XPV8xoh0K7wNHHsnBdvFH3sv52lU7UFteseLG/VanIvcwycVA7+BE1Ulyb20BvwUWZcMTKhaCcmY3ROpvonVMV4N7yBXTL7IDtHzQ4CCcqF66LjF3xUqgErKzolLyCG6Kb7irP/MVTCCwGRxfrPGpMMGvPLgJ881PHMNMIO09T5ig7AzZTX/5PLlwnJLDAPfuHynSGhV4tPqR3gJ4kg4c06c/F1AcjGytKm2Yb5jwMotF7vro4YDLWlnMIpmPg36NgAZsGA0W1spfLSue4xxat0Gdwd0lqDBOgIaMANykwwDKejt5YaNtJYIkrSgu0KjIg0pznY0SCd1qlC6R19g97UrWDoYJGlrvCE05J/5wkjpkre727p5PTRX5FGrSBIfJqhJE/IS876PaHFkx9pGTH3oaY3jJRvLX9Iy3Edoar7cFvJqyUlOhAEiOSAyYgVEGkzHdug+oRHIEOXAExMiTSKU9A6nmRC8mp8iYhwWdP2U/5EkFAdPrZw03YA3gSyNUtMZeh7dDCu8pF5x0VORCTgKp07ehy7NZqKTpIC4UJJ89lnboyAfy5OyXzXtuDRbtAFjZRSyGFTpFrXwkpjSLIQIG3N0Vj4BtzK3wdlkBJrO18MNsgseR4BysJilI0wI6ZahLhBFA0XBmV8d4LUzEcNVb0xbLjLTETYN8OEVqNxkt10W614dd1FlFFVTIgB7/BQQp1sWlNolpIu4ekxUTBV7NmxOFKEBmmN+nA7pvF78/RII5ZHA09OAiE/66MF6HQ+qVEJCHxwymukkNvzqHEh52dULPbVasfQMgTDyBZzx4007YiKdBuUauQOt27Gmy8ISclPmEUCIcuLbkb1mzQSqIa3iE0PJh7UMYQbkpe+hXjTJKdldyt2mVPwywoODGJtBV1lJTgMsuSQBlDMwhEKIfrvsxGQjHPCEfNfMAY2oxvyKcKPUbQySkKG6tj9AQyEW3Q5rpaDJ5Sns9ScLKeizPRbvWYAw4bXkrZdmB7CQopCH8NAmqbuciZChHN8lVGaDbCnmddnqO1PQ4ieMYfcSiBE5zzMz+JV/4eyzrzTEShvqSGzgWimkNxLvUj86iAwcZuIkqdB0VaIB7wncLRmzHkiUQpPBIXbDDLHBlq7vp9xwuC9AiNkIptAYlG7Biyuk8ILdynuUM1cHWJgeB+K3wBP/ineogxkvBNNQ4AkW0hvpBOQGFfeptF2YTR75MexYDUy7Q/9uocGsx41O4IZhViw/2FvAEuGO5g2kyXBUijAggWM08bRhXg5ijgMwDJy40QeY/cQpUDZiIzmvskQpO5G1zyGZA8WByjIQU4jRoFJt56behxtHUUE/om7Rj2psYXGmq3llVOCgGYKNMo4pzwntITtapDqjvQtqpjaJwjHmDzSVGLxMt12gEXAdLi/caHSM3FPRGRf7dB7YC+cD2ho6oL2zGDCkjlf/DFoQVl8GS/56wur3rdV6ggtzZW60MRB3g+U1W8o8cvqIpMkctiGVMzXUFI7FacFLrgtdz4mTEr4aRAaQ2AFQaNeG7GX0yOJgMRYFziXdJf24kg/gBQIZMG/YcPEllRTVNoDYR6oSJ8wQNLuihfw81UpiKPm714bZX1KYjcXJdfclCUOOpvTxr9AAJevTY4HK/G7F3mUc3GOAKqh60zM0v34v+ELyhJZqhkaMA8UMMOU90f8RKEJFj7EqepBVwsRiLbwMo1J2zrE2UYJnsgIAscDmjPjnzI8a719Wxp757wqmSJBjXowhc46QN4RwKIxqEE6E5218OeK7RfcpGjWG1jD7qND+/GTk6M56Ig4yMsU6LUW1EWE+fIYycVV1thldSlbP6ltdC01y3KUfkobkt2q01YYMmxpKRvh1Z48uNKzP/IoRIZ/F6buOymSnW8gICitpJjKWBscSb9JJKaWkvEkqinAJ2kowKoqkqZftRqfRQlLtKoqvTRDi2vg/RrPD/d3a09J8JhGZlEkOM6znTsoMCsuvTmywxTCDhw5dd0GJOHCMPbsj3QLkTE3MInsZsimDQ3HkvthT7U9VA4s6G07sID0FW4SHJmRGwCl+Mu4xf0ezqeXD2PtPDnwMPo86sbwDV+9PWcgFcARUVYm3hrFQrHcgMElFGbSM2A1zUYA3baWfheJp2AINmTJLuoyYD/OwA4a6V0ChBN97E8YtDBerUECv0u0TlxR5yhJCXvJxgyM73Bb6pyq0jTFJDZ4p1Am1SA6sh8nADd1hAcGBMfq4d/UfwnmBqe0Jun1n1LzrgKuZMAnxA3NtCN7Klf4BH+14B7ibBmgt0TGUafVzI4uKlpF7v8NmgNjg90D6QE3tbx8AjSAC+OA1YJvclyPKgT27QpIEgVYpbPYGBsnyCNrGz9XUsCHkW1QAHgL2STZk12QGqmvAB0NFteERkvBIH7INDsNW9KKaAYyDMdBEMzJiWaJHZALqDxQDWRntumSDPcplyFiI1oDpT8wbwe01AHhW6+vAUUBoGhY3CT2tgwehdPqU/4Q7ZLYvhRl/ogOvR9O2+wkkPKW5vCTjD2fHRYXONCoIl4Jh1bZY0ZE1O94mMGn/dFSWBWzQ/VYk+Gezi46RgiDv3EshoTmMSlioUK6MQEN8qeyK6FRninyX8ZPeUWjjbMJChn0n/yJvrq5bh5UcCAcBYSafTFg7p0jDgrXo2QWLb3WpSOET/Hh4oSadBTvyDo10IufLzxiMLAnbZ1vcUmj3w7BQuIXjEZXifwukVxrGa9j+DXfpi12m1RbzYLg9J2wFergEwOxFyD0/JstNK06ZN2XdZSGWxcJODpQHOq4iKqjqkJUmPu1VczL5xTGUfCgLEYyNBCCbMBFT/cUP6pE/mujnHsSDeWxMbhrNilS5MyYR0nJyzanWXBeVcEQrRIhQeJA6Xt4f2eQESNeLwmC10WJVHqwx8SSyrtAAjpGjidcj1E2FYN0LObUcFQhafUKTiGmHWRHGsFCB+HEXgrzJEB5bp0QiF8ZHh11nFX8AboTD0PS4O1LqF8XBks2MpjsQnwKHF6HgaKCVLJtcr0XjqFMRGfKv8tmmykhLRzu+vqQ02+KpJBjaLt9ye1Ab+BbEBhy4EVdIJDrL2naV0o4wU8YZ2Lq04FG1mWCKC+UwkXOoAjneU/xHplMQo2cXUlrVNqJYczgYlaOEczVCs/OCgkyvLmTmdaBJc1iBLuKwmr6qtRnhowngsDxhzKFAi02tf8bmET8BO27ovJKF1plJwm3b0JpMh38+xsrXXg7U74QUM8ZCIMOpXujHntKdaRtsgyEZl5MClMVMMMZkZLNxH9+b8fH6+b8Lev30A9TuEVj9CqAdmwAAHBPbfOBFEATAPZ2CS0OH1Pj/0Q7PFUcC8hDrxESWdfgFRm+7vvWbkEppHB4T/1ApWnlTIqQwjcPl0VgS1yHSmD0OdsCVST8CQVwuiew1Y+g3QGFjNMzwRB2DSsAk26cmA8lp2wIU4p93AUBiUHFGOxOajAqD7Gm6NezNDjYzwLOaSXRBYcWipTSONHjUDXCY4mMI8XoVCR/Rrs/JLKXgEx+qkmeDlFOD1/yTQNDClRuiUyKYCllfMiQiyFkmuTz2vLsBNyRW+xz+5FElFxWB28VjYIGZ0Yd+5wIjkcoMaggxswbT0pCmckRAErbRlIlcOGdBo4djTNO8FAgQ+lT6vPS60BwTRSUAM3ddkEAZiwtEyArrkiDRnS7LJ+2hwbzd2YDQagSgACpsovmjil5wfPuXq3GuH0CyE7FK3M4FgRaFoIkaodORrPx1+JpI9psyNYIFuJogZa0/1AhOWdlHQxdAgbwacsHqPZo8u/ngAH2GmaTdhYnBfSDbBfh8CHq6Bx5bttP2+RdM+MAaYaZ0Y/ADkbNCZuAyAVQa2OcXOeICmDn9Q/eFkDeFQg5MgHEDXq/tVjj+jtd26nhaaolWxs1ixSUgOBwrDhRIGOLyOVk2/Bc0UxvseQCO2pQ2i+Krfhu/WeBovNb5dJxQtJRUDv2mCwYVpNl2efQM9xQHnK0JwLYt/U0Wf+phiA4uw8G91slC832pmOTCAoZXohg1fewCZqLBhkOUBofBWpMPsqg7XEXgPfAlDo2U5WXjtFdS87PIqClCK5nW6adCeXPkUiTGx0emOIDQqw1yFYGHEVx20xKjJVYe0O8iLmnQr3FA9nSIQilUKtJ4ZAdcTm7+ExseJauyqo30hs+1qSW211A1SFAOUgDlCGq7eTIcMAeyZkV1SQJ4j/e1Smbq4HcjqgFbLAGLyKxlMDMgZavK5NAYH19Olz3la/QCTiVelFnU6O/GCvykqS/wZJDhKN9gBtSOp/1SP5VRgJcoVj+kmf2wBgv4gjrgARBWiURYx8xENV3bEVUAAWWD3dYDKAIWk5opaCFCMR5ZjJExiCAw7gYiSZ2rkyTce4eNMY3lfGn+8p6+vBckGlKEXnA6Eota69OxDO9oOsJoy28BXOR0UoXNRaJD5ceKdlWMJlOFzDdZNpc05tkMGQtqeNF2lttZqNco1VtwXgRstLSQ6tSPChgqtGV5h2DcDReIQadaNRR6AsAYKL5gSFsCJMgfsaZ7DpKh8mg8Wz8V7H+gDnLuMxaWEIUPevIbClgap4dqmVWSrPgVYCzAoZHIa5z2Ocx1D/GvDOEqMOKLrMefWIbSWHZ6jbgA8qVBhYNHpx0P+jAgN5TB3haSifDcApp6yymEi6Ij/GsEpDYUgcHATJUYDUAmC1SCkJ4cuZXSAP2DEpQsGUjQmKJfJOvlC2x/pChkOyLW7KEoMYc5FDC4v2FGqSoRWiLsbPCiyg1U5yiHZVm1XLkHMMZL11/yxyw0UnGig3MFdZklN5FI/qiT65T+jOXOdO7XbgWurOAZR6Cv9uu1cm5LjkXX4xi6mWn5r5NjBS0gTliHhMZI2WNqSiSphEtiCAwnafS11JhseDGHYQ5+bqWiAYiAv6Jsf79/VUs4cIl+n6+WOjcgB/2l5TreoAV2717JzZbQIR0W1cl/dEqCy5kJ3ZSIHuU0vBoHooEpiHeQWVkkkOqRX27eD1FWw4BfO9CJDdKoSogQi3hAAwsPRFrN5RbX7bqLdBJ9JYMohWrgJKHSjVl1sy2xAG0E3sNyO0oCbSGOxCNBRRXTXenYKuwAoDLfnDcQaCwehUOIDiHAu5m5hMpKeKM4sIo3vxACakIxKoH2YWF2QM84e6F5C5hJU4g8uxuFOlAYnqtwxmHyNEawLW/PhoawJDrGAP0JYWHgAVUByo/bGdiv2T2EMg8gsS14/rAdzlOYazFE7w4OzxeKiWdm3nSOnQRRKXSlVo8HEAbBfyJMKqoq+SCcTSx5NDtbFwNlh8VhjGGDu7JG5/TAGAvniQSSUog0pNzTim8Owc6QTuSKSTXlQqwV3eiEnklS3LeSXYPXGK2VgeZBqNcHG6tZHvA3vTINhV0ELuQdp3t1y9+ogD8Kk/W7QoRN1UWPqM4+xdygkFDPLoTaumKReKiLWoPHOfY54m3qPx4c+4pgY3MRKKbljG8w4wvz8pxk3AqKsy4GMAkAtmRjRMsCxbb4Q2Ds0Ia9ci8cMT6DmsJG00XaHCIS+o3F8YVVeikw13w+OEDaCYYhC0ZE54kA4jpjruBr5STWeqQG6M74HHL6TZ3lXrd99ZX++7LhNatQaZosuxEf5yRA15S9gPeHskBIq3Gcw81AGb9/O53DYi/5CsQ51EmEh8Rkg4vOciClpy4d04eYsfr6fyQkBmtD+P8sNh6e+XYHJXT/lkXxT4KXU5F2sGxYyzfniMMQkb9OjDN2C8tRRgTyL7GwozH14PrEUZc6oz05Emne3Ts5EG7WolDmU8OB1LDG3VrpQxp+pT0KYV5dGtknU64JhabdqcVQbGZiAxQAnvN1u70y1AnmvOSPgLI6uB4AuDGhmAu3ATkJSw7OtS/2ToPjqkaq62/7WFG8advGlRRqxB9diP07JrXowKR9tpRa+jGJ91zxNTT1h8I2PcSfoUPtd7NejVoH03EUcqSBuFZPkMZhegHyo2ZAITovmm3zAIdGFWxoNNORiMRShgwdYwFzkPw5PA4a5MIIQpmq+nsp3YMuXt/GkXxLx/P6+ZJS0lFyz4MunC3eWSGE8xlCQrKvhKUPXr0hjpAN9ZK4PfEDrPMfMbGNWcHDzjA7ngMxTPnT7GMHar+gMQQ3NwHCv4zH4BIMYvzsdiERi6gebRmerTsVwZJTRsL8dkZgxgRxmpbgRcud+YlCIRpPwHShlUSwuipZnx9QCsEWziVazdDeKSYU5CF7UVPAhLer3CgJOQXl/zh575R5rsrmRnKAzq4POFdgbYBuEviM4+LVC15ssLNFghbTtHWerS1hDt5s4qkLUha/qpZXhWh1C6lTQAqCNQnaDjS7UGFBC6wTu8yFnKJnExCnAs3Ok9yj5KpfZESQ4lTy5pTGTnkAUpxI+yjEldJfSo4y0QhG4i4IwkRFGcjWY8+EzgYYJUK7BXQksLxAww/YYWBMhJILB9e8ePEJ4OP7z+4/wOQDl64iOYDp26DaONPxpKtBxq/aTzRGarm3VkPYTLJKx6Z/Mw2YbBGseJhPMwhhNswrIkyvV2BYzrvZbxLpKwcWJhYmFtVZ+lPEq91FzVp1HlQY1bZVLqeNR9SAUn6n0E28k/UuGkNpP1DBI5ch/EehZfjUQ9aE41NhETExoPT2gGQz0IhWJbEOvTQ4wgcXCHHFBhewYUiFHuhRSAUVmEHeCRQHQkXGFwkAgyzREJCVN7TRnTon36Zw3tPhx4EALwNdwDv+J41YSP4B2CQqz0EFgARZ4ESgBHQgROwAVn9GTI+HYexTUevLUeta4/DqKrbMVS+Yqb8hUwYCrlgKtmAq1YCrFgKrd4qpXiqZcKn1oqdWipjYKpWwVPVYqW6xUpVipKqFR3QKjagVEtAqHpxUMTitsnFaJOKx2cVhswq35RVpyiq9lFVNIKnOQVMkgqtYxVNxiqQjFS7GKlSIVIsQqPIhUWwioigFQ++KkN8VHr49HDw9Ebo9EDo9DTo9Crg9BDg9/Wx7gWx7YWwlobYrOGxWPNisAaAHEyALpkAVDIAeWAArsABVXACYuAD5cAF6wAKFQAQqgAbVAAsoAAlQAUaYAfkwAvogBWQACOgAD9AAHSAAKT4GUdMiOvFngBTwCn2AZ7Dv6B6k/90B8+yRnkV144AIBoAMTQATGgAjNAA4YABgwABZgB/mQCwyAVlwCguASlwCEuAQFwB4uAMlwBYuAJlQAUVAAhUD2KgdpUDaJgaRMDFJgX5MC1JgWJEAokQCWRAHxEAWkQBMRADpEAMkQAYROAEecC484DRpwBDTnwNOdw05tjTmiNOYwtswhYFwLA7BYG4LA2BYGOLAwRYFuLAsxYFQJAohIEyJAMwkAwiQC0JAJgkAeiQBkJAFokAPCQA0JABwcD4Dgc4cDdDgaYcDIDgYgUC6CgWgUClCgUYUAVBQBOFAEYMALgwAgDA9QYAdIn8AZzeBB2L5EcWrenUT1KXienEsuJJ7x5U8XlTjc1NVzUyXFTGb1LlpUtWlTDIjqwE4LsagowoCi2gJLKAkpoBgJQNpAIhNqaEoneI6kiiqQ6Go/n6j0cS+a2gEU8gIHJ+BwfgZX4GL+Bd/gW34FZ+BS/gUH4FN6BTegTvoEv6BJegRnYEF2A79gOvYDl2BdEjCkqkGtwXp0LNToIskOTXzh/F062yJ7AAAAEDAWAAABWhJ+KPEIJgBFxMVP7w2QJBGHASQnOBKXKFIdUK4igKA9IEaYJg);src:url(data:application/vnd.ms-fontobject;base64,n04AAEFNAAACAAIABAAAAAAABQAAAAAAAAABAJABAAAEAExQAAAAAAAAAAIAAAAAAAAAAAEAAAAAAAAAJxJ/LAAAAAAAAAAAAAAAAAAAAAAAACgARwBMAFkAUABIAEkAQwBPAE4AUwAgAEgAYQBsAGYAbABpAG4AZwBzAAAADgBSAGUAZwB1AGwAYQByAAAAeABWAGUAcgBzAGkAbwBuACAAMQAuADAAMAA5ADsAUABTACAAMAAwADEALgAwADAAOQA7AGgAbwB0AGMAbwBuAHYAIAAxAC4AMAAuADcAMAA7AG0AYQBrAGUAbwB0AGYALgBsAGkAYgAyAC4ANQAuADUAOAAzADIAOQAAADgARwBMAFkAUABIAEkAQwBPAE4AUwAgAEgAYQBsAGYAbABpAG4AZwBzACAAUgBlAGcAdQBsAGEAcgAAAAAAQlNHUAAAAAAAAAAAAAAAAAAAAAADAKncAE0TAE0ZAEbuFM3pjM/SEdmjKHUbyow8ATBE40IvWA3vTu8LiABDQ+pexwUMcm1SMnNryctQSiI1K5ZnbOlXKmnVV5YvRe6RnNMFNCOs1KNVpn6yZhCJkRtVRNzEufeIq7HgSrcx4S8h/v4vnrrKc6oCNxmSk2uKlZQHBii6iKFoH0746ThvkO1kJHlxjrkxs+LWORaDQBEtiYJIR5IB9Bi1UyL4Rmr0BNigNkMzlKQmnofBHviqVzUxwdMb3NdCn69hy+pRYVKGVS/1tnsqv4LL7wCCPZZAZPT4aCShHjHJVNuXbmMrY5LeQaGnvAkXlVrJgKRAUdFjrWEah9XebPeQMj7KS7DIBAFt8ycgC5PLGUOHSE3ErGZCiViNLL5ZARfywnCoZaKQCu6NuFX42AEeKtKUGnr/Cm2Cy8tpFhBPMW5Fxi4Qm4TkDWh4IWFDClhU2hRWosUWqcKLlgyXB+lSHaWaHiWlBAR8SeSgSPCQxdVQgzUixWKSTrIQEbU94viDctkvX+VSjJuUmV8L4CXShI11esnp0pjWNZIyxKHS4wVQ2ime1P4RnhvGw0aDN1OLAXGERsB7buFpFGGBAre4QEQR0HOIO5oYH305G+KspT/FupEGGafCCwxSe6ZUa+073rXHnNdVXE6eWvibUS27XtRzkH838mYLMBmYysZTM0EM3A1fbpCBYFccN1B/EnCYu/TgCGmr7bMh8GfYL+BfcLvB0gRagC09w9elfldaIy/hNCBLRgBgtCC7jAF63wLSMAfbfAlEggYU0bUA7ACCJmTDpEmJtI78w4/BO7dN7JR7J7ZvbYaUbaILSQsRBiF3HGk5fEg6p9unwLvn98r+vnsV+372uf1xBLq4qU/45fTuqaAP+pssmCCCTF0mhEow8ZXZOS8D7Q85JsxZ+Azok7B7O/f6J8AzYBySZQB/QHYUSA+EeQhEWiS6AIQzgcsDiER4MjgMBAWDV4AgQ3g1eBgIdweCQmCjJEMkJ+PKRWyFHHmg1Wi/6xzUgA0LREoKJChwnQa9B+5RQZRB3IlBlkAnxyQNaANwHMowzlYSMCBgnbpzvqpl0iTJNCQidDI9ZrSYNIRBhHtUa5YHMHxyGEik9hDE0AKj72AbTCaxtHPUaKZdAZSnQTyjGqGLsmBStCejApUhg4uBMU6mATujEl+KdDPbI6Ag4vLr+hjY6lbjBeoLKnZl0UZgRX8gTySOeynZVz1wOq7e1hFGYIq+MhrGxDLak0PrwYzSXtcuyhXEhwOYofiW+EcI/jw8P6IY6ed+etAbuqKp5QIapT77LnAe505lMuqL79a0ut4rWexzFttsOsLDy7zvtQzcq3U1qabe7tB0wHWVXji+zDbo8x8HyIRUbXnwUcklFv51fvTymiV+MXLSmGH9d9+aXpD5X6lao41anWGig7IwIdnoBY2ht/pO9mClLo4NdXHAsefqWUKlXJkbqPOFhMoR4aiA1BXqhRNbB2Xwi+7u/jpAoOpKJ0UX24EsrzMfHXViakCNcKjBxuQX8BO0ZqjJ3xXzf+61t2VXOSgJ8xu65QKgtN6FibPmPYsXbJRHHqbgATcSZxBqGiDiU4NNNsYBsKD0MIP/OfKnlk/Lkaid/O2NbKeuQrwOB2Gq3YHyr6ALgzym5wIBnsdC1ZkoBFZSQXChZvlesPqvK2c5oHHT3Q65jYpNxnQcGF0EHbvYqoFw60WNlXIHQF2HQB7zD6lWjZ9rVqUKBXUT6hrkZOle0RFYII0V5ZYGl1JAP0Ud1fZZMvSomBzJ710j4Me8mjQDwEre5Uv2wQfk1ifDwb5ksuJQQ3xt423lbuQjvoIQByQrNDh1JxGFkOdlJvu/gFtuW0wR4cgd+ZKesSV7QkNE2kw6AV4hoIuC02LGmTomyf8PiO6CZzOTLTPQ+HW06H+tx+bQ8LmDYg1pTFrp2oJXgkZTyeRJZM0C8aE2LpFrNVDuhARsN543/FV6klQ6Tv1OoZGXLv0igKrl/CmJxRmX7JJbJ998VSIPQRyDBICzl4JJlYHbdql30NvYcOuZ7a10uWRrgoieOdgIm4rlq6vNOQBuqESLbXG5lzdJGHw2m0sDYmODXbYGTfSTGRKpssTO95fothJCjUGQgEL4yKoGAF/0SrpUDNn8CBgBcSDQByAeNkCXp4S4Ro2Xh4OeaGRgR66PVOsU8bc6TR5/xTcn4IVMLOkXSWiXxkZQCbvKfmoAvQaKjO3EDKwkwqHChCDEM5loQRPd5ACBki1TjF772oaQhQbQ5C0lcWXPFOzrfsDGUXGrpxasbG4iab6eByaQkQfm0VFlP0ZsDkvvqCL6QXMUwCjdMx1ZOyKhTJ7a1GWAdOUcJ8RSejxNVyGs31OKMyRyBVoZFjqIkmKlLQ5eHMeEL4MkUf23cQ/1SgRCJ1dk4UdBT7OoyuNgLs0oCd8RnrEIb6QdMxT2QjD4zMrJkfgx5aDMcA4orsTtKCqWb/Veyceqa5OGSmB28YwH4rFbkQaLoUN8OQQYnD3w2eXpI4ScQfbCUZiJ4yMOIKLyyTc7BQ4uXUw6Ee6/xM+4Y67ngNBknxIPwuppgIhFcwJyr6EIj+LzNj/mfR2vhhRlx0BILZoAYruF0caWQ7YxO66UmeguDREAFHYuC7HJviRgVO6ruJH59h/C/PkgSle8xNzZJULLWq9JMDTE2fjGE146a1Us6PZDGYle6ldWRqn/pdpgHKNGrGIdkRK+KPETT9nKT6kLyDI8xd9A1FgWmXWRAIHwZ37WyZHOVyCadJEmMVz0MadMjDrPho+EIochkVC2xgGiwwsQ6DMv2P7UXqT4x7CdcYGId2BJQQa85EQKmCmwcRejQ9Bm4oATENFPkxPXILHpMPUyWTI5rjNOsIlmEeMbcOCEqInpXACYQ9DDxmFo9vcmsDblcMtg4tqBerNngkIKaFJmrQAPnq1dEzsMXcwjcHdfdCibcAxxA+q/j9m3LM/O7WJka4tSidVCjsvo2lQ/2ewyoYyXwAYyr2PlRoR5MpgVmSUIrM3PQxXPbgjBOaDQFIyFMJvx3Pc5RSYj12ySVF9fwFPQu2e2KWVoL9q3Ayv3IzpGHUdvdPdrNUdicjsTQ2ISy7QU3DrEytIjvbzJnAkmANXjAFERA0MUoPF3/5KFmW14bBNOhwircYgMqoDpUMcDtCmBE82QM2YtdjVLB4kBuKho/bcwQdeboqfQartuU3CsCf+cXkgYAqp/0Ee3RorAZt0AvvOCSI4JICIlGlsV0bsSid/NIEALAAzb6HAgyWHBps6xAOwkJIGcB82CxRQq4sJf3FzA70A+TRqcqjEMETCoez3mkPcpnoALs0ugJY8kQwrC+JE5ik3w9rzrvDRjAQnqgEVvdGrNwlanR0SOKWzxOJOvLJhcd8Cl4AshACUkv9czdMkJCVQSQhp6kp7StAlpVRpK0t0SW6LHeBJnE2QchB5Ccu8kxRghZXGIgZIiSj7gEKMJDClcnX6hgoqJMwiQDigIXg3ioFLCgDgjPtYHYpsF5EiA4kcnN18MZtOrY866dEQAb0FB34OGKHGZQjwW/WDHA60cYFaI/PjpzquUqdaYGcIq+mLez3WLFFCtNBN2QJcrlcoELgiPku5R5dSlJFaCEqEZle1AQzAKC+1SotMcBNyQUFuRHRF6OlimSBgjZeTBCwLyc6A+P/oFRchXTz5ADknYJHxzrJ5pGuIKRQISU6WyKTBBjD8WozmVYWIsto1AS5rxzKlvJu4E/vwOiKxRtCWsDM+eTHUrmwrCK5BIfMzGkD+0Fk5LzBs0jMYXktNDblB06LMNJ09U8pzSLmo14MS0OMjcdrZ31pyQqxJJpRImlSvfYAK8inkYU52QY2FPEVsjoWewpwhRp5yAuNpkqhdb7ku9Seefl2D0B8SMTFD90xi4CSOwwZy9IKkpMtI3FmFUg3/kFutpQGNc3pCR7gvC4sgwbupDu3DyEN+W6YGLNM21jpB49irxy9BSlHrVDlnihGKHwPrbVFtc+h1rVQKZduxIyojccZIIcOCmhEnC7UkY68WXKQgLi2JCDQkQWJRQuk60hZp0D3rtCTINSeY9Ej2kIKYfGxwOs4j9qMM7fYZiipzgcf7TamnehqdhsiMiCawXnz4xAbyCkLAx5EGbo3Ax1u3dUIKnTxIaxwQTHehPl3V491H0+bC5zgpGz7Io+mjdhKlPJ01EeMpM7UsRJMi1nGjmJg35i6bQBAAxjO/ENJubU2mg3ONySEoWklCwdABETcs7ck3jgiuU9pcKKpbgn+3YlzV1FzIkB6pmEDOSSyDfPPlQskznctFji0kpgZjW5RZe6x9kYT4KJcXg0bNiCyif+pZACCyRMmYsfiKmN9tSO65F0R2OO6ytlEhY5Sj6uRKfFxw0ijJaAx/k3QgnAFSq27/2i4GEBA+UvTJKK/9eISNvG46Em5RZfjTYLdeD8kdXHyrwId/DQZUaMCY4gGbke2C8vfjgV/Y9kkRQOJIn/xM9INZSpiBnqX0Q9GlQPpPKAyO5y+W5NMPSRdBCUlmuxl40ZfMCnf2Cp044uI9WLFtCi4YVxKjuRCOBWIb4XbIsGdbo4qtMQnNOQz4XDSui7W/N6l54qOynCqD3DpWQ+mpD7C40D8BZEWGJX3tlAaZBMj1yjvDYKwCJBa201u6nBKE5UE+7QSEhCwrXfbRZylAaAkplhBWX50dumrElePyNMRYUrC99UmcSSNgImhFhDI4BXjMtiqkgizUGCrZ8iwFxU6fQ8GEHCFdLewwxYWxgScAYMdMLmcZR6b7rZl95eQVDGVoUKcRMM1ixXQtXNkBETZkVVPg8LoSrdetHzkuM7DjZRHP02tCxA1fmkXKF3VzfN1pc1cv/8lbTIkkYpqKM9VOhp65ktYk+Q46myFWBapDfyWUCnsnI00QTBQmuFjMZTcd0V2NQ768Fhpby04k2IzNR1wKabuGJqYWwSly6ocMFGTeeI+ejsWDYgEvr66QgqdcIbFYDNgsm0x9UHY6SCd5+7tpsLpKdvhahIDyYmEJQCqMqtCF6UlrE5GXRmbu+vtm3BFSxI6ND6UxIE7GsGMgWqghXxSnaRJuGFveTcK5ZVSPJyjUxe1dKgI6kNF7EZhIZs8y8FVqwEfbM0Xk2ltORVDKZZM40SD3qQoQe0orJEKwPfZwm3YPqwixhUMOndis6MhbmfvLBKjC8sKKIZKbJk8L11oNkCQzCgvjhyyEiQSuJcgCQSG4Mocfgc0Hkwcjal1UNgP0CBPikYqBIk9tONv4kLtBswH07vUCjEaHiFGlLf8MgXKzSgjp2HolRRccAOh0ILHz9qlGgIFkwAnzHJRjWFhlA7ROwINyB5HFj59PRZHFor6voq7l23EPNRwdWhgawqbivLSjRA4htEYUFkjESu67icTg5S0aW1sOkCiIysfJ9UnIWevOOLGpepcBxy1wEhd2WI3AZg7sr9WBmHWyasxMcvY/iOmsLtHSWNUWEGk9hScMPShasUA1AcHOtRZlqMeQ0OzYS9vQvYUjOLrzP07BUAFikcJNMi7gIxEw4pL1G54TcmmmoAQ5s7TGWErJZ2Io4yQ0ljRYhL8H5e62oDtLF8aDpnIvZ5R3GWJyAugdiiJW9hQAVTsnCBHhwu7rkBlBX6r3b7ejEY0k5GGeyKv66v+6dg7mcJTrWHbtMywbedYqCQ0FPwoytmSWsL8WTtChZCKKzEF7vP6De4x2BJkkniMgSdWhbeBSLtJZR9CTHetK1xb34AYIJ37OegYIoPVbXgJ/qDQK+bfCtxQRVKQu77WzOoM6SGL7MaZwCGJVk46aImai9fmam+WpHG+0BtQPWUgZ7RIAlPq6lkECUhZQ2gqWkMYKcYMYaIc4gYCDFHYa2d1nzp3+J1eCBay8IYZ0wQRKGAqvCuZ/UgbQPyllosq+XtfKIZOzmeJqRazpmmoP/76YfkjzV2NlXTDSBYB04SVlNQsFTbGPk1t/I4Jktu0XSgifO2ozFOiwd/0SssJDn0dn4xqk4GDTTKX73/wQyBLdqgJ+Wx6AQaba3BA9CKEzjtQYIfAsiYamapq80LAamYjinlKXUkxdpIDk0puXUEYzSalfRibAeDAKpNiqQ0FTwoxuGYzRnisyTotdVTclis1LHRQCy/qqL8oUaQzWRxilq5Mi0IJGtMY02cGLD69vGjkj3p6pGePKI8bkBv5evq8SjjyU04vJR2cQXQwSJyoinDsUJHCQ50jrFTT7yRdbdYQMB3MYCb6uBzJ9ewhXYPAIZSXfeEQBZZ3GPN3Nbhh/wkvAJLXnQMdi5NYYZ5GHE400GS5rXkOZSQsdZgIbzRnF9ueLnsfQ47wHAsirITnTlkCcuWWIUhJSbpM3wWhXNHvt2xUsKKMpdBSbJnBMcihkoDqAd1Zml/R4yrzow1Q2A5G+kzo/RhRxQS2lCSDRV8LlYLBOOoo1bF4jwJAwKMK1tWLHlu9i0j4Ig8qVm6wE1DxXwAwQwsaBWUg2pOOol2dHxyt6npwJEdLDDVYyRc2D0HbcbLUJQj8gPevQBUBOUHXPrsAPBERICpnYESeu2OHotpXQxRGlCCtLdIsu23MhZVEoJg8Qumj/UMMc34IBqTKLDTp76WzL/dMjCxK7MjhiGjeYAC/kj/jY/Rde7hpSM1xChrog6yZ7OWTuD56xBJnGFE+pT2ElSyCnJcwVzCjkqeNLfMEJqKW0G7OFIp0G+9mh50I9o8k1tpCY0xYqFNIALgIfc2me4n1bmJnRZ89oepgLPT0NTMLNZsvSCZAc3TXaNB07vail36/dBySis4m9/DR8izaLJW6bWCkVgm5T+ius3ZXq4xI+GnbveLbdRwF2mNtsrE0JjYc1AXknCOrLSu7Te/r4dPYMCl5qtiHNTn+TPbh1jCBHH+dMJNhwNgs3nT+OhQoQ0vYif56BMG6WowAcHR3DjQolxLzyVekHj00PBAaW7IIAF1EF+uRIWyXjQMAs2chdpaKPNaB+kSezYt0+CA04sOg5vx8Fr7Ofa9sUv87h7SLAUFSzbetCCZ9pmyLt6l6/TzoA1/ZBG9bIUVHLAbi/kdBFgYGyGwRQGBpkqCEg2ah9UD6EedEcEL3j4y0BQQCiExEnocA3SZboh+epgd3YsOkHskZwPuQ5OoyA0fTA5AXrHcUOQF+zkJHIA7PwCDk1gGVmGUZSSoPhNf+Tklauz98QofOlCIQ/tCD4dosHYPqtPCXB3agggQQIqQJsSkB+qn0rkQ1toJjON/OtCIB9RYv3PqRA4C4U68ZMlZn6BdgEvi2ziU+TQ6NIw3ej+AtDwMGEZk7e2IjxUWKdAxyaw9OCwSmeADTPPleyk6UhGDNXQb++W6Uk4q6F7/rg6WVTo82IoCxSIsFDrav4EPHphD3u4hR53WKVvYZUwNCCeM4PMBWzK+EfIthZOkuAwPo5C5jgoZgn6dUdvx5rIDmd58cXXdKNfw3l+wM2UjgrDJeQHhbD7HW2QDoZMCujgIUkk5Fg8VCsdyjOtnGRx8wgKRPZN5dR0zPUyfGZFVihbFRniXZFOZGKPnEQzU3AnD1KfR6weHW2XS6KbPJxUkOTZsAB9vTVp3Le1F8q5l+DMcLiIq78jxAImD2pGFw0VHfRatScGlK6SMu8leTmhUSMy8Uhdd6xBiH3Gdman4tjQGLboJfqz6fL2WKHTmrfsKZRYX6BTDjDldKMosaSTLdQS7oDisJNqAUhw1PfTlnacCO8vl8706Km1FROgLDmudzxg+EWTiArtHgLsRrAXYWdB0NmToNCJdKm0KWycZQqb+Mw76Qy29iQ5up/X7oyw8QZ75kP5F6iJAJz6KCmqxz8fEa/xnsMYcIO/vEkGRuMckhr4rIeLrKaXnmIzlNLxbFspOphkcnJdnz/Chp/Vlpj2P7jJQmQRwGnltkTV5dbF9fE3/fxoSqTROgq9wFUlbuYzYcasE0ouzBo+dDCDzxKAfhbAZYxQiHrLzV2iVexnDX/QnT1fsT/xuhu1ui5qIytgbGmRoQkeQooO8eJNNZsf0iALur8QxZFH0nCMnjerYQqG1pIfjyVZWxhVRznmmfLG00BcBWJE6hzQWRyFknuJnXuk8A5FRDCulwrWASSNoBtR+CtGdkPwYN2o7DOw/VGlCZPusRBFXODQdUM5zeHDIVuAJBLqbO/f9Qua+pDqEPk230Sob9lEZ8BHiCorjVghuI0lI4JDgHGRDD/prQ84B1pVGkIpVUAHCG+iz3Bn3qm2AVrYcYWhock4jso5+J7HfHVj4WMIQdGctq3psBCVVzupQOEioBGA2Bk+UILT7+VoX5mdxxA5fS42gISQVi/HTzrgMxu0fY6hE1ocUwwbsbWcezrY2n6S8/6cxXkOH4prpmPuFoikTzY7T85C4T2XYlbxLglSv2uLCgFv8Quk/wdesUdWPeHYIH0R729JIisN9Apdd4eB10aqwXrPt+Su9mA8k8n1sjMwnfsfF2j3jMUzXepSHmZ/BfqXvzgUNQQWOXO8YEuFBh4QTYCkOAPxywpYu1VxiDyJmKVcmJPGWk/gc3Pov02StyYDahwmzw3E1gYC9wkupyWfDqDSUMpCTH5e5N8B//lHiMuIkTNw4USHrJU67bjXGqNav6PBuQSoqTxc8avHoGmvqNtXzIaoyMIQIiiUHIM64cXieouplhNYln7qgc4wBVAYR104kO+CvKqsg4yIUlFNThVUAKZxZt1XA34h3TCUUiXVkZ0w8Hh2R0Z5L0b4LZvPd/p1gi/07h8qfwHrByuSxglc9cI4QIg2oqvC/qm0i7tjPLTgDhoWTAKDO2ONW5oe+/eKB9vZB8K6C25yCZ9RFVMnb6NRdRjyVK57CHHSkJBfnM2/j4ODUwRkqrtBBCrDsDpt8jhZdXoy/1BCqw3sSGhgGGy0a5Jw6BP/TExoCmNFYjZl248A0osgPyGEmRA+fAsqPVaNAfytu0vuQJ7rk3J4kTDTR2AlCHJ5cls26opZM4w3jMULh2YXKpcqGBtuleAlOZnaZGbD6DHzMd6i2oFeJ8z9XYmalg1Szd/ocZDc1C7Y6vcALJz2lYnTXiWEr2wawtoR4g3jvWUU2Ngjd1cewtFzEvM1NiHZPeLlIXFbBPawxNgMwwAlyNSuGF3zizVeOoC9bag1qRAQKQE/EZBWC2J8mnXAN2aTBboZ7HewnObE8CwROudZHmUM5oZ/Ugd/JZQK8lvAm43uDRAbyW8gZ+ZGq0EVerVGUKUSm/Idn8AQHdR4m7bue88WBwft9mSCeMOt1ncBwziOmJYI2ZR7ewNMPiCugmSsE4EyQ+QATJG6qORMGd4snEzc6B4shPIo4G1T7PgSm8PY5eUkPdF8JZ0VBtadbHXoJgnEhZQaODPj2gpODKJY5Yp4DOsLBFxWbvXN755KWylJm+oOd4zEL9Hpubuy2gyyfxh8oEfFutnYWdfB8PdESLWYvSqbElP9qo3u6KTmkhoacDauMNNjj0oy40DFV7Ql0aZj77xfGl7TJNHnIwgqOkenruYYNo6h724+zUQ7+vkCpZB+pGA562hYQiDxHVWOq0oDQl/QsoiY+cuI7iWq/ZIBtHcXJ7kks+h2fCNUPA82BzjnqktNts+RLdk1VSu+tqEn7QZCCsvEqk6FkfiOYkrsw092J8jsfIuEKypNjLxrKA9kiA19mxBD2suxQKCzwXGws7kEJvlhUiV9tArLIdZW0IORcxEzdzKmjtFhsjKy/44XYXdI5noQoRcvjZ1RMPACRqYg2V1+OwOepcOknRLLFdYgTkT5UApt/JhLM3jeFYprZV+Zow2g8fP+U68hkKFWJj2yBbKqsrp25xkZX1DAjUw52IMYWaOhab8Kp05VrdNftqwRrymWF4OQSjbdfzmRZirK8FMJELEgER2PHjEAN9pGfLhCUiTJFbd5LBkOBMaxLr/A1SY9dXFz4RjzoU9ExfJCmx/I9FKEGT3n2cmzl2X42L3Jh+AbQq6sA+Ss1kitoa4TAYgKHaoybHUDJ51oETdeI/9ThSmjWGkyLi5QAGWhL0BG1UsTyRGRJOldKBrYJeB8ljLJHfATWTEQBXBDnQexOHTB+Un44zExFE4vLytcu5NwpWrUxO/0ZICUGM7hGABXym0V6ZvDST0E370St9MIWQOTWngeoQHUTdCJUP04spMBMS8LSker9cReVQkULFDIZDFPrhTzBl6sed9wcZQTbL+BDqMyaN3RJPh/anbx+Iv+qgQdAa3M9Z5JmvYlh4qop+Ho1F1W5gbOE9YKLgAnWytXElU4G8GtW47lhgFE6gaSs+gs37sFvi0PPVvA5dnCBgILTwoKd/+DoL9F6inlM7H4rOTzD79KJgKlZO/Zgt22UsKhrAaXU5ZcLrAglTVKJEmNJvORGN1vqrcfSMizfpsgbIe9zno+gBoKVXgIL/VI8dB1O5o/R3Suez/gD7M781ShjKpIIORM/nxG+jjhhgPwsn2IoXsPGPqYHXA63zJ07M2GPEykQwJBYLK808qYxuIew4frk52nhCsnCYmXiR6CuapvE1IwRB4/QftDbEn+AucIr1oxrLabRj9q4ae0+fXkHnteAJwXRbVkR0mctVSwEbqhJiMSZUp9DNbEDMmjX22m3ABpkrPQQTP3S1sib5pD2VRKRd+eNAjLYyT0hGrdjWJZy24OYXRoWQAIhGBZRxuBFMjjZQhpgrWo8SiFYbojcHO8V5DyscJpLTHyx9Fimassyo5U6WNtquUMYgccaHY5amgR3PQzq3ToNM5ABnoB9kuxsebqmYZm0R9qxJbFXCQ1UPyFIbxoUraTJFDpCk0Wk9GaYJKz/6oHwEP0Q14lMtlddQsOAU9zlYdMVHiT7RQP3XCmWYDcHCGbVRHGnHuwzScA0BaSBOGkz3lM8CArjrBsyEoV6Ys4qgDK3ykQQPZ3hCRGNXQTNNXbEb6tDiTDLKOyMzRhCFT+mAUmiYbV3YQVqFVp9dorv+TsLeCykS2b5yyu8AV7IS9cxcL8z4Kfwp+xJyYLv1OsxQCZwTB4a8BZ/5EdxTBJthApqyfd9u3ifr/WILTqq5VqgwMT9SOxbSGWLQJUUWCVi4k9tho9nEsbUh7U6NUsLmkYFXOhZ0kmamaJLRNJzSj/qn4Mso6zb6iLLBXoaZ6AqeWCjHQm2lztnejYYM2eubnpBdKVLORZhudH3JF1waBJKA9+W8EhMj3Kzf0L4vi4k6RoHh3Z5YgmSZmk6ns4fjScjAoL8GoOECgqgYEBYUGFVO4FUv4/YtowhEmTs0vrvlD/CrisnoBNDAcUi/teY7OctFlmARQzjOItrrlKuPO6E2Ox93L4O/4DcgV/dZ7qR3VBwVQxP1GCieA4RIpweYJ5FoYrHxqRBdJjnqbsikA2Ictbb8vE1GYIo9dacK0REgDX4smy6GAkxlH1yCGGsk+tgiDhNKuKu3yNrMdxafmKTF632F8Vx4BNK57GvlFisrkjN9WDAtjsWA0ENT2e2nETUb/n7qwhvGnrHuf5bX6Vh/n3xffU3PeHdR+FA92i6ufT3AlyAREoNDh6chiMWTvjKjHDeRhOa9YkOQRq1vQXEMppAQVwHCuIcV2g5rBn6GmZZpTR7vnSD6ZmhdSl176gqKTXu5E+YbfL0adwNtHP7dT7t7b46DVZIkzaRJOM+S6KcrzYVg+T3wSRFRQashjfU18NutrKa/7PXbtuJvpIjbgPeqd+pjmRw6YKpnANFSQcpzTZgpSNJ6J7uiagAbir/8tNXJ/OsOnRh6iuIexxrmkIneAgz8QoLmiaJ8sLQrELVK2yn3wOHp57BAZJhDZjTBzyoRAuuZ4eoxHruY1pSb7qq79cIeAdOwin4GdgMeIMHeG+FZWYaiUQQyC5b50zKjYw97dFjAeY2I4Bnl105Iku1y0lMA1ZHolLx19uZnRdILcXKlZGQx/GdEqSsMRU1BIrFqRcV1qQOOHyxOLXEGcbRtAEsuAC2V4K3p5mFJ22IDWaEkk9ttf5Izb2LkD1MnrSwztXmmD/Qi/EmVEFBfiKGmftsPwVaIoZanlKndMZsIBOskFYpDOq3QUs9aSbAAtL5Dbokus2G4/asthNMK5UQKCOhU97oaOYNGsTah+jfCKsZnTRn5TbhFX8ghg8CBYt/BjeYYYUrtUZ5jVij/op7V5SsbA4mYTOwZ46hqdpbB6Qvq3AS2HHNkC15pTDIcDNGsMPXaBidXYPHc6PJAkRh29Vx8KcgX46LoUQBhRM+3SW6Opll/wgxxsPgKJKzr5QCmwkUxNbeg6Wj34SUnEzOemSuvS2OetRCO8Tyy+QbSKVJcqkia+GvDefFwMOmgnD7h81TUtMn+mRpyJJ349HhAnoWFTejhpYTL9G8N2nVg1qkXBeoS9Nw2fB27t7trm7d/QK7Cr4uoCeOQ7/8JfKT77KiDzLImESHw/0wf73QeHu74hxv7uihi4fTX+XEwAyQG3264dwv17aJ5N335Vt9sdrAXhPOAv8JFvzqyYXwfx8WYJaef1gMl98JRFyl5Mv5Uo/oVH5ww5OzLFsiTPDns7fS6EURSSWd/92BxMYQ8sBaH+j+wthQPdVgDGpTfi+JQIWMD8xKqULliRH01rTeyF8x8q/GBEEEBrAJMPf25UQwi0b8tmqRXY7kIvNkzrkvRWLnxoGYEJsz8u4oOyMp8cHyaybb1HdMCaLApUE+/7xLIZGP6H9xuSEXp1zLIdjk5nBaMuV/yTDRRP8Y2ww5RO6d2D94o+6ucWIqUAvgHIHXhZsmDhjVLczmZ3ca0Cb3PpKwt2UtHVQ0BgFJsqqTsnzZPlKahRUkEu4qmkJt+kqdae76ViWe3STan69yaF9+fESD2lcQshLHWVu4ovItXxO69bqC5p1nZLvI8NdQB9s9UNaJGlQ5mG947ipdDA0eTIw/A1zEdjWquIsQXXGIVEH0thC5M+W9pZe7IhAVnPJkYCCXN5a32HjN6nsvokEqRS44tGIs7s2LVTvcrHAF+RVmI8L4HUYk4x+67AxSMJKqCg8zrGOgvK9kNMdDrNiUtSWuHFpC8/p5qIQrEo/H+1l/0cAwQ2nKmpWxKcMIuHY44Y6DlkpO48tRuUGBWT0FyHwSKO72Ud+tJUfdaZ4CWNijzZtlRa8+CkmO/EwHYfPZFU/hzjFWH7vnzHRMo+aF9u8qHSAiEkA2HjoNQPEwHsDKOt6hOoK3Ce/+/9boMWDa44I6FrQhdgS7OnNaSzwxWKZMcyHi6LN4WC6sSj0qm2PSOGBTvDs/GWJS6SwEN/ULwpb4LQo9fYjUfSXRwZkynUazlSpvX9e+G2zor8l+YaMxSEomDdLHGcD6YVQPegTaA74H8+V4WvJkFUrjMLGLlvSZQWvi8/QA7yzQ8GPno//5SJHRP/OqKObPCo81s/+6WgLqykYpGAgQZhVDEBPXWgU/WzFZjKUhSFInufPRiMAUULC6T11yL45ZrRoB4DzOyJShKXaAJIBS9wzLYIoCEcJKQW8GVCx4fihqJ6mshBUXSw3wWVj3grrHQlGNGhIDNNzsxQ3M+GWn6ASobIWC+LbYOC6UpahVO13Zs2zOzZC8z7FmA05JhUGyBsF4tsG0drcggIFzgg/kpf3+CnAXKiMgIE8Jk/Mhpkc8DUJEUzDSnWlQFme3d0sHZDrg7LavtsEX3cHwjCYA17pMTfx8Ajw9hHscN67hyo+RJQ4458RmPywXykkVcW688oVUrQhahpPRvTWPnuI0B+SkQu7dCyvLRyFYlC1LG1gRCIvn3rwQeINzZQC2KXq31FaR9UmVV2QeGVqBHjmE+VMd3b1fhCynD0pQNhCG6/WCDbKPyE7NRQzL3BzQAJ0g09aUzcQA6mUp9iZFK6Sbp/YbHjo++7/Wj8S4YNa+ZdqAw1hDrKWFXv9+zaXpf8ZTDSbiqsxnwN/CzK5tPkOr4tRh2kY3Bn9JtalbIOI4b3F7F1vPQMfoDcdxMS8CW9m/NCW/HILTUVWQIPiD0j1A6bo8vsv6P1hCESl2abrSJWDrq5sSzUpwoxaCU9FtJyYH4QFMxDBpkkBR6kn0LMPO+5EJ7Z6bCiRoPedRZ/P0SSdii7ZnPAtVwwHUidcdyspwncz5uq6vvm4IEDbJVLUFCn/LvIHfooUBTkFO130FC7CmmcrKdgDJcid9mvVzsDSibOoXtIf9k6ABle3PmIxejodc4aob0QKS432srrCMndbfD454q52V01G4q913mC5HOsTzWF4h2No1av1VbcUgWAqyoZl+11PoFYnNv2HwAODeNRkHj+8SF1fcvVBu6MrehHAZK1Gm69ICcTKizykHgGFx7QdowTVAsYEF2tVc0Z6wLryz2FI1sc5By2znJAAmINndoJiB4sfPdPrTC8RnkW7KRCwxC6YvXg5ahMlQuMpoCSXjOlBy0Kij+bsCYPbGp8BdCBiLmLSAkEQRaieWo1SYvZIKJGj9Ur/eWHjiB7SOVdqMAVmpBvfRiebsFjger7DC+8kRFGtNrTrnnGD2GAJb8rQCWkUPYHhwXsjNBSkE6lGWUj5QNhK0DMNM2l+kXRZ0KLZaGsFSIdQz/HXDxf3/TE30+DgBKWGWdxElyLccJfEpjsnszECNoDGZpdwdRgCixeg9L4EPhH+RptvRMVRaahu4cySjS3P5wxAUCPkmn+rhyASpmiTaiDeggaIxYBmtLZDDhiWIJaBgzfCsAGUF1Q1SFZYyXDt9skCaxJsxK2Ms65dmdp5WAZyxik/zbrTQk5KmgxCg/f45L0jywebOWUYFJQAJia7XzCV0x89rpp/f3AVWhSPyTanqmik2SkD8A3Ml4NhIGLAjBXtPShwKYfi2eXtrDuKLk4QlSyTw1ftXgwqA2jUuopDl+5tfUWZNwBpEPXghzbBggYCw/dhy0ntds2yeHCDKkF/YxQjNIL/F/37jLPHCKBO9ibwYCmuxImIo0ijV2Wbg3kSN2psoe8IsABv3RNFaF9uMyCtCYtqcD+qNOhwMlfARQUdJ2tUX+MNJqOwIciWalZsmEjt07tfa8ma4cji9sqz+Q9hWfmMoKEbIHPOQORbhQRHIsrTYlnVTNvcq1imqmmPDdVDkJgRcTgB8Sb6epCQVmFZe+jGDiNJQLWnfx+drTKYjm0G8yH0ZAGMWzEJhUEQ4Maimgf/bkvo8PLVBsZl152y5S8+HRDfZIMCbYZ1WDp4yrdchOJw8k6R+/2pHmydK4NIK2PHdFPHtoLmHxRDwLFb7eB+M4zNZcB9NrAgjVyzLM7xyYSY13ykWfIEEd2n5/iYp3ZdrCf7fL+en+sIJu2W7E30MrAgZBD1rAAbZHPgeAMtKCg3NpSpYQUDWJu9bT3V7tOKv+NRiJc8JAKqqgCA/PNRBR7ChpiEulyQApMK1AyqcWnpSOmYh6yLiWkGJ2mklCSPIqN7UypWj3dGi5MvsHQ87MrB4VFgypJaFriaHivwcHIpmyi5LhNqtem4q0n8awM19Qk8BOS0EsqGscuuydYsIGsbT5GHnERUiMpKJl4ON7qjB4fEqlGN/hCky89232UQCiaeWpDYCJINXjT6xl4Gc7DxRCtgV0i1ma4RgWLsNtnEBRQFqZggCLiuyEydmFd7WlogpkCw5G1x4ft2psm3KAREwVwr1Gzl6RT7FDAqpVal34ewVm3VH4qn5mjGj+bYL1NgfLNeXDwtmYSpwzbruDKpTjOdgiIHDVQSb5/zBgSMbHLkxWWgghIh9QTFSDILixVwg0Eg1puooBiHAt7DzwJ7m8i8/i+jHvKf0QDnnHVkVTIqMvIQImOrzCJwhSR7qYB5gSwL6aWL9hERHCZc4G2+JrpgHNB8eCCmcIWIQ6rSdyPCyftXkDlErUkHafHRlkOIjxGbAktz75bnh50dU7YHk+Mz7wwstg6RFZb+TZuSOx1qqP5C66c0mptQmzIC2dlpte7vZrauAMm/7RfBYkGtXWGiaWTtwvAQiq2oD4YixPLXE2khB2FRaNRDTk+9sZ6K74Ia9VntCpN4BhJGJMT4Z5c5FhSepRCRWmBXqx+whVZC4me4saDs2iNqXMuCl6iAZflH8fscC1sTsy4PHeC+XYuqMBMUun5YezKbRKmEPwuK+CLzijPEQgfhahQswBBLfg/GBgBiI4QwAqzJkkyYAWtjzSg2ILgMAgqxYfwERRo3zruBL9WOryUArSD8sQOcD7fvIODJxKFS615KFPsb68USBEPPj1orNzFY2xoTtNBVTyzBhPbhFH0PI5AtlJBl2aSgNPYzxYLw7XTDBDinmVoENwiGzmngrMo8OmnRP0Z0i0Zrln9DDFcnmOoBZjABaQIbPOJYZGqX+RCMlDDbElcjaROLDoualmUIQ88Kekk3iM4OQrADcxi3rJguS4MOIBIgKgXrjd1WkbCdqxJk/4efRIFsavZA7KvvJQqp3Iid5Z0NFc5aiMRzGN3vrpBzaMy4JYde3wr96PjN90AYOIbyp6T4zj8LoE66OGcX1Ef4Z3KoWLAUF4BTg7ug/AbkG5UNQXAMkQezujSHeir2uTThgd3gpyzDrbnEdDRH2W7U6PeRvBX1ZFMP5RM+Zu6UUZZD8hDPHldVWntTCNk7To8IeOW9yn2wx0gmurwqC60AOde4r3ETi5pVMSDK8wxhoGAoEX9NLWHIR33VbrbMveii2jAJlrxwytTHbWNu8Y4N8vCCyZjAX/pcsfwXbLze2+D+u33OGBoJyAAL3jn3RuEcdp5If8O+a4NKWvxOTyDltG0IWoHhwVGe7dKkCWFT++tm+haBCikRUUMrMhYKZJKYoVuv/bsJzO8DwfVIInQq3g3BYypiz8baogH3r3GwqCwFtZnz4xMjAVOYnyOi5HWbFA8n0qz1OjSpHWFzpQOpvkNETZBGpxN8ybhtqV/DMUxd9uFZmBfKXMCn/SqkWJyKPnT6lq+4zBZni6fYRByJn6OK+OgPBGRAJluwGSk4wxjOOzyce/PKODwRlsgrVkdcsEiYrqYdXo0Er2GXi2GQZd0tNJT6c9pK1EEJG1zgDJBoTVuCXGAU8BKTvCO/cEQ1Wjk3Zzuy90JX4m3O5IlxVFhYkSUwuQB2up7jhvkm+bddRQu5F9s0XftGEJ9JSuSk+ZachCbdU45fEqbugzTIUokwoAKvpUQF/CvLbWW5BNQFqFkJg2f30E/48StNe5QwBg8zz3YAJ82FZoXBxXSv4QDooDo79NixyglO9AembuBcx5Re3CwOKTHebOPhkmFC7wNaWtoBhFuV4AkEuJ0J+1pT0tLkvFVZaNzfhs/Kd3+A9YsImlO4XK4vpCo/elHQi/9gkFg07xxnuXLt21unCIpDV+bbRxb7FC6nWYTsMFF8+1LUg4JFjVt3vqbuhHmDKbgQ4e+RGizRiO8ky05LQGMdL2IKLSNar0kNG7lHJMaXr5mLdG3nykgj6vB/KVijd1ARWkFEf3yiUw1v/WaQivVUpIDdSNrrKbjO5NPnxz6qTTGgYg03HgPhDrCFyYZTi3XQw3HXCva39mpLNFtz8AiEhxAJHpWX13gCTAwgm9YTvMeiqetdNQv6IU0hH0G+ZManTqDLPjyrOse7WiiwOJCG+J0pZYULhN8NILulmYYvmVcV2MjAfA39sGKqGdjpiPo86fecg65UPyXDIAOyOkCx5NQsLeD4gGVjTVDwOHWkbbBW0GeNjDkcSOn2Nq4cEssP54t9D749A7M1AIOBl0Fi0sSO5v3P7LCBrM6ZwFY6kp2FX6AcbGUdybnfChHPyu6WlRZ2Fwv9YM0RMI7kISRgR8HpQSJJOyTfXj/6gQKuihPtiUtlCQVPohUgzfezTg8o1b3n9pNZeco1QucaoXe40Fa5JYhqdTspFmxGtW9h5ezLFZs3j/N46f+S2rjYNC2JySXrnSAFhvAkz9a5L3pza8eYKHNoPrvBRESpxYPJdKVUxBE39nJ1chrAFpy4MMkf0qKgYALctGg1DQI1kIymyeS2AJNT4X240d3IFQb/0jQbaHJ2YRK8A+ls6WMhWmpCXYG5jqapGs5/eOJErxi2/2KWVHiPellTgh/fNl/2KYPKb7DUcAg+mCOPQFCiU9Mq/WLcU1xxC8aLePFZZlE+PCLzf7ey46INWRw2kcXySR9FDgByXzfxiNKwDFbUSMMhALPFSedyjEVM5442GZ4hTrsAEvZxIieSHGSgkwFh/nFNdrrFD4tBH4Il7fW6ur4J8Xaz7RW9jgtuPEXQsYk7gcMs2neu3zJwTyUerHKSh1iTBkj2YJh1SSOZL5pLuQbFFAvyO4k1Hxg2h99MTC6cTUkbONQIAnEfGsGkNFWRbuRyyaEZInM5pij73EA9rPIUfU4XoqQpHT9THZkW+oKFLvpyvTBMM69tN1Ydwv1LIEhHsC+ueVG+w+kyCPsvV3erRikcscHjZCkccx6VrBkBRusTDDd8847GA7p2Ucy0y0HdSRN6YIBciYa4vuXcAZbQAuSEmzw+H/AuOx+aH+tBL88H57D0MsqyiZxhOEQkF/8DR1d2hSPMj/sNOa5rxcUnBgH8ictv2J+cb4BA4v3MCShdZ2vtK30vAwkobnEWh7rsSyhmos3WC93Gn9C4nnAd/PjMMtQfyDNZsOPd6XcAsnBE/mRHtHEyJMzJfZFLE9OvQa0i9kUmToJ0ZxknTgdl/XPV8xoh0K7wNHHsnBdvFH3sv52lU7UFteseLG/VanIvcwycVA7+BE1Ulyb20BvwUWZcMTKhaCcmY3ROpvonVMV4N7yBXTL7IDtHzQ4CCcqF66LjF3xUqgErKzolLyCG6Kb7irP/MVTCCwGRxfrPGpMMGvPLgJ881PHMNMIO09T5ig7AzZTX/5PLlwnJLDAPfuHynSGhV4tPqR3gJ4kg4c06c/F1AcjGytKm2Yb5jwMotF7vro4YDLWlnMIpmPg36NgAZsGA0W1spfLSue4xxat0Gdwd0lqDBOgIaMANykwwDKejt5YaNtJYIkrSgu0KjIg0pznY0SCd1qlC6R19g97UrWDoYJGlrvCE05J/5wkjpkre727p5PTRX5FGrSBIfJqhJE/IS876PaHFkx9pGTH3oaY3jJRvLX9Iy3Edoar7cFvJqyUlOhAEiOSAyYgVEGkzHdug+oRHIEOXAExMiTSKU9A6nmRC8mp8iYhwWdP2U/5EkFAdPrZw03YA3gSyNUtMZeh7dDCu8pF5x0VORCTgKp07ehy7NZqKTpIC4UJJ89lnboyAfy5OyXzXtuDRbtAFjZRSyGFTpFrXwkpjSLIQIG3N0Vj4BtzK3wdlkBJrO18MNsgseR4BysJilI0wI6ZahLhBFA0XBmV8d4LUzEcNVb0xbLjLTETYN8OEVqNxkt10W614dd1FlFFVTIgB7/BQQp1sWlNolpIu4ekxUTBV7NmxOFKEBmmN+nA7pvF78/RII5ZHA09OAiE/66MF6HQ+qVEJCHxwymukkNvzqHEh52dULPbVasfQMgTDyBZzx4007YiKdBuUauQOt27Gmy8ISclPmEUCIcuLbkb1mzQSqIa3iE0PJh7UMYQbkpe+hXjTJKdldyt2mVPwywoODGJtBV1lJTgMsuSQBlDMwhEKIfrvsxGQjHPCEfNfMAY2oxvyKcKPUbQySkKG6tj9AQyEW3Q5rpaDJ5Sns9ScLKeizPRbvWYAw4bXkrZdmB7CQopCH8NAmqbuciZChHN8lVGaDbCnmddnqO1PQ4ieMYfcSiBE5zzMz+JV/4eyzrzTEShvqSGzgWimkNxLvUj86iAwcZuIkqdB0VaIB7wncLRmzHkiUQpPBIXbDDLHBlq7vp9xwuC9AiNkIptAYlG7Biyuk8ILdynuUM1cHWJgeB+K3wBP/ineogxkvBNNQ4AkW0hvpBOQGFfeptF2YTR75MexYDUy7Q/9uocGsx41O4IZhViw/2FvAEuGO5g2kyXBUijAggWM08bRhXg5ijgMwDJy40QeY/cQpUDZiIzmvskQpO5G1zyGZA8WByjIQU4jRoFJt56behxtHUUE/om7Rj2psYXGmq3llVOCgGYKNMo4pzwntITtapDqjvQtqpjaJwjHmDzSVGLxMt12gEXAdLi/caHSM3FPRGRf7dB7YC+cD2ho6oL2zGDCkjlf/DFoQVl8GS/56wur3rdV6ggtzZW60MRB3g+U1W8o8cvqIpMkctiGVMzXUFI7FacFLrgtdz4mTEr4aRAaQ2AFQaNeG7GX0yOJgMRYFziXdJf24kg/gBQIZMG/YcPEllRTVNoDYR6oSJ8wQNLuihfw81UpiKPm714bZX1KYjcXJdfclCUOOpvTxr9AAJevTY4HK/G7F3mUc3GOAKqh60zM0v34v+ELyhJZqhkaMA8UMMOU90f8RKEJFj7EqepBVwsRiLbwMo1J2zrE2UYJnsgIAscDmjPjnzI8a719Wxp757wqmSJBjXowhc46QN4RwKIxqEE6E5218OeK7RfcpGjWG1jD7qND+/GTk6M56Ig4yMsU6LUW1EWE+fIYycVV1thldSlbP6ltdC01y3KUfkobkt2q01YYMmxpKRvh1Z48uNKzP/IoRIZ/F6buOymSnW8gICitpJjKWBscSb9JJKaWkvEkqinAJ2kowKoqkqZftRqfRQlLtKoqvTRDi2vg/RrPD/d3a09J8JhGZlEkOM6znTsoMCsuvTmywxTCDhw5dd0GJOHCMPbsj3QLkTE3MInsZsimDQ3HkvthT7U9VA4s6G07sID0FW4SHJmRGwCl+Mu4xf0ezqeXD2PtPDnwMPo86sbwDV+9PWcgFcARUVYm3hrFQrHcgMElFGbSM2A1zUYA3baWfheJp2AINmTJLuoyYD/OwA4a6V0ChBN97E8YtDBerUECv0u0TlxR5yhJCXvJxgyM73Bb6pyq0jTFJDZ4p1Am1SA6sh8nADd1hAcGBMfq4d/UfwnmBqe0Jun1n1LzrgKuZMAnxA3NtCN7Klf4BH+14B7ibBmgt0TGUafVzI4uKlpF7v8NmgNjg90D6QE3tbx8AjSAC+OA1YJvclyPKgT27QpIEgVYpbPYGBsnyCNrGz9XUsCHkW1QAHgL2STZk12QGqmvAB0NFteERkvBIH7INDsNW9KKaAYyDMdBEMzJiWaJHZALqDxQDWRntumSDPcplyFiI1oDpT8wbwe01AHhW6+vAUUBoGhY3CT2tgwehdPqU/4Q7ZLYvhRl/ogOvR9O2+wkkPKW5vCTjD2fHRYXONCoIl4Jh1bZY0ZE1O94mMGn/dFSWBWzQ/VYk+Gezi46RgiDv3EshoTmMSlioUK6MQEN8qeyK6FRninyX8ZPeUWjjbMJChn0n/yJvrq5bh5UcCAcBYSafTFg7p0jDgrXo2QWLb3WpSOET/Hh4oSadBTvyDo10IufLzxiMLAnbZ1vcUmj3w7BQuIXjEZXifwukVxrGa9j+DXfpi12m1RbzYLg9J2wFergEwOxFyD0/JstNK06ZN2XdZSGWxcJODpQHOq4iKqjqkJUmPu1VczL5xTGUfCgLEYyNBCCbMBFT/cUP6pE/mujnHsSDeWxMbhrNilS5MyYR0nJyzanWXBeVcEQrRIhQeJA6Xt4f2eQESNeLwmC10WJVHqwx8SSyrtAAjpGjidcj1E2FYN0LObUcFQhafUKTiGmHWRHGsFCB+HEXgrzJEB5bp0QiF8ZHh11nFX8AboTD0PS4O1LqF8XBks2MpjsQnwKHF6HgaKCVLJtcr0XjqFMRGfKv8tmmykhLRzu+vqQ02+KpJBjaLt9ye1Ab+BbEBhy4EVdIJDrL2naV0o4wU8YZ2Lq04FG1mWCKC+UwkXOoAjneU/xHplMQo2cXUlrVNqJYczgYlaOEczVCs/OCgkyvLmTmdaBJc1iBLuKwmr6qtRnhowngsDxhzKFAi02tf8bmET8BO27ovJKF1plJwm3b0JpMh38+xsrXXg7U74QUM8ZCIMOpXujHntKdaRtsgyEZl5MClMVMMMZkZLNxH9+b8fH6+b8Lev30A9TuEVj9CqAdmwAAHBPbfOBFEATAPZ2CS0OH1Pj/0Q7PFUcC8hDrxESWdfgFRm+7vvWbkEppHB4T/1ApWnlTIqQwjcPl0VgS1yHSmD0OdsCVST8CQVwuiew1Y+g3QGFjNMzwRB2DSsAk26cmA8lp2wIU4p93AUBiUHFGOxOajAqD7Gm6NezNDjYzwLOaSXRBYcWipTSONHjUDXCY4mMI8XoVCR/Rrs/JLKXgEx+qkmeDlFOD1/yTQNDClRuiUyKYCllfMiQiyFkmuTz2vLsBNyRW+xz+5FElFxWB28VjYIGZ0Yd+5wIjkcoMaggxswbT0pCmckRAErbRlIlcOGdBo4djTNO8FAgQ+lT6vPS60BwTRSUAM3ddkEAZiwtEyArrkiDRnS7LJ+2hwbzd2YDQagSgACpsovmjil5wfPuXq3GuH0CyE7FK3M4FgRaFoIkaodORrPx1+JpI9psyNYIFuJogZa0/1AhOWdlHQxdAgbwacsHqPZo8u/ngAH2GmaTdhYnBfSDbBfh8CHq6Bx5bttP2+RdM+MAaYaZ0Y/ADkbNCZuAyAVQa2OcXOeICmDn9Q/eFkDeFQg5MgHEDXq/tVjj+jtd26nhaaolWxs1ixSUgOBwrDhRIGOLyOVk2/Bc0UxvseQCO2pQ2i+Krfhu/WeBovNb5dJxQtJRUDv2mCwYVpNl2efQM9xQHnK0JwLYt/U0Wf+phiA4uw8G91slC832pmOTCAoZXohg1fewCZqLBhkOUBofBWpMPsqg7XEXgPfAlDo2U5WXjtFdS87PIqClCK5nW6adCeXPkUiTGx0emOIDQqw1yFYGHEVx20xKjJVYe0O8iLmnQr3FA9nSIQilUKtJ4ZAdcTm7+ExseJauyqo30hs+1qSW211A1SFAOUgDlCGq7eTIcMAeyZkV1SQJ4j/e1Smbq4HcjqgFbLAGLyKxlMDMgZavK5NAYH19Olz3la/QCTiVelFnU6O/GCvykqS/wZJDhKN9gBtSOp/1SP5VRgJcoVj+kmf2wBgv4gjrgARBWiURYx8xENV3bEVUAAWWD3dYDKAIWk5opaCFCMR5ZjJExiCAw7gYiSZ2rkyTce4eNMY3lfGn+8p6+vBckGlKEXnA6Eota69OxDO9oOsJoy28BXOR0UoXNRaJD5ceKdlWMJlOFzDdZNpc05tkMGQtqeNF2lttZqNco1VtwXgRstLSQ6tSPChgqtGV5h2DcDReIQadaNRR6AsAYKL5gSFsCJMgfsaZ7DpKh8mg8Wz8V7H+gDnLuMxaWEIUPevIbClgap4dqmVWSrPgVYCzAoZHIa5z2Ocx1D/GvDOEqMOKLrMefWIbSWHZ6jbgA8qVBhYNHpx0P+jAgN5TB3haSifDcApp6yymEi6Ij/GsEpDYUgcHATJUYDUAmC1SCkJ4cuZXSAP2DEpQsGUjQmKJfJOvlC2x/pChkOyLW7KEoMYc5FDC4v2FGqSoRWiLsbPCiyg1U5yiHZVm1XLkHMMZL11/yxyw0UnGig3MFdZklN5FI/qiT65T+jOXOdO7XbgWurOAZR6Cv9uu1cm5LjkXX4xi6mWn5r5NjBS0gTliHhMZI2WNqSiSphEtiCAwnafS11JhseDGHYQ5+bqWiAYiAv6Jsf79/VUs4cIl+n6+WOjcgB/2l5TreoAV2717JzZbQIR0W1cl/dEqCy5kJ3ZSIHuU0vBoHooEpiHeQWVkkkOqRX27eD1FWw4BfO9CJDdKoSogQi3hAAwsPRFrN5RbX7bqLdBJ9JYMohWrgJKHSjVl1sy2xAG0E3sNyO0oCbSGOxCNBRRXTXenYKuwAoDLfnDcQaCwehUOIDiHAu5m5hMpKeKM4sIo3vxACakIxKoH2YWF2QM84e6F5C5hJU4g8uxuFOlAYnqtwxmHyNEawLW/PhoawJDrGAP0JYWHgAVUByo/bGdiv2T2EMg8gsS14/rAdzlOYazFE7w4OzxeKiWdm3nSOnQRRKXSlVo8HEAbBfyJMKqoq+SCcTSx5NDtbFwNlh8VhjGGDu7JG5/TAGAvniQSSUog0pNzTim8Owc6QTuSKSTXlQqwV3eiEnklS3LeSXYPXGK2VgeZBqNcHG6tZHvA3vTINhV0ELuQdp3t1y9+ogD8Kk/W7QoRN1UWPqM4+xdygkFDPLoTaumKReKiLWoPHOfY54m3qPx4c+4pgY3MRKKbljG8w4wvz8pxk3AqKsy4GMAkAtmRjRMsCxbb4Q2Ds0Ia9ci8cMT6DmsJG00XaHCIS+o3F8YVVeikw13w+OEDaCYYhC0ZE54kA4jpjruBr5STWeqQG6M74HHL6TZ3lXrd99ZX++7LhNatQaZosuxEf5yRA15S9gPeHskBIq3Gcw81AGb9/O53DYi/5CsQ51EmEh8Rkg4vOciClpy4d04eYsfr6fyQkBmtD+P8sNh6e+XYHJXT/lkXxT4KXU5F2sGxYyzfniMMQkb9OjDN2C8tRRgTyL7GwozH14PrEUZc6oz05Emne3Ts5EG7WolDmU8OB1LDG3VrpQxp+pT0KYV5dGtknU64JhabdqcVQbGZiAxQAnvN1u70y1AnmvOSPgLI6uB4AuDGhmAu3ATkJSw7OtS/2ToPjqkaq62/7WFG8advGlRRqxB9diP07JrXowKR9tpRa+jGJ91zxNTT1h8I2PcSfoUPtd7NejVoH03EUcqSBuFZPkMZhegHyo2ZAITovmm3zAIdGFWxoNNORiMRShgwdYwFzkPw5PA4a5MIIQpmq+nsp3YMuXt/GkXxLx/P6+ZJS0lFyz4MunC3eWSGE8xlCQrKvhKUPXr0hjpAN9ZK4PfEDrPMfMbGNWcHDzjA7ngMxTPnT7GMHar+gMQQ3NwHCv4zH4BIMYvzsdiERi6gebRmerTsVwZJTRsL8dkZgxgRxmpbgRcud+YlCIRpPwHShlUSwuipZnx9QCsEWziVazdDeKSYU5CF7UVPAhLer3CgJOQXl/zh575R5rsrmRnKAzq4POFdgbYBuEviM4+LVC15ssLNFghbTtHWerS1hDt5s4qkLUha/qpZXhWh1C6lTQAqCNQnaDjS7UGFBC6wTu8yFnKJnExCnAs3Ok9yj5KpfZESQ4lTy5pTGTnkAUpxI+yjEldJfSo4y0QhG4i4IwkRFGcjWY8+EzgYYJUK7BXQksLxAww/YYWBMhJILB9e8ePEJ4OP7z+4/wOQDl64iOYDp26DaONPxpKtBxq/aTzRGarm3VkPYTLJKx6Z/Mw2YbBGseJhPMwhhNswrIkyvV2BYzrvZbxLpKwcWJhYmFtVZ+lPEq91FzVp1HlQY1bZVLqeNR9SAUn6n0E28k/UuGkNpP1DBI5ch/EehZfjUQ9aE41NhETExoPT2gGQz0IhWJbEOvTQ4wgcXCHHFBhewYUiFHuhRSAUVmEHeCRQHQkXGFwkAgyzREJCVN7TRnTon36Zw3tPhx4EALwNdwDv+J41YSP4B2CQqz0EFgARZ4ESgBHQgROwAVn9GTI+HYexTUevLUeta4/DqKrbMVS+Yqb8hUwYCrlgKtmAq1YCrFgKrd4qpXiqZcKn1oqdWipjYKpWwVPVYqW6xUpVipKqFR3QKjagVEtAqHpxUMTitsnFaJOKx2cVhswq35RVpyiq9lFVNIKnOQVMkgqtYxVNxiqQjFS7GKlSIVIsQqPIhUWwioigFQ++KkN8VHr49HDw9Ebo9EDo9DTo9Crg9BDg9/Wx7gWx7YWwlobYrOGxWPNisAaAHEyALpkAVDIAeWAArsABVXACYuAD5cAF6wAKFQAQqgAbVAAsoAAlQAUaYAfkwAvogBWQACOgAD9AAHSAAKT4GUdMiOvFngBTwCn2AZ7Dv6B6k/90B8+yRnkV144AIBoAMTQATGgAjNAA4YABgwABZgB/mQCwyAVlwCguASlwCEuAQFwB4uAMlwBYuAJlQAUVAAhUD2KgdpUDaJgaRMDFJgX5MC1JgWJEAokQCWRAHxEAWkQBMRADpEAMkQAYROAEecC484DRpwBDTnwNOdw05tjTmiNOYwtswhYFwLA7BYG4LA2BYGOLAwRYFuLAsxYFQJAohIEyJAMwkAwiQC0JAJgkAeiQBkJAFokAPCQA0JABwcD4Dgc4cDdDgaYcDIDgYgUC6CgWgUClCgUYUAVBQBOFAEYMALgwAgDA9QYAdIn8AZzeBB2L5EcWrenUT1KXienEsuJJ7x5U8XlTjc1NVzUyXFTGb1LlpUtWlTDIjqwE4LsagowoCi2gJLKAkpoBgJQNpAIhNqaEoneI6kiiqQ6Go/n6j0cS+a2gEU8gIHJ+BwfgZX4GL+Bd/gW34FZ+BS/gUH4FN6BTegTvoEv6BJegRnYEF2A79gOvYDl2BdEjCkqkGtwXp0LNToIskOTXzh/F062yJ7AAAAEDAWAAABWhJ+KPEIJgBFxMVP7w2QJBGHASQnOBKXKFIdUK4igKA9IEaYJg) format('embedded-opentype'),url(data:application/font-woff;base64,d09GRgABAAAAAFuAAA8AAAAAsVwAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABWAAAABwAAAAcbSqX3EdERUYAAAF0AAAAHwAAACABRAAET1MvMgAAAZQAAABFAAAAYGe5a4ljbWFwAAAB3AAAAsAAAAZy2q3jgWN2dCAAAAScAAAABAAAAAQAKAL4Z2FzcAAABKAAAAAIAAAACP//AANnbHlmAAAEqAAATRcAAJSkfV3Cb2hlYWQAAFHAAAAANAAAADYFTS/YaGhlYQAAUfQAAAAcAAAAJApEBBFobXR4AABSEAAAAU8AAAN00scgYGxvY2EAAFNgAAACJwAAAjBv+5XObWF4cAAAVYgAAAAgAAAAIAFqANhuYW1lAABVqAAAAZ4AAAOisyygm3Bvc3QAAFdIAAAELQAACtG6o+U1d2ViZgAAW3gAAAAGAAAABsMYVFAAAAABAAAAAMw9os8AAAAA0HaBdQAAAADQdnOXeNpjYGRgYOADYgkGEGBiYGRgZBQDkixgHgMABUgASgB42mNgZulmnMDAysDCzMN0gYGBIQpCMy5hMGLaAeQDpRCACYkd6h3ux+DAoPD/P/OB/wJAdSIM1UBhRiQlCgyMADGWCwwAAAB42u2UP2hTQRzHf5ekaVPExv6JjW3fvTQ0sa3QLA5xylBLgyBx0gzSWEUaXbIoBBQyCQGHLqXUqYNdtIIgIg5FHJxEtwqtpbnfaV1E1KFaSvX5vVwGEbW6OPngk8/vvXfv7pt3v4SImojIDw6BViKxRgIVBaZwVdSv+xvXA+Iuzqcog2cOkkvDNE8Lbqs74k64i+5Sf3u8Z2AnIRLbyVCyTflVSEXVoEqrrMqrgiqqsqqqWQ5xlAc5zWOc5TwXucxVnuE5HdQhHdFRHdNJndZZndeFLc/zsKJLQ/WV6BcrCdWkwspVKZVROaw0qUqqoqZZcJhdTnGGxznHBS5xhad5VhNWCuturBTXKZ3RObuS98pb9c57k6ql9rp2v1as5deb1r6s9q1GV2IrHSt73T631424YXzjgPwqt+Rn+VG+lRvyirwsS/KCPCfPytPypDwhj8mjctRZd9acF86y89x55jxxHjkPnXstXfbt/pNjj/nwXW+cHa6/SYvZ7yEwbDYazDcIgoUGzY3h2HtqgUcs1AFPWKgTXrRQF7xkoQhRf7uF9hPFeyzUTTSwY6EoUUJY6AC8bSGMS4Ys1Au3WaiPSGGsMtkdGH2rzJgYHAaYjxIwQqtB1CnYkEZ9BM6ALOpROAfyqI/DBQudgidBETXuqRIooz4DV0AV9UV4GsyivkTEyMMmw1UYGdhkuAYjA5sMGMvIwCbDDRgZeAz1TXgcmDy3YeRhk+cOjCxsMjyAkYFNhscwMrDJ8BQ2886gXoaRhedQvyTSkDZ7uA6HLLQBI5vGntAbGHugTc53cMxC7+E4SKL+ACOzNpk3YWTWJid+iRo5NXIKM3fBItAPW55FdJLY3FeHBDr90606JCIU9Jk+Ms3/Y/8L8jUq3y79bJ/0/+ROoP4v9v/4/mj+i7HBXUd0/elU6IHfHt8Aj9EPGAAoAvgAAAAB//8AAnjaxb0JfBvVtTA+dxaN1hltI1m2ZVuSJVneLVlSHCdy9oTEWchqtrBEJRAgCYEsQNhC2EsbWmpI2dqkQBoSYgKlpaQthVL0yusrpW77aEubfq/ly+ujvJampSTW5Dvnzmi1E+jr//3+Xmbu3Llz77nnbuece865DMu0MAy5jGtiOEZkOp8lTNeUwyLP/DH+rEH41ZTDHAtB5lkOowWMPiwayNiUwwTjE46AI5xwhFrINPXYn/7ENY0dbWHfZAiTZbL8ID/InAd5xz2NpIH4STpDGonHIJNE3OP1KG4ISaSNeBuITAyRLgIxoiEUhFAnmUpEiXSRSGqAQEw0kuyFUIb0k2gnGSApyBFi0il2SI5YLGb5MdFjXCey4mNHzQ7WwLGEdZiPPgYR64we8THZHAt+wnT84D/x8YTpGPgheKH4CMEDVF9xBOIeP3EbQgGH29BGgpGkIxCMTCW9qUTA0Zsir+QUP1mt+P2KusevwIO6Bx/Iaj8/OD5O0VNrZW2EsqZBWbO1skRiEKE0DdlKKaSVO5VAuRpqk8VQJAqY7ydxaK44YJvrO2EWjOoDBoFYzQbDNkON+UbiKoRkywMWWf1j4bEY2iIY1AeMgvmEz/kVo9v4FSc/aMZMrFbjl4zWLL0+Y5FlyzNlEVYDudJohg8gPUP7kcB/mn+G6cd+5PV4Q72dXCgocWJADBgUuDTwiXiGSyZo14HOEQ2lE6k0XDIEusexDzZOMXwt1Dutz+tqmxTvlskNWXXUQIbhaurum9GrePqm9Yaeabjkiqf+bUvzDOvb2Y1E+EX2DnemcTP/zLcuu7xjQXdAtjR0Lo5n4/Hs/GtntMlysHt+29NXbH6se//WbFcyu+r28H0MwzI30DYeYTLMXIA2EG8QlHpAsyS0EfEToR0a3utIxFPJ3kiIHCCrZ66b0e2xEmL1dM9YN/MwS5p01N5jMX/BLKt/1R83l0LyC29M6+iYxo/UNg/EF7c2WyyW5tYl8WnhWg2/hyySbD5UhnDyS7OcU0dnrFw+DfGdI7v4QfYIIzOMq9hFtY55gmvC7jZ2FK7sEdrn6IXBuucYhjsGdQ8z0yEbWkkczjjsE5hNAIZrPx2zOLZDmKNXcXtg7EMqidAEEWg+SJCBBNwxvxJfc/bZa+KKf+xoKZybnq5vaqpPTye7CiF+ZFjxZ8/7Qij0hfOG/cowPA1rT1l4ymWnrKmxxqfErTVrpgwPlz1kC+Oy8NMDz6c+IO38K/x0xkPnLW8Kx6qGAoQdL+TD9V9rb+/ctn//trxz8dUrZrD/zk/ferF0cNt1BzctmX2FZPXt/jnFCQNz4Ah/iKllGiCMs1w5Lkg0kiEwj6VTXCDKsX9rMpnvIj9pcDecXAIXMnqn2dTUbN6w0XQ9ue6FV/nnXCH7S3lPWGltVcLsH75ub3ab7A8M28caNrIeOr3o5Q0yFsYL80xaa0EY/UEczV7icUMY5pnelAkmUAXmHYjvFWFGxuqlSaow3OM+/iYY7/l/hVELF4EjRqNR/bvRbOY+DUGzGR/Oh3EqmE/ugIQQguGt/eMYz/+L0cimjeZfQDI3phXMbMQsqH+CjwVz/hf4idHovgVmB8gLvjbicDcC/NypP536E/9N/puMibExdohBmNwyiaZdJGoigos7GpF222xrfnZhML/7Z+ylaqP63Hr+m7bdUkQ6/2cXqdfmvwixY+s2ksXFeXcE+iX0Z+Iow76DBNgjJ7TOdUK18iPsPflfQD+DPsZG2Aj9VmKMMJ4fYRrhIaxhTDR0Elh2vA6h/AE6xUb29mj3sjmL72petXjejPy+oel60M99tFduCI59N3221xe7apOvxs6aHs7vab1IqY2tv7q2xsHeHGml/cV06u/8S/xTjJ+JYc0bWEX0ukW6YmIbGkJRMdjJ9mYIH5QIdJF4hvRGyK7cC7ctImQRcUET99fGXOoft35GYLMQu+g2smnkgZUrH8AL/9Si217IssJ916nv14ZrJrvdxLkQvrvtBcjgPC0NXOicO8Qf4mcxPqh3hgUw3DDfdvLJXngg7N3dN2zbPJSaed3OfZnMU7dvmznp3C3bruO+Nmue0LFsy7S+6265+fCKFYdvvuW6vmlblnUI8xCXp37CrOZv4B9gauDBlYp7adcUXB5DNCwYImlXOJJKkAdvExXxVvKEYnCo+3eIskP9qrrfIYs71CccBjfXRC52udTHHdaP1A1ui/VvH1otbrLrpNXBsGX5B89QghDyimlvNB2KfkxZ5C9/em3+d1+d//IfFp2+2Oxn/s+9n/79p39S3s8idN6g0yZObwJOgKUpNB3GyU0Ls0PbRzIRq4lcarLKOJBkLRzJQD4j2090XrbA7DW8K3jNF5hlGS5e4V2D17zgss4T20egOJte5iD0bReM9yjTxnQxCRj3c5kFzGJmGbNKmwGw39IJDJcXJZGMkaAB4jyJAKw0jt5IAuIE+A+U3cVAZZrq9zhDyBrU8oosuxcGNTzCKJfla7JjNVmuSb/+tuzN2H+X4vlB+PpdfMXXmuVsNiub1T34SFbjYw5itEvVi0K0Nt9pNJUMI7SLGRhf2xipfCYf8z5OdlGKayOucFeVPeS/dbo3lBrbSMmwUiQN5/ed7g0Ds1s17IuZC5kNzM3MZ6EWCa0DtekdJfAxz+R/OX28sND7yRMTBcf++s8mQCQWHya4qBv/ufeMoWyslPA9DtMxUknxkH/yfTnm2CMYzs+Cq3r7PxY/MXomrvTEsRpfEGHa+WN8E1AHjElb7d06ddA7oK/+5Mdsv9EtPms0jv0Z5kf1FqPxWdFtfFr0kHfgDX0Y+5PRSG7RUj0tQr7rmfX8DH4G5W28kKeJLtmQsQkuwMP1pk16EV4sl7vrMJATfyUWo/GwEco4rh4XFQgaiUX9qxZHrMQqKnz/c2d8b9TysYrAuXpP/Rf/Gr8b1qwwc5a+euLa6S6sneNXToG2XrEJi4R5SGs8Sq2S3d97bsfCRaTdaLwKClRHt37mkudvXbjwVrLhuYeGhh56bvfQkHpk2CwvwClqgWwuBfndC3c8dwmstj81KkagcUgbfPY8Zje0W/82VPWJHmSq6pP8hPWpotc/EexDOK3qU+wngPhOCiO9MJRm8TJefjelrzoKnG2Bn+1NCUmPE4gHFmBN9jrTigRIpsACrc9Gstg58ULkp9467+Gf/eFnD5/31lNrt2967dhrm7bzI+VT5m+fzKhvf2MzpICEm79Bopkn07lt1762adNr127LwVqQLdJ5+lpQDcvHPQtVY5knhYrK6q8/JsiP6EuhGZdFdaNszjvpqvc+PI0CdjN0AXsFOC3ZfALDJwr4q2Xq+GF+GNbsxUg5NLLIEXi8otcDQcUts0D8eQ1iVDRAMBTsYiNdRIxE09EIBJO9A2xqgERTaW86BUFn0OD2xFO97FAgFhF6OoQ7prYt4XwSeUgQHiJyDbeke9IdQntciLQ1FlJMaYcUNvZBg+FB1ubjlnRNvl3o6IEU2w7fdNPhm/hh+FLysUu6++DLHkOkrSHYEjH0tEPe7WdD3uyDgvAgK/m4szFFR7ch0toUgBTdWHr7EpaWru6+6dmbbnqWEbV2EtxAsXiZAPTtGPSbHsotI2leoM8TePEqgSQprs7AGFf8kuOkPdZPXGb55POAW1d/jLST9v5YflasP6v/CO7+GNAPC2BMZWmsOjp2NNbfHwMCJD+LPVL+D/OYlWEEI/9jpPddOFkB5d1GSuKZYggmCCd7JUxD7EXAzxyirYnNDLdDZoFdx14kivkvGc3579Jm36reTTvDgBnaO6vzyQ6chQmlsMoIkIQ2+bBDWBud1Va4pcCn8CPqxlh/fgtG8IPaPH8C5wk6/nZDv69jurV5QhtwE0x2iqOsj9Mx8B9/0EaUdiPfOYYDCi/q9jhWRuupMDEU0+CtX0sDFxv07T/K5niBPqN9+tQjgEc31NGCXFeMcCEuQBIc/BK4CO78u7EPYvl3yaEfK3vcb6qP1R2tI7vUjVDDUdKubsSrNjYKY1qBEa2P50SJoaXiksIoLiCwnxS6EBuBde87botNfdEWwYvF/R0/u5yCqhGeEOR2ynSeyXjt6ka7neyye8kryBSWE52y+RBgogrXPZ8E1yIHoHIFUM+AbJhE7lbMtt8ApL+xmZW7PwbjAO0fAVoXQOuiSP/ksIVdFZ0aulsamKUzwPZ/NYDMJRBPCxsBqLzqHyneXF6Ej9HlIFo7+pg+jUb3unRmGpstGkm6etOuDBGA5wCMefp1gTHcdZlvPBXlOslvYTp1cd8UjYLVd/J5awNrIOKLnIt9MD9qdrKrWCvA6ALm3QV9VrsPm60Q7+RHJHP+2hqfugo/MvI2H/mqr4b9tFnKSRY1Y5Ek80Nm/WIhr1ikKnxGz9TWXrokf9xwujfvcOTtNTWnxd0F37Y2W79tteBqZ4G5qLCuomw+nSr28QESCRVLTyYKILGJOPfcnaIFOsewhRdvv+rWa/Wih0vlbX6Zb75T5C0qNKVFvH1QL/vazSWgC2s6oWXXIuUxQelKiJbowuJDQViatLmLijg9CQBMg8WiPgiw3LEeYRmm5f+XdnvkDnxLLjMLxtvX74C3OlwPQqx4xwIdpPx38LrlDphiyWUWHWKAzzxurS/xTo+P5wGFak62ap1PVFFN4v/y+xuR39WnIO7lsWfwgVsK17wxrs9K8ltIKuhkw7f/6dhK6gQokFKhWX3urrjk/rnI0pgfpGMeuQIUaEM7+GF5q2iMkCaMQwxxOzcvU0eXbsnS9XknXvP7Gtw5dwPXlFu2ecvSHEZgNDsU6x/GdXBYXyOQjzZReSedeEPY6nEv9gJR4oBQJtFO6Kd0fwC6BO4LNHDeBujB6dSNcUQC9zIv2LnAzGk99bUDrdFY+9yGFQtEo0GQPNv6vS2drj4+1jHbv3aJSMUWP+QTZrmbNTjU8wyG/iXNNpskybLcJ3CiTF5Ir+JYzmJwE0mSVhlxbtbmvweB3ulB6Til5UuUZydpgiFVeobhU0WaBqpJ198d+/XeNRTZ9/1OPfG7+2hwzd5W3D+hmyjsRcUg/+Cavb++Vh2ls3L7zT/etOnHNxeerv313vzLVqPai4nJv+K1FC6040/4udw7sAb3laSg0XCkAAs0npBO6VJabS4Elk/U+D4gTXW+j0wnrMlqNamq4tMIYB87tE10i0FR3LZNhJsb7/R561btmes8YBCRkhYNByRtKd55mqTas9FYhJnbRGHuOh3M4QTdgQSqmgRxuzGdSvZGcbMxNQGk5C3ebLjoXIOFM4l+WKHmLTJwRv9E8GWJ6dYvf/FmEyEGr+gyrr1p5zrgkz0Cw2j94Hv8Jdx7dIVegBSNtgsqGsRQEYiIBoXwD0LNvQ5d7s5Z00QzwNhqZA0b+tMG1tQq5nd84uq8R0zPvX35G8uRaze4jcOHzz0w1+Q2BIRvf6J6Kgatnrbiem+CFvAxfkrndzD9MFPP1GWTUHclpASUkCNAQkpCCcCgDSUDAhDZ+CuEkgn8J7i9nMA7pA4lISappxILKfAeSAbIcSDuN2bJcfZILqeO5rLs0MnngSHYRdrHjmaz7JEsEPw51ZqDJDmUIOZIe34WaQeegNsJn1qz8AIpT3yCjyEih/xELkuJ0lEMYTLVCiWpo5oYMleMH6USyYJcD+uOe+kWKpn1Qns34iyYDjkSLvgnZXcgVQNeqINXr48m3iS7cjm8tedyY0f1QvTnHHdsrKby/+SSbPY8/NH6vpl/Esq3Ae4ZU1HC44KFiI9o7CEgab/RqHbj7s5KAg06s39ZP/zxI/mVuF/TbTSy+3Fb8If9/cv7+wt91yy8RfP1QXtW5RzQn7qIiZyuFM5QfJ5E9uVnqT85TanFx0lkP3ukBAMprvsRyi/C8NAJL1xbIIirSvnSj4O5netb4JxmNANHPssHAcHMHsFRgEug816gDBeMbdfiuRcghqYcm0+Xxx/5IAEtN3fqFF3LzAXqwoT0PN0OVTNqxo8sxMkd5Ig6k79Zk7VxxX6gMLOZFQgvpW2RrMW1D0BDihaXQ9wVRoBxPLfpknmkeMtoB/qM9cRc9IqmMD2XUmdZ7GSRKPUZvChf8BoykriM2MnKYbOHX8R7cLdNCxSFFVQqoYswnlWtlFS2mNkhswVpZiQW1J/UKFfipHGlUkM6UKBhMz1istELIHJLMSctu3ugzfaVSOjKvUgc/THK4Sdg2Wscz69leKIkkrwuuWiOe9yGYKQXRumkC3qbRcMwrvhjNXgdZk3RxAUEhuSPvn3nnd++U/3vlVOmrJzCD8JLxV1OHRjrZifbcFDOuRNTGqdgQm1tSNJ2OcQ04YiEXuxtII1ECSQRoQGYioEsgCfchB4ghAtw7FfJre4WZ9hkVi9MtjuWqtdNDlpMrfEG9fOT6q21okg+e4As38MfGquNt7oUws6Ysarj1/efE+yst86YUVNvDdts3Pv5c8m/aP0C+f8/Qb+IMnGq09BgwN01oIOAnAdagI8mBSrqk1gxTDUBOtk2ousEtBH2z4Ir2d3f6k8PXXVlt2qN9RODxRuoJT/v27wm09jRYVc/e++iyx2tyzJb/n3J0htXP87eSsQaf2Ly0s6Zmxela88REy1cf4273mI3iXNJ7KxrZibOm9xm6rl4fqy/t27smU8tOfdW2ucBzg2UfmOIVyLIl3kpYlwphDISTXJXsctmiDtN7fNV6zelgxwnWxsVr83Aj/S5ki1jL/a0GC6+2L6Um+aoddlNFuj+bJ8mH/iaLh8I0/U51NspIEfq0dohwyFXKgm4NggwQ4rRhCOUFtxxo8XnitT4cnGfT93IS8FaT85XE3H5LMY4zIEPL1hw443wz+1UmhTJyJGxZzw+wsKkKZgUiVtKOKMEb2AKHTv61FNc01PQFwKnvsZ/9pPA4RKTASWahmh+8MxwzHxKy74IRn5LGRjsPUUwTu64UYNY38caqd7HKucZ/tHnODtENw/2UfHRMaq1UUPDJQ0OKkWCeet5fYOhII1VRz8+/Elg5j4Gxur3J8o2PJ4rg+2d08T/fwEzSVbyZ9XPro95T477lRKqUSRXQnauHNsISAl27oWi6Fv9z48JMv8r/aMMj8onCP/DuDZOuN+GPPr/+p7bx+7JlbYdppcNhzKU/1Px5aiaGDn/s1iGMaBcleKUo/v9rcxkZj7DBEKOfrayytXNLYiUdBY+pleQXdnscKlQcpzuWluxsieeyuXIK6SdxozitWyGOV3vOHHjguyCQ6fpIYy2JwvrQEF/Qa9Pdf/QqOSqCiE/EE1/XIVKTc2tzWbHnimrEd+Vyz311Ml3P0GVTj7PD5aDnsvCvH36alEaPMePcMegXs7x8igTu4B9v7G9vTHvhCu/kzIdx+BxC0ay9zRSvoS0F2lIxI+X7klU63I40gLQ3w5ep5na+SFnba3z5D64zv+QtM4n4ffG3tq4aNHGRfxgrXPMim+5487abL7xhdseIRn1KDl+7aINixdv0OD+JSPwKf5+xoP6aiTeQIDVlIhMcL1H5R9PYXvprs3fv2bO7MOplCmweuiq2JRZ1zz+9a/v2PH1Hfz9236w+ZrPXvWfAxlj4NLLHpq3c/PQ3uvmvbrjG7fe+o2y/cLdtE6VUlXi0ASb1VLUBVSUWSU4HdvAraTyS8xzM8NxvxFkXV6pUVRiJwcgC5zEeht4rwcp7ki0k41G0qlQhG1Vzlq8alEmnFi58caB5Q9vn988MLhqyVlHvLEWjtQFeupdiocF/tkkOGPW2ibWaBTkeZ/dvPWazXfOnnvL6jkRXpi85sFzZt+55ZptW3bl1cCCHZPD06MhySha7UFzjcjbp8fOecFCirzAG/yVjBX6OFIaadSjQq1nNhyIe8tVbaaSdHlXIWKacMeuZA1uxS95zILhyrxAdsXTL6m7kNQlx2P9uZf2qhufePFFbpI6/OU0WcP99RrCsrwseVot5mtytpf6Y0gm9sdeyKnPQ7onyK4nXlR/rg7H95M1upzu89DH6pgUcikoiihJ6NJKmRxV1x+MJiOA3YwhDRQrWU0u/0rvq0VYXnyCwsLeTJYBq3dAtJDavuzyoVpzZ99Z0+a0uoiFH/xcqgDR7rUFeOrUn6Cywb8ZeNMbhLV5ugP9l0zv9UN5b5mFkjzxUcpPJCn3V402pRxtJd2GrnLdhtVk9ZSZh9W91fCSH5B7ofxPiWL+j3D/uwhBRdyAyozeZwvQzs79soi+BKSnafLviZCcfrpBpLyimfLfTyJtbyruIQKD01tUwJyKEo/ybaxkSNFUMdMkhQoJyRBQFhnUkDQSXhTM+3NmY0EDM7ffLIjqWEGt8lCO6mLia3PukFnghosJD5p5SIho/VDkzQfLE+IrYoJXkD19pdP7OwG/voIUtagiWiZ4PAFTHHlTVhRZ7dYmPar+NJ+8JhmR6DFK5DV1foHoLNO/pHrvZfmWZ15RQlwvoVDKhCWNK3CCch9lfFBuAqUgpFSShmNaPj+i5++WZfKeViJfW5HnUakVL4UCNVkA4+ETfIqx4B5xSaP2L1yn0zn2ltPn4+OqZGmwwEVCaCSqG53ldtL1oLGAhdMLd09MpCCF6tD6ZnAZBY9hDaYsP0jzZ0j5ZjKsF4i1UmLuhbJMCnYJPt5VwFNvmZawXjEvLJqIH8STonZjq7BZ8gKgR20C9MDFqJAX1H64QW2NEup6qgzLP8cvppL/NNTOBTCJABOHeWoXzLhw4Wuy7gaBtjKr9kgKq8ZlRYBS32Lpxc8vIhpNDTfyNXWybMJbn2RyQ5EmWc2QF9wmSZ0KYCE+cPuYO6b15Uotj2Kd4MItLS7gtFbkTdrFND6pvEZqv5Yv7jXAus7Pg7avo7KDot50NX3CPkP+Kps8J9/3mGQIteY/LGPC+L7872SPR2br5fy8MtKBMHedGuM28/MZmPJMrGgi3Gb1S+Si1/L/zrZwO9XH1ce/z7ZQ1WSoY/+pMb5FT4ua0Wm+Jf/298nFmChEQ+Ti71est4mq9VYI6RsymoRJKYidElT2FGnDTZvqtfhGAFTbeqEw68GqtfmbVa/1IFO1/jdWr/8BDRRtQh9XNjubEm4aWVpVonpTGR7PVGc+KJNoBIWF7kYi4gUV3r1U6723i6TxUl3n3/tM27aZfKb7THiHW9VzFSwHJ05VfK6Ar7kaB0XgPPE0BSkSFKsBUpaLihEWoA9wBt8qirh2VSOkZwXEwyrxZ5jyt2rJmSo9gX7cg6jsEUGJU9z9xJPOEM3uQQxKgkh35DNATnVyrmJ3mbCNyIB/yox4wH1bg2DwN7q9kov4pFqny8oSm3RQbGgJ1QQTs6ZMLilOVYJ9v6Wha3HcJ9jddsXp9YhGUXLXt/qMDnvLpPNTXfNa60z5/yjXQOMq+lNmwh5egpYrdfZQZV9rI47xlRkuyTjpzsmCBSWNkAXVoK8sgYWqQJWbo1RLo6QH0YW6pxqfCnRgkd+RiFjUQUQ7poIaYoakgXxwFd9BuuI38H1xBxXSFb/pBDIKQFn7YB3dB36l7sG1FLaKiBdp1KxLvfswap/30lnVESgNnvjbUoT6w9N+Xoio0qcYOIM+heg940YimsucQVvli9NEcft2UZwGQwLuilj1fFr1i3NP94X+PE7Hpvtj6lBJfJ4R6NvWiaL6MgzWHxiN66DExa+dAdAbMYX6HVF8A+7rjEZIXAVbDe7PVI9rmN69JOLV1DOSvRPxWNPZBZf/Nf+Ny65BhYxxxV+77XJ2wfQ389/IQPgajXbwMsuAz/0IaQcXJavKbRqR2IqyZruXjVC2+hdee/5vdnYOedpmVtR3NGXldxSzDSIiBVpkGb9by89UpEPKrSLZmyFDzMab/wXl2CNe7s/qCtTvWgG5kpBmCBlSzDS/r8N4uwBwohRW63JTS1y32f0TQsPfXVGEHQrV8/NCfiOUVirYcBbIeA2+iF68rQIo3B/S628vYESr79ehzS7Q9LEL9UXmik9XVHb1yBO3Ngvt5935+k1efkV51mzzrM0LL3/20avnwMeKuWyOUZg2TasSqZ+KcZQiOn1Iu2Vh497ALUVZiCKt/gh6IvTIj1ZLRjWAkpHKOKovNwp00eqPROiAbiNEKieXwMLcXhVJ1/uzmLP4tfxaHR59cBdJVG1kTAgl9ze9QKUEQ946Hkb+okJ5JRDyf54Axur1D+WS49cLr0tTPEu7UmXrxcSr3XNvumv4yXzInXKH4F7Tc7p17Zt+t/qW2+93k063X7VW6lALxTY7i1nBXMxcxmzQbabxz+tJo+wijYaIGMNS8AoSMgAPt84DdHOoMPfjXhF+kuH1tZvuFQrRCN07xGcXRX9MYxYchDe5BcHj+Z4i+42WyPc8Xofi7bbZJN5nJLJ5qr6IqRtzqNlM17SpFsnkEyTWoABEjz4JXOQvzWYuwdnV5LNGOwTM5v9r4RpQ8ZXsYodks3o31JBlzbYtNotisnm22MxiwGFXam5oN1n0TA/hRvshvTSDwHff4nNzRo9Dum6PaJbMXzDz+x+Fkj4L4bFNBb1asqsgH7Dyh4DvbkPtf5yMDKzEwyoaESMSNS9P9gJVA3/RTlwoMwZvxECFWxIPNw9gi01nOHjP32esZTtmXHnxvZd8ZtakqQ7ekajbXetpNa6ocTVxJtY+uSe69OLz77zh5bDR3xjZMzUz6fxrz1nqrZGcHQHfPVefN+fiK86LeXj+Sc5lPKy+k/vCUI/DaLFYCWHr6nbXuILTIsb5imNKY/rCm28fSMxPhkN1XbNMNZGuqwOBhtTSxWuTk6bw0ZaG86b1hKddePOKuBvmiguYBn4T/yOqOyGRBt7bKUI1GjioBC8aUKwF7Q319UgcmtFGIzCJGBqwQij0ynDsfdFGc3TS3BlNfJ25xmzniMkpXXTPvCaD3ZaZvyzjmZdudBostmhb0ORZNN2sJBeed1HXkrUsywueQH+L0eCPxmsa5ZpgRJSDZ11yDv+jmbd86vxZfc1WcZJ3UkMq1BOOOVtvu/+pB+en186d3GTwWAw2jheaJs09/+LNfZft37DALyrNj1wABMuUKbODyTVnT/KYbJ3Tpq8IrNh92dkxOj5P/YpZx4/ycyiVcDYdn4JbEoKdQi9054iBKsygLW46FRGxAb0NPNCm8BSNCPjoKcj6EAus4SuP3rB+cV99/eTF6294dA8+TK6v74MHVpYNRt/I30e8QGTOOdfGWzzxcy+87a7bLjw37rHw1nPzp0KyyRSeZO+QQhInt3dYgvycjrPOv+T8s1rptaP84VeywdWX2T4ysr0/7TLIs6+x9zib56ye1dM9e/XsZmePY3NDs9zlnNVt4+WgHJbbz3Livg4P9WWgviOMm4kCRT6I8vw0NbUUEnFvOuFKoxQW1gTsvFirsF5pb7qTUCx4i7VmtToveaDxvK9uOaedVvPRpVOnNz0Q6bry7uiSdQ8t7Vy4JQKVS+XPplV2ts4bvCwZu+KzgITtxepaPRzWdpv74muvv6RO0SorX6cu/dqKn/XWnrtp/Zragz13DUCl5myiFW2Ycvb0PtsXnU+tx8pvLFbUspLX68mdegwmOif/NPDONajTGoUh6tU56HBJCTBASVvNUB5VIiKpc9kd7kludodSFz7xQbiOmMk5dOYk56gzL6uaf7N8a6MQOHm0ae6snZpFDfuT3/jdYzjzwkXXIVHoXNuCfQslQZqBZjTsoHMqrkE4jaYdgkGz2ATOgB3cPkSukD01DnV3ttb1wx+6arPqbkcNAHoFPzKUUQ+qL0k97pjbZv1I/egC9zTFbrrlFpNdmea+gIgfWW3wqkcis8ky5FAcRd1If5nNZrl2FFpungc8wpoCl1BpQV/ScS+zjlASyUTVv/AJ46gkJI4bHX4lTnloctxPZE1ckS3+jG2fKIjkQFyzuo8jvYQG1OrGvJPSTu/nSp9PHNTl4z5hK/8gtXVKF6gEKiglgcKiRlCESsQCV5QIlKWKpr34lt/wkSx/JCmP5/cBKQfl/5gd+rOS/+p91/+YCg5CXK2W4M9fu+/6xxX+vnelVuldIDCG0VQTpU9Dw4pRfei+6zWx0MLie0gPbyrkmRU7OwT16JGeyXLHqOLqAfVN1GPlBzWtFNzj0TRTCjogtP1NjIvu5habN5Aoa1k66wGpqriVetJgiGdwDZtKhnN0y4n9sXYnsqGmZfDSR15+5NLBlhoDaedEm7sxmpqRija6ZEEg2EAnTiAC8IrmFbGz1q08P9PSkjl/5bqzYqT9hMmptEXDgTqP3Wiye+sD4Wir4jCeoHbbp5hRfpB7BakUIppIlPCD30dR1GtslDz8OsqbXmejFC/v8wu5X2myq7SJ8Avzv9DFUJySf5uNvq4+Ti7W9D/OZrLChdwxmPNiBRqVjnpK/aGxRCDspVYKAW9AN1JANoo8wP4BJUlGqdgw6m1qPQ2QW3+OfU5/ieLS/NuKpDU3uf8bcAXyBal5jMR2NEAbPAZt0K3hvxHBEDlUxfIGcD+N2gNSNx36nfqlAYow0puatNpRz0e4W2oahKzQHsjf2c16ad/3t2KTtPobnX6D8C8pd0MDP+Kx7wnXqGGlLQcvikMErm6TmfsuxJXbSAxqNjOogJLQBLiKEHAE+JGTS3JoEhTrz8/CB+5YlupJ58aOat8Kv4JvregxwcU5Cp8GFAFm1FyOfto6GS2m1NGTS6CPNKkbsTdCBlnN9onMho55BX8IJZtEQ35lk+htwN5A0V3RCPoD/yXAcv6pAtbZczRUA64JmcUf4q7Q89ZHLeJVZ5D1Ps/t+0iCT3AHVtZC7JDCXfR7OSb/Xja5H3zQbZL1B+ULX1BMTEk3AseSpmnKEK4T9ekMIidUCRQFfcbj7z8gNLvzF7mbhQN8h6ZbRset+nQWdS/ZX3k7WpS8P9sfo0iGS64wV516pOhjI6TZ2dApgI5+LhxywYoWxKUrykKJsIoDsR4mSrCTg0egMPnLW/3Q5Nn8BZEuzqEI7HK3n0+zFmuO3TtWQ5WJoG9YqCD6Gc32SxnbnVPfsxvrFXK2dILl7bLthDp6glhcsfp4bYvbSmj/mQ94uBTw0E73x2jbNRCvC6VL6GCFDwU7eWQDcC5FY5s0slieRDwtAbRsbLXbaXAuu14e2OJw1dc6jQ3ZdY8v7rv2/BWZLqvFWVvvcmwZkK9f5jS4muO9yR5res4kfkRxhV03L1RfPOiPtYi8pd7jNEsOpyTwxpaY/yCZu/Amd5Or9uS3DYaeqVOhH7gZN/8I/wi1fEuLXvyNivibjuKvN+1Nc01HF/3h+ef/sOhox8MPd5SFucPjorQwXT+ytA8EmA5mamHNFDVhBI5pjZbQpugBNkO8MvRub8KVDKST1Wag7D3xlin1ZF7LFP/79nbvCXFOY+PUjrT7/otsPXXZ4exdPzuhZuL5LUXVAn7k7PbhG89uz3b41X01gbjP1xwlu5rrvvf9+pbs6E/Vu7Nk642/PYRaAiUBdrmO6CDTBLPQFA1ur0uXoBR1INDMkypKpoTqnSMx5GiEdTEaSHLs0Alvu/19/5QW9Rv1U1ridT22i+53pzumbs+XFFXYC++CGsTj5JUT/GCgRt3n78i2n71FHG4/u6X++9+raya7os3ZbDmgWfXun44e+u2NZKuGZ0HiF8M4TlMPR+EU6rPKRJ8wOU2RFUFLex3egEsz3YqEAq0cqhAAW19dBZIlVzR61tuIdTnpXH7l+uXrbjPUyep+8cl6aXKWhPHpDcXl9KiTWDNr4mBQc8Tq+NzK/OKSbsfl79o9G20R+brBXYvUg0rLHhtrc4TN81TTOWSZ0gL1ZVlOYH2ery/7XVUjFMbzYpg7UswcqJPQwBd0LKLabJ8IaCr2otcjSkIrGwootKECaUd4XH1+SdazRrfddkBU98t1htvWrbjqSqjaCguxrffM/5zDCpBALUycmajhd+R6ww4SWafuZ5eU+tPid4lgd3gt+b/Y9rQoZNmiXYPXyRHbRs8zX/f4WIFjWZJtUdSD55AP3xtXH+ZipC0EqdBGDA4CoYEU6gRLGPU11QhkLTBiEYPiqOeQgwTCl9aok1Qr5pFf71qEeNxjy/8F0GoqYPv75Yh9j3x4DuJ+uEzHRpAq2lMqb+qfTdiq6kGtzfOWsv0c7lSeMXDHBDe1MT+LUgx0Pg/p87u2UicdIvqQi8DkxhcUwUXCedMpb4NQjwY3npTmgsURJavLwCRyEcN2HfWsDVGfv/u9ZUWUx+PYFueUKwaNvbtu+Xps3eVWbN1GcgVrdMnWJ7WmJz9SD66EBidag0NF1Ukep0t5A7sFCWdhzvYwHv6L/BehXuHqfaBwBEU7hfVLcXvS4VQv+T/vaSIl7cbeMc7ekv9i8S3e1L5xxpvMGcu1EYPbKyCiijjGXcDKckm43PqU2qNWlXusZMiqF82cuVzolUHN9NNR0HZPxFPV9V0wLtvq+k4DqOwVWDlzuQLVdqFiP08cRX7aRlBVfR8cb55bWe5LExnlcsDp1vAP8Q9BucPMk1Ulh4GnN0SAdxcNHv3q9ohx1Ati4S/tkWjIDe3hQdkUGrGRaFBiUdiTSkI41UkMuuQHP+EaSQYlPQTFWJF03BNPpTu5KFAdkWgDukzsZKMG0Q1TAQQglScOaP/dsZ8+fP75D/9Uu5Gs3FY/2SxPld0DHOciXI9gqjcEidXjE+3BLosy0OcX3T7O5g65ROGyzQ2BZs7WbZVnO5ydLe32hMwTQ4wnnKXW6XW5LAa7oaXOIHoUl0FgLQLH2by8wSTWeAx2Y5PDazK3BqZbeJZwXGPaYhX87ZNszoDdaRxotXO1nNlpdvAPFWHDm8PqEE0sZxDEqGzxisFNnuCWetPcGrObN0p23tTZwMuRVodSV8+LTrOV3eRvzjQZiSjaLYS1WEJe0kNsJlZu9LFun7++wW4gRDRbaxw2nrOGm+xOj9cmtbp9ZqeTM1m8UXfQQCSTVSQox6pvtjot/FpHvIUjJovFEoYvHYV9C5Y/xN9OfcalvII37UEhTbTg/AQIaPb4Vz6j5u8/aViycMod/fkDcpu8QZbZoeBi/vbzP3XPsZvOubMtaPHkD9jt6+U2O7vqU/9C9SMvgrXpQNG/E0oJxun+CiElUa0IKQSUwERxOntKSV7ekcuh9VBZBBo3VUcB58ofKBHCwLyf9qFosz9Ibf8dGqwaBMjRig4SGOZ2UkWI7UiO9OfUPdxOYFApUZyfpY7mgEc5rtNGGk2H1lPhAk1Hp/VAMqQEHEUfEYkkUQq1JMdzsX7kklRrTrUi1wMcDjmu1YYfATj7Y+pGpPEBXuoQIj8rR9mgCl4C9yqmF7xnVWxGVniNqtpVmXBvQ6iwni5YQ8a1jYrXtc2J13HvgkvqWxuva1sbr+P2S5ceKGyBwDv2DbrToe1u6BkAJV7xnVLUaq0sJB8pFqcUIPi3yuwxi4JuLr+P30f3OkPQ72aO0xYo3/EsmO3QO5qEF8S0qQH0UsKXv0brnl9+8M7jF174+DsfvPOl1au/RL5/9DsbNnwHL2pHR1NTRxMZhJtHktOOxLxErPF6YlLvpC9YP73x+4ofw+3xVdrHcDE0dQQCmCRgvt9b35xINDf1CDcRSfJ+pYl+Sf8YcurfmXP5F/kj6J82jNsrkWiEuhVlgFfyNkB3S5MUzLhoNiwSCYcxQ7Ui4J0Xh7fmqRbaPa1tzujxkBRlsEHy0/OM4pYLPb7g9O6BQJN6l9zQ0OGyCaZz0vMTbHOzXfQ7a2tsterTcqxeInODoemdktw+1SbVhKwtW9ffe8VKadK0OVuC3bWzyKm5LeddsWTeorWyY9IMtUFutdu5g+Rn533qkocdvLs2HmhU75br/MmWtD8zA3OP2t1ea636jEzqYxJZGAwFiDEd61oTsrRuW3/3pYNi3bS+Rd+GjOfVpAPNd6y64Gsz1GaZleWIPoYL/v9mTeQBENVEguiF1aC4YeXxFETw6QyPfn0m9g8IrMFAvKM1EI11DARnbqibHk/Iojy5rSdgCyZi06y8sS024PeuO4MfwQ5Y9yKRZCqyYaF30vzeHlmUprR21tR0t0yz8KZY66zWuGvxVQB/36kP+K38t2Hu6NQ9SFJfw0AdpqPEK2qTMpf2VCqJwqPoJezTL824b8akoL+x03nhh+oNo5e77psxg9Q5LzebIKD+fsY34f2MtB9fk9v5b8PT6tYrgv4kRPwd0q9z3gdJSJ0653KjCYPwCaR5aUY63eW48O/kdo33yxX9wCiMv2QTrk8eGSI6Ag6moG9t2P/F7GRNlDjl0gw7pJ5aOXXqyqn8SENnXBmbSwUYLyqJjv3UmY1nKr4t80no0faXsaIEiF/BRaIBnItSce4OUif7W6Vm9T9H1X9Vj71BEm+RdmIJQST/ZfVdudUvh9S/qqNvqT98g9SQ3lHibZY0mRVHooyDN/FHmTgzjdozKw28NwQ0hwN6BCoPKaEk3YtKwNhwRLXuk076CGoZNXDQcRwZvreTZY9EZi+d0s4+ztv8iei04JQl6ZbDD2eHV7X4uHuFVfPrOmcs6m6Kr7hssr+1VZFcEZ/PdJkn1hOs8SXS/NFFgqt94PIZzZ3tdaL6Q5vo6piSzdy737pwsX1VyxUrF15iJ4uNkq+rbyg1Z+O8VsNC1UmcvORPRfxtPrfRwL2p/oA1eZp6Z/aGffoewaXcA/xBlKlQLfhQL/oPgBGP3qsA7IQS8qDVNswHKRSheDUvA3Q7MZoRcJMxlEygujn1QdyzfPfq3dEp/bXh5e5YXW2Ngfvza0ZF6UgFL/E0fTq4LBlvTE2qb/KuuzYSXVnjTfM1osvqMHVbm9950quIZlbqaL6YP7jk3kUtA0GnX2nvq53f3WoSsvEdDRnULgo2fN7lNZJgI8/VWi33c3bBZnGY05+dm+3qc7fNmj4YGKLj2nfqFP+g7jdDlxEV5XsJQZP6hYrS1l0VQr4c69Xueixp90gnZPmE5OF22j+SYEWHlZ0K/Hgsh/Ztsbh6h2DNRlvv6jJh9XaJaHCZDiUDKNTMkvb8vsqCyf3ZNdSmO0fa0Y4baJTtpbKzuVzeeSI7fCKr2Z0WypapnXJ4gnoWy3PoUIlIQ1TXdqhQJIXp9Wx5fYdpeWh2TY5D+YVyKd0jw3iumwi/BC3cEy4o83QlZnW79MrCgCjbhWXBlRZVVZZv4rIKpXC01HFlHdHLoeWVl6UVc/J5uGm6CViW5mulYMk+HqNYr0AyUPivLg2oMs2MPqtuhHyRyiwvNJej1Br+fcLyoAyu8D9B7bgmzUqfFobF5nKnK4+t8MPJkI/xHUNWk117jugWF+xazTAALQn6+UE9lhoI5ApGA/iuJOsrlNP28SVVuBVajXmircLel46w2bJS1Q0Ft0KDuikDFL/3pYrid1Q4FvofwRIo4R9h2ftSwc6jHAMqLcCql8YPHtlzGoByNXYN6v8hXnRaOhUvx0sVLCexwupGDR4NOYC7PePa5keIPACnuAdD7dEadRuTIiS6Lb7uskb381My5yjzF8lGCjBRqdwrWJCagfB3yCy7XT1i92hbcZ5Ci1FJkgYMDf6n+jspIsHFjJrTOdzSMuOa9DbDcj/nH9N9bIoGVgzHPWIQuFuYtaMRaq8eCKI0gEF6lPOZjBz3EEvaaxwSUT9U/8JbJZPJJLBLolH1La/RbF9AbC8JJjv/mMnssKjLRBJyqj9QXxNko0Ux/X79epfiXkm6fmKwF/en1HLc6LxloXWKvGa5rVCVL83VuiPcDEX/K5pTXOxHfx6HHB0t2FI0qI2rCZFTrvPWU67zVuS/kTsLnc7IKhFg30e4FOkqNSfH5PtkmUy6Cpiv/36k2sbqCeCFNa+URpoY0sZoYmCgCr3qgZz6s8I0gP1bYiR+D79H56NOz0EVWCTy2/fffvSCCx59W7uRV9995eqrX8GLesOXNm360iZ+T/El3uZqL+FyzSZ8XxpTiI/G0nkT4zznFZ0t4ipMz5v4q9ssqbdKUZt6u82knPCrt6PZwsnn0XySVnyPR1ZXAn72yx48bWJsu7apnI3Hy8bygUK5Js32qcytapqgmn95uexccj205vGgJ+euOeG2SORmKZr/qKzcx9SFctMJdwMUFZDJITs7dnOp1EKZCxg304Cevyfya+vlKqv6aXK1qIj3imL+L6hL+yvUlFfE0VKZ7E8gBY3M/8VoJCFgizH1W6VyC76nH6b7jiibYVxUmVIEspry/LgZIlCeP11Z4zs/AwvVwtGFEut5S1JY4lfyT0N/evOLo+rUEgjcqc9IkGpQbv3iW7Co5b+KgjvpzYdH85PLcc4X21ouwEGl/S4qnUAvoSlXUUhR1eKr2VWFTB+GMl6FsiQsVD1R3urlAAIoSn7JQkmiVVCHSpCwDH/qPepXQ0Db77CJOAImohB+RPWr31ev5g/kE+zTa4lbvZo8xdWPffQu9yJTPCNB66s+zXoJt/0L6hSoCuBIoK8fnBGG87OoRckJpLqyWe4YbpGi50g0+3I3UD85Oa0fzubfoXxPLbW3FDWzigmyJeM0tQkax7PqTy80+UxfUHPlBZIRVNQ+v0xRm8REKPoLmNr0+Uo48v9GFbXPKylqQ2IKm00QddgyWGMROCTxdLB9nCY8P7j2DjlsV/+mfr0C0r/NkeXbbpPlOTBBwT0mVz1zx9S/wJecBF9Wgv3p032iP2v4VSgfgW2G+HUEdEXU6iq4CtpLJfIN9XQG8dwa1VoO8XC2SrPDDyCOQptXgbcPvlAgBfxBoGwftQKeKFrNTASPt3pGGqDt/QRasn2kri+H6L80MJRsmVYJrAKyDItpJUy3/15WYIJqcJ9Q5N/LFJ4c3dc1URpWl9hW6mu50MUIelg4ucTPf15zs5DFo1c0VSp1tKB9jkwIyuM45kb+IP8gHed+6jO3v0KbIknzLy636E8KPTdCuUpB0wLo9JKnAO6pv0vS31EtBha/fJemkgLVVnd8KCk4qBTpQ5m7FbifBKrPJcq0pZAFVG/XbOFz+Tcq2MLrcmV28Nmi/OHskh82bau0k8eWCaPijQPWQ5lUvslwVCfHkXBMIehqUgtDNLeauH1huvZTbYmw+luPjyWoNGEuxRLR7LK5fSyXFUyK7PURQv2v8D3XOt2NJ6liBbmPGOsakw1kbeOs+31Wm5qpH+iJWSzqdPr2O7zc2TmtnrzCig6bBd/vgQmzOlz0STWIlmZEQfupogOZFHUZ7EkUnMn0RrpIMqAgHRJAOjIJ3yGw1I/MAp9q9S3Q/clADNm1wEeO+xbwg5OIYHZLY3ehG5lJk2xhco+6JWybpEVz2wrR6hZyD0QXZbeDVB+onmlimpkWprdAs4WEZDSQppsDlcdCBJJESIYFuAtUnC4GIF2C3Uu2Kv7L1bdz6FxtqxpG4TqQOqOUNAJ2HLvPWA2GgDy4O4vaDrtyl6P+1fAll+SyFcQ28GHqh7fvvf37udylf0fNwhzgz87Y+cf5x9GnF6ygHu18sAbipWeF0YPBgp2GaKeQduxxdEr3SgbH1kvH7tvqSLhedomOvZyts2dw8acu3dY/f+ucuMtCuP/e4zC4XnH3OLZ8ZuxTWxy8dJfU5dhDeKPSlJy5pn/+7u3XrJhmr9C5CuleGflGQocKnlAUaRKp0BAHV0ZwUt9VCqk6zYOgRIuMfePJzdmBdpPJ7/6B23+f+sp9NMDZevovvfYHG5dGPISQq1DojqNckchVrCcCYz/Q0hI0m3NKDRfkgsrnamo+p0CAq1FyvC3a3Nak/s5VX282x9Ufy3E39VAx6o7LpCvO2wK+ch9jNqpJCutcIOooKnYWtDK8gTRVYygRQfwgzKM5+jP2jOZdx3r32Py7rQUPOzAnoRs95NvRAR0qLGU11Taqu1bUYSzMcWjMEir067JQQHfIrLBHsrgv00/Wavd8HRLMEEYFSW3HCSNQehnrHztKqHcDyo4VfZ6gPKCR+gufwA8GegxUEo4A+gd0BASHiH6jYMLIsUdQJTs/C641KN4oCHWolCMLlMfIdtWKScjx7SM5LD9HnfmhrGI0S139UWfUnxgOXdJFW+AMcGjKr6eHAttHF5sUoeArYKDcxMSYcKA/xUDhPiEOEAPafSIUFArN0r24ynI91EPARDXvIDYyvqZaWeroBOUABQA/E+DXC7PWafDLQY2oiwpUEyj4RQtVlUp1GrM7In2p2A7VuiOW6otMiGOo5Mrp05ejVuTy6dNX/k/7mybZQ0nUmfrbx3U4KueDnlHm5wdh8FFeKnoaKKh/TK18StOPhwG9Xo5mqXAxvw/79YQwwDR+nAKQQ4izVXioB84qcppWB7IqjU45z4CE17OvF1Dw+oTFqxtz8dxwtogBnF9MjIl/in+K8s3hM9laIn0TiCbTAXL0T798bPXqx36p3chrv0O+GC9Xaj48Ecv8U8UEeBvUEsDlTepiU5OvlpeNGvpnKF0RvUooWhIjnx6GeBapXCQYTw9DNg6/OC3gZjp76oNTj9Kz6Jqobxb9NDqc08vcKReOpcsQV2K8InXFaXW3aI6Ofr1k48rp7CX7rx+v1UKPsfvzQU0Kc83i2VdILmd2/yX55zT9luN2+Cu4nKfwPcK/CvDVU+pHh8+LaldIf1fA5h3ndT6Fln9/W/9Ce1vndfvJtnPVO2xhm3qbafHVCN1X363UXHq9xuVD8OSD29Z8pZ5cZrern9cAdGW/uib/ud+VK0L9a42r6C90kL8KzxwLQw9NkIQJL0ASU8M+VG0KsUdgdvpgP/6NqqP0/gHZFUfGEijZLHpiIgvV5/Bltrj8Qd7XQd5p4P+7tJo30NMO6VGBwahSPMYiaaBYoLY6uEnciyhhh1Z/vvacG/rjpsvnpzs0B1Id6fmX8119l88XnOxe/uGrzzHcdu7UtY3+2vmXN5zUyj3ZcPl8p1sZSs6/nGXtwrV7Ka0XZdz83fwjjINpZWYw85lL8BRK4nGyIir2RiOsEyipuEcIakpGjWgBjLiHWOgj0Yi34gW1kKPxHt2Na5q+lwg1RdRSpFDNzosb44YJXnAfoEOpZW//6u1lhYA6leevezbI26zNHO811M2dc5HFxpk4i1jPC0s21/BWW5DnPQbn2X1WK43/aM2n18DfSoybbNHijFpamzXI31eRibGUOxSu/lT96YZlq1Yt20DaSBuG6knw2eusHs5EPBfNmVvHKdaQzcDfz9ZsXmLDWGXy2U5OsYSsIn8CS12jQIyD12KKqZrLPy7mSPdICmd6WGHG8NDZkkHuE4h9TU8FpmUO/VjC/EinToFyoNDz2p9XD6g78WgQdPG7Z3R0T/Z5dTM9lsL8Ktek7szl2L+gQwGgwkZHc2g5Su7NvVqwGy2Ua4KSXUwt1X4PaM5paaEu6jQ5zVFyNabxvUksVt2T/4VeamYPlLtffdQsk+2sUTY/zDXl/05W53/Bz9UK3p7LjapZ2ZxOm+UlZXrL3HHGqO8+wVroDaCTTnTxitMxmiAAYQzVJQH+nj3oIHnPaN6Zq6sNSLjBl8tKgVr2mj/9CWi9dnKca8rBQBsd5R1tzVlgrl5pbnPw6kZclCr2CHxMnHohLz+3KRQokzALyeIKFU1TNCiayJdoHvDYe7K6mZLm8S3uJ9dojuaJ62/qN/tjQxnSnhnKPw+LNrLi8ZKyJ3x1YhiI1aNAtP6NzCGzYv3DmaGh/LvQZnt0evgIhTFV0kE/PYxAnOHhCQUZdCWY5JWJwMzlAGl1mpNbDU7yyGnhRMILsYhH3VRAijrPcBU8/Cj1Y9NY6cnGVW0CjTLaz7E3epvaT/LtTV72Rs+0WVVmd0dz/MGTI5F0OsIviaqDlbbO5X6xT3PeXbXHRtf/z+fdka+eKPr8KF7IF4vBsT9MFPuPJMBTBMq9hQxXelQ+bewnf18ap4Ib+mSMrtDU5zqlD8QANa5MBGh/OwOvSDfcV2d66mfEWsbGWmIz6nsyZDWQSmqmxDneYyvjHPmRXHZxeueyRGLZzvRioKnGto9nIPkibAJA16adcOZRQr1iAP3bUyBR7T4RgAWTKxhkCYFwshq+7iV9r0whk50cmRcTg4fy5x4OmmNkHndIA2+YuMbmE9dwGYB4KFTsvnDE6Ah47r/fE3AYI+oXADpkdlENcZ8OZEEf8FFGZNxMs6ZLpG3SUFLL7Q2kcFU/A/Jsw+vWDa/7emewLaoeibaF1B9qUNnuqWK3+UfXYVL1v/omD15xxeDkPnXTOKSVcCbDGtOu0YQNpGAP7U1HU58UrqGu8xIbHtkQ3LVhb7Dx46ET3Ffcm1q0YcOizNmf3bC3VjWfAcpSv3MyTlgJ23FHQgmgvk+gk8pL0mcCDOn08MDAQlf+/SlTZ1z12fnqntOhbOTL9/ZdevbAPN+yby1f/uUtC/ixm8ZBo59LTXEW060hGrTDplNprWd58fwB/b/E27BdS/s7U+rGVCeQ46nzaw9QccnmZerGZZs3Yw9aVHt+Kh6HN4ti6lxIhT/wahnZtWwzlY9QHQ2c79C+dxzvVDKy8GqKWQERO9YAKbpsDUTLdWV5dE8PVPjvj9pqw7ah/PFVtkit7aj6G5xY9mfJrCz1j1e0BcnPol4UjtrCdbahIVtd2HaURujnFJR8CuOuUUfhrGhgKKgjCYNSvCc1WKlEp8wHUaAYynFNyzZn+2MnYv36dbMDBTonl/T/ma5IKAyEGz+4eRnVtaX6tss2o34u8mWorFtuFgm4A6qK/yp/gLEBVat5WnPDdKA574ubuFJ/IUfZ/Y2Nt6mN+ZNNTSTaeI56gKwkXerTe9DDHUw8/H35FY3nNN7GGuBKWhrV9ep+0k1WjNWVaHkW1yA+QHWNu8rtBw2a5YXuE40rs7/GA+j09V3hA98yRnFPOGr8ltGlsFdD/7tRce3LH6Trcneuiy7K7J3khKu+3qUaXPWaX7T6/Kfj9BX2eZq2XAcZT79u1ClJzUtHUqfqSMWBcZS43Ena0cUGLgpkKxB1QM+0Fxz10wgg6r5rltnFpH05pepUq3Y2HfYqeKRntmUFNz+XmcOs1H31U6cC6RTVLfCg7RNBF1UF2/wBgu0fFQtPEU1sSg3VcNsR7dWq3af87tUFn1l3ltXpaJxpNvtcZkH2WmMst3JqRpxUH+WC0E1qOGtP66s1MYv+VLu8/XFXvV/ZbunYYBeVN64ls0ur6NzpV9xzlmQwB5qC4Tq70WC0tk8dWJXeHvkD0h9zJOM0vD86/1NJMaIAolctvlByferCsqOKDKceOfUu1PsmoFCamV5mCrMUOCi6V6FJosMF22AcrKJgQDVhfYh6tepp/lYgvnCEAbJQ1L0rOpajEmRcasMiPfxhgGoVo4rwreQpV6fUJHH2e8fa1s2c13Apl1b89a58ozdoap2sjgLN9uISl7P1DrulyeIkt0zr6JjWocoPOZsaXPb6jtqBblsgsaRre2xHi4nELm0MhG1+x1SXwLpFi53b+aHRYo/IrbZtuWAKu5cSEXfybnnmUCaXGTpQr0xK2O2WWY76f+nAjNVf7nCZHU5XqIkTnpt6VtvsFlPXg1031g/VRdpkkyVpD7jnmax88QwDvg/66NnMRdRXTcGTmQc3cuINwN5IQqi0yzb+YFVHuVqI5s4ADfg5oE4ybDLd28mFSFmYvRoomsWXEdLU2Wl3GJy93ZNb/d5gqmNaqJZSO1l6PVRy0nZIj/45EetjLguh1rLqR+SK0hO6NrsqcNX8zoUdjQYDJ7tb4os6+i+Y0qpY2AWlnLRDWdGFTfGY1gV0zNAtJ7pdo24se0D88AwLY/gZmE9iuP4V5v7CSR/RThaHLh+UeBkXwU6BC7lGOevK65udTv+tS/PfW7qj3ljTcj3b9OkbV85t8xsMj7Ddj7DGpthZKwKPvso/c/1K9aLE12fMWLV1y1D9ua8lyJdWXr/bG+noCFutf/mLILe39ITUV4igr3876fpX5g2zeB52sWnIL4fXHlgeUzOx5QfIvJQyrKQE9wHUqVq+PEaOrz0wVvNbJZVSfsuMzxN4l9PkedFzw9V5Dj+nzpgoT4ZxCxJfC5RWLc74YVHxKlExCYt0JAOMatREhHBSCAtSfod6x6Ls8HCWECLwXZ9nd5Dz1T24JUdWs6fU3++fcnT49Qe+kBs+wdsMZgPXMp3U5S958snPP/EE7bvkOPCuTUDTUQ/UzirLhML9yPahoe1D5Fj5jWsaoveyP00PehdUAHk/seDVWsvDWXXXsyn/4wfpXc2V3/Qxli3jl/5hj/83avSCfpTNxOEKLmTjxOEKuxgNlsQn0xgct724mhynupNW1Ph6o3RYS3/+2TJrzLlkFz+ip3qCHKf6eqW02QJLjBYuuj4sobhCWqa/YHGEHpcnumuWSOhxeaL7sOakNR6vvmo+YcfFA8UFXEPZf9UjyudIOyNwx/i90DdsujS/FX2UAwvWSVK4NxaMhAGw3oowp/uc8CTi7D2rBgZWwb/60faR7SPsEbjkXy4G0XaqhXPwe2cePjxjxuHD6ssQuR1fq6PF0E+o2t1nePTn8TUmxz/A3crMoCc7egESuoTHYc7mYdg6etORoOhR7BBGD+qJopELrl4S6cJNRtEAsLP/OdvnJq0Wo0GolY2Et9VFB2Kf+4bZvVyxfOMz3WdFfSIryj6DwWghre7aQbdiDrkTL3A3vNDuDpk93HqXwam+bWmUJZfNn5ozKV5Pmmq8PF/jVY+2Tlk2M2RzSXKjmbQ4RZcQavEYrN/9rlXwtIQqzxQNMzPPfHYLvuPoO9TbT8bpGw5CQPGd+SyX/Cyf0Vxjd2R9NmsunnXYa8xGHzn+sSfM5J0y0DZEXWWxkXjcR75KBLNLHi7XvX2G8VOrf4Ykg0AMdBESIpo7MgAfyakA6rkqpI6UjNs0px7cMV+D5BF49Tez1VGnYmq0WIijp985m4Sn2gJR9b07riPPFo97OYbUZbxJCpot7H/lpZBicglCPN7WOfJkcHqc3ElWqvvz/1E6bIQrG+tz6WkM1SM9FBTR7FSs8KyBBytSmNEoquJNFN5EQyTiCrnKDx1h58yxCepPHU5nxGoxEQeeOZi2m80DxNxncVhr6BmEfUarxejw+WSiHhWk19bSY7aKR5MsteblJpfTLtjimBouXsm3d3djjYM+wEW0El9dM/ueVRWIsXwe43R7SgbVZqrnqoJ1X/kuF7pcgf8duv4q6vayV5U9zMV91GxO59UUjW8rHV6u799WzKMT7umRCXbYUKM+foaCcwgaoqZUtmodV3p+X7akb4dnU9B9La38RPFUG2SCC90tVA4XwEFhyOpZZrUCsgWYHsczLFBBVGNtstoN1bw0Z+O4fYIbvZVt4EUcJEKOhHeincWqONw+q6w5Go+WGOSR7LhKV+KBqbBPpfUvOf9QqkpDyVhBeyyZQGMsdA5FBUqvFMtUyGq9vjnsAJU4UcrxldP1CCaofyDkSAifoP5QwWx+SyUGxp75BzGAvtG7uQ38LehlyEQMeh0TeE6Bm7tYdXqdkt0uOb3kfYlNwmOdDyacOq/qlFo1v+PTmTi3E/glC9W11b34A22zmLzvb231Q0L2Bgg60OTW4YdstO+YOJnO38TtpH7zy9ymokWyA79qlVSn38HtpFlImFnhu3b4boNWXklOXV0Iwo7lQ1hrZyPFcwtjwFP7iEKSHSSJw509kh8kj6pr+H1jR7km9vcvqN9657vffefkv+fKxge1X+7RdjYUPIESN7gTvRkB/RMYtEkaVkdHApmdBPpnKmz0n1xSWFOyVIuLrinZwpoCRe6kyiVZoHX088F+UX4+WKS4iBTP0IWxGtZgOdMaV4KTayqHQF/VihBwTbgDXTCmKoOBJeNhwJMzEVjtjIFLuU38fPR7hqNG1JS7g/qRCuy3vmQ3W9Vu8qbVbP+SzazGRJH83MzP90Ck2m31mMjP8TiLn5uwD2Ugr2PFvPQjB5BnSJvQxGQZZEB+LopqzGzDbMmbkAPkZVJjeO5FzOSBKCgJze2ZS4Gemc9twrwY6u9H61iUQTcRvtdT9RW3tRxAWwFs2tcuJRnI6xjmBdWjbgFNRHMHiF1uHYBfUR/ut5Ug2jXAaT96+9RH/FToRwIzGbKmVJ1AZQnoabSB1yyIg7ByAridHApPMjyw0OiV6RjSbCuzwLAvFizBliWJua1tsuAgvNPbmljYbpt8lkWam7b3XZiOiKJskMOtmfScnsbPW208knwjuXrXK4Q1iKIgNyYXXDVT9C2Ye/78GQ5BEEXfFdde2RwauOysdJNL5AzCy84ard/nGAVN8alecnFdgu5Gbd5DJTL+hHZK0vApVy3OfU8XTSJg1TlssivsPYUlIqvn66PzrVTymCc4wgF6SDNR0pDf+9Gp+VnsUH5WtpHYsuhOaey8zdwLN47V8MTbm78g687+P3cx6tcAeNpjYGRgYGBk8s0/zBIfz2/zlUGeZQNQhOFCWfF0GP0/8P8c1jusIkAuBwMTSBQAYwQM6HjaY2BkYGAV+d8KJgP/XWG9wwAUQQGLAYqPBl942n1TvUoDQRCe1VM8kWARjNrZGIurBAsRBIuA2vkAFsJiKTYW4guIjT5ARMgTxCLoA1hcb5OgDyGHrY7f7M65e8fpLF++2W/nZ2eTmGfaIJi5I0qGDlZZcD51QzTTJirZPAI9JIwVA+wT8L5nOdMaV0AuMJ+icRHq8of6LSD18fzq8ds7xjpwBnQiSI9V5QVl6NwPvgM15NXn/AtWZyj3W0HjEXitOc/dIdbetPdFTZ+P6t+X7xU0/k6GJtOe1/B3arN0/pmz1J4UZc+D6ExwjD7vioeGd5HvhvU+R+DZcGZ6YBPNfAi0G97iBPwFXqph2cW8+D7kjMfwtinHb6kLb6Wygk3cZytSEoptGrlScdHtLPeri1JKueACMZfU1ViJG1Sq5E43dIt7SZZFl1zuRhb/GOs44xFVDbrJzB5tYs35OmaXTrEmkv0DajnMWQB42mNgYNCCwk0MLxheMPrhgUuY2JiUmOqY2pjWMD1hdmPOY+5hPsLCwWLEksSyiOUOawzrLrYiti/sCuxJ7Kc45DiSOPZxmnG2cG7jvMelweXDNYXrEbcBdxf3KR4OngheLd443g18fHwZfFv4NfiX8T8TEBIIEZggsEpQS7BMcJsQl5CFUI3QAWEp4RLhCyJaIldEbURXiJ4RYxEzE0sQ2yD2TzxIfJkEk4SeRJbENIkNEg8k/klqSGZITpE8InlL8p2UmVSG1A6pb9Jx0ltkjGSmyDySlZF1kc2RnSK7R/aZnJ5cmdwB+ST5SwpuCvsUjRTLFHcoOShNU9qhzKespGyhXKV8SPmBCpOKgUqcyjSVR6omqgmqe9RE1OrUnqkHqO9R/6FholGgsUZzgeYZLTUtL60WbS7tKh0OnQydXTpvdGV0O3S/6Gnopekt0ruhz6fvpl+nv0n/h4GdQYvBJUMhwwTDdYYvjFSM4oxmGd0zVjK2M84w3mYiYZJgssLkkqmO6TzTF2Z2ZjVmd8ylzP3MJ5lfsRCwcLJoszhhyWXpZdlhecZKxirHapbVPesF1ndsJGwCbBbZ/LA1sn1jZ2XXY3fFXsM+z36V/S8HD4cGh2OOTI51ThJOK5zeOUs4OzmXOS9wPuUi4JLgss7lm2uU6zY3NrcSty1u39zN3Mvct7l/8xDzMPLw88jyaPM44ynkaeEZ59niucqLyUvPKwgAn3OqOQAAAQAAARcApwARAAAAAAACAAAAAQABAAAAQAAuAAAAAHjarZK9TgJBEMf/d6CRaAyRhMLqCgsbL4ciglTGRPEjSiSKlnLycXJ86CEniU/hM9jYWPgIFkYfwd6nsDD+d1mBIIUx3mZnfzs3MzszuwDCeIYG8UUwQxmAFgxxPeeuyxrmcaNYxzTuFAewi0fFQSTxqXgM11pC8TgS2oPiCUS1d8Uh8ofiSczpYcVT5LjiCPlY8Qui+ncOr7D02y6/BTCrP/m+b5bdTrPi2I26Z9qNGtbRQBMdXMJBGRW0YOCecxEWYoiTCvxrYBunqHPdoX2bLOyrMKlZg8thDETw5K7Itci1TXlGy0124QRZZLDFU/exhxztMozlosTpMH6ZPge0L+OKGnFKjJ4WRwppHPL0PP3SI2P9jLQwFOu3GRhDfkeyDo//G7IHgzllZQxLdquvrdCyBVvat3seJlYo06gxapUxhU2JWnFygR03sSxnEkvcpf5Y5eibGq315TDp7fKWm8zbUVl71Aqq/ZtNnlkWmLnQtno9ycvXYbA6W2pF3aKfCayyC0Ja7Fr/PW70/HO4YM0OKxFvzf0C1MyPjwAAeNpt1VWUU2cYRuHsgxenQt1d8/3JOUnqAyR1d/cCLQVKO22pu7tQd3d3d3d3d3cXmGzumrWy3pWLs/NdPDMpZaWu1783l1Lpf14MnfzO6FbqVupfGkD30iR60JNe9KYP09CXfvRnAAMZxGCGMG3pW6ZjemZgKDMyEzMzC7MyG7MzB3MyF3MzD/MyH/OzAAuyEAuzCIuyGIuzBGWCRIUqOQU16jRYkqVYmmVYluVYng6GMZwRNGmxAiuyEiuzCquyGquzBmuyFmuzDuuyHuuzARuyERuzCZuyGZuzBVuyFVuzDduyHdszklGMZgd2ZAw7MZZxjGdnJrALu9LJbuzOHkxkT/Zib/ZhX/Zjfw7gQA7iYA7hUA7jcI7gSI7iaI7hWI7jeE7gRE7iZE5hEqdyGqdzBmdyFmdzDudyHudzARdyERdzCZdyGZdzBVdyFVdzDddyHddzAzdyEzdzC7dyG7dzB3dyF3dzD/dyH/fzAA/yEA/zCI/yGI/zBE/yFE/zDM/yHM/zAi/yEi/zCq/yGq/zBm/yFm/zDu/yHu/zAR/yER/zCZ/yGZ/zBV/yFV/zDd/yHd/zAz/yEz/zC7/yG7/zB3/yF3/zD/9mpYwsy7pl3bMeWc+sV9Y765NNk/XN+mX9swHZwGxQNjgb0nPkmInjR0V7Uq/OsaPL5Y7ylE3l8tQNN7kVt+rmbuHW3LrbcDvam1rtzVvdm50TxrU/DBvRtZUY1rV5a3jXFn550Wo/XDNWK3dFmh7X9LimxzU9qulRTY9qelTTo5rlKLt2wk7YiaprL+yFvbAX9pK9ZC/ZS/aSvWQv2Uv2kr1kr2KvYq9ir2KvYq9ir2KvYq9ir2Kvaq9qr2qvaq9qr2qvaq9qr2qvai+3l9vL7eX2cnu5vdxebi+3l9sr7BV2CjuFncJOYaewU9gp7NTs1LyrZq9mr2avZq9mr2avZq9mr26vbq9ur26vbq9ur26vbq9ur26vYa9hr2GvYa9hr2GvYa/R7oXuQ/eh+2j/UU7e3C3cqc/V3fYdof/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6D92H7kP3ofvQfeg+dB+6D92H7kP3ofvQfRT29B/6D/2H/kP/of/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6D/2H/kP/of/Qf+g/9B/6j6nuG3Ya7U5q/0hN3nCTW3Grbu4Wrs/rP+k/6T/pP+k/6T/pP+k+6T7pPek86TzpPOk86TzpOuk66TrpOuk66TrpOlWmPu/36zrpOuk66TrpOuk66TrpOvl/Pek76TvpO+k76TvpO+k76TvpO+k76TvpO7V9t+qtVs/OaOURU6bo6PgPt6rZbwAAAAABVFDDFwAA) format('woff'),url(data:application/x-font-truetype;base64,AAEAAAAPAIAAAwBwRkZUTW0ql9wAAAD8AAAAHEdERUYBRAAEAAABGAAAACBPUy8yZ7lriQAAATgAAABgY21hcNqt44EAAAGYAAAGcmN2dCAAKAL4AAAIDAAAAARnYXNw//8AAwAACBAAAAAIZ2x5Zn1dwm8AAAgYAACUpGhlYWQFTS/YAACcvAAAADZoaGVhCkQEEQAAnPQAAAAkaG10eNLHIGAAAJ0YAAADdGxvY2Fv+5XOAACgjAAAAjBtYXhwAWoA2AAAorwAAAAgbmFtZbMsoJsAAKLcAAADonBvc3S6o+U1AACmgAAACtF3ZWJmwxhUUAAAsVQAAAAGAAAAAQAAAADMPaLPAAAAANB2gXUAAAAA0HZzlwABAAAADgAAABgAAAAAAAIAAQABARYAAQAEAAAAAgAAAAMEiwGQAAUABAMMAtAAAABaAwwC0AAAAaQAMgK4AAAAAAUAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAFVLV04AQAAg//8DwP8QAAAFFAB7AAAAAQAAAAAAAAAAAAAAIAABAAAABQAAAAMAAAAsAAAACgAAAdwAAQAAAAAEaAADAAEAAAAsAAMACgAAAdwABAGwAAAAaABAAAUAKAAgACsAoAClIAogLyBfIKwgvSISIxsl/CYBJvonCScP4APgCeAZ4CngOeBJ4FngYOBp4HngieCX4QnhGeEp4TnhRuFJ4VnhaeF54YnhleGZ4gbiCeIW4hniIeIn4jniSeJZ4mD4////AAAAIAAqAKAApSAAIC8gXyCsIL0iEiMbJfwmASb6JwknD+AB4AXgEOAg4DDgQOBQ4GDgYuBw4IDgkOEB4RDhIOEw4UDhSOFQ4WDhcOGA4ZDhl+IA4gniEOIY4iHiI+Iw4kDiUOJg+P/////j/9r/Zv9i4Ajf5N+132nfWd4F3P3aHdoZ2SHZE9kOIB0gHCAWIBAgCiAEH/4f+B/3H/Ef6x/lH3wfdh9wH2ofZB9jH10fVx9RH0sfRR9EHt4e3B7WHtUezh7NHsUevx65HrMIFQABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAAAAAACjAAAAAAAAAA1AAAAIAAAACAAAAADAAAAKgAAACsAAAAEAAAAoAAAAKAAAAAGAAAApQAAAKUAAAAHAAAgAAAAIAoAAAAIAAAgLwAAIC8AAAATAAAgXwAAIF8AAAAUAAAgrAAAIKwAAAAVAAAgvQAAIL0AAAAWAAAiEgAAIhIAAAAXAAAjGwAAIxsAAAAYAAAl/AAAJfwAAAAZAAAmAQAAJgEAAAAaAAAm+gAAJvoAAAAbAAAnCQAAJwkAAAAcAAAnDwAAJw8AAAAdAADgAQAA4AMAAAAeAADgBQAA4AkAAAAhAADgEAAA4BkAAAAmAADgIAAA4CkAAAAwAADgMAAA4DkAAAA6AADgQAAA4EkAAABEAADgUAAA4FkAAABOAADgYAAA4GAAAABYAADgYgAA4GkAAABZAADgcAAA4HkAAABhAADggAAA4IkAAABrAADgkAAA4JcAAAB1AADhAQAA4QkAAAB9AADhEAAA4RkAAACGAADhIAAA4SkAAACQAADhMAAA4TkAAACaAADhQAAA4UYAAACkAADhSAAA4UkAAACrAADhUAAA4VkAAACtAADhYAAA4WkAAAC3AADhcAAA4XkAAADBAADhgAAA4YkAAADLAADhkAAA4ZUAAADVAADhlwAA4ZkAAADbAADiAAAA4gYAAADeAADiCQAA4gkAAADlAADiEAAA4hYAAADmAADiGAAA4hkAAADtAADiIQAA4iEAAADvAADiIwAA4icAAADwAADiMAAA4jkAAAD1AADiQAAA4kkAAAD/AADiUAAA4lkAAAEJAADiYAAA4mAAAAETAAD4/wAA+P8AAAEUAAH1EQAB9REAAAEVAAH2qgAB9qoAAAEWAAYCCgAAAAABAAABAAAAAAAAAAAAAAAAAAAAAQACAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAEAAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKAL4AAAAAf//AAIAAgAoAAABaAMgAAMABwAusQEALzyyBwQA7TKxBgXcPLIDAgDtMgCxAwAvPLIFBADtMrIHBgH8PLIBAgDtMjMRIRElMxEjKAFA/ujw8AMg/OAoAtAAAQBkAGQETARMAFsAAAEyFh8BHgEdATc+AR8BFgYPATMyFhcWFRQGDwEOASsBFx4BDwEGJi8BFRQGBwYjIiYvAS4BPQEHDgEvASY2PwEjIiYnJjU0Nj8BPgE7AScuAT8BNhYfATU0Njc2AlgPJgsLCg+eBxYIagcCB57gChECBgMCAQIRCuCeBwIHaggWB54PCikiDyYLCwoPngcWCGoHAgee4AoRAgYDAgECEQrgngcCB2oIFgeeDwopBEwDAgECEQrgngcCB2oIFgeeDwopIg8mCwsKD54HFghqBwIHnuAKEQIGAwIBAhEK4J4HAgdqCBYHng8KKSIPJgsLCg+eBxYIagcCB57gChECBgAAAAABAAAAAARMBEwAIwAAATMyFhURITIWHQEUBiMhERQGKwEiJjURISImPQE0NjMhETQ2AcLIFR0BXhUdHRX+oh0VyBUd/qIVHR0VAV4dBEwdFf6iHRXIFR3+ohUdHRUBXh0VyBUdAV4VHQAAAAABAHAAAARABEwARQAAATMyFgcBBgchMhYPAQ4BKwEVITIWDwEOASsBFRQGKwEiJj0BISImPwE+ATsBNSEiJj8BPgE7ASYnASY2OwEyHwEWMj8BNgM5+goFCP6UBgUBDAoGBngGGAp9ARMKBgZ4BhgKfQ8LlAsP/u0KBgZ4BhgKff7tCgYGeAYYCnYFBv6UCAUK+hkSpAgUCKQSBEwKCP6UBgwMCKAIDGQMCKAIDK4LDw8LrgwIoAgMZAwIoAgMDAYBbAgKEqQICKQSAAABAGQABQSMBK4AOwAAATIXFhcjNC4DIyIOAwchByEGFSEHIR4EMzI+AzUzBgcGIyInLgEnIzczNjcjNzM+ATc2AujycDwGtSM0QDkXEys4MjAPAXtk/tQGAZZk/tQJMDlCNBUWOUA0I64eYmunznYkQgzZZHABBdpkhhQ+H3UErr1oaS1LMCEPCx4uTzJkMjJkSnRCKw8PIjBKK6trdZ4wqndkLzVkV4UljQAAAgB7AAAETASwAD4ARwAAASEyHgUVHAEVFA4FKwEHITIWDwEOASsBFRQGKwEiJj0BISImPwE+ATsBNSEiJj8BPgE7ARE0NhcRMzI2NTQmIwGsAV5DakIwFgwBAQwWMEJqQ7ICASAKBgZ4BhgKigsKlQoP/vUKBgZ4BhgKdf71CgYGeAYYCnUPtstALS1ABLAaJD8yTyokCwsLJCpQMkAlGmQMCKAIDK8LDg8KrwwIoAgMZAwIoAgMAdsKD8j+1EJWVEAAAAEAyAGQBEwCvAAPAAATITIWHQEUBiMhIiY9ATQ2+gMgFR0dFfzgFR0dArwdFcgVHR0VyBUdAAAAAgDIAAAD6ASwACUAQQAAARUUBisBFRQGBx4BHQEzMhYdASE1NDY7ATU0NjcuAT0BIyImPQEXFRQWFx4BFAYHDgEdASE1NCYnLgE0Njc+AT0BA+gdFTJjUVFjMhUd/OAdFTJjUVFjMhUdyEE3HCAgHDdBAZBBNxwgIBw3QQSwlhUdZFuVIyOVW5YdFZaWFR2WW5UjI5VbZB0VlshkPGMYDDI8MgwYYzyWljxjGAwyPDIMGGM8ZAAAAAEAAAAAAAAAAAAAAAAxAAAB//IBLATCBEEAFgAAATIWFzYzMhYVFAYjISImNTQ2NyY1NDYB9261LCwueKqqeP0ST3FVQgLYBEF3YQ6teHmtclBFaw4MGZnXAAAAAgAAAGQEsASvABoAHgAAAB4BDwEBMzIWHQEhNTQ2OwEBJyY+ARYfATc2AyEnAwL2IAkKiAHTHhQe+1AeFB4B1IcKCSAkCm9wCXoBebbDBLMTIxC7/RYlFSoqFSUC6rcQJBQJEJSWEPwecAIWAAAAAAQAAABkBLAETAALABcAIwA3AAATITIWBwEGIicBJjYXARYUBwEGJjURNDYJATYWFREUBicBJjQHARYGIyEiJjcBNjIfARYyPwE2MhkEfgoFCP3MCBQI/cwIBQMBCAgI/vgICgoDjAEICAoKCP74CFwBbAgFCvuCCgUIAWwIFAikCBQIpAgUBEwKCP3JCAgCNwgK2v74CBQI/vgIBQoCJgoF/vABCAgFCv3aCgUIAQgIFID+lAgKCggBbAgIpAgIpAgAAAAD//D/8AS6BLoACQANABAAAAAyHwEWFA8BJzcTAScJAQUTA+AmDpkNDWPWXyL9mdYCZv4f/rNuBLoNmQ4mDlzWYP50/ZrWAmb8anABTwAAAAEAAAAABLAEsAAPAAABETMyFh0BITU0NjsBEQEhArz6FR384B0V+v4MBLACiv3aHRUyMhUdAiYCJgAAAAEADgAIBEwEnAAfAAABJTYWFREUBgcGLgE2NzYXEQURFAYHBi4BNjc2FxE0NgFwAoUnMFNGT4gkV09IQv2oWEFPiCRXT0hCHQP5ow8eIvzBN1EXGSltchkYEAIJm/2iKmAVGilucRoYEQJ/JioAAAACAAn/+AS7BKcAHQApAAAAMh4CFQcXFAcBFgYPAQYiJwEGIycHIi4CND4BBCIOARQeATI+ATQmAZDItoNOAQFOARMXARY7GikT/u13jgUCZLaDTk6DAXKwlFZWlLCUVlYEp06DtmQCBY15/u4aJRg6FBQBEk0BAU6Dtsi2g1tWlLCUVlaUsJQAAQBkAFgErwREABkAAAE+Ah4CFRQOAwcuBDU0PgIeAQKJMHt4dVg2Q3mEqD4+p4V4Qzhadnh5A7VESAUtU3ZAOXmAf7JVVbJ/gHk5QHZTLQVIAAAAAf/TAF4EewSUABgAAAETNjIXEyEyFgcFExYGJyUFBiY3EyUmNjMBl4MHFQeBAaUVBhH+qoIHDxH+qf6qEQ8Hgv6lEQYUAyABYRMT/p8RDPn+bxQLDPb3DAsUAZD7DBEAAv/TAF4EewSUABgAIgAAARM2MhcTITIWBwUTFgYnJQUGJjcTJSY2MwUjFwc3Fyc3IycBl4MHFQeBAaUVBhH+qoIHDxH+qf6qEQ8Hgv6lEQYUAfPwxUrBw0rA6k4DIAFhExP+nxEM+f5vFAsM9vcMCxQBkPsMEWSO4ouM5YzTAAABAAAAAASwBLAAJgAAATIWHQEUBiMVFBYXBR4BHQEUBiMhIiY9ATQ2NyU+AT0BIiY9ATQ2Alh8sD4mDAkBZgkMDwr7ggoPDAkBZgkMJj6wBLCwfPouaEsKFwbmBRcKXQoPDwpdChcF5gYXCktoLvp8sAAAAA0AAAAABLAETAAPABMAIwAnACsALwAzADcARwBLAE8AUwBXAAATITIWFREUBiMhIiY1ETQ2FxUzNSkBIgYVERQWMyEyNjURNCYzFTM1BRUzNSEVMzUFFTM1IRUzNQchIgYVERQWMyEyNjURNCYFFTM1IRUzNQUVMzUhFTM1GQR+Cg8PCvuCCg8PVWQCo/3aCg8PCgImCg8Pc2T8GGQDIGT8GGQDIGTh/doKDw8KAiYKDw/872QDIGT8GGQDIGQETA8K++YKDw8KBBoKD2RkZA8K/qIKDw8KAV4KD2RkyGRkZGTIZGRkZGQPCv6iCg8PCgFeCg9kZGRkZMhkZGRkAAAEAAAAAARMBEwADwAfAC8APwAAEyEyFhURFAYjISImNRE0NikBMhYVERQGIyEiJjURNDYBITIWFREUBiMhIiY1ETQ2KQEyFhURFAYjISImNRE0NjIBkBUdHRX+cBUdHQJtAZAVHR0V/nAVHR39vQGQFR0dFf5wFR0dAm0BkBUdHRX+cBUdHQRMHRX+cBUdHRUBkBUdHRX+cBUdHRUBkBUd/agdFf5wFR0dFQGQFR0dFf5wFR0dFQGQFR0AAAkAAAAABEwETAAPAB8ALwA/AE8AXwBvAH8AjwAAEzMyFh0BFAYrASImPQE0NiEzMhYdARQGKwEiJj0BNDYhMzIWHQEUBisBIiY9ATQ2ATMyFh0BFAYrASImPQE0NiEzMhYdARQGKwEiJj0BNDYhMzIWHQEUBisBIiY9ATQ2ATMyFh0BFAYrASImPQE0NiEzMhYdARQGKwEiJj0BNDYhMzIWHQEUBisBIiY9ATQ2MsgVHR0VyBUdHQGlyBUdHRXIFR0dAaXIFR0dFcgVHR389cgVHR0VyBUdHQGlyBUdHRXIFR0dAaXIFR0dFcgVHR389cgVHR0VyBUdHQGlyBUdHRXIFR0dAaXIFR0dFcgVHR0ETB0VyBUdHRXIFR0dFcgVHR0VyBUdHRXIFR0dFcgVHf5wHRXIFR0dFcgVHR0VyBUdHRXIFR0dFcgVHR0VyBUd/nAdFcgVHR0VyBUdHRXIFR0dFcgVHR0VyBUdHRXIFR0ABgAAAAAEsARMAA8AHwAvAD8ATwBfAAATMzIWHQEUBisBIiY9ATQ2KQEyFh0BFAYjISImPQE0NgEzMhYdARQGKwEiJj0BNDYpATIWHQEUBiMhIiY9ATQ2ATMyFh0BFAYrASImPQE0NikBMhYdARQGIyEiJj0BNDYyyBUdHRXIFR0dAaUCvBUdHRX9RBUdHf6FyBUdHRXIFR0dAaUCvBUdHRX9RBUdHf6FyBUdHRXIFR0dAaUCvBUdHRX9RBUdHQRMHRXIFR0dFcgVHR0VyBUdHRXIFR3+cB0VyBUdHRXIFR0dFcgVHR0VyBUd/nAdFcgVHR0VyBUdHRXIFR0dFcgVHQAAAAABACYALAToBCAAFwAACQE2Mh8BFhQHAQYiJwEmND8BNjIfARYyAdECOwgUB7EICPzxBxUH/oAICLEHFAirBxYB3QI7CAixBxQI/PAICAGACBQHsQgIqwcAAQBuAG4EQgRCACMAAAEXFhQHCQEWFA8BBiInCQEGIi8BJjQ3CQEmND8BNjIXCQE2MgOIsggI/vUBCwgIsggVB/70/vQHFQiyCAgBC/71CAiyCBUHAQwBDAcVBDuzCBUH/vT+9AcVCLIICAEL/vUICLIIFQcBDAEMBxUIsggI/vUBDAcAAwAX/+sExQSZABkAJQBJAAAAMh4CFRQHARYUDwEGIicBBiMiLgI0PgEEIg4BFB4BMj4BNCYFMzIWHQEzMhYdARQGKwEVFAYrASImPQEjIiY9ATQ2OwE1NDYBmcSzgk1OASwICG0HFQj+1HeOYrSBTU2BAW+zmFhYmLOZWFj+vJYKD0sKDw8KSw8KlgoPSwoPDwpLDwSZTYKzYo15/tUIFQhsCAgBK01NgbTEs4JNWJmzmFhYmLOZIw8KSw8KlgoPSwoPDwpLDwqWCg9LCg8AAAMAF//rBMUEmQAZACUANQAAADIeAhUUBwEWFA8BBiInAQYjIi4CND4BBCIOARQeATI+ATQmBSEyFh0BFAYjISImPQE0NgGZxLOCTU4BLAgIbQcVCP7Ud45itIFNTYEBb7OYWFiYs5lYWP5YAV4KDw8K/qIKDw8EmU2Cs2KNef7VCBUIbAgIAStNTYG0xLOCTViZs5hYWJizmYcPCpYKDw8KlgoPAAAAAAIAFwAXBJkEsAAPAC0AAAEzMhYVERQGKwEiJjURNDYFNRYSFRQOAiIuAjU0EjcVDgEVFB4BMj4BNTQmAiZkFR0dFWQVHR0BD6fSW5vW6tabW9KnZ3xyxejFcnwEsB0V/nAVHR0VAZAVHeGmPv7ZuHXWm1tbm9Z1uAEnPqY3yHh0xXJyxXR4yAAEAGQAAASwBLAADwAfAC8APwAAATMyFhURFAYrASImNRE0NgEzMhYVERQGKwEiJjURNDYBMzIWFREUBisBIiY1ETQ2BTMyFh0BFAYrASImPQE0NgQBlgoPDwqWCg8P/t6WCg8PCpYKDw/+3pYKDw8KlgoPD/7elgoPDwqWCg8PBLAPCvuCCg8PCgR+Cg/+cA8K/RIKDw8KAu4KD/7UDwr+PgoPDwoBwgoPyA8K+goPDwr6Cg8AAAAAAgAaABsElgSWAEcATwAAATIfAhYfATcWFwcXFh8CFhUUDwIGDwEXBgcnBwYPAgYjIi8CJi8BByYnNycmLwImNTQ/AjY/ASc2Nxc3Nj8CNhIiBhQWMjY0AlghKSYFMS0Fhj0rUAMZDgGYBQWYAQ8YA1AwOIYFLDIFJisfISkmBTEtBYY8LFADGQ0ClwYGlwINGQNQLzqFBS0xBSYreLJ+frJ+BJYFmAEOGQJQMDmGBSwxBiYrHiIoJgYxLAWGPSxRAxkOApcFBZcCDhkDUTA5hgUtMAYmKiAhKCYGMC0Fhj0sUAIZDgGYBf6ZfrF+frEABwBkAAAEsAUUABMAFwAhACUAKQAtADEAAAEhMhYdASEyFh0BITU0NjMhNTQ2FxUhNQERFAYjISImNREXETMRMxEzETMRMxEzETMRAfQBLCk7ARMKD/u0DwoBEzspASwBLDsp/UQpO2RkZGRkZGRkBRQ7KWQPCktLCg9kKTtkZGT+1PzgKTs7KQMgZP1EArz9RAK8/UQCvP1EArwAAQAMAAAFCATRAB8AABMBNjIXARYGKwERFAYrASImNREhERQGKwEiJjURIyImEgJsCBUHAmAIBQqvDwr6Cg/+1A8K+goPrwoFAmoCYAcH/aAICv3BCg8PCgF3/okKDw8KAj8KAAIAZAAAA+gEsAARABcAAAERFBYzIREUBiMhIiY1ETQ2MwEjIiY9AQJYOykBLB0V/OAVHR0VA1L6FR0EsP5wKTv9dhUdHRUETBUd/nAdFfoAAwAXABcEmQSZAA8AGwAwAAAAMh4CFA4CIi4CND4BBCIOARQeATI+ATQmBTMyFhURMzIWHQEUBisBIiY1ETQ2AePq1ptbW5vW6tabW1ubAb/oxXJyxejFcnL+fDIKD68KDw8K+goPDwSZW5vW6tabW1ub1urWmztyxejFcnLF6MUNDwr+7Q8KMgoPDwoBXgoPAAAAAAL/nAAABRQEsAALAA8AACkBAyMDIQEzAzMDMwEDMwMFFP3mKfIp/eYBr9EVohTQ/p4b4BsBkP5wBLD+1AEs/nD+1AEsAAAAAAIAZAAABLAEsAAVAC8AAAEzMhYVETMyFgcBBiInASY2OwERNDYBMzIWFREUBiMhIiY1ETQ2OwEyFh0BITU0NgImyBUdvxQLDf65DSYN/rkNCxS/HQJUMgoPDwr75goPDwoyCg8DhA8EsB0V/j4XEP5wEBABkBAXAcIVHfzgDwr+ogoPDwoBXgoPDwqvrwoPAAMAFwAXBJkEmQAPABsAMQAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JgUzMhYVETMyFgcDBiInAyY2OwERNDYB4+rWm1tbm9bq1ptbW5sBv+jFcnLF6MVycv58lgoPiRUKDd8NJg3fDQoViQ8EmVub1urWm1tbm9bq1ps7csXoxXJyxejFDQ8K/u0XEP7tEBABExAXARMKDwAAAAMAFwAXBJkEmQAPABsAMQAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JiUTFgYrAREUBisBIiY1ESMiJjcTNjIB4+rWm1tbm9bq1ptbW5sBv+jFcnLF6MVycv7n3w0KFYkPCpYKD4kVCg3fDSYEmVub1urWm1tbm9bq1ps7csXoxXJyxejFAf7tEBf+7QoPDwoBExcQARMQAAAAAAIAAAAABLAEsAAZADkAABMhMhYXExYVERQGBwYjISImJyY1EzQ3Ez4BBSEiBgcDBhY7ATIWHwEeATsBMjY/AT4BOwEyNicDLgHhAu4KEwO6BwgFDBn7tAweAgYBB7kDEwKX/dQKEgJXAgwKlgoTAiYCEwr6ChMCJgITCpYKDAJXAhIEsA4K/XQYGf5XDB4CBggEDRkBqRkYAowKDsgOC/4+Cw4OCpgKDg4KmAoODgsBwgsOAAMAFwAXBJkEmQAPABsAJwAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JgUXFhQPAQYmNRE0NgHj6tabW1ub1urWm1tbmwG/6MVycsXoxXJy/ov9ERH9EBgYBJlbm9bq1ptbW5vW6tabO3LF6MVycsXoxV2+DCQMvgwLFQGQFQsAAQAXABcEmQSwACgAAAE3NhYVERQGIyEiJj8BJiMiDgEUHgEyPgE1MxQOAiIuAjQ+AjMyA7OHBwsPCv6WCwQHhW2BdMVycsXoxXKWW5vW6tabW1ub1nXABCSHBwQL/pYKDwsHhUxyxejFcnLFdHXWm1tbm9bq1ptbAAAAAAIAFwABBJkEsAAaADUAAAE3NhYVERQGIyEiJj8BJiMiDgEVIzQ+AjMyEzMUDgIjIicHBiY1ETQ2MyEyFg8BFjMyPgEDs4cHCw8L/pcLBAeGboF0xXKWW5vWdcDrllub1nXAnIYHCw8LAWgKBQiFboJ0xXIEJIcHBAv+lwsPCweGS3LFdHXWm1v9v3XWm1t2hggFCgFoCw8LB4VMcsUAAAAKAGQAAASwBLAADwAfAC8APwBPAF8AbwB/AI8AnwAAEyEyFhURFAYjISImNRE0NgUhIgYVERQWMyEyNjURNCYFMzIWHQEUBisBIiY9ATQ2MyEyFh0BFAYjISImPQE0NgczMhYdARQGKwEiJj0BNDYzITIWHQEUBiMhIiY9ATQ2BzMyFh0BFAYrASImPQE0NjMhMhYdARQGIyEiJj0BNDYHMzIWHQEUBisBIiY9ATQ2MyEyFh0BFAYjISImPQE0Nn0EGgoPDwr75goPDwPA/K4KDw8KA1IKDw/9CDIKDw8KMgoPD9IBwgoPDwr+PgoPD74yCg8PCjIKDw/SAcIKDw8K/j4KDw++MgoPDwoyCg8P0gHCCg8PCv4+Cg8PvjIKDw8KMgoPD9IBwgoPDwr+PgoPDwSwDwr7ggoPDwoEfgoPyA8K/K4KDw8KA1IKD2QPCjIKDw8KMgoPDwoyCg8PCjIKD8gPCjIKDw8KMgoPDwoyCg8PCjIKD8gPCjIKDw8KMgoPDwoyCg8PCjIKD8gPCjIKDw8KMgoPDwoyCg8PCjIKDwAAAAACAAAAAARMBLAAGQAjAAABNTQmIyEiBh0BIyIGFREUFjMhMjY1ETQmIyE1NDY7ATIWHQEDhHVT/tRSdmQpOzspA4QpOzsp/ageFMgUHgMgyFN1dlLIOyn9qCk7OykCWCk7lhUdHRWWAAIAZAAABEwETAAJADcAABMzMhYVESMRNDYFMhcWFREUBw4DIyIuAScuAiMiBwYjIicmNRE+ATc2HgMXHgIzMjc2fTIKD2QPA8AEBRADIUNAMRwaPyonKSxHHlVLBwgGBQ4WeDsXKC4TOQQpLUUdZ1AHBEwPCvvNBDMKDzACBhH+WwYGO1AkDQ0ODg8PDzkFAwcPAbY3VwMCAwsGFAEODg5XCAAAAwAAAAAEsASXACEAMQBBAAAAMh4CFREUBisBIiY1ETQuASAOARURFAYrASImNRE0PgEDMzIWFREUBisBIiY1ETQ2ITMyFhURFAYrASImNRE0NgHk6N6jYw8KMgoPjeT++uSNDwoyCg9joyqgCAwMCKAIDAwCYKAIDAwIoAgMDASXY6PedP7UCg8PCgEsf9FyctF//tQKDw8KASx03qP9wAwI/jQIDAwIAcwIDAwI/jQIDAwIAcwIDAAAAAACAAAA0wRHA90AFQA5AAABJTYWFREUBiclJisBIiY1ETQ2OwEyBTc2Mh8BFhQPARcWFA8BBiIvAQcGIi8BJjQ/AScmND8BNjIXAUEBAgkMDAn+/hUZ+goPDwr6GQJYeAcUByIHB3h4BwciBxQHeHgHFAciBwd3dwcHIgcUBwMurAYHCv0SCgcGrA4PCgFeCg+EeAcHIgcUB3h4BxQHIgcHd3cHByIHFAd4eAcUByIICAAAAAACAAAA0wNyA90AFQAvAAABJTYWFREUBiclJisBIiY1ETQ2OwEyJTMWFxYVFAcGDwEiLwEuATc2NTQnJjY/ATYBQQECCQwMCf7+FRn6Cg8PCvoZAdIECgZgWgYLAwkHHQcDBkhOBgMIHQcDLqwGBwr9EgoHBqwODwoBXgoPZAEJgaGafwkBAQYXBxMIZ36EaggUBxYFAAAAAAMAAADEBGID7AAbADEASwAAATMWFxYVFAYHBgcjIi8BLgE3NjU0JicmNj8BNgUlNhYVERQGJyUmKwEiJjURNDY7ATIlMxYXFhUUBwYPASIvAS4BNzY1NCcmNj8BNgPHAwsGh0RABwoDCQcqCAIGbzs3BgIJKgf9ggECCQwMCf7+FRn6Cg8PCvoZAdIECgZgWgYLAwkHHQcDBkhOBgMIHQcD7AEJs9lpy1QJAQYiBhQIlrJarEcJFAYhBb6sBgcK/RIKBwasDg8KAV4KD2QBCYGhmn8JAQEGFwcTCGd+hGoIFQYWBQAAAAANAAAAAASwBLAACQAVABkAHQAhACUALQA7AD8AQwBHAEsATwAAATMVIxUhFSMRIQEjFTMVIREjESM1IQURIREhESERBSM1MwUjNTMBMxEhETM1MwEzFSMVIzUjNTM1IzUhBREhEQcjNTMFIzUzASM1MwUhNSEB9GRk/nBkAfQCvMjI/tTIZAJY+7QBLAGQASz84GRkArxkZP1EyP4MyGQB9MhkyGRkyAEs/UQBLGRkZAOEZGT+DGRkAfT+1AEsA4RkZGQCWP4MZMgBLAEsyGT+1AEs/tQBLMhkZGT+DP4MAfRk/tRkZGRkyGTI/tQBLMhkZGT+1GRkZAAAAAAJAAAAAASwBLAAAwAHAAsADwATABcAGwAfACMAADcjETMTIxEzASMRMxMjETMBIxEzASE1IRcjNTMXIzUzBSM1M2RkZMhkZAGQyMjIZGQBLMjI/OD+1AEsyGRkyGRkASzIyMgD6PwYA+j8GAPo/BgD6PwYA+j7UGRkW1tbW1sAAAIAAAAKBKYEsAANABUAAAkBFhQHAQYiJwETNDYzBCYiBhQWMjYB9AKqCAj+MAgUCP1WAQ8KAUM7Uzs7UzsEsP1WCBQI/jAICAKqAdsKD807O1Q7OwAAAAADAAAACgXSBLAADQAZACEAAAkBFhQHAQYiJwETNDYzIQEWFAcBBiIvAQkBBCYiBhQWMjYB9AKqCAj+MAgUCP1WAQ8KAwYCqggI/jAIFAg4Aaj9RP7TO1M7O1M7BLD9VggUCP4wCAgCqgHbCg/9VggUCP4wCAg4AaoCvM07O1Q7OwAAAAABAGQAAASwBLAAJgAAASEyFREUDwEGJjURNCYjISIPAQYWMyEyFhURFAYjISImNRE0PwE2ASwDOUsSQAgKDwr9RBkSQAgFCgK8Cg8PCvyuCg8SixIEsEv8fBkSQAgFCgO2Cg8SQAgKDwr8SgoPDwoDzxkSixIAAAABAMj//wRMBLAACgAAEyEyFhURCQERNDb6AyAVHf4+/j4dBLAdFfuCAbz+QwR/FR0AAAAAAwAAAAAEsASwABUARQBVAAABISIGBwMGHwEeATMhMjY/ATYnAy4BASMiBg8BDgEjISImLwEuASsBIgYVERQWOwEyNj0BNDYzITIWHQEUFjsBMjY1ETQmASEiBg8BBhYzITI2LwEuAQM2/kQLEAFOBw45BhcKAcIKFwY+DgdTARABVpYKFgROBBYK/doKFgROBBYKlgoPDwqWCg8PCgLuCg8PCpYKDw/+sf4MChMCJgILCgJYCgsCJgITBLAPCv7TGBVsCQwMCWwVGAEtCg/+cA0JnAkNDQmcCQ0PCv12Cg8PCpYKDw8KlgoPDwoCigoP/agOCpgKDg4KmAoOAAAAAAQAAABkBLAETAAdACEAKQAxAAABMzIeAh8BMzIWFREUBiMhIiY1ETQ2OwE+BAEVMzUEIgYUFjI2NCQyFhQGIiY0AfTIOF00JAcGlik7Oyn8GCk7OymWAgknM10ByGT+z76Hh76H/u9WPDxWPARMKTs7FRQ7Kf2oKTs7KQJYKTsIG0U1K/7UZGRGh76Hh74IPFY8PFYAAAAAAgA1AAAEsASvACAAIwAACQEWFx4BHwEVITUyNi8BIQYHBh4CMxUhNTY3PgE/AQEDIQMCqQGBFCgSJQkK/l81LBFS/nk6IgsJKjIe/pM4HAwaBwcBj6wBVKIEr/waMioTFQECQkJXLd6RWSIuHAxCQhgcDCUNDQPu/VoByQAAAAADAGQAAAPwBLAAJwAyADsAAAEeBhUUDgMjITU+ATURNC4EJzUFMh4CFRQOAgclMzI2NTQuAisBETMyNjU0JisBAvEFEzUwOyodN1htbDD+DCk7AQYLFyEaAdc5dWM+Hy0tEP6Pi05pESpTPnbYUFJ9Xp8CgQEHGB0zOlIuQ3VONxpZBzMoAzsYFBwLEAkHRwEpSXNDM1s6KwkxYUopOzQb/K5lUFqBAAABAMgAAANvBLAAGQAAARcOAQcDBhYXFSE1NjcTNjQuBCcmJzUDbQJTQgeECSxK/gy6Dq0DAw8MHxUXDQYEsDkTNSj8uTEoBmFhEFIDQBEaExAJCwYHAwI5AAAAAAL/tQAABRQEsAAlAC8AAAEjNC4FKwERFBYfARUhNTI+AzURIyIOBRUjESEFIxEzByczESM3BRQyCAsZEyYYGcgyGRn+cAQOIhoWyBkYJhMZCwgyA+j7m0tLfX1LS30DhBUgFQ4IAwH8rhYZAQJkZAEFCRUOA1IBAwgOFSAVASzI/OCnpwMgpwACACH/tQSPBLAAJQAvAAABIzQuBSsBERQWHwEVITUyPgM1ESMiDgUVIxEhEwc1IRUnNxUhNQRMMggLGRMmGBnIMhkZ/nAEDiIaFsgZGCYTGQsIMgPoQ6f84KenAyADhBUgFQ4IAwH9dhYZAQJkZAEFCRUOAooBAwgOFSAVASz7gn1LS319S0sABAAAAAAEsARMAA8AHwAvAD8AABMhMhYdARQGIyEiJj0BNDYTITIWHQEUBiMhIiY9ATQ2EyEyFh0BFAYjISImPQE0NhMhMhYdARQGIyEiJj0BNDYyAlgVHR0V/agVHR0VA+gVHR0V/BgVHR0VAyAVHR0V/OAVHR0VBEwVHR0V+7QVHR0ETB0VZBUdHRVkFR3+1B0VZBUdHRVkFR3+1B0VZBUdHRVkFR3+1B0VZBUdHRVkFR0ABAAAAAAEsARMAA8AHwAvAD8AABMhMhYdARQGIyEiJj0BNDYDITIWHQEUBiMhIiY9ATQ2EyEyFh0BFAYjISImPQE0NgMhMhYdARQGIyEiJj0BNDb6ArwVHR0V/UQVHR2zBEwVHR0V+7QVHR3dArwVHR0V/UQVHR2zBEwVHR0V+7QVHR0ETB0VZBUdHRVkFR3+1B0VZBUdHRVkFR3+1B0VZBUdHRVkFR3+1B0VZBUdHRVkFR0ABAAAAAAEsARMAA8AHwAvAD8AAAE1NDYzITIWHQEUBiMhIiYBNTQ2MyEyFh0BFAYjISImEzU0NjMhMhYdARQGIyEiJgE1NDYzITIWHQEUBiMhIiYB9B0VAlgVHR0V/agVHf5wHRUD6BUdHRX8GBUdyB0VAyAVHR0V/OAVHf7UHRUETBUdHRX7tBUdA7ZkFR0dFWQVHR3+6WQVHR0VZBUdHf7pZBUdHRVkFR0d/ulkFR0dFWQVHR0AAAQAAAAABLAETAAPAB8ALwA/AAATITIWHQEUBiMhIiY9ATQ2EyEyFh0BFAYjISImPQE0NhMhMhYdARQGIyEiJj0BNDYTITIWHQEUBiMhIiY9ATQ2MgRMFR0dFfu0FR0dFQRMFR0dFfu0FR0dFQRMFR0dFfu0FR0dFQRMFR0dFfu0FR0dBEwdFWQVHR0VZBUd/tQdFWQVHR0VZBUd/tQdFWQVHR0VZBUd/tQdFWQVHR0VZBUdAAgAAAAABLAETAAPAB8ALwA/AE8AXwBvAH8AABMzMhYdARQGKwEiJj0BNDYpATIWHQEUBiMhIiY9ATQ2ATMyFh0BFAYrASImPQE0NikBMhYdARQGIyEiJj0BNDYBMzIWHQEUBisBIiY9ATQ2KQEyFh0BFAYjISImPQE0NgEzMhYdARQGKwEiJj0BNDYpATIWHQEUBiMhIiY9ATQ2MmQVHR0VZBUdHQFBAyAVHR0V/OAVHR3+6WQVHR0VZBUdHQFBAyAVHR0V/OAVHR3+6WQVHR0VZBUdHQFBAyAVHR0V/OAVHR3+6WQVHR0VZBUdHQFBAyAVHR0V/OAVHR0ETB0VZBUdHRVkFR0dFWQVHR0VZBUd/tQdFWQVHR0VZBUdHRVkFR0dFWQVHf7UHRVkFR0dFWQVHR0VZBUdHRVkFR3+1B0VZBUdHRVkFR0dFWQVHR0VZBUdAAAG/5wAAASwBEwAAwATACMAKgA6AEoAACEjETsCMhYdARQGKwEiJj0BNDYTITIWHQEUBiMhIiY9ATQ2BQc1IzUzNQUhMhYdARQGIyEiJj0BNDYTITIWHQEUBiMhIiY9ATQ2AZBkZJZkFR0dFWQVHR0VAfQVHR0V/gwVHR3++qfIyAHCASwVHR0V/tQVHR0VAlgVHR0V/agVHR0ETB0VZBUdHRVkFR3+1B0VZBUdHRVkFR36fUtkS68dFWQVHR0VZBUd/tQdFWQVHR0VZBUdAAAABgAAAAAFFARMAA8AEwAjACoAOgBKAAATMzIWHQEUBisBIiY9ATQ2ASMRMwEhMhYdARQGIyEiJj0BNDYFMxUjFSc3BSEyFh0BFAYjISImPQE0NhMhMhYdARQGIyEiJj0BNDYyZBUdHRVkFR0dA2dkZPyuAfQVHR0V/gwVHR0EL8jIp6f75gEsFR0dFf7UFR0dFQJYFR0dFf2oFR0dBEwdFWQVHR0VZBUd+7QETP7UHRVkFR0dFWQVHchkS319rx0VZBUdHRVkFR3+1B0VZBUdHRVkFR0AAAAAAgAAAMgEsAPoAA8AEgAAEyEyFhURFAYjISImNRE0NgkCSwLuHywsH/0SHywsBIT+1AEsA+gsH/12HywsHwKKHyz9RAEsASwAAwAAAAAEsARMAA8AFwAfAAATITIWFREUBiMhIiY1ETQ2FxE3BScBExEEMhYUBiImNCwEWBIaGhL7qBIaGkr3ASpKASXs/NJwTk5wTgRMGhL8DBIaGhID9BIaZP0ftoOcAT7+4AH0dE5vT09vAAAAAAIA2wAFBDYEkQAWAB4AAAEyHgEVFAcOAQ8BLgQnJjU0PgIWIgYUFjI2NAKIdcZzRkWyNjYJIV5YbSk8RHOft7eCgreCBJF4ynVzj23pPz4IIWZomEiEdVijeUjDgriBgbgAAAACABcAFwSZBJkADwAXAAAAMh4CFA4CIi4CND4BAREiDgEUHgEB4+rWm1tbm9bq1ptbW5sBS3TFcnLFBJlbm9bq1ptbW5vW6tab/G8DVnLF6MVyAAACAHUAAwPfBQ8AGgA1AAABHgYVFA4DBy4DNTQ+BQMOAhceBBcWNj8BNiYnLgInJjc2IyYCKhVJT1dOPiUzVnB9P1SbfEokP0xXUEm8FykoAwEbITEcExUWAgYCCQkFEikMGiACCAgFD0iPdXdzdYdFR4BeRiYEBTpjl1lFh3ZzeHaQ/f4hS4I6JUEnIw4IBwwQIgoYBwQQQSlZtgsBAAAAAwAAAAAEywRsAAwAKgAvAAABNz4CHgEXHgEPAiUhMhcHISIGFREUFjMhMjY9ATcRFAYjISImNRE0NgkBBzcBA+hsAgYUFR0OFgoFBmz9BQGQMje7/pApOzspAfQpO8i7o/5wpbm5Azj+lqE3AWMD9XMBAgIEDw4WKgsKc8gNuzsp/gwpOzsptsj+tKW5uaUBkKW5/tf+ljKqAWMAAgAAAAAEkwRMABsANgAAASEGByMiBhURFBYzITI2NTcVFAYjISImNRE0NgUBFhQHAQYmJzUmDgMHPgY3NT4BAV4BaaQ0wyk7OykB9Ck7yLml/nClubkCfwFTCAj+rAcLARo5ZFRYGgouOUlARioTAQsETJI2Oyn+DCk7OymZZ6W5uaUBkKW5G/7TBxUH/s4GBAnLAQINFjAhO2JBNB0UBwHSCgUAAAAAAgAAAAAEnQRMAB0ANQAAASEyFwchIgYVERQWMyEyNj0BNxUUBiMhIiY1ETQ2CQE2Mh8BFhQHAQYiLwEmND8BNjIfARYyAV4BXjxDsv6jKTs7KQH0KTvIuaX+cKW5uQHKAYsHFQdlBwf97QcVB/gHB2UHFQdvCBQETBexOyn+DCk7OylFyNulubmlAZCluf4zAYsHB2UHFQf97AcH+AcVB2UHB28HAAAAAQAKAAoEpgSmADsAAAkBNjIXARYGKwEVMzU0NhcBFhQHAQYmPQEjFTMyFgcBBiInASY2OwE1IxUUBicBJjQ3ATYWHQEzNSMiJgE+AQgIFAgBBAcFCqrICggBCAgI/vgICsiqCgUH/vwIFAj++AgFCq/ICgj++AgIAQgICsivCgUDlgEICAj++AgKyK0KBAf+/AcVB/73BwQKrcgKCP74CAgBCAgKyK0KBAcBCQcVBwEEBwQKrcgKAAEAyAAAA4QETAAZAAATMzIWFREBNhYVERQGJwERFAYrASImNRE0NvpkFR0B0A8VFQ/+MB0VZBUdHQRMHRX+SgHFDggV/BgVCA4Bxf5KFR0dFQPoFR0AAAABAAAAAASwBEwAIwAAEzMyFhURATYWFREBNhYVERQGJwERFAYnAREUBisBIiY1ETQ2MmQVHQHQDxUB0A8VFQ/+MBUP/jAdFWQVHR0ETB0V/koBxQ4IFf5KAcUOCBX8GBUIDgHF/koVCA4Bxf5KFR0dFQPoFR0AAAABAJ0AGQSwBDMAFQAAAREUBicBERQGJwEmNDcBNhYVEQE2FgSwFQ/+MBUP/hQPDwHsDxUB0A8VBBr8GBUIDgHF/koVCA4B4A4qDgHgDggV/koBxQ4IAAAAAQDIABYEMwQ2AAsAABMBFhQHAQYmNRE0NvMDLhIS/NISGRkEMv4OCx4L/g4LDhUD6BUOAAIAyABkA4QD6AAPAB8AABMzMhYVERQGKwEiJjURNDYhMzIWFREUBisBIiY1ETQ2+sgVHR0VyBUdHQGlyBUdHRXIFR0dA+gdFfzgFR0dFQMgFR0dFfzgFR0dFQMgFR0AAAEAyABkBEwD6AAPAAABERQGIyEiJjURNDYzITIWBEwdFfzgFR0dFQMgFR0DtvzgFR0dFQMgFR0dAAAAAAEAAAAZBBMEMwAVAAABETQ2FwEWFAcBBiY1EQEGJjURNDYXAfQVDwHsDw/+FA8V/jAPFRUPAmQBthUIDv4gDioO/iAOCBUBtv47DggVA+gVCA4AAAH//gACBLMETwAjAAABNzIWFRMUBiMHIiY1AwEGJjUDAQYmNQM0NhcBAzQ2FwEDNDYEGGQUHgUdFWQVHQL+MQ4VAv4yDxUFFQ8B0gIVDwHSAh0ETgEdFfwYFR0BHRUBtf46DwkVAbX+OQ4JFAPoFQkP/j4BthQJDv49AbYVHQAAAQEsAAAD6ARMABkAAAEzMhYVERQGKwEiJjURAQYmNRE0NhcBETQ2A1JkFR0dFWQVHf4wDxUVDwHQHQRMHRX8GBUdHRUBtv47DggVA+gVCA7+OwG2FR0AAAIAZADIBLAESAALABsAAAkBFgYjISImNwE2MgEhMhYdARQGIyEiJj0BNDYCrgH1DwkW++4WCQ8B9Q8q/fcD6BUdHRX8GBUdHQQ5/eQPFhYPAhwP/UgdFWQVHR0VZBUdAAEAiP/8A3UESgAFAAAJAgcJAQN1/qABYMX92AIoA4T+n/6fxgIoAiYAAAAAAQE7//wEKARKAAUAAAkBJwkBNwQo/dnGAWH+n8YCI/3ZxgFhAWHGAAIAFwAXBJkEmQAPADMAAAAyHgIUDgIiLgI0PgEFIyIGHQEjIgYdARQWOwEVFBY7ATI2PQEzMjY9ATQmKwE1NCYB4+rWm1tbm9bq1ptbW5sBfWQVHZYVHR0Vlh0VZBUdlhUdHRWWHQSZW5vW6tabW1ub1urWm7odFZYdFWQVHZYVHR0Vlh0VZBUdlhUdAAAAAAIAFwAXBJkEmQAPAB8AAAAyHgIUDgIiLgI0PgEBISIGHQEUFjMhMjY9ATQmAePq1ptbW5vW6tabW1ubAkX+DBUdHRUB9BUdHQSZW5vW6tabW1ub1urWm/5+HRVkFR0dFWQVHQACABcAFwSZBJkADwAzAAAAMh4CFA4CIi4CND4BBCIPAScmIg8BBhQfAQcGFB8BFjI/ARcWMj8BNjQvATc2NC8BAePq1ptbW5vW6tabW1ubAeUZCXh4CRkJjQkJeHgJCY0JGQl4eAkZCY0JCXh4CQmNBJlbm9bq1ptbW5vW6tabrQl4eAkJjQkZCXh4CRkJjQkJeHgJCY0JGQl4eAkZCY0AAgAXABcEmQSZAA8AJAAAADIeAhQOAiIuAjQ+AQEnJiIPAQYUHwEWMjcBNjQvASYiBwHj6tabW1ub1urWm1tbmwEVVAcVCIsHB/IHFQcBdwcHiwcVBwSZW5vW6tabW1ub1urWm/4xVQcHiwgUCPEICAF3BxUIiwcHAAAAAAMAFwAXBJkEmQAPADsASwAAADIeAhQOAiIuAjQ+AQUiDgMVFDsBFjc+ATMyFhUUBgciDgUHBhY7ATI+AzU0LgMTIyIGHQEUFjsBMjY9ATQmAePq1ptbW5vW6tabW1ubAT8dPEIyIRSDHgUGHR8UFw4TARkOGhITDAIBDQ6tBx4oIxgiM0Q8OpYKDw8KlgoPDwSZW5vW6tabW1ub1urWm5ELHi9PMhkFEBQQFRIXFgcIBw4UHCoZCBEQKDhcNi9IKhsJ/eMPCpYKDw8KlgoPAAADABcAFwSZBJkADwAfAD4AAAAyHgIUDgIiLgI0PgEFIyIGHQEUFjsBMjY9ATQmAyMiBh0BFBY7ARUjIgYdARQWMyEyNj0BNCYrARE0JgHj6tabW1ub1urWm1tbmwGWlgoPDwqWCg8PCvoKDw8KS0sKDw8KAV4KDw8KSw8EmVub1urWm1tbm9bq1ptWDwqWCg8PCpYKD/7UDwoyCg/IDwoyCg8PCjIKDwETCg8AAgAAAAAEsASwAC8AXwAAATMyFh0BHgEXMzIWHQEUBisBDgEHFRQGKwEiJj0BLgEnIyImPQE0NjsBPgE3NTQ2ExUUBisBIiY9AQ4BBzMyFh0BFAYrAR4BFzU0NjsBMhYdAT4BNyMiJj0BNDY7AS4BAg2WCg9nlxvCCg8PCsIbl2cPCpYKD2eXG8IKDw8KwhuXZw+5DwqWCg9EZheoCg8PCqgXZkQPCpYKD0RmF6gKDw8KqBdmBLAPCsIbl2cPCpYKD2eXG8IKDw8KwhuXZw8KlgoPZ5cbwgoP/s2oCg8PCqgXZkQPCpYKD0RmF6gKDw8KqBdmRA8KlgoPRGYAAwAXABcEmQSZAA8AGwA/AAAAMh4CFA4CIi4CND4BBCIOARQeATI+ATQmBxcWFA8BFxYUDwEGIi8BBwYiLwEmND8BJyY0PwE2Mh8BNzYyAePq1ptbW5vW6tabW1ubAb/oxXJyxejFcnKaQAcHfHwHB0AHFQd8fAcVB0AHB3x8BwdABxUHfHwHFQSZW5vW6tabW1ub1urWmztyxejFcnLF6MVaQAcVB3x8BxUHQAcHfHwHB0AHFQd8fAcVB0AHB3x8BwAAAAMAFwAXBJkEmQAPABsAMAAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JgcXFhQHAQYiLwEmND8BNjIfATc2MgHj6tabW1ub1urWm1tbmwG/6MVycsXoxXJyg2oHB/7ACBQIyggIagcVB0/FBxUEmVub1urWm1tbm9bq1ps7csXoxXJyxejFfWoHFQf+vwcHywcVB2oICE/FBwAAAAMAFwAXBJkEmQAPABgAIQAAADIeAhQOAiIuAjQ+AQUiDgEVFBcBJhcBFjMyPgE1NAHj6tabW1ub1urWm1tbmwFLdMVyQQJLafX9uGhzdMVyBJlbm9bq1ptbW5vW6tabO3LFdHhpAktB0P24PnLFdHMAAAAAAQAXAFMEsAP5ABUAABMBNhYVESEyFh0BFAYjIREUBicBJjQnAgoQFwImFR0dFf3aFxD99hACRgGrDQoV/t0dFcgVHf7dFQoNAasNJgAAAAABAAAAUwSZA/kAFQAACQEWFAcBBiY1ESEiJj0BNDYzIRE0NgJ/AgoQEP32EBf92hUdHRUCJhcD8f5VDSYN/lUNChUBIx0VyBUdASMVCgAAAAEAtwAABF0EmQAVAAAJARYGIyERFAYrASImNREhIiY3ATYyAqoBqw0KFf7dHRXIFR3+3RUKDQGrDSYEif32EBf92hUdHRUCJhcQAgoQAAAAAQC3ABcEXQSwABUAAAEzMhYVESEyFgcBBiInASY2MyERNDYCJsgVHQEjFQoN/lUNJg3+VQ0KFQEjHQSwHRX92hcQ/fYQEAIKEBcCJhUdAAABAAAAtwSZBF0AFwAACQEWFAcBBiY1EQ4DBz4ENxE0NgJ/AgoQEP32EBdesKWBJAUsW4fHfhcEVf5VDSYN/lUNChUBIwIkRHVNabGdcUYHAQYVCgACAAAAAASwBLAAFQArAAABITIWFREUBi8BBwYiLwEmND8BJyY2ASEiJjURNDYfATc2Mh8BFhQPARcWBgNSASwVHRUOXvkIFAhqBwf5Xg4I/iH+1BUdFQ5e+QgUCGoHB/leDggEsB0V/tQVCA5e+QcHaggUCPleDhX7UB0VASwVCA5e+QcHaggUCPleDhUAAAACAEkASQRnBGcAFQArAAABFxYUDwEXFgYjISImNRE0Nh8BNzYyASEyFhURFAYvAQcGIi8BJjQ/AScmNgP2agcH+V4OCBX+1BUdFQ5e+QgU/QwBLBUdFQ5e+QgUCGoHB/leDggEYGoIFAj5Xg4VHRUBLBUIDl75B/3xHRX+1BUIDl75BwdqCBQI+V4OFQAAAAADABcAFwSZBJkADwAfAC8AAAAyHgIUDgIiLgI0PgEFIyIGFxMeATsBMjY3EzYmAyMiBh0BFBY7ATI2PQE0JgHj6tabW1ub1urWm1tbmwGz0BQYBDoEIxQ2FCMEOgQYMZYKDw8KlgoPDwSZW5vW6tabW1ub1urWm7odFP7SFB0dFAEuFB3+DA8KlgoPDwqWCg8AAAAABQAAAAAEsASwAEkAVQBhAGgAbwAAATIWHwEWHwEWFxY3Nj8BNjc2MzIWHwEWHwIeATsBMhYdARQGKwEiBh0BIREjESE1NCYrASImPQE0NjsBMjY1ND8BNjc+BAUHBhY7ATI2LwEuAQUnJgYPAQYWOwEyNhMhIiY1ESkBERQGIyERAQQJFAUFFhbEFQ8dCAsmxBYXERUXMA0NDgQZCAEPCj0KDw8KMgoP/nDI/nAPCjIKDw8KPQsOCRkFDgIGFRYfAp2mBwQK2woKAzMDEP41sQgQAzMDCgrnCwMe/okKDwGQAlgPCv6JBLAEAgIKDXYNCxUJDRZ2DQoHIREQFRh7LAkLDwoyCg8PCq8BLP7UrwoPDwoyCg8GBQQwgBkUAwgWEQ55ogcKDgqVCgSqnQcECo8KDgr8cg8KAXf+iQoPAZAAAAAAAgAAAAwErwSmACsASQAAATYWFQYCDgQuAScmByYOAQ8BBiY1NDc+ATc+AScuAT4BNz4GFyYGBw4BDwEOBAcOARY2Nz4CNz4DNz4BBI0IGgItQmxhi2KORDg9EQQRMxuZGhYqCFUYEyADCQIQOjEnUmFch3vAJQgdHyaiPT44XHRZUhcYDhItIRmKcVtGYWtbKRYEBKYDEwiy/t3IlVgxEQgLCwwBAQIbG5kYEyJAJghKFRE8Hzdff4U/M0o1JSMbL0QJGCYvcSEhHjZST2c1ODwEJygeW0AxJUBff1UyFAABAF0AHgRyBM8ATwAAAQ4BHgQXLgc+ATceAwYHDgQHBicmNzY3PgQuAScWDgMmJy4BJyY+BDcGHgM3PgEuAicmPgMCjScfCic4R0IgBBsKGAoQAwEJEg5gikggBhANPkpTPhZINx8SBgsNJysiCRZOQQoVNU1bYC9QZwICBAUWITsoCAYdJzIYHw8YIiYHDyJJYlkEz0OAZVxEOSQMBzgXOB42IzElKRIqg5Gnl0o3Z0c6IAYWCwYNAwQFIDhHXGF1OWiqb0sdBxUknF0XNTQ8PEUiNWNROBYJDS5AQVUhVZloUSkAAAAAA//cAGoE1ARGABsAPwBRAAAAMh4FFA4FIi4FND4EBSYGFxYVFAYiJjU0NzYmBwYHDgEXHgQyPgM3NiYnJgUHDgEXFhcWNj8BNiYnJicuAQIGpJ17bk85HBw6T257naKde25POhwcOU9uewIPDwYIGbD4sBcIBw5GWg0ECxYyWl+DiINfWjIWCwQMWv3/Iw8JCSU4EC0OIw4DDywtCyIERi1JXGJcSSpJXGJcSS0tSVxiXEkqSVxiXEncDwYTOT58sLB8OzcTBg9FcxAxEiRGXkQxMEVeRSQSMRF1HiQPLxJEMA0EDyIPJQ8sSRIEAAAABP/cAAAE1ASwABQAJwA7AEwAACEjNy4ENTQ+BTMyFzczEzceARUUDgMHNz4BNzYmJyYlBgcOARceBBc3LgE1NDc2JhcHDgEXFhcWNj8CJyYnLgECUJQfW6l2WSwcOU9ue51SPUEglCYvbIknUGqYUi5NdiYLBAw2/VFGWg0ECxIqSExoNSlrjxcIB3wjDwkJJTgQLQ4MFgMsLQsieBRhdHpiGxVJXGJcSS0Pef5StVXWNBpacm5jGq0xiD8SMRFGckVzEDESHjxRQTkNmhKnbjs3EwZwJA8vEkQwDQQPC1YELEkSBAAAAAP/ngAABRIEqwALABgAKAAAJwE2FhcBFgYjISImJSE1NDY7ATIWHQEhAQczMhYPAQ4BKwEiJi8BJjZaAoIUOBQCghUbJfryJRsBCgFZDwqWCg8BWf5DaNAUGAQ6BCMUNhQjBDoEGGQEKh8FIfvgIEdEhEsKDw8KSwLT3x0U/BQdHRT8FB0AAAABAGQAFQSwBLAAKAAAADIWFREBHgEdARQGJyURFh0BFAYvAQcGJj0BNDcRBQYmPQE0NjcBETQCTHxYAWsPFhgR/plkGhPNzRMaZP6ZERgWDwFrBLBYPv6t/rsOMRQpFA0M+f75XRRAFRAJgIAJEBVAFF0BB/kMDRQpFDEOAUUBUz4AAAARAAAAAARMBLAAHQAnACsALwAzADcAOwA/AEMARwBLAE8AUwBXAFsAXwBjAAABMzIWHQEzMhYdASE1NDY7ATU0NjsBMhYdASE1NDYBERQGIyEiJjURFxUzNTMVMzUzFTM1MxUzNTMVMzUFFTM1MxUzNTMVMzUzFTM1MxUzNQUVMzUzFTM1MxUzNTMVMzUzFTM1A1JkFR0yFR37tB0VMh0VZBUdAfQdAQ8dFfwYFR1kZGRkZGRkZGRk/HxkZGRkZGRkZGT8fGRkZGRkZGRkZASwHRUyHRWWlhUdMhUdHRUyMhUd/nD9EhUdHRUC7shkZGRkZGRkZGRkyGRkZGRkZGRkZGTIZGRkZGRkZGRkZAAAAAMAAAAZBXcElwAZACUANwAAARcWFA8BBiY9ASMBISImPQE0NjsBATM1NDYBBycjIiY9ATQ2MyEBFxYUDwEGJj0BIyc3FzM1NDYEb/kPD/kOFZ/9qP7dFR0dFdECWPEV/amNetEVHR0VASMDGvkPD/kOFfG1jXqfFQSN5g4qDuYOCBWW/agdFWQVHQJYlhUI/piNeh0VZBUd/k3mDioO5g4IFZa1jXqWFQgAAAABAAAAAASwBEwAEgAAEyEyFhURFAYjIQERIyImNRE0NmQD6Ck7Oyn9rP7QZCk7OwRMOyn9qCk7/tQBLDspAlgpOwAAAAMAZAAABEwEsAAJABMAPwAAEzMyFh0BITU0NiEzMhYdASE1NDYBERQOBSIuBTURIRUUFRwBHgYyPgYmNTQ9AZbIFR3+1B0C0cgVHf7UHQEPBhgoTGacwJxmTCgYBgEsAwcNFB8nNkI2Jx8TDwUFAQSwHRX6+hUdHRX6+hUd/nD+1ClJalZcPigoPlxWakkpASz6CRIVKyclIRsWEAgJEBccISUnKhURCPoAAAAB//8A1ARMA8IABQAAAQcJAScBBEzG/p/+n8UCJwGbxwFh/p/HAicAAQAAAO4ETQPcAAUAAAkCNwkBBE392v3ZxgFhAWEDFf3ZAifH/p8BYQAAAAAC/1EAZAVfA+gAFAApAAABITIWFREzMhYPAQYiLwEmNjsBESElFxYGKwERIRchIiY1ESMiJj8BNjIBlALqFR2WFQgO5g4qDuYOCBWW/oP+HOYOCBWWAYHX/RIVHZYVCA7mDioD6B0V/dkVDvkPD/kOFQGRuPkOFf5wyB0VAiYVDvkPAAABAAYAAASeBLAAMAAAEzMyFh8BITIWBwMOASMhFyEyFhQGKwEVFAYiJj0BIRUUBiImPQEjIiYvAQMjIiY0NjheERwEJgOAGB4FZAUsIf2HMAIXFR0dFTIdKh3+1B0qHR8SHQYFyTYUHh4EsBYQoiUY/iUVK8gdKh0yFR0dFTIyFR0dFTIUCQoDwR0qHQAAAAACAAAAAASwBEwACwAPAAABFSE1MzQ2MyEyFhUFIREhBLD7UMg7KQEsKTv9RASw+1AD6GRkKTs7Kcj84AACAAAAAAXcBEwADAAQAAATAxEzNDYzITIWFSEVBQEhAcjIyDspASwqOgH0ASz+1PtQASwDIP5wAlgpOzspyGT9RAK8AAEBRQAAA2sErwAbAAABFxYGKwERMzIWDwEGIi8BJjY7AREjIiY/ATYyAnvmDggVlpYVCA7mDioO5g4IFZaWFQgO5g4qBKD5DhX9pxUO+Q8P+Q4VAlkVDvkPAAAAAQABAUQErwNrABsAAAEXFhQPAQYmPQEhFRQGLwEmND8BNhYdASE1NDYDqPkODvkPFf2oFQ/5Dg75DxUCWBUDYOUPKQ/lDwkUl5cUCQ/lDykP5Q8JFZWVFQkAAAAEAAAAAASwBLAACQAZAB0AIQAAAQMuASMhIgYHAwUhIgYdARQWMyEyNj0BNCYFNTMVMzUzFQSRrAUkFP1gFCQFrAQt/BgpOzspA+gpOzv+q2RkZAGQAtwXLSgV/R1kOylkKTs7KWQpO8hkZGRkAAAAA/+cAGQEsARMAAsAIwAxAAAAMhYVERQGIiY1ETQDJSMTFgYjIisBIiYnAj0BNDU0PgE7ASUBFSIuAz0BND4CNwRpKh0dKh1k/V0mLwMRFQUCVBQdBDcCCwzIAqP8GAQOIhoWFR0dCwRMHRX8rhUdHRUDUhX8mcj+7BAIHBUBUQ76AgQQDw36/tT6AQsTKRwyGigUDAEAAAACAEoAAARmBLAALAA1AAABMzIWDwEeARcTFzMyFhQGBw4EIyIuBC8BLgE0NjsBNxM+ATcnJjYDFjMyNw4BIiYCKV4UEgYSU3oPP3YRExwaEggeZGqfTzl0XFU+LwwLEhocExF2Pw96UxIGEyQyNDUxDDdGOASwFRMlE39N/rmtHSkoBwQLHBYSCg4REg4FBAgoKR2tAUdNfhQgExr7vgYGMT09AAEAFAAUBJwEnAAXAAABNwcXBxcHFycHJwcnBzcnNyc3Jxc3FzcDIOBO6rS06k7gLZubLeBO6rS06k7gLZubA7JO4C2bmy3gTuq0tOpO4C2bmy3gTuq0tAADAAAAZASwBLAAIQAtAD0AAAEzMhYdAQchMhYdARQHAw4BKwEiJi8BIyImNRE0PwI+ARcPAREzFzMTNSE3NQEzMhYVERQGKwEiJjURNDYCijIoPBwBSCg8He4QLBf6B0YfHz0tNxSRYA0xG2SWZIjW+v4+Mv12ZBUdHRVkFR0dBLBRLJZ9USxkLR3+qBghMhkZJCcBkCQbxMYcKGTU1f6JZAF3feGv/tQdFf4MFR0dFQH0FR0AAAAAAwAAAAAEsARMACAAMAA8AAABMzIWFxMWHQEUBiMhFh0BFAYrASImLwImNRE0NjsBNgUzMhYVERQGKwEiJjURNDYhByMRHwEzNSchNQMCWPoXLBDuHTwo/rgcPCgyGzENYJEUNy09fP3pZBUdHRVkFR0dAl+IZJZkMjIBwvoETCEY/qgdLWQsUXYHlixRKBzGxBskAZAnJGRkHRX+DBUdHRUB9BUdZP6J1dSv4X0BdwADAAAAZAUOBE8AGwA3AEcAAAElNh8BHgEPASEyFhQGKwEDDgEjISImNRE0NjcXERchEz4BOwEyNiYjISoDLgQnJj8BJwUzMhYVERQGKwEiJjURNDYBZAFrHxZuDQEMVAEuVGxuVGqDBhsP/qoHphwOOmQBJYMGGw/LFRMSFv44AgoCCQMHAwUDAQwRklb9T2QVHR0VZBUdHQNp5hAWcA0mD3lMkE7+rRUoog0CDRElCkj+CVkBUxUoMjIBAgIDBQIZFrdT5B0V/gwVHR0VAfQVHQAAAAP/nABkBLAETwAdADYARgAAAQUeBBURFAYjISImJwMjIiY0NjMhJyY2PwE2BxcWBw4FKgIjIRUzMhYXEyE3ESUFMzIWFREUBisBIiY1ETQ2AdsBbgIIFBANrAf+qg8bBoNqVW1sVAEuVQsBDW4WSpIRDAIDBQMHAwkDCgH+Jd0PHAaCASZq/qoCUGQVHR0VZBUdHQRP5gEFEBEXC/3zDaIoFQFTTpBMeQ8mDXAWrrcWGQIFAwICAWQoFf6tWQH37OQdFf4MFR0dFQH0FR0AAAADAGEAAARMBQ4AGwA3AEcAAAAyFh0BBR4BFREUBiMhIiYvAQMmPwE+AR8BETQXNTQmBhURHAMOBAcGLwEHEyE3ESUuAQMhMhYdARQGIyEiJj0BNDYB3pBOAVMVKKIN/fMRJQoJ5hAWcA0mD3nGMjIBAgIDBQIZFrdT7AH3Wf6tFSiWAfQVHR0V/gwVHR0FDm5UaoMGGw/+qgemHA4OAWsfFm4NAQxUAS5U1ssVExIW/jgCCgIJAwcDBQMBDBGSVv6tZAElgwYb/QsdFWQVHR0VZBUdAAP//QAGA+gFFAAPAC0ASQAAASEyNj0BNCYjISIGHQEUFgEVFAYiJjURBwYmLwEmNxM+BDMhMhYVERQGBwEDFzc2Fx4FHAIVERQWNj0BNDY3JREnAV4B9BUdHRX+DBUdHQEPTpBMeQ8mDXAWEOYBBRARFwsCDQ2iKBX9iexTtxYZAgUDAgIBMjIoFQFTWQRMHRVkFR0dFWQVHfzmalRubFQBLlQMAQ1uFh8BawIIEw8Mpgf+qg8bBgHP/q1WkhEMAQMFAwcDCQIKAv44FhITFcsPGwaDASVkAAIAFgAWBJoEmgAPACUAAAAyHgIUDgIiLgI0PgEBJSYGHQEhIgYdARQWMyEVFBY3JTY0AeLs1ptbW5vW7NabW1ubAob+7RAX/u0KDw8KARMXEAETEASaW5vW7NabW1ub1uzWm/453w0KFYkPCpYKD4kVCg3fDSYAAAIAFgAWBJoEmgAPACUAAAAyHgIUDgIiLgI0PgENAQYUFwUWNj0BITI2PQE0JiMhNTQmAeLs1ptbW5vW7NabW1ubASX+7RAQARMQFwETCg8PCv7tFwSaW5vW7NabW1ub1uzWm+jfDSYN3w0KFYkPCpYKD4kVCgAAAAIAFgAWBJoEmgAPACUAAAAyHgIUDgIiLgI0PgEBAyYiBwMGFjsBERQWOwEyNjURMzI2AeLs1ptbW5vW7NabW1ubAkvfDSYN3w0KFYkPCpYKD4kVCgSaW5vW7NabW1ub1uzWm/5AARMQEP7tEBf+7QoPDwoBExcAAAIAFgAWBJoEmgAPACUAAAAyHgIUDgIiLgI0PgEFIyIGFREjIgYXExYyNxM2JisBETQmAeLs1ptbW5vW7NabW1ubAZeWCg+JFQoN3w0mDd8NChWJDwSaW5vW7NabW1ub1uzWm7sPCv7tFxD+7RAQARMQFwETCg8AAAMAGAAYBJgEmAAPAJYApgAAADIeAhQOAiIuAjQ+ASUOAwcGJgcOAQcGFgcOAQcGFgcUFgcyHgEXHgIXHgI3Fg4BFx4CFxQGFBcWNz4CNy4BJy4BJyIOAgcGJyY2NS4BJzYuAQYHBicmNzY3HgIXHgMfAT4CJyY+ATc+AzcmNzIWMjY3LgMnND4CJiceAT8BNi4CJwYHFB4BFS4CJz4BNxYyPgEB5OjVm1xcm9Xo1ZtcXJsBZA8rHDoKDz0PFD8DAxMBAzEFCRwGIgEMFhkHECIvCxU/OR0HFBkDDRQjEwcFaHUeISQDDTAMD0UREi4oLBAzDwQBBikEAQMLGhIXExMLBhAGKBsGBxYVEwYFAgsFAwMNFwQGCQcYFgYQCCARFwkKKiFBCwQCAQMDHzcLDAUdLDgNEiEQEgg/KhADGgMKEgoRBJhcm9Xo1ZtcXJvV6NWbEQwRBwkCAwYFBycPCxcHInIWInYcCUcYChQECA4QBAkuHgQPJioRFRscBAcSCgwCch0kPiAIAQcHEAsBAgsLIxcBMQENCQIPHxkCFBkdHB4QBgEBBwoMGBENBAMMJSAQEhYXDQ4qFBkKEhIDCQsXJxQiBgEOCQwHAQ0DBAUcJAwSCwRnETIoAwEJCwsLJQcKDBEAAAAAAQAAAAIErwSFABYAAAE2FwUXNxYGBw4BJwEGIi8BJjQ3ASY2AvSkjv79kfsGUE08hjv9rA8rD28PDwJYIk8EhVxliuh+WYcrIgsW/awQEG4PKxACV2XJAAYAAABgBLAErAAPABMAIwAnADcAOwAAEyEyFh0BFAYjISImPQE0NgUjFTMFITIWHQEUBiMhIiY9ATQ2BSEVIQUhMhYdARQGIyEiJj0BNDYFIRUhZAPoKTs7KfwYKTs7BBHIyPwYA+gpOzsp/BgpOzsEEf4MAfT8GAPoKTs7KfwYKTs7BBH+1AEsBKw7KWQpOzspZCk7ZGTIOylkKTs7KWQpO2RkyDspZCk7OylkKTtkZAAAAAIAZAAABEwEsAALABEAABMhMhYUBiMhIiY0NgERBxEBIZYDhBUdHRX8fBUdHQI7yP6iA4QEsB0qHR0qHf1E/tTIAfQB9AAAAAMAAABkBLAEsAAXABsAJQAAATMyFh0BITIWFREhNSMVIRE0NjMhNTQ2FxUzNQEVFAYjISImPQEB9MgpOwEsKTv+DMj+DDspASw7KcgB9Dsp/BgpOwSwOylkOyn+cGRkAZApO2QpO2RkZP1EyCk7OynIAAAABAAAAAAEsASwABUAKwBBAFcAABMhMhYPARcWFA8BBiIvAQcGJjURNDYpATIWFREUBi8BBwYiLwEmND8BJyY2ARcWFA8BFxYGIyEiJjURNDYfATc2MgU3NhYVERQGIyEiJj8BJyY0PwE2MhcyASwVCA5exwcHaggUCMdeDhUdAzUBLBUdFQ5exwgUCGoHB8deDgj+L2oHB8deDggV/tQVHRUOXscIFALLXg4VHRX+1BUIDl7HBwdqCBQIBLAVDl7HCBQIagcHx14OCBUBLBUdHRX+1BUIDl7HBwdqCBQIx14OFf0maggUCMdeDhUdFQEsFQgOXscHzl4OCBX+1BUdFQ5exwgUCGoHBwAAAAYAAAAABKgEqAAPABsAIwA7AEMASwAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JiQyFhQGIiY0JDIWFAYjIicHFhUUBiImNTQ2PwImNTQEMhYUBiImNCQyFhQGIiY0Advy3Z9fX5/d8t2gXl6gAcbgv29vv+C/b2/+LS0gIC0gAUwtICAWDg83ETNIMykfegEJ/octICAtIAIdLSAgLSAEqF+f3fLdoF5eoN3y3Z9Xb7/gv29vv+C/BiAtISEtICAtIQqRFxwkMzMkIDEFfgEODhekIC0gIC0gIC0gIC0AAf/YAFoEuQS8AFsAACUBNjc2JicmIyIOAwcABw4EFx4BMzI3ATYnLgEjIgcGBwEOASY0NwA3PgEzMhceARcWBgcOBgcGIyImJyY2NwE2NzYzMhceARcWBgcBDgEnLgECIgHVWwgHdl8WGSJBMD8hIP6IDx4eLRMNBQlZN0ozAiQkEAcdEhoYDRr+qw8pHA4BRyIjQS4ODyw9DQ4YIwwod26La1YOOEBGdiIwGkQB/0coW2tQSE5nDxE4Qv4eDyoQEAOtAdZbZWKbEQQUGjIhH/6JDxsdNSg3HT5CMwIkJCcQFBcMGv6uDwEcKQ4BTSIjIQEINykvYyMLKnhuiWZMBxtAOU6+RAH/SBg3ISSGV121Qv4kDwIPDyYAAAACAGQAWASvBEQAGQBEAAABPgIeAhUUDgMHLgQ1ND4CHgEFIg4DIi4DIyIGFRQeAhcWFx4EMj4DNzY3PgQ1NCYCiTB7eHVYNkN5hKg+PqeFeEM4WnZ4eQEjIT8yLSohJyktPyJDbxtBMjMPBw86KzEhDSIzKUAMBAgrKT8dF2oDtURIBS1TdkA5eYB/slVVsn+AeTlAdlMtBUgtJjY1JiY1NiZvTRc4SjQxDwcOPCouGBgwKEALBAkpKkQqMhNPbQACADn/8gR3BL4AFwAuAAAAMh8BFhUUBg8BJi8BNycBFwcvASY0NwEDNxYfARYUBwEGIi8BJjQ/ARYfAQcXAQKru0KNQjgiHR8uEl/3/nvUaRONQkIBGxJpCgmNQkL+5UK6Qo1CQjcdLhJf9wGFBL5CjUJeKmsiHTUuEl/4/nvUahKNQrpCARv+RmkICY1CukL+5UJCjUK7Qjc3LxFf+AGFAAAAAAMAyAAAA+gEsAARABUAHQAAADIeAhURFAYjISImNRE0PgEHESERACIGFBYyNjQCBqqaZDo7Kf2oKTs8Zj4CWP7/Vj09Vj0EsB4uMhX8Ryk7OykDuRUzLar9RAK8/RY9Vj09VgABAAAAAASwBLAAFgAACQEWFAYiLwEBEScBBRMBJyEBJyY0NjIDhgEbDx0qDiT+6dT+zP7oywEz0gEsAQsjDx0qBKH+5g8qHQ8j/vX+1NL+zcsBGAE01AEXJA4qHQAAAAADAScAEQQJBOAAMgBAAEsAAAEVHgQXIy4DJxEXHgQVFAYHFSM1JicuASczHgEXEScuBDU0PgI3NRkBDgMVFB4DFxYXET4ENC4CArwmRVI8LAKfBA0dMydAIjxQNyiym2SWVygZA4sFV0obLkJOMCAyVWg6HSoqFQ4TJhkZCWgWKTEiGBkzNwTgTgUTLD9pQiQuLBsH/s0NBxMtPGQ+i6oMTU8QVyhrVk1iEAFPCA4ZLzlYNkZwSCoGTf4SARIEDh02Jh0rGRQIBgPQ/soCCRYgNEM0JRkAAAABAGQAZgOUBK0ASgAAATIeARUjNC4CIyIGBwYVFB4BFxYXMxUjFgYHBgc+ATM2FjMyNxcOAyMiLgEHDgEPASc+BTc+AScjNTMmJy4CPgE3NgIxVJlemSc8OxolVBQpGxoYBgPxxQgVFS02ImIWIIwiUzUyHzY4HCAXanQmJ1YYFzcEGAcTDBEJMAwk3aYXFQcKAg4tJGEErVCLTig/IhIdFSw5GkowKgkFZDKCHj4yCg8BIh6TExcIASIfBAMaDAuRAxAFDQsRCjePR2QvORQrREFMIVgAAAACABn//wSXBLAADwAfAAABMzIWDwEGIi8BJjY7AREzBRcWBisBESMRIyImPwE2MgGQlhUIDuYOKg7mDggVlsgCF+YOCBWWyJYVCA7mDioBLBYO+g8P+g4WA4QQ+Q4V/HwDhBUO+Q8AAAQAGf//A+gEsAAHABcAGwAlAAABIzUjFSMRIQEzMhYPAQYiLwEmNjsBETMFFTM1EwczFSE1NyM1IQPoZGRkASz9qJYVCA7mDioO5g4IFZbIAZFkY8jI/tTIyAEsArxkZAH0/HwWDvoPD/oOFgOEZMjI/RL6ZJb6ZAAAAAAEABn//wPoBLAADwAZACEAJQAAATMyFg8BBiIvASY2OwERMwUHMxUhNTcjNSERIzUjFSMRIQcVMzUBkJYVCA7mDioO5g4IFZbIAljIyP7UyMgBLGRkZAEsx2QBLBYO+g8P+g4WA4SW+mSW+mT7UGRkAfRkyMgAAAAEABn//wRMBLAADwAVABsAHwAAATMyFg8BBiIvASY2OwERMwEjESM1MxMjNSMRIQcVMzUBkJYVCA7mDioO5g4IFZbIAlhkZMhkZMgBLMdkASwWDvoPD/oOFgOE/gwBkGT7UGQBkGTIyAAAAAAEABn//wRMBLAADwAVABkAHwAAATMyFg8BBiIvASY2OwERMwEjNSMRIQcVMzUDIxEjNTMBkJYVCA7mDioO5g4IFZbIArxkyAEsx2QBZGTIASwWDvoPD/oOFgOE/gxkAZBkyMj7tAGQZAAAAAAFABn//wSwBLAADwATABcAGwAfAAABMzIWDwEGIi8BJjY7AREzBSM1MxMhNSETITUhEyE1IQGQlhUIDuYOKg7mDggVlsgB9MjIZP7UASxk/nABkGT+DAH0ASwWDvoPD/oOFgOEyMj+DMj+DMj+DMgABQAZ//8EsASwAA8AEwAXABsAHwAAATMyFg8BBiIvASY2OwERMwUhNSEDITUhAyE1IQMjNTMBkJYVCA7mDioO5g4IFZbIAyD+DAH0ZP5wAZBk/tQBLGTIyAEsFg76Dw/6DhYDhMjI/gzI/gzI/gzIAAIAAAAABEwETAAPAB8AAAEhMhYVERQGIyEiJjURNDYFISIGFREUFjMhMjY1ETQmAV4BkKK8u6P+cKW5uQJn/gwpOzspAfQpOzsETLuj/nClubmlAZClucg7Kf4MKTs7KQH0KTsAAAAAAwAAAAAETARMAA8AHwArAAABITIWFREUBiMhIiY1ETQ2BSEiBhURFBYzITI2NRE0JgUXFhQPAQYmNRE0NgFeAZClubml/nCju7wCZP4MKTs7KQH0KTs7/m/9ERH9EBgYBEy5pf5wpbm5pQGQo7vIOyn+DCk7OykB9Ck7gr4MJAy+DAsVAZAVCwAAAAADAAAAAARMBEwADwAfACsAAAEhMhYVERQGIyEiJjURNDYFISIGFREUFjMhMjY1ETQmBSEyFg8BBiIvASY2AV4BkKO7uaX+cKW5uQJn/gwpOzspAfQpOzv+FQGQFQsMvgwkDL4MCwRMvKL+cKW5uaUBkKO7yDsp/gwpOzspAfQpO8gYEP0REf0QGAAAAAMAAAAABEwETAAPAB8AKwAAASEyFhURFAYjISImNRE0NgUhIgYVERQWMyEyNjURNCYFFxYGIyEiJj8BNjIBXgGQpbm5pf5wo7u5Amf+DCk7OykB9Ck7O/77vgwLFf5wFQsMvgwkBEy5pf5wo7u8ogGQpbnIOyn+DCk7OykB9Ck7z/0QGBgQ/REAAAAAAgAAAAAFFARMAB8ANQAAASEyFhURFAYjISImPQE0NjMhMjY1ETQmIyEiJj0BNDYHARYUBwEGJj0BIyImPQE0NjsBNTQ2AiYBkKW5uaX+cBUdHRUBwik7Oyn+PhUdHb8BRBAQ/rwQFvoVHR0V+hYETLml/nCluR0VZBUdOykB9Ck7HRVkFR3p/uQOJg7+5A4KFZYdFcgVHZYVCgAAAQDZAAID1wSeACMAAAEXFgcGAgclMhYHIggBBwYrAScmNz4BPwEhIicmNzYANjc2MwMZCQgDA5gCASwYEQ4B/vf+8wQMDgkJCQUCUCcn/tIXCAoQSwENuwUJEASeCQoRC/5TBwEjEv7K/sUFDwgLFQnlbm4TFRRWAS/TBhAAAAACAAAAAAT+BEwAHwA1AAABITIWHQEUBiMhIgYVERQWMyEyFh0BFAYjISImNRE0NgUBFhQHAQYmPQEjIiY9ATQ2OwE1NDYBXgGQFR0dFf4+KTs7KQHCFR0dFf5wpbm5AvEBRBAQ/rwQFvoVHR0V+hYETB0VZBUdOyn+DCk7HRVkFR25pQGQpbnp/uQOJg7+5A4KFZYdFcgVHZYVCgACAAAAAASwBLAAFQAxAAABITIWFREUBi8BAQYiLwEmNDcBJyY2ASMiBhURFBYzITI2PQE3ERQGIyEiJjURNDYzIQLuAZAVHRUObf7IDykPjQ8PAThtDgj+75wpOzspAfQpO8i7o/5wpbm5pQEsBLAdFf5wFQgObf7IDw+NDykPAThtDhX+1Dsp/gwpOzsplMj+1qW5uaUBkKW5AAADAA4ADgSiBKIADwAbACMAAAAyHgIUDgIiLgI0PgEEIg4BFB4BMj4BNCYEMhYUBiImNAHh7tmdXV2d2e7ZnV1dnQHD5sJxccLmwnFx/nugcnKgcgSiXZ3Z7tmdXV2d2e7ZnUdxwubCcXHC5sJzcqBycqAAAAMAAAAABEwEsAAVAB8AIwAAATMyFhURMzIWBwEGIicBJjY7ARE0NgEhMhYdASE1NDYFFTM1AcLIFR31FAoO/oEOJw3+hQ0JFfod/oUD6BUd+7QdA2dkBLAdFf6iFg/+Vg8PAaoPFgFeFR38fB0V+voVHWQyMgAAAAMAAAAABEwErAAVAB8AIwAACQEWBisBFRQGKwEiJj0BIyImNwE+AQEhMhYdASE1NDYFFTM1AkcBeg4KFfQiFsgUGPoUCw4Bfw4n/fkD6BUd+7QdA2dkBJ7+TQ8g+hQeHRX6IQ8BrxAC/H8dFfr6FR1kMjIAAwAAAAAETARLABQAHgAiAAAJATYyHwEWFAcBBiInASY0PwE2MhcDITIWHQEhNTQ2BRUzNQGMAXEHFQeLBwf98wcVB/7cBweLCBUH1APoFR37tB0DZ2QC0wFxBweLCBUH/fMICAEjCBQIiwcH/dIdFfr6FR1kMjIABAAAAAAETASbAAkAGQAjACcAABM3NjIfAQcnJjQFNzYWFQMOASMFIiY/ASc3ASEyFh0BITU0NgUVMzWHjg4qDk3UTQ4CFtIOFQIBHRX9qxUIDtCa1P49A+gVHfu0HQNnZAP/jg4OTdRMDyqa0g4IFf2pFB4BFQ7Qm9T9Oh0V+voVHWQyMgAAAAQAAAAABEwEsAAPABkAIwAnAAABBR4BFRMUBi8BByc3JyY2EwcGIi8BJjQ/AQEhMhYdASE1NDYFFTM1AV4CVxQeARUO0JvUm9IOCMNMDyoOjg4OTf76A+gVHfu0HQNnZASwAgEdFf2rFQgO0JrUmtIOFf1QTQ4Ojg4qDk3+WB0V+voVHWQyMgACAAT/7ASwBK8ABQAIAAAlCQERIQkBFQEEsP4d/sb+cQSs/TMCq2cBFP5xAacDHPz55gO5AAAAAAIAAABkBEwEsAAVABkAAAERFAYrAREhESMiJjURNDY7AREhETMHIzUzBEwdFZb9RJYVHR0V+gH0ZMhkZAPo/K4VHQGQ/nAdFQPoFB7+1AEsyMgAAAMAAABFBN0EsAAWABoALwAAAQcBJyYiDwEhESMiJjURNDY7AREhETMHIzUzARcWFAcBBiIvASY0PwE2Mh8BATYyBEwC/tVfCRkJlf7IlhUdHRX6AfRkyGRkAbBqBwf+XAgUCMoICGoHFQdPASkHFQPolf7VXwkJk/5wHRUD6BQe/tQBLMjI/c5qBxUH/lsHB8sHFQdqCAhPASkHAAMAAAANBQcEsAAWABoAPgAAAREHJy4BBwEhESMiJjURNDY7AREhETMHIzUzARcWFA8BFxYUDwEGIi8BBwYiLwEmND8BJyY0PwE2Mh8BNzYyBExnhg8lEP72/reWFR0dFfoB9GTIZGQB9kYPD4ODDw9GDykPg4MPKQ9GDw+Dgw8PRg8pD4ODDykD6P7zZ4YPAw7+9v5wHRUD6BQe/tQBLMjI/YxGDykPg4MPKQ9GDw+Dgw8PRg8pD4ODDykPRg8Pg4MPAAADAAAAFQSXBLAAFQAZAC8AAAERISIGHQEhESMiJjURNDY7AREhETMHIzUzEzMyFh0BMzIWDwEGIi8BJjY7ATU0NgRM/qIVHf4MlhUdHRX6AfRkyGRklmQVHZYVCA7mDioO5g4IFZYdA+j+1B0Vlv5wHRUD6BQe/tQBLMjI/agdFfoVDuYODuYOFfoVHQAAAAADAAAAAASXBLAAFQAZAC8AAAERJyYiBwEhESMiJjURNDY7AREhETMHIzUzExcWBisBFRQGKwEiJj0BIyImPwE2MgRMpQ4qDv75/m6WFR0dFfoB9GTIZGTr5g4IFZYdFWQVHZYVCA7mDioD6P5wpQ8P/vf+cB0VA+gUHv7UASzIyP2F5Q8V+hQeHhT6FQ/lDwADAAAAyASwBEwACQATABcAABMhMhYdASE1NDYBERQGIyEiJjURExUhNTIETBUd+1AdBJMdFfu0FR1kAZAETB0VlpYVHf7U/doVHR0VAib+1MjIAAAGAAMAfQStBJcADwAZAB0ALQAxADsAAAEXFhQPAQYmPQEhNSE1NDYBIyImPQE0NjsBFyM1MwE3NhYdASEVIRUUBi8BJjQFIzU7AjIWHQEUBisBA6f4Dg74DhX+cAGQFf0vMhUdHRUyyGRk/oL3DhUBkP5wFQ73DwOBZGRkMxQdHRQzBI3mDioO5g4IFZbIlhUI/oUdFWQVHcjI/cvmDggVlsiWFQgO5g4qecgdFWQVHQAAAAACAGQAAASwBLAAFgBRAAABJTYWFREUBisBIiY1ES4ENRE0NiUyFh8BERQOAg8BERQGKwEiJjURLgQ1ETQ+AzMyFh8BETMRPAE+AjMyFh8BETMRND4DA14BFBklHRXIFR0EDiIaFiX+4RYZAgEVHR0LCh0VyBUdBA4iGhYBBwoTDRQZAgNkBQkVDxcZAQFkAQUJFQQxdBIUH/uuFR0dFQGNAQgbHzUeAWcfRJEZDA3+Phw/MSkLC/5BFR0dFQG/BA8uLkAcAcICBxENCxkMDf6iAV4CBxENCxkMDf6iAV4CBxENCwABAGQAAASwBEwAMwAAARUiDgMVERQWHwEVITUyNjURIREUFjMVITUyPgM1ETQmLwE1IRUiBhURIRE0JiM1BLAEDiIaFjIZGf5wSxn+DBlL/nAEDiIaFjIZGQGQSxkB9BlLBEw4AQUKFA78iBYZAQI4OA0lAYr+diUNODgBBQoUDgN4FhkBAjg4DSX+dgGKJQ04AAAABgAAAAAETARMAAwAHAAgACQAKAA0AAABITIWHQEjBTUnITchBSEyFhURFAYjISImNRE0NhcVITUBBTUlBRUhNQUVFAYjIQchJyE3MwKjAXcVHWn+2cj+cGQBd/4lASwpOzsp/tQpOzspASwCvP5wAZD8GAEsArwdFf6JZP6JZAGQyGkD6B0VlmJiyGTIOyn+DCk7OykB9Ck7ZMjI/veFo4XGyMhm+BUdZGTIAAEAEAAQBJ8EnwAmAAATNzYWHwEWBg8BHgEXNz4BHwEeAQ8BBiIuBicuBTcRohEuDosOBhF3ZvyNdxEzE8ATBxGjAw0uMUxPZWZ4O0p3RjITCwED76IRBhPCFDERdo78ZXYRBA6IDi8RogEECBUgNUNjO0qZfHNVQBAAAAACAAAAAASwBEwAIwBBAAAAMh4EHwEVFAYvAS4BPQEmIAcVFAYPAQYmPQE+BRIyHgIfARUBHgEdARQGIyEiJj0BNDY3ATU0PgIB/LimdWQ/LAkJHRTKFB2N/sKNHRTKFB0DDTE7ZnTKcFImFgEBAW0OFR0V+7QVHRUOAW0CFiYETBUhKCgiCgrIFRgDIgMiFZIYGJIVIgMiAxgVyAQNJyQrIP7kExwcCgoy/tEPMhTUFR0dFdQUMg8BLzIEDSEZAAADAAAAAASwBLAADQAdACcAAAEHIScRMxUzNTMVMzUzASEyFhQGKwEXITcjIiY0NgMhMhYdASE1NDYETMj9qMjIyMjIyPyuArwVHR0VDIn8SokMFR0dswRMFR37UB0CvMjIAfTIyMjI/OAdKh1kZB0qHf7UHRUyMhUdAAAAAwBkAAAEsARMAAkAEwAdAAABIyIGFREhETQmASMiBhURIRE0JgEhETQ2OwEyFhUCvGQpOwEsOwFnZCk7ASw7/Rv+1DspZCk7BEw7KfwYA+gpO/7UOyn9RAK8KTv84AGQKTs7KQAAAAAF/5wAAASwBEwADwATAB8AJQApAAATITIWFREUBiMhIiY1ETQ2FxEhEQUjFTMRITUzNSMRIQURByMRMwcRMxHIArx8sLB8/UR8sLAYA4T+DMjI/tTIyAEsAZBkyMhkZARMsHz+DHywsHwB9HywyP1EArzIZP7UZGQBLGT+1GQB9GT+1AEsAAAABf+cAAAEsARMAA8AEwAfACUAKQAAEyEyFhURFAYjISImNRE0NhcRIREBIzUjFSMRMxUzNTMFEQcjETMHETMRyAK8fLCwfP1EfLCwGAOE/gxkZGRkZGQBkGTIyGRkBEywfP4MfLCwfAH0fLDI/UQCvP2oyMgB9MjIZP7UZAH0ZP7UASwABP+cAAAEsARMAA8AEwAbACMAABMhMhYVERQGIyEiJjURNDYXESERBSMRMxUhESEFIxEzFSERIcgCvHywsHz9RHywsBgDhP4MyMj+1AEsAZDIyP7UASwETLB8/gx8sLB8AfR8sMj9RAK8yP7UZAH0ZP7UZAH0AAAABP+cAAAEsARMAA8AEwAWABkAABMhMhYVERQGIyEiJjURNDYXESERAS0BDQERyAK8fLCwfP1EfLCwGAOE/gz+1AEsAZD+1ARMsHz+DHywsHwB9HywyP1EArz+DJaWlpYBLAAAAAX/nAAABLAETAAPABMAFwAgACkAABMhMhYVERQGIyEiJjURNDYXESERAyERIQcjIgYVFBY7AQERMzI2NTQmI8gCvHywsHz9RHywsBgDhGT9RAK8ZIImOTYpgv4Mgik2OSYETLB8/gx8sLB8AfR8sMj9RAK8/agB9GRWQUFUASz+1FRBQVYAAAAF/5wAAASwBEwADwATAB8AJQApAAATITIWFREUBiMhIiY1ETQ2FxEhEQUjFTMRITUzNSMRIQEjESM1MwMjNTPIArx8sLB8/UR8sLAYA4T+DMjI/tTIyAEsAZBkZMjIZGQETLB8/gx8sLB8AfR8sMj9RAK8yGT+1GRkASz+DAGQZP4MZAAG/5wAAASwBEwADwATABkAHwAjACcAABMhMhYVERQGIyEiJjURNDYXESERBTMRIREzASMRIzUzBRUzNQEjNTPIArx8sLB8/UR8sLAYA4T9RMj+1GQCWGRkyP2oZAEsZGQETLB8/gx8sLB8AfR8sMj9RAK8yP5wAfT+DAGQZMjIyP7UZAAF/5wAAASwBEwADwATABwAIgAmAAATITIWFREUBiMhIiY1ETQ2FxEhEQEHIzU3NSM1IQEjESM1MwMjNTPIArx8sLB8/UR8sLAYA4T+DMdkx8gBLAGQZGTIx2RkBEywfP4MfLCwfAH0fLDI/UQCvP5wyDLIlmT+DAGQZP4MZAAAAAMACQAJBKcEpwAPABsAJQAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JgchFSEVISc1NyEB4PDbnl5entvw255eXp4BxeTCcXHC5MJxcWz+1AEs/tRkZAEsBKdentvw255eXp7b8NueTHHC5MJxccLkwtDIZGTIZAAAAAAEAAkACQSnBKcADwAbACcAKwAAADIeAhQOAiIuAjQ+AQQiDgEUHgEyPgE0JgcVBxcVIycjFSMRIQcVMzUB4PDbnl5entvw255eXp4BxeTCcXHC5MJxcWwyZGRklmQBLMjIBKdentvw255eXp7b8NueTHHC5MJxccLkwtBkMmQyZGQBkGRkZAAAAv/y/50EwgRBACAANgAAATIWFzYzMhYUBisBNTQmIyEiBh0BIyImNTQ2NyY1ND4BEzMyFhURMzIWDwEGIi8BJjY7ARE0NgH3brUsLC54qqp4gB0V/tQVHd5QcFZBAmKqepYKD4kVCg3fDSYN3w0KFYkPBEF3YQ6t8a36FR0dFfpzT0VrDhMSZKpi/bMPCv7tFxD0EBD0EBcBEwoPAAAAAAL/8v+cBMMEQQAcADMAAAEyFhc2MzIWFxQGBwEmIgcBIyImNTQ2NyY1ND4BExcWBisBERQGKwEiJjURIyImNzY3NjIB9m62LCsueaoBeFr+hg0lDf6DCU9xVkECYqnm3w0KFYkPCpYKD4kVCg3HGBMZBEF3YQ+teGOkHAFoEBD+k3NPRWsOExNkqWP9kuQQF/7tCg8PCgETFxDMGBMAAAABAGQAAARMBG0AGAAAJTUhATMBMwkBMwEzASEVIyIGHQEhNTQmIwK8AZD+8qr+8qr+1P7Uqv7yqv7yAZAyFR0BkB0VZGQBLAEsAU3+s/7U/tRkHRUyMhUdAAAAAAEAeQAABDcEmwAvAAABMhYXHgEVFAYHFhUUBiMiJxUyFh0BITU0NjM1BiMiJjU0Ny4BNTQ2MzIXNCY1NDYCWF6TGll7OzIJaUo3LRUd/tQdFS03SmkELzlpSgUSAqMEm3FZBoNaPWcfHRpKaR77HRUyMhUd+x5pShIUFVg1SmkCAhAFdKMAAAAGACcAFASJBJwAEQAqAEIASgBiAHsAAAEWEgIHDgEiJicmAhI3PgEyFgUiBw4BBwYWHwEWMzI3Njc2Nz4BLwEmJyYXIgcOAQcGFh8BFjMyNz4BNz4BLwEmJyYWJiIGFBYyNjciBw4BBw4BHwEWFxYzMjc+ATc2Ji8BJhciBwYHBgcOAR8BFhcWMzI3PgE3NiYvASYD8m9PT29T2dzZU29PT29T2dzZ/j0EBHmxIgQNDCQDBBcGG0dGYAsNAwkDCwccBAVQdRgEDA0iBAQWBhJROQwMAwkDCwf5Y4xjY4xjVhYGElE6CwwDCQMLBwgEBVB1GAQNDCIEjRcGG0dGYAsNAwkDCwcIBAR5sSIEDQwkAwPyb/7V/tVvU1dXU28BKwErb1NXVxwBIrF5DBYDCQEWYEZHGwMVDCMNBgSRAhh1UA0WAwkBFTpREgMVCyMMBwT6Y2OMY2MVFTpREQQVCyMMBwQCGHVQDRYDCQEkFmBGRxsDFQwjDQYEASKxeQwWAwkBAAAABQBkAAAD6ASwAAwADwAWABwAIgAAASERIzUhFSERNDYzIQEjNQMzByczNTMDISImNREFFRQGKwECvAEstP6s/oQPCgI/ASzIZKLU1KJktP51Cg8DhA8KwwMg/oTIyALzCg/+1Mj84NTUyP4MDwoBi8jDCg8AAAAABQBkAAAD6ASwAAkADAATABoAIQAAASERCQERNDYzIQEjNRMjFSM1IzcDISImPQEpARUUBisBNQK8ASz+ov3aDwoCPwEsyD6iZKLUqv6dCg8BfAIIDwqbAyD9+AFe/doERwoP/tTI/HzIyNT+ZA8KNzcKD1AAAAAAAwAAAAAEsAP0AAgAGQAfAAABIxUzFyERIzcFMzIeAhUhFSEDETM0PgIBMwMhASEEiqJkZP7UotT9EsgbGiEOASz9qMhkDiEaAnPw8PzgASwB9AMgyGQBLNTUBBErJGT+ogHCJCsRBP5w/nAB9AAAAAMAAAAABEwETAAZADIAOQAAATMyFh0BMzIWHQEUBiMhIiY9ATQ2OwE1NDYFNTIWFREUBiMhIic3ARE0NjMVFBYzITI2AQc1IzUzNQKKZBUdMhUdHRX+1BUdHRUyHQFzKTs7Kf2oARP2/ro7KVg+ASw+WP201MjIBEwdFTIdFWQVHR0VZBUdMhUd+pY7KfzgKTsE9gFGAUQpO5Y+WFj95tSiZKIAAwBkAAAEvARMABkANgA9AAABMzIWHQEzMhYdARQGIyEiJj0BNDY7ATU0NgU1MhYVESMRMxQOAiMhIiY1ETQ2MxUUFjMhMjYBBzUjNTM1AcJkFR0yFR0dFf7UFR0dFTIdAXMpO8jIDiEaG/2oKTs7KVg+ASw+WAGc1MjIBEwdFTIdFWQVHR0VZBUdMhUd+pY7Kf4M/tQkKxEEOykDICk7lj5YWP3m1KJkogAAAAP/ogAABRYE1AALABsAHwAACQEWBiMhIiY3ATYyEyMiBhcTHgE7ATI2NxM2JgMVMzUCkgJ9FyAs+wQsIBcCfRZARNAUGAQ6BCMUNhQjBDoEGODIBK37sCY3NyYEUCf+TB0U/tIUHR0UAS4UHf4MZGQAAAAACQAAAAAETARMAA8AHwAvAD8ATwBfAG8AfwCPAAABMzIWHQEUBisBIiY9ATQ2EzMyFh0BFAYrASImPQE0NiEzMhYdARQGKwEiJj0BNDYBMzIWHQEUBisBIiY9ATQ2ITMyFh0BFAYrASImPQE0NiEzMhYdARQGKwEiJj0BNDYBMzIWHQEUBisBIiY9ATQ2ITMyFh0BFAYrASImPQE0NiEzMhYdARQGKwEiJj0BNDYBqfoKDw8K+goPDwr6Cg8PCvoKDw8BmvoKDw8K+goPD/zq+goPDwr6Cg8PAZr6Cg8PCvoKDw8BmvoKDw8K+goPD/zq+goPDwr6Cg8PAZr6Cg8PCvoKDw8BmvoKDw8K+goPDwRMDwqWCg8PCpYKD/7UDwqWCg8PCpYKDw8KlgoPDwqWCg/+1A8KlgoPDwqWCg8PCpYKDw8KlgoPDwqWCg8PCpYKD/7UDwqWCg8PCpYKDw8KlgoPDwqWCg8PCpYKDw8KlgoPAAAAAwAAAAAEsAUUABkAKQAzAAABMxUjFSEyFg8BBgchJi8BJjYzITUjNTM1MwEhMhYUBisBFyE3IyImNDYDITIWHQEhNTQ2ArxkZAFePjEcQiko/PwoKUIcMT4BXmRkyP4+ArwVHR0VDIn8SooNFR0dswRMFR37UB0EsMhkTzeEUzMzU4Q3T2TIZPx8HSodZGQdKh3+1B0VMjIVHQAABAAAAAAEsAUUAAUAGQArADUAAAAyFhUjNAchFhUUByEyFg8BIScmNjMhJjU0AyEyFhQGKwEVBSElNSMiJjQ2AyEyFh0BITU0NgIwUDnCPAE6EgMBSCkHIq/9WrIiCikBSAOvArwVHR0VlgET/EoBE5YVHR2zBEwVHftQHQUUOykpjSUmCBEhFpGRFiERCCb+lR0qHcjIyMgdKh39qB0VMjIVHQAEAAAAAASwBJ0ABwAUACQALgAAADIWFAYiJjQTMzIWFRQXITY1NDYzASEyFhQGKwEXITcjIiY0NgMhMhYdASE1NDYCDZZqapZqty4iKyf+vCcrI/7NArwVHR0VDYr8SokMFR0dswRMFR37UB0EnWqWamqW/us5Okxra0w6Of5yHSodZGQdKh3+1B0VMjIVHQAEAAAAAASwBRQADwAcACwANgAAATIeARUUBiImNTQ3FzcnNhMzMhYVFBchNjU0NjMBITIWFAYrARchNyMiJjQ2AyEyFh0BITU0NgJYL1szb5xvIpBvoyIfLiIrJ/68Jysj/s0CvBUdHRUNivxKiQwVHR2zBEwVHftQHQUUa4s2Tm9vTj5Rj2+jGv4KOTpMa2tMOjn+ch0qHWRkHSod/tQdFTIyFR0AAAADAAAAAASwBRIAEgAiACwAAAEFFSEUHgMXIS4BNTQ+AjcBITIWFAYrARchNyMiJjQ2AyEyFh0BITU0NgJYASz+1CU/P00T/e48PUJtj0r+ogK8FR0dFQ2K/EqJDBUdHbMETBUd+1AdBLChizlmUT9IGVO9VFShdksE/H4dKh1kZB0qHf7UHRUyMhUdAAIAyAAAA+gFFAAPACkAAAAyFh0BHgEdASE1NDY3NTQDITIWFyMVMxUjFTMVIxUzFAYjISImNRE0NgIvUjsuNv5wNi5kAZA2XBqsyMjIyMh1U/5wU3V1BRQ7KU4aXDYyMjZcGk4p/kc2LmRkZGRkU3V1UwGQU3UAAAMAZP//BEwETAAPAC8AMwAAEyEyFhURFAYjISImNRE0NgMhMhYdARQGIyEXFhQGIi8BIQcGIiY0PwEhIiY9ATQ2BQchJ5YDhBUdHRX8fBUdHQQDtgoPDwr+5eANGiUNWP30Vw0mGg3g/t8KDw8BqmQBRGQETB0V/gwVHR0VAfQVHf1EDwoyCg/gDSUbDVhYDRslDeAPCjIKD2RkZAAAAAAEAAAAAASwBEwAGQAjAC0ANwAAEyEyFh0BIzQmKwEiBhUjNCYrASIGFSM1NDYDITIWFREhETQ2ExUUBisBIiY9ASEVFAYrASImPQHIAyBTdWQ7KfopO2Q7KfopO2R1EQPoKTv7UDvxHRVkFR0D6B0VZBUdBEx1U8gpOzspKTs7KchTdf4MOyn+1AEsKTv+DDIVHR0VMjIVHR0VMgADAAEAAASpBKwADQARABsAAAkBFhQPASEBJjQ3ATYyCQMDITIWHQEhNTQ2AeACqh8fg/4f/fsgIAEnH1n+rAFWAS/+q6IDIBUd/HwdBI39VR9ZH4MCBh9ZHwEoH/5u/qoBMAFV/BsdFTIyFR0AAAAAAgCPAAAEIQSwABcALwAAAQMuASMhIgYHAwYWMyEVFBYyNj0BMzI2AyE1NDY7ATU0NjsBETMRMzIWHQEzMhYVBCG9CCcV/nAVJwi9CBMVAnEdKh19FROo/a0dFTIdFTDILxUdMhUdAocB+hMcHBP+BhMclhUdHRWWHP2MMhUdMhUdASz+1B0VMh0VAAAEAAAAAASwBLAADQAQAB8AIgAAASERFAYjIREBNTQ2MyEBIzUBIREUBiMhIiY1ETQ2MyEBIzUDhAEsDwr+if7UDwoBdwEsyP2oASwPCv12Cg8PCgF3ASzIAyD9wQoPAk8BLFQKD/7UyP4M/cEKDw8KA7YKD/7UyAAC/5wAZAUUBEcARgBWAAABMzIeAhcWFxY2NzYnJjc+ARYXFgcOASsBDgEPAQ4BKwEiJj8BBisBIicHDgErASImPwEmLwEuAT0BNDY7ATY3JyY2OwE2BSMiBh0BFBY7ATI2PQE0JgHkw0uOakkMEhEfQwoKGRMKBQ8XDCkCA1Y9Pgc4HCcDIhVkFRgDDDEqwxgpCwMiFWQVGAMaVCyfExwdFXwLLW8QBxXLdAFF+goPDwr6Cg8PBEdBa4pJDgYKISAiJRsQCAYIDCw9P1c3fCbqFB0dFEYOCEAUHR0UnUplNQcmFTIVHVdPXw4TZV8PCjIKDw8KMgoPAAb/nP/mBRQEfgAJACQANAA8AFIAYgAAASU2Fh8BFgYPASUzMhYfASEyFh0BFAYHBQYmJyYjISImPQE0NhcjIgYdARQ7ATI2NTQmJyYEIgYUFjI2NAE3PgEeARceAT8BFxYGDwEGJi8BJjYlBwYfAR4BPwE2Jy4BJy4BAoEBpxMuDiAOAxCL/CtqQ0geZgM3FR0cE/0fFyIJKjr+1D5YWLlQExIqhhALIAsSAYBALS1ALf4PmBIgHhMQHC0aPzANITNQL3wpgigJASlmHyElDR0RPRMFAhQHCxADhPcICxAmDyoNeMgiNtQdFTIVJgeEBBQPQ1g+yD5YrBwVODMQEAtEERzJLUAtLUD+24ITChESEyMgAwWzPUkrRSgJL5cvfRxYGyYrDwkLNRAhFEgJDAQAAAAAAwBkAAAEOQSwAFEAYABvAAABMzIWHQEeARcWDgIPATIeBRUUDgUjFRQGKwEiJj0BIxUUBisBIiY9ASMiJj0BNDY7AREjIiY9ATQ2OwE1NDY7ATIWHQEzNTQ2AxUhMj4CNTc0LgMjARUhMj4CNTc0LgMjAnGWCg9PaAEBIC4uEBEGEjQwOiodFyI2LUAjGg8KlgoPZA8KlgoPrwoPDwpLSwoPDwqvDwqWCg9kD9cBBxwpEwsBAQsTKRz++QFrHCkTCwEBCxMpHASwDwptIW1KLk0tHwYGAw8UKDJOLTtdPCoVCwJLCg8PCktLCg8PCksPCpYKDwJYDwqWCg9LCg8PCktLCg/+1MgVHR0LCgQOIhoW/nDIFR0dCwoEDiIaFgAAAwAEAAIEsASuABcAKQAsAAATITIWFREUBg8BDgEjISImJy4CNRE0NgQiDgQPARchNy4FAyMT1AMMVnokEhIdgVL9xFKCHAgYKHoCIIx9VkcrHQYGnAIwnAIIIClJVSGdwwSuelb+YDO3QkJXd3ZYHFrFMwGgVnqZFyYtLSUMDPPzBQ8sKDEj/sIBBQACAMgAAAOEBRQADwAZAAABMzIWFREUBiMhIiY1ETQ2ARUUBisBIiY9AQHblmesVCn+PilUrAFINhWWFTYFFKxn/gwpVFQpAfRnrPwY4RU2NhXhAAACAMgAAAOEBRQADwAZAAABMxQWMxEUBiMhIiY1ETQ2ARUUBisBIiY9AQHbYLOWVCn+PilUrAFINhWWFTYFFJaz/kIpVFQpAfRnrPwY4RU2NhXhAAACAAAAFAUOBBoAFAAaAAAJASUHFRcVJwc1NzU0Jj4CPwEnCQEFJTUFJQUO/YL+hk5klpZkAQEBBQQvkwKCAVz+ov6iAV4BXgL//uWqPOCWx5SVyJb6BA0GCgYDKEEBG/1ipqaTpaUAAAMAZAH0BLADIAAHAA8AFwAAEjIWFAYiJjQkMhYUBiImNCQyFhQGIiY0vHxYWHxYAeh8WFh8WAHofFhYfFgDIFh8WFh8WFh8WFh8WFh8WFh8AAAAAAMBkAAAArwETAAHAA8AFwAAADIWFAYiJjQSMhYUBiImNBIyFhQGIiY0Aeh8WFh8WFh8WFh8WFh8WFh8WARMWHxYWHz+yFh8WFh8/shYfFhYfAAAAAMAZABkBEwETAAPAB8ALwAAEyEyFh0BFAYjISImPQE0NhMhMhYdARQGIyEiJj0BNDYTITIWHQEUBiMhIiY9ATQ2fQO2Cg8PCvxKCg8PCgO2Cg8PCvxKCg8PCgO2Cg8PCvxKCg8PBEwPCpYKDw8KlgoP/nAPCpYKDw8KlgoP/nAPCpYKDw8KlgoPAAAABAAAAAAEsASwAA8AHwAvADMAAAEhMhYVERQGIyEiJjURNDYFISIGFREUFjMhMjY1ETQmBSEyFhURFAYjISImNRE0NhcVITUBXgH0ory7o/4Mpbm5Asv9qCk7OykCWCk7O/2xAfQVHR0V/gwVHR1HAZAEsLuj/gylubmlAfSlucg7Kf2oKTs7KQJYKTtkHRX+1BUdHRUBLBUdZMjIAAAAAAEAZABkBLAETAA7AAATITIWFAYrARUzMhYUBisBFTMyFhQGKwEVMzIWFAYjISImNDY7ATUjIiY0NjsBNSMiJjQ2OwE1IyImNDaWA+gVHR0VMjIVHR0VMjIVHR0VMjIVHR0V/BgVHR0VMjIVHR0VMjIVHR0VMjIVHR0ETB0qHcgdKh3IHSodyB0qHR0qHcgdKh3IHSodyB0qHQAAAAYBLAAFA+gEowAHAA0AEwAZAB8AKgAAAR4BBgcuATYBMhYVIiYlFAYjNDYBMhYVIiYlFAYjNDYDFRQGIiY9ARYzMgKKVz8/V1c/P/75fLB8sAK8sHyw/cB8sHywArywfLCwHSodKAMRBKNDsrJCQrKy/sCwfLB8fLB8sP7UsHywfHywfLD+05AVHR0VjgQAAAH/tQDIBJQDgQBCAAABNzYXAR4BBw4BKwEyFRQOBCsBIhE0NyYiBxYVECsBIi4DNTQzIyImJyY2NwE2HwEeAQ4BLwEHIScHBi4BNgLpRRkUASoLCAYFGg8IAQQNGyc/KZK4ChRUFQu4jjBJJxkHAgcPGQYGCAsBKhQaTBQVCiMUM7YDe7YsFCMKFgNuEwYS/tkLHw8OEw0dNkY4MhwBIBgXBAQYF/7gKjxTQyMNEw4PHwoBKBIHEwUjKBYGDMHBDAUWKCMAAAAAAgAAAAAEsASwACUAQwAAASM0LgUrAREUFh8BFSE1Mj4DNREjIg4FFSMRIQEjNC4DKwERFBYXMxUjNTI1ESMiDgMVIzUhBLAyCAsZEyYYGcgyGRn+cAQOIhoWyBkYJhMZCwgyA+j9RBkIChgQEWQZDQzIMmQREBgKCBkB9AOEFSAVDggDAfyuFhkBAmRkAQUJFQ4DUgEDCA4VIBUBLP0SDxMKBQH+VwsNATIyGQGpAQUKEw+WAAAAAAMAAAAABEwErgAdACAAMAAAATUiJy4BLwEBIwEGBw4BDwEVITUiJj8BIRcWBiMVARsBARUUBiMhIiY9ATQ2MyEyFgPoGR4OFgUE/t9F/tQSFQkfCwsBETE7EkUBJT0NISf+7IZ5AbEdFfwYFR0dFQPoFR0BLDIgDiIKCwLr/Q4jFQkTBQUyMisusKYiQTIBhwFW/qr942QVHR0VZBUdHQADAAAAAASwBLAADwBHAEoAABMhMhYVERQGIyEiJjURNDYFIyIHAQYHBgcGHQEUFjMhMjY9ATQmIyInJj8BIRcWBwYjIgYdARQWMyEyNj0BNCYnIicmJyMBJhMjEzIETBUdHRX7tBUdHQJGRg0F/tUREhImDAsJAREIDAwINxAKCj8BCjkLEQwYCAwMCAE5CAwLCBEZGQ8B/uAFDsVnBLAdFfu0FR0dFQRMFR1SDP0PIBMSEAUNMggMDAgyCAwXDhmjmR8YEQwIMggMDAgyBwwBGRskAuwM/gUBCAAABAAAAAAEsASwAAMAEwAjACcAAAEhNSEFITIWFREUBiMhIiY1ETQ2KQEyFhURFAYjISImNRE0NhcRIREEsPtQBLD7ggGQFR0dFf5wFR0dAm0BkBUdHRX+cBUdHUcBLARMZMgdFfx8FR0dFQOEFR0dFf5wFR0dFQGQFR1k/tQBLAAEAAAAAASwBLAADwAfACMAJwAAEyEyFhURFAYjISImNRE0NgEhMhYVERQGIyEiJjURNDYXESEREyE1ITIBkBUdHRX+cBUdHQJtAZAVHR0V/nAVHR1HASzI+1AEsASwHRX8fBUdHRUDhBUd/gwdFf5wFR0dFQGQFR1k/tQBLP2oZAAAAAACAAAAZASwA+gAJwArAAATITIWFREzNTQ2MyEyFh0BMxUjFRQGIyEiJj0BIxEUBiMhIiY1ETQ2AREhETIBkBUdZB0VAZAVHWRkHRX+cBUdZB0V/nAVHR0CnwEsA+gdFf6ilhUdHRWWZJYVHR0Vlv6iFR0dFQMgFR3+1P7UASwAAAQAAAAABLAEsAADABMAFwAnAAAzIxEzFyEyFhURFAYjISImNRE0NhcRIREBITIWFREUBiMhIiY1ETQ2ZGRklgGQFR0dFf5wFR0dRwEs/qIDhBUdHRX8fBUdHQSwZB0V/nAVHR0VAZAVHWT+1AEs/gwdFf5wFR0dFQGQFR0AAAAAAgBkAAAETASwACcAKwAAATMyFhURFAYrARUhMhYVERQGIyEiJjURNDYzITUjIiY1ETQ2OwE1MwcRIRECWJYVHR0VlgHCFR0dFfx8FR0dFQFelhUdHRWWZMgBLARMHRX+cBUdZB0V/nAVHR0VAZAVHWQdFQGQFR1kyP7UASwAAAAEAAAAAASwBLAAAwATABcAJwAAISMRMwUhMhYVERQGIyEiJjURNDYXESERASEyFhURFAYjISImNRE0NgSwZGT9dgGQFR0dFf5wFR0dRwEs/K4DhBUdHRX8fBUdHQSwZB0V/nAVHR0VAZAVHWT+1AEs/gwdFf5wFR0dFQGQFR0AAAEBLAAwA28EgAAPAAAJAQYjIiY1ETQ2MzIXARYUA2H+EhcSDhAQDhIXAe4OAjX+EhcbGQPoGRsX/hIOKgAAAAABAUEAMgOEBH4ACwAACQE2FhURFAYnASY0AU8B7h0qKh3+Eg4CewHuHREp/BgpER0B7g4qAAAAAAEAMgFBBH4DhAALAAATITIWBwEGIicBJjZkA+gpER3+Eg4qDv4SHREDhCod/hIODgHuHSoAAAAAAQAyASwEfgNvAAsAAAkBFgYjISImNwE2MgJ7Ae4dESn8GCkRHQHuDioDYf4SHSoqHQHuDgAAAAACAAgAAASwBCgABgAKAAABFQE1LQE1ASE1IQK8/UwBnf5jBKj84AMgAuW2/r3dwcHd+9jIAAAAAAIAAABkBLAEsAALADEAAAEjFTMVIREzNSM1IQEzND4FOwERFAYPARUhNSIuAzURMzIeBRUzESEEsMjI/tTIyAEs+1AyCAsZEyYYGWQyGRkBkAQOIhoWZBkYJhMZCwgy/OADhGRkASxkZP4MFSAVDggDAf3aFhkBAmRkAQUJFQ4CJgEDCA4VIBUBLAAAAgAAAAAETAPoACUAMQAAASM0LgUrAREUFh8BFSE1Mj4DNREjIg4FFSMRIQEjFTMVIREzNSM1IQMgMggLGRMmGBlkMhkZ/nAEDiIaFmQZGCYTGQsIMgMgASzIyP7UyMgBLAK8FSAVDggDAf3aFhkCAWRkAQUJFQ4CJgEDCA4VIBUBLPzgZGQBLGRkAAABAMgAZgNyBEoAEgAAATMyFgcJARYGKwEiJwEmNDcBNgK9oBAKDP4wAdAMChCgDQr+KQcHAdcKBEoWDP4w/jAMFgkB1wgUCAHXCQAAAQE+AGYD6ARKABIAAAEzMhcBFhQHAQYrASImNwkBJjYBU6ANCgHXBwf+KQoNoBAKDAHQ/jAMCgRKCf4pCBQI/ikJFgwB0AHQDBYAAAEAZgDIBEoDcgASAAAAFh0BFAcBBiInASY9ATQ2FwkBBDQWCf4pCBQI/ikJFgwB0AHQA3cKEKANCv4pBwcB1woNoBAKDP4wAdAAAAABAGYBPgRKA+gAEgAACQEWHQEUBicJAQYmPQE0NwE2MgJqAdcJFgz+MP4wDBYJAdcIFAPh/ikKDaAQCgwB0P4wDAoQoA0KAdcHAAAAAgDZ//kEPQSwAAUAOgAAARQGIzQ2BTMyFh8BNjc+Ah4EBgcOBgcGIiYjIgYiJy4DLwEuAT4EHgEXJyY2A+iwfLD+VmQVJgdPBQsiKFAzRyorDwURAQQSFyozTSwNOkkLDkc3EDlfNyYHBw8GDyUqPjdGMR+TDA0EsHywfLDIHBPCAQIGBwcFDx81S21DBxlLR1xKQhEFBQcHGWt0bCQjP2hJNyATBwMGBcASGAAAAAACAMgAFQOEBLAAFgAaAAATITIWFREUBisBEQcGJjURIyImNRE0NhcVITX6AlgVHR0Vlv8TGpYVHR2rASwEsB0V/nAVHf4MsgkQFQKKHRUBkBUdZGRkAAAAAgDIABkETASwAA4AEgAAEyEyFhURBRElIREjETQ2ARU3NfoC7ic9/UQCWP1EZB8BDWQEsFEs/Ft1A7Z9/BgEARc0/V1kFGQAAQAAAAECTW/DBF9fDzz1AB8EsAAAAADQdnOXAAAAANB2c5f/Uf+cBdwFFAAAAAgAAgAAAAAAAAABAAAFFP+FAAAFFP9R/tQF3AABAAAAAAAAAAAAAAAAAAAAowG4ACgAAAAAAZAAAASwAAAEsABkBLAAAASwAAAEsABwAooAAAUUAAACigAABRQAAAGxAAABRQAAANgAAADYAAAAogAAAQQAAABIAAABBAAAAUUAAASwAGQEsAB7BLAAyASwAMgB9AAABLD/8gSwAAAEsAAABLD/8ASwAAAEsAAOBLAACQSwAGQEsP/TBLD/0wSwAAAEsAAABLAAAASwAAAEsAAABLAAJgSwAG4EsAAXBLAAFwSwABcEsABkBLAAGgSwAGQEsAAMBLAAZASwABcEsP+cBLAAZASwABcEsAAXBLAAAASwABcEsAAXBLAAFwSwAGQEsAAABLAAZASwAAAEsAAABLAAAASwAAAEsAAABLAAAASwAAAEsAAABLAAZASwAMgEsAAABLAAAASwADUEsABkBLAAyASw/7UEsAAhBLAAAASwAAAEsAAABLAAAASwAAAEsP+cBLAAAASwAAAEsAAABLAA2wSwABcEsAB1BLAAAASwAAAEsAAABLAACgSwAMgEsAAABLAAnQSwAMgEsADIBLAAyASwAAAEsP/+BLABLASwAGQEsACIBLABOwSwABcEsAAXBLAAFwSwABcEsAAXBLAAFwSwAAAEsAAXBLAAFwSwABcEsAAXBLAAAASwALcEsAC3BLAAAASwAAAEsABJBLAAFwSwAAAEsAAABLAAXQSw/9wEsP/cBLD/nwSwAGQEsAAABLAAAASwAAAEsABkBLD//wSwAAAEsP9RBLAABgSwAAAEsAAABLABRQSwAAEEsAAABLD/nASwAEoEsAAUBLAAAASwAAAEsAAABLD/nASwAGEEsP/9BLAAFgSwABYEsAAWBLAAFgSwABgEsAAABMQAAASwAGQAAAAAAAD/2ABkADkAyAAAAScAZAAZABkAGQAZABkAGQAZAAAAAAAAAAAAAADZAAAAAAAOAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAMAZABkAAAAEAAAAAAAZP+c/5z/nP+c/5z/nP+c/5wACQAJ//L/8gBkAHkAJwBkAGQAAAAAAGT/ogAAAAAAAAAAAAAAAADIAGQAAAABAI8AAP+c/5wAZAAEAMgAyAAAAGQBkABkAAAAZAEs/7UAAAAAAAAAAAAAAAAAAABkAAABLAFBADIAMgAIAAAAAADIAT4AZgBmANkAyADIAAAAKgAqACoAKgCyAOgA6AFOAU4BTgFOAU4BTgFOAU4BTgFOAU4BTgFOAU4BpAIGAiICfgKGAqwC5ANGA24DjAPEBAgEMgRiBKIE3AVcBboGcgb0ByAHYgfKCB4IYgi+CTYJhAm2Cd4KKApMCpQK4gswC4oLygwIDFgNKg1eDbAODg5oDrQPKA+mD+YQEhBUEJAQqhEqEXYRthIKEjgSfBLAExoTdBPQFCoU1BU8FagVzBYEFjYWYBawFv4XUhemGAIYLhhqGJYYsBjgGP4ZKBloGZQZxBnaGe4aNhpoGrga9hteG7QcMhyUHOIdHB1EHWwdlB28HeYeLh52HsAfYh/SIEYgviEyIXYhuCJAIpYiuCMOIyIjOCN6I8Ij4CQCJDAkXiSWJOIlNCVgJbwmFCZ+JuYnUCe8J/goNChwKKwpoCnMKiYqSiqEKworeiwILGgsuizsLRwtiC30LiguZi6iLtgvDi9GL34vsi/4MD4whDDSMRIxYDGuMegyJDJeMpoy3jMiMz4zaDO2NBg0YDSoNNI1LDWeNeg2PjZ8Ntw3GjdON5I31DgQOEI4hjjIOQo5SjmIOcw6HDpsOpo63jugO9w8GDxQPKI8+D0yPew+Oj6MPtQ/KD9uP6o/+kBIQIBAxkECQX5CGEKoQu5DGENCQ3ZDoEPKRBBEYESuRPZFWkW2RgZGdEa0RvZHNkd2R7ZH9kgWSDJITkhqSIZIzEkSSThJXkmESapKAkouSlIAAQAAARcApwARAAAAAAACAAAAAQABAAAAQAAuAAAAAAAAABAAxgABAAAAAAATABIAAAADAAEECQAAAGoAEgADAAEECQABACgAfAADAAEECQACAA4ApAADAAEECQADAEwAsgADAAEECQAEADgA/gADAAEECQAFAHgBNgADAAEECQAGADYBrgADAAEECQAIABYB5AADAAEECQAJABYB+gADAAEECQALACQCEAADAAEECQAMACQCNAADAAEECQATACQCWAADAAEECQDIABYCfAADAAEECQDJADACkgADAAEECdkDABoCwnd3dy5nbHlwaGljb25zLmNvbQBDAG8AcAB5AHIAaQBnAGgAdAAgAKkAIAAyADAAMQA0ACAAYgB5ACAASgBhAG4AIABLAG8AdgBhAHIAaQBrAC4AIABBAGwAbAAgAHIAaQBnAGgAdABzACAAcgBlAHMAZQByAHYAZQBkAC4ARwBMAFkAUABIAEkAQwBPAE4AUwAgAEgAYQBsAGYAbABpAG4AZwBzAFIAZQBnAHUAbABhAHIAMQAuADAAMAA5ADsAVQBLAFcATgA7AEcATABZAFAASABJAEMATwBOAFMASABhAGwAZgBsAGkAbgBnAHMALQBSAGUAZwB1AGwAYQByAEcATABZAFAASABJAEMATwBOAFMAIABIAGEAbABmAGwAaQBuAGcAcwAgAFIAZQBnAHUAbABhAHIAVgBlAHIAcwBpAG8AbgAgADEALgAwADAAOQA7AFAAUwAgADAAMAAxAC4AMAAwADkAOwBoAG8AdABjAG8AbgB2ACAAMQAuADAALgA3ADAAOwBtAGEAawBlAG8AdABmAC4AbABpAGIAMgAuADUALgA1ADgAMwAyADkARwBMAFkAUABIAEkAQwBPAE4AUwBIAGEAbABmAGwAaQBuAGcAcwAtAFIAZQBnAHUAbABhAHIASgBhAG4AIABLAG8AdgBhAHIAaQBrAEoAYQBuACAASwBvAHYAYQByAGkAawB3AHcAdwAuAGcAbAB5AHAAaABpAGMAbwBuAHMALgBjAG8AbQB3AHcAdwAuAGcAbAB5AHAAaABpAGMAbwBuAHMALgBjAG8AbQB3AHcAdwAuAGcAbAB5AHAAaABpAGMAbwBuAHMALgBjAG8AbQBXAGUAYgBmAG8AbgB0ACAAMQAuADAAVwBlAGQAIABPAGMAdAAgADIAOQAgADAANgA6ADMANgA6ADAANwAgADIAMAAxADQARgBvAG4AdAAgAFMAcQB1AGkAcgByAGUAbAAAAAIAAAAAAAD/tQAyAAAAAAAAAAAAAAAAAAAAAAAAAAABFwAAAQIBAwADAA0ADgEEAJYBBQEGAQcBCAEJAQoBCwEMAQ0BDgEPARABEQESARMA7wEUARUBFgEXARgBGQEaARsBHAEdAR4BHwEgASEBIgEjASQBJQEmAScBKAEpASoBKwEsAS0BLgEvATABMQEyATMBNAE1ATYBNwE4ATkBOgE7ATwBPQE+AT8BQAFBAUIBQwFEAUUBRgFHAUgBSQFKAUsBTAFNAU4BTwFQAVEBUgFTAVQBVQFWAVcBWAFZAVoBWwFcAV0BXgFfAWABYQFiAWMBZAFlAWYBZwFoAWkBagFrAWwBbQFuAW8BcAFxAXIBcwF0AXUBdgF3AXgBeQF6AXsBfAF9AX4BfwGAAYEBggGDAYQBhQGGAYcBiAGJAYoBiwGMAY0BjgGPAZABkQGSAZMBlAGVAZYBlwGYAZkBmgGbAZwBnQGeAZ8BoAGhAaIBowGkAaUBpgGnAagBqQGqAasBrAGtAa4BrwGwAbEBsgGzAbQBtQG2AbcBuAG5AboBuwG8Ab0BvgG/AcABwQHCAcMBxAHFAcYBxwHIAckBygHLAcwBzQHOAc8B0AHRAdIB0wHUAdUB1gHXAdgB2QHaAdsB3AHdAd4B3wHgAeEB4gHjAeQB5QHmAecB6AHpAeoB6wHsAe0B7gHvAfAB8QHyAfMB9AH1AfYB9wH4AfkB+gH7AfwB/QH+Af8CAAIBAgICAwIEAgUCBgIHAggCCQIKAgsCDAINAg4CDwIQAhECEgZnbHlwaDEGZ2x5cGgyB3VuaTAwQTAHdW5pMjAwMAd1bmkyMDAxB3VuaTIwMDIHdW5pMjAwMwd1bmkyMDA0B3VuaTIwMDUHdW5pMjAwNgd1bmkyMDA3B3VuaTIwMDgHdW5pMjAwOQd1bmkyMDBBB3VuaTIwMkYHdW5pMjA1RgRFdXJvB3VuaTIwQkQHdW5pMjMxQgd1bmkyNUZDB3VuaTI2MDEHdW5pMjZGQQd1bmkyNzA5B3VuaTI3MEYHdW5pRTAwMQd1bmlFMDAyB3VuaUUwMDMHdW5pRTAwNQd1bmlFMDA2B3VuaUUwMDcHdW5pRTAwOAd1bmlFMDA5B3VuaUUwMTAHdW5pRTAxMQd1bmlFMDEyB3VuaUUwMTMHdW5pRTAxNAd1bmlFMDE1B3VuaUUwMTYHdW5pRTAxNwd1bmlFMDE4B3VuaUUwMTkHdW5pRTAyMAd1bmlFMDIxB3VuaUUwMjIHdW5pRTAyMwd1bmlFMDI0B3VuaUUwMjUHdW5pRTAyNgd1bmlFMDI3B3VuaUUwMjgHdW5pRTAyOQd1bmlFMDMwB3VuaUUwMzEHdW5pRTAzMgd1bmlFMDMzB3VuaUUwMzQHdW5pRTAzNQd1bmlFMDM2B3VuaUUwMzcHdW5pRTAzOAd1bmlFMDM5B3VuaUUwNDAHdW5pRTA0MQd1bmlFMDQyB3VuaUUwNDMHdW5pRTA0NAd1bmlFMDQ1B3VuaUUwNDYHdW5pRTA0Nwd1bmlFMDQ4B3VuaUUwNDkHdW5pRTA1MAd1bmlFMDUxB3VuaUUwNTIHdW5pRTA1Mwd1bmlFMDU0B3VuaUUwNTUHdW5pRTA1Ngd1bmlFMDU3B3VuaUUwNTgHdW5pRTA1OQd1bmlFMDYwB3VuaUUwNjIHdW5pRTA2Mwd1bmlFMDY0B3VuaUUwNjUHdW5pRTA2Ngd1bmlFMDY3B3VuaUUwNjgHdW5pRTA2OQd1bmlFMDcwB3VuaUUwNzEHdW5pRTA3Mgd1bmlFMDczB3VuaUUwNzQHdW5pRTA3NQd1bmlFMDc2B3VuaUUwNzcHdW5pRTA3OAd1bmlFMDc5B3VuaUUwODAHdW5pRTA4MQd1bmlFMDgyB3VuaUUwODMHdW5pRTA4NAd1bmlFMDg1B3VuaUUwODYHdW5pRTA4Nwd1bmlFMDg4B3VuaUUwODkHdW5pRTA5MAd1bmlFMDkxB3VuaUUwOTIHdW5pRTA5Mwd1bmlFMDk0B3VuaUUwOTUHdW5pRTA5Ngd1bmlFMDk3B3VuaUUxMDEHdW5pRTEwMgd1bmlFMTAzB3VuaUUxMDQHdW5pRTEwNQd1bmlFMTA2B3VuaUUxMDcHdW5pRTEwOAd1bmlFMTA5B3VuaUUxMTAHdW5pRTExMQd1bmlFMTEyB3VuaUUxMTMHdW5pRTExNAd1bmlFMTE1B3VuaUUxMTYHdW5pRTExNwd1bmlFMTE4B3VuaUUxMTkHdW5pRTEyMAd1bmlFMTIxB3VuaUUxMjIHdW5pRTEyMwd1bmlFMTI0B3VuaUUxMjUHdW5pRTEyNgd1bmlFMTI3B3VuaUUxMjgHdW5pRTEyOQd1bmlFMTMwB3VuaUUxMzEHdW5pRTEzMgd1bmlFMTMzB3VuaUUxMzQHdW5pRTEzNQd1bmlFMTM2B3VuaUUxMzcHdW5pRTEzOAd1bmlFMTM5B3VuaUUxNDAHdW5pRTE0MQd1bmlFMTQyB3VuaUUxNDMHdW5pRTE0NAd1bmlFMTQ1B3VuaUUxNDYHdW5pRTE0OAd1bmlFMTQ5B3VuaUUxNTAHdW5pRTE1MQd1bmlFMTUyB3VuaUUxNTMHdW5pRTE1NAd1bmlFMTU1B3VuaUUxNTYHdW5pRTE1Nwd1bmlFMTU4B3VuaUUxNTkHdW5pRTE2MAd1bmlFMTYxB3VuaUUxNjIHdW5pRTE2Mwd1bmlFMTY0B3VuaUUxNjUHdW5pRTE2Ngd1bmlFMTY3B3VuaUUxNjgHdW5pRTE2OQd1bmlFMTcwB3VuaUUxNzEHdW5pRTE3Mgd1bmlFMTczB3VuaUUxNzQHdW5pRTE3NQd1bmlFMTc2B3VuaUUxNzcHdW5pRTE3OAd1bmlFMTc5B3VuaUUxODAHdW5pRTE4MQd1bmlFMTgyB3VuaUUxODMHdW5pRTE4NAd1bmlFMTg1B3VuaUUxODYHdW5pRTE4Nwd1bmlFMTg4B3VuaUUxODkHdW5pRTE5MAd1bmlFMTkxB3VuaUUxOTIHdW5pRTE5Mwd1bmlFMTk0B3VuaUUxOTUHdW5pRTE5Nwd1bmlFMTk4B3VuaUUxOTkHdW5pRTIwMAd1bmlFMjAxB3VuaUUyMDIHdW5pRTIwMwd1bmlFMjA0B3VuaUUyMDUHdW5pRTIwNgd1bmlFMjA5B3VuaUUyMTAHdW5pRTIxMQd1bmlFMjEyB3VuaUUyMTMHdW5pRTIxNAd1bmlFMjE1B3VuaUUyMTYHdW5pRTIxOAd1bmlFMjE5B3VuaUUyMjEHdW5pRTIyMwd1bmlFMjI0B3VuaUUyMjUHdW5pRTIyNgd1bmlFMjI3B3VuaUUyMzAHdW5pRTIzMQd1bmlFMjMyB3VuaUUyMzMHdW5pRTIzNAd1bmlFMjM1B3VuaUUyMzYHdW5pRTIzNwd1bmlFMjM4B3VuaUUyMzkHdW5pRTI0MAd1bmlFMjQxB3VuaUUyNDIHdW5pRTI0Mwd1bmlFMjQ0B3VuaUUyNDUHdW5pRTI0Ngd1bmlFMjQ3B3VuaUUyNDgHdW5pRTI0OQd1bmlFMjUwB3VuaUUyNTEHdW5pRTI1Mgd1bmlFMjUzB3VuaUUyNTQHdW5pRTI1NQd1bmlFMjU2B3VuaUUyNTcHdW5pRTI1OAd1bmlFMjU5B3VuaUUyNjAHdW5pRjhGRgZ1MUY1MTEGdTFGNkFBAAAAAAFUUMMXAAA=) format('truetype'),url(data:image/svg+xml;base64,<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
<svg xmlns="http://www.w3.org/2000/svg">
<metadata></metadata>
<defs>
<font id="glyphicons_halflingsregular" horiz-adv-x="1200" >
<font-face units-per-em="1200" ascent="960" descent="-240" />
<missing-glyph horiz-adv-x="500" />
<glyph horiz-adv-x="0" />
<glyph horiz-adv-x="400" />
<glyph unicode=" " />
<glyph unicode="*" d="M600 1100q15 0 34 -1.5t30 -3.5l11 -1q10 -2 17.5 -10.5t7.5 -18.5v-224l158 158q7 7 18 8t19 -6l106 -106q7 -8 6 -19t-8 -18l-158 -158h224q10 0 18.5 -7.5t10.5 -17.5q6 -41 6 -75q0 -15 -1.5 -34t-3.5 -30l-1 -11q-2 -10 -10.5 -17.5t-18.5 -7.5h-224l158 -158 q7 -7 8 -18t-6 -19l-106 -106q-8 -7 -19 -6t-18 8l-158 158v-224q0 -10 -7.5 -18.5t-17.5 -10.5q-41 -6 -75 -6q-15 0 -34 1.5t-30 3.5l-11 1q-10 2 -17.5 10.5t-7.5 18.5v224l-158 -158q-7 -7 -18 -8t-19 6l-106 106q-7 8 -6 19t8 18l158 158h-224q-10 0 -18.5 7.5 t-10.5 17.5q-6 41 -6 75q0 15 1.5 34t3.5 30l1 11q2 10 10.5 17.5t18.5 7.5h224l-158 158q-7 7 -8 18t6 19l106 106q8 7 19 6t18 -8l158 -158v224q0 10 7.5 18.5t17.5 10.5q41 6 75 6z" />
<glyph unicode="+" d="M450 1100h200q21 0 35.5 -14.5t14.5 -35.5v-350h350q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-350v-350q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v350h-350q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5 h350v350q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xa0;" />
<glyph unicode="&#xa5;" d="M825 1100h250q10 0 12.5 -5t-5.5 -13l-364 -364q-6 -6 -11 -18h268q10 0 13 -6t-3 -14l-120 -160q-6 -8 -18 -14t-22 -6h-125v-100h275q10 0 13 -6t-3 -14l-120 -160q-6 -8 -18 -14t-22 -6h-125v-174q0 -11 -7.5 -18.5t-18.5 -7.5h-148q-11 0 -18.5 7.5t-7.5 18.5v174 h-275q-10 0 -13 6t3 14l120 160q6 8 18 14t22 6h125v100h-275q-10 0 -13 6t3 14l120 160q6 8 18 14t22 6h118q-5 12 -11 18l-364 364q-8 8 -5.5 13t12.5 5h250q25 0 43 -18l164 -164q8 -8 18 -8t18 8l164 164q18 18 43 18z" />
<glyph unicode="&#x2000;" horiz-adv-x="650" />
<glyph unicode="&#x2001;" horiz-adv-x="1300" />
<glyph unicode="&#x2002;" horiz-adv-x="650" />
<glyph unicode="&#x2003;" horiz-adv-x="1300" />
<glyph unicode="&#x2004;" horiz-adv-x="433" />
<glyph unicode="&#x2005;" horiz-adv-x="325" />
<glyph unicode="&#x2006;" horiz-adv-x="216" />
<glyph unicode="&#x2007;" horiz-adv-x="216" />
<glyph unicode="&#x2008;" horiz-adv-x="162" />
<glyph unicode="&#x2009;" horiz-adv-x="260" />
<glyph unicode="&#x200a;" horiz-adv-x="72" />
<glyph unicode="&#x202f;" horiz-adv-x="260" />
<glyph unicode="&#x205f;" horiz-adv-x="325" />
<glyph unicode="&#x20ac;" d="M744 1198q242 0 354 -189q60 -104 66 -209h-181q0 45 -17.5 82.5t-43.5 61.5t-58 40.5t-60.5 24t-51.5 7.5q-19 0 -40.5 -5.5t-49.5 -20.5t-53 -38t-49 -62.5t-39 -89.5h379l-100 -100h-300q-6 -50 -6 -100h406l-100 -100h-300q9 -74 33 -132t52.5 -91t61.5 -54.5t59 -29 t47 -7.5q22 0 50.5 7.5t60.5 24.5t58 41t43.5 61t17.5 80h174q-30 -171 -128 -278q-107 -117 -274 -117q-206 0 -324 158q-36 48 -69 133t-45 204h-217l100 100h112q1 47 6 100h-218l100 100h134q20 87 51 153.5t62 103.5q117 141 297 141z" />
<glyph unicode="&#x20bd;" d="M428 1200h350q67 0 120 -13t86 -31t57 -49.5t35 -56.5t17 -64.5t6.5 -60.5t0.5 -57v-16.5v-16.5q0 -36 -0.5 -57t-6.5 -61t-17 -65t-35 -57t-57 -50.5t-86 -31.5t-120 -13h-178l-2 -100h288q10 0 13 -6t-3 -14l-120 -160q-6 -8 -18 -14t-22 -6h-138v-175q0 -11 -5.5 -18 t-15.5 -7h-149q-10 0 -17.5 7.5t-7.5 17.5v175h-267q-10 0 -13 6t3 14l120 160q6 8 18 14t22 6h117v100h-267q-10 0 -13 6t3 14l120 160q6 8 18 14t22 6h117v475q0 10 7.5 17.5t17.5 7.5zM600 1000v-300h203q64 0 86.5 33t22.5 119q0 84 -22.5 116t-86.5 32h-203z" />
<glyph unicode="&#x2212;" d="M250 700h800q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#x231b;" d="M1000 1200v-150q0 -21 -14.5 -35.5t-35.5 -14.5h-50v-100q0 -91 -49.5 -165.5t-130.5 -109.5q81 -35 130.5 -109.5t49.5 -165.5v-150h50q21 0 35.5 -14.5t14.5 -35.5v-150h-800v150q0 21 14.5 35.5t35.5 14.5h50v150q0 91 49.5 165.5t130.5 109.5q-81 35 -130.5 109.5 t-49.5 165.5v100h-50q-21 0 -35.5 14.5t-14.5 35.5v150h800zM400 1000v-100q0 -60 32.5 -109.5t87.5 -73.5q28 -12 44 -37t16 -55t-16 -55t-44 -37q-55 -24 -87.5 -73.5t-32.5 -109.5v-150h400v150q0 60 -32.5 109.5t-87.5 73.5q-28 12 -44 37t-16 55t16 55t44 37 q55 24 87.5 73.5t32.5 109.5v100h-400z" />
<glyph unicode="&#x25fc;" horiz-adv-x="500" d="M0 0z" />
<glyph unicode="&#x2601;" d="M503 1089q110 0 200.5 -59.5t134.5 -156.5q44 14 90 14q120 0 205 -86.5t85 -206.5q0 -121 -85 -207.5t-205 -86.5h-750q-79 0 -135.5 57t-56.5 137q0 69 42.5 122.5t108.5 67.5q-2 12 -2 37q0 153 108 260.5t260 107.5z" />
<glyph unicode="&#x26fa;" d="M774 1193.5q16 -9.5 20.5 -27t-5.5 -33.5l-136 -187l467 -746h30q20 0 35 -18.5t15 -39.5v-42h-1200v42q0 21 15 39.5t35 18.5h30l468 746l-135 183q-10 16 -5.5 34t20.5 28t34 5.5t28 -20.5l111 -148l112 150q9 16 27 20.5t34 -5zM600 200h377l-182 112l-195 534v-646z " />
<glyph unicode="&#x2709;" d="M25 1100h1150q10 0 12.5 -5t-5.5 -13l-564 -567q-8 -8 -18 -8t-18 8l-564 567q-8 8 -5.5 13t12.5 5zM18 882l264 -264q8 -8 8 -18t-8 -18l-264 -264q-8 -8 -13 -5.5t-5 12.5v550q0 10 5 12.5t13 -5.5zM918 618l264 264q8 8 13 5.5t5 -12.5v-550q0 -10 -5 -12.5t-13 5.5 l-264 264q-8 8 -8 18t8 18zM818 482l364 -364q8 -8 5.5 -13t-12.5 -5h-1150q-10 0 -12.5 5t5.5 13l364 364q8 8 18 8t18 -8l164 -164q8 -8 18 -8t18 8l164 164q8 8 18 8t18 -8z" />
<glyph unicode="&#x270f;" d="M1011 1210q19 0 33 -13l153 -153q13 -14 13 -33t-13 -33l-99 -92l-214 214l95 96q13 14 32 14zM1013 800l-615 -614l-214 214l614 614zM317 96l-333 -112l110 335z" />
<glyph unicode="&#xe001;" d="M700 650v-550h250q21 0 35.5 -14.5t14.5 -35.5v-50h-800v50q0 21 14.5 35.5t35.5 14.5h250v550l-500 550h1200z" />
<glyph unicode="&#xe002;" d="M368 1017l645 163q39 15 63 0t24 -49v-831q0 -55 -41.5 -95.5t-111.5 -63.5q-79 -25 -147 -4.5t-86 75t25.5 111.5t122.5 82q72 24 138 8v521l-600 -155v-606q0 -42 -44 -90t-109 -69q-79 -26 -147 -5.5t-86 75.5t25.5 111.5t122.5 82.5q72 24 138 7v639q0 38 14.5 59 t53.5 34z" />
<glyph unicode="&#xe003;" d="M500 1191q100 0 191 -39t156.5 -104.5t104.5 -156.5t39 -191l-1 -2l1 -5q0 -141 -78 -262l275 -274q23 -26 22.5 -44.5t-22.5 -42.5l-59 -58q-26 -20 -46.5 -20t-39.5 20l-275 274q-119 -77 -261 -77l-5 1l-2 -1q-100 0 -191 39t-156.5 104.5t-104.5 156.5t-39 191 t39 191t104.5 156.5t156.5 104.5t191 39zM500 1022q-88 0 -162 -43t-117 -117t-43 -162t43 -162t117 -117t162 -43t162 43t117 117t43 162t-43 162t-117 117t-162 43z" />
<glyph unicode="&#xe005;" d="M649 949q48 68 109.5 104t121.5 38.5t118.5 -20t102.5 -64t71 -100.5t27 -123q0 -57 -33.5 -117.5t-94 -124.5t-126.5 -127.5t-150 -152.5t-146 -174q-62 85 -145.5 174t-150 152.5t-126.5 127.5t-93.5 124.5t-33.5 117.5q0 64 28 123t73 100.5t104 64t119 20 t120.5 -38.5t104.5 -104z" />
<glyph unicode="&#xe006;" d="M407 800l131 353q7 19 17.5 19t17.5 -19l129 -353h421q21 0 24 -8.5t-14 -20.5l-342 -249l130 -401q7 -20 -0.5 -25.5t-24.5 6.5l-343 246l-342 -247q-17 -12 -24.5 -6.5t-0.5 25.5l130 400l-347 251q-17 12 -14 20.5t23 8.5h429z" />
<glyph unicode="&#xe007;" d="M407 800l131 353q7 19 17.5 19t17.5 -19l129 -353h421q21 0 24 -8.5t-14 -20.5l-342 -249l130 -401q7 -20 -0.5 -25.5t-24.5 6.5l-343 246l-342 -247q-17 -12 -24.5 -6.5t-0.5 25.5l130 400l-347 251q-17 12 -14 20.5t23 8.5h429zM477 700h-240l197 -142l-74 -226 l193 139l195 -140l-74 229l192 140h-234l-78 211z" />
<glyph unicode="&#xe008;" d="M600 1200q124 0 212 -88t88 -212v-250q0 -46 -31 -98t-69 -52v-75q0 -10 6 -21.5t15 -17.5l358 -230q9 -5 15 -16.5t6 -21.5v-93q0 -10 -7.5 -17.5t-17.5 -7.5h-1150q-10 0 -17.5 7.5t-7.5 17.5v93q0 10 6 21.5t15 16.5l358 230q9 6 15 17.5t6 21.5v75q-38 0 -69 52 t-31 98v250q0 124 88 212t212 88z" />
<glyph unicode="&#xe009;" d="M25 1100h1150q10 0 17.5 -7.5t7.5 -17.5v-1050q0 -10 -7.5 -17.5t-17.5 -7.5h-1150q-10 0 -17.5 7.5t-7.5 17.5v1050q0 10 7.5 17.5t17.5 7.5zM100 1000v-100h100v100h-100zM875 1000h-550q-10 0 -17.5 -7.5t-7.5 -17.5v-350q0 -10 7.5 -17.5t17.5 -7.5h550 q10 0 17.5 7.5t7.5 17.5v350q0 10 -7.5 17.5t-17.5 7.5zM1000 1000v-100h100v100h-100zM100 800v-100h100v100h-100zM1000 800v-100h100v100h-100zM100 600v-100h100v100h-100zM1000 600v-100h100v100h-100zM875 500h-550q-10 0 -17.5 -7.5t-7.5 -17.5v-350q0 -10 7.5 -17.5 t17.5 -7.5h550q10 0 17.5 7.5t7.5 17.5v350q0 10 -7.5 17.5t-17.5 7.5zM100 400v-100h100v100h-100zM1000 400v-100h100v100h-100zM100 200v-100h100v100h-100zM1000 200v-100h100v100h-100z" />
<glyph unicode="&#xe010;" d="M50 1100h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM650 1100h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v400 q0 21 14.5 35.5t35.5 14.5zM50 500h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM650 500h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400 q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe011;" d="M50 1100h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM450 1100h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200 q0 21 14.5 35.5t35.5 14.5zM850 1100h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM50 700h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200 q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM450 700h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM850 700h200q21 0 35.5 -14.5t14.5 -35.5v-200 q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM50 300h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM450 300h200 q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM850 300h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5 t35.5 14.5z" />
<glyph unicode="&#xe012;" d="M50 1100h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM450 1100h700q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-700q-21 0 -35.5 14.5t-14.5 35.5v200 q0 21 14.5 35.5t35.5 14.5zM50 700h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM450 700h700q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-700 q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM50 300h200q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5zM450 300h700q21 0 35.5 -14.5t14.5 -35.5v-200 q0 -21 -14.5 -35.5t-35.5 -14.5h-700q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe013;" d="M465 477l571 571q8 8 18 8t17 -8l177 -177q8 -7 8 -17t-8 -18l-783 -784q-7 -8 -17.5 -8t-17.5 8l-384 384q-8 8 -8 18t8 17l177 177q7 8 17 8t18 -8l171 -171q7 -7 18 -7t18 7z" />
<glyph unicode="&#xe014;" d="M904 1083l178 -179q8 -8 8 -18.5t-8 -17.5l-267 -268l267 -268q8 -7 8 -17.5t-8 -18.5l-178 -178q-8 -8 -18.5 -8t-17.5 8l-268 267l-268 -267q-7 -8 -17.5 -8t-18.5 8l-178 178q-8 8 -8 18.5t8 17.5l267 268l-267 268q-8 7 -8 17.5t8 18.5l178 178q8 8 18.5 8t17.5 -8 l268 -267l268 268q7 7 17.5 7t18.5 -7z" />
<glyph unicode="&#xe015;" d="M507 1177q98 0 187.5 -38.5t154.5 -103.5t103.5 -154.5t38.5 -187.5q0 -141 -78 -262l300 -299q8 -8 8 -18.5t-8 -18.5l-109 -108q-7 -8 -17.5 -8t-18.5 8l-300 299q-119 -77 -261 -77q-98 0 -188 38.5t-154.5 103t-103 154.5t-38.5 188t38.5 187.5t103 154.5 t154.5 103.5t188 38.5zM506.5 1023q-89.5 0 -165.5 -44t-120 -120.5t-44 -166t44 -165.5t120 -120t165.5 -44t166 44t120.5 120t44 165.5t-44 166t-120.5 120.5t-166 44zM425 900h150q10 0 17.5 -7.5t7.5 -17.5v-75h75q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5 t-17.5 -7.5h-75v-75q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v75h-75q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5h75v75q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe016;" d="M507 1177q98 0 187.5 -38.5t154.5 -103.5t103.5 -154.5t38.5 -187.5q0 -141 -78 -262l300 -299q8 -8 8 -18.5t-8 -18.5l-109 -108q-7 -8 -17.5 -8t-18.5 8l-300 299q-119 -77 -261 -77q-98 0 -188 38.5t-154.5 103t-103 154.5t-38.5 188t38.5 187.5t103 154.5 t154.5 103.5t188 38.5zM506.5 1023q-89.5 0 -165.5 -44t-120 -120.5t-44 -166t44 -165.5t120 -120t165.5 -44t166 44t120.5 120t44 165.5t-44 166t-120.5 120.5t-166 44zM325 800h350q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-350q-10 0 -17.5 7.5 t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe017;" d="M550 1200h100q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM800 975v166q167 -62 272 -209.5t105 -331.5q0 -117 -45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5 t-184.5 123t-123 184.5t-45.5 224q0 184 105 331.5t272 209.5v-166q-103 -55 -165 -155t-62 -220q0 -116 57 -214.5t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5q0 120 -62 220t-165 155z" />
<glyph unicode="&#xe018;" d="M1025 1200h150q10 0 17.5 -7.5t7.5 -17.5v-1150q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v1150q0 10 7.5 17.5t17.5 7.5zM725 800h150q10 0 17.5 -7.5t7.5 -17.5v-750q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v750 q0 10 7.5 17.5t17.5 7.5zM425 500h150q10 0 17.5 -7.5t7.5 -17.5v-450q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v450q0 10 7.5 17.5t17.5 7.5zM125 300h150q10 0 17.5 -7.5t7.5 -17.5v-250q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5 v250q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe019;" d="M600 1174q33 0 74 -5l38 -152l5 -1q49 -14 94 -39l5 -2l134 80q61 -48 104 -105l-80 -134l3 -5q25 -44 39 -93l1 -6l152 -38q5 -43 5 -73q0 -34 -5 -74l-152 -38l-1 -6q-15 -49 -39 -93l-3 -5l80 -134q-48 -61 -104 -105l-134 81l-5 -3q-44 -25 -94 -39l-5 -2l-38 -151 q-43 -5 -74 -5q-33 0 -74 5l-38 151l-5 2q-49 14 -94 39l-5 3l-134 -81q-60 48 -104 105l80 134l-3 5q-25 45 -38 93l-2 6l-151 38q-6 42 -6 74q0 33 6 73l151 38l2 6q13 48 38 93l3 5l-80 134q47 61 105 105l133 -80l5 2q45 25 94 39l5 1l38 152q43 5 74 5zM600 815 q-89 0 -152 -63t-63 -151.5t63 -151.5t152 -63t152 63t63 151.5t-63 151.5t-152 63z" />
<glyph unicode="&#xe020;" d="M500 1300h300q41 0 70.5 -29.5t29.5 -70.5v-100h275q10 0 17.5 -7.5t7.5 -17.5v-75h-1100v75q0 10 7.5 17.5t17.5 7.5h275v100q0 41 29.5 70.5t70.5 29.5zM500 1200v-100h300v100h-300zM1100 900v-800q0 -41 -29.5 -70.5t-70.5 -29.5h-700q-41 0 -70.5 29.5t-29.5 70.5 v800h900zM300 800v-700h100v700h-100zM500 800v-700h100v700h-100zM700 800v-700h100v700h-100zM900 800v-700h100v700h-100z" />
<glyph unicode="&#xe021;" d="M18 618l620 608q8 7 18.5 7t17.5 -7l608 -608q8 -8 5.5 -13t-12.5 -5h-175v-575q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v375h-300v-375q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v575h-175q-10 0 -12.5 5t5.5 13z" />
<glyph unicode="&#xe022;" d="M600 1200v-400q0 -41 29.5 -70.5t70.5 -29.5h300v-650q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v1100q0 21 14.5 35.5t35.5 14.5h450zM1000 800h-250q-21 0 -35.5 14.5t-14.5 35.5v250z" />
<glyph unicode="&#xe023;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5t-57 214.5t-155.5 155.5t-214.5 57zM525 900h50q10 0 17.5 -7.5t7.5 -17.5v-275h175q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v350q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe024;" d="M1300 0h-538l-41 400h-242l-41 -400h-538l431 1200h209l-21 -300h162l-20 300h208zM515 800l-27 -300h224l-27 300h-170z" />
<glyph unicode="&#xe025;" d="M550 1200h200q21 0 35.5 -14.5t14.5 -35.5v-450h191q20 0 25.5 -11.5t-7.5 -27.5l-327 -400q-13 -16 -32 -16t-32 16l-327 400q-13 16 -7.5 27.5t25.5 11.5h191v450q0 21 14.5 35.5t35.5 14.5zM1125 400h50q10 0 17.5 -7.5t7.5 -17.5v-350q0 -10 -7.5 -17.5t-17.5 -7.5 h-1050q-10 0 -17.5 7.5t-7.5 17.5v350q0 10 7.5 17.5t17.5 7.5h50q10 0 17.5 -7.5t7.5 -17.5v-175h900v175q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe026;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5t-57 214.5t-155.5 155.5t-214.5 57zM525 900h150q10 0 17.5 -7.5t7.5 -17.5v-275h137q21 0 26 -11.5t-8 -27.5l-223 -275q-13 -16 -32 -16t-32 16l-223 275q-13 16 -8 27.5t26 11.5h137v275q0 10 7.5 17.5t17.5 7.5z " />
<glyph unicode="&#xe027;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5t-57 214.5t-155.5 155.5t-214.5 57zM632 914l223 -275q13 -16 8 -27.5t-26 -11.5h-137v-275q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v275h-137q-21 0 -26 11.5t8 27.5l223 275q13 16 32 16 t32 -16z" />
<glyph unicode="&#xe028;" d="M225 1200h750q10 0 19.5 -7t12.5 -17l186 -652q7 -24 7 -49v-425q0 -12 -4 -27t-9 -17q-12 -6 -37 -6h-1100q-12 0 -27 4t-17 8q-6 13 -6 38l1 425q0 25 7 49l185 652q3 10 12.5 17t19.5 7zM878 1000h-556q-10 0 -19 -7t-11 -18l-87 -450q-2 -11 4 -18t16 -7h150 q10 0 19.5 -7t11.5 -17l38 -152q2 -10 11.5 -17t19.5 -7h250q10 0 19.5 7t11.5 17l38 152q2 10 11.5 17t19.5 7h150q10 0 16 7t4 18l-87 450q-2 11 -11 18t-19 7z" />
<glyph unicode="&#xe029;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5t-57 214.5t-155.5 155.5t-214.5 57zM540 820l253 -190q17 -12 17 -30t-17 -30l-253 -190q-16 -12 -28 -6.5t-12 26.5v400q0 21 12 26.5t28 -6.5z" />
<glyph unicode="&#xe030;" d="M947 1060l135 135q7 7 12.5 5t5.5 -13v-362q0 -10 -7.5 -17.5t-17.5 -7.5h-362q-11 0 -13 5.5t5 12.5l133 133q-109 76 -238 76q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5h150q0 -117 -45.5 -224 t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5q192 0 347 -117z" />
<glyph unicode="&#xe031;" d="M947 1060l135 135q7 7 12.5 5t5.5 -13v-361q0 -11 -7.5 -18.5t-18.5 -7.5h-361q-11 0 -13 5.5t5 12.5l134 134q-110 75 -239 75q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5h-150q0 117 45.5 224t123 184.5t184.5 123t224 45.5q192 0 347 -117zM1027 600h150 q0 -117 -45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5q-192 0 -348 118l-134 -134q-7 -8 -12.5 -5.5t-5.5 12.5v360q0 11 7.5 18.5t18.5 7.5h360q10 0 12.5 -5.5t-5.5 -12.5l-133 -133q110 -76 240 -76q116 0 214.5 57t155.5 155.5t57 214.5z" />
<glyph unicode="&#xe032;" d="M125 1200h1050q10 0 17.5 -7.5t7.5 -17.5v-1150q0 -10 -7.5 -17.5t-17.5 -7.5h-1050q-10 0 -17.5 7.5t-7.5 17.5v1150q0 10 7.5 17.5t17.5 7.5zM1075 1000h-850q-10 0 -17.5 -7.5t-7.5 -17.5v-850q0 -10 7.5 -17.5t17.5 -7.5h850q10 0 17.5 7.5t7.5 17.5v850 q0 10 -7.5 17.5t-17.5 7.5zM325 900h50q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-50q-10 0 -17.5 7.5t-7.5 17.5v50q0 10 7.5 17.5t17.5 7.5zM525 900h450q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-450q-10 0 -17.5 7.5t-7.5 17.5v50 q0 10 7.5 17.5t17.5 7.5zM325 700h50q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-50q-10 0 -17.5 7.5t-7.5 17.5v50q0 10 7.5 17.5t17.5 7.5zM525 700h450q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-450q-10 0 -17.5 7.5t-7.5 17.5v50 q0 10 7.5 17.5t17.5 7.5zM325 500h50q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-50q-10 0 -17.5 7.5t-7.5 17.5v50q0 10 7.5 17.5t17.5 7.5zM525 500h450q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-450q-10 0 -17.5 7.5t-7.5 17.5v50 q0 10 7.5 17.5t17.5 7.5zM325 300h50q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-50q-10 0 -17.5 7.5t-7.5 17.5v50q0 10 7.5 17.5t17.5 7.5zM525 300h450q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-450q-10 0 -17.5 7.5t-7.5 17.5v50 q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe033;" d="M900 800v200q0 83 -58.5 141.5t-141.5 58.5h-300q-82 0 -141 -59t-59 -141v-200h-100q-41 0 -70.5 -29.5t-29.5 -70.5v-600q0 -41 29.5 -70.5t70.5 -29.5h900q41 0 70.5 29.5t29.5 70.5v600q0 41 -29.5 70.5t-70.5 29.5h-100zM400 800v150q0 21 15 35.5t35 14.5h200 q20 0 35 -14.5t15 -35.5v-150h-300z" />
<glyph unicode="&#xe034;" d="M125 1100h50q10 0 17.5 -7.5t7.5 -17.5v-1075h-100v1075q0 10 7.5 17.5t17.5 7.5zM1075 1052q4 0 9 -2q16 -6 16 -23v-421q0 -6 -3 -12q-33 -59 -66.5 -99t-65.5 -58t-56.5 -24.5t-52.5 -6.5q-26 0 -57.5 6.5t-52.5 13.5t-60 21q-41 15 -63 22.5t-57.5 15t-65.5 7.5 q-85 0 -160 -57q-7 -5 -15 -5q-6 0 -11 3q-14 7 -14 22v438q22 55 82 98.5t119 46.5q23 2 43 0.5t43 -7t32.5 -8.5t38 -13t32.5 -11q41 -14 63.5 -21t57 -14t63.5 -7q103 0 183 87q7 8 18 8z" />
<glyph unicode="&#xe035;" d="M600 1175q116 0 227 -49.5t192.5 -131t131 -192.5t49.5 -227v-300q0 -10 -7.5 -17.5t-17.5 -7.5h-50q-10 0 -17.5 7.5t-7.5 17.5v300q0 127 -70.5 231.5t-184.5 161.5t-245 57t-245 -57t-184.5 -161.5t-70.5 -231.5v-300q0 -10 -7.5 -17.5t-17.5 -7.5h-50 q-10 0 -17.5 7.5t-7.5 17.5v300q0 116 49.5 227t131 192.5t192.5 131t227 49.5zM220 500h160q8 0 14 -6t6 -14v-460q0 -8 -6 -14t-14 -6h-160q-8 0 -14 6t-6 14v460q0 8 6 14t14 6zM820 500h160q8 0 14 -6t6 -14v-460q0 -8 -6 -14t-14 -6h-160q-8 0 -14 6t-6 14v460 q0 8 6 14t14 6z" />
<glyph unicode="&#xe036;" d="M321 814l258 172q9 6 15 2.5t6 -13.5v-750q0 -10 -6 -13.5t-15 2.5l-258 172q-21 14 -46 14h-250q-10 0 -17.5 7.5t-7.5 17.5v350q0 10 7.5 17.5t17.5 7.5h250q25 0 46 14zM900 668l120 120q7 7 17 7t17 -7l34 -34q7 -7 7 -17t-7 -17l-120 -120l120 -120q7 -7 7 -17 t-7 -17l-34 -34q-7 -7 -17 -7t-17 7l-120 119l-120 -119q-7 -7 -17 -7t-17 7l-34 34q-7 7 -7 17t7 17l119 120l-119 120q-7 7 -7 17t7 17l34 34q7 8 17 8t17 -8z" />
<glyph unicode="&#xe037;" d="M321 814l258 172q9 6 15 2.5t6 -13.5v-750q0 -10 -6 -13.5t-15 2.5l-258 172q-21 14 -46 14h-250q-10 0 -17.5 7.5t-7.5 17.5v350q0 10 7.5 17.5t17.5 7.5h250q25 0 46 14zM766 900h4q10 -1 16 -10q96 -129 96 -290q0 -154 -90 -281q-6 -9 -17 -10l-3 -1q-9 0 -16 6 l-29 23q-7 7 -8.5 16.5t4.5 17.5q72 103 72 229q0 132 -78 238q-6 8 -4.5 18t9.5 17l29 22q7 5 15 5z" />
<glyph unicode="&#xe038;" d="M967 1004h3q11 -1 17 -10q135 -179 135 -396q0 -105 -34 -206.5t-98 -185.5q-7 -9 -17 -10h-3q-9 0 -16 6l-42 34q-8 6 -9 16t5 18q111 150 111 328q0 90 -29.5 176t-84.5 157q-6 9 -5 19t10 16l42 33q7 5 15 5zM321 814l258 172q9 6 15 2.5t6 -13.5v-750q0 -10 -6 -13.5 t-15 2.5l-258 172q-21 14 -46 14h-250q-10 0 -17.5 7.5t-7.5 17.5v350q0 10 7.5 17.5t17.5 7.5h250q25 0 46 14zM766 900h4q10 -1 16 -10q96 -129 96 -290q0 -154 -90 -281q-6 -9 -17 -10l-3 -1q-9 0 -16 6l-29 23q-7 7 -8.5 16.5t4.5 17.5q72 103 72 229q0 132 -78 238 q-6 8 -4.5 18.5t9.5 16.5l29 22q7 5 15 5z" />
<glyph unicode="&#xe039;" d="M500 900h100v-100h-100v-100h-400v-100h-100v600h500v-300zM1200 700h-200v-100h200v-200h-300v300h-200v300h-100v200h600v-500zM100 1100v-300h300v300h-300zM800 1100v-300h300v300h-300zM300 900h-100v100h100v-100zM1000 900h-100v100h100v-100zM300 500h200v-500 h-500v500h200v100h100v-100zM800 300h200v-100h-100v-100h-200v100h-100v100h100v200h-200v100h300v-300zM100 400v-300h300v300h-300zM300 200h-100v100h100v-100zM1200 200h-100v100h100v-100zM700 0h-100v100h100v-100zM1200 0h-300v100h300v-100z" />
<glyph unicode="&#xe040;" d="M100 200h-100v1000h100v-1000zM300 200h-100v1000h100v-1000zM700 200h-200v1000h200v-1000zM900 200h-100v1000h100v-1000zM1200 200h-200v1000h200v-1000zM400 0h-300v100h300v-100zM600 0h-100v91h100v-91zM800 0h-100v91h100v-91zM1100 0h-200v91h200v-91z" />
<glyph unicode="&#xe041;" d="M500 1200l682 -682q8 -8 8 -18t-8 -18l-464 -464q-8 -8 -18 -8t-18 8l-682 682l1 475q0 10 7.5 17.5t17.5 7.5h474zM319.5 1024.5q-29.5 29.5 -71 29.5t-71 -29.5t-29.5 -71.5t29.5 -71.5t71 -29.5t71 29.5t29.5 71.5t-29.5 71.5z" />
<glyph unicode="&#xe042;" d="M500 1200l682 -682q8 -8 8 -18t-8 -18l-464 -464q-8 -8 -18 -8t-18 8l-682 682l1 475q0 10 7.5 17.5t17.5 7.5h474zM800 1200l682 -682q8 -8 8 -18t-8 -18l-464 -464q-8 -8 -18 -8t-18 8l-56 56l424 426l-700 700h150zM319.5 1024.5q-29.5 29.5 -71 29.5t-71 -29.5 t-29.5 -71.5t29.5 -71.5t71 -29.5t71 29.5t29.5 71.5t-29.5 71.5z" />
<glyph unicode="&#xe043;" d="M300 1200h825q75 0 75 -75v-900q0 -25 -18 -43l-64 -64q-8 -8 -13 -5.5t-5 12.5v950q0 10 -7.5 17.5t-17.5 7.5h-700q-25 0 -43 -18l-64 -64q-8 -8 -5.5 -13t12.5 -5h700q10 0 17.5 -7.5t7.5 -17.5v-950q0 -10 -7.5 -17.5t-17.5 -7.5h-850q-10 0 -17.5 7.5t-7.5 17.5v975 q0 25 18 43l139 139q18 18 43 18z" />
<glyph unicode="&#xe044;" d="M250 1200h800q21 0 35.5 -14.5t14.5 -35.5v-1150l-450 444l-450 -445v1151q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe045;" d="M822 1200h-444q-11 0 -19 -7.5t-9 -17.5l-78 -301q-7 -24 7 -45l57 -108q6 -9 17.5 -15t21.5 -6h450q10 0 21.5 6t17.5 15l62 108q14 21 7 45l-83 301q-1 10 -9 17.5t-19 7.5zM1175 800h-150q-10 0 -21 -6.5t-15 -15.5l-78 -156q-4 -9 -15 -15.5t-21 -6.5h-550 q-10 0 -21 6.5t-15 15.5l-78 156q-4 9 -15 15.5t-21 6.5h-150q-10 0 -17.5 -7.5t-7.5 -17.5v-650q0 -10 7.5 -17.5t17.5 -7.5h150q10 0 17.5 7.5t7.5 17.5v150q0 10 7.5 17.5t17.5 7.5h750q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 7.5 -17.5t17.5 -7.5h150q10 0 17.5 7.5 t7.5 17.5v650q0 10 -7.5 17.5t-17.5 7.5zM850 200h-500q-10 0 -19.5 -7t-11.5 -17l-38 -152q-2 -10 3.5 -17t15.5 -7h600q10 0 15.5 7t3.5 17l-38 152q-2 10 -11.5 17t-19.5 7z" />
<glyph unicode="&#xe046;" d="M500 1100h200q56 0 102.5 -20.5t72.5 -50t44 -59t25 -50.5l6 -20h150q41 0 70.5 -29.5t29.5 -70.5v-600q0 -41 -29.5 -70.5t-70.5 -29.5h-1000q-41 0 -70.5 29.5t-29.5 70.5v600q0 41 29.5 70.5t70.5 29.5h150q2 8 6.5 21.5t24 48t45 61t72 48t102.5 21.5zM900 800v-100 h100v100h-100zM600 730q-95 0 -162.5 -67.5t-67.5 -162.5t67.5 -162.5t162.5 -67.5t162.5 67.5t67.5 162.5t-67.5 162.5t-162.5 67.5zM600 603q43 0 73 -30t30 -73t-30 -73t-73 -30t-73 30t-30 73t30 73t73 30z" />
<glyph unicode="&#xe047;" d="M681 1199l385 -998q20 -50 60 -92q18 -19 36.5 -29.5t27.5 -11.5l10 -2v-66h-417v66q53 0 75 43.5t5 88.5l-82 222h-391q-58 -145 -92 -234q-11 -34 -6.5 -57t25.5 -37t46 -20t55 -6v-66h-365v66q56 24 84 52q12 12 25 30.5t20 31.5l7 13l399 1006h93zM416 521h340 l-162 457z" />
<glyph unicode="&#xe048;" d="M753 641q5 -1 14.5 -4.5t36 -15.5t50.5 -26.5t53.5 -40t50.5 -54.5t35.5 -70t14.5 -87q0 -67 -27.5 -125.5t-71.5 -97.5t-98.5 -66.5t-108.5 -40.5t-102 -13h-500v89q41 7 70.5 32.5t29.5 65.5v827q0 24 -0.5 34t-3.5 24t-8.5 19.5t-17 13.5t-28 12.5t-42.5 11.5v71 l471 -1q57 0 115.5 -20.5t108 -57t80.5 -94t31 -124.5q0 -51 -15.5 -96.5t-38 -74.5t-45 -50.5t-38.5 -30.5zM400 700h139q78 0 130.5 48.5t52.5 122.5q0 41 -8.5 70.5t-29.5 55.5t-62.5 39.5t-103.5 13.5h-118v-350zM400 200h216q80 0 121 50.5t41 130.5q0 90 -62.5 154.5 t-156.5 64.5h-159v-400z" />
<glyph unicode="&#xe049;" d="M877 1200l2 -57q-83 -19 -116 -45.5t-40 -66.5l-132 -839q-9 -49 13 -69t96 -26v-97h-500v97q186 16 200 98l173 832q3 17 3 30t-1.5 22.5t-9 17.5t-13.5 12.5t-21.5 10t-26 8.5t-33.5 10q-13 3 -19 5v57h425z" />
<glyph unicode="&#xe050;" d="M1300 900h-50q0 21 -4 37t-9.5 26.5t-18 17.5t-22 11t-28.5 5.5t-31 2t-37 0.5h-200v-850q0 -22 25 -34.5t50 -13.5l25 -2v-100h-400v100q4 0 11 0.5t24 3t30 7t24 15t11 24.5v850h-200q-25 0 -37 -0.5t-31 -2t-28.5 -5.5t-22 -11t-18 -17.5t-9.5 -26.5t-4 -37h-50v300 h1000v-300zM175 1000h-75v-800h75l-125 -167l-125 167h75v800h-75l125 167z" />
<glyph unicode="&#xe051;" d="M1100 900h-50q0 21 -4 37t-9.5 26.5t-18 17.5t-22 11t-28.5 5.5t-31 2t-37 0.5h-200v-650q0 -22 25 -34.5t50 -13.5l25 -2v-100h-400v100q4 0 11 0.5t24 3t30 7t24 15t11 24.5v650h-200q-25 0 -37 -0.5t-31 -2t-28.5 -5.5t-22 -11t-18 -17.5t-9.5 -26.5t-4 -37h-50v300 h1000v-300zM1167 50l-167 -125v75h-800v-75l-167 125l167 125v-75h800v75z" />
<glyph unicode="&#xe052;" d="M50 1100h600q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-600q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 800h1000q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1000q-21 0 -35.5 14.5t-14.5 35.5v100 q0 21 14.5 35.5t35.5 14.5zM50 500h800q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 200h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe053;" d="M250 1100h700q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-700q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 800h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v100 q0 21 14.5 35.5t35.5 14.5zM250 500h700q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-700q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 200h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe054;" d="M500 950v100q0 21 14.5 35.5t35.5 14.5h600q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-600q-21 0 -35.5 14.5t-14.5 35.5zM100 650v100q0 21 14.5 35.5t35.5 14.5h1000q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1000 q-21 0 -35.5 14.5t-14.5 35.5zM300 350v100q0 21 14.5 35.5t35.5 14.5h800q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5zM0 50v100q0 21 14.5 35.5t35.5 14.5h1100q21 0 35.5 -14.5t14.5 -35.5v-100 q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5z" />
<glyph unicode="&#xe055;" d="M50 1100h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 800h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v100 q0 21 14.5 35.5t35.5 14.5zM50 500h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 200h1100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe056;" d="M50 1100h100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM350 1100h800q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v100 q0 21 14.5 35.5t35.5 14.5zM50 800h100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM350 800h800q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-800 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 500h100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM350 500h800q21 0 35.5 -14.5t14.5 -35.5v-100 q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 200h100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM350 200h800 q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe057;" d="M400 0h-100v1100h100v-1100zM550 1100h100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM550 800h500q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-500 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM267 550l-167 -125v75h-200v100h200v75zM550 500h300q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-300q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM550 200h600 q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-600q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe058;" d="M50 1100h100q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM900 0h-100v1100h100v-1100zM50 800h500q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-500 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM1100 600h200v-100h-200v-75l-167 125l167 125v-75zM50 500h300q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-300q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5zM50 200h600 q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-600q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe059;" d="M75 1000h750q31 0 53 -22t22 -53v-650q0 -31 -22 -53t-53 -22h-750q-31 0 -53 22t-22 53v650q0 31 22 53t53 22zM1200 300l-300 300l300 300v-600z" />
<glyph unicode="&#xe060;" d="M44 1100h1112q18 0 31 -13t13 -31v-1012q0 -18 -13 -31t-31 -13h-1112q-18 0 -31 13t-13 31v1012q0 18 13 31t31 13zM100 1000v-737l247 182l298 -131l-74 156l293 318l236 -288v500h-1000zM342 884q56 0 95 -39t39 -94.5t-39 -95t-95 -39.5t-95 39.5t-39 95t39 94.5 t95 39z" />
<glyph unicode="&#xe062;" d="M648 1169q117 0 216 -60t156.5 -161t57.5 -218q0 -115 -70 -258q-69 -109 -158 -225.5t-143 -179.5l-54 -62q-9 8 -25.5 24.5t-63.5 67.5t-91 103t-98.5 128t-95.5 148q-60 132 -60 249q0 88 34 169.5t91.5 142t137 96.5t166.5 36zM652.5 974q-91.5 0 -156.5 -65 t-65 -157t65 -156.5t156.5 -64.5t156.5 64.5t65 156.5t-65 157t-156.5 65z" />
<glyph unicode="&#xe063;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 173v854q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57z" />
<glyph unicode="&#xe064;" d="M554 1295q21 -72 57.5 -143.5t76 -130t83 -118t82.5 -117t70 -116t49.5 -126t18.5 -136.5q0 -71 -25.5 -135t-68.5 -111t-99 -82t-118.5 -54t-125.5 -23q-84 5 -161.5 34t-139.5 78.5t-99 125t-37 164.5q0 69 18 136.5t49.5 126.5t69.5 116.5t81.5 117.5t83.5 119 t76.5 131t58.5 143zM344 710q-23 -33 -43.5 -70.5t-40.5 -102.5t-17 -123q1 -37 14.5 -69.5t30 -52t41 -37t38.5 -24.5t33 -15q21 -7 32 -1t13 22l6 34q2 10 -2.5 22t-13.5 19q-5 4 -14 12t-29.5 40.5t-32.5 73.5q-26 89 6 271q2 11 -6 11q-8 1 -15 -10z" />
<glyph unicode="&#xe065;" d="M1000 1013l108 115q2 1 5 2t13 2t20.5 -1t25 -9.5t28.5 -21.5q22 -22 27 -43t0 -32l-6 -10l-108 -115zM350 1100h400q50 0 105 -13l-187 -187h-368q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5v182l200 200v-332 q0 -165 -93.5 -257.5t-256.5 -92.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400q0 165 92.5 257.5t257.5 92.5zM1009 803l-362 -362l-161 -50l55 170l355 355z" />
<glyph unicode="&#xe066;" d="M350 1100h361q-164 -146 -216 -200h-195q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5l200 153v-103q0 -165 -92.5 -257.5t-257.5 -92.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400q0 165 92.5 257.5t257.5 92.5z M824 1073l339 -301q8 -7 8 -17.5t-8 -17.5l-340 -306q-7 -6 -12.5 -4t-6.5 11v203q-26 1 -54.5 0t-78.5 -7.5t-92 -17.5t-86 -35t-70 -57q10 59 33 108t51.5 81.5t65 58.5t68.5 40.5t67 24.5t56 13.5t40 4.5v210q1 10 6.5 12.5t13.5 -4.5z" />
<glyph unicode="&#xe067;" d="M350 1100h350q60 0 127 -23l-178 -177h-349q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5v69l200 200v-219q0 -165 -92.5 -257.5t-257.5 -92.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400q0 165 92.5 257.5t257.5 92.5z M643 639l395 395q7 7 17.5 7t17.5 -7l101 -101q7 -7 7 -17.5t-7 -17.5l-531 -532q-7 -7 -17.5 -7t-17.5 7l-248 248q-7 7 -7 17.5t7 17.5l101 101q7 7 17.5 7t17.5 -7l111 -111q8 -7 18 -7t18 7z" />
<glyph unicode="&#xe068;" d="M318 918l264 264q8 8 18 8t18 -8l260 -264q7 -8 4.5 -13t-12.5 -5h-170v-200h200v173q0 10 5 12t13 -5l264 -260q8 -7 8 -17.5t-8 -17.5l-264 -265q-8 -7 -13 -5t-5 12v173h-200v-200h170q10 0 12.5 -5t-4.5 -13l-260 -264q-8 -8 -18 -8t-18 8l-264 264q-8 8 -5.5 13 t12.5 5h175v200h-200v-173q0 -10 -5 -12t-13 5l-264 265q-8 7 -8 17.5t8 17.5l264 260q8 7 13 5t5 -12v-173h200v200h-175q-10 0 -12.5 5t5.5 13z" />
<glyph unicode="&#xe069;" d="M250 1100h100q21 0 35.5 -14.5t14.5 -35.5v-438l464 453q15 14 25.5 10t10.5 -25v-1000q0 -21 -10.5 -25t-25.5 10l-464 453v-438q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v1000q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe070;" d="M50 1100h100q21 0 35.5 -14.5t14.5 -35.5v-438l464 453q15 14 25.5 10t10.5 -25v-438l464 453q15 14 25.5 10t10.5 -25v-1000q0 -21 -10.5 -25t-25.5 10l-464 453v-438q0 -21 -10.5 -25t-25.5 10l-464 453v-438q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5 t-14.5 35.5v1000q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe071;" d="M1200 1050v-1000q0 -21 -10.5 -25t-25.5 10l-464 453v-438q0 -21 -10.5 -25t-25.5 10l-492 480q-15 14 -15 35t15 35l492 480q15 14 25.5 10t10.5 -25v-438l464 453q15 14 25.5 10t10.5 -25z" />
<glyph unicode="&#xe072;" d="M243 1074l814 -498q18 -11 18 -26t-18 -26l-814 -498q-18 -11 -30.5 -4t-12.5 28v1000q0 21 12.5 28t30.5 -4z" />
<glyph unicode="&#xe073;" d="M250 1000h200q21 0 35.5 -14.5t14.5 -35.5v-800q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v800q0 21 14.5 35.5t35.5 14.5zM650 1000h200q21 0 35.5 -14.5t14.5 -35.5v-800q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v800 q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe074;" d="M1100 950v-800q0 -21 -14.5 -35.5t-35.5 -14.5h-800q-21 0 -35.5 14.5t-14.5 35.5v800q0 21 14.5 35.5t35.5 14.5h800q21 0 35.5 -14.5t14.5 -35.5z" />
<glyph unicode="&#xe075;" d="M500 612v438q0 21 10.5 25t25.5 -10l492 -480q15 -14 15 -35t-15 -35l-492 -480q-15 -14 -25.5 -10t-10.5 25v438l-464 -453q-15 -14 -25.5 -10t-10.5 25v1000q0 21 10.5 25t25.5 -10z" />
<glyph unicode="&#xe076;" d="M1048 1102l100 1q20 0 35 -14.5t15 -35.5l5 -1000q0 -21 -14.5 -35.5t-35.5 -14.5l-100 -1q-21 0 -35.5 14.5t-14.5 35.5l-2 437l-463 -454q-14 -15 -24.5 -10.5t-10.5 25.5l-2 437l-462 -455q-15 -14 -25.5 -9.5t-10.5 24.5l-5 1000q0 21 10.5 25.5t25.5 -10.5l466 -450 l-2 438q0 20 10.5 24.5t25.5 -9.5l466 -451l-2 438q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe077;" d="M850 1100h100q21 0 35.5 -14.5t14.5 -35.5v-1000q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v438l-464 -453q-15 -14 -25.5 -10t-10.5 25v1000q0 21 10.5 25t25.5 -10l464 -453v438q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe078;" d="M686 1081l501 -540q15 -15 10.5 -26t-26.5 -11h-1042q-22 0 -26.5 11t10.5 26l501 540q15 15 36 15t36 -15zM150 400h1000q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1000q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe079;" d="M885 900l-352 -353l352 -353l-197 -198l-552 552l552 550z" />
<glyph unicode="&#xe080;" d="M1064 547l-551 -551l-198 198l353 353l-353 353l198 198z" />
<glyph unicode="&#xe081;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM650 900h-100q-21 0 -35.5 -14.5t-14.5 -35.5v-150h-150 q-21 0 -35.5 -14.5t-14.5 -35.5v-100q0 -21 14.5 -35.5t35.5 -14.5h150v-150q0 -21 14.5 -35.5t35.5 -14.5h100q21 0 35.5 14.5t14.5 35.5v150h150q21 0 35.5 14.5t14.5 35.5v100q0 21 -14.5 35.5t-35.5 14.5h-150v150q0 21 -14.5 35.5t-35.5 14.5z" />
<glyph unicode="&#xe082;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM850 700h-500q-21 0 -35.5 -14.5t-14.5 -35.5v-100q0 -21 14.5 -35.5 t35.5 -14.5h500q21 0 35.5 14.5t14.5 35.5v100q0 21 -14.5 35.5t-35.5 14.5z" />
<glyph unicode="&#xe083;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM741.5 913q-12.5 0 -21.5 -9l-120 -120l-120 120q-9 9 -21.5 9 t-21.5 -9l-141 -141q-9 -9 -9 -21.5t9 -21.5l120 -120l-120 -120q-9 -9 -9 -21.5t9 -21.5l141 -141q9 -9 21.5 -9t21.5 9l120 120l120 -120q9 -9 21.5 -9t21.5 9l141 141q9 9 9 21.5t-9 21.5l-120 120l120 120q9 9 9 21.5t-9 21.5l-141 141q-9 9 -21.5 9z" />
<glyph unicode="&#xe084;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM546 623l-84 85q-7 7 -17.5 7t-18.5 -7l-139 -139q-7 -8 -7 -18t7 -18 l242 -241q7 -8 17.5 -8t17.5 8l375 375q7 7 7 17.5t-7 18.5l-139 139q-7 7 -17.5 7t-17.5 -7z" />
<glyph unicode="&#xe085;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM588 941q-29 0 -59 -5.5t-63 -20.5t-58 -38.5t-41.5 -63t-16.5 -89.5 q0 -25 20 -25h131q30 -5 35 11q6 20 20.5 28t45.5 8q20 0 31.5 -10.5t11.5 -28.5q0 -23 -7 -34t-26 -18q-1 0 -13.5 -4t-19.5 -7.5t-20 -10.5t-22 -17t-18.5 -24t-15.5 -35t-8 -46q-1 -8 5.5 -16.5t20.5 -8.5h173q7 0 22 8t35 28t37.5 48t29.5 74t12 100q0 47 -17 83 t-42.5 57t-59.5 34.5t-64 18t-59 4.5zM675 400h-150q-10 0 -17.5 -7.5t-7.5 -17.5v-150q0 -10 7.5 -17.5t17.5 -7.5h150q10 0 17.5 7.5t7.5 17.5v150q0 10 -7.5 17.5t-17.5 7.5z" />
<glyph unicode="&#xe086;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM675 1000h-150q-10 0 -17.5 -7.5t-7.5 -17.5v-150q0 -10 7.5 -17.5 t17.5 -7.5h150q10 0 17.5 7.5t7.5 17.5v150q0 10 -7.5 17.5t-17.5 7.5zM675 700h-250q-10 0 -17.5 -7.5t-7.5 -17.5v-50q0 -10 7.5 -17.5t17.5 -7.5h75v-200h-75q-10 0 -17.5 -7.5t-7.5 -17.5v-50q0 -10 7.5 -17.5t17.5 -7.5h350q10 0 17.5 7.5t7.5 17.5v50q0 10 -7.5 17.5 t-17.5 7.5h-75v275q0 10 -7.5 17.5t-17.5 7.5z" />
<glyph unicode="&#xe087;" d="M525 1200h150q10 0 17.5 -7.5t7.5 -17.5v-194q103 -27 178.5 -102.5t102.5 -178.5h194q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-194q-27 -103 -102.5 -178.5t-178.5 -102.5v-194q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v194 q-103 27 -178.5 102.5t-102.5 178.5h-194q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5h194q27 103 102.5 178.5t178.5 102.5v194q0 10 7.5 17.5t17.5 7.5zM700 893v-168q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v168q-68 -23 -119 -74 t-74 -119h168q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-168q23 -68 74 -119t119 -74v168q0 10 7.5 17.5t17.5 7.5h150q10 0 17.5 -7.5t7.5 -17.5v-168q68 23 119 74t74 119h-168q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5h168 q-23 68 -74 119t-119 74z" />
<glyph unicode="&#xe088;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5t-57 214.5t-155.5 155.5t-214.5 57zM759 823l64 -64q7 -7 7 -17.5t-7 -17.5l-124 -124l124 -124q7 -7 7 -17.5t-7 -17.5l-64 -64q-7 -7 -17.5 -7t-17.5 7l-124 124l-124 -124q-7 -7 -17.5 -7t-17.5 7l-64 64 q-7 7 -7 17.5t7 17.5l124 124l-124 124q-7 7 -7 17.5t7 17.5l64 64q7 7 17.5 7t17.5 -7l124 -124l124 124q7 7 17.5 7t17.5 -7z" />
<glyph unicode="&#xe089;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5t57 -214.5 t155.5 -155.5t214.5 -57t214.5 57t155.5 155.5t57 214.5t-57 214.5t-155.5 155.5t-214.5 57zM782 788l106 -106q7 -7 7 -17.5t-7 -17.5l-320 -321q-8 -7 -18 -7t-18 7l-202 203q-8 7 -8 17.5t8 17.5l106 106q7 8 17.5 8t17.5 -8l79 -79l197 197q7 7 17.5 7t17.5 -7z" />
<glyph unicode="&#xe090;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM600 1027q-116 0 -214.5 -57t-155.5 -155.5t-57 -214.5q0 -120 65 -225 l587 587q-105 65 -225 65zM965 819l-584 -584q104 -62 219 -62q116 0 214.5 57t155.5 155.5t57 214.5q0 115 -62 219z" />
<glyph unicode="&#xe091;" d="M39 582l522 427q16 13 27.5 8t11.5 -26v-291h550q21 0 35.5 -14.5t14.5 -35.5v-200q0 -21 -14.5 -35.5t-35.5 -14.5h-550v-291q0 -21 -11.5 -26t-27.5 8l-522 427q-16 13 -16 32t16 32z" />
<glyph unicode="&#xe092;" d="M639 1009l522 -427q16 -13 16 -32t-16 -32l-522 -427q-16 -13 -27.5 -8t-11.5 26v291h-550q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5h550v291q0 21 11.5 26t27.5 -8z" />
<glyph unicode="&#xe093;" d="M682 1161l427 -522q13 -16 8 -27.5t-26 -11.5h-291v-550q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v550h-291q-21 0 -26 11.5t8 27.5l427 522q13 16 32 16t32 -16z" />
<glyph unicode="&#xe094;" d="M550 1200h200q21 0 35.5 -14.5t14.5 -35.5v-550h291q21 0 26 -11.5t-8 -27.5l-427 -522q-13 -16 -32 -16t-32 16l-427 522q-13 16 -8 27.5t26 11.5h291v550q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe095;" d="M639 1109l522 -427q16 -13 16 -32t-16 -32l-522 -427q-16 -13 -27.5 -8t-11.5 26v291q-94 -2 -182 -20t-170.5 -52t-147 -92.5t-100.5 -135.5q5 105 27 193.5t67.5 167t113 135t167 91.5t225.5 42v262q0 21 11.5 26t27.5 -8z" />
<glyph unicode="&#xe096;" d="M850 1200h300q21 0 35.5 -14.5t14.5 -35.5v-300q0 -21 -10.5 -25t-24.5 10l-94 94l-249 -249q-8 -7 -18 -7t-18 7l-106 106q-7 8 -7 18t7 18l249 249l-94 94q-14 14 -10 24.5t25 10.5zM350 0h-300q-21 0 -35.5 14.5t-14.5 35.5v300q0 21 10.5 25t24.5 -10l94 -94l249 249 q8 7 18 7t18 -7l106 -106q7 -8 7 -18t-7 -18l-249 -249l94 -94q14 -14 10 -24.5t-25 -10.5z" />
<glyph unicode="&#xe097;" d="M1014 1120l106 -106q7 -8 7 -18t-7 -18l-249 -249l94 -94q14 -14 10 -24.5t-25 -10.5h-300q-21 0 -35.5 14.5t-14.5 35.5v300q0 21 10.5 25t24.5 -10l94 -94l249 249q8 7 18 7t18 -7zM250 600h300q21 0 35.5 -14.5t14.5 -35.5v-300q0 -21 -10.5 -25t-24.5 10l-94 94 l-249 -249q-8 -7 -18 -7t-18 7l-106 106q-7 8 -7 18t7 18l249 249l-94 94q-14 14 -10 24.5t25 10.5z" />
<glyph unicode="&#xe101;" d="M600 1177q117 0 224 -45.5t184.5 -123t123 -184.5t45.5 -224t-45.5 -224t-123 -184.5t-184.5 -123t-224 -45.5t-224 45.5t-184.5 123t-123 184.5t-45.5 224t45.5 224t123 184.5t184.5 123t224 45.5zM704 900h-208q-20 0 -32 -14.5t-8 -34.5l58 -302q4 -20 21.5 -34.5 t37.5 -14.5h54q20 0 37.5 14.5t21.5 34.5l58 302q4 20 -8 34.5t-32 14.5zM675 400h-150q-10 0 -17.5 -7.5t-7.5 -17.5v-150q0 -10 7.5 -17.5t17.5 -7.5h150q10 0 17.5 7.5t7.5 17.5v150q0 10 -7.5 17.5t-17.5 7.5z" />
<glyph unicode="&#xe102;" d="M260 1200q9 0 19 -2t15 -4l5 -2q22 -10 44 -23l196 -118q21 -13 36 -24q29 -21 37 -12q11 13 49 35l196 118q22 13 45 23q17 7 38 7q23 0 47 -16.5t37 -33.5l13 -16q14 -21 18 -45l25 -123l8 -44q1 -9 8.5 -14.5t17.5 -5.5h61q10 0 17.5 -7.5t7.5 -17.5v-50 q0 -10 -7.5 -17.5t-17.5 -7.5h-50q-10 0 -17.5 -7.5t-7.5 -17.5v-175h-400v300h-200v-300h-400v175q0 10 -7.5 17.5t-17.5 7.5h-50q-10 0 -17.5 7.5t-7.5 17.5v50q0 10 7.5 17.5t17.5 7.5h61q11 0 18 3t7 8q0 4 9 52l25 128q5 25 19 45q2 3 5 7t13.5 15t21.5 19.5t26.5 15.5 t29.5 7zM915 1079l-166 -162q-7 -7 -5 -12t12 -5h219q10 0 15 7t2 17l-51 149q-3 10 -11 12t-15 -6zM463 917l-177 157q-8 7 -16 5t-11 -12l-51 -143q-3 -10 2 -17t15 -7h231q11 0 12.5 5t-5.5 12zM500 0h-375q-10 0 -17.5 7.5t-7.5 17.5v375h400v-400zM1100 400v-375 q0 -10 -7.5 -17.5t-17.5 -7.5h-375v400h400z" />
<glyph unicode="&#xe103;" d="M1165 1190q8 3 21 -6.5t13 -17.5q-2 -178 -24.5 -323.5t-55.5 -245.5t-87 -174.5t-102.5 -118.5t-118 -68.5t-118.5 -33t-120 -4.5t-105 9.5t-90 16.5q-61 12 -78 11q-4 1 -12.5 0t-34 -14.5t-52.5 -40.5l-153 -153q-26 -24 -37 -14.5t-11 43.5q0 64 42 102q8 8 50.5 45 t66.5 58q19 17 35 47t13 61q-9 55 -10 102.5t7 111t37 130t78 129.5q39 51 80 88t89.5 63.5t94.5 45t113.5 36t129 31t157.5 37t182 47.5zM1116 1098q-8 9 -22.5 -3t-45.5 -50q-38 -47 -119 -103.5t-142 -89.5l-62 -33q-56 -30 -102 -57t-104 -68t-102.5 -80.5t-85.5 -91 t-64 -104.5q-24 -56 -31 -86t2 -32t31.5 17.5t55.5 59.5q25 30 94 75.5t125.5 77.5t147.5 81q70 37 118.5 69t102 79.5t99 111t86.5 148.5q22 50 24 60t-6 19z" />
<glyph unicode="&#xe104;" d="M653 1231q-39 -67 -54.5 -131t-10.5 -114.5t24.5 -96.5t47.5 -80t63.5 -62.5t68.5 -46.5t65 -30q-4 7 -17.5 35t-18.5 39.5t-17 39.5t-17 43t-13 42t-9.5 44.5t-2 42t4 43t13.5 39t23 38.5q96 -42 165 -107.5t105 -138t52 -156t13 -159t-19 -149.5q-13 -55 -44 -106.5 t-68 -87t-78.5 -64.5t-72.5 -45t-53 -22q-72 -22 -127 -11q-31 6 -13 19q6 3 17 7q13 5 32.5 21t41 44t38.5 63.5t21.5 81.5t-6.5 94.5t-50 107t-104 115.5q10 -104 -0.5 -189t-37 -140.5t-65 -93t-84 -52t-93.5 -11t-95 24.5q-80 36 -131.5 114t-53.5 171q-2 23 0 49.5 t4.5 52.5t13.5 56t27.5 60t46 64.5t69.5 68.5q-8 -53 -5 -102.5t17.5 -90t34 -68.5t44.5 -39t49 -2q31 13 38.5 36t-4.5 55t-29 64.5t-36 75t-26 75.5q-15 85 2 161.5t53.5 128.5t85.5 92.5t93.5 61t81.5 25.5z" />
<glyph unicode="&#xe105;" d="M600 1094q82 0 160.5 -22.5t140 -59t116.5 -82.5t94.5 -95t68 -95t42.5 -82.5t14 -57.5t-14 -57.5t-43 -82.5t-68.5 -95t-94.5 -95t-116.5 -82.5t-140 -59t-159.5 -22.5t-159.5 22.5t-140 59t-116.5 82.5t-94.5 95t-68.5 95t-43 82.5t-14 57.5t14 57.5t42.5 82.5t68 95 t94.5 95t116.5 82.5t140 59t160.5 22.5zM888 829q-15 15 -18 12t5 -22q25 -57 25 -119q0 -124 -88 -212t-212 -88t-212 88t-88 212q0 59 23 114q8 19 4.5 22t-17.5 -12q-70 -69 -160 -184q-13 -16 -15 -40.5t9 -42.5q22 -36 47 -71t70 -82t92.5 -81t113 -58.5t133.5 -24.5 t133.5 24t113 58.5t92.5 81.5t70 81.5t47 70.5q11 18 9 42.5t-14 41.5q-90 117 -163 189zM448 727l-35 -36q-15 -15 -19.5 -38.5t4.5 -41.5q37 -68 93 -116q16 -13 38.5 -11t36.5 17l35 34q14 15 12.5 33.5t-16.5 33.5q-44 44 -89 117q-11 18 -28 20t-32 -12z" />
<glyph unicode="&#xe106;" d="M592 0h-148l31 120q-91 20 -175.5 68.5t-143.5 106.5t-103.5 119t-66.5 110t-22 76q0 21 14 57.5t42.5 82.5t68 95t94.5 95t116.5 82.5t140 59t160.5 22.5q61 0 126 -15l32 121h148zM944 770l47 181q108 -85 176.5 -192t68.5 -159q0 -26 -19.5 -71t-59.5 -102t-93 -112 t-129 -104.5t-158 -75.5l46 173q77 49 136 117t97 131q11 18 9 42.5t-14 41.5q-54 70 -107 130zM310 824q-70 -69 -160 -184q-13 -16 -15 -40.5t9 -42.5q18 -30 39 -60t57 -70.5t74 -73t90 -61t105 -41.5l41 154q-107 18 -178.5 101.5t-71.5 193.5q0 59 23 114q8 19 4.5 22 t-17.5 -12zM448 727l-35 -36q-15 -15 -19.5 -38.5t4.5 -41.5q37 -68 93 -116q16 -13 38.5 -11t36.5 17l12 11l22 86l-3 4q-44 44 -89 117q-11 18 -28 20t-32 -12z" />
<glyph unicode="&#xe107;" d="M-90 100l642 1066q20 31 48 28.5t48 -35.5l642 -1056q21 -32 7.5 -67.5t-50.5 -35.5h-1294q-37 0 -50.5 34t7.5 66zM155 200h345v75q0 10 7.5 17.5t17.5 7.5h150q10 0 17.5 -7.5t7.5 -17.5v-75h345l-445 723zM496 700h208q20 0 32 -14.5t8 -34.5l-58 -252 q-4 -20 -21.5 -34.5t-37.5 -14.5h-54q-20 0 -37.5 14.5t-21.5 34.5l-58 252q-4 20 8 34.5t32 14.5z" />
<glyph unicode="&#xe108;" d="M650 1200q62 0 106 -44t44 -106v-339l363 -325q15 -14 26 -38.5t11 -44.5v-41q0 -20 -12 -26.5t-29 5.5l-359 249v-263q100 -93 100 -113v-64q0 -21 -13 -29t-32 1l-205 128l-205 -128q-19 -9 -32 -1t-13 29v64q0 20 100 113v263l-359 -249q-17 -12 -29 -5.5t-12 26.5v41 q0 20 11 44.5t26 38.5l363 325v339q0 62 44 106t106 44z" />
<glyph unicode="&#xe109;" d="M850 1200h100q21 0 35.5 -14.5t14.5 -35.5v-50h50q21 0 35.5 -14.5t14.5 -35.5v-150h-1100v150q0 21 14.5 35.5t35.5 14.5h50v50q0 21 14.5 35.5t35.5 14.5h100q21 0 35.5 -14.5t14.5 -35.5v-50h500v50q0 21 14.5 35.5t35.5 14.5zM1100 800v-750q0 -21 -14.5 -35.5 t-35.5 -14.5h-1000q-21 0 -35.5 14.5t-14.5 35.5v750h1100zM100 600v-100h100v100h-100zM300 600v-100h100v100h-100zM500 600v-100h100v100h-100zM700 600v-100h100v100h-100zM900 600v-100h100v100h-100zM100 400v-100h100v100h-100zM300 400v-100h100v100h-100zM500 400 v-100h100v100h-100zM700 400v-100h100v100h-100zM900 400v-100h100v100h-100zM100 200v-100h100v100h-100zM300 200v-100h100v100h-100zM500 200v-100h100v100h-100zM700 200v-100h100v100h-100zM900 200v-100h100v100h-100z" />
<glyph unicode="&#xe110;" d="M1135 1165l249 -230q15 -14 15 -35t-15 -35l-249 -230q-14 -14 -24.5 -10t-10.5 25v150h-159l-600 -600h-291q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h209l600 600h241v150q0 21 10.5 25t24.5 -10zM522 819l-141 -141l-122 122h-209q-21 0 -35.5 14.5 t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h291zM1135 565l249 -230q15 -14 15 -35t-15 -35l-249 -230q-14 -14 -24.5 -10t-10.5 25v150h-241l-181 181l141 141l122 -122h159v150q0 21 10.5 25t24.5 -10z" />
<glyph unicode="&#xe111;" d="M100 1100h1000q41 0 70.5 -29.5t29.5 -70.5v-600q0 -41 -29.5 -70.5t-70.5 -29.5h-596l-304 -300v300h-100q-41 0 -70.5 29.5t-29.5 70.5v600q0 41 29.5 70.5t70.5 29.5z" />
<glyph unicode="&#xe112;" d="M150 1200h200q21 0 35.5 -14.5t14.5 -35.5v-250h-300v250q0 21 14.5 35.5t35.5 14.5zM850 1200h200q21 0 35.5 -14.5t14.5 -35.5v-250h-300v250q0 21 14.5 35.5t35.5 14.5zM1100 800v-300q0 -41 -3 -77.5t-15 -89.5t-32 -96t-58 -89t-89 -77t-129 -51t-174 -20t-174 20 t-129 51t-89 77t-58 89t-32 96t-15 89.5t-3 77.5v300h300v-250v-27v-42.5t1.5 -41t5 -38t10 -35t16.5 -30t25.5 -24.5t35 -19t46.5 -12t60 -4t60 4.5t46.5 12.5t35 19.5t25 25.5t17 30.5t10 35t5 38t2 40.5t-0.5 42v25v250h300z" />
<glyph unicode="&#xe113;" d="M1100 411l-198 -199l-353 353l-353 -353l-197 199l551 551z" />
<glyph unicode="&#xe114;" d="M1101 789l-550 -551l-551 551l198 199l353 -353l353 353z" />
<glyph unicode="&#xe115;" d="M404 1000h746q21 0 35.5 -14.5t14.5 -35.5v-551h150q21 0 25 -10.5t-10 -24.5l-230 -249q-14 -15 -35 -15t-35 15l-230 249q-14 14 -10 24.5t25 10.5h150v401h-381zM135 984l230 -249q14 -14 10 -24.5t-25 -10.5h-150v-400h385l215 -200h-750q-21 0 -35.5 14.5 t-14.5 35.5v550h-150q-21 0 -25 10.5t10 24.5l230 249q14 15 35 15t35 -15z" />
<glyph unicode="&#xe116;" d="M56 1200h94q17 0 31 -11t18 -27l38 -162h896q24 0 39 -18.5t10 -42.5l-100 -475q-5 -21 -27 -42.5t-55 -21.5h-633l48 -200h535q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-50v-50q0 -21 -14.5 -35.5t-35.5 -14.5t-35.5 14.5t-14.5 35.5v50h-300v-50 q0 -21 -14.5 -35.5t-35.5 -14.5t-35.5 14.5t-14.5 35.5v50h-31q-18 0 -32.5 10t-20.5 19l-5 10l-201 961h-54q-20 0 -35 14.5t-15 35.5t15 35.5t35 14.5z" />
<glyph unicode="&#xe117;" d="M1200 1000v-100h-1200v100h200q0 41 29.5 70.5t70.5 29.5h300q41 0 70.5 -29.5t29.5 -70.5h500zM0 800h1200v-800h-1200v800z" />
<glyph unicode="&#xe118;" d="M200 800l-200 -400v600h200q0 41 29.5 70.5t70.5 29.5h300q42 0 71 -29.5t29 -70.5h500v-200h-1000zM1500 700l-300 -700h-1200l300 700h1200z" />
<glyph unicode="&#xe119;" d="M635 1184l230 -249q14 -14 10 -24.5t-25 -10.5h-150v-601h150q21 0 25 -10.5t-10 -24.5l-230 -249q-14 -15 -35 -15t-35 15l-230 249q-14 14 -10 24.5t25 10.5h150v601h-150q-21 0 -25 10.5t10 24.5l230 249q14 15 35 15t35 -15z" />
<glyph unicode="&#xe120;" d="M936 864l249 -229q14 -15 14 -35.5t-14 -35.5l-249 -229q-15 -15 -25.5 -10.5t-10.5 24.5v151h-600v-151q0 -20 -10.5 -24.5t-25.5 10.5l-249 229q-14 15 -14 35.5t14 35.5l249 229q15 15 25.5 10.5t10.5 -25.5v-149h600v149q0 21 10.5 25.5t25.5 -10.5z" />
<glyph unicode="&#xe121;" d="M1169 400l-172 732q-5 23 -23 45.5t-38 22.5h-672q-20 0 -38 -20t-23 -41l-172 -739h1138zM1100 300h-1000q-41 0 -70.5 -29.5t-29.5 -70.5v-100q0 -41 29.5 -70.5t70.5 -29.5h1000q41 0 70.5 29.5t29.5 70.5v100q0 41 -29.5 70.5t-70.5 29.5zM800 100v100h100v-100h-100 zM1000 100v100h100v-100h-100z" />
<glyph unicode="&#xe122;" d="M1150 1100q21 0 35.5 -14.5t14.5 -35.5v-850q0 -21 -14.5 -35.5t-35.5 -14.5t-35.5 14.5t-14.5 35.5v850q0 21 14.5 35.5t35.5 14.5zM1000 200l-675 200h-38l47 -276q3 -16 -5.5 -20t-29.5 -4h-7h-84q-20 0 -34.5 14t-18.5 35q-55 337 -55 351v250v6q0 16 1 23.5t6.5 14 t17.5 6.5h200l675 250v-850zM0 750v-250q-4 0 -11 0.5t-24 6t-30 15t-24 30t-11 48.5v50q0 26 10.5 46t25 30t29 16t25.5 7z" />
<glyph unicode="&#xe123;" d="M553 1200h94q20 0 29 -10.5t3 -29.5l-18 -37q83 -19 144 -82.5t76 -140.5l63 -327l118 -173h17q19 0 33 -14.5t14 -35t-13 -40.5t-31 -27q-8 -4 -23 -9.5t-65 -19.5t-103 -25t-132.5 -20t-158.5 -9q-57 0 -115 5t-104 12t-88.5 15.5t-73.5 17.5t-54.5 16t-35.5 12l-11 4 q-18 8 -31 28t-13 40.5t14 35t33 14.5h17l118 173l63 327q15 77 76 140t144 83l-18 32q-6 19 3.5 32t28.5 13zM498 110q50 -6 102 -6q53 0 102 6q-12 -49 -39.5 -79.5t-62.5 -30.5t-63 30.5t-39 79.5z" />
<glyph unicode="&#xe124;" d="M800 946l224 78l-78 -224l234 -45l-180 -155l180 -155l-234 -45l78 -224l-224 78l-45 -234l-155 180l-155 -180l-45 234l-224 -78l78 224l-234 45l180 155l-180 155l234 45l-78 224l224 -78l45 234l155 -180l155 180z" />
<glyph unicode="&#xe125;" d="M650 1200h50q40 0 70 -40.5t30 -84.5v-150l-28 -125h328q40 0 70 -40.5t30 -84.5v-100q0 -45 -29 -74l-238 -344q-16 -24 -38 -40.5t-45 -16.5h-250q-7 0 -42 25t-66 50l-31 25h-61q-45 0 -72.5 18t-27.5 57v400q0 36 20 63l145 196l96 198q13 28 37.5 48t51.5 20z M650 1100l-100 -212l-150 -213v-375h100l136 -100h214l250 375v125h-450l50 225v175h-50zM50 800h100q21 0 35.5 -14.5t14.5 -35.5v-500q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v500q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe126;" d="M600 1100h250q23 0 45 -16.5t38 -40.5l238 -344q29 -29 29 -74v-100q0 -44 -30 -84.5t-70 -40.5h-328q28 -118 28 -125v-150q0 -44 -30 -84.5t-70 -40.5h-50q-27 0 -51.5 20t-37.5 48l-96 198l-145 196q-20 27 -20 63v400q0 39 27.5 57t72.5 18h61q124 100 139 100z M50 1000h100q21 0 35.5 -14.5t14.5 -35.5v-500q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v500q0 21 14.5 35.5t35.5 14.5zM636 1000l-136 -100h-100v-375l150 -213l100 -212h50v175l-50 225h450v125l-250 375h-214z" />
<glyph unicode="&#xe127;" d="M356 873l363 230q31 16 53 -6l110 -112q13 -13 13.5 -32t-11.5 -34l-84 -121h302q84 0 138 -38t54 -110t-55 -111t-139 -39h-106l-131 -339q-6 -21 -19.5 -41t-28.5 -20h-342q-7 0 -90 81t-83 94v525q0 17 14 35.5t28 28.5zM400 792v-503l100 -89h293l131 339 q6 21 19.5 41t28.5 20h203q21 0 30.5 25t0.5 50t-31 25h-456h-7h-6h-5.5t-6 0.5t-5 1.5t-5 2t-4 2.5t-4 4t-2.5 4.5q-12 25 5 47l146 183l-86 83zM50 800h100q21 0 35.5 -14.5t14.5 -35.5v-500q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v500 q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe128;" d="M475 1103l366 -230q2 -1 6 -3.5t14 -10.5t18 -16.5t14.5 -20t6.5 -22.5v-525q0 -13 -86 -94t-93 -81h-342q-15 0 -28.5 20t-19.5 41l-131 339h-106q-85 0 -139.5 39t-54.5 111t54 110t138 38h302l-85 121q-11 15 -10.5 34t13.5 32l110 112q22 22 53 6zM370 945l146 -183 q17 -22 5 -47q-2 -2 -3.5 -4.5t-4 -4t-4 -2.5t-5 -2t-5 -1.5t-6 -0.5h-6h-6.5h-6h-475v-100h221q15 0 29 -20t20 -41l130 -339h294l106 89v503l-342 236zM1050 800h100q21 0 35.5 -14.5t14.5 -35.5v-500q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5 v500q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe129;" d="M550 1294q72 0 111 -55t39 -139v-106l339 -131q21 -6 41 -19.5t20 -28.5v-342q0 -7 -81 -90t-94 -83h-525q-17 0 -35.5 14t-28.5 28l-9 14l-230 363q-16 31 6 53l112 110q13 13 32 13.5t34 -11.5l121 -84v302q0 84 38 138t110 54zM600 972v203q0 21 -25 30.5t-50 0.5 t-25 -31v-456v-7v-6v-5.5t-0.5 -6t-1.5 -5t-2 -5t-2.5 -4t-4 -4t-4.5 -2.5q-25 -12 -47 5l-183 146l-83 -86l236 -339h503l89 100v293l-339 131q-21 6 -41 19.5t-20 28.5zM450 200h500q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-500 q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe130;" d="M350 1100h500q21 0 35.5 14.5t14.5 35.5v100q0 21 -14.5 35.5t-35.5 14.5h-500q-21 0 -35.5 -14.5t-14.5 -35.5v-100q0 -21 14.5 -35.5t35.5 -14.5zM600 306v-106q0 -84 -39 -139t-111 -55t-110 54t-38 138v302l-121 -84q-15 -12 -34 -11.5t-32 13.5l-112 110 q-22 22 -6 53l230 363q1 2 3.5 6t10.5 13.5t16.5 17t20 13.5t22.5 6h525q13 0 94 -83t81 -90v-342q0 -15 -20 -28.5t-41 -19.5zM308 900l-236 -339l83 -86l183 146q22 17 47 5q2 -1 4.5 -2.5t4 -4t2.5 -4t2 -5t1.5 -5t0.5 -6v-5.5v-6v-7v-456q0 -22 25 -31t50 0.5t25 30.5 v203q0 15 20 28.5t41 19.5l339 131v293l-89 100h-503z" />
<glyph unicode="&#xe131;" d="M600 1178q118 0 225 -45.5t184.5 -123t123 -184.5t45.5 -225t-45.5 -225t-123 -184.5t-184.5 -123t-225 -45.5t-225 45.5t-184.5 123t-123 184.5t-45.5 225t45.5 225t123 184.5t184.5 123t225 45.5zM914 632l-275 223q-16 13 -27.5 8t-11.5 -26v-137h-275 q-10 0 -17.5 -7.5t-7.5 -17.5v-150q0 -10 7.5 -17.5t17.5 -7.5h275v-137q0 -21 11.5 -26t27.5 8l275 223q16 13 16 32t-16 32z" />
<glyph unicode="&#xe132;" d="M600 1178q118 0 225 -45.5t184.5 -123t123 -184.5t45.5 -225t-45.5 -225t-123 -184.5t-184.5 -123t-225 -45.5t-225 45.5t-184.5 123t-123 184.5t-45.5 225t45.5 225t123 184.5t184.5 123t225 45.5zM561 855l-275 -223q-16 -13 -16 -32t16 -32l275 -223q16 -13 27.5 -8 t11.5 26v137h275q10 0 17.5 7.5t7.5 17.5v150q0 10 -7.5 17.5t-17.5 7.5h-275v137q0 21 -11.5 26t-27.5 -8z" />
<glyph unicode="&#xe133;" d="M600 1178q118 0 225 -45.5t184.5 -123t123 -184.5t45.5 -225t-45.5 -225t-123 -184.5t-184.5 -123t-225 -45.5t-225 45.5t-184.5 123t-123 184.5t-45.5 225t45.5 225t123 184.5t184.5 123t225 45.5zM855 639l-223 275q-13 16 -32 16t-32 -16l-223 -275q-13 -16 -8 -27.5 t26 -11.5h137v-275q0 -10 7.5 -17.5t17.5 -7.5h150q10 0 17.5 7.5t7.5 17.5v275h137q21 0 26 11.5t-8 27.5z" />
<glyph unicode="&#xe134;" d="M600 1178q118 0 225 -45.5t184.5 -123t123 -184.5t45.5 -225t-45.5 -225t-123 -184.5t-184.5 -123t-225 -45.5t-225 45.5t-184.5 123t-123 184.5t-45.5 225t45.5 225t123 184.5t184.5 123t225 45.5zM675 900h-150q-10 0 -17.5 -7.5t-7.5 -17.5v-275h-137q-21 0 -26 -11.5 t8 -27.5l223 -275q13 -16 32 -16t32 16l223 275q13 16 8 27.5t-26 11.5h-137v275q0 10 -7.5 17.5t-17.5 7.5z" />
<glyph unicode="&#xe135;" d="M600 1176q116 0 222.5 -46t184 -123.5t123.5 -184t46 -222.5t-46 -222.5t-123.5 -184t-184 -123.5t-222.5 -46t-222.5 46t-184 123.5t-123.5 184t-46 222.5t46 222.5t123.5 184t184 123.5t222.5 46zM627 1101q-15 -12 -36.5 -20.5t-35.5 -12t-43 -8t-39 -6.5 q-15 -3 -45.5 0t-45.5 -2q-20 -7 -51.5 -26.5t-34.5 -34.5q-3 -11 6.5 -22.5t8.5 -18.5q-3 -34 -27.5 -91t-29.5 -79q-9 -34 5 -93t8 -87q0 -9 17 -44.5t16 -59.5q12 0 23 -5t23.5 -15t19.5 -14q16 -8 33 -15t40.5 -15t34.5 -12q21 -9 52.5 -32t60 -38t57.5 -11 q7 -15 -3 -34t-22.5 -40t-9.5 -38q13 -21 23 -34.5t27.5 -27.5t36.5 -18q0 -7 -3.5 -16t-3.5 -14t5 -17q104 -2 221 112q30 29 46.5 47t34.5 49t21 63q-13 8 -37 8.5t-36 7.5q-15 7 -49.5 15t-51.5 19q-18 0 -41 -0.5t-43 -1.5t-42 -6.5t-38 -16.5q-51 -35 -66 -12 q-4 1 -3.5 25.5t0.5 25.5q-6 13 -26.5 17.5t-24.5 6.5q1 15 -0.5 30.5t-7 28t-18.5 11.5t-31 -21q-23 -25 -42 4q-19 28 -8 58q6 16 22 22q6 -1 26 -1.5t33.5 -4t19.5 -13.5q7 -12 18 -24t21.5 -20.5t20 -15t15.5 -10.5l5 -3q2 12 7.5 30.5t8 34.5t-0.5 32q-3 18 3.5 29 t18 22.5t15.5 24.5q6 14 10.5 35t8 31t15.5 22.5t34 22.5q-6 18 10 36q8 0 24 -1.5t24.5 -1.5t20 4.5t20.5 15.5q-10 23 -31 42.5t-37.5 29.5t-49 27t-43.5 23q0 1 2 8t3 11.5t1.5 10.5t-1 9.5t-4.5 4.5q31 -13 58.5 -14.5t38.5 2.5l12 5q5 28 -9.5 46t-36.5 24t-50 15 t-41 20q-18 -4 -37 0zM613 994q0 -17 8 -42t17 -45t9 -23q-8 1 -39.5 5.5t-52.5 10t-37 16.5q3 11 16 29.5t16 25.5q10 -10 19 -10t14 6t13.5 14.5t16.5 12.5z" />
<glyph unicode="&#xe136;" d="M756 1157q164 92 306 -9l-259 -138l145 -232l251 126q6 -89 -34 -156.5t-117 -110.5q-60 -34 -127 -39.5t-126 16.5l-596 -596q-15 -16 -36.5 -16t-36.5 16l-111 110q-15 15 -15 36.5t15 37.5l600 599q-34 101 5.5 201.5t135.5 154.5z" />
<glyph unicode="&#xe137;" horiz-adv-x="1220" d="M100 1196h1000q41 0 70.5 -29.5t29.5 -70.5v-100q0 -41 -29.5 -70.5t-70.5 -29.5h-1000q-41 0 -70.5 29.5t-29.5 70.5v100q0 41 29.5 70.5t70.5 29.5zM1100 1096h-200v-100h200v100zM100 796h1000q41 0 70.5 -29.5t29.5 -70.5v-100q0 -41 -29.5 -70.5t-70.5 -29.5h-1000 q-41 0 -70.5 29.5t-29.5 70.5v100q0 41 29.5 70.5t70.5 29.5zM1100 696h-500v-100h500v100zM100 396h1000q41 0 70.5 -29.5t29.5 -70.5v-100q0 -41 -29.5 -70.5t-70.5 -29.5h-1000q-41 0 -70.5 29.5t-29.5 70.5v100q0 41 29.5 70.5t70.5 29.5zM1100 296h-300v-100h300v100z " />
<glyph unicode="&#xe138;" d="M150 1200h900q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-900q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM700 500v-300l-200 -200v500l-350 500h900z" />
<glyph unicode="&#xe139;" d="M500 1200h200q41 0 70.5 -29.5t29.5 -70.5v-100h300q41 0 70.5 -29.5t29.5 -70.5v-400h-500v100h-200v-100h-500v400q0 41 29.5 70.5t70.5 29.5h300v100q0 41 29.5 70.5t70.5 29.5zM500 1100v-100h200v100h-200zM1200 400v-200q0 -41 -29.5 -70.5t-70.5 -29.5h-1000 q-41 0 -70.5 29.5t-29.5 70.5v200h1200z" />
<glyph unicode="&#xe140;" d="M50 1200h300q21 0 25 -10.5t-10 -24.5l-94 -94l199 -199q7 -8 7 -18t-7 -18l-106 -106q-8 -7 -18 -7t-18 7l-199 199l-94 -94q-14 -14 -24.5 -10t-10.5 25v300q0 21 14.5 35.5t35.5 14.5zM850 1200h300q21 0 35.5 -14.5t14.5 -35.5v-300q0 -21 -10.5 -25t-24.5 10l-94 94 l-199 -199q-8 -7 -18 -7t-18 7l-106 106q-7 8 -7 18t7 18l199 199l-94 94q-14 14 -10 24.5t25 10.5zM364 470l106 -106q7 -8 7 -18t-7 -18l-199 -199l94 -94q14 -14 10 -24.5t-25 -10.5h-300q-21 0 -35.5 14.5t-14.5 35.5v300q0 21 10.5 25t24.5 -10l94 -94l199 199 q8 7 18 7t18 -7zM1071 271l94 94q14 14 24.5 10t10.5 -25v-300q0 -21 -14.5 -35.5t-35.5 -14.5h-300q-21 0 -25 10.5t10 24.5l94 94l-199 199q-7 8 -7 18t7 18l106 106q8 7 18 7t18 -7z" />
<glyph unicode="&#xe141;" d="M596 1192q121 0 231.5 -47.5t190 -127t127 -190t47.5 -231.5t-47.5 -231.5t-127 -190.5t-190 -127t-231.5 -47t-231.5 47t-190.5 127t-127 190.5t-47 231.5t47 231.5t127 190t190.5 127t231.5 47.5zM596 1010q-112 0 -207.5 -55.5t-151 -151t-55.5 -207.5t55.5 -207.5 t151 -151t207.5 -55.5t207.5 55.5t151 151t55.5 207.5t-55.5 207.5t-151 151t-207.5 55.5zM454.5 905q22.5 0 38.5 -16t16 -38.5t-16 -39t-38.5 -16.5t-38.5 16.5t-16 39t16 38.5t38.5 16zM754.5 905q22.5 0 38.5 -16t16 -38.5t-16 -39t-38 -16.5q-14 0 -29 10l-55 -145 q17 -23 17 -51q0 -36 -25.5 -61.5t-61.5 -25.5t-61.5 25.5t-25.5 61.5q0 32 20.5 56.5t51.5 29.5l122 126l1 1q-9 14 -9 28q0 23 16 39t38.5 16zM345.5 709q22.5 0 38.5 -16t16 -38.5t-16 -38.5t-38.5 -16t-38.5 16t-16 38.5t16 38.5t38.5 16zM854.5 709q22.5 0 38.5 -16 t16 -38.5t-16 -38.5t-38.5 -16t-38.5 16t-16 38.5t16 38.5t38.5 16z" />
<glyph unicode="&#xe142;" d="M546 173l469 470q91 91 99 192q7 98 -52 175.5t-154 94.5q-22 4 -47 4q-34 0 -66.5 -10t-56.5 -23t-55.5 -38t-48 -41.5t-48.5 -47.5q-376 -375 -391 -390q-30 -27 -45 -41.5t-37.5 -41t-32 -46.5t-16 -47.5t-1.5 -56.5q9 -62 53.5 -95t99.5 -33q74 0 125 51l548 548 q36 36 20 75q-7 16 -21.5 26t-32.5 10q-26 0 -50 -23q-13 -12 -39 -38l-341 -338q-15 -15 -35.5 -15.5t-34.5 13.5t-14 34.5t14 34.5q327 333 361 367q35 35 67.5 51.5t78.5 16.5q14 0 29 -1q44 -8 74.5 -35.5t43.5 -68.5q14 -47 2 -96.5t-47 -84.5q-12 -11 -32 -32 t-79.5 -81t-114.5 -115t-124.5 -123.5t-123 -119.5t-96.5 -89t-57 -45q-56 -27 -120 -27q-70 0 -129 32t-93 89q-48 78 -35 173t81 163l511 511q71 72 111 96q91 55 198 55q80 0 152 -33q78 -36 129.5 -103t66.5 -154q17 -93 -11 -183.5t-94 -156.5l-482 -476 q-15 -15 -36 -16t-37 14t-17.5 34t14.5 35z" />
<glyph unicode="&#xe143;" d="M649 949q48 68 109.5 104t121.5 38.5t118.5 -20t102.5 -64t71 -100.5t27 -123q0 -57 -33.5 -117.5t-94 -124.5t-126.5 -127.5t-150 -152.5t-146 -174q-62 85 -145.5 174t-150 152.5t-126.5 127.5t-93.5 124.5t-33.5 117.5q0 64 28 123t73 100.5t104 64t119 20 t120.5 -38.5t104.5 -104zM896 972q-33 0 -64.5 -19t-56.5 -46t-47.5 -53.5t-43.5 -45.5t-37.5 -19t-36 19t-40 45.5t-43 53.5t-54 46t-65.5 19q-67 0 -122.5 -55.5t-55.5 -132.5q0 -23 13.5 -51t46 -65t57.5 -63t76 -75l22 -22q15 -14 44 -44t50.5 -51t46 -44t41 -35t23 -12 t23.5 12t42.5 36t46 44t52.5 52t44 43q4 4 12 13q43 41 63.5 62t52 55t46 55t26 46t11.5 44q0 79 -53 133.5t-120 54.5z" />
<glyph unicode="&#xe144;" d="M776.5 1214q93.5 0 159.5 -66l141 -141q66 -66 66 -160q0 -42 -28 -95.5t-62 -87.5l-29 -29q-31 53 -77 99l-18 18l95 95l-247 248l-389 -389l212 -212l-105 -106l-19 18l-141 141q-66 66 -66 159t66 159l283 283q65 66 158.5 66zM600 706l105 105q10 -8 19 -17l141 -141 q66 -66 66 -159t-66 -159l-283 -283q-66 -66 -159 -66t-159 66l-141 141q-66 66 -66 159.5t66 159.5l55 55q29 -55 75 -102l18 -17l-95 -95l247 -248l389 389z" />
<glyph unicode="&#xe145;" d="M603 1200q85 0 162 -15t127 -38t79 -48t29 -46v-953q0 -41 -29.5 -70.5t-70.5 -29.5h-600q-41 0 -70.5 29.5t-29.5 70.5v953q0 21 30 46.5t81 48t129 37.5t163 15zM300 1000v-700h600v700h-600zM600 254q-43 0 -73.5 -30.5t-30.5 -73.5t30.5 -73.5t73.5 -30.5t73.5 30.5 t30.5 73.5t-30.5 73.5t-73.5 30.5z" />
<glyph unicode="&#xe146;" d="M902 1185l283 -282q15 -15 15 -36t-14.5 -35.5t-35.5 -14.5t-35 15l-36 35l-279 -267v-300l-212 210l-308 -307l-280 -203l203 280l307 308l-210 212h300l267 279l-35 36q-15 14 -15 35t14.5 35.5t35.5 14.5t35 -15z" />
<glyph unicode="&#xe148;" d="M700 1248v-78q38 -5 72.5 -14.5t75.5 -31.5t71 -53.5t52 -84t24 -118.5h-159q-4 36 -10.5 59t-21 45t-40 35.5t-64.5 20.5v-307l64 -13q34 -7 64 -16.5t70 -32t67.5 -52.5t47.5 -80t20 -112q0 -139 -89 -224t-244 -97v-77h-100v79q-150 16 -237 103q-40 40 -52.5 93.5 t-15.5 139.5h139q5 -77 48.5 -126t117.5 -65v335l-27 8q-46 14 -79 26.5t-72 36t-63 52t-40 72.5t-16 98q0 70 25 126t67.5 92t94.5 57t110 27v77h100zM600 754v274q-29 -4 -50 -11t-42 -21.5t-31.5 -41.5t-10.5 -65q0 -29 7 -50.5t16.5 -34t28.5 -22.5t31.5 -14t37.5 -10 q9 -3 13 -4zM700 547v-310q22 2 42.5 6.5t45 15.5t41.5 27t29 42t12 59.5t-12.5 59.5t-38 44.5t-53 31t-66.5 24.5z" />
<glyph unicode="&#xe149;" d="M561 1197q84 0 160.5 -40t123.5 -109.5t47 -147.5h-153q0 40 -19.5 71.5t-49.5 48.5t-59.5 26t-55.5 9q-37 0 -79 -14.5t-62 -35.5q-41 -44 -41 -101q0 -26 13.5 -63t26.5 -61t37 -66q6 -9 9 -14h241v-100h-197q8 -50 -2.5 -115t-31.5 -95q-45 -62 -99 -112 q34 10 83 17.5t71 7.5q32 1 102 -16t104 -17q83 0 136 30l50 -147q-31 -19 -58 -30.5t-55 -15.5t-42 -4.5t-46 -0.5q-23 0 -76 17t-111 32.5t-96 11.5q-39 -3 -82 -16t-67 -25l-23 -11l-55 145q4 3 16 11t15.5 10.5t13 9t15.5 12t14.5 14t17.5 18.5q48 55 54 126.5 t-30 142.5h-221v100h166q-23 47 -44 104q-7 20 -12 41.5t-6 55.5t6 66.5t29.5 70.5t58.5 71q97 88 263 88z" />
<glyph unicode="&#xe150;" d="M400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM935 1184l230 -249q14 -14 10 -24.5t-25 -10.5h-150v-900h-200v900h-150q-21 0 -25 10.5t10 24.5l230 249q14 15 35 15t35 -15z" />
<glyph unicode="&#xe151;" d="M1000 700h-100v100h-100v-100h-100v500h300v-500zM400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM801 1100v-200h100v200h-100zM1000 350l-200 -250h200v-100h-300v150l200 250h-200v100h300v-150z " />
<glyph unicode="&#xe152;" d="M400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM1000 1050l-200 -250h200v-100h-300v150l200 250h-200v100h300v-150zM1000 0h-100v100h-100v-100h-100v500h300v-500zM801 400v-200h100v200h-100z " />
<glyph unicode="&#xe153;" d="M400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM1000 700h-100v400h-100v100h200v-500zM1100 0h-100v100h-200v400h300v-500zM901 400v-200h100v200h-100z" />
<glyph unicode="&#xe154;" d="M400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM1100 700h-100v100h-200v400h300v-500zM901 1100v-200h100v200h-100zM1000 0h-100v400h-100v100h200v-500z" />
<glyph unicode="&#xe155;" d="M400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM900 1000h-200v200h200v-200zM1000 700h-300v200h300v-200zM1100 400h-400v200h400v-200zM1200 100h-500v200h500v-200z" />
<glyph unicode="&#xe156;" d="M400 300h150q21 0 25 -11t-10 -25l-230 -250q-14 -15 -35 -15t-35 15l-230 250q-14 14 -10 25t25 11h150v900h200v-900zM1200 1000h-500v200h500v-200zM1100 700h-400v200h400v-200zM1000 400h-300v200h300v-200zM900 100h-200v200h200v-200z" />
<glyph unicode="&#xe157;" d="M350 1100h400q162 0 256 -93.5t94 -256.5v-400q0 -165 -93.5 -257.5t-256.5 -92.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400q0 165 92.5 257.5t257.5 92.5zM800 900h-500q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5 v500q0 41 -29.5 70.5t-70.5 29.5z" />
<glyph unicode="&#xe158;" d="M350 1100h400q165 0 257.5 -92.5t92.5 -257.5v-400q0 -165 -92.5 -257.5t-257.5 -92.5h-400q-163 0 -256.5 92.5t-93.5 257.5v400q0 163 94 256.5t256 93.5zM800 900h-500q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5 v500q0 41 -29.5 70.5t-70.5 29.5zM440 770l253 -190q17 -12 17 -30t-17 -30l-253 -190q-16 -12 -28 -6.5t-12 26.5v400q0 21 12 26.5t28 -6.5z" />
<glyph unicode="&#xe159;" d="M350 1100h400q163 0 256.5 -94t93.5 -256v-400q0 -165 -92.5 -257.5t-257.5 -92.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400q0 163 92.5 256.5t257.5 93.5zM800 900h-500q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5 v500q0 41 -29.5 70.5t-70.5 29.5zM350 700h400q21 0 26.5 -12t-6.5 -28l-190 -253q-12 -17 -30 -17t-30 17l-190 253q-12 16 -6.5 28t26.5 12z" />
<glyph unicode="&#xe160;" d="M350 1100h400q165 0 257.5 -92.5t92.5 -257.5v-400q0 -163 -92.5 -256.5t-257.5 -93.5h-400q-163 0 -256.5 94t-93.5 256v400q0 165 92.5 257.5t257.5 92.5zM800 900h-500q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5 v500q0 41 -29.5 70.5t-70.5 29.5zM580 693l190 -253q12 -16 6.5 -28t-26.5 -12h-400q-21 0 -26.5 12t6.5 28l190 253q12 17 30 17t30 -17z" />
<glyph unicode="&#xe161;" d="M550 1100h400q165 0 257.5 -92.5t92.5 -257.5v-400q0 -165 -92.5 -257.5t-257.5 -92.5h-400q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h450q41 0 70.5 29.5t29.5 70.5v500q0 41 -29.5 70.5t-70.5 29.5h-450q-21 0 -35.5 14.5t-14.5 35.5v100 q0 21 14.5 35.5t35.5 14.5zM338 867l324 -284q16 -14 16 -33t-16 -33l-324 -284q-16 -14 -27 -9t-11 26v150h-250q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5h250v150q0 21 11 26t27 -9z" />
<glyph unicode="&#xe162;" d="M793 1182l9 -9q8 -10 5 -27q-3 -11 -79 -225.5t-78 -221.5l300 1q24 0 32.5 -17.5t-5.5 -35.5q-1 0 -133.5 -155t-267 -312.5t-138.5 -162.5q-12 -15 -26 -15h-9l-9 8q-9 11 -4 32q2 9 42 123.5t79 224.5l39 110h-302q-23 0 -31 19q-10 21 6 41q75 86 209.5 237.5 t228 257t98.5 111.5q9 16 25 16h9z" />
<glyph unicode="&#xe163;" d="M350 1100h400q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-450q-41 0 -70.5 -29.5t-29.5 -70.5v-500q0 -41 29.5 -70.5t70.5 -29.5h450q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400 q0 165 92.5 257.5t257.5 92.5zM938 867l324 -284q16 -14 16 -33t-16 -33l-324 -284q-16 -14 -27 -9t-11 26v150h-250q-21 0 -35.5 14.5t-14.5 35.5v200q0 21 14.5 35.5t35.5 14.5h250v150q0 21 11 26t27 -9z" />
<glyph unicode="&#xe164;" d="M750 1200h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -10.5 -25t-24.5 10l-109 109l-312 -312q-15 -15 -35.5 -15t-35.5 15l-141 141q-15 15 -15 35.5t15 35.5l312 312l-109 109q-14 14 -10 24.5t25 10.5zM456 900h-156q-41 0 -70.5 -29.5t-29.5 -70.5v-500 q0 -41 29.5 -70.5t70.5 -29.5h500q41 0 70.5 29.5t29.5 70.5v148l200 200v-298q0 -165 -93.5 -257.5t-256.5 -92.5h-400q-165 0 -257.5 92.5t-92.5 257.5v400q0 165 92.5 257.5t257.5 92.5h300z" />
<glyph unicode="&#xe165;" d="M600 1186q119 0 227.5 -46.5t187 -125t125 -187t46.5 -227.5t-46.5 -227.5t-125 -187t-187 -125t-227.5 -46.5t-227.5 46.5t-187 125t-125 187t-46.5 227.5t46.5 227.5t125 187t187 125t227.5 46.5zM600 1022q-115 0 -212 -56.5t-153.5 -153.5t-56.5 -212t56.5 -212 t153.5 -153.5t212 -56.5t212 56.5t153.5 153.5t56.5 212t-56.5 212t-153.5 153.5t-212 56.5zM600 794q80 0 137 -57t57 -137t-57 -137t-137 -57t-137 57t-57 137t57 137t137 57z" />
<glyph unicode="&#xe166;" d="M450 1200h200q21 0 35.5 -14.5t14.5 -35.5v-350h245q20 0 25 -11t-9 -26l-383 -426q-14 -15 -33.5 -15t-32.5 15l-379 426q-13 15 -8.5 26t25.5 11h250v350q0 21 14.5 35.5t35.5 14.5zM50 300h1000q21 0 35.5 -14.5t14.5 -35.5v-250h-1100v250q0 21 14.5 35.5t35.5 14.5z M900 200v-50h100v50h-100z" />
<glyph unicode="&#xe167;" d="M583 1182l378 -435q14 -15 9 -31t-26 -16h-244v-250q0 -20 -17 -35t-39 -15h-200q-20 0 -32 14.5t-12 35.5v250h-250q-20 0 -25.5 16.5t8.5 31.5l383 431q14 16 33.5 17t33.5 -14zM50 300h1000q21 0 35.5 -14.5t14.5 -35.5v-250h-1100v250q0 21 14.5 35.5t35.5 14.5z M900 200v-50h100v50h-100z" />
<glyph unicode="&#xe168;" d="M396 723l369 369q7 7 17.5 7t17.5 -7l139 -139q7 -8 7 -18.5t-7 -17.5l-525 -525q-7 -8 -17.5 -8t-17.5 8l-292 291q-7 8 -7 18t7 18l139 139q8 7 18.5 7t17.5 -7zM50 300h1000q21 0 35.5 -14.5t14.5 -35.5v-250h-1100v250q0 21 14.5 35.5t35.5 14.5zM900 200v-50h100v50 h-100z" />
<glyph unicode="&#xe169;" d="M135 1023l142 142q14 14 35 14t35 -14l77 -77l-212 -212l-77 76q-14 15 -14 36t14 35zM655 855l210 210q14 14 24.5 10t10.5 -25l-2 -599q-1 -20 -15.5 -35t-35.5 -15l-597 -1q-21 0 -25 10.5t10 24.5l208 208l-154 155l212 212zM50 300h1000q21 0 35.5 -14.5t14.5 -35.5 v-250h-1100v250q0 21 14.5 35.5t35.5 14.5zM900 200v-50h100v50h-100z" />
<glyph unicode="&#xe170;" d="M350 1200l599 -2q20 -1 35 -15.5t15 -35.5l1 -597q0 -21 -10.5 -25t-24.5 10l-208 208l-155 -154l-212 212l155 154l-210 210q-14 14 -10 24.5t25 10.5zM524 512l-76 -77q-15 -14 -36 -14t-35 14l-142 142q-14 14 -14 35t14 35l77 77zM50 300h1000q21 0 35.5 -14.5 t14.5 -35.5v-250h-1100v250q0 21 14.5 35.5t35.5 14.5zM900 200v-50h100v50h-100z" />
<glyph unicode="&#xe171;" d="M1200 103l-483 276l-314 -399v423h-399l1196 796v-1096zM483 424v-230l683 953z" />
<glyph unicode="&#xe172;" d="M1100 1000v-850q0 -21 -14.5 -35.5t-35.5 -14.5h-150v400h-700v-400h-150q-21 0 -35.5 14.5t-14.5 35.5v1000q0 20 14.5 35t35.5 15h250v-300h500v300h100zM700 1000h-100v200h100v-200z" />
<glyph unicode="&#xe173;" d="M1100 1000l-2 -149l-299 -299l-95 95q-9 9 -21.5 9t-21.5 -9l-149 -147h-312v-400h-150q-21 0 -35.5 14.5t-14.5 35.5v1000q0 20 14.5 35t35.5 15h250v-300h500v300h100zM700 1000h-100v200h100v-200zM1132 638l106 -106q7 -7 7 -17.5t-7 -17.5l-420 -421q-8 -7 -18 -7 t-18 7l-202 203q-8 7 -8 17.5t8 17.5l106 106q7 8 17.5 8t17.5 -8l79 -79l297 297q7 7 17.5 7t17.5 -7z" />
<glyph unicode="&#xe174;" d="M1100 1000v-269l-103 -103l-134 134q-15 15 -33.5 16.5t-34.5 -12.5l-266 -266h-329v-400h-150q-21 0 -35.5 14.5t-14.5 35.5v1000q0 20 14.5 35t35.5 15h250v-300h500v300h100zM700 1000h-100v200h100v-200zM1202 572l70 -70q15 -15 15 -35.5t-15 -35.5l-131 -131 l131 -131q15 -15 15 -35.5t-15 -35.5l-70 -70q-15 -15 -35.5 -15t-35.5 15l-131 131l-131 -131q-15 -15 -35.5 -15t-35.5 15l-70 70q-15 15 -15 35.5t15 35.5l131 131l-131 131q-15 15 -15 35.5t15 35.5l70 70q15 15 35.5 15t35.5 -15l131 -131l131 131q15 15 35.5 15 t35.5 -15z" />
<glyph unicode="&#xe175;" d="M1100 1000v-300h-350q-21 0 -35.5 -14.5t-14.5 -35.5v-150h-500v-400h-150q-21 0 -35.5 14.5t-14.5 35.5v1000q0 20 14.5 35t35.5 15h250v-300h500v300h100zM700 1000h-100v200h100v-200zM850 600h100q21 0 35.5 -14.5t14.5 -35.5v-250h150q21 0 25 -10.5t-10 -24.5 l-230 -230q-14 -14 -35 -14t-35 14l-230 230q-14 14 -10 24.5t25 10.5h150v250q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe176;" d="M1100 1000v-400l-165 165q-14 15 -35 15t-35 -15l-263 -265h-402v-400h-150q-21 0 -35.5 14.5t-14.5 35.5v1000q0 20 14.5 35t35.5 15h250v-300h500v300h100zM700 1000h-100v200h100v-200zM935 565l230 -229q14 -15 10 -25.5t-25 -10.5h-150v-250q0 -20 -14.5 -35 t-35.5 -15h-100q-21 0 -35.5 15t-14.5 35v250h-150q-21 0 -25 10.5t10 25.5l230 229q14 15 35 15t35 -15z" />
<glyph unicode="&#xe177;" d="M50 1100h1100q21 0 35.5 -14.5t14.5 -35.5v-150h-1200v150q0 21 14.5 35.5t35.5 14.5zM1200 800v-550q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v550h1200zM100 500v-200h400v200h-400z" />
<glyph unicode="&#xe178;" d="M935 1165l248 -230q14 -14 14 -35t-14 -35l-248 -230q-14 -14 -24.5 -10t-10.5 25v150h-400v200h400v150q0 21 10.5 25t24.5 -10zM200 800h-50q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h50v-200zM400 800h-100v200h100v-200zM18 435l247 230 q14 14 24.5 10t10.5 -25v-150h400v-200h-400v-150q0 -21 -10.5 -25t-24.5 10l-247 230q-15 14 -15 35t15 35zM900 300h-100v200h100v-200zM1000 500h51q20 0 34.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-34.5 -14.5h-51v200z" />
<glyph unicode="&#xe179;" d="M862 1073l276 116q25 18 43.5 8t18.5 -41v-1106q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v397q-4 1 -11 5t-24 17.5t-30 29t-24 42t-11 56.5v359q0 31 18.5 65t43.5 52zM550 1200q22 0 34.5 -12.5t14.5 -24.5l1 -13v-450q0 -28 -10.5 -59.5 t-25 -56t-29 -45t-25.5 -31.5l-10 -11v-447q0 -21 -14.5 -35.5t-35.5 -14.5h-200q-21 0 -35.5 14.5t-14.5 35.5v447q-4 4 -11 11.5t-24 30.5t-30 46t-24 55t-11 60v450q0 2 0.5 5.5t4 12t8.5 15t14.5 12t22.5 5.5q20 0 32.5 -12.5t14.5 -24.5l3 -13v-350h100v350v5.5t2.5 12 t7 15t15 12t25.5 5.5q23 0 35.5 -12.5t13.5 -24.5l1 -13v-350h100v350q0 2 0.5 5.5t3 12t7 15t15 12t24.5 5.5z" />
<glyph unicode="&#xe180;" d="M1200 1100v-56q-4 0 -11 -0.5t-24 -3t-30 -7.5t-24 -15t-11 -24v-888q0 -22 25 -34.5t50 -13.5l25 -2v-56h-400v56q75 0 87.5 6.5t12.5 43.5v394h-500v-394q0 -37 12.5 -43.5t87.5 -6.5v-56h-400v56q4 0 11 0.5t24 3t30 7.5t24 15t11 24v888q0 22 -25 34.5t-50 13.5 l-25 2v56h400v-56q-75 0 -87.5 -6.5t-12.5 -43.5v-394h500v394q0 37 -12.5 43.5t-87.5 6.5v56h400z" />
<glyph unicode="&#xe181;" d="M675 1000h375q21 0 35.5 -14.5t14.5 -35.5v-150h-105l-295 -98v98l-200 200h-400l100 100h375zM100 900h300q41 0 70.5 -29.5t29.5 -70.5v-500q0 -41 -29.5 -70.5t-70.5 -29.5h-300q-41 0 -70.5 29.5t-29.5 70.5v500q0 41 29.5 70.5t70.5 29.5zM100 800v-200h300v200 h-300zM1100 535l-400 -133v163l400 133v-163zM100 500v-200h300v200h-300zM1100 398v-248q0 -21 -14.5 -35.5t-35.5 -14.5h-375l-100 -100h-375l-100 100h400l200 200h105z" />
<glyph unicode="&#xe182;" d="M17 1007l162 162q17 17 40 14t37 -22l139 -194q14 -20 11 -44.5t-20 -41.5l-119 -118q102 -142 228 -268t267 -227l119 118q17 17 42.5 19t44.5 -12l192 -136q19 -14 22.5 -37.5t-13.5 -40.5l-163 -162q-3 -1 -9.5 -1t-29.5 2t-47.5 6t-62.5 14.5t-77.5 26.5t-90 42.5 t-101.5 60t-111 83t-119 108.5q-74 74 -133.5 150.5t-94.5 138.5t-60 119.5t-34.5 100t-15 74.5t-4.5 48z" />
<glyph unicode="&#xe183;" d="M600 1100q92 0 175 -10.5t141.5 -27t108.5 -36.5t81.5 -40t53.5 -37t31 -27l9 -10v-200q0 -21 -14.5 -33t-34.5 -9l-202 34q-20 3 -34.5 20t-14.5 38v146q-141 24 -300 24t-300 -24v-146q0 -21 -14.5 -38t-34.5 -20l-202 -34q-20 -3 -34.5 9t-14.5 33v200q3 4 9.5 10.5 t31 26t54 37.5t80.5 39.5t109 37.5t141 26.5t175 10.5zM600 795q56 0 97 -9.5t60 -23.5t30 -28t12 -24l1 -10v-50l365 -303q14 -15 24.5 -40t10.5 -45v-212q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v212q0 20 10.5 45t24.5 40l365 303v50 q0 4 1 10.5t12 23t30 29t60 22.5t97 10z" />
<glyph unicode="&#xe184;" d="M1100 700l-200 -200h-600l-200 200v500h200v-200h200v200h200v-200h200v200h200v-500zM250 400h700q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-12l137 -100h-950l137 100h-12q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM50 100h1100q21 0 35.5 -14.5 t14.5 -35.5v-50h-1200v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe185;" d="M700 1100h-100q-41 0 -70.5 -29.5t-29.5 -70.5v-1000h300v1000q0 41 -29.5 70.5t-70.5 29.5zM1100 800h-100q-41 0 -70.5 -29.5t-29.5 -70.5v-700h300v700q0 41 -29.5 70.5t-70.5 29.5zM400 0h-300v400q0 41 29.5 70.5t70.5 29.5h100q41 0 70.5 -29.5t29.5 -70.5v-400z " />
<glyph unicode="&#xe186;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM500 700h-200v-100h200v-300h-300v100h200v100h-200v300h300v-100zM900 700v-300l-100 -100h-200v500h200z M700 700v-300h100v300h-100z" />
<glyph unicode="&#xe187;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM500 300h-100v200h-100v-200h-100v500h100v-200h100v200h100v-500zM900 700v-300l-100 -100h-200v500h200z M700 700v-300h100v300h-100z" />
<glyph unicode="&#xe188;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM500 700h-200v-300h200v-100h-300v500h300v-100zM900 700h-200v-300h200v-100h-300v500h300v-100z" />
<glyph unicode="&#xe189;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM500 400l-300 150l300 150v-300zM900 550l-300 -150v300z" />
<glyph unicode="&#xe190;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM900 300h-700v500h700v-500zM800 700h-130q-38 0 -66.5 -43t-28.5 -108t27 -107t68 -42h130v300zM300 700v-300 h130q41 0 68 42t27 107t-28.5 108t-66.5 43h-130z" />
<glyph unicode="&#xe191;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM500 700h-200v-100h200v-300h-300v100h200v100h-200v300h300v-100zM900 300h-100v400h-100v100h200v-500z M700 300h-100v100h100v-100z" />
<glyph unicode="&#xe192;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM300 700h200v-400h-300v500h100v-100zM900 300h-100v400h-100v100h200v-500zM300 600v-200h100v200h-100z M700 300h-100v100h100v-100z" />
<glyph unicode="&#xe193;" d="M200 1100h700q124 0 212 -88t88 -212v-500q0 -124 -88 -212t-212 -88h-700q-124 0 -212 88t-88 212v500q0 124 88 212t212 88zM100 900v-700h900v700h-900zM500 500l-199 -200h-100v50l199 200v150h-200v100h300v-300zM900 300h-100v400h-100v100h200v-500zM701 300h-100 v100h100v-100z" />
<glyph unicode="&#xe194;" d="M600 1191q120 0 229.5 -47t188.5 -126t126 -188.5t47 -229.5t-47 -229.5t-126 -188.5t-188.5 -126t-229.5 -47t-229.5 47t-188.5 126t-126 188.5t-47 229.5t47 229.5t126 188.5t188.5 126t229.5 47zM600 1021q-114 0 -211 -56.5t-153.5 -153.5t-56.5 -211t56.5 -211 t153.5 -153.5t211 -56.5t211 56.5t153.5 153.5t56.5 211t-56.5 211t-153.5 153.5t-211 56.5zM800 700h-300v-200h300v-100h-300l-100 100v200l100 100h300v-100z" />
<glyph unicode="&#xe195;" d="M600 1191q120 0 229.5 -47t188.5 -126t126 -188.5t47 -229.5t-47 -229.5t-126 -188.5t-188.5 -126t-229.5 -47t-229.5 47t-188.5 126t-126 188.5t-47 229.5t47 229.5t126 188.5t188.5 126t229.5 47zM600 1021q-114 0 -211 -56.5t-153.5 -153.5t-56.5 -211t56.5 -211 t153.5 -153.5t211 -56.5t211 56.5t153.5 153.5t56.5 211t-56.5 211t-153.5 153.5t-211 56.5zM800 700v-100l-50 -50l100 -100v-50h-100l-100 100h-150v-100h-100v400h300zM500 700v-100h200v100h-200z" />
<glyph unicode="&#xe197;" d="M503 1089q110 0 200.5 -59.5t134.5 -156.5q44 14 90 14q120 0 205 -86.5t85 -207t-85 -207t-205 -86.5h-128v250q0 21 -14.5 35.5t-35.5 14.5h-300q-21 0 -35.5 -14.5t-14.5 -35.5v-250h-222q-80 0 -136 57.5t-56 136.5q0 69 43 122.5t108 67.5q-2 19 -2 37q0 100 49 185 t134 134t185 49zM525 500h150q10 0 17.5 -7.5t7.5 -17.5v-275h137q21 0 26 -11.5t-8 -27.5l-223 -244q-13 -16 -32 -16t-32 16l-223 244q-13 16 -8 27.5t26 11.5h137v275q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe198;" d="M502 1089q110 0 201 -59.5t135 -156.5q43 15 89 15q121 0 206 -86.5t86 -206.5q0 -99 -60 -181t-150 -110l-378 360q-13 16 -31.5 16t-31.5 -16l-381 -365h-9q-79 0 -135.5 57.5t-56.5 136.5q0 69 43 122.5t108 67.5q-2 19 -2 38q0 100 49 184.5t133.5 134t184.5 49.5z M632 467l223 -228q13 -16 8 -27.5t-26 -11.5h-137v-275q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v275h-137q-21 0 -26 11.5t8 27.5q199 204 223 228q19 19 31.5 19t32.5 -19z" />
<glyph unicode="&#xe199;" d="M700 100v100h400l-270 300h170l-270 300h170l-300 333l-300 -333h170l-270 -300h170l-270 -300h400v-100h-50q-21 0 -35.5 -14.5t-14.5 -35.5v-50h400v50q0 21 -14.5 35.5t-35.5 14.5h-50z" />
<glyph unicode="&#xe200;" d="M600 1179q94 0 167.5 -56.5t99.5 -145.5q89 -6 150.5 -71.5t61.5 -155.5q0 -61 -29.5 -112.5t-79.5 -82.5q9 -29 9 -55q0 -74 -52.5 -126.5t-126.5 -52.5q-55 0 -100 30v-251q21 0 35.5 -14.5t14.5 -35.5v-50h-300v50q0 21 14.5 35.5t35.5 14.5v251q-45 -30 -100 -30 q-74 0 -126.5 52.5t-52.5 126.5q0 18 4 38q-47 21 -75.5 65t-28.5 97q0 74 52.5 126.5t126.5 52.5q5 0 23 -2q0 2 -1 10t-1 13q0 116 81.5 197.5t197.5 81.5z" />
<glyph unicode="&#xe201;" d="M1010 1010q111 -111 150.5 -260.5t0 -299t-150.5 -260.5q-83 -83 -191.5 -126.5t-218.5 -43.5t-218.5 43.5t-191.5 126.5q-111 111 -150.5 260.5t0 299t150.5 260.5q83 83 191.5 126.5t218.5 43.5t218.5 -43.5t191.5 -126.5zM476 1065q-4 0 -8 -1q-121 -34 -209.5 -122.5 t-122.5 -209.5q-4 -12 2.5 -23t18.5 -14l36 -9q3 -1 7 -1q23 0 29 22q27 96 98 166q70 71 166 98q11 3 17.5 13.5t3.5 22.5l-9 35q-3 13 -14 19q-7 4 -15 4zM512 920q-4 0 -9 -2q-80 -24 -138.5 -82.5t-82.5 -138.5q-4 -13 2 -24t19 -14l34 -9q4 -1 8 -1q22 0 28 21 q18 58 58.5 98.5t97.5 58.5q12 3 18 13.5t3 21.5l-9 35q-3 12 -14 19q-7 4 -15 4zM719.5 719.5q-49.5 49.5 -119.5 49.5t-119.5 -49.5t-49.5 -119.5t49.5 -119.5t119.5 -49.5t119.5 49.5t49.5 119.5t-49.5 119.5zM855 551q-22 0 -28 -21q-18 -58 -58.5 -98.5t-98.5 -57.5 q-11 -4 -17 -14.5t-3 -21.5l9 -35q3 -12 14 -19q7 -4 15 -4q4 0 9 2q80 24 138.5 82.5t82.5 138.5q4 13 -2.5 24t-18.5 14l-34 9q-4 1 -8 1zM1000 515q-23 0 -29 -22q-27 -96 -98 -166q-70 -71 -166 -98q-11 -3 -17.5 -13.5t-3.5 -22.5l9 -35q3 -13 14 -19q7 -4 15 -4 q4 0 8 1q121 34 209.5 122.5t122.5 209.5q4 12 -2.5 23t-18.5 14l-36 9q-3 1 -7 1z" />
<glyph unicode="&#xe202;" d="M700 800h300v-380h-180v200h-340v-200h-380v755q0 10 7.5 17.5t17.5 7.5h575v-400zM1000 900h-200v200zM700 300h162l-212 -212l-212 212h162v200h100v-200zM520 0h-395q-10 0 -17.5 7.5t-7.5 17.5v395zM1000 220v-195q0 -10 -7.5 -17.5t-17.5 -7.5h-195z" />
<glyph unicode="&#xe203;" d="M700 800h300v-520l-350 350l-550 -550v1095q0 10 7.5 17.5t17.5 7.5h575v-400zM1000 900h-200v200zM862 200h-162v-200h-100v200h-162l212 212zM480 0h-355q-10 0 -17.5 7.5t-7.5 17.5v55h380v-80zM1000 80v-55q0 -10 -7.5 -17.5t-17.5 -7.5h-155v80h180z" />
<glyph unicode="&#xe204;" d="M1162 800h-162v-200h100l100 -100h-300v300h-162l212 212zM200 800h200q27 0 40 -2t29.5 -10.5t23.5 -30t7 -57.5h300v-100h-600l-200 -350v450h100q0 36 7 57.5t23.5 30t29.5 10.5t40 2zM800 400h240l-240 -400h-800l300 500h500v-100z" />
<glyph unicode="&#xe205;" d="M650 1100h100q21 0 35.5 -14.5t14.5 -35.5v-50h50q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-300q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h50v50q0 21 14.5 35.5t35.5 14.5zM1000 850v150q41 0 70.5 -29.5t29.5 -70.5v-800 q0 -41 -29.5 -70.5t-70.5 -29.5h-600q-1 0 -20 4l246 246l-326 326v324q0 41 29.5 70.5t70.5 29.5v-150q0 -62 44 -106t106 -44h300q62 0 106 44t44 106zM412 250l-212 -212v162h-200v100h200v162z" />
<glyph unicode="&#xe206;" d="M450 1100h100q21 0 35.5 -14.5t14.5 -35.5v-50h50q21 0 35.5 -14.5t14.5 -35.5v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-300q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h50v50q0 21 14.5 35.5t35.5 14.5zM800 850v150q41 0 70.5 -29.5t29.5 -70.5v-500 h-200v-300h200q0 -36 -7 -57.5t-23.5 -30t-29.5 -10.5t-40 -2h-600q-41 0 -70.5 29.5t-29.5 70.5v800q0 41 29.5 70.5t70.5 29.5v-150q0 -62 44 -106t106 -44h300q62 0 106 44t44 106zM1212 250l-212 -212v162h-200v100h200v162z" />
<glyph unicode="&#xe209;" d="M658 1197l637 -1104q23 -38 7 -65.5t-60 -27.5h-1276q-44 0 -60 27.5t7 65.5l637 1104q22 39 54 39t54 -39zM704 800h-208q-20 0 -32 -14.5t-8 -34.5l58 -302q4 -20 21.5 -34.5t37.5 -14.5h54q20 0 37.5 14.5t21.5 34.5l58 302q4 20 -8 34.5t-32 14.5zM500 300v-100h200 v100h-200z" />
<glyph unicode="&#xe210;" d="M425 1100h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5zM425 800h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5 t17.5 7.5zM825 800h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5zM25 500h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150 q0 10 7.5 17.5t17.5 7.5zM425 500h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5zM825 500h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5 v150q0 10 7.5 17.5t17.5 7.5zM25 200h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5zM425 200h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5 t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5zM825 200h250q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-250q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe211;" d="M700 1200h100v-200h-100v-100h350q62 0 86.5 -39.5t-3.5 -94.5l-66 -132q-41 -83 -81 -134h-772q-40 51 -81 134l-66 132q-28 55 -3.5 94.5t86.5 39.5h350v100h-100v200h100v100h200v-100zM250 400h700q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-12l137 -100 h-950l138 100h-13q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM50 100h1100q21 0 35.5 -14.5t14.5 -35.5v-50h-1200v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe212;" d="M600 1300q40 0 68.5 -29.5t28.5 -70.5h-194q0 41 28.5 70.5t68.5 29.5zM443 1100h314q18 -37 18 -75q0 -8 -3 -25h328q41 0 44.5 -16.5t-30.5 -38.5l-175 -145h-678l-178 145q-34 22 -29 38.5t46 16.5h328q-3 17 -3 25q0 38 18 75zM250 700h700q21 0 35.5 -14.5 t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-150v-200l275 -200h-950l275 200v200h-150q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM50 100h1100q21 0 35.5 -14.5t14.5 -35.5v-50h-1200v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe213;" d="M600 1181q75 0 128 -53t53 -128t-53 -128t-128 -53t-128 53t-53 128t53 128t128 53zM602 798h46q34 0 55.5 -28.5t21.5 -86.5q0 -76 39 -183h-324q39 107 39 183q0 58 21.5 86.5t56.5 28.5h45zM250 400h700q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-13 l138 -100h-950l137 100h-12q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM50 100h1100q21 0 35.5 -14.5t14.5 -35.5v-50h-1200v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe214;" d="M600 1300q47 0 92.5 -53.5t71 -123t25.5 -123.5q0 -78 -55.5 -133.5t-133.5 -55.5t-133.5 55.5t-55.5 133.5q0 62 34 143l144 -143l111 111l-163 163q34 26 63 26zM602 798h46q34 0 55.5 -28.5t21.5 -86.5q0 -76 39 -183h-324q39 107 39 183q0 58 21.5 86.5t56.5 28.5h45 zM250 400h700q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-13l138 -100h-950l137 100h-12q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM50 100h1100q21 0 35.5 -14.5t14.5 -35.5v-50h-1200v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe215;" d="M600 1200l300 -161v-139h-300q0 -57 18.5 -108t50 -91.5t63 -72t70 -67.5t57.5 -61h-530q-60 83 -90.5 177.5t-30.5 178.5t33 164.5t87.5 139.5t126 96.5t145.5 41.5v-98zM250 400h700q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-13l138 -100h-950l137 100 h-12q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5zM50 100h1100q21 0 35.5 -14.5t14.5 -35.5v-50h-1200v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe216;" d="M600 1300q41 0 70.5 -29.5t29.5 -70.5v-78q46 -26 73 -72t27 -100v-50h-400v50q0 54 27 100t73 72v78q0 41 29.5 70.5t70.5 29.5zM400 800h400q54 0 100 -27t72 -73h-172v-100h200v-100h-200v-100h200v-100h-200v-100h200q0 -83 -58.5 -141.5t-141.5 -58.5h-400 q-83 0 -141.5 58.5t-58.5 141.5v400q0 83 58.5 141.5t141.5 58.5z" />
<glyph unicode="&#xe218;" d="M150 1100h900q21 0 35.5 -14.5t14.5 -35.5v-500q0 -21 -14.5 -35.5t-35.5 -14.5h-900q-21 0 -35.5 14.5t-14.5 35.5v500q0 21 14.5 35.5t35.5 14.5zM125 400h950q10 0 17.5 -7.5t7.5 -17.5v-50q0 -10 -7.5 -17.5t-17.5 -7.5h-283l224 -224q13 -13 13 -31.5t-13 -32 t-31.5 -13.5t-31.5 13l-88 88h-524l-87 -88q-13 -13 -32 -13t-32 13.5t-13 32t13 31.5l224 224h-289q-10 0 -17.5 7.5t-7.5 17.5v50q0 10 7.5 17.5t17.5 7.5zM541 300l-100 -100h324l-100 100h-124z" />
<glyph unicode="&#xe219;" d="M200 1100h800q83 0 141.5 -58.5t58.5 -141.5v-200h-100q0 41 -29.5 70.5t-70.5 29.5h-250q-41 0 -70.5 -29.5t-29.5 -70.5h-100q0 41 -29.5 70.5t-70.5 29.5h-250q-41 0 -70.5 -29.5t-29.5 -70.5h-100v200q0 83 58.5 141.5t141.5 58.5zM100 600h1000q41 0 70.5 -29.5 t29.5 -70.5v-300h-1200v300q0 41 29.5 70.5t70.5 29.5zM300 100v-50q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v50h200zM1100 100v-50q0 -21 -14.5 -35.5t-35.5 -14.5h-100q-21 0 -35.5 14.5t-14.5 35.5v50h200z" />
<glyph unicode="&#xe221;" d="M480 1165l682 -683q31 -31 31 -75.5t-31 -75.5l-131 -131h-481l-517 518q-32 31 -32 75.5t32 75.5l295 296q31 31 75.5 31t76.5 -31zM108 794l342 -342l303 304l-341 341zM250 100h800q21 0 35.5 -14.5t14.5 -35.5v-50h-900v50q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe223;" d="M1057 647l-189 506q-8 19 -27.5 33t-40.5 14h-400q-21 0 -40.5 -14t-27.5 -33l-189 -506q-8 -19 1.5 -33t30.5 -14h625v-150q0 -21 14.5 -35.5t35.5 -14.5t35.5 14.5t14.5 35.5v150h125q21 0 30.5 14t1.5 33zM897 0h-595v50q0 21 14.5 35.5t35.5 14.5h50v50 q0 21 14.5 35.5t35.5 14.5h48v300h200v-300h47q21 0 35.5 -14.5t14.5 -35.5v-50h50q21 0 35.5 -14.5t14.5 -35.5v-50z" />
<glyph unicode="&#xe224;" d="M900 800h300v-575q0 -10 -7.5 -17.5t-17.5 -7.5h-375v591l-300 300v84q0 10 7.5 17.5t17.5 7.5h375v-400zM1200 900h-200v200zM400 600h300v-575q0 -10 -7.5 -17.5t-17.5 -7.5h-650q-10 0 -17.5 7.5t-7.5 17.5v950q0 10 7.5 17.5t17.5 7.5h375v-400zM700 700h-200v200z " />
<glyph unicode="&#xe225;" d="M484 1095h195q75 0 146 -32.5t124 -86t89.5 -122.5t48.5 -142q18 -14 35 -20q31 -10 64.5 6.5t43.5 48.5q10 34 -15 71q-19 27 -9 43q5 8 12.5 11t19 -1t23.5 -16q41 -44 39 -105q-3 -63 -46 -106.5t-104 -43.5h-62q-7 -55 -35 -117t-56 -100l-39 -234q-3 -20 -20 -34.5 t-38 -14.5h-100q-21 0 -33 14.5t-9 34.5l12 70q-49 -14 -91 -14h-195q-24 0 -65 8l-11 -64q-3 -20 -20 -34.5t-38 -14.5h-100q-21 0 -33 14.5t-9 34.5l26 157q-84 74 -128 175l-159 53q-19 7 -33 26t-14 40v50q0 21 14.5 35.5t35.5 14.5h124q11 87 56 166l-111 95 q-16 14 -12.5 23.5t24.5 9.5h203q116 101 250 101zM675 1000h-250q-10 0 -17.5 -7.5t-7.5 -17.5v-50q0 -10 7.5 -17.5t17.5 -7.5h250q10 0 17.5 7.5t7.5 17.5v50q0 10 -7.5 17.5t-17.5 7.5z" />
<glyph unicode="&#xe226;" d="M641 900l423 247q19 8 42 2.5t37 -21.5l32 -38q14 -15 12.5 -36t-17.5 -34l-139 -120h-390zM50 1100h106q67 0 103 -17t66 -71l102 -212h823q21 0 35.5 -14.5t14.5 -35.5v-50q0 -21 -14 -40t-33 -26l-737 -132q-23 -4 -40 6t-26 25q-42 67 -100 67h-300q-62 0 -106 44 t-44 106v200q0 62 44 106t106 44zM173 928h-80q-19 0 -28 -14t-9 -35v-56q0 -51 42 -51h134q16 0 21.5 8t5.5 24q0 11 -16 45t-27 51q-18 28 -43 28zM550 727q-32 0 -54.5 -22.5t-22.5 -54.5t22.5 -54.5t54.5 -22.5t54.5 22.5t22.5 54.5t-22.5 54.5t-54.5 22.5zM130 389 l152 130q18 19 34 24t31 -3.5t24.5 -17.5t25.5 -28q28 -35 50.5 -51t48.5 -13l63 5l48 -179q13 -61 -3.5 -97.5t-67.5 -79.5l-80 -69q-47 -40 -109 -35.5t-103 51.5l-130 151q-40 47 -35.5 109.5t51.5 102.5zM380 377l-102 -88q-31 -27 2 -65l37 -43q13 -15 27.5 -19.5 t31.5 6.5l61 53q19 16 14 49q-2 20 -12 56t-17 45q-11 12 -19 14t-23 -8z" />
<glyph unicode="&#xe227;" d="M625 1200h150q10 0 17.5 -7.5t7.5 -17.5v-109q79 -33 131 -87.5t53 -128.5q1 -46 -15 -84.5t-39 -61t-46 -38t-39 -21.5l-17 -6q6 0 15 -1.5t35 -9t50 -17.5t53 -30t50 -45t35.5 -64t14.5 -84q0 -59 -11.5 -105.5t-28.5 -76.5t-44 -51t-49.5 -31.5t-54.5 -16t-49.5 -6.5 t-43.5 -1v-75q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v75h-100v-75q0 -10 -7.5 -17.5t-17.5 -7.5h-150q-10 0 -17.5 7.5t-7.5 17.5v75h-175q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5h75v600h-75q-10 0 -17.5 7.5t-7.5 17.5v150 q0 10 7.5 17.5t17.5 7.5h175v75q0 10 7.5 17.5t17.5 7.5h150q10 0 17.5 -7.5t7.5 -17.5v-75h100v75q0 10 7.5 17.5t17.5 7.5zM400 900v-200h263q28 0 48.5 10.5t30 25t15 29t5.5 25.5l1 10q0 4 -0.5 11t-6 24t-15 30t-30 24t-48.5 11h-263zM400 500v-200h363q28 0 48.5 10.5 t30 25t15 29t5.5 25.5l1 10q0 4 -0.5 11t-6 24t-15 30t-30 24t-48.5 11h-363z" />
<glyph unicode="&#xe230;" d="M212 1198h780q86 0 147 -61t61 -147v-416q0 -51 -18 -142.5t-36 -157.5l-18 -66q-29 -87 -93.5 -146.5t-146.5 -59.5h-572q-82 0 -147 59t-93 147q-8 28 -20 73t-32 143.5t-20 149.5v416q0 86 61 147t147 61zM600 1045q-70 0 -132.5 -11.5t-105.5 -30.5t-78.5 -41.5 t-57 -45t-36 -41t-20.5 -30.5l-6 -12l156 -243h560l156 243q-2 5 -6 12.5t-20 29.5t-36.5 42t-57 44.5t-79 42t-105 29.5t-132.5 12zM762 703h-157l195 261z" />
<glyph unicode="&#xe231;" d="M475 1300h150q103 0 189 -86t86 -189v-500q0 -41 -42 -83t-83 -42h-450q-41 0 -83 42t-42 83v500q0 103 86 189t189 86zM700 300v-225q0 -21 -27 -48t-48 -27h-150q-21 0 -48 27t-27 48v225h300z" />
<glyph unicode="&#xe232;" d="M475 1300h96q0 -150 89.5 -239.5t239.5 -89.5v-446q0 -41 -42 -83t-83 -42h-450q-41 0 -83 42t-42 83v500q0 103 86 189t189 86zM700 300v-225q0 -21 -27 -48t-48 -27h-150q-21 0 -48 27t-27 48v225h300z" />
<glyph unicode="&#xe233;" d="M1294 767l-638 -283l-378 170l-78 -60v-224l100 -150v-199l-150 148l-150 -149v200l100 150v250q0 4 -0.5 10.5t0 9.5t1 8t3 8t6.5 6l47 40l-147 65l642 283zM1000 380l-350 -166l-350 166v147l350 -165l350 165v-147z" />
<glyph unicode="&#xe234;" d="M250 800q62 0 106 -44t44 -106t-44 -106t-106 -44t-106 44t-44 106t44 106t106 44zM650 800q62 0 106 -44t44 -106t-44 -106t-106 -44t-106 44t-44 106t44 106t106 44zM1050 800q62 0 106 -44t44 -106t-44 -106t-106 -44t-106 44t-44 106t44 106t106 44z" />
<glyph unicode="&#xe235;" d="M550 1100q62 0 106 -44t44 -106t-44 -106t-106 -44t-106 44t-44 106t44 106t106 44zM550 700q62 0 106 -44t44 -106t-44 -106t-106 -44t-106 44t-44 106t44 106t106 44zM550 300q62 0 106 -44t44 -106t-44 -106t-106 -44t-106 44t-44 106t44 106t106 44z" />
<glyph unicode="&#xe236;" d="M125 1100h950q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-950q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5zM125 700h950q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-950q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5 t17.5 7.5zM125 300h950q10 0 17.5 -7.5t7.5 -17.5v-150q0 -10 -7.5 -17.5t-17.5 -7.5h-950q-10 0 -17.5 7.5t-7.5 17.5v150q0 10 7.5 17.5t17.5 7.5z" />
<glyph unicode="&#xe237;" d="M350 1200h500q162 0 256 -93.5t94 -256.5v-500q0 -165 -93.5 -257.5t-256.5 -92.5h-500q-165 0 -257.5 92.5t-92.5 257.5v500q0 165 92.5 257.5t257.5 92.5zM900 1000h-600q-41 0 -70.5 -29.5t-29.5 -70.5v-600q0 -41 29.5 -70.5t70.5 -29.5h600q41 0 70.5 29.5 t29.5 70.5v600q0 41 -29.5 70.5t-70.5 29.5zM350 900h500q21 0 35.5 -14.5t14.5 -35.5v-300q0 -21 -14.5 -35.5t-35.5 -14.5h-500q-21 0 -35.5 14.5t-14.5 35.5v300q0 21 14.5 35.5t35.5 14.5zM400 800v-200h400v200h-400z" />
<glyph unicode="&#xe238;" d="M150 1100h1000q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-50v-200h50q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-50v-200h50q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5t-35.5 -14.5h-50v-200h50q21 0 35.5 -14.5t14.5 -35.5t-14.5 -35.5 t-35.5 -14.5h-1000q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5h50v200h-50q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5h50v200h-50q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5h50v200h-50q-21 0 -35.5 14.5t-14.5 35.5t14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe239;" d="M650 1187q87 -67 118.5 -156t0 -178t-118.5 -155q-87 66 -118.5 155t0 178t118.5 156zM300 800q124 0 212 -88t88 -212q-124 0 -212 88t-88 212zM1000 800q0 -124 -88 -212t-212 -88q0 124 88 212t212 88zM300 500q124 0 212 -88t88 -212q-124 0 -212 88t-88 212z M1000 500q0 -124 -88 -212t-212 -88q0 124 88 212t212 88zM700 199v-144q0 -21 -14.5 -35.5t-35.5 -14.5t-35.5 14.5t-14.5 35.5v142q40 -4 43 -4q17 0 57 6z" />
<glyph unicode="&#xe240;" d="M745 878l69 19q25 6 45 -12l298 -295q11 -11 15 -26.5t-2 -30.5q-5 -14 -18 -23.5t-28 -9.5h-8q1 0 1 -13q0 -29 -2 -56t-8.5 -62t-20 -63t-33 -53t-51 -39t-72.5 -14h-146q-184 0 -184 288q0 24 10 47q-20 4 -62 4t-63 -4q11 -24 11 -47q0 -288 -184 -288h-142 q-48 0 -84.5 21t-56 51t-32 71.5t-16 75t-3.5 68.5q0 13 2 13h-7q-15 0 -27.5 9.5t-18.5 23.5q-6 15 -2 30.5t15 25.5l298 296q20 18 46 11l76 -19q20 -5 30.5 -22.5t5.5 -37.5t-22.5 -31t-37.5 -5l-51 12l-182 -193h891l-182 193l-44 -12q-20 -5 -37.5 6t-22.5 31t6 37.5 t31 22.5z" />
<glyph unicode="&#xe241;" d="M1200 900h-50q0 21 -4 37t-9.5 26.5t-18 17.5t-22 11t-28.5 5.5t-31 2t-37 0.5h-200v-850q0 -22 25 -34.5t50 -13.5l25 -2v-100h-400v100q4 0 11 0.5t24 3t30 7t24 15t11 24.5v850h-200q-25 0 -37 -0.5t-31 -2t-28.5 -5.5t-22 -11t-18 -17.5t-9.5 -26.5t-4 -37h-50v300 h1000v-300zM500 450h-25q0 15 -4 24.5t-9 14.5t-17 7.5t-20 3t-25 0.5h-100v-425q0 -11 12.5 -17.5t25.5 -7.5h12v-50h-200v50q50 0 50 25v425h-100q-17 0 -25 -0.5t-20 -3t-17 -7.5t-9 -14.5t-4 -24.5h-25v150h500v-150z" />
<glyph unicode="&#xe242;" d="M1000 300v50q-25 0 -55 32q-14 14 -25 31t-16 27l-4 11l-289 747h-69l-300 -754q-18 -35 -39 -56q-9 -9 -24.5 -18.5t-26.5 -14.5l-11 -5v-50h273v50q-49 0 -78.5 21.5t-11.5 67.5l69 176h293l61 -166q13 -34 -3.5 -66.5t-55.5 -32.5v-50h312zM412 691l134 342l121 -342 h-255zM1100 150v-100q0 -21 -14.5 -35.5t-35.5 -14.5h-1000q-21 0 -35.5 14.5t-14.5 35.5v100q0 21 14.5 35.5t35.5 14.5h1000q21 0 35.5 -14.5t14.5 -35.5z" />
<glyph unicode="&#xe243;" d="M50 1200h1100q21 0 35.5 -14.5t14.5 -35.5v-1100q0 -21 -14.5 -35.5t-35.5 -14.5h-1100q-21 0 -35.5 14.5t-14.5 35.5v1100q0 21 14.5 35.5t35.5 14.5zM611 1118h-70q-13 0 -18 -12l-299 -753q-17 -32 -35 -51q-18 -18 -56 -34q-12 -5 -12 -18v-50q0 -8 5.5 -14t14.5 -6 h273q8 0 14 6t6 14v50q0 8 -6 14t-14 6q-55 0 -71 23q-10 14 0 39l63 163h266l57 -153q11 -31 -6 -55q-12 -17 -36 -17q-8 0 -14 -6t-6 -14v-50q0 -8 6 -14t14 -6h313q8 0 14 6t6 14v50q0 7 -5.5 13t-13.5 7q-17 0 -42 25q-25 27 -40 63h-1l-288 748q-5 12 -19 12zM639 611 h-197l103 264z" />
<glyph unicode="&#xe244;" d="M1200 1100h-1200v100h1200v-100zM50 1000h400q21 0 35.5 -14.5t14.5 -35.5v-900q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v900q0 21 14.5 35.5t35.5 14.5zM650 1000h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400 q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM700 900v-300h300v300h-300z" />
<glyph unicode="&#xe245;" d="M50 1200h400q21 0 35.5 -14.5t14.5 -35.5v-900q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v900q0 21 14.5 35.5t35.5 14.5zM650 700h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v400 q0 21 14.5 35.5t35.5 14.5zM700 600v-300h300v300h-300zM1200 0h-1200v100h1200v-100z" />
<glyph unicode="&#xe246;" d="M50 1000h400q21 0 35.5 -14.5t14.5 -35.5v-350h100v150q0 21 14.5 35.5t35.5 14.5h400q21 0 35.5 -14.5t14.5 -35.5v-150h100v-100h-100v-150q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v150h-100v-350q0 -21 -14.5 -35.5t-35.5 -14.5h-400 q-21 0 -35.5 14.5t-14.5 35.5v800q0 21 14.5 35.5t35.5 14.5zM700 700v-300h300v300h-300z" />
<glyph unicode="&#xe247;" d="M100 0h-100v1200h100v-1200zM250 1100h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM300 1000v-300h300v300h-300zM250 500h900q21 0 35.5 -14.5t14.5 -35.5v-400 q0 -21 -14.5 -35.5t-35.5 -14.5h-900q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe248;" d="M600 1100h150q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-150v-100h450q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-900q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5h350v100h-150q-21 0 -35.5 14.5 t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5h150v100h100v-100zM400 1000v-300h300v300h-300z" />
<glyph unicode="&#xe249;" d="M1200 0h-100v1200h100v-1200zM550 1100h400q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-400q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM600 1000v-300h300v300h-300zM50 500h900q21 0 35.5 -14.5t14.5 -35.5v-400 q0 -21 -14.5 -35.5t-35.5 -14.5h-900q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5z" />
<glyph unicode="&#xe250;" d="M865 565l-494 -494q-23 -23 -41 -23q-14 0 -22 13.5t-8 38.5v1000q0 25 8 38.5t22 13.5q18 0 41 -23l494 -494q14 -14 14 -35t-14 -35z" />
<glyph unicode="&#xe251;" d="M335 635l494 494q29 29 50 20.5t21 -49.5v-1000q0 -41 -21 -49.5t-50 20.5l-494 494q-14 14 -14 35t14 35z" />
<glyph unicode="&#xe252;" d="M100 900h1000q41 0 49.5 -21t-20.5 -50l-494 -494q-14 -14 -35 -14t-35 14l-494 494q-29 29 -20.5 50t49.5 21z" />
<glyph unicode="&#xe253;" d="M635 865l494 -494q29 -29 20.5 -50t-49.5 -21h-1000q-41 0 -49.5 21t20.5 50l494 494q14 14 35 14t35 -14z" />
<glyph unicode="&#xe254;" d="M700 741v-182l-692 -323v221l413 193l-413 193v221zM1200 0h-800v200h800v-200z" />
<glyph unicode="&#xe255;" d="M1200 900h-200v-100h200v-100h-300v300h200v100h-200v100h300v-300zM0 700h50q0 21 4 37t9.5 26.5t18 17.5t22 11t28.5 5.5t31 2t37 0.5h100v-550q0 -22 -25 -34.5t-50 -13.5l-25 -2v-100h400v100q-4 0 -11 0.5t-24 3t-30 7t-24 15t-11 24.5v550h100q25 0 37 -0.5t31 -2 t28.5 -5.5t22 -11t18 -17.5t9.5 -26.5t4 -37h50v300h-800v-300z" />
<glyph unicode="&#xe256;" d="M800 700h-50q0 21 -4 37t-9.5 26.5t-18 17.5t-22 11t-28.5 5.5t-31 2t-37 0.5h-100v-550q0 -22 25 -34.5t50 -14.5l25 -1v-100h-400v100q4 0 11 0.5t24 3t30 7t24 15t11 24.5v550h-100q-25 0 -37 -0.5t-31 -2t-28.5 -5.5t-22 -11t-18 -17.5t-9.5 -26.5t-4 -37h-50v300 h800v-300zM1100 200h-200v-100h200v-100h-300v300h200v100h-200v100h300v-300z" />
<glyph unicode="&#xe257;" d="M701 1098h160q16 0 21 -11t-7 -23l-464 -464l464 -464q12 -12 7 -23t-21 -11h-160q-13 0 -23 9l-471 471q-7 8 -7 18t7 18l471 471q10 9 23 9z" />
<glyph unicode="&#xe258;" d="M339 1098h160q13 0 23 -9l471 -471q7 -8 7 -18t-7 -18l-471 -471q-10 -9 -23 -9h-160q-16 0 -21 11t7 23l464 464l-464 464q-12 12 -7 23t21 11z" />
<glyph unicode="&#xe259;" d="M1087 882q11 -5 11 -21v-160q0 -13 -9 -23l-471 -471q-8 -7 -18 -7t-18 7l-471 471q-9 10 -9 23v160q0 16 11 21t23 -7l464 -464l464 464q12 12 23 7z" />
<glyph unicode="&#xe260;" d="M618 993l471 -471q9 -10 9 -23v-160q0 -16 -11 -21t-23 7l-464 464l-464 -464q-12 -12 -23 -7t-11 21v160q0 13 9 23l471 471q8 7 18 7t18 -7z" />
<glyph unicode="&#xf8ff;" d="M1000 1200q0 -124 -88 -212t-212 -88q0 124 88 212t212 88zM450 1000h100q21 0 40 -14t26 -33l79 -194q5 1 16 3q34 6 54 9.5t60 7t65.5 1t61 -10t56.5 -23t42.5 -42t29 -64t5 -92t-19.5 -121.5q-1 -7 -3 -19.5t-11 -50t-20.5 -73t-32.5 -81.5t-46.5 -83t-64 -70 t-82.5 -50q-13 -5 -42 -5t-65.5 2.5t-47.5 2.5q-14 0 -49.5 -3.5t-63 -3.5t-43.5 7q-57 25 -104.5 78.5t-75 111.5t-46.5 112t-26 90l-7 35q-15 63 -18 115t4.5 88.5t26 64t39.5 43.5t52 25.5t58.5 13t62.5 2t59.5 -4.5t55.5 -8l-147 192q-12 18 -5.5 30t27.5 12z" />
<glyph unicode="&#x1f511;" d="M250 1200h600q21 0 35.5 -14.5t14.5 -35.5v-400q0 -21 -14.5 -35.5t-35.5 -14.5h-150v-500l-255 -178q-19 -9 -32 -1t-13 29v650h-150q-21 0 -35.5 14.5t-14.5 35.5v400q0 21 14.5 35.5t35.5 14.5zM400 1100v-100h300v100h-300z" />
<glyph unicode="&#x1f6aa;" d="M250 1200h750q39 0 69.5 -40.5t30.5 -84.5v-933l-700 -117v950l600 125h-700v-1000h-100v1025q0 23 15.5 49t34.5 26zM500 525v-100l100 20v100z" />
</font>
</defs></svg> ) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\2a"}.glyphicon-plus:before{content:"\2b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:14.33px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled,.btn-default.disabled.active,.btn-default.disabled.focus,.btn-default.disabled:active,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled],.btn-default[disabled].active,.btn-default[disabled].focus,.btn-default[disabled]:active,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default,fieldset[disabled] .btn-default.active,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:active,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled,.btn-primary.disabled.active,.btn-primary.disabled.focus,.btn-primary.disabled:active,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled],.btn-primary[disabled].active,.btn-primary[disabled].focus,.btn-primary[disabled]:active,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary,fieldset[disabled] .btn-primary.active,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:active,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled,.btn-success.disabled.active,.btn-success.disabled.focus,.btn-success.disabled:active,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled],.btn-success[disabled].active,.btn-success[disabled].focus,.btn-success[disabled]:active,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success,fieldset[disabled] .btn-success.active,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:active,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled,.btn-info.disabled.active,.btn-info.disabled.focus,.btn-info.disabled:active,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled],.btn-info[disabled].active,.btn-info[disabled].focus,.btn-info[disabled]:active,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info,fieldset[disabled] .btn-info.active,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:active,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled,.btn-warning.disabled.active,.btn-warning.disabled.focus,.btn-warning.disabled:active,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled],.btn-warning[disabled].active,.btn-warning[disabled].focus,.btn-warning[disabled]:active,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning,fieldset[disabled] .btn-warning.active,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:active,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled,.btn-danger.disabled.active,.btn-danger.disabled.focus,.btn-danger.disabled:active,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled],.btn-danger[disabled].active,.btn-danger[disabled].focus,.btn-danger[disabled]:active,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger,fieldset[disabled] .btn-danger.active,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:active,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:3;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:2;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{min-height:16.43px;padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-15px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-15px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-15px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
</style>
<script>/*!
 * Bootstrap v3.3.5 (http://getbootstrap.com)
 * Copyright 2011-2015 Twitter, Inc.
 * Licensed under the MIT license
 */
if("undefined"==typeof jQuery)throw new Error("Bootstrap's JavaScript requires jQuery");+function(a){"use strict";var b=a.fn.jquery.split(" ")[0].split(".");if(b[0]<2&&b[1]<9||1==b[0]&&9==b[1]&&b[2]<1)throw new Error("Bootstrap's JavaScript requires jQuery version 1.9.1 or higher")}(jQuery),+function(a){"use strict";function b(){var a=document.createElement("bootstrap"),b={WebkitTransition:"webkitTransitionEnd",MozTransition:"transitionend",OTransition:"oTransitionEnd otransitionend",transition:"transitionend"};for(var c in b)if(void 0!==a.style[c])return{end:b[c]};return!1}a.fn.emulateTransitionEnd=function(b){var c=!1,d=this;a(this).one("bsTransitionEnd",function(){c=!0});var e=function(){c||a(d).trigger(a.support.transition.end)};return setTimeout(e,b),this},a(function(){a.support.transition=b(),a.support.transition&&(a.event.special.bsTransitionEnd={bindType:a.support.transition.end,delegateType:a.support.transition.end,handle:function(b){return a(b.target).is(this)?b.handleObj.handler.apply(this,arguments):void 0}})})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var c=a(this),e=c.data("bs.alert");e||c.data("bs.alert",e=new d(this)),"string"==typeof b&&e[b].call(c)})}var c='[data-dismiss="alert"]',d=function(b){a(b).on("click",c,this.close)};d.VERSION="3.3.5",d.TRANSITION_DURATION=150,d.prototype.close=function(b){function c(){g.detach().trigger("closed.bs.alert").remove()}var e=a(this),f=e.attr("data-target");f||(f=e.attr("href"),f=f&&f.replace(/.*(?=#[^\s]*$)/,""));var g=a(f);b&&b.preventDefault(),g.length||(g=e.closest(".alert")),g.trigger(b=a.Event("close.bs.alert")),b.isDefaultPrevented()||(g.removeClass("in"),a.support.transition&&g.hasClass("fade")?g.one("bsTransitionEnd",c).emulateTransitionEnd(d.TRANSITION_DURATION):c())};var e=a.fn.alert;a.fn.alert=b,a.fn.alert.Constructor=d,a.fn.alert.noConflict=function(){return a.fn.alert=e,this},a(document).on("click.bs.alert.data-api",c,d.prototype.close)}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.button"),f="object"==typeof b&&b;e||d.data("bs.button",e=new c(this,f)),"toggle"==b?e.toggle():b&&e.setState(b)})}var c=function(b,d){this.$element=a(b),this.options=a.extend({},c.DEFAULTS,d),this.isLoading=!1};c.VERSION="3.3.5",c.DEFAULTS={loadingText:"loading..."},c.prototype.setState=function(b){var c="disabled",d=this.$element,e=d.is("input")?"val":"html",f=d.data();b+="Text",null==f.resetText&&d.data("resetText",d[e]()),setTimeout(a.proxy(function(){d[e](null==f[b]?this.options[b]:f[b]),"loadingText"==b?(this.isLoading=!0,d.addClass(c).attr(c,c)):this.isLoading&&(this.isLoading=!1,d.removeClass(c).removeAttr(c))},this),0)},c.prototype.toggle=function(){var a=!0,b=this.$element.closest('[data-toggle="buttons"]');if(b.length){var c=this.$element.find("input");"radio"==c.prop("type")?(c.prop("checked")&&(a=!1),b.find(".active").removeClass("active"),this.$element.addClass("active")):"checkbox"==c.prop("type")&&(c.prop("checked")!==this.$element.hasClass("active")&&(a=!1),this.$element.toggleClass("active")),c.prop("checked",this.$element.hasClass("active")),a&&c.trigger("change")}else this.$element.attr("aria-pressed",!this.$element.hasClass("active")),this.$element.toggleClass("active")};var d=a.fn.button;a.fn.button=b,a.fn.button.Constructor=c,a.fn.button.noConflict=function(){return a.fn.button=d,this},a(document).on("click.bs.button.data-api",'[data-toggle^="button"]',function(c){var d=a(c.target);d.hasClass("btn")||(d=d.closest(".btn")),b.call(d,"toggle"),a(c.target).is('input[type="radio"]')||a(c.target).is('input[type="checkbox"]')||c.preventDefault()}).on("focus.bs.button.data-api blur.bs.button.data-api",'[data-toggle^="button"]',function(b){a(b.target).closest(".btn").toggleClass("focus",/^focus(in)?$/.test(b.type))})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.carousel"),f=a.extend({},c.DEFAULTS,d.data(),"object"==typeof b&&b),g="string"==typeof b?b:f.slide;e||d.data("bs.carousel",e=new c(this,f)),"number"==typeof b?e.to(b):g?e[g]():f.interval&&e.pause().cycle()})}var c=function(b,c){this.$element=a(b),this.$indicators=this.$element.find(".carousel-indicators"),this.options=c,this.paused=null,this.sliding=null,this.interval=null,this.$active=null,this.$items=null,this.options.keyboard&&this.$element.on("keydown.bs.carousel",a.proxy(this.keydown,this)),"hover"==this.options.pause&&!("ontouchstart"in document.documentElement)&&this.$element.on("mouseenter.bs.carousel",a.proxy(this.pause,this)).on("mouseleave.bs.carousel",a.proxy(this.cycle,this))};c.VERSION="3.3.5",c.TRANSITION_DURATION=600,c.DEFAULTS={interval:5e3,pause:"hover",wrap:!0,keyboard:!0},c.prototype.keydown=function(a){if(!/input|textarea/i.test(a.target.tagName)){switch(a.which){case 37:this.prev();break;case 39:this.next();break;default:return}a.preventDefault()}},c.prototype.cycle=function(b){return b||(this.paused=!1),this.interval&&clearInterval(this.interval),this.options.interval&&!this.paused&&(this.interval=setInterval(a.proxy(this.next,this),this.options.interval)),this},c.prototype.getItemIndex=function(a){return this.$items=a.parent().children(".item"),this.$items.index(a||this.$active)},c.prototype.getItemForDirection=function(a,b){var c=this.getItemIndex(b),d="prev"==a&&0===c||"next"==a&&c==this.$items.length-1;if(d&&!this.options.wrap)return b;var e="prev"==a?-1:1,f=(c+e)%this.$items.length;return this.$items.eq(f)},c.prototype.to=function(a){var b=this,c=this.getItemIndex(this.$active=this.$element.find(".item.active"));return a>this.$items.length-1||0>a?void 0:this.sliding?this.$element.one("slid.bs.carousel",function(){b.to(a)}):c==a?this.pause().cycle():this.slide(a>c?"next":"prev",this.$items.eq(a))},c.prototype.pause=function(b){return b||(this.paused=!0),this.$element.find(".next, .prev").length&&a.support.transition&&(this.$element.trigger(a.support.transition.end),this.cycle(!0)),this.interval=clearInterval(this.interval),this},c.prototype.next=function(){return this.sliding?void 0:this.slide("next")},c.prototype.prev=function(){return this.sliding?void 0:this.slide("prev")},c.prototype.slide=function(b,d){var e=this.$element.find(".item.active"),f=d||this.getItemForDirection(b,e),g=this.interval,h="next"==b?"left":"right",i=this;if(f.hasClass("active"))return this.sliding=!1;var j=f[0],k=a.Event("slide.bs.carousel",{relatedTarget:j,direction:h});if(this.$element.trigger(k),!k.isDefaultPrevented()){if(this.sliding=!0,g&&this.pause(),this.$indicators.length){this.$indicators.find(".active").removeClass("active");var l=a(this.$indicators.children()[this.getItemIndex(f)]);l&&l.addClass("active")}var m=a.Event("slid.bs.carousel",{relatedTarget:j,direction:h});return a.support.transition&&this.$element.hasClass("slide")?(f.addClass(b),f[0].offsetWidth,e.addClass(h),f.addClass(h),e.one("bsTransitionEnd",function(){f.removeClass([b,h].join(" ")).addClass("active"),e.removeClass(["active",h].join(" ")),i.sliding=!1,setTimeout(function(){i.$element.trigger(m)},0)}).emulateTransitionEnd(c.TRANSITION_DURATION)):(e.removeClass("active"),f.addClass("active"),this.sliding=!1,this.$element.trigger(m)),g&&this.cycle(),this}};var d=a.fn.carousel;a.fn.carousel=b,a.fn.carousel.Constructor=c,a.fn.carousel.noConflict=function(){return a.fn.carousel=d,this};var e=function(c){var d,e=a(this),f=a(e.attr("data-target")||(d=e.attr("href"))&&d.replace(/.*(?=#[^\s]+$)/,""));if(f.hasClass("carousel")){var g=a.extend({},f.data(),e.data()),h=e.attr("data-slide-to");h&&(g.interval=!1),b.call(f,g),h&&f.data("bs.carousel").to(h),c.preventDefault()}};a(document).on("click.bs.carousel.data-api","[data-slide]",e).on("click.bs.carousel.data-api","[data-slide-to]",e),a(window).on("load",function(){a('[data-ride="carousel"]').each(function(){var c=a(this);b.call(c,c.data())})})}(jQuery),+function(a){"use strict";function b(b){var c,d=b.attr("data-target")||(c=b.attr("href"))&&c.replace(/.*(?=#[^\s]+$)/,"");return a(d)}function c(b){return this.each(function(){var c=a(this),e=c.data("bs.collapse"),f=a.extend({},d.DEFAULTS,c.data(),"object"==typeof b&&b);!e&&f.toggle&&/show|hide/.test(b)&&(f.toggle=!1),e||c.data("bs.collapse",e=new d(this,f)),"string"==typeof b&&e[b]()})}var d=function(b,c){this.$element=a(b),this.options=a.extend({},d.DEFAULTS,c),this.$trigger=a('[data-toggle="collapse"][href="#'+b.id+'"],[data-toggle="collapse"][data-target="#'+b.id+'"]'),this.transitioning=null,this.options.parent?this.$parent=this.getParent():this.addAriaAndCollapsedClass(this.$element,this.$trigger),this.options.toggle&&this.toggle()};d.VERSION="3.3.5",d.TRANSITION_DURATION=350,d.DEFAULTS={toggle:!0},d.prototype.dimension=function(){var a=this.$element.hasClass("width");return a?"width":"height"},d.prototype.show=function(){if(!this.transitioning&&!this.$element.hasClass("in")){var b,e=this.$parent&&this.$parent.children(".panel").children(".in, .collapsing");if(!(e&&e.length&&(b=e.data("bs.collapse"),b&&b.transitioning))){var f=a.Event("show.bs.collapse");if(this.$element.trigger(f),!f.isDefaultPrevented()){e&&e.length&&(c.call(e,"hide"),b||e.data("bs.collapse",null));var g=this.dimension();this.$element.removeClass("collapse").addClass("collapsing")[g](0).attr("aria-expanded",!0),this.$trigger.removeClass("collapsed").attr("aria-expanded",!0),this.transitioning=1;var h=function(){this.$element.removeClass("collapsing").addClass("collapse in")[g](""),this.transitioning=0,this.$element.trigger("shown.bs.collapse")};if(!a.support.transition)return h.call(this);var i=a.camelCase(["scroll",g].join("-"));this.$element.one("bsTransitionEnd",a.proxy(h,this)).emulateTransitionEnd(d.TRANSITION_DURATION)[g](this.$element[0][i])}}}},d.prototype.hide=function(){if(!this.transitioning&&this.$element.hasClass("in")){var b=a.Event("hide.bs.collapse");if(this.$element.trigger(b),!b.isDefaultPrevented()){var c=this.dimension();this.$element[c](this.$element[c]())[0].offsetHeight,this.$element.addClass("collapsing").removeClass("collapse in").attr("aria-expanded",!1),this.$trigger.addClass("collapsed").attr("aria-expanded",!1),this.transitioning=1;var e=function(){this.transitioning=0,this.$element.removeClass("collapsing").addClass("collapse").trigger("hidden.bs.collapse")};return a.support.transition?void this.$element[c](0).one("bsTransitionEnd",a.proxy(e,this)).emulateTransitionEnd(d.TRANSITION_DURATION):e.call(this)}}},d.prototype.toggle=function(){this[this.$element.hasClass("in")?"hide":"show"]()},d.prototype.getParent=function(){return a(this.options.parent).find('[data-toggle="collapse"][data-parent="'+this.options.parent+'"]').each(a.proxy(function(c,d){var e=a(d);this.addAriaAndCollapsedClass(b(e),e)},this)).end()},d.prototype.addAriaAndCollapsedClass=function(a,b){var c=a.hasClass("in");a.attr("aria-expanded",c),b.toggleClass("collapsed",!c).attr("aria-expanded",c)};var e=a.fn.collapse;a.fn.collapse=c,a.fn.collapse.Constructor=d,a.fn.collapse.noConflict=function(){return a.fn.collapse=e,this},a(document).on("click.bs.collapse.data-api",'[data-toggle="collapse"]',function(d){var e=a(this);e.attr("data-target")||d.preventDefault();var f=b(e),g=f.data("bs.collapse"),h=g?"toggle":e.data();c.call(f,h)})}(jQuery),+function(a){"use strict";function b(b){var c=b.attr("data-target");c||(c=b.attr("href"),c=c&&/#[A-Za-z]/.test(c)&&c.replace(/.*(?=#[^\s]*$)/,""));var d=c&&a(c);return d&&d.length?d:b.parent()}function c(c){c&&3===c.which||(a(e).remove(),a(f).each(function(){var d=a(this),e=b(d),f={relatedTarget:this};e.hasClass("open")&&(c&&"click"==c.type&&/input|textarea/i.test(c.target.tagName)&&a.contains(e[0],c.target)||(e.trigger(c=a.Event("hide.bs.dropdown",f)),c.isDefaultPrevented()||(d.attr("aria-expanded","false"),e.removeClass("open").trigger("hidden.bs.dropdown",f))))}))}function d(b){return this.each(function(){var c=a(this),d=c.data("bs.dropdown");d||c.data("bs.dropdown",d=new g(this)),"string"==typeof b&&d[b].call(c)})}var e=".dropdown-backdrop",f='[data-toggle="dropdown"]',g=function(b){a(b).on("click.bs.dropdown",this.toggle)};g.VERSION="3.3.5",g.prototype.toggle=function(d){var e=a(this);if(!e.is(".disabled, :disabled")){var f=b(e),g=f.hasClass("open");if(c(),!g){"ontouchstart"in document.documentElement&&!f.closest(".navbar-nav").length&&a(document.createElement("div")).addClass("dropdown-backdrop").insertAfter(a(this)).on("click",c);var h={relatedTarget:this};if(f.trigger(d=a.Event("show.bs.dropdown",h)),d.isDefaultPrevented())return;e.trigger("focus").attr("aria-expanded","true"),f.toggleClass("open").trigger("shown.bs.dropdown",h)}return!1}},g.prototype.keydown=function(c){if(/(38|40|27|32)/.test(c.which)&&!/input|textarea/i.test(c.target.tagName)){var d=a(this);if(c.preventDefault(),c.stopPropagation(),!d.is(".disabled, :disabled")){var e=b(d),g=e.hasClass("open");if(!g&&27!=c.which||g&&27==c.which)return 27==c.which&&e.find(f).trigger("focus"),d.trigger("click");var h=" li:not(.disabled):visible a",i=e.find(".dropdown-menu"+h);if(i.length){var j=i.index(c.target);38==c.which&&j>0&&j--,40==c.which&&j<i.length-1&&j++,~j||(j=0),i.eq(j).trigger("focus")}}}};var h=a.fn.dropdown;a.fn.dropdown=d,a.fn.dropdown.Constructor=g,a.fn.dropdown.noConflict=function(){return a.fn.dropdown=h,this},a(document).on("click.bs.dropdown.data-api",c).on("click.bs.dropdown.data-api",".dropdown form",function(a){a.stopPropagation()}).on("click.bs.dropdown.data-api",f,g.prototype.toggle).on("keydown.bs.dropdown.data-api",f,g.prototype.keydown).on("keydown.bs.dropdown.data-api",".dropdown-menu",g.prototype.keydown)}(jQuery),+function(a){"use strict";function b(b,d){return this.each(function(){var e=a(this),f=e.data("bs.modal"),g=a.extend({},c.DEFAULTS,e.data(),"object"==typeof b&&b);f||e.data("bs.modal",f=new c(this,g)),"string"==typeof b?f[b](d):g.show&&f.show(d)})}var c=function(b,c){this.options=c,this.$body=a(document.body),this.$element=a(b),this.$dialog=this.$element.find(".modal-dialog"),this.$backdrop=null,this.isShown=null,this.originalBodyPad=null,this.scrollbarWidth=0,this.ignoreBackdropClick=!1,this.options.remote&&this.$element.find(".modal-content").load(this.options.remote,a.proxy(function(){this.$element.trigger("loaded.bs.modal")},this))};c.VERSION="3.3.5",c.TRANSITION_DURATION=300,c.BACKDROP_TRANSITION_DURATION=150,c.DEFAULTS={backdrop:!0,keyboard:!0,show:!0},c.prototype.toggle=function(a){return this.isShown?this.hide():this.show(a)},c.prototype.show=function(b){var d=this,e=a.Event("show.bs.modal",{relatedTarget:b});this.$element.trigger(e),this.isShown||e.isDefaultPrevented()||(this.isShown=!0,this.checkScrollbar(),this.setScrollbar(),this.$body.addClass("modal-open"),this.escape(),this.resize(),this.$element.on("click.dismiss.bs.modal",'[data-dismiss="modal"]',a.proxy(this.hide,this)),this.$dialog.on("mousedown.dismiss.bs.modal",function(){d.$element.one("mouseup.dismiss.bs.modal",function(b){a(b.target).is(d.$element)&&(d.ignoreBackdropClick=!0)})}),this.backdrop(function(){var e=a.support.transition&&d.$element.hasClass("fade");d.$element.parent().length||d.$element.appendTo(d.$body),d.$element.show().scrollTop(0),d.adjustDialog(),e&&d.$element[0].offsetWidth,d.$element.addClass("in"),d.enforceFocus();var f=a.Event("shown.bs.modal",{relatedTarget:b});e?d.$dialog.one("bsTransitionEnd",function(){d.$element.trigger("focus").trigger(f)}).emulateTransitionEnd(c.TRANSITION_DURATION):d.$element.trigger("focus").trigger(f)}))},c.prototype.hide=function(b){b&&b.preventDefault(),b=a.Event("hide.bs.modal"),this.$element.trigger(b),this.isShown&&!b.isDefaultPrevented()&&(this.isShown=!1,this.escape(),this.resize(),a(document).off("focusin.bs.modal"),this.$element.removeClass("in").off("click.dismiss.bs.modal").off("mouseup.dismiss.bs.modal"),this.$dialog.off("mousedown.dismiss.bs.modal"),a.support.transition&&this.$element.hasClass("fade")?this.$element.one("bsTransitionEnd",a.proxy(this.hideModal,this)).emulateTransitionEnd(c.TRANSITION_DURATION):this.hideModal())},c.prototype.enforceFocus=function(){a(document).off("focusin.bs.modal").on("focusin.bs.modal",a.proxy(function(a){this.$element[0]===a.target||this.$element.has(a.target).length||this.$element.trigger("focus")},this))},c.prototype.escape=function(){this.isShown&&this.options.keyboard?this.$element.on("keydown.dismiss.bs.modal",a.proxy(function(a){27==a.which&&this.hide()},this)):this.isShown||this.$element.off("keydown.dismiss.bs.modal")},c.prototype.resize=function(){this.isShown?a(window).on("resize.bs.modal",a.proxy(this.handleUpdate,this)):a(window).off("resize.bs.modal")},c.prototype.hideModal=function(){var a=this;this.$element.hide(),this.backdrop(function(){a.$body.removeClass("modal-open"),a.resetAdjustments(),a.resetScrollbar(),a.$element.trigger("hidden.bs.modal")})},c.prototype.removeBackdrop=function(){this.$backdrop&&this.$backdrop.remove(),this.$backdrop=null},c.prototype.backdrop=function(b){var d=this,e=this.$element.hasClass("fade")?"fade":"";if(this.isShown&&this.options.backdrop){var f=a.support.transition&&e;if(this.$backdrop=a(document.createElement("div")).addClass("modal-backdrop "+e).appendTo(this.$body),this.$element.on("click.dismiss.bs.modal",a.proxy(function(a){return this.ignoreBackdropClick?void(this.ignoreBackdropClick=!1):void(a.target===a.currentTarget&&("static"==this.options.backdrop?this.$element[0].focus():this.hide()))},this)),f&&this.$backdrop[0].offsetWidth,this.$backdrop.addClass("in"),!b)return;f?this.$backdrop.one("bsTransitionEnd",b).emulateTransitionEnd(c.BACKDROP_TRANSITION_DURATION):b()}else if(!this.isShown&&this.$backdrop){this.$backdrop.removeClass("in");var g=function(){d.removeBackdrop(),b&&b()};a.support.transition&&this.$element.hasClass("fade")?this.$backdrop.one("bsTransitionEnd",g).emulateTransitionEnd(c.BACKDROP_TRANSITION_DURATION):g()}else b&&b()},c.prototype.handleUpdate=function(){this.adjustDialog()},c.prototype.adjustDialog=function(){var a=this.$element[0].scrollHeight>document.documentElement.clientHeight;this.$element.css({paddingLeft:!this.bodyIsOverflowing&&a?this.scrollbarWidth:"",paddingRight:this.bodyIsOverflowing&&!a?this.scrollbarWidth:""})},c.prototype.resetAdjustments=function(){this.$element.css({paddingLeft:"",paddingRight:""})},c.prototype.checkScrollbar=function(){var a=window.innerWidth;if(!a){var b=document.documentElement.getBoundingClientRect();a=b.right-Math.abs(b.left)}this.bodyIsOverflowing=document.body.clientWidth<a,this.scrollbarWidth=this.measureScrollbar()},c.prototype.setScrollbar=function(){var a=parseInt(this.$body.css("padding-right")||0,10);this.originalBodyPad=document.body.style.paddingRight||"",this.bodyIsOverflowing&&this.$body.css("padding-right",a+this.scrollbarWidth)},c.prototype.resetScrollbar=function(){this.$body.css("padding-right",this.originalBodyPad)},c.prototype.measureScrollbar=function(){var a=document.createElement("div");a.className="modal-scrollbar-measure",this.$body.append(a);var b=a.offsetWidth-a.clientWidth;return this.$body[0].removeChild(a),b};var d=a.fn.modal;a.fn.modal=b,a.fn.modal.Constructor=c,a.fn.modal.noConflict=function(){return a.fn.modal=d,this},a(document).on("click.bs.modal.data-api",'[data-toggle="modal"]',function(c){var d=a(this),e=d.attr("href"),f=a(d.attr("data-target")||e&&e.replace(/.*(?=#[^\s]+$)/,"")),g=f.data("bs.modal")?"toggle":a.extend({remote:!/#/.test(e)&&e},f.data(),d.data());d.is("a")&&c.preventDefault(),f.one("show.bs.modal",function(a){a.isDefaultPrevented()||f.one("hidden.bs.modal",function(){d.is(":visible")&&d.trigger("focus")})}),b.call(f,g,this)})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.tooltip"),f="object"==typeof b&&b;(e||!/destroy|hide/.test(b))&&(e||d.data("bs.tooltip",e=new c(this,f)),"string"==typeof b&&e[b]())})}var c=function(a,b){this.type=null,this.options=null,this.enabled=null,this.timeout=null,this.hoverState=null,this.$element=null,this.inState=null,this.init("tooltip",a,b)};c.VERSION="3.3.5",c.TRANSITION_DURATION=150,c.DEFAULTS={animation:!0,placement:"top",selector:!1,template:'<div class="tooltip" role="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner"></div></div>',trigger:"hover focus",title:"",delay:0,html:!1,container:!1,viewport:{selector:"body",padding:0}},c.prototype.init=function(b,c,d){if(this.enabled=!0,this.type=b,this.$element=a(c),this.options=this.getOptions(d),this.$viewport=this.options.viewport&&a(a.isFunction(this.options.viewport)?this.options.viewport.call(this,this.$element):this.options.viewport.selector||this.options.viewport),this.inState={click:!1,hover:!1,focus:!1},this.$element[0]instanceof document.constructor&&!this.options.selector)throw new Error("`selector` option must be specified when initializing "+this.type+" on the window.document object!");for(var e=this.options.trigger.split(" "),f=e.length;f--;){var g=e[f];if("click"==g)this.$element.on("click."+this.type,this.options.selector,a.proxy(this.toggle,this));else if("manual"!=g){var h="hover"==g?"mouseenter":"focusin",i="hover"==g?"mouseleave":"focusout";this.$element.on(h+"."+this.type,this.options.selector,a.proxy(this.enter,this)),this.$element.on(i+"."+this.type,this.options.selector,a.proxy(this.leave,this))}}this.options.selector?this._options=a.extend({},this.options,{trigger:"manual",selector:""}):this.fixTitle()},c.prototype.getDefaults=function(){return c.DEFAULTS},c.prototype.getOptions=function(b){return b=a.extend({},this.getDefaults(),this.$element.data(),b),b.delay&&"number"==typeof b.delay&&(b.delay={show:b.delay,hide:b.delay}),b},c.prototype.getDelegateOptions=function(){var b={},c=this.getDefaults();return this._options&&a.each(this._options,function(a,d){c[a]!=d&&(b[a]=d)}),b},c.prototype.enter=function(b){var c=b instanceof this.constructor?b:a(b.currentTarget).data("bs."+this.type);return c||(c=new this.constructor(b.currentTarget,this.getDelegateOptions()),a(b.currentTarget).data("bs."+this.type,c)),b instanceof a.Event&&(c.inState["focusin"==b.type?"focus":"hover"]=!0),c.tip().hasClass("in")||"in"==c.hoverState?void(c.hoverState="in"):(clearTimeout(c.timeout),c.hoverState="in",c.options.delay&&c.options.delay.show?void(c.timeout=setTimeout(function(){"in"==c.hoverState&&c.show()},c.options.delay.show)):c.show())},c.prototype.isInStateTrue=function(){for(var a in this.inState)if(this.inState[a])return!0;return!1},c.prototype.leave=function(b){var c=b instanceof this.constructor?b:a(b.currentTarget).data("bs."+this.type);return c||(c=new this.constructor(b.currentTarget,this.getDelegateOptions()),a(b.currentTarget).data("bs."+this.type,c)),b instanceof a.Event&&(c.inState["focusout"==b.type?"focus":"hover"]=!1),c.isInStateTrue()?void 0:(clearTimeout(c.timeout),c.hoverState="out",c.options.delay&&c.options.delay.hide?void(c.timeout=setTimeout(function(){"out"==c.hoverState&&c.hide()},c.options.delay.hide)):c.hide())},c.prototype.show=function(){var b=a.Event("show.bs."+this.type);if(this.hasContent()&&this.enabled){this.$element.trigger(b);var d=a.contains(this.$element[0].ownerDocument.documentElement,this.$element[0]);if(b.isDefaultPrevented()||!d)return;var e=this,f=this.tip(),g=this.getUID(this.type);this.setContent(),f.attr("id",g),this.$element.attr("aria-describedby",g),this.options.animation&&f.addClass("fade");var h="function"==typeof this.options.placement?this.options.placement.call(this,f[0],this.$element[0]):this.options.placement,i=/\s?auto?\s?/i,j=i.test(h);j&&(h=h.replace(i,"")||"top"),f.detach().css({top:0,left:0,display:"block"}).addClass(h).data("bs."+this.type,this),this.options.container?f.appendTo(this.options.container):f.insertAfter(this.$element),this.$element.trigger("inserted.bs."+this.type);var k=this.getPosition(),l=f[0].offsetWidth,m=f[0].offsetHeight;if(j){var n=h,o=this.getPosition(this.$viewport);h="bottom"==h&&k.bottom+m>o.bottom?"top":"top"==h&&k.top-m<o.top?"bottom":"right"==h&&k.right+l>o.width?"left":"left"==h&&k.left-l<o.left?"right":h,f.removeClass(n).addClass(h)}var p=this.getCalculatedOffset(h,k,l,m);this.applyPlacement(p,h);var q=function(){var a=e.hoverState;e.$element.trigger("shown.bs."+e.type),e.hoverState=null,"out"==a&&e.leave(e)};a.support.transition&&this.$tip.hasClass("fade")?f.one("bsTransitionEnd",q).emulateTransitionEnd(c.TRANSITION_DURATION):q()}},c.prototype.applyPlacement=function(b,c){var d=this.tip(),e=d[0].offsetWidth,f=d[0].offsetHeight,g=parseInt(d.css("margin-top"),10),h=parseInt(d.css("margin-left"),10);isNaN(g)&&(g=0),isNaN(h)&&(h=0),b.top+=g,b.left+=h,a.offset.setOffset(d[0],a.extend({using:function(a){d.css({top:Math.round(a.top),left:Math.round(a.left)})}},b),0),d.addClass("in");var i=d[0].offsetWidth,j=d[0].offsetHeight;"top"==c&&j!=f&&(b.top=b.top+f-j);var k=this.getViewportAdjustedDelta(c,b,i,j);k.left?b.left+=k.left:b.top+=k.top;var l=/top|bottom/.test(c),m=l?2*k.left-e+i:2*k.top-f+j,n=l?"offsetWidth":"offsetHeight";d.offset(b),this.replaceArrow(m,d[0][n],l)},c.prototype.replaceArrow=function(a,b,c){this.arrow().css(c?"left":"top",50*(1-a/b)+"%").css(c?"top":"left","")},c.prototype.setContent=function(){var a=this.tip(),b=this.getTitle();a.find(".tooltip-inner")[this.options.html?"html":"text"](b),a.removeClass("fade in top bottom left right")},c.prototype.hide=function(b){function d(){"in"!=e.hoverState&&f.detach(),e.$element.removeAttr("aria-describedby").trigger("hidden.bs."+e.type),b&&b()}var e=this,f=a(this.$tip),g=a.Event("hide.bs."+this.type);return this.$element.trigger(g),g.isDefaultPrevented()?void 0:(f.removeClass("in"),a.support.transition&&f.hasClass("fade")?f.one("bsTransitionEnd",d).emulateTransitionEnd(c.TRANSITION_DURATION):d(),this.hoverState=null,this)},c.prototype.fixTitle=function(){var a=this.$element;(a.attr("title")||"string"!=typeof a.attr("data-original-title"))&&a.attr("data-original-title",a.attr("title")||"").attr("title","")},c.prototype.hasContent=function(){return this.getTitle()},c.prototype.getPosition=function(b){b=b||this.$element;var c=b[0],d="BODY"==c.tagName,e=c.getBoundingClientRect();null==e.width&&(e=a.extend({},e,{width:e.right-e.left,height:e.bottom-e.top}));var f=d?{top:0,left:0}:b.offset(),g={scroll:d?document.documentElement.scrollTop||document.body.scrollTop:b.scrollTop()},h=d?{width:a(window).width(),height:a(window).height()}:null;return a.extend({},e,g,h,f)},c.prototype.getCalculatedOffset=function(a,b,c,d){return"bottom"==a?{top:b.top+b.height,left:b.left+b.width/2-c/2}:"top"==a?{top:b.top-d,left:b.left+b.width/2-c/2}:"left"==a?{top:b.top+b.height/2-d/2,left:b.left-c}:{top:b.top+b.height/2-d/2,left:b.left+b.width}},c.prototype.getViewportAdjustedDelta=function(a,b,c,d){var e={top:0,left:0};if(!this.$viewport)return e;var f=this.options.viewport&&this.options.viewport.padding||0,g=this.getPosition(this.$viewport);if(/right|left/.test(a)){var h=b.top-f-g.scroll,i=b.top+f-g.scroll+d;h<g.top?e.top=g.top-h:i>g.top+g.height&&(e.top=g.top+g.height-i)}else{var j=b.left-f,k=b.left+f+c;j<g.left?e.left=g.left-j:k>g.right&&(e.left=g.left+g.width-k)}return e},c.prototype.getTitle=function(){var a,b=this.$element,c=this.options;return a=b.attr("data-original-title")||("function"==typeof c.title?c.title.call(b[0]):c.title)},c.prototype.getUID=function(a){do a+=~~(1e6*Math.random());while(document.getElementById(a));return a},c.prototype.tip=function(){if(!this.$tip&&(this.$tip=a(this.options.template),1!=this.$tip.length))throw new Error(this.type+" `template` option must consist of exactly 1 top-level element!");return this.$tip},c.prototype.arrow=function(){return this.$arrow=this.$arrow||this.tip().find(".tooltip-arrow")},c.prototype.enable=function(){this.enabled=!0},c.prototype.disable=function(){this.enabled=!1},c.prototype.toggleEnabled=function(){this.enabled=!this.enabled},c.prototype.toggle=function(b){var c=this;b&&(c=a(b.currentTarget).data("bs."+this.type),c||(c=new this.constructor(b.currentTarget,this.getDelegateOptions()),a(b.currentTarget).data("bs."+this.type,c))),b?(c.inState.click=!c.inState.click,c.isInStateTrue()?c.enter(c):c.leave(c)):c.tip().hasClass("in")?c.leave(c):c.enter(c)},c.prototype.destroy=function(){var a=this;clearTimeout(this.timeout),this.hide(function(){a.$element.off("."+a.type).removeData("bs."+a.type),a.$tip&&a.$tip.detach(),a.$tip=null,a.$arrow=null,a.$viewport=null})};var d=a.fn.tooltip;a.fn.tooltip=b,a.fn.tooltip.Constructor=c,a.fn.tooltip.noConflict=function(){return a.fn.tooltip=d,this}}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.popover"),f="object"==typeof b&&b;(e||!/destroy|hide/.test(b))&&(e||d.data("bs.popover",e=new c(this,f)),"string"==typeof b&&e[b]())})}var c=function(a,b){this.init("popover",a,b)};if(!a.fn.tooltip)throw new Error("Popover requires tooltip.js");c.VERSION="3.3.5",c.DEFAULTS=a.extend({},a.fn.tooltip.Constructor.DEFAULTS,{placement:"right",trigger:"click",content:"",template:'<div class="popover" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>'}),c.prototype=a.extend({},a.fn.tooltip.Constructor.prototype),c.prototype.constructor=c,c.prototype.getDefaults=function(){return c.DEFAULTS},c.prototype.setContent=function(){var a=this.tip(),b=this.getTitle(),c=this.getContent();a.find(".popover-title")[this.options.html?"html":"text"](b),a.find(".popover-content").children().detach().end()[this.options.html?"string"==typeof c?"html":"append":"text"](c),a.removeClass("fade top bottom left right in"),a.find(".popover-title").html()||a.find(".popover-title").hide()},c.prototype.hasContent=function(){return this.getTitle()||this.getContent()},c.prototype.getContent=function(){var a=this.$element,b=this.options;return a.attr("data-content")||("function"==typeof b.content?b.content.call(a[0]):b.content)},c.prototype.arrow=function(){return this.$arrow=this.$arrow||this.tip().find(".arrow")};var d=a.fn.popover;a.fn.popover=b,a.fn.popover.Constructor=c,a.fn.popover.noConflict=function(){return a.fn.popover=d,this}}(jQuery),+function(a){"use strict";function b(c,d){this.$body=a(document.body),this.$scrollElement=a(a(c).is(document.body)?window:c),this.options=a.extend({},b.DEFAULTS,d),this.selector=(this.options.target||"")+" .nav li > a",this.offsets=[],this.targets=[],this.activeTarget=null,this.scrollHeight=0,this.$scrollElement.on("scroll.bs.scrollspy",a.proxy(this.process,this)),this.refresh(),this.process()}function c(c){return this.each(function(){var d=a(this),e=d.data("bs.scrollspy"),f="object"==typeof c&&c;e||d.data("bs.scrollspy",e=new b(this,f)),"string"==typeof c&&e[c]()})}b.VERSION="3.3.5",b.DEFAULTS={offset:10},b.prototype.getScrollHeight=function(){return this.$scrollElement[0].scrollHeight||Math.max(this.$body[0].scrollHeight,document.documentElement.scrollHeight)},b.prototype.refresh=function(){var b=this,c="offset",d=0;this.offsets=[],this.targets=[],this.scrollHeight=this.getScrollHeight(),a.isWindow(this.$scrollElement[0])||(c="position",d=this.$scrollElement.scrollTop()),this.$body.find(this.selector).map(function(){var b=a(this),e=b.data("target")||b.attr("href"),f=/^#./.test(e)&&a(e);return f&&f.length&&f.is(":visible")&&[[f[c]().top+d,e]]||null}).sort(function(a,b){return a[0]-b[0]}).each(function(){b.offsets.push(this[0]),b.targets.push(this[1])})},b.prototype.process=function(){var a,b=this.$scrollElement.scrollTop()+this.options.offset,c=this.getScrollHeight(),d=this.options.offset+c-this.$scrollElement.height(),e=this.offsets,f=this.targets,g=this.activeTarget;if(this.scrollHeight!=c&&this.refresh(),b>=d)return g!=(a=f[f.length-1])&&this.activate(a);if(g&&b<e[0])return this.activeTarget=null,this.clear();for(a=e.length;a--;)g!=f[a]&&b>=e[a]&&(void 0===e[a+1]||b<e[a+1])&&this.activate(f[a])},b.prototype.activate=function(b){this.activeTarget=b,this.clear();var c=this.selector+'[data-target="'+b+'"],'+this.selector+'[href="'+b+'"]',d=a(c).parents("li").addClass("active");d.parent(".dropdown-menu").length&&(d=d.closest("li.dropdown").addClass("active")),
d.trigger("activate.bs.scrollspy")},b.prototype.clear=function(){a(this.selector).parentsUntil(this.options.target,".active").removeClass("active")};var d=a.fn.scrollspy;a.fn.scrollspy=c,a.fn.scrollspy.Constructor=b,a.fn.scrollspy.noConflict=function(){return a.fn.scrollspy=d,this},a(window).on("load.bs.scrollspy.data-api",function(){a('[data-spy="scroll"]').each(function(){var b=a(this);c.call(b,b.data())})})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.tab");e||d.data("bs.tab",e=new c(this)),"string"==typeof b&&e[b]()})}var c=function(b){this.element=a(b)};c.VERSION="3.3.5",c.TRANSITION_DURATION=150,c.prototype.show=function(){var b=this.element,c=b.closest("ul:not(.dropdown-menu)"),d=b.data("target");if(d||(d=b.attr("href"),d=d&&d.replace(/.*(?=#[^\s]*$)/,"")),!b.parent("li").hasClass("active")){var e=c.find(".active:last a"),f=a.Event("hide.bs.tab",{relatedTarget:b[0]}),g=a.Event("show.bs.tab",{relatedTarget:e[0]});if(e.trigger(f),b.trigger(g),!g.isDefaultPrevented()&&!f.isDefaultPrevented()){var h=a(d);this.activate(b.closest("li"),c),this.activate(h,h.parent(),function(){e.trigger({type:"hidden.bs.tab",relatedTarget:b[0]}),b.trigger({type:"shown.bs.tab",relatedTarget:e[0]})})}}},c.prototype.activate=function(b,d,e){function f(){g.removeClass("active").find("> .dropdown-menu > .active").removeClass("active").end().find('[data-toggle="tab"]').attr("aria-expanded",!1),b.addClass("active").find('[data-toggle="tab"]').attr("aria-expanded",!0),h?(b[0].offsetWidth,b.addClass("in")):b.removeClass("fade"),b.parent(".dropdown-menu").length&&b.closest("li.dropdown").addClass("active").end().find('[data-toggle="tab"]').attr("aria-expanded",!0),e&&e()}var g=d.find("> .active"),h=e&&a.support.transition&&(g.length&&g.hasClass("fade")||!!d.find("> .fade").length);g.length&&h?g.one("bsTransitionEnd",f).emulateTransitionEnd(c.TRANSITION_DURATION):f(),g.removeClass("in")};var d=a.fn.tab;a.fn.tab=b,a.fn.tab.Constructor=c,a.fn.tab.noConflict=function(){return a.fn.tab=d,this};var e=function(c){c.preventDefault(),b.call(a(this),"show")};a(document).on("click.bs.tab.data-api",'[data-toggle="tab"]',e).on("click.bs.tab.data-api",'[data-toggle="pill"]',e)}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.affix"),f="object"==typeof b&&b;e||d.data("bs.affix",e=new c(this,f)),"string"==typeof b&&e[b]()})}var c=function(b,d){this.options=a.extend({},c.DEFAULTS,d),this.$target=a(this.options.target).on("scroll.bs.affix.data-api",a.proxy(this.checkPosition,this)).on("click.bs.affix.data-api",a.proxy(this.checkPositionWithEventLoop,this)),this.$element=a(b),this.affixed=null,this.unpin=null,this.pinnedOffset=null,this.checkPosition()};c.VERSION="3.3.5",c.RESET="affix affix-top affix-bottom",c.DEFAULTS={offset:0,target:window},c.prototype.getState=function(a,b,c,d){var e=this.$target.scrollTop(),f=this.$element.offset(),g=this.$target.height();if(null!=c&&"top"==this.affixed)return c>e?"top":!1;if("bottom"==this.affixed)return null!=c?e+this.unpin<=f.top?!1:"bottom":a-d>=e+g?!1:"bottom";var h=null==this.affixed,i=h?e:f.top,j=h?g:b;return null!=c&&c>=e?"top":null!=d&&i+j>=a-d?"bottom":!1},c.prototype.getPinnedOffset=function(){if(this.pinnedOffset)return this.pinnedOffset;this.$element.removeClass(c.RESET).addClass("affix");var a=this.$target.scrollTop(),b=this.$element.offset();return this.pinnedOffset=b.top-a},c.prototype.checkPositionWithEventLoop=function(){setTimeout(a.proxy(this.checkPosition,this),1)},c.prototype.checkPosition=function(){if(this.$element.is(":visible")){var b=this.$element.height(),d=this.options.offset,e=d.top,f=d.bottom,g=Math.max(a(document).height(),a(document.body).height());"object"!=typeof d&&(f=e=d),"function"==typeof e&&(e=d.top(this.$element)),"function"==typeof f&&(f=d.bottom(this.$element));var h=this.getState(g,b,e,f);if(this.affixed!=h){null!=this.unpin&&this.$element.css("top","");var i="affix"+(h?"-"+h:""),j=a.Event(i+".bs.affix");if(this.$element.trigger(j),j.isDefaultPrevented())return;this.affixed=h,this.unpin="bottom"==h?this.getPinnedOffset():null,this.$element.removeClass(c.RESET).addClass(i).trigger(i.replace("affix","affixed")+".bs.affix")}"bottom"==h&&this.$element.offset({top:g-b-f})}};var d=a.fn.affix;a.fn.affix=b,a.fn.affix.Constructor=c,a.fn.affix.noConflict=function(){return a.fn.affix=d,this},a(window).on("load",function(){a('[data-spy="affix"]').each(function(){var c=a(this),d=c.data();d.offset=d.offset||{},null!=d.offsetBottom&&(d.offset.bottom=d.offsetBottom),null!=d.offsetTop&&(d.offset.top=d.offsetTop),b.call(c,d)})})}(jQuery);</script>
<script>/**
* @preserve HTML5 Shiv 3.7.2 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
*/
// Only run this code in IE 8
if (!!window.navigator.userAgent.match("MSIE 8")) {
!function(a,b){function c(a,b){var c=a.createElement("p"),d=a.getElementsByTagName("head")[0]||a.documentElement;return c.innerHTML="x<style>"+b+"</style>",d.insertBefore(c.lastChild,d.firstChild)}function d(){var a=t.elements;return"string"==typeof a?a.split(" "):a}function e(a,b){var c=t.elements;"string"!=typeof c&&(c=c.join(" ")),"string"!=typeof a&&(a=a.join(" ")),t.elements=c+" "+a,j(b)}function f(a){var b=s[a[q]];return b||(b={},r++,a[q]=r,s[r]=b),b}function g(a,c,d){if(c||(c=b),l)return c.createElement(a);d||(d=f(c));var e;return e=d.cache[a]?d.cache[a].cloneNode():p.test(a)?(d.cache[a]=d.createElem(a)).cloneNode():d.createElem(a),!e.canHaveChildren||o.test(a)||e.tagUrn?e:d.frag.appendChild(e)}function h(a,c){if(a||(a=b),l)return a.createDocumentFragment();c=c||f(a);for(var e=c.frag.cloneNode(),g=0,h=d(),i=h.length;i>g;g++)e.createElement(h[g]);return e}function i(a,b){b.cache||(b.cache={},b.createElem=a.createElement,b.createFrag=a.createDocumentFragment,b.frag=b.createFrag()),a.createElement=function(c){return t.shivMethods?g(c,a,b):b.createElem(c)},a.createDocumentFragment=Function("h,f","return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&("+d().join().replace(/[\w\-:]+/g,function(a){return b.createElem(a),b.frag.createElement(a),'c("'+a+'")'})+");return n}")(t,b.frag)}function j(a){a||(a=b);var d=f(a);return!t.shivCSS||k||d.hasCSS||(d.hasCSS=!!c(a,"article,aside,dialog,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}mark{background:#FF0;color:#000}template{display:none}")),l||i(a,d),a}var k,l,m="3.7.2",n=a.html5||{},o=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,p=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,q="_html5shiv",r=0,s={};!function(){try{var a=b.createElement("a");a.innerHTML="<xyz></xyz>",k="hidden"in a,l=1==a.childNodes.length||function(){b.createElement("a");var a=b.createDocumentFragment();return"undefined"==typeof a.cloneNode||"undefined"==typeof a.createDocumentFragment||"undefined"==typeof a.createElement}()}catch(c){k=!0,l=!0}}();var t={elements:n.elements||"abbr article aside audio bdi canvas data datalist details dialog figcaption figure footer header hgroup main mark meter nav output picture progress section summary template time video",version:m,shivCSS:n.shivCSS!==!1,supportsUnknownElements:l,shivMethods:n.shivMethods!==!1,type:"default",shivDocument:j,createElement:g,createDocumentFragment:h,addElements:e};a.html5=t,j(b)}(this,document);
};
</script>
<script>/*! Respond.js v1.4.2: min/max-width media query polyfill * Copyright 2013 Scott Jehl
 * Licensed under https://github.com/scottjehl/Respond/blob/master/LICENSE-MIT
 *  */

// Only run this code in IE 8
if (!!window.navigator.userAgent.match("MSIE 8")) {
!function(a){"use strict";a.matchMedia=a.matchMedia||function(a){var b,c=a.documentElement,d=c.firstElementChild||c.firstChild,e=a.createElement("body"),f=a.createElement("div");return f.id="mq-test-1",f.style.cssText="position:absolute;top:-100em",e.style.background="none",e.appendChild(f),function(a){return f.innerHTML='&shy;<style media="'+a+'"> #mq-test-1 { width: 42px; }</style>',c.insertBefore(e,d),b=42===f.offsetWidth,c.removeChild(e),{matches:b,media:a}}}(a.document)}(this),function(a){"use strict";function b(){u(!0)}var c={};a.respond=c,c.update=function(){};var d=[],e=function(){var b=!1;try{b=new a.XMLHttpRequest}catch(c){b=new a.ActiveXObject("Microsoft.XMLHTTP")}return function(){return b}}(),f=function(a,b){var c=e();c&&(c.open("GET",a,!0),c.onreadystatechange=function(){4!==c.readyState||200!==c.status&&304!==c.status||b(c.responseText)},4!==c.readyState&&c.send(null))};if(c.ajax=f,c.queue=d,c.regex={media:/@media[^\{]+\{([^\{\}]*\{[^\}\{]*\})+/gi,keyframes:/@(?:\-(?:o|moz|webkit)\-)?keyframes[^\{]+\{(?:[^\{\}]*\{[^\}\{]*\})+[^\}]*\}/gi,urls:/(url\()['"]?([^\/\)'"][^:\)'"]+)['"]?(\))/g,findStyles:/@media *([^\{]+)\{([\S\s]+?)$/,only:/(only\s+)?([a-zA-Z]+)\s?/,minw:/\([\s]*min\-width\s*:[\s]*([\s]*[0-9\.]+)(px|em)[\s]*\)/,maxw:/\([\s]*max\-width\s*:[\s]*([\s]*[0-9\.]+)(px|em)[\s]*\)/},c.mediaQueriesSupported=a.matchMedia&&null!==a.matchMedia("only all")&&a.matchMedia("only all").matches,!c.mediaQueriesSupported){var g,h,i,j=a.document,k=j.documentElement,l=[],m=[],n=[],o={},p=30,q=j.getElementsByTagName("head")[0]||k,r=j.getElementsByTagName("base")[0],s=q.getElementsByTagName("link"),t=function(){var a,b=j.createElement("div"),c=j.body,d=k.style.fontSize,e=c&&c.style.fontSize,f=!1;return b.style.cssText="position:absolute;font-size:1em;width:1em",c||(c=f=j.createElement("body"),c.style.background="none"),k.style.fontSize="100%",c.style.fontSize="100%",c.appendChild(b),f&&k.insertBefore(c,k.firstChild),a=b.offsetWidth,f?k.removeChild(c):c.removeChild(b),k.style.fontSize=d,e&&(c.style.fontSize=e),a=i=parseFloat(a)},u=function(b){var c="clientWidth",d=k[c],e="CSS1Compat"===j.compatMode&&d||j.body[c]||d,f={},o=s[s.length-1],r=(new Date).getTime();if(b&&g&&p>r-g)return a.clearTimeout(h),h=a.setTimeout(u,p),void 0;g=r;for(var v in l)if(l.hasOwnProperty(v)){var w=l[v],x=w.minw,y=w.maxw,z=null===x,A=null===y,B="em";x&&(x=parseFloat(x)*(x.indexOf(B)>-1?i||t():1)),y&&(y=parseFloat(y)*(y.indexOf(B)>-1?i||t():1)),w.hasquery&&(z&&A||!(z||e>=x)||!(A||y>=e))||(f[w.media]||(f[w.media]=[]),f[w.media].push(m[w.rules]))}for(var C in n)n.hasOwnProperty(C)&&n[C]&&n[C].parentNode===q&&q.removeChild(n[C]);n.length=0;for(var D in f)if(f.hasOwnProperty(D)){var E=j.createElement("style"),F=f[D].join("\n");E.type="text/css",E.media=D,q.insertBefore(E,o.nextSibling),E.styleSheet?E.styleSheet.cssText=F:E.appendChild(j.createTextNode(F)),n.push(E)}},v=function(a,b,d){var e=a.replace(c.regex.keyframes,"").match(c.regex.media),f=e&&e.length||0;b=b.substring(0,b.lastIndexOf("/"));var g=function(a){return a.replace(c.regex.urls,"$1"+b+"$2$3")},h=!f&&d;b.length&&(b+="/"),h&&(f=1);for(var i=0;f>i;i++){var j,k,n,o;h?(j=d,m.push(g(a))):(j=e[i].match(c.regex.findStyles)&&RegExp.$1,m.push(RegExp.$2&&g(RegExp.$2))),n=j.split(","),o=n.length;for(var p=0;o>p;p++)k=n[p],l.push({media:k.split("(")[0].match(c.regex.only)&&RegExp.$2||"all",rules:m.length-1,hasquery:k.indexOf("(")>-1,minw:k.match(c.regex.minw)&&parseFloat(RegExp.$1)+(RegExp.$2||""),maxw:k.match(c.regex.maxw)&&parseFloat(RegExp.$1)+(RegExp.$2||"")})}u()},w=function(){if(d.length){var b=d.shift();f(b.href,function(c){v(c,b.href,b.media),o[b.href]=!0,a.setTimeout(function(){w()},0)})}},x=function(){for(var b=0;b<s.length;b++){var c=s[b],e=c.href,f=c.media,g=c.rel&&"stylesheet"===c.rel.toLowerCase();e&&g&&!o[e]&&(c.styleSheet&&c.styleSheet.rawCssText?(v(c.styleSheet.rawCssText,e,f),o[e]=!0):(!/^([a-zA-Z:]*\/\/)/.test(e)&&!r||e.replace(RegExp.$1,"").split("/")[0]===a.location.host)&&("//"===e.substring(0,2)&&(e=a.location.protocol+e),d.push({href:e,media:f})))}w()};x(),c.update=x,c.getEmValue=t,a.addEventListener?a.addEventListener("resize",b,!1):a.attachEvent&&a.attachEvent("onresize",b)}}(this);
};
</script>
<style type="text/css">.pagedtable {
overflow: auto;
padding-left: 8px;
padding-right: 8px;
}
.pagedtable-wrapper {
border: 1px solid #ccc;
border-radius: 4px;
margin-bottom: 10px;
}
.pagedtable table {
width: 100%;
max-width: 100%;
margin: 0;
}
.pagedtable th {
padding: 0 5px 0 5px;
border: none;
border-bottom: 2px solid #dddddd;
min-width: 45px;
}
.pagedtable-empty th {
display: none;
}
.pagedtable td {
padding: 0 4px 0 4px;
}
.pagedtable .even {
background-color: rgba(140, 140, 140, 0.1);
}
.pagedtable-padding-col {
display: none;
}
.pagedtable a {
-webkit-touch-callout: none;
-webkit-user-select: none;
-khtml-user-select: none;
-moz-user-select: none;
-ms-user-select: none;
user-select: none;
}
.pagedtable-index-nav {
cursor: pointer;
padding: 0 5px 0 5px;
float: right;
border: 0;
}
.pagedtable-index-nav-disabled {
cursor: default;
text-decoration: none;
color: #999;
}
a.pagedtable-index-nav-disabled:hover {
text-decoration: none;
color: #999;
}
.pagedtable-indexes {
cursor: pointer;
float: right;
border: 0;
}
.pagedtable-index-current {
cursor: default;
text-decoration: none;
font-weight: bold;
color: #333;
border: 0;
}
a.pagedtable-index-current:hover {
text-decoration: none;
font-weight: bold;
color: #333;
}
.pagedtable-index {
width: 30px;
display: inline-block;
text-align: center;
border: 0;
}
.pagedtable-index-separator-left {
display: inline-block;
color: #333;
font-size: 9px;
padding: 0 0 0 0;
cursor: default;
}
.pagedtable-index-separator-right {
display: inline-block;
color: #333;
font-size: 9px;
padding: 0 4px 0 0;
cursor: default;
}
.pagedtable-footer {
padding-top: 4px;
padding-bottom: 5px;
}
.pagedtable-not-empty .pagedtable-footer {
border-top: 2px solid #dddddd;
}
.pagedtable-info {
overflow: hidden;
color: #999;
white-space: nowrap;
text-overflow: ellipsis;
}
.pagedtable-header-name {
overflow: hidden;
text-overflow: ellipsis;
}
.pagedtable-header-type {
color: #999;
font-weight: 400;
}
.pagedtable-na-cell {
font-style: italic;
opacity: 0.3;
}
</style>
<script>// Production steps of ECMA-262, Edition 5, 15.4.4.18
// Reference: http://es5.github.io/#x15.4.4.18
if (!Array.prototype.forEach) {

  Array.prototype.forEach = function(callback, thisArg) {

    var T, k;

    if (this === null) {
      throw new TypeError(' this is null or not defined');
    }

    // 1. Let O be the result of calling toObject() passing the
    // |this| value as the argument.
    var O = Object(this);

    // 2. Let lenValue be the result of calling the Get() internal
    // method of O with the argument "length".
    // 3. Let len be toUint32(lenValue).
    var len = O.length >>> 0;

    // 4. If isCallable(callback) is false, throw a TypeError exception.
    // See: http://es5.github.com/#x9.11
    if (typeof callback !== "function") {
      throw new TypeError(callback + ' is not a function');
    }

    // 5. If thisArg was supplied, let T be thisArg; else let
    // T be undefined.
    if (arguments.length > 1) {
      T = thisArg;
    }

    // 6. Let k be 0
    k = 0;

    // 7. Repeat, while k < len
    while (k < len) {

      var kValue;

      // a. Let Pk be ToString(k).
      //    This is implicit for LHS operands of the in operator
      // b. Let kPresent be the result of calling the HasProperty
      //    internal method of O with argument Pk.
      //    This step can be combined with c
      // c. If kPresent is true, then
      if (k in O) {

        // i. Let kValue be the result of calling the Get internal
        // method of O with argument Pk.
        kValue = O[k];

        // ii. Call the Call internal method of callback with T as
        // the this value and argument list containing kValue, k, and O.
        callback.call(T, kValue, k, O);
      }
      // d. Increase k by 1.
      k++;
    }
    // 8. return undefined
  };
}

// Production steps of ECMA-262, Edition 5, 15.4.4.19
// Reference: http://es5.github.io/#x15.4.4.19
if (!Array.prototype.map) {

  Array.prototype.map = function(callback, thisArg) {

    var T, A, k;

    if (this == null) {
      throw new TypeError(' this is null or not defined');
    }

    // 1. Let O be the result of calling ToObject passing the |this|
    //    value as the argument.
    var O = Object(this);

    // 2. Let lenValue be the result of calling the Get internal
    //    method of O with the argument "length".
    // 3. Let len be ToUint32(lenValue).
    var len = O.length >>> 0;

    // 4. If IsCallable(callback) is false, throw a TypeError exception.
    // See: http://es5.github.com/#x9.11
    if (typeof callback !== 'function') {
      throw new TypeError(callback + ' is not a function');
    }

    // 5. If thisArg was supplied, let T be thisArg; else let T be undefined.
    if (arguments.length > 1) {
      T = thisArg;
    }

    // 6. Let A be a new array created as if by the expression new Array(len)
    //    where Array is the standard built-in constructor with that name and
    //    len is the value of len.
    A = new Array(len);

    // 7. Let k be 0
    k = 0;

    // 8. Repeat, while k < len
    while (k < len) {

      var kValue, mappedValue;

      // a. Let Pk be ToString(k).
      //   This is implicit for LHS operands of the in operator
      // b. Let kPresent be the result of calling the HasProperty internal
      //    method of O with argument Pk.
      //   This step can be combined with c
      // c. If kPresent is true, then
      if (k in O) {

        // i. Let kValue be the result of calling the Get internal
        //    method of O with argument Pk.
        kValue = O[k];

        // ii. Let mappedValue be the result of calling the Call internal
        //     method of callback with T as the this value and argument
        //     list containing kValue, k, and O.
        mappedValue = callback.call(T, kValue, k, O);

        // iii. Call the DefineOwnProperty internal method of A with arguments
        // Pk, Property Descriptor
        // { Value: mappedValue,
        //   Writable: true,
        //   Enumerable: true,
        //   Configurable: true },
        // and false.

        // In browsers that support Object.defineProperty, use the following:
        // Object.defineProperty(A, k, {
        //   value: mappedValue,
        //   writable: true,
        //   enumerable: true,
        //   configurable: true
        // });

        // For best browser support, use the following:
        A[k] = mappedValue;
      }
      // d. Increase k by 1.
      k++;
    }

    // 9. return A
    return A;
  };
}

var PagedTable = function (pagedTable) {
  var me = this;

  var source = function(pagedTable) {
    var sourceElems = [].slice.call(pagedTable.children).filter(function(e) {
      return e.hasAttribute("data-pagedtable-source");
    });

    if (sourceElems === null || sourceElems.length !== 1) {
      throw("A single data-pagedtable-source was not found");
    }

    return JSON.parse(sourceElems[0].innerHTML);
  }(pagedTable);

  var options = function(source) {
    var options = typeof(source.options) !== "undefined" &&
      source.options !== null ? source.options : {};

    var columns = typeof(options.columns) !== "undefined" ? options.columns : {};
    var rows = typeof(options.rows) !== "undefined" ? options.rows : {};

    var positiveIntOrNull = function(value) {
      return parseInt(value) >= 0 ? parseInt(value) : null;
    };

    return {
      pages: positiveIntOrNull(options.pages),
      rows: {
        min: positiveIntOrNull(rows.min),
        max: positiveIntOrNull(rows.max),
        total: positiveIntOrNull(rows.total)
      },
      columns: {
        min: positiveIntOrNull(columns.min),
        max: positiveIntOrNull(columns.max),
        total: positiveIntOrNull(columns.total)
      }
    };
  }(source);

  var Measurer = function() {

    // set some default initial values that will get adjusted in runtime
    me.measures = {
      padding: 12,
      character: 8,
      height: 15,
      defaults: true
    };

    me.calculate = function(measuresCell) {
      if (!me.measures.defaults)
        return;

      var measuresCellStyle = window.getComputedStyle(measuresCell, null);

      var newPadding = parsePadding(measuresCellStyle.paddingLeft) +
            parsePadding(measuresCellStyle.paddingRight);

      var sampleString = "ABCDEFGHIJ0123456789";
      var newCharacter = Math.ceil(measuresCell.clientWidth / sampleString.length);

      if (newPadding <= 0 || newCharacter <= 0)
        return;

      me.measures.padding = newPadding;
      me.measures.character = newCharacter;
      me.measures.height = measuresCell.clientHeight;
      me.measures.defaults = false;
    };

    return me;
  };

  var Page = function(data, options) {
    var me = this;

    var defaults = {
      max: 7,
      rows: 10
    };

    var totalPages = function() {
      return Math.ceil(data.length / me.rows);
    };

    me.number = 0;
    me.max = options.pages !== null ? options.pages : defaults.max;
    me.visible = me.max;
    me.rows = options.rows.min !== null ? options.rows.min : defaults.rows;
    me.total = totalPages();

    me.setRows = function(newRows) {
      me.rows = newRows;
      me.total = totalPages();
    };

    me.setPageNumber = function(newPageNumber) {
      if (newPageNumber < 0) newPageNumber = 0;
      if (newPageNumber >= me.total) newPageNumber = me.total - 1;

      me.number = newPageNumber;
    };

    me.setVisiblePages = function(visiblePages) {
      me.visible = Math.min(me.max, visiblePages);
      me.setPageNumber(me.number);
    };

    me.getVisiblePageRange = function() {
      var start = me.number - Math.max(Math.floor((me.visible - 1) / 2), 0);
      var end = me.number + Math.floor(me.visible / 2) + 1;
      var pageCount = me.total;

      if (start < 0) {
        var diffToStart = 0 - start;
        start += diffToStart;
        end += diffToStart;
      }

      if (end > pageCount) {
        var diffToEnd = end - pageCount;
        start -= diffToEnd;
        end -= diffToEnd;
      }

      start = start < 0 ? 0 : start;
      end = end >= pageCount ? pageCount : end;

      var first = false;
      var last = false;

      if (start > 0 && me.visible > 1) {
        start = start + 1;
        first = true;
      }

      if (end < pageCount && me.visible > 2) {
        end = end - 1;
        last = true;
      }

      return {
        first: first,
        start: start,
        end: end,
        last: last
      };
    };

    me.getRowStart = function() {
      var rowStart = page.number * page.rows;
      if (rowStart < 0)
        rowStart = 0;

      return rowStart;
    };

    me.getRowEnd = function() {
      var rowStart = me.getRowStart();
      return Math.min(rowStart + me.rows, data.length);
    };

    me.getPaddingRows = function() {
      var rowStart = me.getRowStart();
      var rowEnd = me.getRowEnd();
      return data.length > me.rows ? me.rows - (rowEnd - rowStart) : 0;
    };
  };

  var Columns = function(data, columns, options) {
    var me = this;

    me.defaults = {
      min: 5
    };

    me.number = 0;
    me.visible = 0;
    me.total = columns.length;
    me.subset = [];
    me.padding = 0;
    me.min = options.columns.min !== null ? options.columns.min : me.defaults.min;
    me.max = options.columns.max !== null ? options.columns.max : null;
    me.widths = {};

    var widthsLookAhead = Math.max(100, options.rows.min);
    var paddingColChars = 10;

    me.emptyNames = function() {
      columns.forEach(function(column) {
        if (columns.label !== null && columns.label !== "")
          return false;
      });

      return true;
    };

    var parsePadding = function(value) {
      return parseInt(value) >= 0 ? parseInt(value) : 0;
    };

    me.calculateWidths = function(measures) {
      columns.forEach(function(column) {
        var maxChars = Math.max(
          column.label.toString().length,
          column.type.toString().length
        );

        for (var idxRow = 0; idxRow < Math.min(widthsLookAhead, data.length); idxRow++) {
          maxChars = Math.max(maxChars, data[idxRow][column.name.toString()].length);
        }

        me.widths[column.name] = {
          // width in characters
          chars: maxChars,
          // width for the inner html columns
          inner: maxChars * measures.character,
          // width adding outer styles like padding
          outer: maxChars * measures.character + measures.padding
        };
      });
    };

    me.getWidth = function() {
      var widthOuter = 0;
      for (var idxCol = 0; idxCol < me.subset.length; idxCol++) {
        var columnName = me.subset[idxCol].name;
        widthOuter = widthOuter + me.widths[columnName].outer;
      }

      widthOuter = widthOuter + me.padding * paddingColChars * measurer.measures.character;

      if (me.hasMoreLeftColumns()) {
        widthOuter = widthOuter + columnNavigationWidthPX + measurer.measures.padding;
      }

      if (me.hasMoreRightColumns()) {
        widthOuter = widthOuter + columnNavigationWidthPX + measurer.measures.padding;
      }

      return widthOuter;
    };

    me.updateSlice = function() {
      if (me.number + me.visible >= me.total)
        me.number = me.total - me.visible;

      if (me.number < 0) me.number = 0;

      me.subset = columns.slice(me.number, Math.min(me.number + me.visible, me.total));

      me.subset = me.subset.map(function(column) {
        Object.keys(column).forEach(function(colKey) {
          column[colKey] = column[colKey] === null ? "" : column[colKey].toString();
        });

        column.width = null;
        return column;
      });
    };

    me.setVisibleColumns = function(columnNumber, newVisibleColumns, paddingCount) {
      me.number = columnNumber;
      me.visible = newVisibleColumns;
      me.padding = paddingCount;

      me.updateSlice();
    };

    me.incColumnNumber = function(increment) {
      me.number = me.number + increment;
    };

    me.setColumnNumber = function(newNumber) {
      me.number = newNumber;
    };

    me.setPaddingCount = function(newPadding) {
      me.padding = newPadding;
    };

    me.getPaddingCount = function() {
      return me.padding;
    };

    me.hasMoreLeftColumns = function() {
      return me.number > 0;
    };

    me.hasMoreRightColumns = function() {
      return me.number + me.visible < me.total;
    };

    me.updateSlice(0);
    return me;
  };

  var data = source.data;
  var page = new Page(data, options);
  var measurer = new Measurer(data, options);
  var columns = new Columns(data, source.columns, options);

  var table = null;
  var tableDiv = null;
  var header = null;
  var footer = null;
  var tbody = null;

  // Caches pagedTable.clientWidth, specially for webkit
  var cachedPagedTableClientWidth = null;

  var onChangeCallbacks = [];

  var clearSelection = function() {
    if(document.selection && document.selection.empty) {
      document.selection.empty();
    } else if(window.getSelection) {
      var sel = window.getSelection();
      sel.removeAllRanges();
    }
  };

  var columnNavigationWidthPX = 5;

  var renderColumnNavigation = function(increment, backwards) {
    var arrow = document.createElement("div");
    arrow.setAttribute("style",
      "border-top: " + columnNavigationWidthPX + "px solid transparent;" +
      "border-bottom: " + columnNavigationWidthPX + "px solid transparent;" +
      "border-" + (backwards ? "right" : "left") + ": " + columnNavigationWidthPX + "px solid;");

    var header = document.createElement("th");
    header.appendChild(arrow);
    header.setAttribute("style",
      "cursor: pointer;" +
      "vertical-align: middle;" +
      "min-width: " + columnNavigationWidthPX + "px;" +
      "width: " + columnNavigationWidthPX + "px;");

    header.onclick = function() {
      columns.incColumnNumber(backwards ? -1 : increment);

      me.animateColumns(backwards);
      renderFooter();

      clearSelection();
      triggerOnChange();
    };

    return header;
  };

  var maxColumnWidth = function(width) {
    var padding = 80;
    var columnMax = Math.max(cachedPagedTableClientWidth - padding, 0);

    return parseInt(width) > 0 ?
      Math.min(columnMax, parseInt(width)) + "px" :
      columnMax + "px";
  };

  var clearHeader = function() {
    var thead = pagedTable.querySelectorAll("thead")[0];
    thead.innerHTML = "";
  };

  var renderHeader = function(clear) {
    cachedPagedTableClientWidth = pagedTable.clientWidth;

    var fragment = document.createDocumentFragment();

    header = document.createElement("tr");
    fragment.appendChild(header);

    if (columns.number > 0)
      header.appendChild(renderColumnNavigation(-columns.visible, true));

    columns.subset = columns.subset.map(function(columnData) {
      var column = document.createElement("th");
      column.setAttribute("align", columnData.align);
      column.style.textAlign = columnData.align;

      column.style.maxWidth = maxColumnWidth(null);
      if (columnData.width) {
        column.style.minWidth =
          column.style.maxWidth = maxColumnWidth(columnData.width);
      }

      var columnName = document.createElement("div");
      columnName.setAttribute("class", "pagedtable-header-name");
      if (columnData.label === "") {
        columnName.innerHTML = "&nbsp;";
      }
      else {
        columnName.appendChild(document.createTextNode(columnData.label));
      }
      column.appendChild(columnName);

      var columnType = document.createElement("div");
      columnType.setAttribute("class", "pagedtable-header-type");
      if (columnData.type === "") {
        columnType.innerHTML = "&nbsp;";
      }
      else {
        columnType.appendChild(document.createTextNode("<" + columnData.type + ">"));
      }
      column.appendChild(columnType);

      header.appendChild(column);

      columnData.element = column;

      return columnData;
    });

    for (var idx = 0; idx < columns.getPaddingCount(); idx++) {
      var paddingCol = document.createElement("th");
      paddingCol.setAttribute("class", "pagedtable-padding-col");
      header.appendChild(paddingCol);
    }

    if (columns.number + columns.visible < columns.total)
      header.appendChild(renderColumnNavigation(columns.visible, false));

    if (typeof(clear) == "undefined" || clear) clearHeader();
    var thead = pagedTable.querySelectorAll("thead")[0];
    thead.appendChild(fragment);
  };

  me.animateColumns = function(backwards) {
    var thead = pagedTable.querySelectorAll("thead")[0];

    var headerOld = thead.querySelectorAll("tr")[0];
    var tbodyOld = table.querySelectorAll("tbody")[0];

    me.fitColumns(backwards);

    renderHeader(false);

    header.style.opacity = "0";
    header.style.transform = backwards ? "translateX(-30px)" : "translateX(30px)";
    header.style.transition = "transform 200ms linear, opacity 200ms";
    header.style.transitionDelay = "0";

    renderBody(false);

    if (headerOld) {
      headerOld.style.position = "absolute";
      headerOld.style.transform = "translateX(0px)";
      headerOld.style.opacity = "1";
      headerOld.style.transition = "transform 100ms linear, opacity 100ms";
      headerOld.setAttribute("class", "pagedtable-remove-head");
      if (headerOld.style.transitionEnd) {
        headerOld.addEventListener("transitionend", function() {
          var headerOldByClass = thead.querySelector(".pagedtable-remove-head");
          if (headerOldByClass) thead.removeChild(headerOldByClass);
        });
      }
      else {
        thead.removeChild(headerOld);
      }
    }

    if (tbodyOld) table.removeChild(tbodyOld);

    tbody.style.opacity = "0";
    tbody.style.transition = "transform 200ms linear, opacity 200ms";
    tbody.style.transitionDelay = "0ms";

    // force relayout
    window.getComputedStyle(header).opacity;
    window.getComputedStyle(tbody).opacity;

    if (headerOld) {
      headerOld.style.transform = backwards ? "translateX(20px)" : "translateX(-30px)";
      headerOld.style.opacity = "0";
    }

    header.style.transform = "translateX(0px)";
    header.style.opacity = "1";

    tbody.style.opacity = "1";
  }

  me.onChange = function(callback) {
    onChangeCallbacks.push(callback);
  };

  var triggerOnChange = function() {
    onChangeCallbacks.forEach(function(onChange) {
      onChange();
    });
  };

  var clearBody = function() {
    if (tbody) {
      table.removeChild(tbody);
      tbody = null;
    }
  };

  var renderBody = function(clear) {
    cachedPagedTableClientWidth = pagedTable.clientWidth

    var fragment = document.createDocumentFragment();

    var pageData = data.slice(page.getRowStart(), page.getRowEnd());

    pageData.forEach(function(dataRow, idxRow) {
      var htmlRow = document.createElement("tr");
      htmlRow.setAttribute("class", (idxRow % 2 !==0) ? "even" : "odd");

      if (columns.hasMoreLeftColumns())
        htmlRow.appendChild(document.createElement("td"));

      columns.subset.forEach(function(columnData) {
        var cellName = columnData.name;
        var dataCell = dataRow[cellName];
        var htmlCell = document.createElement("td");

        if (dataCell === "NA") htmlCell.setAttribute("class", "pagedtable-na-cell");
        if (dataCell === "__NA__") dataCell = "NA";

        var cellText = document.createTextNode(dataCell);
        htmlCell.appendChild(cellText);
        if (dataCell.length > 50) {
          htmlCell.setAttribute("title", dataCell);
        }
        htmlCell.setAttribute("align", columnData.align);
        htmlCell.style.textAlign = columnData.align;
        htmlCell.style.maxWidth = maxColumnWidth(null);
        if (columnData.width) {
          htmlCell.style.minWidth = htmlCell.style.maxWidth = maxColumnWidth(columnData.width);
        }
        htmlRow.appendChild(htmlCell);
      });

      for (var idx = 0; idx < columns.getPaddingCount(); idx++) {
        var paddingCol = document.createElement("td");
        paddingCol.setAttribute("class", "pagedtable-padding-col");
        htmlRow.appendChild(paddingCol);
      }

      if (columns.hasMoreRightColumns())
        htmlRow.appendChild(document.createElement("td"));

      fragment.appendChild(htmlRow);
    });

    for (var idxPadding = 0; idxPadding < page.getPaddingRows(); idxPadding++) {
      var paddingRow = document.createElement("tr");

      var paddingCellRow = document.createElement("td");
      paddingCellRow.innerHTML = "&nbsp;";
      paddingCellRow.setAttribute("colspan", "100%");
      paddingRow.appendChild(paddingCellRow);

      fragment.appendChild(paddingRow);
    }

    if (typeof(clear) == "undefined" || clear) clearBody();
    tbody = document.createElement("tbody");
    tbody.appendChild(fragment);

    table.appendChild(tbody);
  };

  var getLabelInfo = function() {
    var pageStart = page.getRowStart();
    var pageEnd = page.getRowEnd();
    var totalRows = data.length;

    var totalRowsLabel = options.rows.total ? options.rows.total : totalRows;
    var totalRowsLabelFormat = totalRowsLabel.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');

    var infoText = (pageStart + 1) + "-" + pageEnd + " of " + totalRowsLabelFormat + " rows";
    if (totalRows < page.rows) {
      infoText = totalRowsLabel + " row" + (totalRows != 1 ? "s" : "");
    }
    if (columns.total > columns.visible) {
      var totalColumnsLabel = options.columns.total ? options.columns.total : columns.total;

      infoText = infoText + " | " + (columns.number + 1) + "-" +
        (Math.min(columns.number + columns.visible, columns.total)) +
        " of " + totalColumnsLabel + " columns";
    }

    return infoText;
  };

  var clearFooter = function() {
    footer = pagedTable.querySelectorAll("div.pagedtable-footer")[0];
    footer.innerHTML = "";

    return footer;
  };

  var createPageLink = function(idxPage) {
    var pageLink = document.createElement("a");
    pageLinkClass = idxPage === page.number ? "pagedtable-index pagedtable-index-current" : "pagedtable-index";
    pageLink.setAttribute("class", pageLinkClass);
    pageLink.setAttribute("data-page-index", idxPage);
    pageLink.onclick = function() {
      page.setPageNumber(parseInt(this.getAttribute("data-page-index")));
      renderBody();
      renderFooter();

      triggerOnChange();
    };

    pageLink.appendChild(document.createTextNode(idxPage + 1));

    return pageLink;
  }

  var renderFooter = function() {
    footer = clearFooter();

    var next = document.createElement("a");
    next.appendChild(document.createTextNode("Next"));
    next.onclick = function() {
      page.setPageNumber(page.number + 1);
      renderBody();
      renderFooter();

      triggerOnChange();
    };
    if (data.length > page.rows) footer.appendChild(next);

    var pageNumbers = document.createElement("div");
    pageNumbers.setAttribute("class", "pagedtable-indexes");

    var pageRange = page.getVisiblePageRange();

    if (pageRange.first) {
      var pageLink = createPageLink(0);
      pageNumbers.appendChild(pageLink);

      var pageSeparator = document.createElement("div");
      pageSeparator.setAttribute("class", "pagedtable-index-separator-left");
      pageSeparator.appendChild(document.createTextNode("..."))
      pageNumbers.appendChild(pageSeparator);
    }

    for (var idxPage = pageRange.start; idxPage < pageRange.end; idxPage++) {
      var pageLink = createPageLink(idxPage);

      pageNumbers.appendChild(pageLink);
    }

    if (pageRange.last) {
      var pageSeparator = document.createElement("div");
      pageSeparator.setAttribute("class", "pagedtable-index-separator-right");
      pageSeparator.appendChild(document.createTextNode("..."))
      pageNumbers.appendChild(pageSeparator);

      var pageLink = createPageLink(page.total - 1);
      pageNumbers.appendChild(pageLink);
    }

    if (data.length > page.rows) footer.appendChild(pageNumbers);

    var previous = document.createElement("a");
    previous.appendChild(document.createTextNode("Previous"));
    previous.onclick = function() {
      page.setPageNumber(page.number - 1);
      renderBody();
      renderFooter();

      triggerOnChange();
    };
    if (data.length > page.rows) footer.appendChild(previous);

    var infoLabel = document.createElement("div");
    infoLabel.setAttribute("class", "pagedtable-info");
    infoLabel.setAttribute("title", getLabelInfo());
    infoLabel.appendChild(document.createTextNode(getLabelInfo()));
    footer.appendChild(infoLabel);

    var enabledClass = "pagedtable-index-nav";
    var disabledClass = "pagedtable-index-nav pagedtable-index-nav-disabled";
    previous.setAttribute("class", page.number <= 0 ? disabledClass : enabledClass);
    next.setAttribute("class", (page.number + 1) * page.rows >= data.length ? disabledClass : enabledClass);
  };

  var measuresCell = null;

  var renderMeasures = function() {
    var measuresTable = document.createElement("table");
    measuresTable.style.visibility = "hidden";
    measuresTable.style.position = "absolute";
    measuresTable.style.whiteSpace = "nowrap";
    measuresTable.style.height = "auto";
    measuresTable.style.width = "auto";

    var measuresRow = document.createElement("tr");
    measuresTable.appendChild(measuresRow);

    measuresCell = document.createElement("td");
    var sampleString = "ABCDEFGHIJ0123456789";
    measuresCell.appendChild(document.createTextNode(sampleString));

    measuresRow.appendChild(measuresCell);

    tableDiv.appendChild(measuresTable);
  }

  me.init = function() {
    tableDiv = document.createElement("div");
    pagedTable.appendChild(tableDiv);
    var pagedTableClass = data.length > 0 ?
      "pagedtable pagedtable-not-empty" :
      "pagedtable pagedtable-empty";

    if (columns.total == 0 || (columns.emptyNames() && data.length == 0)) {
      pagedTableClass = pagedTableClass + " pagedtable-empty-columns";
    }

    tableDiv.setAttribute("class", pagedTableClass);

    renderMeasures();
    measurer.calculate(measuresCell);
    columns.calculateWidths(measurer.measures);

    table = document.createElement("table");
    table.setAttribute("cellspacing", "0");
    table.setAttribute("class", "table table-condensed");
    tableDiv.appendChild(table);

    table.appendChild(document.createElement("thead"));

    var footerDiv = document.createElement("div");
    footerDiv.setAttribute("class", "pagedtable-footer");
    tableDiv.appendChild(footerDiv);

    // if the host has not yet provided horizontal space, render hidden
    if (tableDiv.clientWidth <= 0) {
      tableDiv.style.opacity = "0";
    }

    me.render();

    // retry seizing columns later if the host has not provided space
    function retryFit() {
      if (tableDiv.clientWidth <= 0) {
        setTimeout(retryFit, 100);
      } else {
        me.render();
        triggerOnChange();
      }
    }
    if (tableDiv.clientWidth <= 0) {
      retryFit();
    }
  };

  var registerWidths = function() {
    columns.subset = columns.subset.map(function(column) {
      column.width = columns.widths[column.name].inner;
      return column;
    });
  };

  var parsePadding = function(value) {
    return parseInt(value) >= 0 ? parseInt(value) : 0;
  };

  me.fixedHeight = function() {
    return options.rows.max != null;
  }

  me.fitRows = function() {
    if (me.fixedHeight())
      return;

    measurer.calculate(measuresCell);

    var rows = options.rows.min !== null ? options.rows.min : 0;
    var headerHeight = header !== null && header.offsetHeight > 0 ? header.offsetHeight : 0;
    var footerHeight = footer !== null && footer.offsetHeight > 0 ? footer.offsetHeight : 0;

    if (pagedTable.offsetHeight > 0) {
      var availableHeight = pagedTable.offsetHeight - headerHeight - footerHeight;
      rows = Math.floor((availableHeight) / measurer.measures.height);
    }

    rows = options.rows.min !== null ? Math.max(options.rows.min, rows) : rows;

    page.setRows(rows);
  }

  // The goal of this function is to add as many columns as possible
  // starting from left-to-right, when the right most limit is reached
  // it tries to add columns from the left as well.
  //
  // When startBackwards is true columns are added from right-to-left
  me.fitColumns = function(startBackwards) {
    measurer.calculate(measuresCell);
    columns.calculateWidths(measurer.measures);

    if (tableDiv.clientWidth > 0) {
      tableDiv.style.opacity = 1;
    }

    var visibleColumns = tableDiv.clientWidth <= 0 ? Math.max(columns.min, 1) : 1;
    var columnNumber = columns.number;
    var paddingCount = 0;

    // track a list of added columns as we build the visible ones to allow us
    // to remove columns when they don't fit anymore.
    var columnHistory = [];

    var lastTableHeight = 0;
    var backwards = startBackwards;

    var tableDivStyle = window.getComputedStyle(tableDiv, null);
    var tableDivPadding = parsePadding(tableDivStyle.paddingLeft) +
      parsePadding(tableDivStyle.paddingRight);

    var addPaddingCol = false;
    var currentWidth = 0;

    while (true) {
      columns.setVisibleColumns(columnNumber, visibleColumns, paddingCount);
      currentWidth = columns.getWidth();

      if (tableDiv.clientWidth - tableDivPadding < currentWidth) {
        break;
      }

      columnHistory.push({
        columnNumber: columnNumber,
        visibleColumns: visibleColumns,
        paddingCount: paddingCount
      });

      if (columnHistory.length > 100) {
        console.error("More than 100 tries to fit columns, aborting");
        break;
      }

      if (columns.max !== null &&
        columns.visible + columns.getPaddingCount() >= columns.max) {
        break;
      }

      // if we run out of right-columns
      if (!backwards && columnNumber + columns.visible >= columns.total) {
        // if we started adding right-columns, try adding left-columns
        if (!startBackwards && columnNumber > 0) {
          backwards = true;
        }
        else if (columns.min === null || visibleColumns + columns.getPaddingCount() >= columns.min) {
          break;
        }
        else {
          paddingCount = paddingCount + 1;
        }
      }

      // if we run out of left-columns
      if (backwards && columnNumber == 0) {
        // if we started adding left-columns, try adding right-columns
        if (startBackwards && columnNumber + columns.visible < columns.total) {
          backwards = false;
        }
        else if (columns.min === null || visibleColumns + columns.getPaddingCount() >= columns.min) {
          break;
        }
        else {
          paddingCount = paddingCount + 1;
        }
      }

      // when moving backwards try fitting left columns first
      if (backwards && columnNumber > 0) {
        columnNumber = columnNumber - 1;
      }

      if (columnNumber + visibleColumns < columns.total) {
        visibleColumns = visibleColumns + 1;
      }
    }

    var lastRenderableColumn = {
        columnNumber: columnNumber,
        visibleColumns: visibleColumns,
        paddingCount: paddingCount
    };

    if (columnHistory.length > 0) {
      lastRenderableColumn = columnHistory[columnHistory.length - 1];
    }

    columns.setVisibleColumns(
      lastRenderableColumn.columnNumber,
      lastRenderableColumn.visibleColumns,
      lastRenderableColumn.paddingCount);

    if (pagedTable.offsetWidth > 0) {
      page.setVisiblePages(Math.max(Math.ceil(1.0 * (pagedTable.offsetWidth - 250) / 40), 2));
    }

    registerWidths();
  };

  me.fit = function(startBackwards) {
    me.fitRows();
    me.fitColumns(startBackwards);
  }

  me.render = function() {
    me.fitColumns(false);

    // render header/footer to measure height accurately
    renderHeader();
    renderFooter();

    me.fitRows();
    renderBody();

    // re-render footer to match new rows
    renderFooter();
  }

  var resizeLastWidth = -1;
  var resizeLastHeight = -1;
  var resizeNewWidth = -1;
  var resizeNewHeight = -1;
  var resizePending = false;

  me.resize = function(newWidth, newHeight) {

    function resizeDelayed() {
      resizePending = false;

      if (
        (resizeNewWidth !== resizeLastWidth) ||
        (!me.fixedHeight() && resizeNewHeight !== resizeLastHeight)
      ) {
        resizeLastWidth = resizeNewWidth;
        resizeLastHeight = resizeNewHeight;

        setTimeout(resizeDelayed, 200);
        resizePending = true;
      } else {
        me.render();
        triggerOnChange();

        resizeLastWidth = -1;
        resizeLastHeight = -1;
      }
    }

    resizeNewWidth = newWidth;
    resizeNewHeight = newHeight;

    if (!resizePending) resizeDelayed();
  };
};

var PagedTableDoc;
(function (PagedTableDoc) {
  var allPagedTables = [];

  PagedTableDoc.initAll = function() {
    allPagedTables = [];

    var pagedTables = [].slice.call(document.querySelectorAll('[data-pagedtable="false"],[data-pagedtable=""]'));
    pagedTables.forEach(function(pagedTable, idx) {
      pagedTable.setAttribute("data-pagedtable", "true");
      pagedTable.setAttribute("pagedtable-page", 0);
      pagedTable.setAttribute("class", "pagedtable-wrapper");

      var pagedTableInstance = new PagedTable(pagedTable);
      pagedTableInstance.init();

      allPagedTables.push(pagedTableInstance);
    });
  };

  PagedTableDoc.resizeAll = function() {
    allPagedTables.forEach(function(pagedTable) {
      pagedTable.render();
    });
  };

  window.addEventListener("resize", PagedTableDoc.resizeAll);

  return PagedTableDoc;
})(PagedTableDoc || (PagedTableDoc = {}));

window.onload = function() {
  PagedTableDoc.initAll();
};
</script>
<script>/*! jQuery UI - v1.11.4 - 2016-01-05
* http://jqueryui.com
* Includes: core.js, widget.js, mouse.js, position.js, draggable.js, droppable.js, resizable.js, selectable.js, sortable.js, accordion.js, autocomplete.js, button.js, dialog.js, menu.js, progressbar.js, selectmenu.js, slider.js, spinner.js, tabs.js, tooltip.js, effect.js, effect-blind.js, effect-bounce.js, effect-clip.js, effect-drop.js, effect-explode.js, effect-fade.js, effect-fold.js, effect-highlight.js, effect-puff.js, effect-pulsate.js, effect-scale.js, effect-shake.js, effect-size.js, effect-slide.js, effect-transfer.js
* Copyright jQuery Foundation and other contributors; Licensed MIT */

(function(e){"function"==typeof define&&define.amd?define(["jquery"],e):e(jQuery)})(function(e){function t(t,s){var n,a,o,r=t.nodeName.toLowerCase();return"area"===r?(n=t.parentNode,a=n.name,t.href&&a&&"map"===n.nodeName.toLowerCase()?(o=e("img[usemap='#"+a+"']")[0],!!o&&i(o)):!1):(/^(input|select|textarea|button|object)$/.test(r)?!t.disabled:"a"===r?t.href||s:s)&&i(t)}function i(t){return e.expr.filters.visible(t)&&!e(t).parents().addBack().filter(function(){return"hidden"===e.css(this,"visibility")}).length}function s(e){return function(){var t=this.element.val();e.apply(this,arguments),this._refresh(),t!==this.element.val()&&this._trigger("change")}}e.ui=e.ui||{},e.extend(e.ui,{version:"1.11.4",keyCode:{BACKSPACE:8,COMMA:188,DELETE:46,DOWN:40,END:35,ENTER:13,ESCAPE:27,HOME:36,LEFT:37,PAGE_DOWN:34,PAGE_UP:33,PERIOD:190,RIGHT:39,SPACE:32,TAB:9,UP:38}}),e.fn.extend({scrollParent:function(t){var i=this.css("position"),s="absolute"===i,n=t?/(auto|scroll|hidden)/:/(auto|scroll)/,a=this.parents().filter(function(){var t=e(this);return s&&"static"===t.css("position")?!1:n.test(t.css("overflow")+t.css("overflow-y")+t.css("overflow-x"))}).eq(0);return"fixed"!==i&&a.length?a:e(this[0].ownerDocument||document)},uniqueId:function(){var e=0;return function(){return this.each(function(){this.id||(this.id="ui-id-"+ ++e)})}}(),removeUniqueId:function(){return this.each(function(){/^ui-id-\d+$/.test(this.id)&&e(this).removeAttr("id")})}}),e.extend(e.expr[":"],{data:e.expr.createPseudo?e.expr.createPseudo(function(t){return function(i){return!!e.data(i,t)}}):function(t,i,s){return!!e.data(t,s[3])},focusable:function(i){return t(i,!isNaN(e.attr(i,"tabindex")))},tabbable:function(i){var s=e.attr(i,"tabindex"),n=isNaN(s);return(n||s>=0)&&t(i,!n)}}),e("<a>").outerWidth(1).jquery||e.each(["Width","Height"],function(t,i){function s(t,i,s,a){return e.each(n,function(){i-=parseFloat(e.css(t,"padding"+this))||0,s&&(i-=parseFloat(e.css(t,"border"+this+"Width"))||0),a&&(i-=parseFloat(e.css(t,"margin"+this))||0)}),i}var n="Width"===i?["Left","Right"]:["Top","Bottom"],a=i.toLowerCase(),o={innerWidth:e.fn.innerWidth,innerHeight:e.fn.innerHeight,outerWidth:e.fn.outerWidth,outerHeight:e.fn.outerHeight};e.fn["inner"+i]=function(t){return void 0===t?o["inner"+i].call(this):this.each(function(){e(this).css(a,s(this,t)+"px")})},e.fn["outer"+i]=function(t,n){return"number"!=typeof t?o["outer"+i].call(this,t):this.each(function(){e(this).css(a,s(this,t,!0,n)+"px")})}}),e.fn.addBack||(e.fn.addBack=function(e){return this.add(null==e?this.prevObject:this.prevObject.filter(e))}),e("<a>").data("a-b","a").removeData("a-b").data("a-b")&&(e.fn.removeData=function(t){return function(i){return arguments.length?t.call(this,e.camelCase(i)):t.call(this)}}(e.fn.removeData)),e.ui.ie=!!/msie [\w.]+/.exec(navigator.userAgent.toLowerCase()),e.fn.extend({focus:function(t){return function(i,s){return"number"==typeof i?this.each(function(){var t=this;setTimeout(function(){e(t).focus(),s&&s.call(t)},i)}):t.apply(this,arguments)}}(e.fn.focus),disableSelection:function(){var e="onselectstart"in document.createElement("div")?"selectstart":"mousedown";return function(){return this.bind(e+".ui-disableSelection",function(e){e.preventDefault()})}}(),enableSelection:function(){return this.unbind(".ui-disableSelection")},zIndex:function(t){if(void 0!==t)return this.css("zIndex",t);if(this.length)for(var i,s,n=e(this[0]);n.length&&n[0]!==document;){if(i=n.css("position"),("absolute"===i||"relative"===i||"fixed"===i)&&(s=parseInt(n.css("zIndex"),10),!isNaN(s)&&0!==s))return s;n=n.parent()}return 0}}),e.ui.plugin={add:function(t,i,s){var n,a=e.ui[t].prototype;for(n in s)a.plugins[n]=a.plugins[n]||[],a.plugins[n].push([i,s[n]])},call:function(e,t,i,s){var n,a=e.plugins[t];if(a&&(s||e.element[0].parentNode&&11!==e.element[0].parentNode.nodeType))for(n=0;a.length>n;n++)e.options[a[n][0]]&&a[n][1].apply(e.element,i)}};var n=0,a=Array.prototype.slice;e.cleanData=function(t){return function(i){var s,n,a;for(a=0;null!=(n=i[a]);a++)try{s=e._data(n,"events"),s&&s.remove&&e(n).triggerHandler("remove")}catch(o){}t(i)}}(e.cleanData),e.widget=function(t,i,s){var n,a,o,r,h={},l=t.split(".")[0];return t=t.split(".")[1],n=l+"-"+t,s||(s=i,i=e.Widget),e.expr[":"][n.toLowerCase()]=function(t){return!!e.data(t,n)},e[l]=e[l]||{},a=e[l][t],o=e[l][t]=function(e,t){return this._createWidget?(arguments.length&&this._createWidget(e,t),void 0):new o(e,t)},e.extend(o,a,{version:s.version,_proto:e.extend({},s),_childConstructors:[]}),r=new i,r.options=e.widget.extend({},r.options),e.each(s,function(t,s){return e.isFunction(s)?(h[t]=function(){var e=function(){return i.prototype[t].apply(this,arguments)},n=function(e){return i.prototype[t].apply(this,e)};return function(){var t,i=this._super,a=this._superApply;return this._super=e,this._superApply=n,t=s.apply(this,arguments),this._super=i,this._superApply=a,t}}(),void 0):(h[t]=s,void 0)}),o.prototype=e.widget.extend(r,{widgetEventPrefix:a?r.widgetEventPrefix||t:t},h,{constructor:o,namespace:l,widgetName:t,widgetFullName:n}),a?(e.each(a._childConstructors,function(t,i){var s=i.prototype;e.widget(s.namespace+"."+s.widgetName,o,i._proto)}),delete a._childConstructors):i._childConstructors.push(o),e.widget.bridge(t,o),o},e.widget.extend=function(t){for(var i,s,n=a.call(arguments,1),o=0,r=n.length;r>o;o++)for(i in n[o])s=n[o][i],n[o].hasOwnProperty(i)&&void 0!==s&&(t[i]=e.isPlainObject(s)?e.isPlainObject(t[i])?e.widget.extend({},t[i],s):e.widget.extend({},s):s);return t},e.widget.bridge=function(t,i){var s=i.prototype.widgetFullName||t;e.fn[t]=function(n){var o="string"==typeof n,r=a.call(arguments,1),h=this;return o?this.each(function(){var i,a=e.data(this,s);return"instance"===n?(h=a,!1):a?e.isFunction(a[n])&&"_"!==n.charAt(0)?(i=a[n].apply(a,r),i!==a&&void 0!==i?(h=i&&i.jquery?h.pushStack(i.get()):i,!1):void 0):e.error("no such method '"+n+"' for "+t+" widget instance"):e.error("cannot call methods on "+t+" prior to initialization; "+"attempted to call method '"+n+"'")}):(r.length&&(n=e.widget.extend.apply(null,[n].concat(r))),this.each(function(){var t=e.data(this,s);t?(t.option(n||{}),t._init&&t._init()):e.data(this,s,new i(n,this))})),h}},e.Widget=function(){},e.Widget._childConstructors=[],e.Widget.prototype={widgetName:"widget",widgetEventPrefix:"",defaultElement:"<div>",options:{disabled:!1,create:null},_createWidget:function(t,i){i=e(i||this.defaultElement||this)[0],this.element=e(i),this.uuid=n++,this.eventNamespace="."+this.widgetName+this.uuid,this.bindings=e(),this.hoverable=e(),this.focusable=e(),i!==this&&(e.data(i,this.widgetFullName,this),this._on(!0,this.element,{remove:function(e){e.target===i&&this.destroy()}}),this.document=e(i.style?i.ownerDocument:i.document||i),this.window=e(this.document[0].defaultView||this.document[0].parentWindow)),this.options=e.widget.extend({},this.options,this._getCreateOptions(),t),this._create(),this._trigger("create",null,this._getCreateEventData()),this._init()},_getCreateOptions:e.noop,_getCreateEventData:e.noop,_create:e.noop,_init:e.noop,destroy:function(){this._destroy(),this.element.unbind(this.eventNamespace).removeData(this.widgetFullName).removeData(e.camelCase(this.widgetFullName)),this.widget().unbind(this.eventNamespace).removeAttr("aria-disabled").removeClass(this.widgetFullName+"-disabled "+"ui-state-disabled"),this.bindings.unbind(this.eventNamespace),this.hoverable.removeClass("ui-state-hover"),this.focusable.removeClass("ui-state-focus")},_destroy:e.noop,widget:function(){return this.element},option:function(t,i){var s,n,a,o=t;if(0===arguments.length)return e.widget.extend({},this.options);if("string"==typeof t)if(o={},s=t.split("."),t=s.shift(),s.length){for(n=o[t]=e.widget.extend({},this.options[t]),a=0;s.length-1>a;a++)n[s[a]]=n[s[a]]||{},n=n[s[a]];if(t=s.pop(),1===arguments.length)return void 0===n[t]?null:n[t];n[t]=i}else{if(1===arguments.length)return void 0===this.options[t]?null:this.options[t];o[t]=i}return this._setOptions(o),this},_setOptions:function(e){var t;for(t in e)this._setOption(t,e[t]);return this},_setOption:function(e,t){return this.options[e]=t,"disabled"===e&&(this.widget().toggleClass(this.widgetFullName+"-disabled",!!t),t&&(this.hoverable.removeClass("ui-state-hover"),this.focusable.removeClass("ui-state-focus"))),this},enable:function(){return this._setOptions({disabled:!1})},disable:function(){return this._setOptions({disabled:!0})},_on:function(t,i,s){var n,a=this;"boolean"!=typeof t&&(s=i,i=t,t=!1),s?(i=n=e(i),this.bindings=this.bindings.add(i)):(s=i,i=this.element,n=this.widget()),e.each(s,function(s,o){function r(){return t||a.options.disabled!==!0&&!e(this).hasClass("ui-state-disabled")?("string"==typeof o?a[o]:o).apply(a,arguments):void 0}"string"!=typeof o&&(r.guid=o.guid=o.guid||r.guid||e.guid++);var h=s.match(/^([\w:-]*)\s*(.*)$/),l=h[1]+a.eventNamespace,u=h[2];u?n.delegate(u,l,r):i.bind(l,r)})},_off:function(t,i){i=(i||"").split(" ").join(this.eventNamespace+" ")+this.eventNamespace,t.unbind(i).undelegate(i),this.bindings=e(this.bindings.not(t).get()),this.focusable=e(this.focusable.not(t).get()),this.hoverable=e(this.hoverable.not(t).get())},_delay:function(e,t){function i(){return("string"==typeof e?s[e]:e).apply(s,arguments)}var s=this;return setTimeout(i,t||0)},_hoverable:function(t){this.hoverable=this.hoverable.add(t),this._on(t,{mouseenter:function(t){e(t.currentTarget).addClass("ui-state-hover")},mouseleave:function(t){e(t.currentTarget).removeClass("ui-state-hover")}})},_focusable:function(t){this.focusable=this.focusable.add(t),this._on(t,{focusin:function(t){e(t.currentTarget).addClass("ui-state-focus")},focusout:function(t){e(t.currentTarget).removeClass("ui-state-focus")}})},_trigger:function(t,i,s){var n,a,o=this.options[t];if(s=s||{},i=e.Event(i),i.type=(t===this.widgetEventPrefix?t:this.widgetEventPrefix+t).toLowerCase(),i.target=this.element[0],a=i.originalEvent)for(n in a)n in i||(i[n]=a[n]);return this.element.trigger(i,s),!(e.isFunction(o)&&o.apply(this.element[0],[i].concat(s))===!1||i.isDefaultPrevented())}},e.each({show:"fadeIn",hide:"fadeOut"},function(t,i){e.Widget.prototype["_"+t]=function(s,n,a){"string"==typeof n&&(n={effect:n});var o,r=n?n===!0||"number"==typeof n?i:n.effect||i:t;n=n||{},"number"==typeof n&&(n={duration:n}),o=!e.isEmptyObject(n),n.complete=a,n.delay&&s.delay(n.delay),o&&e.effects&&e.effects.effect[r]?s[t](n):r!==t&&s[r]?s[r](n.duration,n.easing,a):s.queue(function(i){e(this)[t](),a&&a.call(s[0]),i()})}}),e.widget;var o=!1;e(document).mouseup(function(){o=!1}),e.widget("ui.mouse",{version:"1.11.4",options:{cancel:"input,textarea,button,select,option",distance:1,delay:0},_mouseInit:function(){var t=this;this.element.bind("mousedown."+this.widgetName,function(e){return t._mouseDown(e)}).bind("click."+this.widgetName,function(i){return!0===e.data(i.target,t.widgetName+".preventClickEvent")?(e.removeData(i.target,t.widgetName+".preventClickEvent"),i.stopImmediatePropagation(),!1):void 0}),this.started=!1},_mouseDestroy:function(){this.element.unbind("."+this.widgetName),this._mouseMoveDelegate&&this.document.unbind("mousemove."+this.widgetName,this._mouseMoveDelegate).unbind("mouseup."+this.widgetName,this._mouseUpDelegate)},_mouseDown:function(t){if(!o){this._mouseMoved=!1,this._mouseStarted&&this._mouseUp(t),this._mouseDownEvent=t;var i=this,s=1===t.which,n="string"==typeof this.options.cancel&&t.target.nodeName?e(t.target).closest(this.options.cancel).length:!1;return s&&!n&&this._mouseCapture(t)?(this.mouseDelayMet=!this.options.delay,this.mouseDelayMet||(this._mouseDelayTimer=setTimeout(function(){i.mouseDelayMet=!0},this.options.delay)),this._mouseDistanceMet(t)&&this._mouseDelayMet(t)&&(this._mouseStarted=this._mouseStart(t)!==!1,!this._mouseStarted)?(t.preventDefault(),!0):(!0===e.data(t.target,this.widgetName+".preventClickEvent")&&e.removeData(t.target,this.widgetName+".preventClickEvent"),this._mouseMoveDelegate=function(e){return i._mouseMove(e)},this._mouseUpDelegate=function(e){return i._mouseUp(e)},this.document.bind("mousemove."+this.widgetName,this._mouseMoveDelegate).bind("mouseup."+this.widgetName,this._mouseUpDelegate),t.preventDefault(),o=!0,!0)):!0}},_mouseMove:function(t){if(this._mouseMoved){if(e.ui.ie&&(!document.documentMode||9>document.documentMode)&&!t.button)return this._mouseUp(t);if(!t.which)return this._mouseUp(t)}return(t.which||t.button)&&(this._mouseMoved=!0),this._mouseStarted?(this._mouseDrag(t),t.preventDefault()):(this._mouseDistanceMet(t)&&this._mouseDelayMet(t)&&(this._mouseStarted=this._mouseStart(this._mouseDownEvent,t)!==!1,this._mouseStarted?this._mouseDrag(t):this._mouseUp(t)),!this._mouseStarted)},_mouseUp:function(t){return this.document.unbind("mousemove."+this.widgetName,this._mouseMoveDelegate).unbind("mouseup."+this.widgetName,this._mouseUpDelegate),this._mouseStarted&&(this._mouseStarted=!1,t.target===this._mouseDownEvent.target&&e.data(t.target,this.widgetName+".preventClickEvent",!0),this._mouseStop(t)),o=!1,!1},_mouseDistanceMet:function(e){return Math.max(Math.abs(this._mouseDownEvent.pageX-e.pageX),Math.abs(this._mouseDownEvent.pageY-e.pageY))>=this.options.distance},_mouseDelayMet:function(){return this.mouseDelayMet},_mouseStart:function(){},_mouseDrag:function(){},_mouseStop:function(){},_mouseCapture:function(){return!0}}),function(){function t(e,t,i){return[parseFloat(e[0])*(p.test(e[0])?t/100:1),parseFloat(e[1])*(p.test(e[1])?i/100:1)]}function i(t,i){return parseInt(e.css(t,i),10)||0}function s(t){var i=t[0];return 9===i.nodeType?{width:t.width(),height:t.height(),offset:{top:0,left:0}}:e.isWindow(i)?{width:t.width(),height:t.height(),offset:{top:t.scrollTop(),left:t.scrollLeft()}}:i.preventDefault?{width:0,height:0,offset:{top:i.pageY,left:i.pageX}}:{width:t.outerWidth(),height:t.outerHeight(),offset:t.offset()}}e.ui=e.ui||{};var n,a,o=Math.max,r=Math.abs,h=Math.round,l=/left|center|right/,u=/top|center|bottom/,d=/[\+\-]\d+(\.[\d]+)?%?/,c=/^\w+/,p=/%$/,f=e.fn.position;e.position={scrollbarWidth:function(){if(void 0!==n)return n;var t,i,s=e("<div style='display:block;position:absolute;width:50px;height:50px;overflow:hidden;'><div style='height:100px;width:auto;'></div></div>"),a=s.children()[0];return e("body").append(s),t=a.offsetWidth,s.css("overflow","scroll"),i=a.offsetWidth,t===i&&(i=s[0].clientWidth),s.remove(),n=t-i},getScrollInfo:function(t){var i=t.isWindow||t.isDocument?"":t.element.css("overflow-x"),s=t.isWindow||t.isDocument?"":t.element.css("overflow-y"),n="scroll"===i||"auto"===i&&t.width<t.element[0].scrollWidth,a="scroll"===s||"auto"===s&&t.height<t.element[0].scrollHeight;return{width:a?e.position.scrollbarWidth():0,height:n?e.position.scrollbarWidth():0}},getWithinInfo:function(t){var i=e(t||window),s=e.isWindow(i[0]),n=!!i[0]&&9===i[0].nodeType;return{element:i,isWindow:s,isDocument:n,offset:i.offset()||{left:0,top:0},scrollLeft:i.scrollLeft(),scrollTop:i.scrollTop(),width:s||n?i.width():i.outerWidth(),height:s||n?i.height():i.outerHeight()}}},e.fn.position=function(n){if(!n||!n.of)return f.apply(this,arguments);n=e.extend({},n);var p,m,g,v,y,b,_=e(n.of),x=e.position.getWithinInfo(n.within),w=e.position.getScrollInfo(x),k=(n.collision||"flip").split(" "),T={};return b=s(_),_[0].preventDefault&&(n.at="left top"),m=b.width,g=b.height,v=b.offset,y=e.extend({},v),e.each(["my","at"],function(){var e,t,i=(n[this]||"").split(" ");1===i.length&&(i=l.test(i[0])?i.concat(["center"]):u.test(i[0])?["center"].concat(i):["center","center"]),i[0]=l.test(i[0])?i[0]:"center",i[1]=u.test(i[1])?i[1]:"center",e=d.exec(i[0]),t=d.exec(i[1]),T[this]=[e?e[0]:0,t?t[0]:0],n[this]=[c.exec(i[0])[0],c.exec(i[1])[0]]}),1===k.length&&(k[1]=k[0]),"right"===n.at[0]?y.left+=m:"center"===n.at[0]&&(y.left+=m/2),"bottom"===n.at[1]?y.top+=g:"center"===n.at[1]&&(y.top+=g/2),p=t(T.at,m,g),y.left+=p[0],y.top+=p[1],this.each(function(){var s,l,u=e(this),d=u.outerWidth(),c=u.outerHeight(),f=i(this,"marginLeft"),b=i(this,"marginTop"),D=d+f+i(this,"marginRight")+w.width,S=c+b+i(this,"marginBottom")+w.height,N=e.extend({},y),M=t(T.my,u.outerWidth(),u.outerHeight());"right"===n.my[0]?N.left-=d:"center"===n.my[0]&&(N.left-=d/2),"bottom"===n.my[1]?N.top-=c:"center"===n.my[1]&&(N.top-=c/2),N.left+=M[0],N.top+=M[1],a||(N.left=h(N.left),N.top=h(N.top)),s={marginLeft:f,marginTop:b},e.each(["left","top"],function(t,i){e.ui.position[k[t]]&&e.ui.position[k[t]][i](N,{targetWidth:m,targetHeight:g,elemWidth:d,elemHeight:c,collisionPosition:s,collisionWidth:D,collisionHeight:S,offset:[p[0]+M[0],p[1]+M[1]],my:n.my,at:n.at,within:x,elem:u})}),n.using&&(l=function(e){var t=v.left-N.left,i=t+m-d,s=v.top-N.top,a=s+g-c,h={target:{element:_,left:v.left,top:v.top,width:m,height:g},element:{element:u,left:N.left,top:N.top,width:d,height:c},horizontal:0>i?"left":t>0?"right":"center",vertical:0>a?"top":s>0?"bottom":"middle"};d>m&&m>r(t+i)&&(h.horizontal="center"),c>g&&g>r(s+a)&&(h.vertical="middle"),h.important=o(r(t),r(i))>o(r(s),r(a))?"horizontal":"vertical",n.using.call(this,e,h)}),u.offset(e.extend(N,{using:l}))})},e.ui.position={fit:{left:function(e,t){var i,s=t.within,n=s.isWindow?s.scrollLeft:s.offset.left,a=s.width,r=e.left-t.collisionPosition.marginLeft,h=n-r,l=r+t.collisionWidth-a-n;t.collisionWidth>a?h>0&&0>=l?(i=e.left+h+t.collisionWidth-a-n,e.left+=h-i):e.left=l>0&&0>=h?n:h>l?n+a-t.collisionWidth:n:h>0?e.left+=h:l>0?e.left-=l:e.left=o(e.left-r,e.left)},top:function(e,t){var i,s=t.within,n=s.isWindow?s.scrollTop:s.offset.top,a=t.within.height,r=e.top-t.collisionPosition.marginTop,h=n-r,l=r+t.collisionHeight-a-n;t.collisionHeight>a?h>0&&0>=l?(i=e.top+h+t.collisionHeight-a-n,e.top+=h-i):e.top=l>0&&0>=h?n:h>l?n+a-t.collisionHeight:n:h>0?e.top+=h:l>0?e.top-=l:e.top=o(e.top-r,e.top)}},flip:{left:function(e,t){var i,s,n=t.within,a=n.offset.left+n.scrollLeft,o=n.width,h=n.isWindow?n.scrollLeft:n.offset.left,l=e.left-t.collisionPosition.marginLeft,u=l-h,d=l+t.collisionWidth-o-h,c="left"===t.my[0]?-t.elemWidth:"right"===t.my[0]?t.elemWidth:0,p="left"===t.at[0]?t.targetWidth:"right"===t.at[0]?-t.targetWidth:0,f=-2*t.offset[0];0>u?(i=e.left+c+p+f+t.collisionWidth-o-a,(0>i||r(u)>i)&&(e.left+=c+p+f)):d>0&&(s=e.left-t.collisionPosition.marginLeft+c+p+f-h,(s>0||d>r(s))&&(e.left+=c+p+f))},top:function(e,t){var i,s,n=t.within,a=n.offset.top+n.scrollTop,o=n.height,h=n.isWindow?n.scrollTop:n.offset.top,l=e.top-t.collisionPosition.marginTop,u=l-h,d=l+t.collisionHeight-o-h,c="top"===t.my[1],p=c?-t.elemHeight:"bottom"===t.my[1]?t.elemHeight:0,f="top"===t.at[1]?t.targetHeight:"bottom"===t.at[1]?-t.targetHeight:0,m=-2*t.offset[1];0>u?(s=e.top+p+f+m+t.collisionHeight-o-a,(0>s||r(u)>s)&&(e.top+=p+f+m)):d>0&&(i=e.top-t.collisionPosition.marginTop+p+f+m-h,(i>0||d>r(i))&&(e.top+=p+f+m))}},flipfit:{left:function(){e.ui.position.flip.left.apply(this,arguments),e.ui.position.fit.left.apply(this,arguments)},top:function(){e.ui.position.flip.top.apply(this,arguments),e.ui.position.fit.top.apply(this,arguments)}}},function(){var t,i,s,n,o,r=document.getElementsByTagName("body")[0],h=document.createElement("div");t=document.createElement(r?"div":"body"),s={visibility:"hidden",width:0,height:0,border:0,margin:0,background:"none"},r&&e.extend(s,{position:"absolute",left:"-1000px",top:"-1000px"});for(o in s)t.style[o]=s[o];t.appendChild(h),i=r||document.documentElement,i.insertBefore(t,i.firstChild),h.style.cssText="position: absolute; left: 10.7432222px;",n=e(h).offset().left,a=n>10&&11>n,t.innerHTML="",i.removeChild(t)}()}(),e.ui.position,e.widget("ui.draggable",e.ui.mouse,{version:"1.11.4",widgetEventPrefix:"drag",options:{addClasses:!0,appendTo:"parent",axis:!1,connectToSortable:!1,containment:!1,cursor:"auto",cursorAt:!1,grid:!1,handle:!1,helper:"original",iframeFix:!1,opacity:!1,refreshPositions:!1,revert:!1,revertDuration:500,scope:"default",scroll:!0,scrollSensitivity:20,scrollSpeed:20,snap:!1,snapMode:"both",snapTolerance:20,stack:!1,zIndex:!1,drag:null,start:null,stop:null},_create:function(){"original"===this.options.helper&&this._setPositionRelative(),this.options.addClasses&&this.element.addClass("ui-draggable"),this.options.disabled&&this.element.addClass("ui-draggable-disabled"),this._setHandleClassName(),this._mouseInit()},_setOption:function(e,t){this._super(e,t),"handle"===e&&(this._removeHandleClassName(),this._setHandleClassName())},_destroy:function(){return(this.helper||this.element).is(".ui-draggable-dragging")?(this.destroyOnClear=!0,void 0):(this.element.removeClass("ui-draggable ui-draggable-dragging ui-draggable-disabled"),this._removeHandleClassName(),this._mouseDestroy(),void 0)},_mouseCapture:function(t){var i=this.options;return this._blurActiveElement(t),this.helper||i.disabled||e(t.target).closest(".ui-resizable-handle").length>0?!1:(this.handle=this._getHandle(t),this.handle?(this._blockFrames(i.iframeFix===!0?"iframe":i.iframeFix),!0):!1)},_blockFrames:function(t){this.iframeBlocks=this.document.find(t).map(function(){var t=e(this);return e("<div>").css("position","absolute").appendTo(t.parent()).outerWidth(t.outerWidth()).outerHeight(t.outerHeight()).offset(t.offset())[0]})},_unblockFrames:function(){this.iframeBlocks&&(this.iframeBlocks.remove(),delete this.iframeBlocks)},_blurActiveElement:function(t){var i=this.document[0];if(this.handleElement.is(t.target))try{i.activeElement&&"body"!==i.activeElement.nodeName.toLowerCase()&&e(i.activeElement).blur()}catch(s){}},_mouseStart:function(t){var i=this.options;return this.helper=this._createHelper(t),this.helper.addClass("ui-draggable-dragging"),this._cacheHelperProportions(),e.ui.ddmanager&&(e.ui.ddmanager.current=this),this._cacheMargins(),this.cssPosition=this.helper.css("position"),this.scrollParent=this.helper.scrollParent(!0),this.offsetParent=this.helper.offsetParent(),this.hasFixedAncestor=this.helper.parents().filter(function(){return"fixed"===e(this).css("position")}).length>0,this.positionAbs=this.element.offset(),this._refreshOffsets(t),this.originalPosition=this.position=this._generatePosition(t,!1),this.originalPageX=t.pageX,this.originalPageY=t.pageY,i.cursorAt&&this._adjustOffsetFromHelper(i.cursorAt),this._setContainment(),this._trigger("start",t)===!1?(this._clear(),!1):(this._cacheHelperProportions(),e.ui.ddmanager&&!i.dropBehaviour&&e.ui.ddmanager.prepareOffsets(this,t),this._normalizeRightBottom(),this._mouseDrag(t,!0),e.ui.ddmanager&&e.ui.ddmanager.dragStart(this,t),!0)},_refreshOffsets:function(e){this.offset={top:this.positionAbs.top-this.margins.top,left:this.positionAbs.left-this.margins.left,scroll:!1,parent:this._getParentOffset(),relative:this._getRelativeOffset()},this.offset.click={left:e.pageX-this.offset.left,top:e.pageY-this.offset.top}},_mouseDrag:function(t,i){if(this.hasFixedAncestor&&(this.offset.parent=this._getParentOffset()),this.position=this._generatePosition(t,!0),this.positionAbs=this._convertPositionTo("absolute"),!i){var s=this._uiHash();if(this._trigger("drag",t,s)===!1)return this._mouseUp({}),!1;this.position=s.position}return this.helper[0].style.left=this.position.left+"px",this.helper[0].style.top=this.position.top+"px",e.ui.ddmanager&&e.ui.ddmanager.drag(this,t),!1},_mouseStop:function(t){var i=this,s=!1;return e.ui.ddmanager&&!this.options.dropBehaviour&&(s=e.ui.ddmanager.drop(this,t)),this.dropped&&(s=this.dropped,this.dropped=!1),"invalid"===this.options.revert&&!s||"valid"===this.options.revert&&s||this.options.revert===!0||e.isFunction(this.options.revert)&&this.options.revert.call(this.element,s)?e(this.helper).animate(this.originalPosition,parseInt(this.options.revertDuration,10),function(){i._trigger("stop",t)!==!1&&i._clear()}):this._trigger("stop",t)!==!1&&this._clear(),!1},_mouseUp:function(t){return this._unblockFrames(),e.ui.ddmanager&&e.ui.ddmanager.dragStop(this,t),this.handleElement.is(t.target)&&this.element.focus(),e.ui.mouse.prototype._mouseUp.call(this,t)},cancel:function(){return this.helper.is(".ui-draggable-dragging")?this._mouseUp({}):this._clear(),this},_getHandle:function(t){return this.options.handle?!!e(t.target).closest(this.element.find(this.options.handle)).length:!0},_setHandleClassName:function(){this.handleElement=this.options.handle?this.element.find(this.options.handle):this.element,this.handleElement.addClass("ui-draggable-handle")},_removeHandleClassName:function(){this.handleElement.removeClass("ui-draggable-handle")},_createHelper:function(t){var i=this.options,s=e.isFunction(i.helper),n=s?e(i.helper.apply(this.element[0],[t])):"clone"===i.helper?this.element.clone().removeAttr("id"):this.element;return n.parents("body").length||n.appendTo("parent"===i.appendTo?this.element[0].parentNode:i.appendTo),s&&n[0]===this.element[0]&&this._setPositionRelative(),n[0]===this.element[0]||/(fixed|absolute)/.test(n.css("position"))||n.css("position","absolute"),n},_setPositionRelative:function(){/^(?:r|a|f)/.test(this.element.css("position"))||(this.element[0].style.position="relative")},_adjustOffsetFromHelper:function(t){"string"==typeof t&&(t=t.split(" ")),e.isArray(t)&&(t={left:+t[0],top:+t[1]||0}),"left"in t&&(this.offset.click.left=t.left+this.margins.left),"right"in t&&(this.offset.click.left=this.helperProportions.width-t.right+this.margins.left),"top"in t&&(this.offset.click.top=t.top+this.margins.top),"bottom"in t&&(this.offset.click.top=this.helperProportions.height-t.bottom+this.margins.top)},_isRootNode:function(e){return/(html|body)/i.test(e.tagName)||e===this.document[0]},_getParentOffset:function(){var t=this.offsetParent.offset(),i=this.document[0];return"absolute"===this.cssPosition&&this.scrollParent[0]!==i&&e.contains(this.scrollParent[0],this.offsetParent[0])&&(t.left+=this.scrollParent.scrollLeft(),t.top+=this.scrollParent.scrollTop()),this._isRootNode(this.offsetParent[0])&&(t={top:0,left:0}),{top:t.top+(parseInt(this.offsetParent.css("borderTopWidth"),10)||0),left:t.left+(parseInt(this.offsetParent.css("borderLeftWidth"),10)||0)}},_getRelativeOffset:function(){if("relative"!==this.cssPosition)return{top:0,left:0};var e=this.element.position(),t=this._isRootNode(this.scrollParent[0]);return{top:e.top-(parseInt(this.helper.css("top"),10)||0)+(t?0:this.scrollParent.scrollTop()),left:e.left-(parseInt(this.helper.css("left"),10)||0)+(t?0:this.scrollParent.scrollLeft())}},_cacheMargins:function(){this.margins={left:parseInt(this.element.css("marginLeft"),10)||0,top:parseInt(this.element.css("marginTop"),10)||0,right:parseInt(this.element.css("marginRight"),10)||0,bottom:parseInt(this.element.css("marginBottom"),10)||0}},_cacheHelperProportions:function(){this.helperProportions={width:this.helper.outerWidth(),height:this.helper.outerHeight()}},_setContainment:function(){var t,i,s,n=this.options,a=this.document[0];return this.relativeContainer=null,n.containment?"window"===n.containment?(this.containment=[e(window).scrollLeft()-this.offset.relative.left-this.offset.parent.left,e(window).scrollTop()-this.offset.relative.top-this.offset.parent.top,e(window).scrollLeft()+e(window).width()-this.helperProportions.width-this.margins.left,e(window).scrollTop()+(e(window).height()||a.body.parentNode.scrollHeight)-this.helperProportions.height-this.margins.top],void 0):"document"===n.containment?(this.containment=[0,0,e(a).width()-this.helperProportions.width-this.margins.left,(e(a).height()||a.body.parentNode.scrollHeight)-this.helperProportions.height-this.margins.top],void 0):n.containment.constructor===Array?(this.containment=n.containment,void 0):("parent"===n.containment&&(n.containment=this.helper[0].parentNode),i=e(n.containment),s=i[0],s&&(t=/(scroll|auto)/.test(i.css("overflow")),this.containment=[(parseInt(i.css("borderLeftWidth"),10)||0)+(parseInt(i.css("paddingLeft"),10)||0),(parseInt(i.css("borderTopWidth"),10)||0)+(parseInt(i.css("paddingTop"),10)||0),(t?Math.max(s.scrollWidth,s.offsetWidth):s.offsetWidth)-(parseInt(i.css("borderRightWidth"),10)||0)-(parseInt(i.css("paddingRight"),10)||0)-this.helperProportions.width-this.margins.left-this.margins.right,(t?Math.max(s.scrollHeight,s.offsetHeight):s.offsetHeight)-(parseInt(i.css("borderBottomWidth"),10)||0)-(parseInt(i.css("paddingBottom"),10)||0)-this.helperProportions.height-this.margins.top-this.margins.bottom],this.relativeContainer=i),void 0):(this.containment=null,void 0)},_convertPositionTo:function(e,t){t||(t=this.position);var i="absolute"===e?1:-1,s=this._isRootNode(this.scrollParent[0]);return{top:t.top+this.offset.relative.top*i+this.offset.parent.top*i-("fixed"===this.cssPosition?-this.offset.scroll.top:s?0:this.offset.scroll.top)*i,left:t.left+this.offset.relative.left*i+this.offset.parent.left*i-("fixed"===this.cssPosition?-this.offset.scroll.left:s?0:this.offset.scroll.left)*i}},_generatePosition:function(e,t){var i,s,n,a,o=this.options,r=this._isRootNode(this.scrollParent[0]),h=e.pageX,l=e.pageY;return r&&this.offset.scroll||(this.offset.scroll={top:this.scrollParent.scrollTop(),left:this.scrollParent.scrollLeft()}),t&&(this.containment&&(this.relativeContainer?(s=this.relativeContainer.offset(),i=[this.containment[0]+s.left,this.containment[1]+s.top,this.containment[2]+s.left,this.containment[3]+s.top]):i=this.containment,e.pageX-this.offset.click.left<i[0]&&(h=i[0]+this.offset.click.left),e.pageY-this.offset.click.top<i[1]&&(l=i[1]+this.offset.click.top),e.pageX-this.offset.click.left>i[2]&&(h=i[2]+this.offset.click.left),e.pageY-this.offset.click.top>i[3]&&(l=i[3]+this.offset.click.top)),o.grid&&(n=o.grid[1]?this.originalPageY+Math.round((l-this.originalPageY)/o.grid[1])*o.grid[1]:this.originalPageY,l=i?n-this.offset.click.top>=i[1]||n-this.offset.click.top>i[3]?n:n-this.offset.click.top>=i[1]?n-o.grid[1]:n+o.grid[1]:n,a=o.grid[0]?this.originalPageX+Math.round((h-this.originalPageX)/o.grid[0])*o.grid[0]:this.originalPageX,h=i?a-this.offset.click.left>=i[0]||a-this.offset.click.left>i[2]?a:a-this.offset.click.left>=i[0]?a-o.grid[0]:a+o.grid[0]:a),"y"===o.axis&&(h=this.originalPageX),"x"===o.axis&&(l=this.originalPageY)),{top:l-this.offset.click.top-this.offset.relative.top-this.offset.parent.top+("fixed"===this.cssPosition?-this.offset.scroll.top:r?0:this.offset.scroll.top),left:h-this.offset.click.left-this.offset.relative.left-this.offset.parent.left+("fixed"===this.cssPosition?-this.offset.scroll.left:r?0:this.offset.scroll.left)}},_clear:function(){this.helper.removeClass("ui-draggable-dragging"),this.helper[0]===this.element[0]||this.cancelHelperRemoval||this.helper.remove(),this.helper=null,this.cancelHelperRemoval=!1,this.destroyOnClear&&this.destroy()},_normalizeRightBottom:function(){"y"!==this.options.axis&&"auto"!==this.helper.css("right")&&(this.helper.width(this.helper.width()),this.helper.css("right","auto")),"x"!==this.options.axis&&"auto"!==this.helper.css("bottom")&&(this.helper.height(this.helper.height()),this.helper.css("bottom","auto"))},_trigger:function(t,i,s){return s=s||this._uiHash(),e.ui.plugin.call(this,t,[i,s,this],!0),/^(drag|start|stop)/.test(t)&&(this.positionAbs=this._convertPositionTo("absolute"),s.offset=this.positionAbs),e.Widget.prototype._trigger.call(this,t,i,s)},plugins:{},_uiHash:function(){return{helper:this.helper,position:this.position,originalPosition:this.originalPosition,offset:this.positionAbs}}}),e.ui.plugin.add("draggable","connectToSortable",{start:function(t,i,s){var n=e.extend({},i,{item:s.element});s.sortables=[],e(s.options.connectToSortable).each(function(){var i=e(this).sortable("instance");i&&!i.options.disabled&&(s.sortables.push(i),i.refreshPositions(),i._trigger("activate",t,n))})},stop:function(t,i,s){var n=e.extend({},i,{item:s.element});s.cancelHelperRemoval=!1,e.each(s.sortables,function(){var e=this;e.isOver?(e.isOver=0,s.cancelHelperRemoval=!0,e.cancelHelperRemoval=!1,e._storedCSS={position:e.placeholder.css("position"),top:e.placeholder.css("top"),left:e.placeholder.css("left")},e._mouseStop(t),e.options.helper=e.options._helper):(e.cancelHelperRemoval=!0,e._trigger("deactivate",t,n))})},drag:function(t,i,s){e.each(s.sortables,function(){var n=!1,a=this;a.positionAbs=s.positionAbs,a.helperProportions=s.helperProportions,a.offset.click=s.offset.click,a._intersectsWith(a.containerCache)&&(n=!0,e.each(s.sortables,function(){return this.positionAbs=s.positionAbs,this.helperProportions=s.helperProportions,this.offset.click=s.offset.click,this!==a&&this._intersectsWith(this.containerCache)&&e.contains(a.element[0],this.element[0])&&(n=!1),n
})),n?(a.isOver||(a.isOver=1,s._parent=i.helper.parent(),a.currentItem=i.helper.appendTo(a.element).data("ui-sortable-item",!0),a.options._helper=a.options.helper,a.options.helper=function(){return i.helper[0]},t.target=a.currentItem[0],a._mouseCapture(t,!0),a._mouseStart(t,!0,!0),a.offset.click.top=s.offset.click.top,a.offset.click.left=s.offset.click.left,a.offset.parent.left-=s.offset.parent.left-a.offset.parent.left,a.offset.parent.top-=s.offset.parent.top-a.offset.parent.top,s._trigger("toSortable",t),s.dropped=a.element,e.each(s.sortables,function(){this.refreshPositions()}),s.currentItem=s.element,a.fromOutside=s),a.currentItem&&(a._mouseDrag(t),i.position=a.position)):a.isOver&&(a.isOver=0,a.cancelHelperRemoval=!0,a.options._revert=a.options.revert,a.options.revert=!1,a._trigger("out",t,a._uiHash(a)),a._mouseStop(t,!0),a.options.revert=a.options._revert,a.options.helper=a.options._helper,a.placeholder&&a.placeholder.remove(),i.helper.appendTo(s._parent),s._refreshOffsets(t),i.position=s._generatePosition(t,!0),s._trigger("fromSortable",t),s.dropped=!1,e.each(s.sortables,function(){this.refreshPositions()}))})}}),e.ui.plugin.add("draggable","cursor",{start:function(t,i,s){var n=e("body"),a=s.options;n.css("cursor")&&(a._cursor=n.css("cursor")),n.css("cursor",a.cursor)},stop:function(t,i,s){var n=s.options;n._cursor&&e("body").css("cursor",n._cursor)}}),e.ui.plugin.add("draggable","opacity",{start:function(t,i,s){var n=e(i.helper),a=s.options;n.css("opacity")&&(a._opacity=n.css("opacity")),n.css("opacity",a.opacity)},stop:function(t,i,s){var n=s.options;n._opacity&&e(i.helper).css("opacity",n._opacity)}}),e.ui.plugin.add("draggable","scroll",{start:function(e,t,i){i.scrollParentNotHidden||(i.scrollParentNotHidden=i.helper.scrollParent(!1)),i.scrollParentNotHidden[0]!==i.document[0]&&"HTML"!==i.scrollParentNotHidden[0].tagName&&(i.overflowOffset=i.scrollParentNotHidden.offset())},drag:function(t,i,s){var n=s.options,a=!1,o=s.scrollParentNotHidden[0],r=s.document[0];o!==r&&"HTML"!==o.tagName?(n.axis&&"x"===n.axis||(s.overflowOffset.top+o.offsetHeight-t.pageY<n.scrollSensitivity?o.scrollTop=a=o.scrollTop+n.scrollSpeed:t.pageY-s.overflowOffset.top<n.scrollSensitivity&&(o.scrollTop=a=o.scrollTop-n.scrollSpeed)),n.axis&&"y"===n.axis||(s.overflowOffset.left+o.offsetWidth-t.pageX<n.scrollSensitivity?o.scrollLeft=a=o.scrollLeft+n.scrollSpeed:t.pageX-s.overflowOffset.left<n.scrollSensitivity&&(o.scrollLeft=a=o.scrollLeft-n.scrollSpeed))):(n.axis&&"x"===n.axis||(t.pageY-e(r).scrollTop()<n.scrollSensitivity?a=e(r).scrollTop(e(r).scrollTop()-n.scrollSpeed):e(window).height()-(t.pageY-e(r).scrollTop())<n.scrollSensitivity&&(a=e(r).scrollTop(e(r).scrollTop()+n.scrollSpeed))),n.axis&&"y"===n.axis||(t.pageX-e(r).scrollLeft()<n.scrollSensitivity?a=e(r).scrollLeft(e(r).scrollLeft()-n.scrollSpeed):e(window).width()-(t.pageX-e(r).scrollLeft())<n.scrollSensitivity&&(a=e(r).scrollLeft(e(r).scrollLeft()+n.scrollSpeed)))),a!==!1&&e.ui.ddmanager&&!n.dropBehaviour&&e.ui.ddmanager.prepareOffsets(s,t)}}),e.ui.plugin.add("draggable","snap",{start:function(t,i,s){var n=s.options;s.snapElements=[],e(n.snap.constructor!==String?n.snap.items||":data(ui-draggable)":n.snap).each(function(){var t=e(this),i=t.offset();this!==s.element[0]&&s.snapElements.push({item:this,width:t.outerWidth(),height:t.outerHeight(),top:i.top,left:i.left})})},drag:function(t,i,s){var n,a,o,r,h,l,u,d,c,p,f=s.options,m=f.snapTolerance,g=i.offset.left,v=g+s.helperProportions.width,y=i.offset.top,b=y+s.helperProportions.height;for(c=s.snapElements.length-1;c>=0;c--)h=s.snapElements[c].left-s.margins.left,l=h+s.snapElements[c].width,u=s.snapElements[c].top-s.margins.top,d=u+s.snapElements[c].height,h-m>v||g>l+m||u-m>b||y>d+m||!e.contains(s.snapElements[c].item.ownerDocument,s.snapElements[c].item)?(s.snapElements[c].snapping&&s.options.snap.release&&s.options.snap.release.call(s.element,t,e.extend(s._uiHash(),{snapItem:s.snapElements[c].item})),s.snapElements[c].snapping=!1):("inner"!==f.snapMode&&(n=m>=Math.abs(u-b),a=m>=Math.abs(d-y),o=m>=Math.abs(h-v),r=m>=Math.abs(l-g),n&&(i.position.top=s._convertPositionTo("relative",{top:u-s.helperProportions.height,left:0}).top),a&&(i.position.top=s._convertPositionTo("relative",{top:d,left:0}).top),o&&(i.position.left=s._convertPositionTo("relative",{top:0,left:h-s.helperProportions.width}).left),r&&(i.position.left=s._convertPositionTo("relative",{top:0,left:l}).left)),p=n||a||o||r,"outer"!==f.snapMode&&(n=m>=Math.abs(u-y),a=m>=Math.abs(d-b),o=m>=Math.abs(h-g),r=m>=Math.abs(l-v),n&&(i.position.top=s._convertPositionTo("relative",{top:u,left:0}).top),a&&(i.position.top=s._convertPositionTo("relative",{top:d-s.helperProportions.height,left:0}).top),o&&(i.position.left=s._convertPositionTo("relative",{top:0,left:h}).left),r&&(i.position.left=s._convertPositionTo("relative",{top:0,left:l-s.helperProportions.width}).left)),!s.snapElements[c].snapping&&(n||a||o||r||p)&&s.options.snap.snap&&s.options.snap.snap.call(s.element,t,e.extend(s._uiHash(),{snapItem:s.snapElements[c].item})),s.snapElements[c].snapping=n||a||o||r||p)}}),e.ui.plugin.add("draggable","stack",{start:function(t,i,s){var n,a=s.options,o=e.makeArray(e(a.stack)).sort(function(t,i){return(parseInt(e(t).css("zIndex"),10)||0)-(parseInt(e(i).css("zIndex"),10)||0)});o.length&&(n=parseInt(e(o[0]).css("zIndex"),10)||0,e(o).each(function(t){e(this).css("zIndex",n+t)}),this.css("zIndex",n+o.length))}}),e.ui.plugin.add("draggable","zIndex",{start:function(t,i,s){var n=e(i.helper),a=s.options;n.css("zIndex")&&(a._zIndex=n.css("zIndex")),n.css("zIndex",a.zIndex)},stop:function(t,i,s){var n=s.options;n._zIndex&&e(i.helper).css("zIndex",n._zIndex)}}),e.ui.draggable,e.widget("ui.droppable",{version:"1.11.4",widgetEventPrefix:"drop",options:{accept:"*",activeClass:!1,addClasses:!0,greedy:!1,hoverClass:!1,scope:"default",tolerance:"intersect",activate:null,deactivate:null,drop:null,out:null,over:null},_create:function(){var t,i=this.options,s=i.accept;this.isover=!1,this.isout=!0,this.accept=e.isFunction(s)?s:function(e){return e.is(s)},this.proportions=function(){return arguments.length?(t=arguments[0],void 0):t?t:t={width:this.element[0].offsetWidth,height:this.element[0].offsetHeight}},this._addToManager(i.scope),i.addClasses&&this.element.addClass("ui-droppable")},_addToManager:function(t){e.ui.ddmanager.droppables[t]=e.ui.ddmanager.droppables[t]||[],e.ui.ddmanager.droppables[t].push(this)},_splice:function(e){for(var t=0;e.length>t;t++)e[t]===this&&e.splice(t,1)},_destroy:function(){var t=e.ui.ddmanager.droppables[this.options.scope];this._splice(t),this.element.removeClass("ui-droppable ui-droppable-disabled")},_setOption:function(t,i){if("accept"===t)this.accept=e.isFunction(i)?i:function(e){return e.is(i)};else if("scope"===t){var s=e.ui.ddmanager.droppables[this.options.scope];this._splice(s),this._addToManager(i)}this._super(t,i)},_activate:function(t){var i=e.ui.ddmanager.current;this.options.activeClass&&this.element.addClass(this.options.activeClass),i&&this._trigger("activate",t,this.ui(i))},_deactivate:function(t){var i=e.ui.ddmanager.current;this.options.activeClass&&this.element.removeClass(this.options.activeClass),i&&this._trigger("deactivate",t,this.ui(i))},_over:function(t){var i=e.ui.ddmanager.current;i&&(i.currentItem||i.element)[0]!==this.element[0]&&this.accept.call(this.element[0],i.currentItem||i.element)&&(this.options.hoverClass&&this.element.addClass(this.options.hoverClass),this._trigger("over",t,this.ui(i)))},_out:function(t){var i=e.ui.ddmanager.current;i&&(i.currentItem||i.element)[0]!==this.element[0]&&this.accept.call(this.element[0],i.currentItem||i.element)&&(this.options.hoverClass&&this.element.removeClass(this.options.hoverClass),this._trigger("out",t,this.ui(i)))},_drop:function(t,i){var s=i||e.ui.ddmanager.current,n=!1;return s&&(s.currentItem||s.element)[0]!==this.element[0]?(this.element.find(":data(ui-droppable)").not(".ui-draggable-dragging").each(function(){var i=e(this).droppable("instance");return i.options.greedy&&!i.options.disabled&&i.options.scope===s.options.scope&&i.accept.call(i.element[0],s.currentItem||s.element)&&e.ui.intersect(s,e.extend(i,{offset:i.element.offset()}),i.options.tolerance,t)?(n=!0,!1):void 0}),n?!1:this.accept.call(this.element[0],s.currentItem||s.element)?(this.options.activeClass&&this.element.removeClass(this.options.activeClass),this.options.hoverClass&&this.element.removeClass(this.options.hoverClass),this._trigger("drop",t,this.ui(s)),this.element):!1):!1},ui:function(e){return{draggable:e.currentItem||e.element,helper:e.helper,position:e.position,offset:e.positionAbs}}}),e.ui.intersect=function(){function e(e,t,i){return e>=t&&t+i>e}return function(t,i,s,n){if(!i.offset)return!1;var a=(t.positionAbs||t.position.absolute).left+t.margins.left,o=(t.positionAbs||t.position.absolute).top+t.margins.top,r=a+t.helperProportions.width,h=o+t.helperProportions.height,l=i.offset.left,u=i.offset.top,d=l+i.proportions().width,c=u+i.proportions().height;switch(s){case"fit":return a>=l&&d>=r&&o>=u&&c>=h;case"intersect":return a+t.helperProportions.width/2>l&&d>r-t.helperProportions.width/2&&o+t.helperProportions.height/2>u&&c>h-t.helperProportions.height/2;case"pointer":return e(n.pageY,u,i.proportions().height)&&e(n.pageX,l,i.proportions().width);case"touch":return(o>=u&&c>=o||h>=u&&c>=h||u>o&&h>c)&&(a>=l&&d>=a||r>=l&&d>=r||l>a&&r>d);default:return!1}}}(),e.ui.ddmanager={current:null,droppables:{"default":[]},prepareOffsets:function(t,i){var s,n,a=e.ui.ddmanager.droppables[t.options.scope]||[],o=i?i.type:null,r=(t.currentItem||t.element).find(":data(ui-droppable)").addBack();e:for(s=0;a.length>s;s++)if(!(a[s].options.disabled||t&&!a[s].accept.call(a[s].element[0],t.currentItem||t.element))){for(n=0;r.length>n;n++)if(r[n]===a[s].element[0]){a[s].proportions().height=0;continue e}a[s].visible="none"!==a[s].element.css("display"),a[s].visible&&("mousedown"===o&&a[s]._activate.call(a[s],i),a[s].offset=a[s].element.offset(),a[s].proportions({width:a[s].element[0].offsetWidth,height:a[s].element[0].offsetHeight}))}},drop:function(t,i){var s=!1;return e.each((e.ui.ddmanager.droppables[t.options.scope]||[]).slice(),function(){this.options&&(!this.options.disabled&&this.visible&&e.ui.intersect(t,this,this.options.tolerance,i)&&(s=this._drop.call(this,i)||s),!this.options.disabled&&this.visible&&this.accept.call(this.element[0],t.currentItem||t.element)&&(this.isout=!0,this.isover=!1,this._deactivate.call(this,i)))}),s},dragStart:function(t,i){t.element.parentsUntil("body").bind("scroll.droppable",function(){t.options.refreshPositions||e.ui.ddmanager.prepareOffsets(t,i)})},drag:function(t,i){t.options.refreshPositions&&e.ui.ddmanager.prepareOffsets(t,i),e.each(e.ui.ddmanager.droppables[t.options.scope]||[],function(){if(!this.options.disabled&&!this.greedyChild&&this.visible){var s,n,a,o=e.ui.intersect(t,this,this.options.tolerance,i),r=!o&&this.isover?"isout":o&&!this.isover?"isover":null;r&&(this.options.greedy&&(n=this.options.scope,a=this.element.parents(":data(ui-droppable)").filter(function(){return e(this).droppable("instance").options.scope===n}),a.length&&(s=e(a[0]).droppable("instance"),s.greedyChild="isover"===r)),s&&"isover"===r&&(s.isover=!1,s.isout=!0,s._out.call(s,i)),this[r]=!0,this["isout"===r?"isover":"isout"]=!1,this["isover"===r?"_over":"_out"].call(this,i),s&&"isout"===r&&(s.isout=!1,s.isover=!0,s._over.call(s,i)))}})},dragStop:function(t,i){t.element.parentsUntil("body").unbind("scroll.droppable"),t.options.refreshPositions||e.ui.ddmanager.prepareOffsets(t,i)}},e.ui.droppable,e.widget("ui.resizable",e.ui.mouse,{version:"1.11.4",widgetEventPrefix:"resize",options:{alsoResize:!1,animate:!1,animateDuration:"slow",animateEasing:"swing",aspectRatio:!1,autoHide:!1,containment:!1,ghost:!1,grid:!1,handles:"e,s,se",helper:!1,maxHeight:null,maxWidth:null,minHeight:10,minWidth:10,zIndex:90,resize:null,start:null,stop:null},_num:function(e){return parseInt(e,10)||0},_isNumber:function(e){return!isNaN(parseInt(e,10))},_hasScroll:function(t,i){if("hidden"===e(t).css("overflow"))return!1;var s=i&&"left"===i?"scrollLeft":"scrollTop",n=!1;return t[s]>0?!0:(t[s]=1,n=t[s]>0,t[s]=0,n)},_create:function(){var t,i,s,n,a,o=this,r=this.options;if(this.element.addClass("ui-resizable"),e.extend(this,{_aspectRatio:!!r.aspectRatio,aspectRatio:r.aspectRatio,originalElement:this.element,_proportionallyResizeElements:[],_helper:r.helper||r.ghost||r.animate?r.helper||"ui-resizable-helper":null}),this.element[0].nodeName.match(/^(canvas|textarea|input|select|button|img)$/i)&&(this.element.wrap(e("<div class='ui-wrapper' style='overflow: hidden;'></div>").css({position:this.element.css("position"),width:this.element.outerWidth(),height:this.element.outerHeight(),top:this.element.css("top"),left:this.element.css("left")})),this.element=this.element.parent().data("ui-resizable",this.element.resizable("instance")),this.elementIsWrapper=!0,this.element.css({marginLeft:this.originalElement.css("marginLeft"),marginTop:this.originalElement.css("marginTop"),marginRight:this.originalElement.css("marginRight"),marginBottom:this.originalElement.css("marginBottom")}),this.originalElement.css({marginLeft:0,marginTop:0,marginRight:0,marginBottom:0}),this.originalResizeStyle=this.originalElement.css("resize"),this.originalElement.css("resize","none"),this._proportionallyResizeElements.push(this.originalElement.css({position:"static",zoom:1,display:"block"})),this.originalElement.css({margin:this.originalElement.css("margin")}),this._proportionallyResize()),this.handles=r.handles||(e(".ui-resizable-handle",this.element).length?{n:".ui-resizable-n",e:".ui-resizable-e",s:".ui-resizable-s",w:".ui-resizable-w",se:".ui-resizable-se",sw:".ui-resizable-sw",ne:".ui-resizable-ne",nw:".ui-resizable-nw"}:"e,s,se"),this._handles=e(),this.handles.constructor===String)for("all"===this.handles&&(this.handles="n,e,s,w,se,sw,ne,nw"),t=this.handles.split(","),this.handles={},i=0;t.length>i;i++)s=e.trim(t[i]),a="ui-resizable-"+s,n=e("<div class='ui-resizable-handle "+a+"'></div>"),n.css({zIndex:r.zIndex}),"se"===s&&n.addClass("ui-icon ui-icon-gripsmall-diagonal-se"),this.handles[s]=".ui-resizable-"+s,this.element.append(n);this._renderAxis=function(t){var i,s,n,a;t=t||this.element;for(i in this.handles)this.handles[i].constructor===String?this.handles[i]=this.element.children(this.handles[i]).first().show():(this.handles[i].jquery||this.handles[i].nodeType)&&(this.handles[i]=e(this.handles[i]),this._on(this.handles[i],{mousedown:o._mouseDown})),this.elementIsWrapper&&this.originalElement[0].nodeName.match(/^(textarea|input|select|button)$/i)&&(s=e(this.handles[i],this.element),a=/sw|ne|nw|se|n|s/.test(i)?s.outerHeight():s.outerWidth(),n=["padding",/ne|nw|n/.test(i)?"Top":/se|sw|s/.test(i)?"Bottom":/^e$/.test(i)?"Right":"Left"].join(""),t.css(n,a),this._proportionallyResize()),this._handles=this._handles.add(this.handles[i])},this._renderAxis(this.element),this._handles=this._handles.add(this.element.find(".ui-resizable-handle")),this._handles.disableSelection(),this._handles.mouseover(function(){o.resizing||(this.className&&(n=this.className.match(/ui-resizable-(se|sw|ne|nw|n|e|s|w)/i)),o.axis=n&&n[1]?n[1]:"se")}),r.autoHide&&(this._handles.hide(),e(this.element).addClass("ui-resizable-autohide").mouseenter(function(){r.disabled||(e(this).removeClass("ui-resizable-autohide"),o._handles.show())}).mouseleave(function(){r.disabled||o.resizing||(e(this).addClass("ui-resizable-autohide"),o._handles.hide())})),this._mouseInit()},_destroy:function(){this._mouseDestroy();var t,i=function(t){e(t).removeClass("ui-resizable ui-resizable-disabled ui-resizable-resizing").removeData("resizable").removeData("ui-resizable").unbind(".resizable").find(".ui-resizable-handle").remove()};return this.elementIsWrapper&&(i(this.element),t=this.element,this.originalElement.css({position:t.css("position"),width:t.outerWidth(),height:t.outerHeight(),top:t.css("top"),left:t.css("left")}).insertAfter(t),t.remove()),this.originalElement.css("resize",this.originalResizeStyle),i(this.originalElement),this},_mouseCapture:function(t){var i,s,n=!1;for(i in this.handles)s=e(this.handles[i])[0],(s===t.target||e.contains(s,t.target))&&(n=!0);return!this.options.disabled&&n},_mouseStart:function(t){var i,s,n,a=this.options,o=this.element;return this.resizing=!0,this._renderProxy(),i=this._num(this.helper.css("left")),s=this._num(this.helper.css("top")),a.containment&&(i+=e(a.containment).scrollLeft()||0,s+=e(a.containment).scrollTop()||0),this.offset=this.helper.offset(),this.position={left:i,top:s},this.size=this._helper?{width:this.helper.width(),height:this.helper.height()}:{width:o.width(),height:o.height()},this.originalSize=this._helper?{width:o.outerWidth(),height:o.outerHeight()}:{width:o.width(),height:o.height()},this.sizeDiff={width:o.outerWidth()-o.width(),height:o.outerHeight()-o.height()},this.originalPosition={left:i,top:s},this.originalMousePosition={left:t.pageX,top:t.pageY},this.aspectRatio="number"==typeof a.aspectRatio?a.aspectRatio:this.originalSize.width/this.originalSize.height||1,n=e(".ui-resizable-"+this.axis).css("cursor"),e("body").css("cursor","auto"===n?this.axis+"-resize":n),o.addClass("ui-resizable-resizing"),this._propagate("start",t),!0},_mouseDrag:function(t){var i,s,n=this.originalMousePosition,a=this.axis,o=t.pageX-n.left||0,r=t.pageY-n.top||0,h=this._change[a];return this._updatePrevProperties(),h?(i=h.apply(this,[t,o,r]),this._updateVirtualBoundaries(t.shiftKey),(this._aspectRatio||t.shiftKey)&&(i=this._updateRatio(i,t)),i=this._respectSize(i,t),this._updateCache(i),this._propagate("resize",t),s=this._applyChanges(),!this._helper&&this._proportionallyResizeElements.length&&this._proportionallyResize(),e.isEmptyObject(s)||(this._updatePrevProperties(),this._trigger("resize",t,this.ui()),this._applyChanges()),!1):!1},_mouseStop:function(t){this.resizing=!1;var i,s,n,a,o,r,h,l=this.options,u=this;return this._helper&&(i=this._proportionallyResizeElements,s=i.length&&/textarea/i.test(i[0].nodeName),n=s&&this._hasScroll(i[0],"left")?0:u.sizeDiff.height,a=s?0:u.sizeDiff.width,o={width:u.helper.width()-a,height:u.helper.height()-n},r=parseInt(u.element.css("left"),10)+(u.position.left-u.originalPosition.left)||null,h=parseInt(u.element.css("top"),10)+(u.position.top-u.originalPosition.top)||null,l.animate||this.element.css(e.extend(o,{top:h,left:r})),u.helper.height(u.size.height),u.helper.width(u.size.width),this._helper&&!l.animate&&this._proportionallyResize()),e("body").css("cursor","auto"),this.element.removeClass("ui-resizable-resizing"),this._propagate("stop",t),this._helper&&this.helper.remove(),!1},_updatePrevProperties:function(){this.prevPosition={top:this.position.top,left:this.position.left},this.prevSize={width:this.size.width,height:this.size.height}},_applyChanges:function(){var e={};return this.position.top!==this.prevPosition.top&&(e.top=this.position.top+"px"),this.position.left!==this.prevPosition.left&&(e.left=this.position.left+"px"),this.size.width!==this.prevSize.width&&(e.width=this.size.width+"px"),this.size.height!==this.prevSize.height&&(e.height=this.size.height+"px"),this.helper.css(e),e},_updateVirtualBoundaries:function(e){var t,i,s,n,a,o=this.options;a={minWidth:this._isNumber(o.minWidth)?o.minWidth:0,maxWidth:this._isNumber(o.maxWidth)?o.maxWidth:1/0,minHeight:this._isNumber(o.minHeight)?o.minHeight:0,maxHeight:this._isNumber(o.maxHeight)?o.maxHeight:1/0},(this._aspectRatio||e)&&(t=a.minHeight*this.aspectRatio,s=a.minWidth/this.aspectRatio,i=a.maxHeight*this.aspectRatio,n=a.maxWidth/this.aspectRatio,t>a.minWidth&&(a.minWidth=t),s>a.minHeight&&(a.minHeight=s),a.maxWidth>i&&(a.maxWidth=i),a.maxHeight>n&&(a.maxHeight=n)),this._vBoundaries=a},_updateCache:function(e){this.offset=this.helper.offset(),this._isNumber(e.left)&&(this.position.left=e.left),this._isNumber(e.top)&&(this.position.top=e.top),this._isNumber(e.height)&&(this.size.height=e.height),this._isNumber(e.width)&&(this.size.width=e.width)},_updateRatio:function(e){var t=this.position,i=this.size,s=this.axis;return this._isNumber(e.height)?e.width=e.height*this.aspectRatio:this._isNumber(e.width)&&(e.height=e.width/this.aspectRatio),"sw"===s&&(e.left=t.left+(i.width-e.width),e.top=null),"nw"===s&&(e.top=t.top+(i.height-e.height),e.left=t.left+(i.width-e.width)),e},_respectSize:function(e){var t=this._vBoundaries,i=this.axis,s=this._isNumber(e.width)&&t.maxWidth&&t.maxWidth<e.width,n=this._isNumber(e.height)&&t.maxHeight&&t.maxHeight<e.height,a=this._isNumber(e.width)&&t.minWidth&&t.minWidth>e.width,o=this._isNumber(e.height)&&t.minHeight&&t.minHeight>e.height,r=this.originalPosition.left+this.originalSize.width,h=this.position.top+this.size.height,l=/sw|nw|w/.test(i),u=/nw|ne|n/.test(i);return a&&(e.width=t.minWidth),o&&(e.height=t.minHeight),s&&(e.width=t.maxWidth),n&&(e.height=t.maxHeight),a&&l&&(e.left=r-t.minWidth),s&&l&&(e.left=r-t.maxWidth),o&&u&&(e.top=h-t.minHeight),n&&u&&(e.top=h-t.maxHeight),e.width||e.height||e.left||!e.top?e.width||e.height||e.top||!e.left||(e.left=null):e.top=null,e},_getPaddingPlusBorderDimensions:function(e){for(var t=0,i=[],s=[e.css("borderTopWidth"),e.css("borderRightWidth"),e.css("borderBottomWidth"),e.css("borderLeftWidth")],n=[e.css("paddingTop"),e.css("paddingRight"),e.css("paddingBottom"),e.css("paddingLeft")];4>t;t++)i[t]=parseInt(s[t],10)||0,i[t]+=parseInt(n[t],10)||0;return{height:i[0]+i[2],width:i[1]+i[3]}},_proportionallyResize:function(){if(this._proportionallyResizeElements.length)for(var e,t=0,i=this.helper||this.element;this._proportionallyResizeElements.length>t;t++)e=this._proportionallyResizeElements[t],this.outerDimensions||(this.outerDimensions=this._getPaddingPlusBorderDimensions(e)),e.css({height:i.height()-this.outerDimensions.height||0,width:i.width()-this.outerDimensions.width||0})},_renderProxy:function(){var t=this.element,i=this.options;this.elementOffset=t.offset(),this._helper?(this.helper=this.helper||e("<div style='overflow:hidden;'></div>"),this.helper.addClass(this._helper).css({width:this.element.outerWidth()-1,height:this.element.outerHeight()-1,position:"absolute",left:this.elementOffset.left+"px",top:this.elementOffset.top+"px",zIndex:++i.zIndex}),this.helper.appendTo("body").disableSelection()):this.helper=this.element},_change:{e:function(e,t){return{width:this.originalSize.width+t}},w:function(e,t){var i=this.originalSize,s=this.originalPosition;return{left:s.left+t,width:i.width-t}},n:function(e,t,i){var s=this.originalSize,n=this.originalPosition;return{top:n.top+i,height:s.height-i}},s:function(e,t,i){return{height:this.originalSize.height+i}},se:function(t,i,s){return e.extend(this._change.s.apply(this,arguments),this._change.e.apply(this,[t,i,s]))},sw:function(t,i,s){return e.extend(this._change.s.apply(this,arguments),this._change.w.apply(this,[t,i,s]))},ne:function(t,i,s){return e.extend(this._change.n.apply(this,arguments),this._change.e.apply(this,[t,i,s]))},nw:function(t,i,s){return e.extend(this._change.n.apply(this,arguments),this._change.w.apply(this,[t,i,s]))}},_propagate:function(t,i){e.ui.plugin.call(this,t,[i,this.ui()]),"resize"!==t&&this._trigger(t,i,this.ui())},plugins:{},ui:function(){return{originalElement:this.originalElement,element:this.element,helper:this.helper,position:this.position,size:this.size,originalSize:this.originalSize,originalPosition:this.originalPosition}}}),e.ui.plugin.add("resizable","animate",{stop:function(t){var i=e(this).resizable("instance"),s=i.options,n=i._proportionallyResizeElements,a=n.length&&/textarea/i.test(n[0].nodeName),o=a&&i._hasScroll(n[0],"left")?0:i.sizeDiff.height,r=a?0:i.sizeDiff.width,h={width:i.size.width-r,height:i.size.height-o},l=parseInt(i.element.css("left"),10)+(i.position.left-i.originalPosition.left)||null,u=parseInt(i.element.css("top"),10)+(i.position.top-i.originalPosition.top)||null;i.element.animate(e.extend(h,u&&l?{top:u,left:l}:{}),{duration:s.animateDuration,easing:s.animateEasing,step:function(){var s={width:parseInt(i.element.css("width"),10),height:parseInt(i.element.css("height"),10),top:parseInt(i.element.css("top"),10),left:parseInt(i.element.css("left"),10)};n&&n.length&&e(n[0]).css({width:s.width,height:s.height}),i._updateCache(s),i._propagate("resize",t)}})}}),e.ui.plugin.add("resizable","containment",{start:function(){var t,i,s,n,a,o,r,h=e(this).resizable("instance"),l=h.options,u=h.element,d=l.containment,c=d instanceof e?d.get(0):/parent/.test(d)?u.parent().get(0):d;c&&(h.containerElement=e(c),/document/.test(d)||d===document?(h.containerOffset={left:0,top:0},h.containerPosition={left:0,top:0},h.parentData={element:e(document),left:0,top:0,width:e(document).width(),height:e(document).height()||document.body.parentNode.scrollHeight}):(t=e(c),i=[],e(["Top","Right","Left","Bottom"]).each(function(e,s){i[e]=h._num(t.css("padding"+s))}),h.containerOffset=t.offset(),h.containerPosition=t.position(),h.containerSize={height:t.innerHeight()-i[3],width:t.innerWidth()-i[1]},s=h.containerOffset,n=h.containerSize.height,a=h.containerSize.width,o=h._hasScroll(c,"left")?c.scrollWidth:a,r=h._hasScroll(c)?c.scrollHeight:n,h.parentData={element:c,left:s.left,top:s.top,width:o,height:r}))},resize:function(t){var i,s,n,a,o=e(this).resizable("instance"),r=o.options,h=o.containerOffset,l=o.position,u=o._aspectRatio||t.shiftKey,d={top:0,left:0},c=o.containerElement,p=!0;c[0]!==document&&/static/.test(c.css("position"))&&(d=h),l.left<(o._helper?h.left:0)&&(o.size.width=o.size.width+(o._helper?o.position.left-h.left:o.position.left-d.left),u&&(o.size.height=o.size.width/o.aspectRatio,p=!1),o.position.left=r.helper?h.left:0),l.top<(o._helper?h.top:0)&&(o.size.height=o.size.height+(o._helper?o.position.top-h.top:o.position.top),u&&(o.size.width=o.size.height*o.aspectRatio,p=!1),o.position.top=o._helper?h.top:0),n=o.containerElement.get(0)===o.element.parent().get(0),a=/relative|absolute/.test(o.containerElement.css("position")),n&&a?(o.offset.left=o.parentData.left+o.position.left,o.offset.top=o.parentData.top+o.position.top):(o.offset.left=o.element.offset().left,o.offset.top=o.element.offset().top),i=Math.abs(o.sizeDiff.width+(o._helper?o.offset.left-d.left:o.offset.left-h.left)),s=Math.abs(o.sizeDiff.height+(o._helper?o.offset.top-d.top:o.offset.top-h.top)),i+o.size.width>=o.parentData.width&&(o.size.width=o.parentData.width-i,u&&(o.size.height=o.size.width/o.aspectRatio,p=!1)),s+o.size.height>=o.parentData.height&&(o.size.height=o.parentData.height-s,u&&(o.size.width=o.size.height*o.aspectRatio,p=!1)),p||(o.position.left=o.prevPosition.left,o.position.top=o.prevPosition.top,o.size.width=o.prevSize.width,o.size.height=o.prevSize.height)},stop:function(){var t=e(this).resizable("instance"),i=t.options,s=t.containerOffset,n=t.containerPosition,a=t.containerElement,o=e(t.helper),r=o.offset(),h=o.outerWidth()-t.sizeDiff.width,l=o.outerHeight()-t.sizeDiff.height;t._helper&&!i.animate&&/relative/.test(a.css("position"))&&e(this).css({left:r.left-n.left-s.left,width:h,height:l}),t._helper&&!i.animate&&/static/.test(a.css("position"))&&e(this).css({left:r.left-n.left-s.left,width:h,height:l})}}),e.ui.plugin.add("resizable","alsoResize",{start:function(){var t=e(this).resizable("instance"),i=t.options;e(i.alsoResize).each(function(){var t=e(this);t.data("ui-resizable-alsoresize",{width:parseInt(t.width(),10),height:parseInt(t.height(),10),left:parseInt(t.css("left"),10),top:parseInt(t.css("top"),10)})})},resize:function(t,i){var s=e(this).resizable("instance"),n=s.options,a=s.originalSize,o=s.originalPosition,r={height:s.size.height-a.height||0,width:s.size.width-a.width||0,top:s.position.top-o.top||0,left:s.position.left-o.left||0};e(n.alsoResize).each(function(){var t=e(this),s=e(this).data("ui-resizable-alsoresize"),n={},a=t.parents(i.originalElement[0]).length?["width","height"]:["width","height","top","left"];e.each(a,function(e,t){var i=(s[t]||0)+(r[t]||0);i&&i>=0&&(n[t]=i||null)}),t.css(n)})},stop:function(){e(this).removeData("resizable-alsoresize")}}),e.ui.plugin.add("resizable","ghost",{start:function(){var t=e(this).resizable("instance"),i=t.options,s=t.size;t.ghost=t.originalElement.clone(),t.ghost.css({opacity:.25,display:"block",position:"relative",height:s.height,width:s.width,margin:0,left:0,top:0}).addClass("ui-resizable-ghost").addClass("string"==typeof i.ghost?i.ghost:""),t.ghost.appendTo(t.helper)},resize:function(){var t=e(this).resizable("instance");t.ghost&&t.ghost.css({position:"relative",height:t.size.height,width:t.size.width})},stop:function(){var t=e(this).resizable("instance");t.ghost&&t.helper&&t.helper.get(0).removeChild(t.ghost.get(0))}}),e.ui.plugin.add("resizable","grid",{resize:function(){var t,i=e(this).resizable("instance"),s=i.options,n=i.size,a=i.originalSize,o=i.originalPosition,r=i.axis,h="number"==typeof s.grid?[s.grid,s.grid]:s.grid,l=h[0]||1,u=h[1]||1,d=Math.round((n.width-a.width)/l)*l,c=Math.round((n.height-a.height)/u)*u,p=a.width+d,f=a.height+c,m=s.maxWidth&&p>s.maxWidth,g=s.maxHeight&&f>s.maxHeight,v=s.minWidth&&s.minWidth>p,y=s.minHeight&&s.minHeight>f;s.grid=h,v&&(p+=l),y&&(f+=u),m&&(p-=l),g&&(f-=u),/^(se|s|e)$/.test(r)?(i.size.width=p,i.size.height=f):/^(ne)$/.test(r)?(i.size.width=p,i.size.height=f,i.position.top=o.top-c):/^(sw)$/.test(r)?(i.size.width=p,i.size.height=f,i.position.left=o.left-d):((0>=f-u||0>=p-l)&&(t=i._getPaddingPlusBorderDimensions(this)),f-u>0?(i.size.height=f,i.position.top=o.top-c):(f=u-t.height,i.size.height=f,i.position.top=o.top+a.height-f),p-l>0?(i.size.width=p,i.position.left=o.left-d):(p=l-t.width,i.size.width=p,i.position.left=o.left+a.width-p))}}),e.ui.resizable,e.widget("ui.selectable",e.ui.mouse,{version:"1.11.4",options:{appendTo:"body",autoRefresh:!0,distance:0,filter:"*",tolerance:"touch",selected:null,selecting:null,start:null,stop:null,unselected:null,unselecting:null},_create:function(){var t,i=this;this.element.addClass("ui-selectable"),this.dragged=!1,this.refresh=function(){t=e(i.options.filter,i.element[0]),t.addClass("ui-selectee"),t.each(function(){var t=e(this),i=t.offset();e.data(this,"selectable-item",{element:this,$element:t,left:i.left,top:i.top,right:i.left+t.outerWidth(),bottom:i.top+t.outerHeight(),startselected:!1,selected:t.hasClass("ui-selected"),selecting:t.hasClass("ui-selecting"),unselecting:t.hasClass("ui-unselecting")})})},this.refresh(),this.selectees=t.addClass("ui-selectee"),this._mouseInit(),this.helper=e("<div class='ui-selectable-helper'></div>")},_destroy:function(){this.selectees.removeClass("ui-selectee").removeData("selectable-item"),this.element.removeClass("ui-selectable ui-selectable-disabled"),this._mouseDestroy()},_mouseStart:function(t){var i=this,s=this.options;this.opos=[t.pageX,t.pageY],this.options.disabled||(this.selectees=e(s.filter,this.element[0]),this._trigger("start",t),e(s.appendTo).append(this.helper),this.helper.css({left:t.pageX,top:t.pageY,width:0,height:0}),s.autoRefresh&&this.refresh(),this.selectees.filter(".ui-selected").each(function(){var s=e.data(this,"selectable-item");s.startselected=!0,t.metaKey||t.ctrlKey||(s.$element.removeClass("ui-selected"),s.selected=!1,s.$element.addClass("ui-unselecting"),s.unselecting=!0,i._trigger("unselecting",t,{unselecting:s.element}))}),e(t.target).parents().addBack().each(function(){var s,n=e.data(this,"selectable-item");return n?(s=!t.metaKey&&!t.ctrlKey||!n.$element.hasClass("ui-selected"),n.$element.removeClass(s?"ui-unselecting":"ui-selected").addClass(s?"ui-selecting":"ui-unselecting"),n.unselecting=!s,n.selecting=s,n.selected=s,s?i._trigger("selecting",t,{selecting:n.element}):i._trigger("unselecting",t,{unselecting:n.element}),!1):void 0}))},_mouseDrag:function(t){if(this.dragged=!0,!this.options.disabled){var i,s=this,n=this.options,a=this.opos[0],o=this.opos[1],r=t.pageX,h=t.pageY;return a>r&&(i=r,r=a,a=i),o>h&&(i=h,h=o,o=i),this.helper.css({left:a,top:o,width:r-a,height:h-o}),this.selectees.each(function(){var i=e.data(this,"selectable-item"),l=!1;
i&&i.element!==s.element[0]&&("touch"===n.tolerance?l=!(i.left>r||a>i.right||i.top>h||o>i.bottom):"fit"===n.tolerance&&(l=i.left>a&&r>i.right&&i.top>o&&h>i.bottom),l?(i.selected&&(i.$element.removeClass("ui-selected"),i.selected=!1),i.unselecting&&(i.$element.removeClass("ui-unselecting"),i.unselecting=!1),i.selecting||(i.$element.addClass("ui-selecting"),i.selecting=!0,s._trigger("selecting",t,{selecting:i.element}))):(i.selecting&&((t.metaKey||t.ctrlKey)&&i.startselected?(i.$element.removeClass("ui-selecting"),i.selecting=!1,i.$element.addClass("ui-selected"),i.selected=!0):(i.$element.removeClass("ui-selecting"),i.selecting=!1,i.startselected&&(i.$element.addClass("ui-unselecting"),i.unselecting=!0),s._trigger("unselecting",t,{unselecting:i.element}))),i.selected&&(t.metaKey||t.ctrlKey||i.startselected||(i.$element.removeClass("ui-selected"),i.selected=!1,i.$element.addClass("ui-unselecting"),i.unselecting=!0,s._trigger("unselecting",t,{unselecting:i.element})))))}),!1}},_mouseStop:function(t){var i=this;return this.dragged=!1,e(".ui-unselecting",this.element[0]).each(function(){var s=e.data(this,"selectable-item");s.$element.removeClass("ui-unselecting"),s.unselecting=!1,s.startselected=!1,i._trigger("unselected",t,{unselected:s.element})}),e(".ui-selecting",this.element[0]).each(function(){var s=e.data(this,"selectable-item");s.$element.removeClass("ui-selecting").addClass("ui-selected"),s.selecting=!1,s.selected=!0,s.startselected=!0,i._trigger("selected",t,{selected:s.element})}),this._trigger("stop",t),this.helper.remove(),!1}}),e.widget("ui.sortable",e.ui.mouse,{version:"1.11.4",widgetEventPrefix:"sort",ready:!1,options:{appendTo:"parent",axis:!1,connectWith:!1,containment:!1,cursor:"auto",cursorAt:!1,dropOnEmpty:!0,forcePlaceholderSize:!1,forceHelperSize:!1,grid:!1,handle:!1,helper:"original",items:"> *",opacity:!1,placeholder:!1,revert:!1,scroll:!0,scrollSensitivity:20,scrollSpeed:20,scope:"default",tolerance:"intersect",zIndex:1e3,activate:null,beforeStop:null,change:null,deactivate:null,out:null,over:null,receive:null,remove:null,sort:null,start:null,stop:null,update:null},_isOverAxis:function(e,t,i){return e>=t&&t+i>e},_isFloating:function(e){return/left|right/.test(e.css("float"))||/inline|table-cell/.test(e.css("display"))},_create:function(){this.containerCache={},this.element.addClass("ui-sortable"),this.refresh(),this.offset=this.element.offset(),this._mouseInit(),this._setHandleClassName(),this.ready=!0},_setOption:function(e,t){this._super(e,t),"handle"===e&&this._setHandleClassName()},_setHandleClassName:function(){this.element.find(".ui-sortable-handle").removeClass("ui-sortable-handle"),e.each(this.items,function(){(this.instance.options.handle?this.item.find(this.instance.options.handle):this.item).addClass("ui-sortable-handle")})},_destroy:function(){this.element.removeClass("ui-sortable ui-sortable-disabled").find(".ui-sortable-handle").removeClass("ui-sortable-handle"),this._mouseDestroy();for(var e=this.items.length-1;e>=0;e--)this.items[e].item.removeData(this.widgetName+"-item");return this},_mouseCapture:function(t,i){var s=null,n=!1,a=this;return this.reverting?!1:this.options.disabled||"static"===this.options.type?!1:(this._refreshItems(t),e(t.target).parents().each(function(){return e.data(this,a.widgetName+"-item")===a?(s=e(this),!1):void 0}),e.data(t.target,a.widgetName+"-item")===a&&(s=e(t.target)),s?!this.options.handle||i||(e(this.options.handle,s).find("*").addBack().each(function(){this===t.target&&(n=!0)}),n)?(this.currentItem=s,this._removeCurrentsFromItems(),!0):!1:!1)},_mouseStart:function(t,i,s){var n,a,o=this.options;if(this.currentContainer=this,this.refreshPositions(),this.helper=this._createHelper(t),this._cacheHelperProportions(),this._cacheMargins(),this.scrollParent=this.helper.scrollParent(),this.offset=this.currentItem.offset(),this.offset={top:this.offset.top-this.margins.top,left:this.offset.left-this.margins.left},e.extend(this.offset,{click:{left:t.pageX-this.offset.left,top:t.pageY-this.offset.top},parent:this._getParentOffset(),relative:this._getRelativeOffset()}),this.helper.css("position","absolute"),this.cssPosition=this.helper.css("position"),this.originalPosition=this._generatePosition(t),this.originalPageX=t.pageX,this.originalPageY=t.pageY,o.cursorAt&&this._adjustOffsetFromHelper(o.cursorAt),this.domPosition={prev:this.currentItem.prev()[0],parent:this.currentItem.parent()[0]},this.helper[0]!==this.currentItem[0]&&this.currentItem.hide(),this._createPlaceholder(),o.containment&&this._setContainment(),o.cursor&&"auto"!==o.cursor&&(a=this.document.find("body"),this.storedCursor=a.css("cursor"),a.css("cursor",o.cursor),this.storedStylesheet=e("<style>*{ cursor: "+o.cursor+" !important; }</style>").appendTo(a)),o.opacity&&(this.helper.css("opacity")&&(this._storedOpacity=this.helper.css("opacity")),this.helper.css("opacity",o.opacity)),o.zIndex&&(this.helper.css("zIndex")&&(this._storedZIndex=this.helper.css("zIndex")),this.helper.css("zIndex",o.zIndex)),this.scrollParent[0]!==this.document[0]&&"HTML"!==this.scrollParent[0].tagName&&(this.overflowOffset=this.scrollParent.offset()),this._trigger("start",t,this._uiHash()),this._preserveHelperProportions||this._cacheHelperProportions(),!s)for(n=this.containers.length-1;n>=0;n--)this.containers[n]._trigger("activate",t,this._uiHash(this));return e.ui.ddmanager&&(e.ui.ddmanager.current=this),e.ui.ddmanager&&!o.dropBehaviour&&e.ui.ddmanager.prepareOffsets(this,t),this.dragging=!0,this.helper.addClass("ui-sortable-helper"),this._mouseDrag(t),!0},_mouseDrag:function(t){var i,s,n,a,o=this.options,r=!1;for(this.position=this._generatePosition(t),this.positionAbs=this._convertPositionTo("absolute"),this.lastPositionAbs||(this.lastPositionAbs=this.positionAbs),this.options.scroll&&(this.scrollParent[0]!==this.document[0]&&"HTML"!==this.scrollParent[0].tagName?(this.overflowOffset.top+this.scrollParent[0].offsetHeight-t.pageY<o.scrollSensitivity?this.scrollParent[0].scrollTop=r=this.scrollParent[0].scrollTop+o.scrollSpeed:t.pageY-this.overflowOffset.top<o.scrollSensitivity&&(this.scrollParent[0].scrollTop=r=this.scrollParent[0].scrollTop-o.scrollSpeed),this.overflowOffset.left+this.scrollParent[0].offsetWidth-t.pageX<o.scrollSensitivity?this.scrollParent[0].scrollLeft=r=this.scrollParent[0].scrollLeft+o.scrollSpeed:t.pageX-this.overflowOffset.left<o.scrollSensitivity&&(this.scrollParent[0].scrollLeft=r=this.scrollParent[0].scrollLeft-o.scrollSpeed)):(t.pageY-this.document.scrollTop()<o.scrollSensitivity?r=this.document.scrollTop(this.document.scrollTop()-o.scrollSpeed):this.window.height()-(t.pageY-this.document.scrollTop())<o.scrollSensitivity&&(r=this.document.scrollTop(this.document.scrollTop()+o.scrollSpeed)),t.pageX-this.document.scrollLeft()<o.scrollSensitivity?r=this.document.scrollLeft(this.document.scrollLeft()-o.scrollSpeed):this.window.width()-(t.pageX-this.document.scrollLeft())<o.scrollSensitivity&&(r=this.document.scrollLeft(this.document.scrollLeft()+o.scrollSpeed))),r!==!1&&e.ui.ddmanager&&!o.dropBehaviour&&e.ui.ddmanager.prepareOffsets(this,t)),this.positionAbs=this._convertPositionTo("absolute"),this.options.axis&&"y"===this.options.axis||(this.helper[0].style.left=this.position.left+"px"),this.options.axis&&"x"===this.options.axis||(this.helper[0].style.top=this.position.top+"px"),i=this.items.length-1;i>=0;i--)if(s=this.items[i],n=s.item[0],a=this._intersectsWithPointer(s),a&&s.instance===this.currentContainer&&n!==this.currentItem[0]&&this.placeholder[1===a?"next":"prev"]()[0]!==n&&!e.contains(this.placeholder[0],n)&&("semi-dynamic"===this.options.type?!e.contains(this.element[0],n):!0)){if(this.direction=1===a?"down":"up","pointer"!==this.options.tolerance&&!this._intersectsWithSides(s))break;this._rearrange(t,s),this._trigger("change",t,this._uiHash());break}return this._contactContainers(t),e.ui.ddmanager&&e.ui.ddmanager.drag(this,t),this._trigger("sort",t,this._uiHash()),this.lastPositionAbs=this.positionAbs,!1},_mouseStop:function(t,i){if(t){if(e.ui.ddmanager&&!this.options.dropBehaviour&&e.ui.ddmanager.drop(this,t),this.options.revert){var s=this,n=this.placeholder.offset(),a=this.options.axis,o={};a&&"x"!==a||(o.left=n.left-this.offset.parent.left-this.margins.left+(this.offsetParent[0]===this.document[0].body?0:this.offsetParent[0].scrollLeft)),a&&"y"!==a||(o.top=n.top-this.offset.parent.top-this.margins.top+(this.offsetParent[0]===this.document[0].body?0:this.offsetParent[0].scrollTop)),this.reverting=!0,e(this.helper).animate(o,parseInt(this.options.revert,10)||500,function(){s._clear(t)})}else this._clear(t,i);return!1}},cancel:function(){if(this.dragging){this._mouseUp({target:null}),"original"===this.options.helper?this.currentItem.css(this._storedCSS).removeClass("ui-sortable-helper"):this.currentItem.show();for(var t=this.containers.length-1;t>=0;t--)this.containers[t]._trigger("deactivate",null,this._uiHash(this)),this.containers[t].containerCache.over&&(this.containers[t]._trigger("out",null,this._uiHash(this)),this.containers[t].containerCache.over=0)}return this.placeholder&&(this.placeholder[0].parentNode&&this.placeholder[0].parentNode.removeChild(this.placeholder[0]),"original"!==this.options.helper&&this.helper&&this.helper[0].parentNode&&this.helper.remove(),e.extend(this,{helper:null,dragging:!1,reverting:!1,_noFinalSort:null}),this.domPosition.prev?e(this.domPosition.prev).after(this.currentItem):e(this.domPosition.parent).prepend(this.currentItem)),this},serialize:function(t){var i=this._getItemsAsjQuery(t&&t.connected),s=[];return t=t||{},e(i).each(function(){var i=(e(t.item||this).attr(t.attribute||"id")||"").match(t.expression||/(.+)[\-=_](.+)/);i&&s.push((t.key||i[1]+"[]")+"="+(t.key&&t.expression?i[1]:i[2]))}),!s.length&&t.key&&s.push(t.key+"="),s.join("&")},toArray:function(t){var i=this._getItemsAsjQuery(t&&t.connected),s=[];return t=t||{},i.each(function(){s.push(e(t.item||this).attr(t.attribute||"id")||"")}),s},_intersectsWith:function(e){var t=this.positionAbs.left,i=t+this.helperProportions.width,s=this.positionAbs.top,n=s+this.helperProportions.height,a=e.left,o=a+e.width,r=e.top,h=r+e.height,l=this.offset.click.top,u=this.offset.click.left,d="x"===this.options.axis||s+l>r&&h>s+l,c="y"===this.options.axis||t+u>a&&o>t+u,p=d&&c;return"pointer"===this.options.tolerance||this.options.forcePointerForContainers||"pointer"!==this.options.tolerance&&this.helperProportions[this.floating?"width":"height"]>e[this.floating?"width":"height"]?p:t+this.helperProportions.width/2>a&&o>i-this.helperProportions.width/2&&s+this.helperProportions.height/2>r&&h>n-this.helperProportions.height/2},_intersectsWithPointer:function(e){var t="x"===this.options.axis||this._isOverAxis(this.positionAbs.top+this.offset.click.top,e.top,e.height),i="y"===this.options.axis||this._isOverAxis(this.positionAbs.left+this.offset.click.left,e.left,e.width),s=t&&i,n=this._getDragVerticalDirection(),a=this._getDragHorizontalDirection();return s?this.floating?a&&"right"===a||"down"===n?2:1:n&&("down"===n?2:1):!1},_intersectsWithSides:function(e){var t=this._isOverAxis(this.positionAbs.top+this.offset.click.top,e.top+e.height/2,e.height),i=this._isOverAxis(this.positionAbs.left+this.offset.click.left,e.left+e.width/2,e.width),s=this._getDragVerticalDirection(),n=this._getDragHorizontalDirection();return this.floating&&n?"right"===n&&i||"left"===n&&!i:s&&("down"===s&&t||"up"===s&&!t)},_getDragVerticalDirection:function(){var e=this.positionAbs.top-this.lastPositionAbs.top;return 0!==e&&(e>0?"down":"up")},_getDragHorizontalDirection:function(){var e=this.positionAbs.left-this.lastPositionAbs.left;return 0!==e&&(e>0?"right":"left")},refresh:function(e){return this._refreshItems(e),this._setHandleClassName(),this.refreshPositions(),this},_connectWith:function(){var e=this.options;return e.connectWith.constructor===String?[e.connectWith]:e.connectWith},_getItemsAsjQuery:function(t){function i(){r.push(this)}var s,n,a,o,r=[],h=[],l=this._connectWith();if(l&&t)for(s=l.length-1;s>=0;s--)for(a=e(l[s],this.document[0]),n=a.length-1;n>=0;n--)o=e.data(a[n],this.widgetFullName),o&&o!==this&&!o.options.disabled&&h.push([e.isFunction(o.options.items)?o.options.items.call(o.element):e(o.options.items,o.element).not(".ui-sortable-helper").not(".ui-sortable-placeholder"),o]);for(h.push([e.isFunction(this.options.items)?this.options.items.call(this.element,null,{options:this.options,item:this.currentItem}):e(this.options.items,this.element).not(".ui-sortable-helper").not(".ui-sortable-placeholder"),this]),s=h.length-1;s>=0;s--)h[s][0].each(i);return e(r)},_removeCurrentsFromItems:function(){var t=this.currentItem.find(":data("+this.widgetName+"-item)");this.items=e.grep(this.items,function(e){for(var i=0;t.length>i;i++)if(t[i]===e.item[0])return!1;return!0})},_refreshItems:function(t){this.items=[],this.containers=[this];var i,s,n,a,o,r,h,l,u=this.items,d=[[e.isFunction(this.options.items)?this.options.items.call(this.element[0],t,{item:this.currentItem}):e(this.options.items,this.element),this]],c=this._connectWith();if(c&&this.ready)for(i=c.length-1;i>=0;i--)for(n=e(c[i],this.document[0]),s=n.length-1;s>=0;s--)a=e.data(n[s],this.widgetFullName),a&&a!==this&&!a.options.disabled&&(d.push([e.isFunction(a.options.items)?a.options.items.call(a.element[0],t,{item:this.currentItem}):e(a.options.items,a.element),a]),this.containers.push(a));for(i=d.length-1;i>=0;i--)for(o=d[i][1],r=d[i][0],s=0,l=r.length;l>s;s++)h=e(r[s]),h.data(this.widgetName+"-item",o),u.push({item:h,instance:o,width:0,height:0,left:0,top:0})},refreshPositions:function(t){this.floating=this.items.length?"x"===this.options.axis||this._isFloating(this.items[0].item):!1,this.offsetParent&&this.helper&&(this.offset.parent=this._getParentOffset());var i,s,n,a;for(i=this.items.length-1;i>=0;i--)s=this.items[i],s.instance!==this.currentContainer&&this.currentContainer&&s.item[0]!==this.currentItem[0]||(n=this.options.toleranceElement?e(this.options.toleranceElement,s.item):s.item,t||(s.width=n.outerWidth(),s.height=n.outerHeight()),a=n.offset(),s.left=a.left,s.top=a.top);if(this.options.custom&&this.options.custom.refreshContainers)this.options.custom.refreshContainers.call(this);else for(i=this.containers.length-1;i>=0;i--)a=this.containers[i].element.offset(),this.containers[i].containerCache.left=a.left,this.containers[i].containerCache.top=a.top,this.containers[i].containerCache.width=this.containers[i].element.outerWidth(),this.containers[i].containerCache.height=this.containers[i].element.outerHeight();return this},_createPlaceholder:function(t){t=t||this;var i,s=t.options;s.placeholder&&s.placeholder.constructor!==String||(i=s.placeholder,s.placeholder={element:function(){var s=t.currentItem[0].nodeName.toLowerCase(),n=e("<"+s+">",t.document[0]).addClass(i||t.currentItem[0].className+" ui-sortable-placeholder").removeClass("ui-sortable-helper");return"tbody"===s?t._createTrPlaceholder(t.currentItem.find("tr").eq(0),e("<tr>",t.document[0]).appendTo(n)):"tr"===s?t._createTrPlaceholder(t.currentItem,n):"img"===s&&n.attr("src",t.currentItem.attr("src")),i||n.css("visibility","hidden"),n},update:function(e,n){(!i||s.forcePlaceholderSize)&&(n.height()||n.height(t.currentItem.innerHeight()-parseInt(t.currentItem.css("paddingTop")||0,10)-parseInt(t.currentItem.css("paddingBottom")||0,10)),n.width()||n.width(t.currentItem.innerWidth()-parseInt(t.currentItem.css("paddingLeft")||0,10)-parseInt(t.currentItem.css("paddingRight")||0,10)))}}),t.placeholder=e(s.placeholder.element.call(t.element,t.currentItem)),t.currentItem.after(t.placeholder),s.placeholder.update(t,t.placeholder)},_createTrPlaceholder:function(t,i){var s=this;t.children().each(function(){e("<td>&#160;</td>",s.document[0]).attr("colspan",e(this).attr("colspan")||1).appendTo(i)})},_contactContainers:function(t){var i,s,n,a,o,r,h,l,u,d,c=null,p=null;for(i=this.containers.length-1;i>=0;i--)if(!e.contains(this.currentItem[0],this.containers[i].element[0]))if(this._intersectsWith(this.containers[i].containerCache)){if(c&&e.contains(this.containers[i].element[0],c.element[0]))continue;c=this.containers[i],p=i}else this.containers[i].containerCache.over&&(this.containers[i]._trigger("out",t,this._uiHash(this)),this.containers[i].containerCache.over=0);if(c)if(1===this.containers.length)this.containers[p].containerCache.over||(this.containers[p]._trigger("over",t,this._uiHash(this)),this.containers[p].containerCache.over=1);else{for(n=1e4,a=null,u=c.floating||this._isFloating(this.currentItem),o=u?"left":"top",r=u?"width":"height",d=u?"clientX":"clientY",s=this.items.length-1;s>=0;s--)e.contains(this.containers[p].element[0],this.items[s].item[0])&&this.items[s].item[0]!==this.currentItem[0]&&(h=this.items[s].item.offset()[o],l=!1,t[d]-h>this.items[s][r]/2&&(l=!0),n>Math.abs(t[d]-h)&&(n=Math.abs(t[d]-h),a=this.items[s],this.direction=l?"up":"down"));if(!a&&!this.options.dropOnEmpty)return;if(this.currentContainer===this.containers[p])return this.currentContainer.containerCache.over||(this.containers[p]._trigger("over",t,this._uiHash()),this.currentContainer.containerCache.over=1),void 0;a?this._rearrange(t,a,null,!0):this._rearrange(t,null,this.containers[p].element,!0),this._trigger("change",t,this._uiHash()),this.containers[p]._trigger("change",t,this._uiHash(this)),this.currentContainer=this.containers[p],this.options.placeholder.update(this.currentContainer,this.placeholder),this.containers[p]._trigger("over",t,this._uiHash(this)),this.containers[p].containerCache.over=1}},_createHelper:function(t){var i=this.options,s=e.isFunction(i.helper)?e(i.helper.apply(this.element[0],[t,this.currentItem])):"clone"===i.helper?this.currentItem.clone():this.currentItem;return s.parents("body").length||e("parent"!==i.appendTo?i.appendTo:this.currentItem[0].parentNode)[0].appendChild(s[0]),s[0]===this.currentItem[0]&&(this._storedCSS={width:this.currentItem[0].style.width,height:this.currentItem[0].style.height,position:this.currentItem.css("position"),top:this.currentItem.css("top"),left:this.currentItem.css("left")}),(!s[0].style.width||i.forceHelperSize)&&s.width(this.currentItem.width()),(!s[0].style.height||i.forceHelperSize)&&s.height(this.currentItem.height()),s},_adjustOffsetFromHelper:function(t){"string"==typeof t&&(t=t.split(" ")),e.isArray(t)&&(t={left:+t[0],top:+t[1]||0}),"left"in t&&(this.offset.click.left=t.left+this.margins.left),"right"in t&&(this.offset.click.left=this.helperProportions.width-t.right+this.margins.left),"top"in t&&(this.offset.click.top=t.top+this.margins.top),"bottom"in t&&(this.offset.click.top=this.helperProportions.height-t.bottom+this.margins.top)},_getParentOffset:function(){this.offsetParent=this.helper.offsetParent();var t=this.offsetParent.offset();return"absolute"===this.cssPosition&&this.scrollParent[0]!==this.document[0]&&e.contains(this.scrollParent[0],this.offsetParent[0])&&(t.left+=this.scrollParent.scrollLeft(),t.top+=this.scrollParent.scrollTop()),(this.offsetParent[0]===this.document[0].body||this.offsetParent[0].tagName&&"html"===this.offsetParent[0].tagName.toLowerCase()&&e.ui.ie)&&(t={top:0,left:0}),{top:t.top+(parseInt(this.offsetParent.css("borderTopWidth"),10)||0),left:t.left+(parseInt(this.offsetParent.css("borderLeftWidth"),10)||0)}},_getRelativeOffset:function(){if("relative"===this.cssPosition){var e=this.currentItem.position();return{top:e.top-(parseInt(this.helper.css("top"),10)||0)+this.scrollParent.scrollTop(),left:e.left-(parseInt(this.helper.css("left"),10)||0)+this.scrollParent.scrollLeft()}}return{top:0,left:0}},_cacheMargins:function(){this.margins={left:parseInt(this.currentItem.css("marginLeft"),10)||0,top:parseInt(this.currentItem.css("marginTop"),10)||0}},_cacheHelperProportions:function(){this.helperProportions={width:this.helper.outerWidth(),height:this.helper.outerHeight()}},_setContainment:function(){var t,i,s,n=this.options;"parent"===n.containment&&(n.containment=this.helper[0].parentNode),("document"===n.containment||"window"===n.containment)&&(this.containment=[0-this.offset.relative.left-this.offset.parent.left,0-this.offset.relative.top-this.offset.parent.top,"document"===n.containment?this.document.width():this.window.width()-this.helperProportions.width-this.margins.left,("document"===n.containment?this.document.width():this.window.height()||this.document[0].body.parentNode.scrollHeight)-this.helperProportions.height-this.margins.top]),/^(document|window|parent)$/.test(n.containment)||(t=e(n.containment)[0],i=e(n.containment).offset(),s="hidden"!==e(t).css("overflow"),this.containment=[i.left+(parseInt(e(t).css("borderLeftWidth"),10)||0)+(parseInt(e(t).css("paddingLeft"),10)||0)-this.margins.left,i.top+(parseInt(e(t).css("borderTopWidth"),10)||0)+(parseInt(e(t).css("paddingTop"),10)||0)-this.margins.top,i.left+(s?Math.max(t.scrollWidth,t.offsetWidth):t.offsetWidth)-(parseInt(e(t).css("borderLeftWidth"),10)||0)-(parseInt(e(t).css("paddingRight"),10)||0)-this.helperProportions.width-this.margins.left,i.top+(s?Math.max(t.scrollHeight,t.offsetHeight):t.offsetHeight)-(parseInt(e(t).css("borderTopWidth"),10)||0)-(parseInt(e(t).css("paddingBottom"),10)||0)-this.helperProportions.height-this.margins.top])},_convertPositionTo:function(t,i){i||(i=this.position);var s="absolute"===t?1:-1,n="absolute"!==this.cssPosition||this.scrollParent[0]!==this.document[0]&&e.contains(this.scrollParent[0],this.offsetParent[0])?this.scrollParent:this.offsetParent,a=/(html|body)/i.test(n[0].tagName);return{top:i.top+this.offset.relative.top*s+this.offset.parent.top*s-("fixed"===this.cssPosition?-this.scrollParent.scrollTop():a?0:n.scrollTop())*s,left:i.left+this.offset.relative.left*s+this.offset.parent.left*s-("fixed"===this.cssPosition?-this.scrollParent.scrollLeft():a?0:n.scrollLeft())*s}},_generatePosition:function(t){var i,s,n=this.options,a=t.pageX,o=t.pageY,r="absolute"!==this.cssPosition||this.scrollParent[0]!==this.document[0]&&e.contains(this.scrollParent[0],this.offsetParent[0])?this.scrollParent:this.offsetParent,h=/(html|body)/i.test(r[0].tagName);return"relative"!==this.cssPosition||this.scrollParent[0]!==this.document[0]&&this.scrollParent[0]!==this.offsetParent[0]||(this.offset.relative=this._getRelativeOffset()),this.originalPosition&&(this.containment&&(t.pageX-this.offset.click.left<this.containment[0]&&(a=this.containment[0]+this.offset.click.left),t.pageY-this.offset.click.top<this.containment[1]&&(o=this.containment[1]+this.offset.click.top),t.pageX-this.offset.click.left>this.containment[2]&&(a=this.containment[2]+this.offset.click.left),t.pageY-this.offset.click.top>this.containment[3]&&(o=this.containment[3]+this.offset.click.top)),n.grid&&(i=this.originalPageY+Math.round((o-this.originalPageY)/n.grid[1])*n.grid[1],o=this.containment?i-this.offset.click.top>=this.containment[1]&&i-this.offset.click.top<=this.containment[3]?i:i-this.offset.click.top>=this.containment[1]?i-n.grid[1]:i+n.grid[1]:i,s=this.originalPageX+Math.round((a-this.originalPageX)/n.grid[0])*n.grid[0],a=this.containment?s-this.offset.click.left>=this.containment[0]&&s-this.offset.click.left<=this.containment[2]?s:s-this.offset.click.left>=this.containment[0]?s-n.grid[0]:s+n.grid[0]:s)),{top:o-this.offset.click.top-this.offset.relative.top-this.offset.parent.top+("fixed"===this.cssPosition?-this.scrollParent.scrollTop():h?0:r.scrollTop()),left:a-this.offset.click.left-this.offset.relative.left-this.offset.parent.left+("fixed"===this.cssPosition?-this.scrollParent.scrollLeft():h?0:r.scrollLeft())}},_rearrange:function(e,t,i,s){i?i[0].appendChild(this.placeholder[0]):t.item[0].parentNode.insertBefore(this.placeholder[0],"down"===this.direction?t.item[0]:t.item[0].nextSibling),this.counter=this.counter?++this.counter:1;var n=this.counter;this._delay(function(){n===this.counter&&this.refreshPositions(!s)})},_clear:function(e,t){function i(e,t,i){return function(s){i._trigger(e,s,t._uiHash(t))}}this.reverting=!1;var s,n=[];if(!this._noFinalSort&&this.currentItem.parent().length&&this.placeholder.before(this.currentItem),this._noFinalSort=null,this.helper[0]===this.currentItem[0]){for(s in this._storedCSS)("auto"===this._storedCSS[s]||"static"===this._storedCSS[s])&&(this._storedCSS[s]="");this.currentItem.css(this._storedCSS).removeClass("ui-sortable-helper")}else this.currentItem.show();for(this.fromOutside&&!t&&n.push(function(e){this._trigger("receive",e,this._uiHash(this.fromOutside))}),!this.fromOutside&&this.domPosition.prev===this.currentItem.prev().not(".ui-sortable-helper")[0]&&this.domPosition.parent===this.currentItem.parent()[0]||t||n.push(function(e){this._trigger("update",e,this._uiHash())}),this!==this.currentContainer&&(t||(n.push(function(e){this._trigger("remove",e,this._uiHash())}),n.push(function(e){return function(t){e._trigger("receive",t,this._uiHash(this))}}.call(this,this.currentContainer)),n.push(function(e){return function(t){e._trigger("update",t,this._uiHash(this))}}.call(this,this.currentContainer)))),s=this.containers.length-1;s>=0;s--)t||n.push(i("deactivate",this,this.containers[s])),this.containers[s].containerCache.over&&(n.push(i("out",this,this.containers[s])),this.containers[s].containerCache.over=0);if(this.storedCursor&&(this.document.find("body").css("cursor",this.storedCursor),this.storedStylesheet.remove()),this._storedOpacity&&this.helper.css("opacity",this._storedOpacity),this._storedZIndex&&this.helper.css("zIndex","auto"===this._storedZIndex?"":this._storedZIndex),this.dragging=!1,t||this._trigger("beforeStop",e,this._uiHash()),this.placeholder[0].parentNode.removeChild(this.placeholder[0]),this.cancelHelperRemoval||(this.helper[0]!==this.currentItem[0]&&this.helper.remove(),this.helper=null),!t){for(s=0;n.length>s;s++)n[s].call(this,e);this._trigger("stop",e,this._uiHash())}return this.fromOutside=!1,!this.cancelHelperRemoval},_trigger:function(){e.Widget.prototype._trigger.apply(this,arguments)===!1&&this.cancel()},_uiHash:function(t){var i=t||this;return{helper:i.helper,placeholder:i.placeholder||e([]),position:i.position,originalPosition:i.originalPosition,offset:i.positionAbs,item:i.currentItem,sender:t?t.element:null}}}),e.widget("ui.accordion",{version:"1.11.4",options:{active:0,animate:{},collapsible:!1,event:"click",header:"> li > :first-child,> :not(li):even",heightStyle:"auto",icons:{activeHeader:"ui-icon-triangle-1-s",header:"ui-icon-triangle-1-e"},activate:null,beforeActivate:null},hideProps:{borderTopWidth:"hide",borderBottomWidth:"hide",paddingTop:"hide",paddingBottom:"hide",height:"hide"},showProps:{borderTopWidth:"show",borderBottomWidth:"show",paddingTop:"show",paddingBottom:"show",height:"show"},_create:function(){var t=this.options;this.prevShow=this.prevHide=e(),this.element.addClass("ui-accordion ui-widget ui-helper-reset").attr("role","tablist"),t.collapsible||t.active!==!1&&null!=t.active||(t.active=0),this._processPanels(),0>t.active&&(t.active+=this.headers.length),this._refresh()},_getCreateEventData:function(){return{header:this.active,panel:this.active.length?this.active.next():e()}},_createIcons:function(){var t=this.options.icons;t&&(e("<span>").addClass("ui-accordion-header-icon ui-icon "+t.header).prependTo(this.headers),this.active.children(".ui-accordion-header-icon").removeClass(t.header).addClass(t.activeHeader),this.headers.addClass("ui-accordion-icons"))},_destroyIcons:function(){this.headers.removeClass("ui-accordion-icons").children(".ui-accordion-header-icon").remove()},_destroy:function(){var e;this.element.removeClass("ui-accordion ui-widget ui-helper-reset").removeAttr("role"),this.headers.removeClass("ui-accordion-header ui-accordion-header-active ui-state-default ui-corner-all ui-state-active ui-state-disabled ui-corner-top").removeAttr("role").removeAttr("aria-expanded").removeAttr("aria-selected").removeAttr("aria-controls").removeAttr("tabIndex").removeUniqueId(),this._destroyIcons(),e=this.headers.next().removeClass("ui-helper-reset ui-widget-content ui-corner-bottom ui-accordion-content ui-accordion-content-active ui-state-disabled").css("display","").removeAttr("role").removeAttr("aria-hidden").removeAttr("aria-labelledby").removeUniqueId(),"content"!==this.options.heightStyle&&e.css("height","")},_setOption:function(e,t){return"active"===e?(this._activate(t),void 0):("event"===e&&(this.options.event&&this._off(this.headers,this.options.event),this._setupEvents(t)),this._super(e,t),"collapsible"!==e||t||this.options.active!==!1||this._activate(0),"icons"===e&&(this._destroyIcons(),t&&this._createIcons()),"disabled"===e&&(this.element.toggleClass("ui-state-disabled",!!t).attr("aria-disabled",t),this.headers.add(this.headers.next()).toggleClass("ui-state-disabled",!!t)),void 0)},_keydown:function(t){if(!t.altKey&&!t.ctrlKey){var i=e.ui.keyCode,s=this.headers.length,n=this.headers.index(t.target),a=!1;switch(t.keyCode){case i.RIGHT:case i.DOWN:a=this.headers[(n+1)%s];break;case i.LEFT:case i.UP:a=this.headers[(n-1+s)%s];break;case i.SPACE:case i.ENTER:this._eventHandler(t);break;case i.HOME:a=this.headers[0];break;case i.END:a=this.headers[s-1]}a&&(e(t.target).attr("tabIndex",-1),e(a).attr("tabIndex",0),a.focus(),t.preventDefault())}},_panelKeyDown:function(t){t.keyCode===e.ui.keyCode.UP&&t.ctrlKey&&e(t.currentTarget).prev().focus()},refresh:function(){var t=this.options;this._processPanels(),t.active===!1&&t.collapsible===!0||!this.headers.length?(t.active=!1,this.active=e()):t.active===!1?this._activate(0):this.active.length&&!e.contains(this.element[0],this.active[0])?this.headers.length===this.headers.find(".ui-state-disabled").length?(t.active=!1,this.active=e()):this._activate(Math.max(0,t.active-1)):t.active=this.headers.index(this.active),this._destroyIcons(),this._refresh()},_processPanels:function(){var e=this.headers,t=this.panels;this.headers=this.element.find(this.options.header).addClass("ui-accordion-header ui-state-default ui-corner-all"),this.panels=this.headers.next().addClass("ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom").filter(":not(.ui-accordion-content-active)").hide(),t&&(this._off(e.not(this.headers)),this._off(t.not(this.panels)))},_refresh:function(){var t,i=this.options,s=i.heightStyle,n=this.element.parent();this.active=this._findActive(i.active).addClass("ui-accordion-header-active ui-state-active ui-corner-top").removeClass("ui-corner-all"),this.active.next().addClass("ui-accordion-content-active").show(),this.headers.attr("role","tab").each(function(){var t=e(this),i=t.uniqueId().attr("id"),s=t.next(),n=s.uniqueId().attr("id");t.attr("aria-controls",n),s.attr("aria-labelledby",i)}).next().attr("role","tabpanel"),this.headers.not(this.active).attr({"aria-selected":"false","aria-expanded":"false",tabIndex:-1}).next().attr({"aria-hidden":"true"}).hide(),this.active.length?this.active.attr({"aria-selected":"true","aria-expanded":"true",tabIndex:0}).next().attr({"aria-hidden":"false"}):this.headers.eq(0).attr("tabIndex",0),this._createIcons(),this._setupEvents(i.event),"fill"===s?(t=n.height(),this.element.siblings(":visible").each(function(){var i=e(this),s=i.css("position");"absolute"!==s&&"fixed"!==s&&(t-=i.outerHeight(!0))}),this.headers.each(function(){t-=e(this).outerHeight(!0)}),this.headers.next().each(function(){e(this).height(Math.max(0,t-e(this).innerHeight()+e(this).height()))}).css("overflow","auto")):"auto"===s&&(t=0,this.headers.next().each(function(){t=Math.max(t,e(this).css("height","").height())}).height(t))},_activate:function(t){var i=this._findActive(t)[0];i!==this.active[0]&&(i=i||this.active[0],this._eventHandler({target:i,currentTarget:i,preventDefault:e.noop}))},_findActive:function(t){return"number"==typeof t?this.headers.eq(t):e()},_setupEvents:function(t){var i={keydown:"_keydown"};t&&e.each(t.split(" "),function(e,t){i[t]="_eventHandler"}),this._off(this.headers.add(this.headers.next())),this._on(this.headers,i),this._on(this.headers.next(),{keydown:"_panelKeyDown"}),this._hoverable(this.headers),this._focusable(this.headers)},_eventHandler:function(t){var i=this.options,s=this.active,n=e(t.currentTarget),a=n[0]===s[0],o=a&&i.collapsible,r=o?e():n.next(),h=s.next(),l={oldHeader:s,oldPanel:h,newHeader:o?e():n,newPanel:r};
t.preventDefault(),a&&!i.collapsible||this._trigger("beforeActivate",t,l)===!1||(i.active=o?!1:this.headers.index(n),this.active=a?e():n,this._toggle(l),s.removeClass("ui-accordion-header-active ui-state-active"),i.icons&&s.children(".ui-accordion-header-icon").removeClass(i.icons.activeHeader).addClass(i.icons.header),a||(n.removeClass("ui-corner-all").addClass("ui-accordion-header-active ui-state-active ui-corner-top"),i.icons&&n.children(".ui-accordion-header-icon").removeClass(i.icons.header).addClass(i.icons.activeHeader),n.next().addClass("ui-accordion-content-active")))},_toggle:function(t){var i=t.newPanel,s=this.prevShow.length?this.prevShow:t.oldPanel;this.prevShow.add(this.prevHide).stop(!0,!0),this.prevShow=i,this.prevHide=s,this.options.animate?this._animate(i,s,t):(s.hide(),i.show(),this._toggleComplete(t)),s.attr({"aria-hidden":"true"}),s.prev().attr({"aria-selected":"false","aria-expanded":"false"}),i.length&&s.length?s.prev().attr({tabIndex:-1,"aria-expanded":"false"}):i.length&&this.headers.filter(function(){return 0===parseInt(e(this).attr("tabIndex"),10)}).attr("tabIndex",-1),i.attr("aria-hidden","false").prev().attr({"aria-selected":"true","aria-expanded":"true",tabIndex:0})},_animate:function(e,t,i){var s,n,a,o=this,r=0,h=e.css("box-sizing"),l=e.length&&(!t.length||e.index()<t.index()),u=this.options.animate||{},d=l&&u.down||u,c=function(){o._toggleComplete(i)};return"number"==typeof d&&(a=d),"string"==typeof d&&(n=d),n=n||d.easing||u.easing,a=a||d.duration||u.duration,t.length?e.length?(s=e.show().outerHeight(),t.animate(this.hideProps,{duration:a,easing:n,step:function(e,t){t.now=Math.round(e)}}),e.hide().animate(this.showProps,{duration:a,easing:n,complete:c,step:function(e,i){i.now=Math.round(e),"height"!==i.prop?"content-box"===h&&(r+=i.now):"content"!==o.options.heightStyle&&(i.now=Math.round(s-t.outerHeight()-r),r=0)}}),void 0):t.animate(this.hideProps,a,n,c):e.animate(this.showProps,a,n,c)},_toggleComplete:function(e){var t=e.oldPanel;t.removeClass("ui-accordion-content-active").prev().removeClass("ui-corner-top").addClass("ui-corner-all"),t.length&&(t.parent()[0].className=t.parent()[0].className),this._trigger("activate",null,e)}}),e.widget("ui.menu",{version:"1.11.4",defaultElement:"<ul>",delay:300,options:{icons:{submenu:"ui-icon-carat-1-e"},items:"> *",menus:"ul",position:{my:"left-1 top",at:"right top"},role:"menu",blur:null,focus:null,select:null},_create:function(){this.activeMenu=this.element,this.mouseHandled=!1,this.element.uniqueId().addClass("ui-menu ui-widget ui-widget-content").toggleClass("ui-menu-icons",!!this.element.find(".ui-icon").length).attr({role:this.options.role,tabIndex:0}),this.options.disabled&&this.element.addClass("ui-state-disabled").attr("aria-disabled","true"),this._on({"mousedown .ui-menu-item":function(e){e.preventDefault()},"click .ui-menu-item":function(t){var i=e(t.target);!this.mouseHandled&&i.not(".ui-state-disabled").length&&(this.select(t),t.isPropagationStopped()||(this.mouseHandled=!0),i.has(".ui-menu").length?this.expand(t):!this.element.is(":focus")&&e(this.document[0].activeElement).closest(".ui-menu").length&&(this.element.trigger("focus",[!0]),this.active&&1===this.active.parents(".ui-menu").length&&clearTimeout(this.timer)))},"mouseenter .ui-menu-item":function(t){if(!this.previousFilter){var i=e(t.currentTarget);i.siblings(".ui-state-active").removeClass("ui-state-active"),this.focus(t,i)}},mouseleave:"collapseAll","mouseleave .ui-menu":"collapseAll",focus:function(e,t){var i=this.active||this.element.find(this.options.items).eq(0);t||this.focus(e,i)},blur:function(t){this._delay(function(){e.contains(this.element[0],this.document[0].activeElement)||this.collapseAll(t)})},keydown:"_keydown"}),this.refresh(),this._on(this.document,{click:function(e){this._closeOnDocumentClick(e)&&this.collapseAll(e),this.mouseHandled=!1}})},_destroy:function(){this.element.removeAttr("aria-activedescendant").find(".ui-menu").addBack().removeClass("ui-menu ui-widget ui-widget-content ui-menu-icons ui-front").removeAttr("role").removeAttr("tabIndex").removeAttr("aria-labelledby").removeAttr("aria-expanded").removeAttr("aria-hidden").removeAttr("aria-disabled").removeUniqueId().show(),this.element.find(".ui-menu-item").removeClass("ui-menu-item").removeAttr("role").removeAttr("aria-disabled").removeUniqueId().removeClass("ui-state-hover").removeAttr("tabIndex").removeAttr("role").removeAttr("aria-haspopup").children().each(function(){var t=e(this);t.data("ui-menu-submenu-carat")&&t.remove()}),this.element.find(".ui-menu-divider").removeClass("ui-menu-divider ui-widget-content")},_keydown:function(t){var i,s,n,a,o=!0;switch(t.keyCode){case e.ui.keyCode.PAGE_UP:this.previousPage(t);break;case e.ui.keyCode.PAGE_DOWN:this.nextPage(t);break;case e.ui.keyCode.HOME:this._move("first","first",t);break;case e.ui.keyCode.END:this._move("last","last",t);break;case e.ui.keyCode.UP:this.previous(t);break;case e.ui.keyCode.DOWN:this.next(t);break;case e.ui.keyCode.LEFT:this.collapse(t);break;case e.ui.keyCode.RIGHT:this.active&&!this.active.is(".ui-state-disabled")&&this.expand(t);break;case e.ui.keyCode.ENTER:case e.ui.keyCode.SPACE:this._activate(t);break;case e.ui.keyCode.ESCAPE:this.collapse(t);break;default:o=!1,s=this.previousFilter||"",n=String.fromCharCode(t.keyCode),a=!1,clearTimeout(this.filterTimer),n===s?a=!0:n=s+n,i=this._filterMenuItems(n),i=a&&-1!==i.index(this.active.next())?this.active.nextAll(".ui-menu-item"):i,i.length||(n=String.fromCharCode(t.keyCode),i=this._filterMenuItems(n)),i.length?(this.focus(t,i),this.previousFilter=n,this.filterTimer=this._delay(function(){delete this.previousFilter},1e3)):delete this.previousFilter}o&&t.preventDefault()},_activate:function(e){this.active.is(".ui-state-disabled")||(this.active.is("[aria-haspopup='true']")?this.expand(e):this.select(e))},refresh:function(){var t,i,s=this,n=this.options.icons.submenu,a=this.element.find(this.options.menus);this.element.toggleClass("ui-menu-icons",!!this.element.find(".ui-icon").length),a.filter(":not(.ui-menu)").addClass("ui-menu ui-widget ui-widget-content ui-front").hide().attr({role:this.options.role,"aria-hidden":"true","aria-expanded":"false"}).each(function(){var t=e(this),i=t.parent(),s=e("<span>").addClass("ui-menu-icon ui-icon "+n).data("ui-menu-submenu-carat",!0);i.attr("aria-haspopup","true").prepend(s),t.attr("aria-labelledby",i.attr("id"))}),t=a.add(this.element),i=t.find(this.options.items),i.not(".ui-menu-item").each(function(){var t=e(this);s._isDivider(t)&&t.addClass("ui-widget-content ui-menu-divider")}),i.not(".ui-menu-item, .ui-menu-divider").addClass("ui-menu-item").uniqueId().attr({tabIndex:-1,role:this._itemRole()}),i.filter(".ui-state-disabled").attr("aria-disabled","true"),this.active&&!e.contains(this.element[0],this.active[0])&&this.blur()},_itemRole:function(){return{menu:"menuitem",listbox:"option"}[this.options.role]},_setOption:function(e,t){"icons"===e&&this.element.find(".ui-menu-icon").removeClass(this.options.icons.submenu).addClass(t.submenu),"disabled"===e&&this.element.toggleClass("ui-state-disabled",!!t).attr("aria-disabled",t),this._super(e,t)},focus:function(e,t){var i,s;this.blur(e,e&&"focus"===e.type),this._scrollIntoView(t),this.active=t.first(),s=this.active.addClass("ui-state-focus").removeClass("ui-state-active"),this.options.role&&this.element.attr("aria-activedescendant",s.attr("id")),this.active.parent().closest(".ui-menu-item").addClass("ui-state-active"),e&&"keydown"===e.type?this._close():this.timer=this._delay(function(){this._close()},this.delay),i=t.children(".ui-menu"),i.length&&e&&/^mouse/.test(e.type)&&this._startOpening(i),this.activeMenu=t.parent(),this._trigger("focus",e,{item:t})},_scrollIntoView:function(t){var i,s,n,a,o,r;this._hasScroll()&&(i=parseFloat(e.css(this.activeMenu[0],"borderTopWidth"))||0,s=parseFloat(e.css(this.activeMenu[0],"paddingTop"))||0,n=t.offset().top-this.activeMenu.offset().top-i-s,a=this.activeMenu.scrollTop(),o=this.activeMenu.height(),r=t.outerHeight(),0>n?this.activeMenu.scrollTop(a+n):n+r>o&&this.activeMenu.scrollTop(a+n-o+r))},blur:function(e,t){t||clearTimeout(this.timer),this.active&&(this.active.removeClass("ui-state-focus"),this.active=null,this._trigger("blur",e,{item:this.active}))},_startOpening:function(e){clearTimeout(this.timer),"true"===e.attr("aria-hidden")&&(this.timer=this._delay(function(){this._close(),this._open(e)},this.delay))},_open:function(t){var i=e.extend({of:this.active},this.options.position);clearTimeout(this.timer),this.element.find(".ui-menu").not(t.parents(".ui-menu")).hide().attr("aria-hidden","true"),t.show().removeAttr("aria-hidden").attr("aria-expanded","true").position(i)},collapseAll:function(t,i){clearTimeout(this.timer),this.timer=this._delay(function(){var s=i?this.element:e(t&&t.target).closest(this.element.find(".ui-menu"));s.length||(s=this.element),this._close(s),this.blur(t),this.activeMenu=s},this.delay)},_close:function(e){e||(e=this.active?this.active.parent():this.element),e.find(".ui-menu").hide().attr("aria-hidden","true").attr("aria-expanded","false").end().find(".ui-state-active").not(".ui-state-focus").removeClass("ui-state-active")},_closeOnDocumentClick:function(t){return!e(t.target).closest(".ui-menu").length},_isDivider:function(e){return!/[^\-\u2014\u2013\s]/.test(e.text())},collapse:function(e){var t=this.active&&this.active.parent().closest(".ui-menu-item",this.element);t&&t.length&&(this._close(),this.focus(e,t))},expand:function(e){var t=this.active&&this.active.children(".ui-menu ").find(this.options.items).first();t&&t.length&&(this._open(t.parent()),this._delay(function(){this.focus(e,t)}))},next:function(e){this._move("next","first",e)},previous:function(e){this._move("prev","last",e)},isFirstItem:function(){return this.active&&!this.active.prevAll(".ui-menu-item").length},isLastItem:function(){return this.active&&!this.active.nextAll(".ui-menu-item").length},_move:function(e,t,i){var s;this.active&&(s="first"===e||"last"===e?this.active["first"===e?"prevAll":"nextAll"](".ui-menu-item").eq(-1):this.active[e+"All"](".ui-menu-item").eq(0)),s&&s.length&&this.active||(s=this.activeMenu.find(this.options.items)[t]()),this.focus(i,s)},nextPage:function(t){var i,s,n;return this.active?(this.isLastItem()||(this._hasScroll()?(s=this.active.offset().top,n=this.element.height(),this.active.nextAll(".ui-menu-item").each(function(){return i=e(this),0>i.offset().top-s-n}),this.focus(t,i)):this.focus(t,this.activeMenu.find(this.options.items)[this.active?"last":"first"]())),void 0):(this.next(t),void 0)},previousPage:function(t){var i,s,n;return this.active?(this.isFirstItem()||(this._hasScroll()?(s=this.active.offset().top,n=this.element.height(),this.active.prevAll(".ui-menu-item").each(function(){return i=e(this),i.offset().top-s+n>0}),this.focus(t,i)):this.focus(t,this.activeMenu.find(this.options.items).first())),void 0):(this.next(t),void 0)},_hasScroll:function(){return this.element.outerHeight()<this.element.prop("scrollHeight")},select:function(t){this.active=this.active||e(t.target).closest(".ui-menu-item");var i={item:this.active};this.active.has(".ui-menu").length||this.collapseAll(t,!0),this._trigger("select",t,i)},_filterMenuItems:function(t){var i=t.replace(/[\-\[\]{}()*+?.,\\\^$|#\s]/g,"\\$&"),s=RegExp("^"+i,"i");return this.activeMenu.find(this.options.items).filter(".ui-menu-item").filter(function(){return s.test(e.trim(e(this).text()))})}}),e.widget("ui.autocomplete",{version:"1.11.4",defaultElement:"<input>",options:{appendTo:null,autoFocus:!1,delay:300,minLength:1,position:{my:"left top",at:"left bottom",collision:"none"},source:null,change:null,close:null,focus:null,open:null,response:null,search:null,select:null},requestIndex:0,pending:0,_create:function(){var t,i,s,n=this.element[0].nodeName.toLowerCase(),a="textarea"===n,o="input"===n;this.isMultiLine=a?!0:o?!1:this.element.prop("isContentEditable"),this.valueMethod=this.element[a||o?"val":"text"],this.isNewMenu=!0,this.element.addClass("ui-autocomplete-input").attr("autocomplete","off"),this._on(this.element,{keydown:function(n){if(this.element.prop("readOnly"))return t=!0,s=!0,i=!0,void 0;t=!1,s=!1,i=!1;var a=e.ui.keyCode;switch(n.keyCode){case a.PAGE_UP:t=!0,this._move("previousPage",n);break;case a.PAGE_DOWN:t=!0,this._move("nextPage",n);break;case a.UP:t=!0,this._keyEvent("previous",n);break;case a.DOWN:t=!0,this._keyEvent("next",n);break;case a.ENTER:this.menu.active&&(t=!0,n.preventDefault(),this.menu.select(n));break;case a.TAB:this.menu.active&&this.menu.select(n);break;case a.ESCAPE:this.menu.element.is(":visible")&&(this.isMultiLine||this._value(this.term),this.close(n),n.preventDefault());break;default:i=!0,this._searchTimeout(n)}},keypress:function(s){if(t)return t=!1,(!this.isMultiLine||this.menu.element.is(":visible"))&&s.preventDefault(),void 0;if(!i){var n=e.ui.keyCode;switch(s.keyCode){case n.PAGE_UP:this._move("previousPage",s);break;case n.PAGE_DOWN:this._move("nextPage",s);break;case n.UP:this._keyEvent("previous",s);break;case n.DOWN:this._keyEvent("next",s)}}},input:function(e){return s?(s=!1,e.preventDefault(),void 0):(this._searchTimeout(e),void 0)},focus:function(){this.selectedItem=null,this.previous=this._value()},blur:function(e){return this.cancelBlur?(delete this.cancelBlur,void 0):(clearTimeout(this.searching),this.close(e),this._change(e),void 0)}}),this._initSource(),this.menu=e("<ul>").addClass("ui-autocomplete ui-front").appendTo(this._appendTo()).menu({role:null}).hide().menu("instance"),this._on(this.menu.element,{mousedown:function(t){t.preventDefault(),this.cancelBlur=!0,this._delay(function(){delete this.cancelBlur});var i=this.menu.element[0];e(t.target).closest(".ui-menu-item").length||this._delay(function(){var t=this;this.document.one("mousedown",function(s){s.target===t.element[0]||s.target===i||e.contains(i,s.target)||t.close()})})},menufocus:function(t,i){var s,n;return this.isNewMenu&&(this.isNewMenu=!1,t.originalEvent&&/^mouse/.test(t.originalEvent.type))?(this.menu.blur(),this.document.one("mousemove",function(){e(t.target).trigger(t.originalEvent)}),void 0):(n=i.item.data("ui-autocomplete-item"),!1!==this._trigger("focus",t,{item:n})&&t.originalEvent&&/^key/.test(t.originalEvent.type)&&this._value(n.value),s=i.item.attr("aria-label")||n.value,s&&e.trim(s).length&&(this.liveRegion.children().hide(),e("<div>").text(s).appendTo(this.liveRegion)),void 0)},menuselect:function(e,t){var i=t.item.data("ui-autocomplete-item"),s=this.previous;this.element[0]!==this.document[0].activeElement&&(this.element.focus(),this.previous=s,this._delay(function(){this.previous=s,this.selectedItem=i})),!1!==this._trigger("select",e,{item:i})&&this._value(i.value),this.term=this._value(),this.close(e),this.selectedItem=i}}),this.liveRegion=e("<span>",{role:"status","aria-live":"assertive","aria-relevant":"additions"}).addClass("ui-helper-hidden-accessible").appendTo(this.document[0].body),this._on(this.window,{beforeunload:function(){this.element.removeAttr("autocomplete")}})},_destroy:function(){clearTimeout(this.searching),this.element.removeClass("ui-autocomplete-input").removeAttr("autocomplete"),this.menu.element.remove(),this.liveRegion.remove()},_setOption:function(e,t){this._super(e,t),"source"===e&&this._initSource(),"appendTo"===e&&this.menu.element.appendTo(this._appendTo()),"disabled"===e&&t&&this.xhr&&this.xhr.abort()},_appendTo:function(){var t=this.options.appendTo;return t&&(t=t.jquery||t.nodeType?e(t):this.document.find(t).eq(0)),t&&t[0]||(t=this.element.closest(".ui-front")),t.length||(t=this.document[0].body),t},_initSource:function(){var t,i,s=this;e.isArray(this.options.source)?(t=this.options.source,this.source=function(i,s){s(e.ui.autocomplete.filter(t,i.term))}):"string"==typeof this.options.source?(i=this.options.source,this.source=function(t,n){s.xhr&&s.xhr.abort(),s.xhr=e.ajax({url:i,data:t,dataType:"json",success:function(e){n(e)},error:function(){n([])}})}):this.source=this.options.source},_searchTimeout:function(e){clearTimeout(this.searching),this.searching=this._delay(function(){var t=this.term===this._value(),i=this.menu.element.is(":visible"),s=e.altKey||e.ctrlKey||e.metaKey||e.shiftKey;(!t||t&&!i&&!s)&&(this.selectedItem=null,this.search(null,e))},this.options.delay)},search:function(e,t){return e=null!=e?e:this._value(),this.term=this._value(),e.length<this.options.minLength?this.close(t):this._trigger("search",t)!==!1?this._search(e):void 0},_search:function(e){this.pending++,this.element.addClass("ui-autocomplete-loading"),this.cancelSearch=!1,this.source({term:e},this._response())},_response:function(){var t=++this.requestIndex;return e.proxy(function(e){t===this.requestIndex&&this.__response(e),this.pending--,this.pending||this.element.removeClass("ui-autocomplete-loading")},this)},__response:function(e){e&&(e=this._normalize(e)),this._trigger("response",null,{content:e}),!this.options.disabled&&e&&e.length&&!this.cancelSearch?(this._suggest(e),this._trigger("open")):this._close()},close:function(e){this.cancelSearch=!0,this._close(e)},_close:function(e){this.menu.element.is(":visible")&&(this.menu.element.hide(),this.menu.blur(),this.isNewMenu=!0,this._trigger("close",e))},_change:function(e){this.previous!==this._value()&&this._trigger("change",e,{item:this.selectedItem})},_normalize:function(t){return t.length&&t[0].label&&t[0].value?t:e.map(t,function(t){return"string"==typeof t?{label:t,value:t}:e.extend({},t,{label:t.label||t.value,value:t.value||t.label})})},_suggest:function(t){var i=this.menu.element.empty();this._renderMenu(i,t),this.isNewMenu=!0,this.menu.refresh(),i.show(),this._resizeMenu(),i.position(e.extend({of:this.element},this.options.position)),this.options.autoFocus&&this.menu.next()},_resizeMenu:function(){var e=this.menu.element;e.outerWidth(Math.max(e.width("").outerWidth()+1,this.element.outerWidth()))},_renderMenu:function(t,i){var s=this;e.each(i,function(e,i){s._renderItemData(t,i)})},_renderItemData:function(e,t){return this._renderItem(e,t).data("ui-autocomplete-item",t)},_renderItem:function(t,i){return e("<li>").text(i.label).appendTo(t)},_move:function(e,t){return this.menu.element.is(":visible")?this.menu.isFirstItem()&&/^previous/.test(e)||this.menu.isLastItem()&&/^next/.test(e)?(this.isMultiLine||this._value(this.term),this.menu.blur(),void 0):(this.menu[e](t),void 0):(this.search(null,t),void 0)},widget:function(){return this.menu.element},_value:function(){return this.valueMethod.apply(this.element,arguments)},_keyEvent:function(e,t){(!this.isMultiLine||this.menu.element.is(":visible"))&&(this._move(e,t),t.preventDefault())}}),e.extend(e.ui.autocomplete,{escapeRegex:function(e){return e.replace(/[\-\[\]{}()*+?.,\\\^$|#\s]/g,"\\$&")},filter:function(t,i){var s=RegExp(e.ui.autocomplete.escapeRegex(i),"i");return e.grep(t,function(e){return s.test(e.label||e.value||e)})}}),e.widget("ui.autocomplete",e.ui.autocomplete,{options:{messages:{noResults:"No search results.",results:function(e){return e+(e>1?" results are":" result is")+" available, use up and down arrow keys to navigate."}}},__response:function(t){var i;this._superApply(arguments),this.options.disabled||this.cancelSearch||(i=t&&t.length?this.options.messages.results(t.length):this.options.messages.noResults,this.liveRegion.children().hide(),e("<div>").text(i).appendTo(this.liveRegion))}}),e.ui.autocomplete;var r,h="ui-button ui-widget ui-state-default ui-corner-all",l="ui-button-icons-only ui-button-icon-only ui-button-text-icons ui-button-text-icon-primary ui-button-text-icon-secondary ui-button-text-only",u=function(){var t=e(this);setTimeout(function(){t.find(":ui-button").button("refresh")},1)},d=function(t){var i=t.name,s=t.form,n=e([]);return i&&(i=i.replace(/'/g,"\\'"),n=s?e(s).find("[name='"+i+"'][type=radio]"):e("[name='"+i+"'][type=radio]",t.ownerDocument).filter(function(){return!this.form})),n};e.widget("ui.button",{version:"1.11.4",defaultElement:"<button>",options:{disabled:null,text:!0,label:null,icons:{primary:null,secondary:null}},_create:function(){this.element.closest("form").unbind("reset"+this.eventNamespace).bind("reset"+this.eventNamespace,u),"boolean"!=typeof this.options.disabled?this.options.disabled=!!this.element.prop("disabled"):this.element.prop("disabled",this.options.disabled),this._determineButtonType(),this.hasTitle=!!this.buttonElement.attr("title");var t=this,i=this.options,s="checkbox"===this.type||"radio"===this.type,n=s?"":"ui-state-active";null===i.label&&(i.label="input"===this.type?this.buttonElement.val():this.buttonElement.html()),this._hoverable(this.buttonElement),this.buttonElement.addClass(h).attr("role","button").bind("mouseenter"+this.eventNamespace,function(){i.disabled||this===r&&e(this).addClass("ui-state-active")}).bind("mouseleave"+this.eventNamespace,function(){i.disabled||e(this).removeClass(n)}).bind("click"+this.eventNamespace,function(e){i.disabled&&(e.preventDefault(),e.stopImmediatePropagation())}),this._on({focus:function(){this.buttonElement.addClass("ui-state-focus")},blur:function(){this.buttonElement.removeClass("ui-state-focus")}}),s&&this.element.bind("change"+this.eventNamespace,function(){t.refresh()}),"checkbox"===this.type?this.buttonElement.bind("click"+this.eventNamespace,function(){return i.disabled?!1:void 0}):"radio"===this.type?this.buttonElement.bind("click"+this.eventNamespace,function(){if(i.disabled)return!1;e(this).addClass("ui-state-active"),t.buttonElement.attr("aria-pressed","true");var s=t.element[0];d(s).not(s).map(function(){return e(this).button("widget")[0]}).removeClass("ui-state-active").attr("aria-pressed","false")}):(this.buttonElement.bind("mousedown"+this.eventNamespace,function(){return i.disabled?!1:(e(this).addClass("ui-state-active"),r=this,t.document.one("mouseup",function(){r=null}),void 0)}).bind("mouseup"+this.eventNamespace,function(){return i.disabled?!1:(e(this).removeClass("ui-state-active"),void 0)}).bind("keydown"+this.eventNamespace,function(t){return i.disabled?!1:((t.keyCode===e.ui.keyCode.SPACE||t.keyCode===e.ui.keyCode.ENTER)&&e(this).addClass("ui-state-active"),void 0)}).bind("keyup"+this.eventNamespace+" blur"+this.eventNamespace,function(){e(this).removeClass("ui-state-active")}),this.buttonElement.is("a")&&this.buttonElement.keyup(function(t){t.keyCode===e.ui.keyCode.SPACE&&e(this).click()})),this._setOption("disabled",i.disabled),this._resetButton()},_determineButtonType:function(){var e,t,i;this.type=this.element.is("[type=checkbox]")?"checkbox":this.element.is("[type=radio]")?"radio":this.element.is("input")?"input":"button","checkbox"===this.type||"radio"===this.type?(e=this.element.parents().last(),t="label[for='"+this.element.attr("id")+"']",this.buttonElement=e.find(t),this.buttonElement.length||(e=e.length?e.siblings():this.element.siblings(),this.buttonElement=e.filter(t),this.buttonElement.length||(this.buttonElement=e.find(t))),this.element.addClass("ui-helper-hidden-accessible"),i=this.element.is(":checked"),i&&this.buttonElement.addClass("ui-state-active"),this.buttonElement.prop("aria-pressed",i)):this.buttonElement=this.element},widget:function(){return this.buttonElement},_destroy:function(){this.element.removeClass("ui-helper-hidden-accessible"),this.buttonElement.removeClass(h+" ui-state-active "+l).removeAttr("role").removeAttr("aria-pressed").html(this.buttonElement.find(".ui-button-text").html()),this.hasTitle||this.buttonElement.removeAttr("title")},_setOption:function(e,t){return this._super(e,t),"disabled"===e?(this.widget().toggleClass("ui-state-disabled",!!t),this.element.prop("disabled",!!t),t&&("checkbox"===this.type||"radio"===this.type?this.buttonElement.removeClass("ui-state-focus"):this.buttonElement.removeClass("ui-state-focus ui-state-active")),void 0):(this._resetButton(),void 0)},refresh:function(){var t=this.element.is("input, button")?this.element.is(":disabled"):this.element.hasClass("ui-button-disabled");t!==this.options.disabled&&this._setOption("disabled",t),"radio"===this.type?d(this.element[0]).each(function(){e(this).is(":checked")?e(this).button("widget").addClass("ui-state-active").attr("aria-pressed","true"):e(this).button("widget").removeClass("ui-state-active").attr("aria-pressed","false")}):"checkbox"===this.type&&(this.element.is(":checked")?this.buttonElement.addClass("ui-state-active").attr("aria-pressed","true"):this.buttonElement.removeClass("ui-state-active").attr("aria-pressed","false"))},_resetButton:function(){if("input"===this.type)return this.options.label&&this.element.val(this.options.label),void 0;var t=this.buttonElement.removeClass(l),i=e("<span></span>",this.document[0]).addClass("ui-button-text").html(this.options.label).appendTo(t.empty()).text(),s=this.options.icons,n=s.primary&&s.secondary,a=[];s.primary||s.secondary?(this.options.text&&a.push("ui-button-text-icon"+(n?"s":s.primary?"-primary":"-secondary")),s.primary&&t.prepend("<span class='ui-button-icon-primary ui-icon "+s.primary+"'></span>"),s.secondary&&t.append("<span class='ui-button-icon-secondary ui-icon "+s.secondary+"'></span>"),this.options.text||(a.push(n?"ui-button-icons-only":"ui-button-icon-only"),this.hasTitle||t.attr("title",e.trim(i)))):a.push("ui-button-text-only"),t.addClass(a.join(" "))}}),e.widget("ui.buttonset",{version:"1.11.4",options:{items:"button, input[type=button], input[type=submit], input[type=reset], input[type=checkbox], input[type=radio], a, :data(ui-button)"},_create:function(){this.element.addClass("ui-buttonset")},_init:function(){this.refresh()},_setOption:function(e,t){"disabled"===e&&this.buttons.button("option",e,t),this._super(e,t)},refresh:function(){var t="rtl"===this.element.css("direction"),i=this.element.find(this.options.items),s=i.filter(":ui-button");i.not(":ui-button").button(),s.button("refresh"),this.buttons=i.map(function(){return e(this).button("widget")[0]}).removeClass("ui-corner-all ui-corner-left ui-corner-right").filter(":first").addClass(t?"ui-corner-right":"ui-corner-left").end().filter(":last").addClass(t?"ui-corner-left":"ui-corner-right").end().end()},_destroy:function(){this.element.removeClass("ui-buttonset"),this.buttons.map(function(){return e(this).button("widget")[0]}).removeClass("ui-corner-left ui-corner-right").end().button("destroy")}}),e.ui.button,e.widget("ui.dialog",{version:"1.11.4",options:{appendTo:"body",autoOpen:!0,buttons:[],closeOnEscape:!0,closeText:"Close",dialogClass:"",draggable:!0,hide:null,height:"auto",maxHeight:null,maxWidth:null,minHeight:150,minWidth:150,modal:!1,position:{my:"center",at:"center",of:window,collision:"fit",using:function(t){var i=e(this).css(t).offset().top;0>i&&e(this).css("top",t.top-i)}},resizable:!0,show:null,title:null,width:300,beforeClose:null,close:null,drag:null,dragStart:null,dragStop:null,focus:null,open:null,resize:null,resizeStart:null,resizeStop:null},sizeRelatedOptions:{buttons:!0,height:!0,maxHeight:!0,maxWidth:!0,minHeight:!0,minWidth:!0,width:!0},resizableRelatedOptions:{maxHeight:!0,maxWidth:!0,minHeight:!0,minWidth:!0},_create:function(){this.originalCss={display:this.element[0].style.display,width:this.element[0].style.width,minHeight:this.element[0].style.minHeight,maxHeight:this.element[0].style.maxHeight,height:this.element[0].style.height},this.originalPosition={parent:this.element.parent(),index:this.element.parent().children().index(this.element)},this.originalTitle=this.element.attr("title"),this.options.title=this.options.title||this.originalTitle,this._createWrapper(),this.element.show().removeAttr("title").addClass("ui-dialog-content ui-widget-content").appendTo(this.uiDialog),this._createTitlebar(),this._createButtonPane(),this.options.draggable&&e.fn.draggable&&this._makeDraggable(),this.options.resizable&&e.fn.resizable&&this._makeResizable(),this._isOpen=!1,this._trackFocus()},_init:function(){this.options.autoOpen&&this.open()},_appendTo:function(){var t=this.options.appendTo;return t&&(t.jquery||t.nodeType)?e(t):this.document.find(t||"body").eq(0)},_destroy:function(){var e,t=this.originalPosition;this._untrackInstance(),this._destroyOverlay(),this.element.removeUniqueId().removeClass("ui-dialog-content ui-widget-content").css(this.originalCss).detach(),this.uiDialog.stop(!0,!0).remove(),this.originalTitle&&this.element.attr("title",this.originalTitle),e=t.parent.children().eq(t.index),e.length&&e[0]!==this.element[0]?e.before(this.element):t.parent.append(this.element)},widget:function(){return this.uiDialog},disable:e.noop,enable:e.noop,close:function(t){var i,s=this;if(this._isOpen&&this._trigger("beforeClose",t)!==!1){if(this._isOpen=!1,this._focusedElement=null,this._destroyOverlay(),this._untrackInstance(),!this.opener.filter(":focusable").focus().length)try{i=this.document[0].activeElement,i&&"body"!==i.nodeName.toLowerCase()&&e(i).blur()}catch(n){}this._hide(this.uiDialog,this.options.hide,function(){s._trigger("close",t)})}},isOpen:function(){return this._isOpen},moveToTop:function(){this._moveToTop()},_moveToTop:function(t,i){var s=!1,n=this.uiDialog.siblings(".ui-front:visible").map(function(){return+e(this).css("z-index")}).get(),a=Math.max.apply(null,n);return a>=+this.uiDialog.css("z-index")&&(this.uiDialog.css("z-index",a+1),s=!0),s&&!i&&this._trigger("focus",t),s},open:function(){var t=this;return this._isOpen?(this._moveToTop()&&this._focusTabbable(),void 0):(this._isOpen=!0,this.opener=e(this.document[0].activeElement),this._size(),this._position(),this._createOverlay(),this._moveToTop(null,!0),this.overlay&&this.overlay.css("z-index",this.uiDialog.css("z-index")-1),this._show(this.uiDialog,this.options.show,function(){t._focusTabbable(),t._trigger("focus")}),this._makeFocusTarget(),this._trigger("open"),void 0)},_focusTabbable:function(){var e=this._focusedElement;e||(e=this.element.find("[autofocus]")),e.length||(e=this.element.find(":tabbable")),e.length||(e=this.uiDialogButtonPane.find(":tabbable")),e.length||(e=this.uiDialogTitlebarClose.filter(":tabbable")),e.length||(e=this.uiDialog),e.eq(0).focus()},_keepFocus:function(t){function i(){var t=this.document[0].activeElement,i=this.uiDialog[0]===t||e.contains(this.uiDialog[0],t);i||this._focusTabbable()}t.preventDefault(),i.call(this),this._delay(i)},_createWrapper:function(){this.uiDialog=e("<div>").addClass("ui-dialog ui-widget ui-widget-content ui-corner-all ui-front "+this.options.dialogClass).hide().attr({tabIndex:-1,role:"dialog"}).appendTo(this._appendTo()),this._on(this.uiDialog,{keydown:function(t){if(this.options.closeOnEscape&&!t.isDefaultPrevented()&&t.keyCode&&t.keyCode===e.ui.keyCode.ESCAPE)return t.preventDefault(),this.close(t),void 0;if(t.keyCode===e.ui.keyCode.TAB&&!t.isDefaultPrevented()){var i=this.uiDialog.find(":tabbable"),s=i.filter(":first"),n=i.filter(":last");t.target!==n[0]&&t.target!==this.uiDialog[0]||t.shiftKey?t.target!==s[0]&&t.target!==this.uiDialog[0]||!t.shiftKey||(this._delay(function(){n.focus()}),t.preventDefault()):(this._delay(function(){s.focus()}),t.preventDefault())}},mousedown:function(e){this._moveToTop(e)&&this._focusTabbable()}}),this.element.find("[aria-describedby]").length||this.uiDialog.attr({"aria-describedby":this.element.uniqueId().attr("id")})},_createTitlebar:function(){var t;this.uiDialogTitlebar=e("<div>").addClass("ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix").prependTo(this.uiDialog),this._on(this.uiDialogTitlebar,{mousedown:function(t){e(t.target).closest(".ui-dialog-titlebar-close")||this.uiDialog.focus()}}),this.uiDialogTitlebarClose=e("<button type='button'></button>").button({label:this.options.closeText,icons:{primary:"ui-icon-closethick"},text:!1}).addClass("ui-dialog-titlebar-close").appendTo(this.uiDialogTitlebar),this._on(this.uiDialogTitlebarClose,{click:function(e){e.preventDefault(),this.close(e)}}),t=e("<span>").uniqueId().addClass("ui-dialog-title").prependTo(this.uiDialogTitlebar),this._title(t),this.uiDialog.attr({"aria-labelledby":t.attr("id")})},_title:function(e){this.options.title||e.html("&#160;"),e.text(this.options.title)
},_createButtonPane:function(){this.uiDialogButtonPane=e("<div>").addClass("ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"),this.uiButtonSet=e("<div>").addClass("ui-dialog-buttonset").appendTo(this.uiDialogButtonPane),this._createButtons()},_createButtons:function(){var t=this,i=this.options.buttons;return this.uiDialogButtonPane.remove(),this.uiButtonSet.empty(),e.isEmptyObject(i)||e.isArray(i)&&!i.length?(this.uiDialog.removeClass("ui-dialog-buttons"),void 0):(e.each(i,function(i,s){var n,a;s=e.isFunction(s)?{click:s,text:i}:s,s=e.extend({type:"button"},s),n=s.click,s.click=function(){n.apply(t.element[0],arguments)},a={icons:s.icons,text:s.showText},delete s.icons,delete s.showText,e("<button></button>",s).button(a).appendTo(t.uiButtonSet)}),this.uiDialog.addClass("ui-dialog-buttons"),this.uiDialogButtonPane.appendTo(this.uiDialog),void 0)},_makeDraggable:function(){function t(e){return{position:e.position,offset:e.offset}}var i=this,s=this.options;this.uiDialog.draggable({cancel:".ui-dialog-content, .ui-dialog-titlebar-close",handle:".ui-dialog-titlebar",containment:"document",start:function(s,n){e(this).addClass("ui-dialog-dragging"),i._blockFrames(),i._trigger("dragStart",s,t(n))},drag:function(e,s){i._trigger("drag",e,t(s))},stop:function(n,a){var o=a.offset.left-i.document.scrollLeft(),r=a.offset.top-i.document.scrollTop();s.position={my:"left top",at:"left"+(o>=0?"+":"")+o+" "+"top"+(r>=0?"+":"")+r,of:i.window},e(this).removeClass("ui-dialog-dragging"),i._unblockFrames(),i._trigger("dragStop",n,t(a))}})},_makeResizable:function(){function t(e){return{originalPosition:e.originalPosition,originalSize:e.originalSize,position:e.position,size:e.size}}var i=this,s=this.options,n=s.resizable,a=this.uiDialog.css("position"),o="string"==typeof n?n:"n,e,s,w,se,sw,ne,nw";this.uiDialog.resizable({cancel:".ui-dialog-content",containment:"document",alsoResize:this.element,maxWidth:s.maxWidth,maxHeight:s.maxHeight,minWidth:s.minWidth,minHeight:this._minHeight(),handles:o,start:function(s,n){e(this).addClass("ui-dialog-resizing"),i._blockFrames(),i._trigger("resizeStart",s,t(n))},resize:function(e,s){i._trigger("resize",e,t(s))},stop:function(n,a){var o=i.uiDialog.offset(),r=o.left-i.document.scrollLeft(),h=o.top-i.document.scrollTop();s.height=i.uiDialog.height(),s.width=i.uiDialog.width(),s.position={my:"left top",at:"left"+(r>=0?"+":"")+r+" "+"top"+(h>=0?"+":"")+h,of:i.window},e(this).removeClass("ui-dialog-resizing"),i._unblockFrames(),i._trigger("resizeStop",n,t(a))}}).css("position",a)},_trackFocus:function(){this._on(this.widget(),{focusin:function(t){this._makeFocusTarget(),this._focusedElement=e(t.target)}})},_makeFocusTarget:function(){this._untrackInstance(),this._trackingInstances().unshift(this)},_untrackInstance:function(){var t=this._trackingInstances(),i=e.inArray(this,t);-1!==i&&t.splice(i,1)},_trackingInstances:function(){var e=this.document.data("ui-dialog-instances");return e||(e=[],this.document.data("ui-dialog-instances",e)),e},_minHeight:function(){var e=this.options;return"auto"===e.height?e.minHeight:Math.min(e.minHeight,e.height)},_position:function(){var e=this.uiDialog.is(":visible");e||this.uiDialog.show(),this.uiDialog.position(this.options.position),e||this.uiDialog.hide()},_setOptions:function(t){var i=this,s=!1,n={};e.each(t,function(e,t){i._setOption(e,t),e in i.sizeRelatedOptions&&(s=!0),e in i.resizableRelatedOptions&&(n[e]=t)}),s&&(this._size(),this._position()),this.uiDialog.is(":data(ui-resizable)")&&this.uiDialog.resizable("option",n)},_setOption:function(e,t){var i,s,n=this.uiDialog;"dialogClass"===e&&n.removeClass(this.options.dialogClass).addClass(t),"disabled"!==e&&(this._super(e,t),"appendTo"===e&&this.uiDialog.appendTo(this._appendTo()),"buttons"===e&&this._createButtons(),"closeText"===e&&this.uiDialogTitlebarClose.button({label:""+t}),"draggable"===e&&(i=n.is(":data(ui-draggable)"),i&&!t&&n.draggable("destroy"),!i&&t&&this._makeDraggable()),"position"===e&&this._position(),"resizable"===e&&(s=n.is(":data(ui-resizable)"),s&&!t&&n.resizable("destroy"),s&&"string"==typeof t&&n.resizable("option","handles",t),s||t===!1||this._makeResizable()),"title"===e&&this._title(this.uiDialogTitlebar.find(".ui-dialog-title")))},_size:function(){var e,t,i,s=this.options;this.element.show().css({width:"auto",minHeight:0,maxHeight:"none",height:0}),s.minWidth>s.width&&(s.width=s.minWidth),e=this.uiDialog.css({height:"auto",width:s.width}).outerHeight(),t=Math.max(0,s.minHeight-e),i="number"==typeof s.maxHeight?Math.max(0,s.maxHeight-e):"none","auto"===s.height?this.element.css({minHeight:t,maxHeight:i,height:"auto"}):this.element.height(Math.max(0,s.height-e)),this.uiDialog.is(":data(ui-resizable)")&&this.uiDialog.resizable("option","minHeight",this._minHeight())},_blockFrames:function(){this.iframeBlocks=this.document.find("iframe").map(function(){var t=e(this);return e("<div>").css({position:"absolute",width:t.outerWidth(),height:t.outerHeight()}).appendTo(t.parent()).offset(t.offset())[0]})},_unblockFrames:function(){this.iframeBlocks&&(this.iframeBlocks.remove(),delete this.iframeBlocks)},_allowInteraction:function(t){return e(t.target).closest(".ui-dialog").length?!0:!!e(t.target).closest(".ui-datepicker").length},_createOverlay:function(){if(this.options.modal){var t=!0;this._delay(function(){t=!1}),this.document.data("ui-dialog-overlays")||this._on(this.document,{focusin:function(e){t||this._allowInteraction(e)||(e.preventDefault(),this._trackingInstances()[0]._focusTabbable())}}),this.overlay=e("<div>").addClass("ui-widget-overlay ui-front").appendTo(this._appendTo()),this._on(this.overlay,{mousedown:"_keepFocus"}),this.document.data("ui-dialog-overlays",(this.document.data("ui-dialog-overlays")||0)+1)}},_destroyOverlay:function(){if(this.options.modal&&this.overlay){var e=this.document.data("ui-dialog-overlays")-1;e?this.document.data("ui-dialog-overlays",e):this.document.unbind("focusin").removeData("ui-dialog-overlays"),this.overlay.remove(),this.overlay=null}}}),e.widget("ui.progressbar",{version:"1.11.4",options:{max:100,value:0,change:null,complete:null},min:0,_create:function(){this.oldValue=this.options.value=this._constrainedValue(),this.element.addClass("ui-progressbar ui-widget ui-widget-content ui-corner-all").attr({role:"progressbar","aria-valuemin":this.min}),this.valueDiv=e("<div class='ui-progressbar-value ui-widget-header ui-corner-left'></div>").appendTo(this.element),this._refreshValue()},_destroy:function(){this.element.removeClass("ui-progressbar ui-widget ui-widget-content ui-corner-all").removeAttr("role").removeAttr("aria-valuemin").removeAttr("aria-valuemax").removeAttr("aria-valuenow"),this.valueDiv.remove()},value:function(e){return void 0===e?this.options.value:(this.options.value=this._constrainedValue(e),this._refreshValue(),void 0)},_constrainedValue:function(e){return void 0===e&&(e=this.options.value),this.indeterminate=e===!1,"number"!=typeof e&&(e=0),this.indeterminate?!1:Math.min(this.options.max,Math.max(this.min,e))},_setOptions:function(e){var t=e.value;delete e.value,this._super(e),this.options.value=this._constrainedValue(t),this._refreshValue()},_setOption:function(e,t){"max"===e&&(t=Math.max(this.min,t)),"disabled"===e&&this.element.toggleClass("ui-state-disabled",!!t).attr("aria-disabled",t),this._super(e,t)},_percentage:function(){return this.indeterminate?100:100*(this.options.value-this.min)/(this.options.max-this.min)},_refreshValue:function(){var t=this.options.value,i=this._percentage();this.valueDiv.toggle(this.indeterminate||t>this.min).toggleClass("ui-corner-right",t===this.options.max).width(i.toFixed(0)+"%"),this.element.toggleClass("ui-progressbar-indeterminate",this.indeterminate),this.indeterminate?(this.element.removeAttr("aria-valuenow"),this.overlayDiv||(this.overlayDiv=e("<div class='ui-progressbar-overlay'></div>").appendTo(this.valueDiv))):(this.element.attr({"aria-valuemax":this.options.max,"aria-valuenow":t}),this.overlayDiv&&(this.overlayDiv.remove(),this.overlayDiv=null)),this.oldValue!==t&&(this.oldValue=t,this._trigger("change")),t===this.options.max&&this._trigger("complete")}}),e.widget("ui.selectmenu",{version:"1.11.4",defaultElement:"<select>",options:{appendTo:null,disabled:null,icons:{button:"ui-icon-triangle-1-s"},position:{my:"left top",at:"left bottom",collision:"none"},width:null,change:null,close:null,focus:null,open:null,select:null},_create:function(){var e=this.element.uniqueId().attr("id");this.ids={element:e,button:e+"-button",menu:e+"-menu"},this._drawButton(),this._drawMenu(),this.options.disabled&&this.disable()},_drawButton:function(){var t=this;this.label=e("label[for='"+this.ids.element+"']").attr("for",this.ids.button),this._on(this.label,{click:function(e){this.button.focus(),e.preventDefault()}}),this.element.hide(),this.button=e("<span>",{"class":"ui-selectmenu-button ui-widget ui-state-default ui-corner-all",tabindex:this.options.disabled?-1:0,id:this.ids.button,role:"combobox","aria-expanded":"false","aria-autocomplete":"list","aria-owns":this.ids.menu,"aria-haspopup":"true"}).insertAfter(this.element),e("<span>",{"class":"ui-icon "+this.options.icons.button}).prependTo(this.button),this.buttonText=e("<span>",{"class":"ui-selectmenu-text"}).appendTo(this.button),this._setText(this.buttonText,this.element.find("option:selected").text()),this._resizeButton(),this._on(this.button,this._buttonEvents),this.button.one("focusin",function(){t.menuItems||t._refreshMenu()}),this._hoverable(this.button),this._focusable(this.button)},_drawMenu:function(){var t=this;this.menu=e("<ul>",{"aria-hidden":"true","aria-labelledby":this.ids.button,id:this.ids.menu}),this.menuWrap=e("<div>",{"class":"ui-selectmenu-menu ui-front"}).append(this.menu).appendTo(this._appendTo()),this.menuInstance=this.menu.menu({role:"listbox",select:function(e,i){e.preventDefault(),t._setSelection(),t._select(i.item.data("ui-selectmenu-item"),e)},focus:function(e,i){var s=i.item.data("ui-selectmenu-item");null!=t.focusIndex&&s.index!==t.focusIndex&&(t._trigger("focus",e,{item:s}),t.isOpen||t._select(s,e)),t.focusIndex=s.index,t.button.attr("aria-activedescendant",t.menuItems.eq(s.index).attr("id"))}}).menu("instance"),this.menu.addClass("ui-corner-bottom").removeClass("ui-corner-all"),this.menuInstance._off(this.menu,"mouseleave"),this.menuInstance._closeOnDocumentClick=function(){return!1},this.menuInstance._isDivider=function(){return!1}},refresh:function(){this._refreshMenu(),this._setText(this.buttonText,this._getSelectedItem().text()),this.options.width||this._resizeButton()},_refreshMenu:function(){this.menu.empty();var e,t=this.element.find("option");t.length&&(this._parseOptions(t),this._renderMenu(this.menu,this.items),this.menuInstance.refresh(),this.menuItems=this.menu.find("li").not(".ui-selectmenu-optgroup"),e=this._getSelectedItem(),this.menuInstance.focus(null,e),this._setAria(e.data("ui-selectmenu-item")),this._setOption("disabled",this.element.prop("disabled")))},open:function(e){this.options.disabled||(this.menuItems?(this.menu.find(".ui-state-focus").removeClass("ui-state-focus"),this.menuInstance.focus(null,this._getSelectedItem())):this._refreshMenu(),this.isOpen=!0,this._toggleAttr(),this._resizeMenu(),this._position(),this._on(this.document,this._documentClick),this._trigger("open",e))},_position:function(){this.menuWrap.position(e.extend({of:this.button},this.options.position))},close:function(e){this.isOpen&&(this.isOpen=!1,this._toggleAttr(),this.range=null,this._off(this.document),this._trigger("close",e))},widget:function(){return this.button},menuWidget:function(){return this.menu},_renderMenu:function(t,i){var s=this,n="";e.each(i,function(i,a){a.optgroup!==n&&(e("<li>",{"class":"ui-selectmenu-optgroup ui-menu-divider"+(a.element.parent("optgroup").prop("disabled")?" ui-state-disabled":""),text:a.optgroup}).appendTo(t),n=a.optgroup),s._renderItemData(t,a)})},_renderItemData:function(e,t){return this._renderItem(e,t).data("ui-selectmenu-item",t)},_renderItem:function(t,i){var s=e("<li>");return i.disabled&&s.addClass("ui-state-disabled"),this._setText(s,i.label),s.appendTo(t)},_setText:function(e,t){t?e.text(t):e.html("&#160;")},_move:function(e,t){var i,s,n=".ui-menu-item";this.isOpen?i=this.menuItems.eq(this.focusIndex):(i=this.menuItems.eq(this.element[0].selectedIndex),n+=":not(.ui-state-disabled)"),s="first"===e||"last"===e?i["first"===e?"prevAll":"nextAll"](n).eq(-1):i[e+"All"](n).eq(0),s.length&&this.menuInstance.focus(t,s)},_getSelectedItem:function(){return this.menuItems.eq(this.element[0].selectedIndex)},_toggle:function(e){this[this.isOpen?"close":"open"](e)},_setSelection:function(){var e;this.range&&(window.getSelection?(e=window.getSelection(),e.removeAllRanges(),e.addRange(this.range)):this.range.select(),this.button.focus())},_documentClick:{mousedown:function(t){this.isOpen&&(e(t.target).closest(".ui-selectmenu-menu, #"+this.ids.button).length||this.close(t))}},_buttonEvents:{mousedown:function(){var e;window.getSelection?(e=window.getSelection(),e.rangeCount&&(this.range=e.getRangeAt(0))):this.range=document.selection.createRange()},click:function(e){this._setSelection(),this._toggle(e)},keydown:function(t){var i=!0;switch(t.keyCode){case e.ui.keyCode.TAB:case e.ui.keyCode.ESCAPE:this.close(t),i=!1;break;case e.ui.keyCode.ENTER:this.isOpen&&this._selectFocusedItem(t);break;case e.ui.keyCode.UP:t.altKey?this._toggle(t):this._move("prev",t);break;case e.ui.keyCode.DOWN:t.altKey?this._toggle(t):this._move("next",t);break;case e.ui.keyCode.SPACE:this.isOpen?this._selectFocusedItem(t):this._toggle(t);break;case e.ui.keyCode.LEFT:this._move("prev",t);break;case e.ui.keyCode.RIGHT:this._move("next",t);break;case e.ui.keyCode.HOME:case e.ui.keyCode.PAGE_UP:this._move("first",t);break;case e.ui.keyCode.END:case e.ui.keyCode.PAGE_DOWN:this._move("last",t);break;default:this.menu.trigger(t),i=!1}i&&t.preventDefault()}},_selectFocusedItem:function(e){var t=this.menuItems.eq(this.focusIndex);t.hasClass("ui-state-disabled")||this._select(t.data("ui-selectmenu-item"),e)},_select:function(e,t){var i=this.element[0].selectedIndex;this.element[0].selectedIndex=e.index,this._setText(this.buttonText,e.label),this._setAria(e),this._trigger("select",t,{item:e}),e.index!==i&&this._trigger("change",t,{item:e}),this.close(t)},_setAria:function(e){var t=this.menuItems.eq(e.index).attr("id");this.button.attr({"aria-labelledby":t,"aria-activedescendant":t}),this.menu.attr("aria-activedescendant",t)},_setOption:function(e,t){"icons"===e&&this.button.find("span.ui-icon").removeClass(this.options.icons.button).addClass(t.button),this._super(e,t),"appendTo"===e&&this.menuWrap.appendTo(this._appendTo()),"disabled"===e&&(this.menuInstance.option("disabled",t),this.button.toggleClass("ui-state-disabled",t).attr("aria-disabled",t),this.element.prop("disabled",t),t?(this.button.attr("tabindex",-1),this.close()):this.button.attr("tabindex",0)),"width"===e&&this._resizeButton()},_appendTo:function(){var t=this.options.appendTo;return t&&(t=t.jquery||t.nodeType?e(t):this.document.find(t).eq(0)),t&&t[0]||(t=this.element.closest(".ui-front")),t.length||(t=this.document[0].body),t},_toggleAttr:function(){this.button.toggleClass("ui-corner-top",this.isOpen).toggleClass("ui-corner-all",!this.isOpen).attr("aria-expanded",this.isOpen),this.menuWrap.toggleClass("ui-selectmenu-open",this.isOpen),this.menu.attr("aria-hidden",!this.isOpen)},_resizeButton:function(){var e=this.options.width;e||(e=this.element.show().outerWidth(),this.element.hide()),this.button.outerWidth(e)},_resizeMenu:function(){this.menu.outerWidth(Math.max(this.button.outerWidth(),this.menu.width("").outerWidth()+1))},_getCreateOptions:function(){return{disabled:this.element.prop("disabled")}},_parseOptions:function(t){var i=[];t.each(function(t,s){var n=e(s),a=n.parent("optgroup");i.push({element:n,index:t,value:n.val(),label:n.text(),optgroup:a.attr("label")||"",disabled:a.prop("disabled")||n.prop("disabled")})}),this.items=i},_destroy:function(){this.menuWrap.remove(),this.button.remove(),this.element.show(),this.element.removeUniqueId(),this.label.attr("for",this.ids.element)}}),e.widget("ui.slider",e.ui.mouse,{version:"1.11.4",widgetEventPrefix:"slide",options:{animate:!1,distance:0,max:100,min:0,orientation:"horizontal",range:!1,step:1,value:0,values:null,change:null,slide:null,start:null,stop:null},numPages:5,_create:function(){this._keySliding=!1,this._mouseSliding=!1,this._animateOff=!0,this._handleIndex=null,this._detectOrientation(),this._mouseInit(),this._calculateNewMax(),this.element.addClass("ui-slider ui-slider-"+this.orientation+" ui-widget"+" ui-widget-content"+" ui-corner-all"),this._refresh(),this._setOption("disabled",this.options.disabled),this._animateOff=!1},_refresh:function(){this._createRange(),this._createHandles(),this._setupEvents(),this._refreshValue()},_createHandles:function(){var t,i,s=this.options,n=this.element.find(".ui-slider-handle").addClass("ui-state-default ui-corner-all"),a="<span class='ui-slider-handle ui-state-default ui-corner-all' tabindex='0'></span>",o=[];for(i=s.values&&s.values.length||1,n.length>i&&(n.slice(i).remove(),n=n.slice(0,i)),t=n.length;i>t;t++)o.push(a);this.handles=n.add(e(o.join("")).appendTo(this.element)),this.handle=this.handles.eq(0),this.handles.each(function(t){e(this).data("ui-slider-handle-index",t)})},_createRange:function(){var t=this.options,i="";t.range?(t.range===!0&&(t.values?t.values.length&&2!==t.values.length?t.values=[t.values[0],t.values[0]]:e.isArray(t.values)&&(t.values=t.values.slice(0)):t.values=[this._valueMin(),this._valueMin()]),this.range&&this.range.length?this.range.removeClass("ui-slider-range-min ui-slider-range-max").css({left:"",bottom:""}):(this.range=e("<div></div>").appendTo(this.element),i="ui-slider-range ui-widget-header ui-corner-all"),this.range.addClass(i+("min"===t.range||"max"===t.range?" ui-slider-range-"+t.range:""))):(this.range&&this.range.remove(),this.range=null)},_setupEvents:function(){this._off(this.handles),this._on(this.handles,this._handleEvents),this._hoverable(this.handles),this._focusable(this.handles)},_destroy:function(){this.handles.remove(),this.range&&this.range.remove(),this.element.removeClass("ui-slider ui-slider-horizontal ui-slider-vertical ui-widget ui-widget-content ui-corner-all"),this._mouseDestroy()},_mouseCapture:function(t){var i,s,n,a,o,r,h,l,u=this,d=this.options;return d.disabled?!1:(this.elementSize={width:this.element.outerWidth(),height:this.element.outerHeight()},this.elementOffset=this.element.offset(),i={x:t.pageX,y:t.pageY},s=this._normValueFromMouse(i),n=this._valueMax()-this._valueMin()+1,this.handles.each(function(t){var i=Math.abs(s-u.values(t));(n>i||n===i&&(t===u._lastChangedValue||u.values(t)===d.min))&&(n=i,a=e(this),o=t)}),r=this._start(t,o),r===!1?!1:(this._mouseSliding=!0,this._handleIndex=o,a.addClass("ui-state-active").focus(),h=a.offset(),l=!e(t.target).parents().addBack().is(".ui-slider-handle"),this._clickOffset=l?{left:0,top:0}:{left:t.pageX-h.left-a.width()/2,top:t.pageY-h.top-a.height()/2-(parseInt(a.css("borderTopWidth"),10)||0)-(parseInt(a.css("borderBottomWidth"),10)||0)+(parseInt(a.css("marginTop"),10)||0)},this.handles.hasClass("ui-state-hover")||this._slide(t,o,s),this._animateOff=!0,!0))},_mouseStart:function(){return!0},_mouseDrag:function(e){var t={x:e.pageX,y:e.pageY},i=this._normValueFromMouse(t);return this._slide(e,this._handleIndex,i),!1},_mouseStop:function(e){return this.handles.removeClass("ui-state-active"),this._mouseSliding=!1,this._stop(e,this._handleIndex),this._change(e,this._handleIndex),this._handleIndex=null,this._clickOffset=null,this._animateOff=!1,!1},_detectOrientation:function(){this.orientation="vertical"===this.options.orientation?"vertical":"horizontal"},_normValueFromMouse:function(e){var t,i,s,n,a;return"horizontal"===this.orientation?(t=this.elementSize.width,i=e.x-this.elementOffset.left-(this._clickOffset?this._clickOffset.left:0)):(t=this.elementSize.height,i=e.y-this.elementOffset.top-(this._clickOffset?this._clickOffset.top:0)),s=i/t,s>1&&(s=1),0>s&&(s=0),"vertical"===this.orientation&&(s=1-s),n=this._valueMax()-this._valueMin(),a=this._valueMin()+s*n,this._trimAlignValue(a)},_start:function(e,t){var i={handle:this.handles[t],value:this.value()};return this.options.values&&this.options.values.length&&(i.value=this.values(t),i.values=this.values()),this._trigger("start",e,i)},_slide:function(e,t,i){var s,n,a;this.options.values&&this.options.values.length?(s=this.values(t?0:1),2===this.options.values.length&&this.options.range===!0&&(0===t&&i>s||1===t&&s>i)&&(i=s),i!==this.values(t)&&(n=this.values(),n[t]=i,a=this._trigger("slide",e,{handle:this.handles[t],value:i,values:n}),s=this.values(t?0:1),a!==!1&&this.values(t,i))):i!==this.value()&&(a=this._trigger("slide",e,{handle:this.handles[t],value:i}),a!==!1&&this.value(i))},_stop:function(e,t){var i={handle:this.handles[t],value:this.value()};this.options.values&&this.options.values.length&&(i.value=this.values(t),i.values=this.values()),this._trigger("stop",e,i)},_change:function(e,t){if(!this._keySliding&&!this._mouseSliding){var i={handle:this.handles[t],value:this.value()};this.options.values&&this.options.values.length&&(i.value=this.values(t),i.values=this.values()),this._lastChangedValue=t,this._trigger("change",e,i)}},value:function(e){return arguments.length?(this.options.value=this._trimAlignValue(e),this._refreshValue(),this._change(null,0),void 0):this._value()},values:function(t,i){var s,n,a;if(arguments.length>1)return this.options.values[t]=this._trimAlignValue(i),this._refreshValue(),this._change(null,t),void 0;if(!arguments.length)return this._values();if(!e.isArray(arguments[0]))return this.options.values&&this.options.values.length?this._values(t):this.value();for(s=this.options.values,n=arguments[0],a=0;s.length>a;a+=1)s[a]=this._trimAlignValue(n[a]),this._change(null,a);this._refreshValue()},_setOption:function(t,i){var s,n=0;switch("range"===t&&this.options.range===!0&&("min"===i?(this.options.value=this._values(0),this.options.values=null):"max"===i&&(this.options.value=this._values(this.options.values.length-1),this.options.values=null)),e.isArray(this.options.values)&&(n=this.options.values.length),"disabled"===t&&this.element.toggleClass("ui-state-disabled",!!i),this._super(t,i),t){case"orientation":this._detectOrientation(),this.element.removeClass("ui-slider-horizontal ui-slider-vertical").addClass("ui-slider-"+this.orientation),this._refreshValue(),this.handles.css("horizontal"===i?"bottom":"left","");break;case"value":this._animateOff=!0,this._refreshValue(),this._change(null,0),this._animateOff=!1;break;case"values":for(this._animateOff=!0,this._refreshValue(),s=0;n>s;s+=1)this._change(null,s);this._animateOff=!1;break;case"step":case"min":case"max":this._animateOff=!0,this._calculateNewMax(),this._refreshValue(),this._animateOff=!1;break;case"range":this._animateOff=!0,this._refresh(),this._animateOff=!1}},_value:function(){var e=this.options.value;return e=this._trimAlignValue(e)},_values:function(e){var t,i,s;if(arguments.length)return t=this.options.values[e],t=this._trimAlignValue(t);if(this.options.values&&this.options.values.length){for(i=this.options.values.slice(),s=0;i.length>s;s+=1)i[s]=this._trimAlignValue(i[s]);return i}return[]},_trimAlignValue:function(e){if(this._valueMin()>=e)return this._valueMin();if(e>=this._valueMax())return this._valueMax();var t=this.options.step>0?this.options.step:1,i=(e-this._valueMin())%t,s=e-i;return 2*Math.abs(i)>=t&&(s+=i>0?t:-t),parseFloat(s.toFixed(5))},_calculateNewMax:function(){var e=this.options.max,t=this._valueMin(),i=this.options.step,s=Math.floor(+(e-t).toFixed(this._precision())/i)*i;e=s+t,this.max=parseFloat(e.toFixed(this._precision()))},_precision:function(){var e=this._precisionOf(this.options.step);return null!==this.options.min&&(e=Math.max(e,this._precisionOf(this.options.min))),e},_precisionOf:function(e){var t=""+e,i=t.indexOf(".");return-1===i?0:t.length-i-1},_valueMin:function(){return this.options.min},_valueMax:function(){return this.max},_refreshValue:function(){var t,i,s,n,a,o=this.options.range,r=this.options,h=this,l=this._animateOff?!1:r.animate,u={};this.options.values&&this.options.values.length?this.handles.each(function(s){i=100*((h.values(s)-h._valueMin())/(h._valueMax()-h._valueMin())),u["horizontal"===h.orientation?"left":"bottom"]=i+"%",e(this).stop(1,1)[l?"animate":"css"](u,r.animate),h.options.range===!0&&("horizontal"===h.orientation?(0===s&&h.range.stop(1,1)[l?"animate":"css"]({left:i+"%"},r.animate),1===s&&h.range[l?"animate":"css"]({width:i-t+"%"},{queue:!1,duration:r.animate})):(0===s&&h.range.stop(1,1)[l?"animate":"css"]({bottom:i+"%"},r.animate),1===s&&h.range[l?"animate":"css"]({height:i-t+"%"},{queue:!1,duration:r.animate}))),t=i}):(s=this.value(),n=this._valueMin(),a=this._valueMax(),i=a!==n?100*((s-n)/(a-n)):0,u["horizontal"===this.orientation?"left":"bottom"]=i+"%",this.handle.stop(1,1)[l?"animate":"css"](u,r.animate),"min"===o&&"horizontal"===this.orientation&&this.range.stop(1,1)[l?"animate":"css"]({width:i+"%"},r.animate),"max"===o&&"horizontal"===this.orientation&&this.range[l?"animate":"css"]({width:100-i+"%"},{queue:!1,duration:r.animate}),"min"===o&&"vertical"===this.orientation&&this.range.stop(1,1)[l?"animate":"css"]({height:i+"%"},r.animate),"max"===o&&"vertical"===this.orientation&&this.range[l?"animate":"css"]({height:100-i+"%"},{queue:!1,duration:r.animate}))},_handleEvents:{keydown:function(t){var i,s,n,a,o=e(t.target).data("ui-slider-handle-index");switch(t.keyCode){case e.ui.keyCode.HOME:case e.ui.keyCode.END:case e.ui.keyCode.PAGE_UP:case e.ui.keyCode.PAGE_DOWN:case e.ui.keyCode.UP:case e.ui.keyCode.RIGHT:case e.ui.keyCode.DOWN:case e.ui.keyCode.LEFT:if(t.preventDefault(),!this._keySliding&&(this._keySliding=!0,e(t.target).addClass("ui-state-active"),i=this._start(t,o),i===!1))return}switch(a=this.options.step,s=n=this.options.values&&this.options.values.length?this.values(o):this.value(),t.keyCode){case e.ui.keyCode.HOME:n=this._valueMin();break;case e.ui.keyCode.END:n=this._valueMax();break;case e.ui.keyCode.PAGE_UP:n=this._trimAlignValue(s+(this._valueMax()-this._valueMin())/this.numPages);break;case e.ui.keyCode.PAGE_DOWN:n=this._trimAlignValue(s-(this._valueMax()-this._valueMin())/this.numPages);break;case e.ui.keyCode.UP:case e.ui.keyCode.RIGHT:if(s===this._valueMax())return;n=this._trimAlignValue(s+a);break;case e.ui.keyCode.DOWN:case e.ui.keyCode.LEFT:if(s===this._valueMin())return;n=this._trimAlignValue(s-a)}this._slide(t,o,n)},keyup:function(t){var i=e(t.target).data("ui-slider-handle-index");this._keySliding&&(this._keySliding=!1,this._stop(t,i),this._change(t,i),e(t.target).removeClass("ui-state-active"))}}}),e.widget("ui.spinner",{version:"1.11.4",defaultElement:"<input>",widgetEventPrefix:"spin",options:{culture:null,icons:{down:"ui-icon-triangle-1-s",up:"ui-icon-triangle-1-n"},incremental:!0,max:null,min:null,numberFormat:null,page:10,step:1,change:null,spin:null,start:null,stop:null},_create:function(){this._setOption("max",this.options.max),this._setOption("min",this.options.min),this._setOption("step",this.options.step),""!==this.value()&&this._value(this.element.val(),!0),this._draw(),this._on(this._events),this._refresh(),this._on(this.window,{beforeunload:function(){this.element.removeAttr("autocomplete")}})},_getCreateOptions:function(){var t={},i=this.element;return e.each(["min","max","step"],function(e,s){var n=i.attr(s);void 0!==n&&n.length&&(t[s]=n)}),t},_events:{keydown:function(e){this._start(e)&&this._keydown(e)&&e.preventDefault()},keyup:"_stop",focus:function(){this.previous=this.element.val()},blur:function(e){return this.cancelBlur?(delete this.cancelBlur,void 0):(this._stop(),this._refresh(),this.previous!==this.element.val()&&this._trigger("change",e),void 0)},mousewheel:function(e,t){if(t){if(!this.spinning&&!this._start(e))return!1;this._spin((t>0?1:-1)*this.options.step,e),clearTimeout(this.mousewheelTimer),this.mousewheelTimer=this._delay(function(){this.spinning&&this._stop(e)},100),e.preventDefault()}},"mousedown .ui-spinner-button":function(t){function i(){var e=this.element[0]===this.document[0].activeElement;e||(this.element.focus(),this.previous=s,this._delay(function(){this.previous=s}))}var s;s=this.element[0]===this.document[0].activeElement?this.previous:this.element.val(),t.preventDefault(),i.call(this),this.cancelBlur=!0,this._delay(function(){delete this.cancelBlur,i.call(this)}),this._start(t)!==!1&&this._repeat(null,e(t.currentTarget).hasClass("ui-spinner-up")?1:-1,t)},"mouseup .ui-spinner-button":"_stop","mouseenter .ui-spinner-button":function(t){return e(t.currentTarget).hasClass("ui-state-active")?this._start(t)===!1?!1:(this._repeat(null,e(t.currentTarget).hasClass("ui-spinner-up")?1:-1,t),void 0):void 0},"mouseleave .ui-spinner-button":"_stop"},_draw:function(){var e=this.uiSpinner=this.element.addClass("ui-spinner-input").attr("autocomplete","off").wrap(this._uiSpinnerHtml()).parent().append(this._buttonHtml());this.element.attr("role","spinbutton"),this.buttons=e.find(".ui-spinner-button").attr("tabIndex",-1).button().removeClass("ui-corner-all"),this.buttons.height()>Math.ceil(.5*e.height())&&e.height()>0&&e.height(e.height()),this.options.disabled&&this.disable()},_keydown:function(t){var i=this.options,s=e.ui.keyCode;switch(t.keyCode){case s.UP:return this._repeat(null,1,t),!0;case s.DOWN:return this._repeat(null,-1,t),!0;case s.PAGE_UP:return this._repeat(null,i.page,t),!0;case s.PAGE_DOWN:return this._repeat(null,-i.page,t),!0}return!1},_uiSpinnerHtml:function(){return"<span class='ui-spinner ui-widget ui-widget-content ui-corner-all'></span>"},_buttonHtml:function(){return"<a class='ui-spinner-button ui-spinner-up ui-corner-tr'><span class='ui-icon "+this.options.icons.up+"'>&#9650;</span>"+"</a>"+"<a class='ui-spinner-button ui-spinner-down ui-corner-br'>"+"<span class='ui-icon "+this.options.icons.down+"'>&#9660;</span>"+"</a>"},_start:function(e){return this.spinning||this._trigger("start",e)!==!1?(this.counter||(this.counter=1),this.spinning=!0,!0):!1},_repeat:function(e,t,i){e=e||500,clearTimeout(this.timer),this.timer=this._delay(function(){this._repeat(40,t,i)},e),this._spin(t*this.options.step,i)},_spin:function(e,t){var i=this.value()||0;this.counter||(this.counter=1),i=this._adjustValue(i+e*this._increment(this.counter)),this.spinning&&this._trigger("spin",t,{value:i})===!1||(this._value(i),this.counter++)},_increment:function(t){var i=this.options.incremental;return i?e.isFunction(i)?i(t):Math.floor(t*t*t/5e4-t*t/500+17*t/200+1):1},_precision:function(){var e=this._precisionOf(this.options.step);return null!==this.options.min&&(e=Math.max(e,this._precisionOf(this.options.min))),e},_precisionOf:function(e){var t=""+e,i=t.indexOf(".");return-1===i?0:t.length-i-1},_adjustValue:function(e){var t,i,s=this.options;return t=null!==s.min?s.min:0,i=e-t,i=Math.round(i/s.step)*s.step,e=t+i,e=parseFloat(e.toFixed(this._precision())),null!==s.max&&e>s.max?s.max:null!==s.min&&s.min>e?s.min:e},_stop:function(e){this.spinning&&(clearTimeout(this.timer),clearTimeout(this.mousewheelTimer),this.counter=0,this.spinning=!1,this._trigger("stop",e))},_setOption:function(e,t){if("culture"===e||"numberFormat"===e){var i=this._parse(this.element.val());return this.options[e]=t,this.element.val(this._format(i)),void 0}("max"===e||"min"===e||"step"===e)&&"string"==typeof t&&(t=this._parse(t)),"icons"===e&&(this.buttons.first().find(".ui-icon").removeClass(this.options.icons.up).addClass(t.up),this.buttons.last().find(".ui-icon").removeClass(this.options.icons.down).addClass(t.down)),this._super(e,t),"disabled"===e&&(this.widget().toggleClass("ui-state-disabled",!!t),this.element.prop("disabled",!!t),this.buttons.button(t?"disable":"enable"))},_setOptions:s(function(e){this._super(e)}),_parse:function(e){return"string"==typeof e&&""!==e&&(e=window.Globalize&&this.options.numberFormat?Globalize.parseFloat(e,10,this.options.culture):+e),""===e||isNaN(e)?null:e
},_format:function(e){return""===e?"":window.Globalize&&this.options.numberFormat?Globalize.format(e,this.options.numberFormat,this.options.culture):e},_refresh:function(){this.element.attr({"aria-valuemin":this.options.min,"aria-valuemax":this.options.max,"aria-valuenow":this._parse(this.element.val())})},isValid:function(){var e=this.value();return null===e?!1:e===this._adjustValue(e)},_value:function(e,t){var i;""!==e&&(i=this._parse(e),null!==i&&(t||(i=this._adjustValue(i)),e=this._format(i))),this.element.val(e),this._refresh()},_destroy:function(){this.element.removeClass("ui-spinner-input").prop("disabled",!1).removeAttr("autocomplete").removeAttr("role").removeAttr("aria-valuemin").removeAttr("aria-valuemax").removeAttr("aria-valuenow"),this.uiSpinner.replaceWith(this.element)},stepUp:s(function(e){this._stepUp(e)}),_stepUp:function(e){this._start()&&(this._spin((e||1)*this.options.step),this._stop())},stepDown:s(function(e){this._stepDown(e)}),_stepDown:function(e){this._start()&&(this._spin((e||1)*-this.options.step),this._stop())},pageUp:s(function(e){this._stepUp((e||1)*this.options.page)}),pageDown:s(function(e){this._stepDown((e||1)*this.options.page)}),value:function(e){return arguments.length?(s(this._value).call(this,e),void 0):this._parse(this.element.val())},widget:function(){return this.uiSpinner}}),e.widget("ui.tabs",{version:"1.11.4",delay:300,options:{active:null,collapsible:!1,event:"click",heightStyle:"content",hide:null,show:null,activate:null,beforeActivate:null,beforeLoad:null,load:null},_isLocal:function(){var e=/#.*$/;return function(t){var i,s;t=t.cloneNode(!1),i=t.href.replace(e,""),s=location.href.replace(e,"");try{i=decodeURIComponent(i)}catch(n){}try{s=decodeURIComponent(s)}catch(n){}return t.hash.length>1&&i===s}}(),_create:function(){var t=this,i=this.options;this.running=!1,this.element.addClass("ui-tabs ui-widget ui-widget-content ui-corner-all").toggleClass("ui-tabs-collapsible",i.collapsible),this._processTabs(),i.active=this._initialActive(),e.isArray(i.disabled)&&(i.disabled=e.unique(i.disabled.concat(e.map(this.tabs.filter(".ui-state-disabled"),function(e){return t.tabs.index(e)}))).sort()),this.active=this.options.active!==!1&&this.anchors.length?this._findActive(i.active):e(),this._refresh(),this.active.length&&this.load(i.active)},_initialActive:function(){var t=this.options.active,i=this.options.collapsible,s=location.hash.substring(1);return null===t&&(s&&this.tabs.each(function(i,n){return e(n).attr("aria-controls")===s?(t=i,!1):void 0}),null===t&&(t=this.tabs.index(this.tabs.filter(".ui-tabs-active"))),(null===t||-1===t)&&(t=this.tabs.length?0:!1)),t!==!1&&(t=this.tabs.index(this.tabs.eq(t)),-1===t&&(t=i?!1:0)),!i&&t===!1&&this.anchors.length&&(t=0),t},_getCreateEventData:function(){return{tab:this.active,panel:this.active.length?this._getPanelForTab(this.active):e()}},_tabKeydown:function(t){var i=e(this.document[0].activeElement).closest("li"),s=this.tabs.index(i),n=!0;if(!this._handlePageNav(t)){switch(t.keyCode){case e.ui.keyCode.RIGHT:case e.ui.keyCode.DOWN:s++;break;case e.ui.keyCode.UP:case e.ui.keyCode.LEFT:n=!1,s--;break;case e.ui.keyCode.END:s=this.anchors.length-1;break;case e.ui.keyCode.HOME:s=0;break;case e.ui.keyCode.SPACE:return t.preventDefault(),clearTimeout(this.activating),this._activate(s),void 0;case e.ui.keyCode.ENTER:return t.preventDefault(),clearTimeout(this.activating),this._activate(s===this.options.active?!1:s),void 0;default:return}t.preventDefault(),clearTimeout(this.activating),s=this._focusNextTab(s,n),t.ctrlKey||t.metaKey||(i.attr("aria-selected","false"),this.tabs.eq(s).attr("aria-selected","true"),this.activating=this._delay(function(){this.option("active",s)},this.delay))}},_panelKeydown:function(t){this._handlePageNav(t)||t.ctrlKey&&t.keyCode===e.ui.keyCode.UP&&(t.preventDefault(),this.active.focus())},_handlePageNav:function(t){return t.altKey&&t.keyCode===e.ui.keyCode.PAGE_UP?(this._activate(this._focusNextTab(this.options.active-1,!1)),!0):t.altKey&&t.keyCode===e.ui.keyCode.PAGE_DOWN?(this._activate(this._focusNextTab(this.options.active+1,!0)),!0):void 0},_findNextTab:function(t,i){function s(){return t>n&&(t=0),0>t&&(t=n),t}for(var n=this.tabs.length-1;-1!==e.inArray(s(),this.options.disabled);)t=i?t+1:t-1;return t},_focusNextTab:function(e,t){return e=this._findNextTab(e,t),this.tabs.eq(e).focus(),e},_setOption:function(e,t){return"active"===e?(this._activate(t),void 0):"disabled"===e?(this._setupDisabled(t),void 0):(this._super(e,t),"collapsible"===e&&(this.element.toggleClass("ui-tabs-collapsible",t),t||this.options.active!==!1||this._activate(0)),"event"===e&&this._setupEvents(t),"heightStyle"===e&&this._setupHeightStyle(t),void 0)},_sanitizeSelector:function(e){return e?e.replace(/[!"$%&'()*+,.\/:;<=>?@\[\]\^`{|}~]/g,"\\$&"):""},refresh:function(){var t=this.options,i=this.tablist.children(":has(a[href])");t.disabled=e.map(i.filter(".ui-state-disabled"),function(e){return i.index(e)}),this._processTabs(),t.active!==!1&&this.anchors.length?this.active.length&&!e.contains(this.tablist[0],this.active[0])?this.tabs.length===t.disabled.length?(t.active=!1,this.active=e()):this._activate(this._findNextTab(Math.max(0,t.active-1),!1)):t.active=this.tabs.index(this.active):(t.active=!1,this.active=e()),this._refresh()},_refresh:function(){this._setupDisabled(this.options.disabled),this._setupEvents(this.options.event),this._setupHeightStyle(this.options.heightStyle),this.tabs.not(this.active).attr({"aria-selected":"false","aria-expanded":"false",tabIndex:-1}),this.panels.not(this._getPanelForTab(this.active)).hide().attr({"aria-hidden":"true"}),this.active.length?(this.active.addClass("ui-tabs-active ui-state-active").attr({"aria-selected":"true","aria-expanded":"true",tabIndex:0}),this._getPanelForTab(this.active).show().attr({"aria-hidden":"false"})):this.tabs.eq(0).attr("tabIndex",0)},_processTabs:function(){var t=this,i=this.tabs,s=this.anchors,n=this.panels;this.tablist=this._getList().addClass("ui-tabs-nav ui-helper-reset ui-helper-clearfix ui-widget-header ui-corner-all").attr("role","tablist").delegate("> li","mousedown"+this.eventNamespace,function(t){e(this).is(".ui-state-disabled")&&t.preventDefault()}).delegate(".ui-tabs-anchor","focus"+this.eventNamespace,function(){e(this).closest("li").is(".ui-state-disabled")&&this.blur()}),this.tabs=this.tablist.find("> li:has(a[href])").addClass("ui-state-default ui-corner-top").attr({role:"tab",tabIndex:-1}),this.anchors=this.tabs.map(function(){return e("a",this)[0]}).addClass("ui-tabs-anchor").attr({role:"presentation",tabIndex:-1}),this.panels=e(),this.anchors.each(function(i,s){var n,a,o,r=e(s).uniqueId().attr("id"),h=e(s).closest("li"),l=h.attr("aria-controls");t._isLocal(s)?(n=s.hash,o=n.substring(1),a=t.element.find(t._sanitizeSelector(n))):(o=h.attr("aria-controls")||e({}).uniqueId()[0].id,n="#"+o,a=t.element.find(n),a.length||(a=t._createPanel(o),a.insertAfter(t.panels[i-1]||t.tablist)),a.attr("aria-live","polite")),a.length&&(t.panels=t.panels.add(a)),l&&h.data("ui-tabs-aria-controls",l),h.attr({"aria-controls":o,"aria-labelledby":r}),a.attr("aria-labelledby",r)}),this.panels.addClass("ui-tabs-panel ui-widget-content ui-corner-bottom").attr("role","tabpanel"),i&&(this._off(i.not(this.tabs)),this._off(s.not(this.anchors)),this._off(n.not(this.panels)))},_getList:function(){return this.tablist||this.element.find("ol,ul").eq(0)},_createPanel:function(t){return e("<div>").attr("id",t).addClass("ui-tabs-panel ui-widget-content ui-corner-bottom").data("ui-tabs-destroy",!0)},_setupDisabled:function(t){e.isArray(t)&&(t.length?t.length===this.anchors.length&&(t=!0):t=!1);for(var i,s=0;i=this.tabs[s];s++)t===!0||-1!==e.inArray(s,t)?e(i).addClass("ui-state-disabled").attr("aria-disabled","true"):e(i).removeClass("ui-state-disabled").removeAttr("aria-disabled");this.options.disabled=t},_setupEvents:function(t){var i={};t&&e.each(t.split(" "),function(e,t){i[t]="_eventHandler"}),this._off(this.anchors.add(this.tabs).add(this.panels)),this._on(!0,this.anchors,{click:function(e){e.preventDefault()}}),this._on(this.anchors,i),this._on(this.tabs,{keydown:"_tabKeydown"}),this._on(this.panels,{keydown:"_panelKeydown"}),this._focusable(this.tabs),this._hoverable(this.tabs)},_setupHeightStyle:function(t){var i,s=this.element.parent();"fill"===t?(i=s.height(),i-=this.element.outerHeight()-this.element.height(),this.element.siblings(":visible").each(function(){var t=e(this),s=t.css("position");"absolute"!==s&&"fixed"!==s&&(i-=t.outerHeight(!0))}),this.element.children().not(this.panels).each(function(){i-=e(this).outerHeight(!0)}),this.panels.each(function(){e(this).height(Math.max(0,i-e(this).innerHeight()+e(this).height()))}).css("overflow","auto")):"auto"===t&&(i=0,this.panels.each(function(){i=Math.max(i,e(this).height("").height())}).height(i))},_eventHandler:function(t){var i=this.options,s=this.active,n=e(t.currentTarget),a=n.closest("li"),o=a[0]===s[0],r=o&&i.collapsible,h=r?e():this._getPanelForTab(a),l=s.length?this._getPanelForTab(s):e(),u={oldTab:s,oldPanel:l,newTab:r?e():a,newPanel:h};t.preventDefault(),a.hasClass("ui-state-disabled")||a.hasClass("ui-tabs-loading")||this.running||o&&!i.collapsible||this._trigger("beforeActivate",t,u)===!1||(i.active=r?!1:this.tabs.index(a),this.active=o?e():a,this.xhr&&this.xhr.abort(),l.length||h.length||e.error("jQuery UI Tabs: Mismatching fragment identifier."),h.length&&this.load(this.tabs.index(a),t),this._toggle(t,u))},_toggle:function(t,i){function s(){a.running=!1,a._trigger("activate",t,i)}function n(){i.newTab.closest("li").addClass("ui-tabs-active ui-state-active"),o.length&&a.options.show?a._show(o,a.options.show,s):(o.show(),s())}var a=this,o=i.newPanel,r=i.oldPanel;this.running=!0,r.length&&this.options.hide?this._hide(r,this.options.hide,function(){i.oldTab.closest("li").removeClass("ui-tabs-active ui-state-active"),n()}):(i.oldTab.closest("li").removeClass("ui-tabs-active ui-state-active"),r.hide(),n()),r.attr("aria-hidden","true"),i.oldTab.attr({"aria-selected":"false","aria-expanded":"false"}),o.length&&r.length?i.oldTab.attr("tabIndex",-1):o.length&&this.tabs.filter(function(){return 0===e(this).attr("tabIndex")}).attr("tabIndex",-1),o.attr("aria-hidden","false"),i.newTab.attr({"aria-selected":"true","aria-expanded":"true",tabIndex:0})},_activate:function(t){var i,s=this._findActive(t);s[0]!==this.active[0]&&(s.length||(s=this.active),i=s.find(".ui-tabs-anchor")[0],this._eventHandler({target:i,currentTarget:i,preventDefault:e.noop}))},_findActive:function(t){return t===!1?e():this.tabs.eq(t)},_getIndex:function(e){return"string"==typeof e&&(e=this.anchors.index(this.anchors.filter("[href$='"+e+"']"))),e},_destroy:function(){this.xhr&&this.xhr.abort(),this.element.removeClass("ui-tabs ui-widget ui-widget-content ui-corner-all ui-tabs-collapsible"),this.tablist.removeClass("ui-tabs-nav ui-helper-reset ui-helper-clearfix ui-widget-header ui-corner-all").removeAttr("role"),this.anchors.removeClass("ui-tabs-anchor").removeAttr("role").removeAttr("tabIndex").removeUniqueId(),this.tablist.unbind(this.eventNamespace),this.tabs.add(this.panels).each(function(){e.data(this,"ui-tabs-destroy")?e(this).remove():e(this).removeClass("ui-state-default ui-state-active ui-state-disabled ui-corner-top ui-corner-bottom ui-widget-content ui-tabs-active ui-tabs-panel").removeAttr("tabIndex").removeAttr("aria-live").removeAttr("aria-busy").removeAttr("aria-selected").removeAttr("aria-labelledby").removeAttr("aria-hidden").removeAttr("aria-expanded").removeAttr("role")}),this.tabs.each(function(){var t=e(this),i=t.data("ui-tabs-aria-controls");i?t.attr("aria-controls",i).removeData("ui-tabs-aria-controls"):t.removeAttr("aria-controls")}),this.panels.show(),"content"!==this.options.heightStyle&&this.panels.css("height","")},enable:function(t){var i=this.options.disabled;i!==!1&&(void 0===t?i=!1:(t=this._getIndex(t),i=e.isArray(i)?e.map(i,function(e){return e!==t?e:null}):e.map(this.tabs,function(e,i){return i!==t?i:null})),this._setupDisabled(i))},disable:function(t){var i=this.options.disabled;if(i!==!0){if(void 0===t)i=!0;else{if(t=this._getIndex(t),-1!==e.inArray(t,i))return;i=e.isArray(i)?e.merge([t],i).sort():[t]}this._setupDisabled(i)}},load:function(t,i){t=this._getIndex(t);var s=this,n=this.tabs.eq(t),a=n.find(".ui-tabs-anchor"),o=this._getPanelForTab(n),r={tab:n,panel:o},h=function(e,t){"abort"===t&&s.panels.stop(!1,!0),n.removeClass("ui-tabs-loading"),o.removeAttr("aria-busy"),e===s.xhr&&delete s.xhr};this._isLocal(a[0])||(this.xhr=e.ajax(this._ajaxSettings(a,i,r)),this.xhr&&"canceled"!==this.xhr.statusText&&(n.addClass("ui-tabs-loading"),o.attr("aria-busy","true"),this.xhr.done(function(e,t,n){setTimeout(function(){o.html(e),s._trigger("load",i,r),h(n,t)},1)}).fail(function(e,t){setTimeout(function(){h(e,t)},1)})))},_ajaxSettings:function(t,i,s){var n=this;return{url:t.attr("href"),beforeSend:function(t,a){return n._trigger("beforeLoad",i,e.extend({jqXHR:t,ajaxSettings:a},s))}}},_getPanelForTab:function(t){var i=e(t).attr("aria-controls");return this.element.find(this._sanitizeSelector("#"+i))}}),e.widget("ui.tooltip",{version:"1.11.4",options:{content:function(){var t=e(this).attr("title")||"";return e("<a>").text(t).html()},hide:!0,items:"[title]:not([disabled])",position:{my:"left top+15",at:"left bottom",collision:"flipfit flip"},show:!0,tooltipClass:null,track:!1,close:null,open:null},_addDescribedBy:function(t,i){var s=(t.attr("aria-describedby")||"").split(/\s+/);s.push(i),t.data("ui-tooltip-id",i).attr("aria-describedby",e.trim(s.join(" ")))},_removeDescribedBy:function(t){var i=t.data("ui-tooltip-id"),s=(t.attr("aria-describedby")||"").split(/\s+/),n=e.inArray(i,s);-1!==n&&s.splice(n,1),t.removeData("ui-tooltip-id"),s=e.trim(s.join(" ")),s?t.attr("aria-describedby",s):t.removeAttr("aria-describedby")},_create:function(){this._on({mouseover:"open",focusin:"open"}),this.tooltips={},this.parents={},this.options.disabled&&this._disable(),this.liveRegion=e("<div>").attr({role:"log","aria-live":"assertive","aria-relevant":"additions"}).addClass("ui-helper-hidden-accessible").appendTo(this.document[0].body)},_setOption:function(t,i){var s=this;return"disabled"===t?(this[i?"_disable":"_enable"](),this.options[t]=i,void 0):(this._super(t,i),"content"===t&&e.each(this.tooltips,function(e,t){s._updateContent(t.element)}),void 0)},_disable:function(){var t=this;e.each(this.tooltips,function(i,s){var n=e.Event("blur");n.target=n.currentTarget=s.element[0],t.close(n,!0)}),this.element.find(this.options.items).addBack().each(function(){var t=e(this);t.is("[title]")&&t.data("ui-tooltip-title",t.attr("title")).removeAttr("title")})},_enable:function(){this.element.find(this.options.items).addBack().each(function(){var t=e(this);t.data("ui-tooltip-title")&&t.attr("title",t.data("ui-tooltip-title"))})},open:function(t){var i=this,s=e(t?t.target:this.element).closest(this.options.items);s.length&&!s.data("ui-tooltip-id")&&(s.attr("title")&&s.data("ui-tooltip-title",s.attr("title")),s.data("ui-tooltip-open",!0),t&&"mouseover"===t.type&&s.parents().each(function(){var t,s=e(this);s.data("ui-tooltip-open")&&(t=e.Event("blur"),t.target=t.currentTarget=this,i.close(t,!0)),s.attr("title")&&(s.uniqueId(),i.parents[this.id]={element:this,title:s.attr("title")},s.attr("title",""))}),this._registerCloseHandlers(t,s),this._updateContent(s,t))},_updateContent:function(e,t){var i,s=this.options.content,n=this,a=t?t.type:null;return"string"==typeof s?this._open(t,e,s):(i=s.call(e[0],function(i){n._delay(function(){e.data("ui-tooltip-open")&&(t&&(t.type=a),this._open(t,e,i))})}),i&&this._open(t,e,i),void 0)},_open:function(t,i,s){function n(e){l.of=e,o.is(":hidden")||o.position(l)}var a,o,r,h,l=e.extend({},this.options.position);if(s){if(a=this._find(i))return a.tooltip.find(".ui-tooltip-content").html(s),void 0;i.is("[title]")&&(t&&"mouseover"===t.type?i.attr("title",""):i.removeAttr("title")),a=this._tooltip(i),o=a.tooltip,this._addDescribedBy(i,o.attr("id")),o.find(".ui-tooltip-content").html(s),this.liveRegion.children().hide(),s.clone?(h=s.clone(),h.removeAttr("id").find("[id]").removeAttr("id")):h=s,e("<div>").html(h).appendTo(this.liveRegion),this.options.track&&t&&/^mouse/.test(t.type)?(this._on(this.document,{mousemove:n}),n(t)):o.position(e.extend({of:i},this.options.position)),o.hide(),this._show(o,this.options.show),this.options.show&&this.options.show.delay&&(r=this.delayedShow=setInterval(function(){o.is(":visible")&&(n(l.of),clearInterval(r))},e.fx.interval)),this._trigger("open",t,{tooltip:o})}},_registerCloseHandlers:function(t,i){var s={keyup:function(t){if(t.keyCode===e.ui.keyCode.ESCAPE){var s=e.Event(t);s.currentTarget=i[0],this.close(s,!0)}}};i[0]!==this.element[0]&&(s.remove=function(){this._removeTooltip(this._find(i).tooltip)}),t&&"mouseover"!==t.type||(s.mouseleave="close"),t&&"focusin"!==t.type||(s.focusout="close"),this._on(!0,i,s)},close:function(t){var i,s=this,n=e(t?t.currentTarget:this.element),a=this._find(n);return a?(i=a.tooltip,a.closing||(clearInterval(this.delayedShow),n.data("ui-tooltip-title")&&!n.attr("title")&&n.attr("title",n.data("ui-tooltip-title")),this._removeDescribedBy(n),a.hiding=!0,i.stop(!0),this._hide(i,this.options.hide,function(){s._removeTooltip(e(this))}),n.removeData("ui-tooltip-open"),this._off(n,"mouseleave focusout keyup"),n[0]!==this.element[0]&&this._off(n,"remove"),this._off(this.document,"mousemove"),t&&"mouseleave"===t.type&&e.each(this.parents,function(t,i){e(i.element).attr("title",i.title),delete s.parents[t]}),a.closing=!0,this._trigger("close",t,{tooltip:i}),a.hiding||(a.closing=!1)),void 0):(n.removeData("ui-tooltip-open"),void 0)},_tooltip:function(t){var i=e("<div>").attr("role","tooltip").addClass("ui-tooltip ui-widget ui-corner-all ui-widget-content "+(this.options.tooltipClass||"")),s=i.uniqueId().attr("id");return e("<div>").addClass("ui-tooltip-content").appendTo(i),i.appendTo(this.document[0].body),this.tooltips[s]={element:t,tooltip:i}},_find:function(e){var t=e.data("ui-tooltip-id");return t?this.tooltips[t]:null},_removeTooltip:function(e){e.remove(),delete this.tooltips[e.attr("id")]},_destroy:function(){var t=this;e.each(this.tooltips,function(i,s){var n=e.Event("blur"),a=s.element;n.target=n.currentTarget=a[0],t.close(n,!0),e("#"+i).remove(),a.data("ui-tooltip-title")&&(a.attr("title")||a.attr("title",a.data("ui-tooltip-title")),a.removeData("ui-tooltip-title"))}),this.liveRegion.remove()}});var c="ui-effects-",p=e;e.effects={effect:{}},function(e,t){function i(e,t,i){var s=d[t.type]||{};return null==e?i||!t.def?null:t.def:(e=s.floor?~~e:parseFloat(e),isNaN(e)?t.def:s.mod?(e+s.mod)%s.mod:0>e?0:e>s.max?s.max:e)}function s(i){var s=l(),n=s._rgba=[];return i=i.toLowerCase(),f(h,function(e,a){var o,r=a.re.exec(i),h=r&&a.parse(r),l=a.space||"rgba";return h?(o=s[l](h),s[u[l].cache]=o[u[l].cache],n=s._rgba=o._rgba,!1):t}),n.length?("0,0,0,0"===n.join()&&e.extend(n,a.transparent),s):a[i]}function n(e,t,i){return i=(i+1)%1,1>6*i?e+6*(t-e)*i:1>2*i?t:2>3*i?e+6*(t-e)*(2/3-i):e}var a,o="backgroundColor borderBottomColor borderLeftColor borderRightColor borderTopColor color columnRuleColor outlineColor textDecorationColor textEmphasisColor",r=/^([\-+])=\s*(\d+\.?\d*)/,h=[{re:/rgba?\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*(?:,\s*(\d?(?:\.\d+)?)\s*)?\)/,parse:function(e){return[e[1],e[2],e[3],e[4]]}},{re:/rgba?\(\s*(\d+(?:\.\d+)?)\%\s*,\s*(\d+(?:\.\d+)?)\%\s*,\s*(\d+(?:\.\d+)?)\%\s*(?:,\s*(\d?(?:\.\d+)?)\s*)?\)/,parse:function(e){return[2.55*e[1],2.55*e[2],2.55*e[3],e[4]]}},{re:/#([a-f0-9]{2})([a-f0-9]{2})([a-f0-9]{2})/,parse:function(e){return[parseInt(e[1],16),parseInt(e[2],16),parseInt(e[3],16)]}},{re:/#([a-f0-9])([a-f0-9])([a-f0-9])/,parse:function(e){return[parseInt(e[1]+e[1],16),parseInt(e[2]+e[2],16),parseInt(e[3]+e[3],16)]}},{re:/hsla?\(\s*(\d+(?:\.\d+)?)\s*,\s*(\d+(?:\.\d+)?)\%\s*,\s*(\d+(?:\.\d+)?)\%\s*(?:,\s*(\d?(?:\.\d+)?)\s*)?\)/,space:"hsla",parse:function(e){return[e[1],e[2]/100,e[3]/100,e[4]]}}],l=e.Color=function(t,i,s,n){return new e.Color.fn.parse(t,i,s,n)},u={rgba:{props:{red:{idx:0,type:"byte"},green:{idx:1,type:"byte"},blue:{idx:2,type:"byte"}}},hsla:{props:{hue:{idx:0,type:"degrees"},saturation:{idx:1,type:"percent"},lightness:{idx:2,type:"percent"}}}},d={"byte":{floor:!0,max:255},percent:{max:1},degrees:{mod:360,floor:!0}},c=l.support={},p=e("<p>")[0],f=e.each;p.style.cssText="background-color:rgba(1,1,1,.5)",c.rgba=p.style.backgroundColor.indexOf("rgba")>-1,f(u,function(e,t){t.cache="_"+e,t.props.alpha={idx:3,type:"percent",def:1}}),l.fn=e.extend(l.prototype,{parse:function(n,o,r,h){if(n===t)return this._rgba=[null,null,null,null],this;(n.jquery||n.nodeType)&&(n=e(n).css(o),o=t);var d=this,c=e.type(n),p=this._rgba=[];return o!==t&&(n=[n,o,r,h],c="array"),"string"===c?this.parse(s(n)||a._default):"array"===c?(f(u.rgba.props,function(e,t){p[t.idx]=i(n[t.idx],t)}),this):"object"===c?(n instanceof l?f(u,function(e,t){n[t.cache]&&(d[t.cache]=n[t.cache].slice())}):f(u,function(t,s){var a=s.cache;f(s.props,function(e,t){if(!d[a]&&s.to){if("alpha"===e||null==n[e])return;d[a]=s.to(d._rgba)}d[a][t.idx]=i(n[e],t,!0)}),d[a]&&0>e.inArray(null,d[a].slice(0,3))&&(d[a][3]=1,s.from&&(d._rgba=s.from(d[a])))}),this):t},is:function(e){var i=l(e),s=!0,n=this;return f(u,function(e,a){var o,r=i[a.cache];return r&&(o=n[a.cache]||a.to&&a.to(n._rgba)||[],f(a.props,function(e,i){return null!=r[i.idx]?s=r[i.idx]===o[i.idx]:t})),s}),s},_space:function(){var e=[],t=this;return f(u,function(i,s){t[s.cache]&&e.push(i)}),e.pop()},transition:function(e,t){var s=l(e),n=s._space(),a=u[n],o=0===this.alpha()?l("transparent"):this,r=o[a.cache]||a.to(o._rgba),h=r.slice();return s=s[a.cache],f(a.props,function(e,n){var a=n.idx,o=r[a],l=s[a],u=d[n.type]||{};null!==l&&(null===o?h[a]=l:(u.mod&&(l-o>u.mod/2?o+=u.mod:o-l>u.mod/2&&(o-=u.mod)),h[a]=i((l-o)*t+o,n)))}),this[n](h)},blend:function(t){if(1===this._rgba[3])return this;var i=this._rgba.slice(),s=i.pop(),n=l(t)._rgba;return l(e.map(i,function(e,t){return(1-s)*n[t]+s*e}))},toRgbaString:function(){var t="rgba(",i=e.map(this._rgba,function(e,t){return null==e?t>2?1:0:e});return 1===i[3]&&(i.pop(),t="rgb("),t+i.join()+")"},toHslaString:function(){var t="hsla(",i=e.map(this.hsla(),function(e,t){return null==e&&(e=t>2?1:0),t&&3>t&&(e=Math.round(100*e)+"%"),e});return 1===i[3]&&(i.pop(),t="hsl("),t+i.join()+")"},toHexString:function(t){var i=this._rgba.slice(),s=i.pop();return t&&i.push(~~(255*s)),"#"+e.map(i,function(e){return e=(e||0).toString(16),1===e.length?"0"+e:e}).join("")},toString:function(){return 0===this._rgba[3]?"transparent":this.toRgbaString()}}),l.fn.parse.prototype=l.fn,u.hsla.to=function(e){if(null==e[0]||null==e[1]||null==e[2])return[null,null,null,e[3]];var t,i,s=e[0]/255,n=e[1]/255,a=e[2]/255,o=e[3],r=Math.max(s,n,a),h=Math.min(s,n,a),l=r-h,u=r+h,d=.5*u;return t=h===r?0:s===r?60*(n-a)/l+360:n===r?60*(a-s)/l+120:60*(s-n)/l+240,i=0===l?0:.5>=d?l/u:l/(2-u),[Math.round(t)%360,i,d,null==o?1:o]},u.hsla.from=function(e){if(null==e[0]||null==e[1]||null==e[2])return[null,null,null,e[3]];var t=e[0]/360,i=e[1],s=e[2],a=e[3],o=.5>=s?s*(1+i):s+i-s*i,r=2*s-o;return[Math.round(255*n(r,o,t+1/3)),Math.round(255*n(r,o,t)),Math.round(255*n(r,o,t-1/3)),a]},f(u,function(s,n){var a=n.props,o=n.cache,h=n.to,u=n.from;l.fn[s]=function(s){if(h&&!this[o]&&(this[o]=h(this._rgba)),s===t)return this[o].slice();var n,r=e.type(s),d="array"===r||"object"===r?s:arguments,c=this[o].slice();return f(a,function(e,t){var s=d["object"===r?e:t.idx];null==s&&(s=c[t.idx]),c[t.idx]=i(s,t)}),u?(n=l(u(c)),n[o]=c,n):l(c)},f(a,function(t,i){l.fn[t]||(l.fn[t]=function(n){var a,o=e.type(n),h="alpha"===t?this._hsla?"hsla":"rgba":s,l=this[h](),u=l[i.idx];return"undefined"===o?u:("function"===o&&(n=n.call(this,u),o=e.type(n)),null==n&&i.empty?this:("string"===o&&(a=r.exec(n),a&&(n=u+parseFloat(a[2])*("+"===a[1]?1:-1))),l[i.idx]=n,this[h](l)))})})}),l.hook=function(t){var i=t.split(" ");f(i,function(t,i){e.cssHooks[i]={set:function(t,n){var a,o,r="";if("transparent"!==n&&("string"!==e.type(n)||(a=s(n)))){if(n=l(a||n),!c.rgba&&1!==n._rgba[3]){for(o="backgroundColor"===i?t.parentNode:t;(""===r||"transparent"===r)&&o&&o.style;)try{r=e.css(o,"backgroundColor"),o=o.parentNode}catch(h){}n=n.blend(r&&"transparent"!==r?r:"_default")}n=n.toRgbaString()}try{t.style[i]=n}catch(h){}}},e.fx.step[i]=function(t){t.colorInit||(t.start=l(t.elem,i),t.end=l(t.end),t.colorInit=!0),e.cssHooks[i].set(t.elem,t.start.transition(t.end,t.pos))}})},l.hook(o),e.cssHooks.borderColor={expand:function(e){var t={};return f(["Top","Right","Bottom","Left"],function(i,s){t["border"+s+"Color"]=e}),t}},a=e.Color.names={aqua:"#00ffff",black:"#000000",blue:"#0000ff",fuchsia:"#ff00ff",gray:"#808080",green:"#008000",lime:"#00ff00",maroon:"#800000",navy:"#000080",olive:"#808000",purple:"#800080",red:"#ff0000",silver:"#c0c0c0",teal:"#008080",white:"#ffffff",yellow:"#ffff00",transparent:[null,null,null,0],_default:"#ffffff"}}(p),function(){function t(t){var i,s,n=t.ownerDocument.defaultView?t.ownerDocument.defaultView.getComputedStyle(t,null):t.currentStyle,a={};if(n&&n.length&&n[0]&&n[n[0]])for(s=n.length;s--;)i=n[s],"string"==typeof n[i]&&(a[e.camelCase(i)]=n[i]);else for(i in n)"string"==typeof n[i]&&(a[i]=n[i]);return a}function i(t,i){var s,a,o={};for(s in i)a=i[s],t[s]!==a&&(n[s]||(e.fx.step[s]||!isNaN(parseFloat(a)))&&(o[s]=a));return o}var s=["add","remove","toggle"],n={border:1,borderBottom:1,borderColor:1,borderLeft:1,borderRight:1,borderTop:1,borderWidth:1,margin:1,padding:1};e.each(["borderLeftStyle","borderRightStyle","borderBottomStyle","borderTopStyle"],function(t,i){e.fx.step[i]=function(e){("none"!==e.end&&!e.setAttr||1===e.pos&&!e.setAttr)&&(p.style(e.elem,i,e.end),e.setAttr=!0)}}),e.fn.addBack||(e.fn.addBack=function(e){return this.add(null==e?this.prevObject:this.prevObject.filter(e))}),e.effects.animateClass=function(n,a,o,r){var h=e.speed(a,o,r);return this.queue(function(){var a,o=e(this),r=o.attr("class")||"",l=h.children?o.find("*").addBack():o;l=l.map(function(){var i=e(this);return{el:i,start:t(this)}}),a=function(){e.each(s,function(e,t){n[t]&&o[t+"Class"](n[t])})},a(),l=l.map(function(){return this.end=t(this.el[0]),this.diff=i(this.start,this.end),this}),o.attr("class",r),l=l.map(function(){var t=this,i=e.Deferred(),s=e.extend({},h,{queue:!1,complete:function(){i.resolve(t)}});return this.el.animate(this.diff,s),i.promise()}),e.when.apply(e,l.get()).done(function(){a(),e.each(arguments,function(){var t=this.el;e.each(this.diff,function(e){t.css(e,"")})}),h.complete.call(o[0])})})},e.fn.extend({addClass:function(t){return function(i,s,n,a){return s?e.effects.animateClass.call(this,{add:i},s,n,a):t.apply(this,arguments)}}(e.fn.addClass),removeClass:function(t){return function(i,s,n,a){return arguments.length>1?e.effects.animateClass.call(this,{remove:i},s,n,a):t.apply(this,arguments)}}(e.fn.removeClass),toggleClass:function(t){return function(i,s,n,a,o){return"boolean"==typeof s||void 0===s?n?e.effects.animateClass.call(this,s?{add:i}:{remove:i},n,a,o):t.apply(this,arguments):e.effects.animateClass.call(this,{toggle:i},s,n,a)}}(e.fn.toggleClass),switchClass:function(t,i,s,n,a){return e.effects.animateClass.call(this,{add:i,remove:t},s,n,a)}})}(),function(){function t(t,i,s,n){return e.isPlainObject(t)&&(i=t,t=t.effect),t={effect:t},null==i&&(i={}),e.isFunction(i)&&(n=i,s=null,i={}),("number"==typeof i||e.fx.speeds[i])&&(n=s,s=i,i={}),e.isFunction(s)&&(n=s,s=null),i&&e.extend(t,i),s=s||i.duration,t.duration=e.fx.off?0:"number"==typeof s?s:s in e.fx.speeds?e.fx.speeds[s]:e.fx.speeds._default,t.complete=n||i.complete,t}function i(t){return!t||"number"==typeof t||e.fx.speeds[t]?!0:"string"!=typeof t||e.effects.effect[t]?e.isFunction(t)?!0:"object"!=typeof t||t.effect?!1:!0:!0}e.extend(e.effects,{version:"1.11.4",save:function(e,t){for(var i=0;t.length>i;i++)null!==t[i]&&e.data(c+t[i],e[0].style[t[i]])},restore:function(e,t){var i,s;for(s=0;t.length>s;s++)null!==t[s]&&(i=e.data(c+t[s]),void 0===i&&(i=""),e.css(t[s],i))},setMode:function(e,t){return"toggle"===t&&(t=e.is(":hidden")?"show":"hide"),t},getBaseline:function(e,t){var i,s;switch(e[0]){case"top":i=0;break;case"middle":i=.5;break;case"bottom":i=1;break;default:i=e[0]/t.height}switch(e[1]){case"left":s=0;break;case"center":s=.5;break;case"right":s=1;break;default:s=e[1]/t.width}return{x:s,y:i}},createWrapper:function(t){if(t.parent().is(".ui-effects-wrapper"))return t.parent();var i={width:t.outerWidth(!0),height:t.outerHeight(!0),"float":t.css("float")},s=e("<div></div>").addClass("ui-effects-wrapper").css({fontSize:"100%",background:"transparent",border:"none",margin:0,padding:0}),n={width:t.width(),height:t.height()},a=document.activeElement;try{a.id}catch(o){a=document.body}return t.wrap(s),(t[0]===a||e.contains(t[0],a))&&e(a).focus(),s=t.parent(),"static"===t.css("position")?(s.css({position:"relative"}),t.css({position:"relative"})):(e.extend(i,{position:t.css("position"),zIndex:t.css("z-index")}),e.each(["top","left","bottom","right"],function(e,s){i[s]=t.css(s),isNaN(parseInt(i[s],10))&&(i[s]="auto")}),t.css({position:"relative",top:0,left:0,right:"auto",bottom:"auto"})),t.css(n),s.css(i).show()},removeWrapper:function(t){var i=document.activeElement;return t.parent().is(".ui-effects-wrapper")&&(t.parent().replaceWith(t),(t[0]===i||e.contains(t[0],i))&&e(i).focus()),t},setTransition:function(t,i,s,n){return n=n||{},e.each(i,function(e,i){var a=t.cssUnit(i);a[0]>0&&(n[i]=a[0]*s+a[1])}),n}}),e.fn.extend({effect:function(){function i(t){function i(){e.isFunction(a)&&a.call(n[0]),e.isFunction(t)&&t()}var n=e(this),a=s.complete,r=s.mode;(n.is(":hidden")?"hide"===r:"show"===r)?(n[r](),i()):o.call(n[0],s,i)}var s=t.apply(this,arguments),n=s.mode,a=s.queue,o=e.effects.effect[s.effect];return e.fx.off||!o?n?this[n](s.duration,s.complete):this.each(function(){s.complete&&s.complete.call(this)}):a===!1?this.each(i):this.queue(a||"fx",i)},show:function(e){return function(s){if(i(s))return e.apply(this,arguments);var n=t.apply(this,arguments);return n.mode="show",this.effect.call(this,n)}}(e.fn.show),hide:function(e){return function(s){if(i(s))return e.apply(this,arguments);var n=t.apply(this,arguments);return n.mode="hide",this.effect.call(this,n)}}(e.fn.hide),toggle:function(e){return function(s){if(i(s)||"boolean"==typeof s)return e.apply(this,arguments);var n=t.apply(this,arguments);return n.mode="toggle",this.effect.call(this,n)}}(e.fn.toggle),cssUnit:function(t){var i=this.css(t),s=[];return e.each(["em","px","%","pt"],function(e,t){i.indexOf(t)>0&&(s=[parseFloat(i),t])}),s}})}(),function(){var t={};e.each(["Quad","Cubic","Quart","Quint","Expo"],function(e,i){t[i]=function(t){return Math.pow(t,e+2)}}),e.extend(t,{Sine:function(e){return 1-Math.cos(e*Math.PI/2)},Circ:function(e){return 1-Math.sqrt(1-e*e)},Elastic:function(e){return 0===e||1===e?e:-Math.pow(2,8*(e-1))*Math.sin((80*(e-1)-7.5)*Math.PI/15)},Back:function(e){return e*e*(3*e-2)},Bounce:function(e){for(var t,i=4;((t=Math.pow(2,--i))-1)/11>e;);return 1/Math.pow(4,3-i)-7.5625*Math.pow((3*t-2)/22-e,2)}}),e.each(t,function(t,i){e.easing["easeIn"+t]=i,e.easing["easeOut"+t]=function(e){return 1-i(1-e)},e.easing["easeInOut"+t]=function(e){return.5>e?i(2*e)/2:1-i(-2*e+2)/2}})}(),e.effects,e.effects.effect.blind=function(t,i){var s,n,a,o=e(this),r=/up|down|vertical/,h=/up|left|vertical|horizontal/,l=["position","top","bottom","left","right","height","width"],u=e.effects.setMode(o,t.mode||"hide"),d=t.direction||"up",c=r.test(d),p=c?"height":"width",f=c?"top":"left",m=h.test(d),g={},v="show"===u;o.parent().is(".ui-effects-wrapper")?e.effects.save(o.parent(),l):e.effects.save(o,l),o.show(),s=e.effects.createWrapper(o).css({overflow:"hidden"}),n=s[p](),a=parseFloat(s.css(f))||0,g[p]=v?n:0,m||(o.css(c?"bottom":"right",0).css(c?"top":"left","auto").css({position:"absolute"}),g[f]=v?a:n+a),v&&(s.css(p,0),m||s.css(f,a+n)),s.animate(g,{duration:t.duration,easing:t.easing,queue:!1,complete:function(){"hide"===u&&o.hide(),e.effects.restore(o,l),e.effects.removeWrapper(o),i()
}})},e.effects.effect.bounce=function(t,i){var s,n,a,o=e(this),r=["position","top","bottom","left","right","height","width"],h=e.effects.setMode(o,t.mode||"effect"),l="hide"===h,u="show"===h,d=t.direction||"up",c=t.distance,p=t.times||5,f=2*p+(u||l?1:0),m=t.duration/f,g=t.easing,v="up"===d||"down"===d?"top":"left",y="up"===d||"left"===d,b=o.queue(),_=b.length;for((u||l)&&r.push("opacity"),e.effects.save(o,r),o.show(),e.effects.createWrapper(o),c||(c=o["top"===v?"outerHeight":"outerWidth"]()/3),u&&(a={opacity:1},a[v]=0,o.css("opacity",0).css(v,y?2*-c:2*c).animate(a,m,g)),l&&(c/=Math.pow(2,p-1)),a={},a[v]=0,s=0;p>s;s++)n={},n[v]=(y?"-=":"+=")+c,o.animate(n,m,g).animate(a,m,g),c=l?2*c:c/2;l&&(n={opacity:0},n[v]=(y?"-=":"+=")+c,o.animate(n,m,g)),o.queue(function(){l&&o.hide(),e.effects.restore(o,r),e.effects.removeWrapper(o),i()}),_>1&&b.splice.apply(b,[1,0].concat(b.splice(_,f+1))),o.dequeue()},e.effects.effect.clip=function(t,i){var s,n,a,o=e(this),r=["position","top","bottom","left","right","height","width"],h=e.effects.setMode(o,t.mode||"hide"),l="show"===h,u=t.direction||"vertical",d="vertical"===u,c=d?"height":"width",p=d?"top":"left",f={};e.effects.save(o,r),o.show(),s=e.effects.createWrapper(o).css({overflow:"hidden"}),n="IMG"===o[0].tagName?s:o,a=n[c](),l&&(n.css(c,0),n.css(p,a/2)),f[c]=l?a:0,f[p]=l?0:a/2,n.animate(f,{queue:!1,duration:t.duration,easing:t.easing,complete:function(){l||o.hide(),e.effects.restore(o,r),e.effects.removeWrapper(o),i()}})},e.effects.effect.drop=function(t,i){var s,n=e(this),a=["position","top","bottom","left","right","opacity","height","width"],o=e.effects.setMode(n,t.mode||"hide"),r="show"===o,h=t.direction||"left",l="up"===h||"down"===h?"top":"left",u="up"===h||"left"===h?"pos":"neg",d={opacity:r?1:0};e.effects.save(n,a),n.show(),e.effects.createWrapper(n),s=t.distance||n["top"===l?"outerHeight":"outerWidth"](!0)/2,r&&n.css("opacity",0).css(l,"pos"===u?-s:s),d[l]=(r?"pos"===u?"+=":"-=":"pos"===u?"-=":"+=")+s,n.animate(d,{queue:!1,duration:t.duration,easing:t.easing,complete:function(){"hide"===o&&n.hide(),e.effects.restore(n,a),e.effects.removeWrapper(n),i()}})},e.effects.effect.explode=function(t,i){function s(){b.push(this),b.length===d*c&&n()}function n(){p.css({visibility:"visible"}),e(b).remove(),m||p.hide(),i()}var a,o,r,h,l,u,d=t.pieces?Math.round(Math.sqrt(t.pieces)):3,c=d,p=e(this),f=e.effects.setMode(p,t.mode||"hide"),m="show"===f,g=p.show().css("visibility","hidden").offset(),v=Math.ceil(p.outerWidth()/c),y=Math.ceil(p.outerHeight()/d),b=[];for(a=0;d>a;a++)for(h=g.top+a*y,u=a-(d-1)/2,o=0;c>o;o++)r=g.left+o*v,l=o-(c-1)/2,p.clone().appendTo("body").wrap("<div></div>").css({position:"absolute",visibility:"visible",left:-o*v,top:-a*y}).parent().addClass("ui-effects-explode").css({position:"absolute",overflow:"hidden",width:v,height:y,left:r+(m?l*v:0),top:h+(m?u*y:0),opacity:m?0:1}).animate({left:r+(m?0:l*v),top:h+(m?0:u*y),opacity:m?1:0},t.duration||500,t.easing,s)},e.effects.effect.fade=function(t,i){var s=e(this),n=e.effects.setMode(s,t.mode||"toggle");s.animate({opacity:n},{queue:!1,duration:t.duration,easing:t.easing,complete:i})},e.effects.effect.fold=function(t,i){var s,n,a=e(this),o=["position","top","bottom","left","right","height","width"],r=e.effects.setMode(a,t.mode||"hide"),h="show"===r,l="hide"===r,u=t.size||15,d=/([0-9]+)%/.exec(u),c=!!t.horizFirst,p=h!==c,f=p?["width","height"]:["height","width"],m=t.duration/2,g={},v={};e.effects.save(a,o),a.show(),s=e.effects.createWrapper(a).css({overflow:"hidden"}),n=p?[s.width(),s.height()]:[s.height(),s.width()],d&&(u=parseInt(d[1],10)/100*n[l?0:1]),h&&s.css(c?{height:0,width:u}:{height:u,width:0}),g[f[0]]=h?n[0]:u,v[f[1]]=h?n[1]:0,s.animate(g,m,t.easing).animate(v,m,t.easing,function(){l&&a.hide(),e.effects.restore(a,o),e.effects.removeWrapper(a),i()})},e.effects.effect.highlight=function(t,i){var s=e(this),n=["backgroundImage","backgroundColor","opacity"],a=e.effects.setMode(s,t.mode||"show"),o={backgroundColor:s.css("backgroundColor")};"hide"===a&&(o.opacity=0),e.effects.save(s,n),s.show().css({backgroundImage:"none",backgroundColor:t.color||"#ffff99"}).animate(o,{queue:!1,duration:t.duration,easing:t.easing,complete:function(){"hide"===a&&s.hide(),e.effects.restore(s,n),i()}})},e.effects.effect.size=function(t,i){var s,n,a,o=e(this),r=["position","top","bottom","left","right","width","height","overflow","opacity"],h=["position","top","bottom","left","right","overflow","opacity"],l=["width","height","overflow"],u=["fontSize"],d=["borderTopWidth","borderBottomWidth","paddingTop","paddingBottom"],c=["borderLeftWidth","borderRightWidth","paddingLeft","paddingRight"],p=e.effects.setMode(o,t.mode||"effect"),f=t.restore||"effect"!==p,m=t.scale||"both",g=t.origin||["middle","center"],v=o.css("position"),y=f?r:h,b={height:0,width:0,outerHeight:0,outerWidth:0};"show"===p&&o.show(),s={height:o.height(),width:o.width(),outerHeight:o.outerHeight(),outerWidth:o.outerWidth()},"toggle"===t.mode&&"show"===p?(o.from=t.to||b,o.to=t.from||s):(o.from=t.from||("show"===p?b:s),o.to=t.to||("hide"===p?b:s)),a={from:{y:o.from.height/s.height,x:o.from.width/s.width},to:{y:o.to.height/s.height,x:o.to.width/s.width}},("box"===m||"both"===m)&&(a.from.y!==a.to.y&&(y=y.concat(d),o.from=e.effects.setTransition(o,d,a.from.y,o.from),o.to=e.effects.setTransition(o,d,a.to.y,o.to)),a.from.x!==a.to.x&&(y=y.concat(c),o.from=e.effects.setTransition(o,c,a.from.x,o.from),o.to=e.effects.setTransition(o,c,a.to.x,o.to))),("content"===m||"both"===m)&&a.from.y!==a.to.y&&(y=y.concat(u).concat(l),o.from=e.effects.setTransition(o,u,a.from.y,o.from),o.to=e.effects.setTransition(o,u,a.to.y,o.to)),e.effects.save(o,y),o.show(),e.effects.createWrapper(o),o.css("overflow","hidden").css(o.from),g&&(n=e.effects.getBaseline(g,s),o.from.top=(s.outerHeight-o.outerHeight())*n.y,o.from.left=(s.outerWidth-o.outerWidth())*n.x,o.to.top=(s.outerHeight-o.to.outerHeight)*n.y,o.to.left=(s.outerWidth-o.to.outerWidth)*n.x),o.css(o.from),("content"===m||"both"===m)&&(d=d.concat(["marginTop","marginBottom"]).concat(u),c=c.concat(["marginLeft","marginRight"]),l=r.concat(d).concat(c),o.find("*[width]").each(function(){var i=e(this),s={height:i.height(),width:i.width(),outerHeight:i.outerHeight(),outerWidth:i.outerWidth()};f&&e.effects.save(i,l),i.from={height:s.height*a.from.y,width:s.width*a.from.x,outerHeight:s.outerHeight*a.from.y,outerWidth:s.outerWidth*a.from.x},i.to={height:s.height*a.to.y,width:s.width*a.to.x,outerHeight:s.height*a.to.y,outerWidth:s.width*a.to.x},a.from.y!==a.to.y&&(i.from=e.effects.setTransition(i,d,a.from.y,i.from),i.to=e.effects.setTransition(i,d,a.to.y,i.to)),a.from.x!==a.to.x&&(i.from=e.effects.setTransition(i,c,a.from.x,i.from),i.to=e.effects.setTransition(i,c,a.to.x,i.to)),i.css(i.from),i.animate(i.to,t.duration,t.easing,function(){f&&e.effects.restore(i,l)})})),o.animate(o.to,{queue:!1,duration:t.duration,easing:t.easing,complete:function(){0===o.to.opacity&&o.css("opacity",o.from.opacity),"hide"===p&&o.hide(),e.effects.restore(o,y),f||("static"===v?o.css({position:"relative",top:o.to.top,left:o.to.left}):e.each(["top","left"],function(e,t){o.css(t,function(t,i){var s=parseInt(i,10),n=e?o.to.left:o.to.top;return"auto"===i?n+"px":s+n+"px"})})),e.effects.removeWrapper(o),i()}})},e.effects.effect.scale=function(t,i){var s=e(this),n=e.extend(!0,{},t),a=e.effects.setMode(s,t.mode||"effect"),o=parseInt(t.percent,10)||(0===parseInt(t.percent,10)?0:"hide"===a?0:100),r=t.direction||"both",h=t.origin,l={height:s.height(),width:s.width(),outerHeight:s.outerHeight(),outerWidth:s.outerWidth()},u={y:"horizontal"!==r?o/100:1,x:"vertical"!==r?o/100:1};n.effect="size",n.queue=!1,n.complete=i,"effect"!==a&&(n.origin=h||["middle","center"],n.restore=!0),n.from=t.from||("show"===a?{height:0,width:0,outerHeight:0,outerWidth:0}:l),n.to={height:l.height*u.y,width:l.width*u.x,outerHeight:l.outerHeight*u.y,outerWidth:l.outerWidth*u.x},n.fade&&("show"===a&&(n.from.opacity=0,n.to.opacity=1),"hide"===a&&(n.from.opacity=1,n.to.opacity=0)),s.effect(n)},e.effects.effect.puff=function(t,i){var s=e(this),n=e.effects.setMode(s,t.mode||"hide"),a="hide"===n,o=parseInt(t.percent,10)||150,r=o/100,h={height:s.height(),width:s.width(),outerHeight:s.outerHeight(),outerWidth:s.outerWidth()};e.extend(t,{effect:"scale",queue:!1,fade:!0,mode:n,complete:i,percent:a?o:100,from:a?h:{height:h.height*r,width:h.width*r,outerHeight:h.outerHeight*r,outerWidth:h.outerWidth*r}}),s.effect(t)},e.effects.effect.pulsate=function(t,i){var s,n=e(this),a=e.effects.setMode(n,t.mode||"show"),o="show"===a,r="hide"===a,h=o||"hide"===a,l=2*(t.times||5)+(h?1:0),u=t.duration/l,d=0,c=n.queue(),p=c.length;for((o||!n.is(":visible"))&&(n.css("opacity",0).show(),d=1),s=1;l>s;s++)n.animate({opacity:d},u,t.easing),d=1-d;n.animate({opacity:d},u,t.easing),n.queue(function(){r&&n.hide(),i()}),p>1&&c.splice.apply(c,[1,0].concat(c.splice(p,l+1))),n.dequeue()},e.effects.effect.shake=function(t,i){var s,n=e(this),a=["position","top","bottom","left","right","height","width"],o=e.effects.setMode(n,t.mode||"effect"),r=t.direction||"left",h=t.distance||20,l=t.times||3,u=2*l+1,d=Math.round(t.duration/u),c="up"===r||"down"===r?"top":"left",p="up"===r||"left"===r,f={},m={},g={},v=n.queue(),y=v.length;for(e.effects.save(n,a),n.show(),e.effects.createWrapper(n),f[c]=(p?"-=":"+=")+h,m[c]=(p?"+=":"-=")+2*h,g[c]=(p?"-=":"+=")+2*h,n.animate(f,d,t.easing),s=1;l>s;s++)n.animate(m,d,t.easing).animate(g,d,t.easing);n.animate(m,d,t.easing).animate(f,d/2,t.easing).queue(function(){"hide"===o&&n.hide(),e.effects.restore(n,a),e.effects.removeWrapper(n),i()}),y>1&&v.splice.apply(v,[1,0].concat(v.splice(y,u+1))),n.dequeue()},e.effects.effect.slide=function(t,i){var s,n=e(this),a=["position","top","bottom","left","right","width","height"],o=e.effects.setMode(n,t.mode||"show"),r="show"===o,h=t.direction||"left",l="up"===h||"down"===h?"top":"left",u="up"===h||"left"===h,d={};e.effects.save(n,a),n.show(),s=t.distance||n["top"===l?"outerHeight":"outerWidth"](!0),e.effects.createWrapper(n).css({overflow:"hidden"}),r&&n.css(l,u?isNaN(s)?"-"+s:-s:s),d[l]=(r?u?"+=":"-=":u?"-=":"+=")+s,n.animate(d,{queue:!1,duration:t.duration,easing:t.easing,complete:function(){"hide"===o&&n.hide(),e.effects.restore(n,a),e.effects.removeWrapper(n),i()}})},e.effects.effect.transfer=function(t,i){var s=e(this),n=e(t.to),a="fixed"===n.css("position"),o=e("body"),r=a?o.scrollTop():0,h=a?o.scrollLeft():0,l=n.offset(),u={top:l.top-r,left:l.left-h,height:n.innerHeight(),width:n.innerWidth()},d=s.offset(),c=e("<div class='ui-effects-transfer'></div>").appendTo(document.body).addClass(t.className).css({top:d.top-r,left:d.left-h,height:s.innerHeight(),width:s.innerWidth(),position:a?"fixed":"absolute"}).animate(u,t.duration,t.easing,function(){c.remove(),i()})}});</script>
<style type="text/css">

.tocify {
width: 20%;
max-height: 90%;
overflow: auto;
margin-left: 2%;
position: fixed;
border: 1px solid #ccc;
webkit-border-radius: 6px;
moz-border-radius: 6px;
border-radius: 6px;
}

.tocify ul, .tocify li {
list-style: none;
margin: 0;
padding: 0;
border: none;
line-height: 30px;
}

.tocify-header {
text-indent: 10px;
}

.tocify-subheader {
text-indent: 20px;
display: none;
}

.tocify-subheader li {
font-size: 12px;
}

.tocify-subheader .tocify-subheader {
text-indent: 30px;
}

.tocify-subheader .tocify-subheader .tocify-subheader {
text-indent: 40px;
}

.tocify .tocify-item > a, .tocify .nav-list .nav-header {
margin: 0px;
}

.tocify .tocify-item a, .tocify .list-group-item {
padding: 5px;
}
.tocify .nav-pills > li {
float: none;
}


</style>
<script>/* jquery Tocify - v1.9.1 - 2013-10-22
 * http://www.gregfranko.com/jquery.tocify.js/
 * Copyright (c) 2013 Greg Franko; Licensed MIT */

// Immediately-Invoked Function Expression (IIFE) [Ben Alman Blog Post](http://benalman.com/news/2010/11/immediately-invoked-function-expression/) that calls another IIFE that contains all of the plugin logic.  I used this pattern so that anyone viewing this code would not have to scroll to the bottom of the page to view the local parameters that were passed to the main IIFE.
(function(tocify) {

    // ECMAScript 5 Strict Mode: [John Resig Blog Post](http://ejohn.org/blog/ecmascript-5-strict-mode-json-and-more/)
    "use strict";

    // Calls the second IIFE and locally passes in the global jQuery, window, and document objects
    tocify(window.jQuery, window, document);

  }

  // Locally passes in `jQuery`, the `window` object, the `document` object, and an `undefined` variable.  The `jQuery`, `window` and `document` objects are passed in locally, to improve performance, since javascript first searches for a variable match within the local variables set before searching the global variables set.  All of the global variables are also passed in locally to be minifier friendly. `undefined` can be passed in locally, because it is not a reserved word in JavaScript.
  (function($, window, document, undefined) {

    // ECMAScript 5 Strict Mode: [John Resig Blog Post](http://ejohn.org/blog/ecmascript-5-strict-mode-json-and-more/)
    "use strict";

    var tocClassName = "tocify",
      tocClass = "." + tocClassName,
      tocFocusClassName = "tocify-focus",
      tocHoverClassName = "tocify-hover",
      hideTocClassName = "tocify-hide",
      hideTocClass = "." + hideTocClassName,
      headerClassName = "tocify-header",
      headerClass = "." + headerClassName,
      subheaderClassName = "tocify-subheader",
      subheaderClass = "." + subheaderClassName,
      itemClassName = "tocify-item",
      itemClass = "." + itemClassName,
      extendPageClassName = "tocify-extend-page",
      extendPageClass = "." + extendPageClassName;

    // Calling the jQueryUI Widget Factory Method
    $.widget("toc.tocify", {

      //Plugin version
      version: "1.9.1",

      // These options will be used as defaults
      options: {

        // **context**: Accepts String: Any jQuery selector
        // The container element that holds all of the elements used to generate the table of contents
        context: "body",

        // **ignoreSelector**: Accepts String: Any jQuery selector
        // A selector to any element that would be matched by selectors that you wish to be ignored
        ignoreSelector: null,

        // **selectors**: Accepts an Array of Strings: Any jQuery selectors
        // The element's used to generate the table of contents.  The order is very important since it will determine the table of content's nesting structure
        selectors: "h1, h2, h3",

        // **showAndHide**: Accepts a boolean: true or false
        // Used to determine if elements should be shown and hidden
        showAndHide: true,

        // **showEffect**: Accepts String: "none", "fadeIn", "show", or "slideDown"
        // Used to display any of the table of contents nested items
        showEffect: "slideDown",

        // **showEffectSpeed**: Accepts Number (milliseconds) or String: "slow", "medium", or "fast"
        // The time duration of the show animation
        showEffectSpeed: "medium",

        // **hideEffect**: Accepts String: "none", "fadeOut", "hide", or "slideUp"
        // Used to hide any of the table of contents nested items
        hideEffect: "slideUp",

        // **hideEffectSpeed**: Accepts Number (milliseconds) or String: "slow", "medium", or "fast"
        // The time duration of the hide animation
        hideEffectSpeed: "medium",

        // **smoothScroll**: Accepts a boolean: true or false
        // Determines if a jQuery animation should be used to scroll to specific table of contents items on the page
        smoothScroll: true,

        // **smoothScrollSpeed**: Accepts Number (milliseconds) or String: "slow", "medium", or "fast"
        // The time duration of the smoothScroll animation
        smoothScrollSpeed: "medium",

        // **scrollTo**: Accepts Number (pixels)
        // The amount of space between the top of page and the selected table of contents item after the page has been scrolled
        scrollTo: 0,

        // **showAndHideOnScroll**: Accepts a boolean: true or false
        // Determines if table of contents nested items should be shown and hidden while scrolling
        showAndHideOnScroll: true,

        // **highlightOnScroll**: Accepts a boolean: true or false
        // Determines if table of contents nested items should be highlighted (set to a different color) while scrolling
        highlightOnScroll: true,

        // **highlightOffset**: Accepts a number
        // The offset distance in pixels to trigger the next active table of contents item
        highlightOffset: 40,

        // **theme**: Accepts a string: "bootstrap", "jqueryui", or "none"
        // Determines if Twitter Bootstrap, jQueryUI, or Tocify classes should be added to the table of contents
        theme: "bootstrap",

        // **extendPage**: Accepts a boolean: true or false
        // If a user scrolls to the bottom of the page and the page is not tall enough to scroll to the last table of contents item, then the page height is increased
        extendPage: true,

        // **extendPageOffset**: Accepts a number: pixels
        // How close to the bottom of the page a user must scroll before the page is extended
        extendPageOffset: 100,

        // **history**: Accepts a boolean: true or false
        // Adds a hash to the page url to maintain history
        history: true,

        // **scrollHistory**: Accepts a boolean: true or false
        // Adds a hash to the page url, to maintain history, when scrolling to a TOC item
        scrollHistory: false,

        // **hashGenerator**: How the hash value (the anchor segment of the URL, following the
        // # character) will be generated.
        //
        // "compact" (default) - #CompressesEverythingTogether
        // "pretty" - #looks-like-a-nice-url-and-is-easily-readable
        // function(text, element){} - Your own hash generation function that accepts the text as an
        // argument, and returns the hash value.
        hashGenerator: "compact",

        // **highlightDefault**: Accepts a boolean: true or false
        // Set's the first TOC item as active if no other TOC item is active.
        highlightDefault: true

      },

      // _Create
      // -------
      //      Constructs the plugin.  Only called once.
      _create: function() {

        var self = this;

        self.extendPageScroll = true;

        // Internal array that keeps track of all TOC items (Helps to recognize if there are duplicate TOC item strings)
        self.items = [];

        // Generates the HTML for the dynamic table of contents
        self._generateToc();

        // Adds CSS classes to the newly generated table of contents HTML
        self._addCSSClasses();

        self.webkit = (function() {

          for (var prop in window) {

            if (prop) {

              if (prop.toLowerCase().indexOf("webkit") !== -1) {

                return true;

              }

            }

          }

          return false;

        }());

        // Adds jQuery event handlers to the newly generated table of contents
        self._setEventHandlers();

        // Binding to the Window load event to make sure the correct scrollTop is calculated
        $(window).load(function() {

          // Sets the active TOC item
          self._setActiveElement(true);

          // Once all animations on the page are complete, this callback function will be called
          $("html, body").promise().done(function() {

            setTimeout(function() {

              self.extendPageScroll = false;

            }, 0);

          });

        });

      },

      // _generateToc
      // ------------
      //      Generates the HTML for the dynamic table of contents
      _generateToc: function() {

        // _Local variables_

        // Stores the plugin context in the self variable
        var self = this,

          // All of the HTML tags found within the context provided (i.e. body) that match the top level jQuery selector above
          firstElem,

          // Instantiated variable that will store the top level newly created unordered list DOM element
          ul,
          ignoreSelector = self.options.ignoreSelector;


        // Determine the element to start the toc with
        // get all the top level selectors
        firstElem = [];
        var selectors = this.options.selectors.replace(/ /g, "").split(",");
        // find the first set that have at least one non-ignored element
        for(var i = 0; i < selectors.length; i++) {
          var foundSelectors = $(this.options.context).find(selectors[i]);
          for (var s = 0; s < foundSelectors.length; s++) {
            if (!$(foundSelectors[s]).is(ignoreSelector)) {
              firstElem = foundSelectors;
              break;
            }
          }
          if (firstElem.length> 0)
            break;
        }

        if (!firstElem.length) {

          self.element.addClass(hideTocClassName);

          return;

        }

        self.element.addClass(tocClassName);

        // Loops through each top level selector
        firstElem.each(function(index) {

          //If the element matches the ignoreSelector then we skip it
          if ($(this).is(ignoreSelector)) {
            return;
          }

          // Creates an unordered list HTML element and adds a dynamic ID and standard class name
          ul = $("<ul/>", {
            "id": headerClassName + index,
            "class": headerClassName
          }).

          // Appends a top level list item HTML element to the previously created HTML header
          append(self._nestElements($(this), index));

          // Add the created unordered list element to the HTML element calling the plugin
          self.element.append(ul);

          // Finds all of the HTML tags between the header and subheader elements
          $(this).nextUntil(this.nodeName.toLowerCase()).each(function() {

            // If there are no nested subheader elemements
            if ($(this).find(self.options.selectors).length === 0) {

              // Loops through all of the subheader elements
              $(this).filter(self.options.selectors).each(function() {

                //If the element matches the ignoreSelector then we skip it
                if ($(this).is(ignoreSelector)) {
                  return;
                }

                self._appendSubheaders.call(this, self, ul);

              });

            }

            // If there are nested subheader elements
            else {

              // Loops through all of the subheader elements
              $(this).find(self.options.selectors).each(function() {

                //If the element matches the ignoreSelector then we skip it
                if ($(this).is(ignoreSelector)) {
                  return;
                }

                self._appendSubheaders.call(this, self, ul);

              });

            }

          });

        });

      },

      _setActiveElement: function(pageload) {

        var self = this,

          hash = window.location.hash.substring(1),

          elem = self.element.find('li[data-unique="' + hash + '"]');

        if (hash.length) {

          // Removes highlighting from all of the list item's
          self.element.find("." + self.focusClass).removeClass(self.focusClass);

          // Highlights the current list item that was clicked
          elem.addClass(self.focusClass);

          // Triggers the click event on the currently focused TOC item
          elem.click();

        } else {

          // Removes highlighting from all of the list item's
          self.element.find("." + self.focusClass).removeClass(self.focusClass);

          if (!hash.length && pageload && self.options.highlightDefault) {

            // Highlights the first TOC item if no other items are highlighted
            self.element.find(itemClass).first().addClass(self.focusClass);

          }

        }

        return self;

      },

      // _nestElements
      // -------------
      //      Helps create the table of contents list by appending nested list items
      _nestElements: function(self, index) {

        var arr, item, hashValue;

        arr = $.grep(this.items, function(item) {

          return item === self.text();

        });

        // If there is already a duplicate TOC item
        if (arr.length) {

          // Adds the current TOC item text and index (for slight randomization) to the internal array
          this.items.push(self.text() + index);

        }

        // If there not a duplicate TOC item
        else {

          // Adds the current TOC item text to the internal array
          this.items.push(self.text());

        }

        hashValue = this._generateHashValue(arr, self, index);

        // Appends a list item HTML element to the last unordered list HTML element found within the HTML element calling the plugin
        item = $("<li/>", {

          // Sets a common class name to the list item
          "class": itemClassName,

          "data-unique": hashValue

        });

        if (this.options.theme !== "bootstrap3") {

          item.append($("<a/>", {

            "text": self.text()

          }));

        } else {

          item.text(self.text());

        }

        // Adds an HTML anchor tag before the currently traversed HTML element
        self.before($("<div/>", {

          // Sets a name attribute on the anchor tag to the text of the currently traversed HTML element (also making sure that all whitespace is replaced with an underscore)
          "name": hashValue,

          "data-unique": hashValue

        }));

        return item;

      },

      // _generateHashValue
      // ------------------
      //      Generates the hash value that will be used to refer to each item.
      _generateHashValue: function(arr, self, index) {

        var hashValue = "",
          hashGeneratorOption = this.options.hashGenerator;

        if (hashGeneratorOption === "pretty") {

          // prettify the text
          hashValue = self.text().toLowerCase().replace(/\s/g, "-");

          // fix double hyphens
          while (hashValue.indexOf("--") > -1) {
            hashValue = hashValue.replace(/--/g, "-");
          }

          // fix colon-space instances
          while (hashValue.indexOf(":-") > -1) {
            hashValue = hashValue.replace(/:-/g, "-");
          }

        } else if (typeof hashGeneratorOption === "function") {

          // call the function
          hashValue = hashGeneratorOption(self.text(), self);

        } else {

          // compact - the default
          hashValue = self.text().replace(/\s/g, "");

        }

        // add the index if we need to
        if (arr.length) {
          hashValue += "" + index;
        }

        // return the value
        return hashValue;

      },

      // _appendElements
      // ---------------
      //      Helps create the table of contents list by appending subheader elements

      _appendSubheaders: function(self, ul) {

        // The current element index
        var index = $(this).index(self.options.selectors),

          // Finds the previous header DOM element
          previousHeader = $(self.options.selectors).eq(index - 1),

          currentTagName = +$(this).prop("tagName").charAt(1),

          previousTagName = +previousHeader.prop("tagName").charAt(1),

          lastSubheader;

        // If the current header DOM element is smaller than the previous header DOM element or the first subheader
        if (currentTagName < previousTagName) {

          // Selects the last unordered list HTML found within the HTML element calling the plugin
          self.element.find(subheaderClass + "[data-tag=" + currentTagName + "]").last().append(self._nestElements($(this), index));

        }

        // If the current header DOM element is the same type of header(eg. h4) as the previous header DOM element
        else if (currentTagName === previousTagName) {

          ul.find(itemClass).last().after(self._nestElements($(this), index));

        } else {

          // Selects the last unordered list HTML found within the HTML element calling the plugin
          ul.find(itemClass).last().

          // Appends an unorderedList HTML element to the dynamic `unorderedList` variable and sets a common class name
          after($("<ul/>", {

            "class": subheaderClassName,

            "data-tag": currentTagName

          })).next(subheaderClass).

          // Appends a list item HTML element to the last unordered list HTML element found within the HTML element calling the plugin
          append(self._nestElements($(this), index));
        }

      },

      // _setEventHandlers
      // ----------------
      //      Adds jQuery event handlers to the newly generated table of contents
      _setEventHandlers: function() {

        // _Local variables_

        // Stores the plugin context in the self variable
        var self = this,

          // Instantiates a new variable that will be used to hold a specific element's context
          $self,

          // Instantiates a new variable that will be used to determine the smoothScroll animation time duration
          duration;

        // Event delegation that looks for any clicks on list item elements inside of the HTML element calling the plugin
        this.element.on("click.tocify", "li", function(event) {

          if (self.options.history) {

            window.location.hash = $(this).attr("data-unique");

          }

          // Removes highlighting from all of the list item's
          self.element.find("." + self.focusClass).removeClass(self.focusClass);

          // Highlights the current list item that was clicked
          $(this).addClass(self.focusClass);

          // If the showAndHide option is true
          if (self.options.showAndHide) {

            var elem = $('li[data-unique="' + $(this).attr("data-unique") + '"]');

            self._triggerShow(elem);

          }

          self._scrollTo($(this));

        });

        // Mouseenter and Mouseleave event handlers for the list item's within the HTML element calling the plugin
        this.element.find("li").on({

          // Mouseenter event handler
          "mouseenter.tocify": function() {

            // Adds a hover CSS class to the current list item
            $(this).addClass(self.hoverClass);

            // Makes sure the cursor is set to the pointer icon
            $(this).css("cursor", "pointer");

          },

          // Mouseleave event handler
          "mouseleave.tocify": function() {

            if (self.options.theme !== "bootstrap") {

              // Removes the hover CSS class from the current list item
              $(this).removeClass(self.hoverClass);

            }

          }
        });

        // only attach handler if needed (expensive in IE)
        if (self.options.extendPage || self.options.highlightOnScroll || self.options.scrollHistory || self.options.showAndHideOnScroll) {
          // Window scroll event handler
          $(window).on("scroll.tocify", function() {

            // Once all animations on the page are complete, this callback function will be called
            $("html, body").promise().done(function() {

              // Local variables

              // Stores how far the user has scrolled
              var winScrollTop = $(window).scrollTop(),

                // Stores the height of the window
                winHeight = $(window).height(),

                // Stores the height of the document
                docHeight = $(document).height(),

                scrollHeight = $("body")[0].scrollHeight,

                // Instantiates a variable that will be used to hold a selected HTML element
                elem,

                lastElem,

                lastElemOffset,

                currentElem;

              if (self.options.extendPage) {

                // If the user has scrolled to the bottom of the page and the last toc item is not focused
                if ((self.webkit && winScrollTop >= scrollHeight - winHeight - self.options.extendPageOffset) || (!self.webkit && winHeight + winScrollTop > docHeight - self.options.extendPageOffset)) {

                  if (!$(extendPageClass).length) {

                    lastElem = $('div[data-unique="' + $(itemClass).last().attr("data-unique") + '"]');

                    if (!lastElem.length) return;

                    // Gets the top offset of the page header that is linked to the last toc item
                    lastElemOffset = lastElem.offset().top;

                    // Appends a div to the bottom of the page and sets the height to the difference of the window scrollTop and the last element's position top offset
                    $(self.options.context).append($("<div/>", {

                      "class": extendPageClassName,

                      "height": Math.abs(lastElemOffset - winScrollTop) + "px",

                      "data-unique": extendPageClassName

                    }));

                    if (self.extendPageScroll) {

                      currentElem = self.element.find('li.' + self.focusClass);

                      self._scrollTo($('div[data-unique="' + currentElem.attr("data-unique") + '"]'));

                    }

                  }

                }

              }

              // The zero timeout ensures the following code is run after the scroll events
              setTimeout(function() {

                // _Local variables_

                // Stores the distance to the closest anchor
                var closestAnchorDistance = null,

                  // Stores the index of the closest anchor
                  closestAnchorIdx = null,

                  // Keeps a reference to all anchors
                  anchors = $(self.options.context).find("div[data-unique]"),

                  anchorText;

                // Determines the index of the closest anchor
                anchors.each(function(idx) {
                  var distance = Math.abs(($(this).next().length ? $(this).next() : $(this)).offset().top - winScrollTop - self.options.highlightOffset);
                  if (closestAnchorDistance == null || distance < closestAnchorDistance) {
                    closestAnchorDistance = distance;
                    closestAnchorIdx = idx;
                  } else {
                    return false;
                  }
                });

                anchorText = $(anchors[closestAnchorIdx]).attr("data-unique");

                // Stores the list item HTML element that corresponds to the currently traversed anchor tag
                elem = $('li[data-unique="' + anchorText + '"]');

                // If the `highlightOnScroll` option is true and a next element is found
                if (self.options.highlightOnScroll && elem.length) {

                  // Removes highlighting from all of the list item's
                  self.element.find("." + self.focusClass).removeClass(self.focusClass);

                  // Highlights the corresponding list item
                  elem.addClass(self.focusClass);

                }

                if (self.options.scrollHistory) {

                  if (window.location.hash !== "#" + anchorText) {

                    window.location.replace("#" + anchorText);

                  }
                }

                // If the `showAndHideOnScroll` option is true
                if (self.options.showAndHideOnScroll && self.options.showAndHide) {

                  self._triggerShow(elem, true);

                }

              }, 0);

            });

          });
        }

      },

      // Show
      // ----
      //      Opens the current sub-header
      show: function(elem, scroll) {

        // Stores the plugin context in the `self` variable
        var self = this,
          element = elem;

        // If the sub-header is not already visible
        if (!elem.is(":visible")) {

          // If the current element does not have any nested subheaders, is not a header, and its parent is not visible
          if (!elem.find(subheaderClass).length && !elem.parent().is(headerClass) && !elem.parent().is(":visible")) {

            // Sets the current element to all of the subheaders within the current header
            elem = elem.parents(subheaderClass).add(elem);

          }

          // If the current element does not have any nested subheaders and is not a header
          else if (!elem.children(subheaderClass).length && !elem.parent().is(headerClass)) {

            // Sets the current element to the closest subheader
            elem = elem.closest(subheaderClass);

          }

          //Determines what jQuery effect to use
          switch (self.options.showEffect) {

            //Uses `no effect`
            case "none":

              elem.show();

              break;

              //Uses the jQuery `show` special effect
            case "show":

              elem.show(self.options.showEffectSpeed);

              break;

              //Uses the jQuery `slideDown` special effect
            case "slideDown":

              elem.slideDown(self.options.showEffectSpeed);

              break;

              //Uses the jQuery `fadeIn` special effect
            case "fadeIn":

              elem.fadeIn(self.options.showEffectSpeed);

              break;

              //If none of the above options were passed, then a `jQueryUI show effect` is expected
            default:

              elem.show();

              break;

          }

        }

        // If the current subheader parent element is a header
        if (elem.parent().is(headerClass)) {

          // Hides all non-active sub-headers
          self.hide($(subheaderClass).not(elem));

        }

        // If the current subheader parent element is not a header
        else {

          // Hides all non-active sub-headers
          self.hide($(subheaderClass).not(elem.closest(headerClass).find(subheaderClass).not(elem.siblings())));

        }

        // Maintains chainablity
        return self;

      },

      // Hide
      // ----
      //      Closes the current sub-header
      hide: function(elem) {

        // Stores the plugin context in the `self` variable
        var self = this;

        //Determines what jQuery effect to use
        switch (self.options.hideEffect) {

          // Uses `no effect`
          case "none":

            elem.hide();

            break;

            // Uses the jQuery `hide` special effect
          case "hide":

            elem.hide(self.options.hideEffectSpeed);

            break;

            // Uses the jQuery `slideUp` special effect
          case "slideUp":

            elem.slideUp(self.options.hideEffectSpeed);

            break;

            // Uses the jQuery `fadeOut` special effect
          case "fadeOut":

            elem.fadeOut(self.options.hideEffectSpeed);

            break;

            // If none of the above options were passed, then a `jqueryUI hide effect` is expected
          default:

            elem.hide();

            break;

        }

        // Maintains chainablity
        return self;
      },

      // _triggerShow
      // ------------
      //      Determines what elements get shown on scroll and click
      _triggerShow: function(elem, scroll) {

        var self = this;

        // If the current element's parent is a header element or the next element is a nested subheader element
        if (elem.parent().is(headerClass) || elem.next().is(subheaderClass)) {

          // Shows the next sub-header element
          self.show(elem.next(subheaderClass), scroll);

        }

        // If the current element's parent is a subheader element
        else if (elem.parent().is(subheaderClass)) {

          // Shows the parent sub-header element
          self.show(elem.parent(), scroll);

        }

        // Maintains chainability
        return self;

      },

      // _addCSSClasses
      // --------------
      //      Adds CSS classes to the newly generated table of contents HTML
      _addCSSClasses: function() {

        // If the user wants a jqueryUI theme
        if (this.options.theme === "jqueryui") {

          this.focusClass = "ui-state-default";

          this.hoverClass = "ui-state-hover";

          //Adds the default styling to the dropdown list
          this.element.addClass("ui-widget").find(".toc-title").addClass("ui-widget-header").end().find("li").addClass("ui-widget-content");

        }

        // If the user wants a twitterBootstrap theme
        else if (this.options.theme === "bootstrap") {

          this.element.find(headerClass + "," + subheaderClass).addClass("nav nav-list");

          this.focusClass = "active";

        }

        // If the user wants a twitterBootstrap theme
        else if (this.options.theme === "bootstrap3") {

          this.element.find(headerClass + "," + subheaderClass).addClass("list-group");

          this.element.find(itemClass).addClass("list-group-item");

          this.focusClass = "active";

        }

        // If a user does not want a prebuilt theme
        else {

          // Adds more neutral classes (instead of jqueryui)

          this.focusClass = tocFocusClassName;

          this.hoverClass = tocHoverClassName;

        }

        //Maintains chainability
        return this;

      },

      // setOption
      // ---------
      //      Sets a single Tocify option after the plugin is invoked
      setOption: function() {

        // Calls the jQueryUI Widget Factory setOption method
        $.Widget.prototype._setOption.apply(this, arguments);

      },

      // setOptions
      // ----------
      //      Sets a single or multiple Tocify options after the plugin is invoked
      setOptions: function() {

        // Calls the jQueryUI Widget Factory setOptions method
        $.Widget.prototype._setOptions.apply(this, arguments);

      },

      // _scrollTo
      // ---------
      //      Scrolls to a specific element
      _scrollTo: function(elem) {

        var self = this,
          duration = self.options.smoothScroll || 0,
          scrollTo = self.options.scrollTo,
          currentDiv = $('div[data-unique="' + elem.attr("data-unique") + '"]');

        if (!currentDiv.length) {

          return self;

        }

        // Once all animations on the page are complete, this callback function will be called
        $("html, body").promise().done(function() {

          // Animates the html and body element scrolltops
          $("html, body").animate({

            // Sets the jQuery `scrollTop` to the top offset of the HTML div tag that matches the current list item's `data-unique` tag
            "scrollTop": currentDiv.offset().top - ($.isFunction(scrollTo) ? scrollTo.call() : scrollTo) + "px"

          }, {

            // Sets the smoothScroll animation time duration to the smoothScrollSpeed option
            "duration": duration

          });

        });

        // Maintains chainability
        return self;

      }

    });

  })); //end of plugin
</script>
<script>

/**
 * jQuery Plugin: Sticky Tabs
 *
 * @author Aidan Lister <aidan@php.net>
 * adapted by Ruben Arslan to activate parent tabs too
 * http://www.aidanlister.com/2014/03/persisting-the-tab-state-in-bootstrap/
 */
(function($) {
  "use strict";
  $.fn.rmarkdownStickyTabs = function() {
    var context = this;
    // Show the tab corresponding with the hash in the URL, or the first tab
    var showStuffFromHash = function() {
      var hash = window.location.hash;
      var selector = hash ? 'a[href="' + hash + '"]' : 'li.active > a';
      var $selector = $(selector, context);
      if($selector.data('toggle') === "tab") {
        $selector.tab('show');
        // walk up the ancestors of this element, show any hidden tabs
        $selector.parents('.section.tabset').each(function(i, elm) {
          var link = $('a[href="#' + $(elm).attr('id') + '"]');
          if(link.data('toggle') === "tab") {
            link.tab("show");
          }
        });
      }
    };


    // Set the correct tab when the page loads
    showStuffFromHash(context);

    // Set the correct tab when a user uses their back/forward button
    $(window).on('hashchange', function() {
      showStuffFromHash(context);
    });

    // Change the URL when tabs are clicked
    $('a', context).on('click', function(e) {
      history.pushState(null, null, this.href);
      showStuffFromHash(context);
    });

    return this;
  };
}(jQuery));

window.buildTabsets = function(tocID) {

  // build a tabset from a section div with the .tabset class
  function buildTabset(tabset) {

    // check for fade and pills options
    var fade = tabset.hasClass("tabset-fade");
    var pills = tabset.hasClass("tabset-pills");
    var navClass = pills ? "nav-pills" : "nav-tabs";

    // determine the heading level of the tabset and tabs
    var match = tabset.attr('class').match(/level(\d) /);
    if (match === null)
      return;
    var tabsetLevel = Number(match[1]);
    var tabLevel = tabsetLevel + 1;

    // find all subheadings immediately below
    var tabs = tabset.find("div.section.level" + tabLevel);
    if (!tabs.length)
      return;

    // create tablist and tab-content elements
    var tabList = $('<ul class="nav ' + navClass + '" role="tablist"></ul>');
    $(tabs[0]).before(tabList);
    var tabContent = $('<div class="tab-content"></div>');
    $(tabs[0]).before(tabContent);

    // build the tabset
    var activeTab = 0;
    tabs.each(function(i) {

      // get the tab div
      var tab = $(tabs[i]);

      // get the id then sanitize it for use with bootstrap tabs
      var id = tab.attr('id');

      // see if this is marked as the active tab
      if (tab.hasClass('active'))
        activeTab = i;

      // remove any table of contents entries associated with
      // this ID (since we'll be removing the heading element)
      $("div#" + tocID + " li a[href='#" + id + "']").parent().remove();

      // sanitize the id for use with bootstrap tabs
      id = id.replace(/[.\/?&!#<>]/g, '').replace(/\s/g, '_');
      tab.attr('id', id);

      // get the heading element within it, grab it's text, then remove it
      var heading = tab.find('h' + tabLevel + ':first');
      var headingText = heading.html();
      heading.remove();

      // build and append the tab list item
      var a = $('<a role="tab" data-toggle="tab">' + headingText + '</a>');
      a.attr('href', '#' + id);
      a.attr('aria-controls', id);
      var li = $('<li role="presentation"></li>');
      li.append(a);
      tabList.append(li);

      // set it's attributes
      tab.attr('role', 'tabpanel');
      tab.addClass('tab-pane');
      tab.addClass('tabbed-pane');
      if (fade)
        tab.addClass('fade');

      // move it into the tab content div
      tab.detach().appendTo(tabContent);
    });

    // set active tab
    $(tabList.children('li')[activeTab]).addClass('active');
    var active = $(tabContent.children('div.section')[activeTab]);
    active.addClass('active');
    if (fade)
      active.addClass('in');

    if (tabset.hasClass("tabset-sticky"))
      tabset.rmarkdownStickyTabs();
  }

  // convert section divs with the .tabset class to tabsets
  var tabsets = $("div.section.tabset");
  tabsets.each(function(i) {
    buildTabset($(tabsets[i]));
  });
};

</script>
<script>
window.initializeCodeFolding = function(show) {

  // handlers for show-all and hide all
  $("#rmd-show-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('show');
    });
  });
  $("#rmd-hide-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('hide');
    });
  });

  // index for unique code element ids
  var currentIndex = 1;

  // select all R code blocks
  var rCodeBlocks = $('pre.r, pre.python, pre.bash, pre.sql, pre.cpp, pre.stan');
  rCodeBlocks.each(function() {

    // create a collapsable div to wrap the code in
    var div = $('<div class="collapse r-code-collapse"></div>');
    if (show)
      div.addClass('in');
    var id = 'rcode-643E0F36' + currentIndex++;
    div.attr('id', id);
    $(this).before(div);
    $(this).detach().appendTo(div);

    // add a show code button right above
    var showCodeText = $('<span>' + (show ? 'Hide' : 'Code') + '</span>');
    var showCodeButton = $('<button type="button" class="btn btn-default btn-xs code-folding-btn pull-right"></button>');
    showCodeButton.append(showCodeText);
    showCodeButton
        .attr('data-toggle', 'collapse')
        .attr('data-target', '#' + id)
        .attr('aria-expanded', show)
        .attr('aria-controls', id);

    var buttonRow = $('<div class="row"></div>');
    var buttonCol = $('<div class="col-md-12"></div>');

    buttonCol.append(showCodeButton);
    buttonRow.append(buttonCol);

    div.before(buttonRow);

    // update state of button on show/hide
    div.on('hidden.bs.collapse', function () {
      showCodeText.text('Code');
    });
    div.on('show.bs.collapse', function () {
      showCodeText.text('Hide');
    });
  });

}
</script>
<script>
window.initializeSourceEmbed = function(filename) {
  $("#rmd-download-source").click(function() {
    var src = $("#rmd-source-code").html();
    var a = document.createElement('a');
    a.href = "data:text/x-r-markdown;base64," + src;
    a.download = filename;
    document.body.appendChild(a);
    a.click();
    document.body.removeChild(a);
  });
};
</script>
<style type="text/css">.hljs-literal {
color: rgb(88, 72, 246);
}
.hljs-number {
color: rgb(0, 0, 205);
}
.hljs-comment {
color: rgb(76, 136, 107);
}
.hljs-keyword {
color: rgb(0, 0, 255);
}
.hljs-string {
color: rgb(3, 106, 7);
}
</style>
<script src="data:application/javascript;base64,/*! highlight.js v9.12.0 | BSD3 License | git.io/hljslicense */
!function(e){var n="object"==typeof window&&window||"object"==typeof self&&self;"undefined"!=typeof exports?e(exports):n&&(n.hljs=e({}),"function"==typeof define&&define.amd&&define([],function(){return n.hljs}))}(function(e){function n(e){return e.replace(/&/g,"&amp;").replace(/</g,"&lt;").replace(/>/g,"&gt;")}function t(e){return e.nodeName.toLowerCase()}function r(e,n){var t=e&&e.exec(n);return t&&0===t.index}function a(e){return k.test(e)}function i(e){var n,t,r,i,o=e.className+" ";if(o+=e.parentNode?e.parentNode.className:"",t=B.exec(o))return w(t[1])?t[1]:"no-highlight";for(o=o.split(/\s+/),n=0,r=o.length;r>n;n++)if(i=o[n],a(i)||w(i))return i}function o(e){var n,t={},r=Array.prototype.slice.call(arguments,1);for(n in e)t[n]=e[n];return r.forEach(function(e){for(n in e)t[n]=e[n]}),t}function u(e){var n=[];return function r(e,a){for(var i=e.firstChild;i;i=i.nextSibling)3===i.nodeType?a+=i.nodeValue.length:1===i.nodeType&&(n.push({event:"start",offset:a,node:i}),a=r(i,a),t(i).match(/br|hr|img|input/)||n.push({event:"stop",offset:a,node:i}));return a}(e,0),n}function c(e,r,a){function i(){return e.length&&r.length?e[0].offset!==r[0].offset?e[0].offset<r[0].offset?e:r:"start"===r[0].event?e:r:e.length?e:r}function o(e){function r(e){return" "+e.nodeName+'="'+n(e.value).replace('"',"&quot;")+'"'}s+="<"+t(e)+E.map.call(e.attributes,r).join("")+">"}function u(e){s+="</"+t(e)+">"}function c(e){("start"===e.event?o:u)(e.node)}for(var l=0,s="",f=[];e.length||r.length;){var g=i();if(s+=n(a.substring(l,g[0].offset)),l=g[0].offset,g===e){f.reverse().forEach(u);do c(g.splice(0,1)[0]),g=i();while(g===e&&g.length&&g[0].offset===l);f.reverse().forEach(o)}else"start"===g[0].event?f.push(g[0].node):f.pop(),c(g.splice(0,1)[0])}return s+n(a.substr(l))}function l(e){return e.v&&!e.cached_variants&&(e.cached_variants=e.v.map(function(n){return o(e,{v:null},n)})),e.cached_variants||e.eW&&[o(e)]||[e]}function s(e){function n(e){return e&&e.source||e}function t(t,r){return new RegExp(n(t),"m"+(e.cI?"i":"")+(r?"g":""))}function r(a,i){if(!a.compiled){if(a.compiled=!0,a.k=a.k||a.bK,a.k){var o={},u=function(n,t){e.cI&&(t=t.toLowerCase()),t.split(" ").forEach(function(e){var t=e.split("|");o[t[0]]=[n,t[1]?Number(t[1]):1]})};"string"==typeof a.k?u("keyword",a.k):x(a.k).forEach(function(e){u(e,a.k[e])}),a.k=o}a.lR=t(a.l||/\w+/,!0),i&&(a.bK&&(a.b="\\b("+a.bK.split(" ").join("|")+")\\b"),a.b||(a.b=/\B|\b/),a.bR=t(a.b),a.e||a.eW||(a.e=/\B|\b/),a.e&&(a.eR=t(a.e)),a.tE=n(a.e)||"",a.eW&&i.tE&&(a.tE+=(a.e?"|":"")+i.tE)),a.i&&(a.iR=t(a.i)),null==a.r&&(a.r=1),a.c||(a.c=[]),a.c=Array.prototype.concat.apply([],a.c.map(function(e){return l("self"===e?a:e)})),a.c.forEach(function(e){r(e,a)}),a.starts&&r(a.starts,i);var c=a.c.map(function(e){return e.bK?"\\.?("+e.b+")\\.?":e.b}).concat([a.tE,a.i]).map(n).filter(Boolean);a.t=c.length?t(c.join("|"),!0):{exec:function(){return null}}}}r(e)}function f(e,t,a,i){function o(e,n){var t,a;for(t=0,a=n.c.length;a>t;t++)if(r(n.c[t].bR,e))return n.c[t]}function u(e,n){if(r(e.eR,n)){for(;e.endsParent&&e.parent;)e=e.parent;return e}return e.eW?u(e.parent,n):void 0}function c(e,n){return!a&&r(n.iR,e)}function l(e,n){var t=N.cI?n[0].toLowerCase():n[0];return e.k.hasOwnProperty(t)&&e.k[t]}function p(e,n,t,r){var a=r?"":I.classPrefix,i='<span class="'+a,o=t?"":C;return i+=e+'">',i+n+o}function h(){var e,t,r,a;if(!E.k)return n(k);for(a="",t=0,E.lR.lastIndex=0,r=E.lR.exec(k);r;)a+=n(k.substring(t,r.index)),e=l(E,r),e?(B+=e[1],a+=p(e[0],n(r[0]))):a+=n(r[0]),t=E.lR.lastIndex,r=E.lR.exec(k);return a+n(k.substr(t))}function d(){var e="string"==typeof E.sL;if(e&&!y[E.sL])return n(k);var t=e?f(E.sL,k,!0,x[E.sL]):g(k,E.sL.length?E.sL:void 0);return E.r>0&&(B+=t.r),e&&(x[E.sL]=t.top),p(t.language,t.value,!1,!0)}function b(){L+=null!=E.sL?d():h(),k=""}function v(e){L+=e.cN?p(e.cN,"",!0):"",E=Object.create(e,{parent:{value:E}})}function m(e,n){if(k+=e,null==n)return b(),0;var t=o(n,E);if(t)return t.skip?k+=n:(t.eB&&(k+=n),b(),t.rB||t.eB||(k=n)),v(t,n),t.rB?0:n.length;var r=u(E,n);if(r){var a=E;a.skip?k+=n:(a.rE||a.eE||(k+=n),b(),a.eE&&(k=n));do E.cN&&(L+=C),E.skip||(B+=E.r),E=E.parent;while(E!==r.parent);return r.starts&&v(r.starts,""),a.rE?0:n.length}if(c(n,E))throw new Error('Illegal lexeme "'+n+'" for mode "'+(E.cN||"<unnamed>")+'"');return k+=n,n.length||1}var N=w(e);if(!N)throw new Error('Unknown language: "'+e+'"');s(N);var R,E=i||N,x={},L="";for(R=E;R!==N;R=R.parent)R.cN&&(L=p(R.cN,"",!0)+L);var k="",B=0;try{for(var M,j,O=0;;){if(E.t.lastIndex=O,M=E.t.exec(t),!M)break;j=m(t.substring(O,M.index),M[0]),O=M.index+j}for(m(t.substr(O)),R=E;R.parent;R=R.parent)R.cN&&(L+=C);return{r:B,value:L,language:e,top:E}}catch(T){if(T.message&&-1!==T.message.indexOf("Illegal"))return{r:0,value:n(t)};throw T}}function g(e,t){t=t||I.languages||x(y);var r={r:0,value:n(e)},a=r;return t.filter(w).forEach(function(n){var t=f(n,e,!1);t.language=n,t.r>a.r&&(a=t),t.r>r.r&&(a=r,r=t)}),a.language&&(r.second_best=a),r}function p(e){return I.tabReplace||I.useBR?e.replace(M,function(e,n){return I.useBR&&"\n"===e?"<br>":I.tabReplace?n.replace(/\t/g,I.tabReplace):""}):e}function h(e,n,t){var r=n?L[n]:t,a=[e.trim()];return e.match(/\bhljs\b/)||a.push("hljs"),-1===e.indexOf(r)&&a.push(r),a.join(" ").trim()}function d(e){var n,t,r,o,l,s=i(e);a(s)||(I.useBR?(n=document.createElementNS("http://www.w3.org/1999/xhtml","div"),n.innerHTML=e.innerHTML.replace(/\n/g,"").replace(/<br[ \/]*>/g,"\n")):n=e,l=n.textContent,r=s?f(s,l,!0):g(l),t=u(n),t.length&&(o=document.createElementNS("http://www.w3.org/1999/xhtml","div"),o.innerHTML=r.value,r.value=c(t,u(o),l)),r.value=p(r.value),e.innerHTML=r.value,e.className=h(e.className,s,r.language),e.result={language:r.language,re:r.r},r.second_best&&(e.second_best={language:r.second_best.language,re:r.second_best.r}))}function b(e){I=o(I,e)}function v(){if(!v.called){v.called=!0;var e=document.querySelectorAll("pre code");E.forEach.call(e,d)}}function m(){addEventListener("DOMContentLoaded",v,!1),addEventListener("load",v,!1)}function N(n,t){var r=y[n]=t(e);r.aliases&&r.aliases.forEach(function(e){L[e]=n})}function R(){return x(y)}function w(e){return e=(e||"").toLowerCase(),y[e]||y[L[e]]}var E=[],x=Object.keys,y={},L={},k=/^(no-?highlight|plain|text)$/i,B=/\blang(?:uage)?-([\w-]+)\b/i,M=/((^(<[^>]+>|\t|)+|(?:\n)))/gm,C="</span>",I={classPrefix:"hljs-",tabReplace:null,useBR:!1,languages:void 0};return e.highlight=f,e.highlightAuto=g,e.fixMarkup=p,e.highlightBlock=d,e.configure=b,e.initHighlighting=v,e.initHighlightingOnLoad=m,e.registerLanguage=N,e.listLanguages=R,e.getLanguage=w,e.inherit=o,e.IR="[a-zA-Z]\\w*",e.UIR="[a-zA-Z_]\\w*",e.NR="\\b\\d+(\\.\\d+)?",e.CNR="(-?)(\\b0[xX][a-fA-F0-9]+|(\\b\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)",e.BNR="\\b(0b[01]+)",e.RSR="!|!=|!==|%|%=|&|&&|&=|\\*|\\*=|\\+|\\+=|,|-|-=|/=|/|:|;|<<|<<=|<=|<|===|==|=|>>>=|>>=|>=|>>>|>>|>|\\?|\\[|\\{|\\(|\\^|\\^=|\\||\\|=|\\|\\||~",e.BE={b:"\\\\[\\s\\S]",r:0},e.ASM={cN:"string",b:"'",e:"'",i:"\\n",c:[e.BE]},e.QSM={cN:"string",b:'"',e:'"',i:"\\n",c:[e.BE]},e.PWM={b:/\b(a|an|the|are|I'm|isn't|don't|doesn't|won't|but|just|should|pretty|simply|enough|gonna|going|wtf|so|such|will|you|your|they|like|more)\b/},e.C=function(n,t,r){var a=e.inherit({cN:"comment",b:n,e:t,c:[]},r||{});return a.c.push(e.PWM),a.c.push({cN:"doctag",b:"(?:TODO|FIXME|NOTE|BUG|XXX):",r:0}),a},e.CLCM=e.C("//","$"),e.CBCM=e.C("/\\*","\\*/"),e.HCM=e.C("#","$"),e.NM={cN:"number",b:e.NR,r:0},e.CNM={cN:"number",b:e.CNR,r:0},e.BNM={cN:"number",b:e.BNR,r:0},e.CSSNM={cN:"number",b:e.NR+"(%|em|ex|ch|rem|vw|vh|vmin|vmax|cm|mm|in|pt|pc|px|deg|grad|rad|turn|s|ms|Hz|kHz|dpi|dpcm|dppx)?",r:0},e.RM={cN:"regexp",b:/\//,e:/\/[gimuy]*/,i:/\n/,c:[e.BE,{b:/\[/,e:/\]/,r:0,c:[e.BE]}]},e.TM={cN:"title",b:e.IR,r:0},e.UTM={cN:"title",b:e.UIR,r:0},e.METHOD_GUARD={b:"\\.\\s*"+e.UIR,r:0},e});hljs.registerLanguage("sql",function(e){var t=e.C("--","$");return{cI:!0,i:/[<>{}*#]/,c:[{bK:"begin end start commit rollback savepoint lock alter create drop rename call delete do handler insert load replace select truncate update set show pragma grant merge describe use explain help declare prepare execute deallocate release unlock purge reset change stop analyze cache flush optimize repair kill install uninstall checksum restore check backup revoke comment",e:/;/,eW:!0,l:/[\w\.]+/,k:{keyword:"abort abs absolute acc acce accep accept access accessed accessible account acos action activate add addtime admin administer advanced advise aes_decrypt aes_encrypt after agent aggregate ali alia alias allocate allow alter always analyze ancillary and any anydata anydataset anyschema anytype apply archive archived archivelog are as asc ascii asin assembly assertion associate asynchronous at atan atn2 attr attri attrib attribu attribut attribute attributes audit authenticated authentication authid authors auto autoallocate autodblink autoextend automatic availability avg backup badfile basicfile before begin beginning benchmark between bfile bfile_base big bigfile bin binary_double binary_float binlog bit_and bit_count bit_length bit_or bit_xor bitmap blob_base block blocksize body both bound buffer_cache buffer_pool build bulk by byte byteordermark bytes cache caching call calling cancel capacity cascade cascaded case cast catalog category ceil ceiling chain change changed char_base char_length character_length characters characterset charindex charset charsetform charsetid check checksum checksum_agg child choose chr chunk class cleanup clear client clob clob_base clone close cluster_id cluster_probability cluster_set clustering coalesce coercibility col collate collation collect colu colum column column_value columns columns_updated comment commit compact compatibility compiled complete composite_limit compound compress compute concat concat_ws concurrent confirm conn connec connect connect_by_iscycle connect_by_isleaf connect_by_root connect_time connection consider consistent constant constraint constraints constructor container content contents context contributors controlfile conv convert convert_tz corr corr_k corr_s corresponding corruption cos cost count count_big counted covar_pop covar_samp cpu_per_call cpu_per_session crc32 create creation critical cross cube cume_dist curdate current current_date current_time current_timestamp current_user cursor curtime customdatum cycle data database databases datafile datafiles datalength date_add date_cache date_format date_sub dateadd datediff datefromparts datename datepart datetime2fromparts day day_to_second dayname dayofmonth dayofweek dayofyear days db_role_change dbtimezone ddl deallocate declare decode decompose decrement decrypt deduplicate def defa defau defaul default defaults deferred defi defin define degrees delayed delegate delete delete_all delimited demand dense_rank depth dequeue des_decrypt des_encrypt des_key_file desc descr descri describ describe descriptor deterministic diagnostics difference dimension direct_load directory disable disable_all disallow disassociate discardfile disconnect diskgroup distinct distinctrow distribute distributed div do document domain dotnet double downgrade drop dumpfile duplicate duration each edition editionable editions element ellipsis else elsif elt empty enable enable_all enclosed encode encoding encrypt end end-exec endian enforced engine engines enqueue enterprise entityescaping eomonth error errors escaped evalname evaluate event eventdata events except exception exceptions exchange exclude excluding execu execut execute exempt exists exit exp expire explain export export_set extended extent external external_1 external_2 externally extract failed failed_login_attempts failover failure far fast feature_set feature_value fetch field fields file file_name_convert filesystem_like_logging final finish first first_value fixed flash_cache flashback floor flush following follows for forall force form forma format found found_rows freelist freelists freepools fresh from from_base64 from_days ftp full function general generated get get_format get_lock getdate getutcdate global global_name globally go goto grant grants greatest group group_concat group_id grouping grouping_id groups gtid_subtract guarantee guard handler hash hashkeys having hea head headi headin heading heap help hex hierarchy high high_priority hosts hour http id ident_current ident_incr ident_seed identified identity idle_time if ifnull ignore iif ilike ilm immediate import in include including increment index indexes indexing indextype indicator indices inet6_aton inet6_ntoa inet_aton inet_ntoa infile initial initialized initially initrans inmemory inner innodb input insert install instance instantiable instr interface interleaved intersect into invalidate invisible is is_free_lock is_ipv4 is_ipv4_compat is_not is_not_null is_used_lock isdate isnull isolation iterate java join json json_exists keep keep_duplicates key keys kill language large last last_day last_insert_id last_value lax lcase lead leading least leaves left len lenght length less level levels library like like2 like4 likec limit lines link list listagg little ln load load_file lob lobs local localtime localtimestamp locate locator lock locked log log10 log2 logfile logfiles logging logical logical_reads_per_call logoff logon logs long loop low low_priority lower lpad lrtrim ltrim main make_set makedate maketime managed management manual map mapping mask master master_pos_wait match matched materialized max maxextents maximize maxinstances maxlen maxlogfiles maxloghistory maxlogmembers maxsize maxtrans md5 measures median medium member memcompress memory merge microsecond mid migration min minextents minimum mining minus minute minvalue missing mod mode model modification modify module monitoring month months mount move movement multiset mutex name name_const names nan national native natural nav nchar nclob nested never new newline next nextval no no_write_to_binlog noarchivelog noaudit nobadfile nocheck nocompress nocopy nocycle nodelay nodiscardfile noentityescaping noguarantee nokeep nologfile nomapping nomaxvalue nominimize nominvalue nomonitoring none noneditionable nonschema noorder nopr nopro noprom nopromp noprompt norely noresetlogs noreverse normal norowdependencies noschemacheck noswitch not nothing notice notrim novalidate now nowait nth_value nullif nulls num numb numbe nvarchar nvarchar2 object ocicoll ocidate ocidatetime ociduration ociinterval ociloblocator ocinumber ociref ocirefcursor ocirowid ocistring ocitype oct octet_length of off offline offset oid oidindex old on online only opaque open operations operator optimal optimize option optionally or oracle oracle_date oradata ord ordaudio orddicom orddoc order ordimage ordinality ordvideo organization orlany orlvary out outer outfile outline output over overflow overriding package pad parallel parallel_enable parameters parent parse partial partition partitions pascal passing password password_grace_time password_lock_time password_reuse_max password_reuse_time password_verify_function patch path patindex pctincrease pctthreshold pctused pctversion percent percent_rank percentile_cont percentile_disc performance period period_add period_diff permanent physical pi pipe pipelined pivot pluggable plugin policy position post_transaction pow power pragma prebuilt precedes preceding precision prediction prediction_cost prediction_details prediction_probability prediction_set prepare present preserve prior priority private private_sga privileges procedural procedure procedure_analyze processlist profiles project prompt protection public publishingservername purge quarter query quick quiesce quota quotename radians raise rand range rank raw read reads readsize rebuild record records recover recovery recursive recycle redo reduced ref reference referenced references referencing refresh regexp_like register regr_avgx regr_avgy regr_count regr_intercept regr_r2 regr_slope regr_sxx regr_sxy reject rekey relational relative relaylog release release_lock relies_on relocate rely rem remainder rename repair repeat replace replicate replication required reset resetlogs resize resource respect restore restricted result result_cache resumable resume retention return returning returns reuse reverse revoke right rlike role roles rollback rolling rollup round row row_count rowdependencies rowid rownum rows rtrim rules safe salt sample save savepoint sb1 sb2 sb4 scan schema schemacheck scn scope scroll sdo_georaster sdo_topo_geometry search sec_to_time second section securefile security seed segment select self sequence sequential serializable server servererror session session_user sessions_per_user set sets settings sha sha1 sha2 share shared shared_pool short show shrink shutdown si_averagecolor si_colorhistogram si_featurelist si_positionalcolor si_stillimage si_texture siblings sid sign sin size size_t sizes skip slave sleep smalldatetimefromparts smallfile snapshot some soname sort soundex source space sparse spfile split sql sql_big_result sql_buffer_result sql_cache sql_calc_found_rows sql_small_result sql_variant_property sqlcode sqldata sqlerror sqlname sqlstate sqrt square standalone standby start starting startup statement static statistics stats_binomial_test stats_crosstab stats_ks_test stats_mode stats_mw_test stats_one_way_anova stats_t_test_ stats_t_test_indep stats_t_test_one stats_t_test_paired stats_wsr_test status std stddev stddev_pop stddev_samp stdev stop storage store stored str str_to_date straight_join strcmp strict string struct stuff style subdate subpartition subpartitions substitutable substr substring subtime subtring_index subtype success sum suspend switch switchoffset switchover sync synchronous synonym sys sys_xmlagg sysasm sysaux sysdate sysdatetimeoffset sysdba sysoper system system_user sysutcdatetime table tables tablespace tan tdo template temporary terminated tertiary_weights test than then thread through tier ties time time_format time_zone timediff timefromparts timeout timestamp timestampadd timestampdiff timezone_abbr timezone_minute timezone_region to to_base64 to_date to_days to_seconds todatetimeoffset trace tracking transaction transactional translate translation treat trigger trigger_nestlevel triggers trim truncate try_cast try_convert try_parse type ub1 ub2 ub4 ucase unarchived unbounded uncompress under undo unhex unicode uniform uninstall union unique unix_timestamp unknown unlimited unlock unpivot unrecoverable unsafe unsigned until untrusted unusable unused update updated upgrade upped upper upsert url urowid usable usage use use_stored_outlines user user_data user_resources users using utc_date utc_timestamp uuid uuid_short validate validate_password_strength validation valist value values var var_samp varcharc vari varia variab variabl variable variables variance varp varraw varrawc varray verify version versions view virtual visible void wait wallet warning warnings week weekday weekofyear wellformed when whene whenev wheneve whenever where while whitespace with within without work wrapped xdb xml xmlagg xmlattributes xmlcast xmlcolattval xmlelement xmlexists xmlforest xmlindex xmlnamespaces xmlpi xmlquery xmlroot xmlschema xmlserialize xmltable xmltype xor year year_to_month years yearweek",literal:"true false null",built_in:"array bigint binary bit blob boolean char character date dec decimal float int int8 integer interval number numeric real record serial serial8 smallint text varchar varying void"},c:[{cN:"string",b:"'",e:"'",c:[e.BE,{b:"''"}]},{cN:"string",b:'"',e:'"',c:[e.BE,{b:'""'}]},{cN:"string",b:"`",e:"`",c:[e.BE]},e.CNM,e.CBCM,t]},e.CBCM,t]}});hljs.registerLanguage("r",function(e){var r="([a-zA-Z]|\\.[a-zA-Z.])[a-zA-Z0-9._]*";return{c:[e.HCM,{b:r,l:r,k:{keyword:"function if in break next repeat else for return switch while try tryCatch stop warning require library attach detach source setMethod setGeneric setGroupGeneric setClass ...",literal:"NULL NA TRUE FALSE T F Inf NaN NA_integer_|10 NA_real_|10 NA_character_|10 NA_complex_|10"},r:0},{cN:"number",b:"0[xX][0-9a-fA-F]+[Li]?\\b",r:0},{cN:"number",b:"\\d+(?:[eE][+\\-]?\\d*)?L\\b",r:0},{cN:"number",b:"\\d+\\.(?!\\d)(?:i\\b)?",r:0},{cN:"number",b:"\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",r:0},{b:"`",e:"`",r:0},{cN:"string",c:[e.BE],v:[{b:'"',e:'"'},{b:"'",e:"'"}]}]}});hljs.registerLanguage("perl",function(e){var t="getpwent getservent quotemeta msgrcv scalar kill dbmclose undef lc ma syswrite tr send umask sysopen shmwrite vec qx utime local oct semctl localtime readpipe do return format read sprintf dbmopen pop getpgrp not getpwnam rewinddir qqfileno qw endprotoent wait sethostent bless s|0 opendir continue each sleep endgrent shutdown dump chomp connect getsockname die socketpair close flock exists index shmgetsub for endpwent redo lstat msgctl setpgrp abs exit select print ref gethostbyaddr unshift fcntl syscall goto getnetbyaddr join gmtime symlink semget splice x|0 getpeername recv log setsockopt cos last reverse gethostbyname getgrnam study formline endhostent times chop length gethostent getnetent pack getprotoent getservbyname rand mkdir pos chmod y|0 substr endnetent printf next open msgsnd readdir use unlink getsockopt getpriority rindex wantarray hex system getservbyport endservent int chr untie rmdir prototype tell listen fork shmread ucfirst setprotoent else sysseek link getgrgid shmctl waitpid unpack getnetbyname reset chdir grep split require caller lcfirst until warn while values shift telldir getpwuid my getprotobynumber delete and sort uc defined srand accept package seekdir getprotobyname semop our rename seek if q|0 chroot sysread setpwent no crypt getc chown sqrt write setnetent setpriority foreach tie sin msgget map stat getlogin unless elsif truncate exec keys glob tied closedirioctl socket readlink eval xor readline binmode setservent eof ord bind alarm pipe atan2 getgrent exp time push setgrent gt lt or ne m|0 break given say state when",r={cN:"subst",b:"[$@]\\{",e:"\\}",k:t},s={b:"->{",e:"}"},n={v:[{b:/\$\d/},{b:/[\$%@](\^\w\b|#\w+(::\w+)*|{\w+}|\w+(::\w*)*)/},{b:/[\$%@][^\s\w{]/,r:0}]},i=[e.BE,r,n],o=[n,e.HCM,e.C("^\\=\\w","\\=cut",{eW:!0}),s,{cN:"string",c:i,v:[{b:"q[qwxr]?\\s*\\(",e:"\\)",r:5},{b:"q[qwxr]?\\s*\\[",e:"\\]",r:5},{b:"q[qwxr]?\\s*\\{",e:"\\}",r:5},{b:"q[qwxr]?\\s*\\|",e:"\\|",r:5},{b:"q[qwxr]?\\s*\\<",e:"\\>",r:5},{b:"qw\\s+q",e:"q",r:5},{b:"'",e:"'",c:[e.BE]},{b:'"',e:'"'},{b:"`",e:"`",c:[e.BE]},{b:"{\\w+}",c:[],r:0},{b:"-?\\w+\\s*\\=\\>",c:[],r:0}]},{cN:"number",b:"(\\b0[0-7_]+)|(\\b0x[0-9a-fA-F_]+)|(\\b[1-9][0-9_]*(\\.[0-9_]+)?)|[0_]\\b",r:0},{b:"(\\/\\/|"+e.RSR+"|\\b(split|return|print|reverse|grep)\\b)\\s*",k:"split return print reverse grep",r:0,c:[e.HCM,{cN:"regexp",b:"(s|tr|y)/(\\\\.|[^/])*/(\\\\.|[^/])*/[a-z]*",r:10},{cN:"regexp",b:"(m|qr)?/",e:"/[a-z]*",c:[e.BE],r:0}]},{cN:"function",bK:"sub",e:"(\\s*\\(.*?\\))?[;{]",eE:!0,r:5,c:[e.TM]},{b:"-\\w\\b",r:0},{b:"^__DATA__$",e:"^__END__$",sL:"mojolicious",c:[{b:"^@@.*",e:"$",cN:"comment"}]}];return r.c=o,s.c=o,{aliases:["pl","pm"],l:/[\w\.]+/,k:t,c:o}});hljs.registerLanguage("ini",function(e){var b={cN:"string",c:[e.BE],v:[{b:"'''",e:"'''",r:10},{b:'"""',e:'"""',r:10},{b:'"',e:'"'},{b:"'",e:"'"}]};return{aliases:["toml"],cI:!0,i:/\S/,c:[e.C(";","$"),e.HCM,{cN:"section",b:/^\s*\[+/,e:/\]+/},{b:/^[a-z0-9\[\]_-]+\s*=\s*/,e:"$",rB:!0,c:[{cN:"attr",b:/[a-z0-9\[\]_-]+/},{b:/=/,eW:!0,r:0,c:[{cN:"literal",b:/\bon|off|true|false|yes|no\b/},{cN:"variable",v:[{b:/\$[\w\d"][\w\d_]*/},{b:/\$\{(.*?)}/}]},b,{cN:"number",b:/([\+\-]+)?[\d]+_[\d_]+/},e.NM]}]}]}});hljs.registerLanguage("diff",function(e){return{aliases:["patch"],c:[{cN:"meta",r:10,v:[{b:/^@@ +\-\d+,\d+ +\+\d+,\d+ +@@$/},{b:/^\*\*\* +\d+,\d+ +\*\*\*\*$/},{b:/^\-\-\- +\d+,\d+ +\-\-\-\-$/}]},{cN:"comment",v:[{b:/Index: /,e:/$/},{b:/={3,}/,e:/$/},{b:/^\-{3}/,e:/$/},{b:/^\*{3} /,e:/$/},{b:/^\+{3}/,e:/$/},{b:/\*{5}/,e:/\*{5}$/}]},{cN:"addition",b:"^\\+",e:"$"},{cN:"deletion",b:"^\\-",e:"$"},{cN:"addition",b:"^\\!",e:"$"}]}});hljs.registerLanguage("go",function(e){var t={keyword:"break default func interface select case map struct chan else goto package switch const fallthrough if range type continue for import return var go defer bool byte complex64 complex128 float32 float64 int8 int16 int32 int64 string uint8 uint16 uint32 uint64 int uint uintptr rune",literal:"true false iota nil",built_in:"append cap close complex copy imag len make new panic print println real recover delete"};return{aliases:["golang"],k:t,i:"</",c:[e.CLCM,e.CBCM,{cN:"string",v:[e.QSM,{b:"'",e:"[^\\\\]'"},{b:"`",e:"`"}]},{cN:"number",v:[{b:e.CNR+"[dflsi]",r:1},e.CNM]},{b:/:=/},{cN:"function",bK:"func",e:/\s*\{/,eE:!0,c:[e.TM,{cN:"params",b:/\(/,e:/\)/,k:t,i:/["']/}]}]}});hljs.registerLanguage("bash",function(e){var t={cN:"variable",v:[{b:/\$[\w\d#@][\w\d_]*/},{b:/\$\{(.*?)}/}]},s={cN:"string",b:/"/,e:/"/,c:[e.BE,t,{cN:"variable",b:/\$\(/,e:/\)/,c:[e.BE]}]},a={cN:"string",b:/'/,e:/'/};return{aliases:["sh","zsh"],l:/\b-?[a-z\._]+\b/,k:{keyword:"if then else elif fi for while in do done case esac function",literal:"true false",built_in:"break cd continue eval exec exit export getopts hash pwd readonly return shift test times trap umask unset alias bind builtin caller command declare echo enable help let local logout mapfile printf read readarray source type typeset ulimit unalias set shopt autoload bg bindkey bye cap chdir clone comparguments compcall compctl compdescribe compfiles compgroups compquote comptags comptry compvalues dirs disable disown echotc echoti emulate fc fg float functions getcap getln history integer jobs kill limit log noglob popd print pushd pushln rehash sched setcap setopt stat suspend ttyctl unfunction unhash unlimit unsetopt vared wait whence where which zcompile zformat zftp zle zmodload zparseopts zprof zpty zregexparse zsocket zstyle ztcp",_:"-ne -eq -lt -gt -f -d -e -s -l -a"},c:[{cN:"meta",b:/^#![^\n]+sh\s*$/,r:10},{cN:"function",b:/\w[\w\d_]*\s*\(\s*\)\s*\{/,rB:!0,c:[e.inherit(e.TM,{b:/\w[\w\d_]*/})],r:0},e.HCM,s,a,t]}});hljs.registerLanguage("python",function(e){var r={keyword:"and elif is global as in if from raise for except finally print import pass return exec else break not with class assert yield try while continue del or def lambda async await nonlocal|10 None True False",built_in:"Ellipsis NotImplemented"},b={cN:"meta",b:/^(>>>|\.\.\.) /},c={cN:"subst",b:/\{/,e:/\}/,k:r,i:/#/},a={cN:"string",c:[e.BE],v:[{b:/(u|b)?r?'''/,e:/'''/,c:[b],r:10},{b:/(u|b)?r?"""/,e:/"""/,c:[b],r:10},{b:/(fr|rf|f)'''/,e:/'''/,c:[b,c]},{b:/(fr|rf|f)"""/,e:/"""/,c:[b,c]},{b:/(u|r|ur)'/,e:/'/,r:10},{b:/(u|r|ur)"/,e:/"/,r:10},{b:/(b|br)'/,e:/'/},{b:/(b|br)"/,e:/"/},{b:/(fr|rf|f)'/,e:/'/,c:[c]},{b:/(fr|rf|f)"/,e:/"/,c:[c]},e.ASM,e.QSM]},s={cN:"number",r:0,v:[{b:e.BNR+"[lLjJ]?"},{b:"\\b(0o[0-7]+)[lLjJ]?"},{b:e.CNR+"[lLjJ]?"}]},i={cN:"params",b:/\(/,e:/\)/,c:["self",b,s,a]};return c.c=[a,s,b],{aliases:["py","gyp"],k:r,i:/(<\/|->|\?)|=>/,c:[b,s,a,e.HCM,{v:[{cN:"function",bK:"def"},{cN:"class",bK:"class"}],e:/:/,i:/[${=;\n,]/,c:[e.UTM,i,{b:/->/,eW:!0,k:"None"}]},{cN:"meta",b:/^[\t ]*@/,e:/$/},{b:/\b(print|exec)\(/}]}});hljs.registerLanguage("julia",function(e){var r={keyword:"in isa where baremodule begin break catch ccall const continue do else elseif end export false finally for function global if import importall let local macro module quote return true try using while type immutable abstract bitstype typealias ",literal:"true false ARGS C_NULL DevNull ENDIAN_BOM ENV I Inf Inf16 Inf32 Inf64 InsertionSort JULIA_HOME LOAD_PATH MergeSort NaN NaN16 NaN32 NaN64 PROGRAM_FILE QuickSort RoundDown RoundFromZero RoundNearest RoundNearestTiesAway RoundNearestTiesUp RoundToZero RoundUp STDERR STDIN STDOUT VERSION catalan e|0 eu|0 eulergamma golden im nothing pi γ π φ ",built_in:"ANY AbstractArray AbstractChannel AbstractFloat AbstractMatrix AbstractRNG AbstractSerializer AbstractSet AbstractSparseArray AbstractSparseMatrix AbstractSparseVector AbstractString AbstractUnitRange AbstractVecOrMat AbstractVector Any ArgumentError Array AssertionError Associative Base64DecodePipe Base64EncodePipe Bidiagonal BigFloat BigInt BitArray BitMatrix BitVector Bool BoundsError BufferStream CachingPool CapturedException CartesianIndex CartesianRange Cchar Cdouble Cfloat Channel Char Cint Cintmax_t Clong Clonglong ClusterManager Cmd CodeInfo Colon Complex Complex128 Complex32 Complex64 CompositeException Condition ConjArray ConjMatrix ConjVector Cptrdiff_t Cshort Csize_t Cssize_t Cstring Cuchar Cuint Cuintmax_t Culong Culonglong Cushort Cwchar_t Cwstring DataType Date DateFormat DateTime DenseArray DenseMatrix DenseVecOrMat DenseVector Diagonal Dict DimensionMismatch Dims DirectIndexString Display DivideError DomainError EOFError EachLine Enum Enumerate ErrorException Exception ExponentialBackOff Expr Factorization FileMonitor Float16 Float32 Float64 Function Future GlobalRef GotoNode HTML Hermitian IO IOBuffer IOContext IOStream IPAddr IPv4 IPv6 IndexCartesian IndexLinear IndexStyle InexactError InitError Int Int128 Int16 Int32 Int64 Int8 IntSet Integer InterruptException InvalidStateException Irrational KeyError LabelNode LinSpace LineNumberNode LoadError LowerTriangular MIME Matrix MersenneTwister Method MethodError MethodTable Module NTuple NewvarNode NullException Nullable Number ObjectIdDict OrdinalRange OutOfMemoryError OverflowError Pair ParseError PartialQuickSort PermutedDimsArray Pipe PollingFileWatcher ProcessExitedException Ptr QuoteNode RandomDevice Range RangeIndex Rational RawFD ReadOnlyMemoryError Real ReentrantLock Ref Regex RegexMatch RemoteChannel RemoteException RevString RoundingMode RowVector SSAValue SegmentationFault SerializationState Set SharedArray SharedMatrix SharedVector Signed SimpleVector Slot SlotNumber SparseMatrixCSC SparseVector StackFrame StackOverflowError StackTrace StepRange StepRangeLen StridedArray StridedMatrix StridedVecOrMat StridedVector String SubArray SubString SymTridiagonal Symbol Symmetric SystemError TCPSocket Task Text TextDisplay Timer Tridiagonal Tuple Type TypeError TypeMapEntry TypeMapLevel TypeName TypeVar TypedSlot UDPSocket UInt UInt128 UInt16 UInt32 UInt64 UInt8 UndefRefError UndefVarError UnicodeError UniformScaling Union UnionAll UnitRange Unsigned UpperTriangular Val Vararg VecElement VecOrMat Vector VersionNumber Void WeakKeyDict WeakRef WorkerConfig WorkerPool "},t="[A-Za-z_\\u00A1-\\uFFFF][A-Za-z_0-9\\u00A1-\\uFFFF]*",a={l:t,k:r,i:/<\//},n={cN:"number",b:/(\b0x[\d_]*(\.[\d_]*)?|0x\.\d[\d_]*)p[-+]?\d+|\b0[box][a-fA-F0-9][a-fA-F0-9_]*|(\b\d[\d_]*(\.[\d_]*)?|\.\d[\d_]*)([eEfF][-+]?\d+)?/,r:0},o={cN:"string",b:/'(.|\\[xXuU][a-zA-Z0-9]+)'/},i={cN:"subst",b:/\$\(/,e:/\)/,k:r},l={cN:"variable",b:"\\$"+t},c={cN:"string",c:[e.BE,i,l],v:[{b:/\w*"""/,e:/"""\w*/,r:10},{b:/\w*"/,e:/"\w*/}]},s={cN:"string",c:[e.BE,i,l],b:"`",e:"`"},d={cN:"meta",b:"@"+t},u={cN:"comment",v:[{b:"#=",e:"=#",r:10},{b:"#",e:"$"}]};return a.c=[n,o,c,s,d,u,e.HCM,{cN:"keyword",b:"\\b(((abstract|primitive)\\s+)type|(mutable\\s+)?struct)\\b"},{b:/<:/}],i.c=a.c,a});hljs.registerLanguage("coffeescript",function(e){var c={keyword:"in if for while finally new do return else break catch instanceof throw try this switch continue typeof delete debugger super yield import export from as default await then unless until loop of by when and or is isnt not",literal:"true false null undefined yes no on off",built_in:"npm require console print module global window document"},n="[A-Za-z$_][0-9A-Za-z$_]*",r={cN:"subst",b:/#\{/,e:/}/,k:c},i=[e.BNM,e.inherit(e.CNM,{starts:{e:"(\\s*/)?",r:0}}),{cN:"string",v:[{b:/'''/,e:/'''/,c:[e.BE]},{b:/'/,e:/'/,c:[e.BE]},{b:/"""/,e:/"""/,c:[e.BE,r]},{b:/"/,e:/"/,c:[e.BE,r]}]},{cN:"regexp",v:[{b:"///",e:"///",c:[r,e.HCM]},{b:"//[gim]*",r:0},{b:/\/(?![ *])(\\\/|.)*?\/[gim]*(?=\W|$)/}]},{b:"@"+n},{sL:"javascript",eB:!0,eE:!0,v:[{b:"```",e:"```"},{b:"`",e:"`"}]}];r.c=i;var s=e.inherit(e.TM,{b:n}),t="(\\(.*\\))?\\s*\\B[-=]>",o={cN:"params",b:"\\([^\\(]",rB:!0,c:[{b:/\(/,e:/\)/,k:c,c:["self"].concat(i)}]};return{aliases:["coffee","cson","iced"],k:c,i:/\/\*/,c:i.concat([e.C("###","###"),e.HCM,{cN:"function",b:"^\\s*"+n+"\\s*=\\s*"+t,e:"[-=]>",rB:!0,c:[s,o]},{b:/[:\(,=]\s*/,r:0,c:[{cN:"function",b:t,e:"[-=]>",rB:!0,c:[o]}]},{cN:"class",bK:"class",e:"$",i:/[:="\[\]]/,c:[{bK:"extends",eW:!0,i:/[:="\[\]]/,c:[s]},s]},{b:n+":",e:":",rB:!0,rE:!0,r:0}])}});hljs.registerLanguage("cpp",function(t){var e={cN:"keyword",b:"\\b[a-z\\d_]*_t\\b"},r={cN:"string",v:[{b:'(u8?|U)?L?"',e:'"',i:"\\n",c:[t.BE]},{b:'(u8?|U)?R"',e:'"',c:[t.BE]},{b:"'\\\\?.",e:"'",i:"."}]},s={cN:"number",v:[{b:"\\b(0b[01']+)"},{b:"(-?)\\b([\\d']+(\\.[\\d']*)?|\\.[\\d']+)(u|U|l|L|ul|UL|f|F|b|B)"},{b:"(-?)(\\b0[xX][a-fA-F0-9']+|(\\b[\\d']+(\\.[\\d']*)?|\\.[\\d']+)([eE][-+]?[\\d']+)?)"}],r:0},i={cN:"meta",b:/#\s*[a-z]+\b/,e:/$/,k:{"meta-keyword":"if else elif endif define undef warning error line pragma ifdef ifndef include"},c:[{b:/\\\n/,r:0},t.inherit(r,{cN:"meta-string"}),{cN:"meta-string",b:/<[^\n>]*>/,e:/$/,i:"\\n"},t.CLCM,t.CBCM]},a=t.IR+"\\s*\\(",c={keyword:"int float while private char catch import module export virtual operator sizeof dynamic_cast|10 typedef const_cast|10 const for static_cast|10 union namespace unsigned long volatile static protected bool template mutable if public friend do goto auto void enum else break extern using asm case typeid short reinterpret_cast|10 default double register explicit signed typename try this switch continue inline delete alignof constexpr decltype noexcept static_assert thread_local restrict _Bool complex _Complex _Imaginary atomic_bool atomic_char atomic_schar atomic_uchar atomic_short atomic_ushort atomic_int atomic_uint atomic_long atomic_ulong atomic_llong atomic_ullong new throw return and or not",built_in:"std string cin cout cerr clog stdin stdout stderr stringstream istringstream ostringstream auto_ptr deque list queue stack vector map set bitset multiset multimap unordered_set unordered_map unordered_multiset unordered_multimap array shared_ptr abort abs acos asin atan2 atan calloc ceil cosh cos exit exp fabs floor fmod fprintf fputs free frexp fscanf isalnum isalpha iscntrl isdigit isgraph islower isprint ispunct isspace isupper isxdigit tolower toupper labs ldexp log10 log malloc realloc memchr memcmp memcpy memset modf pow printf putchar puts scanf sinh sin snprintf sprintf sqrt sscanf strcat strchr strcmp strcpy strcspn strlen strncat strncmp strncpy strpbrk strrchr strspn strstr tanh tan vfprintf vprintf vsprintf endl initializer_list unique_ptr",literal:"true false nullptr NULL"},n=[e,t.CLCM,t.CBCM,s,r];return{aliases:["c","cc","h","c++","h++","hpp"],k:c,i:"</",c:n.concat([i,{b:"\\b(deque|list|queue|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<",e:">",k:c,c:["self",e]},{b:t.IR+"::",k:c},{v:[{b:/=/,e:/;/},{b:/\(/,e:/\)/},{bK:"new throw return else",e:/;/}],k:c,c:n.concat([{b:/\(/,e:/\)/,k:c,c:n.concat(["self"]),r:0}]),r:0},{cN:"function",b:"("+t.IR+"[\\*&\\s]+)+"+a,rB:!0,e:/[{;=]/,eE:!0,k:c,i:/[^\w\s\*&]/,c:[{b:a,rB:!0,c:[t.TM],r:0},{cN:"params",b:/\(/,e:/\)/,k:c,r:0,c:[t.CLCM,t.CBCM,r,s,e]},t.CLCM,t.CBCM,i]},{cN:"class",bK:"class struct",e:/[{;:]/,c:[{b:/</,e:/>/,c:["self"]},t.TM]}]),exports:{preprocessor:i,strings:r,k:c}}});hljs.registerLanguage("ruby",function(e){var b="[a-zA-Z_]\\w*[!?=]?|[-+~]\\@|<<|>>|=~|===?|<=>|[<>]=?|\\*\\*|[-/+%^&*~`|]|\\[\\]=?",r={keyword:"and then defined module in return redo if BEGIN retry end for self when next until do begin unless END rescue else break undef not super class case require yield alias while ensure elsif or include attr_reader attr_writer attr_accessor",literal:"true false nil"},c={cN:"doctag",b:"@[A-Za-z]+"},a={b:"#<",e:">"},s=[e.C("#","$",{c:[c]}),e.C("^\\=begin","^\\=end",{c:[c],r:10}),e.C("^__END__","\\n$")],n={cN:"subst",b:"#\\{",e:"}",k:r},t={cN:"string",c:[e.BE,n],v:[{b:/'/,e:/'/},{b:/"/,e:/"/},{b:/`/,e:/`/},{b:"%[qQwWx]?\\(",e:"\\)"},{b:"%[qQwWx]?\\[",e:"\\]"},{b:"%[qQwWx]?{",e:"}"},{b:"%[qQwWx]?<",e:">"},{b:"%[qQwWx]?/",e:"/"},{b:"%[qQwWx]?%",e:"%"},{b:"%[qQwWx]?-",e:"-"},{b:"%[qQwWx]?\\|",e:"\\|"},{b:/\B\?(\\\d{1,3}|\\x[A-Fa-f0-9]{1,2}|\\u[A-Fa-f0-9]{4}|\\?\S)\b/},{b:/<<(-?)\w+$/,e:/^\s*\w+$/}]},i={cN:"params",b:"\\(",e:"\\)",endsParent:!0,k:r},d=[t,a,{cN:"class",bK:"class module",e:"$|;",i:/=/,c:[e.inherit(e.TM,{b:"[A-Za-z_]\\w*(::\\w+)*(\\?|\\!)?"}),{b:"<\\s*",c:[{b:"("+e.IR+"::)?"+e.IR}]}].concat(s)},{cN:"function",bK:"def",e:"$|;",c:[e.inherit(e.TM,{b:b}),i].concat(s)},{b:e.IR+"::"},{cN:"symbol",b:e.UIR+"(\\!|\\?)?:",r:0},{cN:"symbol",b:":(?!\\s)",c:[t,{b:b}],r:0},{cN:"number",b:"(\\b0[0-7_]+)|(\\b0x[0-9a-fA-F_]+)|(\\b[1-9][0-9_]*(\\.[0-9_]+)?)|[0_]\\b",r:0},{b:"(\\$\\W)|((\\$|\\@\\@?)(\\w+))"},{cN:"params",b:/\|/,e:/\|/,k:r},{b:"("+e.RSR+"|unless)\\s*",k:"unless",c:[a,{cN:"regexp",c:[e.BE,n],i:/\n/,v:[{b:"/",e:"/[a-z]*"},{b:"%r{",e:"}[a-z]*"},{b:"%r\\(",e:"\\)[a-z]*"},{b:"%r!",e:"![a-z]*"},{b:"%r\\[",e:"\\][a-z]*"}]}].concat(s),r:0}].concat(s);n.c=d,i.c=d;var l="[>?]>",o="[\\w#]+\\(\\w+\\):\\d+:\\d+>",u="(\\w+-)?\\d+\\.\\d+\\.\\d(p\\d+)?[^>]+>",w=[{b:/^\s*=>/,starts:{e:"$",c:d}},{cN:"meta",b:"^("+l+"|"+o+"|"+u+")",starts:{e:"$",c:d}}];return{aliases:["rb","gemspec","podspec","thor","irb"],k:r,i:/\/\*/,c:s.concat(w).concat(d)}});hljs.registerLanguage("yaml",function(e){var b="true false yes no null",a="^[ \\-]*",r="[a-zA-Z_][\\w\\-]*",t={cN:"attr",v:[{b:a+r+":"},{b:a+'"'+r+'":'},{b:a+"'"+r+"':"}]},c={cN:"template-variable",v:[{b:"{{",e:"}}"},{b:"%{",e:"}"}]},l={cN:"string",r:0,v:[{b:/'/,e:/'/},{b:/"/,e:/"/},{b:/\S+/}],c:[e.BE,c]};return{cI:!0,aliases:["yml","YAML","yaml"],c:[t,{cN:"meta",b:"^---s*$",r:10},{cN:"string",b:"[\\|>] *$",rE:!0,c:l.c,e:t.v[0].b},{b:"<%[%=-]?",e:"[%-]?%>",sL:"ruby",eB:!0,eE:!0,r:0},{cN:"type",b:"!!"+e.UIR},{cN:"meta",b:"&"+e.UIR+"$"},{cN:"meta",b:"\\*"+e.UIR+"$"},{cN:"bullet",b:"^ *-",r:0},e.HCM,{bK:b,k:{literal:b}},e.CNM,l]}});hljs.registerLanguage("css",function(e){var c="[a-zA-Z-][a-zA-Z0-9_-]*",t={b:/[A-Z\_\.\-]+\s*:/,rB:!0,e:";",eW:!0,c:[{cN:"attribute",b:/\S/,e:":",eE:!0,starts:{eW:!0,eE:!0,c:[{b:/[\w-]+\(/,rB:!0,c:[{cN:"built_in",b:/[\w-]+/},{b:/\(/,e:/\)/,c:[e.ASM,e.QSM]}]},e.CSSNM,e.QSM,e.ASM,e.CBCM,{cN:"number",b:"#[0-9A-Fa-f]+"},{cN:"meta",b:"!important"}]}}]};return{cI:!0,i:/[=\/|'\$]/,c:[e.CBCM,{cN:"selector-id",b:/#[A-Za-z0-9_-]+/},{cN:"selector-class",b:/\.[A-Za-z0-9_-]+/},{cN:"selector-attr",b:/\[/,e:/\]/,i:"$"},{cN:"selector-pseudo",b:/:(:)?[a-zA-Z0-9\_\-\+\(\)"'.]+/},{b:"@(font-face|page)",l:"[a-z-]+",k:"font-face page"},{b:"@",e:"[{;]",i:/:/,c:[{cN:"keyword",b:/\w+/},{b:/\s/,eW:!0,eE:!0,r:0,c:[e.ASM,e.QSM,e.CSSNM]}]},{cN:"selector-tag",b:c,r:0},{b:"{",e:"}",i:/\S/,c:[e.CBCM,t]}]}});hljs.registerLanguage("fortran",function(e){var t={cN:"params",b:"\\(",e:"\\)"},n={literal:".False. .True.",keyword:"kind do while private call intrinsic where elsewhere type endtype endmodule endselect endinterface end enddo endif if forall endforall only contains default return stop then public subroutine|10 function program .and. .or. .not. .le. .eq. .ge. .gt. .lt. goto save else use module select case access blank direct exist file fmt form formatted iostat name named nextrec number opened rec recl sequential status unformatted unit continue format pause cycle exit c_null_char c_alert c_backspace c_form_feed flush wait decimal round iomsg synchronous nopass non_overridable pass protected volatile abstract extends import non_intrinsic value deferred generic final enumerator class associate bind enum c_int c_short c_long c_long_long c_signed_char c_size_t c_int8_t c_int16_t c_int32_t c_int64_t c_int_least8_t c_int_least16_t c_int_least32_t c_int_least64_t c_int_fast8_t c_int_fast16_t c_int_fast32_t c_int_fast64_t c_intmax_t C_intptr_t c_float c_double c_long_double c_float_complex c_double_complex c_long_double_complex c_bool c_char c_null_ptr c_null_funptr c_new_line c_carriage_return c_horizontal_tab c_vertical_tab iso_c_binding c_loc c_funloc c_associated  c_f_pointer c_ptr c_funptr iso_fortran_env character_storage_size error_unit file_storage_size input_unit iostat_end iostat_eor numeric_storage_size output_unit c_f_procpointer ieee_arithmetic ieee_support_underflow_control ieee_get_underflow_mode ieee_set_underflow_mode newunit contiguous recursive pad position action delim readwrite eor advance nml interface procedure namelist include sequence elemental pure integer real character complex logical dimension allocatable|10 parameter external implicit|10 none double precision assign intent optional pointer target in out common equivalence data",built_in:"alog alog10 amax0 amax1 amin0 amin1 amod cabs ccos cexp clog csin csqrt dabs dacos dasin datan datan2 dcos dcosh ddim dexp dint dlog dlog10 dmax1 dmin1 dmod dnint dsign dsin dsinh dsqrt dtan dtanh float iabs idim idint idnint ifix isign max0 max1 min0 min1 sngl algama cdabs cdcos cdexp cdlog cdsin cdsqrt cqabs cqcos cqexp cqlog cqsin cqsqrt dcmplx dconjg derf derfc dfloat dgamma dimag dlgama iqint qabs qacos qasin qatan qatan2 qcmplx qconjg qcos qcosh qdim qerf qerfc qexp qgamma qimag qlgama qlog qlog10 qmax1 qmin1 qmod qnint qsign qsin qsinh qsqrt qtan qtanh abs acos aimag aint anint asin atan atan2 char cmplx conjg cos cosh exp ichar index int log log10 max min nint sign sin sinh sqrt tan tanh print write dim lge lgt lle llt mod nullify allocate deallocate adjustl adjustr all allocated any associated bit_size btest ceiling count cshift date_and_time digits dot_product eoshift epsilon exponent floor fraction huge iand ibclr ibits ibset ieor ior ishft ishftc lbound len_trim matmul maxexponent maxloc maxval merge minexponent minloc minval modulo mvbits nearest pack present product radix random_number random_seed range repeat reshape rrspacing scale scan selected_int_kind selected_real_kind set_exponent shape size spacing spread sum system_clock tiny transpose trim ubound unpack verify achar iachar transfer dble entry dprod cpu_time command_argument_count get_command get_command_argument get_environment_variable is_iostat_end ieee_arithmetic ieee_support_underflow_control ieee_get_underflow_mode ieee_set_underflow_mode is_iostat_eor move_alloc new_line selected_char_kind same_type_as extends_type_ofacosh asinh atanh bessel_j0 bessel_j1 bessel_jn bessel_y0 bessel_y1 bessel_yn erf erfc erfc_scaled gamma log_gamma hypot norm2 atomic_define atomic_ref execute_command_line leadz trailz storage_size merge_bits bge bgt ble blt dshiftl dshiftr findloc iall iany iparity image_index lcobound ucobound maskl maskr num_images parity popcnt poppar shifta shiftl shiftr this_image"};return{cI:!0,aliases:["f90","f95"],k:n,i:/\/\*/,c:[e.inherit(e.ASM,{cN:"string",r:0}),e.inherit(e.QSM,{cN:"string",r:0}),{cN:"function",bK:"subroutine function program",i:"[${=\\n]",c:[e.UTM,t]},e.C("!","$",{r:0}),{cN:"number",b:"(?=\\b|\\+|\\-|\\.)(?=\\.\\d|\\d)(?:\\d+)?(?:\\.?\\d*)(?:[de][+-]?\\d+)?\\b\\.?",r:0}]}});hljs.registerLanguage("awk",function(e){var r={cN:"variable",v:[{b:/\$[\w\d#@][\w\d_]*/},{b:/\$\{(.*?)}/}]},b="BEGIN END if else while do for in break continue delete next nextfile function func exit|10",n={cN:"string",c:[e.BE],v:[{b:/(u|b)?r?'''/,e:/'''/,r:10},{b:/(u|b)?r?"""/,e:/"""/,r:10},{b:/(u|r|ur)'/,e:/'/,r:10},{b:/(u|r|ur)"/,e:/"/,r:10},{b:/(b|br)'/,e:/'/},{b:/(b|br)"/,e:/"/},e.ASM,e.QSM]};return{k:{keyword:b},c:[r,n,e.RM,e.HCM,e.NM]}});hljs.registerLanguage("makefile",function(e){var i={cN:"variable",v:[{b:"\\$\\("+e.UIR+"\\)",c:[e.BE]},{b:/\$[@%<?\^\+\*]/}]},r={cN:"string",b:/"/,e:/"/,c:[e.BE,i]},a={cN:"variable",b:/\$\([\w-]+\s/,e:/\)/,k:{built_in:"subst patsubst strip findstring filter filter-out sort word wordlist firstword lastword dir notdir suffix basename addsuffix addprefix join wildcard realpath abspath error warning shell origin flavor foreach if or and call eval file value"},c:[i]},n={b:"^"+e.UIR+"\\s*[:+?]?=",i:"\\n",rB:!0,c:[{b:"^"+e.UIR,e:"[:+?]?=",eE:!0}]},t={cN:"meta",b:/^\.PHONY:/,e:/$/,k:{"meta-keyword":".PHONY"},l:/[\.\w]+/},l={cN:"section",b:/^[^\s]+:/,e:/$/,c:[i]};return{aliases:["mk","mak"],k:"define endef undefine ifdef ifndef ifeq ifneq else endif include -include sinclude override export unexport private vpath",l:/[\w-]+/,c:[e.HCM,i,r,a,n,t,l]}});hljs.registerLanguage("java",function(e){var a="[À-ʸa-zA-Z_$][À-ʸa-zA-Z_$0-9]*",t=a+"(<"+a+"(\\s*,\\s*"+a+")*>)?",r="false synchronized int abstract float private char boolean static null if const for true while long strictfp finally protected import native final void enum else break transient catch instanceof byte super volatile case assert short package default double public try this switch continue throws protected public private module requires exports do",s="\\b(0[bB]([01]+[01_]+[01]+|[01]+)|0[xX]([a-fA-F0-9]+[a-fA-F0-9_]+[a-fA-F0-9]+|[a-fA-F0-9]+)|(([\\d]+[\\d_]+[\\d]+|[\\d]+)(\\.([\\d]+[\\d_]+[\\d]+|[\\d]+))?|\\.([\\d]+[\\d_]+[\\d]+|[\\d]+))([eE][-+]?\\d+)?)[lLfF]?",c={cN:"number",b:s,r:0};return{aliases:["jsp"],k:r,i:/<\/|#/,c:[e.C("/\\*\\*","\\*/",{r:0,c:[{b:/\w+@/,r:0},{cN:"doctag",b:"@[A-Za-z]+"}]}),e.CLCM,e.CBCM,e.ASM,e.QSM,{cN:"class",bK:"class interface",e:/[{;=]/,eE:!0,k:"class interface",i:/[:"\[\]]/,c:[{bK:"extends implements"},e.UTM]},{bK:"new throw return else",r:0},{cN:"function",b:"("+t+"\\s+)+"+e.UIR+"\\s*\\(",rB:!0,e:/[{;=]/,eE:!0,k:r,c:[{b:e.UIR+"\\s*\\(",rB:!0,r:0,c:[e.UTM]},{cN:"params",b:/\(/,e:/\)/,k:r,r:0,c:[e.ASM,e.QSM,e.CNM,e.CBCM]},e.CLCM,e.CBCM]},c,{cN:"meta",b:"@[A-Za-z]+"}]}});hljs.registerLanguage("stan",function(e){return{c:[e.HCM,e.CLCM,e.CBCM,{b:e.UIR,l:e.UIR,k:{name:"for in while repeat until if then else",symbol:"bernoulli bernoulli_logit binomial binomial_logit beta_binomial hypergeometric categorical categorical_logit ordered_logistic neg_binomial neg_binomial_2 neg_binomial_2_log poisson poisson_log multinomial normal exp_mod_normal skew_normal student_t cauchy double_exponential logistic gumbel lognormal chi_square inv_chi_square scaled_inv_chi_square exponential inv_gamma weibull frechet rayleigh wiener pareto pareto_type_2 von_mises uniform multi_normal multi_normal_prec multi_normal_cholesky multi_gp multi_gp_cholesky multi_student_t gaussian_dlm_obs dirichlet lkj_corr lkj_corr_cholesky wishart inv_wishart","selector-tag":"int real vector simplex unit_vector ordered positive_ordered row_vector matrix cholesky_factor_corr cholesky_factor_cov corr_matrix cov_matrix",title:"functions model data parameters quantities transformed generated",literal:"true false"},r:0},{cN:"number",b:"0[xX][0-9a-fA-F]+[Li]?\\b",r:0},{cN:"number",b:"0[xX][0-9a-fA-F]+[Li]?\\b",r:0},{cN:"number",b:"\\d+(?:[eE][+\\-]?\\d*)?L\\b",r:0},{cN:"number",b:"\\d+\\.(?!\\d)(?:i\\b)?",r:0},{cN:"number",b:"\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",r:0}]}});hljs.registerLanguage("javascript",function(e){var r="[A-Za-z$_][0-9A-Za-z$_]*",t={keyword:"in of if for while finally var new function do return void else break catch instanceof with throw case default try this switch continue typeof delete let yield const export super debugger as async await static import from as",literal:"true false null undefined NaN Infinity",built_in:"eval isFinite isNaN parseFloat parseInt decodeURI decodeURIComponent encodeURI encodeURIComponent escape unescape Object Function Boolean Error EvalError InternalError RangeError ReferenceError StopIteration SyntaxError TypeError URIError Number Math Date String RegExp Array Float32Array Float64Array Int16Array Int32Array Int8Array Uint16Array Uint32Array Uint8Array Uint8ClampedArray ArrayBuffer DataView JSON Intl arguments require module console window document Symbol Set Map WeakSet WeakMap Proxy Reflect Promise"},a={cN:"number",v:[{b:"\\b(0[bB][01]+)"},{b:"\\b(0[oO][0-7]+)"},{b:e.CNR}],r:0},n={cN:"subst",b:"\\$\\{",e:"\\}",k:t,c:[]},c={cN:"string",b:"`",e:"`",c:[e.BE,n]};n.c=[e.ASM,e.QSM,c,a,e.RM];var s=n.c.concat([e.CBCM,e.CLCM]);return{aliases:["js","jsx"],k:t,c:[{cN:"meta",r:10,b:/^\s*['"]use (strict|asm)['"]/},{cN:"meta",b:/^#!/,e:/$/},e.ASM,e.QSM,c,e.CLCM,e.CBCM,a,{b:/[{,]\s*/,r:0,c:[{b:r+"\\s*:",rB:!0,r:0,c:[{cN:"attr",b:r,r:0}]}]},{b:"("+e.RSR+"|\\b(case|return|throw)\\b)\\s*",k:"return throw case",c:[e.CLCM,e.CBCM,e.RM,{cN:"function",b:"(\\(.*?\\)|"+r+")\\s*=>",rB:!0,e:"\\s*=>",c:[{cN:"params",v:[{b:r},{b:/\(\s*\)/},{b:/\(/,e:/\)/,eB:!0,eE:!0,k:t,c:s}]}]},{b:/</,e:/(\/\w+|\w+\/)>/,sL:"xml",c:[{b:/<\w+\s*\/>/,skip:!0},{b:/<\w+/,e:/(\/\w+|\w+\/)>/,skip:!0,c:[{b:/<\w+\s*\/>/,skip:!0},"self"]}]}],r:0},{cN:"function",bK:"function",e:/\{/,eE:!0,c:[e.inherit(e.TM,{b:r}),{cN:"params",b:/\(/,e:/\)/,eB:!0,eE:!0,c:s}],i:/\[|%/},{b:/\$[(.]/},e.METHOD_GUARD,{cN:"class",bK:"class",e:/[{;=]/,eE:!0,i:/[:"\[\]]/,c:[{bK:"extends"},e.UTM]},{bK:"constructor",e:/\{/,eE:!0}],i:/#(?!!)/}});hljs.registerLanguage("tex",function(c){var e={cN:"tag",b:/\\/,r:0,c:[{cN:"name",v:[{b:/[a-zA-Zа-яА-я]+[*]?/},{b:/[^a-zA-Zа-яА-я0-9]/}],starts:{eW:!0,r:0,c:[{cN:"string",v:[{b:/\[/,e:/\]/},{b:/\{/,e:/\}/}]},{b:/\s*=\s*/,eW:!0,r:0,c:[{cN:"number",b:/-?\d*\.?\d+(pt|pc|mm|cm|in|dd|cc|ex|em)?/}]}]}}]};return{c:[e,{cN:"formula",c:[e],r:0,v:[{b:/\$\$/,e:/\$\$/},{b:/\$/,e:/\$/}]},c.C("%","$",{r:0})]}});hljs.registerLanguage("xml",function(s){var e="[A-Za-z0-9\\._:-]+",t={eW:!0,i:/</,r:0,c:[{cN:"attr",b:e,r:0},{b:/=\s*/,r:0,c:[{cN:"string",endsParent:!0,v:[{b:/"/,e:/"/},{b:/'/,e:/'/},{b:/[^\s"'=<>`]+/}]}]}]};return{aliases:["html","xhtml","rss","atom","xjb","xsd","xsl","plist"],cI:!0,c:[{cN:"meta",b:"<!DOCTYPE",e:">",r:10,c:[{b:"\\[",e:"\\]"}]},s.C("<!--","-->",{r:10}),{b:"<\\!\\[CDATA\\[",e:"\\]\\]>",r:10},{b:/<\?(php)?/,e:/\?>/,sL:"php",c:[{b:"/\\*",e:"\\*/",skip:!0}]},{cN:"tag",b:"<style(?=\\s|>|$)",e:">",k:{name:"style"},c:[t],starts:{e:"</style>",rE:!0,sL:["css","xml"]}},{cN:"tag",b:"<script(?=\\s|>|$)",e:">",k:{name:"script"},c:[t],starts:{e:"</script>",rE:!0,sL:["actionscript","javascript","handlebars","xml"]}},{cN:"meta",v:[{b:/<\?xml/,e:/\?>/,r:10},{b:/<\?\w+/,e:/\?>/}]},{cN:"tag",b:"</?",e:"/?>",c:[{cN:"name",b:/[^\/><\s]+/,r:0},t]}]}});hljs.registerLanguage("markdown",function(e){return{aliases:["md","mkdown","mkd"],c:[{cN:"section",v:[{b:"^#{1,6}",e:"$"},{b:"^.+?\\n[=-]{2,}$"}]},{b:"<",e:">",sL:"xml",r:0},{cN:"bullet",b:"^([*+-]|(\\d+\\.))\\s+"},{cN:"strong",b:"[*_]{2}.+?[*_]{2}"},{cN:"emphasis",v:[{b:"\\*.+?\\*"},{b:"_.+?_",r:0}]},{cN:"quote",b:"^>\\s+",e:"$"},{cN:"code",v:[{b:"^```w*s*$",e:"^```s*$"},{b:"`.+?`"},{b:"^( {4}|	)",e:"$",r:0}]},{b:"^[-\\*]{3,}",e:"$"},{b:"\\[.+?\\][\\(\\[].*?[\\)\\]]",rB:!0,c:[{cN:"string",b:"\\[",e:"\\]",eB:!0,rE:!0,r:0},{cN:"link",b:"\\]\\(",e:"\\)",eB:!0,eE:!0},{cN:"symbol",b:"\\]\\[",e:"\\]",eB:!0,eE:!0}],r:10},{b:/^\[[^\n]+\]:/,rB:!0,c:[{cN:"symbol",b:/\[/,e:/\]/,eB:!0,eE:!0},{cN:"link",b:/:\s*/,e:/$/,eB:!0}]}]}});hljs.registerLanguage("json",function(e){var i={literal:"true false null"},n=[e.QSM,e.CNM],r={e:",",eW:!0,eE:!0,c:n,k:i},t={b:"{",e:"}",c:[{cN:"attr",b:/"/,e:/"/,c:[e.BE],i:"\\n"},e.inherit(r,{b:/:/})],i:"\\S"},c={b:"\\[",e:"\\]",c:[e.inherit(r)],i:"\\S"};return n.splice(n.length,0,t,c),{c:n,k:i,i:"\\S"}});"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
#rmd-source-code {
  display: none;
}
</style>


</head>

<body>

<style type="text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>

<style type="text/css">
.kable-table {
  border: 1px solid #ccc;
  border-radius: 4px;
  overflow: auto;
  padding-left: 8px;
  padding-right: 8px;
  margin-bottom: 20px;
  max-height: 350px;
}

.kable-table table {
  margin-bottom: 0px;
}

.kable-table table>thead>tr>th {
  border: none;
  border-bottom: 2px solid #dddddd;
}

.kable-table table>thead {
  background-color: #fff;
}
</style>


<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>
<script>
$(document).ready(function () {
  window.initializeSourceEmbed("01-normalizing_scRNA-seq.Rmd");
  window.initializeCodeFolding("show" === "show");
});
</script>



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="fluid-row" id="header">

<div class="btn-group pull-right">
<button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
<li role="separator" class="divider"></li>
<li><a id="rmd-download-source" href="#">Download Rmd</a></li>
</ul>
</div>



<h1 class="title toc-ignore">scRNA-seq Normalization</h1>

</div>


<!-- rnb-text-begin -->
<p><strong>CCDL 2019</strong></p>
<p>In this notebook, we’ll perform quality control analyses and normalization of scRNA-seq count data.</p>
<p>As opposed to bulk RNA-seq, there are there are a few main things to look out for in single-cell RNA-seq:</p>
<p><strong>Single-cell RNA-seq…</strong></p>
<ul>
<li>Requires more PCR amplification (and therefore more PCR-associated biases and error).<br />
</li>
<li>Has more zeroes in the gene expression data (most genes aren’t expressed across cell types).<br />
People sometimes refer to this as “sparsity” or “dropout.”</li>
</ul>
<p>For this tutorial, we will be using a pair of single-cell analysis specific R packages: <code>scater</code> and <code>scran</code>. This tutorial is in part based on the <a href="https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html">scran tutorial.</a></p>
<div id="set-up" class="section level2">
<h2>Set Up</h2>
<p>For these analyses, we will need <code>scater</code> and <code>scran</code> packages, which have been installed already on your Docker container.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBSZWFkIGluIGEgZnVuY3Rpb24gZm9yIGZpbHRlcmluZyB0aGUgZ2VuZSBtYXRyaXggZnJvbSBhIHNvdXJjZSBzY3JpcHRcbnNvdXJjZShmaWxlLnBhdGgoXCJzY3JpcHRzXCIsIFwiR2VuZU1hdHJpeEZpbHRlci5SXCIpKVxuIyBNYWdyaXR0ciBwaXBlXG5gJT4lYCA8LSBkcGx5cjo6YCU+JWBcbiMgTG9hZCB0aGUgYWxyZWFkeSBpbnN0YWxsZWQgcGFja2FnZXNcbmxpYnJhcnkoc2NhdGVyKVxubGlicmFyeShzY3JhbilcbmBgYCJ9 -->
<pre class="r"><code># Read in a function for filtering the gene matrix from a source script
source(file.path(&quot;scripts&quot;, &quot;GeneMatrixFilter.R&quot;))
# Magrittr pipe
`%&gt;%` &lt;- dplyr::`%&gt;%`
# Load the already installed packages
library(scater)
library(scran)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
<div id="import-single-cell-rna-seq-counts-matrix" class="section level2">
<h2>Import single-cell RNA-seq counts matrix</h2>
<p>This <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465">data set</a> we are using is glioblastoma data that was Fluorescence-Activated Cell sorted and then was processed by paired end sequencing using Smart-seq2 protocol <a href="https://www.ncbi.nlm.nih.gov/pubmed/29091775">(Darmanis et al. <em>Cell Reports.</em> 2017.).</a>. We’re using a subset of this dataset (n = 786 cells). You have been provided with a counts matrix from single-cell RNA-seq data that has already been processed using <code>Salmon</code> and <code>tximport</code> and saved the counts to a tsv file. You already know how to process this type of non-tag-based data, because you can follow essentially the same steps as we did in the bulk RNA-seq modules. As far as <em>tag-based</em> scRNA-seq data, we will explain how to process raw data, like fastq files, in the next section, so you can also get it to a count matrix and could use these same steps to filter and normalize the data.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBSZWFkIGluIHRoZSBkYXRhXG5zY19kYXRhIDwtIHJlYWRyOjpyZWFkX3RzdihmaWxlLnBhdGgoXCJkYXRhXCIsIFwidW5maWx0ZXJlZF9kYXJtYW5pc19jb3VudHMudHN2XCIpLCBcbiAgICAgICAgICAgICAgICAgICAgICAgICAgIHByb2dyZXNzID0gRkFMU0UpXG5gYGAifQ== -->
<pre class="r"><code># Read in the data
sc_data &lt;- readr::read_tsv(file.path(&quot;data&quot;, &quot;unfiltered_darmanis_counts.tsv&quot;), 
                           progress = FALSE)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiUGFyc2VkIHdpdGggY29sdW1uIHNwZWNpZmljYXRpb246XG5jb2xzKFxuICAuZGVmYXVsdCA9IGNvbF9kb3VibGUoKSxcbiAgZ2VuZXMgPSBcdTAwMWJbMzFtY29sX2NoYXJhY3RlcigpXHUwMDFiWzM5bVxuKVxuU2VlIHNwZWMoLi4uKSBmb3IgZnVsbCBjb2x1bW4gc3BlY2lmaWNhdGlvbnMuXG4ifQ== -->
<pre><code>Parsed with column specification:
cols(
  .default = col_double(),
  genes = [31mcol_character()[39m
)
See spec(...) for full column specifications.</code></pre>
<!-- rnb-output-end -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBSZWFkIGluIHRoZSBtZXRhZGF0YVxuc2NfbWV0YWRhdGEgPC0gcmVhZHI6OnJlYWRfdHN2KGZpbGUucGF0aChcImRhdGFcIiwgXCJ1bmZpbHRlcmVkX2Rhcm1hbmlzX21ldGFkYXRhLnRzdlwiKSwgXG4gICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgcHJvZ3Jlc3MgPSBGQUxTRSlcbmBgYCJ9 -->
<pre class="r"><code># Read in the metadata
sc_metadata &lt;- readr::read_tsv(file.path(&quot;data&quot;, &quot;unfiltered_darmanis_metadata.tsv&quot;), 
                               progress = FALSE)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiUGFyc2VkIHdpdGggY29sdW1uIHNwZWNpZmljYXRpb246XG5jb2xzKFxuICAuZGVmYXVsdCA9IGNvbF9jaGFyYWN0ZXIoKSxcbiAgY2hhbm5lbF9jb3VudCA9IFx1MDAxYlszMm1jb2xfZG91YmxlKClcdTAwMWJbMzltLFxuICB0YXhpZF9jaDEgPSBcdTAwMWJbMzJtY29sX2RvdWJsZSgpXHUwMDFiWzM5bSxcbiAgY29udGFjdF9waG9uZSA9IFx1MDAxYlszMm1jb2xfZG91YmxlKClcdTAwMWJbMzltLFxuICBjb250YWN0X3ppcC5wb3N0YWxfY29kZSA9IFx1MDAxYlszMm1jb2xfZG91YmxlKClcdTAwMWJbMzltLFxuICBkYXRhX3Jvd19jb3VudCA9IFx1MDAxYlszMm1jb2xfZG91YmxlKClcdTAwMWJbMzltLFxuICBwbGF0ZS5pZC5jaDEgPSBcdTAwMWJbMzJtY29sX2RvdWJsZSgpXHUwMDFiWzM5bSxcbiAgdHNuZS5jbHVzdGVyLmNoMSA9IFx1MDAxYlszMm1jb2xfZG91YmxlKClcdTAwMWJbMzltXG4pXG5TZWUgc3BlYyguLi4pIGZvciBmdWxsIGNvbHVtbiBzcGVjaWZpY2F0aW9ucy5cbiJ9 -->
<pre><code>Parsed with column specification:
cols(
  .default = col_character(),
  channel_count = [32mcol_double()[39m,
  taxid_ch1 = [32mcol_double()[39m,
  contact_phone = [32mcol_double()[39m,
  contact_zip.postal_code = [32mcol_double()[39m,
  data_row_count = [32mcol_double()[39m,
  plate.id.ch1 = [32mcol_double()[39m,
  tsne.cluster.ch1 = [32mcol_double()[39m
)
See spec(...) for full column specifications.</code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
<div id="filter-the-counts-matrix" class="section level2">
<h2>Filter the counts matrix</h2>
<p>Single-cell RNA-seq experiments are generally much larger dimensions than bulk RNA-seq, since every individual cell is being measured. However, due to the high risk of technical artifacts, we don’t necessarily want to keep the data for all the cells and genes.</p>
<p>There are different approaches to filtering your data. In an ideal world, all RNA-seq data would come with the percent mapped reads statistics associated with it. This should be one of the first things you filter your data by. Side note: even the highest quality single-cell RNA-seq data still has much lower percent mapped reads than bulk RNA-seq, so don’t expect the same range of percent mapped reads of bulk RNA-seq. However, because a lot of publicly available data does not have percent mapped read statistics associated with it, we are going to attempt to get around this by trying other strategies of filtering first.</p>
<div id="total-counts-as-a-quality-measure" class="section level4">
<h4>Total counts as a quality measure</h4>
<p>One basic of method of exploring if a particular cell’s data is trustworthy is seeing how many counts were measured total, over all the genes. Let’s take a look at this data from how many counts are there per sample: Note that for <code>apply</code> the second argument indicates whether the function you specify is being applied across all the rows or the columns (1 = rows, 2 = columns). Also note that in this case we are excluding the <code>genes</code> column for the calculation.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBNYWtlIGEgdmVjdG9yIG9mIHRoZSB0b3RhbCBudW1iZXIgb2YgY291bnRzIHBlciBzYW1wbGVcbnRvdGFsX2NvdW50cyA8LSBhcHBseShzY19kYXRhWywgLTFdLCAyLCBzdW0pXG4jIFRha2UgYSBsb29rIGF0IHRoZSBzdW1tYXJ5IHN0YXRpc3RpY3MgZm9yIHRoZSB0b3RhbCBjb3VudHNcbnN1bW1hcnkodG90YWxfY291bnRzKVxuYGBgIn0= -->
<pre class="r"><code># Make a vector of the total number of counts per sample
total_counts &lt;- apply(sc_data[, -1], 2, sum)
# Take a look at the summary statistics for the total counts
summary(total_counts)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiICAgIE1pbi4gIDFzdCBRdS4gICBNZWRpYW4gICAgIE1lYW4gIDNyZCBRdS4gICAgIE1heC4gXG4gICAgNDguMiAgODgxMjAuMyAxMzM2NDAuNyAxNDQ2ODQuMyAxOTI3MDguMyA1MTQyMDYuNSBcbiJ9 -->
<pre><code>    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
    48.2  88120.3 133640.7 144684.3 192708.3 514206.5 </code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Yikes, one of the samples has only 48 counts. It’s highly likely that this ‘cell’ is either an empty well or the well it was in didn’t get sequenced properly. Regardless of what may have happened, we can’t trust that ‘cell’. These kinds of things are to be expected; it’s highly unlikely that when obtain a single-cell RNA-seq dataset that all of cells within the set have reliable, high quality data.</p>
<p>This is why exploring your data and filtering it is important before we attempt to look for any biologically meaningful information.</p>
<p>Let’s visualize the distribution of total counts to see if the 48 count sample is the only one we should get rid of.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBMZXQncyB0YWtlIGEgbG9vayBhdCB3aGF0IHRoaXMgbG9va3MgbGlrZTpcbmhpc3QodG90YWxfY291bnRzLCBjb2wgPSBcImxpZ2h0Ymx1ZVwiLCBicmVha3MgPSA0MCk7XG4gICMgbGV0J3MgcGxvdCB3aGF0IHdlIG1pZ2h0IHdhbnQgdG8gYmUgb3VyIGN1dG9mZjpcbiAgYWJsaW5lKHYgPSA1MDAwMCwgY29sID0gXCJyZWRcIilcbmBgYCJ9 -->
<pre class="r"><code># Let&#39;s take a look at what this looks like:
hist(total_counts, col = &quot;lightblue&quot;, breaks = 40);
  # let&#39;s plot what we might want to be our cutoff:
  abline(v = 50000, col = &quot;red&quot;)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-plot-begin eyJjb25kaXRpb25zIjpbXSwiaGVpZ2h0Ijo0MzIuNjMyOSwic2l6ZV9iZWhhdmlvciI6MCwid2lkdGgiOjcwMH0= -->
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAArwAAAGwCAYAAABLkLalAAAEGWlDQ1BrQ0dDb2xvclNwYWNlR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19zn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7N0HmBRF2sDxd5ew5ChBQIEFVDCAiBIkiKIIekZQDxAwIMYTBUVPDOiJAT0Vz+zpiSKgoJ4BEAEjmAFJSs45w5IW2P7qrft6nE3dM8uG6Zl/Pc/udqjprv7V9Ow71dXVSY5JQkIAAQQQQAABBBBAIE4FkuP0uDgsBBBAAAEEEEAAAQSsAAEvbwQEEEAAAQQQQACBuBYg4I3r6uXgEEAAAQQQQAABBAh4eQ8ggAACCCCAAAIIxLUAAW9cVy8HhwACCCCAAAIIIEDAy3sAAQQQQAABBBBAIK4FCHjjuno5OAQQQAABBBBAAAECXt4DCCCAAAIIIIAAAnEtQMAb19XLwSGAAAIIIIAAAggQ8PIeQAABBBBAAAEEEIhrAQLeuK5eDg4BBBBAAAEEEECAgJf3AAIIIIAAAggggEBcCxDwxnX1cnAI/CnwzDPPyNlnn21/li1b9ueK/5+aNGlSaP2ECRNC65988snQ8oyMjNDyaCeWL18uO3fujPZl5A8T+OWXX+SCCy6QGjVqSIMGDaRXr15ha3OezC/3I9lOfr2Hcj7C2F96JHaxf3SUEIFgCBDwBqOeKCUCRyywaNEi+fLLL+1PWlpatu2tX78+tH7dunWh9b///ntoueM4oeWRTuzZs0eGDBkiTZo0kc2bN0f6MvJlEdi9e7ecd955ol9GNm3aJPqlZcOGDVly/TmbX+75sZ0jfQ/9eVTBmsoPu2AdMaVFIHYFisdu0SgZAgjEgsCpp54q27Zts0VJSkqKukhPPfWUPProo1G/jhdkFtAvLNu3b7cLr7nmGhk2bJh4fQHJL/f82k7mo0mMOewSo545ymAI0MIbjHqilAgUmcDf/vY3+e9//2t/kpMzf2QcOnRIVq5caVsb9+7de0RlPHz4sOilX/3rl7SleMeOHX7ZMq1fs2ZNpnmd0X1pa/Yff/whGzduzLY+twW6rfT09EyrNfhctWpVROXP9ML/n/E7/rVr14Ze1qVLF6lZs6YcffTRoWV5nVDLrVu35vXl9nXa+qwBudbfvn37jmhbfi9WpxUrVkTk7Gfqt69I1u/atUv06kheUn6eP3nZP69BIKEEzIc0CQEEEkDgxhtv1P4I9ue3337LdsRvvPFGaP1rr70WWt+/f3+nbNmy9sf8g7bLzaV05+qrr3aKFy8eeo1p/XU6derkmKAv9NrLLrvMKVGiRChP6dKlHROohdbrxMSJE53zzz/fKV++vM2XkpJit7Nw4cJM+XTm66+/do477jibzwTfTsuWLR1zad+pWrWqLZ9pSbavmTFjRqjMn376aeg1xxxzjGOCO5vn4YcfdipVqhQqm9rUq1fPmTp1ql3v/tJ96PHffPPNzpgxY5zatWvb15QqVcq54YYbHBNUOS+88IJz1FFH2eUVKlRwBgwY4L7c928kx2/6Xjvq4tafTmuZTL/rHLfv526+nDi33Xabc8IJJ4S2qcf19NNP2+NxN+q3HdNVwTnzzDND29DylSxZ0unTp49jAl93M07fvn1Dedz3UGhlhBMmmLbvE9ehcuXKzuWXX+588MEH2bYQiam+aN68eaH3ybPPPptpO6b7iF3XuHHj0PLrrrvOLmvTpo2jx37GGWc4+j7U465bt64zefLkUF4vu0jPn9DGmEAAgSMW0EtiJAQQSACB8IB3+PDhznvvvZfp56abbgoFJeEBb07BSvv27W1eDXhPPvlk+6MBr/7j79ixo2NubrOiXbt2DW3TDdY0IHST6Y9qAyR3XfhfDY6/+eYbN6vz448/OsWKFQttT7ej+TWAdpc/9NBDNv93330Xymdu8ApNt2jRwq7/5JNPQsvKlCnjVKlSJTSvAbpp8Q3t95RTTrHrdD+6P83rBl06f9JJJ9nlWYPn559/PrSN3CYiPf7WrVuHyhdu9Nlnn+W4aS93DUT1i0n4dsKnL7zwwtA2vbajgb4GfO5ra9WqFQr+dFnv3r1D28npPRRaGcGEadF13PrWbeuXDXe/+r4bP358aCuRmuoL5syZE9qOnhPhyX2PayDrJnOToM1frVo1RwNufa+Ev3f0vWRazG12Lzt3237nj7tf/iKAwJELEPAeuSFbQCAQAuEBrxss5PbXK+A1XQBCQcIrr7wSOvZ3333XMf19bWuetrpqMpfhbUuiu58vvvjCWbp0qV2nLcFu4KItr2+++aZjuhU4TzzxhFOuXDm7j0aNGjnaGqnJDRJ0W7pf06XA0Zbc448/PlSeBx980OYND3g1v7bemW4ZtjVZM2jgooGKtuLpdjRAf+6550Lb+fe//223o7/cgFe3owG1BnqmG0emoPeuu+5ytOXSjKIQavXW1kevFM3xq4vWieuox6+vD29FDd+Xl7u2VLvb0aBUWyq//fZbR1uR3eVu/XttR1vbtfVTg75p06bZ3ZtROJxjjz3Wbkfr1E1HGvB269bNblO/2Oix63tCg1X3qoCWQ+swGlMtW14DXnXSqxKmW419P2iLtms3btw4e9i52UVz/rh+/EUAgSMXIOA9ckO2gEAgBMIDXr0Mm/XHbaHVf9xuwKMHljVYMX0WQy2qGnBo0PTWW285ph9jjg4aJLrBwOLFi0N5wrtQmCHTQst1QrfpvkYv2x84cCDUeqgBbnh67LHHQnlzCnjNMF7h2TNNa7CrSYMTc4NRaDt6ad9NbsCrx6rBrps0yNIyqtv+/fvdxY4G6br8tNNOCy3LaSKa49fXf/TRR6Hyhbdo5rRtXZabuxuQajeQ8O4F+kXENW/VqlVos7ltx83gGmod6ReN5s2b2+2ol5uyvofc5ZH8VXP3i9FFF12U6SXahUC/ROkXAk3Rmh5JwBt+FUDL4Nq99NJLoTLmZBft+RPaGBMIIHBEApnvQDFnLAkBBOJfYNasWfamHxNMhP6aVs2IDtwEMmL6f9q8erPSyJEjxbRwibmkLaa/q3z88ccRbeerr74K5bv44otD0zoRPj937lx7k5I7BrC5HJ8prw7V5ZVOP/30HFebbg1iunGICQDF9F+VQYMGhfKZT9XQtDuhx2e+JLizYrpc2Onq1auL6eIQWm761tppEwCGluU0Ec3x5/T6vCwz/WDtjXX6Wh3P17SYhjaTmpoqpnuKnTd9W7UxJLQutwnzRUH++c9/SocOHcR0OZC2bdvKzJkzbfZIXp/bdsOXr169WswXCrvI9N8OXyXnnnuu6PtB60DTkZhmLa/7fsu0w7AZ0987NKfH7ia/es+v88fdH38RQCAygT8/vSPLTy4EEEDABjnvv/++DTjcYE8Dhp9++skGq+ayrq+SGzBqxvDAS+dNq6n+CaXwedPvMbRcJ/xGGDB9azPl15lbbrlFTGuhaJBvLsnL7bffLubGs1C+8MDWXWhaGd1J+9ctk+kakWl5pDPRHH+k2/TLF77PrI76WveY3L9e29Og2LRiyz333CM//PCDnHPOOaIPN9EgVFNOhl7by21deODpF0yGH5/feyrr/kxrd6ZFXvtSH/d9ry/Kuq9MG8phRr8kHOn5k8NmWYQAAh4CBLweOKxCAIGcBbRlWFsDzWV2OzyYGdnAtpa6uc1oBu5kpr/hrWjmRqzQOh32LDyFtxLrfrQV1g0qzGXz8KxiRmHINJ91xowakGmRtkq++OKLdtn1118v5lK+mD6+Yu68D+WLJliLJDgMbThsIprjD3tZniZddx3KzG2Z1Bbu8GBSh5czl/jt9k888cRQ8Bu+Q3c7usz0ebZDuWngt2DBAjE30IkZnULMjVv2JdEYhu8j67S5aSwUXM6fPz/Tan2C25VXXilmxA3Rh6lEaxoeIGd9GEtehxrLVMCwmXC7vJ4/YZtjEgEEohQg4I0SjOwIJLqA6T9qL+ebIa1k6NChoi2f+sjiRx55RNzgMrxVVVtQ3aTj17pPB9PL325QpAGntnhpFwltaTVDTdmXNGzYUMzNajbgMTeB2WW//vqr6PSX5qlxgwcPtq207vZz+usGyu46HS/WTaYfqzspr7/+emg6a/ATWpGPE9Ecf152m5u7emrS8Xe1pVsDXe3iYm5mC+3GDL8Vms5tO66j+poRC2x+/fKg9aJJnzKWH0nfI507d7abMjfH2Qdu6BjM+phlfaCJGW1E3n77bTE3OtouFZG+p3SDGvy771l9gp3pj2y7cowYMUJyGrc52uPJyS7a8yfafZIfAQRyETDfOkkIIJAAAuE3rUUzDm/WG450ZAAdisx8pNgfHbapXbt2jmkts/M6DqsOIeYmHbvWzWtaQ+3NZ+6NTiZQCd0Ap3lM8BTKa1oO7bi77nZ0XF7zoIXQenebpo9uaJneJKQpfJSG8JEkdJ15apxTsWJF+xotj47+oMeg29N5/Wu6OGhWm9yb1po2beousn/1pjTNax6ZnGl5s2bN7HIdrswvRXP80d60lpu7eVqbHb/Y9Qs312U6nFZ4ym07DzzwgD1OfY3pQ+uYLg12xAbXUJfrvjRlfQ+Fbz+S6SVLljg65FdOZdb96RB7borGVF9j+h+HtqtDy+l7TLdZv359u1zfG25yhyXT9eFJR+dwyxY+nm9OdnrTWjTnT/h+mEYAgbwL0MJrPqVICCAQuYC26GpLmz7eVi9nawuhGdbKPmHLBHuiLWVmfNbQBv/yl7/YVlpdYD6qRG/qcvvdmgBCtL+vdifQlja91Ksthu7NT25rpL5Wb1jS7gxmiCr7lDGd1/6j77zzjq62yW2tc+dz+qutkdoqqDehaXnMWL92v9qtQluqNekxFEaK5vijLU9u7tr6bkYVsPVXp04de+y6bXXRxxWHdyfR5blt5+9//7v06NFDs8imTZvsDWNXXHGF7eZiF5pf+eXYoEED0ZZ9fV+YYDNUZhOMyqhRo6R79+7uLiVaU72yYEalsNvVlmN9/40dO9Yed2ijeZzIyU5bvqM5f/K4a16GAAJZBOzX1CzLmEUAAQQiEtC75/XSrz5WWPvZhndlyLoBM0aqvWSswYsGLVmTbsu05ImuD+9b6ebTO/C1C4IGOeF3xWsgZB4oYbNp31wdeSGSpDcpmfGCbdCu2yzq5Hf8eS2fn/sK85he7Qag9eeVctuOaTG37wEzHFuO9ea1zbys04BR3yfm4Q/20co5vZfc7UZjql/C9DHBpmXXfXm+/c3NLprzJ98Kw4YQSFABAt4ErXgOG4GgCWhrr7YkazJPxbJ9Ts0g/rYfsdvKq/06deQAEgIIIIAAAuECBLzhGkwjgEDMCowePTp0CT2nQl566aWhm91yWs+y2BHQm+J0VIdIknYv0HF+SQgggMCRCGQe0PJItsRrEUAAgQIU+Otf/2rvxNexXnV4Kh1lQO/M1zvtr7rqKrnjjjsKcO9sOj8FtGuKefRwRJvMOv5xRC8iEwIIIJBFgBbeLCDMIoBAMATcG9yCUVpKiQACCCBQlAIEvEWpz74RQAABBBBAAAEEClyAYckKnJgdIIAAAggggAACCBSlAAFvUeqzbwQQQAABBBBAAIECFyDgLXBidoAAAggggAACCCBQlAIEvEWpz74RQAABBBBAAAEEClyAgLfAidkBAggggAACCCCAQFEKBD7gdRzHjsepj7ckIYAAAggggAACCCCQVSCQAe/atWtl8ODBdsD5kiVLSvXq1UUHMq9YsaI0bdpUBg4cKGlpaVmPlflCFnCWLRNn9epC3iu7QwABBBBAAAEEMgsEbhzelStXStu2bSUpKUm6d+8uqampUqVKFTuvrbzLly+XcePGibb8Tp06VRo0aJD5iJkrNIHDd98rSdWqSfJddxbaPtkRAggggAACCCCQVSBwjxYePny4bdmdMmWKpKSkZD0eOz9s2DDp0qWLjBw5UoYOHZpjHhYigAACCCCAAAIIJIZA4Lo0zJ49W3r37p1rsKvVVqJECenbt69MmjQpMWqRo0QAAQQQQAABBBDIVSBwAW+bNm1k+vTpuR6Qu2LatGlSu3Ztd5a/CCCAAAIIIIAAAgkqELguDT169BANejdu3Cg9e/a0fXT1hrXk5GTRPrwrVqyQUaNGyYQJE0S7PZAQQAABBBBAAAEEElsgcAFvs2bNZM6cOdK/f3/p06ePZGRkZKvBTp06yeTJk6VDhw7Z1rEAAQQQQAABBBBAILEEAhfwavU0bNjQjsCQnp4uq1atsq26Bw8elFq1akmdOnXsEGWJVY0cLQIIIIAAAggggEBuAoEMeN2D0ZvTNPjVn6xp3759dlHp0qWzrmIeAQQQQAABBBBAIIEEAnfTmtaNDjdWv359KVWqlLRu3TrHm9i0r692eSAhgAACCCCAAAIIJLZA4ALeL774wgaydevWlbvuuss+Vrh9+/byr3/9K7FrkqNHAAEEEEAAAQQQyFEgcF0aXnnlFencuXNojN1HHnlEHnzwQbntttukfPnyR9yqO2bMGBkwYECOWOELt2/fLm+88YYdKSJ8OdMI6FjRp556qlQ56ihPjD3m8ddnn9NJJnz6iWc+ViKAAAIIIIDAkQkELuDVRwuHd1XQRww//PDDcvjwYenXr58de1dHachruuSSS6Rjx46+L+/atatoKzMJgawC+sCT83pdJ1363ph1Vab5fWm7ZcTNV2daxgwCCCCAAAII5L9A4AJeHYlBHypx6623ZtJ49NFHZfXq1dKtWzf55ptvMq2LZkb7BeuPXypZsqQULx44Pr/DYn0+CRQ3N1RWrFrNc2sHD+y340d7ZmIlAggggAACCByxQOD68OrNaJ988oltzZ01a1YmAO1icNZZZ9nxd+fOnZtpHTMIIIAAAggggAACiSkQuID3yiuvlAceeEDefvtt+xNebdriOnbsWNFuCUuXLg1fxTQCCCCAAAIIIIBAggoE8pr8/fffb0do2LFjR7ZqS0lJkTfffFNuvPFG+/jhbBlYgAACCCCAAAIIIJBQAoEMeLWGtJ9tzZo1c62sli1b5rqOFQgggAACCCCAAAKJIxC4Lg2JUzUcKQIIIIAAAggggEB+CAQu4G3RooWUKVMmop9evXrlhxHbQAABBBBAAAEEEAiwQOC6NDzzzDNy+eWXS3p6ugwZMsRzWKfjjjsuwFVD0RFAAAEEEEAAAQTyQyBwAW+7du3ku+++k2bNmtmHTQwaNCg/HNgGAggggAACCCCAQJwKBK5Lg9aDttzqI4X/8Y9/yNatW+O0ajgsBBBAAAEEEEAAgfwQCFwLr3vQAwYMkObNm9tWXncZfxFAAAEEEEAAAQQQyCoQ2IC3WLFi0rFjx6zHwzwCCCCAAAIIIIAAApkEAtmlIdMRMIMAAggggAACCCCAgIcAAa8HDqsQQAABBBBAAAEEgi9AwBv8OuQIEEAAAQQQQAABBDwECHg9cFiFAAIIIIAAAgggEHwBAt7g1yFHgAACCCCAAAIIIOAhQMDrgcMqBBBAAAEEEEAAgeALEPAGvw45AgQQQAABBBBAAAEPAQJeDxxWIYAAAggggAACCARfILAPngg+PUdQ0AIzZ86MaBepqalSqVKliPKSCQEEEEAAAQSCJ0DAG7w6o8QRCIwePVp69OghDZqc5Jl7y4b1UrVKZVm6eLFnPlYigAACCCCAQHAFCHiDW3eU3EPg999/l4v7D5CLbxzgkUtk3bLF8sY9t3jmYSUCCCCAAAIIBFuAPrzBrj9KjwACCCCAAAIIIOAjQMDrA8RqBBBAAAEEEEAAgWALEPAGu/4oPQIIIIAAAggggICPAAGvDxCrEUAAAQQQQAABBIItQMAb7Pqj9AgggAACCCCAAAI+AgS8PkCsRgABBBBAAAEEEAi2AAFvsOuP0iOAAAIIIIAAAgj4CBDw+gCxGgEEEEAAAQQQQCDYAoEPeB3Hkc2bN8u2bduCXROUHgEEEEAAAQQQQKBABAL5pLW1a9fKiBEjZOzYsaLThw4dsjgVKlSQevXqSadOnWTo0KFSrly5AkFjo4kpMPCuu2XkyLekQsVKngDLFi+Srn1v9MzDSgQQQAABBBAoPIHABbwrV66Utm3bSlJSknTv3l1SU1OlSpUqdl5beZcvXy7jxo2T8ePHy9SpU6VBgwaFp8me4lrg088+kxueeEnKmC9WXumdxx+QDHPlgYQAAggggAACsSEQuIB3+PDhthV3ypQpkpKSkqPisGHDpEuXLqY1bqRt6c0xEwsRiFKgWLFiUrJ0KamV2sjzlcWKl/Bcz0oEEEAAAQQQKFyBwPXhnT17tvTu3TvXYFf5SpQoIX379pVJkyYVriZ7QwABBBBAAAEEEIg5gcAFvG3atJHp06f7Qk6bNk1q167tm48MCCCAAAIIIIAAAvEtELguDT169BANejdu3Cg9e/a0fXSrVq0qycnJdqSGFStWyKhRo2TChAmi3R5ICCCAAAIIIIAAAoktELiAt1mzZjJnzhzp37+/9OnTRzIyMrLVoI7SMHnyZOnQoUO2dSxAAAEEEEAAAQQQSCyBwAW8Wj0NGza0IzCkp6fLqlWrRFt1Dx48KLVq1ZI6deqItviSEEAAAQQQQAABBBBQgUAGvG7VlSxZ0nZpqFixougd9Do8GQkBBBBAAAEEEEAAgXCBwN20poXXh00MHjzYDk+mQW/16tVtq64Gvk2bNpWBAwdKWlpa+HEyjQACCCCAAAIIIJCgAoFr4eXBEwn6TuWwEUAAAQQQQACBPAoELuAt6AdP6A1xn376qS+ntjJv3rzZNx8ZEMgPgWXLlsno0aMj2tSgQYM8x6mOaCNkQgABBBBAII4EAhfw6oMndHSG3J6ypnXjPnji+eefj/pJa4cOHZI9e/b4VvHhw4dzHCHC94VkQCAPAi1btZbq5glvqSc183z1lDFvyeo1a+Xll170zMdKBBBAAAEEEkkgcAGv++CJfv36edZTXh880bx5c9Efv6Tbr1Gjhl821iOQLwJ1jq0rF9w8UBqc4v3eLF+5qhTbvz1f9slGEEAAAQQQiBeBwAW8PHgiXt56HAcCCCCAAAIIIFA4AoELeHnwROG8MdgLAggggAACCCAQLwKBC3gVngdPxMvbj+NAAAEEEEAAAQQKXiCQAa/LomPwavCrPyQEEEAAAQQQQAABBHISCOSDJ3I6EJYhgAACCCCAAAIIIJCTAAFvTiosQwABBBBAAAEEEIgbgcAFvC1atJAyZcpE9NOrV6+4qSgOBAEEEEAAAQQQQCBvAoHrw/vMM8/I5ZdfLunp6TJkyBBJTs49Zj/uuOPypsKrEEAAAQQQQAABBOJGIHABb7t27eS7774THZ5Mn3amj1ElIXAkAvo+Wrlype8mNB8JAQQQQAABBIInELiAV4m15faRRx6Rhx56SK6//nqpWrVq8OQpcUwI7N6+VVYuWypntDnTtzy7d+70zUMGBBBAAAEEEIg9gUAGvMo4YMAA+whgWt1i700VpBLt25MmVWocLY9/Nt232Ld39H6sr+8GyIAAAggggAACRSIQ2IC3WLFi0rFjxyJBY6cIIIAAAggggAACwRHI/Y6v4BwDJUUAAQQQQAABBBBAIFcBAt5caViBAAIIIIAAAgggEA8CBLzxUIscAwIIIIAAAggggECuAgS8udKwAgEEEEAAAQQQQCAeBAh446EWOQYEEEAAAQQQQACBXAUCO0pDrkfECgQQyFeBjIwMOXDgQETbLF26dET5yIQAAggggEBhCtDCW5ja7AuBAAr06t1HypQpI5UqV/b80TwvvPhiAI+QIiOAAAIIxLsALbzxXsMcHwJHKLDCPHb5rlfflcant/Hc0lfjRsnMWbM887ASAQQQQACBohCghbco1NknAggggAACCCCAQKEJEPAWGjU7QgABBBBAAAEEECgKAQLeolBnnwgggAACCCCAAAKFJkDAW2jU7AgBBBBAAAEEEECgKAQIeItCnX0igAACCCCAAAIIFJoAAW+hUbMjBBBAAAEEEEAAgaIQIOAtCnX2iQACCCCAAAIIIFBoAgS8hUbNjhBAAAEEEEAAAQSKQoCAtyjU2ScCCCCAAAIIIIBAoQkQ8BYaNTtCAAEEEEAAAQQQKAoBAt6iUGefCCCAAAIIIIAAAoUmQMBbaNTsCAEEEEAAAQQQQKAoBAh4i0KdfSKAAAIIIIAAAggUmkDxQtsTO0IAgZgS2LJli+zcudO3TAcPHvTNQwYEEEAAAQRiWYCAN5Zrh7IhUEACu3btkmrVqsnRx9T13cOO7duko28uMiCAAAIIIBC7AoEPeB3HEW2pKlasmFSpUiV2pSkZAjEksG/fPqlU9Sh59OOvfUs1qHMr3zxkQAABBBBAIJYFAtmHd+3atTJ48GCpV6+elCxZUqpXry5Vq1aVihUrStOmTWXgwIGSlpYWy+6UDQEEEEAAAQQQQKCQBALXwrty5Upp27atJCUlSffu3SU1NdW27Or8tm3bZPny5TJu3DgZP368TJ06VRo0aFBIlOwGAQQQQAABBBBAIBYFAhfwDh8+3LbsTpkyRVJSUnI0HTZsmHTp0kVGjhwpQ4cOzTEPCxGIR4FD5gazr7/+Wq7vd4Pn4e3Zu0cyDh/2zMNKBBBAAAEE4kUgcF0aZs+eLb1798412NWKKVGihPTt21cmTZoUL/XEcSAQkcDSOTOlVPVakl69rufP/kpHS3p6ekTbJBMCCCCAAAJBFwhcC2+bNm1k+vTp0q9fP0/7adOmSe3atT3zsBKBeBSoWLWadOzey/PQ1iz5Qya99YpnHlYigAACCCAQLwKBC3h79OghGvRu3LhRevbsafvo6g1rycnJtg/vihUrZNSoUTJhwgTRbg8kBBBAAAEEEEAAgcQWCFzA26xZM5kzZ470799f+vTpIxkZGdlqsFOnTjJ58mTp0KFDtnV+Cw4dOiT79+/3yyaHTf/HnPbt+0IyHJGA1o8OqeWXuFzvJ8R6BBBAAAEEEkcgcAGvVk3Dhg3tCAwa1KxatUq0VVefBlWrVi2pU6eOHaIsr1U4ZswYuemmm3xfvnfvXvnjjz9sa7NvZjLkm8AZLVvJrJm/SukyZT23uc/clHVx/wGeeViJAAIIIIAAAokhEMiA160aHYNXg1/90aQtrjosmY7HW7x43g6tV69eoj9+qXXr1tKkSRO/bKzPZ4Ht5lG4j344VY6u5z3c3OPXXZHPe2ZzCCCAAAIIIBBUgcCN0qDQ8+bNkxtvvFGuvfZaOwSTLnv66aelZs2aNvitXLmyvPrqq7qYhAACCCCAAAIIIJDgAnlrBi1CNA12Tz/9dPuEtaOOOkq0C8Jzzz0nDz30kFx11VVyzjnnyPvvv2+7JdSvX1/OPffcIiwtu0YAAQQQQAABBBAoaoHAtfA+/vjjcuqpp8q6detkyZIlcsstt8gNN9xgHyf82muv2aBXn7J2/vnny4svvljUvuwfAQQQQAABBBBAoIgFAhfwLlq0SHRosrJly9rHC1999dWWUB8zHJ66desmy5YtC1/ENAIIIIAAAggggEACCgQu4NWRGPShEm5yp7/66it3kf2rXR90xAYSAggggAACCCCAQGILBK4Pr96s1qVLF9uPV/vwTp06Ve6880559NFHRcdo1TF4J06caPv1vvnmm4lduxw9AggggAACCCCAgAQu4NW+ue+9954NaHeaIap0NIa+ffvaJ6/dcccd4jiO7eqg0253B+oZAQQQQAABBBBAIHEFfAPeb7/9Vj744AMbVDZt2jQmpLS/btY+u++8844MHz5cZs6cKSeddJLUrVs3JspKIRBAAAEEEEAAAQSKVsC3D2+lSpXk888/F32kr/48++yzsnnz5qItdS57P/roo+WCCy4g2M3Fh8UIIIAAAggggEAiCvgGvCeffLIsWLBAfv75Z2nfvr3tK1u7dm255JJL5MMPP7SP9E1EOI4ZAQQQQAABBBBAIBgCvgGvexgtWrSQESNG2PFvtYtD1apVbTcHHTVhwIAB8scff7hZ+YsAAggggAACCCCAQMwIRBzwuiVetWqVzJo1y/aV3bVrl6SmpsqPP/4oTZo0kfvvv9/Nxl8EEEAAAQQQQAABBGJCwPemNS3lli1bZOzYsaI3hv3www9So0YNOwKCzp944on2QPRxvldccYVceuml0rx585g4OAqBAAIIIIAAAggggIBvwPvNN9/YsW2V6sILL5SPP/7YjoNbvHjml3bt2tVqbty4EVUE/ifgiL3Bcad5Op5fSk5OloyMDL9sJs9h3zxkQAABBBBAAAEEwgUyR63ha/5/ukyZMvLkk09Kz549pVq1ajnk+N+ilJQU2bBhg239zTUTKxJKYK552t07kybKmPfHeh73hjWr5bB5aEjtevU98+nKrZs2+eYhAwIIIIAAAgggEC7gG/DqzWqnnXaajB49Wlq1amX77OoGbr75Zvnb3/4mJ5xwgt2etvhqVwcSAq7A1q1b5eS2HeXE5193F+X494eJ/5WPXvqnPPLhlzmuD194a/tTwmeZRgABBBBAAAEEfAUiummtc+fOdkQGbcHVpE8z++233+wDHnRcXhICCCCAAAIIIIAAArEq4BvwLl68WL7++muZZy5Pt2nTxh5HUlKSTJ8+Xf75z3/KfffdJ+np6bF6fJQLAQQQQAABBBBAIMEFfAPeqVOninZrOO6447JR9erVS/bu3Ss6VBkJAQQQQAABBBBAAIFYFPANeLWP7k8//SSrV6/OVv5PP/1UihUrJvrkNRICCCCAAAIIIIAAArEo4HvTmt6oVrNmTTvu7gMPPGBbevfv328fNTxkyBDp3r27lC5dOhaPjTIhEPMC6ekH5L333vMt5759e33zkAEBBBBAAAEEchbwDXhLlSolM2bMkAsuuEDOOeecTFvp1q2bvPjii5mWMYMAApEJ7Ny6RbaYYdaee+Nt3xfQbciXiAwIIIAAAgjkKuAb8OorjznmGJkzZ46sWbNGZs+ebbsxaFeH+vX9x03Ndc+sQCDBBQ4fOijlK1eR6x//l6/EHZ1O980TpAx6lUh//JIOd1iuXDm/bKxHAAEEEEDAUyCigFdbeHX4MQ14cxqR4ZdffvHcCSsRQAABV+DAgQO2G1TZ8uXdRbn+3bN7tyxYsEAaN26cax5WIIAAAggg4CfgG/DqzWrnn3++VKxYUVq3bi0VKlTw2ybrEUAAgVwFNOAtU7acPP/N3FzzuCv+dfs1snz5cgJeF4S/CCCAAAJ5EvANeHVYsuTkZNuloXLlynnaCS9CAAEEEEAAAQQQQKCoBHyHJdOnqtWpU0cIdouqitgvAggggAACCCCAwJEI+Aa8HTt2lGXLlsn8+fOPZD+8FgEEEEAAAQQQQACBIhHw7dKgw5L16NFDOnToIFdeeaVt7dWHTYSnu+++O3yWaQQQQAABBBBAAAEEYkbAN+D97bff5P3337cFfuedd3IsOAFvjiwsRAABBBBAAAEEEIgBAd+At3PnzrJz584YKCpFQAABBBBAAAEEEEAgegHfPrzhm9yzZ48drSEtLU10aCESAggggAACCCCAAAKxLhBRwKsPnNDHCOsTj5o2bWoHgh88eLAMGjRI9u7dG+vHSPkQQCDOBebNmydJSUlSokQJzx+9/6BK1apxrsHhIYAAAghkFfDt0qBPVrvooovk0KFD8swzz8h9991nt9GuXTu57rrrZNeuXfLqq69m3S7zCCCAQKEJzJw5U9pdeJlc/cDjnvt0MjLk5jNP9MzDSgQQQACB+BPwDXinTJkia9eulUWLFtmnrT344INW4fLLL7dPXevTp4/oWL3aukJCAIHEFdi/d4989/lkOa9LF08E/RJ9+PBhzzx5WamfQcVLlPR86aGD6XxWeQqxEgEEEIhPAd+Ad/HixXLiiSfaYDcrwemnny7r16+XFStWSP369bOuLpR5Dba3bNki9lJllSqFsk92ggAC2QVW/TFfajY8QRp36Z59ZdiS/Xt2y/RvvwtbwiQCCCCAAAIFK+Ab8GogO336dNm8ebNUq1YtU2nGjBkjxYsXl9q1a2daXtAz2uI8YsQIGTt2rG191u4WmipUqCD16tWTTp06ydChQ22f44IuC9tHAIE/BcqUryDN2p/z54IcprZtXC/CBaEcZFiEAAIIIFBQAr4B79lnn20fNtG1a1e58847JcP0gdNW308++URefvll+1CKkiW9LyPmZ+FXrlwpbdu2tZclu3fvLqmpqVLFtOzq5cxt27bJ8uXLZdy4cTJ+/HiZOnWqNGjQID93z7YQQAABBBBAAAEEAibgG/DqyAwffvih9O3b1wa3eny9evWyh3nJJZfIc889V6iHPHz4cNuKq32LU1JSctz3sGHDpIvpRzhy5Ejb0ptjJhYigAACCCCAAAIIJISAb8CrCqeccor88ssv8uuvv9rWXW3R1X69jRs3LnSk2bNni94ol1uwqwXSoYk0QH/++ecJeAu9htghAggggAACCCAQWwIRBbxa5OTkZNGb1PSnKFObNm1sn+J+/fp5FmPatGmF3rfYs0CsRAABBBBAAAEEECgSAd+A94cffpB77rnHs3BfffWV5/r8XNmjRw/RoHfjxo3Ss2dP20e3qhlIXgNy7cOrI0aMGjVKJkyYINrtgYQAAggggAACCCCQ2AK+Aa92HTj66KMzKenDJv744w/RJ7ANGDAg07qCnmnWrJl9vHH//v1t1wa9iS5r0lEaJk+eLB06dMi6inkEEEAAAQQQQACBBBPwDXhPPfVUGT16dDYWHf/2tttuk1WrVmVbV9ALGjZsaEdg0AHsdf/aqnvw4EGpVauWHVFCW3zzmnQItjfeeMP35UuXLrVDovlmJAMCCCCAAAIIIIBAkQr4Bry5lU6HARs0aJBo8Pnaa68VyZi3evOc7l9/NG3atMnzZrbcjiV8efXq1eXMM88MX5TjtHbjKFOmTI7rWIgAAggggAACCCAQOwJ5Dnj1ELQfrT4idOfOnYUa8P7000/y8MMP25bn8uXL2zGBtbVZx+jVJ641bdpUnn32WWnXrl3U0o0aNRL98Usa5FeuXNkvG+sRQOAIBA6ZKzf65VI/Y7zS999/bx9x7pWHdQgggAACiSvgG/Bqd4GPP/44k5AGuXqDmAZ9J5xwQqGOhvDjjz9K69atpXPnzrZMelPdpZdeaoPcW2+91bbw6oMnzjvvPNuPNy9Bb6aDZQYBBIpMYM3SJfLa4kVy4ryFnmX44YuJctrZ53nmYSUCCCCAQOIK+Aa8CxYskLvuuiubUNmyZUVvICvsB09of+ILL7wwFIS/+uqrUrNmTdFAWB9zrElbe/XGtf/85z95auXNdrAsQACBIhFINud073sellPane25/2Kl75V9u3d55nFX6v0HW7dudWdz/aufJxUrVsx1PSsQQAABBIIj4Bvw6iOFDxw4EDNHpA/A0KHJ3LRjxw65+OKLQ8Guu1zzvPDCC+4sfxFAAAHJMFen9ApV6v/3+/ci2WU+W/RBN9pFioQAAgggEGwB34A31g5Pn/r2/vvvy7XXXiulSpWSs846S/RxwyNGjLD9d7W82oIzceLEInkSXKx5UR4EEPhTICPjsCSZMbuf/XL2nwtzmXr17ptkyZIlBLy5+LAYAQQQCJKAb8AbyYMnwg9Yh/RKTU0NX5Sv04MHD5bTTjtNWrZsKffee699CIX2I9Yxd6+55hrRm9j0wROff/65fSJbvu6cjSGAAAIIIIAAAggETsA34NWRCEqXLm0DyHPOOUeaNGki69ats08x0+4EHTt2lBo1aoQO3O1HG1qQzxN169aVb775Rv7xj39Ir1697OVJdxc6hq4mHTtYb7TTwJiEAAIIIIAAAgggkNgCvgFvuXLlbD82fUzv2Wf/eeOI9oPTURHmzp2b44MpCpJVg+53331Xnn/+eVm+fLl9AIQOW6QPnjjmmGPk+OOPL8jds20EEEAAAQQQQACBAAn4Brz6iF5tMQ0PdvX4dLzbIUOG2CebrV+/PtvjhwvDQJ+opj8tWrQojN2xDwQQQAABBBBAAIEACiT7lblChQqiD3pIS0vLlnXDhg2SbG4A0VZgEgIIIIAAAggggAACsSjgG/DqeLb79++3N4T99ttvNvDVR/h+8MEH0r17d7niiivsjWKxeHCUCQEEEEAAAQQQQAAB3y4NOvC63gz2l7/8xT5oIpxMg91XXnklfBHTCCCAAAIIIIAAAgjElIBvwKul1YHXly5dKvPnz7c3sOlT1k4++WT7WOGYOhoKgwACCCCAAAIIIIBAFoGIAl59TYkSJaRRo0a2z66Os6vzJAQQQAABBBBAAAEEYl3Atw+vHsCaNWukW7du9uY0be1dsGCB6AMgBg0aJHv37o31Y6R8CCCAAAIIIIAAAgks4Bvwpqeny0UXXSSLFi2SZ555RsqUKWO52rVrJ6+//roMGDAggfk4dAQQQAABBBBAAIFYF/Dt0qAPnFi7dq0NePUGtgcffNAe0+WXXy46ZFmfPn3EcRxJSkqK9WOlfAgggAACCCCAAAIJKODbwrt48WI58cQTRYPdrOn0008XfejEihUrsq5iHgEEEEAAAQQQQACBmBDwDXjr169vhyXbvHlztgKPGTNGihcvLrVr1862jgUIIIAAAggggAACCMSCgG+XBn2kcJ06daRr165y5513SkZGhmir7yeffCIvv/yy9OjRQ0qWLBkLx0IZEEAAAQQQQAABBBDIJuAb8Opjgz/88EPp27evDW51C7169bIbuuSSS+S5557LtlEWIIAAAggggAACCCAQKwK+Ae+OHTvsAyf0aWvz5s2zrbvaoqv9ehs3bhwrx0E5EEAAAQQQQAABBBDIUcA34P3ss89si+7q1atFb1LTHxICCCCAAAIIIIAAAkER8L1p7aijjrLHsnPnzqAcE+VEAAEEEEAAAQQQQCAk4NvC26xZM9t3t3379nLppZdKgwYNJCUlJbQBndCb2UgIIIAAAggggAACCMSigG/AO2vWLPnoo49s2UePHp3jMRDw5sjCQgQQQAABBBBAAIEYEPANeM8//3zZs2dPDBSVIiCAAAIIIIAAAgggEL1Ajn14Dx06JPv3749+a7wCAQQQQAABBBBAAIEYE8gx4NWxdVNTUzMVdc2aNfL7779nWsYMAggggAACCCCAAAKxLpBjwJtToZ955hm5+uqrc1rFMgQQQAABBBBAAAEEYlYg4oA3Zo+AgiGAAAIIIIAAAggg4CFAwOuBwyoEEEAAAQQQQACB4AsQ8Aa/DjkCBBBAAAEEEEAAAQ8BAl4PHFYhgAACCCCAAAIIBF8g13F4N2/eLCeddFLoCDds2GDH4w1f5q6cN2+eO8lfBBBAIOEEZsyYIWlpab7HrU+q1B8SAggggEDhCuQY8DZs2FAuvPDCTCVp1KhRpnlmEEAAAQREvvzySzn77LOlWZv2vhyzZ3wjhw8fluRkLq75YpEBAQQQyEeBHAPeiy++WPQnCMlxHNmyZYsUK1ZMqlSpEoQiU0YEEIgjAf38aXXeBXLDEy/4HtV1zeuLfmaREEAAAQQKVyCQzQxr166VwYMHS7169aRkyZJSvXp1qVq1qlSsWFGaNm0qAwcOjOjyYuFSszcEEEAAAQQQQACBohDIsYW3KAoS6T5Xrlwpbdu2laSkJOnevbt9Ipy27Or8tm3bZPny5TJu3DgZP368TJ06lf5ykcKSDwEEEEAAAQQQiFOBwAW8w4cPty27U6ZMkZSUlByrZdiwYdKlSxcZOXKkDB06NMc8LEQAAQS8BHbv2C6DzJWk517w7qqwYukyqdnwBK9NsQ4BBBBAoIgFAhfwzp49W/r06ZNrsKueJUqUkL59+8rzzz9PwFvEbzB2j0BQBTavXyddr7lZqh5d2/MQkqZMkJ2bNnjmYSUCCCCAQNEKBC7gbdOmjUyfPl369evnKTdt2jSpXdv7H5XnBliJAAIJLaDdpMpUqCgntm7n6TB3xleyM8kzCysRQAABBIpYIHABb48ePUSD3o0bN0rPnj1tH129YU2H+dE+vCtWrJBRo0bJhAkTRLs9kBBAAAEEEEAAAQQSWyBwAW+zZs1kzpw50r9/f9u1ISMjI1sNdurUSSZPniwdOnTIts5vwfbt22XVqlV+2exDOPbv3++bjwwIIIAAAggggAACRSsQuIBXufTBGDoCQ3p6ug1OtVX34MGDUqtWLalTp44doiyvrN988408+OCDvi9funSprFmzxjcfGRBAAAEEEEAAAQSKViCQAa9LpmPwavCrP5r27NljuzVoF4e8pkgfutG6devQfvO6L16HAAIIIIAAAgggUPACgXzwRG4sH3zwgZx66qm5rWY5AggggAACCCCAQAIKBK6F96mnnrI3puVUVwsXLrStvLfeeqtd3bx5c7n22mtzysoyBBBAAAEEEEAAgQQRCFzA++uvv8qYMWOkcuXK2YYd27Fjh+3L+/XXX9vqy+3BFLFat/v27ZPbbh8gu3bt8i3iccc1kocfesiOTuGbmQwIIIAAAggggEACCwQu4NUhx0455RTRp6n17t1bBg4cGAr63n77bbnjjjtk7ty5gazSGTNmyJffzZBzel3vW/4nh94tt992m1SrVs03LxkQQAABBBBAAIFEFghcwKvj7d57771y3nnn2XF4P/30U/sI4bp168ZFPZarWFFad73E91jee+ph3zwFlWHJkiX24R9+2y+zdatIrbJ+2ViPAAIIIIAAAggUqEDgAl5X47TTTpOZM2fKoEGDbIvviBEjQi29bh7+FoxAo0aN5ORWbaVy9ZqeO+i6fLnUrHWsZx5WIoAAAggggAACBS0Q2IBXYcqUKSMvvviiXHDBBfbmtNKlSxe0V1xvX59UF8nDNGocXUt6P/CEVD3a+9HNB0wXDRICCEQvcPjwYfs0yUheqd2aSpQoEUlW8iCAAAIJKxDogNetNQ14td/u7bffLps3b3YX8zcKgWXLlv3vMc3Va/i+ancEN9X5boQMCCCQq8B99z8gTzw2TKr6nI9bN22Ua6/vJ/9+7dVct8UKBBBAAAGRuAh4tSKrV68uo0ePpk7zKLBhwwZp3Ow0uevN8b5buPnME33zkAEBBPIusNR8Ae336LO+/flnfzNVfp/4ft53xCsRQACBBBGIqwdPJEidcZgIIIAAAggggAACUQgQ8EaBRVYEEEAAAQQQQACB4AkQ8AavzigxAggggAACCCCAQBQCBLxRYJEVAQQQQAABBBBAIHgCcXPTWvDoKTECCCBw5AI7tm+XiRMn+m4oNTVVjj/+eN98ZEAAAQTiUYCANx5rlWNCAIGEEFi3bLH8/OOPMuSxJ32Pd+a3X8nevXuF8cp9qciAAAJxKEDAG4eVyiEhgEBsCjiOI4cOHRL965n81v//iw+YALZxi1Zy64j/eG5OV97cponoAy1ICCCAQCIKEPAmYq1zzAggUGQC2sKalJTkuf8SpUpL76atPfOwEgEEEEAgcgEC3sityIkAAggcscBrPy+R5GLFPLcz5LJzPNezEgEEEEAgOgFGaYjOi9wIIIAAAggggAACARMg4A1YhVFcBBBAAAEEEEAAgegECHij8yI3AggggAACCCCAQMAECHgDVmEUFwEEEEAAAQQQQCA6AQLe6LzIjQACCCCAAAIIIBAwAQLegFUYxUUAAQQQQAABBBCIToCANzovciOAAAIIIIAAAggETICAN2AVRnERQAABBBBAAAEEohMg4I3Oi9wIIIAAAggggAACARPgSWsBqzCKiwACCBSkwPwFC+T2AXdIhYoVPXezb+9eufOOAXJup06e+ViJAAIIxIIAAW8s1AJlQAABBGJE4PEnnpSKx50i1eoc61miPbt2ygMPDSXg9VRiJQIIxIoAAW+s1ATlQAABBGJAIDk5WSpUOUpann+RZ2nm//CdrCjOvxBPJFYigEDMCNCHN2aqgoIggAACCCCAAAIIFIQAAW9BqLJNBBBAAAEEEEAAgZgRIOCNmaqgIAgggAACCCCAAAIFIUDAWxCqbBMBBBBAAAEEEEAgZgQIeGOmKigIAggggAACCCCAQEEIBD7gdRxHNm/eLNu2bSsIH7aJAAIIIIAAAgggEHCBQAa8a9eulcGDB0u9evWkZMmSUr16dalatapUNAOlN23aVAYOHChpaWkBrxqKjwACCCCAAAIIIJAfAoEbRHHlypXStm1bSUpKku7du0tqaqpUqVLFzmsr7/Lly2XcuHEyfvx4mTp1qjRo0CA/nNgGAggggAACCCCAQEAFAhfwDh8+3LbsTpkyRVJSUnJkHzZsmHTp0kVGjhwpQ4cOzTEPCxFAAAEEEEAAAQQSQyBwXRpmz54tvXv3zjXY1WorUaKE9O3bVyZNmpQYtchRIoAAAggggAACCOQqELiAt02bNjJ9+vRcD8hdMW3aNKldu7Y7G3d/9Wa9AwcO+P6kp6fH3bFzQAgggAACCCCAQDQCgevS0KNHD9Ggd+PGjdKzZ0/bR1dvWNPnv2sf3hUrVsioUaNkwoQJot0e4jXt27tHjjnmGCluWrO90qGDB2XGjBnSunVrr2ysQwABBBBAAAEE4lYgcAFvs2bNZM6cOdK/f3/p06ePZGRkZKucTp06yeTJk6VDhw7Z1sXNgqRkeXbqr1KhSlXPQxrz5EPy888/E/B6KrESAQQQQAABBOJZIHABr1ZGw4YN7QgMerl+1apVtlX3oGnJrFWrltSpU8cOUZbXSps4caI8/vjjvi+fP3++HRGiVatWvnmLMsPu7dvk4X88Ku+Oec+zGBs3rJfk0mU987ASAQQQQAABBBAIokAgA14XWm9O0+BXf7Kmffv22UWlS5fOuspzvnnz5vLII4945tGVN910kx3/1zdjEWdYv3KZtL28p5zQwjswXzp3lvzw2YdFXFp2jwACCCCAAAII5L9AIANeHW7swQcflHXr1okGqE899ZSceeaZmXS0r68GxO+9592ymelFZqZGjRr2J+vyrPMVKlSQsmWD0SJapnwFOf60llkPIdP8+mVLMs0zgwACCCCAAAIIxItA4EZp+OKLL2zf3bp168pdd91lHyvcvn17+de//hUvdcJxIIAAAggggAACCOSjQOBaeF955RXp3LlzaIxd7X6grb233XablC9f3gbD+ejDphBAAAEEEEAAAQQCLhC4gFcfLayjM7hJHzH88MMPy+HDh6Vfv3527F0dpYGEAAIIIIAAAggggIAKBK5Lg47EoA+VyJoeffRRueqqq6Rbt2522LKs65lHAAEEEEAAAQQQSEyBwAW8ejPaJ598YltzZ82alanW3njjDTnrrLPs+Ltz587NtI4ZBBBAAAEEEEAAgcQUCFzAe+WVV8oDDzwgb7/9tv0Jr7bixYvL2LFj5ZJLLpGlS5eGr2IaAQQQQAABBBBAIEEFAteHV+vp/vvvtyM07NixI1u1paSkyJtvvik33nijffxwtgwsQAABBBJQILlYMel+5VVSomRJz6PXJ1l27Hm8Zx5WIoAAAkETCGTAq8ilSpWSmjVr5urdsqX3uLO5vpAVCCCAQBwK7E3bLfXany9JycU8j+7nWbM917MSAQQQCKJAYAPeIGJTZgQQQKCoBIqZLl+ntDtbSpXxfmDOp68+V1RFZL8IIIBAgQkErg9vgUmwYQQQQAABBBBAAIG4FCDgjctq5aAQQAABBBBAAAEEXAECXleCvwgggAACCCCAAAJxKUDAG5fVykEhgAACCCCAAAIIuAIEvK4EfxFAAAEEEEAAAQTiUoCANy6rlYNCAAEEEEAAAQQQcAUIeF0J/iKAAAIIIIAAAgjEpQABb1xWKweFAAIIIIAAAggg4Arw4AlXgr8IIIAAAlEJ7DSPd//oo498X5OamiqnnHKKbz4yIIAAAgUlQMBbULJsFwEEEIhjgfUrlsrc32bLY8+/5HuUP02bLNu3b5dKlSr55iUDAgggUBACBLwFoco2EUAAgTgXSN+3Txqc3ExufPpV3yNdeHZzOXjwoG++gshw6NAhOXDggO+mk5KSpEyZMr75yIAAAsEUIOD96CXIAAAneklEQVQNZr1RagQQQACBCAROaNJEli5eLCmlSnvmPrB/n3z11VfSoUMHz3ysRACBYAoQ8Aaz3ig1AggggEAEAmlpe+SpiTOkSs1anrnff/Yx+fHHHwl4PZVYiUBwBRilIbh1R8kRQAABBBBAAAEEIhAg4I0AiSwIIIAAAggggAACwRWgS0Nw646SI4AAAgkr8M4778jSpUt9j78gbpYbM2aMLFy40HffOhTbpZde6puPDAggUPACBLwFb8weEEAAAQTyUWDGjBly9dVXy0U3/M13q2lpab55oskwa9Ys+etf/xrRvh966DJZtGiRNGrUKJpdkBcBBApAgIC3AFDZJAIIIIBAwQk4jiONT20hl9x0p+9OJr/zb9880WTQfTdoclJE+/518iei+UkIIFD0AvThLfo6oAQIIIAAAggggAACBShAwFuAuGwaAQQQQAABBBBAoOgF6NJQ9HVACRBAAIG4Fti/f7/UqFFDSpX2efhDerpkmCejlfZ54tm+vXul3gkn5qvZlnVrZPBbr8hDQ4d6blf3fXS9VM88rEQAgdgTIOCNvTqhRAgggEBcCWRkHJZ/Tv5RUsqU9Tyux6+7QtpceJm0v/Qqz3zTPx4n33ww2jNPtCu3blgnl996l5zz176eL5057XP55LURnnlYiQACsSdAwBt7dUKJEEAAgTgTSJKk5GJSyifgNbkkOYJ8ycWKFYhPktmuXxmTk+kJWCD4bBSBAhbgzC1gYDaPAAIIIIAAAgggULQCBLxF68/eEUAAAQQQQAABBApYgIC3gIHZPAIIIIAAAggggEDRCtCHt2j92TsCCCCAQJwKZGRkyMqVK02/ZO+2paSkpIgfUFGzZk0pV65cnIpxWAgUnAABb8HZsmUEEEAAgQQW2L51i/zl4oularUangrrVq2w62sdW88z35aNG6Rho4Yyf+5cz3ysRACB7AIEvNlNWIIAAggggMARC+iIE0NGfSI166Z6buuxa7tLk5Zt5eL+t3vmW7dssbxxzy2eeViJAAI5CxDw5uzCUgQQQAABBGJKYP/ePbJ2zWo5pm4933Lt2btPDqXvl4qVKnvmXbNqpfzwww/SsmVLz3xFuVIfXHJy06ayf/8B32LsSUuT3xfMtw868c1MhoQSCHzA6ziObNmyRYqZ8ROrVKmSUJXHwSKAAAIIJI7Arm1bzDjB5eTWF972PehHel4kA57/j5SvcpRn3ukfvy8ffPhhTAe8O3bskI0bN8l973zseSy68q0H7pQFCxYQ8PpKJV6GQAa8a9eulREjRsjYsWNFpw+ZR1FqqlChgtSrV086deokQ83jIenYn3hvaI4YAQQQiGeBYsWLS/U6x/oeYlJykpRIKeWbV2+omzhxkmzZutVzmwcPHrQPDylezPsGPN1IraNryQP3D5ESJUp4bjOalcXNtiI5bi3n4088Ke+8+67n5vft2ydPPPaYHHPMMZ75oln568yZ8uxzz0nJkiU9X7Znzx77gJXSpUt55tOVbdq0keuuucY339x58+SJJ5+UlJQUz7zaWj7kvvuk8QkneOaLx5WBC3j1jte2bduK3tXavXt3SU1NtS27Or9t2zZZvny5jBs3TsaPHy9Tp06VBg0axGO9cUwIIIAAAggcscCS32ZKxWMbyKEa9T23tcPcMPf5O6/LVQOHeObTlW+MfF0u6NpFWrVq5Zs3vzNs2bBemp19vhwqX8Fz03s2b5QHHnpI3vz3vz3zRbNywJ0DpcbJp0tyxUqeL1s042vZn7ZbTj/vQs98OsrH9ddeG1HAO+juwVK2fmMpVtn7SvfebVvlrsH3yKf//chz3/G4MnAB7/Dhw20r7pQpU3L9JjNs2DDp0qWLjBw50rb0RlNxGlD/+uuvvi/Zar4N79y50zdftBn27Nolv0yZ6Puyw6ZV+/efpkux4t7foLesXyu7t2/13eaWDWtE+4dFsm9tUV8062cpMX+OZzkb7N4lyWabv/kcz0Zzh/KBfXsj27f59r583m+ydskiz32vX75E6jQ83neb0ew7Pf2ArF74u2xavcpz3+nmWNJ2bPPd97aN6+TA/n2++XRnmm+dOabtmzZ67lvrMG3nDt9t6qXRgwcO+ObTne3bkyabTb/BPTu935e67yST3+89tH/PbtO3MN03n+57rzkftm3a4Jt3vynj3l07ffMdOpgujvkn4ldG3ffuHdsjOnf2mX9ce3fvjmibut1fp31uv7DrdG5pp/l8ieS8Tdu1Q/aa/UdyPPqZMee7L23LUm771eXbN2+KaN+7tm6Wfaa/ZKT7juTzauuGyD6vtprPtX2mlSySfR86dFAWzfxJipf0bvnatHplRMe9YdVy81kZ+efVsrmzZc3ihV7ksmHFUjnmuMa+xxPN55We36sXLjCfVys9933AtHRqq3G5St6B0gHTMqgNS375dGfrVi6XDz74QGbPnu25b135008/yRlnnOGZT//X7o3wvab/S7SMZXwC3t3bt8mMr6fLyy+/7LlvXbl06dKIGs/Wr18vDdpXkbIVvAPeEua9mJ78v3J67dxxMuzqSMq4YsVyaXlqG9/6OXz4sGxYss9rt3G7Lsn0gXWCdHTautunTx/p16+fZ7Hfeecdef755+XHH3/0zJd15Zdffin/+te/si7ONj/PXD7QfOeee262dXldoJc5rr2+n+w1H0B+aZcJBMqUKSPFzQeVV9pi/nllZDhSvUYNr2yiwdyBA+lSvnx5z3y6crf5556SUtJctvH+B9Ltp19luynf1OZNPbepl6D0Mkuk+y5pLm2llPK+FLRh3TrRN/bRtWrl277TzHFrX/HSxt0r6U0TWpd+5vrBo/m0K45fStudZi4nJknZsmU9s+r29pl/xkdVq+aZL6p9m4DKMe+h8j7l1H3vN4F51arefQa11SLNGEVy3PpeyzBOFSt5/wPRy5P7zU9ln378+nGn507FihU9fXSl/pPV4LhSZe+bfnS/+01wUcmnjLpN7YsYSb7t27eLGZjV93gOmP1qIFIhguPR4y5n3j/J5j3slbaZYDvD/KM96ijv91C0+47k82rr5s1y2Jj7nTv282q/+byq4P95pe+1kiVL+H5ebdywwbLUMOPceiX9vNpn3ucVfAIq3Ybuu4T5DPT9vDKBkr438/Xzyuy7mOmuEMnnlZbT77j1M2Ov+WyJ5HNa61H/9/iNP6xGCxculOOPP14nc01qU9G0XPqVUTegnxl67vh9Xmk9fj7hM9996zb13Ink82rJsmXSvn0HKeszTnKkn1e674XmBrxIHJcsXSZt27XzPW5tsBry97/LmW1a6+YTKgUu4L377rtl06ZN8p///Mezoq41lwH0n4t+yyQVjcDhu++VJBN4Jd91Z9EUgL0igAACCCCAAAJGwLt5MAaJevToYTtxb9y4UXr27GkvM1StWtV+A9I+vCtWrJBRo0bJhAkTRLs9kBBAAAEEEEAAAQQSWyBwAW+zZs1kzpw50r9/f9u1QS+PZk06SsPkyZOlQ4cOWVcxjwACCCCAAAIIIJBgAoELeLV+GjZsaEdgSDc3vqxatcq26mp/nFqmv2adOnVMH8KqCVaNHC4CCCCAAAIIIIBAbgKBDHjdg9Gx7jT41R8SAggggAACCCCAAAI5CfiPIJ3Tq1iGAAIIIIAAAggggEBABAh4A1JRFBMBBBBAAAEEEEAgbwIEvHlz41UIIIAAAggggAACAREg4A1IRVFMBBBAAAEEEEAAgbwJBPqmtbwdcuy+6ueff5bXX39dTjnllNgtZBQlazlzluwvX05+e8H7iWxRbLJIs+ojp/WJQOV8nqJTpIVM0J3ruNz6oBm/JzYlKE+RH/b3338vLVq0kBLmKYmk2BLQkY70KWr169ePrYJRGvOU1AzRc+fdd99FIx8ECHjzATG/NjF//nz7wIx4+aewsFGqpcn4/ff8IirS7Xz88ceiYzwfdZT3o3OLtJAJuvM//vhD1qxZY/9BJChBTB/2+PHjzePIU6R06dIxXc5ELJx+kdchPvXx7qTYEtAvImPHjiXgzadqCdyjhfPpuGNyM1OnTpXHHnuMJ8TFZO2I6ENP3nrrLWnatGmMljBxizVy5Eg7NrfWDyn2BKqZR4z/br748mUx9upG/+fs3r1bhg0bFnuFS/AS6ReR8uXLy4EDBxJcIn8Onz68+ePIVhBAAAEEEEAAAQRiVICAN0YrhmIhgAACCCCAAAII5I8AAW/+OLIVBBBAAAEEEEAAgRgVIOCN0YqhWAgggAACCCCAAAL5I0DAmz+ObAUBBBBAAAEEEEAgRgUIeGO0YigWAggggAACCCCAQP4IEPDmjyNbQQABBBBAAAEEEIhRAQLeGK0YioUAAggggAACCCCQPwI8eCJ/HPNlK2lpabJ582Ye8Zgvmvm/kcWLF8sxxxwjpUqVyv+Ns8UjEtDHCuvg+Vo/pNgTWLBggX3sc7FixWKvcAleIv2fo0/0qlmzZoJLxN7hO44j+gTWk046KfYKF8ASEfAGsNIoMgIIIIAAAggggEDkAnRpiNyKnAgggAACCCCAAAIBFCDgDWClUWQEEEAAAQQQQACByAUIeCO3IicCCCCAAAIIIIBAAAUIeANYaRQZAQQQQAABBBBAIHIBAt7IrciJAAIIIIAAAgggEEABAt4AVhpFRgABBBBAAAEEEIhcgIA3cityIoAAAggggAACCARQgIA3gJVGkRFAAAEEEEAAAQQiFyDgjdyKnAgggAACCCCAAAIBFCDgDWClUWQEEEAAAQQQQACByAUIeCO38sypz7wurBTJviLJU1jljYX97Nu3r1CKgXt0zBkZGdG94AhzUz/RARamV2HuKzoFcqsA9RO77wPqJrK6IeCNzCnHXGlpaTJ48GBp1KiRVKlSRS677DLZunVrjnmPdGEk+4okz5GWI4ivf+ONN+Soo44q0KLPmjVLevbsKZUrV5bU1FR55JFHMu1P66Zx48ZywgknZPrp169fpnyJMjN69Gg56aSTJCUlxZ47V155paxZs6ZADj+S82L8+PGZ6sWtJ12eaOngwYPyzDPPSKtWraRcuXLSoUMH+fHHHwuM4a233pKzzjpLypQpI2eccYZ89dVXmfbFuZOJI9PMbbfdJjVq1Mi0LD9nIjl39DPMPV/C/+7Zsyc/ixKIbRX2e5VzJ7q3RfHospM7XODvf/+7TJgwQV5++WUpWbKk/O1vf5NOnTrJzJkzJSkpKTzrEU9Hsq9I8hxxQQK2gY8++khuuukmKVasWIGVfO/evdKtWzdp2bKlfPnll/Lbb7/JLbfcIsnJyXLffffZ/c6bN0/++OMPGTBggJQvXz5UloYNG4amE2Xi008/lR49ekifPn3khRdekEWLFsnjjz8uXbt2lV9++cWeS/lpEcl5MWPGDNF67Nu3b6Zd169fP9N8Isy8+uqrMnToUBk2bJg0a9ZMRowYIeecc46tGw1o8jN98803csMNN8jTTz8tzz77rLzyyivSpUsXG2CfcsopdlecOzmL62eNnj/VqlXLOUM+LI3k3Jk0aZI0adLEfv6F77JEiRLhswkxXZjvVc6dPLylTFM4KQ8Cc+bMcUxA45iAKvTqBQsWaL8Gx3wAhJZFM2ECAOfqq6/O9pJI9hVJnmwbjuMFO3fudEyLq62P4447zilduvQRH22FChWcDz74INt2HnzwQUfXmW4ToXUmYHBMq7Kzf/9+u8z8I3dMa5lz+PDhUJ5EnTj//POdevXqOaY7Q4hgzJgxtq6+/vrr0LJoJo7k3NH9mIDOMcFuNLuMy7wm6Hdq1qzp3H333aHj03OpbNmyzqOPPhpaFs2Efkbqez+nZK562PM0fJ1p+Xeuvfba0CLOnRBFaGLXrl32HDrxxBOd6tWrh5bnZeJIzh1zRdOet5999lledh13r8nv9yrnTv6+RejSkIcvCfqSL774wrZEaWuEm/SStQmuxJz87iLRSxw333yzmH/w9rL6JZdcIitXrgytD58wwZHk1Nc0kn1Fkid8X/E+/euvv8p3330n2sJ744035tri/u9//1tOPfVU2+qql1M//vjjXGm0Lg8dOpRt/eeff25bpUqVKhVad/HFF8uWLVvk559/tsu01Vdby7TVt7D7rYYKFSMTeglUW/LCr4K4LakmuAqVcu7cuXLeeefZLg/abUhbm/Rye07pSM4d3Z75wmjfBzqdyPVjvhiKug8ZMkQpbNq+fbscOHBAsrbYRXru6Dmj507WpF1Yfv/9d7n00kszrdJzR6+cuYlzx5X48++gQYNsl6Arrrjiz4VhU4V17uh+NOlnqKZEPnf0+CN5r0YTE3DuqGr+JQLePFouWbLEXkrSrgzhqVatWrJx40a7yHw3sZcCtR+gXr597bXXRP95nH766bJt27bwl3lOR7KvSPJ47iTOVjZv3txeKtd/nrml4cOH22BYL8e988470rZtW9EvJNH221T72rVrZ9qNO79hwwa7XAMq7VZhWhGlUqVKcswxx8gTTzyRkP8gtK+7BrLhadSoUVK8eHFp0aKFXayBkHYR0fPln//8p9x11132/Ondu3f4y3ynIzkvtI42b95su5ycffbZol9c9P3z7bff+m4/HjNof3ftdqNfLvSyqXYJ0svm3bt3Dx1ufpw7ixcvtttzzxV34zqv9eEGT5w7rsz//k6ePFnee+8925Uu85r/zRXmuaMBnmn9l5deekm0e5Z+tmkQrvWXiMnvvZpfMQHnTt7eXfThzZubmEtKtuUp68v1piU34B07dqz89NNPtrXCbQk2l3PtP4+nnnrK9pHTVquFCxfazWhePSEuv/xyO6+tWtq3MZJ9RZIna1njeb5ixYqeh7djxw75xz/+IaYLiehNbZo0OF63bp3cc889tg70S0n4TWX6D1jrzVx+t/n1S8xFF11k66dq1ap2mftLP/g1ue8F/SA03RmkY8eOtq+i7lP3oy1nDzzwgPuyhPw7ffp02+J75513ytFHH20NtIVRb2LSVnq9sU2TGmtfaQ1+NSDNr3NH/2lr0nLo1RgNerUFWv/qlQK3L6nNlEC/9MY1vSlXk7bm6lUqTZGcO9pP+80337T5165da/+6n2s6o767d++2y7OeO/oZqueK3gCsgTbnjmWyv/QKyHXXXSf6hSPrFwU3V2GeO1o3enPa0qVLbb9vvdKo//e0X76eOwV574R7vLH01++9GklMwLlTcDVKwJtHW728p5ensya9TJuenm4XawuRBl7aWqI3xbipadOmoXntwuDezaqXLzTgdefd7g2R7CuSPO7++Ssye/ZsG6iG14W66JcM/VDSFgqtS7cuXDMNUN1l7uV1tQ+/PK953XnNo/WqLSC6bW3d12T6KNrg97HHHrMBnF5KTsT0ww8/yF/+8hc7IoDeKOUmbVk888wz7T9Nd5meS3rO6Ws04M2vc0e7IumNWtqC6X5R0S86erXm4YcflnHjxrlFSKi/f/3rX21Xnbffftu28uqXQQ2mIjl39H3vnid6zmjSS7nueaFfHrVFX5O7zM6E/dLPUc6dMBAzqTe96o2D119/feYVYXOFee7oFatzzz1XdJQVTTpSjXYN05t29UpZbl0uwoobN5ORvFcjiQk4dwrwLWECLFIeBEzrnL1pIOtLTQue07lzZ7vY3HVuO/Sb6sv217SWZH2pYz40HNOClW15JPuKJE+2DSfIAnNJ3DGthZmO9j//+U+2OgmvJ9Panim/zuhNiuZSYrblDRo0cEyrY6blpv+u3f7rr7+eaXn4jLnD2uYxQ5qFL06Y6alTp9qbofScMa19oeM2gZJn3WS11hceybkT2nGWibPOOssxl2mzLE3MWdPP1jGt7/ZGw2jPHfOFwdZnVjkzmo1dbvq5Z1r14osv2uV6Y1ZuKRHPHXOvgGO+HDjm6pBjRmiwPybgdMyXNDu9atUqJxbOHdNFyH5W3nvvvblVX0ItD3+vRhsTcO7k71slexNlAQbX8bRpvfSqrYB66S08aX9AHYdVk16qq1u3rs2jLRrhP8uWLQt/med0JPuKJI/nThJspY6brElbC8PrxZ12W2IjYTF3tYvbV9fNv379ejtpgmF76VZb+LXbSXhyb3LTFuJES+Yftlx44YV2GD+9QUnHe3WTtnbrz8CBA3OsmyeffNLN6vs3kvPCBAr2fZB1Y1o/iVY3eslcb7oNv3lQXXS4RX1P641m+XXu6HmjyT1X7Iz5peeS249Yuz1w7vxPRlvWzb//0NUh7R5lvnzYLiY6rV2tCvvc0SE4tZ98eHKvNma9vyU8TzxOR/Jeza+YgHMnb+8gAt68udmb0fSSXfgg6RrE6g0D2vdPkxkyxo7IoIO262U7/dEPLB2DVPuCRpp0DEy/fUWSJ9L9JUI+rRutD+2+4NaN/tUxlfXStt71H2nSYEDHogz/8qNBgxmqTE477TTRgEovz+t4puFJL/npZfpEG4t3/vz5Nth1bxB0A3/XRutBbyTUG3M0ufWj/QL1ff7999+7WX3/RnJeaH/qNm3a2JvW3A1qP1XTAh26ic5dHu9/NdjULyLaBSc86ftZ+1RrN4/8Onf0y4huK3xUG92n9mF0P0M5d/6sBR1tRkf4Cf+544477JcDXda/f397rhTmudOrV6/QPSduSXWkG728r599iZQiea/mV0zAuZPHd1b+Nhgn1tb0UqyOGWluOnNMy4djPqQdc6d/aHxRc4e5Y77ROWbIFue///2vY27esJe+TUd+Ry/nZU2mj5ujPzklv33payLJk9O2431ZTl0a9JivuuoqxwScjrkBxDH/6J0PP/zQdn24//77cyQxN7E5pl9htnWmb6NjWjUc02/N0TzmS5BjWsEcM5B+KG/r1q3teJk6jq9eqjU39NjXmD68oTyJMmG+INhxkc0XAMf0nc30Yx7OYRnUyXykOebGQMcEyPZ8MUGpPd9MH7dsVEdy7uj5a1rGHNMX0TGtaI650dAx/8gdc7Oc3Xe2ncX5AvNkNcc8vSv0mfXQQw85+pkVfl5Ec+7oOaPnRU7JBNaO+cLj6KVb06rsmIdd2HlzE1QoO+dOiCLbhI73nXUc3sI8d8wDQ+x5qmORb9q0yTEPjnHMvQqOCXZD/wezFTqOF/i9V6ONCTh38vfNoi2OpDwKaJCr/4T1H7O5AcP+w9SHT4Qn7Z+pAa/mMS1VjrnZxjGtiOFZIpqOZF+R5IloZ3GWKbeAV//Bah84rTutH/Ot2TE3Kzk6+H60ybRK2QdN6HY02DVPj3JMK0doM+bbv31/6Hr9MTcyOmYEjtD6RJnQLweuQU5/w/s8azCslppPA9ILLrjA/kON1iqS80Lr79hjjw2VTfvumkvp0e4qLvJrHenDQdz60cDfPDEw05e9/Dp39Bwxo3PYL3+6P30QhXlcaiZHzp1MHJlmcgp4NUNhnTumC5ij94+Y7gv2/aKfpeYm1Fy/4GQqfBzORPJeza+YgHMn+jdQkr7EfNCQjkBAHzCgd4+7fdty2pReItXL5G7fm5zyRLIskn1FkieSfSVKHr0bXIdOqmeGXdLL53lNeirppcU6deqE7kDPui19H2j9aD/vnEb5yJqfebFdQnR4Ku2feCTJ77zQ+jPBsX2gjGnhPJJdxcVrdVg+vU9B36u59WXOr3NHPxtNC6GYLx252nHu5EqT6wq9zF4Y546+D1asWGE/+7TrS6KnSN6r+RUTcO5E/m4j4I3cipwIIIAAAggggAACARTgprUAVhpFRgABBBBAAAEEEIhcgIA3cityIoAAAggggAACCARQgIA3gJVGkRFAAAEEEEAAAQQiFyDgjdyKnAgggAACCCCAAAIBFCDgDWClUWQEEEAAAQQQQACByAUIeCO3IicCCCCAAAIIIIBAAAUIeANYaRQZAQQQQAABBBBAIHIBAt7IrciJAAIIIIAAAgggEEABAt4AVhpFRgABBBBAAAEEEIhcgIA3cityIoAAAggggAACCARQgIA3gJVGkRFAAAEEEEAAAQQiFyDgjdyKnAgggAACCCCAAAIBFCDgDWClUWQEEEAAAQQQQACByAUIeCO3IicCCCCAAAIIIIBAAAUIeANYaRQZAQQQQAABBBBAIHIBAt7IrciJAAIIIIAAAgggEEABAt4AVhpFRgABBBBAAAEEEIhcgIA3cityIoAAAggggAACCARQgIA3gJVGkRFAAAEEEEAAAQQiFyDgjdyKnAgggAACCCCAAAIBFCDgDWClUWQEEEAAAQQQQACByAUIeCO3IicCCCCAAAIIIIBAAAUIeANYaRQZAQQQQAABBBBAIHIBAt7IrciJAAIIIIAAAgggEEABAt4AVhpFRgABBBBAAAEEEIhcgIA3cityIoBAAgls2bJF3nrrraiOOCMjQ1566SXZv39/xK/773//K0lJSbJhw4aIXxNLGX/88Uf57rvvYqlIlAUBBBDIJkDAm42EBQgggIBI//79ZezYsVFRvPfee3LzzTfL4cOHo3pdUDMfOnRIWrduLUuWLAnqIVBuBBBIEAEC3gSpaA4TAQSiE9DWWm15jSbpaxIpOY4j+kNCAAEEYl2AgDfWa4jyIYBAoQvcdttt8tVXX9lL9S1atJD169fbMnz77bfSvn17qVixojRq1EgGDx4sBw4csOs+/vhjGTJkiJ1u27atvP3223Z62bJlctlll0m9evWkfPnyctppp8moUaPsuiP59eqrr0qHDh2kWrVqcvXVV9vyutvTFuZnnnlGGjdubPd5xhlnyPjx493V9m+rVq1kzJgxmZb9/e9/ty3buvD7778Xfd3ixYulc+fOUqlSJWnatKl88MEH9jV79+4V3Yamhx56SHr37m2nd+7cKddcc43UrFlTqlSpIh07dpQffvjBruMXAgggUFQCBLxFJc9+EUAgZgUuueQSSU1NtUHq7bffLhUqVJBff/3VBm+VK1eWkSNHigbFr7zyig1m9UCaNGliA0OdvuWWW2xgq8GfBribN2+WoUOH2iC0VKlS0qtXL5k3b55mzVN644037D7OOussef3112XPnj1y0UUXifY71vTggw/aYPzKK6+U0aNHS/PmzaVbt27y5ptvhvY3a9YsW67QAjOxfPlyWbRokV20a9cu+eWXX+T888+XWrVq2bKXKVPGbkeD+JIlS1oDzXzeeefJVVddZV+nLl9++aUNgjUoT05OtoH51q1b7Xp+IYAAAkUiYC5HkRBAAAEEsgiYoNfp2rVraGm7du0c09rrmG4LoWWvvfaaXs93vvjiC7vMtNza+bS0NDs/ceJE57jjjnNWrlwZeo1pMbV5TDBol3300Ud23rQih/J4TRw8eNAxQbNz3333hbKZm+QcE6A7Tz/9tLNq1SqnRIkSzuOPPx5arxPnnnuuU716dce0SNvlJmB1RowYkSmPCVodE0TbZZMmTbLlevLJJ0N5Vq9ebZeZQN8uS09Pt/MmkA7lqVu3rnPvvfeG5tetW+dcd911zvz580PLmEAAAQQKW6B4kUTZ7BQBBBAIkID5YLatnY8++mimfr3a+qlpxowZ0qlTp2xHpOsXLlxol+/evVt+//130VENtNVTuwTkJen2dBSIyy+/PPTylJQUWbp0qZ3/5JNPxATF0rNnz9B6nejSpYuYwNzm064OkSZtRXZTnTp1RFuotfU3t9SyZUsxgbfs2LFDLr74Ytsqrq3QJAQQQKAoBejSUJT67BsBBAIhsHHjRtm3b5+9tP9/7d0xSiVLFIDhfmgiGpqIICMaCIK4EPfgDgQ1cQWCIKILMDM3ExQNzRUTN2BioEvoOf/h3eb2dd5YI+PV4v0F49zuru4uv4bhTN1Tp4cHTABI6kPMYg7v7n0+OTlplpeXM++XfFZyfQmg+fORRtBMI0f2V420BBbbzc3N9Q6T70v73Vh7J/y7MTs729tNKsPvFufFrHemPZyenmY6BDnG5Dr/XypX9LDcUECBbyNgwPttHoUDUUCB7yrA4itmZV9eXt4MMdIXmqWlpTf72UHQt7293ZATfHt7m7Oe1N2lfTTgJYeYxgzqcCMoZxaZAJNrv76+Dh/OY+wYHmukJPT6kAM8Oq4/rVRBvjOL8sjZZaEcC94iLSJzgHs3c0MBBRQYo4AB7xixvZUCCtQjQKA3mJVkVpOqDNfX171f4P7+vnl+fm7W1tZy/yA4HMyAXlxc5GK2w8PDrFcbubXN3d1dBpWDa/cuWLCxsrKSM7gE0MMtcnSbnZ2dZnV1NXePjpVtgtEfUS2CNjMz01u0Rk3dx8fHPFb6g/8E0Aa/C6kWpDFQj5jrU52CzwTZLJKzKaCAAl8lYMD7VfLeVwEFvrUAJcRioVVzeXmZVRAo2XV+ft4cHR01VF94eHhoNjc3swICZchonEM7OztrqGRACTOCSKoWkFdLqa9Bbu1Hc3jn5+fzGlRioEwas7L7+/uZHxyLxTL43tjYaHZ3d7NUGRUceIEGZcq2trZyfPyg3BrpB4yNMcbCsq7KQ9fpnQ8TExOZ03t1dZW5yeT3MsMcC+qam5ubnGVmtheLQUrFO5f0sAIKKPA5AvH1lU0BBRRQYEQgZkTbmBHNKgQRFGZ1huPj4zaC2txHpYT4ur6NGd7uzEgjaNfX1/N4vHGtZZtqD1NTU+3k5GQbebdtBMN5Xiwiy/P+tEoDJ0U6QxuBcxsBZ94ryoW1jG3QIghuoyRZd5zqDMOVE+gXAXsbs8V5PmPjent7e2+qNERO8OCy+TcmBwcH3b5I2cjfLQLd3EeViJjZbaNubzc27h2zwN05flBAAQXGLfAPN/ycUNqrKqCAAnUL8M8jubDk8A4a+6LMWC4KozrCrxr5tXylH4FkHmY2l/xfFrn9zcZCuqenp2ZhYSHr4o5emxQDXpqxuLg4eqjb5nxepMF4P9oYB2kc09PT3SVIc4jgt4kyZZn/3B3wgwIKKPAFAga8X4DuLRVQQAEFFFBAAQXGJ2Ad3vFZeycFFFDgPwV4kxuvCH6vUfOX1/zaFFBAAQXKBZzhLbeypwIKKPBpAiwuY3HXe41XGLNYzKaAAgooUC5gwFtuZU8FFFBAAQUUUECBCgUsS1bhQ3PICiiggAIKKKCAAuUCBrzlVvZUQAEFFFBAAQUUqFDAgLfCh+aQFVBAAQUUUEABBcoFDHjLreypgAIKKKCAAgooUKGAAW+FD80hK6CAAgoooIACCpQLGPCWW9lTAQUUUEABBRRQoEIBA94KH5pDVkABBRRQQAEFFCgXMOAtt7KnAgoooIACCiigQIUCBrwVPjSHrIACCiiggAIKKFAuYMBbbmVPBRRQQAEFFFBAgQoFDHgrfGgOWQEFFFBAAQUUUKBcwIC33MqeCiiggAIKKKCAAhUKGPBW+NAcsgIKKKCAAgoooEC5gAFvuZU9FVBAAQUUUEABBSoUMOCt8KE5ZAUUUEABBRRQQIFyAQPecit7KqCAAgoooIACClQoYMBb4UNzyAoooIACCiiggALlAga85Vb2VEABBRRQQAEFFKhQwIC3wofmkBVQQAEFFFBAAQXKBX4CwBHMYqaCwwYAAAAASUVORK5CYII=" /></p>
<!-- rnb-plot-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBBbHdheXMgZ29vZCB0byBkb3VibGUgY2hlY2sgdGhhdCB5b3VyIGRhdGEgYW5kIHlvdXIgbWV0YWRhdGEgbWF0Y2hlc1xuYWxsLmVxdWFsKGNvbG5hbWVzKHNjX2RhdGFfZmlsdGVyZWQgKSwgc2NfbWV0YWRhdGFfZmlsdGVyZWQkZ2VvX2FjY2Vzc2lvbilcblxuYGBgIn0= -->
<pre class="r"><code># Always good to double check that your data and your metadata matches
all.equal(colnames(sc_data_filtered ), sc_metadata_filtered$geo_accession)
</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiWzFdIFRSVUVcbiJ9 -->
<pre><code>[1] TRUE</code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>We got rid of some of the ridiculously low total count samples.</p>
</div>
<div id="number-of-genes-a-cell-expresses-as-a-quality-measure" class="section level3">
<h3>Number of genes a cell expresses as a quality measure</h3>
<p>What if a particular cell has all its counts belonging to one gene? This cell would not have helpful data for us, so we should look to remove any cells we suspect might not have a useful amount of its transcriptome measured.</p>
<p>But before we can determine how many genes we consider a particular cell to be expressing we need to determine a numeric cutoff for what we consider to be a detected gene. How many counts must there be for you to consider a gene expressed? For single-cell data, it is probably best to be a bit lenient due to the nature of the data. Here let’s go for a simple detection cutoff of &gt; 0</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuZGV0ZWN0aW9uX21hdCA8LSBhcy5tYXRyaXgoc2NfZGF0YV9maWx0ZXJlZFssIC0xXSkgPiAwIFxuYGBgIn0= -->
<pre class="r"><code>detection_mat &lt;- as.matrix(sc_data_filtered[, -1]) &gt; 0 </code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Now that we have turned our data into a matrix of <code>TRUE/FALSE</code> for detection, we can sum this data by column to effectively get a vector of how many genes were measured in each cell.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBNYWtlIGEgdmVjdG9yIHRoYXQgY29udGFpbnMgdGhlIG51bWJlciBvZiBnZW5lcyBleHByZXNzZWQgYnkgYSBwYXJ0aWN1bGFyIGNlbGxcbm51bV9nZW5lc19leHAgPC0gYXBwbHkoZGV0ZWN0aW9uX21hdCwgMiwgc3VtKVxuIyBMZXQncyB0YWtlIGEgbG9vayBhdCB3aGF0IHRoaXMgbG9va3MgbGlrZTpcbnFwbG90KG51bV9nZW5lc19leHAsIGdlb20gPSBcImRlbnNpdHlcIikgKyBcbiAgZ2VvbV9kZW5zaXR5KGZpbGwgPSBcImxpZ2h0Ymx1ZVwiKVxuYGBgIn0= -->
<pre class="r"><code># Make a vector that contains the number of genes expressed by a particular cell
num_genes_exp &lt;- apply(detection_mat, 2, sum)
# Let&#39;s take a look at what this looks like:
qplot(num_genes_exp, geom = &quot;density&quot;) + 
  geom_density(fill = &quot;lightblue&quot;)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-plot-begin eyJjb25kaXRpb25zIjpbXSwiaGVpZ2h0Ijo0MzIuNjMyOSwic2l6ZV9iZWhhdmlvciI6MCwid2lkdGgiOjcwMH0= -->
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAArwAAAGwCAYAAABLkLalAAAEGWlDQ1BrQ0dDb2xvclNwYWNlR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19zn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7J0HfBTV9sd/6ZVAQo8UEekdSaF3kC69d0RBQUAREURRfAj4BKWIWAClI4IUAeGJVEX0Ifx5SFN6FQwtEAgJ/zmjG5KQ3bRtc+d3P59kZ2duOed7ZnfP3j33XI/7WgELCZAACZAACZAACZAACShKwFNRvagWCZAACZAACZAACZAACegE6PDyRiABEiABEiABEiABElCaAB1epc1L5UiABEiABEiABEiABOjw8h4gARIgARIgARIgARJQmgAdXqXNS+VIgARIgARIgARIgAS8iSBjBO7evZuxiqxFAiRAAiRAAiRAAiTgVAKenp7w9rbu1lq/4lQx3X+wK1euJAkpUPPnz4+YmBjExcUlnVfxIDAwEPfu3YPqDn9wcDCCgoJw8eJFFc2YQqfQ0FD93k1xUsEn8hqNjY3FzZs3FdTugUq+vr76m/ytW7cenFTwyN/fH3LvXrhwAapn0zTLazR37txISEjA1atXFbxjH6gkPoN8xly/fv3BSQWPxNnMmzcvxF9yhc/g5+eHsLAwq2QZ0mAVDS+QAAmQAAmQAAmQAAmoQIAOrwpWpA4kQAIkQAIkQAIkQAJWCdDhtYqGF0iABEiABEiABEiABFQg4LQY3jt37uDnn3+Gh4cHIiIi4OPjkya/y5cv45dffsGjjz6KUqVKpahz+PBhnDx5ElWrVkWePHlSXJM4vX379qFmzZopzh87dgynT59GdHQ0AgICkq7t3bsXIpOllCxZ0mbsh6UeH0mABEiABEiABEiABIxFwCkzvLdv30afPn2wZcsWLFiwAC+//HKaiw7ECe3bty+OHDmi11m5cmUSzalTp2LKlCmQOv3798epU6eSrsnCsTfeeANff/110jk5GDFiBD766COIoyzj//TTT/p1WYQ1cuRISP+Wv7Nnz6ZoyyckQAIkQAIkQAIkQAJqEHDKDO/SpUsRFRWFYcOG6dSeeeYZ7N69W591TY5x2rRpmDBhAipVqoROnTphwIABaNGiBc6dO4ft27fjyy+/hKx2XLJkCRYuXIjRo0fjjz/+wKuvvoqcOXPqf5b+Dhw4gD///BNffPGFfkpmi6VdZGQkTpw4gUKFCmHSpEmW6nwkARIgARIgARIgARJQlIBTHF4JK2jSpEkSQglJOHjwYAqHV2Zdz5w5g4oVK+r1JKWQpMSSmdfjx4/r58XZlSLt161bpx9LKp4xY8boaTC++eYb/Zz8K1u2LObMmZP0/Nq1a0kpxI4ePao7vBs2bNDDGho3bqyPlVRZO+jXr58ePiHnihcvjlmzZiW/rB+HhIQgR44cD51X6YQwlxRAqqcBEj0l3EZSqqheRFcz6Cn2lFRzyUOZVLSt6GnRVUX9LDqJjlJSh7NZrqv0aJbXqJeXl55SzwzvR2JTSZtlhiITkK4oycNU0xrfKQ6v5E0U59BS5Fic2+Tl0qVL+oeT5U1Nrgm0v/76C+fPn08xeyvtLXlxy5cvr3fz/fff64+Wf3JzWT7opG+Z6ZUwBikSMiFhDjKTLM70xx9/jHnz5qV4I5UZ6ccff1yvL853apCSby4+Pl7PIahXUvSf6JmYmKj/KaqirpbElMtfajurqLO86ZpBT0sOaXmdqlzkvU7+ZNJA5SLOkfyZ4d41y2tUcivLZIoZbCr5sl2Rm9aZ7wniv1l8I/EbnF0kp7Ot4hSHV96kkgsib8wWZ9QiXOo6cl7qyQsi9bW02lv6Sf0oDu2oUaP02GBZuCbl6aef1v/kA1GKvNhktrdHjx76c/knYRfJizjdliIfLjJzJLHJ3HjCQsXYj5IUXF6oqicGFytJUnsz6CnvMfLa5sYTxn5tWqSXzwL5u3HjhvK/OJnlNSqTDOIbqP5+JD6DWTaeEL9Kfnl3hXOf3gy6UxatyU9QMlNrKXIcHh5ueao/yo4rsitS8m96lnryc0fq9gULFkzRPq0nv/32G4YPH47nnnsOLVu2TKoiYRLJxylSpIg+i5xUgQckQAIkQAIkQAIkQALKEHCKw1u7dm2sX79enw2VtGO7du1ClSpVdIjyXP5kdk3CCFavXq2f37Ztmz4TJd90JY2ZLEKT9GIyu7tmzRp98ZktK0ifkg3i9ddfR926dVNUlb5nz56tn5NvIt999x3q16+fog6fkAAJkAAJkAAJkAAJqEHAKSENjRo1ws6dO9G1a1c91qxLly4oVqyYTnDu3Ll67KRkcBg8eHBSOjL5CWDcuHF6HYnZHThwoJ61QfZJLlq0KLp162bTAsuWLdP3537hhReS6knbVatWoWPHjnqGBkmBJpkcZNGaLIRjIQESIAESIAESIAESUI+AhxYwft9ZaknslcTVyWyurXL16lXkypXroSqy+ERCESQWxh5FZnclPji9uA8ZK3UMryxki4mJYQyvPQzhBn3IPSVx2RcvXnQDaRwrgvxqIveu6kVeoxImxRheNSwt8bty78oiaCd+bLkEnlleoxLKKDG88pmvcjFTDK+EoEpSAVfF8MrEprVi2/O01iqL5zOawistZ1eGtKykz+LwDzWzLFp76AJPkAAJkAAJkAAJkAAJKEPAKTG8ytCiIiRAAiRAAiRAAiRAAoYj4NQZXsPRocAkQAIk4CACkgpx8eLF+m6RkuJQfrKXjDGypqBnz57phn45SCx2SwIkQAJKEuAMr5JmpVIkQALuSuCDDz5ASW2rc9nN8T9ahpjb9z0QmLcg4jy8sEPLYCM7RxZ77DG0a9cuabdHd9WFcpEACZCAUQhwhtcolqKcJEAChiZw8uRJtGvfHufPnUORUmXRvc+zqFKvCXy1mV1LSdDSLh7b9wt2rV2h/a1EzVq10ESb8ZU0irJTEwsJkAAJkEDWCHCGN2vc2IoESIAEMkxAwhfqaPnAr2jZMfq+MRlvLPkGUU+2TuHsSmdeWgabUk9Eoe/rkzF53XZUb/6Uvgtk2XLlsGnTpgyPx4okQAIkQAIpCdDhTcmDz0iABEjArgQkJ/gAbTvzsAKPYPyS9ajdplOG+g/NVwD93/w3Rn2yFAEhudBHyxs+ZMiQDLVlJRIgARIggZQE6PCm5MFnJEACJGA3ArLD5IgXX0Shx0tjzOcrkb/Io5nuW2Z83/ryW0Q3a4MVK1agevUauHbtWqb7YQMSIAESMDMBOrxmtj51JwEScBiBQ4cO4dlnByFfoaJ4afYCBOd8eDOdjA7uHxiEpydMRe/XJuL0mdOoVq0a9u/fn9HmrEcCJEACpidAh9f0twABkAAJ2JtAXFwc2jz1FHy1nSVHzJqP4FyhdhmibruueOXTZfDw9kXLVq30GV+7dMxOSIAESEBxAnR4FTcw1SMBEnA+gbZt2+KmtpX64Hc/RJ7wwnYV4PFKT2iL3tZp4RHFMPSFF/DGG2/YtX92RgIkQAIqEqDDq6JVqRMJkIDLCMyaNQv79u1Dm2eHo0xEDYfIEVYgXI8JrlirPmbMmIH2WrozFhIgARIgAesE6PBaZ8MrJEACJJApAhcuXMA7kyaheMUqaNn/uUy1zWxliesdMvVjNO35NP7zn/8gOro6rl+/ntluWJ8ESIAETEGADq8pzEwlSYAEnEGgS5cu+jAD3nwPnl5eDh/S09MTnUeMQZ9xk3DmzBl9MduBAwccPi4HIAESIAGjEaDDazSLUV4SIAG3JCD5do8cOYKnBr+I/EWLOVXGOm074+VPluC+lzeaN2+B+fPnO3V8DkYCJEAC7k6ADq+7W4jykQAJuD2BxMREjBkzRl9I1qR7f5fIW7JKBMYvXY9HSpTG6FdfRe/evSFysZAACZAACQB0eHkXkAAJkEA2CYwePRqxsbHo8cqb8PbxyWZvWW8elr8gxsxfgQYde+hbEVeuXBlHjx7NeodsSQIkQAKKEKDDq4ghqQYJkIBrCMiuZ4uXLEH5GnVRrnpt1wiRbFRvH1/0GP0WBk/5EDdvx6FBw4aYOHFisho8JAESIAHzEaDDaz6bU2MSIAE7EhgyZAju3bunLx6zY7fZ7qpao2aYsGITSlaNxPTp0xEZGcnZ3mxTZQckQAJGJUCH16iWo9wkQAIuJ3D+/Hls2fI9arRoh0eKl3S5PKkFkBCHl2Yv1Lck/vPKX2jQoCFe0DarYGxvalJ8TgIkoDoBOryqW5j6kQAJOIyAOI/379/HU4OGOWyM7Hbs4eEB2ZL4X6u2oEr9xli+fDlKly4NySrBQgIkQAJmIUCH1yyWpp4kQAJ2JSCzu7t++AG12nS0+/bBdhX0n85C8+XX43pHzPocgaF5MGzYMFSvXh379+93xHDskwRIgATcigAdXrcyB4UhARIwCoEXX3xRn91tOcCxO6rZm0f56nX02N7OL47FxctX0Kx5c31r4suXL9t7KPZHAiRAAm5DgA6v25iCgpAACRiFQExMDLbv2IHqzdsaYnY3NVdJnda0xwBMXrsdDTr1xO7dP6Fq1ScgTrwswGMhARIgAdUI0OFVzaLUhwRIwOEEJO9uguYYNu/7rMPHcuQAwblC0X3UeEz4ahPK1aiDxYsXo5QW3/vpp586clj2TQIkQAJOJ0CH1+nIOSAJkICRCcgM6IaNG1GpTkOEP1bCyKokyV6g6GN44f1PMXLOIoQVLITXXnsN1apVw969e5Pq8IAESIAEjEyADq+RrUfZSYAEnE7gnXfewd07d9Cs9zNOH9vRA5aJqIHxyzag15i3EXP9Blq2aoWePXsiLi7O0UOzfxIgARJwKAE6vA7Fy85JgARUI7Bg4UIUKVVO39BBNd1EH09PT9Tr0B2TtPjeOm274D/ffYey5crp4Q4q6kudSIAEzEGADq857EwtSYAE7EBg7dq1uK5tJdykR3879ObeXQTnzIXeY/+FMfNWaGEOj+gL2po2bQpZsMdCAiRAAkYj4KElTb9vNKFdIW/qn/T8/f1x9+5d5Xcs8vLy0lMvqb4zk7e3N0TXO9pP1aoXH22Ffnx8vOpqws/PDwkJCXbNOlCxYkWcu/Qnpm76Cd4+vsoztCh4T7tfvpn7IVbPeR9y/7w/bRp69+5tueyUR5l59vX1NUV4hVleo2JPcUHM8H4knzGqZ0CRTW7kfVc+R13hWgrf4OBgq+9HdHitokl5QZLMW4q88ebPn1+f6UjtCFvqqPIYGBiov0jFuVe5yIskKCgIFy9eVFlNXbfQ0FBTzNLJazQ2NhY3b960i03Pnj2LyKgoLTPDILR/fqRd+jRaJ6eP/IaPRg/FuePHUKtmTSxatAjyQe6MIpMMcu9euHDBJR+mztDRMoZZXqO5c+fWv5RevXrVorqSj+IzyGfM9evXldTPopS8F+TNmxdXrlzRJwQt5531KM52WFiY1eEY0mAVDS+QAAmQwAMC48aN0x2teu27PThpsqPCJcvgjSXr0LTnAOzQ8hCXr1CBO7WZ7B6guiRgVAJ0eI1qOcpNAiTgVAJbtnyPijXrIbcWz2rmIqEcnYePgWxRnAAPtGjZErNnzzYzEupOAiRgAAJ0eA1gJIpIAiTgWgKff/65Fjt6W89e4FpJ3Gd02aL4reXfoli5SnjzzTfRq1cv9xGOkpAACZBAKgJ0eFMB4VMSIAESSE1AZjBDcudBxVr1U18y9fNcefNh1CdL0ahrH2zevBnR0dHKxyma2uBUngQMTIAOr4GNR9FJgAQcT0AWSZ08dUrPSeupZfJgSUnAS1uo0u3lN9D39Uk4oy3si4iMxB9//JGyEp+RAAmQgIsJ0OF1sQE4PAmQgHsT+Ne//oX7iYmo1aaTewvqYulqP9UZL81eiPiERDRs2BC7d+92sUQcngRIgAQeEKDD+4AFj0iABEjgIQIbNm7Ud1XLV6jIQ9d4IiWB0tWiMfbzlQgIyYWOHTthw4YNKSvwGQmQAAm4iAAdXheB57AkQALuT2Dr1q24eeMGanN2N8PGCn+sBMZ+sQq5wwvh6YEDsWrVqgy3ZUUSIAEScBQBOryOIst+SYAEDE/gvffeg4+vH55o1MzwujhTgbD8BTF67pfIX6QYnh8yhE6vM+FzLBIggTQJ0OFNEwtPkgAJmJ2AbKf96759qNa4OfwDg8yOI9P6h4TlxiufLkOBoo9hyNCh+PbbbzPdBxuQAAmQgL0I0OG1F0n2QwIkoBSBefPmIV7bUrtGi3ZK6eVMZYJzheLlj5cgT8FCGDDgaezZs8eZw3MsEiABEkgiQIc3CQUPSIAESOABgfnz50MctjKRNR6c5FGmCchM78iPFyM4NAydOnViyrJME2QDEiABexCgw2sPiuyDBEhAKQK3bt3SHbPqzduCuXezb1qJ6X1p9gJ4ePugWfPm3Jwi+0jZAwmQQCYJ0OHNJDBWJwESUJ/ArFmzkJCQgOhmbdRX1kkaSvaGodM+QWxsLJo2fdJJo3IYEiABEvibAB1e3gkkQAIkkIrA8uXLEVYgHMXKV0p1hU+zQ0Dy9PZ69W2cPHkC/fr1y05XbEsCJEACmSJAhzdTuFiZBEhAdQLXrl3D2bPnUL1FW9VVdYl+ddp1Qb0O3bFR29BD4qRZSIAESMAZBOjwOoMyxyABEjAMgenTpyMxMQGRTVoaRmajCdrt5ddRpHR5vDZuHBexGc14lJcEDEqADq9BDUexSYAEHEPg66+/Rr5CRVG4ZBnHDMBe4e3ji+fenaU/tm3HtG+8JUiABBxPgA6v4xlzBBIgAYMQiImJwfnz5xH5ZCuDSGxcMfOEF0af1yfhz0uXMHjwYOMqQslJgAQMQYAOryHMRCFJgAScQWDmzJlaOEMiwxmcAVsbQ8JGJFb669WrsWPHDieNymFIgATMSIAOrxmtTp1JgATSJLBmzRrkeaQwCpUoneZ1nrQ/ge6jxiMkLI+2E9sA/cuG/UdgjyRAAiQA0OHlXUACJEACGoGbN2/q2RkimzKcwZk3RGCOEPQfP0XfjGLQoEHOHJpjkQAJmIgAHV4TGZuqkgAJWCcwe/ZsPTtDtYbNrFfiFYcQKF+jrh7asG7dOuzfv98hY7BTEiABcxOgw2tu+1N7EiCBfwisWrUKofny49GyFcjEBQS6vPga/AKD0JcbUriAPockAfUJ0OFV38bUkARIIB0Cd+/exclTp/BEo+bp1ORlRxHIERqGziPG4vy5c5DFgywkQAIkYE8CdHjtSZN9kQAJGJLAF198gYR791C1flNDyq+K0LWf6oSi2oYU7/7734iLi1NFLepBAiTgBgTo8LqBESgCCZCAawksXboUsniqZJUI1wpi8tE9PDzQ49W3cEdzdocMGWJyGlSfBEjAngTo8NqTJvsiARIwJIHDR47os7ueXl6GlF8loYtXqILoZm2wYcNGnD59WiXVqAsJkIALCdDhdSF8Dk0CJOB6ApIZIF6L4a1Sr7HrhaEEOoEOQ0cB2mzvM888QyIkQAIkYBcCdHjtgpGdkAAJGJXA/Pnz4e3jg7LRtY2qgnJyhxUIR5Pu/bBv33788ssvyulHhUiABJxPwGkO7507d7Bz507s2rUL8fHxVjW9fPkyNm7ciMOHDz9UR859++23kDqpiySNl/5TF2ttMipP6v74nARIQC0Ce/f+inKas+sXEKCWYgbXpkX/wVqaskC88MILBteE4pMACbgDAac4vLdv30afPn2wZcsWLFiwAC+//DLu37//kP579+5F3759cUSLp5M6K1euTKozdepUTJkyBVKnf//+OKWlELIUWc37xhtv4Ouvv7ac0h+ttcmoPCk64xMSIAHlCBw8eBCxsTdRmeEMbmfbwBw50aLfYPzxxx/6Z4fbCUiBSIAEDEXAKQ6vrICOiorC2LFjMWvWLNy6dQu7d+9+CNS0adMwYcIEfXXunDlz8Nlnn0HyY544cQLbt2+HnBs1ahS6du2KhQsX6u3lzVCc6Rs3bqToz1abjMqTokM+IQESUI7AjBkzdJ0q1W6onG4qKNSoa18E5wzFK6+8ooI61IEESMCFBLydMfaxY8fQpEmTpKGqVq0KmVmJjo5OOndPy4F55swZVKxYUT+XP39+BGo/Z509exbHjx/Xz3t6/u2fS3tZaCJFnOcxY8bgypUr+Oabb/Rz8k8cYekrrTYZkUecbUvoRL58+dChQ4ekvi0HAdpPoD5a7J/KRfRLTEyEn5+fymrC19dXWyPjgRw5ciitpyjn7e1tCj3FnnLfyqO1snHjtyhSqixy5c1nrQrPu5CAhJlIaMPS997WJz1at26N4OBgF0rknKHN8hr18vIyxfuuvAfJZ6nqny8Wf0t8I1f4DOKr2CpOcXgvXLiAkJCQJDnkWJzb5OXSpUsICgpK8eGUM2dO/PXXXzh//jzk2FKkvTi4UsqXL68/fv/99/qj5Z+tNhmRZ9u2bZBZYimPP/44evbsqR8n/ydOkuoOr7xQ0wo/Sc5BhWOLUyQvVNWLvCmZQU+xozgO8qGaVrl+/Tri7sRp2RkefBlPqx7PuZZAvQ49sPbTmRg2bBjE4TXDvWuW16joKe+9ZrCp6Gntvci1rzD7j+4KZ1e0SG+zGqc4vGLkhISEJKoym5v6Bk9dRypLPX9/f/0mSa99Uuf/HKTuL/mYtq5Z+pk3b57lUH8UB9pS5EUqM9DXrl1LF7CljVEfZZZd2EloicpFZo3kC5d88VK9hIaGIiYmRnU19ddobGwsZEFrWkVCqO5rMwKV6jCcIS0+7nJOZnmb9XkWX34wCTIRUbJkSeW/hJvlNZo7d27dN7h69aq73G4OkUN8BvmMkS/ZKheZYMibNy/Enq7wGcTRls9xa8UpMbx58uTRZ2otQsisbXh4uOWp/ig3vnw4SfYES7HUE4BybClyXLBgQcvTNB9ttcmIPGl2ypMkQALKEFizZg1yhIahaJm/fyVSRjEFFanfsQf8A4MwaNAgBbWjSiRAAs4g4BSHt3bt2li/fr0+GypxsZKarEqVKrp+8lz+5JuBLGxbvXq1fl6+ycu3XPmLiIjAgQMH9F13ZLZRPqgiIyNt8rHVxpY8NjvlRRIgAWUIHPv9d1Su2yhFGJUyyimmSEBQMBp364vffjuEQ4cOKaYd1SEBEnAGAaeENDRq1EjPkSvZFWRqv0uXLihWrJiu39y5c/U4WInPGjx4cFI6Mqk3btw4vY7E7A4cOBADBgxAWJg2I1O0KLp162aTj602tuSx2SkvkgAJKEFg8+bN+u5qFWvVV0IfMyjRsEsfrJ83GyNHjkyaGDGD3tSRBEjAPgQ8tAVJDyfEtU/fD/UiqcMkdldmc20Vif/IlSvXQ1VkwwoJecjMKl1bbTIqjwiSVgyvxEGmFyT9kBIGO2G2GN6LFy8azEKZF9cs8YESZ28thldSGW7e/B9M37YPMnvIYgwCX0x8DVtXLMKvWj52CU1TtZjlNcoYXrXuYEsMryQVcFUMr0yKWitOCWmwDC4pOdJzdqVuWs6unJeMCJlxdtNrk1F5pB8WEiABdQj89NNPKFGlGp1dg5m0Sff+eprE0aNHG0xyiksCJOBqAk51eF2tLMcnARIgAcnEIRlWuNmE8e6F/EUeRZW6jfHtpk169hjjaUCJSYAEXEWADq+ryHNcEiABlxD49NNP9bRWFWrVc8n4HDR7BJr2HKDHX8tW8ywkQAIkkFECdHgzSor1SIAElCCwYcMG5MyTD48UL6mEPmZTomTVSBQuWQZzPv7YbKpTXxIggWwQoMObDXhsSgIkYDwCJ06eRGVuNmE8wyWTWGJ578TF6ekuk53mIQmQAAlYJUCH1yoaXiABElCNwCYt9jNe2zWQ4QzGtmzkk60QmCME77zzjrEVofQkQAJOI0CH12moORAJkICrCSxatAgeWo7vMhE1XC0Kx88GAR9fP9TTdl+TzUPOnj2bjZ7YlARIwCwE6PCaxdLUkwRIAHv27MHjFasiIDgHaRicQL323fTFh+PHjze4JhSfBEjAGQTo8DqDMscgARJwOQHZKCYm5iq4u5rLTWEXAfKEF0LFmvWwSds1j4UESIAE0iNAhzc9QrxOAiSgBIHPP/9cmxFMRPkadZXQh0oA9Tv11BevffbZZ8RBAiRAAjYJ0OG1iYcXSYAEVCGwbt06BIbkRJHS5VRRyfR6VNBmeCXF3EcffWR6FgRAAiRgmwAdXtt8eJUESEARAseO/Q5xkDw8PBTRiGp4agsQ63XojjNnzuKklm6OhQRIgASsEaDDa40Mz5MACShDYN++fYiLu43y1esooxMV+ZtA7ac6aQf3MWHCBCIhARIgAasE6PBaRcMLJEACqhCYN2+erkq56NqqqEQ9/iEQlr8gymlfZL777jsyIQESIAGrBOjwWkXDCyRAAqoQ2LFjBwoWexy58uZTRSXqkYxA3XZdcPv2bSxfvjzZWR6SAAmQwAMCdHgfsOARCZCAggQSExNx4eJFpiNT0LYWlSrVaYSgnLkwc+ZMyyk+kgAJkEAKAnR4U+DgExIgAdUIrFy5Egn37qFcdC3VVKM+/xDw9vFBzVYd9J3Xrl27Ri4kQAIk8BABOrwPIeEJEiABlQh89dVX8PL2RsmqUSqpRV1SEajVpiMSExIwadKkVFf4lARIgAQAOry8C0iABJQmsHfvryhRJQK+/v5K62l25Qo9XgpFSpXD6tWrzY6C+pMACaRBgA5vGlB4igRIQA0Cf/31F+QnbqYjU8Oe6WkhKcrE5pKGjoUESIAEkhOgw5ucBo9JgASUIiBbzsp2wkxHppRZrSoT9WRreHp5YfLkyVbr8AIJkIA5CdDhNafdqTUJmILA2rVrERCcg9sJm8LaQHCuUFSs3QA//PCjSTSmmiRAAhklQIc3o6RYjwRIwHAEDh0+rG9KwO2EDWe6LAtcs1V7fVc9yc7BQgIkQAIWAnR4LST4SAIkoBSBw5qzeys2lunIlLJq+spU0mZ4/YOC8eGHH6ZfmTVIgARMQ4AOr2lMTUVJwFwELA5P2Sjm3zWT5b19fCGxvIcOHcLdu3fNpDp1JQESsEGADq8NOLxEAiRgXAIbN25EWIFw5H2ksHGVoORZIlC9RVvc0zYbmT17dpbasxEJkIB6BOjwqmdTakQCJKAROHHyJCrUqEsWJiRQonI1hOYrgEWLFplQe6pMAiSQFgE6vGlR4TkSIAFDE9i1axfu3rmDMlE1Da0Hhc8aAVmkWL1lO5w5cxYxMTFZ64StSIAElCJAh1cpc1IZEiABIWCZ2SsTUYNATEogulkbJCYm4P333zcpAapNAiSQnIDHfa0kP8HjtAncvHkzxYXg4GAt9U2cHieW4oJiT3x8fLQPjUQkaHvUq1x8fX0husZqq/pVL/7aFrty76pcSpcuDY+AYLy5fKPKalK3dAiMbd8Iibdu4ujRo+nUdK/LZniNCvGAgAD98+WO9muMykV+cZDPGNX19PT0RGBgIG7duqXb1dk2FV8lJCTE6rDeVq/wQgoCt2/fTnouN684vLICWPUbWJSWxR/x8fFJ+qt4IDb19vZGcjurqKfoJI69ynrKm96FixfRqFtzVU1IvTJIoHqLdvhqxhQcOXIEhQsbZ/Gi6q9Ri/nECZTXq8rvR6KrOILyp7qeXtouh+Lwim/kCp9BPsNtFdtXbbU02TVx+ixFblwpMuuZ/LzlukqPoqMZ9JQ3XSmq21N0lB91VNZz9erVSNBer2UjGb8r9jZziXqyFVZMn4xJkyZh2rRphkGh+ms0uSHMoKv4DPIZo/L7bnKbuspnEIfbVmEMry06vEYCJGA4Al999ZU2m+KFklUjDSc7BbYvgTzhhVGsfCVs2rTJvh2zNxIgAcMRoMNrOJNRYBIgAVsEfvnlv3isQmX4BQTaqsZrJiFQvXlbPVOD7LzHQgIkYF4CdHjNa3tqTgLKEZDFeDExf2nbCddWTjcqlDUCEY21WG4tRv+9997LWgdsRQIkoAQBOrxKmJFKkAAJCIFly5bpsXJlIpmOjHfE3wRy5smHUk9E4futW4mEBEjAxATo8JrY+FSdBFQjsGbNGnhrK78lpIGFBCwEop5sjRvXr2Pv3r2WU3wkARIwGQE6vCYzONUlAZUJHDhwAKWqRsHbx1dlNalbJglUa9gMHtpK+Q8++CCTLVmdBEhAFQJ0eFWxJPUgAZMTkC1kr2uzeNxO2OQ3QhrqB+cKRVltm+kdO3emcZWnSIAEzECADq8ZrEwdScAEBGQ7Ycnpye2ETWDsLKgoYQ2x2o6Zu3fvzkJrNiEBEjA6ATq8Rrcg5ScBEtAJbNy4EX6BQShapjyJkMBDBKrWbwJPLTH9jBkzHrrGEyRAAuoToMOrvo2pIQmYgsBvhw5ps7vV9S08TaEwlcwUgcAcOVGuem388OOPmWrHyiRAAmoQoMOrhh2pBQmYmsCFCxf0n6vLcDthU98H6Skf1bQ1bsXGYseOHelV5XUSIAHFCNDhVcygVIcEzEhgwYIFutrMv2tG62dc5yr1GsPL2xuzZs3KeCPWJAESUIIAHV4lzEglSMDcBDZt2oSgkFx4pHhJc4Og9jYJBATnQPkadbH7p59s1uNFEiAB9QjQ4VXPptSIBExH4Njvv6NsdC1tB1kP0+lOhTNHILJpK9y+dQvbt2/PXEPWJgESMDQBOryGNh+FJwESOH78uO7AyII1FhJIj0Dluo20sAYfzJw5M72qvE4CJKAQATq8ChmTqpCAGQkkxe9G1DCj+tQ5kwQCgoJRoVY97NnzcyZbsjoJkICRCdDhNbL1KDsJkAC+//57hITlQf6ixUiDBDJEILJxC9y+fQtbt27NUH1WIgESMD4BOrzGtyE1IAFTEzh+4gTKafG7LCSQUQKV/glrYLaGjBJjPRIwPgE6vMa3ITUgAdMSOKRtNhF3+zZKRzKcwbQ3QRYUl7CGirXr4+eff8lCazYhARIwIgE6vEa0GmUmARLQCSxcuFB/5II13hCZJRDxT1jDd999l9mmrE8CJGBAAnR4DWg0ikwCJPA3gW3btiE0XwHkCS9MJCSQKQKSrcHbxwezZ8/OVDtWJgESMCYBOrzGtBulJgES0AicOnVai9+tTRYkkGkC/oFBqFirAX7+hWENmYbHBiRgQAJ0eA1oNIpMAiQA7N+/H3fuxKE08+/ydsgigcimLfUY8M2bN2exBzYjARIwCgE6vEaxFOUkARJIQWDRokX6czq8KbDwSSYIVKrTkGENmeDFqiRgZAJ0eI1sPcpOAiYmsGPHDi12txDC8hc0MQWqnh0CfgGBqFSnEf77373Z6YZtSYAEDECADq8BjEQRSYAEHiZw5uxZlI1i/t2HyfBMZgjoYQ1xt/Htt99mphnrkgAJGIwAHV6DGYzikgAJAL9oC43u3rnD+F3eDNkmIAvXfHz98OGHH2a7L3ZAAiTgvgTo8LqvbSgZCZCAFQJLlizRr5SuVt1KDZ4mgYwR8AsIgKQo+/XXfRlrwFokQAKGJECH15Bmo9AkYG4Cu3btQr5CRZArbz5zg6D2diEQ+WQrPePH6tWr7dIfOyEBEnA/At7OEumO9vPjzz//DA8PD0RERMBHS/idVrl8+bL+c+Wjjz6KUqVKpahy+PBhnDx5ElWrVkWePHmSrqXV5vz58zhx4kRSHTmQsaOjo/Vze/fu1d7g7iRdL1myJMLCwpKe84AESMB9CUj8bo1WHdxXQEpmKAIVa9aDr78/Pv74Y7Ru3dpQslNYEiCBjBFwisN7W9vrvl+/fihXrhzOnTuH5cuX47333tMd0ORiihM6btw4NGnSBLNmzUKfPn3Qtm1bvcrUqVPxv//9DyVKlMDMmTMxffp0FClSBNbaHDt2DGvXrk3qXsa9du0a5Bv8vXv3MHLkSDzxxBNJ13v06EGHN4kGD0jAfQns2bMH8XfvgtsJu6+NjCaZj58/qjZ4Ev/d/I3RRKe8JEACGSTgFId36dKliIqKwrBhw3SxnnnmGezevTtpttUi67Rp0zBhwgRUqlQJnTp1woABA9CiRQvdSd6+fTu+/PJLeHp6QuL3Fi5ciNGjR8Nam9q1a0P+pMhMbv/+/TFq1Cj9ucz8FipUCJMmTdKf8x8JkIBxCFjid0tV+/vXGuNITkndmUDUk63x4zersGzZMv3zx51lpWwkQAKZJ+AUh1dmW2XW1lIkJOHgwYMpHF6ZdT1z5gwqVqyoV8ufPz8CAwNxVvvp8vjx4/p5cXalSPt169bpM7XW2hQrVkyvK/8+/fRTVKhQATVr1tTPHT16VHd4N2zYoDvDjRs31sdKaqAdHDp0KCnkIUBb1JA83EFCI6R4eXlZDc3QKyjwT3SUcv/+fQW0sa6CRU9roTbWWxrvity/Rtbzhx9+QP4ijyJn7rzGg0+J3ZaAbFEdEBSMzz77DN27d3epnEZ/jWYUnugpn+tGfj/KiK5m0dPyOeoq30g42ypOcXgvXLiAkJCQJDnkWBzV5OXSpUsICgpKEeaQM2dO/PXXX5B4XDm2FGl/5coV2GpjcXgljOGrr77SZ4Qt7Y8cOQKJB5aZZHGmJW5r3rx5KeKChw8fjj/++ENvIrHEaS1mSK6TpW8+GptA8thwY2tiW3o/Pz/bFdz46lktPKlWm05uLCFFMyIBb21dSbUmLfDD2q/0CQ7LBIurdDHyazSzzMyiq0yemaHkypXLJWreunXL5rhOcXjF209ISEgSRGZzUxs+dR2pLPX8tYUEqa9Z2qc+n7yNZTCZxY2MjITMGFvK008/DfmTGWQpEvIg9SSO11I++OADxMXF6U9FVlkYZynyLSJ37ty4fv067mqxhCoX4S+85U/lIveC6CpfsFQvOXLkwI0bNwyppmRnkPhdhjMY0nxuL3T0k22wfeVSTJw4ERJ656pi5NdoZpjJRFZiYqJh348yqqv4DPIZExsbm9EmhqwnPlloaCiuXr3qEp9Bxrf4dWkBdIrDK7NmyR0JOS5cuHAKecSBlJtBnE/Ltz2pFx4eDpkh3r9/f1J9OV+wYEHd6bTWxlL5m2++wXPPPWd5qj9KmITIZAEji99kFjl5kcVxyUvy65Zv/uLEx8fHJ6+m3LH81GQGPS1fyFS3p9ygEp5iVD0XLVqkv8ZKM35Xufcad1BIvkjlCA3D/Pnz9YXWrpLJyK/RzDATPcXhNer7UUZ1FZ/BDHpaQh9d5TNYfDNrdnFKHl5ZPLZ+/Xp9xlRmSmWWpkqVKrpM8lz+vL299YVtltCBbdu26d8U5NuCpDE7cOAATp8+rX9rWLNmjT5ra6uNdC7OsyxQK1++fAr9pe/Zs2fr52QK/LvvvkP9+vVT1OETEiAB9yOgx+8WLYaQsAdpCd1PSkpkVALygRnVrI0WznYc6f08alQdKTcJmJWAUxzeRo0a6fG5Xbt21X8m6tChAywxtnPnzsWCBQt0/oMHD9YzMXTr1g1z5szRszDIBYmVHThwoJ61oWfPnvrPH1JHirU2cu3UqVPImzdv0kyunJPSsWNH3Lx5E3379tVX48oiOPljIQEScG8C57Vfe8pG1XJvISmdoQlEaw5vQsI9PTWmoRWh8CRAAikIeGhT0E5bfi9xgxIPKzOztorEf6QV9Cw/e8isbXBw8EPNrbV5qGKyE/INXmI+LCEUyS49dJg6pEFigmNiYpLifB9qoMgJCfuQ+F3VY5XlnpJFkxcvXlTEctbVkF9N5N41Wvnxxx/Rrl07DJo8ExGNWxhNfMprIAIvt6gFP4/7kJzPrihGfY1mlpWEMsrP3/L5rXKRXw7kM0bW/ahcxLeTSUZJKuAKn0F8ueQZtVKzdsoMr2VQCcRPz9mVumk5u3Je4knTcnZttZFr1oo4cxlxdq2153kSIAHnEZB83lJKPRHlvEE5kikJ1GjZDue0dR3JFyubEgSVJgGFCDjV4VWIG1UhARJwMgHG7zoZuImHkzje+9piqilTppiYAlUnAbUI0OFVy57UhgSUJcD4XWVN63aKFXy0OIqWLq9vcOR2wlEgEiCBLBGgw5slbGxEAiTgTAISvyv5d5mOzJnUzT1WjVbt9XSasisoCwmQgPEJ0OE1vg2pAQkoT4Dxu8qb2O0UjHqyFTy0xUaTJ092O9koEAmQQOYJ0OHNPDO2IAEScDIBxu86GTiH03M9V6hRF9u37yANEiABBQjQ4VXAiFSBBFQnwPhd1S3snvrVbN0Bt2/fgmVDJPeUklKRAAlkhAAd3oxQYh0SIAGXEbDE7zIdmctMYNqBK9dtBP+gYMyYMcO0DKg4CahCgA6vKpakHiSgKIHly5frmtHhVdTAbqyWj68fqrdoi98OHeJWw25sJ4pGAhkhQIc3I5RYhwRIwGUEdu3ahfxFHkXO3HldJgMHNi+BWq07IkHbbXLq1KnmhUDNSUABAnR4FTAiVSABlQnIjldlImuqrCJ1c2MCxcpVRMFixbF06TI3lpKikQAJpEeADm96hHidBEjAZQT27NnD/Lsuo8+BLQTqtu+mbTP8J/bv3285xUcSIAGDEaDDazCDUVwSMBMB5t81k7XdV9caLdrB08sLEydOdF8hKRkJkIBNAnR4beLhRRIgAVcSkPy7+QoVQc48+VwpBsc2OYHgXKF4osGTkPsxMTHR5DSoPgkYkwAdXmPajVKTgCkInDl7jvG7prC0+yspYQ13te2tZ86c6f7CUkISIIGHCNDhfQgJT5AACbgDgV9++UWL372DUtWi3UEcymByAmUiayBPeCHMnTfP5CSoPgkYkwAdXmPajVKTgPIEHsTv0uFV3tgGUNDDwwMNOvXChQsXcODAAQNITBFJgASSE6DDm5wGj0mABNyGgOTfzRNeGKH58ruNTBTE3ARqtekIL23x2vjx480NgtqTgAEJ0OE1oNEoMgmYgcCZs2dRJqqGGVSljgYhIIvXIpu2wu6ffkJcXJxBpKaYJEACQoAOL+8DEiABtyOwb98+3L1zB6WrVXc72SiQuQk07NIb9+LjMWnSJHODoPYkYDACdHgNZjCKSwJmILBkyRJdzVJPRJlBXepoIAKPla+MoqXLY/HixQaSmqKSAAnQ4eU9QAIk4HYEdu7cibAC4QjLX9DtZKNAJNCk5wBcv34dK1asIAwSIAGDEKDDaxBDUUwSMBOB06fPoGxkTTOpTF0NRCCicQuEhOXG5MmTDSQ1RSUBcxOgw2tu+1N7EnA7Av/73/9w504c8++6nWUokIWAt48PGnfrhzNnzkDizVlIgATcnwAdXve3ESUkAVMRYPyuqcxtWGXrdewBb18/jB492rA6UHASMBMBOrxmsjZ1JQEDENi2bRtyabl3ZVcrFhJwVwJBITlRt31X7N//fzirpdBjIQEScG8CdHjd2z6UjgRMR+DU6dOM3zWd1Y2pcNMeA3TBX3rpJWMqQKlJwEQE6PCayNhUlQTcncDhw4dxR0voX6oatxN2d1tRPiB3wUcQ3fwp7NCyisTExBAJCZCAGxOgw+vGxqFoJGA2Ag/id+nwms32RtW3Rb9BSEhIwIsvvmhUFSg3CZiCgLcptLSDkj7aqlxL8fDw0A9lT/Xk5y3XVXoUHaXcv39fJbUe0sWip+r2FMU9PT3d9r6V+N2cufMiX6EiD9mIJ0jAHQkULPY4Iho1x+bN3+L27dsICQnJtpjyGWOG9yLR053fj7JtyH86EB3NoKflc9RVvpHFN7NmNzq81sikOu/n55d0xgJV3pAsx0kXFTvw9vbWX6jyYlW5WF6oye2sqr5iS3fV8/iJE6hcr4mq6KmXogRaDRyKPZvWYcSIEfjiiy+yraW8H7nrazTbyiXrwOLwqq6r6GkGm1r8BF9fX13fZKZ2ymF6E3N0eDNohps3bybVFKMGBwcjTos1lD+VS2BgIO7du4e7d++qrKZuT/kCk9zOqirsrnr+8ccfiNNmyEpXq64qeuqlKIFCj5eCbEaxbt03OK0tugwNDc2Wpu76Gs2WUmk0FkdXwkFUf98Vn0GcXtX1lAmyoKAg/ZcOV/gM6X1xUnvaLo0XGE+RAAm4J4FFixbpgpV6Iso9BaRUJGCDQJtnh2nO2z0MHTrURi1eIgEScBUBOryuIs9xSYAEUhDYunUrcoSGIX/RYinO8wkJGIFA+GMlUKNVe2z5/nt9ltcIMlNGEjATATq8ZrI2dSUBNybwx/HjKBNZ040lpGgkYJtA20EjtJ+uPfHss8/arsirJEACTidAh9fpyDkgCZBAagInT57E7Vu3wHCG1GT43EgEwgqEo0mP/vj111+xZ88eI4lOWUlAeQJ0eJU3MRUkAfcnsHjxYl3I0txwwv2NRQltEmjRbzACgnNg0KBBNuvxIgmQgHMJ0OF1Lm+ORgIkkAaBLVu2IChnLkhOUxYSMDKBwBwhaD/kZZw7dw5z5841siqUnQSUIkCHVylzUhkSMCaB37WUZGUiahhTeEpNAqkI1G3XFbKIbcLbbyuf0jGV6nxKAm5LgA6v25qGgpGAOQicPXsWt2Jjtfy73E7YHBZXX0tPbeOIXmPe1uPSn3/+efUVpoYkYAACdHgNYCSKSAIqE7DE75aiw6uymU2nW8mqkaje4il8s349Dh48aDr9qTAJuBsBOrzuZhHKQwImI/Ddd99B4h7lJ2AWElCJQOcRY+HrH4BevXqppBZ1IQFDEqDDa0izUWgSUIfA0aPH9O2EZetNFhJQiUBIWB500ZxeWcA2ZcoUlVSjLiRgOAJ0eA1nMgpMAuoQuHDhAmJjb6J0RHV1lKImJJCMQO22nSHhDdNnzMD58+eTXeEhCZCAMwnQ4XUmbY5FAiSQgkBS/O4TUSnO8wkJqEJAfrno94Y2u6s9duzYURW1qAcJGI4AHV7DmYwCk4A6BDZv3oyAoGAUKlFaHaWoCQmkIpCvcFF0fGE0/tDS702dOjXVVT4lARJwBgE6vM6gzDFIgATSJHD06FE9nIHxu2ni4UmFCDTq2kcPbZg6dRqOHz+ukGZUhQSMQYAOrzHsRClJQDkCly5dws2bkn+X8bvKGZcKPURAvtQNeOs9ePp4o0OHDg9d5wkSIAHHEqDD61i+7J0ESMAKgb/jd++jVDXG71pBxNOKEcgTXgg9X3lLX7w2cuRIxbSjOiTg3gTo8Lq3fSgdCShLQOJ3/fX43TLK6kjFSCA1gZqtO6Bao2aQL3w//vhj6st8TgIk4CACdHgdBJbdkgAJ2CZw+MgRlNKyM3h68m3INileVY1A79feQY7Q3PqGFHFxcaqpR31IwC0J8JPGLc1CoUhAbQKXL1/W4neZf1dtK1M7awSCQnLimXem42ZsLDp37mytGs+TAAnYkQAdXjvCZFckQAIZI6DH796/ry1Yi85YA9YiAcUIyL3fst9z2LNnD+bMmaOYdlSHBNyPAB1e97MJJSIB5Qls2rQJfoFBKFyyrPK6UkESsEagzbPDUKx8ZUx4+22mKrMGiedJwE4E6PDaCSS7IQESyDiBQ4cPo/QT0YzfzTgy1lSQgJe3NwZNmg4vbx+0bddOQQ2pEgm4DwE6vO5jC0pCAqYgcOXKlX/idxnOYAqDU0mbBPKEF0afce/g0sWLGDRokM26vEgCJJB1AnR4s86OLUmABLJAYNGiRYAev8sNJ7KAj00UJBD1ZGvUbNUeq9esgYT7sJAACdifAB1e+zNljyRAAjYIJMXvlmL8rg1MvGQyAt1feRO5C4Tj2Wef1X8BMZn6VJcEHE6ADq/DEXMAEiCB5AT0+F1thTrz7yanwmOzE/DXFnE+O2kG4u7cQdOmTc2Og/qTgN0J0OG1O1J2SAIkYI0A8+9aI8PzJAA8pmVsaD3wBfz3v//FJ598QiQkQAJ2JECH144w2RUJkIBtApb43TIRjN+1TYpXzUqg1YDn8WjZinhrwgRcuHDBrBioNwnYnQAdXrsjZYckQALWCEj8rn9QMAqVKGOtCs+TgKkJeHp5YeC/pmnrOu+jU6dOpmZB5UnAngSc5vDe0eKSdu7ciV27diE+Pt6qDvKT58aNG3FYy9OZusi5b7/9FlInebHW5tSpU/jxxx+T/o4ePZrULKPyJDXgAQmQQLYJHD58BKW12V3G72YbJTtQmECBoo+h/dBROHbsGGbOnKmwplSNBJxHwCkO7+3bt9GnTx9s2bIFCxYswMsvv6x/e02t5t69e9G3b18cOXJEr7Ny5cqkKlOnTsWUKVMgdfr37w9xZqXYaiMxUHPnzoX0I38//fST3iaj8uiV+Y8ESMAuBC5duqSvPi9TjeEMdgHKTpQm0LhbP30Xtsna517qSR6lFadyJOAgAt4O6jdFt0uXLkVUVBSGDRumn3/mmWewe/duREenTDw/bdo0TNDilipVqqT/lDNgwAC0aNEC586dw/bt2/Hll1/qM0NLlizBwoULMXr0aFhr4+vrC5nRnTRpEooUKZIleVI04hMSIIFsEZDXrJaAV5/hzVZHbEwCJiAgv4L0Hz8Fr3duhu7du+u/fJpAbapIAg4j4BSHV36WadKkSZISVatWxcGDB1M4vPfu3cOZM2dQsWJFvV7+/PkRGBiIs2fP6nuMy3nLz6DSft26dbDVRtr/9ddf+PPPP7Ft2zbUq1cPhQoV0vvOiDw9e/bEiRMn9PolSpTAxx9/rB8n/5czZ06EhIQkP6XcsYeHR5qz8aopKnrKX758+VRT7SF95HXkCj3lF57AHCFa/G7ph2TiCRIggYcJhD9WAi37P4+vP5qGzZs3o1u3bg9XMvAZeS/y1rZXdsX7kbOxyeeLv7+/s4d1yXi5cuVyybhxcXE2x3WKwysrTZM7hnIszm3yIj93BgUF6U6H5bw4lOK0nj9/HnJsKdJetie11ebmzZuQON09e/YgICBAn12WcAmZMc6IPA0aNNDHljHz5s0LCYNIXoKDg3H37l3d6U5+XrVjeTOSxRMJCQmqqZZCH/lFwMfH5yE7p6ikyBN5003vjcERqv526BDKRNZK8Rp3xDjskwRUItCi/2D88M0qDBo8GK1atdIdRFX0k8/mxMRE/bNaFZ3S0kOcXfl8EZ9B5SJfYGSiUnwvsauzS3pjOsXh9dJWnSZ3mGRmVm705CV1Hbkm9eTDOfU1S/vU55O3KVWqlB63Gxoaqg/z+OOP47PPPtMd3tTtLP3pFf/5J85x8iJOt6WIUcXhFSfYFY6DRQ5nPMrNK3xUf6GKPcW5v3HjhjOwunQMV+gpv9TEal9CS0fWcKnuHJwEjEbA28cXvV/7F6YM7Kavhfnwww+NpoJVeWWiQXwD1d93LT6D6nrKZ4v4DOIbucJn8PPzs3qvyQWnLFrLkydP0mypDCqztuHh4XKYVHLnzo3Y2NgU3/Qs9WSGVY4tRY4LFiwIW22uXr2KmJgYSxM9jvfixYv6t46MyJPUkAckQALZJvDFF1/ofZTmgrVss2QH5iNQJqIGIpu2wpq1a/HHH3+YDwA1JgE7EHCKw1u7dm2sX79enw2V1aaSmqxKlSq6+PJc/uSbgSxsW716tX5e4m5ldlb+IiIicODAAZw+fVqfbVyzZg0iIyNttrl27RqGDx+uf9OQn+TXam8UdevW1eOAbcljB6bsggRIIBWB7777DsG5QvFI8RKprvApCZBARgh0HjEWXt4+eiajjNRnHRIggZQEnBLS0KhRIz0Hb9euXXWHs0uXLihWrJguiaQNk9gWyeAwWItRkpRlkkJMfgIYN26cXkdidgcOHAjJ2hAWFoaiRYsmBe9ba1O8eHG0a9dObyc/yUsfb731lt6fLXlS4uEzEiABexD4/fc/UKFOQ3t0xT5IwJQEQvPlR+unh2DFjCn6BFKzZs1MyYFKk0BWCXhos5/3s9o4s+0kfkVid2U211aRcIS0VvnJhhUSDC3xlqmLtTYSxCwL2MThTV0yKo+0Sx3DK1kgJGSCMbypqRrzudxTsmhSwl5UL/KrSfJwH0frK1lR6tSpo8UhTkTddl0dPRz7JwFlCcTficMrrevB495d/VdPoysqYYkSwyuf3yoXSwzv9evXVVZT9+0kBFWSCrgqhlcmRa0Vp4Q0WAbPkSNHus6u1E3L2ZXzMhOclrNrq43caGk5u9Imo/JIXRYSIIGsEZAuT+ekAAA+qklEQVTNZqRIHCILCZBA1gn4+Pmj4wuv6Gtapk+fnvWO2JIETEjAqQ6vCflSZRIwPQHJv5tL+zk2X+GipmdBACSQXQJRzdqgSKmymPb++y5J/ZRd+dmeBFxFgA6vq8hzXBIwCYFTp06jfHQdk2hLNUnAsQQkp2vHYa/i9q1beP311x07GHsnAYUI0OFVyJhUhQTcjcC+ffu0uPs45t91N8NQHkMTKBddC6WqReMLLVzIFbGShoZH4U1LgA6vaU1PxUnA8QQYv+t4xhzBnAQ6DHkZd7VF3KNGjTInAGpNApkkQIc3k8BYnQRIIOMEtm/frsXuPgpJqcRCAiRgPwLFK1ZFhZr18JWWxlP1bEH2o8aezEyADq+ZrU/dScDBBM5pW3KXr17bwaOwexIwJ4GnBo1A/N27eOWVV8wJgFqTQCYI0OHNBCxWJQESyDgB2V3tnpY7u0xUzYw3Yk0SIIEMEyhWrqI+y7ty1SrO8maYGiualQAdXrNannqTgIMJLFu2DNBWlJeJqO7gkdg9CZiXQOtnXtBneS07k5qXBDUnAdsE6PDa5sOrJEACWSSwe/duFC1VDoE5cmaxBzYjARJIj0DxClVQulp1LP/yS+blTQ8Wr5uaAB1eU5ufypOAYwhIqqQ/L19GuRrMv+sYwuyVBB4QaDVwCO7ExWHixIkPTvKIBEggBQE6vClw8AkJkIA9CHz11VdITEhAuaha9uiOfZAACdggINt2Fy1dHvPnf26jFi+RgLkJ0OE1t/2pPQk4hMAqbRGNt48PHq/8hEP6Z6ckQAIpCTTvNxg3b97A3LlzU17gMxIgAZ0AHV7eCCRAAnYnIDuslawaBR9fP7v3zQ5JgAQeJvBEwyeRu+AjeP/99x++yDMkQAKgw8ubgARIwK4ELmuxu9evX9fy7zJ+165g2RkJ2CDg6emJJ3sNxKVLl7B161YbNXmJBMxJgA6vOe1OrUnAYQTmz5+P+/fvo2w043cdBpkdk0AaBGq16QT/oGC88cYbaVzlKRIwNwE6vOa2P7UnAbsT2LhxI4Jy5kLhkmXs3jc7JAESsE7ALyAA9Tv2wJGjR3Hy5EnrFXmFBExIgA6vCY1OlUnAkQSOHvsdFWrU1fac8HDkMOybBEggDQINu/TWz44dOzaNqzxFAuYlQIfXvLan5iRgdwJ79+7V8oHeRnnN4WUhARJwPoGw/AVRrWEzbNu2HZIPm4UESOBvAnR4eSeQAAnYjcDnn3+u91WO8bt2Y8qOSCCzBBp364f4+LuYNGlSZpuyPgkoS4AOr7KmpWIk4HwC27dvR/hjJZAzTz7nD84RSYAEdAKS/7pIqbJYtGgRiZAACfxDgA4vbwUSIAG7ELh37x4uXryIirXq26U/dkICJJB1AjLLe+3aNaxfvz7rnbAlCShEgA6vQsakKiTgSgLLly9HgradcIWa9VwpBscmARLQCEQ2bYnAHCF45513yIMESEAjQIeXtwEJkIBdCKxYsULfWa1ElWp26Y+dkAAJZJ2Aj58/6rbvht9//x3nz5/PekdsSQKKEKDDq4ghqQYJuJrA/v/7P5SJrAFvH19Xi8LxSYAENAL1O3ZHorYJDDei4O1AApzh5T1AAiRgBwLHjx/HzRs3UIHxu3agyS5IwD4E8oQX1nNib9q02T4dshcSMDABzvAa2HgUnQTchcCnn36qi8L4XXexCOUggb8JNOjcC3FabuzPPvuMSEjA1ATo8Jra/FSeBOxDYPPmzchbqAjyaX8sJEAC7kNAvoSG5iuAD2fPdh+hKAkJuIAAHV4XQOeQJKASgcTERJw9dw6V6jRUSS3qQgJKEPD09NRieXvg3NmzOHr0qBI6UQkSyAoB76w0MmOb0NDQh9QOCgpCQEDAQ+dVOuHl5YX72qIHcWpULt7e3vDw8EBadlZNbx8fH7vquXDhQiRoOXiZf1e1O4X6qEKgdtvOWPXhVLz99tv4+uuv3UYted+VzxgzvO+KrqrrKZ+hUoKDg3W/wdk3muSCt1Xo8Nqik+xaTExM0jP5xuzv74/Y2FgtNiou6byKB4GBgZCbSPU92eUFKl9gkttZRXuKTvKma089JX7Xx9cPpZ6IUhUZ9SIBQxPImTsvqjZoiu+//w+uXLkC+Qxzh5I7d249d/fVq1fdQRyHySC85TPm+vXrDhvDHToWpz5v3ry4efOmS3wGPz8/mxjc4663KSIvkgAJuDMBSUdWNrqW7vS6s5yUjQTMTKBeh+6aE3IHM2fONDMG6m5iAnR4TWx8qk4C2SVw6NAhxGrf5hm/m12SbE8CjiUgObLzhBfC3LlzHTsQeycBNyVAh9dNDUOxSMAIBD766CNdzEq1GxhBXMpIAqYlIPGV9Tv1xMWLF3Hw4EHTcqDi5iVAh9e8tqfmJJBtAt9//z0KlSitpz3KdmfsgARIwKEEarbqAA8tnvStt95y6DjsnATckQAdXne0CmUiAQMQkAUYf/75J6rWb2IAaSkiCZBASFhuPNHgSez64QflM+/Q2iSQmgAd3tRE+JwESCBDBGTnJklXV6lOowzVZyUSIAHXE6jXsTvi797F9OnTXS8MJSABJxKgw+tE2ByKBFQisGrVKoSE5cGjZSuopBZ1IQGlCZSJqKHvijhv/nyl9aRyJJCaAB3e1ET4nARIIF0CMrP7x/Hjem5PS7LxdBuxAgmQgFsQqN+hBxevuYUlKIQzCdDhdSZtjkUCihBYsWIF7sXHo0q9xopoRDVIwDwEarbuoG8+MX78ePMoTU1NT4AOr+lvAQIggcwTWLBgAXy13QYltycLCZCAsQjkCA1DtUbN8ePu3fpOmsaSntKSQNYI0OHNGje2IgFTE5Dd1WSxmrePr6k5UHkSMCoB2XmNi9eMaj3KnRUCdHizQo1tSMDEBLZu3Yo7cXFMR2bie4CqG59A6WrRyFf4Uczn4jXjG5MaZIgAHd4MYWIlEiABC4E5c+bAy9sbFWvVt5ziIwmQgAEJNOjcE5cuXcLevXsNKD1FJoHMEaDDmzlerE0Cpifw0097UDa6FgKCc5ieBQGQgJEJyM5rXt4+3HnNyEak7BkmQIc3w6hYkQRI4JdffkFs7E19wQtpkAAJGJtAUEhORD3ZGj///AvitDAlFhJQmQAdXpWtS91IwM4EZsyYAQ9PT1Spy3RkdkbL7kjAJQTqd+qhZWqIx7vvvuuS8TkoCTiLAB1eZ5HmOCSgAIGdu3ZBFrsE5wpVQBuqQAIkULxCFRQqURqLFy8mDBJQmgAdXqXNS+VIwH4E9u3bh5s3biCyaSv7dcqeSIAEXE6gUZc+iImJwZYtW1wuCwUgAUcRoMPrKLLslwQUI/D+++9DthGuWr+pYppRHRIwN4GoZm3gHxiEiRMnmhsEtVeaAB1epc1L5UjAfgR27NyJ0hHVIbs0sZAACahDwC8gALWf6oyDv/2Gy5cvq6MYNSGBZATo8CaDwUMSIIG0CezZs0cPZ4jWZoJYSIAE1CPQoFNPJCYk4PXXX1dPOWpEAhoBOry8DUiABNIlIOEMnl5eqNrgyXTrsgIJkIDxCOQvWgzltPza69dvMJ7wlJgEMkCADm8GILEKCZidwA8//ogKNepC8naykAAJqEmgcbd+Wj7e2/joo4/UVJBamZoAHV5Tm5/Kk0D6BL799lvcvnUL1Vu0Tb8ya5AACRiWQAVtu/A8jxTGhx9+aFgdKDgJWCNAh9caGZ4nARLQCUyfPh2+/v6ozM0meEeQgNIEJAtLk+79cenSJezScm6zkIBKBOjwqmRN6kICdiaQmJiI/fv/D080bK47vXbunt2RAAm4GYFabTrCLyAQ48aNczPJKA4JZI8AHd7s8WNrElCawCeffIL4+Luo2aq90npSORIggb8JSD7eeh2749ChQzh9+jSxkIAyBOjwKmNKKkIC9icwf/585MydV8+/a//e2SMJkIA7EmjctS/ua4KNHj3aHcWjTCSQJQJ0eLOEjY1IQH0CkoD+5MmTqNm6Azw9+VahvsWpIQn8TSCsQDgim7TEtu3bcUtbsMpCAioQ4KeYClakDiTgAALvvPMOJIZXYvpYSIAEzEWgWZ9ncC8+HmPGjDGX4tRWWQJOc3jv3LmDndrWpLLyM157EVkrMqu0ceNGHD58+KEqck5SJKXe+tBWm2PHjmHLli24fft2iv727t2LH7Xcopa/v/76K8V1PiEBsxNYu24dHqtQGQWKPmZ2FNSfBExHoEipcigbVROrvv5a/+JrOgBUWDkCXm9oxdFaibPZt29fxMbG6k6vOKBNmzaFpEBJXsQJHTZsGHLlyqUnvvbXUiGVKVNGrzJ16lSsWrUKd+/excyZM1GzZk3kzJkTttqMGDEC//3vf/UX67Rp01C0aFE88sgjuHfvHnr37o1r167pgfkSnF+sWDHkz58/uTgpjm/evJn0XOQODg7WEnTH6X0lXVDwwMfHR+eXoG05qXLx9fWF/Mk9qnoJCAjQ711bem7XfspcuGAB2jw7HEXLlLdVlddIgAQUJRCaryC2rVqqv1/UqVPH7loGBgbi/v376b4f2X1gJ3coPoN8vsjEn8pFQt+CgoL0CUZX+Aze3t6QzzdrxdvaBXueX7p0KaKionRnVvp95plnsHv3bkRHR6cYRpzSCRMmoFKlSujUqRMGDBiAFi1a4Ny5c5AP4C+//FKPJVyyZAkWLlyoB9Rba3PkyBH8+eef+OKLL/QxSpUqBWkXGRmJEydOoFChQpg0aVKK8fmEBEjgbwLy2vDx80dk05ZEQgIkYFICZSJroFi5ipinLV4dO3asSSlQbVUIOMXhlbCCJk2aJDGrWrUqDh48mMLhlVnXM2fOoGLFino9mW2Vb39nz57F8ePH9fOWhTPSfp32c6utNmXLlsWcOXOSxpTZXJmRlXL06FHd4d2wYYP+jatx48b6WEmVtYPZs2frDrOcE1m6d++e/LJ+LN8k5FubykW+MUkcp8y2q1xkJlu+hYeEhKispq6b2NSWnvI6kdy71Vu1g6QoYiEBEjAvgVZPD8UHwwbon4kvv/yyXUF4eXnpk1i23o/sOqCLOpPPFvmMMYOeglh8N1f4DOnNKjvF4b1w4UIKQ4vRxblNXmRnF5kKlxvDUiRkQWJrz58/r4cvWM5L+ytXrui7wVhrIyEKlqlt6VtmekeOHKl3IbO/Eg8sM8niTH/88ceYN28e8uTJYxkCv/zyC06dOqU/f+yxx9CvX7+ka5YDuYHFeVC5WOwhPzupXCxfpvz8/FRWU9dNdLWl52uvvaZ9mYxHvXbdlGdBBUmABGwTqFSnIQo9XgpTtV9g5b3BnkXei+Szxdb7kT3Hc2Vf6b3vulI2e48tvpErSnohI07x1uRbXHLPW2ZmLc6oBUrqOnJe6sm3hNTXLO1Tn0/extKvOLSjRo3SY4gtIRRPP/005E++hUgRSDLb26NHD0sz3QlOeqIdiNNtKXLjyqzv9evXk2aNLddUexRGwltip1UuEpMtX54kDEb1EhoaipiYGKtqype/wiXLoFj5Slbr8AIJkIA5CMikR5tBwzHzxWd1h3fo0KF2Uzx37ty6b3D16lW79emOHYnPIJ8x4jOoXGQCMG/evPr6KFf4DPLFST7HrRWnZGmQmdPkWRDkODw8PIVMcuPLgqHkHrqlngBM3b5gwYKw1UY6/+233zB8+HA899xzaNnyQSyihEkkH6dIkSIpHNoUgvEJCZiIwNatW3VnuEHnXibSmqqSAAnYIlC1flMUKlEa02fMsFWN10jArQk4xeGtXbs21q9fr8+GSgoxSU1WpUoVHYw8lz/5ZiAL21avXq2f37ZtG2QmSv4iIiJw4MABfZtDmW1cs2aNvvjMVhvpU+KNXn/9ddStWzeFEaRvidGVIkm1v/vuO9SvXz9FHT4hATMSkEWjErcb3ayNGdWnziRAAmkQkFne9kNeRqyWrWjixIlp1OApEnB/Ah5a/IzDgzPFSR0/frzutMrUfpcuXdCx49/J7KdMmaIHc0s6MtnVSZxUCVWQeuPGjUPJkiV1imvXrsX06dMRFhampxeTD2ZxeK21mTVrFhYvXpwiJljaSmoz+VlBVqFL9gf5CVsWrQ0ZMsTmblJphTTIz8KWhXDub+qsSWi2kIaLFy9mDZSBWlkLaZAviZW1L6KNuvRBl5fsG6tnIDwUlQRIwAqBCb2ewtmjh3BYS+VpjwXbDGmwAtqgpy0hDbLGylUhDeLnWStOcXgtg9+4cUOP3RUotorE80gu3tRFNqyQUASJhUldrLVJXS/5c5ndFec6IwHzdHjNEcNrZodX0gWu0b5YTly1BfmLPJr8pcJjEiABEsCRvXvwTr+OetpQSQma3UKHN7sE3au9uzu8TglpsJgkR44cGcpqkJazK33Iyr+0nF25Zq2NXLNWZPYyI86utfY8TwKqEJDUcxu1XQwr1W5AZ1cVo1IPErAzgZJVIlC5biN89dVKPVOSnbtndyTgUAJOdXgdqgk7JwESyDIBCS26q/160qRH/yz3wYYkQALqE+g0/FU9N7tkOmIhASMRoMNrJGtRVhJwEIG5c+fpq7DLRNRw0AjslgRIQAUCBYo+hgZdemH3Tz/h559/VkEl6mASAnR4TWJoqkkC1gjIltvXr19D8z7PWqvC8yRAAiSQROCpZ4chICgYEvfPQgJGIUCH1yiWopwk4CACkyZPRmi+Aoho8iBXtYOGYrckQAIKEAjMkROdhr2q56//4IMPFNCIKpiBAB1eM1iZOpKAFQKbN2/GRW3r7+b9BsMrnewpVrrgaRIgARMSqN22s7YbY2W8995UfWctEyKgygYjQIfXYAajuCRgTwJjx45FUM5cqN2mkz27ZV8kQAKKE5DNKPqOe0ffer5bt26Ka0v1VCBAh1cFK1IHEsgCgR07duDUqVNopsXu+vr7Z6EHNiEBEjAzAdluuHm/Qdj7669YtmyZmVFQdwMQoMNrACNRRBJwBIFRo0YhIDgHGnTq6Yju2ScJkIAJCLQeOBThxYpj1CuvMLTBBPY2sop0eI1sPcpOAlkksH37dhw/cQLN+w6Cf2BQFnthMxIgAbMT8PbxxdNvT0P83Xi0b9/e7DiovxsToMPrxsahaCTgKAIvvfQSgkJyolHXPo4agv2SAAmYhEDR0uXR7vmXcPDgQfz73/82idZU02gE6PAazWKUlwSySeDLL7/E6dOn0erpIfALCMxmb2xOAiRAAsCTvZ9B6WrVMW3a+9i3bx+RkIDbEaDD63YmoUAk4FgCw4eP0PPuMnbXsZzZOwmYiYCnpyeemfg+AnKEoGOnTrh165aZ1KeuBiBAh9cARqKIJGAvAjNnzsTly3+i/dBRkNg7FhIgARKwF4GcefJh8JRZiL15Ey1atLBXt+yHBOxCgA6vXTCyExJwfwL37t3Dv997D0VKl0P15k+5v8CUkARIwHAESleLRpcXX8Phw4fx3HPPGU5+CqwuATq86tqWmpFACgJDhw5F3O3b6DbydUjSeBYSIAEScASBxt37oVabjli5ahW49bAjCLPPrBCgw5sVamxDAgYjcPLkSaxZuxYRjVugZNVIg0lPcUmABIxGoNeYf6HUE1GYPHkyVq5caTTxKa+CBOjwKmhUqkQCqQn06tULnp5e6DxiTOpLfE4CJEACdifg7eODoVM/RsFij2PoCy9g69atdh+DHZJAZgjQ4c0MLdYlAQMSWLJkCY4ePYo2g4YjrEC4ATWgyCRAAkYkIDs5vjR7AXLlzY9evXrj559/NqIalFkRAnR4FTEk1SCBtAjExcXh1TFj9FmWJt37p1WF50iABEjAYQQkc8OoT5YgMGcudOjQEXv37nXYWOyYBGwRoMNriw6vkYDBCfTu3Rvi9PZ9fZKWhszH4NpQfBIgASMSyBNeGK98shT+OXKgdes2nOk1ohEVkJkOrwJGpAokkBaB9evXY8eOHWjctS8er/REWlV4jgRIgAScQiB/0WIY/elyBIeGoX37DozpdQp1DpKcAB3e5DR4TAKKELipJX5/fsgQ5AkvpO1xP1IRragGCZCAkQmI0ztm/gqE5MmLHj17YunSpUZWh7IbjAAdXoMZjOKSQEYIdO7cWc+5O3DiB/ALCMhIE9YhARIgAYcTkPCGsZ+vRIGij6FPnz54T9sMh4UEnEGADq8zKHMMEnAiAdk+WBaGtH5mGIpXqOLEkTkUCZAACaRPQBayjZ77JUpUicBrr72GkSP5K1T61FgjuwTo8GaXINuTgBsROHDgAN6ZNAklKldDqwHPu5FkFIUESIAEHhAIzBGCFz/8ApFNW2HhwoVo167dg4s8IgEHEKDD6wCo7JIEXEFAsjF06NAB/kHBeHbSTHh6eblCDI5JAiRAAhki4O3ji4H/eh8ttS/nP/74IyIiInDt2rUMtWUlEsgsATq8mSXG+iTgpgRatGiBGzduYPDkWQjNl99NpaRYJEACJPCAgIeHB9o99xIGvPUeLly8iGrVqmH//v0PKvCIBOxEgA6vnUCyGxJwJYEhWkaG3377DZ2Gv4oykTVcKQrHJgESIIFME6jRsp22QcVSePj4oWXLv8McMt0JG5CADQIe97Vi4zov/UNA0jwlL8HBwXpC/3v37iU/rdyxj7ZZQWJiIhISEpTTLblCvr6+EF1jY2OTnzbE8bvvvovxb76Jmq3ao//4dw0hM4UkARIggbQI/HXxPD4YNgCnDx9Ely5dMGfOnLSqGeqczGLLZ8ydO3cMJXdmhfX09ERgYCBu3bql+w2ZbZ/d+uKrhISEWO2GDq9VNCkvXLp0KemE3Lx58+bVY41Uv4EDtJRW4uzevXs3SX8VD4KCgiC6Xr582VDqLVu2DC+88AJKVo3CiFmfczc1Q1mPwpIACaRFIP5OHOa9NRo/rFuJEiVKYMOGDZBJJqMWcQTlM0ZCzlQuXtq6kdy5cyMmJgbx8fFOV1UmrUJDQ62O6231Ci+kIJB8hlNuXilmmPm06Jhc/xRgFHkiekoxkp7ffvsthg0bjvDiJTFk6hw6u4rci1SDBMxOwMfPH09PmIrHylfG4ilvomKlSlimbVJRpYox0yzKD+mWz1KVbSuTgVJcpau3t22XljG8Kt991E1ZAlu2bMGAAU/rO6m9NHsBAoJzKKsrFSMBEjAngYZdeuPlT5bAU8vm0LpNG0iOcRYSyCoBOrxZJcd2JOAiAjKz27t3H4TmL6B9GCxFSFgeF0nCYUmABEjAsQRKaptTvLlsg76Jzttvv42OHTtC9bUzjiVq3t7p8JrX9tTcgAQkZrf/gAHIXfARvPLZl0w/ZkAbUmQSIIHMEZCd2V7+eAma9XkWO3fuRKXKlXH06NHMdcLapidAh9f0twABGIXAlClTMHzECDyixeyOnreCzq5RDEc5SYAEsk1ANtLp+MIrGPr+p4jTFlE3bNgIn376abb7ZQfmIUCH1zy2pqYGJtCrVy9MnToV5aJraTO7y7UwhtwG1oaikwAJkEDWCFSu01ALcdiIwqXK4rXXXtN3l2SIQ9ZYmq0VHV6zWZz6GoqApMOTnYc2b96MRt36YtgHc+EfGGQoHSgsCZAACdiTgIR0var9ytWs9zPYtWsXyleogL1799pzCPalIAE6vAoalSqpQWDFihWIjIrCJS03sGy72W3k65Cf9VhIgARIwOwEvLQUVB2HjdbzjyfCE61at8Zbb71ldizU3wYBOrw24PASCbiCQFxcHDp37oyhQ4cid3ghvL5oLWTbTRYSIAESIIGUBMpXr4MJX21C2aha+PDDD1GrVi0k3ygqZW0+MzMBOrxmtj51dzsCS5YsQbny5bF9xw406toXbyxepy9ScztBKRAJkAAJuAkBSc04YuZ8dB81HqdOn0ZkZBQ+//xzN5GOYrgLATq87mIJymFqAocOHdJnJkZoWRhy5iuIV7WUY11HjoPsOMRCAiRAAiSQPgHZqOKNJd+gQLHieOWVV9CiRQtcv349/YasYQoCdHhNYWYq6a4Ezp8/j3bt2qFRo0Y4ox13Gv6qnmT98cpPuKvIlIsESIAE3JZA+GMlMPaLr9F64FD8um+fnrNX1kOwkAAdXt4DJOACAsePH0f79u0RERmJn/bsQb2OPTB57Q482WsgvH18XCARhyQBEiABNQjIe+hTg0bgNc3xDS3wCIZo6yFatWrF2V41zJtlLejwZhkdG5JA5gls2rQJ9erVQ63atXVHt3abTnhn9Vb0eOVN5tbNPE62IAESIAGrBB4tW0H7xWw9WvQbjP9qactkhzbG9lrFpfwFj/taUV5LOygoPz1biqenJ/Lnz4+YmBjIinqVS2BgoL5v+V1tZxuVS3BwMIKCgnDx4kW7qyn3yaRJk7Dq669x/do1BGhj1e/YU1+UlitvPruPxw5JgARIgARSEjj52wF8Ou5FnDl2GKVLl8aiRYtQoECBlJUc9Ex8BvmMUT2e2FtLFZc3b15cuXIFrvAZ/Pz8EBYWZtWK3lav8AIJkECWCdy+fRsfffQRli5ditOnzyAxMQHFylVE++E9ENW0NXz9uRgty3DZkARIgAQySaBomfJ4Xct6s+HzOVj14VQ9x/nAp5/G2LFjM9kTqxuVAGd4M2g5zvByhje9W+WytkGEOLlr167F6TOak5uQgNB8BVC9RVs9j64spmAhARIgARJwLYFLp09i/luj8dueXfqM5CeffIKIiAiHCcUZXoehTdExZ3hT4OATErAvAdnyd+HChdi2bZse3iIRQmEFwtG4Wz9Ua9QMxStWte+A7I0ESIAESCBbBPIVLoqRcxbhh29WYfHk8XiqbVvU1jasEMdXQg9Y1CTAkAY17UqtHERg//79euzX9u3bcebMWcTH34WHFp8ljm3lOg1RSft7pHhJB43ObkmABEiABOxFoHrzp1Cpdn18NeNdbFm+AOXLV8Dzzz+Hl156yV5DsB83IsCQhgwagyEN5gxp2Kut7F22bBl27typhync+WeRYp5HCqNCjbooV702ykTU0Bai5cjgncRqJEACJEAC7kbg1OH/4Yt/jcXv+/cid+7ceO+999C4cWO7iMmQBrtgTLeT9EIa6PCmi/DvCnR41Xd4vby88Nlnn0FSh4mje0HL2BD/T3aKsPwF9b3aS1WLRpnIGpDnLCRAAiRAAmoR+P/2zgS6qurq4xuQMIQQkpCQMCMQBGWeq1TpACoOi9VqcUT5FFosVaiiLIdqaemya1XUSMUqDlBxWFCQSSgiAoICgiACMhgCgRAICUMIIRB43/lvvbeXR+7NyyN5yXv5n7WSd+5wzj3nt9+5b99999nnS+Pm8MHkSXI8N0c6pKbK1KlTpUOHDpfUSSq8l4Qv4MJUeANG5X0iFd7IUngxwWz58uXy+eefy7fffqvuCScLTorv/HmRGjWkmZlgltqjj7Tv1ktSe/algus9PHiUBEiABCKGwBnzJm/JjNdl4ZtTpNgYPXr16iVTpkyRZs2aBdVHKrxBYStzISq8ZUZWcgEqvOGh8CL2X05OjkBe+02khMzMTNm3b5/ms7KyJMcougUFBbblFtJGJIU2V3U1YcO6yuVXdZPWJnxYvWhOXCh5JHAvCZAACVQPAifyjsj819PUvxc97t+vn7o6tGjRokwAqPCWCVfQJ1PhDRrdhQWp8Jafwnvy5ElZt26dYALY7t275cCBAxqoOj8/X04XFelCF+dMSK/zxtqKqAc//F0ojx+2flwzxXz4fOflPNZQcVlHpXZUHYkzbgjJrdpIk5ZtJKVNW2lqJpc1b5cq9WNiS6qc+0iABEiABEhAjmTtl4XTpsiquR8qjS5dOsvEiROlZ8+eAdGhwhsQpks+iQrvJSP8oQIqvMEpvOvXr5clS5bIhg0bZM+ePXL8+AkpKrpwdTpM+IpNSNSldaNj43Qlsqi69QRKKtZEr2lWb6lh3Azw50y6bfbVrFlLaplzLouKkqg6daVOvfo6iSw6tpHExMVr3fVjGjqLMk8CJEACJEACZSKQl50li42rw4pZMzVCT5OkJLnvvvvkoYceEqwy5pao8LqRKd/9VHjLiScV3tIVXrgPzJkzR2PS7tixQ46ZZXTPFRerBKCEtkjtqH8pbdoZK2trSWzeUhJMzNraRkllIgESIAESIIFwIHAq/7hae5d9MF2OHMhUw0ynjh3l7rvvlmHDhl2k/FLhDY1UqfCWE2cqvBcqvKdOnZKPP/5YsPDC5s2b5WB2tlghu6DAYhndtp27G3/YrtK6U2dJNGG8mEiABEiABEggUgjA3W7nxnWyZsFsWb90kZw2E5/xtjG5SRNduW3QoEGCPyxmgb8TJ05EStdL7Aes3ImJieqiiPk0oU5UeMuJeHVWeOGKsGjRIlm7dq1s27ZNssyEsEKj8GKwI6JBsvGJbWeiGbTt0l2VXPjG4omWiQRIgARIgASqA4His2dlx4a1smX1Z7L1i5VyIH2XzimB610d46IX2zBG4/tCIWzcuLE0atRIlWAoaTgHc1agJJ42ESJgUMLkavwhjz/sLzJzXPBXbN6cOue5oDz+oHDWNm6AqDM6OlpiYn64ZnJysmCiXfv27aVz584SG1sx81ao8EbINz3SFd69e/dq7FmE6Nq5c6dg+/DhHCk4VSAYyJrMgIKlFtEMWnfsbKy3naWV+WREgwj5krMbJEACJEAC5ULgVP4Jydi2RbCgxcE9u+XQvgzJNZPfTuTlmihBRQFdA6t4Yl4K3prWqVdP/5DX+S1mzkpNEzsexiUYn86bid4IoXbGzJEpMgpyYUG+FJ7MF4RY80+wQtc19TQ0SjiU4bZt25pV5q6SfiYKRZcuXfxPD3ibCm/AqKr2ieGq8OIVCvxpocTCUgs/22zjfoDQXceNj22hGQxnzBMjni6thEHUuGlzaWp8beFvm3J5OxOXNlWjGmDQMZEACZAACZAACQRHAArvaWO9PXO6UOe5QGGFcovFjzBRG0ptVN26Jh8V3AUcpWCwyj+aZxTtHDmWc1iOHs6WvINZkntwvxzKzJDDmXvl5LGjdgm0o575nY8zFmhYha+44gqNRgFluLQ4xFR4f8QIM/xXX32lZvfevXur2d0m7MhgQQDM6G/duvVFq5tAcYPlsUePHvpKwCoWTJlA22Ndo6opvOjz9u3bVZHNyMjQOLOHDx9W35kTCO9lFNmzZ86a1x4/TBqz+oHP6IaxGns2PqXZD5/JKWbyWDNJMNuw4DYycWnpkuAkxjwJkAAJkAAJRCaB0+ZNLhTfQ/v2SPbedMnOSJes9N2aLzLHrARlvL5RhuGS0bJlS9XRYBnu06ePKsdUeA2pwsJCGTFihFx55ZWC4P/wL8E61fA5cSYs5/rMM8+okzcmQ91nwn0MHTpUT5k8ebJs3bpVfVDWrFkjaWlpCjyYMoG2x9m2UCq8COW1YsUK41JwWBdQOGSWuD169Kggfi3i1EKRPX/+nLN5+nQY0yhOGiU20XizjRonSWxikuinyVvbCP+F1xlMJEACJEACJEACJOBFAItvQAHO3rtHDkEZ/lEhzjHRKYrP/m9iGt4M14GbREwDiY+Pl6ZNm0qqWZr58ccflyjjfhGKVCUmrb399tty7NgxeeSRR7TPo0aNkvvvv1/9RZwQhg8fLuPGjZOuXbsKlLwHHnhAZs+erUoy9s+aNUstj++//76+np8wYYIEUybQ9jjbFkqFt5Wxbp/9cYZjrctqa3zaRklGkU1MNgqtnyJrFFzsi4lLuOgBwtl+5kmABEiABEiABEigPAjADQMuEjn79wqUXyzOAR/lI1mZuo1j54rPqhHzt7/9bXlcstQ6SlN4Q2Lqw2paCM1hJbgkYLY/fEKshFmHWArWcphuYsJ61K9fX1fhgu8p9luv2VF+4cKFOlOxrGVwvUDas2XLFrVM43y0IyUlBVlNlmUa5vuKeHKB0Jq37yhD/m+0WQUMCyZcaAn/sRn6Acd4/JleOXczTwIkQAIkQAIkQAIVTgDukPiTnn3ta+UdOihvPDVW5wdVhJ5kX8iRsXREx64LsiFReDFJqmHD/610hTwUVWfC63uE0bCUSRxD6Iy8vDx9re8Mo4Hyubm5+sq/rGVQbyDtGT9+vKSnp+N09VOZN2+e5p3/EPKjIlJdo/Du2bpZXhk3qiKqZ50kQAIkQAIkQAIkUOEEmjdvruHYKvxC5gII3+aVQqLwYuYhYsZZCdZczAJ0Jv9zcAzn1TUzFf2PWeX99wdSBuf4l7PqwzErTZ06VePdYRttQFQDK0Eph9M2IiBg8lt5p6VLl8qmTZvKu9oS64OrCeIBuiU8mSGCAxhFcoJVHX3NNxP+wiGVJjevPuCNRWk3Bq/y4XIMD6SIa1kRYzRYBpciN7dr4k0TLBuVEejdrU0VsR/xRfHdxX1XY4BXxEVc6kREGxhanAYZl1PLZXd1GaMwWOH3BfNqKiLhu4Jr4De/MhO+N/iNwWTySE7gjAU2ED8YOgO+xwMHDrxAf6rI/uNeiGu6pZAovFAOYam1EvIId+FMCQkJCgk/TvhiIOE8OD7DIvvNN9/Yp2M/XAyCKYNKAmlPq1at7Osh4+/Di31Q4itCEYQ7x+DBg3GJSk/48qCPkf5jikGKGyN8xyM9xcXF6STISO8nxhFuvJjsGckJD2q40Uf6QwwMD/ju4vcg1ApvqL8/1WWM4jccv6N4EIzkhAfS6rrSWkXoSG7fldIebEKyHNaAAQN0GVo83SCcFqIsdO/eXduMbfzhht23b1+xXAdWrlypNzcMfIQxw4IIiCELePPnz9cwGMGUwUW92uMGkvtJgARIgARIgARIgATCk0BILLy/+MUvZPXq1XLHHXfoq7dhw4ZJmzZtlNhbb72lMXkRwWH06NEC39k5c+boeQhRhoRXSSNHjtSoDQh3AevrnXfeqceCKePVHq2U/0iABEiABEiABEiABCKGQA3zasgXqt7APxK+u7DMeiU3P7ezZsUQuDzg1YB/CqZMoO3BtfxdGvC6FLFxI90nhy4N/t+08N+uLq9L6dIQ/t9VZw/o0uCkERl5ujREhhytXkC3S0xM1KACleEGCXdYGEXdkrfm6VYqyP2BRjVwm0SFSQv4KykFUybQ9pR0Pe4jARIgARIgARIgARIIDwIh8eENDxRsJQmQAAmQAAmQAAmQQCQSoMIbiVJln0iABEiABEiABEiABGwCVHhtFMyQAAmQAAmQAAmQAAlEIgEqvJEoVfaJBEiABEiABEiABEjAJkCF10bBDAmQAAmQAAmQAAmQQCQSoMIbiVJln0iABEiABEiABEiABGwCVHhtFMyQAAmQAAmQAAmQAAlEIgEqvJEoVfaJBEiABEiABEiABEjAJkCF10bBDAmQAAmQAAmQAAmQQCQSoMIbiVJln0iABEiABEiABEiABGwCNXwm2VvMuBI4ffq0fezcuXOyf/9+ady4sURHR9v7IzFTs2ZNwVck0r8mx44dk4KCAmnWrFkkivGCPtWqVUvwHY70dODAAR2fbsuOR0r/a9SoIfg7f/58pHSpxH5gfB45ckSaN28u+A5HcqouY/Tw4cOC3xj8lkZ6qg4yPXPmjBw8eFCaNGkidevWDblI8V2KiopyvS4VXlc07gfy8vKkf//+MnnyZLnxxhvdT+SRsCGQlpYm7777rnz55Zdh02Y21JtAv3795K677pIxY8Z4n8ijYUFg4cKFMm7cOPniiy8kPj4+LNrMRnoTuPPOOyUpKUlefPFF7xN5NCwI7NixQ2655Rb597//Lb17965ybaZLQ5UTCRtEAiRAAiRAAiRAAiRQngSo8JYnTdZFAiRAAiRAAiRAAiRQ5QjUetakKteqMGhQnTp1pG/fvny1FgayCqSJ8P1p0aKFdO/ePZDTeU4YEIBMe/bsWS38ssNAHJfcRMgzMTFR+vTpI5dddtkl18cKKp8A/Fo7deokbdu2rfzGsAWXTABjtEGDBjpGY2JiLrm+8q6APrzlTZT1kQAJkAAJkAAJkAAJVCkCdGmoUuJgY0iABEiABEiABEiABMqbAF0agiCK0DirVq3S0E7VIZxKEIiqXJGTJ0/Kxo0bNZwcQsrhD+GNkIqKimTt2rW6Lzk5+YKQR5R1lROlNmjp0qXSqlUrDWlktRAzhCHj2NhYqV+/vrXbU75uZezCzISEwHfffScYo84Qcl9//bXs3bvXHrOQab169bQ9buPSayyHpCO8iJw9e1bWr18vubm5Gp4KIfOsFIzc3MpYdfKz4gkcPXpUVq9erePP6aqwb98+2b59uz1GCwsLJSEhQRvkJbfKuu9S4S3jdwU34UceeURvzK+99prGmuvYsWMZa+HpoSaAwYrQN4j7iB9X/P3yl78UDND7779fY/DinOXLl8vgwYM1rillHWopBXa92bNny/PPPy/33HOP7cuJEIFz584VxIGcMmWKXH311ar4esnXrUxgreBZ5UVgz549ek+9/PLLpX379lptcXGxDB8+XI4fP26P1zZt2qgC5TYuvWRdXm1lPd4EsrOzZcSIEXr/3Lp1q+A38qabbtJxGozc3Mp4t4JHy5PARx99JP/4xz/0fvree+/J7t27BSEfkbD/888/l4yMDB2nGLddunQRL7lV6n0XC08wBU7g3nvv9W3atEkLmMHtM4PZZ6wKgVfAMyuFgLnx+t5+++2Lrv3WW2/5zAC0948cOdJn4nzqNmVtY6kSGWM58k2YMMEHGV1zzTU+sxiMtssoTL6hQ4f6zGIaum1uyr5JkyZp3k2+XmWqRGerSSPMQ4rv1ltv9d1xxx2+RYsW2b3etWuXzyi89rYz4zYu3WTtLMt8xRJ4+eWXfdOmTbMv8vTTT/vmz5+v28HIza2MfQFmKpSAUWB9v/rVr3zp6el6HbP4i2/IkCE+sxaBbg8bNsxn3sJc1AY3uVX2fZc+vGV4FMLTC16F4wkGCauJ4DUbVnRiqtoEzA+ozh6dOXOmvm4zI1QbjKfVHj162I1Hftu2bUJZ20iqTAarw8GyAAuuM5mbsY5JzBBGsmSIvJt8vcqgHFNoCMBFwShIatl1vvrGeIXL0eLFiwUWplOnTmmDvMalm6xD0xNeBQRGjRqlb14sGrDQw/IejNy8ylj187NiCSCKxjvvvCN4u4KEFWfheoR7McYkFuHKycnRhSagGyF5ya2y77tUeFVEgf3D63AsJey8McNfEEJnqtoE8AOKVdQQTm769Ony2GOPaYPxCq5hw4Z245GH7xllbSOpMhnIDqv4+IekwlKWGIdWsmSIbTf5epWx6uFnxRMYNGiQ7fNnPYTiqjt37hT4+eXn5+vn7bffrssKe41LN1lXfC94BYsAlnWtXbu2bn766adqILrhhhs876ducvOStXU9flY8Aeg8SFi6/KWXXpLrr79el4L+/vvvdX4E/LWhAMPVE6shesmtsu+7DGZYhu9LSWth42mmMtaMLkOzeaoh8Oabb6rfNayAN998s5jXqHoz9pcp5Amrk/9+QKSsq+ZXyV9WlgzRWrdjbvurZg+rX6sefPBBwZ81+RCT0WDthd89flydyRqXlKmTSuXm582bp1a/F154Qd+smVfhZZabvzzRI0vWldu76nd1jL8///nPgofSJ598UgFg7tKcOXMkLi5Ot9u1a6e/s7169SqzrENFlBbeMpDG7EMMXAjfSrDuNm3a1NrkZxUkgIlMmO1tvfKGFQLuKLAsIMqG00JvyZOyroKCdGkSFiPwl2FKSoqe7SZfrzIul+HuEBKAm5jzPtuyZUuBdchrXLrJOoTN5qUMgRkzZsiHH34oaWlpGkkFUIKRm1cZgg4dAbguPProo4LoDBMnThT8fiIdO3ZMEL3BShijhw4dUgXYTU+q7PsuFV5LWgF84lUqVlfD0yvSypUrVbjWE04AVfCUSiCAV2ywNMClAQlhVOC20K1bNxkwYIB8/PHH6puEMCpr1qzR1dYo60oQVJCX7N27t3z77beSmZmpFiAzSUZX+kF1bvL1KhNkM1isHAng3jp16lStET+4eD0+cOBAdWdxuwe7ybocm8WqSiFgJh7KsmXL5NVXX1WjgnW61/3UTW5eZax6+VnxBP70pz9Jhw4d5IknnrggZCf8s8eOHas+2rD8LliwQK699lpViN3GaGXfd7nSWhm/L7AUjh8/XgUPi+EzzzwjqampZayFp4eawIYNG+Rf//qXxoiEkz0GL0JX4RXZc889pwoT5Glmncptt92mzaOsQy2lwK+HH8lPPvlEfbJRCjdbWJTi4+PVqvSXv/xFlSMv+bqVCbwVPLO8COBHFRMS4e+JdOLECQ09l5WVpZNi4MowZswYfUvjNi69ZF1e7WQ93gR+/etfq5XPOc/FzPKXhx9+WN+ylfTb6SU3N1l7t4JHy4sAjEMmKo5W55TpK6+8ohOFMaEN92HIEHMnYAFOSkpylTUqqsz7LhXeIL8ZMOc7g6QHWQ2LhZgAnkoxMJ2DF03A5Bj47sKq4J8oa38iVXMbAe/xGhxrufsnN/l6lfGvg9uhJwDrLnw5MWHRP7mNSzdZ+5fnduUQCEZubmUqpwe8qpMAJrMhcgN+V/2Tm9wq675LhddfQtwmARIgARIgARIgARKIKAL04Y0ocbIzJEACJEACJEACJEAC/gSo8PoT4TYJkAAJkAAJkAAJkEBEEaDCG1HiZGdIgARIgARIgARIgAT8CVDh9SfCbRIgARIgARIgARIggYgiQIU3osTJzpAACZAACZAACZAACfgToMLrT4TbJEACJEACJEACJEACEUWACm9EiZOdIQESIAESIAESIAES8CdAhdefCLdJgARIgARIgARIgAQiigAV3ogSJztDAiRQ3gSwKtCDDz4o6enpuiT14MGD5fe//70cPHhQL1VYWKjHv//+e/vS+/fv131YaQgJS3H+97//lddee01uueUW+d3vfqf1ZWRkyEMPPaTLWc+aNcsuH0gG7cHS5jfeeKNgad5Vq1bptlUWy/M+9dRTcv3118s999wjy5Ytsw7pEttefbJOnD59utx+++3a5smTJ+sSotYx9O3JJ5+UQYMGyW9+8xt54403xOfzWYdL/fRqH5aGnjRpkl0Hzh09erQsXrzYbvuWLVtUDjfffLO8+OKLutqTXYAZEiABEvAjwJXW/IBwkwRIgAScBE6fPq3LTnfo0EH69+8vvXr1UgUWy1BD6cJy1VhmfPXq1fKTn/xEi27evFm6desmBw4ckKZNmwqUsk2bNknXrl1VQXz99dcFS3LWrl1bbrjhBl3SGgreunXrpGfPns7Ll5jPzc3VdiQnJ8vw4cPl/fff1/pTUlJk+/btUlBQoNfHcp8PPPCA1vvee+8Jrgvlt7Q+4aIPP/ywzJw5U0aOHCkxMTHy0ksvSd++fWXu3Lnapptuukny8vL0+JEjR+Svf/2rjB8/XiZMmFBim507S2vfihUr5Gc/+5ngIWDo0KFy7733yldffaV/NWvWVHk0adJEhgwZIv369ZPnn39e2eJ8/2XDnddlngRIoBoTME/kTCRAAiRAAi4EjAUXZkvf008/bZ+xdOlS3ZeVleUzlk7NG4XXPm6UW91nFF7dZ5RDn1GYfcZarNv/+c9/9PgLL7xglxkwYMAF17APlJB54oknfElJST6jNNtHO3Xq5Lviiit02yjPvujoaG2bdQL2GSVRy5TWpx07dviMYukzCq9V3Ldr1y5t82effab74uLifFOnTrWPz58/32eUYXvbK1Na+1DWKM8+o8D7jFXcV7duXZ95iNAqrbYbRV+38W/Dhg3atuXLl9v7mCEBEiABJ4HLqrGuz66TAAmQQMAE+vTpY5/bsmVLzcNSWb9+fXu/VwbWXViFkdq2baufcDewUmJiomRnZ1ubnp+wdsIy7LRmwuI6b948LQdLMay/f/vb3+x6YG0+dOiQwN0C10Jy6xPqNz8Usn79eoG12koNGjRQK+u1116rlmW4GcyYMUPdKm699Va58sorrVM9P0trX4sWLWTixIliHixk1KhR8s9//lO6dOlyQZ3ov5W6d++ufdq4caNcd9111m5+kgAJkIBNgD68NgpmSIAESMCdgLGY2gfxWh0JSqGVnHn4/fqnhIQE/10Cl4NgElwaoBQ6E9wqrHT06FFVxNFO6w/nw93AajvOdesT/HOhnNepU8cuj3JjxoyxlVr49H700UcCV4+XX35ZrrrqKvVxttrg9RlI+3B9KNhIJfFs1aqVfQko/uj/yZMn7X3MkAAJkICTAC28ThrMkwAJkEAZCURFRWkJp7KFyWgVmWBJhTXTmZYsWWJvtmvXTifJYfKXpeBiUh38jGHdhf+wV0J5KJnwPbb8ks+dOyfvvPOOpKamyqlTp+SDDz4QWJXxh/pgkYUf77PPPivGBcGreimtfSj897//Xb777jtJS0uTxx57TH7+85/byjaOf/rpp+q/i/y+ffvEuFxIjx49sMlEAiRAAhcRoIX3IiTcQQIkQAKBE6hXr540b95cpk2bJvn5+Rp9AYpmRaaxY8cKFNw//vGP8s0332iUBiizVoIbAFwXnnvuOYE1FREl7rrrLlmwYIFYCrp1bkmfAwcOVMstokBs3bpVJ7lBkX388cfVKo0+w83A+BILrMGIVJGTkyPNmjUrVdnF9Upr39dff60RJ6ZMmaKRGOC+cPfdd8uZM2fs5iKCBFwj4KZh/KtVif7pT39qH2eGBEiABJwEqPA6aTBPAiRAAkEQePXVVzXsl5nIpdETHn300SBqCbwILJnw14WV85prrpG1a9dqNAbLstq7d2959913xUwqE0RuMJPZVBmFtTSQhOgRcFeAJbdz587SuHFj7R/8dZGHCwFCrcFqDCU3Pj5ePvnkE5k9e3Yg1YtX+6A8QzmHT/Btt92m9YEvrLgIs2Yl1HHdddcJXBuglC9cuDBoFxGrTn6SAAlELgGGJYtc2bJnJEACISSA1/qYGAYF0HIjqKjLYyIZ/GuhyFrpD3/4g5joCmr5tfbhE22CG0Mgll1nOSuPsGvFxcVSkg8yzoErB5RUayKcVS7Qz7K2zwqpBmX/6quv1rBwwV470DbyPBIggfAnQAtv+MuQPSABEqgCBKDkYmLYpSq78JWFgun2B8Uai0wgJjDcGKBwwlUBCz9gAQj/BAU8WGUXdcXGxroquziOiWVOhRPtc2u7tR/lrHQp7UO/nNe26uQnCZAACfgT4KQ1fyLcJgESIIFKJICFJ0x8X9cWYCEG+LZmZmbqog+Y2IWJZJgwNmLECNdyoToAX1+4UrilWrVq2avUuZ3jtR/uFFByL0WJ96qfx0iABCKTAF0aIlOu7BUJkEA1IVBUVKTuDdWku+wmCZAACQRFgApvUNhYiARIgARIgARIgARIIFwI0Ic3XCTFdpIACZAACZAACZAACQRFgApvUNhYiARIgARIgARIgARIIFwIUOENF0mxnSRAAiRAAiRAAiRAAkERoMIbFDYWIgESIAESIAESIAESCBcCVHjDRVJsJwmQAAmQAAmQAAmQQFAE/h/CxrTCyKb84wAAAABJRU5ErkJggg==" /></p>
<!-- rnb-plot-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>This distribution makes it seem like maybe any cells expressing less than 750 genes might be a good cutoff?</p>
<p>But as you may have noticed in our <code>sc_metadata</code> we have cell-type information for these data. Let’s use the power of our single-cell resolution data to our advantage and look at this data with cell-type labels.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBMZXQncyBtYXRjaCB0aGlzIHRvIHRoZSBtZXRhZGF0YSBzbyB3ZSBjYW4gcGxvdCBzYW1wbGVzIGJ5IHRoZWlyIGNlbGwgdHlwZVxubnVtX2dlbmVzX2V4cF9kZiA8LSBkYXRhLmZyYW1lKCdnZW9fYWNjZXNzaW9uJyA9IGNvbG5hbWVzKGRldGVjdGlvbl9tYXQpLCBcbiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICBudW1fZ2VuZXNfZXhwKSAlPiUgXG4gIGRwbHlyOjppbm5lcl9qb2luKHNjX21ldGFkYXRhLCBieSA9ICdnZW9fYWNjZXNzaW9uJylcbmBgYCJ9 -->
<pre class="r"><code># Let&#39;s match this to the metadata so we can plot samples by their cell type
num_genes_exp_df &lt;- data.frame(&#39;geo_accession&#39; = colnames(detection_mat), 
                               num_genes_exp) %&gt;% 
  dplyr::inner_join(sc_metadata, by = &#39;geo_accession&#39;)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiQ29sdW1uIGBnZW9fYWNjZXNzaW9uYCBqb2luaW5nIGZhY3RvciBhbmQgY2hhcmFjdGVyIHZlY3RvciwgY29lcmNpbmcgaW50byBjaGFyYWN0ZXIgdmVjdG9yXG4ifQ== -->
<pre><code>Column `geo_accession` joining factor and character vector, coercing into character vector</code></pre>
<!-- rnb-output-end -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBQbG90IHRoZXNlIGRhdGFcbmdncGxvdChudW1fZ2VuZXNfZXhwX2RmLCBhZXMoeCA9IG51bV9nZW5lc19leHApKSArIFxuICBnZW9tX2RlbnNpdHkoYWVzKGNvbG9yID0gY2VsbC50eXBlLmNoMSkpICtcbiAgZ2VvbV92bGluZSh4aW50ZXJjZXB0ID0gMjAwLCBjb2xvciA9IFwicmVkXCIpXG5gYGAifQ== -->
<pre class="r"><code># Plot these data
ggplot(num_genes_exp_df, aes(x = num_genes_exp)) + 
  geom_density(aes(color = cell.type.ch1)) +
  geom_vline(xintercept = 200, color = &quot;red&quot;)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-plot-begin eyJjb25kaXRpb25zIjpbXSwiaGVpZ2h0Ijo0MzIuNjMyOSwic2l6ZV9iZWhhdmlvciI6MCwid2lkdGgiOjcwMH0= -->
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAArwAAAGwCAYAAABLkLalAAAEGWlDQ1BrQ0dDb2xvclNwYWNlR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19zn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7J0HYFRV1sf/M5NOOgkhAULvRYpUFQuiKJbPhljWrtjLFnVX17auu+q62FZ3XctaUMSCiFixURQEROm9hg7pPVO+e27ywiSZTGaSqcn/QPLarb938+bMeeeea3IoAYUESIAESIAESIAESIAEWikBcyvtF7tFAiRAAiRAAiRAAiRAApoAFV4OBBIgARIgARIgARIggVZNgApvq7697BwJkAAJkAAJkAAJkAAVXo4BEiABEiABEiABEiCBVk2ACm+rvr3sHAmQAAmQAAmQAAmQQAQRHCVgtVpht9uPngilvaIioLgEyOwYSq1iW0iABEiABEigxQTMZjMiIqiStBgkC2iUAEeXE5rCwkJUVFToMykpKTCZTMjNzXVKEbzduOnPIvalV3Bk/a+NNkIeGAkJCSgoKGg0TahdkAdceno6Dh8+jKqqqlBrXqPtSUxMRHFxceh+QXLR8lAb0y6a2OBUOI7pyMhIpKWlhd2YTkpKQpH6Yh2yX/objA5AxrRIXl6ei6uheYpj2vV9iYmJqb2frlPwLAm0jABdGlrGj7lJgARIgARIgARIgARCnAAV3hC/QWweCZAACZAACZAACZBAywhQ4W0ZP+YmARIgARIgARIgARIIcQJUeEP8BrF5JEACJEACJEACJEACLSNAhbdl/JibBEiABEiABEiABEggxAlQ4Q3xG8TmkQAJkAAJkAAJkAAJtIwAFd6W8WNuEiABEiABEiABEiCBECdAhTfEbxCbRwIkQAIkQAIkQAIk0DICVHhbxo+5SYAESIAESIAESIAEQpwAFd4Qv0FsHgmQAAmQAAmQAAmQQMsIBGxpYVmyd/ny5Xq53pEjR0KW33QlssTsihUr0K1bN/Tt27dOko0bN2Lnzp0YPny4XrrTuCjLSkqegQMHIjMz0ziNXbt2Ye/evbXH7du3R+/evWuPuUMCJEACJEACJEACJND6CQRE4S0rK8M111yjFVJRQN977z3885//1MqvM+KVK1figQcewGmnnYYXXngBV111Fc477zydZPr06Vi7dq1WWP/1r3/hueeeQ3Z2NubMmYP3338f48eP19t+/frhzjvv1HlefvllHDhwAMnJyfp4yJAhVHidgXOfBEiABEiABEiABNoAgYAovO+++y5Gjx5dq4hOmzYNS5cuxZgxY+ogfvrpp/Hoo4/imGOOwZQpU3Dddddh8uTJ2kq7cOFCrdCazWbMnDkTM2bMwN13340333wTTz75JLp3747LLrtM57vyyiuRkpKCzZs34/HHH9eKcZ2KeEACJEACJEACJEACJNBmCARE4d2yZYu22hpUxSVh3bp1dRReq9WKnJwciBVWJCMjA3FxcdizZw+2b9+uz4uyKyL5582bB4vFgtdffx3t2rXT58vLy1FcXAybzYbS0lLk5ubi0KFDWLBgAU466SR07txZpzN+VVZWQqzPhkgbTCaTcai39Y/rXAzoQXW73LXHuGZsA9q8ZlZmtFW2xn4ziwp4tnBss0AKJ85GW41twG9yCyoMx/ERbm02xoWxbcHtClhWo63GNmAVt6Ai57Y677egSGYlgYATCIjCu3//fiQmJtZ2TvZFuXWWgwcPasXV+Y8pKSlJK6379u2D7Bsi+Y8cOaIPDWXXbrfjmWeewaRJk7R/7+rVqyF+w8uWLUNsbKy2Ll999dXaYmyU8+mnn+Kee+4xDvH8889j4sSJtcey07FjxzrHwTqwJ8TDrhR+T9ojXxTCTcS/OtzEGHvh1m5PxlCo9YljOjB3JBw5CxmO6cCMj7S0NL9VJEYqCgn4k0BAFF6xxIrV1RCxpIoS6iz108g1SRcTE6Mtue7yi2L7yCOPwOFw4L777tPF9u/fH7Nnz9auDXKiV69eePXVV+sovMceeyz+8Y9/6PTyq0ePHpAJcCKizIjyLRbjUJAoZYmOsjtq2+eqTWIBF17h9OCQ+y5fYAoLC+uMEVf9C6VzMn5l3MkXrXCRYIxp87pyWL5Xb13Gqy9sA2O8RsUx7TWyZmcQZVfekoXTmI6Pj9f9DZXntCfwOaZdUzLe4Lq+yrMk0HICAVF45VuhuBcYIvtdunQxDvVWLHwlJSVaiYiOjtbnJF1WVhbEQrxq1ara9HLeiMYgyp1YaTt16oQ//OEPWjmWhPn5+VqJEl9eEZngJhPY5GFu/GGJi4Ozm4OUKw98EVFoROE1jvXJIP4yK+U/Eg637ZF+SfSLUGmzJ7giIqqHoLiXVFVVeZIlJNJERUVpzuGkHAR6TEesq0T8X9XffbQJlvnFKHooFbbeUV7dv3Ac00YEmnAb0/LcDTeF1zCchNMzj2Pa9SNAjDUUEvAngYDE4T3hhBPw2Wef6YephB374YcfMGzYMN0vOZYfUXxkYtvHH3+sz4vfrSir8iNhzNasWYPdu3drq+/cuXMxatQone7BBx/U4cvuvffeWmVXLhQUFOCuu+7SPrpi+f3kk09w4okn1iq7OjN/kQAJ+IeAehsR90oBbN0ikP/vDrBlR6Ddy4X+qYulkgAJkAAJkEATBAJi4T311FOxePFiXHLJJVrhnDp1qo6qIG177bXXtFVSQondfPPNOvKCuCLIt2AJUSYir7xvuOEGHbUhNTUVXbt2xaWXXor169djyZIl+mfWrFk6rfwSX1yZ/Hb++efrfOIaIWX85S9/qU3DHRIgAf8RiFhTCcs+G4ruV29YotSbkinxiH8yH5aNlbD19c7K679WsmQSIAESIIG2QsCkrJ+OQHW2qKhIuwoYr7Ebq1fcEYzYuc5p5JW3+E0aflvO1xrbl1fO4t8lCm9TIi4NUr6IWJbFpUHOhYLETX8WsS+9giPrf220OfIlISEhQVu3G00UYhdkLKSnp2srfzi5NMh4knEVTi4NgRzT7Z5Tyu3WKhROV5Nc1N8RlMU36fZDqBocjdJpRyegNjUcw3FMi0uDuHHJm6twGtMyMVie0eE2pmUMGXMvmhpPoXCdY9r1XRCXBnlGUUjAXwQCYuE1Gi/KmCfiStmVfPJBYvjHeVKOpJGHiyfKrqflMR0JkEATBKwORP5cgfIzVbQQUXZFzCZUjopB1JJqH/nqk/xNAiRAAiRAAoEhEBAf3sB0hbWQAAmEAoGIzVUwVThQdUz15FOjTdaBUTDnq0mjOVbjFLckQAIkQAIkEBACVHgDgpmVkEDbIRCxugKOWBNsPSPrdLqqfxQcyuAbubbabajORR6QAAmQAAmQgB8JUOH1I1wWTQJtkUDkWhViTim3sNS4MxgQ4sw6akPEpvAJP2c0nVsSIAESIIHwJkCFN7zvH1tPAqFFQPnvWnZUwdq3rnXXaKStR6SezGYcc0sCJEACJEACgSBAhTcQlFkHCbQRAqLsmpQBt7EFJqzKzcFy0AZTcfisUNdGbh27SQIkQAKtmgAV3lZ9e9k5EggsAZmw5lBPFauy5LoSsfCKWLbRrcEVH54jARIgARLwDwEqvP7hylJJoE0SiFCxd2VVNVlswpXYOkfAoXResQRTSIAESIAESCBQBKjwBoo06yGBNkBAFFlbN9fWXd19FY/XlhUBy26GJmsDw4FdJAESIIGQIUCFN2RuBRtCAmFOQMXeNavlhG3d3K9nY+uiFF7G4g3zm83mkwAJkEB4EaDCG173i60lgZAlYNmtJqyphcqtXd1YeFXrtcK7V1l41XLDFBIgARIgARIIBAEqvIGgzDpIoA0QsOywQlRYW9cmLLydI3UkB/N+Wxugwi6SAAmQAAmEAgEqvKFwF9gGEmgFBMRNwd7BAsS4f6zY1cQ1Ecse+vG2gtvOLpAACZBAWBBw/8kUFl1gI0mABEKBgCi8EoWhKbG3N+tIDeb9VHibYsXrJEACJEACviFAhdc3HFkKCbR5AtrC64HCCxWpwZ6hJq6pCW4UEiABEiABEggEASq8gaDMOkiglRMwFdphVj+eWHgFha2jRUV0oIW3lQ8Ldo8ESIAEQoYAFd6QuRVsCAmELwEjzJinCq89U1l4OWktfG84W04CJEACYUaACm+Y3TA2lwRCkYBZhRnTERqUIuuJ2DKVhbfADpSqHwoJkAAJkAAJ+JkAFV4/A2bxJNAWCFiUe4JDTUZDtOslheszEB9eEctB+vHWZ8NjEiABEiAB3xOgwut7piyRBNocAVF4bR5adwWOTcKXKTFT4dUc+IsESIAESMC/BKjw+pcvSyeBNkHAvFctKZzlmTuDAHGkqNBkSuc1H6KFt00MEHaSBEiABIJMgApvkG8AqyeBsCdgdWjF1a78cj0WCU2Wrvx4DzJSg8fMmJAESIAESKDZBKjwNhsdM5IACQgB8wEbTGrGmjcuDZJPFF768AoJCgmQAAmQgL8JUOH1N2GWTwKtnIClZsU0e4YXFl7FRJYhpktDKx8c7B4JkAAJhAgBKrwhciPYDBIIVwIy8cyhniRisfVGZOIaFV5viDEtCZAACZBAcwlQ4W0uOeYjARLQBMxqAQmt7Cq/XG/E3t4CUxVgKuDENW+4MS0JkAAJkID3BKjwes+MOUiABJwIWA5Y4a07g2S3p1VbhM1HuPiEE07ukgAJkAAJ+IEAFV4/QGWRJNCWCIhLg61mIQlv+i0WXhHzEVp4veHGtCRAAiRAAt4ToMLrPTPmIAESMAjYa0KSeTlhTbLrWLzqCWQ+TIXXwMktCZAACZCAfwh4HineP/WHVKlRUVGIi4vTbYqMjNTblJSUkGijKSYWJpMZTbVH2t1UmpDoUE0jTKZqv8+EhAQ4HCq2VZhIREQELBbvJmkFu2t+GdMHlBOu7QDieiQhLiXB+y62P4K44kiVt/G/M2HNMe09Wm9zCOekpCRvswU1vV/GdAB6xDHdELLVypjcDanwjC8JUOF1ollZWYmioiJ9Rj5gRRnLy8tzShG83bjyMsQ67G7bYzabIYpjQUFB8BrqZc3y4E9PT9fcq6qU8hQmkpiYiOLiYtjt4eN/6o8xHbGlAqLmFsaVwZbn/QdWQqpagGJvKUoa+TsLxzEtSlh0dHTYjWlRduX5F25jWh4ZofKc9uTxxTHtmlJMTIzrCzxLAj4iQJcGH4FkMSTQFgmI/66IzcuQZAYre6oKTUaXBgMHtyRAAiRAAn4iQIXXT2BZLAm0BQKi8NoTlFtKbPMeJTJxzZxLH962MFbYRxIgARIIJoHmfUoFs8WsmwRIIGQIyMIR3i444dx4e6oZpnzlFqImv1FIgARIgARIwF8EqPD6iyzLJYE2QMAiFt705k8FsKeoxSeUrmsqCB9f6DZwW9lFEiABEmh1BKjwtrpbyg6RQOAIaJeGZoQkM1ooFl4Rcy4VXoMJtyRAAiRAAr4nQIXX90xZIgm0DQKVDm2ZtdWsmNacTsukNRH68TaHHvOQAAmQAAl4SoAKr6ekmI4ESKAOAVkhTaIot8SH15FshkMVYs7jxLU6cHlAAiRAAiTgUwJUeH2Kk4WRQNshIBPWROwtsPDCYoIjUU1co0tD2xk47CkJkAAJBIEAFd4gQGeVJNAaCNQqvM2MwWswED9eWngNGtySAAmQAAn4gwAVXn9QZZkk0AYI6JBkyjqLqOrloZvbZYnUwElrzaXHfCRAAiRAAp4QoMLrCSWmIQESaEDA0sIYvEaBjhRl4c2nD6/Bg1sSIAESIAHfE6DC63umLJEE2gQBWRK4JRPWDEj2ZBWLVxafoJAACZAACZCAnwhQ4fUTWBZLAq2dgFZ4WzJhrQaQXSy8JWr1CRXmjEICJEACJEAC/iBAhdcfVFkmCbR2AlYVg1dZZe1pLX+ESGgyEbo1tPZBw/6RAAmQQPAItPzTKnhtZ80kQAJBIqBj8CqDbItCktW0XVwaROjWUAOEGxIgARIgAZ8ToMLrc6QskARaPwFReEV8ovAqlwYRcx79eDUI/iIBEiABEvA5ASq8PkfKAkmg9RMwH65WTn2h8DqSalZbY6SG1j9w2EMSIAESCBIBKrxBAs9qSSCcCUgMXkesWiUtzgePEHPNamu08IbzkGDbSYAESCCkCfjg0yqk+8fGkQAJ+IGAuDTYfBChwWiaXU1cMxfQpcHgwS0JkAAJkIBvCVDh9S1PlkYCbYKAhCRztK+ebOaLDotbA6M0+IIkyyABEiABEnBFgAqvKyo8RwIk4JaAWHh9EZLMqISLTxgkuCUBEiABEvAHASq8/qDKMkmglRPQi0740sJLl4ZWPmLYPRIgARIILgEqvMHlz9pJIOwImArtMFX5JiSZ0Xm7cmkwiQ+vnautGUy4JQESIAES8B0BKry+Y8mSSKBNEKiNwetDC69MWjMpXddUxIlrbWIQsZMkQAIkEGACVHgDDJzVkUC4ExB3BhFfxOA1WDiSqifAMVKDQYRbEiABEiABXxKgwutLmiyLBNoAAbHwOkxK4U313eNDLLwiXF64DQwgdpEESIAEgkDAd59YQWg8qyQBEgg8AR2STJYDVgtG+EocNQqvOZ8uDb5iynJIgARIgASOEqDCe5QF90iABDwgYD5i96k7g1QpK7Y5IpWFt6DaXcKDZjAJCZAACZAACXhMgAqvx6iYkARIQAjoGLw+nLBmUHUkcrU1gwW3JEACJEACviUQMIW3oqICixcvxg8//ICqKhXTqBE5fPgwvvjiC2zcuLFBCjn35ZdfQtI4S15eHubPn499+/Y5n9b7jeVpkJAnSIAEPCLgL4W3NjSZR61gIhIgARIgARLwnEBAFN6ysjJcddVV+Pbbb/HWW2/h7rvvhsPRMN7mypUrcfXVV2PTpk06zezZs2t7Mn36dDz55JOQNNdeey127dqlr82ZMwe33347tm/fjocffhhPP/10k3lqE3CHBEjAOwI2h55YZm/v+0eHXUVqYJQG724HU5MACZAACXhGIMKzZC1L9e6772L06NG48847dUHTpk3D0qVLMWbMmDoFi7L66KOP4phjjsGUKVNw3XXXYfLkydi7dy8WLlyI999/H2azGTNnzsSMGTO0Uvzmm29qRbh79+647LLLdL4rr7wSBQUFLvP88Y9/rFMnD0iABDwnYM5Vi06o76p2f7g0qMUnzFsbf/vjeSuZkgRIgARIgATqEgiIwrtlyxacdtpptTUPHz4c69atq6PwWq1W5OTkYMiQITpdRkYG4uLisGfPHm29lfOi7IpI/nnz5sFiseD1119Hu3bt9Pny8nIUFxfDZrNh27Ztuqz6eXTCml/79+/H2rVra0+J0pycnKyPJZ/JZEJ0dHTt9WDuWCwRMKl/7tojbRYm7tIEsw+u6pb2ikRFRdXeX1fpQu2cwdluD5+oAsbfQkvGh7moXN8KS2YMzD7+2zCnRsLyS0Wd8RvOYzoyMpJj2s9/uL4Y035uYoPiOaYbINEnjHvp+irPkkDLCQRE4RXFMjExsba1si/KrbMcPHhQK66iZBqSlJSE3Nxc7Zsr+4ZI/iNHjuhDQ9kVxeOZZ57BpEmTkJaW5jaPUc6SJUtwzz33GId4/vnnMXHixNpj2UlNTa1zHKwDe1ws7CoMlCftaYlCE6z+OY+PYLXB23pjYmK8zRIS6T0ZQ4011FaeC7HBJvdOhynJt48Pa5YV1sJCpCSnwFQv5Fk4jmnnZ1ZjPEPtfDhyFoYtGdPBugfhyNqfY7q0tDRYt4L1thECvv3EagSaWMPE6mqIWHNjY2ONQ72tn0ZOSjpRKupfq59fJsQ98sgj2i/4vvvuc1le/TyS6PTTT8eoUaN0evkl3zAPHDigj+UPW5Tv/Pz82uvB3IlVlusYu6O2fa7aIu2Pj49HoVIawkUiIiLQvn17/cXG3WTGUOtPQkICSkpKEE4WXuPtRUvGdPTWIkSr8GEHy9UXzmpjr89uTaS5DHHKYH5o6z44EmtWXgvjMS1fyuW5Ey4iXzrlDVlbG9OBvj/h+JyWtxXypcKfY1re8slbXQoJ+ItAQBResbiKpdYQ2e/SpYtxqLei9IgCIcqr8c1X0mVlZUEsxKtWrapNL+czMzP1sXwrFCttp06d8Ic//EErx3IhPT290TxGQaJ0OyveUq7U7yyh8vCvnuTnaPLDSNKFSpudOTa2b7RVtsZ+Y2lD6bzBOdzaLF/iWtRmtayw+O+2qIxGbqQtofqCI88Ke/zRNz0G60ayhdzp6r9VFVs4zP4Wjfb649766yYZrMOpzcLCYO0vLr4u1+Abbu32NQeWF94EfD/V2gWPE044AZ999hnEx1ZCiklosmHDhumUciw/YumTiW0ff/yxPr9gwQKkpKTon5EjR2LNmjXYvXu3tpjMnTu31jL74IMPom/fvrj33ntrlV0pwF0eXQF/kQAJeE1AhyRLrba+ep25iQwSpUGEkRqaAMXLJEACJEACXhMIiIX31FNP1TF4L7nkEu02MHXqVMgEMZHXXnsN8rpEIjjcfPPNteHI5LXPAw88oNPIq7YbbrhBR22Q1ypdu3bFpZdeivXr10P8cOVn1qxZOq38El9cmeTmKk9tIu6QAAl4TUAUXlu2fx4bsvCEiKkwfCYCeg2QGUiABEiABIJCwD+fXPW6Itbbv/zlLygqKtIuBHJsiLghGCKKrIQwEx9Dw9/QuHbWWWdpn1txORA/VZH+/fvr0GNGmvpbV3nqp+ExCZCA5wTMuTZUDfNP5BJHggkO5clAC6/n94MpSYAESIAEPCNwVPP0LH2LUslEH0+kvrJr5BFLsPx4I83J4035TEsCbYZApQPmIuUjnuYflwY1S1RNVlPhAGnhbTNDih0lARIggUARCIgPb6A6w3pIgAT8R0DcGUT8seiE0WpReM1UeA0c3JIACZAACfiIABVeH4FkMSTQ2gkcVXj999iwq9XWTAX04W3tY4n9IwESIIFAE/DfJ1ege8L6SIAE/EqgVuH1U5QGaTwtvH69hSycBEiABNosASq8bfbWs+Mk4B0B8xE7HLFqVlmc/x4bdvrwendTmJoESIAESMAjAv775PKoeiYiARIIFwJi4bX70borHBzKpYFRGsJlRLCdJEACJBA+BKjwhs+9YktJIKgEtMLb3r+PDG3hrXAA8kMhARIgARIgAR8R8O+nl48ayWJIgASCT0BcGvwWkqyme8biE+bC6ogQwe81W0ACJEACJNAaCFDhbQ13kX0ggQAQkEUn/BmSTLogFl4RUyEtvBoEf5EACZAACfiEABVen2BkISTQugmYSu0wlalFJ9r7adGJGnyGhZeLT7Tu8cTekQAJkECgCVDhDTRx1kcCYUjAVLvohH8fGYaFly4NYThI2GQSaIUEKioq8PDDD2P37t26d2+++Sa++OKLRntaWVmJ8vLyRq8H68KqVavw5JNPelS9zWbDo48+itzcXI/Sh0si/356hQsFtpMESMAtAfHfFfG3hRexZjjU6uG08Lq9HbxIAiQQIAKivD700EN1FN7PP//cZe15eXkYMmQIdu3a5fJ6ME/++uuvePzxxz1qwh/+8Af8+c9/RkFBgUfpwyVRRLg0lO0kARIIHoFALDph9M6RoEKTFdGH1+DBLQmQQHgQyM/Px8aNG8OjsS5aKVbsG2+8Ed98842Lq+F/ihbe8L+H7AEJ+J2ADkmWoBadiFI/fhYdmqyAURr8jJnFk0CbIrBs2TL89re/xTnnnIPnnnsOBw4cqO1/YWEh7r//fkyaNAm/+c1v8PXXX9de83SnpKQEf/zjH3VyKUuswLfccgsWLFhQp4gPP/xQuxZUVVXh+uuvx+rVq3Hrrbfi7LPPxtNPP43i4uI66d944w1MmTJFt3v69OmwWq11rtc/cNdPSbtixQpceeWVtfU5HEeNC9deey3sdjvmzp1bv9hWcUyFt1XcRnaCBPxLoDoGr38nrBk90MsLF1W7UBjnuCUBEiCB5hJYunQpJkyYgG3btuG8886DKJ0XXHCBLk4U1REjRuCzzz7Dueeei4iICEyePBniq+uNSL5jjjlGZxk8eDA6deqE/fv345///GedYsQ9QkT8ZF9++WVMnDgRZWVlWqF9/vnntTJqKKF33HEHfve736F3794YN24cnnjiCVx44YU6v6tf7vop6cVF4ZJLLoG0r3///vjTn/5Ux81B2iMcOnfu7Kr4sD9Hl4awv4XsAAn4n0AgYvAavRALr2WPeyuGkZZbEiABEmiKwO9//3tcccUVEIVSRKypYjXdsGEDZs+ejX379mH58uVISkrCTTfdhD59+kD8WC+//PKmiq69Hh0djalTp2ol8uKLL9ZlXH311Vqxlslfqamp+OWXX7Bu3bo65YpV+ZVXXtHliOItP99//z2ysrJ0e9966y2tpEoCUXZF+ZXrJ554Ym3dxo67fkoasQ7PnDkTw4cP11l27tyJ7777Dvfee68+zs7O1tvW+osKb2u9s+wXCfiQgPmwDVVDo31YYuNFiYXXtIEW3sYJ8QoJkICnBMRaKoqmKIOGpKWl1fqp/vTTT+jYsSP+9re/GZexZ88e7fKQk5ODxMTE2vPe7ogyK4rurFmztG+suCecfvrpyMzMrI3kcMYZZ9QWO2zYMKSnp+Pnn3/G3r17IW0XFwWZcGZIfHy8Vs7rK7xN9VPKiYmJwdChQ42itOIrCnVbEbo0tJU7zX6SQHMJqIeuXnQiLTAuDWLhNRdS4W3u7WI+EiCBowTEZUH8Ytu1a3f0pNOeRFaIi4uD2ayeOzU/Xbp00f64ctwSETcH8QkWpVJcGN5++21cddVVdYrs2rVr7bHJZEJycrJur0yAk/xiOTbaJdvbbrsNAwcOrM1j7DTVT0knDJz75LxvlNOat7Twtua7y761CQK5lUCM0kXj/KSPSogwk/IwsLdv2cPf05uhLbyqTyhXSm9cYOr0tG1MRwIkEF4ExCLaoUMHHT3h1FNP1Y2XiVni0iDKY69evfDll1/quLOGArh161YsXrxYW1vFv9ZTEYVVxPDBlX1RcP/xj39oK6/E6JVJc84iERHGjBmjT0k4s82bN2vLa1RUFGRim7hfiP+uiCjNr7/+unaX0CecfjXVT6ekbXaXnyZt9taz4+FOYKOazHvZz8DJPwLHLwLuWAPkeP5s9rj7AYvBW9MiCUsmYubEtRoi3JAACbSEgERDEKVTFFtROiVKw6JFizBy5EhMmzYN4rogi0uItVf8eS+77DJ88sknEKXTnYgPsMS2FWusiLgviEgkBCOG7YABAzBq1CgdIUJ8fMVi6yzi5iBuFRI1QmLfigI+fvx4nHzyyejbty8eeOABrF27VrtAyIS3e+65p9bNon797vrpXGdb3afC21bvPPsd1gSWq+frFSuBEhW965G+wJ09gLVFwKVKAV6vtr4U8d8V8fuiEzWNNlZbMxUeDZdTc4kbEiABEvCawH333YfTTjsNZ555pp6YJj61r732mnZlEKV3xowZ+Pe//619a/v166cjLIhS3JRISDGZ8GWsSCb+vuK3KwrzI488UptdJq9JxIb67gySQOo/6aSTIK4NotjOmzdPK7SRkZGYM2cOSktLdVQF8TuWcGkSPUL2RerX766fOkMb/2VSpnd+qtQMAhm0soygSEpKCuT1hDGQa5IEbRM3/VnEvvQKjqw/6rxevzHyOiYhIaH2m2X966F4LD5K4qR/+PBh/fomFNvoqk3yYBO/MHk1FmjZq1atnLIc6KVc0l4YctSVQVwbpq0C8quAmSOA9vWME80d09GflSB2RhHy38hQZlf/x+E1H7Ai6a7DKPpDCuwjYsNuTMsHlXwghduYlhnqRUVFQRnTzf0bkjEtIpa5cJFwfE4HYkzLhCrjfvrrXsrnu1hjMzLUs8yFyGQ1+TxqyrLrImudU/J3JH7BFku1n9l///tfbVWW5X0NkRXcYmNj9eS54447Tn9uS92uRKzFEmGhffv2ri43ONdUPxtkaCMnaOFtIzea3Ww9BO7fAMSq5+gzg44qu9K7VKXgPqfOlSkd/LHNvuuvxOB1pKhHRQCUXWm1YeGlS4Pv7iFLIgESgHYnaEzZFT4SO7elyq6UI4YnUXa3b9+uQ4j9/e9/14tLyDVXInU2puxKevky6qmyK+nFbcJdPyVNWxQqvG3xrrPPYUvgq0PKP0wtb35/H/UQjGzYjY4xwB96AvMPAz/kNrzenDOBjMGr2xdrhkNNp5XJchQSIAESCFcC7733Hk455RSMHj0a1113XZ1uyBtkX1iT6xTKA7cEqPC6xcOLJBA6BMT56PntyucrGTjRzZutc9TbukEJwFPbZLZwy9svPryB8t81WqtXW6PCa+DglgRIIAwJ3H333ZBwYRKOzIgAYXRDrLAHDx6EuDNQAkOACm9gOLMWEmgxgW+PADtUFIYbst0XJZFxZBLblhLgK2XpbanoZYUDFIPXaKtdRWowMUqDgYNbEiCBMCUgvsmU0CBAhTc07gNbQQJNEngrBxioLLejqufpuE0vVuBhScDLO90ma/qi1QFTvj0oFl66NDR9e5iCBEiABEjAMwJUeD3jxFQkEFQCW5W1Vnx3p2Z53oyrOgMbVT4JYdZcEeuuxGUI1KITRju52ppBglsSIAESIAFfEFBTQyjOBIyVUoxz9Y+N84HfVoeDaqo9cr2pNIFve+M1Gm0Nx3YHss0f7ncgUf21TuqgJnOJz4IHcmKaA53V27SZe5Xfb8rRPJ7mlyosudUTx+xpER7X60HTmkyiV1vbVFVbpzdtbrLwACUI5PjwVZfCrc3GuDC2vuLgz3KMthpbf9blq7Kd2+q876vyWQ4JBIIAFV4nyhJrUOLiici+iIQDCQUxx0TrD/+m2iPtbipNKPTHaIPx8JRlEYMR09Zoh7db4SyxeAMRxtpqd+Czg4U4p3MkOqTGedXUy3uU46n15bDFJDRvTJcos7KShB7KjyK+OqakVw1obuJ0FUy4qEyH9wnXMS3r1gdifDQXcf18wlnCKYWTSBxvkXB65kl7w3VMy3PaX2Nals31h9hVuSYV89Yf4lDhxMw1uoI/ymeZviVAhdeJp6xbLYsJiBhB+o0lA52SBWU3ToJUqyn37toTjgHN5QNLnPolULfwDxcJ5MITC9VktSNqUYnTUyrV/Vc7XshE9X3tSRWp4e3NhbhtsEV/aXI3huoXHbO7WN0fE/KtRWpFi/pX/XccFVWJduUOFBwuQEL7xLBaTEWUGRnT8iwJpzEtSmO4LjzhzZj236j1rORwfE4HYkz7a3KXQy0VXHrfPZ7dHC9TRV95DcynTPAyF5MHiwAV3mCRZ70k4CGBL1Xs3Sy1/PqQRA8zOCVLU4tRjFXLu889AKXwOl3wcFeHJAtwhAZpmkNFaRDRkRrchGDTifiLBEiABJogYJ94OhwdOzaRysPL6q2bZcYbHiZmslAhQIU3VO4E20ECLghUKRdaCUd2fgue02epuLz3rgc2F9nQRxyBvZBgKbzGamuM1ODFzWJSEiCBRgk4+vaDo7dasccXIu4XVHh9QTKgZTBKQ0BxszIS8I6ARGYosgKnul5i3aPCTlIWUrV4GebkeOcOIYVXK7yBf0zIpDVdf6F//Pp04fxFAiRAAiTQZggE/pOszaBlR0mg5QS+V9bdVDV/UlZOa67Eqrlm45XSO29P8xReW7BdGprbceYjARIgARIggRoCVHg5FEgghAksUArv8coH13w0qlizWnuashBvLbZjfYEyF3so4k5gUvMI7cFQeNuZ4FBPJ7o0eHizmIwESIAESMAtASq8bvHwIgkEj8DOUiBHRdMR62xLRZTmGGXp/Xyv55EwxJ1BJBgKrwonAUe8Wl5YKd0UEiABEiABEmgpASq8LSXI/CTgJwI/5CnLrip7tAqB21IRZffEDpH4fJ/nbg3mQ0FUeFWHxY/XTIW3pbee+UmABIJM4PDhw3jppZe8Cq9YVlbm81b7o0yfN9KPBVLh9SNcFk0CLSGwOFf57qpQZF4GVmi0ykmZkVhXYMNeD2Owi4XXoRRlR3JwHhMSqYEW3kZvJy+QAAmECYGXX34Z999/P1577TWPWnznnXfis88+8yitp4kkJvigQYM8Td4q0wXnk6xVomSnSMB3BCQc2XK10MM4H1h3jVad0jES4gosfsGeiI7Q0F5pvC11IPakMhdpJBYvFV4XYHiKBEggrAi8+uqreOqpp/Dvf/+7Qbvz8vKwatUqveiLXBQr7Lp161BYWIiKiora9AcPHsTu3btrj40dWaFu06ZNMFaqk/IqK4++yZN9WZhFrMw7duzAoUMqsHuNlKsFrdasWaPrMs615i0V3tZ8d9m3sCWwpkg9+JTS6wt3BgNCSpQZI1IjsEBZjj2R6pBkSuENkmgLr4odTCEBEiCBcCWwcOFCyIqiv/nNb/SyzN98801tV95++22MGDECDzzwAPr37w85njdvHn755Rc8++yz+OGHH7B3716cfvrpOOOMM3Dcccdh8uTJtcrtokWL0FEtpnHLLbdg4MCB2ip833334dFHH62t48knn8Rf//pX/WO323HFFVfAarVC2tGtWzeINblr164Qpby1CxXe1n6H2b+wJLBU+e9K7NzBLQhH5qrjE5SVd5kqu9wDPVJ8eO3pwVN4xYeXFl5Xd5HnSIAEwoWAKJJXXnmlbu5VV12FF198sbbpcu2JJ57ARx99hDlz5kCssxdeeCFGjRqlXSBOPvlk/P73v0fv3r2xYsUKbNu2TS9Z/vXXX+syLrroIrzxxhv46quv8Oabb+Kdd97B1VdfjRkzZtTWIeelXlF8LRaLVopFAX/sscd0uvnz52PDhg14/PHHtUJem7EV7lDhbYU3lV0KfwLLlDvD8CQg0sd/oSdnRKLSAUj5TUmwLbzapaFENdamfigkQAIkEGYEioqK8N5772l3gn/+8584cuSIVmz379+veyKKqPyIBXfp0qW47LLLGvRwyZIlWgmWC6KoXnDBBVqxFfcGKX/ChAk6z8iRI7XyK9vY2FhtHZYyExIStPXXuWBxaxDrsCjLUv8999yjlW2pqzWLjz9OWzMq9o0EAkOgUrkyrCoERiT7vr6ByRFIiwIWNeHWYCq1w1zqCKqFV7s0CAKutub7gcASSYAE/E5g1qxZ2jqbkZEBcScQ94MhQ4ZAJrGJXH755di1axcuvfRSfPjhhzjhhBMatKlTp051fHLFx1dcEkSRFYutw3HUICC+v3IsSqy4R4il17Au1y84KioK1113HW688Ub98/HHH+u21k/Xmo6p8Lamu8m+tAoCa5SyK1bYY5WF1x9ynJoIJxEg3IlYd0XsaS1/RDgcdpTlqUkYe7/XW4fdA38KVbcjsWa1DRVZgkICJEAC4UbglVde0T6y4pZg/Ig1VUKUySSz8847D2JVFaVUlODt27dDoinExMToyWvS3ylTpmjFVZTc4uJifPDBBxg7diySk5Mxbtw4zJ07V2MRn9xrr71WhTA3aX/hTz75RLsvXHLJJfq6KLgiMoktPT0do0ePxs6dOzFmzBhkZ2drVwipozVLRGvuHPtGAuFIYHmBWiRC6ZkDfOy/a7AYpxahmHNALWqhwjx2jjXO1t3WxuBtgQ+vKLqHN76BA2tfgLXsYG0FEbHp6ND/BqT3uxomc+OPILuK0qBFFF4fRquobQh3SIAESMBPBMQv9tdff8X5559fp4Zzzz0XN910k56cJkrwrbfeiocffhh79uzB9OnTERkZifHjx+s0Ys0V6+/s2bPRs2dPbdmV/GKVFZFQZxdffLGepCZK8jPPPKPPi0VZLMmi5LZvX71ykViDjz/+eGRmZuooEFKnKMPiQyzW57vuuktboHUBrfRX4582rbTD7BYJhDqBlUrhHaLi7/raf9fo9xilPIrtVBa2mNKYwntQxeBVieypFiObV1tbVTG2f38jivcvQnK3c5Da40JEx3dBZUkOcrd9gL0/P4b8XZ+h+4n/RmRsB5dly6Q1LV4sh+yyIJ4kARIggQAT6NevH0pKShrUKkqohAgzZOXKlcjNzdUWW7O5+pl3++2344YbbtAKq5wT662EFouLi9PnjLwStSEnJ0f7BhuKrXFNLL3isuAs3333HUpLS3U5ohSLhVf8ecXi2xak5hOlLXSVfSSB0CdgV64M4r87VCm8/pLkSKB/PPCjUngbEx2hQWLwWmrcChpL6OK8raoEW+dfhtLDP6PHya+i2/HPIjFrPKITuyMh8wR0Pe5p9Jo4E5XFu7D58/NRUbzbRSnKpUFZeLV3Gn14XfLhSRIggdZBIDU1FYaya/RILLbO58SFwXBLMNIYW2dld9myZdqH1whnZqQxtqI0O0tbUXalz1R4ne8890kgyAS2KoNAsXqDf4yf/HeN7o1VVl4JT9ZYAAQdoaEZ7gzixrBj4a0oz9+EnhPeQmKnU4wq62zjM0aj96TZ+tyWr6aiqlT5WNQXteCFo50JJvrw1ifDYxIgARJwSSArK0v7+Eo8X7HyUo4SoMJ7lAX3SCDoBFYq6648oo7xo4VXOiluDbKmw1q1wIUraW4M3gOrn0XR3m/RVVl126UPd1V07Tlxceg58R04bJXY9u3VsFlLa68ZO9qtgQqvgYNbEiABEnBLQKI6TJs2DR06uHYVc5u5lV+kwtvKbzC7F14ExJ2hh3rjlOBn7/qhyoIcrf76ZYELV2JRPrzeLjpRcngl9iuFt8OAG5HUZaKrYhucE6VX3B7KC7dh1+LfNrjuSFRuFVR4G3DhCRIgARIgAe8I+Plj1bvGMDUJtHUCovDKghP+FrXKMIapepaqBSiu71q3NlOJHaYy72LwOuxV2P3jPYhJ6oPMob+rW2ATR3HtByN77BPYueh2FdHhP8gYOK02R7WFt3WHyqntLHdIgAT8RsD88n/UnARfqTx6doHf2sqC/UPAV3ffP61jqSTQhggUVAE7Vaiwq7oEptOj1cIWL+4AKtRCF2LtNaQ2JFkHzyM0HNrwGsoLNqPPpI9UqDE1K85LSVGRHEoO/Yx9vzyhXCFGIL7DsboErfAeCB2F15yzG5HLfoI59whsWZ1QNXoMHKnVYX+87DKTkwAJBICASYXhin35Fb/U5C6sol8qZKEtIkCFt0X4mJkEfEdgtbLuikhIskDIKKXwPqMMFb+oMGijlU+vIWblziBi83DSmrU8V7kyPIfUnhchLu0Yoxivt1nD/6QiO6xUlt5b0XfyZ4iIToGsthYSLg0qTmX0vLmIXLxQLYiRCHtGR0Qt/RFR6rji7HOV4jvW6/4yAwmQgP8JlORtxKLXhvmlooETn0f20Ov9UjYL9T0BKry+Z8oSSaBZBFarCWRxyqgqPryBkP5qYYt4Vd9Pyq2hvsLrUOcdKU5mXzcNOrDmeYhLQ+bQ37tJ1fQlsyUKXU94HpvmTcZO5c8rvr0SmkzPrpPlM4M44zhm1juI+PUXVJ5+BirHn6RejSpAFeWI+fgjxMz+ACoiPKqOO6HpTjIFCZBAUAhkDr4JsUk9fVK3w2HD9sX3+KQsFhI4AlR4A8eaNZGAWwKi8A5USqiKxhUQkRC7I5SVd5lSeJ3FfMhaPWHNg4ZUlu7H4U1vIX3A9Y0uIOFcdlP7Momty7gnseP7aTioVmjLTlRLZSqXC1OxQym/AQJTr5GRixYg8peVKLvwYliPHXn0anQMyi+aCodaGUmsv/ZOnWHr1v3ode6RAAmEDIGU7IlIyjrOJ+1x2K1UeH1CMrCFeGbCCWybWBsJtEkCa5VLw2A/LSfcGFBxa5DQZKXVXgw6mY7Q4KH/7sG1L8KkLLOyVLCvJLnL6Ujvfx32/foUiq3rdbGmQqX1BkHMBw8i+vNPUalcFuoou05tqTj7/2DPzELMezOBKuWITSEBEiABEgg5AlR4Q+6WsEFtkUCOmqyWr+ZmDQqwwnusUnityltAljM2xNMYvFXlh3Fky0yk9b1S+dv6NrRE1rB71eS1Y7F74190s8xFwVF4oz/6QFmWE1Ax+SwDT8Otcm8onzIVprw8RH33TcPrPEMCJEACJBB0AlR4g34L2AASANYoK6vIoABNWKuuDejbDkhUjk21bg3KV1YUXpsHFt7DG/6ni0nvd7VRnM+2Mvu52/gXYGtXqcu0Hyn2WdmeFhSxZjUitm1Vyu45QFS022wyia1q7HGI+v5bNcnO6duD21y8SAIkQAIkECgCVHgDRZr1kIAbAuuUwpuqonlluNer3JTQvEsyD2yEMs4ur/HjNeWqGLzqrby9g3v3fru1TPnuvonUHhchMiateZU3kUvKzZ40XafKXfEabFVq3eVAiYrKEPXl57Aqn1zroMEe1VoxQS22oay9Ed985VF6JiIBEiABEggcgYApvBUVFVi8eDF++OEH5ebWuJ/b4cOH8cUXX2Djxo0NKMi5L7/8EpKmvhQXF+vync/v2rULS5Ysqf3ZvHmz82Xuk0DIEBALb6DdGYzOj1RuDaJwix+v+O+K2DNUFAI3krvtfdgqC5Sv7TVuUrX8UmxaXzhilc9FXgW2fv0bWFWdgZCItasViwOoPG2S59XFxaFy3PGwLPkRjsKaGHOe52ZKEiCBVkZg//79eOedd1pZrxp2R3S71atX6wtvvvkmSkoCaJxo2JxGzwRE4S0rK8NVV12Fb7/9Fm+99RbuvvtuOCTMUD1ZuXIlrr76amzatEmnmT17dm2K6dOn48knn4SkufbaayHKrCHl5eV46KGHMGfOHOOU3r788st47bXXIOXIz08//VTnOg9IIBQI2NWfwnr1xn5AgP13jb6Lwitq7s9KlzQfrF7kwd6ES8Ohja8jIetkxCT2MIrx3zY5CmkdLtILW2z54kJUFB392/dXpVHffwdbdlfYengXxqjquONVPDcHqr792l9NY7kkQAJhQkCMbH/605/CpLXNb+bbb7+Nr76qfrP1u9/9Drm5uc0vzI85A6Lwvvvuuxg9ejTuv/9+vPDCCygtLcXSpUsbdOvpp5/Go48+ittuuw0vvfQSXn31VVRWVmLHjh1YuHChPnfPPffgkksuwYwZM3T+bdu2aWW6qEiZqOqJDLb77rsPjz/+uP657LLL6qXgIQkEn8AuNWFNrKsSkiwY0rvGj1fcGmTRCXu8CY64xh8NRft/QEXBFqT3vSIwzU22INqajt6nzYLNWopNn56F/J3z/Fa3eddOWHJ2o/KEE72uwxGfAPuwEaj6er76FlFtLfe6EGYgARJodQSsKlZ3gfLvt6nnwoYNG2rfdOepya5iCTZE3lbLG/FC9ZZo69atxmls374dYtwzRNKJMdEQSS/5nOtZv369LsdII1sxNkq5u3fvdj7dYF/aKcZH2TqLtGHNmjUNynVOE6r77h31fNTqLVu24LTTTqstbfjw4Vi3bh3GjBlTe05uUk5ODoYMGaLPZWRkIE69ItyzZ4++0XLebK7+EJb88+ZVf+CJ8ixK7ZEjR/Dpp5/Wlifn5VvGoUOHsGDBApx00kno3Llz7XXZWbt2bW05cjxp0iT06NFDdhERUY0mQc3QDgWxqEkzJuVw6a49cj1SxQR1lyYU+uLcBuOeyr22K7/JcBHhHB8f7/JNhbd92F4oLj7lGNmxHRJiGlc0vS23fnp3Y7p/Yim+PGzHg7lmmDKj3I6hnB/fVZbdbsjqe7Yek/Xr8eWxjGlTchEiSq3okD0KKRd9i03f3oodC29B6u5P0PO4x1Rbsn1ZpZrBp76MJycjdoxaPa3mmeNNBaYJp8Kx/CfEbt2MmCFDvcka1LS+HNOB6oi7MR2oNnhbD5/Trom5euvrOmV4nl22bBluvvlmREVF6eem6Du33347xAVAFNWLL74Yf/nLX/DnP/8ZojPt27dPu2+KniT60YEDB7QuJAbE4447DmL869mzJ377299qIPIW/aKLLkK3bt1wyy23aEU1NTUVv/76K2bNmoVTTz0VolyLniNupfn5+VrfkrffMiadZdGiRTjvvPMwdOhQrRjLG/YzzjgD33zzDS699FIMGjQIK1aswFNPPYVrrrnGOWtI7wdE4ZVvL4lqOU5DZF9utrMcVPEu27VrVwd8UlKSVlrlxsu+IZJfFFwRAS/y3Xff6a3xS77ByCCSQRYbG4s777xTu0tMnjzZSKKVaWclWazQklbEUMSM49pMQdpxRMqtMtW2r7FmyMANlTY31kZX56OjAzxby1UjvDgnnC0Wixc5Gk+6oUQpc0rR7ZqiTK1+FHdjumdiFX7KVdaDA1aYO8UgqubvoH5zKkoP4sj2eeg17kH9hbT+dX8c21LUbL79VYhWbYqN7YxjL/gIe9e9jU3f/xErZo5B9vBb0GPU3So02tFnTHPb4SguQvnKnxGhlguOVM+jZknffqjoopTwHxYjJoyWHJYxHRMT06wuByuTuzEdrDZ5Ui+f0w0pOVsvG15tHWfE0CfKbJcuXXDKKadg/vz52vdV9KFjjz1WK7zSU3ljLbqLzFcS499//vMfXH/99RDFc+bMmVrhdUdEXD/FCjtw4EA88cQT+O9//6sVXlF8hw0bhn//+99aIT7nnHPw448/Yty4cXWKE8X5jTfe0EqutOO5557T+4899ph+uz5hwgStgI8fP17rVXUyh/BBQBReUQyczeLybaW+UlY/jTCTdPIArn/NVf76jPv376/9dlNSUvSlXr16aRcJZ4VXrM7OlmexCIviLSL55KEUKr4occoJPNZhx5Ga9ulG1vslD3+x7sprk3ARsdCkp6frb57uJjOGWn/kS5e8UvKFVXqFGnJ91XLCxtjzV1/djelz2wNv71B/c7vLUdHPjPJGxtnBdS9pq3ZUxiS/t1c4yJhOVHqnI7eyTn0Raaei79mjIcsa71zxPHav+h+yht6N1F4X1/nS7C1LWVUtWr3Cy+8/AI5GGDRVplhKE8efiIq338KhLZvhSDz6Zb2pvMG8LkYF+aD1xZgOVD+M57tYrsJFwvE5LWM6LS3Nr89p+awXo1drlm7K+irKroi8TRY9RSQrK0srkKLbiBx//PH6OSafjfKZfvrpp+vzkk7eWDclUo8ouyK9e/fW86dk/7333pMNxBosIp8577//fh2FV1wd5DkgSq3IyJEjtfIrb8vF8iuKsFilReTvTgIDhIsEROGVPxRnxVH2jZtugGrfvr2e2SdWWcPaJ+nkBouFeNWqVUZSXVZmZmbtsasdMdeLT4vxQMzOztYDSh7mhlXAVT6eI4FAE9ig3M8v6RToWuvW10d9zmRaVSiuEgdK3ERoOLJ1FpI6TVDLCKfXLcCPRw7lw2uShSdkoqv6EmqIJSoBWcP/iPZ9Lse+n/+G3UvvRe72D9H1uOmIatc8oJHKFcHWr3+LldRIFZO34p0ZiFDW4qoTTzaazC0JkEAbJiBucM5ivFER45qzNJbOOY3kMRRkOS8GGEPElcGQ+mWL8nzCCScYl2t1JOOEKNhiZHR2MRHLtOhx4o5x3XXXaddJSX/jjTdCjInhIv5zGHQiIHA/++wz7XAtJnoJTSZmdRE5lh+x9IlLwccff6zPy7cYUVblR75hiHlevnnIDZ47dy5GjRql0zX2S6ycd911l3bqlhv3ySef4MQTT6Sy2xgwng8KAVlhrUjNCegfZFdxs3reTnRUT06wZ7j+Hlxy+Bc9WS2150WBZZUUAZPSd01KGXcl0fFd9CIVPU75n4rgsAMb550JmVjnrZjVfAGL+nJdNeJYb7M2SG9SHxp2ZSWO/GVlg2s8QQIkQAItJSBGQpmHJCJGwZ9//rnJIsVPWAIGjBgxQs+hkshX4rLgLMlq/oK4OIieJSJ+uxIZq0OHDlpH27lzp84rRkSJquWsdDuXE4r7rj/ZfNxScZaWOG0SXUGsq1OnTkX37t11LRI2TF6XiI+tOHRLyDJxopZ0DzzwgE4jr49vuOEG/c1Cvrl07dpVO067a6Y4c59//vk6n9wQKUMcwikkEEoEJByZSL+6X/yrTwb499jy6tdppekWRLmoO3fbB7BEpyKxU4AtlkkW3RpToR2O+Ma/oydmnYR+kz/D9gU3q5i9V6DbCc8hOfsMFz1xfSpy5QrMduy/AABAAElEQVQV8zcW1n4DXCfw8qxt6HBEzXgDZvUBYVfPLAoJkAAJ+IrAlVdeqV0yBwwYoCdQjx2rJtk2IVOmTMEHH3yAbt26aYutuD2IXlZfJKKWKMcSNUus0M8884xO8vDDD+v04hcsb8vFqNixY8f62UP22KSsn67NJn5osviFiO+uMbO2sSrEHUG+ZdQX8fEUl4f65v766ZyP5aaIqV8U3qZEXCikfBGxLIeUD+/0ZxH70is4sv7XRrsRjr5hhg+vWPnbog/vc9uBWXuBhcc1elt9dqGpMV3wbjE6zCvBgmcyMDqlbrV2WyXWfjASKT3OR+djH6x70Y9HekwfjIT59p0oeiBVKaOuVPG6DZC27lx0Gwpy5qP7iS8hqXO1L1rdVPWO1HOi3d8ehVVZZSvOv7DeRe8ODX/Hw+qNVPT996Lq2FGouHCKd4UEITV9eAMDPRyf07Vj2o/PaVGs5Bnlayk6vA6LXhuGAZM/RFKWbx60DrsVS17phIETn0f20Ot93WSvyhPfWvH19UbE3VPcEwyXisbySnAAsSTXl+bUWb+MYBw3bi7xQ2vEN6QpZVeqdaXsynn5o/NG2ZU88nDxRNmVtBQSCDQBsfCGgnVX+t0xz4bdypq6oqAhhaK93+uV1VK7n9/wor/POFl4PanKbIlS1t1/ISFzvA5fVppb/drPXV7Lju0wFxXCeowPw4iJtXjgIFj27HZXNa+RAAmQQLMJeKvsSkWiEzWl7Eo6V8qunG9OnZIv2BJQhTfYnWX9JBBqBDaGkMIboUKSFbS3QBagqC95Oz5CtFpVLa794PqX/H+sfHhFxKXBUzGZI5TS+zyiE7pix/fTlLKuZga6kYhVv6oFNxJg697DTSrvL1kHDYFFhVU01YRR9L4E5iABEiABEvAFASq8vqDIMkigGQSOVKpJm+onVCy85n02OLIisLoQqHTSLW1VJdo9IKXbuc3opQ+yRJpgjzPB7IXCK7VaItupyWwvwlp+BDnLqucDuGyNcmeIWLMa1kFKmVdvhHwpVon4oMqMWLfGl8WyLBIgARIgAS8J+Pbp7mXlTE4CbZnABmXdFekbAhPWUKrC9SmFMrmLBZXKq1+UXkMKcr6Cw1aB5G5nG6cCvnUkqhXgvFR4pZExyiqdNeJ+5G2frZT2r122W7szqAUnrIOHuLzeopPKrUGsxhHrmnaraFE9zEwCJEACJOCWQECiNLhtAS+SQBslIO4MUSocWHe16ESwxbK/OiRZh+4RiD0ELFd+vCNq5o3m75iL2NSBWnkMVjtF4TUXOJmdvWhI+96XIn/nXG3lTeg4DuaI6tUUjSIi1q6GXQW897U7g1G+dcAgRH8yB1DLnavl6YzT3JIACQSQwOZvpsFk8dGKnoGb6x9AQq2/Kiq8rf8es4chSkAU3l5qwQeLUnqDLeb91SHJTJkWDFVuFuLHO60rlO9rIYr2LUDHY34b1Cbam2nhlUZLtJXOo/6CDZ+cgQNrX0Rmvb5ErF2jojMM9Lk7gwHMqlZTipn7ESI2bYBVhSqjkAAJBI5Au9S+OO0uFzNxfdAEk6k6ZKIPimIRASDgscIrYSwY7SAAd4RVtBkCG0uAoU1HywsIj6gfypWvqVrMTEVEOFathPvSLqBKGVSLxJ3BXqXi2U4OSDsaq8SRZFaTv6oau9zk+Zik3kjr+xvI0shi8Y2Kq44dad6TA7MKg6j9d5sspW6C0iIzcvdGoSTPAmuVCdFxyiWkQxUyutVN50htD1t6B0RsWE+Fty4aHpGA3wnsLtqIG7/wYfQVpxbfNuJfOLNncMOSOTWHu00Q8Fjh7dOnD8aPH6/XYJal6WTpOQoJkEDzCJQrD4Kd6g331Kzm5fd1LlF27R2qXfrFlaFiB7BGBTZI2vmpcmdQr+QTsn1dpVfl2RPEh7fa7cKrjE6JOw6+A7lb38eBVc+gy5i/6SviW+tQ8ShtvXo7pWx8t6rChB2rY7FzTRwKVHxgEUuEAxFRdlSWmdVynNWK75ATgGxxCa6x3ttk1bVlSxVk9S3CxxPjdCP4iwRIwCUBBxywq3+ndbkWmXE9XKbx9qRdrUr55qYHVMlqwgMlbAh4rPC+9957eP311/UqG3HKD+3yyy/Xyq+s1EEhARLwjsAWZd1Vqg/6hMKENdUOyxEbrD2rF3UYpJY5jlG670+5FRizbyE6DrnDu875IbVYeE3F6sPFrn5kHeRmSER0MjIGTMO+VdPRYeBNWomX6AnWPn2hAoS7LbGizITNP8Vj68o4bc3N7FGB3seWID27AnGJcidV05Q+nrsvEjnr22HF/FisW6KWRT87D+2zqmDt2w9RC76DOWc37NnKV4RCAiQQUAJjMs7GwNTjfVKnTS08IQovJbwIVJt0PGjzCSecgJdfflmv2Tx9+nSsX79er8c8cuRI/Otf/0JBgX98ZDxoGpOQQNgREHcGkT7KhzcUxKJCktkzq5W+SPVUGKJcLZYeLFTuDJVI8mJ5Xn/1RfvwKl3XVFStXDa3nrR+V6twZQk4sOY5mPLzdIxc7b/bSIE25dq8YUk7fP6fDti8QoU5G1yGM6YdxLgL8tB1UFmtsivZzeqlV1rnKow8swSX3K0Wyolx4Lu32uv8tq7dtCU5YuOGRmriaRIgARIgAX8S8FjhNRohSwNPnToVDzzwAK677jr88ssvuPvuu9GpUyf8/ve/D6vlYY0+cUsCgSYgE9Y6xQDx7g2LAWmWSa2wZip3wJZ11E1ppHJrWF2eBEtS36BGZzAAiIVXpDmhyYwyZCuxeTv0vw6522bDtmoBHGpCm03FynUlORtj8OXL6Vi7MAGd+5Vj0g0HccyEwjpKrqt8ci6lAzDxqnwkpFl1/pyt8bD17KUmrm1sLAvPkwAJkAAJ+JGAVwrvli1b8NBDD6F3794YN24cduzYgZkzZyIvLw/ff/89PvroI33dj+1l0STQKghsUgpvSMTfVTQt+6ojNBgWXgE8PKEKlYjCwcwrQ4K3PbFaGW9uaDLnTqT1vUKFJovBwe1vwabcCxwqJJmzFByKwIKZqVg6JwXtUmw49arDGDGpALHx3lmXLerLzESVN6tXOX6am4z9GWO0SwPKypyr4z4JkAAJkEAACHis8J5yyila0Z01axauv/567N69G5988gkuuOACRKlJHyNGjMB5552HNWvWBKDZrIIEwpvAJuXSECruDOa9aoU1hdPW8ai5uXvpYkTay7Gx3SkhAVri8Iq01MIrZViiEpHWYyoOmlaiom83OaVF/HRXfpWI+f9LQ0mBBWP+LxfjL85FUnr1FwIjnTdbk2r2qLPzkaSiNyzaMh4VaIeIrZu9KYJpSYAESIAEfEDAY4V3+PDhWLJkCdatW6ddGDIzMxtUf/vtt+PVV19tcJ4nSIAEjhLYUw6UqAlOoaLwWvZaYU9TFtToo5PBSvd8hT7W1fi1qjp819HWB2fPkWBS7gfKT7aZi0/Ub3VG5PGwm2zYH7dOTzbbvDwOX7zUQUVfiMWA44tx2rWH0KlPRf1szToWS++Yc/Nhs1mwOPkmWDZT4W0WSGYigTAjIAZAmfBfX95++23k5ubWP93i48WLF2P16tXNLqes5u3TwYMH8eGHHza7nFDN6LHC++uvv8JVRIb//e9/uPTSS3X/unTpgvbt24dqX9kuEggJAuLOIBIyLg17lMLr5L/rUKsIyXLCw+IK8IuaiyrxeIMuytdWLy/sI4U3btthpJV1xzZlav/ylTSs+iYRmcr14PTrDqH/2GIVasy3PW6XZMPQiQXYbRqKXZvVjEAKCZBAqycwf/58TJkyBXPnzq3T13vuuQd79uypc84XB6JIf/XVV80q6ptvvsFNN92k84rCO3v27GaVE8qZ3D7Wly5dWgtv0aJF+Pvf/46YGDXTpkZsNpv22xWfXgoJkIBnBMSdIU4ZVGXSWiiIRSm8laOPNqYsdzWsZQcxtmMq3spRk9dUPN7hajGKYItEajAX+kb7LthQhC3xn+BIfk8kp+/DKVdEIqVj8xe28IRN9oBy7Fl6AMsPnIvTDuxDZEYIQPWk4UxDAiTQbALZ2dmYNm0ajj/+eKSkpLgsp7y8HDJHStLWX+BLlM+KigqIQdGQI0eOIDU1FVu3bkXHjh0RH+86vmVJSQm2bdumDZFZWUeDvldVVWHDhg215+0qPri8vZdoW7LIWL9+/fDss88a1entrl27kJSUpH/qXAijA7cKb69evfDb3/5WwxZA8+bNq7PgRGRkJLp164Y//elPYdRlNpUEgktALLy91TwpZbQMvpTbYcq1w9bp6KOgIGe+mtQVh5HZAxG7F1imlhkOBYVXx+JtocJbqfx0135uwTbLvYg3l6sPmN8jpf1ypey+E5B7MfT0Ynz5ZhLWzY/CMZcFpEpWQgIkEEQCJ598snJnskFcPt98880GLRHLqrwlHzRoEFasWIGnnnoK11xzDfbu3Yurr74ahw8fxqFDhzB48GB8/PHHWgc75phjMGDAAF2uhIgVHezWW2+tU/ZLL72Ev/3tbzqfGC//7//+D//5z3+Qk5ODCRMmoGfPnnoulry5f+aZZyDppS4JOyuLi4m1d+XKlVoBluPS0lJdtyxA9vTTT9epK1wO3Lo0iHuC+IQsX74cZ555JhYuXKj35Vh+fvzxR232Hjp0aLj0l+0kgaATCKUJaxY1YU30bnvWUYW3MOdrJGSegOjIaAxVRkhReENB7Co0WUt8eHeti8EXL3fArq3JGFE+CxOVn26PkV1QfOBHlBcExq82NisOAy3fYuuebig8Uh15IhTYsg0kQAL+IyDW0q+//hpz5sxpUMljjz2GGTNmQNwfxOr6+OOPqxUbHTrMq7w9FyVYrLTFxcW6DKMAsRhLmRIaVkLCOrtISH5xbZDroiSLwivrKIgl99NPP4UozLKVsjt37ox2KlKNlHHiiSfiwQcfNKrQ2/vvv18r1+LWKuVIG0UZD0dxq/A6d0igNWY2d07HfRIggcYJlKnJartVVKrert9ANZ7RT1csOdURCAwLb1XpAZTlrUVi5wm6RonH+6vy463wjSdBi3rRXB/eynITfpydjGWfpKB9p0qcHTcd/TtvhDk2Cildz9FRGw5vfrtFbfMmc99euxDjKMQ6Fd+XQgIk0PoJiCuDWFdvvPHGOpPVxHIr7qJvvPGGXrlWfHslzKsECJCfCy+8UMOJUCtBSkSsd945+iZKomKJdOjQAcOGDdNGSH1C/TKp14f//e9/8e2332rL8hVXXKGV3crKSpx22mlacR0yZAgefvhhbUV2p9v99NNPOOecc3TR8lb/888/h7N7hFFnOGyPmnVctFa0eQEhJm+Js2u1ug7Pk5GRoc3xLorgKRIgAScCm5X/roQA61s39KtTisDuiv+u+MY6Eqq/+xbs+UY3IDHrJL0dpRTeZ1WDRekdlRLYttWvrdrCq74xeCH5ByKUspuCynIzRk7OR3bPXMQ/shoVZ56tS5F4vCk9LkDetg+QNfQeHZ/Xi+KbldTUuweOWfkhlmy6Bnn7i5U7hevnarMKZyYSIIGQJHD22Wdj4sSJuO222+q0T8K6yiJeokyKiFIs7qSymJcoqIZIBIXGdDCx/sbFxRlJtfuBRNaSOidNmqQja4kPsFh+xQ1VfH9Fp/vggw8wduxY/fa+NnO9nbS0tDoLiu3cuVP7/rpTkusVETKHbi281157Lf7617/qxp577rk49dRTXf488sgjIdMhNoQEQpmAKLwivUJE4TUrC6+t89HvvYVK4Y1NHYzIWLVUmJIBygjZTr15/ykE3Bq0D6/MKyvzzNy8d3M0vpuRhohoByZceRjZA8vUSmebYFKv9axqUoYhab0uha2yEPm7PjVO+XVr7d4DvSu/R1x0Cdb/QCuvX2GzcBIIIQLiK/vdd99h3759ulXp6ekYPXo0RIkcM2aMnrQmfrui2Ep0B3F1kH1RaA3l1OjOu+++q3cl9JnkP/bYY41L2r1BJrb94x//0O6oUqeIlPX666/j5ptv1n68L7zwgp6gJouISUACIyyZTlzzSyzLErFB/JBl8txJJ50EmUgXjuJW4RWHZTG1ixgzBaXD9X8kFAaFBEigaQIyYS1UlhSW1oqF11B47bZKFO9bhMROp9R2xKIcfI9VVt5QUHjtSdU+r5748e5aG4sfP0pBenYFTr7sCOLVimkiERs3wJ7aHo609No+xiT3RlzacBzZMrP2nF93xBKT2QGDYhdh35YYFBw8+oXDr/WycBIggaASMFwbRHk0RNwK7rvvPoiLgbgb3HXXXTrygkxkkwW+ZHJZ3759tUIr1l9DZH5V//79ccYZZ+gVb50jQIjv7/nnn68XBJNJZuI20aNHDx0JQhRpibggfrxSbvfu3fUkNZmLtWDBAohx01mknAMHDmirsyjVYqGWssJR3D5pDRO7dEx8SL788kvIimtigpdvDjKJTWb+yTcSCgmQQNMEZMKaRGgICalwwHzQVqvwlhz8SS3CUKYU3pPrNE/cGv65VS2Wod68t3P7xKiTzecHYuEVMUksXjfrYeRsjMGyeUno0r8cxyo3BnN1NuVL4oBl0wZYBw1p0Lb2vadi9493o6JwO6ITuze47usTth690PuXOViVeio2LI3HaLUaG4UESKB1EbjzzjsbdOiss87SrgXGhXHjxmkLrfjzisXXEAkaIBEc8vPztbuC6F3OIpPQJI2EChOfXZF//etftUlee+01bRkW3c05nKwkED9cibogLg4yYU2kT58+OkqDuFFIejF4ikj5MqFO2pGQkFAnUpdOEEa/jI+CJpss30DkRokpXiy69957r84joTYM03qThTABCbRxAuLSEDIrrCnrrjwm7TUuDYV7voUlOhVx7esqhKOVwiv2iBXKjzeYIj68Iu4svIdzItXktGRk9a7QPru1yq7kUzOLzUVFddwZdIHqV3LXs3QotiNb3zNO+XVr7dETkcUF6NNvP3I2xOiljP1aIQsnARIIaQLOyq5zQ5OTk7WR0fmcsS/XDGXXOOe8FT/b+squcV18fg1l1zhnVg/MxtJLXRZLeEeW8VjhffHFF3UYi65du+pYcpMnT9YhNsR/d+bMAL0KNO4KtyQQhgT2lwNFykraJ1QiNOyqXmjB1qXabFu49zskZo1XD9C6jwXxN05V8ymW5gUXukRpkAl/jSm8pYVmNUEtFckZVRh1dh7qdUO7MziUtUOsq/XFouIOi9Irk9ccDs98hOuX4c2xTfnxOpRVpnfUckREOrBleaiY/b3pBdOSAAkEg4BEfJAFJyjeEaj7ydZIXlnzuUhZRk466STt1CwO0EZIDPEvkWDFFBIgAfcExJ1BJFRcGiy7VYSGVKVEtjOjsjhHvc7fqhTek3QbnX/J2zJxa1ga7LfuEWp54XYm5dJw1P/NaKeah4Ylc1Jgtjgw9rw8l0sDWzatV8puT6jp0Ea2OtvUHheiquwAivYtrHPeLwexsbB3zETMrk3oPqQUO1bHoqqi+rWkX+pjoSRAAq2GgBgc61tnW03n/NgRjxRecYYW87f47MpMQfHxkFAXMuNP3BlkJREKCZCAewKi8Earv7jsWPfpAnVVFF7Duntw/Su62oTM8S6rH6NCkok7xpGjUXJcpvP3SfHjdWXhXb84XoX4ilS+sHmIaefCQiurBKmZzNa+/RttYnzGKETFZyN36/uNpvHlBZua+GHZvhU9h5fCWmnCzjUhMjB82UmWRQIkQAIhQsCjKSjiIyI+vLK8nKzUIbP0xJwuE9YkVq8ES6aQAAm4J7BZRWjoqSbom0PEkGfZZUXl+Golqyx3DSJiMxARk+qyE6Lwiohbw5kZ1fvB+C2RGvSkNafK8/ZHYOOSePQdVYK0LtVuGk6X9W7EZhWOTE3QcA5HVj+NHKeqmLwH1r6gwpQVqQUp/BsyzNa9J6IWL0K8dT8yeyVi28p26DWi1FWzeI4ESKCFBF5YcyuiLb75UqkeJZQwJOCRwiv9kklqEv6ivLwco0aN0l2944479H5rMa2Lw3Z0dLTum+yLom8c65NB/GWxRKgJRu7bI+0Vp/JQabMnuAwneJmBKszDRQzO8gXQU9lcWqmW6pV76PqVuqflNDedwVePj3wbzIV2mHrGICrSgvL89cgYeF2jY6eb+rPoFleJnwpNOC87cO2vP6ZNqREwH7bVtlM+eH75KhEJqXYMPaVSuTJU//3WZ2TZskmHIovK6lT/Up3jjH5TsX/VdBTv/RId+l5a55qnB8aYlig3BnOXefv20z7JMbt3KWW9O757OwZ5e9uhY/fgLEQhbZW/Q5m5HS5i8A2nZ56MaWl3OLXZ4zHdgoFj3MsWFOEya+eEPph9Xq7Lay09GWl2/bxpabnM7x8CHiu8Un0/Faxd4rFt2bJFt0bWYJY1lWUmYGZmpn9aGMBSjQ9XqVL2nY8D2AyXVZnFLKj+Gw8eV4lCrc2u2lj/nPGQk627vtXPF+xjYS1tlq0nUmFzYEepA1OzI4LWT6Otwtm0s0I329QzFiWHV8BWVYzU7FPctu24NAvmH7C5TeMJC2/SNBjTKUrh3aIU25rZwpuWRyF3XwQmXlmMqOhGZhArBc68fh0cI0bW5musDe2SuyOh42gcUW4NmQN+01gyt+eNthnbRhOrcD9QfrzmbVvRedwJOlbwlp9j0alXcKy8xt9gOCm8zmO6Uc4hdkHabLAOsaY12hxjLBvbRhO24IJxL1tQhMus20rzceYPr6prvv4iZ8KjA07H1M5DXdbLk6FHwGOFV3x3Zfk7icVWXy666CLMmjWr/umwOzZWEpGGy7dv+QOUWHWhIHFVVYhQH9zu2iMPUYm55y5NKPTFuQ3SXontJ28OqlQfw0Wk3bIqjacW3g3KnUHpvOgWWaXuT3D66Tymo9UKGDJ3qyS1CodXfQVzpIqvGD/A7dgZqd7wz9gFrD5Uip4BCipQf0yLf64lz6rbKZO8Vn4dj879VOzgjCJ1zvXoMSsLaju1UlFZz16wNZbIKWtS1/9DztI/Iv/QZkS1c28RdspWuyuWXTECeDKmo7t1Q8SGDWoslaLbEBPWLkxA7qFy137ItTX4Z0fa7c2Y9k8rvCvVsJKG0zOv/pj2rsfBSe3NmG5uCxsLh9Xc8ox88gWuRC2qc2GHAegcrb5k+kDssOPZ3UtRaW84gdYHxbMIPxHwWOGVpejEZ/emm25CampdP79wXFPZTzxZLAm4JCArrImESkiyiJ01K6ypNweF+xYgIWOsCuPl/nEwUkVqEBvqj8qPN1AKr4bm9MuerKzqMnFOLS+8cVmSmuxlxqATi5xSNNyV1dUckVHVERoaXm5wJrnrmdiz7EHkbf8IGYNuaXDdlye0H++SH2HKz0O3wWat8ErEhn5jakJ6+LIylkUCbZjA2Wl9MSqps08IWFXoQlF4KeFFwP0nXE1fCgsL9dLCjz/+ODp06BBePWRrSSAECEiEg3S1UE5y4Nxf3fbasrNKRSyIgrUiD2VHVqP9qClu08vFePW0GKIMJD8od7jLffO50WSd9RPYk6vdFqr2Q8eu7TGsBO2S3FtZIjash1Uttalef9QvzuVxRFSSXl45MApvd90Gy7ZtiB6eohbMKMeOVXHoO7pEvWFy2TyeJAESIAESaAYBj2YJJSYm6vWWly9f3owqmIUESEBCkoXKCmsot8O8zwZr90gVc3axujkONBaOrP6dO05Fa1iuVlyr9HyuXv0iWnRsLC+8Z0n1MptNWUJNKn64OWc3bP36e1VvSvdzUV6wGaW5a73K521iR2KSioXcXoUn26azdj+mFCX5ETi0q+4yot6Wy/QkQAIkQAJ1CXhm8lB5Hn30Udx1113IycnRyq/4MBoiVt+BAwcah9ySAAnUIyAuDeeEyMI4ETvUksLKn9jWLQJF+xeq2LNdEJ2QXa/Frg/HKW+m53eoZYaVK//Yup5NrjP4+KyxvHDeOgt6nFyC6Dj3mrdYd0WsXiq8iZ1O0X7N+TvmIC7Vv882WXUtQim8Mo2wQ9dKxCVa9UIUsk8hARIgARLwDYGjWmsT5YkPb0FBAaZNm9YgZWuZtNagYzxBAj4gIIs15Kp5aqFi4bVsr4JDvduxqfBiRasXqdXVTvS4lwPUssgpyi1jsfLjDYbCKxZemWsdU2FFt5FNT/6zbFgHuwpFJpZUb8RsiUZy9hnI2zEXmcP+6HE0Dm/qMNJalcIbuWIZTMXKFzk+AV0Hl2Hj0ni18lohIqOltxQSIAESIIGWEvDIpUEqkXBkMoPX1c+MGTNa2g7mJ4FWS0D8d0VCSeG1dY5ARfkOVJXsUe4MJ1Q30IPf4lc6Vrk1LPZPWMsmW2Czm1Cu4gZ3TC5DbIJ7665aChIRmzbB2n9Ak+W6SpDS7VxUle5DycGfXF322Tlb9xo/3u3bdZldB6noHyoU7+71vgmS77OGsiASIAGvCKxZswbvvfdegzxvv/02cnOD9BBt0Jq2c8JjhVfCv0h4j48++ghPPPGEvllyMyVQuYQsoZAACbgmIO4MEUpR7KZWWQsFidhaBVtPZd3dv0g1x4T4jHFeNes45cqwTYUA21fuVTafJM7ZEItSSyTS2zVduWXrZpiqKpXC2zyXhPiOY9XKc+nKyvuxT9reWCGO9mmwJyTCsqPaj1cm4aVnV2KnitZAIQESCF8C8+fPx5QpUzB37tw6nZDVaffs2VPnHA/8T8BjhXf9+vXo378/rrrqKu3PKxbfP//5zzj++OOxf7+aMk0hARJwSUAmrHVXym6kx39tLovxyUlHiVphTSasicK7bxFi2w9GhJexKUXhlQACC4NgoNiyoh1sCRbl0uA+MoPAilin3BnUhFu7WiCnOWIymZHcdTLyd30Kh5hc/Shi5TUmrkk1YuXN3ReFoiONLKbhx7awaBIgAd8RyM7O1q6geXnKD6wRkZjdYkCUiFiGFEvscPVW3RC5VlFRvWDQkSNHdNz6bSq6i7Ns3ry5QRmSp0hN3l2nnodW9darLYvHH8HXXnstxo4dq8OTdenSRTP73//+p1eMeeedd9oyQ/adBNwSEJeGUHFnwMYyraxau1tQvP9HtarY8W7b7uqi+PAOUotQLAqwwpu7NxL5ByIRmaUUbrU0sltRb6Mi1q2FdUDzrLtG2SndzoZNhW4r2r/YOOWXrUxcM+/bB7VahS6/U59yWCLt2LkmRF4L+KXXLJQEWj+Bk08+GRMmTMDtt9/usrPffPMNunXrhjvvvBNdu3bFq6++qtOJFfjFF1+szSPGxg8//FAfDxo0CGeddRZGjhyJRYsW4eOPP4acu+GGGyD6mZFPjJIXX3wxhg8fjiuvvBKDBw+uoxDXFt5GdjxSeGUVm59++gmPPPIIJESZIRkZGbjjjjvw6aefGqe4JQEScCJgVXOOtorCqyZ7hYQohdehIl4Vxa1XywkXKv9d7xVe6cfxysq7VBksKppwo/Vln7f9EoeYeBtisgFzvvuKJRSZuahQKbyDWtSEdukjEKlWW8tXk9f8KaLwmpSSbtm5Q1cTEeVA577l2LU2VrmS+bNmlk0CJOBvAs8++yy+/vprzJkzp0FVjz32GGQelLg/bFCrLsp6B54s733uuefi8OHDGDZsGK644go8//zz+Pbbb7F69Wo89NBDtSuXbtmyBZvUXIZly5bpVU2//PLLBm1oKyc8UnglBJksh1hSUjP7xonO2rVr9TWnU9wlARKoIbBD+bpWKYUlZCy868tg7RGJ4kOL1cpq0RCFrjkyvr0yRiqdc1nDlcabU1yTeWQZYfHf7Ta4FI4Utdpagarc/v/sXQV4XGXWfjMWd3drmqTubrSlQltavFDc3XaBRRe2sIsssrDAYj9QtIVCi5e6u3vSNu7aeEb/c2466SSZydxJZpJJek+fdObe++m5M3fOd773vMeyJag4dgQGijvQJSRabdtaAf/YuajMWQU9pSd1lOhDw2Bwd28Da6ivkaM4S+LkdZTepXYlDXSFBvz9/fHBBx/g7rvvbhGsVlJSInholyxZAvbgsleXoQ87duywOqxJkyYJ7DEMN+W0zJMnTxbqMISib9++ggHNJy655JJmlpnExEQB3mC18V5aQJTBy4FpM2bMEHh4eZXAwl5fjjRk1/msWbN6qXqkaUka6JwGmhkanMXDSwavLomwoYTf9QwZAabf6oik0nyCyA7bVNaR2rbXYcYCHcHP4oiyi7OtMY+wS5VlL6/iyOEm7l0TvnDbe22q4Rc7D3pNNarzN3a0Cev1iP5CF0c43nOBa1whKLqJkzfriBS8Zl2BUglJA86tgXnz5uHiiy/GAw880GKgbF/dfvvtgjHMBjHDE5IoM6QLPRNMMbeM6TUVL6+mH5WIiAjodDro9eefh2yfGesGBNB23Dlhx6UY77GxfG97FWXw8qQ//PBDwcgdNWoUMog+h3EpixYtwtSpUy1iU3qbsqT5SBqwVQMn6RkVQJhXNg67XXIp4KFaB3UfoLZkH+F3x3d4SPQsxkR6jnaVwctGHxuAnn46GPyaHluWYA2y4iLIyXOi7T+ww/MzregROICSc8SiIusX09N2f69lgzcnh7YEmviFWccx/euRn+4GrZoOJJE0IGmgR2vgP//5DzZs2IACxuuTBAcHY/To0cjKysKYMWPA3tlbbrlFMFYDAwPBO+gsTAywb98+4X3r/9jgTUlJaWaC4OA3/hs+vGO7d63b703HohNPsFLZzb5582YBZ8KrkiFDhgh/vUkh0lwkDdhTA8zQkORpzxY70daxpojfKj/a7tc3dsrg5VFMIVjDj0TQwrRrjsQoV5W5oDxfheGzm/AT+nMGr4sFHK/i8CEYyLOrpR8BewkHr5Wc+JRgDQ3kFXezV7Mt2hFwvOSpkedkN0Mx2OA9sd0beWluAnNDiwrSgaQBSQM9SgNGaAN7e43ywgsv4NprrxXoXtlLyxltw8LChCAz3lnv168f2JvLpAGWZPHixYIDkoPUGBLBRAJss0nSUgPtGrwMiDa6xY3VGBvCf0bhlQfjR/z8/IynpFdJA5IGzmmAjcHZIU6ijiMEKI5S4WzNZmIA8BEoyTozstGUgMKVnK0bCdbgSIP39AE55IqmIC4er9HglVlgamCDV5ucCqg6BtcwpxOmJys68l9U5W2gDGyOgXDpI6NgIE5zpiczYo+9A3QIiCBOXvJwM1WZJLZroJ4o5fIbq1GurYfaQLR8xFPiLVchWOlJfx6QsStdEkkDDtAAMy+0FmZXMIUVjBs3TvDwMp6XPb5GSUhIAAectT7P140eYmPZKVOmCLy+rcu++eabxiLC64WeJKxdg5eVaHSpt9BaqwMptXArhUiHkgZIAxW0M11CcU5O4+Flg3ewJ9GRbaNkE2MII9YEDejozXKXA2PI6F1PBu8dsR1txXq90wcVCO/TAGYuEMSNcGhuLpBVnMesGVtxKSmGvLAA6oumGk/Z5dXdPxWuPgmozPrVYQYv5HJK9xxLON6MFmOOJS/v/tU+qKuSwcOn7ZxbFJYOiDlEiy1VOcLfvppC5KjPc5u2Vo+bixxJ7gEY5BmKUV4RGOEdAXdZuz+LrZuQjiUN2EUDpsauaYOWzpuWMb63payxzoX02u43e/ny5UQL2cQLuXTpUvAxU2YMHjxYwJSsWrUKH330kUBNdiEpTZqrpAExGmDvLkuyEwSsuVQQb22BBvqFLqhLO4TI4c81Da6T/zOs4YU0oJjgwSH2c6g2j6q8QIGqMhkGTG7p3dQzU4MZD6/y4IEmOEMH0wk3d2zmDXt5S459TGl/CdagcBysQbVpA7mxybClABOWqNR6HFzng+xj7kgZ05YpRygk/Yc88uJ+VXIEv5SnoUavQaTKGyPJiL3BYyBiKLlKoMIdKpkcBvp3VtuIYk0tMhvP4nhdKf6oOC3UVZEBPNY7ErP8EzHFN5Z2MNr9iZS0LmlA0kAP0kC73+bk5GRhKux+nz59Or744otmRob4+HgBU8LRgew2Hz++4wEwPUhf0lAlDYjWAAescUrhBCfIHaA82kSpdTbwEAjAC6+wcaLn0V7ByWTw8obwBvLyXh3RXsmOXcs57gaVmwGh8U0Zhoyt6P3lkJW39XYqyODVsrFrRziDsU8/oicrOvwOqvLXk5d3tvG0XV8FHO+aPyHLy4U+OkZom+cfnkicvJSEQjJ426q7VF2Ll3O2YEXZSbhT2un5gcm4NKCv4LltW/rcGTOLs7T6Mmw6m43VlRl4Mms9vAj2MMe/D64MTEWiO21lSCJpQNJAj9ZAuwavcWac3u7s2bMCVYbxnPGVk09s3erYLETGvqRXSQM9SQPOlFJYwQZvhBKldeuhcAuCu995HH5ndBpI7BODKBfN+lL7G7yccIEN3tj+hLsk+ISpsIdXXtAyTaYsP4/YGYqhnukYY9TdL5lgDYnnYA2O6YMhDQaCNjCO12jw8rxjCL+7/Qd3SjesREB4E4uDqT4u1Pdf5x/Ey6c2ECWzAXeGDcOi4AHwIKO3I9LXPRD8d3vYUJyqL8dK8hT/Up6OpaXHyFMcjuuo7Uk+Mc2cph3pQ6ojaUDSQPdpQJTB6+HhIXhzH3vsMXz66afw9fUVcjwzvcbLL78sRBV23xSkniUNOKcG2MOb7CQMDcoj5CEd44PK3E2E37Uc7dsRTU4NAt4h2Gk12Z/eop4o4nph466+Wo74AU2wKtNaBvbwHiMj3kSU+/cRtpcovFIoYM1B4hd7iWNhDRS0xsFrCjJ4NZOmNM8iLKERKnedELwmGbxAuaYez2VvxLbqXFwakooHgocikALQ7CV9CNf7l8gxuD98BFZVnMHXpUfwSMZqxBI04obggZgbkCTAI+zVn9SO4zXw9zPr4dnBxVDr0Z2LJmh9Wjp2cg2I/nni/M4LFixASEgIIiMjhShBxvfecccdEg+vk99kaXhdrwEN7bafoRixeaFd33frHmX5WsjK9ND016OG+BmjEhe1LtKp42lk8L55BthMsIZL7DjfvBMMZ9AjIlGPalo8mIo+gDG8pGTOtiYjUAVhXhUH9kMzcBDhSEQ/1kybFPXeL4bYGhwMa9BSdLZq5w6CntDczjEIMJw3pl8DsijV8OCpVW083qIG30sK/Upe19fydghQmv8NmI+ZwX0FKiZHTI8xvJcG9hX+dlfnY0nxYbyYuwXvF+4lj29/XBnUT2B8cETfUpv20UC8ZwCOTH3UPo21asWDYC+S9BwNiP5l4JR0u3btEv4OHToEHx8fjBw5UuCI6znTlUYqaaBrNMDGrpbsFWcIWFMeaKRtcqAyaK8weW874XeNmox2b0qdvIZgDXY1eIl7NqJvo1njjjG8QrY1SjHM3l75qXTIqqugHT7SOCyHvLr7p5xja/jNsTjeDeshKyqEPiy8eR6xA+pwaq8n8k+5ISq5rde7uWAvfsPG7t+zNwlG5jfJlyE1OLrLZjuSGBz4j+EOnxcfwnsFe/FJ0UFcEZgiwB1CVU6yndNlGukZHWXTYvnStdUOGezzQ3xwdXzHIDQOGZDUaLsaEG3wcivulOud8zUbcza327J0UdLABawBhjOwOIXBu7+ReGlVqCjbCFevSLh6xzYNzo7/s5f3/3KAeiKDYLqyzkpFoYJouBSI7Ms/VG0jjBjDy8LUZDoyeJV7dkEfECik5xUuOPA/vxiCNZz4P4clodDFxsNAnl35mdMtDF6/UC18gjQUvOZ+QRq8n5Jx+U7Bbsz0S8DzMZO6jUGB4Q6LY6fgPoI7fFlyGN+XHcc3xA4xk5gdGEOc4kFfBkmcRgNa2ikpbTTg0nA5ojzsw7msI2fGB2e09LyjN5L0GA3YZPD2mFlJA5U00M0aOEEGbwjtdvl18+LfpY62+k+oUb/QGxWE3/WPmuQQzVxMfOnvZxGsoRyYcZ47vcN9cTpduVKP0DjG6bY1eA3NBi/lkK9VQ3H0CNTTZnS4P1sqNiehyN8Iv+iZtlQVV5ZwyPqISCFwTTNuQos6sQPrcWSDNxpqZXDzJEjHBSJv5+/GZ8UHcT3hZx+JGOUUgWNhKi/8NXIs7gwdJhi935Ycxa8VpzDMMwwLCe7AtGaKTnJdXyC3t0umeVWUHKMD7bAap9FqCUrFBq8kPUsDTW6SnjVmabSSBpxeA+zhTXEC/l3lvka4kNe1bmAN6irS4B/tGIM3kXZzmX5tdYl9bg0bvByoJbewJGdIA/tWmJpMsW+PgHfVjHAsnME4M05CofKOI7aG34yn7P7K9GTyM2fatBvTr4mPOJuwvBeKvJW/UzB27w0bjkcjRzuFsWuqex+FK24NHYJf+1+LxTGT0UCJLx7PXIs5R7/F+wR7yFc7ZjvddAzSe0kDkgasa0AyeK3rSCohacBmDZysdRI4w+4GaGMoeYN+pzAHR3l4uXH28nLgGsMaOiM1lXJUlSoRQdnVLAoRHBt8KHCtTAfVju0C966B4gq6ShjWUJW3lmANLfmB7dW/NiERstoayIqLWjTJXt2wxEZkHr4wDF42GDlQjNkSmC7MmUVJ3tw5xN7wZfICLEm6FKMpgcUSwvrOO7YU95z6Hb+Vn6LvhkQp58z3UBpb79aAZPD27vsrza4bNJBPdhpTdHW7h5fgDBywphntRumEt8PNJw5u3tEO0whDGeppl51hDZ2RAvLuulBEGht27QkzNcizKoiBohSaMePaK2r3a0xPptfUoLpgs93b5gZ1cedxvK07iBtYh+oypcDJ2/pabzr+qvgIPirajztChwoe1J40twGeIfhH7GT8OWAR/hY1jjK/qfFM9gZMO/oVnsxYh7Wlp6HRd3Jl2JMUIo1V0oATaEAyeJ3gJkhD6F0aYPwuS3cbvKpdDXAhh5J6vBuqi7bDL7IlHrRplPb7v885WMOq4s61WXDaFUHRaiHDWnst6QOInSGnCrrgEOj6JLVX1O7XPAIGQOUVLSShsHvj3CBxnzNDgzlYAy8EXD10yDzUe728qyjV7+v5O7CQaL/uCR/uEBV3RaPeRFt1Fc3hi77z8UPKlYRBHoAjdcW48+gKTDvyJZ7J2oANZ7PQSDAISSQNSBpwrAYkg9ex+pVavwA1wAavF8VGRHWzPaLaXA9tkhINXqVQV2c6LGDN9BbPZFgDeXjrOui8Uje4oDRHhfD24AznOjR4EGVZlQya8WTIn+OrNR2Lo98zJ+/Z3DUEa2iZAMNe/eqIj1eecbpNc8zJG0uZ13KOu0PbC3fI99cUCkklpvnGCUFhbRTQQ0/EufnhXoJm/Nx/IVYOu17g8D1cW4xHKaHFVDJ+nyDc7+rKM7RLIhm/PfQWWxx2eno6/ve//+GVV17Bzp07odOdf0AWFBRg5cqVQt0dO3bgwIEDFtux5YKB2Ck+/PBDovO2D5PE6tWrcfp02+eRLWMSU5Yz+zpKuszgbWxsFFIQb9u2DRqN5ad0aWkpVq1ahZMnT7aZM5/7888/wWVaS01NjdkUx+3Vad2GdCxpwB4aYIO3u+nIONmE8rgGjRe5C3AGnpd/1ER7TK/dNmaFgLxVwIa2X9F26xkvFmW40gPaBeFW4Axc3qU8g/b+vaEZNsJYvUtfm2AN1aTfLQ7pV5fQh7iFq+FC6ZJbC8MatGoZ8k5286qq9cA6eZzbWCUYgKnuQXgxdgrlFLEPjVQnh2X36gO8Q/FgxEis7Hc1viU+4UXEPnGqvoKM3nWYdvgL/I1e2fOrMdCXSZIerYH33nsPEyZMQFpaGtRqtZCZlqldjYYdn//Xv/4lzJGNYXsZvHpKxnPXXXfZzeB9//33sXv3bofei4cffhi///67w/roEoOXb+zNN9+M9evX48svv8Tjjz9u9ibs378ft9xyi/DB4DI//vhj88TffPNNvPbaa+Ayt912G7Kzs5uvcca3559/vnmVZLzQXh1jGelV0oC9NXCcgrK7G87guqYOBncXqMeSwUtwBpVXLHHwRth7qm3aiyOmBp77721ttDZlzZ0oOO0G7wAtvPzPe0DMlXM5W0kBXScp+YQSLloLVA7mKtrxnEfgIKg8oxzG1qAlpgb2zSiIj7e1eAfqEBilRsZBUngvkToK6Ho44094yJR4I/7ibuPZ7Wp19nUPJM/vcCxPvRLfp1yBG0MG4WR9mWD4zzzyFV6nrHIZDZVdPSypPzto4PDhw3jqqafAnts33ngDzz77rOCY42y1t956a5sebr/9dlx99dXN50tKSpCTkwOtVovKypafgeLiYuFac+Fzb9i5yEY0G7ymwp5e9tBye6ZSVkaRxiR83tSuMpZh73RdHWVSMhGuw47LM62YZLhsVVWVScmmt9zu2bNnhQMeF8/LVKppYV9RUYFjx44J9XkOLJbGbFrXlvdd8kuxdOlSjB49Gmy9s/Cqg1cyY8aMaTHWt956Cy+++CIGDx4s3HS++XPmzEF+fj42b96M77//HjLaz/v222/x1VcE/n/ySUHh/IHy9fUV/owNZmZmWqxjLCO9ShqwtwbKaXe7mP5Sve3dsvj2XGr1cF1fj8aLyRhydREMXu+w8eIb6GTJ2eTlfYecr5W0kWMLDzE7s4rOuCKWvJfWRLV+LeDWtPXlUq6DwaNL1u5thuUbMxvlp5fCoNfAhQw1uwrjeMMjIKcfKc3osW2ajh9Uhz2/+RGjhYISUvTsbXD+YeOgrnx1DT5LmocAZe/yXLe5eRZOJLj5424yfvnvaF0Jfi5Pw0r6+4oSW4wh1ocbQwbSa5SF2tJpZ9PAZ599huuuuw7x8fHNQ3OhXYuXX34Z/fv3FwzZ5gv05rnnnkNgYKBgJLNjb82aNUJW2+DgYIGOb+3atYI9xI5B3ulmw3HgwIH46aefIJfLsWLFCsEh2LptNiZnzZolGKlsOA8aNEhwKPJY2N4aP348srKyBKP3qquuAttiRUVFmDRpEvz9/VFYWAhXV9fmoQ4YMEBoY8+ePYKjsby8XBgzj3Pfvn3C/O655x7BeJ05c6ZgMPP4uL1///vfQl3ereexs0ycOBHXXHON4N3mecXGxmLIkCEWx9w8EBvfdInBe+rUKcyYcZ4UftiwYYIlb2rw8gomNzdXUATPITQ0lOI2PJCXl4eMjAzhPBu7LFz/119/Fd7zyuPpp58Grzh+++034Rz/xysPvqnm6hgLbdq0Ce+++67xUDDIhw9vCpBQKJpUwx8+pxB3D/rAy4QvQ3vj4Q+V04y5vYGeu8ZfOBZesNgLa3SuaYe+sJ6VyrYGzpEixnNWY0ykLwJ9u+Tr1Wae+p+LaKufMp4tjIRMVQR1TQ7C+syAStWU993Rn49r3XV480wlttV54oYEtzbjs3SiMJM80g0yJA91pc9w01i5bOvPtIEe8prduyCbfBnwFRnVak+4BHbPCkMx6FqUHP8ILrWHERh3sTA1e36mtf0HQL9zBzzNPId8xwGH1htQcDIA8cnte8Qt6dz0POvZ+NwzPd8V79/J2CZs4b83YD7GhvYV3aVxvI7+TIsekMiCrT/T5qpNons+KToF/yDP98rCY/g4Zw/uPf0HBvuE47GEiZgYeN6IMlffnufs+Zm2NK72oI6W6jj7efa0Tp8+vc0w2QDm348jR460ucYn2HDdvn072GPK5RYuXNgM5fzrX/+KpKQkAfrJdtO0adPAhvBFF12EG2+8UXg/cuRIfEbGNrfBsmzZMgwdOlTAETN++NJLLxWujRtHDxGS1NRUsGOSHYXcNnujH3vsMcyfPx+vvvqqYF9FRbVcaPG1P/74QzBm2WPNY54yZYrgJeb+2WH5zDPPoF+/fvjkk08EY3vevHlgz/T1118v7PYzpvnQoUMC1IMdmFu3bhU83zyXDz74oN0xCwO38b8u+UXm1YGPCUcmv2fj1lRYCZ6e9MNlgtliI4hXDgzq5vdG4fpGNzyvNFg2bNggvBr/a6+OsQwb1KY3kQ0CI5icH0gsxmNjne56lZH7i5hHrY6HDXxnGbMYXfH95i80b3O03oIRU7+7yljS86FyDW3FUhIGYizolvtQSm7VH2iLao4fDETbVXZio6Ain7CxzQsKR48rlBwBYwIVWJ7dgOti2y4KLN2zrOMKKF0NCI7Wku7Ol2qta/2K5eTdJQ/g3Elk8GZDz4sM0wrnqzr8nWfQUIKKRKEw7QfKujZV6M/4mWY9d3oR15eMvzV/QkvbjS4RES3mw0m8kobKkLZXjuEzKNuceFW3aMd40FrPxvOOft1cnonXz2zGHdEjMSsoyabvjbM9p8XqyhZdqyDDVWEDcGVof6wtO43/ZG7D9QeWYaJ/HJ7rcxESPR3vlLHrZ9qCkjr9XbHQbneeDggIAEMuWwv/1vG2fVBQUPNWv2kZNlR5d9vopLjiiisEA5DLMDzi//7v/4TivODja9988w3CwsLg7u6OESOaYhrmzp3bbE999913QvmbCVrKwvYW75gbDV4uyxIXFyc8s9iRuGvXruY+eUFpLCsUpP/YW8ufi+PHj8ONskMyLpklJiYGfem5xd5pboMNWRb+nWcDmYU91Oz5ZezykiVLYByXcPHcf9bGbFpW7PsuMXj5oWT6I8urEr4xptK6DF/jcqzI1tfM1Tdti9+LqcMfDOOHg+uwcW3EybAbn2+m8Zivd6d40JfGnbb92hsPP0S9vb3NfoG6c+zt9c1fWL7HjOHpSSt8XnRxoGRrI30fBWv1JWO3+hxeqb25m72mNRAVlYbYB2gBQIkVdHFkxaiavOBmy7c66fVGOeQEY6iao4SBtq6KifPT1Tse9Vp3uBHmqqs+0zPpN/iFNOBoYSUiRTp5M48FISS2AVXV57FqrT/TspxseOzaicZ58ymgRw1fbxc05lSjOyGOPtGzUHL6O4QOfZ5gDWS004OdP9P8+ej0ZzokFF70HKo7sB8aWqC3lohUBY5uC8aRHY2Io7TDnRF2KvD3sPVnujNtWqtbrK7FQyd/wVBKx3tnwMB2n2/m2uLnNEt7z0Vz9brzXOvPtC1jGaEIwpLEeVhFbA7/yd+F2Xs+x22U5Y0zvXHiC0eJXT/TFgbJ35neJgwVYMPyiSeeaDE19mSGhIQIDjdzzAfh4eFg/K9RjPhXPmZvKge/GYVjpNgmYuOYnzf8/WX7h//4s2YUNjAZOmAU43eHj03fcx1efHB7psY6fwZMxcuLgjVIImghzvadsV8+xwYzj4kNetNnIMMm2Hhmry/X27hxo+BZZoirOWlvzObKWzt3XhvWSnbiOk+ajUmj8HuerKmwEmpra4VVj/G8sRzjQlrX5w9Ee9KROu21J12TNCBGAxywltr0HBBT/HwZys3utrIGvvcUw+f5cni9USm8+t1VDI/3KyHPtMxsYmzE9ecaSjShRt0tPjB4NX21awq3wSu0JVbeWN6Rr5x1je3034rE9VJfLcPZYmX7ySboQe628kfo6XlgTDTBXLwyyrbWncJZ13Tqs6gmXdtdyJOtj4wiHO8ps037BGoRHNOI0/vbGsNmKzjRSS3tWv0tax3kZNC/HDeVXrvk58iJNNCxofCidZZ/osDruzCoPz4q3I+b0lYiq6EpKKhjrUq1HKGBRYsWCYuxl156qXkhycxRd999txCvZKnPyy67TPCGMhyUd7M5ZskoHNTGx2xQ8qJ6+fLlGDt2LJKTk4WFtpHlgA1to6OR8bFsVDJkk6GkTABgjXFhypQpQrwUG7K8Y87GqTlhWy4lJQU///yzcJlhGvzHfbH3mckHeBzs0eY22bvMwl5ehqPyLr3RHuRFj5G9oiNjFhpu578uecLwqoJvAq8WGJDM1GSMJ2HhY/5jTx8HtjH4moXxtbzq4D/Gg7ACjdGKrNhRo0YJ5Sz915E6ltqSzksaEKOBKoobyqXdK5sD1sir6/XvSrgtI4/gKDdU/T0Ale8Eo+oF2g6b7QHlYTV8niqD1yvlUJw4v7I3HZPrb7Vw/6YGDZd4CJnV+Jq6No/+crvF4PWmvaPJQQDDicUI05GxWFIraQAAQABJREFUhMY3Reeaq6Pcvg3y3Bw0zr+ct3CEIvogNnhbRiObq+vIcx4Ea1B6hBNbwy8O6Uab2IeYGsjgpR8ec5IwtA6VhSqU57f0wJgr60znOG3wwdoi/DP2IgQpe57B3t26dJcr8WjkaHySNBdVlOJ6UdoKgce3u8cl9X9eAwzT5HgjxqkmEK92YmKiEJDP+FjG21oSDtpirO7UqVMF/C4bhUYPKwfBsS3EbbGRy7vUbEDzQoiNy/vvv18IiPvhhx+EOCjug41kNo7jCLLAQWocgHbttdda6l44z3ADjqHiPphWzWizmau0ePFi3HfffUIQGgfHMcSCjdjLL79cCH7r06ePMM4HHnhA0AO3wbhkZt26+RzMgs8xTIKD3T7++OMOjZnbaE+6BNLAoG124bOC2V3OEzVGLX766afCjWQGh3vvvVegLOObxuU4YpGFt4/vvPNOAQTNmBj+MPBNb086Uqe99qRrkgasaYC9uyy2eng9/3cWisONqH3UD5rh57f1dIEEBUqibaXLvcBJJNx+qoX3P8qhjVVAM8KNMnHJ4VJngHJHvcC520CsDPWLzgdv1RTtEMbjFTpWeO3q/+aFAg8S+8whYqkZ5NN+74XEzuAbooG7l3mjzoUWxa5//Er6Gdkiq5qedKQ8ZNlIbr9X+1zlHxr28pafWQ7D6H9So/Y1PDmLnMvG9ZAV5Ave3tajjkgiuJOXDqf2eWBURM/w8u2ozsWnxQdxJ6UNHundcrev9fyk4/Y1MNgzFN8kX47nsjYIPL6nQysElof2a0lXu0oDvNvMAWHsKWVogil8gMfA2FfG5bK8/vrrwivDHNgwNdKEcRAZ74Cz8G74unXrBM8xxyEZcb58jZ2GHHjGO+JsKxmF4UqMn2XKMC5vCh9pHU9lCpdgJgWGC7E9ZQqPYI+vqbDnlo1jZo3g+RqF+2UsL7fBcEuGWZgKwzoWLFjQfOrBBx8UbD0eI/dnaczNFWx80yUGL3tveQXA+DDG7vKxUXilYxQ2ZPmDwcrx8/MznhZeGVTNeA52ixuxI6YFWOH8ZyrW6piWld5LGuisBo7XkKlD2/icYlesqNbVQbWtAbV3+7YwdlvUVxB7wUUeUE92h3JPI1SbyPj9pRYujQaBp1UXr0DNw2Qsk3fYVNjgZfyu0oN4wrpBxtPzNoBsv58L2zd42XFZlOmKRPJUmhODjjD/334Fg6cXGuZd2qKIPlDW7ZAGHhAnoSg58YkAawiMndZijJ090MVSIAn9UMhPpZs1eOl3AQlDa3FsqzcGTamGm4VFQ2fHYa/65Zp6Sqm7EcMIt3tHWNNOn73avlDb4RTGzF38XuFefFi0H4WaWjwbPUGCiTjRB4INuNbGrqXhcQCakXWByzCEgT22ptLaRjK9Zmrsmp5nw9VWaa+f1m2ZGrum11q3wawN7Oxkz7Qp3RnXMTXG+bgjY+Z65uS85Wnuqp3PsYUvRlorx1iHXfpGt77xnLXXjtSx1qZ0XdKAOQ0cIw9vX8LvKskAESMulTp4fFUN9Tg3qCe1DOI0W1/mIhi1gmFLQQUuNWTwMlCWgtTMSU3hdniHnw9SMFfGkefITgdz8v5EsIbH+hCm14JeeCues4aFJpj31Bq+/w6y/DzU33lPEzuDyaDZw+tC8GaXs8SI4NvSe2BSzOFvPYKGNcMa7G3wkksGbPQqyODVTL7I7FziB9fh+DZvnDnggX4TaOXlpMLBMM9lbwTjd18iKIOE27XfjeKdhvsodXGEygsv5WxFrU6Nf8ZNdWgwm/1GL7VkqgGGQhw8eFDwjrLHdcuWLUKwmmmZnvyeIQ4MbzBNstEV8+lSg7crJiT1IWmguzRwhAze8f7ie3dfWgPKoou6G8QtBFu0TD9uBmIosCTq2vxuw++ajunSMPJO5BFtYBkw4/xOl2kRIdmEQqVHYGRbfLJix3aqvA6Nl8wlxor4FvX4gDG8LLJSPXTdaPAKsIbYOZSE4jvaurQeYCgM2sJ/NZoKlDXmg181+gby4hvg0acRvnuPwbX2Evh6RFDK3ZarB1cPA2L61wvBa8ljaiB30if7N6VHsY3gDP+Om45QlQ1bIRZ0JZ1uq4HLAlPgRR7fpzLX0986KSCwrYp6xBn2BnMSiN4oRgqzrp6bkz4Wu1oNUn+SBjqngSqycfIoYK2fSNtVVkA0MgRNqL/GyyGeye7G7xq1yWmGk8muWUGwBksGb2GGG9GRNRJmy1ir6VVx5DBUP34PjBsPzaQpLS+eO9IHGw1eHXSJ9sXOmu2wnZN+ZPCWHP8YVfmbiXLo6nZKnr+kNxD2tmofjlZuQnrVbmTVHEWt9jwtW3NJnhqTbez7nbyiSoS6xyPaMxUJXoPRx2cE4rwHImlELTIPuSP7qDviB3eOoqy5Xzu+SaN0uUyldTkZZFP94uzYstRUaw1c7EdpqWOBp7LW44XszXghpokztXU56VjSwIWkAcngvZDutjRXh2ng6LmAtf4iDV63n2uJOow4ZGc6xsvV3fhdU0XPJy/vv08TTpcQC5yUwlQaamWoLFIgYUit6WnIjx2F2zdfQpfaD8rrbgCFGLe4bjww+BFnJNm8stLupSbj8XASCpVnFMozfgKGtG/w5tQcx4bCr7Gr9CdUa8rhJvdEks9ITA2/AeHufRDoFglvZQDBQJqgLhptPfQfvoaSQTEoSAlGQd0pMo6PYE/pr9AZiK9c7oV+fuMRF/FfnNgVgbhB9RS1bdRS97826rV4OmsDbbd74y+RbLlL4mgNzPBPQB3tNvwjhxZgxIJxf8RIR3fp1O2/eFwDHyVR6dhBDHZoQ2qi6zUgGbxdr3Opx16oATZ43chDmSjCfmW8qWpLPRrmk/vTAv62syqqKSL8btiEzjZjl/pziK3hjTNNwWu3k9fJVIopWI0QuC3oyBT79sDt+2XQ9U1G4/U3QdUqste0Plt1jOOVlXS/wcvjYi9vWfrX0BN+0pwcrtiAX3PexYmzlC5Y4YcxwQswIugS9PUdSZ7b9h/HbkFjkXSiCnXT72puWq2rx5nqA+Qh3gJuO8vvDozLX4N3176DwQNDMTJojmBMN1fopjdv55P3mrKDfN53Ptxl7c+zm4bYK7tdEJiMYgpg+1/hPsS4+uLSQMrcd4FJvJcc++ZZwFN1Uhe+lgITOtmuVN0xGpCePI7Rq9TqBaaBw2TwMh2ZXIRXTbWRtpvJRdA4vcl7Z29VCfjdmhzi3x1t76Y71J4fbcdPDWqCNdwWQ+atiY4KiX/XO0ADDx896YSy+6xeBdd1a6AZPAQNVxONoQmji6XOBS5eJ/Dw8vj8YilX/LEPUJKxCnK/83Rwp6r2YmnGiwRb2CNAEe7o+xZGBc+lAMdWLm9Lk6TzuqS+UPy0gtMYgQg2hZIquTtSqB/+uyLuMVT0K8TGonz4nb4On7gOwZennsXYkAWYFn4zor1S22ndcZeYgoyxuw9QQFWqB30QJOlSDdwZNgxZjWfxUu4WxLv5YaAnRZJeQFJfLsPa900eOnac+9i5LvAeZscGpaYcqgHJ4HWoeqXGLxQNsId3pkgnguuGeqIgc3UIdpf17Sz4XdN7fxnBGv4sAfYQTezIc4yDZN+iOFOF6H60AKD0mO5Lv6bEGsfROHU61BfPbGkZmzbW6j3jeOUZnQsUa9Vkhw89AgfA1SceBceXIYqyH51Vl+DbjMXYXvwjIjyS8EDqRxgeNKtD7WuTKJMSKY3ZGrSDBpttw981DCMnumLHigF4MvQgDik/wKaipQJ8op/fBMyJug/9/bvO839W24C/Z22i1MGhuCnE/JjNTkQ6aVcNPBc9EZmUie2xzDUCZ6+/oiWFoV07c7LGiNUQ1RUuiElVw8OXFtb2EIo2PrHTFeoGezQmtdFVGpAM3q7StNRPr9UAY1NLaAd7gAiKQ3maGvJCHepuElG4gxpjg1flHUc0WYQlcBIZQ+wV4eTM/KHgvMFbWaREY50c4V658Hz7XUqiUYv662+EdsAgm0bNHl7VLuf55QmIn09e3g+RHj0Un6Q9IeBwr09cLOBzZS4dp04zUIp2PZHJy9NPWjR4WXERSY3wDtSgcG8fXHXtk1gQ+yh2lKzE77kf4LUj1wpY4SvjnkCyr+N3AF7M2SLgSBfHTiFmCcd42Wz6sFyghV0JRvJa3DRcm/YjcSCvx38TZgmZuS4kdfQdpUZYnH0wvMwdzgavJD1LA63ionvW4KXRShpwBg0crmoaxUARAWuqrQ3Q+8igHahy2NDZ4PUOda7AIKIQxgLy8q4hLy8zWrAUZaggl2kRu+JVGCiHeu0DD9ts7HI77OF1qWdeYjt5b7jRTogqZjJ+9KvGx2mPE4vCELww9HdMj7iZDL6OG7vG4bCXV5F20nho9pXtytRxNSjNcaU/4i4n2MTE0Kvx0rA1uC/1A2KBOIt/HboSbx+7AyUNOWbbsMfJX8rTsfZsJp6IGicEq9mjTamNjmsgwtWb2BomY3t1Hr4oOdzxhqSakgZ6qAYkg7eH3jhp2M6jAcbv+hNONcoaJFdPGNWdDVCPpu1EtgAdIOraAqhrsuHpZAYvT5UNXg3BGH4ppgOCMJTsqERYw1HoRw1H3X0PwhAkEhPSSm/6kCZD0hkC145XbscLJ+5AnrsMVxiG4NmhPyHMI6HViDt+yIF8MkpPKisinrd2JCqlgbDRWiH7mrEYcwWPpAC5F4etxq1JrxEd2h48tXcqBdG9B6ZHs6fkq6vxSu42TPONw9yAJHs2LbXVCQ1M8Y3F1UGp+G/BHjBNnCSSBi4kDUgG74V0t6W5OkQD7OEV491VnNBAVqWHZozj8HPn8bvO5eFlxYfRtCcEEKwhRwPlO/8jGEgkggep0LjgclC+8Q7fG52Ri7fYvkabLQPiDGI/Z7+DVw8vRLB7NJ6N/Cticgl6oCbQsh1FSxmKDERYLBfl5a1GSbYr/bXcTeCkFZPCFuLlERsxIfQqfJf5L7x4cAEKak/bZaR60sVzlDrYQ6bEM5TeVhLn0sDDEaMRSfRwz9I90lDGO0kkDVwoGpAM3gvlTkvzdIgGdOSx5JTCA0VAcpW7z8EZkjmLgGOkppjwu16xUHmQO9UJ5Sp9DtIbldjmPpayzMkRPD6w06M0+J/j4u0marJ6bQ3BA27H8qxXMTPydvLqrkD/gXfCoFfjbPYfnZ5fiwZc3YSMc4qTJ1qcNncQldoAnyANjm42j7XxUPjgpj7/xN8GLhOC6x7dNBE7i38y15RN5z4vPoR9tYV4npId+F5AwVE2KakbC7sRnncxQRtON1Tgo8L93TgSqWtJA12rgY67Vbp2nFJvkgacUgPptbQ7T06SQSIMXtWeBoGdwVFwBlZQTeEOeIU5n3eXx6bYsxuzfvgeodOew77gixFTqYVPoB28sgQPYRyvrMQ+ASk8VrFSXJ+Jt47ditKGXNyb8r5ANSYng8KDtvK9QkagInMlAvtcI7Y5UeW0ySlw/ZMMaTVFS6osB84wlrffhGpibAhAwSlXhFN6YnPClGb/GLYKn595Au8evwezIw/iqvgn6WNquz/kZF0Z3i/ci4VB/TDWJ8pcd9I5J9DAAKImuyFkID4rOoAZfvHo405bL5I4TAPp6YRnX7sWZwmONGXKFIwYMQLyc/zi9QTv+uKLL5r7dnV1RXx8PCZOnNgisDA3Nxfr169HaWkpLrnkEiQnJzfXkd6I04DtTzRx7UqlJA1cEBo4WMVpE4ihwbwTrVkHMtrGl5URnGGYZQOluXAH36jrCgm/m0X8u2M72ILjqil3bKNkEkthIH7dBfHuUBa6wT/WfHKGjoxCoCbrYkjDCcbrHpgHTv7w7JCVgrFrOvaAhAW0ANmGxpo809Odfq8jg9dFp4P81CmrbUX2bURAuBpHNnkzzbFF8VT44smR34DZG37P+x/+e/xOYV4WK5i5wNnUOJVttMoHD0WMMlNCOuVMGriL+HnDVF5gJg2G5EjiGA289957mDBhAtLS0qBWq/HII49g8uTJFMZAdIwkbATfdddd2Lt3L/bv349Nmzbh9ttvx+zZs6FnOgiSb7/9FiNHjhSulZeX47LLLsO9994rXJP+E68ByeAVryuppKSBNho4RBBNzq7mZWWvRLWvEQYqo+nfEk/ZpsFOnGjG74Y4nm7KlmEq9uyC64ofoBk5mpJJLMQsNxV81Qpk+DXY0ky7ZfWh5OHtQoN3c9Eyovi6TuDWfW7IL0IyidYD9KesayxFh99pfalTx/qwcOh9fQXOYjENDZxShapSJbIOW4uqBC6NfVBgcjhcvgH/PnID6rWE1xEpb+XvQo66Cv+MvQhMgyWJc2uAoQ1PRo3HobpirChvn/nDuWfivKM7fPgwnnrqKezYsQNvvPEGnn32WWzduhWRkZG49dZbWwz83Xffxfvvv49PPvkE+/btE4zbAwcOCIbw3XffLXh3P/roIyxevBhbtmzBDz/8IJRp0Yh00K4GJIO3XfVIFyUNtK8B9vAOFgFnUBxshLYfGbucf9hBwumEVV4xUHmGO6QH9gJVNBYiq+YoMmsOo6g+A2p9+0ar/NhRuC3/Dtphw9F4+ZVCMgldniv0Lgb8APt5eHWhiiaDl5gwHCmsg+WZrxK/7l/IozsPTwz8Fj4q8zhklUcIwUsmoL7iiN2HpE3pJ9rgDYrWEJyhAUe3eFMQnXV2EGZyeHTAF8iuPSIY9XVa+pBbka1VOVhaegz3hg1Hsod5fVhpQrrcDRpg2MnFBGng1M+cJEQS+2rgs88+w3XXXSdAFIwtM1vKyy+/jJUrV0KrNQ/D4ucMQx58fHywatUqXH311UhJSTE2gQDi4z558iQmTZrUfE56Y10D0jLcuo6kEpIGzGqgTG1ADv1G3GnN4G3QQ5GuQf1CK7gHs72IPyngd+1MR1apLsKO4pU4VLEOZ6oPoEFHoOVWEuoejxTfMRgSMB0D/Cc3p8uV5efB/ZsvwVRaDVdc3Zw5rfCMK1xD1Ugj/e2pBEacy7zWqlmbDtnD60JwYFmZjvC8jnmsaSkI7aO0R7GTkjgsiHlESOhgbZCBiVcia+vDaDh7Cm6+fawVF31dm5JKFHfbIcvPhz4iwmo99vKu/r9gpO3yJFxvjdXyqYTr/euAr8jLuwivk6f38YHfwFXelM64deUKbT2ez96EEV7huDFkUOvL0rGTa+DRiDG47MR3eK9gL56MHu/ko+1Zw2MYw/Tp09sMmjG6SqUSR44cQVhYmHD9ww8/FIzcoqIi/PrrrwLOtw+xsuzZs0eARLRuxJd2eSSxTQOO+WWwbQxSaUkDPVID+88xTg2x8txRHlULxpiGKLgcJc343UEP26WL/Lp0rMz+D3aX/CIETiSTQTsr8i5EevSFrypYSKLAnr9SSlzA3t6jFZuxsfAbeBAWlKmuZvhfh9gly6APDET9dTeAnuTCuLSUdKI0V4XU8dVgM21Zvn0MXh0ZvCyyIscYvDWaCiFRw+nqfbij71sYH3qF0J+1/3yiZ0Cm8EBFxgqED/mrteKir+sSiZ6MqNw4FbNahMHrHaBD4tA6nNzlhbhBdfDwsU5H1cdnOB7tv0Qwet85diceGfAZ5C5tfzLY2FUTj+/imCn0ubDuQRY9Salgl2ggVOWJG4MH4sOi/ZjsG4NxPtFd0u+F0Al7Yhsa2nrOGZvb2NiIIMqeaBSGP8iIctDT01PA8F5//fXCpeDgYCFQzVhOeu24Bto+vTrellRT0sAFpYEDZw0IIIaxGCvQSMVhNfR+MuijHUhHRtnVWLw66eFlD+7yzFewJv9zwbC9Iu5xgbPVW2k9ijur5gi2ELZ1U+G3WJP7CabEJ2Lu5HcQQFHHRmFeWL3OBeEJjbiSfgfeywTKCNkQ2Mm1gJ4gDQxmkFHaZgww9maf1+L6LLxx9EZUqUvJ6/klUv3GiW5YTsaub/RMweANG/yXFlHXohsxV1Clgq5PEhTHj0I9dZq5Em3O8SIj+5g7Dq33wZj55FoXIX19RwmY3reO3oLP05/ErX1fa1Hr25Kj2ExwhlcpbS0bTpL0TA0wY8OXJUfwadFByeC14y0cP348vv/+ezzxxBMtWmUcb0hICKKiolBYWChce+edd4iOvK1JNnz4cCForUUDdHDTTTdh6tSpwmvra9KxeQ04DlBovj/prKSBXqMB9vBa8+7yZJXH1A4NVuM+mvC7xL/bCfzuqaq9eGbvxdhQ8DVt1z+CV0duwZzoeyHG2OUxxHoNwKLEf+Ad3ZtYeHIAdkYW4olTCwTMayMxGbAwnMHNSwffEC0ua9rJw48FwqXO/adyAfPxyu0cuHaqah8WH7wUGn0jnh78o03GrnFC/vGXQV2bi9ri3cZTdnnVpvaHLCcbLtXWMbbcocrNgP4Tq5F30r1NMor2BjQ4YCpu7PMSNhV9i99z/9dc9FR9OThQbX5AX0wnHKgkPVcDnnKVkCRkL/Enb6vK7bkTcbKRL1q0CJWVlXjppZeaGRcYe8tBaC+++KKo0V577bWCUcy4X2Z5YNzv119/LWB758+fL6oNqVCTBiSDV/okSBrogAbqtQYh4cQwK3AGF8qsJsvVNgWsdaAfsVVqCrd3in/38/Sn8NLBy+Cl9MMLw/7A/JiHoZK5ie2+uZyssAD+f6zHvODb8NqYnZgWcTN+y30fT+6dgv1lqwWDNyy+USgfQF7diymb8Hdk8HICj86KLoyYGgrNB4F0pO2dJT/hlUNXI8g1Gs8N+RmRnn070gy8w8ZD4R5MXt4fOlTfUiVtaj/hkuL4cUtF2pyPH1wHv1AN9q/2IU97m8sWT1wUfj0ujrgVyzL+iSMVG9FAFGR/y1qHcKK1ejxyrMV60oWeo4GZfgno5x6E/9AihrPlSdJ5DTA8gfG4hw4dQkJCAhITE4UAtMceeww33nijqA68vb2FALeNGzciNjYW0dHRWLJkiWD0+vnZIQBC1Ch6RyHJ4O0d91GaRRdrYF+FDmTzYrgVg1dxnPC7NDaBocFBY1TXFhD/bnaH+Hc5Gvjr089jfeEXSPAeimcGryCqrT4dGynh0ty+I65df380XjKP8Lw+WJjwDBYPW40Qt1h8svcV1J1VwCemtLn9awjIW0j276ay5lMdfqMPU0Ce33mDl3WyIusNvH/iPgwi7+aTg74T4B0dHZiLTA7/uAWozPqNjMwmY7+jbZnWM1AEtz46Bopj4lkgGGI7dMZZVJcpkL7HNgjCwoRnwVju90/cjxez1yKnsQovx06Fu9xxUB3T+UrvHasBZg94mPiT0xvK8XvFKcd2dgG1zhjcpUuX4syZM0IA2sGDB3HzzTc3a4CD1viZYw7OYCyUlJSE33//HTk5OQI7wx9//CHAGYzXpVdxGpAMXnF6kkpJGmihgV3lWnhQnFSyV4vTbQ7Y4NXTVjtjTB0lRv5dbxsTTugp0OijtIfxZ/4nuCb+aSF5gkLWcTCtcvNGyPJy0XDlNbR/fr4dNqD/NmgZ5srfIzoyDf5bPgm7Sn4W1DGUFgx9ye5amt957egiiJqMgtbIPdXhxhp1dXjvxD1Ykf0m5kbfj/tTP4BKbgWkLaK3gITLodMQH27uGhGlxRfR9utPCSjSQREwoisFhGvAnt7j27xQVyX+J4AD1u5JeRcV8lT8VpmDh8JHShRkorXeMwqO8I7AWO9I/K9wHzQG64GNPWNWzjFKDkjzJ2dAZ4SNYqYqk6RjGhD/tOtY+1ItSQO9UgM7if6K4QxyK0HpipOEuUo5b/w5QhkCftc7DkqPUNHN6+nH7IOTD2F78Qrc3vcNzI66W3RdcwVdKsrhuuZPaEaNgS7OPJ7To2QMAiMbEOffl4zKe4U0ttWaciyMBLZXAJl15loWf05PkAaBmqzEhr16k+Y5TfDig/NxoHwt7k5+R8g6xl4ve4i7fyrc/FJRfma5PZprbkPbfwBcCNOnSDvRfE7MmwGTq6FQGnBgjZUtilaNVRtckeU6Cz6aE3CvX93qqnTYGzRwPy1k8tTVWFFm22eqN8xdmkPv1oBk8Pbu+yvNzgEaaCTHx36CNIz0s2IM1ekhzyb8brKDDV5KX2urd/ez9L+Rl/Un3JH8pkAj1lk1uf7yEwwqVzTOnmO2KU2ji0BHFpVkIHqrz3Fb39cFLOhTe6ciTP4nvMlb3lkvL3t4WWQFthu8e0t/x9/3XyJkFnt68A8YE7LA7Dw6c5K9vFX5G6FpOA/p6Ex7XFcfEgodbZkqjhy2qSkOYBt0UTUKTrkhL+08i0Z7jWgI9PtE5jr4Ktxxa0AQfiLaOg7qk6R3aSDVIwjTfOPwcdEBSixj+3epd2lDmk1v0oBk8PamuynNpUs0cJDYGdRk9I60sjvFySYooRh5eB2HcVTX5AoMAF5hY0XPfXnmaxRx/w1F3v8T40IuF13PUkF5ehqUR480Gbvu5rf/izJcYdATHVlig9DMxNCr8dLwtYj1HICPTt6GONU6rCzUo64Tv6/6EDkM9ESTF4jH8aqJPeKLU8/gneN3oq/PSLww9HfEeQ20NNVOnfePn0/1DajM+KlT7bSurO0/sCnrmoWsTa3LG49j+tcjJLZR8PLygsSavJ6/A+n1ZXiZKMiui/8LpVNOoXv3EFiHkvQuDdwVNgylmjosLxMfENm7NCDNpjdqwHHAwt6oLWlOkgZIA7uJwtSTPJL9OHFaO3BRRZoaBncX6KIc9zWrpnTCLGL5d9dkLcHPOW8TC8Mj4Mj7TgsFqrn+vJLmGA3t8BEWmys47Qovfy04AYJRAlzD8deBXwoJKz49/RpqdRfhrfTdeDJ5RMf4aglfwrAGeZ44g/d01X58nPYIihuyCcP8DCXWuLNj/RonZOVV6R4C7/BJAqwhOPVWK6XFX9YOGATXDesIy5sGHaUctkU4gG31p8E4sskbU660XHNVxWksKz2ORyJGY5BniFDwjuS38Pz+Ofgh698UnPis5crSlR6ngT7uAZRyOAGfFR3C5YEpUNK/ni77V7tB5d7OA7unT1Aav1UNOO6X2GrXUgFJAz1TA7vI4B0dSIwALvp26XsUJzXQJtEPhcy696yjmmA6MlefPmBjypocK9uG9w4+KHh1L4t91FpxUdeVe3YT920R6u6+rzl1cOuKHPtSeNoNsQPNg3Qnh10L5npdtPcoVhZ5Q99wGfH5voB478Gtm7J6LASuWTF467XVWJ71Gtbmf0ZUY8n4+5BfEONlm6FodSAWCgQkXIGsLQ+gvoIwsP4pFkrZdlpP5PV6P38oKVOTrQavl78OqeOqcZQM3n6jGmlMbfvOaKjEP3I2Y4pvLDhBgVGiPVMxjwL7OCPf6OBLO3S/jG1Jr86ngTtCh+KqyuX4sewkbogY4nwDFDkinyADrvsbFyYvhV3FAA92egg8PHZtWGrMQRqQDF4TxXIEpOpcdLmRIoQ58JxB5ISP5AAaa+Ph/NzWyjjDfIxj4MhVFg8Pj2ZibuM1Z3ytJS6yw9U1eDaOPJZetIVuia+SiGXlZ4qgv9zfofejtngHAuNmWe2jvKEAr2y9Hon+Q/Hg0PehlIvDbbZ7D4gEHWv/BAYPgcfAQRaLFmXKoG6QIXGgzOI4veGN5wfH4fbd9chWx+KFA3MxPvxyXNv3acpmF2CxXutOZcTx67LqrNnyGr0aa3OWYGn6y2jQ1mAhtb0g4SEoZPb1XrX3mfZMvRy5u55CTe5PCIkZ2Xr4HT8eNhyyHdsocNGDftdt+2EfPhXIP6nH1hWuuPReL7jIznvB6nQaPJG2DsGURe2NfnPgrWj5uVnY70nsKf8Nn59+Eq9O2ECLQNv67viEm2o623NazHz4Od4TntND6LdvVlkSlpQcxi0Jo4WpOfI5bfFZKkap7ZSRF6vh8czpdnfj2qne7iXlzeHAFDOrxHZrSRe7SwOSwWuieZ1OJ+S35lP8QOIHU329c+DT3LQayOl3qL3x8Hj5B6C9MibTdYq3PF5+iHJecc4g4+yypVQvJEkYGygTcqRzTnRzIstUw7PBgMYEGXQO+gw1VGUSfjcfHsGj273nOkoS8Or+G4RhPjnya2gJgKxF5z/XinVroKiqQuPM2TC0M8czRzwpy5ce3iG1NE5z2mo6N9rbgGiCAHt7vYU5McOxIvNNbCv4EePCF2Ba2M3o6zfKcuVzVxRhLnCnZB8NRTUw+DQZX3Xk0d1UQFnCcj5AWWMeRodcKvADB7tHQ9OoBfnhrbZrSwFrn2n/uEtRlPYdwgY/QcalfQxEF2JrcKP70XjoIPQpqbYMVyg7fLYGaz73w751Lug3/rwn/vHTq5FTfxZfkaGu0OhRr2l7A29JegWL9y/AyvR3MDv6Lpv77kwFfk6z9KRnHi+I+K8njPm2kMG4qjQdy3IP4s6UCQ59ThvvZWc+D2brcsBFZiN0I9xhCLCTyUO/xYrV1TBUnYdome1bOulUGrDT3XeqOXV4MLzCNBpdxtWm8bjDjdqpYpNhdX585prlhyiXc5YxmxujpXO82OgJ495MAfYB9BubRNyxtbVai15p1YkGGAjJ0BhHHmwHGfKVeZsFdboHjWxXd9+eWSxE07847lcEuIWjvK7c0m0Qf76hHm7r1wq4XU1AYLtzzD2pQhgFq+nJ8LawPmju92pymLx5xgWPJdyI8SOvwsair7GaeIK3kuEb5p4gbJ0P8p+KOO+B5E1s+/gysMFLrdWdLsX+sO3YW/Y7ZXgjujRKDTw8aDYe7PcJpUDuL/TnqM8bLzxZLH2m/eMvR2naV6jIWQufSHKv2kMio6AiWINs/16o+yTZ3CKneu43RoujW9wRnlQjYK2/KD4Exu4ujpmMBJWvxc9YotdwTAy9BsszXsPIwLnwU4Xa3H9HKzjbc1rMPPg5bfpbI6ZOd5VJVPlhok80Psrfi9uSx1n8TNtjfEZvvT3aMteGbr4v9APMB9WaK9/uOdrBY4NXkp6lgba/GD1r/NJoJQ10qQaYL3Ys7WAZjRpLnSvSKeEE02R5NEE2LJXrzPkaoiNjbleFW4DFZg6Wr8MfeR/i8tjHMCBoosVytl5Qbd5EVBVqNE6f0W7VqlIFaisVGDhF3I/DAjJ438tsoih7KMEds6LuwOUpD2FLxgpsLV6O33M/IMzoW1BS2uMoj2QEuUXDS+EHOcES1LoGWoSUgXK7YfmWxfhjwHLBSJ4ddQ8ZZJQi2C2q3bF21UXP4OFw9Y5H+env7Wfw0uC1gwZDuWsHcBl5rGnnxFYZfrEGmUdl2LfKF+5zj1CK2d1YGNQPcwKsG9BXxz9Fi4s/sDTjJdyV/LatXUvlnVgDt4YOwS3pP+OngmOYqApz4pFKQ5M00L4GbH8qtt+edFXSQK/VQFEjkEG7vbfFWJ+i4hRtlDuaf5cYGvzjmOrKvJxVlwgsBP38xgtZw8yX6sDZujqotm6GZuRoGMir2J7kpblBpjAgNI6UJ0J86Ik0lxyEywuAu+MAd1ovMC50SOB04Y89tcz9erp6L3JrTxJEIRd5dScJZqIlI9gVvsoQVAVUYab+ekwZ8RAZvPEieu36Ihy8Vnj4bWgbz0Lh6muXAWgIS63aRDjaNNIHZWCzVZQEzx06owpbv/fH2s3FGJIUikcjx4hqxptw1pfH/gVfnn4O08JvQh+f4aLqSYWcXwODPUMxwjsc/z2zDRNTLnf+AUsjlDRgQQOSwWtBMdJpSQOtNbDlHBJgXPs2Hlxq9ULyA+2cJnxh63bscVxfmQYtJTDwChtnsblP0v5CLBJ63NH3LSKKsJ+nmY1daDRQXzTVYt/GC2zwhlEgmUJ1PhDKeM3S67WUee07Mnh/KwKuoPemwkZtqt9Y4c/0vOl7j4QKeJYGo8adoBZOKv6UhKLg4OuozPoJQX2b8NWdHaqeYA36oCAoD+zvkMHL/fvH1yInvAzjT07BtPFFUNjwuZkafiPWFXyJr04/j+eG/GR1F6Sz85Xqd50GbgsfhnvSfsWWs9kY7UHbMJJIGuiBGrDfr2APnLw0ZEkDtmhgcxlx73oBgVYSp8nPUMIJaliX6DiDl+EMFPEErxDzgVzryfA4VLEeN1NAkb+rHbchGxqavLsjRsHg69eu+mor5ThbrEREUkO75VpfTCR89Ghq+uu81lfEHTPvsTzXvoFo4noWX0rlGSEsVhjWYE/RDB4KxfGjBB4X51E37ZtxpU9nrccvyb/Cw0WFnC0hppetvpeRJ/66hOeQUXMAO0pWWC0vFeg5GpjgG0O846H4pGB/zxm0NFJJA600IBm8rRQiHUoaMKcBClDHDsLvThThNFSc1lCaXTJ4ox23gVJduBUeAYMgV7WlzSuuz8I3Z/4h8O2ODLrE3HQ6fE61bauA3VVPuchqG7kn3QSKq/A+thm83PB15NlNq6UkHxXiPcPGAeliFJDVGuBC6Z+dWQISr0Jd2UE0VKbbbZiaIcPgQt53xTEyem2UtzK3YV1lJh5LGo5Bk2qQfdQDxVlWVnet+hjgPxkD/afgu8yXKRuh7fe9VXPSoRNp4N74sdhTnY+jdSVONCrnHUppaSk++eQTs9SVP/zwAzIzM7tk8F988QXFNtDDVBJIBq/0IZA0IEIDnGyinozeSZbjw5pbkZPBq4sn766DEk4YKL99bdFOeIWPb+7T+Ia9dJ+k/xUeCl9K3vAP42n7vFKQmnLrJmiHDoPB37oi8sjgZeyuys12o3USLSyi3ICvcm2vq4tu8qzLs5zby+sXPQsypRdlXvvOPveHWjEEB0NH0AYlsTXYIn8QG8N/s3fgDkopO9M/EQlD6+AfpsaB1RTZbuO64Zr4p1HRWIA1+Z/aMgSprJNrYH54f4QRH/MSYu+QxLoGfH198dRTT2Hz5iY2HWON4uJi3HjjjWa5wo1l7Pn6l7/8BeXldmDmseeguqktyeDtJsVL3fYsDWwgOEMwObv6t3WotpkIe3i1DoQz1JcfgU5TBe+wtgbv2oLPcfLsDtya9Co8yei1pyh374ILBaw1TplqtdkagjNUFKoQldIxLx+vFdjLu55o4HLqaKVhg+jDKSEI2byKLI0Ntbq+qEzhBv/YuSjP+BG8iLGXaIYOhzw9DS7V1aKaPFRbjOezN2FWUBLuDR8u1GFmNQ5gqy6XI203YUxskCjPFEwgVoxfct5FrYZWipL0Cg0oiE7t+tDBWEu7APmN4j5bvWLiHZwE8wqzYfvVV1+1aOHrr7/G3LlzERhIq3oSpi88deoU0tIoLsOEwlJDOzWHKXtifn5+i/p8UE3f7aysrObzXLay8vx3jXntq4gj3ZwUFRXh0KFDQhvG6zU1NQLH8pkzZyg8w7mfm8Yxd+RVMng7ojWpzgWlAU6mtoEMr8n0fDpHsWpx/ryNLqskLuSEJi+jxYKduFBduIWgAq7wDB7WopWyhjx8l/EvjA+5EoMCrEMOWlS2dkAPZdXmDdBSggNDsHVsZ+5xYmegTCm24ndNhzE/DHCjvAxLMmzEo5K1rItRQp7p3B5enivDGrT1Jagu2Gg69U691xJbA39QFQet4y3ZcHkk40/0cfPH6ymzWwSa+YdpBE/viW3eqKuy7afiMmJs0BBN3K+573VqLlJl59LAFcGpUFGylJ8r0p1rYE46mltvvRXfffcdMThSVspz8vnnn+P2228Xjo4ePYqBAwfivvvuw5VXXonU1FTBG5ubm4sBAwbgiSeewMyZM7Fw4UJjdTz99NPo06cPbrvtNowYMQJnz57Ftm3bMGvWrOYyDJm4+eabm4/5DRvTCxYswOzZs8Fe3+joaPzyyy9CGfZEz5s3D6NGjcJjjz3Wol5vOrDtKdabZi7NRdKASA0cIWcGZafERUHWKygoYI1FgDRYL96hEozf9QyhVLJy2vM3kc9O/Q2ucg9cS4FD9hY2nmTkQVCL8O5y3znH3RGW0Ailq+2QBOPYvQgCfRkFhH+bpUadjQ5QXSwFrmU7v6fCyMlbdtqOsAZKCavrmwzl3j1GVZp9rdap8WDGKqgo2OzNhBm0uGi7SOs/sRpypR6H1vmYbcPSyQDXcEyLuJmShnyKSjXRbUjSKzTgJVchgHYmauizI4l1DbABy3+//fabUJg9q2ygTps2TTjesGGDYHyuWrVK8Lr6+flh7dq1QvnBgwcLr3v37kVUVBTYC7tlyxZ8/PHHyMjIwJo1awQD1mi0WhvN8ePHwTCLffv2YfXq1YLhvGTJkuZqbBCXlJTgjTfeaD7X295IBm9vu6PSfOyugbXk3fUiTyMzB1gTZmjQe7hAH0YVHCB6Tq5QvJfgDBNatL6jeAUOV2zA9YTb9VJa4U1rUVPcgWojeXcT+0AfFW21QmWxAlWlSkT3q7da1lqBRVEuqCFH7cpCayVbXtfFKiErJCu5wTY4RMtWuuYoIPFKVOWuIU7e81uSne1ZM2wE5AX5kJnZDuW2NURX91jGGhSqa/F2wkwEKz3Mdsn464GTq5GX5o6iTNsC2OZG3ydkw/sp+22zbUsnJQ1cCBpgT+yXX34pTPWzzz4TPLPGxEX33HMPwsLC8Oyzzwoe1vT0dCFl/YwZM7Bz504MGjQIL7zwAm655RZ4eXkJ5y6++GJ4eDR9X5955hksWrRIlBrZk/z444/j9ddfxw033IAPPvigRXrriRMnCjs8nAmwt0rvnVlvvWPSvLpcA6spKJnhDEoR3xb28AreXWvYhw7OorZ4D+E9G+Edft7gZZzkV2eex+CAaRgVPK+DLVuuJj95HPKiQqgni4NJZB91J8+uHuGUTrizEu3ugulhCgpeA3EKi29NG6+AC5VX9ABYgz8loTBQ2uUKyiZnL9FS4gmDO92HvbvNNvlizmbsrSnAq3HTkOQeYLaM8WTswHohgO3gWh+rqaGNdfiVF16zIu/AxsJvUNKQY3pJei9p4ILRwNVXX42NGzeirKwMS5cubQE1eOihh/Daa68hKSkJb731FsaNGyewOsTFxeH06dN48803UVFRgbFjx+LgwYMIIp5tU4wt43bz8vIEXTIW2CjsDW4t69evBxvLbGw/8MADgvFrWoYN6t4uIn7Ce7sKpPlJGrCsgeMEZ8glu21GsOUyplfkGVrHwhkKthAVGaV+DRjQ3O2yzH+iUVeHGxNfaj5nzzfs3dWFhQvb5NbaJcchco65IzK5AXKCJNhDbktwRQ7dg00UOChWGMNroKebPMP5YQ0qjzBawEyiVMP2gzVwamGBk/fAPo6KaaG2Dwr34efydDwVPR7jfKJaXDN3wGu3wdMpgK1MgTP7zXuCzdXjczPJ4HWTewrpoC2Vkc5LGujNGmBDcv78+fjrX/+KYcOGITIysnm6DFFgPC0Ht7m6umL37t0C1pZxvvfee68AfXjvvfeQkpIi0JhdeumlAusDG88sTz75pGBEcwBcJtGcGQ3d33//vbkP45vt27dj+vTpePTRRwXsL8MsTI1nY7ne/CoZvL357kpz67QG/iDvLsMZxrfvBBP6cSmjgLUqDlizk6VnZvTVhZuFhAUu5zJgpVftETxoHCQU6Hb+QWqmaodOySh4QnHmNNSTJouqX5jhioZaOeIG1okqL6bQmCAFUsj5sIS8vKJFRYFrxIPMEJOeIBy8Vl9xlP6O2W24GkoOIiP+TSERxblWV5adBBu8t4UOwWWBKaL7CozQIKZ/PY5t8Ya6nixgkeKu8MbsqLuxteh7FNVniKwlFZM00Ls0wEFqDGcwBqsZZ8cBYmwIsyHKsIWLLrpIYGxgr3B2djYYx5ucnIz4+HgheM3f3x/333+/ELTGxjMzLrBhzAFuXLdv377o35/o4wgm0VoYxsBBcpMmTRI8xtwue5EvJHHcL/OFpEVprr1SA7yFzultp5N3VxSc4Zw3URendIg+tA3lqC8/isCkRUL7OoMWn596EkwDNSPyNof0qdq0AXofX2gpg5cYyTrsDi9/LQIj7Wto3kCOyKdPAOxxTxVBDcdjZWiJIq1nBNf4Rl8seO7LTi1D1Mjnxajaahk9BbqwZ165h7xGAwZha1UOXszZgjn+fXBf+Air9VsXGDCJsbxuOLbVG0PI4ytWLo64BavyPhS8vHcm/0dsNamcpIFeo4ExY8aYTUBx3XXXCQwMDE0ICGjpVfnjjz9QRzSQzK3u6enZrAtmVGAjmanHvClA1SjLli0TqMm4LFOiGYV5f43CAXDsHea+GNrwyiuvCJfefvttY5Fe/Sp5eHv17ZUm1xkN7KYYImZnmBMirhXePjcQ5lQf6piANWZnYPEOnyi8rsn7FLm1J3BTn38JwUHCSTv+51JRDsWRQ1BPoP7k1ufUUCtD/ik3xA2yn3fXOJ2ZtOgIoZipz23w8mr7UOBavg4utc4fuCaTu8I/foGA49XbMQJeM3IU5CdP4HhxJh7PXIsRXhF4LmaSUa02vbp769F3VK0Aa2B+XrHCzCGXRN2D7RRYWVh3Rmw1qZykgQtCAxwk1trYNU6cg9NMjV3jeZVK1cLYNZ5nlgdTY9d43vSV4Q/GoDnT8xfCe8ngvRDusjTHDmngJ2IGCHMFRopgZ+AOFITf1XKGNQY9OkCqCzZD5RUDV69oymRViB+zX8fE0GuQ5GO7t07M8FRbt5BrWwnNqDFiioO9uyyxAzrPztC6Q/awX0uIjT/JWVFIeF4xok1Sgu+E/JR9vc1i+u5ImYDEq6FTVxJjw+qOVDdbh5NQZHur8GDuOkS7+uDf8dOgPAeHMVvBysm+o2rg6qnH4Q220ZRNDb9JCGJbmfOWlR6ky5IGJA1IGnCMBiSD1zF6lVrt4RqoJiqs1URHtoCgUGLtV3kmMTTEOQ4lVF2wCd4RTVjabzNeJK+uElfHP+UYTTfUQ7lrJzQjR1P2h5Z8v+Y65OQcZw54ILJvA9zIIHKEXBkBIr2ndMN54lrXRyoEj7sivWfAGjwC+lMwYn+UnfpW3ARFlKogLPOt0xPg1qDBf+NmwJN4VDsjClrP9Z9QjQLy5Jfm0IFIcZW7k5f3buwoXolCCcsrUmtSMUkDkgbsqQHJ4LWnNqW2eo0GGLurJruNs32JEZfKpgxrjko40VCZDk1dIXwIznC8cjt2lqzEFXGPw1vZEvclZqxiyih37qDUPJomOIOICgWnXCkblwKJQ2tFlO5YER9aS1xBiSi+L6DUmrQgsSqUcY1TPCvSeoaHl+cTmHgNZV3bAnWtSKu+HSXU6zR44PQqnHWV49P1mQjNzGmntPhLTFPmE6TBoQ55ef3wU7aE4xWvbamkpAFJA/bSgOPcUfYaodSOpIFu0MCyfGAc2ZIR1p2bwuiYjoxF66CAtSry7oIyYrmHjMQXR65EnNdATAlbJPRp9/84jfDWzdAOHASDn7+o5tP3eMIvVIOgaMcal9dT8No3ZAt+R/fn1hjrQ9Mmq+D2CxnhHIFIBrCzi1/8fOTte4m8vMsQPviRDg9Xy4klCLN7pqECH/SZgxi3IuhpEaNLTu1wm8aKvOMxgJJRbFsegNyTbogiCjoxwl7eWZF34fvMV7Ag5mGEuMeJqSaVkTRgFw0ovq6AwbfKLm2BHieS9DwNSB7ennfPpBE7WAN7KVjtFMVdXUNb6GJFQXAGgysFrIWLD+YR2zaXq87fCM/gYVhfuhz5dWmUUe1Fst8c8/UV0ghXVREV2RRRQ6woVNL2tiuSRjjOu2scSDgtQGZRECHDGtgDb020KYTjbTRA3gMSUPBcFMSx7BdzCXHyLqPobBETtKCAF7I3YWd1Hl6hxBIDPUOgGT2W6MmOweUsfbjtIOGJjbS4acTRTd42JaOYFnETPBW++Dnnv3YYhdSEpAERGnCXwWWwJ2QqoikkSj27/DW4CG26BImH9YgYqVTEwRqQPLwOVrDUfM/TABtTkWRYTbQBLcAMDbpY+jo5wIvI6YRrinfCq9+tWJH1BiaEXIU+PsMcpljV5o3EJZwIfSS5U0XIyZ2ecPfWISrF/sFq5rq/JRr4lYLXfiHYyeUEcWhPtH1UMNAaRHFcDV1Cz/hxCuxzLbE1/EiLnA3wiZza3vTMXns7fzd+rTiF56InYqJvkxucUw27/vGbgMtWXzzTbD1bT3LK4fVfBiHzkAcShohj5uAkFJyM4ses13FpzEMIdqObKYmkAQdqQEaRx65v9XVgD1LTPUUDjnER9ZTZS+OUNNBKAzlks62nYLVFkbbZruxBdBScoaZoJwy6RqxxOUk7aQZcFf9kq1Hb71CengZ5QYFo725VmRx5tK2dNLIGMsc4t9tMLomSUEygxcinBEm1mm6YvO5s6LLB21PEK3QUXH0SUZr+jc1D/qbkKD4rPoh7w4ZjQWDy+fpEb8RcyspdO9pkXjtfyLZ3AZSMIiKpAce3eUHXhOgR1cD0iJvhpvDCrznviiovFZI0IGlA0oA9NCAZvPbQotRGr9HA52REeZKj9jIrnkPTCbvU6CEv1TmMoaEqfwNKvHyw/exqXBbzKHxVwabd2/W9auN66EJCCeuZIqrdE9u94eqhR8JgcR4+UY2KKHQbOQazaXGyhhYn1kTTXwUlG7xWrWNrLXXd9cCka1GVtxZqClQUK6srz+DfedtxZWAqbg8b2qaaesxYyKqroTh6pM21jp7oP7EaDTUynN53nhjfWlucfW1GxK3YXLQM5Y0UgSiJpAFJA5IGukADksHbBUqWuugZGihqBFaQfXEdeXc9bPBWMh0Zi6MYGtjg3RDognD3PpgeeYvQlyP+k+XnQ3EqvSmNsAgutrMlCuQcc0Py6FrIuxgtMIy4kYcSFezHWdY1oSWD16WecLznAgut1+j+EgEJVxIdnhLlIinK9tYU4JmsDZjiG4u/RY0zOwF9dAx0UdFQbt9q9npHTvoEaYWUwyd3eEHTKD4o8GLKDKiUueK33Pc70q1UR9KApAFJAzZrQDJ4bVaZVKG3auAjMp5c6RtxPRm8tgjDGRgnqiPeV3tLY3U29utOIw/lWJT4gkMyqhnHrNq0HnpvH2iHiMMHH9noTdhd8u46kIrMODZzr7cTPPVkLbCpzNzV8+e0SYTjJYNceYRWND1EFK5+8IudgzKCNRj0unZHfbq+Ao9krMb/s3cd8FFU3fdsL8mmF9IrhN6b9G4BUeyioggWbJ8d/dt7++x+Kqggiih2xELvvRNaqCkkkF422+v/vokLKZtkk2zIAu/yC7s7O+/NmzOzM3fuO/fcLupwvJ4wkmjk9TuelksGUYGUExDney+y2nmwDlaLCEypw1NjiWuM2rD29AKUW86WPvW0PV+PI8AR4Ag0FQHu8DYVMb7+BYkAmx7/jaK7t1GeVkATo5VMocEeR86utH5Ho7mgFeb+g43BTvQMGoWuwcOa202j7UTFxZDu3Vuluytt3HHPP6FA/gklug7TQtL46o1uvzkrDAkFOhGfdzY9qDRoVHzB1lEOafr54/Cy/QnrcCusxgJU5C6rd/cKLXrcf+IfhEvV+CBpHD2wNXwwGI/XQXxe2SbvRXn9guxIIkrL0e1+sFAWvKfGktdEpDSyJHeWp034ehwBjgBHoNkIcIe32dDxhhcSAh+eIEeXfIUpzUgaFyqssZLCrWB/FXwDMyk/TE59uRV6P9ul8pcf6YNTkK86u9T9O5agtHdlAEKiLYjr7JkGq/ueWr70ngRgXyWwsbThvqzdFVUFKIyOhlf0oW+ZDB2rvFZ8+Bu3o9LZLXjgxBKiJjvxScpldP4q3K5XYyE9zLBS0bLdOwEjPeV5yToO1MFhF+HwNnoC8dBY0ZRRUVOw6vS3qLQ2cgA97JOvxhHgCHAE6kOAO7z1IcOXXzQI7CBpUpb89EBS07i7AkAmB8SnKWEtwfsO76nKQ9gmzsFw5UAS6SfPrpVMpNNBkpMNy4CBHpURztjsD125BL3GVXhcdrmVho4RFOVNo5n0zxuJ8lp7KCAiZoBs//mj1sAwC0u7HbqCzTCWH6kBoZU0eh/PXIFTFh0+Tr4MUXLPHU0r0RpAxUVk27fW6LMlHxi1hVXZO77TDya957eVy8Zd/cwAAEAASURBVGPvoccsJ4/ytgR83pYjwBHwCAHPr0wedVf/SmazGRs3bsSmTZtgtVYl+bhbu5imVpcuXYrDhw/X+ZotW7ZsGdg61a2+Njk5OdiyZcuZv6NHj1Zvxt9zBGClgN/rdFp0JH9hUrumAyLJsUFEVXdsSQ1PJTe9Z2D+4cehIidtYspTzWnucRsZVVVjnqt19LhG25TlS8ESlNL66xEU0QQtqkZ7bt4KjK56byKQrgU2NBAkdMRKYQ+TUGTz/KI1BCdOhEQRjOKMuTUAepkKS7BEtXeosESamrz+JpgzMAi2Ll0hZ8lrDu9FvNMGEqGafgzs/PDUmOLIiHY3Y+XpedBZyzxtxtfjCHAEOAJNRuCcOLxGmjq74447sHr1asyfPx9PPvkkVREiL6GW7d69G1OnTsWRI0eEdX777bcza7z//vt45513wNaZNm0amDPLrKE2X375JebOnQvWD/vbtm3bmf74G44AQ2DOSeAEKWo9275pursu9IQKa+R02eO9G+HdU7oSB43pGGEIQVCoZ0lkrjE16dVkEhwfa9/+cGo0DTa1UWLS9j+DoQm1odNg4hH4iI0Kq3pg+TSr4QFZeysg20UOr5trT8Mt2+5bsUSJsPa3oDTzV9jMVQ7h56d3CoUlnokbgksCiHTeDLMOHgpxWRmkBw80o7X7JkyeLrW3ASf2qGGs9PzWckXsDNgcVizL+8p9x3wpR4AjwBHwAgKeX5VasLGFCxdiwIABePbZZ/Hpp5/CYDBg69a602kffPABXn31VTz44IOYPXs25syZA4vFgqysLKxfv15YNnPmTNx888347rvvhBHV14Z9ySK6zzzzDN566y3h75ZbbmnBXvCmFxoCGbqqhKdbyWfoRhJXzTGm0CCUE6YCB94ym8OC74+/iBiLHAPDJnirW7f9CFE++o1Zho9w+331hTuXBEKvlaD/leVtlqhWfTzV3z+QCBwgH3xVzcmf6qvA2kcBsZY0k4/WP8NUo4GPfAjrMIWcdAeKj8zHn6VHMbtgN6ZH9qxZWKKJY7UnJpGqSCyE6H4T2za0eof+rACJExlNiPIGK6IwrN1NWH5qDgw2CtVz4whwBDgCrYDAOXF4jx07ht69z0ap2PuDBw/W2B2bzYbc3Fx0795dWB4ZGQk1ZRPn5eXhxIkTwnKxuGq4rvYNtWFOdWlpKYqKioSoMuu7trEos524bK6/2t/zzxcuAkaiCjx1CIhXAQ8mNX8/WUlhm5cT1pad+gqFpmwML7EhMHZM8wfXWEuLGTJWRrhXHziDQxpc+xDxdnMzVOhNvN3A8LanMtQe7FCa1e9JDy0fZ1K0ve7kkbC6rbMcDj8R5NvaNtGu9tgb+yxTRyA48Sqsz1kORmW4PDgF90X1baxZo99bKMorSJTRNdZbJlc50b6vHpl71TBoPb+9jI+7HxaHiUd5vXUgeD8cAY5AHQS8TzysswkgPz8fAQFnQ2jsfW0HtLCwEH5+fkQlPBspCwwMFJzW01TqlL13GWtfUlKChtroKBGH8Ya3b98OlUqFhx9+WKBLjB8/3tUNFi1aBBYxdtknn3yCsWPHuj4Kr1FRUTU+t9UHB003O8jh92Q87EHhfLOwMJqXPof24JZynDIZ8c/YMCQGN4+OoFaoYc47DeXlEQiMivTK6EuN+Vi8+SNcou6FKOdRpPa8lqKpHmTfN2HrrnPI8vefJCNlhP8NNyG4Xf0E5kPEBDpINN/eo4CBY6niA9jfuTVPzumXpRZMXFmCdaZATE52/xuwDrVCvLMSQU+0/u/am+d0Yf+H8OHeRegqk+KzfjdA4QUtOMfIcTAuXofAHTug7FvlQHuCc2NHPoQmJU7sBrL3RGLUjY2tXfV9FKJwafHtWHFyDqb0eRZ+srP3C096cJ3TnqzrK+t4A+tzvS/snJbQucfu1d7GnAWpuHEEWhOBc+LwSiQSIYrq2hEWmWVOaHWrvQ77jq2nVCrpB+a+fe3l1dukpaUJvN3g4GBhM6mpqQJForrD27NnT7z44ovC9+y/hIQEVFRUCJ/ZxYg533o9JWL4gMmIaymjiLRrfO6GxMbL8GKc6fPFWNReQ848e0BhkfZzYd9lW7GQtHNf7SpHnNhAmDZ9qwxny0EtZDYnDDGUZ96cTtxs9vN9j0MEMfoVmhEYMxI6PYtGeici6brBCjcWehgU/7UYzr79UMl+i/WM/0S6DJt+U1HpYCs6DzXWt5qbPfHOoqac053kwOgICd7Yq8WoIAtUkrMPz67RiPrLIV9ihXZLAZydlK7FXn1l1yV/f1Ky8NI5XWY14u5jWxFAv+97M7+Dsc/dMIklzR+zzg7pvFKI15I6h+VmOI9YoS8+COn9STBLrG7zK5q6sU6XKLB3jQKM4qAJ8SwxbnzsA1ia+TUW7nsH16Y87tEma5zTHrVo+5Wack63/WirRlD9nHZQoiMLJjV0L2rOuBku3DgCrYnAOXF42VMhoxe4jL2Pi4tzfRReQ0NDBeeS/ZAUiqqIFlsvOjpaiBCnp6efWZ8tZ0+XDbUpLy+HVquFy+GNj49HQUEBJSWTjNS/1IjExESwP5exfl1PmWwM7Afo+uxap61e1aRsISWHt6HxsP2Sks5mQ+u01fjr2y4bL3N4TeTQN6TeUV/7pi7fQnk/rxCbhikyXBlqIayaJ1PFxm3NqASLDRva2eH0QnTiuHY31p76ATfEPAzR8Y/hn3aXV49l9XNavnY15PQwZxg+st6xH9upJr1dFWI7mtBzbLk3ZVs9PmxNPacfTACu3Q58dtgoqDfU2VAHJ6QBYjhWlcGYcHbWqM56LVggk8kEh9cb5zSTH7v/2N9gTu/n7XrDfPC/yDv4A0KSJzVrhKJSOzSvk7Nb4oBpghp2OndVC7aT89sbjkPHYXo2BPagljseCd2NOLg5ArtWStBvvM6jsfohFIMjrsefWZ9hRPhtUEkbTqJknbruFefTNa+p57RH4LXyStXPaSedkywY5W3MWRCBG0egNRHwnGTVglEMHToU//zzj+DUMAkxJk3Wq1cvoUf2mf0xB4Iltv3xxx/C8nXr1gnOKnNY+/Xrh/379+PkyZPCD23x4sXo379/g23Y0+cjjzwiRDsZV/fPP//E8OHDzzi7Ldgd3vQ8ReAI3XcfpaT0XuTnPNO+5TshFJwIl8Dp1/KfETtH5x9/DtHq9uhtoulckQQBMaNbPkh3PZhptoAcXoG7GxZeZw1WyXb3sgBydgOR3NMgJKlRQazzwpKIyXB9NPA1qW8UkCBDHaMiHpZLlKRMQVFzis77ur1+cgP26gtIfmwMOkUPgn+7wSjY/wlFYT2LmtbYP9KM1rxZBpHBCe0roTBdp4F1SBDMY8ywxf1IhSgc8HujBDC3HBep3Im0ATrkHFShssTzaPSV8Q/CZNcJCWw1xs4/cAQ4AhyBFiJwTm5jY8aMETg/TF3hnnvuwXXXXYekpCRh6Ew2jEmVMbvvvvvw888/Y/LkyYIiw9NPPy0sZ5zdu+++G9OnT8dtt92GyspKYZ2G2qSkpOCaa64R2rH+9uzZg9tvv13oj/938SGQSyyPGfuAKJo8eL8rFSDwwpnPEtbsXtLfXVfwAzJ1e3FL8suozF0B/wh6oFO0DldWvmE9RBRRN4+pyVdnZwUrKLHmu1Ah6ajH6AoqLqGlmY7z63y5L7Hq+L533P24LSNUEOuckO3wDlXE/VZavvSbwnQsKj2Cp2IHo7+GvHiydt0fhll7HGWZi5q8AfVXWogLbdA9FQymS+wyy9BhlOmXB8fEfIjzbVDP07q+atErK0TBpMoObmw8UuvaULgyDkMir8eSvC+4YoMLFP7KEeAIeAWBs1c9r3TnvhMWvX3llVcER5Vxd9lnlz3xxBOutwKHlkmYMTpCUFDNm/2ECRNw6aWXCtwhxo9zGePd1teGObjMQWZcuupJc662/PXiQIBF+u4mRoycHLfPuleVEG7xnpMUgCTbBmsvKvPVQmNSTD9nvYW+oZcjTdUZ+4u2I6bvCy3s1X1zJ/0W5OvWUHnZAXCGkLTBv0YBZjAKw4H1GsiVTgy7uQRhseeXfJdrXwKJZ8Kq5rGCItcShaV/sOubqldWFc+WKIVilRHWgTVzCWqu2XafNmhz8OGpbbg5rAuuDet4ZiD+Ef2giRqK/PT3SblhAkRiRqpp3GTbTVBsNEF/V0AdzWhneATsnTpDtJ9m4SbfD9W8SlgGK6k4RcuSJSU0tI6X6LB3RQA6DpQi0MNCJVfGPYQNBT+TYsOXuDrh0cZ3jq/BEeAIcAQ8QMALcS4PtvLvKoyrWd3Zra9lbWfXtZ6LR+T6XP21vjaML8Wd3epIXVzvi4mie9dekOQRae72ACJadg8/C16uBSKL0yuSZL9lv0vTuHrclPw8KnKXC5qrgbGNVz07OxjP3zkoUY1V17KMOhvdrSiUYsmscKSvCkRMBxPGTC06b51dFxLXkwhDFwosvkpOLzv2tc08Rg0plRkWn/Y9ibUTpjI8nbUaAzQxeDRmQO2hI6rnk7DoclBy9Ps637ldQBQFFrW1UpKmZaR79QrL0OFAUSHscdmwpcig/pqivA56CmqhJfcwgJUdZg9SnlqYMlbQ5V1KDq/eWu5pM74eR4AjwBFoEIFz6vA2OBL+JUfAywiUkLM7nZzdSvJpviBnN86LwTzR8SqCqL2FGry5+gysPDUP42PvA7vRl2f/DXVYL8j9yGPzsjlJk9qxZhWYc+MkmpDFJMIeir6tmBcmbIlRGPqNrxAivF7e9Dnvjqi6eK49wKgss7Lrbt4ymE4GlQiK5b4lhaS1mfHIieUIlanwVsIoSNyQp9Wh3RCUMAGn0z+A3VJZd+dqLVH+pYeo3AHDHfVLfdmTU0iUOgGydathuE0DSZ4d8jUtV3thYhKdqSrf6eNKlJzyLBrNhj8x7kGqvmbBP3mzau0N/8gR4AhwBJqHAHd4m4cbb+XjCJT+6+yW06z8l+TssmQmr9pxExwhYnIcW/YT+pYS1UIU0bgijsqrWipQmb8RQfGXe3Wors4cv/4EquYC89ARpJOqxtLZEchKV5NDosO46UVI7eNbzp9r3M197URBxdtJDGZuDnCotl9IlfHMo1RQrDZCpHcTAm7uRlvQzk6JaDOzVqHMZsQHSeOgaUB/OarXTDisOkpg+7jBLbJ9Yw6vhfbVEX2WSuaukWjMOEiyMuFUnoalrwKq3yjL0wuJfQldSes5xIb9az2P8rLqayOjbhUKUWgtDZTPc7cjfBlHgCPAEXCDQMvu1m465Is4Am2NAHNy7yLOLnN6WWQ3peU02zq7JDpmhj2ZhF9bYFsKf8fhii24JeVFyMVKaE8yOgNVV2sFh1dwZHbtRNHQqVj5Qyx2Lw9ERKKZHN1CdBpEeqwN+0It2Mu2bTojkfS16WHnmQwSH6jl15ouoxPD6oRihW84+oyzu02XhzcSRyFRWTOHoTaKCv84hHeajqKMuTBpT9T++sxnxT+kI077aLzqbN7DmS9rv+nTFw5SxZGvXQPTNf6CdJl8fcujvCxI3WVIJYpPKlCQ6flvZkLcA8IIF59s2KmvvRv8M0eAI8ARcIcAd3jdocKXnbcIaIm+cA85u1S3QeDstm8FZ1fgNlKEtyV0BqNNhx8yX0X34JHoFVrF1y3L/hOq0O5gzoxXjTi7zkXLsD78Efy1hTihNN0//OZiDJhYDnVALS/Qqxtu+87kdIV7nXK+sslvq63a4AyVwDKEorx/k1PoBSmuluztX6VHMb9oPx6I6ofBAZ4d/8huD0CqDEXutufcb5pkyBRLDbAMV4Hta2MmonwHK9FdpAf3w+FfAWt3OZQMG5bR2EKLSTMhKNKK/esCPO4uQB6GcTHTsPr0fJSY8lo4At6cI8ARuNgR4A7vxX4GXED7b7RTdv4+4CQ5N5+TGkOaB0Gt5uy++LSdZL2cFOH1nJNYezuLct6HzlpG0d2XhK9sZqIznN6AYOJmetvyFufhD+NjyBH3wiVXWjF6SjHC4igMfpEYozY8mAT8cApYVlRzp01XUTlzkihTLiHHro3skKEYr5De7rigZEyNpCkJD00iVQtqHjqiwZRm/l6nlYI4uExz1zTe86c+G6l3ULlGyNevFdoxLq80naZKWmhM2q7rcC3KC2TIPeR5gYErYmdQGWU1fst5r4Uj4M05AhyBix0B7vBe7GfABbL/VgpUPnYAyCDa4SfdqjL0W2vXpKS/y6y5Ed48/WGBm8hu5pEq8sTIynP+FugMQQnjhc/e+M9sEGPzTxpsOdwf4ep8XPe4FV0HO0jKyhu9n1993B4LjAgFnidqw9Fqvq0jSkrKBSoo/6DELu25j3Yzvu5jmcuRoAjEC/HDmgwq43sHxI5B3o6XYTOVnm1PCguKJQZY+yjgaNcEvopcQYU5BkO2czud3xbYoyVeS+yLTLQINJr9pNjAipt4YmppAMbH3YeNBT8hT3/EkyZ8HY4AR4Aj4BaBi/DW5xYHvvA8R+AluheyssH/7Qz0DmzdnZGcsNIUsRTOoManid2NZN6xZ4REtQlx95/5ujzrD/iF9yV1hpgzy1rypjBLjuVzw1CUJcVQ82wMnEpVtOpP0m/Jps6LtizC+BpRG9pRcPEhmgVgCh4uM15HUwE0a69aWOladE5ebf8mqRkcNryXNBYqcRMc02ojjO3/KpwOK3K3n6U2yHabISm0w3y559FdV5fWQYOFt7KNG2AerQbrS1TioYfq6qSe124jtDBUSHBsl+fjGhM9FUHySPyU9UY9vfLFHAGOAEegcQS4w9s4RnwNH0fgsyxgcQHJUHUAhlEUr7VNSg4v2iuatZmNJKh/RLsVt6a8DLmkSifNaiiArmALFRKY2Kw+qzdidMuDG/2x/scQaBSVuLrsCcSMDYVTcxF7u/8C5E/+5MddhQq6uJ+cXt2/ErzswcV4rT/kpNggzajmCVcHthXesyS1nbrTeDNxJGIUnisY1B6KXN2OqA3Pk6TdX2cqsDG5NVucFLZOnieJufp1+lPJ4b79IN+yCZYBdIsg3BRriSfkBQui4hNMtSFjkz8sRnoK8cBYQuc1CY9jT+kKIcnTgyZ8FY4AR4AjUAcB7vDWgYQvOJ8QWFJIfN1s0tuNByZ5X7q2LhQ0VSzJoghve895iK5OmIg+S1TrHXopeoaOcS1GWRaViRVJBG3VMwub8cZmEWHzb8E4RA5vWt8KXFb0DBTxgbAOuKQZvV2YTZgW8yfk9LIkNlZqmmk0MzNfrhbKRKtnVQCU7NXatqTsOL77N0ltoIb4Fi200JTridowVkhgs2fmCbxb87jma/FZSLoOVH5aemAbOb/E6WUOrxeS19hudhlaSZQGUZNKDg+mcsOx6o744QRFs700jhZCzptzBDgC5xkC3OE9zw4YH+5ZBBhf94XDwGiqm/BA4tnlrflOkmujCmt072+Gw8umZM12A25JfqnGEEtP/IqAaMqOV4bUWN6UD4ZKMVZ/F4rCbDkGXFWOPqXzINHrYLruBnKmPYukNWV75/O6XSnY/SnxvI8Tl3faHiowRooeoEoV+hlBEJfaof6Kqoy1oh01luLlnHUYHZjYpCS1xoYUP/AtiEi717pgNSWeiYTywI21qe97Z2gobN26C8lr5qFySIooee1QFXe9vjaeLmeV1zoM0Ala0NoSz2gcYiKe35j8DDJ1e7GlqG6Cnqfb5utxBDgCFy8C3OG9eI/9eb3nFXTvfWQ/EEuB1leJm3mufDrG3xWsiQ7vMe1OrMlfgEkJjyJUGXMGe2PZQZjKMxCSfO2ZZU19oy2WYs23YTRFLMaIySVItO2AfMc2ilpOgDMsvKndXRTr9yKeNytIUkQPL5N3Aenk4zpipTBMDYBiowmKxfQ01QpWSZXUWJJalFyDl+KHe3UL7IEpYcCHCD3SF2Ud6UlQ2bLLu2XEKIi1WnrAI5kyKrLiDU1e1w6n9ddB6e/AXqr056l1Cx4B9vdT1puw2L1DsfB023w9jgBH4PxHoGVXxPN///kenIcIsBlNVkiAae6+T9PT6ubljjVrz6XHrbBH0AY1nm/URglFXx99CrF+HUlXdHqN7ZYe/xkSeQBNR4+usdzTD6WnZVizIBRSuRMjby1GsLIEyl9+hK19B7iSjzzt62JbrzPRZhf0BkKI5jqVIr2fZwH6YWqYiN6g+l7nNXUCF65sKv6ZnDVUSc2Ed5PG0HnbfFk7V5+1X0NyqSy1NQjH/V+B9tTa2l836bMjOga2DmmQU7lhy2CiNWw1ARb68XnB2K53H6mlGQkF8o54Tg+6Kek5lJnzwSoUcuMIcAQ4Ak1BgDu8TUGLr+sTCHyQCawvBV5KA+Kr8r7O2bgkzOFNaZqjsiRvNvIMhzE19W1IRGencFlmPdNPDUq8CmJJ05PgSvJkWL8wBH6Bdgy/pRhqfxuUP3wHJxUQMN1w87kLe58z9L2/oSjytb7tRVFeCrrPIi74NTuA38dphLLD6rlaqBZUkv6cd5y82QW7sUF7UojsNlZJrbl7qlhlhC2JzrH2Ycje+DAsehIfboEJUd7iItgjcwTtadkucnq9ZLEdTQhPMGPvygAqq+0Z7SbGrwNSNL2xgWTKSs2nvTQS3g1HgCNwMSDAHd6L4ShfQPt4kPyPr08CA4KAMed6tp6iW5KTNthSPXd4C41ZWJTzAUZFTUFKAHlW1awidwXs5lKEphDPtolWckqGDT+REkOoDcNuKoFC5YR82T+UUJcJ042TSZWh+Vn/TRzKeb86q8b2WEpVtDeSnjuezBBhZLdArB3rD8VfemieL4HkcMvUGzZoczArfxemRvTAqKDEVsFMXGiDdL8F5pFqJAz5iPSWZchaN4MSxJo/dntyCmwJidTvctjipZBv8J7Dy0DoOaYCJr0Yh0i1wVN7pMtcMH3ehZQAyo0jwBHgCHiKwNlwk6ct+HocgTZCgFVSe+oQkERR3Q+JynCujakziGgMNorwehqPnXt0JvykQbgucWad4ZYc/QHKoE5Qh1IGVROsvECKDUx2LMSGoTeUQqZwQrovHYo1q2EeMw52ojNwazoCrCIb4/XuKgcWUmB0eqo/uqvkeHuDFikvlSInTobsnkqUd1bARgUZZBIRReyJ+0sBYKbrILzSe4nUgWCLCcZKB8rIP/ST6fB0zib094/G/VF9mz4wD1vIqbIaiJ5hGaSETKVG4tBPcWz5TVSU4iXEDXjNw17qrmYZOQrqr+cQTaaSzjEVRHoqVe3nnVhJQKgdaZTAdniLP+I6G8FkyxozP1kQrk18Et8c+z/sK16HbmHDGmvCv+cIcAQ4AkxhkRtH4PxA4P0TwClyIL4n3qXKcwqt13ZOeozkyGi79kTPIrzr8n/AoYpN+E/nr6CS1oy4WnS5VEp4HWL7vdSk8VWWSgSNXUZjGPKvsys+lQfljz+Q5mpnWEaPbVJ/fOW6CPSm2QP2Z6CHm21lcsztE4rQHSYM3m3EJX9WQrq4EpUyktUKk2J/qAw721G0PUaB8jNJYtQQZdU6ZtHLm7Bb68TNFSK0p5oL3ShXqx8lziV7Xn+hWn9u3rLKaiQdZhlIT4PqKmfUP6IvYvo8Izi86rCeNJNwvZuGjS+yd+wMe1Q0abitppP/Csi2m2AZ0XzJs9pb7HiJDrkZKuxcEohRt5Z4VAlwRLtbsC7/e8xKfxQfjNhcu0v+mSPAEeAI1EGAO7x1IOELfBGBreQ/sKjbI8lET/R89tOru8IcXjtN60LeON+w3FJAmqGvoF/YBPQKHVdnHCXHfiDpXQWCk66u8119C0w6sRDZZRFd5uzKlU4qh1sBFUXfHKEhMN50C+ft1gdeM5azZMgRJHk3IoyOd3tyJG9WQUvRTSvJc4mPWdAxx4Ze2SZM22+Ak1bRdZajbKwfzH38EBgUjMKSMjx0eAOyzWW4K2IIJE4NMg3AIaLl/FkgFHdDDHGIx9A2JrYDUlvg/Mr2miEucxCdgcZZzcI7ToW+eA9ytz4LVTDNJoQ0b2rEMmo0VN99C3vspZBv9q7DK6GfVO/LKrDu+xAc2e5HEV/Si2vEmEzZlNTX8cqeiVh0/GOMCpvaSAv+NUeAI3CxI8Ad3ov9DDgP9p9F2pjebg+Kik1puUZ/s/dYQk6OtZdnGeXfUPlgEd2Ub0t5pc72WLIac3iDk64SFBrqrOBmAUvq2fhLCOw2EUZQFEzpR5PoVBhANfdLmkt3wHjHNCqH5SnRws0G+CKPEBDRVL68L+FMf4zGwFwzI3FnZbvMUBGlQPNBGWydDVD/XwB+rdyHTPt+TInqhntiakb4WcGL7USdWF0M/HIamJcLMKm0qXHA8FCPhlJjJTklq9mJZmFPI05DLYsf+CaOkPRd1tp70eGKPyFVUPi6iWbr0g328AiIKw5SMYpu9KBFtIYA79Aa2FDC4yxI7mXAwQ0aRKWYERDWOLUhWdMTlyZOww9H3kA3/9EIV8Y3ca/46hwBjsDFhID3rlgXE2p8X88pAh8SlaGE8m5eThPqA5zTbbs2JiqzQ1LsIH5s43SGLYWLsKtkqVBgIkBO4btaVp79N2ymYoR1uK3WN+4/Mhm2bYuDUEki/YOuLYN/ED0BWK1QfTMX4pISGKdOg5MiitzaBgFHhBTmy/xQ+WYYdI9S8YoCG4zTD2HfyhO4JrQjHo4ZUGdgGgo1jKJT45WOwKpBwGv0aqLD+tD+Kl3gHeQMe2rs3JTtNgvJau7aiKUqJA2fRUoIFcje8B+qVMZc9SYaKX9YRo2ByLBRICszWoO3rdvwSqg0dmz7M4gS7TzrfUqnl4gjH4B5R//PswZ8LY4AR+CiRYA7vBftoT8/dnx3BfADURnuTQQSvUcbbPLOS49UFZywNeLwVliKSCP0WfQMGYNBkde43U5Rxlz4hffxeHo5fbUGp48r0G9CGUKiaBx2O00vfwNJdhaMU6bCEdOGYW+3e3jxLmRleHPe1GBjXBE++b03XjzQs1EwFHQVnhBJ53mfqrLHVvJHp+0FZh6setBrrAMFS1ajPizDatIZqrdTaBKRMOg94o2vRX76h9W/8vi9rUdPOEjGwhlYVqXJ63FLz1ZkWtL9JpSjokiKA+trRsTr68FPFoi7uv0X+8vXYlPBr/WtxpdzBDgCHAF2meTGEfBNBNiN/+UjQBpxG2+Pa9sxSo9a4AgU0w2/YRbQvGNPw0n/7mj/ptsB64t2wVCyB+Ed73T7fe2FWekqHNvhj64U/YrpQDVwydlVMmf3yGEYb5kCe2r72k345zZEwErR04dPL8djN6RDO0oBzTc6qObRU5uHNpToDAvJ8X2WDutG0pq+ZjuwsqiBxpSsJl9tgLW/kqToGr6cB8aNRUTX+1Gw70MqSrGmgU7r+YqivOaRY4ivvIdoDWaB1lDPms1eHBptRadBOhzZ5oeCzLr0DHcdD46ehN6hl+K7Ey+APXBy4whwBDgC7hBo+ArprgVfxhE4RwjMOQlkUZLPC0RlkDaeJ9aqo2IRXluHhukMGwt+EagMjLcbJKeQnRsrPPQF5H6xCIy7zM23NRexwhK7lgUivouhKpHHRoUlyNmVZhyC6eZbYe/cpWYD/qnNEXgndxP26woxu/d1kMyIhLWrHIplRkHZwNPBielcvz4a+LUf0IUCnY9SpPe1o8RiccNEYMlqjGpjHu3Z9EdU90fh326QQG2w6PM8HdKZ9Wy9ehNXmJxKotm0Bq2BbYg5vGHE6WXUBmOlZ7eoKamvCQ+a80iqjBtHgCPAEXCHgGdXE3ct+TKOQCsikEOztF9kk5hTTNVNvxU31XjXrOBEJnN46484lZhPYT6VO+0bejkuiZjktk9zZRYqcpYI0V2RmCQAGjB2o9/8ezDpklrR+1KKEFosFCmcA+nhDJgm3wZbt+4NtOZftQUCv5ccxs8lGXg8fhAuCUkQhqB7KpgSHRXw+185JDlVtBhPxxZBuXH/6wY8nkLOLyW2sfLHxZaarRVLDaScICVJuvrPzeot2HmXOORjquynpKIU9zW9KIVEQnzlQaRUcopk0JpANK4+iEbei8jh739luSBPtmVRsEd8XvaAeWvyy/TAuQSbCjm1oRGI+dccgYsSAe7wXpSH3fd3mkW0gimg+kBi24+VyZEJBSfS3Ed4nZRV9uXhRyATK3B7PVQGtheFBz4nVYZAhLS/qcGdYgk7W8jZZVG0SyaVQWKhrP8vZ5HTfULg7Nq6khfEzacQSNcX4vXcjRgfnIpbI6s9jFC4Vv9AIBzhEvi9R06cwU2YtoE9Yc7fbUTR/qIHkEd5YpN3AUf/Ve0Sn6bKaukWmMZ5Ft11bUaqDEXC0P/BULofp3a+6lrs8autdx9SbMiF5Dip4JFaQ2uYyt+BARPLUZYvw+7lJF/hgTHOfO/QyzD/2HO87LAHePFVOAIXGwLc4b3Yjvh5sL9LC8nhI93dJ1OpSlXDlNlzsjdSKivrVIhgT3Lv8C7JmyUUmJjW4V1oZCFux2TRn0LpiV8oujuVKnE17KDsWREg3OgHXlUGta0Y6s8+gbiwgDL/7yHZKUrn5+ZTCBRZDXg8czk6KEPwbNyQumOjghS6R4JJ0ssB9WzP+bzVO+pNPt98KrjiRxMDd+wGmIqDYomeKp6JYBmirL6qR+9ZUYroXk+h+Mg3KMta7FGbMysRl9c0IZ4eyEQ0BuIdtZIxqbIeo7XISlfjKOnzemJT278FqViO2Yf/Q5XvWscZ92QcfB2OAEfA9xDgDq/vHRPfGxHpvEKvh6isFKJSqoRUQXdbI3EOmF6Wl01P8pvvUORoMAU4x4Z7ufNmdifNsMCWSs4uqyNby7J1+/Fz1tsYHXU7eoSMqvXt2Y8F+z+lQhNKhHW84+xCN++y9qmQudcP3UdpESHKhPp/H0NkMcMw4wGq8JbkpgVf1JYIWCgc/xg5u8y1ejdpLBRi909ojmgpDNMDIN9mhnwFEdObYaxIxbxeVAODfL//2+qAlNQZBO7umQpvTes0ovNdCIwdh5NbnoJJe6JJja1De5CzXQR5K9EaXINJIW3elN56MKWSvCPE8WjE2APn9A7vIaNiM/7O/ayRtfnXHAGOwMWEgPur88WEAN/XOgiI83IhPXoE4pxsSPJPQ1ROU7HM6a1lTppvdfprSKaIpmyDguAMDoGD/VHVL+E904aVe8YtdHX9WTZQTlTHp9q7lrTxq90JlrBmmlA3wmS2G/Bpxn1op0rGjcnP1jtQVka49PhCRHS5D1KiNNRnZflS7KYktbjORnRUbIVy1gI4wsKrdHYD6m9XX398eesj8MrJ9cgwlmB2yhWIlNc9R6qPwDJYBek+C9TfamHrKIeDuLdNtQBq8jkxJjZ/poedfpIr+qnhJqbscbfxg/6Lw3+PJz7vDITduNrjdmC6vJeooFjhB+nuI7D16uB52yau2YMe/gxaCWlRB2PoDSWU0NYwF7p7yEiMjb4Tv2a9g46BlyA1gELj3DgCHIGLHoGmX3EvesguTABElZWQbd0M2c7tVKK0DE6plJJhYunG3AmOkNAqx5ZV8iInV0TSWDCTLJHBAFGllqZqySEmp1hyMoc4fVqIqkV+nWo1HOSsOTXkGDPn2M8PTpUKTiXJKFF/EpUatrAwiKlq2HGbAgty22F6OzPipSypyz2F4FweAckJ4u+anVQ9q67j/vXRpwWu4As9F0Murn9aOT/9A4iJxhDRaXq9Q7cYRQJv1z/YhoGK36Gc/yfsnTpXlQvmFdTqxa0tv5hXkI6/yo7h+bih6OlPtYE9MMMdGko8tMDv43JUvkoaZLK6swaNdaMyOnD1bgNW9VDhoVwJ3g6g8sTNnA2RyAOQOPQzHF16DY6tewLR/d5obPNnvjddlUQObxGUP2dA15OeUBnhuBWMChZiwJVlWP9jqFBtcNiNpQhmetQN2I1Jz+Codjs9kM7Ay72WwF9GD9/cOAIcgYsaAe7wVjv8copGqslBYyaTVTlbwcG+caEUKVVCqdrGxsPG3dg61XYZTnJQHX8thnPDOmGxqHdfiPoPgIgcXRcG1ddv7L2TpLNA1b+cJVQzlegPztJSiMvLgIoKel8CUNQYBsq6IQfXRYmgd2Du5BsDZiBaWYqH570DhYP6YdHhwCCIQogXGx4OURRpNcXFQ5SQAIbHObFM2g9ySjR9yaOQn2UArc5bgI0FP+O+7h+hW+wl9Q5FV3JQ4O6mDH4ZYZHxbtdjVMMlv8moEpYYlwdQUYqVKyEadxmkk66FkiJp3jLX8WzK+eGtbbekHyk9fPnamFcUH8NHp7dhamwfTE0dWGP3RP86fhr2kFft4e/MSk/TNeaxLAT/QpILMzxzlM+0ZW9+J5I7/TyG/ycao09YMPOQFZ9r/DAuqu5DWY129XwIDh5KFKU3cGTNYwiKGYKoTpPrWbPWYro0OhOLIDkVicDjxyDu17/WCt79OH66A399Aaz/KRSjb7YgtoPzzDXK3fnx9IAFeGTtYMw98TieG/ALXT9bxyFvzl764jnd0H64sGPntJiUPhT0EO4O84b6aOw7G7t3cOMItCIC3OGtBq6FpJ8qKdLJjP2Y2Y+8jKKdvmBqkxEq8owaGo+YnCN2Qaog57JRI4qCbPNG0ghdQnctVqVpBKyDhlAU1r+qqU7XaBf1rsAcVeacsr+GjLipUpqXDdX44+uMMmzPC8AnIafgmDwZJuIIixhvWFshRI/FR4hisWmjQK1gzGFnaCis3XtS5LULHOQEt5b57yAs28ug09Mr+enMTuoP4dM9D2Fg+FXoHzipwWNyfPWTkPlFwy/+hnrXO7DOH3lHFRgpnQP/g+tgvHEyTRHTNKwnx7FqSB7972vntCeDbtI57UmHXljnCFEYHjq6GJdoYnFfaI86x5U9WDCHgF1LrFQCuo5FUMLZDf5Qf18GXYoT1n71zw7UbisqsSPwtxKYx6phlOnxOgVWHyW/+d5tenzUVY/B9GzYHFPHXovw9ptwePUjNPuSDFVQmkfdKIYHkFxePKwLv4c+MYkEs1v3ljLoWhGWfRWOpV/LMeKWEqR0qbpeubsuKhCEae3fxceH7sLcvc9jUsKjHu1Ta6/ki+d0Y/tc/Zx2EG/dTDN87jBvrJ+GvlfSrB83jkBrItC6V6fWHDnvu9kIiCgCq1q4gHRBs2Ht05d0NccLlINmd9jchnKiSNANUhcYineK7BhOM7xDu0ZT8KoeR5kiAOKCfMi2bIKYItPyTRugWL2SeMNM67QPOQ79Be5wc4dTp52V+Ls0/Wya9O9DAK1gsGnx8cG7Ea6Ox51p79RpUn1BRe4KKuW6HgmC7intqxtjiTgZW/zR3boYcY7dQnIaLxXsBigfWcQUGf5zYhmi5P54M3EU5TE2LwJvJk647BBRGz6vgJa4vI4ozy7FjP/LFENc56SMNv9uF5Lv20eO7wHgM1Ks6x3UPLDSRnwAbcEegc/b4fI/IJGdPe/r69EyQAnVN1qK8kbQ73IzrEMoWtyKJlc5MWpKMTb9GoJ1C0OgnGJDTCpNkdRjfcIuw4S4B7Ao530k+HcRKrLVsypfzBHgCFzgCDTvan2Bg3Ih7550fzr8PnpPUFowTL8HputvahtntxrIb+6rhI5ms2amVlvo7i05x8wZNF97AyVyTYfu+ZdhoFd7YjLk69fC7+03hOIMkhMk8+AFY+WERRSgY9WymLHpaSZ3pLUWYWaf76CU1J+k5LCZkLf9RfhFDEBw4pVuR6MtFGPHH/6IsexBt8jd0D/4sLB/blfmC9scAaPdiofJ2WXlgz9KuhT+kuZRCIQdodkj/X1BcFA5YP93y2g2o36nzbXjrLIZU3kwTmZc+LOXbsa0+bAr0JH80wf2AwerJqlczTx+lZATnzziczAJvZNbZnrUzhkkoaIX9DAn6gvFymX0RNg8BQqPNvbvSuoAB4bfXIIQ4vEu/VqGo7vPYuGun2sSnkCP4FGYlfGQMDvjbh2+jCPAEbjwEWj4SnHh7//Fs4fkrMmX/g3V/G9gT06B/j+PwZ5K86FtbIcqnZhzzIB7ksRgsktNMqr6xHRpTTfeDN0zL8B85VWkV1tIWqefQf3px5AcOdyk7mqvzDLqnSrS302u4nP/mv1f7CldQbJH7yPWv0Pt1Wt8zk9/HxZDPmL7v1JjueuDtVCLzfOkUFuLMbAflQq+czqJrNbvQLva8de2QcBOTu5T2auQaSrDByQ/Fq3QtHggTn/S530siJJEHfAjpxeUHFmfiQtsgoavtYcclhF1dZxVlOP5CUV344jafm86cPxf+k19/dW3nFEZ4ga8jvLsv1CUMbe+1WostwxSQlxKYWWjkpze5TW+a60PMoUTQ64vRWIXB9b+KMfBDf6ulIA6mxRTFP7ejp8gVBmD9w/cgQpLUZ11+AKOAEfgwkeAO7wX/jGmEKUVyu++gXz1KpjHjBOqdVF2XpvvOcvneSnDjkR/Ce5MaOGpSPwvxkHWP/YkjLfcJuyzes4XUH3+CSRZmc3aV1m6uSq6S9Wythb9gcUnP8LEuP+gb9jlDfZnKElH4aEvENnlXuJC1nWMxXv3YdtXDpgdagy6LA+4dCSYzBM330XgrdxN2KA9idcSRqKrH5FwvWSOOOKHk9MrPW6F5g3Sua6w1+mZObv+r5NyCkV19TPq5ytoiBXBKA2sQuHd5PSeJKns5lhI8jUIbT8ZeTtfg75oZ6NdWPvTkyo53M7wy4W8AFERJdWdA2PVuUfeZEXPkTYc2qQhlZMgSvx0n5imkmrwSJevYXWYBKeXSQpy4whwBC4uBPhd9kI/3izZ7avZkB46CNPNt8JCDm9ryQc1FcqfTwP7tMBbfQJJAMH9jaqpfTLH0datBwwPPQLjrbeTdJoR6s//J1AdGP/XU2MlUyWZNkqMU+CYdheVDn4UfUMvp8SXxxrswmE3IXvTo1AGpCKy20M11yVlCuVPP2DvIikKJGkktVQCv16xNdfhn3wOgS/yd+Hnkgw8GTMII4MSvT4+W2cFdE+HQHzKhoCZJZRISnJ/dP4xmgMrUqF5tkRILNU9HQxnQMOX7BBiWczuQUlxtNpde4F8JoHSDIvp+wJUwZ2Iz3sfrKbiBntgkWprT5IYzI8VpAeVf/ze4Pre/JIJL/QdZ0N/ki0ryFRi1Teh0Ba750OHK+PxcOevccpwBJ8cugc2h5uEQm8OjvfFEeAI+BQCDV89fWqofDBNRUBESgvqWZ9BcvoUjHfeBVuPnk3totXWL6HM8g9PAFdEijC8nfuErhZtnO6Etq7dYHj4MRivuxHivDyoP3gXil9+FLSCG+ubRXeZC57foRAfHpyKWL+OuCvtw0aljVhUzFKZjfjB70NcjeMpOXwIfu+/g0MZMTiiHIUeYyoR2ZlCVNx8GoFfijPwGTm8d0b0wI3hnVttrKwQhfbNMNK9lkH1tRZB9xYi6K5C+M2hIhVd5Kh8hQq6RLp35GoPKpJ+TrOpOAXVTMF0cnoLzbXXaPyzWKJA0rDP4LCbkb3+QTgpM78hswxRQZLnoMTRSULRGul+yqI7hxbXyYSRtxUTrUFETm8YlSN2L1uYEtAL93X6HAfK1uPLI4/y8sPn8BjxTXEE2hoB7vC29RFope2zghAq4rKyMsCGu2fAnpLaSltqXrdvH6tqN7N9K5+CLOLbtx/0TzwFy6U05bqPkvbeeYP4zP+QFnD9c76yXWZYYoA382+jaJkfHu4yFwqJ+5uoC4Hy7L9RcuRbRPV8EuoQSp0nY7JqygXzoZ77FY75j8IuxfVo309H5VL5lKoLN199XV5+Aq/nbsBVIR3wQHS/Vh+mM0QC/cPB0H4YRtSFQOjvDkDFu/SeljkDm/ZwFEun6hcU6TWQn8qc3qJmOL1y/1hSGPkIuoItOLX7zQb339pbAYdaRFHeeNjoWqP4cxFpHTZjow1upeEvA8NtgoJDdHsTdi4JwpZFQbCY6s4c9QwZjWkd3sWWot/xzbGnG+6Uf8sR4AhcMAi0srdxweB0Xu3IGWeXdGyN99znc5n/G2iGdgnljTySAoSRxNI5MdJGtYwYBd0TT8PafyDk69bA/63XIV+1kpKFas372kiObK8Ja2IXw2jX4vGu8xEoD29wmKaKo8jZ/AQCYscgnFVUI01n+aoV8PvvWxTxOozjwx7AZtO1YJGobiOamUbf4Aj4l95EYCPxdZ/JWo1RgYl4Nm6IN7tutC9HuBSWoSohOc1TuTJ3nSYSTf9LcnqZAsqd5PQWNMP/DIgeRg9wj6OIOOllWX+420zVMirOYr1ECflGE0xXTqIKjJVQLCfVhnNsLJmt/5Xl6HtFOVEcFFg+Jxz5mcTzqGWDI6/FlJTXsCZ/Ab499lytb/lHjgBH4EJEwLM5sgtxzy/QfWKV01SzPxe4q8a776Vp0GZUcmpFbFjE6ZWjpBUaCFzTFkMjJQSm5mAZOoz4kcvpbylJmq2BZeAgumEPIo5kIBwHqFwyBX/XxS7BY12+RTt1coOI2EylOLF6GmSqCCT0fRPyDesgX7taKL1spap1OWmTsOnvaEQmmYUbMeMdcvNdBLZVnsJjmSvQTxON1ylJrblau76wh8kk/DGHmEyMz3v7bmAWUR0SmpivGtn1fhhK9tED3ZNQBKScmb2ovX/m4SpSaTBCmkeRVXq4ZA981h694KAS5efaEroaERZrwY6/A7GRKrMldjeg+0gtmEPsslHRU2Bz2rDgxAtEbbBhSurrjVKWXG35K0eAI3D+IcAjvOffMat/xBTRdX5IGrtUutcXnV028A+It1tK/N0XSLygLR0/Z1AwzNfdAP3jM4WKbYKO75uvAfNn48hff6FUXYwrRz+FRA15CA2Yw2bEiTXTYCe5qjTHjQh8530o/v5ToJDoH3kCWd0mC85uKN18B15dRrzeBjrjX7U5Ajt1p0lrdym6kRLDuyQ/JrsADhiL9M4lp1dCD1p37AH2U6JoUy1+EPHf/eORufZu2Ew0RePG7Kly2GOkpAZjhGXkaDjCwqH8eSGRiekptw3ML8iOYTeXovuoCpw8qBIqtJ2iiobVbVzMnbg15RWszp+PL448Qo5v24y1+pj4e44AR6B1EOAOb+vgeu57JT6q8stZQHkZjNPu8bnILgNkRzmw8BQwIxFgN2FfMGdIKMyTrqUs+edQetlwvBH8JTod7wG731H0+HWPEKWSHthPGfR5QoljEZX7ZbJLTtL4FW9cg+yFE2EqTEeXI32g2XeKknYGkBP9lKCIkVMUh82/BSM8zoLB15RCwudTfOGQ1zuGHRTZfZCc3U7qMCosMQ5K8YVzwBin95teAEtom0bR3qWF9cLg9guJzA9JI76Ew6pH5rp7KYnNvcKBeaQKsj2U8FlJ1eCup2RRUkaRnyNtXncDZw/V7fsaMObOImhCbfR7DKG/IBgrz976xkTfQSWI/4vNhb/ho4PTYbbXz+13tw2+jCPAETg/ELhwrujnB96tM0pKDlHP+RLi4iKISJXAERzSOttpQa+MyvD8YaCrhqZW41rQUSs1LZNW4l2/DxCZn4ogYyiMIyMB/WlBu1hkpZB0LXOI7DgevwVa/wJ0UBOd4YZroG9PYWsqhsEsY4sfDqzTICbNhP4TynlktxZ+vvZxszYXj2YuRxd1OD5KvhQqCYnZXmAWSlRWRm94JgN48hBJAhKV/D9JoCi2Zzuq0MQjcdinOL7yNpzc+gziL3m7TkPLMBWVLSf+7iojTNcmwDJ8JP2GVpL6RCc44hPqrH+uFvizaO+Npcjer0L6qgAh2tt5SCVSKXmU6lJgaLsboZYG4vOMB/Bm+g2UpDqnUd7+uRo73w5HgCPgHQS4w+sdHNuuFyoqoZo3F2KSHjNRZNcvke5gFIX0NWOqDMXkN/6vW9XUqi+NL1efgfcO3A47Ra1eKvkCjiAxTJMvoWIQgwAH6aGS0oW4nMLTRiNEVC3DJrEjK+t1VJSWkAMwGypKVHNNhDLh+51LApGboRLUGFiCWltSN3wJZ18dy6ryLKGKWm+/dng/eRxUF1Bktzbmanoee68z8NVJ+i1SPZaddFq/2hFIIa6vJ6ZpNwix/V5G7rZniM+bLBRXqd6OafJaBlDFtdUGmK72E3S/pUcyoPrhO6ru+CgJBCurr37O3zNub7sUE/atDhAc3+x9avQaV4HQGCv6hF2Gmd0XkgzhnXh5z5X4T+c5iPcnsLhxBDgCFwQCHj7bXxD7euHtBHHjVFRBjVUSM06ZCkdysk/u4yrSrf8tH3iYhpek9q0h7i5Zhlf3Xk3SY2o812URQnZRdjxlm8NVCINkzZwUMbcnJcPeuQuMSeE4fPz/oC3fh5RRXyGQnF2XlZ6WYeW8MJw6phSS07qP5M6uCxtfff2NFZTIWonBmtiqyO4F7Oy6jgF7AJseXxXtrSAFhxt2Au8Tt77Sejahy7Wuu9ewDrcISiSnSaqsLOvPOquYx6mp1LADsu1mQCqF6aZbieKgg/KXn+us2xYLFCon/T4rMHxyCdger/kuVEhuM+nFSA3og+d7LiYJQj+6LlyFLYW/t8UQ+TY5AhyBVkCAO7ytAOo56ZIij8rv50NCXFLjLVNgZ9PpPmhMCukFojIMDgZujvadAbLklJ+z3qJozjS0D+iH53ouQtSBcIj1TrBpWXemL96Nw/9cCauxED2uXoyA6OHCaiyqu2+NBqvnh0IsdWL0lGKwSBI330ZgFhWUeOXkBowPScU7SWOo2l8VHcW3R+290fUipZRf+hLFiEQUvsulaf0VlZiV5RSSShvbSnTv/0Ng3GXIoaqCuoKtNVZnyWu2FBmU/+iF5Y6ICIr2ToIsfQ9kmzbUWLctP4TFWjH69mJKatMi74gSy74Mx7FdaoTJ48np/QPdg0fi88MPYt7Rp2GhksTcOAIcgfMbAe7wno/Hjzm7CxeAJVOZbposRB59cTdIzhYzDxJHkCJKr9C0qa9M7Rcas/FG+nX46+T/cFX8w3ikyzzi7wVAsdYAW4IU9oS6/M2iw/NwbNkNgvRYh8sXQRNBcksOIHOvCkvpRnl0hx86DtQJzm5AGIXNuPksAlaqGvZ89lowh3daZE+8FD8cUkbkvAhNRT7+QzTz8kd/YFyUFJ+Twzt2C3F799MympWpr2CFiPBKGPIB1KHdSKVkOoxlRAquZqbxfqQ/baUH8ir+u61PP1hI/1rx5x+QnDhebc22fUsTOEJS26XTixCVasLeFYFYQbM0ulNBeKDzbNyc/ALWFSzES7sn4KSu5j627cj51jkCHIGmIsA5vE1FrK3XZ87uTz9Amr6XsqBvgq07ZaH4qLHSwXtIAolpf7KEmbY2FtVdceprIbLrRwkqT3T7Hp2DBgvDEhfZCFMLjLcH1Bgmk2DK2TIT2twVCE29CTH9XqRyqyoc2iLFvvVUBatCItwou43QQhPiYvLW6IJ/8CEESq1GPJ61Avv1hXg+biiuDk3zodG13VCiicXzdk81ZsRpseiUE/8UAs/RzAwzVvm7PXF842nig70Pod9yIN05NFLi6vb/GhUbZ2DfqhnoPmYulIGUQ0Bm7a+APUIC5R96Ui2p+vGbJ14NcWE+VPPnQX8flSsm2TJfMaW/A/3GVyCphwF7yOld+30Y4jobMWzE3UjrOUBIZntxz3hMjH8I42Pvh1Rc96HYV/aFj4MjwBFwjwB3eN3j4rNLlT9+T1XA9sBEGrK23n18dpzshvlNLvAg3f8GEJ2hre1wxRZ8d/xF5OgPYGjkjbgp+Tkwp9dlTDAfchHMQ+jO/68xfmLu9ucBEqWPH/wJHNJJ2L1cTQlpStisIkSnmtF/YhlCotxLNLn64a++gQBzcp8gvq7JYcOnKZejLxWW4FYTgTD6DUyNd9IfkE+z+Dso//UAqTkcI3bCauLiF1LAls3cnDWSXdHMpz+ive+2IVhmRzsKG8cqRZg0yA9X/K7FaYr0BrQnB5HxeW+bCvX/PhRUZQz3PQCLugUVAAAyKElEQVSnPzX0IRNoDkRJOrFHjQPrNTh9LBydBg3Eiz2X4Nfct/F79nvYWrQYt1ORirTAAT40cj4UjgBHoDEEuMPbGEK+8r3DCRGVqxWcXdK3tPUm8p2PGrtBMt7umDDS/GxjCbITlbvpJvU+0stWI9avI57u/hPdqAbWRM7kIK1QA8zDyNlVi2HWnUQeOboVuesh0jwEu+JRbP4nBCYdRaz87EjuaUC3wWKIFZVEayBeAzefR2Bh0UG8e2oLUpTBeDdxDKIVvuVo+SKA7ejnMIH9RZ4dHYmUoJImMrT0jFdJzB09vWeli8sMpTi+fz4qxIGwaK5FnskfT0eq0MNfhyNzdHh5UjCGkFri6DA/9J56F/w//wSqL2fDcPcM+s35ViYrY7ekkFxZbEcj9q8LEPj5WekqjB7zGgb2nIivjz4lUKIGhE/EdYlPIVzZthc5LeGfSQ8k7GGEHRM7HSMlUVVC6RkjjqLyMXQMXTm4Z48kf8cRuPgQ4A7v+XDMSXpMumeXULHIdNMtsPXwXRrDKYoKPUT8P6bG0Fa8XTtFZHeXLCf6whxkUGQ3XBmP6R3ew6CIa+nCX5erqVhDcmMGJwxjxcjZNhs5ewthMD8Eo+UP2E/LoVDbEd3eTJq6RkTEWwTdzoCAAOh058PJc3GPsdxmwssn12NNRTauDknDzNhLSJGDX/aae1YwHn4Awcf+aloILKHX4tjyyXAc+RjJo76GKrgr9GZ/jP1ai70VFnxrk+O7PCBCHo6JVz2O2/6ZjegvPhMK5Tj9/Wt25wOfFGon+lz2L81heQA2/BiKmA4j8eTIpdim/wa/5byLp3YMx8ioWzEh7n4Eyas9GbTi+E30kLGhtOqPFfM52Ug+nYoued2JqTWIHjhGURCCUVO4cQQuRgRIVpQ9s3NjCJSWlsJsNgtgBAcHU5KVSFjWpuhQuWD1vDmQ/7UMkqNZKDmyr97hiCkDQ6PRkAwvzUO2gZVR1OeO3SRXS0HPb3tVVXVqbBhSmuYMDw9HcXExrOTYN9ccTgeOaXdge/Ff2FK0iCSWSpDg1xWXxtyFARETqaxqnTt01aYsTmgeKkJJkBXLomR0/LsLywPDzcTNtSIqxYxgoizUTrircnh151WE12fO6SYc5Jac0xu0OXg5Z71AYfi/uCG4LDilCVtu/qoymQxhYWEtPqebP4LmtQwMDERlZctmLZiCyfGVU2ChWZLEoZ8gIGoEAmYWw0kzJ+Uvhgq6v38R3YlVejPRrNWY4gxML9qBTjdcAVb1sKnGzmlmZWVlTW3apPXZXTJrnwr712qIziRG2gAd4vvkY1n+51h26kvYSMN7SOR1dL25G1Hqhs+z5p7Te+my/stpYFlR1TWWRW4HBFU5s6nEsY6izxq6zLFHepq0EnTPsw1UBIcezHdS2130x+gonenZYlJUVeSe6TJ7YtXP6Uv3fouRgYl4LKbWTJknHTWwjlKphOt4NrAa/4oj0GwE6vECmt0fb+hFBERUOU099yuKPuph7dOXsptzvNi7d7tiU5z3ptONh17nUgCalTBtTWMJaHn6IzhauQMZ5ZtxoHw99LZy+EuD0S98AvF0b0Cypv5IeEWRFCcPqeC3pBS9tA5siE6BUnUYHQYeQ1yXAKgoiYXb+YkAi+q+m7cFf5UdQx//KFJhGIZoOacwnIujKVNFoP24n6j88AxSb5iGqJ4zIb3lDmjeLoNqgxH9h6rQn3zUJ8gnXJQvwgJZe9wY1gn91mfhXppF6dvZN3nV7IE3qbuRIrwmHNygQcYmf2TvS0L/kc9iXL9pWE6zSSwhdk3+AnQNYlHfW9AjZEyLk9sYPWE5ObjzTgIHyXGNoPy/yTHA5RRMZomE9RmrnsecXzbTNoKiuswY9WRNCfBnAfDaUeCDE8C15PjeGtv61+uqEfD/OQJtiwB3eNsW/3q3Ljl6BKoF38KpUsEw40Eov/uh3nXb+otycnLvIWc3zwjM7uF51abGxs0mH7TWYpSY81BsykWhKRv5huNEV1gGs8NI0QqiF9C/OL9OQiJaj5DRxM/tT7QF92ELJiyfc1AllBfVFsmgcmhx/clCnIrPRp/ppxAY3fvfIXFnt7Fj44vfO+h8WVR6GB+f2g4LRfyfjh2E60I7CTM1vjjeC3VMEnq4SCFKQ96u13B69xvQx25H156vUZGcSlh7KuDUiOnBFLiFHK2bYqRYkWPAFxkaTCsKRb/Vpbi/ZxB6BbM4pe+ZXOlEzzFaQc1h78oAbF0UjLA4NUaOmokr+t+HTYW/YOXpefj40N3QyELQP2wiGNe3fUDfJp2HzNFljumXFOPIoetqT6IkvNsZGEnOq4Sc7+YYw5zxsdlfLvW5gOglP52qep1Iy+5KAJhaBzeOwIWKAHd4fe3I0k1bvmYV5MuWCNW9TLdMgdOvgUf5Nh4/KyzBIrtFlDAxi5zdzg0E0kx2PXTWMiES63o1OCrgLLKisCIXFeZioiKUCnSECksR2J/jTNFe0vMVKxGpSkSIMhqdgoaQMPwIIYrLNHTrMzYVWZgtRyZlXbMKaKAF/v4bEBn8OfpnD6HE8f7wmzmAqqm5d5Lr65cv9y0EdlSewnuntiLDWIIxQUl4PHogRcN893fjW+h5fzQi4knH9n0B/uF9cXLL09gTeRv6HppD6gxaKjFM8/D/GnPeLk1QY1yMAmtW7sTHtljckS7GMLURD3RSIc33qL3CyAPDbRh2UykVrFBQUluAUGExoasfBg6dgpG9b8WJyj3YWPAzthX/QQ7w1wiUhQsR3x4ho9A1ZCiJWri/UBLLQ5CE+yyripvLCva8nAawIiHetFji8T6ZStfuRHJ4c4mCRn+/5pMWMynFPZ7CI77exJr35TsIcIfXd44FRFptlcYuRXctQ4bBfPl4epz3XUdsv5YE6g84yId04LnkfagwZOLvigJoLcWosBYJr4xLqyUnVkd/LCLrztRSDUmEBQt0BH9ZMGLUaaSPO0S4SQQr2iFEEY1QRYyQFMJ41Z6YxSgizp1akBfSl0uJolCBiPBZUOJdaMIjkah6BhGrE2G8TcOdXU8A9dF1DhiK8OnpHdhcmYf2yhDMSrkC/bjcmM8craCE8VCH90HutmdxOO01dN76AnS/H4Do6i41xiiSSjDy0j4YeeIElqz7Cx+2uwQ37FDiigAj7uukFtQGajTw0gdRpRYi4v+KKe9BRPxlRh8TmSgLjOUTUIESIUOVrsFOhZxm29SCjJqTuM6OkBA4g+ha1YG4/ilFOLbTDxmb/UmyUIXUPnp0GNALt6X2xC0pL+EQUa52lSzFntLlVMTie5qTohLGQb3R3r8fOpC0WXtNHyp8E4SVJPv2aRZwnHi3A8nRfb1TFT/XS7vqthuWfMicXkZrePEwsLGUaA80juuJWTIt3jf0090OnC/kCDQDAe7wNgO01mgi3bUTysW/w0kOnWHKVJ+pnmawaVFkyiE6QQ7RCthfLorNudhV2R37zPeTA3kUyaKp+P4YhQfIpCI5AikLO4AiGgHyUCT4dxOm9tj0HuPXModWcG7p1Z8u8oHKMLSLjPJagk9ZvhTHd/sJ/Fyi+SI06ghC/V+H1P4D/CP6IrLb2wj0H4aAp4thT5PAfCmR3Liddwhso4ju14V7sYUc3RiaQn+FqqVdHpxKdBbPHojOux0+jwcsV7dD8ogvUZG6CoWfrkfYzwNwSPcE1EMpoS12LMSSqsIUwi4mJ+OyhARcunUrfj6sxacxQ7B0qwPXSApxV0c1IsPrn82pFyKScxRTPoS4sJD+CiAuolf2mRJlRdUSZSm4ChCFzEnJU04ZjUlCtAqaERLZ6EJCycwio4HeExH2X3OSI8zKJjuiotElJg7J45NwMDuNqi76Cw/a7fvrkdpbjy7BQ4S/2/AKThmO4mDFRhyp3IK1+d/j79zPUOEcjWLR/0Hr6IAkZT5ebl9BHN14yCUUhj1HxugO/6VnEJaL8fXJKprDr5Qgd3MMlZ6OA8JJ4owbR+B8R4CrNFQ7gm2h0iAuyIfij98hPX4M1s5dYJ50HXHc6k53qd//CKrZX6Hk0N5qI675trnZv3pbBQqMmdX+slBgYp+zBPqBayvMmdXIO+Go7VmctA5ED7+DmBJ9gITmIxAsbydEYP1kZ6crXe0aevWGSoOd7kEssnJ8txplTEZMZUVYO6KFWJ+E2JGBgJhRiOgygxzefkKasv8bZZDkWFH5ehgc4c2LoHOVhoaOqve+q35OG+xW/FN2HD8WH8RRUykSFYG4I6IHrghJ9anSwNUz2luiPOI9FD3ryRsqDY1tyWm2Q/1sJiTFTuwccCcsIRWC0xsQPRIaiupKldWUGsi5NO9Jx3dZVswJ6g6TRIbri9Jxl/4QoqVW2DukUWljIrXKpNBTdBYWUlMxksSgXkcR2/KqyC1zarUUvf13YCwnwhFOTiopwzio0psjNBSOYIrWUtRWKILBag03ZCZjVTS4pAQS5jjn50N8Og/iggLQzRROUp3RxnbDXtlEZJa2h1TuRGpfPen66um6JLjUYOe0P8mw/Xa8giK6ZhwzEFVLloNEyWewWhbASUQuFgWOUqdSwKAL4v260GtXeu0sBAwaGp63viuhybg5xB/+kTi+jHYyOU6MR3qFQawrA1dp8BbKvJ9zjQB3eKshfi4dXlF5OeSrVkC2fatwsTVPuAq2rt2qjabm25Y6vMypLTJmkyNLziw5smccXHJsGW/WZSqJhniySQJXNkKZiAhVAiKUCQhTxGNNaTt8mCkShM1npgIT27laNf+1JQ5veaEUWelq5BxQwWoWIyjsNAI1cyExvQ4J3XiCk69BeMepVO6UBsuMMkH8Pi6HbKcZuieDYevWfCkJ7vBWQdra/zvpZnvAXo6fcvdiRXkmSd7ZMFATg5vCumBoQFyTEoFae6yu/rnD60LC/auowg7Ni3TNMVlxYtKvKDT8SDJm5F2Ryf3jSL+3MxQBKcJ7Fh2WKsNhNMvxYybp+FqioSeJwbTK03h730J00ZJHVs3YDJmTClk4A4h2QJJlTnKIHSQRJzi3EZFugwnVmjf/LUWRJbknIcnOgiTzBCRZWaSKQEUr1FfjmHyIwIxITClD7GAn1lOxmvl5IhzTOdCNYhv3JABD//XzzXajUA0yW7cPWbr9yKG/PIoK250UeiULkUch/owTTM4wvWc6461lRZSjMZcivkwOjbnrV0WJsc6wGOPoQYHLkrUW6rzf1kKAO7zVkD0XDq84/zTk69dBunsnXd3lsAwbCcvQYRSlaHjOqDGHlzmt5dZ8GMVlyCk9ihITKRuYTxIdgf3l1IjUKsQ0NUjJX1WOLXNukyhKm4xIZZJAQ6gGCZvRw+qSKm7ZUarmM5aSGp5MIXmc5vuK1bunpLGm6fAatGIhSYTJilWWyCiCYkJIyAooRa9CJtpNN8suCEm9ESFJk8Cyxc+YwQF/cnal+yzQPxAI68CWTRdyh/cMsl5/Y6RI7jbdKaytyMFabTbKSGYsUuaH8URZmBjaAfEU2fVl4w5v40dHVEJO7+ulFH11QH9fEPSpBdAVboWhJB2m8gyYtZmwmYhMWsuMIn8sCHoW29Xj6VqnQZotHeOsSzHQsQNquRISqqTHosRSVThk5CzL1VHkOMfS+yh6OGokeltrWy36aLdDnJcLKeVjmI+cxrbyPsiRDoXYqUC2xgJDu1KM6a7FwJRq16h6Nsg0fhkVIlvPHOAD5BAfxEndQRjslERBxoIUzPFNoEiwyxmOVlN0WdzwPaWezbldXOmU4pdiJeYc1VF+hhMDwzMxq3Oy23Wbu5Dr8DYXOd7OUwTOmcPLCjrs2LFDiMj069eP/Dv3P0ZWgGDnzp1ITExEWlpajf04fPgwsrOz0bt3b0HY3fVlc9q42lZ/bS2HV0TFI6T79lJkcQckJ3Pg8POHddBgWAYNEThj1cfA3jONWcadNVBUVkfaskzRIPrThUj+di3mLr2TIrLFxLUqIRWDQpTTH1MzqJ4QxqS6AqnqT7gyFmFU9jKcorOuSG0EObpB8ojam6zzmXG5/iRx+B/ygGySsGGVev6TBPStxVhg0mFOu5kKMFAogKJvTkac/f/2zgU6qurq43tmkkyeJIRHSOQRRBDltWB9gC++KhXFolg+UcDWglRhVUulD1GXVdtaXda1Cq2Pqqu60NpPLYWqIFIVQUG0flRRBCoSXoVAMIRHyPsx99v/E+5kMsydTJJJmsn8D+ty7z33nHPP+Z2TM/vuu+8+Jmgt9Ettl066bo/X7M+4iUZEIvBCuC3cqZP+rmQ5Waxfc2hITv5CMlMekbTkVZKc1V+y+k2WrPxrJCWr6ZhBWs+uGkl76qS4j+uP6wIVdsckI7pNgQJvm/A1yQxThW368dmW8iLZrLa5Wyu+1vHskz4q5F6mHheuzRsmgyW9U2pzmzTk9AkF3lBUzoxzqYYz7TF9CN1WI9WTUqRqhn5AqgtU2MFXVym1lUdU8C2RuuoTUl+ri2LUlkn1qX9LVeUx+SjpUlldO1p21J0lKVIpF/o2ywU16+S8ivfEqjwsFuak0wHzELTH3ox81R6fbbbk03v4Dm6PcFi/fcOHaGt0Ht2mFheZOjfOKKqUwYXJujxzP/HoR7xnyXbp26tIcob6JOGcvsbUQgd6RNWBMgMaYAjA+402eLscq1FVrAaPK1E//h2sLhvPN24b4boRS6vj+4qWhroa/KIkSreMHnLoyDH5n/8rl949i+XlMbktLSpsegq8YfHwYhQIdIjAW6l2VXPnzpVhw4bJoUOHxOv1yuLFi8/4AduyZYvcf//9csUVV8jatWtlzpw5Mm3aNNPMJUuWyPbt22Xw4MHy4YcfyuOPPy79+/eX1uRx4tYeAm/Zy4/LV8Xv6+TmlvLcLCkd0FvKe6VLha/MPKFDsK2sO9Ug4OoTO86r6tXDeFCYucIl01a7Ze5Sr6S5UiTDnSbpqu3o5ukmmQnZKsT2Uk8GeXJW98E6+feWJG+2eBIzdEtzFDbtW8DLgq+uQgrLyuXDEp+8fyJZl87MVIdgLhmfuE+mejbJ8PotUl99vOGHp6ZUf3xKzY8P8kUU1DeuOyGlYfOk6h5bipoepKhtW7rU6RJA0CbDLU9lhb62LB0kZafO1+2/pK4ethPVkupdrwLuq5KR/o5k5Jwt3fUL8PTci8WrP2ShgvtgnSSvLJOkD6qkvr/a+S3IEp/6/YxGoMDbcop4OCqqLZe9VSdkt9rgfqVCC9yI7ak6bl6XpugD0ui0PjI2PU8u6tZXBqdkG3vH/+TqgS1vJV7WxO9Kay3mpX/w3tXlkrK8TD0huKR6SppUT1STBPXVG0nAylxfldbL/+4qNYJlkcq4WEp3vD4Xj8uokBEJh6Rv/W6pU5OJ6lP7dNur2uM9UlvR8JEt7uFOSFMBWAXhjIGSlDFA5xP9YEznFLNBM6zjMpJwUpUEn6vSFcv9bjouUqBvxPCFABbamKJ+bi9Xc+MUjYANr++YW3auOy4H/52tSgu9oHNwj/p9kmPtkp7dT0hWX0tS8rOkXj+Is9R8QDNFUgWjHNlvtMDb5YAKwhCGD1cUqElEw8d2afqhMLzgnJU4Un8lRklW3RBJq+2vZmC9pKggRZLTfMY8rLrCLTWVulWr8A27oqCw4+zDct/0oMg2nlLgbSNAZm+WQIcIvM8//7ycUJvVhQsXmgrNnz9fbr75ZrnggguaVHD27Nnyk5/8REaNGiVH9COAW265RVasWGGEZMQvX77cTBavvPKK7N27V+655x5pTZ4mNw04aQ+B98G1F8pu70H/XRJcXp300nRS1slFkiRZBeEknfST9AONxNoqSaypMJtX7U29Ppck+1SbqdvIt3RBhw1qP/VQqj4oqJsciAiqUfX59OsCv1bVf5umB2rzZrSs+jU08uqd5Kg7Rw678+SAZ6Ds85yrdRwtRxNUcFShZGDNVvmvyjVyUcVr0lNO6mvCbEnw6qtC3TxJWbplGnMBT2K6+bGA4OqCFlfv43JjitegE7gFja++jvNBA1xfZYRqaG0gJNfqxytVFRk6qXaXGtjlneojlZX5qrkZpG2CuYElyamFkpldINl9Dkvv/pWS2r2veDMHqZDs4FlBmXn+XScJ22skaXOVvk7Ue3dzS9VU/RG9QvMknDlxm7q24j8KvE2h1agLpxPax1jlrKS2Ukq0n4tVuC2q0U21coU1p6Sw+pRUnx6rCfp6eaA3S4am9pBhqb1keGpvGaICLuIDQ+BHa4HxnfmYAm/Le8ddXC/JK/ThdJO+TtJQO9IrdbpIRe15ieLLU4HTHfpv116KFksL44F5u2pSN6h5MNxr7dBjnTqNAAx/vkPULXO+TgP9dHrJSaiUzJp94i0vMOYTRhA2AvF+fbDXzHbQ8ZiYkiNJaXlqFqG+ulJ0rko+W44n9pMidx855Osh+2vTZWe5Rw6oRhchS6sLgfuSbJH/Vlk1K+hlZvCYLj/hkaIv1Zf5Ll0OuDhDqusazK0SfRWS5SuUDN/XkppcISkZPknq5pHELK94slR50E0bk6ENS1GvEqqcgBe1+jqXLn2sm37TUFMFodUllSp4nyitlDJdaq26LFG13xnirm9q0lXnUlj621CTvkcSMk6pEsKn869L0lK9kq7ld0vPkLxeuaqMqJDVK9LkUF6J3DMDdKMXKPBGjyVLCk0gskfX0Hkjji0oKDBaWzsDTBJ27NjRROCtU4Hv4MGDMnLkSJMsJydHUvXjg8LCQiPcIh4TBQLyr169WrWCLc9jCjj9X4l+abt7925/FO6Jr2cRcC/4fE1SO9u2hMySiXL97jkq1GLC1hk5RMALI3yV2/AmS9uIA3PS0F5MZrUnXlDh8RU5uvdtSdfJtSxVZzf/b4BOdgGlIz0mf7PXY58K1SoKahrVLEiCOUY10NLzdBuu2tNEqdGFHQ6pCF6tqXDlBi1/hqhnSlNt1LAxNByb/7XshmZpCtUEmHZgb+L1zPKIy+dWbwkqdPtUy6t7V70eG92HiFfrqR4uJSWhWuoSqqQucZfUJlXoXgXjmnqx9A3dsUO95MQnKEs1JvX7pedJrW2Stkbvk1LjlrQqj2RW6laeIAkaV+eypDC7Wr4aUSUFfSql/qjW/KXG2oc6cmt+T02i1Gae/tUKlSggDt0Dxv+pEOrWoeLs+uEaxjP2WJEMwU6PfZ32nD46ifd0vEmnx/a4wh4btP7Bx9AdmeGt+8Dg1jFwTklf/XE+LoM8OfqApw9veldda8vsG0dUpRTIfrMF5m841rGEeusDVOwEl3q0cku9fpzUSDl6tbcgtNTr31Rm4yv7aJQOG1do4dujzhHVT+W3lAluOftQqgzYkyw9tySJPt5Lndsnpal1Up6ipl5enfdOh2Pd6kSnEqnXiUHfUYlOM6qMtGScXh+rg0utaKXSSlZjB69U6VbtS5Janen2aZn7zPw6UFPm60yEUW1JjppYlKTbI7+Rgj5H61XMnzp3qlAYGLqr5jTHKpGLpUznMtVUS7neoUrHLOZut2zTko3vcL0f/plJW3c6QrRMzNCIawwqH2sutUWuSRVPnW61XhVOc6XuZJKUH9GtMWnQEeqNYO9tTqrecNdKL1et9NC9z1Ojf6v7xcLeUy21utW4VQHhqpMyV5UcStQ3d9U6bjG0VFMdHHTUyfDSm6Qou1J/G5u3Pw7OH+7ccAqXgNdIoI0EOkTgLVLXLdCI2QHHEG4Dw9fqIzEtTV+/m4mo4Qrc5EDreviwfn2vx3ZAfgirrcljl4H9xo0b5a677vJHPfHEEzJp0iT/OQ564HVSG8Kc9XfKgNLIhKhwt3Ed0YlQ57DrvkAqezKzc9iTnNO5Hd/cHpN00yf/5nKEvo762HVseJUWOp1TLMQubK0LEHoHlCSbbdIXWa0rhLmiSKBPFMtiUfFGIEEl2eyyJN3a0nLMQ62ZiyK9p0rrKp6rx9pIM0SYDpJndB9s9FWXlokN9W1N2CtJ+jvdo0d+azI75qlQ7TEDCbQngQ4ReD3qoLtev1q1AzSzKeoPMTAEp8E1pMNrjuBrdv7g+EjyBN5z4sSJsmrVKn8UBO7i4mJzDqEawvdJXYGnLeHY3bXywVst+0Nu+sxvdAIyQnUT5xVZsuzScnGfUn1ERoOQC80ENjzZQ1z1qNNEK4B1RHUPvqFfS9CQW+XHxmCOG+5naxOgVVHVrVbUjm9M3uyRvqp02+PjtBYPpUDxYe6re5+Wiw1K4zrV5lR6fKpBg+ak2dIjTqBv4/V1oJqXpEemSXRpnS28QzSVjfg2UU8YrCECkmAsJk7/S4AjfYVWr470MVbsdDg253ZEFGtZUezSlbbaAEnr5NEPjur1w7aYCXgNrr5ha2tVwGoHzXR1uf4t6DBNzohuh3nUz229miD9p8d0pP3s0j8/r+pT3fp6wapVLa3uMQWBCt42oR1+QqeHoP88xE2qjqsJWfc2jNUQZTaJMkXrf+g8/Vv0wYG4gW0M1PQQCcLf/3TKJsW2+AS3MNMc/mu8n6VzakVPB0VDwJjGO57xF5/j/61s8f0dMsAUCG91GUigvQh0iMDbU/0gQlNrBxz369fPPjV7aFLL1ZsBvDngozYEpMvLyxNoiLdu3Wri7Pjc3FyjfW1pHn8hegChFpsdcD/cH6Hh1V6D0G1fb80+P7+X5M9v+1N/akV3SdzilsnzBjlWI9g2zDFhJ7oQiZeGTlRdf1UwbsrKytTe2Pxy+OM78wHsHfEQF/i32Jnri7rF4pimDW/HjapAG96Ou2vb7tRVxjQUT9EM+C1gIIH2JADFTruHCRMmyJo1a6RK1yiHCzF4WRg9erS5L86xYbCPHz9eVq5caeI3bNggmMywwY3Ztm3b5MCBA0brC63suHHjWpWn3RvLG5AACZAACZAACZAACXQqAh3ySHX55ZfLpk2bZNasWUZjM3PmTBk4cKABsXTpUuPGBx4cbrvtNlm0aJG8+uqrJh1clCFAmzZv3jzjtSE7O1sG6FrrN954o7nWmjwmI/8jARIgARIgARIgARKICwId4pbMJnlK1zuH7W5zry7gwiwr68yPjbAuPUwObE8KdrnYtyZPYH4cB5o0dLbXv82ttIb6x+KrMpo0oOc6JnS2MR1Jq2NxTNOkIZKejU4ajGkEuCWLlcAxHbqn6JYsNBfGRo9Ah2h47erCgXwkIZSwi3z4IcEWKrQmT6hyGEcCJEACJEACJEACJNC1CHSIDW/XQsbWkAAJkAAJkAAJkAAJxBIBCryx1FusKwmQAAmQAAmQAAmQQIsJUOBtMTJmIAESIAESIAESIAESiCUCFHhjqbdYVxIgARIgARIgARIggRYToMDbYmTMQAIkQAIkQAIkQAIkEEsEKPDGUm+xriRAAiRAAiRAAiRAAi0mQIG3xciYgQRIgARIgARIgARIIJYIUOCNpd5iXUmABEiABEiABEiABFpMgAJvi5ExAwmQAAmQAAmQAAmQQCwRoMAbS73FupIACZAACZAACZAACbSYgMvS0OJcXTRDTU2N+Hw+07ri4mIBmt69e3eK1rqWvyquN1aL7/lnw9bH4/FIfX192DSd6SKYHz58WPr06SNer7czVS1sXWKNMxrz9ddfmzZ1ljEdFnDAxVhjzTEd0HntfMgx3c6ATxdfXV0tRUVFkpubK0lJSe1yU7fb3W5lt0uFWWjMEaDA69Blt99+u5SXl8vzzz/vkILR0SCwY8cOmTZtmrz88ssyZsyYaBTJMhwIzJ8/X+rq6uS5555zSMHoaBD44osvZPr06bJs2TIZNWpUNIpkGQ4Ebr31VnPlj3/8o0MKRkeDwGeffSYzZsyQFStWyPDhw6NRJMsggQ4nQJOGDkfOG5IACZAACZAACZAACXQkAQq8HUmb9yIBEiABEiABEiABEuhwAp5faOjwu8bADWFPdO6555otBqobs1V0uVySkZEh48aNk/T09JhtRyxUHGN66NChMmTIkFiobszWEWO6W7duHNMd0IOw7+aYbn/QgWM6LS2t/W/IO5BAOxCgDW87QGWRJEACJEACJEACJEACnYcATRo6T1+wJiRAAiRAAiRAAiRAAu1AgCYNIaDCBcvHH38sBw8eNO6y8NqMofUEtmzZIvv37zc8wTQ1NVVSUlJMgTt37pRPP/1UMjMzTbx9F/aBTSKy/TvvvCMDBgwQmC3Y4ejRo7Jx40bjpq5nz552tNk7cQ+Xp0kBcXpSVlYmmzdvlv79+/sJhBvf4Xg69YG/4Dg9qK2tNYxLSkokJydH8DrdDk48w80XTnnsMuN5f/z4cdm0aZOZj2FaZgeOaZsE912JAAXeoN6srKyUm2++2bgkw0Swfv16ufLKK5tMukFZeBqGANxgzZ49W06ePClffvml2QYOHGh+yJYsWSKvvfaawG/pk08+KRdffLERfNkHYYCGuARXQb/5zW/kpptukoSEBJMCP1gLFy6UrKwseeaZZyQ5OVnOO+88c82Je7g8IW4bd1FVVVVy//33y1dffSWTJk0y7Q83vsPxdOqDuIMa1GD4ep07d66Zb7dv327G7tVXX23GtRPPcPOFU56g28bl6euvvy6//e1vzZwLt5AFBQVywQUXGNeFTnN2OJ4c03E5jGKr0Vh4gqGRwNKlSy39w/VHzJs3z/roo4/85zxoGYFdu3ZZOnmekWnv3r2W+t+1dJEMc00nXOvhhx82x+yDM3CFjFBNmHXPPfdYGKOXXHKJpQKZP933vvc9S31nmnMVIiwVGizVglnhuDvl8Rcaxwe7d++21A+pYX3nnXf6STiNbyRw4hmuD/wFx+nBY489ZqmfaH/r77vvPmvVqlXm3IlnuPnCKY//BnF6oA9q1nXXXWft2bPHEFCf89aUKVOsY8eOWRzTcToo4qDZje8/Y0tOb7fa4ik3cAEEHGNxBIbWEdDJU/r27St///vfBRqFiooKU5BOtDJy5Ej/K/hAzuyDyFhjRT1oZKAdDwzQOsJ0BHwR8FoYZiSFhYXixD1cnsCy4/UY4/bee++VWbNmNUHgNL7D8XTqgyYFx+kJFkfBmwo74M0QNLjheDrNF+Hy2OXH6x5mei+88ILgbRsC3l7AXAdzCsd0vI6Krt9uCrxBfYxXanApZAccw5aMoXUE8PoXtoqnTp0y+xtuuEFgU4flhGG3a4dAzuwDm0r4PZZinjp1qt+MwU6N5VbhOijQ9hGsVXvjyD1cHrvceN5jdakRI0acgcBpfIfjGW7sn3GDOIvAsrWJiYmm1evWrTMPbldddZVZFttpTDvNF+H6IM6whmyu7V7M5/PJ73//e5k8ebLA1p9jOiQuRnYBAg0Gf12gIdFqAp588ZRrB2gJ7A+s7DjuIyeApT+xQcOIgI9LoO2FrakTZ/ZB5HxDpQzmhzQYx7DjDb5mj+/g+MA8oe7BuAYCTuMbNr6B4zuQZzBruw/ItJHAypUr5c9//rMsXrzY+OfGMu8t5RnMObAPGu8U30eYj3/1q1+Jvs02bzBAg2M6vsdEV249NbxBvYsnXGjC7IDjvLw8+5T7FhLAa3RMqnbA1+3QcPXq1esMzrm5uSYZ+8Cm1bp9jx49zEeXgdztcezEPVye1tUiPnI5je9wPJ36ID6INd/KF198UZYtWyaPP/648TyCHOF4Os0X4fI0X4uunwJmOj/72c/Mwj8PPvigQLuOwDHd9fs+XltIgTeo5ydMmCBr1qwxNk149f7hhx/K6NGjg1LxNFICGzZskKefftokxwSL15SXXXaZjB07VrZt2yYHDhww2kf9MMWsTIWE7INI6YZOB+35+PHjBVoyBPRB9+7dzebEPVye0HdhrM021PgOx9OpD0hU5M0335R3331XnnrqKWN7bjMJx9NpvgiXxy43nvcPPPCAWUn07rvvNm9+bBZOc3Y4nhzTNj3uOzMBrrQW1Dt4vfjLX/7SCGPwaTpz5ky5/vrrg1LxNFICpaWlxmXWoUOHpLi42LhzWrBggflY7Y033jBanOzsbKPJ+fWvf21MHdgHkdJtTIcf/bVr1wrsehHg93jRokXmhwzjGO607CWFnbiHy9N4p/g+eu+994xQ9uijjxoQ4cZ3OJ5OfRDfdEWmT58uR44caWJ/rt4E5I477nAc0+Hmi3B9EM+s//Wvf4l6dzEIAm39n3jiCcnPz3ecs8Px5JiO5xEVG22nwOvQT/jICra7eKplaDsBaHdhU2cLZHaJcDKPV+/p6el2lH/PPvCjaPXBiRMnjC/e4ALCcXfKE1wGzxsJOI1vpHDiGa4PGkvmUTABJ57h5gunPMFl87yRAMd0IwsedQ0CFHi7Rj+yFSRAAiRAAiRAAiRAAg4EaMPrAIbRJEACJEACJEACJEACXYMABd6u0Y9sBQmQAAmQAAmQAAmQgAMBCrwOYBhNAiRAAiRAAiRAAiTQNQhQ4O0a/chWkAAJkAAJkAAJkAAJOBCgwOsAhtEkQAIkQAIkQAIkQAJdgwAF3q7Rj2wFCZAACZAACZAACZCAAwEKvA5gGE0CJEACJEACJEACJNA1CFDg7Rr9yFaQAAmQAAmQAAmQAAk4EKDA6wCG0STQVQlgha9bb71V9uzZI3fffbdceeWV8sMf/lAOHz7sb/Lvfvc7WbZsmf8cB/fdd5+8++67Jg5LkL799tvyzDPPyNSpU+UHP/iBKW/fvn1y++23m+W4ly9f3iR/cyeoD5ZA/ta3viUPPPCAbNy40ZwH5vvTn/4kN9xwg7nnkiVLBMvKIkTSJqT77LPPTNuvuOIKWbhwoWDJaztgNa57771XcG3GjBny7LPPimVZ9uWI9k71W7NmjcyfP1+OHj3qLwf1x3LaCOC5evVqWbx4sUyZMkXuuusuwfKvDCRAAiRAAtEhQIE3OhxZCgnEDIH6+nojzEGwPHLkiBEeIchC0LPDm2++KR999JF9avYQgLdt22aO33rrLfn+978vq1atkssvv1w++OADueaaa+Tb3/62dOvWTYYOHWqExk8++aRJGU4nJSUl8s1vflPeeecdU5/333/flPfXv/7Vn+WOO+6Qn/70pzJ48GC56KKL5NFHH5Xp06eb65G0ad26dXLhhRdKWVmZEcg//vhjGTlypF/o/e53vyvr16+XG2+8UcaOHSt33nmnPPLII/77N3cQrn4TJkwwDws/+tGPTDHgtmjRIpk4caI5B885c+bIX/7yF5k5c6bhfNlll0lhYWFzt+V1EiABEiCBSAioBoOBBEggjghUVlZCbWmpxtbfahU0TZxqPE3cpEmTLNWA+q/jYMiQIZZqfk3c1VdfbZ177rmWalbN+d/+9jeTXzWU5hz/qZDX5B7+CyEOVNNs9e7d2/L5fP6r559/vqWCsznfuXOn5Xa7rZdeesl/fdeuXeae7733nhVJm0aPHm1NmzbNnx8HiFONtInr3r279fTTT/uvq1Bqvfbaa/7zcAfN1Q95//GPf1gJCQnWc889Z/Xq1ct66KGH/EWCpz4o+HniwqBBg6zbbrvNn4YHJEACJEACrSeQEIlQzDQkQAJdj8C4ceP8jerfv785Li8v98c1dzBq1ChRAc4kU+HM7CdPnuzPpkKdFBUV+c/DHfzzn/+Uq666Slwulz+ZCoGycuVKc47rOs3J5s2b5fPPP/enSU9PF1wbP368iXNqU3V1tcmXm5trzDjsAjwej8mP89mzZ4sKmPLiiy8as4prr71Whg0bZicNu2+uft/4xjdMHX/+858bzTi02TAnCQzQlNs8EQ+Ne6Qa8sByeEwCJEACJHAmAZo0nMmEMSQQFwTS0tL87VTtqTmGUGmHwGPE1dTU2JfMvkePHk3OcQJzhtYEmDT069evSdasrCz/OexrIQx6vV5BXe1twYIFTYRSpzaVlpaKao8FArKdF3vVZMt1111n7gOb2tdff11Ucy2PPfaYDB8+/Ayh1F+hoINI62e3CTbHwSE/P79JVHZ2trTkAaRJZp6QAAmQAAk0IUANbxMcPCEBEgABCJawdbUDPg47ePCgfRr1PTSpn376aZNyYddqh3POOcd8mAY7YdjvIsBu94UXXhA1tbCTOe6hbYYwnpeXJw8//LA/HT68S0xMlIqKCmM/C60yNgjHDz74oKjZgfziF7+Q5ORkf55QB5HUb/v27UaAxgdqKBM2yIFaXtgYBwbYM48ZMyYwisckQAIkQAKtJEANbyvBMRsJdGUCECLfeOMNKSgoEGhH8bEYBMxgrW+0GPz4xz8WCLi4z9atW42Xhk2bNvmLxwdc0LzCiwMEx6qqKiM0wptBpFpleJKAuQLMJNCWDRs2CMwW4DkhJSVF/vCHPxgBFNpatQmW4uJiOeuss5oVdlHJ5uoH7fh3vvMd8yEevFhAg4y2bNmyxd9GmGosXbrU3Bt7eJSAmQUDCZAACZBA2wlQ4G07Q5ZAAl2OAATPAQMGGI8IOTk5RrsKG9NAG9toNhqaTAii0HJecsklAg8Kt9xyi1/YhBYW5gbQxI4YMUJ69uxpvB5AgMVxJAGuzuABAZ4dMjIyjDAJTwzXX3+9aRc0r7t37zZCLswJ1q5dKytWrIikaKMlDlc/2O7CBdqTTz5pyps1a5ZxPwYhGMI1wqWXXmo0yjB7QPqnnnrK78XBJOB/JEACJEACrSbgUo1No9Feq4thRhIgga5IAG7LIBympqa2a/Og3YQZBdyZ2QEuvNT7gdH82nHYnzx50vjfDWVDHJjO6Rj2s2hX3759QyaBKQeEUJhBtCa0pn4w1YANM7TMcEUG04v2erhoTZuYhwRIgARinQA1vLHeg6w/CbQjAWh32yrswh4WNsDhNiwyAR+5MGOAwAlzCiz8gAUggkNmZqa0VthFWdAWOwm7uI4P2wKF3Ujqj3x2aGv9YEZBYdemyT0JkAAJRIcAP1qLDkeWQgIk4EAAH2ipf1uHqyJwDXbgwAGzzZs3T7788kvzIRo+GJs7d65jvo66EEn9A1epa029YMYATToDCZAACZBA+xCgSUP7cGWpJEACrSQAn7kwb2AgARIgARIggWgRoMAbLZIshwRIgARIgARIgARIoFMSoA1vp+wWVooESIAESIAESIAESCBaBCjwRoskyyEBEiABEiABEiABEuiUBCjwdspuYaVIgARIgARIgARIgASiRYACb7RIshwSIAESIAESIAESIIFOSYACb6fsFlaKBEiABEiABEiABEggWgT+H/GP4cGLQKLzAAAAAElFTkSuQmCC" /></p>
<!-- rnb-plot-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
<div id="number-of-samples-that-express-a-gene-as-a-quality-measure" class="section level3">
<h3>Number of samples that express a gene as a quality measure</h3>
<p>Now we have an idea of what samples we probably want to get rid of. But what if our data contains genes that we can’t reliably measure in these cells? Some gene types, like many long non-coding RNAs for example, are very tissue specific and wouldn’t be useful for us to look at it. For example if only one cell is supposedly expressing a particular gene at a low level, do we think that is biologically relevant, or is it more likely a technical artifact? Because our single-cell data is already riddled with zeroes, it’s best just to get rid of these genes that we suspect are not being reliably measured by many cells. To do this, we will use our detection matrix to add up how many samples express each of these genes.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBNYWtlIGEgdmVjdG9yIHRoYXQgY29udGFpbnMgdGhlIG51bWJlciBvZiBjZWxscyB0aGF0IGV4cHJlc3MgYSBwYXJ0aWN1bGFyIGdlbmVcbm51bV9zYW1wbGVzX3RoYXRfZXhwIDwtIGFwcGx5KGRldGVjdGlvbl9tYXQsIDEsIHN1bSlcbiMgTGV0J3MgdGFrZSBhIGxvb2sgYXQgd2hhdCB0aGlzIGxvb2tzIGxpa2U6XG5xcGxvdChudW1fc2FtcGxlc190aGF0X2V4cCwgZ2VvbSA9IFwiZGVuc2l0eVwiKSArIFxuICBnZW9tX2RlbnNpdHkoZmlsbCA9IFwibGlnaHRibHVlXCIpICtcbiAgZ2VvbV92bGluZSh4aW50ZXJjZXB0ID0gNSwgY29sID0gXCJyZWRcIilcbmBgYCJ9 -->
<pre class="r"><code># Make a vector that contains the number of cells that express a particular gene
num_samples_that_exp &lt;- apply(detection_mat, 1, sum)
# Let&#39;s take a look at what this looks like:
qplot(num_samples_that_exp, geom = &quot;density&quot;) + 
  geom_density(fill = &quot;lightblue&quot;) +
  geom_vline(xintercept = 5, col = &quot;red&quot;)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-plot-begin eyJjb25kaXRpb25zIjpbXSwiaGVpZ2h0Ijo0MzIuNjMyOSwic2l6ZV9iZWhhdmlvciI6MCwid2lkdGgiOjcwMH0= -->
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAArwAAAGwCAYAAABLkLalAAAEGWlDQ1BrQ0dDb2xvclNwYWNlR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19zn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7N0HfBRl/vjxb3pIQgIhhCSUUKUKAlIUsSCCZzlFPQW8UxHFOzzLFVDPcurhNU9BPev/p9i7p6CIIIhUqdICSEd6r4EUUv7zfXBiAtmQsju7O/t5fI27OzvzzPO8n8ny3WefeSas2EpCQgABBBBAAAEEEEDApQLhLq0X1UIAAQQQQAABBBBAwAgQ8HIiIIAAAggggAACCLhagIDX1c1L5RBAAAEEEEAAAQQIeDkHEEAAAQQQQAABBFwtQMDr6ualcggggAACCCCAAAKRoUiQn5/vfLUPHxbJyRVpkOr8sTkiAggggAACCCDgYoHw8HCJjPQc1oaF4rRkO3bscKzJo6KiJCUlRbLv+oNE/e8z2b9wjmPHduuB4uLipKCgQPzyxcWFqLGxsVK3bl3ZuXOnMEuhdxo4MTFRjh49KoWFhd7JMMRzSUhIkPj4eNm1a1eIS3iv+vo3f+DAAe9lGOI5JSUlmWBr3759IS7hvepX9RyNiYmR5ORkjwVgSINHGt5AAAEEEEAAAQQQcIMAAa8bWpE6IIAAAggggAACCHgUIOD1SMMbCCCAAAIIIIAAAm4QIOB1QytSBwQQQAABBBBAAAGPAgS8Hml4AwEEEEAAAQQQQMANAgS8bmhF6oAAAggggAACCCDgUYCA1yMNbyCAAAIIIIAAAgi4QYCA1w2tSB0QQAABBBBAAAEEPAoQ8Hqk4Q0EEEAAAQQQQAABNwgQ8LqhFakDAggggAACCCCAgEcBAl6PNLyBAAIIIIAAAggg4AYBAl43tCJ1QAABBBBAAAEEEPAoQMDrkYY3EEAAAQQQQAABBNwgQMDrhlakDggggAACCCCAAAIeBQh4PdLwBgIIIIAAAggggIAbBAh4/diKo0aNkvPPP18ef/xxP5aCQyOAAAIIIIAAAu4WiHR39QK3dgMGDJB58+ZJ7brJ8tJLL8nGjRtl7NixgVtgSoYAAggggAACCASpAD28fmi48ePHy7z586X/TbfLM998LxcPvFkmTZokH374oR9KwyERQAABBBBAAAF3CxDw+qF9//KXByUlvaFcM/xP5ugD//Sw1EvLkFFPPOGH0nBIBBBAAAEEEEDA3QIEvA6378qVK2X//n3S/ze3S1RMrDl6RGSkXDnsHtm7Z49o7y8JAQQQQAABBBBAwHsCBLzes6xUTv/5z38kLDxcuvW7vMz2515xjdRKqC0vvPBCmfW8QAABBBBAAAEEEKiZAAFvzfyqvPesWbOlXY9ekpicUmbfyKgo6XrxL2TVDz+UWc8LBBBAAAEEEEAAgZoJEPDWzK9KexcWFEh29hHpfGG/cvc7u+8v5Hh+PhevlavDSgQQQAABBBBAoHoCBLzVc6vWXkePHjP7te9xXrn7t7PWR8fWknfeeafc91mJAAIIIIAAAgggUHUBAt6qm1V7j9y8XKlTP1UaZDYrNw8d1tC22zkMayhXh5UIIIAAAggggED1BAh4q+dWrb0KCwqlw7kXVriv9vJmH8mW3bt3V7gdbyKAAAIIIIAAAghUToCAt3JONd5q3759Umz917prjwrzatv9XOv9Ynnrrbcq3I43EUAAAQQQQAABBConQMBbOacab7Vu3VqTR/Mzz6owr4YtW0tcYpJMmTKlwu14EwEEEEAAAQQQQKByAgS8lXOq8VZbtmyVsLAwSctsXmFeuk3rLt1l/YYNFW7HmwgggAACCCCAAAKVEyDgrZxTjbc6ePCg6EVpGtCeLrXo2EWOZh+VY8dOzOpwuu15HwEEEEAAAQQQQMCzAAGvZxuvvVNUVCS5eXlWwBtdqTw14C0uLpIvvviiUtuzEQIIIIAAAggggIBnAQJezzZee+fbb781AWyE1cNbmdS0XUdz++FJkyZVZnO2QQABBBBAAAEEEKhAgIC3AhxvvfXVV1+ZrCIjKxfwxtSqJQ1bnCHLly/3VhHIBwEEEEAAAQQQCFkBAl4Hmn7BggUSEREp4RERlT5a8w5nyZ69eyu9PRsigAACCCCAAAIIlC9AwFu+i1fXrlmzRhKT61Upz8y2HSQvN5cbUFRJjY0RQAABBBBAAIFTBQh4TzXx+pp9+/dL7eSUKuXbpE17s/3EiROrtB8bI4AAAggggAACCJQVIOAt6+H1V3v27DE9tbXrJFcp78at2popzGbPnl2l/dgYAQQQQAABBBBAoKwAAW9ZD6+/+vjjj02etetWLeCNjo2V1CZNZcWKFV4vExkigAACCCCAAAKhJEDA6+PWrmXNuKApopIzNJQuTvP2nWTXrt2lV/EcAQQQQAABBBBAoIoCBLxVBKvq5uHh1SdudEZbyck5Jvn5+VU9LNsjgAACCCCAAAII/CRQ/WgMQp8L6Fy8xcXFMmvWLJ8fiwMggAACCCCAAAJuFSDgDeCWbdSytSndzJkzA7iUFA0BBBBAAAEEEAhsAQLeAG6f5LQMiakVJ0uXLg3gUlI0BBBAAAEEEEAgsAUcC3jz8vJEp9iaM2eOHD9+3KPKXuvuYpMmTZLVq1efso2umzx5sug2pdPu3btFb9+7cePG0qtd8bxRq9ayadMmV9SFSiCAAAIIIIAAAv4QcCTgzcnJkVtuuUWmTZsmb7/9towcOdKMTT25wosXL5YhQ4aI3plMt/n0009LNhk9erQ8+eSTotsMHTpUNm/ebN774osvZMSIEbJ161Z56KGHZMKECSX7uOFJI2s+3gMHD7qhKtQBAQQQQAABBBDwi0CkE0f94IMPpEePHnLvvfeaw91xxx0yb9486dmzZ5nDjxkzRkaNGiWdOnWS66+/Xm677Ta5/PLLZfv27aLjWHVOW5314P3335d33nlH7r//ftPj+9hjj0nTpk3Nfs8995zZp0zGQfxCL1zTWwxnZ2dLQkJCENeEoiOAAAIIIIAAAv4RcCTgXbdunfTr16+khl26dJGVK1eWCXgLCgpML23Hjh3Ndg0aNJC4uDjZtm2bGaqg6+0pvnR/7ckNCwuTZ5991myvwyR0yIQGviend999V44ePWpWp6WlyQUXXHDyJj57HRUVVaO805u1NPsvXLjQVYF8TVDUNDIyUmpqW5MyuGlftdSkf28k7wjouameRUVF3skwxHNRT/28j4+PD3EJ71U/IiICT+9xmn+TMPUiqJVVVT11VquKkiMB786dOyUxMbGkHPpchyCUTjoOVz/M9EPNTklJSbJ//37ZsWOH6HM76f779u2zX8rhw4flxhtvtOaszZH/9//+X8l6+8nYsWOtGzjsMi81WL7iiivst3z+GB0dXaNjpDdrYfbXgHfgwIE1youdEahIoHbt2hW9zXtVFOALWRXBKrE552glkKqwCZ5VwKrkpphWEqqSm1XFU2PAipIjAa9G6YWFhSXl0N5c+w5k9sqTt9H1ul2sdYvdk987eX8NgMePHy8zZswQHS4xbty4Mvl//fXX9mHMowbQTiW7Z7m6x0tukC5RMbEyf/580S8OpBM9kXoOcEMO75wN+jdWt25d86XwdN+QvXNE9+ein0n6t1/6c8/9tfZdDXU4l3aI2B0XvjtS6OSsf/MHDhwInQr7uKbaKae/lpXujPPxIV2ffVXP0ZiYmAp/qXTkorWUlBTTU2u3jvbaZmRk2C/NY7169cw/EDqbg53s7erXr3/K/unp6SbgmTt3rtlce4Z1qEKdOnVkxYoVdhaueExr2lx+/PFHV9SFSiCAAAIIIIAAAk4LOBLw9u7dWyZOnCi51sVXOqWYTk3WuXNnU1d9rYt+M9IL27SnVpP21mp0r0u3bt0kKytLtmzZYnp9P//8c+nevbsZw/nSSy+Z3k/dR6ct029XmZmZ+tI1qZF14do+60sCCQEEEEAAAQQQQKDqAo4Maejbt6+5oGzQoEHmwjMdi9qsWTNTWh1fq2PddAaH4cOHl0xHpheoPfLII2Yb/Xlw2LBhZtaG5ORkE9AOHjzYjPfV/TTofeWVV0S7s3XGBu0RdlNKs8bxzvtqvLkAxr5wz031oy4IIIAAAggggIAvBRwJeLX39m9/+5scOXLEjK21rwrXiukcunbSnlmdwuygNe+sDk0onfRCs/79+4sOeSg9PddZZ51lAl43T9uVltnCBLt6xzW7Z7y0Dc8RQAABBBBAAAEEPAs4MqTBPrxebVc62LXXn/x4crBrv689waWDXXu9PnpaX3qbYH2elnmiN1wvXCMhgAACCCCAAAIIVE3A0YC3akVja1sgtXFT83TZsmX2Kh4RQAABBBBAAAEEKilAwFtJKH9uFlOrliTWS5H169f7sxgcGwEEEEAAAQQQCEoBAt4gabb0pi3MDTiCpLgUEwEEEEAAAQQQCBgBAt6AaYqKC5JmBbyHj2RXvBHvIoAAAggggAACCJwiQMB7CklgrmjQpKnk5+Vyd7HAbB5KhQACCCCAAAIBLEDAG8CNU7poaZnNRW/7unDhwtKreY4AAggggAACCCBwGgEC3tMABcrb2sOriYDXMPA/BBBAAAEEEECg0gIEvJWm8u+GKQ0bi3VrOVm5cqV/C8LREUAAAQQQQACBIBMg4A2SBouKjpE69erLhg0bgqTEFBMBBBBAAAEEEAgMAQLewGiHSpWiQdPmsmvXrkpty0YIIIAAAggggAACJwQIeIPoTGhg3XHtSDZTkwVRk1FUBBBAAAEEEAgAAQLeAGiEyhYh1bpwLS83V4qKiiq7C9shgAACCCCAAAIhL0DAG0SnQGqjTDM12dKlS4Oo1BQVAQQQQAABBBDwrwABr3/9q3T01MaZZvsFCxZUaT82RgABBBBAAAEEQlmAgDeIWt8OeFesWBFEpaaoCCCAAAIIIICAfwUIeP3rX6Wjx8bFS3xikmzcuLFK+7ExAggggAACCCAQygIEvEHW+trLu3379iArNcVFAAEEEEAAAQT8J0DA6z/7ah05tUkzOXT4cLX2ZScEEEAAAQQQQCAUBQh4g6zVUxs1kVxrajISAggggAACCCCAQOUECHgr5xQwW9W3At7CggLZtm1bwJSJgiCAAAIIIIAAAoEsQMAbyK1TTtnqN2xi1s6fP7+cd1mFAAIIIIAAAgggcLIAAe/JIgH+2p6abPny5QFeUoqHAAIIIIAAAggEhgABb2C0Q6VLUad+A4mIjJK1a9dWeh82RAABBBBAAAEEQlmAgDfIWj8sLEzqpWfI5s2bg6zkFBcBBBBAAAEEEPCPAAGvf9xrdNTUxk1l3759NcqDnRFAAAEEEEAAgVARIOANwpbWmRqOHjsWhCWnyAgggAACCCCAgPMCBLzOm9f4iPUbNpb8vDwpsKYnIyGAAAIIIIAAAghULEDAW7FPQL6rAW9xcbEwU0NANg+FQgABBBBAAIEAEyDgDbAGqUxxUqyAV9P3339fmc3ZBgEEEEAAAQQQCGkBAt4gbH775hOrVq0KwtJTZAQQQAABBBBAwFkBAl5nvb1ytLjaiRIbHy/r16/3Sn5kggACCCCAAAIIuFmAgDdIW1d7eXfs2BGkpafYCCCAAAIIIICAcwIEvM5Ze/VIeovhgwcPejVPMkMAAQQQQAABBNwoQMAbpK2qF67l5OYGaekpNgIIIIAAAggg4JwAAa9z1l49Uv2MxnI8P1+ys7O9mi+ZIYAAAggggAACbhMg4A3SFk3JaGRKvmjRoiCtAcVGAAEEEEAAAQScESDgdcbZ60ex5+JdtmyZ1/MmQwQQQAABBBBAwE0CBLxB2pop1pAGTatXrw7SGlBsBBBAAAEEEEDAGQECXmecvX6U6NhYiU+qI5s2bfJ63mSIAAIIIIAAAgi4SYCAN4hbM7VRE9m1a1cQ14CiI4AAAggggAACvhcg4PW9sc+OkNq4qRw8dMhn+ZMxAggggAACCCDgBgEC3iBuxXrWTA15uXlBXAOKjgACCCCAAAII+F6AgNf3xj47gk5NVlBwXA4cOOCzY5AxAggggAACCCAQ7AIEvEHcgnrzCU0LFy4M4lpQdAQQQAABBBBAwLcCBLy+9fVp7vbNJ7Kysnx6HDJHAAEEEEAAAQSCWSAymAtf3bLHxcVVd9cq7xcZ6TviehkNTXnWrVsnTtapyghe3iE6OlrU1Ze2Xi5yQGcXFRVlylerVq2ALmcwFU7PTfUsKioKpmIHbFn1HA0LCwupzzlfN0ZERASeXkTWv/nw8HBMvWjq7XPUd9GYFyvt7awKCwu9naVf8ouMipbE5HpmLl631KkykBpEaH1Dqc6VcanuNvqhokldi4uLq5sN+5USUEc9Pwl4S6HU4Kl9bvI3XwPEk3a1z9GTVvOymgL2ZyfnaDUBy9mtqueofuGoKIVkwJuX59zMBgUFBRX51/i9+tZcvDt37hQn61TjQtcwAw3Q1DU/P7+GObG7CmjPmSY9h+wPbbOC/1VbICYmxpyf/ONXbcIyO9q/QoTS51wZAB+80F8F8fQebKx1Myj9LMXUe6ZVPUf1c7eiVHE4XNGevBcQAqmNMuXw4SMBURYKgQACCCCAAAIIBKIAAW8gtkoVyqRz8ebm5VZhDzZFAAEEEEAAAQRCS4CAN8jbu37DxlJo/by/e/fuIK8JxUcAAQQQQAABBHwjQMDrG1fHck35aS7eRYsWOXZMDoQAAggggAACCASTAAFvMLVWOWXVIQ2amIu3HBxWIYAAAggggAAClgABb5CfBskN0s2VoWvWrAnymlB8BBBAAAEEEEDANwIEvL5xdSzXSGtC9qR69WXz5s2OHZMDIYAAAggggAACwSRAwBtMreWhrDoXLxetecBhNQIIIIAAAgiEvAABrwtOgfqNM+VIdrYLakIVEEAAAQQQQAAB7wsQ8Hrf1PEcU6wL1/Jynbt7nOMV5IAIIIAAAggggEANBAh4a4AXKLvWt6YmKywskG3btgVKkSgHAggggAACCCAQMAIEvAHTFNUvSIp18wlNCxYsqH4m7IkAAggggAACCLhUgIDXBQ2rQxo0rVixwgW1oQoIIIAAAggggIB3BQh4vevpl9zqpqZJeHiErFu3zi/H56AIIIAAAggggEAgCxDwBnLrVLJs4RERUie1gWzZsqWSe7AZAggggAACCCAQOgIEvC5p61RrajLm4nVJY1INBBBAAAEEEPCqAAGvVzn9l1l968K1o8eO+a8AHBkBBBBAAAEEEAhQAQLeAG2YqhZLZ2rIy82t6m5sjwACCCCAAAIIuF6AgNclTaxz8RYVFXHhmkvak2oggAACCCCAgPcECHi9Z+nXnOy5eL///nu/loODI4AAAggggAACgSZAwBtoLVLN8ugYXk1ZWVnVzIHdEEAAAQQQQAABdwoQ8LqkXRPr1ZfIqChZv369S2pENRBAAAEEEEAAAe8IEPB6x9HvuYSFhUlyWkPZunWr38tCARBAAAEEEEAAgUASIOANpNaoYVkaNGkqe/furWEu7I4AAggggAACCLhLgIDXRe2p43iP5eS4qEZUBQEEEEAAAQQQqLkAAW/NDQMmhxRrarL8vDwpKCgImDJREAQQQAABBBBAwN8CBLz+bgEvHl+nJisuLpbly5d7MVeyQgABBBBAAAEEgluAgDe4269M6es3OjE12eLFi8us5wUCCCCAAAIIIBDKAgS8Lmp9ey7elStXuqhWVAUBBBBAAAEEEKiZAAFvzfwCau+42kkSGx8vGzZsCKhyURgEEEAAAQQQQMCfAgS8/tT3wbHrWxeubd++3Qc5kyUCCCCAAAIIIBCcAgS8wdluHkvdILOZHDh40OP7vIEAAggggAACCISaAAGvy1pcpybLYS5el7Uq1UEAAQQQQACBmggQ8NZELwD3rd+oiRQcPy6HDh0KwNJRJAQQQAABBBBAwHkBAl7nzX16RHumhgULFvj0OGSOAAIIIIAAAggEiwABb7C0VCXLWb9hE7Pl0qVLK7kHmyGAAAIIIIAAAu4WIOB1WfvWy2gkEhYmq1evdlnNqA4CCCCAAAIIIFA9AQLe6rkF7F6RUVFSJyVVNm3aFLBlpGAIIIAAAggggICTAgS8Tmo7dKw0a2qyXbt2OXQ0DoMAAggggAACCAS2AAFvYLdPtUpXv3GmHMnOrta+7IQAAggggAACCLhNgIDXbS1q1Se1Uabk5eW5sGZUCQEEEEAAAQQQqLoAAW/VzQJ+j1Srh7e4qEiysrICvqwUEAEEEEAAAQQQ8LUAAa+vhf2Qvz01GXPx+gGfQyKAAAIIIIBAwAkQ8AZck9S8QKmNT8zFu3LlyppnRg4IIIAAAggggECQCxDwBnkDllf8uNpJUis+QdauXVve26xDAAEEEEAAAQRCSoCA16XNndqkqWzfvt2ltaNaCCCAAAIIIIBA5QUcC3h11oDZs2fLnDlz5Pjx4x5LuHfvXpk0aVK5dwrTu4dNnjxZdJvS6cCBAzJlyhTZsWNH6dUh/Twts7kcOHgwpA2oPAIIIIAAAgggoAKOBLw5OTlyyy23yLRp0+Ttt9+WkSNHSnFx8SktsHjxYhkyZIisWbPGbPPpp5+WbDN69Gh58sknRbcZOnSobN682bw3btw4ufvuu2Xjxo3y2GOPyZgxY0r2CeUnOlNDbk5uKBNQdwQQQAABBBBAwAg4EvB+8MEH0qNHD3nooYfkhRdekGPHjsm8efNOaQINVkeNGiV33XWXvPLKK/Laa69Jfn6+uU3uzJkzzbr77rtPBg0aJO+8844UFhbKW2+9JY8//rjcfvvt8vTTT5ueXu3xDfWU2rip5VMgW7ZsCXUK6o8AAggggAACIS4Q6UT9161bJ/369Ss5VJcuXURnEOjZs2fJuoKCAtm6dat07NjRrGvQoIHExcXJtm3bTO+trg8PPxGf6/4TJkyQiIgIeeONNyQ+Pt7sk5ubK9nWHcY0EC6djh49WtKjrPuEhYWVftunz508VumKaA+vprlz50qTJidmbSj9frA/V1d/2Qa73cnltx3tx5Pf53X1BDhHq+dW0V6coxXpVP09PKtu5mkP++8dU09C1VvvTU9HAt6dO3dKYmJiSW31uQa3pdPu3btN4Fq6cklJSbJ//34zNlef20n337dvn3lpB7tF1o0WnnnmGbn00kslJSXF3tQ86jrNX1PXrl3l3XffNc+d+J9dPieOVfoYdsCrXzbS0tJKv8VzBMoV0C+ZJO8J+Otv33s1CLyc+Czzbpvg6V1PzQ1T75pWxVNHD1SUHAl4tVe1dK+r9ubWqlWrTLlO3kbf1O1iY2NNT25F++sFcTqsQccFP/jgg2Xy1Rc6lELHEWuqV6+eHHTwYi7tdfZHSqpXX6JiYmXZsmWO1teJukZHR5vzqfQ54cRx3XqMqKgo82Xz0KFDJb+EuLWuTtVLP9/0c0m/iJNqLhATEyO6HD58uOaZkYMR0F9QTxcgQFV5AfXUX6H1V2aSdwSqeo6W7jAtrwSOBLza46o9tXbS540bN7ZfmkcNRHXogf4joR9smnS7jIwM0R5iDdzspOvT09PNS/2D1XG9DRs2lBEjRpjg2N7Ofuzfv7/91Dw6OZuDBu3+SnoDik2bNpUE+/4qh7ePqye1uur4blLNBfSLovZG6pfC8i4mrfkRQi8H/RKhX3b5UuadttcOEf13we648E6uoZ2Ldibh6b1zQDtiIiMjMfUeqenwrMo5aseOnorgyEVrvXv3lokTJ5p/AHRKMZ2arHPnzqZM+loXPVH0wrbx48eb9TNmzJC6deuapVu3bpKVlWUuwNJA5/PPP5fu3bub7f76179K69at5f777y832PVU8VBYn96spewr9UUjFOpMHRFAAAEEEEAAgZMFHOnh7du3r5mDV2dX0C7/gQMHSrNmzUxZxo4dK9obcu+998rw4cNLpiPT7R555BGzjY7ZHTZsmNx2222SnJwsmZmZMnjwYFm1apW5KEsvzPrwww9L6vbf//635OK3kpUh+KRBk2ayeNrkEKw5VUYAAQQQQAABBH4WcCTg1d7bv/3tb3LkyBEzdldf20mHIdhJA1mdwkzH2NapU8debR6vuOIK0aEJOuQhISHBrGvbtq3odGWk8gUaWHdbK7R6xHWmCx3yQUIAAQQQQAABBEJRwJEhDTZs7dq1zdAF+7Wnx5ODXXs77Qm2g117HY+eBbSHV5MOISEhgAACCCCAAAKhKuBowBuqyP6qtz012dKlS/1VBI6LAAIIIIAAAgj4XYCA1+9N4LsC6NRk0bG1ZPXq1b47CDkjgAACCCCAAAIBLkDAG+ANVNPiNbDuuMbthWuqyP4IIIAAAgggEMwCBLzB3HqVKHt681ZMTVYJJzZBAAEEEEAAAfcKEPC6t21NzdKaNmcibJe3MdVDAAEEEEAAgYoFCHgr9gn6d9OsmRqKCgtl7dq1QV8XKoAAAggggAACCFRHgIC3OmpBtE+DzOamtLNmzQqiUlNUBBBAAAEEEEDAewIEvN6zDMic0jJPzMXL1GQB2TwUCgEEEEAAAQQcECDgdQDZn4eolVBbEurUZUiDPxuBYyOAAAIIIICAXwUIeP3K78zBM5q1NLcXduZoHAUBBBBAAAEEEAgsAQLewGoPn5QmrWkLOXzkiE/yJlMEEEAAAQQQQCDQBQh4A72FvFC+9GYtJDc31yxeyI4sEEAAAQQQQACBoBIg4A2q5qpeYdOtHl4pLpbvvvuuehmwFwIIIIAAAgggEMQCBLxB3HiVLXqaNYZX07x58yq7C9shgAACCCCAAAKuESDgdU1Teq5ISkYjiYiMkqysLM8b8Q4CCCCAAAIIIOBSAQJelzZs6WqFh4dLauNM2bhxY+nVPEcAAQQQQAABBEJCgIA3JJpZpGHLM2TP3r0hUluqiQACCCCAAAII/CxAwPuzhauf6Vy8OceOubqOVA4BBBBAAAEEEChPgIC3PBUXrsto3koKCwsZx+vCtqVKCCCAAAIIIFCxAAFvxT6ueVcDXk0zZsxwTZ2oCAIIIIAAAgggUBkBAt7KKLlgm7SmzSUsLEwWLVrkgtpQBQQQQAABBBBAoPICBLyVtwrqLSOjoiWlYWNZt25dUNeDwiOAAAIIIIAAAlUVIOCtqlgQb9/4jLayc9euIK4BRUcAAQQQQAABBKouQMBbdbOg3UPH8R47ykwNQduAFBwBBBBAAAEEqiVAwFsttuDcqWGLM6yZGgqYqSE4m49SI4AAAggggEA1BQh4qwkXjLs1atXGFHvatGnBWHzKjAACCCCAAAIIVEuAgLdabMG5U4MmzSQ8IoKZGoKz+Sg1AggggAACCFRTgIC3mnDBuFtkVJRo0LtmzZpgLD5lRgABBBBAAAEEqiVAwFsttuDdqUmb9rJ7z57grQAlRwABBBBAAAEEqihAwFtFsGDfvFHL1pKTkyMFBQXBXhXKjwACCCCAAAIIVEqAgLdSTO7ZSC9cKy4qkpkzZ7qnUtQEAQQQQAABBBCoQICAtwIcN77V+KeZGqZPn+7G6lEnBBBAAAEEEEDgFAEC3lNI3L0iOS1DasUnyOLFi91dUWqHAAIIIIAAAgj8JEDAG4Kngl64tnHjxhCsOVVGAAEEEEAAgVAUIOANwVZv0rq9HDp0OARrTpURQAABBBBAIBQFCHhDsNWbtG4nx4/n08sbgm1PlRFAAAEEEAhFAQLeEGz1xq3bmlpPmjQpBGtPlRFAAAEEEEAg1AQIeEOtxa36ZjRvZW4x/N1334Vg7akyAggggAACCISaAAFvqLW4Vd/IqGhp2KKVrFq1KgRrT5URQAABBBBAINQECHhDrcV/qm/T9p1kz959IVp7qo0AAggggAACoSRAwBtKrV2qrpltOkhebo7s3r271FqeIoAAAggggAAC7hMg4HVfm1aqRpltO5jtJkyYUKnt2QgBBBBAAAEEEAhWAQLeYG25Gpa7yRltJSw8XGbOnFnDnNgdAQQQQAABBBAIbAEC3sBuH5+VLiomVjKatZQVK1b47BhkjAACCCCAAAIIBIIAAW8gtIKfytCiYxfZvWePn47OYRFAAAEEEEAAAWcECHidcQ7IozTr0Mm6cC1Xtm3bFpDlo1AIIIAAAggggIA3BCK9kUmw5VG3bl3HihwbG+vYsap6oGbtO5pdvv76a/nDH/5Q1d39tn1ERIQUFxdLUVGR38rgpgOHW2O5NdWpU8dN1fJrXSIjI0UXPU9JNRfQv/mwsDBx8rO75qUO7ByioqLw9GITqSfnqBdBrayqeo4WFBRUWICQDHgPHDhQIYo338y1elADNTVs0dq6CUWU6C2Gb7nllkAt5inliouLEz2x8/PzT3mPFVUX0C9l0dHRcvDgQQK0qvOVu0diYqIcPXpUCgsLy32flVUTSEhIkPj4eHHys7tqJQy+rfXLA57ea7ekpCTzJRdT75lW9RyNiYmp8OAMaaiQx91vRlg9UDo92cqVK91dUWqHAAIIIIAAAiEtQMAb0s0v0uqsbrLfwR7vEOem+ggggAACCCDgBwECXj+gB9IhW3TqIoXW8IBZs2YFUrEoCwIIIIAAAggg4DUBAl6vUQZnRi2tqck0jR8/PjgrQKkRQAABBBBAAIHTCBDwngbI7W8npaRK3QbpMn/+fLdXlfohgAACCCCAQIgKEPCGaMOXrnabrj1ky5atpVfxHAEEEEAAAQQQcI0AAa9rmrL6FTnDCnhzco7Jxo0bq58JeyKAAAIIIIAAAgEqQMAboA3jZLHO6NzNHO6DDz5w8rAcCwEEEEAAAQQQcESAgNcR5sA+SHqzlhKfWEe+/fbbwC4opUMAAQQQQAABBKohQMBbDTQ37tLm7J6yfv0GN1aNOiGAAAIIIIBAiAsQ8Ib4CWBXv033c61boWbLtm3b7FU8IoAAAggggAACrhAg4HVFM9a8Em27nWMyeeutt2qeGTkggAACCCCAAAIBJEDAG0CN4c+iZDRvJQl16srUqVP9WQyOjQACCCCAAAIIeF2AgNfrpMGbYftzzpf1GxjHG7wtSMkRQAABBBBAoDwBAt7yVEJ0XbsevSQ3J0eysrJCVIBqI4AAAggggIAbBQh43diq1axTB6uHV9Prr79uHvkfAggggAACCCDgBgECXje0opfqUDc1TdKaNpcZM2Z4KUeyQQABBBBAAAEE/C9AwOv/NgioEnQ8r4/s2LlTioqKAqpcFAYBBBBAAAEEEKiuAAFvdeVcut+ZvS6UwoIC+fDDD11aQ6qFAAIIIIAAAqEmQMAbai1+mvq27tpdomNj5f333z/NlryNAAIIIIAAAggEhwABb3C0k2OljIyKljPPvVCWM1ODY+YcCAEEEEAAAQR8K0DA61vfoMy90wUXS86xY7J48eKgLD+FRgABBBBAAAEESgsQ8JbW4LkR0AvXwsLC5MUXX0QEAQQQQAABBBAIegEC3qBvQu9XIDG5nrTq3E1mzpzp/czJEQEEEEAAAQQQcFiAgNdh8GA5XLdLLpdDhw7J2rVrg6XIlBMBBBBAAAEEEChXgIC3XBZWdunT3yCMGTMGDAQQQAABBBBAIKgFCHiDuvl8V3i965oOa5g69RvfHYScEUAAAQQQQAABBwQIeB1ADtZDnHP5ADl8+JAsWrQoWKtAuRFAAAEEEEAAASHg5STwKHB238skPCJCnn76aY/b8AYCCCCAAAIIIBDoAgS8gd5CfixfQlId6XzBJTLnu+/8WAoOjQACCCCAAAII1EyAgLdmfq7fu9dVv5K83Fx5++23XV9XKogAAggggAAC7hQg4HVnu3qtVh17XSi16ybLSy+95LU8yQgBBBBAAAEEEHBSgIDXSe0gPJaO4b3gmsGycdMm2bZtWxDWgCIjgAACCCCAQKgLEPCG+hlQifpfcO0gKS4ulkcffbQSW7MJAggggAACCCAQWAIEvIHVHgFZmnrpDaVT7z7y9ZQpUlRUFJBlpFAIIIAAAggggIAnAQJeTzKsLyPQ78ahkp+XxxRlZVR4gQACCCCAAALBIEDAGwytFABlbNv9XGnY4gx57bXXAqA0FAEBBBBAAAEEEKi8AAFv5a1CfstfDPmdHDx4UN5///2QtwAAAQQQQAABBIJHgIA3eNrK7yXt0f9KSU7LkH/+859+LwsFQAABBBBAAAEEKitAwFtZKbaTiMhIueK238vu3bvlo48+QgQBBBBAAAEEEAgKAQLeoGimwCnkeb/8leisDY8+9ljgFIqSIIAAAggggAACFQgQ8FaAw1unCkRGRcm1vx8hB/bvl+eee+7UDViDAAIIIIAAAggEmAABb4A1SDAUp8cvrpLMNh1k9Ogxkp+fHwxFpowIIIAAAgggEMICBLwh3PjVrXpYWJgMvu9Ryc3NkTvvvLO62bAfAggggAACCCDgiAABryPM7jtIq7POlnMuHyATv/pKVq5c6b4KUiMEEEAAAQQQcI0AAa9rmtL5itzwxwclplYtufHXv3b+4BwRAQQQQAABBBCopAABbyWh2OxUgcTkFBk88jHZtXOnPPzww6duwBoEEEAAAQQQQCAABBwLePPy8mT27NkyZ84cOX78uMeq7927VyZNmiSrV68+ZRtdN3nyZNFtTk7Z2dkm/5PX89q3Ar2uvFY69u4jY19/XZYuXerbg5E7AggggAACCCBQDQFHAt6cnBy55ZZbZNq0afL222/LyJEjpbi4+JTiLl68WIYMGSJr1qwx23z66acl24wePVqefPJJ0W2GDh0qmzdvLnkvNzdXHn30URk3blzJOp44J3Dro/+WuIREueGGgdaFbLnOHZgjIYAAAggggAAClRBwJOD94IMPpEePHvLQQw/JCy+8IMeOHZN58+adUrwxY8bIqFGj5K677pJXXnlFXnvtNTPt1aZNm2TmzJlm3X333SeDBg2Sd955x+y/YcMGE0wfOXLklPxY4YyADm347b/+K4ePHJarr77amYNyFAQQQAABBBBAoJICkZXcrkabrVu3Tvr161eSR5cuXcyV/T179ixZV1BQIFu3bpWOHTuadQ0aNJC4uDjZtm2bbNy40awPDz8Rn+v+EyZMMNtp8Pzggw/Kvn375MsvvyzJr/STx6y7gh08eNCsat68udx2222l3/bp89jYWJ/mHyiZt+vRS66988/yyX+fNON5fXlTioiICPMLQVFRUaBUP6jLoZ6akpKSgroegVT4KOsGLfZ5GkjlCtayRFq3NdfpEOvUqROsVQi4cus5iqf3mkU9NUbB1LumVfHUOLKi5EjAu9O6qCkxMbGkHPpcg9vSaffu3RIfH28+1Oz1+g/wfuuOXjt27Cjzj7HurwGupg4dOpjHb7/91jyW979du3aZfPQ9zdP+B768bb29Tj+kQyVdPvRO2bL2B9Mz36ZNG7n77rt9UnX9UNEhMaFk6xPInzK1v0g6+Xfhy/oEQt56btqugVCeYC+D/bfOOeq9llRTPL3rqblh6l3TqngWFhZWeHBHAl4tcOmCaBRey5rOqnQ6eRt9T7fTHtKT3ytv/9J5nfxch1GUThpAO5V0/HIopaGP/0f279wu991/v/kC44shDtrzr+cAd3nzzpmlf2N169Y1XwrLG1vvnaOEVi76pfzo0aNlPvdCS8C7tU1ISDCfJ3ZHh3dzD83c9G/+wIEDoVl5H9RaO9P0lwjOUe/hVvUcjYmJqfDgjozhTUlJKelh1dJor21GRkaZgtWrV8/8A6GzOdjJ3q5+/fqn7J+enm5vxmMACURFx8g9z74maZnN5S6rh1dn1SAhgAACCCCAAAL+FHAk4O3du7dMnDjRXMGvU4rp1GSdO3c29dbXuug3I72wbfz48Wb9jBkzTK+TRvjdunWTrKws2bJli+nZ+/zzz6V79+7+dOPYFQjEJybJiFfelZT0RjLUGi+tbU9CAAEEEEAAAQT8JeBIwNu3b1/zc5TOrnDHHXfIddddJ82aNTN1Hjt2rJmqTF8MHz5cPv74Yxk8eLCZkeGBBx4w2+jPg8OGDTMXm/3mN78RnZFBtyEFrkBSvfpy36sfSv2GTeR2q+3sWTUCt8SUDAEEEEAAAQTcKhBmjdk7dUJcH9VWA1Udu6u9uRUlnVGhvCvz9IYVOuRBx3PVJDk5hve9996TP/3pTzL9ihuk04K58r+vptek6EG3b/bBA/LU8Jtk8+oV8rvf/tZMTVfTSjCGt6aCZfe3x/DqxaUOfhyULYTLXjGG17sNao/h1QuQSd4RqOr4SO8c1b25MIbX+21b1XNUx/AmJyd7LIgjPbz20WvXrn3aYFe3LS/Y1fU67UdNg13Nh+ScQEKdunK/1dPbsdeFZg5m7eUnIYAAAggggAACTgo4GvA6WTGOFTgCMVav/l1j/k/633S7TJ8+Xbp27Sram0hCAAEEEEAAAQScECDgdUKZY5g5SW/4w4My7IkxsseaQ7nnOeeUXKAIDwIIIIAAAggg4EsBAl5f6pL3KQI9L7ta/vruF5Kc1lB+97vfyZAhQ4Q7pp3CxAoEEEAAAQQQ8KIAAa8XMcmqcgINW5whf31vglxw7SCZNGmSuVveggULKrczWyGAAAIIIIAAAlUUIOCtIhibe0dAx/Xe9ODfzU0qCsPCZcCAAWZaOnp7veNLLggggAACCCDwswAB788WPPODQKfefeSJT7+R7pf+Uj777DNp1669fPPNN34oCYdEAAEEEEAAAbcKEPC6tWWDqF4JSXXMxWz3WrckDo+JFb25yDXXXCOHDx8OolpQVAQQQAABBBAIVAEC3kBtmRAsV0ert/cf46bJJTcOlXnz5knHjp3kmWeeCUEJqowAAggggAAC3hQg4PWmJnnVWCCmVpwM/NND5qK2hme0lX/961/SuUsX4aK2GtOSAQIIIIAAAiErQMAbsk0f2BVv0rqdPPTmpzLk0X9L9rEcudq6qO26666TAwcOBHbBKR0CCCCAAAIIBJwAAW/ANQkFsgXCwsKk91XXy7++mCEX33CTfDd3rpx1Vmd54IEH7E14RAABBBBAAAEETitAwHtaIjbwt0Bc7SQZPPJRefzDSdKy89ny4osvSrPmzeWjjz7yd9E4PgIIIIAAAggEgQABbxA0EkU8IdCwRSsZ8fK7cveY/5PYxDpyzz33SI8ePWThwoUQIYAAAggggAACHgUIeD3S8EagCpx1QV954n9TZdCIv8reAwflqquulssuu0x+/PHHQC0y5UIAAQQQQAABPwoQ8PoRn0NXXyAyKkouGTxEnvxyllx68zBZnrVCep13ngwaNEj27dtX/YzZEwEEEEAAAQRcJ0DA67omDa0K6fjeX91zv7mwrdeV18mMGTPkrM6dZciQIXLo0KHQwqC2CCCAAAIIIFCuAAFvuSysDDaB5AbpMuSv/5K/W7cpPrvvZTJp8mTpcOaZcvPNN8vevXuDrTqUFwEEEEAAAQS8KEDA60VMsvK/QIPMZnLHP56VUR9PNoHvlClTTI/vAGse37Vr1/q/gJQAAQQQQAABBBwXIOB1nJwDOiGQ0byVCXz//tk0M5fvwkXfy4UXXiTnWeN8v/jiCyeKwDEQQAABBBBAIEAECHgDpCEohm8EGjRpKjc//A95atJ3cuXtv5dd+/bLsGHDpHXrNjJy5Eju3OYbdnJFAAEEEEAgoAQIeAOqOSiMrwQSk1Pk6t/9UUZ/PV9uf2KMpDZrIW+//bac2bGjnH/++fLGG29IUVGRrw5PvggggAACCCDgRwECXj/ic2jnBSKjouWcy66WB9/4VEZ98rVccuOtsnPvPnO7Yr172+WXXy6ffPKJ8wXjiAgggAACCCDgMwECXp/RknGgC+g43xv+8KCMmbJQ/vj8G9Kt/5Wy8ofVctddd0lm06bSr18/ef755yU3NzfQq0L5EEAAAQQQQKACAQLeCnB4KzQEwiMipMO5F8htjz8lz01fIvc8+5r0uPQq2fDjZnniiSekZatW5hbGf/nLX7ibW2icEtQSAQQQQMBlAgS8LmtQqlMzAR3y0Kl3H7n1sSfl2WmL5f5XP5R+Nw6VPImQ119/Xc455xxp3aaNDBw4UD777LOaHYy9EUAAAQQQQMARgUhHjsJBEAhCAe35PaNLd7Nc/4e/yO6tm2XZzG9kqbXMnvOduavb3ffcI00aN5b+/fvLbbfdJunp6UFYU4qMAAIIIICAuwUIeN3dvtTOiwKpjZpI30G3mCXfGtf7w4I5smzWNFkyfYq8+OKL8uJLL0lSUpL06N5dbrrpJunTp48Xj05WCCCAAAIIIFBdAQLe6sqxX0gLRMfGSkdr6IMuv37gb7Jj47oTwe+MqTJlylSZbN3aODomRs6wxv9ec801JgCOi4sLaTMqjwACCCCAgL8ECHj9Jc9xXSWQ3qyl6NL/N7dLTvYRyfpupjX0Yarp/X388cdllHXxW1qDBtK3b1/53e9+J5mZma6qP5VBAAEEEEAgkAUIeAO5dShbUArUSqgt3S65zCx6M4sNyxfLEqvnd/E3k+TNN980S6I19KFnjx4ydOhQ6d27d1DWk0IjgAACCCAQLAIEvMHSUpQzKAXCw8OlZaeuZrnurpGyZ9sW0+u7+NvJJUMfYqzhEe3atpXrr79ebrzxRomM5M8yKBubQiOAAAIIBKwA/7IGbNNQMDcK1G/YWC4ZPMQsx44clqw502Xxt1+bmR8eeOABeejhh6VRw4byi1/8Qn77299KamqqGxmoEwIIIIAAAo4KEPA6ys3BEPhZIK52onS37u6mS2FBgaxdskCWfDtFvp82SV6yZnx4+eWXpW7dumbIgwa/nTp1+nlnniGAAAIIIIBApQUIeCtNxYYI+E4gwhrG0Obsc8wy8M8Py/YNa83QBw1+x40fL+PGjZO4+Hjp0rmzDBkyxPQA+6405IwAAggggIC7BAh43dWe1MYlAhnNW4kulw35nRzev1eWWhe9fT9tssz9bobMmjXLTHmm4351zO+gQYNExwqTEEAAAQQQQKB8Af6VLN+FtQgEjEBicor0vvoGueeZV+W56cvkzqdekq59L5Mf1q6TESNGSNNmzeTiiy+WV199VQqsoREkBBBAAAEEECgrQMBb1oNXCAS0QEytWtK1z6Vy+6jR8uy0xfKnF96SXr/8lWzevkMeti54a968uQl+X3vtNdEp0UgIIIAAAgggIELAy1mAQJAKREZFSftzesvND/1dnpm6SEa88q70HjDQBL8PPfSQ6fnt16+fvP/++0FaQ4qNAAIIIICAdwQIeL3jSC4I+FUgPCJC2nY7V37zl1Em+P3zS+/IuVdcK+s3bpI//vGP0rRpU7nqqqvkq6++8ms5OTgCCCCAAAL+EOCiNX+oc0wEfCigwW+7Hr3M8usH/iYr5s6UeV+Nl0VTv5Jbb71VYq1hET26d5d77rlHevbs6cOSkDUCCCCAAAKBIUDAGxjtQCkQ8ImADnvo1LuPWfIfzrVucTxF5k4cZ830ME2mT58uCQkJcsEFF8ioUaMkLS3NJ2UgUwQQQAABBPwtQMDr7xbg+Ag4JBBt3cK4e78rzKJ3eVv0zVcyb+J4+fLLL2XChAmSmJQkF1rB77333itt2rRxqFQcBgEEEEAAAd8LEPD63pgjIBBwAnqXt95XXW+Ww/v3ycIpX8r8SZ/L+M+txbrRhQa/5/fuLXfeeSd3eAu41qNACCCAAAJVFSDgraoY2yPgMoHE5HrS5/rfmOXQ3t1mrO/8yV+YXt8vvvjCGvZQW7p1O1uGDh0qffr0cVntqQ4CCCCAQCgIEPCGQitTRwQqKZCUkip9brjJLNrzu+Tbr2Xh1C9lxsyZMm3aNImJiZXWrc+Q6667Tn79619LrDVMgoQAAggggECgC4QVWynQC+nt8h09etTbWXrM7/XXX5ff//73Mv2KG6TTgrnyv6+me9yWNxAIVIGco9mSNWe6CYCXWLc5zsk+IjobRLp1oZve5W348OHSoUOHQC2+4+WKjo6W48ePSwh+vPrEOsq6+FKXY8eO+ST/UMw0JiZG8vLyQrHqPqmz/s3rLd5zc3N9kn8oZlrVc1RvtlS7dm2PVCHZw5udne0RxNtv5OfneztL8kPAcYFa8QnS7ZLLzVJUWCjrln0vy2Z+I0umT5E333zTLLXi4qRd27Zy5ZVXmt5fnQEiVJN+6GpwVmhZkWouEB8fL5GRkeLkZ3fNSx3YOeDp3fZJTEzkHPUuaZU99UtxRSkkA14nb7lKD09Fpx/vBaOA9uye0bmbWa67+z45sHunLJ893Vq+NXP+Llq0SB57/HGpl5wsZ599tgwcOFD0jm+hlPTvXj9nnPyscbOv/TmKp/da2T5HvZdjaOeknph69xzwtmdIBrzebRJyQyC0Beqmpsn5A24wi/b+blyxVLK+m2kNgfhWJk2ebO7upvMBN8zIkPPPP19uvvlmadeuXWijUXsEEEAAAUcFCHgd5eZgCLhbQHt/W3TsYpar7rjHjPX9YeFc0/ObZfUCv/XWW2bRu721bNFC+vfvbwLglJQUd8NQOwQQQAABvwoQ8PqVn4Mj4G6BWtaUZp0vvMQsWtP9u3aY4HflvNnmIrinnnpKnnr6aUm05gXu2PFMGTBggFx77bWiF4CQEEAAAQQQ8JYAAa+3JMkHAQROK5DcIL3khhc6Pmvr2h9MALxi7iyZO3+udcvjWTJi5Eipb/X49uzZUwYPHiy9rRtgkBBAAAEEEKiJAAFvTfTYFwEEqi0QFhYmjc9oa5ZLbxomBcfzZe2ShVYAPEtWzJkh46w7vo0bN06irN7ezCZNzE0vbrrpJmnevHm1j8mOCCCAAAKhKUDAG5rtTq0RCDiByKhoadvtXLNcd9dIyT50UFbNnyMr5840M0C88sorokucNUVVm9at5bLLLpMbb7xRkqzbIJMQQAABBBCoSICAtyId3kMAAb8JJCTVseb9vcwsWojdW348Mf5Xe4DnzZLvv/9ennji7ybg7dSpo/zyl79k/K/fWosDI4AAAoEtQMAb2O1D6RBA4CeB1MaZostFv/q1md/2x1VZstIKfFdaAfCs2XNk+vTpZvxvct260qVLF3MDjKuuuspMXg4iAggggEBoCxDwhnb7U3sEglJAb+HZrH1Hs1x+63A5np8n65Yukh8WfGeC4ClTpspkaw7ge+69V2KsMcDnnnuuGQKhAXCcdUc4EgIIIIBAaAkQ8IZWe1NbBFwpEBUdUzL+d8DwP0leTo6ssy6AW71onqxaMEe+tXp/p06dKn8eMUL0FqBntGolF110kfzqV7+Shg0butKESiGAAAII/CxAwPuzBc8QQMAlAjHWjS3an9PbLFql43m5siFrqaxZPF/Wfr9AllnB8IIFC+Tf//636E0w9C5wXbt2Nb3Affv2Fe1BJiGAAAIIuEeAgNc9bUlNEEDAg0BUTKy07trDLLpJUVGRbF+/xgyD0KEQaxcvkA8//NAserc47QXWO8HpUAi9GI5bIXuAZTUCCCAQJAIEvEHSUBQTAQS8J6A9uI1atTHLhdfdaDLOPnhANixfbPUEL5H1yxZLlvV84cKF8uyzz5q5gOslJ0vbtm3NjTB0LHB6err3CkROCCCAAAI+FSDg9SkvmSOAQLAIJNSpKx179zGLllnvBLd78ybZsGKpbDRB8PcyY+YsmTZtmjz++OMSExsrDVJT5cwzz5QLL7xQrrjiCuYEDpbGppwIIBByAgS8IdfkVBgBBCojoHeCa5DZzCznXHa12aXg+HHZtn61FQBbQbAVCGtP8JdffikTJkyQESNGmhkgMjLSpVu3buaiOB0PHGsFxiQEEEAAAf8KEPD62F97iUgIIOAOgcioKMls08Es9lAInRFi8+oVJgDeuGKZrF/6vbz33ntmCbOGTsRbd4ZrmplZclFc79693YFBLRBAAIEgEiDg9XFjaS8RCQEE3CugM0K0Outss9i1PHbkkBUAL5dNVi+wDonQIDgr6w154403RC+K09sh2xfFXX311dLaulUyCQEEEEDAdwIEvL6zJWcEEAhRgbjaSdK+53lmsQkO7tltxgKbXmDrgrhl1rhgnRrtmWeesS6Ki5H6KfXMeGAdBnHllVeamSLsfXlEAAH+VYIwAAAhwklEQVQEEKiZAAFvzfzYGwEEEKiUQJ36qdL5on5m0R10uNPOHzeY8cA6O4ROj/b111Nk0qRJMnLkSGsoRII0bZopvXr1kmuvvVY6dOhQqeOwEQIIIIDAqQIEvKeasAYBBBDwuYAOd0pv2sIs515xjTlefm6u/PhDlhkCYeYHXrJAsl5+WV62Fu0FTmuQKmeffbbotGjcIMPnTcQBEEDARQIEvC5qTKqCAALBLRBtzehw8njgPdu2mN5fvTmG3ir5008/NUtEZKTo3MBnnXWWGQKhwyCio6ODG4DSI4AAAj4SIOD1ESzZIoAAAt4QqN+wsehiT42mF8StXbJI1ljB7w+L5sqUKVNl8uTJcs+990qdOnWkU8eO5hbJAwYMMNOkeaMM5IEAAggEuwABb7C3IOVHAIGQEtAL4jpZN8jQRZNOi7Z+2fey5nsrAF7wXcnNMUbed5+ZDaJD+/bSv39/ue6667gxRkidKVQWAQRKCxDwltbgOQIIIBBkAjotWrsevcwivxM5np9n3SJ5iRn+sHrhXJk3f4HMmjVLHnnkEUlIqC0tW7Ywd4bTALhZs2ZBVluKiwACCFRPgIC3em7shQACCASkgF7c1rprD7PIsLtF7w6nF8Kt+X6+WX6wHhcvXiyjR482t0fOSE+XLl26yKWXXiqXXHIJ44ADslUpFAII1FSAgLemgqfZnzutnQaItxFAwKcCene4Fmd2Nssvbr7DTIe2Y+M6WWeNA9aZINYsni+ffPKJWcKtO8MlJCSYnl+dDUKHQpx33nk+LR+ZI4AAAk4IEPD6WJk7rfkYmOwRQKBKAvqZlNG8lVnOv2ag2VcvhNuQZd0VzhoKscG6IcY6a0zw0qVL5dVXXzV3hkusXVsyf7o9cp8+feT888+XSGuWCBICCCAQLAJ8YgVLS1FOBBBAwEcCeiFch3PON4t9iP27dsimlcusZbn8uCpL1lmBsAbBr732mmhPcLzVE9y4USPp1KmTXHTRRWZe4FhrWjUSAgggEIgCBLyB2CqUCQEEEPCzQHKDdNGly0X9S0pyYPcu2bx6hQmAf1y1XDauWCor33tP3rMW7TmOi4+XhhkZ0tGaGk2DYB0SERcXV7I/TxBAAAF/CRDw+lieMbw+BiZ7BBBwTKBuagPRxZ4STQ+cffDAiQDYujBOe4J1SMTHH39sFhMEx8VLRka6uUGGBsH9+vUjCHasxTgQAgjYAgS8toSPHhnD6yNYskUAgYAQSKhTV9qf09ssdoF0TPCPq1bIJqsXeNOKZSYI/uijj0QX/UysZfX66uwQHTp0kF69epme4JSUFHt3HhFAAAGvCxDwep2UDBFAAIHQFtAxwW27n2sWW6J0EKw9wRutnuDPxo2Tzz77TEaMGCExMbHWneKSpGnTpiYQ7tGjh/Tu3ZubZdiAPCKAQI0ECHhrxHf6nRnScHojtkAAAfcLlBcE5x47KlvX/CCb16yUrWt/sJ6vkiXLs2TevHlmhghVibBmg4izbq6RYY0NTktLM1OmtW7dWtq1ayftrbvIMUbY/ecONUTAGwIEvN5QJA8EEEAAgSoLxFrje1ue1dUspXc+uGe36FzBOzatl92bN8kua9n54wZZ/913Mn369NKbWgFxlHWzjCipZc0QoXMI161bV3R4RIMGDSTdGjbRuHFjadKkiTRv3lxSU1PL7MsLBBAIHQEC3tBpa2qKAAIIBIVAnfqpoosOizg5Hd6/Tw5YU6bt37ldDuzeKRocH9y7Ww5Zy8E9u2TTtu2StWKlFBYcP3lXM35Yb8QRHRVtDaGIltrW/MIaINevX98Ex3ZgrD3I3Hb5FD5WIBDUAgS8Qd18FB4BBBAILYHE5HqiS2bbDhVWPOdothw5sN9a9pX7eHjfXhMkb9iyVZYtXy5FhYVl8tOL6yI1MI6Olvj4OBMYaw+x9hrrOOMWLVqIBsb6qPMSkxBAILAFCHgDu30oHQIIIIBANQRqxSeILqmNmlRq7+xDB60AeI8c3rfnRI9xqZ5j7VHeZ/Umr984X47n5Z6Snx0Yx8bGSGJiYkmvsQ6r0LHH9rCKli1bchHeKXqsQMAZAQJeZ5w5CgIIIIBAAAskJNURXRq2aFVhKbXn+MTwiRPDKA5ZAfIhKzg2jxosWzfnWLtxkyxeskSKi4pOySvM6g3W2zJHWUMrtPdYL7rTIDkpKcmMPdbxx9qL3Mi6i50OsSBIPoWQFQhUS4CAt1ps7IQAAgggEIoCds9xWmbzCquvM/QcO3K4zJCK7AMH5MjB/eZmHXrDjuxDB+Tw/r2yxxp6sXHzFtFZK8Ta7+SkQbIGyLrExsSYi/Pq1Kkj9erVK7k4zw6QdeyxBswkBBAoK0DAW9aDVwgggAACCNRYQMcAxycmmeV0wbF9sCKrR/ioNbQiW4Ni61HHH5sgWcchWwGyGY9sXbSnPcynuzhPp3PTnuRo7Um2gmTtSdZZLHTRHmUNlmOtmS30uV68Zy/6frx1i2hddB9ddLta1tRw+sh4Zbu1eAw2AQLeYGsxyosAAggg4EoBDSZr1002S2UrqL3CpS/Oy7Z6i7XnWANkDZ6PHtYA+sTznfsOSM5PPcknX6RX2ePpdhrMn1isi/Ws5+HhP7/W51qPCF0iIkSDeH1tB9gaSGtwrUM4dIaM0r3UOtZZe6qjraEeJAS8LUDA623Rk/LjxhMngfASAQQQQMBrAjqXsS71GzauUp4Fx/OtIRTHJM8KmPNzc6wl17ogL0+O51uLdWFegTWtW8HxE4tO8VZYUGCWop8eCwtPvLbfO7FtvhTk55/IQ/Ox8syz8s49elSO5hyVfXv2St6PmyUv55jJy1OBIyIizQ1HdH5lT0M4NDjW4Rs6xlkDZxICpxMg4D2dUA3f12/BJAQQQAABBAJJQGeWSEjSpY5fiqUBt45xzjlyRI4eOSTHDh8ywziO6qPpkT7RS63DOA5bs2fo9HHLs7LKDZRPHuNs9yAnJyebCwH1Dn26aO+xPWOGDtUghZYAAW9otTe1RQABBBBAwO8CGnAnJqeYpSqFMfMrW+OYTSBsHq25lq0L/+xhHYd/CpB1jPOKlausXur8crM3wy5+GuccZd2tT29EouOUdciFjmPWoNmePUMDZ+1F1kXnYtYbleiiwzFIwSNAwBs8bUVJEUAAAQQQCGkBe5aM1MaZlXLIy8mxxjKfmBEj++CJMc1mbLMZ33zI9C7nWD3N2rOsy97Dh2Xz9h1mqIcO0zhdCrfGKetYZV30IsFI61HHIOsFfvZFfycH0Dp+WWfZsIPo0o/0PJ9OvPrvE/BW365SezKGt1JMbIQAAggggIDXBWKsXltdktMyqpy3jlvWHuXco0fMOGQdi5x7LNtad+LRfp2bbW1jjYXOtbbN0cfsI3LMWvZZy7Zde6wxyzpO+tQblngqkPY+6zCNsLDwUhcE2hcGnnjUfa1nJVkUy0/T2VkP+vzE7Hb6eGL9z4+6y0/b/rS3PfTSvhhRj296wMM1kA+XXKvsehMVeyYP7fEufTtuHUetiwb8gZwcK12eNRh+4cKF5srObt26mfkEy4PZu3evLFq0SJpat27U2zaWTqtXr5Yff/xRunTpYsbl2O9VtI+9DY8IIIAAAggggEBlBXRqN/uGJJXdx9N2GnDqxXp6kaAJlK0gOE8vGtR1Vi/0iQsHrUfrgkH74kG9AFB7me0LA/VCweLCIikqLjpxU5PSczZb1wvZM2eYYFlny7ACZn1eEkBrEK3/6bVFumiy8rDz05k79GLFQuuYx3+6+DDfKl+OFdDnWIH+/m07rGB/rSnziZ3L/t/ccdAaGpJgDQvRG6jccsstcuONN5bdyI+vHAl4c6zGvPXWW6V9+/ayfft2+eijj+Tpp58+gV6q8osXL5ZHHnlE+vXrJy+88ILBGjBggNli9OjRsmLFCmnVqpU8//zz8txzz5m70FS0T6ms/fbU/ubktwJwYAQQQAABBBDwq4DGAvaMGkkpfi1KjQ+ugbHOE63zRR+2xlHrzVP0boMH9+ySA9YtuPfv3C5rVi6X//znP6EX8H7wwQfSo0cPuffeew30HXfcIfPmzZOePXuWgR8zZoyMGjVKOnXqJNdff73cdtttcvnll5sgeebMmfLxxx+bbyrvv/++vPPOO/LAAw+Ip30CZR4/At4yTcwLBBBAAAEEEAhiAR23nJhczywZzcu/Ffe/bx8oB7ZsDKhaOtLDu27dOtNra9dchySsXLmyTMBbYI2V2bp1q3Ts2NFspuNFdPD2tm3bZOPGjWa9dstr0v0nTJhgdb173kfn57PTkiVLJN/qntekY1D0Kkunkg5k16T3WdefCH5YONepQ3McBBBAAAEEEEDAcYGj1sV/mmrS+agdhlXZ344RPVXWkYB3586dZnoPuxA61YcGt6XT7t27zXQgpXtE9UrG/fv3y44dO8xdWeztdf99+/ZJRfuUDnjvuusus63u37VrV3n33XftrHz+2LBhQ3OMrO+mSweJEP3WQ0IAAQQQQAABBNwsoHFYTadu09tiVzYds8ZEV5QcCXi1l7PQGvNhJ+2Z1fnuSqeTt9H3dDud2uPk9+z9T15fep/Seb/xxhsmL12nx92zZ0/pt336/OKLL5apU6dKylPPSPLc+TL2qX/49HgHrWlXdLoTNyf9xqe3q9TzwG3pqHX1b1RUVJW+1dbUQI+nv6Yctr6R21fy1jTPUN1f/dRRf6HSX5X0PCXVXED/0dO/+yPWTQp8nbT9dBqp0/UW+bocvs5f/+ZPFyD4ugy+yF//XdBZBbQNnUwaW+g5o5/hpBMC3bt3r1G8pZ2b+vdY2aSzRFQ0rZsjAa9erac9tXbS53q3k9JJvwXoiaKzOdgRvW6XkZEh2kO8bNmyks11fXp6uvnm4Gmfko2tJ82bNy/90vQYl1nhwxfaY92nTx/J/vRziVqyTPr37+/Do4VG1npC64eaPUwlNGrtu1rql0qdB1L/zgh4veOsH9T62VT6i753cg7NXDR40RsC7Nq1KzQBfFBr/Zs/cOCAD3IOzSz1F2kNuPTXZ9LPAjXpmNJ/j6qyv3aCVpRODIqtaAsvvNe7d2+ZOHGi+dalU4jNmTNHOnfubHLW17roiaIXto0fP96snzFjRsmkzDqNWZZ1S8EtW7aYyn/++eei3xwq2scLxSYLBBBAAAEEEEAAARcIONLD27dvX5k9e7YMGjTIdPkPHDhQ7DG2Y8eONT/h6gwOw4cPl5EjR8qnn35qttMpyjRpb8mwYcPMrA064XFmZqYMHjzYvOdpH/Mm/0MAAQQQQAABBBAIeYEwq8u47C03fEii4690nIv2zFaUPI1DPW5NhqxDHsobm+Npn/KOoxfBOZV0fKQO6ci+6w8S9b/PZP/COU4d2rXHYUiDd5uWIQ3e9dTcGNLgXVOGNHjXU3NjSIN3TRnS4F3P6pyjOhxWO0U9pYojT097VXO9TglWmeTpoisNHnUpL3nap7xtWYcAAggggAACCCAQOgKOjOENHU5qigACCCCAAAIIIBBoAgS8gdYilAcBBBBAAAEEEEDAqwIEvF7lJDMEEEAAAQQQQACBQBMg4A20FqE8CCCAAAIIIIAAAl4VIOD1KieZIYAAAggggAACCASaAAFvoLUI5UEAAQQQQAABBBDwqgABr1c5yQwBBBBAAAEEEEAg0AQIeAOtRSgPAggggAACCCCAgFcFCHi9yklmCCCAAAIIIIAAAoEmQMAbaC1CeRBAAAEEEEAAAQS8KuDorYW9WvIaZKb3EHcq5eXlyerVqyWjfTsJO5Jt7l/u1LHdepzw8HApLi42i1vr6GS9jh49as7Rxo0bi9qSai4QEREhupC8I3Do0CHZtWuXNGrUyDsZkos5P538t9Dt5Pv27ZPjx49LWlqa26vqWP30M7Qq5+jp/v0KswKHYsdKH4IHWrx4sQwcOFD+97//Sfv27UNQgCoHusD48eNlxIgRsmDBAklMTAz04lK+EBR4/vnn5fXXXzfnaAhWnyoHgcB9990n69atk08++SQIShuaRaQ7JzTbnVojgAACCCCAAAIhI0DAGzJNTUURQAABBBBAAIHQFIh41EqhWXVnaq1jSvRn4u7du0t8fLwzB+UoCFRBICwsTFJTU805yrjTKsCxqWMC+jnapEkT6dy5s2PH5EAIVEVAPztbtmwp7dq1q8pubOugAGN4HcTmUAgggAACCCCAAALOCzCkwXlzjogAAggggAACCCDgoEBITkvmoK+Z7unHH3+ULl26SEpKipOH5lgIGIHs7GzJysoqo9GzZ8+S1zptXnnn6N69e2XRokXStGlTad26dcn2PEHAmwI//PCDxMbGmvPMzlenc1y4cKHocJtu3bpJVFSU/ZbHz9SK9inZmScIVEPg5HOUz9RqIAbALozh9WEjjB49Wj777DPJz88XnVanV69ekpSU5MMjkjUCpwrMnj1bxowZI7t37xb94NblkksuMRt6Okd1Or17771X6tSpIy+//LIJSNq2bXtq5qxBoAYCGzduNOdZ8+bNpVWrViannJwcGTJkiOj80HruTps2Tfr372+CX0/na0X71KB47IqAlHeO8pkapCeGzsNL8r6A9UdSPGDAgOLCwkKT+XvvvVf897//3fsHIkcETiNgBazF1hymp2xV0Tl60003FS9ZssTss3PnzuIrrrii2OpBOyUPViBQXQGrM6D4qquuKh40aFDxl19+WZLN2LFji63AtuT1sGHDir/77rviis5XT/uUZMITBKoh4Okc5TO1GpgBsAtjeH30RWXDhg3SsWPHkjtX6ZCGlStX+uhoZIuAZ4G1a9dKQkKCvPvuu2bifutzx2zs6RwtKCiQrVu3mvNXN2zQoIHExcXJtm3bPB+EdxCookCtWrXk1VdfNT27OnTBTjp5v35e2sn+7PR0vup2nvax8+ARgeoIeDpH+Uytjqb/9yHg9VEb7Nixo8zwBZ2aTG89SELAaQH9cJ47d67ExMTIm2++ae6qpmXwdI7q0AedQq90EKJDcfbv3+900TmeiwX69esn9erVMzW0v4TpC+sXhTJ3/LM/Oz2drxXtYzLnfwhUU8DTOcpnajVB/bwbF635qAF0Tj5rOENJ7tprpt8WSQg4LfDaa6+Zsbg6l+mVV14p1s/IpgfX0zl68notr56/emERCQFfC5x8/tmfnZ7Wa3kqes/X5SX/0BPgMzU425weXh+1W/369cv0iGnvWHp6uo+ORrYIlC+gF0zqDAwa7GqKjo42QxS0F83TOaq9bnrBkF71bic9fzMyMuyXPCLgMwGdzab0rwn2uefpfNWCeNrHZ4Uk45AV4DM1eJuegNdHbadT6ehUUFu2bDG9Y59//rm5k5WPDke2CJQroNM5Pf3002ZIg26watUqM7TmrLPOMtM9lXeORkZGSo8ePWT8+PEmzxkzZkjdunXNUu5BWImAFwV69+4tEydOlNzcXNGp8ebMmWPusFbRZ6qnfbxYLLJCwAjwmRq8JwJDGnzUdjruzLq6WG677TZJTk6WzMxMGTx4sI+ORrYIlC+g43B1erFXXnnFLHv27JH7779fNKit6BwdPny4jBw5Uj799FPTO/zII4+UfwDWIuBlgb59+5rpyKzZG8y5N3DgQGnWrJk5iqfP1Ir28XLxyC7EBfhMDd4TgFsL+7jtjh8/bn4a1qvkSQj4U+DQoUMmyC19MZqWp6Jz9ODBg2b8rz/LzbFDU+DIkSPmugf9clY6VXS+etqn9P48R8BbAnymekvSmXwIeJ1x5igIIIAAAggggAACfhJgDK+f4DksAggggAACCCCAgDMCBLzOOHMUBBBAAAEEEEAAAT8JEPD6CZ7DIoAAAggggAACCDgjQMDrjDNHQQABBBBAAAEEEPCTAAGvn+A5LAIIIIAAAggggIAzAgS8zjhzFAQQQMDnAm+99ZZMmjTJ58exD3D48GH7qbz33nsyYcKEktfVeaJ3sdIbTpAQQAABbwsQ8HpblPwQQAABPwlowPvVV185cvTf//738uyzz5Yc6913361RwHvgwAHp2LGjbN68uSRPniCAAALeEiDg9ZYk+SCAAAIhJDBv3jyv1lZvcrJ69Wqv5klmCCCAgC1AwGtL8IgAAgEtMHfuXHniiSdk4cKFMnToULnsssvkqaeeksLCQlPunJwcuf3222X9+vUl9di6datZp8GU3qFL31+7dq25bbLuP2rUKLO//hR//fXXyz333CPLli0r2f90TzTfBx98UPr16yc33HCD/N///Z8UFxeX7LZhwwZza+dLL71UrrnmGnnyySdFf7bXVNXy2NsvX75ctHf1yiuvlDFjxkh2dnbJ8U5+okMOHnroIdHj/+Y3v5GpU6eW2eSNN96Qq6++2lg+8MADsn///jLve3qh7ps2bZJx48bJP/7xj5LNtO6vvvqq/PKXv5SbbrpJpk2bVvKePvn888/l1ltvlYsvvti8P3HiRPP+0aNHRY+vScv79ddfm+en+19F9dO877jjDtm7d29JNqNHjzZtriv++9//mh7pp59+Wi6//HK57777ZNWqVSXb8gQBBNwlQMDrrvakNgi4VkADVQ3wNHBr3Lix9OzZU/7yl7+YIFgrrYGkBpy7du0qMdi3b59Zd+zYMRPY6vv9+/eXiIgI6dGjh/z73/82weAjjzwi5557rqxbt84EgCUZnObJr3/9axPUDR48WLp16yYjRoyQf/7zn2avjRs3mp/oNYjUMrdq1Ur+9re/mYBON9BAvSrlsbe/5JJLRIN7DSo1aLv55pvLBNl2kTWI7Nq1q2jgd9VVV4neolcDOx32oOnNN9+UP/zhD9K7d28TrH/zzTfSt29fe/cKH9u0aSPx8fHSsGFDadeuXcm2b7/9trz//vvyi1/8QvTLgD6uXLnSvP/888+LOjVv3tyUWdtEv3QsWLDAlK1Tp05muzPPPFPS0tJK8vT05HT103ppgH/33XebLDTYHjlypPTp08e81rHOt9xyi3zwwQcycOBAycrKkosuuki2bdvm6ZCsRwCBYBawvpGTEEAAgYAXsAI07TottnpgS8o6bNiw4vPOO8+8tgIs8/7s2bNL3l+yZIlZZwUxxVaQaJ7ff//9Je8PGjTIrNuyZYtZt3PnTvO69DFKNi7nSd26dYtfeumlknesoKr4s88+M6+tQLPY6s0stgLVkveHDBlSbAVi5nVVy2NvbwW4JfktWrTIlNfqSTXrrGC4+N577zXP//73vxdbQWmxuthJ1zVo0KC4qKioWO10e32uyeoZL7Z6O4uti8bszSt8PPvss4utAL5kmyuuuKLYCoSLrS8eZp0V0BbXqlWr2ArqzWvrS0Xxyy+/XLK91WNdXKdOnWJrHLBZZ/WGm7pYwxpKtqnoyenqp/tavwoUW4F+sdXrXFy/fv1i6xeCkiy1vImJicVaDju1aNGiePjw4fZLHhFAwEUCkcEcrFN2BBAILQErgBLtAbRTkyZNZP78+fbLSj1agVrJdlaAIx06dJBGjRqZdSkpKebRCnzLHKdkh5OeaO+qFSCZXlPtrdSe1Pbt25utdBjBhRdeKNpzqj+Va0/nlClTTum9rGx5tIdYk/aa2qlz585iBXLy/fffm2PZ6/VRXbSntPSQA+291B5wHeqhvZqaV8uWLU1PqxUAyl133WV6W0vnU5Xn2qMcFRVldtG20t7f7du3m9ePPfaY7NixQ/73v//JDz/8IEuXLjU91dWdleF09dNfAbQXX4dI6BAYHUZhfdkpUx3t0daebzvp0BTrS4T9kkcEEHCRAEMaXNSYVAUBtwvExcWVqWJ4ePgpP+dbHRIl2+i415NTcnJymVW1a9cueR0WFlbyvDJPdEyojmNt3bq1mbFAg2c7qNKxwM2aNTPBll7gpTMQXHDBBadkW9XyZGZmluSh5bV6Scsdx6uzHqiXGtmLBoE6VlZf68/3Vg+4CXxnzZplhnZogKhDEaqbrB7TMrtq+ez20OEo6qEB+O7du+W6666T1NTUMttX5cXp6mfnpT6ayjsXmjZtat6z/6dtoUMlSAgg4D6Bn7/auq9u1AgBBEJIIDo62tS29EVcemGVr5KOQdXxn9ozqos1NMCM0dUL6x599FGz6FhX7dXVMcOaNLC0L7Krbrm0x1jHL2vSKbx0bHOXLl1OyU57bidPnmwu0tIAV5Ne0GcN+TC9wjqGVYN9La8uixcvlu7du5t5fPUCPG8mHXOsF4X95z//Mb3Imrc66NhmddNkf9mwA2SzsoL/na5+uuuKFSvMFxAd66xtomO27S8k+r5alk56sVx5lqW34TkCCASnAD28wdlulBoBBE4S0J/QdWiCzhJw5MgR0RkSdBYGXyU93gsvvGACKO0V1aBuz5495kKu2NhYM5xAL1jT9RrEaU/wxx9/LHl5eTUqkl5spj/n69CEhx9+2AxJOP/880/JU2co0KELOpRAe0N1OMGNN94oX3zxheiXAx1SoAGnBsxaPh3GUVBQIDrMozKpXr16ZqiG5nu6pMMGdHs9hga4+mVBLyZTC3tIg93TrUMKDh06dLoszQwMFdVPL2LU+upsFnfeeafpgdeLEzWwt5MajB071rSRPmqPtw5TISGAgPsECHjd16bUCIGQFXjxxRfNlfnWxWSiY2P//Oc/+8xCeyS151B7TXW2Ag3YtDf3k08+Mcf84x//KElJSWJdJCY6Nvi5554z05Lp9qXvUFbVAupsEBdeeKHo0AbtwdQp1U4eSqB56nbvvPOOWBfVSXp6umhvs5ZTy6FJA07t0T3nnHNMT6/26j7zzDPGzWxwmv/pdGYawGsep0s6rld7V3UGBx3GoGOLExISzHAKOwDVOui4Zw1SH3/88dNledr66dhdHT+ss0Nosi5QNLNUaP76JUSTOmrvtg570O31/LFncTAb8D8EEHCNQJj1zf7nAW+uqRYVQQCBUBXQHkS9OEuDO/unfF9b6DAKDaL0ArKTk06Npj2qpccKn7xNZV5rT6j2KuvP8L169TK9oOUdr7y81EO3tYd9lN7G9tLecXtYQen3K3quvajaK3zy2OqK9tFeWQ14S18sVnp77Z3X/OxhIKXf8/S8ovp52kd7fnVMs/bS6/4ZGRlVrr+nvFmPAAKBJ8AY3sBrE0qEAAI1ENAgVwOZmiYdY1pRf4Aexw6otbdSl/KS/pTv7aSBa2WDXT22Bv+eUnleGsRWlDQY1eBYy1FeEF3RvvaMGJ620S8GGoSfrgylA+aK6ufpOKXX13T/0nnxHAEEAlOAgDcw24VSIYCAnwV0ii17Sq3yijJgwACx5pUt7y2frNMA01MvrTcPqGNoS099Vl7eOt5Vb2Lhq6QXmOlQDE9JA+7KjB32tL+u12EMNe11ryh/3kMAgcASYEhDYLUHpUEAAQQQQACB/9++HRoBAMAgENt/a5Z4gcgCPS4YTAkQiAU8rcWgzhEgQIAAAQIECHwJGLxffUhDgAABAgQIECAQCxi8MahzBAgQIECAAAECXwIG71cf0hAgQIAAAQIECMQCBm8M6hwBAgQIECBAgMCXgMH71Yc0BAgQIECAAAECscAAxgXDElcKwCgAAAAASUVORK5CYII=" /></p>
<!-- rnb-plot-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
<div id="apply-a-filter-based-on-our-criteria" class="section level3">
<h3>Apply a filter based on our criteria</h3>
<p>Now that we’ve explored our data, let’s used the information we’ve learned to actually filter our data.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuc2NfZGF0YV9maWx0ZXJlZCA8LSBHZW5lTWF0cml4RmlsdGVyKHNjX2RhdGFfZmlsdGVyZWQsIFxuICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIG1pbl9jb3VudHMgPSAwLCBcbiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICBudW1fc2FtcGxlcyA9IDUsIFxuICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIG51bV9nZW5lcyA9IDIwMClcbmBgYCJ9 -->
<pre class="r"><code>sc_data_filtered &lt;- GeneMatrixFilter(sc_data_filtered, 
                                     min_counts = 0, 
                                     num_samples = 5, 
                                     num_genes = 200)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiRXJyb3IgaW4gYFsuZGF0YS5mcmFtZWAoZ2VuZSwgZ2VuZXMua2VlcCkgOiB1bmRlZmluZWQgY29sdW1ucyBzZWxlY3RlZFxuIn0= -->
<pre><code>Error in `[.data.frame`(gene, genes.keep) : undefined columns selected</code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Let’s see how our filter did in regards to getting rid of samples that are likely bad quality according to percent mapped reads data.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxucGVyY19tYXBfcmVhZHMgPC0gcmVhZHI6OnJlYWRfdHN2KGZpbGUucGF0aChcImRhdGFcIixcbiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgXCJkYXJtYW5pc19wZXJjZW50X21hcHBlZF9yZWFkcy50c3ZcIikpICU+JVxuICBkcGx5cjo6bXV0YXRlKFwia2VwdFwiID0gZ2VvX2FjY2Vzc2lvbiAlaW4lIGNvbG5hbWVzKHNjX2RhdGFfZmlsdGVyZWQpKSBcblxuYGBgIn0= -->
<pre class="r"><code>perc_map_reads &lt;- readr::read_tsv(file.path(&quot;data&quot;,
                                            &quot;darmanis_percent_mapped_reads.tsv&quot;)) %&gt;%
  dplyr::mutate(&quot;kept&quot; = geo_accession %in% colnames(sc_data_filtered)) 
</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiUGFyc2VkIHdpdGggY29sdW1uIHNwZWNpZmljYXRpb246XG5jb2xzKFxuICBnZW9fYWNjZXNzaW9uID0gXHUwMDFiWzMxbWNvbF9jaGFyYWN0ZXIoKVx1MDAxYlszOW0sXG4gIHBlcmNfbWFwcGVkX3JlYWRzID0gXHUwMDFiWzMybWNvbF9kb3VibGUoKVx1MDAxYlszOW1cbilcbiJ9 -->
<pre><code>Parsed with column specification:
cols(
  geo_accession = [31mcol_character()[39m,
  perc_mapped_reads = [32mcol_double()[39m
)</code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>The samples we filtered out do appear to include the lower percent mapped reads so this is decent support for our filtering methods.</p>
</div>
</div>
<div id="set-up-a-singlecellexperiment-object-from-count-data" class="section level2">
<h2>Set up a SingleCellExperiment object from count data</h2>
<p>For the rest of this module and some of the upcoming modules, we are going to use some single-cell RNA-seq R packages called <code>scater</code> and <code>scran</code> to do some processing for us. The functions in these packages require the data to be in their own special object type (not an uncommon thing for R packages to do) called <code>SingleCellExperiment</code>. So first, we are going to set up our data in this format.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBQcmVwIGEgbWF0cml4IHdpdGgganVzdCB0aGUgcm91bmRlZCBjb3VudHMgc28gd2UgY2FuIGNyZWF0ZSBhbiBzY2Ugb2JqZWN0XG5jb3VudHNfbWF0IDwtIHJvdW5kKGFzLm1hdHJpeChmaWx0ZXJlZF9zY19kYXRhKSlcbmBgYCJ9 -->
<pre class="r"><code># Prep a matrix with just the rounded counts so we can create an sce object
counts_mat &lt;- round(as.matrix(filtered_sc_data))</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiRXJyb3IgaW4gYXMubWF0cml4KGZpbHRlcmVkX3NjX2RhdGEpIDogXG4gIG9iamVjdCAnZmlsdGVyZWRfc2NfZGF0YScgbm90IGZvdW5kXG4ifQ== -->
<pre><code>Error in as.matrix(filtered_sc_data) : 
  object &#39;filtered_sc_data&#39; not found</code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Special object type are difficult to use at times if we don’t know what they look like. Based on things we’ve learned in the previous modules, use this space to explore the <code>SingleCellExperiment</code> object you just made.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBUaGUgU2luZ2xlQ2VsbEV4cGVyaW1lbnQgaXMgYSBzcGVjaWFsIHR5cGUgb2Ygb2JqZWN0IHVzZWQgYnkgc2NhdGVyIGFuZCBzY3JhblxuIyBSIHBhY2thZ2VzLiBcbiMgVXNlIHdoYXQgd2UndmUgbGVhcm5lZCBzbyBmYXIgdG8gaWRlbnRpZnkgaG93IG1hbnkgZ2VuZXMgYW5kIGNlbGxzIHBhc3NlZFxuIyB0aGUgZmlsdGVyaW5nIHN0ZXBcbmBgYCJ9 -->
<pre class="r"><code># The SingleCellExperiment is a special type of object used by scater and scran
# R packages. 
# Use what we&#39;ve learned so far to identify how many genes and cells passed
# the filtering step</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Let’s examine what our filtered counts data looks like with a gene mean density plot.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBMZXQncyBjYWxjdWxhdGUgdGhlIGdlbmUgbWVhbnMgYW5kIG1ha2UgYSBkZW5zaXR5IHBsb3RcbmdlbmVfbWVhbnMgPC0gYXBwbHkoY291bnRzKHNjZV9yYXcpLCAxLCBtZWFuKVxuIyBQbG90IHRoZSBtZWFuc1xucXBsb3QoZ2VuZV9tZWFucywgZ2VvbSA9IFwiZGVuc2l0eVwiLCB4bGFiID0gXCJjb3VudHNcIikgKyBcbiAgZ2VvbV9kZW5zaXR5KGZpbGwgPSBcImxpZ2h0Ymx1ZVwiKVxuYGBgIn0= -->
<pre class="r"><code># Let&#39;s calculate the gene means and make a density plot
gene_means &lt;- apply(counts(sce_raw), 1, mean)
# Plot the means
qplot(gene_means, geom = &quot;density&quot;, xlab = &quot;counts&quot;) + 
  geom_density(fill = &quot;lightblue&quot;)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-plot-begin eyJjb25kaXRpb25zIjpbXSwiaGVpZ2h0Ijo0MzIuNjMyOSwic2l6ZV9iZWhhdmlvciI6MCwid2lkdGgiOjcwMH0= -->
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAArwAAAGwCAYAAABLkLalAAAEGWlDQ1BrQ0dDb2xvclNwYWNlR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19zn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7d0LtFRXffjx332/gcs7RP4UjYm6FISYQMzC1hZJNCbVpY3galpQIpWaSuIKMWs19F8hmlKb0MTYJD7QBqtEs0iIlZK/ESWBiCaS2gSEUGMMyDNc7uVeuO/5z2/DmTvPc+ees8+ZOTPfnUXuee199vmcM2d+s2efPRWxeBISAggggAACCCCAAAIlKlBZosfFYSGAAAIIIIAAAgggYAQIeLkQEEAAAQQQQAABBEpagIC3pE8vB4cAAggggAACCCBAwMs1gAACCCCAAAIIIFDSAgS8JX16OTgEEEAAAQQQQACB6lIl6O/vl8HBwVI9PI4LAQQQQAABBBBA4LxAZWWlVFfnDmtzr4k44alTp6Svry+0o2hqapKWlhY5cuRIaPuMwo5aW1ulra0tClUNrY7jx48312Z7e3to+yz2HVVVVYm+hjo6Ooq9qqHVT2/cEyZMkNdff116e3tD22+x76i5uVl6enpCvb8Xu4m+9zQ0NMixY8eKvaqh1o/3n0zuiRMnypkzZ6SzszNzZcSX1NXVydixY3MeBV0actKwAgEEEEAAAQQQQKAUBAh4S+EscgwIIIAAAggggAACOQUIeHPSsAIBBBBAAAEEEECgFAQIeEvhLHIMCCCAAAIIIIAAAjkFCHhz0rACAQQQQAABBBBAoBQECHhL4SxyDAgggAACCCCAAAI5BQh4c9KwAgEEEEAAAQQQQKAUBAh4S+EscgwIIIAAAggggAACOQUIeHPSsAIBBBBAAAEEEECgFAQIeEvhLHIMCCCAAAIIIIAAAjkFCHhz0rACAQQQQAABBBBAoBQECHhL4SxyDAgggAACCCCAAAI5BQh4c9KwAgEEEEAAAQQQQKAUBAh4S+EscgwIIIAAAggggAACOQUIeHPSsAIBBBBAAAEEEECgFAQIeC2dxX/5l3+RqVOnWiqNYhBAAAEEEEAAAQRsCRDwWpJ85JFH5NChQ5ZKoxgEEEAAAQQQQAABWwIEvJYkKyoqTEm9vb2WSqQYBBBAAAEEEEAAARsCBLw2FONlVFVVmZJOnz5tqUSKQQABBBBAAAEEELAhQMBrQzFehtPC29nZaalEikEAAQQQQAABBBCwIRBawNvT0yM7duyQnTt3Sl9fX866nzhxQrZu3Sr79u3L2Oa3v/2tbN++XbSsYkuVlecou7q6iq1q1AcBBBBAAAEEEChrgVAC3rNnz8rixYtl27ZtsmHDBlm5cqXEYrEM+N27d8uSJUtk//79ZptNmzYltvnSl74k9913n7z44ovy8Y9/XP7whz8k1hXDBAFvMZwF6oAAAggggAACCGQKVGcusr9k48aNMmfOHFmxYoUpfNmyZbJr1y6ZO3duys7WrVsna9askZkzZ8r1118vS5culWuuuUbOnDkjP/vZz+Txxx+Xuro6qa6uls2bN8vf/M3fpOQv5IwT8NKloZBngX0jgAACCCCAAAKZAqEEvAcOHJAFCxYk9j579mzZs2dPSsDb398vBw8elBkzZpjtJk2aJI2NjWaoLx3ftqamRp577jl5+9vfblp5/+RP/iRRnk689NJL8swzzySWvfe975XJkycn5oOecB5aGxwclKampqB3F5ny1QWP1NOlH470QxsuQy5qoq9xTFJNdK6+vt7YDK0p76na2lrR60X/ks4J6GtHnyPh9ZN6RfD+k+qhc3qd6GunHK+VUALeI0eOyKhRoxLyOq3BbXI6duyYOQHOw1+6bvTo0XLy5EmZPn266eLw+c9/3pyoiy++WK677rrk7CbgffDBBxPLtJX4oosuSswHPaEBjKaBgQFpbm4OeneRKV/PJx6pp0tNnKA3dU15z6mL8zoqb4nUo29oaEhdUOZzep1ogEcaElATTdxrh0x0Sl0wyTTRgLcUX0Pd3d2pB5s2F0rAq5+yNBB0krbmpt/E07fRbXU7bd14+eWXZe3atbJ+/XqZMmWK3H///aJ9eu+44w6nSNMFQrtBOEkffjt69KgzG/hfp0/y4cOHQ91v4Afmcwetra3S1tbms5TSyj5+/Hjz4GZ7e3tpHZiPo3FaYjo6OnyUUlpZNfifMGGCef0wvvfQudUARh9cdnv4eWjr8phqaWkx76nacEQaEuD9Z8jCmZo4caLpJlqK3S+1y6v2DMiVQnloTd/gtaXWSTqtgWtyGjdunOgIB8kjMDjbvfDCC3LFFVeYFls9mI997GOmD3By/kJPa4udpuE+YRS6nuwfAQQQQAABBBAoN4FQAt558+bJli1bTDCoLa86NNmsWbOMtc7rP23N0Afb9GE0TTr8mH4603/6cNvevXtFR3vQlO2BN7OigP9zAl6njgWsCrtGAAEEEEAAAQQQSBIIpUvD/PnzzRi8ixYtMn0XFy5caPrlaj20m4L2JdERHJYvX54YjkwDyFWrVpmq6kNrOlrD5z73OdGv9rRJ/tOf/nTSYRR+koC38OeAGiCAAAIIIIAAAtkEQgl4tfV29erVoj+7q313kx9MufXWWxP1mjZtmugQZqdOnZIxY8YkluuEBsnalUFbUN36aKRkCnGGgDdEbHaFAAIIIIAAAgiMQCCUgNepj3aszyelB7tOHn3ishiDXa2fE/DSh9c5W/xFAAEEEEAAAQSKQyCUPrzFcajB1oKAN1hfSkcAAQQQQAABBLwKEPB6lUvL54yDmDzKRNomzCKAAAIIIIAAAggUQICA1xI6Aa8lSIpBAAEEEEAAAQQsCxDwWgalhdcyKMUhgAACCCCAAAI+BQh4fQKmZyfgTRdhHgEEEEAAAQQQKKwAAa9lf34C1DIoxSGAAAIIIIAAAj4FCHh9AqZnJ+BNF2EeAQQQQAABBBAorAABr2X/vr4+yyVSHAIIIIAAAggggIAfAQJeP3pZ8hLwZkFhEQIIIIAAAgggUEABAl7L+AS8lkEpDgEEEEAAAQQQ8ClAwOsTMD17f39/+iLmEUAAAQQQQAABBAooQMBrGZ+A1zIoxSGAAAIIIIAAAj4FCHh9AqZnHxgYSF/EPAIIIIAAAggggEABBQh4LeMPDAxaLpHiEEAAAQQQQAABBPwIEPD60cuSd3CQgDcLC4sQQAABBBBAAIGCCRDwWqYn4LUMSnEIIIAAAggggIBPAQJen4Dp2QditPCmmzCPAAIIIIAAAggUUoCA17J+jC4NlkUpDgEEEEAAAQQQ8CdAwOvPLyP34GAsYxkLEEAAAQQQQAABBAonQMBr2T5GlwbLohSHAAIIIIAAAgj4EyDg9eeXkZuH1jJIWIAAAggggAACCBRUgIDXMn8sRpcGy6QUhwACCCCAAAII+BIg4PXFl5mZh9YyTViCAAIIIIAAAggUUoCANwD97u7uAEqlSAQQQAABBBBAAAEvAgS8XtSGydPZ2TnMFqxGAAEEEEAAAQQQCEuAgDcA6dOnTwdQKkUigAACCCCAAAIIeBEg4PWiNkyejo6OYbZgNQIIIIAAAggggEBYAgS8AUifOnUqgFIpEgEEEEAAAQQQQMCLAAGvF7Vh8jAW7zBArEYAAQQQQAABBEIUIOANAJuxeANApUgEEEAAAQQQQMCjAAGvRzi3bLTwuumwDgEEEEAAAQQQCFeAgDcAbwLeAFApEgEEEEAAAQQQ8ChAwOsRzi0bAa+bDusQQAABBBBAAIFwBQh4A/CmD28AqBSJAAIIIIAAAgh4FCDg9Qjnlo0WXjcd1iGAAAIIIIAAAuEKVIe7u/D2Vl9fL83NzaHtsKqqKrEv3Xdra2tivpwnampqsEi7APRaqaiowCXJRT3UhdfNEIqaaNL7GN8aDblUV1dLbW0tJkMkoiaVlZW8fpJMdJL3nzSQ+KxeJw0NDcYmc220l/T397seQMkGvN3d3dLX1+d68DZXDgwMJIrTnxZua2tLzJfzhAYwWKReAePHjzfXZnt7e+qKMp7TYLepqUn4lcKhi0CDmAkTJkhnZ6f09vYOrSjzKf0A0NPTE+r9vdjJW1paTBDDvTb1TPH+k+qhcxMnTpSzZ8+a+0rm2mgvqaurcz0AujS48nhbSWuMNzdyIYAAAggggAACQQgQ8AagSh/eAFApEgEEEEAAAQQQ8ChAwOsRzi0bAa+bDusQQAABBBBAAIFwBQh4A/CmS0MAqBSJAAIIIIAAAgh4FCDg9Qjnlo0WXjcd1iGAAAIIIIAAAuEKEPAG4E0LbwCoFIkAAggggAACCHgUIOD1COeWLXmIMrftWIcAAggggAACCCAQvAABbwDGdGkIAJUiEUAAAQQQQAABjwIEvB7h3LLRpcFNh3UIIIAAAggggEC4AgS8AXjTwhsAKkUigAACCCCAAAIeBQh4PcK5ZSPgddNhHQIIIIAAAgggEK4AAW8A3nRpCACVIhFAAAEEEEAAAY8CBLwe4dyy0cLrpsM6BBBAAAEEEEAgXAEC3gC8aeENAJUiEUAAAQQQQAABjwIEvB7h3LIxDq+bDusQQAABBBBAAIFwBQh4A/CmhTcAVIpEAAEEEEAAAQQ8ChDweoRzy0YfXjcd1iGAAAIIIIAAAuEKEPAG4E0LbwCoFIkAAggggAACCHgUIOD1COeWjRZeNx3WIYAAAggggAAC4QoQ8AbgTQtvAKgUiQACCCCAAAIIeBQg4PUI55aNFl43HdYhgAACCCCAAALhChDwBuBNC28AqBSJAAIIIIAAAgh4FCDg9Qjnlo1xeN10WIcAAggggAACCIQrQMAbgDctvAGgUiQCCCCAAAIIIOBRgIDXI5xbNgJeNx3WIYAAAggggAAC4QoQ8AbgzUNrAaBSJAIIIIAAAggg4FGAgNcjnFs2Al43HdYhgAACCCCAAALhChDwBuBNl4YAUCkSAQQQQAABBBDwKEDA6xHOLRsBr5sO6xBAAAEEEEAAgXAFCHgD8CbgDQCVIhFAAAEEEEAAAY8CBLwe4dyyMQ6vmw7rEEAAAQQQQACBcAUIeAPwpoU3AFSKRAABBBBAAAEEPAoQ8HqEc8vGKA1uOqxDAAEEEEAAAQTCFSDgDcCbFt4AUCkSAQQQQAABBBDwKEDA6xHOLRstvG46rEMAAQQQQAABBMIVIOANwJsW3gBQKRIBBBBAAAEEEPAoQMDrEc4tGwGvmw7rEEAAAQQQQACBcAVCC3h7enpkx44dsnPnTunr68t5lCdOnJCtW7fKvn37MrZ56aWXzLq2traMdcW0gIC3mM4GdUEAAQQQQACBchcIJeA9e/asLF68WLZt2yYbNmyQlStXSragcPfu3bJkyRLZv3+/2WbTpk2J8/PAAw/IvffeK7///e9l6dKl8vzzzyfWFdsE4/AW2xmhPggggAACCCBQzgLVYRz8xo0bZc6cObJixQqzu2XLlsmuXbtk7ty5Kbtft26drFmzRmbOnCnXX3+9CWyvueYa0Vbfp556StavXy/Nzc1y6aWXyp49e8zflAKKZCZbMF8kVaMaCCCAAAIIIIBA2QmEEvAeOHBAFixYkMCdPXu2CViTA97+/n45ePCgzJgxw2w3adIkaWxslEOHDsmLL74o73jHO+T48ePyzDPPyJVXXilaRnJ6+umn5fvf/35i0Y033igXXXRRYj7oiaqqqsQuampqZMyYMYn5cp7AIvPs67VSUVHBNZJEox7V1dWYpJnorH7IZ+SXIRi9TmprazEZIjGvHe4pSSDnJ3n/yTSprKyU+vp6c81kro32Eo0j3VIoAe+RI0dk1KhRiXrotAa3yenYsWPS1NRkAgFn+ejRo+XkyZMm0P3d734nd911l1x88cXy0EMPmS4PyQGz9hHWbZ2kB64ntlCpkPsu1DFn26/ehLHIlMEl1UQ9MMk00SW4ZLpwT8k04TpJNdE5TDJNStlFz7dbCiXg1Rat5H6tGow2NDSk1Ct9G12p2+knEW3d0GD20UcfFd1OW4G1f29ywDt//nzRf07SbhDJAbCzPKi/ycenfZbD3HdQx2Sj3NbWVin2hwxtHOdIyhg/frx5cLO9vX0k2Up6W31d6wfejo6Okj7OkRyctmROmDBBTp8+Lb29vSPJWtLbaou3NnC4Pfxc0gBZDq6lpcW8p/K+k4rD+0+qh85NnDhRNEbp7OzMXBnxJXV1da5HEEoTqL7BJ78QdXrKlCkpFRs3bpx0dXWZG5mzwtlOb/qXXHKJCXZ1nbbyah/eYkvOpwv68BbbmaE+CCCAAAIIIFDOAqEEvPPmzZMtW7ZId3e3eQBNhyabNWuWcdeWWP2nrRn6YNvmzZvN8u3bt4t+OtN/V1xxhezdu1eOHj1q1ukDbNqnt+jS+eZ0At6iOzNUCAEEEEAAAQTKWCCULg3a1UDH4F20aJHpz7lw4UKZPn26YdeRF7RjuY7gsHz58sRwZNpHa9WqVWYbbYLXdfpPv87S4Hjt2rVFd9q0hTcWrxUBb9GdGiqEAAIIIIAAAmUsEErAqwHq6tWrTV807bur80669dZbnUmZNm2a6BBmp06dynha++qrr5b3ve99pt+JPsxWzIknqov57FA3BBBAAAEEECg3gaHIM4Qj1471+aRcQ3rpgy3FHuzq8dHCm89ZZhsEEEAAAQQQQCAcgVD68IZzKIXfCw+tFf4cUAMEEEAAAQQQQCBdgIA3XcTvfLwfL10a/CKSHwEEEEAAAQQQsCdAwGvP0pSkwx4T8FpGpTgEEEAAAQQQQMCHAAGvD7xsWSsqIM3mwjIEEEAAAQQQQKBQAkRntuXjTbw8tGYblfIQQAABBBBAAAHvAgS83u2y5jRj8cZ0NF4SAggggAACCCCAQDEIEPBaPws8tGadlAIRQAABBBBAAAEfAgS8PvCyZa2ojP/aGi282WhYhgACCCCAAAIIFESAgNcye4UQ8FompTgEEEAAAQQQQMCXAAGvL74smePj8NLCm8WFRQgggAACCCCAQIEECHgtw8fjXcbhtWxKcQgggAACCCCAgB8BAl4/elny6ji8tPBmgWERAggggAACCCBQIAECXtvwdGmwLUp5CCCAAAIIIICALwECXl98mZn1p4Vp4c10YQkCCCCAAAIIIFAoAQJe2/LxFt7BwUHbpVIeAggggAACCCCAgEcBAl6PcLmyVVbShzeXDcsRQAABBBBAAIFCCBDwBqBOl4YAUCkSAQQQQAABBBDwKEDA6xEuV7YKHlrLRcNyBBBAAAEEEECgIAIEvLbZCXhti1IeAggggAACCCDgS4CA1xdfZmbG4c00YQkCCCCAAAIIIFBIAQJey/rxBl6GJbNsSnEIIIAAAggggIAfAQJeP3pZ8tKHNwsKixBAAAEEEEAAgQIKEPDaxqcPr21RykMAAQQQQAABBHwJEPD64svMTB/eTBOWIIAAAggggAAChRQg4LWsT5cGy6AUhwACCCCAAAII+BQg4PUJmJ6dgDddhHkEEEAAAQQQQKCwAgS8tv3jfXgHBwdtl0p5CCCAAAIIIIAAAh4FCHg9wuXKVqnjkpEQQAABBBBAAAEEikaAgNf2qWCUBtuilIcAAggggAACCPgSIOD1xZeZmT68mSYsQQABBBBAAAEECilAwGtZn4DXMijFIYAAAggggAACPgUIeH0CpmdnHN50EeYRQAABBBBAAIHCChDw2vanD69tUcpDAAEEEEAAAQR8CRDw+uLLzEyXhkwTliCAAAIIIIAAAoUUIOC1rE/AaxmU4hBAAAEEEEAAAZ8C1T7zF232+vp6aW5uDq1+VVVVZl8VlZVSGf/X2toa2r6LeUc1NTVYpJ0gvVb0gxHXyBCMeqgLJqkmOqf3sVgsNrSizKeqq6ultrYWk6TrQE1430kCOT/J+0+miV4nDQ0Nojallvr7+10PqWQD3u7ubunr63M9eJsrBwYGTHHx923R6ba2NpvFR7YsDWCwSD1948ePN9dme3t76ooyntNgt6mpSTo6OspYIfXQNYiZMGGCdHZ2Sm9vb+rKMp7TDwA9PT2h3t+LnbulpcUEMdxrU88U7z+pHjo3ceJEOXv2rLmvZK6N9pK6ujrXA6BLgyuPh5U8tOYBjSwIIIAAAggggEBwAgS8lm0rpEIGBwctl0pxCCCAAAIIIIAAAl4FCHi9yuXIp314SQgggAACCCCAAALFI0B0ZvlcMEqDZVCKQwABBBBAAAEEfAoQ8PoETM9OwJsuwjwCCCCAAAIIIFBYAQJe2/48tGZblPIQQAABBBBAAAFfAgS8vvgyM+sYd4yZmenCEgQQQAABBBBAoFACBLy25XUgXhICCCCAAAIIIIBA0QgQ8Fo+FfThtQxKcQgggAACCCCAgE8BAl6fgOnZCXjTRZhHAAEEEEAAAQQKK0DAa9m/ooI+vJZJKQ4BBBBAAAEEEPAlQMDriy8zMy28mSYsQQABBBBAAAEECilAwGtZXwNeEgIIIIAAAggggEDxCBDwWj4XtPBaBqU4BBBAAAEEEEDApwABr0/A9OwVjMObTsI8AggggAACCCBQUAECXsv8dGmwDEpxCCCAAAIIIICATwECXp+AGdn5aeEMEhYggAACCCCAAAKFFCDgtaxPH17LoBSHAAIIIIAAAgj4FCDg9QmYnp0+vOkizCOAAAIIIIAAAoUVIOC17E8fXsugFIcAAggggAACCPgUIOD1CZienS4N6SLMI4AAAggggAAChRUg4LXuXyGxWMx6qRSIAAIIIIAAAggg4E2AgNebW85clfFxeEkIIIAAAggggAACxSNAdGb5XJguDZbLpDgEEEAAAQQQQAAB7wIEvN7tsueMj8Mb79OQfR1LEUAAAQQQQAABBEIXIOC1TH7uoTXLhVIcAggggAACCCCAgGcBAl7PdNkzVlQoKS282XVYigACCCCAAAIIhC9AwGvZnD68lkEpDgEEEEAAAQQQ8ClAwOsTMCO76cObsZQFCCCAAAIIIIAAAgUSIOC1DH8u3qVLg2VWikMAAQQQQAABBDwLEPB6psuesULH4SXezY7DUgQQQAABBBBAoAACBLyW0SskPiwZEa9lVYpDAAEEEEAAAQS8CxDwerfLnjPep4EG3uw0LEUAAQQQQAABBAohQMBrWV1HaSDitYxKcQgggAACCCCAgA8BAl4feNmyah/eGBFvNhqWIYAAAggggAACBREg4LXMrj14SQgggAACCCCAAALFI0DAa/lc0KXBMijFIYAAAggggAACPgVCC3h7enpkx44dsnPnTunr68tZ7RMnTsjWrVtl3759Wbc5ffq0bNu2Leu6olhoHlrjsbWiOBdUAgEEEEAAAQQQiAuEEvCePXtWFi9ebALVDRs2yMqVKyUWywwKd+/eLUuWLJH9+/ebbTZt2pRxku6++2751re+lbG8WBaYcXiLpTLUAwEEEEAAAQQQQCCcgHfjxo0yZ84c+fu//3v56le/KmfOnJFdu3Zl8K9bt07WrFkjN910kzz00EPyzW9+U3p7exPbPfnkk/Lqq68m5otxwozDmxnLF2NVqRMCCCCAAAIIIFAWAtVhHOWBAwdkwYIFiV3Nnj1b9uzZI3Pnzk0s6+/vl4MHD8qMGTPMskmTJkljY6McOnRIpk+fLkePHpX/+I//kM985jNy3333JfI5E5s3b5Z7773XmZUvfelLMmvWrMR80BNVVVVmF6YPb/zJtQkTJgS9y0iUXxkftQKL1FOl14r+q62tTV1R5nN6rdTV1ZW5Qubhjx49OnNhGS/R60TfG7J9S1iuLGqi7z3ca1OvAN5/Uj10Tk2ampqkoaEhc2XEl2jXWbcUSsB75MgRGTVqVKIeOq3BbXI6duyYOQkmYDy/Qm/0J0+elGnTpsmdd94pK1asyPmGeOGFF8qf/umfJopsbm6W4Q4+sbGFicTNV/vwxrtrhLlvC9UPrAgNYLBI5a2vr5fBwcGUby9Styi/OX3d19TUYJJ06tWkurraPPOg1wvpnIBeJwMDA+Y1hMmQiV4r3GtTrwjef1I9dE4DXW1gdHuWKjNXNJbofcEthRLwamtWckUUO/3TRfo2WmndToOD7373u/KWt7xF3vnOd8revXuzHs+ll14q+s9J+vBbR0eHMxv4X+cNqaJSA14Jdd+BH5yPHbS2tmKR5qctu3qzCfP6TKtC0c3q619bHTAZOjUawGhLpnYBS+7aNbRFeU45jRml+Ibt9Yy2tLSYb414/aQK8v6T6qFzGlPpB6POzs7MlRFfMtw3hKEEvOPHjzcttY6lttpOnTrVmTV/x40bJ11dXeZEOJXW7aZMmSJf/OIX5fjx4/LYY48lWk///M//XB5//PGUMophxvTh5YcniuFUUAcEEEAAAQQQQMAIhDJKw7x582TLli3S3d0t2vKqQ5M5/Wt1Xv9pa4Y+2KZ9cTVt375d9NOZ/vvOd74j+sCa/tN+utqntxiDXa13cpcMnSchgAACCCCAAAIIFFYglBbe+fPnmzF4Fy1aZDpML1y40ASteujr1683ffe0f+7y5csTw5Fpx+pVq1YVVsfL3s/34fWSlTwIIIAAAggggAAC9gVCCXi19Xb16tWiPxqhfXd13km33nqrM2keTtMhzE6dOiVjxoxJLE+eeOtb3yrf/va3kxcV1bQG6iQEEEAAAQQQQACB4hEYijxDqJN2rM8n5Qp288lb8G3iLbwkBBBAAAEEEEAAgeIRoDnS8rkg3LUMSnEIIIAAAggggIBPAQJen4AZ2enDm0HCAgQQQAABBBBAoJACBLyW9enDaxmU4hBAAAEEEEAAAZ8CBLw+ATOy04c3g4QFCCCAAAIIIIBAIQUIeC3r6w9P6C+tkRBAAAEEEEAAAQSKQ4CA1/J5OPfDE0S8llkpDgEEEEAAAQQQ8CxAwOuZLnvGCsbhzQ7DUgQQQAABBBBAoEACBLy24ePjktGlwTYq5SGAAAIIIIAAAt4FCHi922XNqX14SQgggAACCCCAAALFI0DAa/lc0IfXMijFIYAAAggggAACPgUIeH0CpmenD2+6CPMIIIAAAggggEBhBQh4bfubX1qzXSjlIYAAAggggAACCHgVIOD1KpcjH10acsCwGAEEEEAAAQQQKJAAAa9leB5aswxKcQgggAACCCCAgE8BAl6fgOnZKyr5pbV0E+YRQAABBBBAAIFCChDwWtanS4NlUIpDAAEEEEAAAQR8ChDw+gTMyB5/aI2EAAIIIIAAAgggUDwCBLyWzwV9eC2DUhwCCCCAAAIIIOBTgIDXJ2B6djMObyx9KfMIIIAAAggggAAChRIg4LUsr314Y/H/SAgggAACCCCAAALFIUDAa/s80IXXtijlIYAAAggggAACvgQIeH3xZWY2fXhp4M2EYQkCCCCAAAIIIFAgAQJey/CmDy9dGiyrUhwCCCCAAAIIIOBdgIDXu13WnOf68GZdxUIEEEAAAQQQQACBAggQ8NpGZxxe26KUhwACCCCAAAII+BIg4PXFl5nZ/NJajE68mTIsQQABBBBAAAEECiNAwGvZ/dxPC1sulOIQQAABBBBAAAEEPAsQ8Hqmy56RgDe7C0sRQAABBBBAAIFCCRDw2pY3Pzxhu1DKQwABBBBAAAEEEPAqQMDrVS5HPvrw5oBhMQIIIIAAAgggUCABAl7L8BUVkFompTgEEEAAAQQQQMCXANGZL77MzPThzTRhCQIIIIAAAgggUEgBAl7L+gS8lkEpDgEEEEAAAQQQ8ClAwOsTMCP7+R+eGBwczFjFAgQQQAABBBBAAIHwBarD32U4e9SW1srK8OP55BbeQuw/HN3891Ko85B/DQu3JdfHkL1acK0MeeiUc33gkuqiHphkmiRfM6lry3eO6yT7uS9Xl5INeGtqaqShoSH72Q5gafKbkxbf1NQktbW1AewpWkVWV1dLS0tLtCodcG2rqqrMGzYuQ9B6A+ZaGfLQKTXR1NjYKHxjZCjM//TertcKJqkmer1wTxky0SnuKakeOqexSl1dXeIDdeYW0V0yMDDgWvmSDXh7e3ulq6vL9eBtrnRuvs6bVFtbm9TX19vcRSTLam1tlfb29kjWPahK6xt2X18fLknA+iFAPyR2dHQkLS3vSX2z1g/teh/T+xnpnEBzc7P09PSY1xAm5wQ00NVrhXtt6hXB+0+qh85psNvd3S2dnZ2ZKyO+RI/NLYX/nb9bbUph3flWmf7+/lI4Go4BAQQQQAABBBCIvAABr+VTWHl+HF6nxddy8RSHAAIIIIAAAgggMEIBAt4Rgg27+fkW3lgsNuymbIAAAggggAACCCAQvAABr2Xj8/EuD1RYdqU4BBBAAAEEEEDAqwABr1e5XPnOR7zDPS2YKzvLEUAAAQQQQAABBOwKEPDa9RT68FoGpTgEEEAAAQQQQMCnAAGvT8CM7OdbeHloLUOGBQgggAACCCCAQEEECHgtszvj8PLQmmVYikMAAQQQQAABBDwKEPB6hMuZ7XwLL+Pw5hRiBQIIIIAAAgggEKoAAa9l7kq6NFgWpTgEEEAAAQQQQMCfAAGvP7/M3AS8mSYsQQABBBBAAAEECihAwGsZnz68lkEpDgEEEEAAAQQQ8ClAwOsTMD27E/AyDm+6DPMIIIAAAggggEBhBAh4LbtXVJ4jZVgyy7AUhwACCCCAAAIIeBQg4PUIlztbhVnFsGS5hViDAAIIIIAAAgiEKUDAa1nb6dJAC69lWIpDAAEEEEAAAQQ8ChDweoTLle38IA3COLy5hFiOAAIIIIAAAgiEK0DAa9nb6cNLlwbLsBSHAAIIIIAAAgh4FCDg9QiXK1uFnOvDS5eGXEIsRwABBBBAAAEEwhUg4LXtzQ9P2BalPAQQQAABBBBAwJcAAa8vvszMPLSWacISBBBAAAEEEECgkAIEvJb1nT68dGmwDEtxCCCAAAIIIICARwECXo9wubKd68ErwkNruYRYjgACCCCAAAIIhCtAwGvbmz68tkUpDwEEEEAAAQQQ8CVAwOuLLzOz04d3YGAgcyVLEEAAAQQQQAABBEIXIOC1TE4fXsugFIcAAggggAACCPgUIOD1CZienXF400WYRwABBBBAAAEECitAwGvb/3wfXh5asw1LeQgggAACCCCAgDcBAl5vbjlz0Yc3Jw0rEEAAAQQQQACBgggQ8Fpmr6g8NzAZLbyWYSkOAQQQQAABBBDwKEDA6xEuVzanhZcfnsglxHIEEEAAAQQQQCBcAQJe69608FonpUAEEEAAAQQQQMCHAAGvD7xsWZ0WXsbhzabDMgQQQAABBBBAIHwBAl7L5ozDaxmU4hBAAAEEEEAAAZ8CBLw+AdOzOy289OFNl2EeAQQQQAABBBAojAABr2V354cnGKXBMizFIYAAAggggAACHgVCC3h7enpkx44dsnPnTunr68tZ3RMnTsjWrVtl3759GdscOHBAtm3bJmfPns1YVzQLzj2zJrTwFs0ZoSIIIIAAAgggUOYCoQS8GqAuXrzYBKsbNmyQlStXSrYW0N27d8uSJUtk//79ZptNmzYlTs8tt9wiDz74oAmEtaxf/OIXiXXFNEEf3mI6G9QFAQQQQAABBBAQqQ4DYePGjTJnzhxZsWKF2d2yZctk165dMnfu3JTdr1u3TtasWSMzZ86U66+/XpYuXSrXXHONCYCPHz8uDz/8sNn+kksuke9973ty+eWXp+QvhhmnD2+2gL4Y6kcdEEAAAQQQQACBchMIJeDVrggLFixI2M6ePVv27NmTEvD29/fLwYMHZcaMGWa7SZMmSWNjoxw6dEje9ra3yUMPPZTI397eLt3d3Yl5ndBuEtptwkk6LJgTfDrLwvib3Ie3EPsP4xhHug8csovhMuTiWDh/h9aU71SyRfJ0+YoMHbl6YDLk4Uxh4kgM/cVkyMKZKtfXTygB75EjR2TUqFGOtZnW4DY5HTt2TJqamlJuYqNHj5aTJ0/K9OnTpaGhwWyu22lL76233pqcXR555BH5whe+kFi2fv16efe7352YD3qiuvo8ZfxGrKm5uVkmT54c9G4jUT4OmaeppqbGfKDLXFPeS/QeQEoVGDduXOoC5hDIIcC9NhMGk0wTjU/0X6mlM2fOuB5SKAFvVVWVJP8Qg7bmOgGsU7v0bXS5bldfX+9sIq+88orcdtttpp9vencIDW7/6Z/+KbHtBRdcIKdOnUrMBz3hPKRWWXmuW3RnZ2eo+w/6+LyWr630w12EXsuOar6WlhZzbRf1w5ch4+rrpq6urrgfSC2AiTYUnD59OuX+GXI1im53ep3oe0Pye0rRVTLkCun7ZG1trXR0dIS85+LeHe8/medH7ym9vb0Z35Jnbhm9JcO15ocS8I4fP9601Dp82mo7depUZ9b81VaMrq4u0y1Bb2iadLspU6aY6b1798rtt98uN998s/zxH/+xWZb8P20F1n9O0tEewgwonIBXzo/SoN0rwty/c9zF9ldvxDiknhVtxdQ3a1yGXPQDr/7DZMjE+dZI35z0H+mcgF4nen91G+2n3Kz0WtFvjXj9pJ553n9SPXROG1z0tVOK14oTO2Ye9bkloYzSMG/ePNmyZYv5RKGBqA5NNmvWLFMDndd/+oLVB9s2b95slm/fvl1aW1vNP12vIzv8wz/8Q9ZgN9fBFWK58wmDh9YKoc8+EUAAAQQQQACBTIFQWnjnz59vxuBdtGiR6FeXCxcuTLTGal9b/WSqIzgsX748MRyZbrdq1SpTY+2fq90TPvvZzyaOYOzYsfLYY48l5otloqLi3GcIvm4rljNCPRBAAAEEEECg3AVCCXi19Xb16tWmL5r23XW+qlP85IfPpk2bJjqEmQa3Y8aMSZwbDYT1XxRSdbwflaZS/LogCv7UEQEEEEAAAQQQSBcIJeB1dqp9R/JJycFuPtsX0zbVNecC3vRh04qpjtQFAQQQQAABBBAoJ4FQ+vCWE2h1vHuGJgLecjrrHCsCCCCAAAIIFLMAAa/ls1NVTcBrmZTiEEAAAQQQQAABXwIEvL74MjPXnO/Dm/yrb5lbsQQBBBBAAAEEEEAgLAECXsvSTpcGAl7LsBSHAAIIIIAAAgh4FCDg9QiXK5vTpYGB4nMJsRwBBBBAAAEEEAhXgIDXsndl/FeA9McnaOG1DEtxCCCAAAIIIICARwECXo9wbtmq4uMO87OXbkKsQwABBBBAAAEEwhMg4A3AWrs10KUhAFiKRAABBBBAAAEEPAgQ8HpAGy6LtvAS8A6nxHoEEEAAAQQQQCAcAQLeAJx1pAYC3gBgKRIBBBBAAAEEEPAgQMDrAW24LPrzwvThHU6J9QgggAACCCCAQDgCBLwBOGvA29/fH0DJFIkAAggggAACCCAwUgEC3pGK5bF9dW0NAW8eTmyCAAIIIIAAAgiEIUDAG4AyXRoCQKVIBBBAAAEEEEDAowABr0c4t2x0aXDTYR0CCCCAAAIIIBCuAAFvAN7VtbUyMDAQQMkUiQACCCCAAAIIIDBSAQLekYrlsb0GvDy0lgcUmyCAAAIIIIAAAiEIEPAGgFwTH6WBFt4AYCkSAQQQQAABBBDwIEDA6wFtuCzawjs4ODjcZqxHAAEEEEAAAQQQCEGAgDcAZH1obXAwFkDJFIkAAggggAACCCAwUgEC3pGK5bF9VfynhQdjtPDmQcUmCCCAAAIIIIBA4AIEvAEQV1fHA166NAQgS5EIIIAAAggggMDIBQh4R242bI7qeAtvLEaXhmGh2AABBBBAAAEEEAhBgIA3AORzfXjp0hAALUUigAACCCCAAAIjFiDgHTHZ8Bm0Dy8tvMM7sQUCCCCAAAIIIBCGAAFvAMrapYE+vAHAUiQCCCCAAAIIIOBBgIDXA9pwWXQcXlp4h1NiPQIIIIAAAgggEI4AAW8AztqHl4A3AFiKRAABBBBAAAEEPAgQ8HpAGy6LDksWY1iy4ZhYjwACCCCAAAIIhCJAwBsAsz60pqm/vz+A0ikSAQQQQAABBBBAYCQCBLwj0cpzW+3SoKmrqyvPHGyGAAIIIIAAAgggEJQAAW8AsjpKg6YzZ84EUDpFIoAAAggggAACCIxEgIB3JFp5busEvLTw5gnGZggggAACCCCAQIACBLwB4Dp9eAl4A8ClSAQQQAABBBBAYIQCBLwjBMtn85rzfXjPnj2bz+ZsgwACCCCAAAIIIBCgQHWAZRe06KqqKqmsDC+er6ioSBxv1fmAt6+vT+rq6hLLy3FCz0G5G6Sfd71WcElVUQ99zXKtDLmoh6aa+DMByfeXoS3Kc6q6utr8kmWY9/dil3auFV4/qWeK+2yqh87pvURfQ6V4rQx3TyjZgNcJKjJPdzBLkt+QnD68PT095k08mD1Go1R1cW7G0ahx8LV0rk1chqz1RsW1MuShU87NW/9yrQzZcJ0MWThTvH4cidS/XCupHs5cubqUbMCrY+BqC2tYaTDphyacgLe9vb3sR2rQT5GMVpF6FTY2NpoxmnEZctGATt+0MRky0VaYlpYW0Q/Ovb29QyvKfEqvEzUJ8/5e7OT6+tHrhddP6pni/SfVQ+eam5vNa6cUr5XhWq3D+84/071klzjj8NKHt2RPMQeGAAIIIIAAAhESIOAN4GQ5ozR0d3cHUDpFIoAAAggggAACCIxEgIB3JFp5but0aaCFN08wNkMAAQQQQAABBAIUIOANANcJeGnhDQCXIhFAAAEEEEAAgREKEPCOECyfzatrzg1FRsCbjxbbIIAAAggggAACwQoQ8Abg67Tw6pPEJAQQQAABBBBAAIHCChDwBuDPQ2sBoFIkAggggAACCCDgUYCA1yOcW7b6xib9ORNpa2tz24x1CCCAAAIIIIAAAiEIEPAGgKy/YqJBLwFvALgUiQACCCCAAAIIjFCAgHeEYPlu3jRqtOgvrZEQQAABBBBAAAEECitAwBuQf0vrWOns7AyodIpFAAEEEEAAAQQQyFeAgDdfqRFu1zxmrHR1dY0wF5sjgAACCCCAAAII2BYg4LUter685tFjpLe3N6DSKRYBBBBAAAEEEEAgXwEC3nylRrhdUzzg7evvH2EuNkcAAQQQQAABBBCwLUDAa1v0fHka8PYT8AakS7EIIIAAAggggED+AgS8+VuNaMumUWNkIB7wDg4OjigfGyOAAAIIIIAAAgjYFSDgteuZKE1beDUdOnQosYwJBBBAAAEEEEAAgfAFCHgDMm+Oj8Or6bXXXgtoDxSLAAIIIIAAAgggkI8AAW8+Sh62oYXXAxpZEEAAAQQQQACBAAQIeANA1SKdgPfw4cMB7YFiEUAAAQQQQAABBPIRIODNR8nDNvrQmqZjx455yE0WBBBAAAEEEEAAAVsCBLy2JNPKaTrfh/fEiRNpa5hFAAEEEEAAAQQQCFOAgDcg7cqqKqlraJSTJ08GtAeKRQABBBBAAAEEEMhHgIA3HyWP24ydPIVhyTzakQ0BBBBAAAEEELAlQMBrSzJLORe+6c1y4vXXs6xhEQIIIIAAAggggEBYAgS8AUpPmvZGOXPmTIB7oGgEEEAAAQQQQACB4QQIeIcT8rF+cjzg1Z8X5scnfCCSFQEEEEAAAQQQ8ClAwOsT0C375GnTzepnnnnGbTPWIYAAAggggAACCAQoQMAbIK52adC0e/fuAPdC0QgggAACCCCAAAJuAgS8bjo+1zWPHiONLaNk3759PksiOwIIIIAAAggggIBXAQJer3J55rvwTRfLgf/93zy3ZjMEEEAAAQQQQAAB2wIEvLZF08p71/wPSFv8xydo5U2DYRYBBBBAAAEEEAhJgIA3YOjLr/qgVFRUyJe//OWA90TxCCCAAAIIIIAAAtkECHizqVhcNnr8RHnLZVfIj596Sjo7Oy2WTFEIIIAAAggggAAC+QgQ8Oaj5HObj950m/T29spHP/pRnyWRHQEEEEAAAQQQQGCkAgS8IxXzsP30t8+UD336Zvn1r38tH/nIR2RwcNBDKWRBAAEEEEAAAQQQ8CIQWsDb09MjO3bskJ07d0pfX1/Oup44cUK2bt2a9SEvffDrySefFN0maunapTfJtTfeJM8++6y8Y8YM2bNnT9QOgfoigAACCCCAAAKRFAgl4D179qwsXrxYtm3bJhs2bJCVK1dKLBbLANMfaFiyZIns37/fbLNp06bENvfcc4/88z//s/kRh09+8pPy+9//PrEuKhMfXv45uemer0lPX79cdfXVcuedd0al6tQTAQQQQAABBBCIrEB1GDXfuHGjzJkzR1asWGF2t2zZMtm1a5fMnTs3Zffr1q2TNWvWyMyZM+X666+XpUuXyjXXXCN/+MMf5Omnn5Yf/OAHUllZKd/73vfkO9/5jtx+++0p+aMwM+tP3ierf/D/5Ot33Cz333+/OY73vOc90tLSIqNHj5YbbrhBpk2bFoVDoY4IIIAAAggggEAkBEIJeA8cOCALFixIgMyePdt8pZ8c8Pb398vBgwdlRvzrfk2TJk2SxsZGOXTokLzyyitmuQa7mjT/f/7nf5pp539Hjx6VV1991ZmVCy64wORPLAh4Qlus9d9vnvt5XnvS7g0TLvw/8szm78vmzZsTeb761a/GhzGrlNq6WqmurpbK+JBmo0aNkubmZjkSP8aurjPxoL9C6uvqZMyYMdLa2mrW6b7VR7fTYdC0C4n2Fe7o6DBdQHSdBtT6T4Prmpoas63+1QfqtBVet2lqajLrEhXyOaHlu3Vh0eK1nnr+u7q6zN4aGhqkvr5eBgYG5MyZM3IyPo6xbqPHpnWsra01x+Dk1frrdrovzad/NWl+3bcem5rkOja10+30OtXp8ePHy9ixY02Zulz9zbmI71uT7k/rqts6lmZF0v9037pdd3e3WaplVFVVmWmtvx6PlqHl63EVMmldHGutrx67YzjSemlZetynTp0y3voaVntdpudBkx6vzuu1qS56XjSpsV4HThltbW1mWy3DsTMbuvxPPdvb2039dVqvd6/H4rKblFV6rHod5FvHlMzDzOj1rK9lvdbKKemINvo60X/pSZ31erWZtDzdp14vtpKeMz13em/S10DyPcDWPpxynPuTvq6KKem9W+8nYaXTp09LXfy90blu9LWv9wHSkID66H12JK+hcePGmet3qJTcU1quvrfptfjpT3861POv8YFbCiXgPXLkSAqWvvA1uE1Ox44dM298emNwkt589AVz+PDhlBuR5n/99dedzczfH//4x/KFL3whsWz9+vXy7ne/OzEf9IS+qAfjJ3rtjQt97yoWG5SeeODRc74kfRGnp974jVQDhih27Ug/FuYRQAABBBBAoLQE9IPY2rVrQzso/XDplkIJeNM/keunC23FS07p2+g63U5bONLXZcuvXR8uu+yyRJHaanT8+PHEfNATX/nKV2ThwoWirVJhJG1V0hZeTdqSoB8UtGVMW+h0Xj/h6jK94JKT2mlLm34K0231r26j2zstayP55JdcdrZpPX9OK2e29c4yPcf6yVPrrK0iWk/9tKaf0LX1TKedi1nXOWU6x+h8atVP8079nXV6bE5ri7PO2a/zV/evddUPWdpCqOVrPl2u07pPJznbavmOpbPO+Zu8b12mddc6aNIWS62HtihpuemvBbNRyP/TY1JrPWb9dJ7LabhqOcetx6TnUc+HHrdzjWl+9dT96Tbqots451rzaBm6Xl/Duq2u12szn6T70jz6AVTLyHV+8ikr3230PGpdtd42k2MUxjHYrLeNsvSY9XpMv39p2XqN6vXp9RrNVj+9bvS6t/1ti9Zf7yvOdWmzzsnHofc/3ZceQzElbazR+3dYSe8nen04102+7z9h1a8Y9qP3Kr2njuTbAG3hHen7lL5+58+fH2ocpudd319zpdRoKNdWPpfrV8TaUusknZ46daoza/4qqL5Y9Q1PX7yadLspU6aIthDrkF5O0uXaZSE5afDnBIC6XEdy0JMaVtKL6LrrrjN1DWufUdiPdrkI60NAFDy0jvp60GtTA2vSOQEnwNVvLUjnBPTmPWHCBPNt1kjenErdT4NS58NUqR9rvsenQaUGJPpNKWlIgPefIQtnauLEiaahQT+oB520oSLfxgobddH3Ebfk3uHBLecI1s2bN0+2bNliWl40ENWhyWbNmmVK0Hn9pzd3fbDN6c+6fft20z9VL1htuX3xxRfltddeMy1iTzzxhFx++eUjqAGbIoAAAggggAACCJSrQCgtvNqsrWPwLlq0yHx1qV/9T58+3ZhrX1tt+tYRHJYvX54Yjky/4ly1apXZRvvsfupTnzKjNuhXlTqKwcc//vFyPWccNwIIIIAAAggggMAIBCrifYtCe/xX+/Po1y5O/5pc9Uzun5q8jX4NrF9l5dPPqhBdGvRrJe1+QRoS4CulIQtnii4NjsTQX7o0DFk4U3RpcCRS/9KlIdVD5+jSkGmiS3j/yXQJs0tD5t6DXaLdYd1GBQmlhdc5xHw7ryf3xXXy6l9tCdZ/JAQQQAABBBBAAAEE8hUIpQ9vvpVhOwQQQAABBBBAAAEEbAsQ8NoWpTwEEEAAAQQQQACBohIg4C2q00FlEEAAAQQQQAABBGwLEPDaFqU8BBBAAAEEEEAAgaISIOAtqtNBZRBAAAEEEEAAAQRsCxDw2halPAQQQAABBBBAAIGiEiDgLarTQWUQQAABBBBAAAEEbAsQ8NoWpTwEEEAAAQQQQACBohIg4C2q00FlEEAAAQQQQAABBGwLEPDaFqU8BBBAAAEEEEAAgaISqIjFU1HVyFJlent7ZXBw0FJpwxfT0dEh7e3tMnXq1OE3LqMtqqqqZGBgoIyOePhDPXz4sAz3m9/Dl1J6W3CtpJ5TvYfptTJp0iSpr69PXVnGc5WVlaJvWyX61uXpzLa1tcmZM2fkwgsv9JS/VDNxT8k8s4cOHZKmpiYZM2ZM5sqIL9F7Q21tbc6jKNmAN+cRB7TiG9/4hqxbt07+53/+J6A9UGypCFx33XUyY8YMWbNmTakcEscRgMC+fftEr5UNGzbIZZddFsAeKLJUBNauXSs/+tGP5Kc//WmpHBLHEZDAlVdeKR/96Efl5ptvDmgPxVssXRqK99xQMwQQQAABBBBAAAELAgS8FhApAgEEEEAAAQQQQKB4Bar+bzwVb/WiUzPtO6L9p971rndFp9LUtCAC2q/sHe94h/zRH/1RQfbPTqMhoPeU5uZmufzyy6WlpSUalaaWBRGoqKgw95OZM2cWZP/sNDoC+v4ze/ZsecMb3hCdSluqKX14LUFSDAIIIIAAAggggEBxCtCloTjPC7VCAAEEEEAAAQQQsCRAlwYLkD09PbJr1y45ePCgTJ48WfQrA1J5CfT19ckvf/lLef31180wUvoVo5NOnDghTz/9tBmebfz48c5icbtucuVJZGYi8gLbtm0TvR50iDon6cgMv/rVr2T06NHS2NjoLBa36yFXnkRmJiIr8NJLL8nu3btl3Lhx0tDQkDgOL9eD2/0mUTATkRTo7+83Mcjp06dlwoQJKceQ6/7gdj24XV8phUdshoDX5wk7e/asLFmyRLq6umTHjh2ib2JXXXWVJAc8PndB9iIXOHLkiHziE58w51zfoB588EH54Ac/KNXV1ebNasWKFWbMQ12u46m+9a1vFbfrRt/gsuUpcgaqNwKB7du3yx133CHvf//7E+Nh3nPPPfLYY4+Jjr97//33iw4fpIGv2/WQK88IqsKmRSrwwAMPyOOPP27GTL377rvloosukilTpni6HtzuN0V6+FQrTwEdf/kv//IvRfv8a6OL3kP0vqIp1/3B7Xpwu9/kWaXi3Ux/eILkXWD9+vWx+EWVKOBTn/pU7Nlnn03MM1H6Avfee28sPg5z4kDjgUzsiSeeMPN/9Vd/FXvhhRfMdDwwjsUD4Vj8k3XM7brJlSexAyYiLXD8+PHYX//1X5tr4Xe/+505lldeeSX24Q9/OBb/kRYz/93vfjf2xS9+0Uznuh7c8kQaiMrH4j8OEIuPlRqLt9gZjeeffz728MMPe74e3O43cEdbYPPmzbFVq1YlDmLhwoWx3/zmNzG3+4Pb9ZDrfpPYQYQn6MPr87PIgQMHzBOPTjH69OOePXucWf6WgcCyZcvkhhtXL7/5AAAJ40lEQVRuSByp/uKefoLWr5m0m4v+yIQm/cUs/Zpaf+km13XjliexAyYiKxB/r5C77rpLPvOZz6T8etpvf/tbc51oK40m5z7idj3kyhNZHCqeEIgHuGYkl/iHI/mv//ovefOb32xa8bxeD7nuN4kdMhFZgQsuuEBefvllOXr0qHlfOXXqlLS2torb/SHX9eB2fUUWKKniBLxJGF4m9evsUaNGJbLqtPbjJJWPgP6UYU1NjTngn/zkJybI1a+Ujh07Zr6OTO7eol9Rnzx5UnJdN255yke0dI/0Bz/4gfn58fThC/UnhPXacJJzH3G7HnLlccrgb3QFNNCNt/6bD0faTSr+jYD8/Oc/d72nuF0Pue430RWi5o7ApZdeKm9729tk0aJFpntlvIVWJk6caH6WPNs9RfPluh7c7jfO/qL8tzrKlS+Guqf/Vrd+Qkp+uKAY6kgdwhGIf7VkfgZW+9vp+Knarzv+FXXKzvX60H68ua6b9OWa2cmTUhAzkROIf8UoW7ZskX/7t3/LqHv6eXfuI+nLNaNzPaSvc/JkFM6CyAkMDg6aD8aPPvqouVfot0SbNm2SW265ZcT3FD14rpXIXQJ5VzjenU70YTW9Pjo7O+W2224zY+y6nfNc69KXayWc+03eFSriDWnh9Xly9ClrbbFzkk7rgwWk8hKI96+TRx55RO677z6ZNm2aOXh9slqDXn0a1knO9ZHrunHL45TB32gKPPXUU/Lqq6/KtddeKwsWLDCtLEuXLpVf/OIX5snq9PuIflXpdj3o09jZ8kRTh1onC+i5veSSSxIj/lx88cWmq5zX6yHX/SZ5n0xHU0C7v3zoQx8y3xDpj1+9733vc72n6FHmuh7crq9o6qTWmoA31WPEc/PmzTOtNt3d3WbooJ07d8qsWbNGXA4Zoivwox/9SDSY0ZY77afrJB2lYc6cOaItv5r0yXztW6X/cl03bnmccvkbTQENbvU6efLJJ80/HcLw61//uvkltcsuu0xefPFFee2110yLSvyhR7Pc7XrIlSeaOtQ6WeCKK66QvXv3mn6ZulyvG/11Rq/XQ677TfI+mY6mgI7mokMZatLW2Oeee07mzp0rbveHXNeD2/UVTZ3UWvNLa6keI57TC+wf//EfzZuVPnASf0JS/uIv/mLE5ZAhugLxp6nNG1NyX92PfOQj8tnPfta06K1cudK01Oj1EX+aVrS1xu260VbAbHmiK0TNswnofeLLX/5y4huBH/7wh+YbgrFjx5pla9asMQGO2/WQK0+2/bEsWgL6sNrXvvY10z1KA5G1a9eaFn8v14Pb/SZaKtQ2XUAfktZhDPUBaf1GUQPd5cuXm2HKct0f3K4Ht+srfd9RmyfgtXTGtA+N9t3VGxMJgXQBfXJ2zJgx6YtN36tc102uPBmFsKBkBPQHTLQLjPYBT0+5rge3POllMB8tAX0GQPtlJj985ByBl+uB9ylHr/T+6rfMGn+kxyBu9we36yHX9RVlOQLeKJ896o4AAggggAACCCAwrAB9eIclYgMEEEAAAQQQQACBKAsQ8Eb57FF3BBBAAAEEEEAAgWEFCHiHJWIDBBBAAAEEEEAAgSgLEPBG+exRdwQQQAABBBBAAIFhBQh4hyViAwQQQAABBBBAAIEoCxDwRvnsUXcEEEDAp0BHR4fPEsiOAAIIFL8AAW/xnyNqiAACCAQioL8SePXVVwdSNoUigAACxSRAwFtMZ4O6IIAAAiEK6M8Z668zkRBAAIFSFyDgLfUzzPEhgEBRC5w4cULuuusuufbaa+Xzn/+8PP/884n66i9tPfDAA/LhD39YrrvuOrn77rtFfznJSevWrZNHHnnEmTV/77jjDnnqqafM9Fe+8hXZunWrfPOb35QPfehD5qfPf/zjH5t1P/vZz+TRRx81P0l64403Sltbm1n+7W9/22z7gQ98QG6//XY5efKkWc7/EEAAgSgLEPBG+exRdwQQiLSAtq6+//3vl4cffth0LdCfFb7yyivllVdeMcf1iU98wgTBb37zm+Vd73qXCYx1+1gsZtZrl4Rnn302xUADYG251aTB7rJly+Rb3/qW/Nmf/ZloAH3VVVfJSy+9JBMnTpQ3vOEN0tjYKJdffrnU1tbKv//7v8vNN98s8+bNk4997GPyk5/8RObPn59SPjMIIIBAFAWqo1hp6owAAgiUgsA3vvENOXr0qBw4cMAEnHpMvb29oq2w73znO00A+vjjj5vWXV2nwa4Gp7pMW2zzSU1NTfLTn/5UKisr5W//9m9NoKstwH/3d38nc+bMMfvWFl5NO3bsMIH1LbfcIhUVFSbw1X1pIF5XV5fP7tgGAQQQKEoBWniL8rRQKQQQKAeB3bt3m6BSW1eddP/994sGoLpOg0xtmXWStvJOnjxZfvnLXzqLhv2reTTY1aR/L7zwQuns7Myab+HChbJ9+3a56KKL5KabbpKXX37Z/CXYzcrFQgQQiJAAAW+EThZVRQCB0hLQ1l1tgc2WTp06JWPGjElZr62u2hVBuyY4yene4MxrC3FySi+/qqoqeXXK9Hvf+1554YUXTF/fZ555xnSz0FZgrQsJAQQQiLIAAW+Uzx51RwCBSAu88Y1vlH379qUcgz6YtmbNGtPKqgGxBqBOOnz4sPz617+WWbNmmUXa8prcWtvf328eQnO2H+6vBtDJSfv86kNqd955p2lh/tWvfmX2p8tJCCCAQJQFCHijfPaoOwIIRFrgk5/8pGhL6j333GOGB9PgdvXq1aYfrfbXnTZtmqxatcp0LTh48KDcdtttpoX3Pe95jznuiy++WH74wx+afrj6AxKf+9znTOtveqtvLqSxY8eKBtH79+8XDZb/+7//W2644QazPy3jyJEjZvmb3vSmXEWwHAEEEIiEAAFvJE4TlUQAgVIU0JZaHRlBW3THjRtnHk7Th8n0xyAaGhrkiSeekEOHDslb3vIW0+K7Z88eM+TYBRdcYDg0wNWgWEdxmDRpkhmyTEdVSG+5zWWngbN2cbjkkkvkueeeMw+y6UNxV1xxhbS0tJiRGv71X//VBOC5ymA5AgggEAWBivin+HPj20ShttQRAQQQKEEBvQ1rYKsPlGULVrWbgS5vbW3NevTa9UEDVB1izEty+gs7eQcHB019dNiybPVxtuMvAgggEBUBAt6onCnqiQACCCCAAAIIIOBJgC4NntjIhAACCCCAAAIIIBAVAQLeqJwp6okAAggggAACCCDgSYCA1xMbmRBAAAEEEEAAAQSiIkDAG5UzRT0RQAABBBBAAAEEPAkQ8HpiIxMCCCCAAAIIIIBAVAQIeKNypqgnAggggAACCCCAgCcBAl5PbGRCAAEEEEAAAQQQiIoAAW9UzhT1RAABBBBAAAEEEPAk8P8BZvOsnAEfZZIAAAAASUVORK5CYII=" /></p>
<!-- rnb-plot-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>The unnormalized, counts data has so many zeroes, which we should expect. For this dataset, 79.5% of the data points are zeroes, which is not out of the ordinary for single-cell RNA-seq data. This means our data consists of a few genes that are expressed a lot, and the rest of the genes are barely detected.</p>
</div>
<div id="normalization-of-count-data" class="section level2">
<h2>Normalization of count data</h2>
<p>In whatever data we are working with, we are always looking to maximize biological variance and minimize technical variance. This is where normalization methods usually come into the workflow. With this high amount of zeroes in our data, this makes normalization particularly tricky. So in order to normalize with less of these zeroes in the way, we normalize cells in groups with other cells like them; a method determine by <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0947-7">Lun et al 2016</a>.</p>
<p><em>Overall strategy for <code>scran::computeSumFactors</code> based normalization:</em> 1) Group cells with other like cells by clustering.<br />
2) Compute sum factors for each cell cluster grouping.<br />
3) Normalize using these pooled sum factors to create log counts per million reads (CPMs).</p>
<p>Because of differences in sequencing depth, it’s pretty common to try to correct for this by using total per million reads. You have probably seen data reported that was, in part, normalized based the total per million reads for a sample such as counts per million reads (CPMs), (or TPMs, FPKMs, and RPKMs, which you can <a href="https://haroldpimentel.wordpress.com/2014/05/08/what-the-fpkm-a-review-rna-seq-expression-units/">read about more here</a> on your own).</p>
<div id="normalization-step-1-estimate-cell-clusters" class="section level4">
<h4>Normalization step 1: Estimate cell clusters:</h4>
<p>Using a quick clustering, let’s determine groupiungs for our cells that we will normalize by.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBDYWxjdWxhdGUgY2x1c3RlcnMgKGNhbiBhZGp1c3QgbWluLnNpemUgZGVwZW5kaW5nIG9uIGNsdXN0ZXJzIHNpemVzIHlvdSBtYXkgXG4jIGV4cGVjdCBiYXNlZCBvbiBwcmlvciBrbm93bGVkZ2UpXG5xY2x1c3QgPC0gc2NyYW46OnF1aWNrQ2x1c3RlcihzY2VfcmF3KVxuIyBMb29rIGF0IHRoZSBncm91cCBzaXplcyBiZWZvcmUgd2UgY29udGludWU6XG5zdW1tYXJ5KHFjbHVzdClcbmBgYCJ9 -->
<pre class="r"><code># Calculate clusters (can adjust min.size depending on clusters sizes you may 
# expect based on prior knowledge)
qclust &lt;- scran::quickCluster(sce_raw)
# Look at the group sizes before we continue:
summary(qclust)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-output-begin eyJkYXRhIjoiICAxICAgMiAgIDMgICA0ICAgNSBcbjU4OSA0NDMgMjM4IDIxOCAyMTMgXG4ifQ== -->
<pre><code>  1   2   3   4   5 
589 443 238 218 213 </code></pre>
<!-- rnb-output-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
<div id="normalization-step-2-compute-sum-factors" class="section level4">
<h4>Normalization step 2: Compute sum factors:</h4>
<p>Compute factors for each sample using the clusters we just made.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBDYWxjdWxhdGUgdGhlIHNpemUgZmFjdG9ycyBiYXNlZCBvbiB0aGUgY2x1c3RlcnNcbnNjZV9yYXcgPC0gc2NyYW46OmNvbXB1dGVTdW1GYWN0b3JzKHNjZV9yYXcsIGNsdXN0ZXJzID0gcWNsdXN0KVxuYGBgIn0= -->
<pre class="r"><code># Calculate the size factors based on the clusters
sce_raw &lt;- scran::computeSumFactors(sce_raw, clusters = qclust)</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>The authors of <code>scran</code> recommend removing cells that have negative size factors, so we will do that here. Of course, if you have a lot of cells that turn up has negative size factors, you should probably reconsider your filter and/or whether something has affected the quality of your data.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-frame-begin {"metadata":{"classes":["tbl_df","tbl","data.frame"],"ncol":58,"nrow":1698},"rdf":"H4sIAAAAAAAAA+39z5MlyZHniUWjGj1odKO6etAzOxyOkMGVlSVaZJD93M3MzQ0iy+nI37OyQGNRmOUcKFISyAxUhUxkRHZEZAM15AqLB/4DXJIivPK/4JU8U4SX/Qf2hOPeyAO5Q38vM8PdVPXlq0zTzHqZ+Un8eOGq7qaqX/+a2k93/9Xdf9v/8N/+8ODg4HsHn3z/jw6+98n05x9/b/q/Pzr444M/Xf/+/uDgkx9Pf/xs+t9n099/eKH44YPbPz98dHJ2dtgpyaAk+YXkz+arohYZZ41aVJSo77So16JBi7TFXlvstcWw0iLtRNBOhKBFGomQlCjqs6IOKL0U/fmMV78yZJ0hC4YsGTLLRjZko5YFw5dg+HKD3FJm+HeD3VJm+BwMn4Phc7B8LloWjTiiEUc04ohGHNGIIxpxRCOOaMQRjTiS4XMyfE6Gz8nwORn+WfxLhn/J8s/AeTB8Hgz/BsO/wSgvG+Vlo7xslJcNn4uBQTEwKAYGRZfXr/T96Ffav36l/etXVnkag77TGPSdYbcz7HaG3U5zt+80Ln2ncek7jUvfGXF0RhxGXuuNvNb3RhxGrut7Iw4j1/VGruuNvNYbea038lVv5KveqPt9Mq416ltv1LfeqG+9Ud96o771gxGHUQd7ow72g+HzYPg8GP4Z9bc36m+fDf+MOt0bdbrPhn/ZwD4bPmcDZyNH9NmIYzTiGI04RiOO0YhjNHweDZ9Hw+fR8Hk0fC6Gz8XwuRj+GTmxN3Jib+TEsNJ2g5Enw0qXF4ycGIz8F4xcF4y8Foy8Foy8Fox8FYx8FYycE4z2PBjteTDa82C058Foz4PRngcjvwSjjQ9GzglGLglGnY5GOxMN7KOBczRyezTwi0a/M/bav2jk7Gj0CaPR14tGfo5G3ywa9zIa9yga2EcjZ0cjX0Ujb0Qjb0QjH0QjH0QjH0QjH0QjH0QjH8Ri+GLkiFiM2Iz+VTTyRjTyRjLyRjLyRjL6V8noX6WV9i+ttH9ppXFORm5KRm5KRn8tGfkqGfUoGfUoGf21ZNStZOSwZPTNktE3S0auS0auS0b9TUbfLBl9s2TU82SMTZPRX0tGfy0ZOSIZOSIZOSIZ/bpkjFeTMV5NRv8vGf2/ZIxXk5GbkjFeTUa+SsbYNBljzmTktWTksGS0KcloU5KR15LRpiSjz5qM/JeM/mky+qfJ6IumwbBr9EXTYPhitGXJ6J8mo3+ajP5pMvqnyeifJiPfJ6N/moz+aTL6osnoiyajL5qMNiUZbUoy+qJpNHw2+qfJaI+S0R4loz1KRnuUjPYoGe1RMtqjZLRHyWiPktEeJaNvm4w2KlltVNFxDEa7NRjt1mC0W4PRbg1GuzUY7dZgtFuD0W4NRns0GO3RYLRHg9H2DNHAwMgbg5E3BqMvOhi5ZDByyWD0WQcjvwxGfhmM8e9g5JfByBuDUacHo/4OxvhyMOr0YNTpwajTg1F/B6OuDkZdyEb/IBv9g2z0D7LRP8hG/yAbfYFs9AWy0RfIRl8gG218Ntr4bLTn2WjPs9F2Z6PtzsZYLRscz8b4LRtjg2y0tdloa7NRZ7JRZ7JRZ7JRZ7JRZ7JRZ7JRZ7JRZ7JRZ7LRTmejHmWjvcwG77PB+2y0KdloA7KRs7ORn0djPWA0+lKj0Zcajb7UaHBjNO7vaNyP0cB5NHAeDZxHY25uNLAfjT7SaPSRRiPXjUYfaTT6SKNxf0ejPzQauXM0+kOjkTtHI3eOBl9Go+8zGrlzNPo+o5FPR6PvMxp9n9Hg6Wj0fUaj7zMafZ/R6PuMBu9HI9+PRt9nNPo+o9H3GY16NBp9n9GoW8Xo+xSj71OMvk8x+j7F6PsUo+9TjL5PMcbnxRifF6M/VIz+UDH6Q8UYnxdj3rEY/aZitMnFaJOL0SYXo00uRptcjPa3GO1vMdrfYrS/xWh/i9GuFqNdLcaYuBhj4mLk52KsuRZjvq4YubgY7W8x2tVitKvFaFeL0V4WIz8XIz8XIz8XIxcXI+8WI+8WY2xajFxcjFxcjLxbjH5sMfJuMfJuMfJuMfJuMfJuMfJuMfJuMXJsMXJsMfJpMfJpMfJpMfJpMfJpMfJpMfJpMfJpMfJpucmnP7qRdaubhFoJe0sYLGGyhIMlzJZwNIQ3/e1KaLl0kxkqoeVSsFwKlkvBdMmCLlrQRcv5aDkfLTyjFVG0IopWRNGKKFoRRSuiZEWUrIiSFVGyIkpWRMmKKFkRJSuiZEWUrIgGK6LBimiwIhqsiAYrosGKaLAiGqyIBiuiwYooWxFlK6JsRZStiLIVUbYiylZE2YooW86PlvOj5edo+Vmsy4sVe7HKLGaZhp+dlRU7Kyt2VlbsVgaenZUqOytVdlaq7KxU2a0s5zvL+c5AqeusiDoros6KqLMi6qyIOsv5znK+t5zvLed7y/necr63nO8t53sLeSt/dlbd7C1AeivM3vKzt/zsLT/nbUeV0HC+7w3n+95yKRjI91Zb3FttcR+siKwGurca6N5qoHurge6tBrq3GujeaqB7q4HurQa6txro3mqge6uB7q0Gurca6N4iWG810L3VQPdWA91bbXFvtcW91cL2VgvbWy1sb7WwvZX8+2JZLxZ0xXApWOk3WOk3WOk3WOk3WOk3WOk3WOk3WOk3WOk3WOk3WOk3WJk2WIklWJk2dJbzVvoNVl4KVvoNVqYNVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVgYLVrIKVl4KVl4KVrYJVrYJVrYJVi89WJ3SYPU/g9X/DFb/M1j9z2B1NUO2IrJSULD6n8Hqfwar/xlGy6XRAmS0/BwtP0fLT6ujG6yObrA6usHq6AYr/QYr/YZiRWTm5GJFZPWoo9Wjjisjomjl+Wjl+Wjl+Wgl6mgl6mil32j1fqOVk6OVk6OVaaOVaaPVp41Wpo1WqoxWVoxWVoxWAoxWAoxWAoxWAoxWAoxWXopWLyhavYto9S6i1buI1vg9WuP3aI3fozV+j9b4PVpD9WgN1aM1HIhWVkxW9yBZrEsWwZLV6Cer0U9WRzdZ2TtZPcBk3aNk3aNk3Y5k3Y5k3Y5k3Y5k3Y5kTack63Yk63Yk83ZYMyfJarmS1XIlq+VKVsuVrJmTZDVnyWrOktWcJas5m7dmVUIrIquNm3dnVUIrIqvhS1bDl6yGL1kNX7IavmQ1fMlq+JLV8CWr4UtWw5eshi9ZDV+yGr5kNXyD1fANVsM3WG3cYLVxgzVsGaxhy2C1hoPVGg5WXhqsYctgtZuDlcEGq90crLQ2WGltsNLaYLWwg9XCDtZYZrCa3cEaywzWWGawuu6D1fANVt97sFrDwepmD9agfrBm3QdrpD9YI/3BmnUfrBZhsFqEwWqLByvPD1ZKH6yUPlgt7GCl9MFK6YOV0gcrpQ9WSh+s7D1Y2XuwcvJg5eTBysmDlZMHKycPVk4erJw8WDl5sHJytkibrfFmtsab2RpaZove2aJ3tqa8sjXezFZFyBYVs9U5yRbBssWlbDEkW3czW7djtJLVaCWr0UpWo5WsRitZjVayGq28NFpzLKM1cBitOZbRGk2M1mhitOZYRmuIMVpDjNEaYozWEGO0hhijNccyWnMso8X50aL3aGXv0Zr7Ha2kOlr5c7T6yaOVKkcrrY1WWitWC1usxrRYtClWG1csLhWLNsW678W678W678W678W678W678W678W6xcW6xcUaChXrvhfrvhcrARYr1xUr1xVrwq1YCbBYCbBYPYFi9QSK1RMoFmmL1RMoFpOLlX6LlX6Lxfli9RmKNWAsVu0o1oCxWL2LYo0Ni9W7KFaLUKxqWKxmolgdiWKNDYvVuyhW76JYw8BidTmK1UgVq8tRrJarWB2JYnUkitWRKNbgrliDu2IN7oo1uCvW4K5Yg7tijeOKNY4r1jiuWOO4Yo3jrI1SvbVRqre2P/XW9qfe2unUr4yR1CTUN65fGRODk9A8U6M0CS0/jTw/CTVKk9By3uge9CujmeitDV29taFrEloRGS1Cb23o6q0NXb21d6u39m711t6t3tq71Vt7t3pr71Zv7d3qrb1bvbV3q7f2bvXWNq3e2qbVW9u0emubVm9t0+qtbVq9tU2rt3Zk9daOrN7akdVbO7J6a0dWb+3I6q0dWb21I6u3dmT11j6r3tpn1Vv7rHprS1VvbanqrS1VvbWlqre2VPXWlqre2lI1CS0/jfHmJLT8NFqESWi5ZGTv3trp1Fs7nXprU1NvbWrqrf1LvbVVqbe2KvXWVqXe2qo0CY0wrf1LvbV/qbe2KvXWVqXe2qrUW7uS+s5K1NZWpb6zEnVnJerOStSdlag7K1F3VqLujP5831nZe/26Nfm2wM9/3vcxxJtsfSO5SauzJEnJTZ6bJfqcQUmykijrN+lqlih/BmVrULYGZeumHzpLlPWsrN9koFmiYr9JSDeSUZ0zqiiKlMxLQLNEejgv/swSaT0pxJJCLCnEksInKXySwicpfJLCJyl85mWcWaI8zMrDLO/pvHQzS5TPo/J5VD6r+zWv1cwS5fOofB6Vz6PycFQeFuVhUR4W5Y/mT1H+FOVPURgW6eG8hDJLpD/z6scsUeV0qpxO4jz0SqIy0hBVOVH5E1U5KmvNqwGzREWhMts83z9LJPKDyn6Dyn6Dyn6Dqu+Dqu+DyoeDygDzmsAsUT6rnDConDConDCoLDqoLDGonDConDCo+j6o+j6oujyoujyoujMo9mbF3qzYm1cSsbySiOWV9DmvpM95JRHLql5kVS+yqhdZ1YvcKX865U+n/OmUP53yp9P+SFTn1wHMEoWhqrnzywFmifK5Vz73yude+dwrD4PyMCgPg/IwKA+D8jAoD4PyR2WkrDJSVhkpq55VVjlqfuHALFHlqPyTVbbJKrdklVuyyi1Z5Y2s8kZWGSCr1mp+nH+WqHJUa5VVa5VVa5V1fS8yrlFlgFHV7lHVuFHl8FGhOioMR5VFR4XPqHpEo+oRjSpDjqqXMqoeyKjy4aj6EqO6O6NCflSoFpUhi8obRdXloupyUfW0qHpaVD0tqp4WVU+LqqfzqsUskfyZFzFmicSwqLpcVF0uqi4XVZeL6m8U1d+YFytmifInKQxVligqSxTVbykqbxTF+aI4X1S/pahaUFQmKaqXUlQvpahsU1S2Kap+FdVLKaqXUlQdLGosU1S/pah+S1E1t6iaW1TNLapvU9RYpqixTFH9n6L6P0WNZYrKEkWNZYrKG0WNXIoapxSVW4rKJEVl7KIydtG5RWbsuJI9tLiS+WeSRCVJSpKVRNmSva9JoqzL1iGuZH9skih/ZH9skgxKojyUeXWSKJ9lfyyuZO8rrmTvK65k7yuuZMaOK5mx40r2vuK8HDFLlIcyz8eVzPNxJfN8XMk8H1cyz8eVzPNxJfN8XMk8H1cyz8eVzPNxJXtxcSUzf1zJzB/ntYhZonyWbUFcybYgrmRbEFeyLYgr2RbElWwL4kq2BXEl83xcyTwfVzLPx5XM6nEl5xwmifJQ1eVO9r4mifSwU7W7kz202Kn63qn63snxV+xUfe9UXe5UjetU/erkeCd2qsZ1qsZ1qsZ1qn51qjZ1ir2dbE8nifJHtqexk+3pJFH+yNZzkmjrykPZesZOtoyxky3jJFH3S7aDsZOt3iRRtuRoYpIoW3J8ETvZy42dbK1iJ1ur2CmGd4rhvWJ4rxjeK4b3iuG9YnivGN4rhveqjesV53vV7vSKq73iaq8ydq8yba8yZK/yYS/nTmMvexexl72L2MveRezVXe7V/QoK56AwDArDoDAMcpYmBoVqUD2HoHoOQWWboHoOQfUcgrpfQfUTgspaQfUTgspaQWWtoO57UL2CoLJWUL2CoPJYUL2CoHoFQXEsqF5BUL2CoHoFQfUKguJqUHk1qF5BUL2CoHoFQXE+qF5BULUgqF5BUL2CoHoFQfUKguoVBNUrCKpXEOR4MAY5HoxB9ROC6icE1U8Icjy4XtJQEuWhatGCatGCatGCatGCatGCar+Car+Car+Car+Car+CapuCapuCHJGtp5aURJU8Kp/lvE0MKvsF1X4F1TZF1TZF1TZF1e5ElQ+jyodR5cOosl9UuS6qXBfVKCmq7BdV9osq10XVZ4sq10WV66LKdVHluqhyXVS5LqpcF1VmiyqzRZXHospjUeWxqPJYVHlMrePHqPJYVHksqjwWVR5T+wFiVFkrqqwVVY6KKkdFNXJR+wqmghU+qg8ZVT2Nqp5GVU+jqqdq1X6SKDTkXMokUR7KVeAYVX2PqkcUVY8oqpwQVR8pqr5xVHlD7TSIUfWNo8otUfW1ouobR5V/kso/SeWfpPpsSfWNk8pRSfXrkurXJZXHkspjSeWxpPp1SWW2pPp1SeW6pHJdUrkuqVyXVK5LqqeXVPZLKvsl1fdLqs+fVD5MKvsllf2SylpJZa2kslZSWSup3JJUbkkqtySVW5LqESWVbZLKNkllm6SyTVJ9pKT6SEn1kZLqIyXVR1J7e6La2xOT6iMl1SNSu32i2u0T1W6fqHb7xKR6TWr/T1T7f6La7ROTyj+DqhdqV0NU+xOi2mkQ1U6DOCh/1N6DqPYexEH19AbV01P7E6LanxAH1e4Mcg4/qj0MUe1hiINqiQbVEg2qJRpUSzSolmhQLdGgWqJBtUSDaonULos4qJZoUC3RoJmgWqJBtUSDaokG1e6oPR5R7eiIakdHVDs6otrREdWOhZhVzyqrnpVa649ZZa2sslZWWSurrJVV1lL7AaLaDxCzylpqh0DMKmupPQNR7RmIas9AVHsGYlbjuKyyltpXENW+gkmi/FE5Qe09mCTKQ5UTssoJWeWErHJCVjkhq5yQVU7IKidklROyyglZ5YSsckJWOSGrnJBVTsgqA2RV37Oq72r/xiRRtlRdVvsupi62Okf1f0bV/xlV/2dU/Z9R9XZGNfobVe0eVf9nVP2fUfV/RjXWG9W81qhGf6Ma/Y1qXmtUPatR9axG1bMaVc9qVFlrVFlrVOPBUeWxUY0HR9VnG1WfbVTzWqPKh6PKh6PKh6PKbKPKbGpPziRRJas8pvbtxFHlqFHlKLW3J44qR40q26jdPlHt9omjyiSjyiSjyiSjyiSjyiSjqu9qR1BUe3tiUW1lUW1lUeOvosZfRY2/ihp/FTX+Kmq0VdRoq6heZVG5Re1viWrvSlR7V6LauxLV3pWo9kuklcx1Se1hSGoPQ1pJnJPaw5BWEtWkdjWklUQ1qX0OSe1zSGpXQ1K7GpLa1ZDUroakdjUktashqV0NSe1qSGpXwyRRGMqsntQ+h6T2OSS1zyGpfQ5J7XNIap9DUvscktrnkNQ+h6T2OSS1zyGpfQ5J7XNIap9DUvscktrnkNQ+h6T2OSS1zyGpfQ5J7XNIap9DUvscktrVkNSuhqT2MCS1hyGtZFuQVrItSGqfQ1L7HJLa55BWMieklWwvktoLkVYyS6SVzBJpJduUtJJtSlrJfm9ayVYmrWS/N61kvzetZG9wkqhIZb8urWRbMEmUP3JENklUyXKMltTOkKR2hiS1MySpnSFJ7fpIao9H6lT2U7s+UifblKT2gaROZb9OZb9OZb9OZT+1nyR1Ktep3SNJ7R5JavdI6lQeU/tJUqfymNphkjqVxzqVxzrFqE6OLyaJ8lCOJiaJsqVY18nZg6R2dCS1oyOpXRZJ7bJIai9E6tV979U9VbsjUq9Q7VUG6FUG6FUG6FUG6FUG6FUG6FV97+U4N/Wy5zlJlD+yLzpJlD9ynDtJlD+ydzpJFD6yd5p62TtNvRznThLls+Jhr1jXq1zXy3mt1KuMFFT+Cao/pvaKJLV/I6n9G0mtiSe1Ap6Custq5TqplesU1F0O6n4Fdb+Cul9B3S+1up3U6nZSq9spqLsT1N0Jqrcc1P1SK+ApqEyi1sRTUHkjyJmKpNbNU1CZJKg2Tq2tp6DauKAZpdo4tf6e1Pp7iiprRcVDtUY/SaTPUXFVreMntY6fohoXqHX8FFXOVCv7Sa3jJ7WOn9Q6foqqrVQr+ymqUYBa2Z8kyh/Veqp1/KTW8ZNax09RtYxqZT+plf0UVQ9frfUntdaf1Fp/Uiv7Sa3sJ7Wyn9TKflIr+0mt7Ce1sp/Uqn1Sq/ZJrdEn9aR/Uk/6J/WEflJP6Cf1hH5ST+gn9YR+Uk/oJ/WEflJP6E8SFbvKomqnwSRRPqucqXYaJLXTIKl9BUntK0hqX0FS+wqS2leQ1L6CpPYVJLWvIKl9BUntK0hqF0FSuwiS2kWQ1C6CpHYRJLWLIKldBEntGUhqz0BSewaS2jOQ1J6BpPYMJLVnIKk9A0ntGUhqP0BS+wGS2g+Q1Op/Uqv/Sa3+J7X6n9Tqf1Kr/0mt/k8S5Y8aXySVM5PKmUllSLVDIKk1+qTW6JNakU9qRT6p9fekVtuTWm1ParU9qdX2pN6tkdT6e1Lr70mttie12p7UantSa+tJvVsjqdX2pN6tkdS7NZJ6t0ZS79ZI6t0aSb1bI6l3ayT1bo20frdG/V6Zf/LLZ785O310eHF++IuLfzhcdesXrGWUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUqUKFGiRIkSJUpPZf09xT/7z5+dTfK1akCECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUKECBEiRIgQIUK0jyLx/ij93ilEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEeyWq3x/1yee/OuJP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/uRP/tzfPw8OPp3++sO/+u8PNv/45Zdffvnll19++eWXX3755Zdffvnll19++eWXX3755Zdffvnl92P+PajfI/X925fHp+cccMABBxxwwAEHHHDAAQcccMABBxxwwAEHHHDAAQcccMABBxxwwAEHHHDAAQcccMABBxxwwAEHHHDAAQcccPC2DurnWf784cWTi8Or46enJ+dXyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4YMGTJkyJAhQ4bsO5DV74n7jx6fHn95fnF1evWzwy/PTi9+c3Z8dX3x5Bg1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjRo1atSoUaNGjXpW199H/qunZ8fXJ4enj3922K1W039XXQ7o0L0DXUSHzkmX0KFDhw7dO9UN6NChQ4fOXZfR7b1uRIcO3Y2uoEOHDh06dOjQtenGFTp076muQ4fOSdejQ4fuvdO9Yv8Lui26V+zzQIcOHTp06NC9hu4Ve3jQoUOH7oPRvWJPDTp06NChewu6V+yTQ4cOXbuubB/jrbrt89Do0KH7mHSvyhPb8ws6dH667fvE31y3fa8YOmdd/4r78Cpd2L6PBR06dOjeL90r+lnoPlLd9v1L6Lx12+cc0X2suleMO9ChQ4cOHTpD1/fb5zzQoUP3LXVh+7OP6NB997rtczro3nfd9rmZV+u2zzO8uW77Wuub67bvb0WHDh06dOj2Q7d9nhYdOnTo0KHbD90r5oXRoXPU1d9Z+8HvTs7Ofnb4YKyP77/k6p8+P759s6n7heBet6qvuN3JK8QJR1mYKMKFoT5+KPWdMBjq47u5duDoxuUXJ9wRBTwUDj1MwqC4/ijICEWB96WBKI4FyA9lRMKBe2Nt8L40eCSO7wkI7igIyhbMXh7L80UE94VHd29u8p++vEnSRXGX7oVXg35HsOCuvMsigns77D3sJSTdqwu8Lwq4XV59Ux/0OxC4HXdgLM5/kLew4KUDomo+VDRUN0lYvCs8ui0wv72Npzd1f2tdv0FZVP47woX7woV7Ut/LGGSQkjeqrogSb0uqi/usqoYA5Z44PpLpSoB2u5cYKRQFRndVCOKE+yKEByIhqYwn84PksuT+kUxAsvKlHff9rmaGcOmeTLqS7bI2jFtu2011k+2Q8PlIFPAgS5clygIEmfZVfZMN1R2Jojh+uOMuyYSw9bbL6ro1xUlMo8RQcPVIlnBHYKC4K+uvzJJH4lhWV9WWyrbxfpR3QXgg6989mXaF/kh4/GBb/VdJT3Z4XhZY6gtUa721P7GlKsjbJtO2xFARU97n+/I+qwxzO+2IQaIoa5sERbZFRzoLyswvcFeZX/ZZVHs57Ij6aFeNf6hyzK42XvX0dpBLsvvhrlspG/0HMvOqWynZdKfbgYLsp9yRdJQ9L9kgyo5R2HEjZddPtriyG6I6o7tGHPcFm1WvRNaoh4LNspci+wyyT3FPpABFbpnqH6Qd9122wLK+3ZGtnezNbhtUbRshSGbKUZlMSnIUJ4eBuk8ieSnzdichkTdNNOC6MgqX70seyXGexHxHnta9oG3dLNUBV2l1W+NyNMogZYMb5Y2UKMi8Krm9Y+gnhx2yv3tna59haw98W495a6dCNjZy3FK2gbQNVpkRVEdHJuo7imzb6su2IGSfWuJ8Xx7vmKWQOUf2b2ULLftBsqmS45QH8j7qSQQRkWSS6sLLHCZugu6By8moXcy6p7qnskaqiRFZ32Sml6hLLkueiOqoiCWrj0yDu7i+c0JONW5HaqCza2glh2ZqBLuj0y8bH9WAy6GYbLx2zD7JsaPst+neq5qW2DF0UzSQHu9O5GpEvKOBlfVRdrtk/Vat0a6GQDJJzatKJkpUpcEdw1PFCunPjqqkujg7apJsVuQ9lE2pmrRQPWE5klMNvMzpqmGSUwhyAk41XBLTYcdNVYMgOR0mhzTb5tu2DR+2Tv3IjufW8fiOyWyd9eVMsQxh1wyf7J3LllR2VWVFkD2mrdMmW2qynLq+v6vvIKei1HBBzkGojqQeJwYJ0o7Osexuq/6LmheJ8oJtQ36ZM2UHSPUvts71bq0u27gk276dXJFJSI385K3bMbusxm3basOWtlhmeTmOk4NxWf9lfpCDdTlnqDCXbbUa3e+YXtDMVP1MNYDYtcSo1sskl9V8164pO1ljJfF2rfjJHo9s+9Taj7x+Z8gyb+/ISbLDIkcPcqyrVjnkbZXdUNlH3JZzt2UHtfakeqm7Vm1l2ypr7q5pWFkVZIdr56yRzPOq3ZAZc8cYU3Vods1X3pMGVEMjWbR1TnXbGE829w/UPO6O6QI50aSqjuqxyKB2zFeotawdyUGNQtUiiJxnkkTcMfrQbc6OhlF2hNXIfed8iczZO4YaqlMoId7WqG2dy9uxneBhfk0SyC6abIWkXkIql1B3rg7K7CJZKIeUepS9bVrsZYSy4d6R/9ToRK0bq3QgiSprjlpv3LF+qLKD6turDQPbVqa21E3VKkmPdyz2q6UIyTuZjET5svcmh1tq9kY6LMeUqgN6R6S/ezs7N7JvolrqbcPkrSO4HYNSOb5SXUhZ9+T5OxKw6ipIkmztxG/tvu3YKySnpNXmA+my7FXLvoOa8VJzETuWBtSQT85U6lHwtqq1bTVT5VTVZZMl7thFovel7FgCerh1Un3rxKHMwjs2hciBsm5au124y+Mkg961T0xOCqmWQK5GqKyp9nHIGi5RUSOmXRNXajPctkHaluZUDqDUJLPqncs+ihw3q0y+Y3yhVkdUv3PH8oocwqkR2Y7hw84pl+0jZcWcrZlZTh9u2yO1vQuwbUJy633adWf1vM+OkaGcONJdLUmuXfsBJDtVD1/2h2WDtG31flteUsl35zKRakNVMy/pt2v1T44FVR9f8ldOHMke844esupRq5HgruQt05ZcoLizdaVKZvutDc7W2bttbZ5arFbtsLxgxyYi2Uiq3pnKCjKGXd2z7dOgkmtbs72aDJZ3SiZXeSxz546FXTnnr1pZ1QWUA9JtW5lfGpDVa9eM+46ZF9kabZ0r2pJx1I486e+uzS5yuKk2Zclp3B0pTzaXciCjkrTsMausv+2ObbvFuxZRZJdDLn3pAfWO9dAHIp0Yi7o7RlJy5ekVOxxUVdu2fiATzu5+y47GTe272rapePsayI51VtkzklN6ep/Hjp3UskHX+yrlsHvb6vs2lNXMqkxQu1ZNti5KbCOTvk+yAd+VA+VKjnryQaL8uj1cSTXJHDXul8OKbTllG7NU+yyJobLYjrQrlxD1LqwdS4hy7kNPX8thyY5elBwj78paekS7bc/htg6I7KHoTYS7ljBlYlTkVvN4ejuqbJ6U1zsa+F1T2HLiTY0JdjwRp3YI7Zjkl11LlchVgydzkuwIqueMdi1TbJuN2dYbln0g2aDKKSW90WLbEuGbroLK4afewrRtKWWLRzLjqF1Wstu3Y+pFrS3Lm7BzWVXxdsdEiep47sgfcoJLTcTIDCpz9o51ErmwooYfsvsg85McfaiKohKWmn9Ww7wddVfm4F3POcrJ2DvSRTU1o4dQOx42lflPzkOqZmLH3jtVuWW+Unvltu1V29YBkouaW9e7ti0n7ZxUl0OGrQ/wbpuc0o3GjtUXmVJl06huvBpmyX0aqjrsespQtWs7dr/pJ+i2Ts1ua76npFY/if+j69Orq2cnPzv89bMnF5cIESJEiBAhQoQIESJE+B4L//Kl8Jcnl6dPvzq5/BoFChQoUKBAgQIFChQoUKD4rhR7OHBGiBAhQoQIESJEiBAhwjcT7uewCwUKFChQoEDxdhV72ClBiHCPusJ76CtC/9uM4j1R7CGpECJEiBAhQoQIESJEiBAhQoQIEe6ncD8nuJgSQ4gQIUKECBEiRIjwIxbW74b9y6fH16cn59eHp49/dnj711983qNAgQIFChQoUKBAgQIFChQoUKBAgQIFChQoUKBAgQIFChQoUKBAgQIFChQoUKBAgQIFChQoUKBAgQLFR6PoUKBAgQIFChQoUKBAgQIFChQoUKBAgQIFCjdFRIECBQoUKFCgQIECBQoUKFCgQIECBQoU+62ov9fw2fXV+cnho7NnV9cnlz877Dpn+V/U8v4dmUWOHPn+yEUaGPfUzQ9VLuAPtri8Omd/ZPdQhAuT9yFhNInzWyz7fRXvPSZv08G9D54o31LwHzUm+yN+vbvjci/fpskPzMGPWgxjCf67FoPJeyje+5u2P0004rck3jKT4yLeMh30ujP7q1cbbTzdadrhXeDoBNjbuKc7CvHx0OdWbPNln3x8K+i+WQbYt+k+5O+X/DuiVWMKRf5hyd/z5Paeu/+BwvbthkUf+s16T8LdMzf3zJ09c/M9QQc57Pxw2Pmeh/UOJ2S+g7m09O7L3iJ+sxmgt4H325zpfJt4f4cztLLybJkqes2urMuE49tE/G3eNhe/d9z7NsDT65Wy9zfiu6gR++e4S2X7ltFvyflbml7E28XfsvfyTkpBvI/i1+tfvc2zX5NtO5Z7m1x5vZ0I3wGEOwrxgXBbtm7r1X0He4R3PCjyDsWvC+3brPk7qmFbE7EF8i0P+GwRv54rw97kVBfx2wzH5dZvKWSL365se4fi79Dvb1mn3mov7fUq7OvV4++w1r8mtE3x7M9TDXv/MI9Ln26LeIuDLoV8Bx1Dl67OFiK/XtkfaPZtQvCjFu/PTUP8Hoo/MPrsTxO965UA++PhByYG8PdRzF1DvK/itmVElzlyl7L3D9kPW8zrJRAjRowY8QcupqlDjBjxnokPqrdc/5NHJ2dnh9dfPz352eEvTi6enh1fXZ8+ervKf75Q/t3Z6ZcXj0/OH19ePPr6+uQ79AolSpQofZT/dKH810+ePFvn4Emyr+6i3KX88UL5Xx1fPXp2dnz5QvXpsj375Z1v3crBEerYx6Z89T1DixYtWrRo0aJFixYtWrRo0aJFixbt/mjtme926e6588Y15OVs/tHV9fI626PXvuAtzZ9+d4j74unr2+uU5uSq2116hT/75qpTWH45Ze+WN1CifONKx14slB+hcn8qAwF+IMp3ivibdGf2DrEP/i593DG8J26+JzHsjycoUe4bJ/fHE6J/ZwG2S7+bmea/rEJ9dnlx/vaL3qrwnej0A/9dLRG8xZuxj6sczlOkTrPFb/EmvO3b6OL6O4F+q6evKGmvb8x3UEf2wb232Eq8InImbVGiROmofNOu57u+7m1G6HGGf+9z/+7MW7l3HmbfbCvFfm6taiXF++r3K27U/sx2vCVjr3jG/M1Ub8WVvWu73v3deE/ItntI9a3eZLB3d2Z/EP5gqLdHeen9yWfvutD9bNM/Nu27GUi09vDevoX9Gra83f7yK2tsq2kawjeEdu/c/YCU7wlBUKKEzU1Z9H3UfmThfmxabi9aiIMW7T5qX3tPzttbUnx79vYTerRo0aJFixYtWrRo0aJFixYt2nZt/XnAf3p+M4lqTai+Ne2Pl9pfnXy53AT1HbmEFi1atKRBtG/75rqr9hOB/dQC43utfZfVChUqVKhQoUKFChUqVKhQoUKFCtVHotqfKakPfmLtvXByT1TvO1bvu//7VoXRon332j2k+h66hPZj1n5UhPyogt1P7Xs0gthPAD+qG0c476/D7104e+gSWrTvEV330CWg+G6DRYUKFSpUqFChYnkLLVq0H4x2T5LbW8p7exLCh6oCfFR7mLjeo2z53iGwn9p9I8MHw7H91L53we6hw/vTGdhDYN+Xqo0K1Qepehv1aA8zLa0VWm4B2o9ZC9U/jP7hBxzah6rilqGCA6hQoUKFChUqVKhQoUKFChUqVKhQ3ajq77z9+Ork7OTR9enF+c8O/8350+Pz85PHqFChQoUKFSpUqN4b1T9fqH5++ujy4suz0+Of3Lkb019zBmdwBmdwBmdwBmdwBmdwBmdwBmdwBmdwBmdwBmdwBmfcnPE/WZxxdHV9efHo6+uTq588vPfLoztHP+cszuIszuIszuIszuIszuIszuIsv7P+2eKsX5w8u7w4v/rJr7/6ukOPHj169OjRo0ePHj169OjRo0ePHj169OjRf7j6/+lC/3dnp19ePD45f/xyLeHBnZen7evyBmdxFmdx1od91rdM0pzGaZz2Nk5791V+X3bwc8Z3+czD/3hx1r3zxxfXX51M5s5+cvvzO5zCKZzCKZzCKZzCKe/+lH3pcHHG656xz0MtTuO093TMzlmcxVmcxVmcxVkf51l78iJ3VKhQoUKFCtX7rtqXeUPO4AzO4AzO4AzO4AzO4AzO4AzO4AzO4AzO4Ix9O+Pg4LODg0/+8OLMP316cfb10eGvfnGEAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA4MMU1O+b+Pzq9PzLs5OfPjo5Ozu8uri8ng4PT88Py/DT361FT8+Or0+uDn97cXZ28buTx4e/+frw0d1fHB0+vTx5enx5vH6lxeGzdRmHnz85vrz+6dXJ3/cUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdEUTdFvt+iDg88ODj75wwsT/7fnms8nzVTOZXXpxW+fF7cs/+z0N5fHl18bJn5x8vvrk8vjw3/761s3f18ff/nk5Pz6+WnHjx5dXD5en3p9cXj91cnh1fXx+ePjy8dTYRfXF48uzjYePI/5cBPzr35xtHb6hQdzIJ/PGBEAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARAAARCAYwAHB58eHHzyh7/9v/8/Dtb/+OWXX3755Zdffvnll19++eWXX3755Zdffvnll19++eWXX3755fdj/p3+fbZeR3+xIeCffr5Yf//t5cWTw18/e3JxiRYtWrRo0aJFixYtWrRo0aJFixbta2j/udL+8uTy9OlXJ5dfcwZncAZncAZncAZncAZncAZncAZncAZncAZncMZ7dMZ+zkCiRYsWLVq0aNGiRYsWLVq0aNGifb+0+zLbxRmcwRmcwRmcwRmcwRmc4XXGfva80aJF+/5qyTto3z/tvrTJnPHhnrGfzEeLFi1atGjRokWLFi1atGjRokWLFi1atGjRokX79rT7slK1L2fs511CixYtWrRo0aJFixYtWrRo0aJt19bfUP5Rt1pN/111Odx6MBrC+8ML4acL4e2uN6T3upVRwO3OLMA69ShbDhTL1cEQPjTP7CyfgiG8mw1Hj24iXZ56xyr0oeX9w2T5ZJV5FEycLEv3TfNRCeOth/qWTkJ9ebx1Wzsab90blU9xsm5df2QJ72lI4wSeDn+S6psX51tSCc3LrejvW+7fvaHeUvrADkrzZArKEhr3eXJKs3Syb11+V0efpvpkCA2X0q2HujqmCWfretPSfavQ29aZBsvSzOeleQvRqVDreuOOJquOTYVqQqSZpZWjOm9NQtMngyaTTwZLJ6nl/m19myeh6ZTOhevbbElvGzRdg6Lz5iS1XL1vuXrPPNO6fXdtqDTNJ/MWfvdN/G5b98+oEcnKB9OZFtL3LOGRJTSy4eSSFf1t855YDdzklB2+dep9K/wHOvEns91JVpIdrHo6WNV8uHWk795gpbPp8qR8GkxKDgui1Odq/4e5PVgKjXZzsHLfYHVQhpk9S/MPDPgHq9mfQLEKfaAbrmHRcixPNarEYDXmg5l9BqvXMty6Y94pS/jQwtnkiZFQB4uRw8zICjxN88FsY9ZAWYVG8z7p2rd23/TfwtSqkoOV5gar6ZpMWUIjoQ1mV2yw+lLDnHrru295aqSkyX3LqbvWmUc60HzrgWZUnlPnpwvhPSOmPPfYl8KjYlxvdQWz1b3NVr8lW1U/W+zJVsObrfuUzVqWLfblRRs1S0czdY9z/7g+VdsarTs1WrlntJAere7IuKgSS+k9I8uO1pBtNNvz0epgj1Yna5xb2Tp+C8AjE0AjT463Huo6PZpJYTT7k6NVqUcrJ45WrRit6jvOdfLT6kzzVAuVB0Y7WUxaFasGlDlTL6+3UC1Wf3q63irU6BEUq+9UrJxYrF5KMUlVrLFQsbpuxeokF2soV+Yh79K8NYtQZqJ9WkFqnWpklTJzsirUOtPouBarizvhrLNnMWtvsdruCX0rJouSxerPFSv7lLlBrcxbNLlv3byHVkzW4L5Y1axYczjFSv7FmgIq1rxSMbvNxZpXKnPjUYFnFWq13MWaLipmkipzlqiCMrlvVR1jCqtYXZxiNbLF6ssXszdT5qFEXcl1Oi6Lpq+mtFXskU6oxeyil7nrU5PKRNVCwJjJKFYnt1iTDsWaSijWWLLMY+6afhYo1lC4WLMGxWp6itXHHVeGq5NQNwdrobqrk9BopNaFaqQnqc6ok9DopK+LVVV1Et5RVWXjgEJlkuoZko1XFgB6NDsJ9V1d+28KLfd1c7zGzxLqYePauiXUvfm1IfNGKU5tcLIgMfLs2BnpYxJq9k9CPb6ehHrcNwl1x3cSGkPhtSUdfme03Buh5f491cfYeGq5eltTqrPyxCTV3Zl1ABYoty2n9PTK2pJKU2tDtnlVz9dCbb63anRv1eje6OGvz9R5epIaHZ9JeqTh663e3CTVky7rUy3/9Sz0xivLvm691k5ppHuji7gOyhIa7O2NAfL6cgtTPTe1BlpXqN6qEf2ie1+jb0Gqp3zWnlpOGQmlN6bbN/Z1PesXmbO+0RYqupO2Flpe6fHF2n/tajA6JJPwrq48wUQqWOy31vkmoR60TkI9YzZai39rnyzrRuYMFnWt1a9JaKSTaKWTaLkUjfm6tSGdTaLRlVifqaGLRk9sfblBkmiNN9dS3ZeIVmdyfaoGP1o31Fp7XF9uoXfXMm90uiepkSOjyTJrmXFdquWU0cBGq47EmSZL88YcwGgtKK59Ms3rBjJaw5PRWpAcrfXcMZpteTQ60uvrzfCtQq0G3lr73JxqCa1qYlT8aAwDNjFZd9S6T0bWsxZu1y7pNjOavdhoLB+swbMhsWhqDLk2rlq+6im00VyjX59qhaWHvBuhxQmjcx+tbB7NTlu0+ofRmC0eF2v8deW1pFavJc38+XQhtLrSyeR/snpI5qLmaK1fjtbq+WityY7W6veYLFSs1dNJaGSPZGVUa518bcgSGs12MiaBRms7wbpMy7qRZK212zEZK02juR49JqsjmKyWz9o5MCZjonU0NwmM1haLDaEs6ZEptapUstKPtaFgfb2uZ8ms58lYVhitFfHN9ZZQZ5RktvHJ6rJbmwxGa+PH5qYq4WJN69OF0ELPWtIbraXjcbByv7WkNlrL0evLrTL1lPZoLXFPQmPAYS29rq3rmjdYQ+jBTLGDMaM9CY3B8mAOAqxF4rVTuj4P1q0frLQ7mCQdrKo/WCOLwZp+Gsxx7WA13NYS+WhtkBithev1vbfONGrTYM3fDGY/epgXBCrz1n0y5knXMelug7kVY4OUGb91Tx/ozDtYc+rrAMy6Y91+K3MMVhsxGJt+xsFqXrNVebLV5lrLyesz9e3L1gRCnvPWp8szjXuSrYY8W4zO1rRCtnoc5gr7WqrBz9ZwM1spLltVL1vdeHODwDomy5LRkmVrvJStHJetOdpsJa5s8jFb6ShbfY7FpoNloRZJs0XSbAxNStQdkVXX6zM3QutMtYdsLdQsW0vVXMFaqNL2RmiZV5NXzy+3LKkctzlVgr/x1DpTNUUb8xL8jVBNQDyXymy2ccp0VVW9jdBCSjUGm0KtM1U3dFOm6ZQegm8KsFBR/aANqCYnrHuqeicb+M1ArctV4tkYsnii+jGbMi0/VXXeuKRhNqZkVl02Zs3XUiPOrGev1kI9fFtL78iWaC28pzrmz6VWAaoPvRaqbuDGAQ1fnmvp0pKe/tnY14zIegC3OdMSqhy18UnfKKPV2VxueW9Qz2hyN/dJOz/qYdVaqPfdr6W6fZvSqRV90X27jVDf0WJsPl1LjUiNnQ0boSaPtdlkY99y30gnxjL8xifTU7UOtupHi+f9aPgfOt1nWQv1Xq3NqRr/0OnxyqYA05QiRej0XqG10Khmk1SNoTZCy6eHCr/QGQvuz4OyTKmJkk2klv86+ayDUi3XBlTLVTVe21yvb/XaKVNohWokr2AteW5gUQTenGqhojYHbAAwT9UJYBOWdQNNqC3/9TaE57fVvIFmqJZTulau3Vf1f12mVVX0SupzVpr3yiKwmud7DqoS9hamxvrkWqi7OKHXe0XXQgu9Xi/HbK63fDI42RspORgrthtHVSsfemNG+7nUsq+29W6ERk3vrRZ1HYDlqxoebc7UlTpYHc8QrH7KWqqrajCZEiyqWU9Sbk41wjIeetycqgkU9Jz+5nKzUDVm3lyvE/hg3ixjwmgtNAo1psvWQjW1tBbq/mwwNtWvhVb+tx7U2Dil77X1AMPmVMsrtYdzIzSvNwg4WBV4sJLSYPRzN5b09Vlvj1gLdU8nZGM0GIyJhM2ZltBo07JZKbPZ/Fp955D1poFNqTqBZKtPF7LeWbYWWrUn68m151Hp6mdNBmyKtXw16n82FqQ31+uamvWyxMa+KbTKVFtBNpHa8es6bY5IQtarb8/vtck/K36rAcp6RXwttHoaWc/Wb7wyhbqmWQOdYA1f1kFZoFg9NWtMFIxZq01MFlHU2uvGvOWoHg8GY3Ztc7npp1nPrTKN1tuY3drcZUto9JKzleSzHg/2vbVVey1Vs1trobK0Fqp8thaqzLcWKp82l0tENpdbLql6uxHKO7cpUzJ8c6ZKBmupeuZ2Y96ERN2RDSSWUPWFN0ITZlltN+6bnqoR/sZTmaI3jqq+/PNSZQ3fOGCZUttTnoNisUfvBXjOKcsDvWrfB2P/9Fqo2ui1UHWy10LdHVxLVUVZC1XdXQs1UyfhHXVbgrUBeOOpbd/037RvBaVz/Eaq6mQw9j9vMJXp9DlSigBrp6xCVR9hA5/sdz2PyfJUTe5ukFJVNRhblddCNW/TW4P5fjOWs6RGBQjWtuC1VHUx++W4eXm9kRXWUisA1SBsvLKEaozVm8PujdT01CrUIHqvm7O1UKf5adShJvg2Qtnt7zdDGQ2/tWlpLTXYa7xtpA9J9zrWQtVIroVqz9PmTOM+G/trNkKNXtJTxmuhVSON9yCshcZ9Nl5DsTnTuly3EyEZ7URIRv9+LVUTIRtLmmbWm0E29m1T1qlqJLcxZSKlG6qQzNqbjKWdTVhmsUamSHoF9Xmw5g00wzLPtG6gWnLZXG4x1WjljE1HG0etMtW+5o0h606rkdjzy3VCTFaPaAOUJVQ9mmC8hGRj36ynVkxGc5isNJOMMecGPSt83R9bm7corUc8G6mVe9Q0znP3zXtiVinLUzXl9ZyPOlJjM9JaqBYwN2daQiOfGy9cWAuNJtrYx7W5XHcGrBdrbAq1vFeby55b0rdksKrjoHcU9NYc2HOnrDN1dRysdGy8lmXjvWndOlMPb9YRWS6p9cON85ok1rTa5nrNfGNzV7+cAFsWaqWIwWwijD1bm6B0hrf2HPXWvOIGfes2G3nH2PLUB2PH23NCWEHpOfDn51oOGB0kY7J0A6B5+y3/jb7AYPXDB6t/Y73S5/n9swo14bMoaQwjjA17vTV9vLlPuskdjDWl5+5bUiubGvO/fTBW39dCA/1sDYOysarQm3NN/XKubik0Oo3GrG5vTUpuzjTitzZTraVqB3BvTiv21izQBgANqjGBtkHFitRoDow3uDyHzw7KklodxM0k2Ms3Yh58Myn/6P/3+VcXl9eHlyfHjw+vL0+fPDk9//Jnh7+8PD2/Ovn7w+uLSfPk4h9ODq9uTrs6/MlPp7O+ODs5Pwyrv95cdXj91cnhb08vr64Pu9Xhb54eXpxvROm/++b/8tOT88fTJevTpmt+uz7jr//l86ueF/e70+uvDs8ufnf498+Oz06vv355cRAXr9VfXJ5++dX1YZ/++vB40vz29Oz65HJz8aOLJ0/PTn6/vv6ll2ePvnhycv3VxePDk/Pry4unXx/+r9ay668uT6Zwzh4fDumvbx0e/i9PDp9dnTw+vH/0+a//yzvrmB+fTKVOMZ4cTqFfXp48nS6YipjOmUB5dnL+6ORqY/45NhNwX11crQuZsDt89Oz6+PHx0+vJ/snh6laXDv+7b/6bJ2ukbq0tTYGdPzenMb64fPrV8fnh0+MJyMOzk6ur6ezjNcgToKfnk+T8ywmp316cTfFOBfzm6+cXHp8dXvz28Pzk95PTx4cb4ydTAc/d+fUa5wfHZxeXJ4eTSz+9mqA8/3IK4evD7q9vQQJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkACSAAJIAEkgASQABJAAkgACSABJIAEkAASQAJIAAkgASSABJAAEkCCj4AEB58dHHzyh4M/PliT4emv1jSYbsKX50+meH/28gadTBdvpOtYLzY356svu3I4lXPx5OT5Dfz810e/emFzQ4aNl+uji6fXpxfn6/tx8ez6/uaG/frrpyeHt7/+/D+fJNM9mRU/f3Z2ffrk+Okvnhz//rBfrXUbu9OZ042YwPjy5xMQ40J+9/ZS09WlnV49Ob5+9NWmtFKKrVxf/1+sz1jdWsWbkv/1mjHn6zIXbrwQTud2q82/G83Pj69Prh4cPxW6ydrnRz+fMJ/Ycv/0ZKLc6fMiLq5Pf3s44Q/u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7u4A7uHw/uBwefHRx88ofn+B/8b395cvnTCdSTDfCHx1dXa8yf34qX+C9uyaOL88n36+c35dHFs/Prqwn3y8OTSfr14aac5zfk4a8/P/n76QY8WZs/ubw6ebS+JT89v5gKf3r99eTp1eH5xUsa4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZu4AZuvK9uHBx8dnDwyR9euPOPH5ycXzw5+eI3z07PHv/s8Ksvu4IGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo0aNCgQYMGDRo075mm/i7LyefPnj4923wX5vjy6y9+e3p2cvXFby8unxxff3F8/viLRxfn15tvxlwf/+anj0/OTp+cbj4Sc/L768PNyYe/O73+6vDy+HfPvzPzD8dnz05efDrm+NFXh1fHT6biMYYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGMMYxjCGMYxhDGMYwxjGMIYxjGEMYxjDGMYwhjGMYQxjGJPGDg4+Ozj45A8vjP7gwS//i25MOXDMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHP8kR7Xz6F/9vnTry8vrv7lv7x7fPnk+Pz0Cjly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5MiRI0eOHDly5Mg/OvlB9R73/9HVRn/r8Qv133755Pj07NajiyecwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmcwAmc8OGccHDw6cHBJ384+v/8Z//7//r/dXDAL7/88ssvv/zyyy+//PLLL7/88ssvv/zyyy+//PLLL7/88ssvvx/z7/Tvs/U6+osF9x99fn3y9KuT88P/8tnxvztBiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJEiBAhQoQIESJE6C+s3wf56e3Ti5PzL0/PT04uT8+/RIoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkSJEiRYoUKVKkry2t3/v648+vj89/e3H5+PDfnJ/+w8nl1en116hQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVKhQoUKFChUqVPulqr+/9I/vnB1f/rvDOyfn1yeX//LwXlit0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBgwYNGjRo0KBBg+Y909TfZfnBy0+3cMwxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc8wxxxxzzDHHHHPMMcccc/yRHtfPof/wzvHZ6SQ/Pz1GggQJEiRIkCBBggQJEiRIkCBBggQJEiRIkCBBggQJEiRIkCBBggQJEiRIkCBBggQJEiRIkCBBggQJEiRIkCBBggQJEiRIkCBBggQJEiTvpeTg4NP1e2X+9v/9J59Nfx/wyy+//PLLL7/88ssvv/zyyy+//PLLL7/88ssvv/zyyy+//PLL78f8e1B/n+WTf/P5EX/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/yJ3/y5/7++eI9Ugf84x//+Mc//vGPf/zjH//4xz/+8Y9//OMf//jHP/7xj3/84x//+Mc//vFv/e+zg/k9Un/1r8/Onj05PT8+/MXJ768/P/n7w7RaoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDh06dOjQoUOHDt071dXfWfvjR3d/ccTf/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/M3f/P2d/V3vc//0+vL4/OrR5enT64snp4+QIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFClSpEiRIkWKFOlrS+v3vv6jX/3i6Kefn/w9hxxyyCGHHHLIIYcccsghhxxyyCGHHHLIIYcccsghhxxyyCGHHHLIIYcccsghhxxyyCGHHHLIIYcccsghhxxyyCGHHHLIIYcc2of186n/i9unF58fP3l6dvKzw6+ur59e/exv/uZ3v/vdrfNHvzm9dX725Nb56Ve3vrz4h7/5zenF1ea8v/n86Oe/WKXYd91q8Cil61x8SS6lrBxK6WPxKCW5+JI80O1TcCkluqDrcadXZXSJyKcUD750nQcu/eDCusGFdUPvgotPKS41IDr4sioevqxK71LKmF1KcahHq1IcWDeV4oJLcch1UykOdXoqxaFOT6W43GmPvDuV4pExVy6t/WrlketWK48stVp5tEerlQu6nQu6Lr3DlUsbsHJpA1adRy9o5dLarzqPHvyqc8gMUyku3O1cuNu7cLd3YZ1HOz2V4sK63oV1vQtfehe+9C58CS58CS65LriwLrjwJbjwJbrg4tH3nkpxwcVnDOsyQ7AaPGpjcMlSwSVLBZcsFVzyS3DJL8ElvwSX2hiCR58hBBfWBZd7FFzuUXS5Ry75JbjklxBd7nR0udMubUBwaQNCdMl10YW7LnPNwaUlCS5zzcFlrjm4zDWH5MJdl1ni4DKnGlzmVEN2KWV06dcNLuOAwWUckF16ZNkFF5d7tMouo8/s0mvOLr3m7DLnkV1YN7rwxacejS58GV34MrrwZXThy+jCF491kqkUH9a55DqPVYWpFBfuFhfueqz8TKW4cNdj5WcqxYN1vcs+j37l44vLirvLCkfvssLRu6xw9D7r0y7rAb3LekDvsh7Qu8zk973LPXKZUepdVhV6lxml3mWuuXeZa+5dZrc6l95hl1129Lj0DjuPvtTU8fApxWM/w+iy52R02XMyevSlpg6Zxw6N0WX/y+jRl5pK8dhFMxaH2jh1VF12mLq0AZ1L9u5cVjg6l/XGziV7dy4z+Z3LTH7nMpPfucx7dy4zs11y4UtyaRtd5lQ7l9nQbnDhrscM5DTgc9m96DHXPJXi0aqVwWUnpcc9mkpx2Uk5uLRHHivuUykud9pj3ntVPOa9p1JcaoDPvPfKZRbH5amUlcfMSe+ybt+7rNv3Luv2vcu6fe+ybt+7rE/3LuvTvcv6dO+yE6F32YnQu+xE6F12IvQu6/a9y4p777LiPpXiwjqPdfveZd1+KsWFdR793d5lD0HvsodgKsUlY3o8I9ZHj9nQqRSPiKLHbOhUikdtjB5zqlMpHrUxeoztp1I8amN06b9El/5LdOm/RJf+S3Tpv0SXnkd06XlEl9Y+urT20WN/XR9dWrXo0qpFj7nmqRSXXOfSwkaXFjZ6zORPuLjUAI/V/6kUF1w8nticSvFAN3k8sTmV4sHd5PHE5lSKB1+Sx2zFVIoH65LHbpGpFA/uJo/dIn3y2OcxleLCOo8nWadSXFjn8STrVIoL6zx2rkyluIzVXGpA8Jn1c6kBwWO/VB9c6lFwqUfBpR4Fl3oUXOpRcKlHwaUeBZ965LEDbCrFhbse+8imUly467GHoHd5YrN3eWKzd3licyrFhbseT6f3Lk9sTqW48MVlRil5PGXQDx5rfFMpHnd6yB53evB4EmQqxSNLDR47BqdSXO6Rx/MkUykeWWrw2L04leLCF5f5l8FlbD+4zEAOLnMeg8s85uCyDju4zIYOLiuog8ts6OAyd9h7PAs1leLB3d5j/+5UikcN6F3qdO+x83YqxaM29h5PMU2luLDOZQayd5mBDC5zh8Fl7nBwwWVwwSW7jMqzy6g8u4yns8tIOLuMYbPLGDa7jD6zy7gxu4wbs8uIL7vsIcguY7XssuKeXdbKs8sqd3bpY2aPleWw8tgFHFye2g8uz9sHlyd8Q+cy1+zyntne5c25fXKZf0kuO1eSS21MLrUxuYyyksveiuSyN7S47HQqLveouNyj4nKPissYtriMYYvLGLa47MVxec5mKsWFdS47V1ye1umLxxNVvcszP73Lsy29y/MkUyku3HWZJS4u87vFZWa2uMypFpcZJZevEPSjR+9wKsXHF486PXo8lzWV4sGX0eNNYlMpHnwZXWrj6FIbR5fa6PJ+hqkUF3RdZmZd3onQu7zNYCrFBV2PtxlMpbi0JC7zmMVl92LxGX267FEqLvOYLl/omUpxGR+57MUpLjOQxWcG0mdM4jOP6TKyyS7j6ezyTGF22ZOfXZ5Xyy5PmmWPp/anUlzmMV16Html55Fdeh7ZpeeRXfoM2aeFdVmFGl325I8ua1mjy+7F0WXf4eiy8jO6zO+OLrN+o8us3+gy6ze6tI2jyzPuo0t7NLrMtI0u+19Gl/m60WWmbXSZaUsuc6rJZU41ubyHILnwJbk8sZlcWJdcemTJpUeWXOaak0sNSB7fMphKceGLx1uqplJc+OLS300u85jJZSY/ufS9k8tsaHJZVUgus6HJ47sKUykufHHZTR9c+BJc+BJcdlIGl7nD4DKyCT5PhPu8i8ZlZja6tEfR5402Lq1adGnVokurFl1atejSqkWXVi26tGrRJUtFlywVXVq16NKqRZeVwujSNkaXtjF6fHNoKsWFuy4tbHSZgYwuz6tFl3nM6PK8WnSZDY0uz6tFlznV6PG2/akUl+ePXOYxo88TeC7vRBhc1hsHl3ciDC6rloPLvPfgsvY5uKx9Di7PcAwuK6iDy5Mgg8tM/uCyDju4rKAOLusBg8vTF4PLk/KDy47kweVJkMHnfc0+7wBz2S0SXdbVosu7rqLLm36iy/6X6NKqRZdWLbq0atGlVYsurVp0adWiTy/IpW2MLm1jdGkbo8vuoujSNkaXtjG6rJVHlxY2urSw0eUdPdHnHT0eM21h5dLCunxvos8uGTO7ZIbscqezy3uUsstbwrPLbpHs0jvMLr3D7PJkYnbpHWaXnSvZ5Xm17PJMYXbZ/5Jd9lZkl10R2WU/Q3bZz5BddtFkl5Wf7LJmk11WW7LLOkl2WeHIHiscweVrqn32WOGYfPHoM3Qerf1UisfbDDqPFY6pFJc3CHisKoTOYyZ/KsXlHnnM5AeXr++G3uNpndR7jLKSyxe1k8tXrKdSHLibXL7LnVzeATaV4nKnPfZWTLi48MWj7z2V4oKLx9vekssb1pLLG9amUhxa2KkUl3rkMUucXN7TloLHlyKSy9veksvb3qZSPPgSPOa9k8sb7pPLG+6Tyxvuk8sb7pPLG+6Tyxvuk8sb7pPLG+6Tyxvuk8sb7pPLG+6Tyxvuk8sb7qPLekB0mcmPLjP50WUOPrrMWE+luODi0ceMLt/Aiy5fr4suX68LLj344LKqEFy+dhhc7lFwuUfB5duAUykuuHg8KT+V4oKLx5OJweWrfsHlG3jBZQd7cNl7Hlx2jQeX/d7BZad2cNljHVz2WAeX3dHR5XvC0eVLwFMpDqybSnFhncd+7+CyUzu47LEOLrujJ3Q9eh4uT3fF4FEbp1JcWOcx7z019i4ReezUnkpxichjHjO6PCMWXXYvTl0pj3vksgcyuuwYDMHlbdjBY70xuDyxGYLLiljwWLUMLpkhuGSGqRSPnkfw+JpHiB6zocFlniG4zDMEl5m24DJHFlxmt4LLvFRw+WZicPlmYnD5TmFw+cLgVIoLuh7rasFlLSu4rJMEl7WJ6LKWFVy+XBSDx8zJ5I1Hv6732Os3leLRl+o95g6n/3j0MXuPfYfre+1SigtfPPZATv9xYZ3HfN30Hxe+eMz6TaW4sM5jV+d6BcmlFBfWeewNnUpxYZ3HDtOpFBfueuxTjb3LTFvvsdt1KsWFux57ZqdSXLjrMRKeSnHhrsd4eirFhbseo/LYu8zX9R6z51MpLtx1mYPvPWYrplJcuOsx5zGV4sI6j3nv2LvMe/ceu4CnUlzukcfuxeiywzS6jI9ScNmR7DLPkFzmGaZSXHZdeTz3mVzmPKZSXO60x8zJVIrLriuX/d7BZXe0yyzOVIoLdz2eb5xKceGux/ONUykurPMYN06luNRGj5FNcln9Ty7r9lMpLnzxGAcklxWx5LNW3rvs6uw91gOmUlzGJB7tdHR5cmj9JgKXUjxWxDqPecypFJen3ly+s9y57BjsPOYxp1I81mw6l32Hncc8Zug85g6nUlzutMseyM5j7nAqxeVOe7SwUykufPGYOwydy97QlcuTzyuXJ59XLrv0Vi5P+K48xvZTKR6sW3nMEISVx66rqRSfbya6fP3FY7aiLy5vWCsu7wArLu+6Ki5vkSku71wZfZ7w9ciYyWW1JbmstiSX1ZbkstqSXFY4kssKR3JZ4UguKxzJZYUjuaxwJJcVjuSywpFcVjiSywpHclnhmEpx4a7HOklyWSdJvccO06kUF9Z5vC0luay2JJc9Ssllj1Jy2Rc0leLCXZd5TJfdIiF5vHNlKsVjHJA83lsRBpedlIPLHuvB440TweVt+8HlbfvB5W37weVt+8Hl24DB5duAUykeo0+XN/8Hl+8UBpfvFIbkMnPi8rXD4PK1w+DyncLg8p3C4PKdwuDyncLg8p3C4PKdwuDyncLg8p3CqRQX1rnM+rl87XAqxYW7Ls+IuXwzcSrFhbsuTxsnl+fVksvzasnlPZDJ5Q2O2aUXlF16QS7vGp9K8UA3u/QZssuzcy5vT59K8cgM2eUJvOzylGR2eY4vuzzHl12e43N5w/1UisuddnkDkst78qdSXPji8mRi9lgPCC5v259KcWGdy5uhXN7ZH4LLeqPLFxEmX1ye/ffY1xxc3jgRXN7yEFy+URVcvugUokt75PKFnhBdaoDP++uiy96t6LLrKrrsl3JZKQwuT1QFl5XC4LLGF1zW1YLLilhwWcsKLis/wWXlJ7is/EyleGQplze5B5d3sAeXt4QHl7eEB583K7i8mTu4vFM7uLxTO7i8DTu4vDs6uLyvObi8aTm4vCM5uHzDN7h8wze4fMM3uDzdFVyeywou31meSnFZnXPpHbp823gqxWUF1eVOu3wheSrFpQa4zDMMLvMMLt98Di7ffA4u33wOLt98nkpxqdMuo6zB5T3Wg8tYbXB5G/bg8pzN4PK+oOhRj6bBp8tYzWV2y+X9UsHlnU7BZddVcHnrUHB500/w2XXl9HYdl7VPp/dLubx/12UkPJXi8sSmy1t8XUbC0WUMG13GsNHl61LRZfQZXUaf0WX0GV2+xRRdvqIUXb5/FF1GwtFlJBzXI+GDg88ODj75w4vS0ue/OtpRztXl8b+6Prl88p99/qt/25Vh6jJ0LRcPLRfnhotv9gu/0cVNlseWi0vDxUPLrbrZ6PdGF7fc56EF7aEF7aEF7Zv9cm90ccutyi236mZT3Btd3FKrbrbAvcnFY4vlsYWepeHiPDSQJLfU5/nj4G90ccOtyi3JILckg9ySDHJLfc4t9Tm31OfcUp9zS33OLfU55xaS5BaS5CaSNLQYeWxh2NjCsLGFYWMLw1rydh5bGNaS9PPYwrCxhWGlhSSlhSSlhSSlhSSl5T43tc+l5T6XpvvckEnGVQNJxlXDfR5XDfd5XLXE3LXE3LXE3LXE3DWQZAwNFWMMDRVjDA0VYwwtMbdMOIyx5T7Hlvscm2JuqRippWKkFrRTC9qpBe2W6Z0xtdSq1FKrWiaWxpaJpbFlYmlsGcWOLaPYsWVWamwZAo9DC8NahsBjyxB4bBnFji2j2LFlLDm2jCXHlmHCWFrcbul7ji19z7Gl7zm29D1LS9+zrBoYVlo6rqWl41pWDSQpqwaSzG9/e6OLW+5zS3+7dC33uaWzXlo66/Nb8t7o4oY0VLoWknQNmaS0DFBK18KwvoVhfQvD+haG9S0M61tI0reQpG8hSd9Ckr6FJKGFJKGFJKHlPrcMvkvL4Lu0DL5LbEG7ZeReWlb7S8vgu7SMn0vLELi0jGJLyyi2tIxiS8tYsrSMJUvLoKy0zOmXljn90jKnX1rm9EvLnH5pmdMvLeOq0rIgUFoGZaVhaBRXDZ31uGqYiZsufvP7PF385mjHVcP0znTxm5NkurgF7YatP3HVMMkyXfzmaSiuGpaQ46phnmS6+M1rVVw1pN64akiAcdWQhqaLGxjWNYwlY9cwupkubkgGXcMYY7q44T53DWOM6eKG+tw1jDGmi1tuVWi5VQ0DlOniFno2rEtOF7eQpGF0E7uG0U3sGkY3sWtYl4xdw7pknD+K9EYXt9znhs3f08UtJGkYDsauYS02dg1jyeniFoa19AC7hrXY2LV0H7uGgWjsGpZTY9ewnBq7hiFw7BqGwNPFLSRpWE6dLm4hSUtPv2vYjjxd3MKwhrXY6eIWhrUMULqWAUrXsAo8XdzCsIa9zNPFLQxrWH+eLm5hWMtwsGvYyxy7lrFk17Adebq45Va1jGK7loFo1zCZNl3cwu2WUWzXMJk2XdwEWAM9+4Z9BtPFDfTsG7YKTBc3oN03bBWYLm64VX3L1GXfsM8g9g37DKaLW+5zwz6D6eIWkrTMDfUN+wwWX2Z+o4tbSNKwVWDxEec3uriFJC2TaX3DVoHFx6Lf6OIWkrTMAfYtc4B9yxxg3zIH2LfMAfYtc4B9yxxg3zIH2LfMAfYtc4B9w/aI9RugWy5uYVhoYVjL7GPfMvvYt8w+9i2zj33L7GPfMvvYt8w+9i2zj33LHGDfMgfYt8wB9i3TeP3YUjFaRjd9y+imb9gqEPuWoVHfNDRq2aQQWsZVoWVcFRr2b08XN9zn0DK6CS2d9dDS3w4N+3qni5tibmFYS2c9tPS3Q0uXObR0pULLOkZoWccILesYoWUdI7SsY4SWpYjQshQRWpYiQstSRGhZEAgtCwKhZU4/tMzph5Zp+dAyLR8atmzG0NKhCQ37PWNo2e4UWqaoQ8sUdWjpxIWWTlxo6cSFlk5caOnEhZZOXGzpxMWWTlxs6cTFlpn12NIDjC1zvbGlNxRbekOxZTIttkwsxZZJltgyTxIb3rEUY8vSYmxZWowtS4uxpbmJLXk7tuSw1JJJUksmSS2ZJDU8zjtd3ECS1LLAl1oW+FLLEDi1LPCllgW+1JK3U8saXWoZ9qeWNbrUMuxPLcP+1NJWpZZlttQy7E8ty2ypZc4gtSyzpZZlttTSM0gty2ypZZkttSyzpZZlttTSG0otc0OpZZkttSyzpZZlttTSA0wty2yppfuYWpbZUssyW2pZZksty2ypZZkttSyzpZZlttSyTz+17NNPLWt0qWWNLrWs0aWWffqp4Wnz6eIWkrRMjqeWyfHUMjmeWibHU8vkeGqZ304t89upZX47tcxvp5b57dQyy5xaZplTy+bv1LL5O7XMk6SGlzpPF7fE3DLVkVrmt1PLLHNqmWVOLbPMQ8tc79AyTzK0zJMMLfMkQ8tUx9AyWzG0zFYMLduRh5apjqFlqmNoma0YWnY4DC2zFUPLbMXQMlsxtMxWDC2zFUPLbMXQMlsxtEw4DC0TDkPLnMHQMmcwtMwZDC1zBkPLnMHQMmcwtMwZDC1zBkPLnMHQMmcwtMwZDC3D/qFl2D+0jNxbvs0WW77NNl3ccp9bdrK0fDkrtnz8KrZ8/Gq6uOVWtYwxhpYHTIeWAcrQ8nTq0PJ06tDwsZzp4pb63LJ4PbQsXrd8IW26uIVhLbu0hpax5NCyS2to2aU1tIxih5bV/qFll9bQMn5u+YzedHELw1p2OAwtu7Ryy8g9t+ytyC17K3LLnEFumTPILXMGuWVvRW6ZcMgteytyy2xFbpmtyC2zFblltiK3zFbklo0ZuWWqI7dMdeSWXR25ZQdibpknyS1THbllqiO3DPtzy7A/twz7c8uwP7eMYnPLKDa3jGJzyyg2tyxe55YhcG4ZAueWIXBuGQLnlpXv3LLynVtWvnPLynduWfnOLSvfuWXlO7esfOeWxevcsnjd8tnq2PLZ6uniFpK0LJvnlmXzlg9mx5ZvXseWj83Glg9yxpbP6MWW79HFlg+7xZYPu00Xt8Tc8ibClq/CTRe3kKRlY0bL9+hiy/foposbMsnY8ibCli/hxZYv4U0XtzCsZaJ4bJkoHlsmiseWieKxZaJ4bJkoHlsmiseWieKWzyVOF7cwrGWieGxq6FomiseWieKxZaJ4bJnrbfmyZWz5OGVs+ThlbPk4ZWz5OGVs+ZhdLC0L9qVlwb7la2GxtAz7S8uwv7QM+0vLsL+0DPtbvlMWW75TNl3cQs+WYX9pGfa3fCFturiFJC0DlJYvpE0Xt5CkZdjf8m222PJttunilvvcMqIrLSO60jKiKy0jutIyoistI7rSMqIrLSO60jKiKy0jutIyoistI7rSMqIrLSO60jKiKy0jutIyoistI7rSMigrLeOq0jKuavm+5HRxC9oto5uWby2mVcPi9XTxm9/n6eI3TwbTxW9OkuniNyfJdPGb1+fp4jdn2HTxmzMsrRrWn6eLW+5zw/rzdHHLrWp4tn+6uOU+N2y1ny5uuc8NC/bTxW/eM5gubiFJw4L9dHFLMmgY9k8XtzCsYc5guriFYQ07HNKqYYfDdHELwxrmSaaLWxjWME8yXdxyqxomHFLLF4mni1sAa5gzSC3fQp4ubkG7YeSeWr7CPF3ccqsahsCp5RPOqeUTzmnVMBCdLm4BrGEgOl3cwu2GgWhaNYyrUsvHo1PLV5inixtI0jWsJkwXNzCsa9i/PV3cwLCuYf/2dHEDw7qGLdjTxQ0k6Rq2waSuZRTb8r3Y1PLV1tTy1dbU8tXW1PLV1tTy3cHUtcxWtHwJL7V8CW+6uIGeLV/Cmy5uSEMtn9GbLm5IQy3f4Est3+BLLZ/RSy2f0Ustn9FLLZ/RSy2f0Ustn9FLLZ/RSy2f0Ustn9GbLm7IJC3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy3f4Est3+BLLd/gSy2f0Ustn9FLLV/CSy1fwpsubrnPLbOPLd/gSy3f4Est3+CbLm4hScu8Z8vX/6aLWxjWMhzsW2Zc+5YZ175hr9R0cQvDGvZKTRe3kKRhS0jqWyYQ+4a9FdPFLfRs2B4xXdxynxveEjJd3EKShp3jqeWDnKnlg5yp5YOcqeWzmKnly5bTxQ1ot3wWc7q4CbCGZBBapjpCy1RHaJnqCC1THS3fEZ0ubrnPLXMGLZ8CTS2fAp0ubrlVLXMGoWXOoOUjpNPFLfe5Zc4gtDTuoWG/ZwoN+z2ni1vQbulWhJZuRWh4FC61fJwytXycMrV8LjG1fC4xtXz9b7q4xXJLi9Hy6cAUW9JQbBmUxZZBWWwZlMWWQVlsGZTFlkFZbBlXxYZnUKaLW+5zwzMo08Ut97ll905seAZlurjlPrds/YktW39iy9af2LL1JzY8gzJd3MKwlj5JbOlWxJbZitjwbH+KLYPv2DJ+ji37DFq+5plavi+ZWr4vmVq+qJRaPoqUUktz0/KBoNTygaDp4gaGpZa8nVrydmrJ26klb7d8mii1fJootXyaaLq4IfWmltSbWrZ4pZa83fJRpOnilvvcMpZMLWPJ1PDgYWr5llNKLQPR1DKz3vIVqeniFpK0NO4t369KLd+vSi3fr5oubmFYS5+k5ctZ08UNDBtaekMt3+xKLd/smi5uIEnLN7umixtI0vLBr9Tyza7U8s2u1PLNrunilvvcspowtGxfbPng13Rxy31uWYpo+WZXavlmV2r5Ztd0cUt9btlE2PLBr+nilvvcsomw5WthqeVrYanlg1+p5YNfqeWDX6nlg1+p5YNfqeWDX6nlg1+p5bNbqeWzW6nly1lpaNmZNrTMkwwtD5gOLVMdQ8tUx9AyLT+0zJMMLdPyQ8u0/NAyLT+0TO+0fNgttXzYbbq4hWEtMzQtH3ZLLR92Sy3fZkst32ZLLd9mSy3fZkst32ZLLd9mSy3fZkst32ZLLd9mSy3fZkstn1dLLZ9XSy2fV0stn1dLLZ9XSy2fV0stn1dLLV9ISy1fSEstX0hLLV9ISy1fSEstX0hLLV9ISy1fSEstX0hLLd8pSy3fKUst3ylLLZ8aSy2fGkstnxpLLZ8aSy2fGkstnxpLLZ8amy5uuc8t+z1zywxNbpmhyS2TLC0fOUstX85KLV/OSi0fv0otH79KLd+vSi2foEotn6BKLZ+gSi2foJoubqFnyz7Alu9XpZZPUKWWT1Cllk9QpZavSE0Xt9znlgmH3DLhkFsmHHLLhENumXDILRMOuWXCIbfsJ8ktsxU5D9OFnx0cfPKHF4X85rfXT6erp//XV395cvE3V8dPnp6dXP3Ng89/3vcxnJ+fv/wzhvI3V8+ePj1bCL7oVqvpv6suh1sPxluPrv7h1pf/vt1SjMJSjEtL94fa0qMGS0laSktLt7vezVTqhanUL03d61Zu+CUZVRJR+QU1SEtDbckxqCxN5aWpo+xnSRI9VUS/X9wsDSthaVhVVWrwsySr1FBVqYeOMUnyDRX5HnR+liT5hpp8wc+S5N5Qce9udqtQwygtjRXLZUJqCUrSfKhofsfvRmVJ81zR/KFf1c2dtNRVlpKfJZnOc5XOH/jdpxykpVBRIriRb5QxjX2dzf0oMcp8NNZNvJ+lIi2VOvNFL0spibo7CWZL8dZDtw5Ski1UWrZQkyU39NIgKtQkWFq67Vah0hCkpbC0dG/0onmSbWFatoVx4p4jfEmaSktTR46WBmlpqOBza6KSbKLSUNH8jl8TlWQTlYaK53fcei1JNlFp2URF1cK3WJI1Klc16q4ferKJSssmakLPL/PJJirlqu7e96u7WdbdXNXdu3LA0WJK1t1c1d0Hjmkiy8qbq8orm/gWS1laylWacLQk00SuW0O/liPLLJHrLOE2ikqjzBJjnSX80BtllhjrLOGX+WSnLy07fenWPT+WjzJLjGFpyZHlsneZlr3LdOuh26xOGmWSGNPS0h3HJDHKJDEOS1OelJBVdxyXlu473ihZdceytHTbL6Yiq25ZVZTwawyLrLqlW1qSI8MG8hVZoUpVoTxbKDmKSqWqUbcd4ZM1qlQ1yrF/VGSFKkNdd/0syVa35OpG+fUui6y6paq6chT15paGlahQk6CqUG6T9MOqk5aqCvXQj+bDKktT1Y3y68gOq1Faqm7UXb9R1NDJO9VVd+quW90dul5aqvoSt916YkMvLfW1JT+ey2WvIZSafW6UiPI+xVXdE3M0JatUrKqUWrdpMSXvVOzrLOu2RDTI1bwh1p0xvyoVB2mpajru+1WpKDNSrDLSPb+Y5ALlkKobdd+t3zKkIC2FOvf5JfQUpamq33LfbRQ6yEXXIdXk82s65KLrkGryOTYdctF1SBX7brt1XAY5eT6kuj10G3MMcvJ8SFVC95tUHOTy7jDUDa9fcyin6Yehyuf3HC3JLDFUWeLI0ZLMEkOVJfyWUwa5IDAM9eDGL/PJ9YBhSHW769fEywWBYRjq3oRfuyuXx4ehyhJ3/VKfXHsYhnpmwjEouSIw5Kry3vdLfXJFYMj1hIHbAu8gZ8+HXBHdcXfBIGfPh5xqU36WJCeWE82D4xzSICeah+VE8+A4AzfIebFhrCwd+XVb5ITBUCr0/BYEspwwyMsJgwm95EXzLCcM8nLCYPAc3eRVL031VVB+CT2vgjQV6qjcKm9eRWkqVqRwy315laSltLTktzUorwZpaahicktIWU4i5eUk0uC4UTbLSaS8qiqv7DQ3kE/OIeWuqrwP/BreLCeRcldVKb9tfbmTNaqratSR343qZIXqqgolJ4BbbpSsUF2qE5LjjZI1qqtqlN/YOneyRnVVjfLbApc7WaO6qkY5zjXnrkhTVRvvtyU397Ly9lXlveN3o3rZ8vZVy3vP0ZLMEn2VJeQSb4slmSX6Okv4MaKXWaKvsoTfglTuZZboqyzhN7jJvUwSfZUk5OCmxZJMEn2VJOSSQEOS6GXN7Ut9n/xuVJBVN9SdZkdLsuqGquo+iG7wBVl1Q1/3JdzmW3KQdTeIuuuIn6y8oa68fm1UkJU3pLon69fEB1mnQlWn/FYfslz8yrEiut/+jCzXvnKsiH7kaEkSPVZE91sSyPKJuRwr8jnu3M9ykS3HesTmZynJG5WqGyUX2Rrgk0tfOdWdCccaJRekcqrulN+sdpbLRDnVA14/nsu1m5zqYahjTLLlTfW8mF8+kms3ebl2k2898Oudy3WOvFznyGqVqIHmcp0jL9c5st5g0BKUpMRyoj6rR1AbLMntdnm53S7fOipu8BWZJUq/tOT43EiW2+1yqe6U39NlWW63y8vtdtlxU26W2+3ycrtd9pzqk9vt8nK7XfbsNMvZ81wqmvttrcpFZr5Slpb8+hKjnKcfV1Xmc5wCGeWU9riqUp/fmGOUk4pjJ+qu25hjlBtBxuVGkNFzjXKUTfy4bOJH9XReS1BFWip1UH6kkGvk41Dh59ebGGUjNS4bqdFxpnmUj2KNy0exRsfmcJSPYo3LR7FGxx2so1xMHpeLyaMeWzeYkk/DjMunYcap4+LWHo7y0ZFxrKLye/fIKNd4x7GqUo674Eb58MO4fPhhdHy6cZS9sbFUd8pvp/soOy5jqdKE3PLUcKNkIz+Wsc59bm1HkcvJZdXXdcqt7ShypaN0dVRuU/VFTjaXvrpTD91mxoqcbC591R46ziwWOTFbwqo25VZ7i5yZLaGqU35TY0XOzJZQ0c9vUaXIidkSqmbKb3hd5LxsCVVC95uvKnJetoSa525NbwmS52GoY3IMKktTuTLllmZLkAkpjHWNctu1U+QzHWX5TEfx7KIXOdtclrPNxXHUW+Rsc1nONhe1etgAn5xtLsvZ5uLaSsnZ5rKcbS6Ozx0W+UhHWT7SMcHneKNklVo+0lEctxGWKKtUHJeW/LaFlyhrVKxqlN+iSkmy4U2rpSW/3bJFrj+UVFVdxw56kQsQJVV11+9lCUU+e1OWz94Ux2cSilzpKKnKEn6P6BU541JSlST8XnRS5IRLSUPdcPiRTyaJVCUJxzcrFjm3U1LdGrrNIhW5qFKGVd1E+QUln4gpQ115/Tghn4gpQ1133WaRilwpKkNVd/2eOixypagMVd31exqryNm+MtQNvF+zKx+IKUNVd++5LRQV+TxMGaq66zhdVeQUZhnGuivmZ0lmiaHKEg/8soR88qbkKkt4jm7kozclV1nC7+mHImeAS66yhN9cc5EzwCVXWUJuRGqxJLNErscBfh1Z+TRRyVWWkLOyLZZklshVlpC7cltoLrNEznVj6Jcl5CNSJY91jfKzJLNErrOE342S7+Iq46runftZkklirJKE3/ugi3wXVxnrJOHXvZTLHGUUQ3g/8slXZJWxqrt+L+4u8r1VZaxoLtfIWyxJmo+lbuHdbpRcUCmlnj9y7DLLDRqlVKnP7y3XRW7QKKVKfZ6T53LxppQ69fklJLlDo5R6GNXEvjhbiivxfONaUPG8abagttRLS319o/wsRWmpSkhtO41rS0laqvNRU/+otpSlpXqir6nLXFsq0lKpGw6/GyWeBFwLKlNNHaTakqR5J+bfmlbzalOS513Fc7W9pcVUkKZC3XQ43ipZp7q6kW9akKotyTrVpTpPNLVStalBmqqH8U17xmpLsvp2ooPueKdGaapupfyyXyczRVe3Un7Zr5eJoq8SRdszZrUlWXv7qvbe9UOvl5W3ryuvXxvfy7rb13W3qd9SW5J1t69nz/2ybC+rbj/Ugw6/LNvLutvnOqE7ckLW3b6qu7cd75Ssu31Vd9s2GFSWgqy7YVU3Un75PMhWfrG7ZVx5Vqkg08Rid8tkqW0tubYk08Rid8vakl8fKcg0sdjdMllq3DRRm5J5YrG9ZX2jHJvDIBPFYn/LZKptRaq2JPPEYnvLZKnxAdHalMwTi/0t61vVNI1UW5J5YrG9ZbIktzI0wBdlnlg8tbmhn1+ajTJPxCpPtL2HtbYk80Tsa6L7Jb8oE0WsEkXbe55qSzJRxCpROPaRoswTscoT8j2iLZZkmoh1mvCru1GmiVilibZdXLUlmSVilSXa1kNrSzJLxCpLtL39prKUZJZIVZZoe6KjtiSTRKo7E35NfJL1KVX1qW1Tbm1J1qdU1ae2t1PWlmR9SkOdzf1ybJLkS3UT5dhwFJljF19JGDvP2dIi6+5irnmy5DjiLRK+xVzzZKntZWZLS514GnAtWFpqe39LbamTlrqlpbaH2WpLvbTULy3d9WvgO/GE41pQUcIt9XVyAr1bTKCvLbnV3U5OoHeLCfSNJbe624mX9q0FFc+bNnvWloq0JGqUX5WSk81dV9FPNvENQclpxa6rEpLjDGYnZ/u6vsoTbRuAa0syT/RVnmh7mK22JG9UX98oP57LecWur9JE2ztsa0syTfRVmpB7ZRvIJ2f7ur6qu45D605OwXV9xXO/ydJOzhd0y/mCrnEzQ2VpkBVqsVd27B3nxbpBVqjFVtnJkt+8WDfICrXYKjtZanu4trYkK9Riq+w6Jr+1w26QNWqxV3Yy1bipuTYlq9RiC+tkSrKvJSjJ88UW1slS47MPtSlZeRdbWCdTba/LrS3JNn6xhXUdlF/lzZLouSJ62wcIa0uS6FkQ3Y99Wfb7Fpsw1xnJrznMkue55rlfc5gl+XJFvraHVGpLkny5Il/b5rTKUpHkKxX5HIdscmjdlYp8ba/jqi3JJFvqJOvXRhXJ8lKz3C9JFMnyUrG87Qn82pLM5qXK5o5DazkF0pWqPqkna1tMyQpV6grl10T1chKkX1V9sbZXNdeWOmmp6ov5DaN6OQnSr6os4Tcp28s5kH5VZYm2D6LWlqK0FGv2uU2C9HISpF/VacKP6L2cBOlXQ90Xc0uzvdyx2K+qRNH2oEBtqUhLVe31mz/v5Yaxvqvga3uwu7IkR/F9XzeHflVKjuL75Sg+OO5h7eX2jH65PSOo5ygbaC4X/fvlon9wzedyLbmPVVB+I95eLvD2ywXe4NlyjNLSWFlqe/NSbWmUlsalpbaX/NeWZJJYPGaxvk9+NC+SEaWmuR/3ZJ+5L9V98lsk6mWvr1/2+oLjeC3IvkRY9iVi41OotaUkLaWlJb8p2SBb3bBsdaPjlGyQjW5YNrrRkeVBfC9oLVhaavvARG2pSEulYoTbjGyQzyOExfMIa/QcLXXSUlfF5NZqBLlCFLq6Prlt9AzyaYSweBphjZ5fNzbIpxHC4mmETZJwm6kKsssXuqrutr0TurYk625X112/TkuQi3mhqyqv316xIJ8RCF1Vef26R0Gu5YW+qlJyYqLFkqxSfV2l/BoO2TcPfcVz+XqnBvLJnfuhrxrDxscoa1OyRvVVjfJbuQly537oc91GOQYla1Rf1Si/9YAgd+6HvqpR8uUZDZbkzv0QqubQb7dTkCPDEKq66zeCD3LjfghV3ZUd2QbyyY37IVTNYeOLAWtTMk2EKk20vXqktiTTRKjSxF1HSsgsEaos0fYNpNqSzBKhyhKNj8XXpmSWCFWWaHs3W21JZolQZQm55N8An9y3H2KVJRx3wAU51xJilSb8HjgMct9+iHWa8LtRclYnxCpN+D2IFeS2/RBjnZD8+i1y336IdZpwvFEyTcS6M+HXwst9+yFWaaLt1S21JZklYp0lHC3JLBGrLOH3rHCQ+/ZDElnCz5KsuqmquvJDIC2WZNVNddV1jElW3VRV3Tt+ExPyCYGQqporpxQb0pF8QiCkoR5xOJJPVt1UVd22TybXlmTVTVXVbXxdX21K1t1U1d22b31VluT2wTCIuuvXaZb7B8NQDwQcR2xyA2EYqjzR9imB2pLME4MYCTgGJRPFUCWKtlcU1ZZknRpyzXQ/+GSdGkSd8utgyl19YajH1n4tr9zVF3LFPr+10CB39YUcavj82Cd3wIVc3Sm/J1SC3AEXcn2j/NLsKJPfWCc/v9w3ytw3VrlPPcfWYkpyYgx1RvfjxCjzxLjIE0l1mxuCKvJOLb5MNFlyfOQwyIXXsFx4Ta5jXrmzLyx39iXHfdpBboILy01wyXN8GOUmuLiqbpXf+DDKTXBxuQkuOS69RrlhLC43jKXGN2rXlrK0VFUpv+fvo1zkjctF3uSYZqNc5I2rinx+z6pHufQau4oRfpOlUS6Ixq7KEn4rUlGuh8au4p7fAz5Rvp0tLt7OtmaEo6VBWqoTn+N9kvWpq5sot05zlKuhsatY3va5lsqSfIYy9nVj6Mc9uRoal6uhyXHlK8rV0NjXLPe7T3I1NPYVy+WjHG/e6Ea5GBr7oe5JuPXEolwNjX1Fc7+nRqJcDI191Wz47W6JcjE09lWF8tupGOUSZQxVs9H2SazakqxQoapQjnMgUS4cxlDVKL9ny6JcOIyhbjf8BjdRrhzGUDccnqZklQpVlXKcmohy6TCGsWaFnyVZp0LdFfNLs3LpMMaqkfJ7/2qUK4cxdvXYxg8+uXQYY1V9277PXFuS/ctY9S/9NkZGuXQYYz22cYxJVt5YVV65xNtCCVl3Y67vk2NQskbFqkb5PYMa5TJbTHWN8huwyddjxVQPrP3aeLmgF1PdwfS7T/Kp5Lh8KnlQG/sayCefSo7Lp5IH14ZDTsrG5aTs4Li3KspJ2biclB0c3yQV5aRsXE7KDo5rvFFOysblpOzg+GhjHCXNFx9sWd8nv66snP2Ny9nfwfFLDHGU7cbi0zBr9PxaqFFW3bGqun6PWsdRVt2xqrp+j49HOXUel1Png+PjPVE+hhXHOkf4zbXIx7DiWOUIv1eLRbkcEMuqzkZ+NJdPfMVSJwk/Ssj3McRSJQm/931FucQRS5UkHJ8UjvKFDLFUWcLvUaIoX8gQS5Ul/PZfRrlsE0uVJRy7fPLRvFjqLOG2wBblCxliGeu663ijZJoodVfCreFIcikqrao04fdId5JLUWlVZQm/N5En+QxlWvV1hXLrXSb5Qoa0qtKE3860JNfX0qrKEn57FZN8MDStqizhN0uf5IOhaTXUSdbPUpaWqizht7MlyTXDtKqyhN+sRJJrhmlVJQm/txkn+WBo6qok4fi8YZLLk6mrsoT8tHCLJZkluipLtH2jr7Ykk0QnBhyO8Mks0dVZwm1XZJIroamrs4TfdpMkl0JTVw85/PpiSa6Fpq7KE34P+CT5DGrqqjwhV5Jb4JN5oit17fWDTy67pr4edLhNFyT5uGvqqzzR9nHh2pKsvX1Vex0nSpNc4U19VXv9Vq2TXHdNfV2l/NKsXHZNfV2j3MbWSa7mpeVqXnacL0hyNS8tV/Oy466qJBfz0nIxL3v2kORaXlqu5WXHZa8k123Sct0mO75UMcl1mxQrRshJ+oYkIddtUqzuk2cTLxduUqxulN/2tyQfxErLB7Gy59hQrhCl5QpRdnx/Y5IrRGm5QpQd95MmuUKUlitE2XO6IMklorRcIsqOG8+TfOYrparu+r00KMlnvlKq6q7fykOSz3ylVGVzx8kW+cxXSlWW8HvYNclHvlKqkoTj7G+Sz3ylVGcJR0rILJGqLOG3ByTJR77SUGUJv1c/JPkYVhqqquu39JDkY1hpqKqu38fEknwKKw11s+uXYwdZdYeq6noObeQL49NQ1V2/Ge0kHy1LQ93A+zW78tGyNFRVV665ttwoWXWHqup6DqKybHdz1e46DqLkalQqdbPrRolBziANixmkEh33rg4yxw5zjl1NtckxJpmPhjkfbSz5xSQr1DBXqLUl2ZV4c5oPskINc4VaW3IcBgyyRg1DdaP8XnU4ZNEYToKlJb/1yUFW3WGuuhtLfuSTz58O8/OnG/TcFqMG+fzpkEONnh8lcpSm4tKU39LDIHeKDfNOsU1Qbs3uIHeKDfNOsU2NcoxJZolcZwm3XQyD3JI25CpL+PWPBrklbch1kvB7J+Ug96QN46o25bYeMMhNacNYpQnH5ahB7kobxipP+E22DHJX2jBWecJvnnSQu9KGsUoTfivJg9yVNoxVmvB7hm2Qu9KGsUoTfk8sDXJX2jDWacKT5zJPjFWecHwz2yD3pQ1jnSj80qzclzaUOk/4pVm5LW0odW/Cr4MpBwJDqbOEXw9JbksbSpUl/HbJDnJX2lCqLOE3Sz/IXWlDSTXN/SzJuluquus3zTzIvWJDqaqu3x6aLLdV5VXFcr+dv1luq8qriuVy9rLFUpCWapa7cS/LXVV5VbPcrS3Mchk594uYPF/7m+WjcjksuJc9vxaa5bNyOZSlKb90lOUTbDl2VVBuDXyWS3k5DktLd/1a3SyX8nLMS1NyvNsSlORErDihXszWYkpyIpbalB9+cjEvp9XSlN+zhlku5uVU0c/vmZEs1/Jy6iv6+bUcci0vp1Ch5/asYZZreTnFpSXHd0FnuZiXU6rY59a9zHIxL6cqT/i93iTLFbac6srraEnW3VTVXb+58yxX2PJQVV2/5d0s36mYh6rq+nX6slzLy0NVdf02GWc5d56HULdRfpZk1R3qquvXwssFtjxU9cnvQc0s1wPycj1gdHyXRZbrAXm5HjDqvkSLKVl1l+sBo+d2kywXBPJyQaB4Zj65IJCXCwLF8fGKLBcE8nJBoKgZ7YYbJRcE8nJBoOjvqbcEJSvvckGgeCYkuSCQlwsCxfEdUlkuCOTlgkBxfN4rywWBvFwQKJ7fA89yRSAvVwSK4xJblisCebkiUBynZLNcEMhjlSXkVvqGGyXXA/JYZQnHV2lnuR6QxypNyLfntQQl08RiPaA3mo6WoGSaWCwITKYcK6+cps+LafrQOW7IzXLuPC/mzidL6m2eLaZklVrMna+Dcmx55URzXkw0T6b8Hr7PcqI5Lyaa1/j5cUJONOfFRPMaPrce5iifdR3nZ13XluSU9ptTYpSPYY1dFZPjHMgoH44au6pK+b1jYpTPRo3zs1EbS240H+WzUWNXVV75nHrLjSrSUlV31Vxpgyn5xNI4P7H0PE34cUK+k3LsqzTh947XUc5qj32VJvweERjls1FjX1Upv6nmUT4bNfZVjfL7XMYon40a+1w3Un5VSr4pcuxLbcotKPnZvDHUWdavNzbKl1KOoapSfq/7GuVLKcdQ1yi/jCS/mzeGUDcdbh2XUT4wN4aqSjl+H3KUT8yN8xNzmzzrNmob5RNzYxjqoPzSrHxkbgxV9ZWPAbYEJRvEUDeInkHJRBHqROGXZ+XrL8coEoWfJZknYpUn/PZWjfLtl2OsO+h+lTfKPBFDjZ7b1OIoH6McY6w7SX7sk1/OG2Oqs5+fJZknYpUn5IOALZZkmoi5zkh+lmSaiPWg122+b5SrlGOq20O3gfwoVynHFOpus1+Vkit6Y6qHUo49F/l83pgqTvg9czjKxcMx1ZxwmzIY5eLhmErdSLlZkgtt42KhLfSebZRcaBsXC22TJb8Jl1E+yjYOaWnJb6fxKBfaxsVCW9AP3jRUKLlQNC4WioLrRu1Rrt+MuaLEkdua1CjXb8ZcUcLvZdCjXFQZc0UJx+GNXFQZc0UJvxW9US6qjItFlfV98mui5JrKuFhTWVcotw13o1xTGRdrKmtLjk2UnEAfx6E25cc+uVF7LFXlldun3zyoIjeWllVVd9U4tMVUkKZE5XXLSEVuLS2LraXr2utWp4p8YV9ZVXnC76UMRb6wr6wq+snNxi03KktLizwRPB8nKvKNfWU1Lk057pct8pV9ZVUqU27TSEW+R6903dKSY1+2yBfpla5fmvIbCxT5GHTpQk0Kt0Rb5ApOWa7gBM8VnCJ3R5ZUmfJ7xLHIUUdJVaXye1agyLFASaWGz+1GyY2EZbGRcI2e442SOwnLUNUpv9d+FfngXFk8OLeGr2mlMs2W0kqsia4FsyXjNY6vF1RtqpOmuqWptnmQ2lIvLfVLS203qrYUpKVQwddUd2tLSVpKS0tt7+6uLQ3S0rC01PbkV20pS0t5aUnOTbTQfJSWxqWlxnXe2lSRpkpdo5oSUmWqk5W3qypv24PktSVZo7q+rrt+lbeTVaqrqlTbWLS2FKWluLQk3zXcAp+svF1VeRtfx1WbkrW3q2pv27CjtiRrb1fV3rYZq9qSrL1dVXvb5rVrS7LydlXlbXvMrLLUy7rbr2pK+MHXy4Z3ufMkq4fjWyzJNLHceNL6BERtSWaJ5cYT/ehciyWZJZYbT1pfrlhbklmiT1VMjpZkklhuccmNu0FqSzJJLLe45Na5ndqUzBL9WNHcseHtZZpY7qZpfRywshRkmljuptHPvbZYklkiVFlCPmvRcKOCzBKhr2+UX+oLMk2EOk349ZCCTBOhShONEwa1KZknQpUn2r5cUFuSeSKIPNE04VKbkoki5JoUjnVKJopQJYq2t9TXlmSeCFWeaJvvqyxFmSfiqs5IfvBFmShilSjaZpFqSzJRxLo74WhJ5olY5Ym2Tei1JZknYqx57mdJpolYpQn5bosW8sm6G6u66znkjbLuxqruyq1cDUElWaPSqs59fkElWaNSVaPkSKolKFmjUl93x/ya3iSrVKqrlF/uS7JKpapKNW4zqE3JOpWqOiWX5FuCkk1vqprexi2ftSlZfVNVfds+J1tbkrU31bXX0ZJseVPV8sqnDxssDTJPDHWecLQk08RQpYm2bU+1JZkmhjpN+GXZQWaJIdT9Fr+ENMg0McS6lfKzJLPEUGWJtlet15Zk1R2qqtv2Tq7akqy6Q1V15ZfFGiyNknxjTT6/CjXKHDtWObbtZTG1JZmOxlJnczdLRaajUqWjtlfz1pZkfSp1ffKbgiuyPpW6PvndpyIZUSpGtG1Pqy3Jmluqmus4p11kzS31vJijJcnyUrPcrYHq5LJ1t6pY7rfu2slV625VT4u5zV92YsPYWvDSUt+X1o9Z1KYGaWpYmmp7eXxtaZSWxqUlT0oUaaksLfmtCHSdpMS8t2ptyW+Vo5PLrt287Lq25EhzuerazauuG/TcUmwnV127rqJ52ys3a0tJWkpLS34TVZ1cdO26qj75Lbp2cim066r61PaAWW1J1qeuqk+Oa9adXDbs+ormsr/cEJRcNuz6iuaOw41Orht2fcVzvwa+k+uGXV/xXH4LtcWS5Hlf8bztobnaUpaWcmXJ8T7JGtVXNcpvnqqTq4ZdX7dQjjVKLht287LhhhJ+6VyO1rqxqrty92oDfKOsu2NVd9WEbIspWXfHWN8pt8m3bpT0G2v6+XFCjkG7sU7obntbOjkG7cqqzrJ+neYi+2Klq++UX9tRJP9KqPvnfvyTQ+uu1G2HXzPVy/FNP49v+rByXAzt5a7cft6Vu7bkt1+sl0OBfjEUmCy1PeFYWZId9L6rYmp8dqQ2laSptDTlN4zvZSepX3SSJkt+k1W9bOX7RSs/WfIbSfVylr6fZ+nXluSkYgMl5Cx9P1QVyjH59XKavh/qGuXIPjlP3w8VJ9qeMastRWkpVpXXMSZZo4aqRj3wSxNy8ryfJ8837HPM54PMfUOV++TSTYOlLKtUrqqU39a0Pssqles2ym05qs+yRmWRz906fn2WNSpXNeq+442SNSrHOiP5WZI1Klc1So5EW27UIC0NdZbwq1FZNlK5aqTavpdWW5JpIldpwm/Rus8yS+RFlugcNzX3o8wS42ppqe2Fw7UlmSXGrorJr92VY95+MebdvOnE0ZSsu4uR6CYot0FHL1dD+3Go7pRjb2KUVWrMS1N+z/j0cvGrL+PSkt9+sSCXpMKqIrrj1GKQTx2GVahMudXeIBeKwqrihN9qcpALEKGrYvLbMxHkAkToqirV9mKLypLcWBVSrmuUW28iyP1OIVUJve39N5Ul2WsOy15z5ziMD3JzSxhynWb9LI3S0iJL9I4bQYLsNIdlp7n1s921JUnzZa8vNH47o7YkGbHs9QW1dthQoWSvLyx7fcFzbB1kry8se33R83mOILt9Ydnti45DtiC7fWHZ7YuOa+RBdvvCstuXHPeUBtkZC8vOWHLcshNkXyws+2JJ7V5tIN8oa9SYqpj8umJB9vrCsteXPPsSstMXlp2+5LjwH+SaSlisqUyW2r5sV1mSKx2hVDR3nNgJcqUjLFY61uzzy+dF1qhS1SjHrphcUgmLJZX1jfIjn1xRCaWqu23fs6gtybpb6rrriJ6suqWqun4r10FuVgylqrqOj9YGOWALpaq7bW9GrS3JxrCUihJu44Aoh4ZxVTeGfuk8yuW8uKqyhN8Th1Eu58VVX3PCMaggTdVpwu2B1yj3YMZVnSb88nmUL/SJqypP+G2ciHIQH1dVnnCcroryjT5xVSUKtSLVYmqUpupE4Xir5Ht2YlfVX785zCi3lsauqr7yeY4G+OTKdez6Oic5wierbxdqUvhZknWqq+qU3ypllDs+Y1fVqbbv19aWZJXqqir10JF8skZ1dbfZbSQV5d7S2FVNr9+mkyjffRP7ukL5cU9uYo19XaH87pPcnxH7qj4dOcYkm8O+ag5vu3XGotwJEvuK5X4v6ohyv2fsxxo9txWVKDd8xr6muWMXSW74jEF0Mf0syRoVqhrl91KGKN8TE0M9DnWb047yNTExVDXKb14sytfExFDPIfllCfmWmBhS3ZX1syTbwlC1hX7zv1G+uSWGuoVy7LTIV7fEUNVdv+ddo3x1S4xV1fXbvh/lm1tirKeQ/AaH8s0tMfZ1s+s2YRCjrLuxqrt+W92jfHVLjLFO537wSfLFemrCseGQrzmJaVVXKT9LkhOpnqj3q1Hy3RkxVQmp7TX1tSXZbUlivOuXkAYZ1LAIavAcB8h117hcdx3UV5pbLMl8vlx3HTzrrtwWGZfbIgfH1dAoV3jjcoV3uHXkF5PcFhmX2yIHx+1iUW6LjMttkUPjZ9lqSzKbL3dFDo0fiqwtyWy+XB8f9KxYS1CyK7ZcIB88O31ygTzmKknIxYcW+GSSyLmmhF8LL1etY67qruOsjly1jmNVd+WbFVssybo7VnXX72X4UT42F8dQ89zPkqxRY1Wj/N5fG+X6eByrCuW3mhfl8ngcqwrV9int2pKsUGNVofyerY1yeTyOVavrt98kykcO41i3un5toVyIj6WquY7dWLkOH0tVcx/4tbpyHT6WqtWV785oSOZydTyWqubedUzmcnk8lqrq+j3yFeXyeCx11fVLEnJ5PJaq6vq9vzHK1fFYqqrrOXkpl8djqequ46J1kuvjaVU3u26cSHLROq2qKiUntN8cviTXrNMq1AnJEb4oTVW112/lIck167Sq2123nliSa9ZpVVVev1nmJJes06ruyLr1WpJcsU6rqvLKN4K0kK9IS3W765cmklwbT11dd/0ewE9yHTl1oY7Kj37yGYHUVVXKb5t2kkvWqauqlN93nZJcSE5d3cF0jEkSvas7mH6VVy4kp64eGvpVXvm9ltRXPPd7XDPJJevU19M6bsspSS4kp160UX4VSq4kpz7WHRc/S7JC9XUb5Uc++Tak1NcdTLfRTZJr1qmvu32ONJcVqq8qlN/DrkmuI6dQ09yte57kSmgK9ZDDbVNVkus2KYo+s1+7K9dtUhIDXkdTstOcqk6z42pUki+NT6nOSH6toXyTe0oLUmTHV/YluUaUlmtE2bPdla9XT2msYnJ77iHJxw3T8nFD42stDabkm1vS8s0trm8IT/LVLWn56hb9cYkWS7JKLd/cor/N0WJJ1qjli1s8P5SW5IOhaahqlN/W8ySXQtNyKTS7Dm/kWmga6srryT5Zp4aqTskeZkNQcoky5apKOU73JblGmXJVpfze/5vkGmXKVZXyW6NMco0y5VDD52dJ9pqXa5TZcxgqlyhTriqv31pykkuUKVeVVz4Z2kJzWXdzrlsOvzQhH+FNeazThKMpmSZynSYcM5JcDk1jlSfmxfiDg88ODj75wwuLP/zFycXTs+Or69NHTZK/+Luz0y8vHp+cP768ePT19Ylz8UiQvC+SP/vXT548Oz85fHRydrYXDu2b5Af/1fHVo2dnx5cvjj/5u1/eeXUa+UAg/UDCeItwIEKECBEiRIgQIUKECNHbFi2GoDv/3DJG/dYzYD84urpenrEo+hWqNxtiOcT1WgG8Vsn2yd+u9G8bqizNt/RvZ/Nb823fBuhIvnXtYh79A5XsTfXcG0f2OMRXZ9p9C3dv7vE7MfvhxPZeUerjkXxcd3pvXHudnnHjyOtPfnHy7PLi/M1Or49ea0jyrYNpGWy+ZnD+49rXG7Z8u4HUawb1JlF/GxNvHEpduDzRO7qWu/UOTL4aDEYuSN4rye4mwOOMBk9eT/zabcNbjP9NYHm9onbN6737KeRvv3XpHViVcOxNr1Pu/dp1/CZl7lui8fHvNaFfdhW277fbm6HDvqP/Npj8rrjucc1erNLtTVJ3ETebe3fNfFNzp8n17Yt7rzLNt4r9w5DsTROC5AO6dXuR8vfUrX0VARc4I9oX0Stmhd94quaNy9gLRBAhQoQIESJEiBAh+jai+jUpf/7l2enFb9bDr4snx8iQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOGDBkyZMiQIUOG7DuQ1e+J83oT/j/61cmXe/xFIyRI3raEKvDtwXmtw32LBTK8s1vPIYcccsghhxxyyOFHeLgPo4m9Ded9gX1/bL0NeiDZk9HlnheL5L2+EXvlzJ6H+b6n2e/Q4Xdm+sOL8b2j2ccj+Tjv+F65tycjCw455JDDxsN9S/VIkOwD5/eken4Y84H7ebhXvdO3TeA9r93vrJD3qHF8Z868m3rRmKzeDR8+kqbhIz983Xu65xXtvUv+hIlk/2/f3qajD+DwIwwZfDjkkEMOOeSQQw45fN8P6+fgv3/711983nPAAQcccMABBxxwwAEHHHDAAQcccMABBxxwwAEHHHDAAQd7cNBxwAEHHHDAAQcccMABB3t6EDnggAMOOOCAAw444ICDtoODg0/Xz7Mc/T//2z/5h28ODt6n398dvCe/3/C7T7+/P+B3L36/+W5+vz74wH+/4Zfft/f77w8+kN9v3o/f//UBv9XvN2/n939zwC+/xu83/L6L3//6gN+9+v3m4/z9372I/z/l91v9/s9f/P7km/fr968Ptvx+825+bx3Uv3/zDb9v83d1sKe/37yb3/5gT3+/4fdD+P0/fGS//83Be/77zX78/h8PxO839u//6WDL7zf8fgy//+cDfj+K32/ev9+D+r2QP/g350+Pz89PHnPMMcccc/xeH//Fz08fXV58eXZ6/JM7d2P6a8SIESNGjBgxYsSIESNGjPhjEv/46Or68uLR19cnVz95eO+XR3eOfo4KFSpUqFChQoXqu1D96Bcnzy4vzq9+8uuvvu4QIkSIECFChAgRIkSIECFChO+P8K/+7uz0y4vHJ+ePX476H9x5qduTeQdUqFC9PdWrUgA6dO9C507q/dnPsPfiNwP4s3vnjy+uvzqZSjv7ye3P7yBHjhw58g9Lvj/t1N6L96lDhW4/dHsyxkOFChUqVO9e9V2/aIBjjjnmmBebIEaMGDFixIgRI0aMGLGzuH5v7/d//ezJxSUHHHDAAQcccPBRHPzpL08uT59+dXL5NQIECBAgQIAAAYIPV/Dd9zs54IADDjjggIN3drAvHRAECBAg+PAF333O5+DdNqnfvUv7fLAv9fL9EXz394wDDjjggAMOOOCAAw7e6wO69hxwwAEHHHDwwR0cHHy6frbtb//Tg82/rb8HO/T8vtnv4Xds/wcvfv/jF78Hb8evf/Xff8fxHm75/Rc79K2/b1r+ruu8/X5bOPwL8etdfqs/2/zbdZ7XdW/r923H+13j8bbLf9d+HYrf7zqObb//8Y7fA/H7Mv/+J+I8ebwtb39be9vsbDtfyrddt82P/+QNr9tWzjZ8v+v+Sevvt43jYE/89Y7rbV3vZefwLfnzpuW97nVv2877Wg/fdXxe9WHb77dtf/7qW+pf/h68pv1d7eEu+6/bnr5sHw63yF/X/i6/tl0v/ZX2/0rI39SPb4uHV3nS7+c/N+PVd/X7sh58W/m+/P7twYvfbf0rcSzjutHL35fXH4rfb2tPyrfZk/04ae/l77Z++sEW+bZfaXfH/ZW4Kf1PXvz+oP5V5/2zV5fzre/3jnK2+XtzH/7Zt4trpx+vef1LnHbWtx/Yv1L/bXGRdpWdQ+ffXTzd9vsvdshlOdvqnzzvxbGa/zuo5W73+fD1zn9X/NvXX6843pivchzyrn+/a/vvm1/ev7I/dyB+d80vfdf+78u8Nr8f9i882+vfg/odsd97ML786/7w4q9Pbnf9yz/vdauX+tvdrL8RHuWby8tNkS8L+t7DWdbdFBNe/nX35bWfHN0Y/N6dmxMf3hT9MN0Uc3PeUZi9ubnk/nxxvPnrJsCHsw83Bd57qf3k/lzM0c1f925cvLNwsagIvnd31t5Yvn9T9t0bwD55sDBzg8S9oEO9c4Pi3RmxG8v3jFIe9rOznb7k/s2Jt4uG6QbZpYe3oxHfjfZBVih+7+ENiR4ubssCiJty7t6Uffsm0tv6Xk0MNHg3RXhDwTs3Rd6/KfLeLOtn27NDM8qL+39zze35tt5gtrjpN+7eu/nraKb8TQi3+9nvRTQ3ft9dmL4R3r8x/eCG4IvaMTNzvofzHT6aaT1TJhno3V1ielP4vbnCzXd2vtujAmWiypwdbiwe3Zx4Q5BFBVhgP1fhBVfmlHFnjubmr4cGEjMhDfBmSlWVYo4qzrHc3K+j+cw7Nz4u7tzMrrn+HN38NVNqkZnm3HND0inqmxJn1tybq9yN7OjG3gPNwkUFmVP0945enrjMa0b2XNzqGZS7RiyLWzRjNteZJYdvsuvS9hzNzJTbRoY4WtaVuRbfRLuoxXP2XWSiwfDtyGLfwwWLrdy3aEEM+Of7+dCCaE6DcxVZQjSjfsPepZdz3r0z356iYlyQbs4WC4DmxuSukWIXDZXVst8gtcy1MzNu4Ftk3Tlbzhn03g0R71lp7EEy0Juz18yVO3N+mVs03dFYttPzPZp7JDPJ5/7K7cUtmF2c79BcN7vZ2RmSmyT3wLj/M7MX+WzuDN0zauQyY+t0v2hCF9VrmQRu8u8yPd+klWUanIk99w3uG12auXmf27k7Rras2lDdIlYpdE4HUce4qDT3jFZ7maDnKnlncSM0B5bG53byru4nLDuRN1jMfJ5bt7tGzp6Txm0jp1SdyBsf7luN7VwDboJetqGjcnaJ/b1goHd70ZWduTLX3aw8X/QxbnLFEvqZDEdGT3C+r+bA4o4xhljmrkV3Y+6WLHpURmM8J4ZFkps7I3MCMfrqc99nbiGW7djKAG/uqhwZjfYDu5ou+mBGonpoJP6ZcYscYVXnGfHFuGu+M3NcczFG1+mekaUtSixSs8GI20YbdbToGN3AsGj15j7LIvHNdXWRKOYu5DycWCSPOarBgGnRbZiHAXO3QI8hlg230bFe9BGXPTpjiLmswfPYbzZtjUXmNvW+0XTNN3rO5UbndsGweVh538qXc5d90VzfXfS++jnWmbVhdtxoAu9ZOXjRk42zWncIF53Wu4vkYY7qDAosMZ/zjInvTO1Fj6bXbiz6NjpLLxLgA6Pezj2WuTs3s/Cu7sYuxi+LSOestugBGl3Kh0ZbXw90rImexTzJfA8XfX5ruDGzar4d1gzNnKPnPLOYZZjPMx2b66bB8Tn1zu323NNajNpnoOYmam5jdH1b8nIxa7FotazZq6RDvm8NyuZbfUe3QdU8x9yj0llvWevnZmlmkTUWujdfvEgFM9rGmGvZg5kT4IPF6M3oND608voi985uGL3Q20aEMy0X/aRohLCYtTLa92U+MBLP3Ogtenpmr3aul0aTfkePOBedzZlFy7GHMcF4w2kbxLlJmHPDLLuvO6323M3M3/mu3LbIaAwElqMso5Ys2vzFfNiRdbNmHizme4wZnQUvFy3vYjpRz2MsWLTIErM9Y4pwMfCe78ZM7ptr5xZi7nYsessPdYdu2Tjd4L1oeM3Z92VO052xqq9idJjm3sai2Rl10A+MardIkjOwRkNbNRHGfPY8dFxMPs4G5zby/iJxrJRFc8j8wBoFLftdmhzLmaJF3bqjxy+Lpv/BYog1M9iYZnhoDFqr4ctc94zp1wdmiuqsaOe/0uyatSIwd7MX9Xkeey9q0n2rHZ39XfQorO78YolCd00WyWnuWtxeduvmeGY+GP3Z+0ZrvhjVj4Zj85Bi7qwseiNGw212gK3+2ALnqibOgxg9216nQz3wqbCwsFr2qo1ezr0FXfT9rqqJMad+W4OwXI/Ts4BLni8qnjn9sMhLi/TXadgWd3Tu3Sxa4Pnuzd3vuUU02sFFK9lpx5YVdKb+PBC/Y8xoLOaflinBGHMss8xiam2Rw2a1McU9J6FFm7Bg5WzbahSsQdSiiV/W38VQb0ZjrmTzX3N9Mia15tHxIlstGpW5y6QXEZdrQNZY1ugHzznC6HcvVz7nec7ZmjURPHeOFpP28+DNqB5zInqga/uC1HcWNVjjsYTLGvDPSfXIuI/LaarZncXiiLWSelvT0p69XNCkmnece1lmBjaSymKOXi/81eN4YzZqzt8PRu3lcs1g7sEHHcNiNmau7fOoZRHhYuSVVITLcb4xBF+CfmTlwLtWTZlnERbr3HOEr27f5tsw47zoFc5NuebuEvtFPpshXdQBo8rdNZqDRSwPdPWqhpaDDnXO7HN/zGL+sm+lV2Gq/RfmQos1STRXnIcWI6ulrTldLGwaic8aXs6DiUX7bOxfWcxlG4PiO4sxpbWOMnP8gTEqNofjule8bPnmfD2np7lLvpwU1RM4u+aT5m7ScsJcD/uXYxJj7nmurotu6aLPa/RvzSmoxV0zOrWLpslg69zVP9Kt46LVWDTr89yBtZViTpkz0+9aOXpB+sWYcdGJMZg11zxrn899YxCx7BzfMRtQvfVjMepZVHBjnWNBt5nzi1ULvcqwTNbzhJEx/7GcmjCHrvcMv+d2ZNF9X1Z0Y15grlpHyYBv0em4b41drV05i3xirEw8MCbFtrTdU2U4+OPvTX/90QvJ9x+dHV9dHWy2C94If/j4+Pr41m8vj5+ciNP/9PLid7fOJ/n6kh+tS/xm+r//8B/+v/9MnPjHV09PHk22f7x2YDpt/fuzF8cHz8/93n94YfZ7L6758aOLs7OTR9cXl188+ur48vjR9cnlS7s3qungf6CcvSjns7mcxxfPfnN28iEERTmUQzkkDcqhHCoF5XwkN32vnKEcyvmQyvkgK9e7Dqqetng5B7Eu7Gcvhden1zdu/OjLk4svjh89Orm6Or04fyH8k6vr4+tnVy+O/uLq2W+enG70Xzw+vn555Wdnx1fXXzx7uhYt5X98/fXTm9Inb8/PT86+eHTx7Pz6pryLZ5ePTr5Y+zaF83LO5s8vLr88Pj+9erKQ/fgm2tOr69NHVwvVPzFUt16p7F+lDK9Sxlcp06uUw6uU+VXKl1Nzf/7kYrq9z86WUP3Vye+v1xd88fTy4vpiIsBC908t3Q0wf3p9/PvTx4vT/+zxydWjy9On1/Pd/9FCdHPhX6znutZlbphy/uUL8V8K8c35StFvU4Rtipeo/9nTs+Pr315cPvni9PFLUB5dnF+vg1xz6IZrL2QnT45Pz6Tw6VcX5y/P/McvhWfHv7m4PJ6qztdS8/jk6fHl9ZOTG9L+5UvN6fl0j66f3dD9L14qjh8/vpw8lx4+Or3+Wjqzrl4vr/+PXgr//enTW08vJtW6tjxWxW9q0I2jn27Aurz4XVWzPls7d/ls7fYXT6YyXqLwl2env7k8vvz6i6uTdaqYb/WnN4pNlbyp2S+lE5OuT758afQHlyfTnZiv/uHL45u7/ldXz54+PTtZO7C+/renE227m+hPzs5urZPDrZl+P3p8evzl+cXV6dVC+On5ycXTdXY5fbSUPp1sTQXfOn28kP75mhwntexHN2EuhD+8nlLYs6Xtz66vzqfjs2dXU81byH/wu7Wj6+Pdifkv5sT85bP57sukvH5b+8EfvXzryvOyvv+yrE9eJs7popcF/KPHJ789fnb28sb+8dW/O30qXHiZ43+wruPPp5Ff5v+bokn5pHxSPimflP9aKb/qQ/+ji021uHqxOvd9kX7/6FII/me/Only8Q8nh+cnX07eT39cnf77k8PfHq9bg8M1IOv++A83Fzz/96Pnf//J/3X+e70GWCXy779M0yfnX57esPr7E5tv7vinEzU23f1bTy9Pb/jxw0l6dev6YuLYS8m6mXkhed44/Q//fwTfxMoPyScA"} -->
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["title"],"name":[1],"type":["chr"],"align":["left"]},{"label":["geo_accession"],"name":[2],"type":["chr"],"align":["left"]},{"label":["status"],"name":[3],"type":["chr"],"align":["left"]},{"label":["submission_date"],"name":[4],"type":["chr"],"align":["left"]},{"label":["last_update_date"],"name":[5],"type":["chr"],"align":["left"]},{"label":["type"],"name":[6],"type":["chr"],"align":["left"]},{"label":["channel_count"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["source_name_ch1"],"name":[8],"type":["chr"],"align":["left"]},{"label":["organism_ch1"],"name":[9],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1"],"name":[10],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.1"],"name":[11],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.2"],"name":[12],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.3"],"name":[13],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.4"],"name":[14],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.5"],"name":[15],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.6"],"name":[16],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.7"],"name":[17],"type":["chr"],"align":["left"]},{"label":["characteristics_ch1.8"],"name":[18],"type":["chr"],"align":["left"]},{"label":["molecule_ch1"],"name":[19],"type":["chr"],"align":["left"]},{"label":["extract_protocol_ch1"],"name":[20],"type":["chr"],"align":["left"]},{"label":["extract_protocol_ch1.1"],"name":[21],"type":["chr"],"align":["left"]},{"label":["taxid_ch1"],"name":[22],"type":["dbl"],"align":["right"]},{"label":["description"],"name":[23],"type":["chr"],"align":["left"]},{"label":["description.1"],"name":[24],"type":["chr"],"align":["left"]},{"label":["data_processing"],"name":[25],"type":["chr"],"align":["left"]},{"label":["data_processing.1"],"name":[26],"type":["chr"],"align":["left"]},{"label":["data_processing.2"],"name":[27],"type":["chr"],"align":["left"]},{"label":["data_processing.3"],"name":[28],"type":["chr"],"align":["left"]},{"label":["data_processing.4"],"name":[29],"type":["chr"],"align":["left"]},{"label":["platform_id"],"name":[30],"type":["chr"],"align":["left"]},{"label":["contact_name"],"name":[31],"type":["chr"],"align":["left"]},{"label":["contact_email"],"name":[32],"type":["chr"],"align":["left"]},{"label":["contact_phone"],"name":[33],"type":["dbl"],"align":["right"]},{"label":["contact_laboratory"],"name":[34],"type":["chr"],"align":["left"]},{"label":["contact_department"],"name":[35],"type":["chr"],"align":["left"]},{"label":["contact_institute"],"name":[36],"type":["chr"],"align":["left"]},{"label":["contact_address"],"name":[37],"type":["chr"],"align":["left"]},{"label":["contact_city"],"name":[38],"type":["chr"],"align":["left"]},{"label":["contact_state"],"name":[39],"type":["chr"],"align":["left"]},{"label":["contact_zip.postal_code"],"name":[40],"type":["dbl"],"align":["right"]},{"label":["contact_country"],"name":[41],"type":["chr"],"align":["left"]},{"label":["data_row_count"],"name":[42],"type":["dbl"],"align":["right"]},{"label":["instrument_model"],"name":[43],"type":["chr"],"align":["left"]},{"label":["library_selection"],"name":[44],"type":["chr"],"align":["left"]},{"label":["library_source"],"name":[45],"type":["chr"],"align":["left"]},{"label":["library_strategy"],"name":[46],"type":["chr"],"align":["left"]},{"label":["relation"],"name":[47],"type":["chr"],"align":["left"]},{"label":["relation.1"],"name":[48],"type":["chr"],"align":["left"]},{"label":["supplementary_file_1"],"name":[49],"type":["chr"],"align":["left"]},{"label":["cell.type.ch1"],"name":[50],"type":["chr"],"align":["left"]},{"label":["diagnosis.ch1"],"name":[51],"type":["chr"],"align":["left"]},{"label":["neoplastic.ch1"],"name":[52],"type":["chr"],"align":["left"]},{"label":["patient.id.ch1"],"name":[53],"type":["chr"],"align":["left"]},{"label":["plate.id.ch1"],"name":[54],"type":["dbl"],"align":["right"]},{"label":["selection.ch1"],"name":[55],"type":["chr"],"align":["left"]},{"label":["tissue.ch1"],"name":[56],"type":["chr"],"align":["left"]},{"label":["tsne.cluster.ch1"],"name":[57],"type":["dbl"],"align":["right"]},{"label":["well.ch1"],"name":[58],"type":["chr"],"align":["left"]}],"data":[{"1":"GBM cell 1","2":"GSM2243439","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421106","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963641","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243439/suppl/GSM2243439_1001000173.G8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G8"},{"1":"GBM cell 6","2":"GSM2243444","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421111","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963646","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243444/suppl/GSM2243444_1001000173.F6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F6"},{"1":"GBM cell 7","2":"GSM2243445","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421105","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963647","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243445/suppl/GSM2243445_1001000173.B12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B12"},{"1":"GBM cell 14","2":"GSM2243452","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421100","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963654","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243452/suppl/GSM2243452_1001000173.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E10"},{"1":"GBM cell 17","2":"GSM2243455","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421249","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963657","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243455/suppl/GSM2243455_1001000173.B1.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"2","58":"B1"},{"1":"GBM cell 18","2":"GSM2243456","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421250","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963658","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243456/suppl/GSM2243456_1001000173.B10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B10"},{"1":"GBM cell 19","2":"GSM2243457","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: A7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421251","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963659","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243457/suppl/GSM2243457_1001000173.A7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A7"},{"1":"GBM cell 21","2":"GSM2243459","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421253","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963661","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243459/suppl/GSM2243459_1001000173.F9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F9"},{"1":"GBM cell 22","2":"GSM2243460","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: G6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.G6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421254","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963662","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243460/suppl/GSM2243460_1001000173.G6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G6"},{"1":"GBM cell 26","2":"GSM2243464","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421245","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963666","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243464/suppl/GSM2243464_1001000173.H9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H9"},{"1":"GBM cell 27","2":"GSM2243465","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: G1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421098","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963667","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243465/suppl/GSM2243465_1001000173.G1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G1"},{"1":"GBM cell 28","2":"GSM2243466","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421258","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963668","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243466/suppl/GSM2243466_1001000173.B3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B3"},{"1":"GBM cell 29","2":"GSM2243467","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421259","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963669","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243467/suppl/GSM2243467_1001000173.D7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D7"},{"1":"GBM cell 30","2":"GSM2243468","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: A12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421115","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963670","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243468/suppl/GSM2243468_1001000173.A12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A12"},{"1":"GBM cell 31","2":"GSM2243469","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421260","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963671","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243469/suppl/GSM2243469_1001000173.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C1"},{"1":"GBM cell 32","2":"GSM2243470","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421261","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963672","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243470/suppl/GSM2243470_1001000173.H7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H7"},{"1":"GBM cell 33","2":"GSM2243471","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: H5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421262","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963673","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243471/suppl/GSM2243471_1001000173.H5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H5"},{"1":"GBM cell 34","2":"GSM2243472","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421112","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963674","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243472/suppl/GSM2243472_1001000173.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G2"},{"1":"GBM cell 35","2":"GSM2243473","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421113","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963675","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243473/suppl/GSM2243473_1001000173.A3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A3"},{"1":"GBM cell 44","2":"GSM2243482","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421242","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963684","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243482/suppl/GSM2243482_1001000173.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B11"},{"1":"GBM cell 46","2":"GSM2243484","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420913","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963686","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243484/suppl/GSM2243484_1001000173.F1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F1"},{"1":"GBM cell 56","2":"GSM2243494","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000173","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000173.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420923","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963696","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243494/suppl/GSM2243494_1001000173.H4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H4"},{"1":"GBM cell 120","2":"GSM2243558","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420987","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963760","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243558/suppl/GSM2243558_1001000174.H8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H8"},{"1":"GBM cell 121","2":"GSM2243559","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420988","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963761","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243559/suppl/GSM2243559_1001000174.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H1"},{"1":"GBM cell 123","2":"GSM2243561","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420991","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963763","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243561/suppl/GSM2243561_1001000174.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B5"},{"1":"GBM cell 125","2":"GSM2243563","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420993","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963765","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243563/suppl/GSM2243563_1001000174.E8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E8"},{"1":"GBM cell 126","2":"GSM2243564","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420994","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963766","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243564/suppl/GSM2243564_1001000174.F11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F11"},{"1":"GBM cell 127","2":"GSM2243565","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: A1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420995","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963767","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243565/suppl/GSM2243565_1001000174.A1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A1"},{"1":"GBM cell 128","2":"GSM2243566","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420990","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963768","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243566/suppl/GSM2243566_1001000174.E7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E7"},{"1":"GBM cell 130","2":"GSM2243568","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420997","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963770","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243568/suppl/GSM2243568_1001000174.C12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C12"},{"1":"GBM cell 131","2":"GSM2243569","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420998","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963771","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243569/suppl/GSM2243569_1001000174.C9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C9"},{"1":"GBM cell 132","2":"GSM2243570","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420999","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963772","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243570/suppl/GSM2243570_1001000174.D7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D7"},{"1":"GBM cell 133","2":"GSM2243571","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421000","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963773","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243571/suppl/GSM2243571_1001000174.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D2"},{"1":"GBM cell 134","2":"GSM2243572","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421001","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963774","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243572/suppl/GSM2243572_1001000174.C4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C4"},{"1":"GBM cell 135","2":"GSM2243573","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421002","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963775","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243573/suppl/GSM2243573_1001000174.F8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F8"},{"1":"GBM cell 136","2":"GSM2243574","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421008","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963776","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243574/suppl/GSM2243574_1001000174.D10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D10"},{"1":"GBM cell 137","2":"GSM2243575","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421009","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963777","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243575/suppl/GSM2243575_1001000174.G11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G11"},{"1":"GBM cell 138","2":"GSM2243576","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421010","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963778","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243576/suppl/GSM2243576_1001000174.A3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A3"},{"1":"GBM cell 139","2":"GSM2243577","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421011","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963779","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243577/suppl/GSM2243577_1001000174.E3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E3"},{"1":"GBM cell 140","2":"GSM2243578","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: H5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421012","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963780","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243578/suppl/GSM2243578_1001000174.H5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H5"},{"1":"GBM cell 141","2":"GSM2243579","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: C6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421013","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963781","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243579/suppl/GSM2243579_1001000174.C6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C6"},{"1":"GBM cell 142","2":"GSM2243580","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421014","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963782","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243580/suppl/GSM2243580_1001000174.D3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D3"},{"1":"GBM cell 143","2":"GSM2243581","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000174","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000174.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421015","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963783","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243581/suppl/GSM2243581_1001000174.D9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D9"},{"1":"GBM cell 144","2":"GSM2243582","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421016","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963784","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243582/suppl/GSM2243582_1001000175.E1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E1"},{"1":"GBM cell 145","2":"GSM2243583","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421017","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963785","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243583/suppl/GSM2243583_1001000175.A3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A3"},{"1":"GBM cell 146","2":"GSM2243584","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421018","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963786","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243584/suppl/GSM2243584_1001000175.H2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H2"},{"1":"GBM cell 147","2":"GSM2243585","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421019","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963787","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243585/suppl/GSM2243585_1001000175.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C11"},{"1":"GBM cell 148","2":"GSM2243586","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421020","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963788","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243586/suppl/GSM2243586_1001000175.D9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D9"},{"1":"GBM cell 150","2":"GSM2243588","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421022","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963790","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243588/suppl/GSM2243588_1001000175.F3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F3"},{"1":"GBM cell 151","2":"GSM2243589","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421023","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963791","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243589/suppl/GSM2243589_1001000175.B9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B9"},{"1":"GBM cell 152","2":"GSM2243590","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421024","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963792","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243590/suppl/GSM2243590_1001000175.H4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H4"},{"1":"GBM cell 153","2":"GSM2243591","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421025","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963793","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243591/suppl/GSM2243591_1001000175.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G2"},{"1":"GBM cell 155","2":"GSM2243593","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421027","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963795","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243593/suppl/GSM2243593_1001000175.G11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G11"},{"1":"GBM cell 156","2":"GSM2243594","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421028","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963796","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243594/suppl/GSM2243594_1001000175.B4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B4"},{"1":"GBM cell 157","2":"GSM2243595","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421029","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963797","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243595/suppl/GSM2243595_1001000175.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D2"},{"1":"GBM cell 158","2":"GSM2243596","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421030","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963798","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243596/suppl/GSM2243596_1001000175.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C1"},{"1":"GBM cell 159","2":"GSM2243597","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421031","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963799","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243597/suppl/GSM2243597_1001000175.G7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G7"},{"1":"GBM cell 160","2":"GSM2243598","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421032","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963800","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243598/suppl/GSM2243598_1001000175.E8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E8"},{"1":"GBM cell 162","2":"GSM2243600","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: H6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421034","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963802","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243600/suppl/GSM2243600_1001000175.H6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H6"},{"1":"GBM cell 163","2":"GSM2243601","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421035","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963803","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243601/suppl/GSM2243601_1001000175.H11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H11"},{"1":"GBM cell 169","2":"GSM2243607","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421041","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963809","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243607/suppl/GSM2243607_1001000175.F8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F8"},{"1":"GBM cell 170","2":"GSM2243608","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421042","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963810","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243608/suppl/GSM2243608_1001000175.D12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D12"},{"1":"GBM cell 172","2":"GSM2243610","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421044","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963812","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243610/suppl/GSM2243610_1001000175.D4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D4"},{"1":"GBM cell 173","2":"GSM2243612","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421045","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963813","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243612/suppl/GSM2243612_1001000175.B6.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"3","58":"B6"},{"1":"GBM cell 178","2":"GSM2243622","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421050","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963818","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243622/suppl/GSM2243622_1001000175.B8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"9","58":"B8"},{"1":"GBM cell 195","2":"GSM2243639","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421067","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963835","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243639/suppl/GSM2243639_1001000175.H1.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"2","58":"H1"},{"1":"GBM cell 196","2":"GSM2243640","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: E11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421323","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963836","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243640/suppl/GSM2243640_1001000175.E11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E11"},{"1":"GBM cell 197","2":"GSM2243641","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421324","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963837","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243641/suppl/GSM2243641_1001000175.B10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B10"},{"1":"GBM cell 198","2":"GSM2243642","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421325","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963838","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243642/suppl/GSM2243642_1001000175.C10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C10"},{"1":"GBM cell 201","2":"GSM2243645","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: C2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421328","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963841","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243645/suppl/GSM2243645_1001000175.C2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C2"},{"1":"GBM cell 202","2":"GSM2243646","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421329","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963842","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243646/suppl/GSM2243646_1001000175.F4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F4"},{"1":"GBM cell 203","2":"GSM2243647","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421330","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963843","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243647/suppl/GSM2243647_1001000175.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E2"},{"1":"GBM cell 208","2":"GSM2243652","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421335","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963848","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243652/suppl/GSM2243652_1001000175.F2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F2"},{"1":"GBM cell 209","2":"GSM2243653","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421336","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963849","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243653/suppl/GSM2243653_1001000175.D11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D11"},{"1":"GBM cell 210","2":"GSM2243654","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421337","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963850","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243654/suppl/GSM2243654_1001000175.F5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F5"},{"1":"GBM cell 211","2":"GSM2243655","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421338","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963851","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243655/suppl/GSM2243655_1001000175.C8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C8"},{"1":"GBM cell 212","2":"GSM2243656","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421339","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963852","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243656/suppl/GSM2243656_1001000175.F12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"1","58":"F12"},{"1":"GBM cell 213","2":"GSM2243657","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421340","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963853","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243657/suppl/GSM2243657_1001000175.B7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B7"},{"1":"GBM cell 214","2":"GSM2243658","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421344","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963854","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243658/suppl/GSM2243658_1001000175.D3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D3"},{"1":"GBM cell 215","2":"GSM2243659","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421345","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963855","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243659/suppl/GSM2243659_1001000175.C9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C9"},{"1":"GBM cell 216","2":"GSM2243660","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421346","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963856","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243660/suppl/GSM2243660_1001000175.D6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D6"},{"1":"GBM cell 217","2":"GSM2243661","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: E6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421347","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963857","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243661/suppl/GSM2243661_1001000175.E6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E6"},{"1":"GBM cell 218","2":"GSM2243662","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: A6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421341","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963858","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243662/suppl/GSM2243662_1001000175.A6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A6"},{"1":"GBM cell 219","2":"GSM2243663","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421342","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963859","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243663/suppl/GSM2243663_1001000175.H8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"H8"},{"1":"GBM cell 220","2":"GSM2243664","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421343","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963860","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243664/suppl/GSM2243664_1001000175.B2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B2"},{"1":"GBM cell 221","2":"GSM2243665","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421348","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963861","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243665/suppl/GSM2243665_1001000175.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"B11"},{"1":"GBM cell 222","2":"GSM2243666","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421349","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963862","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243666/suppl/GSM2243666_1001000175.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"E10"},{"1":"GBM cell 223","2":"GSM2243667","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: D8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421350","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963863","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243667/suppl/GSM2243667_1001000175.D8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"D8"},{"1":"GBM cell 224","2":"GSM2243668","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421351","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963864","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243668/suppl/GSM2243668_1001000175.F10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F10"},{"1":"GBM cell 226","2":"GSM2243670","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421353","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963866","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243670/suppl/GSM2243670_1001000175.F7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"F7"},{"1":"GBM cell 227","2":"GSM2243671","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421354","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963867","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243671/suppl/GSM2243671_1001000175.G3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G3"},{"1":"GBM cell 230","2":"GSM2243674","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: A12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421357","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963870","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243674/suppl/GSM2243674_1001000175.A12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"A12"},{"1":"GBM cell 231","2":"GSM2243675","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000175","12":"well: A1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000175.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421358","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963871","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243675/suppl/GSM2243675_1001000175.A1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A1"},{"1":"GBM cell 234","2":"GSM2243678","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421361","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963874","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243678/suppl/GSM2243678_1001000176.B9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B9"},{"1":"GBM cell 235","2":"GSM2243679","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421362","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963875","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243679/suppl/GSM2243679_1001000176.D9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D9"},{"1":"GBM cell 245","2":"GSM2243689","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421372","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963885","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243689/suppl/GSM2243689_1001000176.A5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"A5"},{"1":"GBM cell 254","2":"GSM2243698","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421381","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963894","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243698/suppl/GSM2243698_1001000176.E3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E3"},{"1":"GBM cell 256","2":"GSM2243700","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421068","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963896","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243700/suppl/GSM2243700_1001000176.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D5"},{"1":"GBM cell 257","2":"GSM2243701","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421069","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963897","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243701/suppl/GSM2243701_1001000176.B10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B10"},{"1":"GBM cell 258","2":"GSM2243702","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421070","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963898","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243702/suppl/GSM2243702_1001000176.D11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D11"},{"1":"GBM cell 259","2":"GSM2243703","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421071","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963899","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243703/suppl/GSM2243703_1001000176.B12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B12"},{"1":"GBM cell 260","2":"GSM2243704","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421072","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963900","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243704/suppl/GSM2243704_1001000176.E7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E7"},{"1":"GBM cell 261","2":"GSM2243705","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421073","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963901","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243705/suppl/GSM2243705_1001000176.H7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H7"},{"1":"GBM cell 262","2":"GSM2243706","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421074","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963902","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243706/suppl/GSM2243706_1001000176.E1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E1"},{"1":"GBM cell 263","2":"GSM2243707","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421075","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963903","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243707/suppl/GSM2243707_1001000176.G8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G8"},{"1":"GBM cell 264","2":"GSM2243708","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421076","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963904","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243708/suppl/GSM2243708_1001000176.D3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D3"},{"1":"GBM cell 266","2":"GSM2243710","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: G10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421078","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963906","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243710/suppl/GSM2243710_1001000176.G10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G10"},{"1":"GBM cell 268","2":"GSM2243712","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421080","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963908","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243712/suppl/GSM2243712_1001000176.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"B3"},{"1":"GBM cell 269","2":"GSM2243713","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421081","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963909","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243713/suppl/GSM2243713_1001000176.A8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A8"},{"1":"GBM cell 270","2":"GSM2243714","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421082","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963910","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243714/suppl/GSM2243714_1001000176.G7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G7"},{"1":"GBM cell 271","2":"GSM2243715","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421083","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963911","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243715/suppl/GSM2243715_1001000176.D10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D10"},{"1":"GBM cell 272","2":"GSM2243716","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421084","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963912","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243716/suppl/GSM2243716_1001000176.F5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F5"},{"1":"GBM cell 273","2":"GSM2243717","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421085","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963913","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243717/suppl/GSM2243717_1001000176.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A3"},{"1":"GBM cell 274","2":"GSM2243718","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421086","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963914","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243718/suppl/GSM2243718_1001000176.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H11"},{"1":"GBM cell 275","2":"GSM2243719","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421087","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963915","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243719/suppl/GSM2243719_1001000176.H9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"H9"},{"1":"GBM cell 276","2":"GSM2243720","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: C5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421088","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963916","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243720/suppl/GSM2243720_1001000176.C5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C5"},{"1":"GBM cell 277","2":"GSM2243721","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421089","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963917","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243721/suppl/GSM2243721_1001000176.E5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E5"},{"1":"GBM cell 278","2":"GSM2243722","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421090","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963918","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243722/suppl/GSM2243722_1001000176.H2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H2"},{"1":"GBM cell 279","2":"GSM2243723","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421091","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963919","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243723/suppl/GSM2243723_1001000176.A1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"A1"},{"1":"GBM cell 280","2":"GSM2243724","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421092","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963920","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243724/suppl/GSM2243724_1001000176.C1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C1"},{"1":"GBM cell 281","2":"GSM2243725","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421093","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963921","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243725/suppl/GSM2243725_1001000176.F7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F7"},{"1":"GBM cell 282","2":"GSM2243726","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421094","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963922","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243726/suppl/GSM2243726_1001000176.G3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G3"},{"1":"GBM cell 283","2":"GSM2243727","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421095","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963923","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243727/suppl/GSM2243727_1001000176.B8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B8"},{"1":"GBM cell 285","2":"GSM2243729","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421097","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963925","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243729/suppl/GSM2243729_1001000176.C11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C11"},{"1":"GBM cell 286","2":"GSM2243730","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421206","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963926","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243730/suppl/GSM2243730_1001000176.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D1"},{"1":"GBM cell 287","2":"GSM2243731","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: G4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421207","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963927","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243731/suppl/GSM2243731_1001000176.G4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G4"},{"1":"GBM cell 288","2":"GSM2243732","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421208","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963928","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243732/suppl/GSM2243732_1001000176.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G12"},{"1":"GBM cell 289","2":"GSM2243733","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421209","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963929","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243733/suppl/GSM2243733_1001000176.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A11"},{"1":"GBM cell 290","2":"GSM2243734","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: C3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421210","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963930","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243734/suppl/GSM2243734_1001000176.C3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"C3"},{"1":"GBM cell 291","2":"GSM2243735","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421211","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963931","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243735/suppl/GSM2243735_1001000176.E10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E10"},{"1":"GBM cell 293","2":"GSM2243737","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421213","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963933","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243737/suppl/GSM2243737_1001000176.E2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E2"},{"1":"GBM cell 296","2":"GSM2243740","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421216","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963936","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243740/suppl/GSM2243740_1001000176.H4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H4"},{"1":"GBM cell 297","2":"GSM2243741","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421217","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963937","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243741/suppl/GSM2243741_1001000176.A4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A4"},{"1":"GBM cell 298","2":"GSM2243742","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421218","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963938","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243742/suppl/GSM2243742_1001000176.C8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C8"},{"1":"GBM cell 300","2":"GSM2243744","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421220","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963940","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243744/suppl/GSM2243744_1001000176.B11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B11"},{"1":"GBM cell 301","2":"GSM2243745","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421221","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963941","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243745/suppl/GSM2243745_1001000176.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H1"},{"1":"GBM cell 307","2":"GSM2243751","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: F4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421228","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963947","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243751/suppl/GSM2243751_1001000176.F4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F4"},{"1":"GBM cell 308","2":"GSM2243752","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421222","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963948","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243752/suppl/GSM2243752_1001000176.H10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H10"},{"1":"GBM cell 310","2":"GSM2243754","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421230","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963950","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243754/suppl/GSM2243754_1001000176.H8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H8"},{"1":"GBM cell 312","2":"GSM2243756","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421232","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963952","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243756/suppl/GSM2243756_1001000176.E4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"E4"},{"1":"GBM cell 314","2":"GSM2243758","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: D8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421234","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963954","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243758/suppl/GSM2243758_1001000176.D8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"D8"},{"1":"GBM cell 315","2":"GSM2243759","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000176","12":"well: A2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000176.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421235","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963955","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243759/suppl/GSM2243759_1001000176.A2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A2"},{"1":"GBM cell 316","2":"GSM2243760","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421176","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963956","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243760/suppl/GSM2243760_1001000177.G9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G9"},{"1":"GBM cell 319","2":"GSM2243763","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421179","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963959","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243763/suppl/GSM2243763_1001000177.D6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D6"},{"1":"GBM cell 320","2":"GSM2243764","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: E11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421180","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963960","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243764/suppl/GSM2243764_1001000177.E11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E11"},{"1":"GBM cell 324","2":"GSM2243768","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421184","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963964","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243768/suppl/GSM2243768_1001000177.B1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B1"},{"1":"GBM cell 347","2":"GSM2243791","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420884","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963987","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243791/suppl/GSM2243791_1001000177.A9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A9"},{"1":"GBM cell 348","2":"GSM2243792","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420885","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963988","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243792/suppl/GSM2243792_1001000177.F11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F11"},{"1":"GBM cell 350","2":"GSM2243794","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420887","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963990","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243794/suppl/GSM2243794_1001000177.G2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G2"},{"1":"GBM cell 351","2":"GSM2243795","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420888","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963991","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243795/suppl/GSM2243795_1001000177.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A3"},{"1":"GBM cell 352","2":"GSM2243796","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420889","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963992","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243796/suppl/GSM2243796_1001000177.H7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H7"},{"1":"GBM cell 353","2":"GSM2243797","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420890","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963993","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243797/suppl/GSM2243797_1001000177.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D1"},{"1":"GBM cell 354","2":"GSM2243798","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420891","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963994","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243798/suppl/GSM2243798_1001000177.D4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D4"},{"1":"GBM cell 355","2":"GSM2243799","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420892","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963995","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243799/suppl/GSM2243799_1001000177.C8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C8"},{"1":"GBM cell 356","2":"GSM2243800","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420893","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963996","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243800/suppl/GSM2243800_1001000177.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H11"},{"1":"GBM cell 359","2":"GSM2243803","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: G4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420896","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1963999","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243803/suppl/GSM2243803_1001000177.G4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G4"},{"1":"GBM cell 368","2":"GSM2243812","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000177","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000177.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420901","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964008","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243812/suppl/GSM2243812_1001000177.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F12"},{"1":"GBM cell 411","2":"GSM2243855","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: F10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421121","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964051","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243855/suppl/GSM2243855_1001000178.F10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F10"},{"1":"GBM cell 412","2":"GSM2243856","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: B5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421122","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964052","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243856/suppl/GSM2243856_1001000178.B5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B5"},{"1":"GBM cell 415","2":"GSM2243859","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: F11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421125","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964055","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243859/suppl/GSM2243859_1001000178.F11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F11"},{"1":"GBM cell 421","2":"GSM2243865","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: D8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421131","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964061","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243865/suppl/GSM2243865_1001000178.D8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D8"},{"1":"GBM cell 424","2":"GSM2243868","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: B3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421134","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964064","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243868/suppl/GSM2243868_1001000178.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"B3"},{"1":"GBM cell 428","2":"GSM2243872","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: B1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421138","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964068","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243872/suppl/GSM2243872_1001000178.B1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B1"},{"1":"GBM cell 429","2":"GSM2243873","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: F8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421139","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964069","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243873/suppl/GSM2243873_1001000178.F8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F8"},{"1":"GBM cell 430","2":"GSM2243874","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: A11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421140","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964070","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243874/suppl/GSM2243874_1001000178.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A11"},{"1":"GBM cell 437","2":"GSM2243881","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: E12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421147","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964077","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243881/suppl/GSM2243881_1001000178.E12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E12"},{"1":"GBM cell 440","2":"GSM2243884","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: F6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421150","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964080","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243884/suppl/GSM2243884_1001000178.F6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F6"},{"1":"GBM cell 445","2":"GSM2243889","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: D11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421155","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964085","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243889/suppl/GSM2243889_1001000178.D11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D11"},{"1":"GBM cell 446","2":"GSM2243890","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: D7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421156","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964086","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243890/suppl/GSM2243890_1001000178.D7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"D7"},{"1":"GBM cell 447","2":"GSM2243891","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: G7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421157","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964087","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243891/suppl/GSM2243891_1001000178.G7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G7"},{"1":"GBM cell 451","2":"GSM2243895","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: E6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421161","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964091","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243895/suppl/GSM2243895_1001000178.E6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E6"},{"1":"GBM cell 454","2":"GSM2243898","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: F12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421164","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964094","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243898/suppl/GSM2243898_1001000178.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F12"},{"1":"GBM cell 458","2":"GSM2243902","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: A10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421168","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964098","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243902/suppl/GSM2243902_1001000178.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A10"},{"1":"GBM cell 474","2":"GSM2243918","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: F5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420951","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964114","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243918/suppl/GSM2243918_1001000178.F5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F5"},{"1":"GBM cell 481","2":"GSM2243925","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420958","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964121","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243925/suppl/GSM2243925_1001000178.H3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H3"},{"1":"GBM cell 482","2":"GSM2243926","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: G12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420959","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964122","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243926/suppl/GSM2243926_1001000178.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G12"},{"1":"GBM cell 486","2":"GSM2243930","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: G11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420963","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964126","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243930/suppl/GSM2243930_1001000178.G11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G11"},{"1":"GBM cell 487","2":"GSM2243931","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420964","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964127","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243931/suppl/GSM2243931_1001000178.H4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H4"},{"1":"GBM cell 488","2":"GSM2243932","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420965","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964128","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243932/suppl/GSM2243932_1001000178.H2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H2"},{"1":"GBM cell 489","2":"GSM2243933","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: G9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420966","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964129","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243933/suppl/GSM2243933_1001000178.G9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G9"},{"1":"GBM cell 490","2":"GSM2243934","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420967","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964130","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243934/suppl/GSM2243934_1001000178.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H1"},{"1":"GBM cell 491","2":"GSM2243935","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420968","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964131","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243935/suppl/GSM2243935_1001000178.H8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H8"},{"1":"GBM cell 492","2":"GSM2243936","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420969","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964132","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243936/suppl/GSM2243936_1001000178.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H11"},{"1":"GBM cell 493","2":"GSM2243937","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: H6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420970","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964133","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243937/suppl/GSM2243937_1001000178.H6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H6"},{"1":"GBM cell 494","2":"GSM2243938","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000178","12":"well: G10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000178.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420971","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964134","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243938/suppl/GSM2243938_1001000178.G10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G10"},{"1":"GBM cell 495","2":"GSM2243939","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05420972","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964135","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243939/suppl/GSM2243939_1001000179.F10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F10"},{"1":"GBM cell 497","2":"GSM2243941","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421004","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964137","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243941/suppl/GSM2243941_1001000179.H7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"H7"},{"1":"GBM cell 498","2":"GSM2243942","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421005","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964138","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243942/suppl/GSM2243942_1001000179.C1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C1"},{"1":"GBM cell 500","2":"GSM2243944","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05421007","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964140","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243944/suppl/GSM2243944_1001000179.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A10"},{"1":"GBM cell 501","2":"GSM2243945","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422336","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964141","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243945/suppl/GSM2243945_1001000179.B4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B4"},{"1":"GBM cell 502","2":"GSM2243946","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422337","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964142","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243946/suppl/GSM2243946_1001000179.C7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C7"},{"1":"GBM cell 503","2":"GSM2243947","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422338","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964143","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243947/suppl/GSM2243947_1001000179.D9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"D9"},{"1":"GBM cell 504","2":"GSM2243948","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422339","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964144","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243948/suppl/GSM2243948_1001000179.E6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E6"},{"1":"GBM cell 505","2":"GSM2243949","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422340","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964145","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243949/suppl/GSM2243949_1001000179.A4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A4"},{"1":"GBM cell 506","2":"GSM2243950","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: G3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422341","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964146","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243950/suppl/GSM2243950_1001000179.G3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G3"},{"1":"GBM cell 507","2":"GSM2243951","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422342","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964147","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243951/suppl/GSM2243951_1001000179.D11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D11"},{"1":"GBM cell 508","2":"GSM2243952","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422343","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964148","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243952/suppl/GSM2243952_1001000179.H5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H5"},{"1":"GBM cell 509","2":"GSM2243953","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422344","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964149","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243953/suppl/GSM2243953_1001000179.D6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D6"},{"1":"GBM cell 510","2":"GSM2243954","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422345","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964150","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243954/suppl/GSM2243954_1001000179.B8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B8"},{"1":"GBM cell 511","2":"GSM2243955","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422346","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964151","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243955/suppl/GSM2243955_1001000179.C4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C4"},{"1":"GBM cell 512","2":"GSM2243956","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422347","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964152","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243956/suppl/GSM2243956_1001000179.F1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F1"},{"1":"GBM cell 513","2":"GSM2243957","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422348","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964153","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243957/suppl/GSM2243957_1001000179.B10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B10"},{"1":"GBM cell 515","2":"GSM2243959","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422350","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964155","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243959/suppl/GSM2243959_1001000179.F5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"F5"},{"1":"GBM cell 516","2":"GSM2243960","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422351","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964156","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243960/suppl/GSM2243960_1001000179.C10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C10"},{"1":"GBM cell 517","2":"GSM2243961","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422352","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964157","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243961/suppl/GSM2243961_1001000179.D4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D4"},{"1":"GBM cell 518","2":"GSM2243962","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422353","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964158","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243962/suppl/GSM2243962_1001000179.H3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H3"},{"1":"GBM cell 519","2":"GSM2243963","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422354","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964159","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243963/suppl/GSM2243963_1001000179.F4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F4"},{"1":"GBM cell 520","2":"GSM2243964","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422355","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964160","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243964/suppl/GSM2243964_1001000179.D3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D3"},{"1":"GBM cell 521","2":"GSM2243965","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422356","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964161","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243965/suppl/GSM2243965_1001000179.B7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B7"},{"1":"GBM cell 522","2":"GSM2243966","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422357","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964162","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243966/suppl/GSM2243966_1001000179.E9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E9"},{"1":"GBM cell 523","2":"GSM2243967","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422358","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964163","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243967/suppl/GSM2243967_1001000179.E12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E12"},{"1":"GBM cell 524","2":"GSM2243968","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422359","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964164","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243968/suppl/GSM2243968_1001000179.A1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A1"},{"1":"GBM cell 525","2":"GSM2243969","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: G5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422360","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964165","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243969/suppl/GSM2243969_1001000179.G5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G5"},{"1":"GBM cell 526","2":"GSM2243970","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422428","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964166","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243970/suppl/GSM2243970_1001000179.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H11"},{"1":"GBM cell 527","2":"GSM2243971","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422429","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964167","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243971/suppl/GSM2243971_1001000179.E3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"E3"},{"1":"GBM cell 528","2":"GSM2243972","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422430","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964168","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243972/suppl/GSM2243972_1001000179.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B3"},{"1":"GBM cell 529","2":"GSM2243973","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422431","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964169","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243973/suppl/GSM2243973_1001000179.C3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C3"},{"1":"GBM cell 530","2":"GSM2243974","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422432","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964170","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243974/suppl/GSM2243974_1001000179.A9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A9"},{"1":"GBM cell 531","2":"GSM2243975","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422433","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964171","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243975/suppl/GSM2243975_1001000179.F6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F6"},{"1":"GBM cell 532","2":"GSM2243976","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422434","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964172","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243976/suppl/GSM2243976_1001000179.H9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H9"},{"1":"GBM cell 533","2":"GSM2243977","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422435","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964173","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243977/suppl/GSM2243977_1001000179.C12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C12"},{"1":"GBM cell 534","2":"GSM2243978","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422436","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964174","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243978/suppl/GSM2243978_1001000179.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H1"},{"1":"GBM cell 535","2":"GSM2243979","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: G6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.G6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422437","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964175","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243979/suppl/GSM2243979_1001000179.G6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G6"},{"1":"GBM cell 536","2":"GSM2243980","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422438","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964176","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243980/suppl/GSM2243980_1001000179.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D1"},{"1":"GBM cell 537","2":"GSM2243981","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422439","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964177","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243981/suppl/GSM2243981_1001000179.F9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F9"},{"1":"GBM cell 538","2":"GSM2243982","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422440","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964178","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243982/suppl/GSM2243982_1001000179.B1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B1"},{"1":"GBM cell 540","2":"GSM2243984","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422442","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964180","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243984/suppl/GSM2243984_1001000179.A12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A12"},{"1":"GBM cell 541","2":"GSM2243985","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422443","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964181","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243985/suppl/GSM2243985_1001000179.A7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A7"},{"1":"GBM cell 544","2":"GSM2243988","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422446","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964184","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243988/suppl/GSM2243988_1001000179.C8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C8"},{"1":"GBM cell 545","2":"GSM2243989","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422447","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964185","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243989/suppl/GSM2243989_1001000179.E1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E1"},{"1":"GBM cell 547","2":"GSM2243991","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422449","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964187","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243991/suppl/GSM2243991_1001000179.H10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H10"},{"1":"GBM cell 552","2":"GSM2243996","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422454","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964192","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243996/suppl/GSM2243996_1001000179.D7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D7"},{"1":"GBM cell 553","2":"GSM2243997","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: G11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422455","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964193","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243997/suppl/GSM2243997_1001000179.G11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G11"},{"1":"GBM cell 554","2":"GSM2243998","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422490","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964194","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243998/suppl/GSM2243998_1001000179.H6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"H6"},{"1":"GBM cell 555","2":"GSM2243999","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422491","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964195","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2243nnn/GSM2243999/suppl/GSM2243999_1001000179.F8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F8"},{"1":"GBM cell 557","2":"GSM2244001","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422456","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964197","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244001/suppl/GSM2244001_1001000179.C5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C5"},{"1":"GBM cell 558","2":"GSM2244002","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: G1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422457","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964198","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244002/suppl/GSM2244002_1001000179.G1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G1"},{"1":"GBM cell 560","2":"GSM2244004","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422489","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964200","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244004/suppl/GSM2244004_1001000179.E2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E2"},{"1":"GBM cell 561","2":"GSM2244005","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422493","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964201","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244005/suppl/GSM2244005_1001000179.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A3"},{"1":"GBM cell 563","2":"GSM2244007","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422495","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964203","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244007/suppl/GSM2244007_1001000179.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D5"},{"1":"GBM cell 565","2":"GSM2244009","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422497","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964205","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244009/suppl/GSM2244009_1001000179.B11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"B11"},{"1":"GBM cell 566","2":"GSM2244010","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422498","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964206","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244010/suppl/GSM2244010_1001000179.B9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"B9"},{"1":"GBM cell 567","2":"GSM2244011","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422499","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964207","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244011/suppl/GSM2244011_1001000179.D12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D12"},{"1":"GBM cell 568","2":"GSM2244012","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422500","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964208","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244012/suppl/GSM2244012_1001000179.F11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F11"},{"1":"GBM cell 569","2":"GSM2244013","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422501","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964209","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244013/suppl/GSM2244013_1001000179.E11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E11"},{"1":"GBM cell 570","2":"GSM2244014","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422502","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964210","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244014/suppl/GSM2244014_1001000179.A8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A8"},{"1":"GBM cell 571","2":"GSM2244015","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: G10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422503","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964211","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244015/suppl/GSM2244015_1001000179.G10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"G10"},{"1":"GBM cell 572","2":"GSM2244016","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422504","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964212","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244016/suppl/GSM2244016_1001000179.E4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E4"},{"1":"GBM cell 573","2":"GSM2244017","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422505","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964213","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244017/suppl/GSM2244017_1001000179.C11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C11"},{"1":"GBM cell 574","2":"GSM2244018","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422506","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964214","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244018/suppl/GSM2244018_1001000179.H2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H2"},{"1":"GBM cell 575","2":"GSM2244019","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422507","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964215","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244019/suppl/GSM2244019_1001000179.F3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F3"},{"1":"GBM cell 576","2":"GSM2244020","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422508","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964216","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244020/suppl/GSM2244020_1001000179.F7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F7"},{"1":"GBM cell 578","2":"GSM2244022","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: D2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422510","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964218","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244022/suppl/GSM2244022_1001000179.D2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"D2"},{"1":"GBM cell 579","2":"GSM2244023","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422511","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964219","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244023/suppl/GSM2244023_1001000179.E8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E8"},{"1":"GBM cell 580","2":"GSM2244024","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422512","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964220","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244024/suppl/GSM2244024_1001000179.A6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A6"},{"1":"GBM cell 581","2":"GSM2244025","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: C2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422513","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964221","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244025/suppl/GSM2244025_1001000179.C2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"C2"},{"1":"GBM cell 582","2":"GSM2244026","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: A11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422514","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964222","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244026/suppl/GSM2244026_1001000179.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"A11"},{"1":"GBM cell 583","2":"GSM2244027","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: F12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422515","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964223","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244027/suppl/GSM2244027_1001000179.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"F12"},{"1":"GBM cell 584","2":"GSM2244028","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: B6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422516","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964224","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244028/suppl/GSM2244028_1001000179.B6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B6"},{"1":"GBM cell 585","2":"GSM2244029","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: H8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422517","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964225","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244029/suppl/GSM2244029_1001000179.H8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"H8"},{"1":"GBM cell 586","2":"GSM2244030","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000179","12":"well: E10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000179.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422306","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964226","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244030/suppl/GSM2244030_1001000179.E10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"E10"},{"1":"GBM cell 587","2":"GSM2244031","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: E8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422307","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964227","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244031/suppl/GSM2244031_1001000180.E8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"E8"},{"1":"GBM cell 588","2":"GSM2244032","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: B4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422308","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964228","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244032/suppl/GSM2244032_1001000180.B4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B4"},{"1":"GBM cell 589","2":"GSM2244033","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: B9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422309","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964229","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244033/suppl/GSM2244033_1001000180.B9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B9"},{"1":"GBM cell 590","2":"GSM2244034","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: G10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422310","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964230","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244034/suppl/GSM2244034_1001000180.G10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"G10"},{"1":"GBM cell 591","2":"GSM2244035","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: E11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422311","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964231","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244035/suppl/GSM2244035_1001000180.E11.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"2","58":"E11"},{"1":"GBM cell 592","2":"GSM2244036","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: F5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422312","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964232","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244036/suppl/GSM2244036_1001000180.F5.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"2","58":"F5"},{"1":"GBM cell 593","2":"GSM2244037","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: A11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422313","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964233","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244037/suppl/GSM2244037_1001000180.A11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"A11"},{"1":"GBM cell 594","2":"GSM2244038","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: G9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422314","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964234","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244038/suppl/GSM2244038_1001000180.G9.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"G9"},{"1":"GBM cell 595","2":"GSM2244039","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: C7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422315","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964235","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244039/suppl/GSM2244039_1001000180.C7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"C7"},{"1":"GBM cell 596","2":"GSM2244040","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: G12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422316","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964236","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244040/suppl/GSM2244040_1001000180.G12.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"G12"},{"1":"GBM cell 597","2":"GSM2244041","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: H1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422317","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964237","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244041/suppl/GSM2244041_1001000180.H1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"H1"},{"1":"GBM cell 598","2":"GSM2244042","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: E10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422318","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964238","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244042/suppl/GSM2244042_1001000180.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"E10"},{"1":"GBM cell 599","2":"GSM2244043","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: A5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422319","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964239","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244043/suppl/GSM2244043_1001000180.A5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"A5"},{"1":"GBM cell 600","2":"GSM2244044","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: D2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422320","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964240","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244044/suppl/GSM2244044_1001000180.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"D2"},{"1":"GBM cell 601","2":"GSM2244045","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: F2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422321","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964241","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244045/suppl/GSM2244045_1001000180.F2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"F2"},{"1":"GBM cell 602","2":"GSM2244046","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: F1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422322","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964242","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244046/suppl/GSM2244046_1001000180.F1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"F1"},{"1":"GBM cell 603","2":"GSM2244047","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: H7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422323","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964243","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244047/suppl/GSM2244047_1001000180.H7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"H7"},{"1":"GBM cell 604","2":"GSM2244048","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: B7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422324","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964244","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244048/suppl/GSM2244048_1001000180.B7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B7"},{"1":"GBM cell 605","2":"GSM2244049","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: A8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422325","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964245","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244049/suppl/GSM2244049_1001000180.A8.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"2","58":"A8"},{"1":"GBM cell 606","2":"GSM2244050","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: D8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422326","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964246","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244050/suppl/GSM2244050_1001000180.D8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"D8"},{"1":"GBM cell 607","2":"GSM2244051","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: E4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422327","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964247","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244051/suppl/GSM2244051_1001000180.E4.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"E4"},{"1":"GBM cell 610","2":"GSM2244054","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: H4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422330","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964250","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244054/suppl/GSM2244054_1001000180.H4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"H4"},{"1":"GBM cell 611","2":"GSM2244055","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: B2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422331","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964251","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244055/suppl/GSM2244055_1001000180.B2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B2"},{"1":"GBM cell 612","2":"GSM2244056","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: G11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422332","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964252","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244056/suppl/GSM2244056_1001000180.G11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"G11"},{"1":"GBM cell 615","2":"GSM2244059","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000180","12":"well: C12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000180.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422581","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964255","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244059/suppl/GSM2244059_1001000180.C12.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"2","58":"C12"},{"1":"GBM cell 649","2":"GSM2244093","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: C5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422671","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964289","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244093/suppl/GSM2244093_1001000181.C5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"C5"},{"1":"GBM cell 654","2":"GSM2244098","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: F7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422676","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964294","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244098/suppl/GSM2244098_1001000181.F7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"F7"},{"1":"GBM cell 655","2":"GSM2244099","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: E1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422677","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964295","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244099/suppl/GSM2244099_1001000181.E1.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"E1"},{"1":"GBM cell 656","2":"GSM2244100","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: G3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422678","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964296","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244100/suppl/GSM2244100_1001000181.G3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"G3"},{"1":"GBM cell 657","2":"GSM2244101","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: B5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422679","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964297","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244101/suppl/GSM2244101_1001000181.B5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B5"},{"1":"GBM cell 658","2":"GSM2244102","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: D10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422680","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964298","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244102/suppl/GSM2244102_1001000181.D10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"D10"},{"1":"GBM cell 659","2":"GSM2244103","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: F8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422681","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964299","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244103/suppl/GSM2244103_1001000181.F8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"F8"},{"1":"GBM cell 660","2":"GSM2244104","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: F1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422682","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964300","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244104/suppl/GSM2244104_1001000181.F1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"F1"},{"1":"GBM cell 661","2":"GSM2244105","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: F12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422683","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964301","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244105/suppl/GSM2244105_1001000181.F12.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"F12"},{"1":"GBM cell 662","2":"GSM2244106","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: E3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422684","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964302","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244106/suppl/GSM2244106_1001000181.E3.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"E3"},{"1":"GBM cell 665","2":"GSM2244109","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: E11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422687","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964305","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244109/suppl/GSM2244109_1001000181.E11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"E11"},{"1":"GBM cell 668","2":"GSM2244112","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: B7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422690","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964308","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244112/suppl/GSM2244112_1001000181.B7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B7"},{"1":"GBM cell 674","2":"GSM2244118","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: C11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422697","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964314","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244118/suppl/GSM2244118_1001000181.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"C11"},{"1":"GBM cell 676","2":"GSM2244120","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: A4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422639","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964316","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244120/suppl/GSM2244120_1001000181.A4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"A4"},{"1":"GBM cell 677","2":"GSM2244121","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: D7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422693","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964317","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244121/suppl/GSM2244121_1001000181.D7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"D7"},{"1":"GBM cell 678","2":"GSM2244122","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: B2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422640","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964318","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244122/suppl/GSM2244122_1001000181.B2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B2"},{"1":"GBM cell 679","2":"GSM2244123","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: B9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422641","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964319","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244123/suppl/GSM2244123_1001000181.B9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"B9"},{"1":"GBM cell 680","2":"GSM2244124","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: C4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422642","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964320","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244124/suppl/GSM2244124_1001000181.C4.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"C4"},{"1":"GBM cell 682","2":"GSM2244126","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: A11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422644","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964322","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244126/suppl/GSM2244126_1001000181.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"7","58":"A11"},{"1":"GBM cell 684","2":"GSM2244128","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: A6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422646","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964324","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244128/suppl/GSM2244128_1001000181.A6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"A6"},{"1":"GBM cell 693","2":"GSM2244137","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000181","12":"well: A7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000181.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422655","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964333","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244137/suppl/GSM2244137_1001000181.A7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"9","58":"A7"},{"1":"GBM cell 716","2":"GSM2244160","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422286","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964356","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244160/suppl/GSM2244160_1001000182.E8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"E8"},{"1":"GBM cell 717","2":"GSM2244161","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422287","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964357","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244161/suppl/GSM2244161_1001000182.B4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B4"},{"1":"GBM cell 718","2":"GSM2244162","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422288","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964358","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244162/suppl/GSM2244162_1001000182.F6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"F6"},{"1":"GBM cell 719","2":"GSM2244163","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422289","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964359","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244163/suppl/GSM2244163_1001000182.B12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B12"},{"1":"GBM cell 720","2":"GSM2244164","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422290","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964360","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244164/suppl/GSM2244164_1001000182.C10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"C10"},{"1":"GBM cell 722","2":"GSM2244166","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: A7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422292","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964362","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244166/suppl/GSM2244166_1001000182.A7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"A7"},{"1":"GBM cell 723","2":"GSM2244167","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422293","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964363","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244167/suppl/GSM2244167_1001000182.D11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"D11"},{"1":"GBM cell 724","2":"GSM2244168","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422294","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964364","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244168/suppl/GSM2244168_1001000182.G8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"G8"},{"1":"GBM cell 725","2":"GSM2244169","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422295","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964365","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244169/suppl/GSM2244169_1001000182.D7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"D7"},{"1":"GBM cell 728","2":"GSM2244172","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422298","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964368","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244172/suppl/GSM2244172_1001000182.H2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"H2"},{"1":"GBM cell 729","2":"GSM2244173","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422299","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964369","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244173/suppl/GSM2244173_1001000182.B10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B10"},{"1":"GBM cell 731","2":"GSM2244175","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422301","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964371","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244175/suppl/GSM2244175_1001000182.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"E2"},{"1":"GBM cell 732","2":"GSM2244176","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422302","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964372","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244176/suppl/GSM2244176_1001000182.A9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"A9"},{"1":"GBM cell 734","2":"GSM2244178","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422698","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964374","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244178/suppl/GSM2244178_1001000182.B8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B8"},{"1":"GBM cell 735","2":"GSM2244179","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422699","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964375","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244179/suppl/GSM2244179_1001000182.C8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"C8"},{"1":"GBM cell 748","2":"GSM2244192","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422711","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964388","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244192/suppl/GSM2244192_1001000182.G3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"G3"},{"1":"GBM cell 749","2":"GSM2244193","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: A2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422712","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964389","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244193/suppl/GSM2244193_1001000182.A2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"A2"},{"1":"GBM cell 750","2":"GSM2244194","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422714","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964390","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244194/suppl/GSM2244194_1001000182.C4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"C4"},{"1":"GBM cell 751","2":"GSM2244195","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422716","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964391","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244195/suppl/GSM2244195_1001000182.E7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"E7"},{"1":"GBM cell 752","2":"GSM2244196","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422718","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964392","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244196/suppl/GSM2244196_1001000182.F5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"F5"},{"1":"GBM cell 753","2":"GSM2244197","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422719","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964393","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244197/suppl/GSM2244197_1001000182.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B5"},{"1":"GBM cell 754","2":"GSM2244198","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422721","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964394","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244198/suppl/GSM2244198_1001000182.F10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"F10"},{"1":"GBM cell 755","2":"GSM2244199","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422723","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964395","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244199/suppl/GSM2244199_1001000182.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"C11"},{"1":"GBM cell 756","2":"GSM2244200","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422724","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964396","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244200/suppl/GSM2244200_1001000182.H9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"H9"},{"1":"GBM cell 757","2":"GSM2244201","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: A4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422726","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964397","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244201/suppl/GSM2244201_1001000182.A4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"9","58":"A4"},{"1":"GBM cell 758","2":"GSM2244202","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: D8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422729","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964398","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244202/suppl/GSM2244202_1001000182.D8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"D8"},{"1":"GBM cell 759","2":"GSM2244203","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422731","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964399","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244203/suppl/GSM2244203_1001000182.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"G2"},{"1":"GBM cell 760","2":"GSM2244204","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422734","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964400","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244204/suppl/GSM2244204_1001000182.F11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"F11"},{"1":"GBM cell 761","2":"GSM2244205","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422736","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964401","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244205/suppl/GSM2244205_1001000182.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"E10"},{"1":"GBM cell 762","2":"GSM2244206","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422738","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964402","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244206/suppl/GSM2244206_1001000182.D12.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"2","58":"D12"},{"1":"GBM cell 763","2":"GSM2244207","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422739","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964403","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244207/suppl/GSM2244207_1001000182.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"H1"},{"1":"GBM cell 765","2":"GSM2244209","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422743","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964405","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244209/suppl/GSM2244209_1001000182.H4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"H4"},{"1":"GBM cell 772","2":"GSM2244216","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: C7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423065","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964412","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244216/suppl/GSM2244216_1001000182.C7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"C7"},{"1":"GBM cell 779","2":"GSM2244223","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000182","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000182.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423080","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964419","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244223/suppl/GSM2244223_1001000182.A8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"A8"},{"1":"GBM cell 780","2":"GSM2244224","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423082","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964420","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244224/suppl/GSM2244224_1001000183.A3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"A3"},{"1":"GBM cell 787","2":"GSM2244231","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423097","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964427","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244231/suppl/GSM2244231_1001000183.D4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"D4"},{"1":"GBM cell 792","2":"GSM2244236","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423107","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964432","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244236/suppl/GSM2244236_1001000183.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"C11"},{"1":"GBM cell 797","2":"GSM2244241","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422990","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964437","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244241/suppl/GSM2244241_1001000183.B4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B4"},{"1":"GBM cell 799","2":"GSM2244243","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422994","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964439","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244243/suppl/GSM2244243_1001000183.H9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"H9"},{"1":"GBM cell 839","2":"GSM2244283","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422530","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964479","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244283/suppl/GSM2244283_1001000183.G7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"G7"},{"1":"GBM cell 844","2":"GSM2244288","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422535","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964484","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244288/suppl/GSM2244288_1001000183.E5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"E5"},{"1":"GBM cell 845","2":"GSM2244289","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422536","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964485","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244289/suppl/GSM2244289_1001000183.B3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B3"},{"1":"GBM cell 847","2":"GSM2244291","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422538","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964487","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244291/suppl/GSM2244291_1001000183.D1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"D1"},{"1":"GBM cell 849","2":"GSM2244293","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422540","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964489","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244293/suppl/GSM2244293_1001000183.F7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"F7"},{"1":"GBM cell 853","2":"GSM2244297","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000183","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000183.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422547","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964493","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244297/suppl/GSM2244297_1001000183.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"E2"},{"1":"GBM cell 858","2":"GSM2244302","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422545","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964498","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244302/suppl/GSM2244302_1001000184.A1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A1"},{"1":"GBM cell 861","2":"GSM2244305","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422933","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964501","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244305/suppl/GSM2244305_1001000184.B2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B2"},{"1":"GBM cell 862","2":"GSM2244306","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422936","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964502","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244306/suppl/GSM2244306_1001000184.B9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B9"},{"1":"GBM cell 863","2":"GSM2244307","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422938","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964503","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244307/suppl/GSM2244307_1001000184.B3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B3"},{"1":"GBM cell 864","2":"GSM2244308","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422940","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964504","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244308/suppl/GSM2244308_1001000184.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C1"},{"1":"GBM cell 865","2":"GSM2244309","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422942","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964505","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244309/suppl/GSM2244309_1001000184.A6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A6"},{"1":"GBM cell 866","2":"GSM2244310","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422944","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964506","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244310/suppl/GSM2244310_1001000184.B6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"B6"},{"1":"GBM cell 867","2":"GSM2244311","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422946","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964507","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244311/suppl/GSM2244311_1001000184.A4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A4"},{"1":"GBM cell 868","2":"GSM2244312","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422949","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964508","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244312/suppl/GSM2244312_1001000184.A8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A8"},{"1":"GBM cell 869","2":"GSM2244313","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422951","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964509","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244313/suppl/GSM2244313_1001000184.B10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B10"},{"1":"GBM cell 870","2":"GSM2244314","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422953","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964510","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244314/suppl/GSM2244314_1001000184.A10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A10"},{"1":"GBM cell 872","2":"GSM2244316","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422957","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964512","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244316/suppl/GSM2244316_1001000184.A9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A9"},{"1":"GBM cell 873","2":"GSM2244317","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422959","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964513","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244317/suppl/GSM2244317_1001000184.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B11"},{"1":"GBM cell 874","2":"GSM2244318","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422961","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964514","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244318/suppl/GSM2244318_1001000184.A5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A5"},{"1":"GBM cell 875","2":"GSM2244319","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422963","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964515","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244319/suppl/GSM2244319_1001000184.B4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"B4"},{"1":"GBM cell 877","2":"GSM2244321","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422968","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964517","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244321/suppl/GSM2244321_1001000184.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B5"},{"1":"GBM cell 878","2":"GSM2244322","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: A2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422971","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964518","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244322/suppl/GSM2244322_1001000184.A2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A2"},{"1":"GBM cell 880","2":"GSM2244324","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422976","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964520","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244324/suppl/GSM2244324_1001000184.D9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D9"},{"1":"GBM cell 881","2":"GSM2244325","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422978","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964521","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244325/suppl/GSM2244325_1001000184.G11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G11"},{"1":"GBM cell 882","2":"GSM2244326","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422980","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964522","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244326/suppl/GSM2244326_1001000184.F6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F6"},{"1":"GBM cell 883","2":"GSM2244327","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422982","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964523","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244327/suppl/GSM2244327_1001000184.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C11"},{"1":"GBM cell 884","2":"GSM2244328","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422984","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964524","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244328/suppl/GSM2244328_1001000184.D7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D7"},{"1":"GBM cell 885","2":"GSM2244329","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422986","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964525","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244329/suppl/GSM2244329_1001000184.G5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G5"},{"1":"GBM cell 886","2":"GSM2244330","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: C5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422987","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964526","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244330/suppl/GSM2244330_1001000184.C5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C5"},{"1":"GBM cell 887","2":"GSM2244331","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422865","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964527","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244331/suppl/GSM2244331_1001000184.H4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H4"},{"1":"GBM cell 888","2":"GSM2244332","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422867","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964528","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244332/suppl/GSM2244332_1001000184.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E2"},{"1":"GBM cell 889","2":"GSM2244333","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422869","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964529","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244333/suppl/GSM2244333_1001000184.F10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F10"},{"1":"GBM cell 890","2":"GSM2244334","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422870","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964530","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244334/suppl/GSM2244334_1001000184.E3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"E3"},{"1":"GBM cell 891","2":"GSM2244335","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422872","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964531","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244335/suppl/GSM2244335_1001000184.D5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D5"},{"1":"GBM cell 892","2":"GSM2244336","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: C3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422873","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964532","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244336/suppl/GSM2244336_1001000184.C3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C3"},{"1":"GBM cell 893","2":"GSM2244337","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422876","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964533","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244337/suppl/GSM2244337_1001000184.H10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H10"},{"1":"GBM cell 894","2":"GSM2244338","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422878","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964534","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244338/suppl/GSM2244338_1001000184.H5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"8","58":"H5"},{"1":"GBM cell 895","2":"GSM2244339","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422880","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964535","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244339/suppl/GSM2244339_1001000184.E8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"E8"},{"1":"GBM cell 896","2":"GSM2244340","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422882","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964536","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244340/suppl/GSM2244340_1001000184.D10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D10"},{"1":"GBM cell 897","2":"GSM2244341","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422884","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964537","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244341/suppl/GSM2244341_1001000184.F3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F3"},{"1":"GBM cell 898","2":"GSM2244342","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422886","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964538","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244342/suppl/GSM2244342_1001000184.E5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E5"},{"1":"GBM cell 899","2":"GSM2244343","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422888","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964539","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244343/suppl/GSM2244343_1001000184.H8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H8"},{"1":"GBM cell 900","2":"GSM2244344","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422890","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964540","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244344/suppl/GSM2244344_1001000184.F12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F12"},{"1":"GBM cell 901","2":"GSM2244345","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422892","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964541","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244345/suppl/GSM2244345_1001000184.D3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D3"},{"1":"GBM cell 902","2":"GSM2244346","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422894","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964542","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244346/suppl/GSM2244346_1001000184.F1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F1"},{"1":"GBM cell 903","2":"GSM2244347","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422896","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964543","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244347/suppl/GSM2244347_1001000184.F9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F9"},{"1":"GBM cell 904","2":"GSM2244348","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422898","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964544","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244348/suppl/GSM2244348_1001000184.H9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H9"},{"1":"GBM cell 905","2":"GSM2244349","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422901","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964545","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244349/suppl/GSM2244349_1001000184.H2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"H2"},{"1":"GBM cell 906","2":"GSM2244350","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422903","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964546","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244350/suppl/GSM2244350_1001000184.D1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D1"},{"1":"GBM cell 908","2":"GSM2244352","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422907","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964548","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244352/suppl/GSM2244352_1001000184.G4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G4"},{"1":"GBM cell 909","2":"GSM2244353","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422909","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964549","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244353/suppl/GSM2244353_1001000184.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H1"},{"1":"GBM cell 910","2":"GSM2244354","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: C6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422911","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964550","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244354/suppl/GSM2244354_1001000184.C6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C6"},{"1":"GBM cell 911","2":"GSM2244355","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422913","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964551","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244355/suppl/GSM2244355_1001000184.F7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F7"},{"1":"GBM cell 912","2":"GSM2244356","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422915","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964552","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244356/suppl/GSM2244356_1001000184.G12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G12"},{"1":"GBM cell 913","2":"GSM2244357","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422917","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964553","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244357/suppl/GSM2244357_1001000184.G7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"G7"},{"1":"GBM cell 914","2":"GSM2244358","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422919","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964554","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244358/suppl/GSM2244358_1001000184.E12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E12"},{"1":"GBM cell 915","2":"GSM2244359","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422921","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964555","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244359/suppl/GSM2244359_1001000184.D6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D6"},{"1":"GBM cell 916","2":"GSM2244360","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422744","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964556","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244360/suppl/GSM2244360_1001000184.D12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D12"},{"1":"GBM cell 917","2":"GSM2244361","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422923","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964557","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244361/suppl/GSM2244361_1001000184.H11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H11"},{"1":"GBM cell 918","2":"GSM2244362","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422746","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964558","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244362/suppl/GSM2244362_1001000184.H3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H3"},{"1":"GBM cell 919","2":"GSM2244363","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422747","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964559","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244363/suppl/GSM2244363_1001000184.F11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"F11"},{"1":"GBM cell 920","2":"GSM2244364","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: H6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422750","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964560","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244364/suppl/GSM2244364_1001000184.H6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H6"},{"1":"GBM cell 923","2":"GSM2244367","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422756","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964563","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244367/suppl/GSM2244367_1001000184.E1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E1"},{"1":"GBM cell 924","2":"GSM2244368","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422758","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964564","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244368/suppl/GSM2244368_1001000184.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E10"},{"1":"GBM cell 925","2":"GSM2244369","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422760","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964565","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244369/suppl/GSM2244369_1001000184.G9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G9"},{"1":"GBM cell 928","2":"GSM2244372","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422766","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964568","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244372/suppl/GSM2244372_1001000184.D4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D4"},{"1":"GBM cell 929","2":"GSM2244373","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: E11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422769","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964569","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244373/suppl/GSM2244373_1001000184.E11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E11"},{"1":"GBM cell 934","2":"GSM2244378","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422778","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964574","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244378/suppl/GSM2244378_1001000184.F8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"F8"},{"1":"GBM cell 935","2":"GSM2244379","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422780","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964575","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244379/suppl/GSM2244379_1001000184.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G2"},{"1":"GBM cell 936","2":"GSM2244380","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422782","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964576","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244380/suppl/GSM2244380_1001000184.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D2"},{"1":"GBM cell 937","2":"GSM2244381","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422784","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964577","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244381/suppl/GSM2244381_1001000184.C12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C12"},{"1":"GBM cell 939","2":"GSM2244383","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000184","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000184.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422788","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964579","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244383/suppl/GSM2244383_1001000184.D11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D11"},{"1":"GBM cell 943","2":"GSM2244387","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422796","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964583","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244387/suppl/GSM2244387_1001000185.F9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"F9"},{"1":"GBM cell 946","2":"GSM2244390","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422812","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964586","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244390/suppl/GSM2244390_1001000185.G11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G11"},{"1":"GBM cell 949","2":"GSM2244393","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422804","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964589","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244393/suppl/GSM2244393_1001000185.D10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D10"},{"1":"GBM cell 952","2":"GSM2244396","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: A7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422814","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964592","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244396/suppl/GSM2244396_1001000185.A7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A7"},{"1":"GBM cell 955","2":"GSM2244399","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422820","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964595","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244399/suppl/GSM2244399_1001000185.F12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F12"},{"1":"GBM cell 956","2":"GSM2244400","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422822","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964596","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244400/suppl/GSM2244400_1001000185.D3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D3"},{"1":"GBM cell 957","2":"GSM2244401","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422824","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964597","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244401/suppl/GSM2244401_1001000185.B9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B9"},{"1":"GBM cell 960","2":"GSM2244404","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422830","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964600","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244404/suppl/GSM2244404_1001000185.F4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F4"},{"1":"GBM cell 963","2":"GSM2244407","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422836","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964603","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244407/suppl/GSM2244407_1001000185.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D2"},{"1":"GBM cell 964","2":"GSM2244408","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422838","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964604","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244408/suppl/GSM2244408_1001000185.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G2"},{"1":"GBM cell 965","2":"GSM2244409","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422840","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964605","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244409/suppl/GSM2244409_1001000185.B7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B7"},{"1":"GBM cell 967","2":"GSM2244411","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422844","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964607","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244411/suppl/GSM2244411_1001000185.E3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E3"},{"1":"GBM cell 969","2":"GSM2244413","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422848","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964609","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244413/suppl/GSM2244413_1001000185.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B5"},{"1":"GBM cell 970","2":"GSM2244414","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422850","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964610","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244414/suppl/GSM2244414_1001000185.E8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E8"},{"1":"GBM cell 971","2":"GSM2244415","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422853","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964611","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244415/suppl/GSM2244415_1001000185.G8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G8"},{"1":"GBM cell 972","2":"GSM2244416","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422855","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964612","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244416/suppl/GSM2244416_1001000185.A3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A3"},{"1":"GBM cell 973","2":"GSM2244417","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422857","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964613","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244417/suppl/GSM2244417_1001000185.F6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F6"},{"1":"GBM cell 975","2":"GSM2244419","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422861","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964615","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244419/suppl/GSM2244419_1001000185.D9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D9"},{"1":"GBM cell 976","2":"GSM2244420","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: G5","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422863","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964616","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244420/suppl/GSM2244420_1001000185.G5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G5"},{"1":"GBM cell 978","2":"GSM2244422","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422549","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964618","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244422/suppl/GSM2244422_1001000185.D7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D7"},{"1":"GBM cell 980","2":"GSM2244424","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: E6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422551","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964620","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244424/suppl/GSM2244424_1001000185.E6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E6"},{"1":"GBM cell 981","2":"GSM2244425","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422552","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964621","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244425/suppl/GSM2244425_1001000185.B8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B8"},{"1":"GBM cell 982","2":"GSM2244426","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422553","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964622","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244426/suppl/GSM2244426_1001000185.D11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D11"},{"1":"GBM cell 983","2":"GSM2244427","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422554","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964623","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244427/suppl/GSM2244427_1001000185.C8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"C8"},{"1":"GBM cell 984","2":"GSM2244428","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: A4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422555","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964624","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244428/suppl/GSM2244428_1001000185.A4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A4"},{"1":"GBM cell 985","2":"GSM2244429","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422556","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964625","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244429/suppl/GSM2244429_1001000185.G7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G7"},{"1":"GBM cell 987","2":"GSM2244431","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422558","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964627","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244431/suppl/GSM2244431_1001000185.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"C1"},{"1":"GBM cell 988","2":"GSM2244432","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422559","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964628","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244432/suppl/GSM2244432_1001000185.H11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H11"},{"1":"GBM cell 990","2":"GSM2244434","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422561","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964630","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244434/suppl/GSM2244434_1001000185.H2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"H2"},{"1":"GBM cell 991","2":"GSM2244435","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422562","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964631","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244435/suppl/GSM2244435_1001000185.G12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"G12"},{"1":"GBM cell 992","2":"GSM2244436","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: A12","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422563","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964632","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244436/suppl/GSM2244436_1001000185.A12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A12"},{"1":"GBM cell 993","2":"GSM2244437","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422564","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964633","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244437/suppl/GSM2244437_1001000185.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E10"},{"1":"GBM cell 994","2":"GSM2244438","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422565","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964634","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244438/suppl/GSM2244438_1001000185.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"H1"},{"1":"GBM cell 995","2":"GSM2244439","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422566","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964635","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244439/suppl/GSM2244439_1001000185.B6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B6"},{"1":"GBM cell 996","2":"GSM2244440","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422567","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964636","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244440/suppl/GSM2244440_1001000185.E4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E4"},{"1":"GBM cell 997","2":"GSM2244441","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422568","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964637","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244441/suppl/GSM2244441_1001000185.F11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F11"},{"1":"GBM cell 998","2":"GSM2244442","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422569","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964638","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244442/suppl/GSM2244442_1001000185.B1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B1"},{"1":"GBM cell 999","2":"GSM2244443","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: A6","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422570","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964639","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244443/suppl/GSM2244443_1001000185.A6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"9","58":"A6"},{"1":"GBM cell 1000","2":"GSM2244444","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422571","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964640","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244444/suppl/GSM2244444_1001000185.F1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F1"},{"1":"GBM cell 1002","2":"GSM2244446","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: F3","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422573","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964642","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244446/suppl/GSM2244446_1001000185.F3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"F3"},{"1":"GBM cell 1003","2":"GSM2244447","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422574","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964643","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244447/suppl/GSM2244447_1001000185.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B11"},{"1":"GBM cell 1005","2":"GSM2244449","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422576","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964645","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244449/suppl/GSM2244449_1001000185.A9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"A9"},{"1":"GBM cell 1006","2":"GSM2244450","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422577","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964646","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244450/suppl/GSM2244450_1001000185.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"E2"},{"1":"GBM cell 1007","2":"GSM2244451","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422368","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964647","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244451/suppl/GSM2244451_1001000185.B4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"B4"},{"1":"GBM cell 1008","2":"GSM2244452","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000185","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S2","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000185.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422369","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964648","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244452/suppl/GSM2244452_1001000185.D1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Tumor","57":"11","58":"D1"},{"1":"GBM cell 1031","2":"GSM2244475","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: A2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422393","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964671","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244475/suppl/GSM2244475_1001000186.A2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A2"},{"1":"GBM cell 1032","2":"GSM2244476","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422394","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964672","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244476/suppl/GSM2244476_1001000186.E10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"E10"},{"1":"GBM cell 1034","2":"GSM2244478","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422396","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964674","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244478/suppl/GSM2244478_1001000186.H4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H4"},{"1":"GBM cell 1035","2":"GSM2244479","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422397","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964675","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244479/suppl/GSM2244479_1001000186.E1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"E1"},{"1":"GBM cell 1036","2":"GSM2244480","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: G4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422458","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964676","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244480/suppl/GSM2244480_1001000186.G4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"G4"},{"1":"GBM cell 1037","2":"GSM2244481","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422392","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964677","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244481/suppl/GSM2244481_1001000186.D8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"D8"},{"1":"GBM cell 1038","2":"GSM2244482","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422459","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964678","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244482/suppl/GSM2244482_1001000186.H7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H7"},{"1":"GBM cell 1039","2":"GSM2244483","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: G1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422460","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964679","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244483/suppl/GSM2244483_1001000186.G1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"G1"},{"1":"GBM cell 1040","2":"GSM2244484","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422461","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964680","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244484/suppl/GSM2244484_1001000186.D6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"D6"},{"1":"GBM cell 1041","2":"GSM2244485","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422462","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964681","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244485/suppl/GSM2244485_1001000186.E7.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"E7"},{"1":"GBM cell 1042","2":"GSM2244486","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422463","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964682","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244486/suppl/GSM2244486_1001000186.C4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C4"},{"1":"GBM cell 1043","2":"GSM2244487","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: F7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422464","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964683","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244487/suppl/GSM2244487_1001000186.F7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"F7"},{"1":"GBM cell 1044","2":"GSM2244488","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422465","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964684","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244488/suppl/GSM2244488_1001000186.H3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H3"},{"1":"GBM cell 1045","2":"GSM2244489","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: F1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422466","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964685","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244489/suppl/GSM2244489_1001000186.F1.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"F1"},{"1":"GBM cell 1046","2":"GSM2244490","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422467","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964686","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244490/suppl/GSM2244490_1001000186.E11.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"E11"},{"1":"GBM cell 1047","2":"GSM2244491","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422468","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964687","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244491/suppl/GSM2244491_1001000186.D4.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"2","58":"D4"},{"1":"GBM cell 1048","2":"GSM2244492","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: B7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422469","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964688","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244492/suppl/GSM2244492_1001000186.B7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B7"},{"1":"GBM cell 1049","2":"GSM2244493","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422470","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964689","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244493/suppl/GSM2244493_1001000186.D12.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"D12"},{"1":"GBM cell 1050","2":"GSM2244494","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422471","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964690","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244494/suppl/GSM2244494_1001000186.C1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C1"},{"1":"GBM cell 1051","2":"GSM2244495","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422472","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964691","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244495/suppl/GSM2244495_1001000186.H5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H5"},{"1":"GBM cell 1052","2":"GSM2244496","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: A9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422473","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964692","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244496/suppl/GSM2244496_1001000186.A9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"A9"},{"1":"GBM cell 1053","2":"GSM2244497","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: F9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422474","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964693","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244497/suppl/GSM2244497_1001000186.F9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"F9"},{"1":"GBM cell 1054","2":"GSM2244498","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: F11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422475","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964694","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244498/suppl/GSM2244498_1001000186.F11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"F11"},{"1":"GBM cell 1055","2":"GSM2244499","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422476","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964695","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244499/suppl/GSM2244499_1001000186.E6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"E6"},{"1":"GBM cell 1056","2":"GSM2244500","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422477","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964696","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244500/suppl/GSM2244500_1001000186.C7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C7"},{"1":"GBM cell 1057","2":"GSM2244501","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: F5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422478","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964697","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244501/suppl/GSM2244501_1001000186.F5.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"F5"},{"1":"GBM cell 1058","2":"GSM2244502","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422479","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964698","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244502/suppl/GSM2244502_1001000186.C10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C10"},{"1":"GBM cell 1059","2":"GSM2244503","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422480","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964699","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244503/suppl/GSM2244503_1001000186.C6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C6"},{"1":"GBM cell 1060","2":"GSM2244504","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422481","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964700","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244504/suppl/GSM2244504_1001000186.H2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H2"},{"1":"GBM cell 1061","2":"GSM2244505","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422482","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964701","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244505/suppl/GSM2244505_1001000186.D9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"D9"},{"1":"GBM cell 1062","2":"GSM2244506","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422483","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964702","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244506/suppl/GSM2244506_1001000186.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"D1"},{"1":"GBM cell 1063","2":"GSM2244507","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: G9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422484","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964703","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244507/suppl/GSM2244507_1001000186.G9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"G9"},{"1":"GBM cell 1064","2":"GSM2244508","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: A6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422485","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964704","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244508/suppl/GSM2244508_1001000186.A6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A6"},{"1":"GBM cell 1065","2":"GSM2244509","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: B2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422486","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964705","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244509/suppl/GSM2244509_1001000186.B2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B2"},{"1":"GBM cell 1066","2":"GSM2244510","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: A10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422614","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964706","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244510/suppl/GSM2244510_1001000186.A10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A10"},{"1":"GBM cell 1067","2":"GSM2244511","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422487","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964707","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244511/suppl/GSM2244511_1001000186.E9.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"10","58":"E9"},{"1":"GBM cell 1068","2":"GSM2244512","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422608","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964708","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244512/suppl/GSM2244512_1001000186.D7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"8","58":"D7"},{"1":"GBM cell 1069","2":"GSM2244513","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: G10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422609","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964709","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244513/suppl/GSM2244513_1001000186.G10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"G10"},{"1":"GBM cell 1070","2":"GSM2244514","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422610","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964710","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244514/suppl/GSM2244514_1001000186.H6.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"10","58":"H6"},{"1":"GBM cell 1071","2":"GSM2244515","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422611","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964711","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244515/suppl/GSM2244515_1001000186.D11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"D11"},{"1":"GBM cell 1072","2":"GSM2244516","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422612","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964712","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244516/suppl/GSM2244516_1001000186.C12.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C12"},{"1":"GBM cell 1073","2":"GSM2244517","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422613","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964713","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244517/suppl/GSM2244517_1001000186.D5.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"D5"},{"1":"GBM cell 1074","2":"GSM2244518","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: G8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422615","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964714","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244518/suppl/GSM2244518_1001000186.G8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"G8"},{"1":"GBM cell 1075","2":"GSM2244519","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: A12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422616","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964715","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244519/suppl/GSM2244519_1001000186.A12.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A12"},{"1":"GBM cell 1076","2":"GSM2244520","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: E8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422617","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964716","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244520/suppl/GSM2244520_1001000186.E8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"E8"},{"1":"GBM cell 1077","2":"GSM2244521","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: H9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422618","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964717","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244521/suppl/GSM2244521_1001000186.H9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H9"},{"1":"GBM cell 1079","2":"GSM2244523","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: B11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422620","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964719","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244523/suppl/GSM2244523_1001000186.B11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B11"},{"1":"GBM cell 1080","2":"GSM2244524","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: G5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422621","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964720","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244524/suppl/GSM2244524_1001000186.G5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"G5"},{"1":"GBM cell 1082","2":"GSM2244526","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: C9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422623","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964722","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244526/suppl/GSM2244526_1001000186.C9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"C9"},{"1":"GBM cell 1083","2":"GSM2244527","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000186","12":"well: D3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000186.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422624","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964723","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244527/suppl/GSM2244527_1001000186.D3.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"D3"},{"1":"GBM cell 1090","2":"GSM2244534","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: F7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422631","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964730","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244534/suppl/GSM2244534_1001000187.F7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"F7"},{"1":"GBM cell 1091","2":"GSM2244535","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: B9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422632","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964731","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244535/suppl/GSM2244535_1001000187.B9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B9"},{"1":"GBM cell 1092","2":"GSM2244536","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: D1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422633","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964732","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244536/suppl/GSM2244536_1001000187.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"7","58":"D1"},{"1":"GBM cell 1093","2":"GSM2244537","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: B6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422634","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964733","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244537/suppl/GSM2244537_1001000187.B6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B6"},{"1":"GBM cell 1099","2":"GSM2244543","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: G3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422637","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964739","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244543/suppl/GSM2244543_1001000187.G3.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"10","58":"G3"},{"1":"GBM cell 1100","2":"GSM2244544","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: B4","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422398","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964740","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244544/suppl/GSM2244544_1001000187.B4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B4"},{"1":"GBM cell 1102","2":"GSM2244546","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: G10","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422403","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964742","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244546/suppl/GSM2244546_1001000187.G10.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"G10"},{"1":"GBM cell 1103","2":"GSM2244547","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: D2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422404","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964743","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244547/suppl/GSM2244547_1001000187.D2.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"10","58":"D2"},{"1":"GBM cell 1104","2":"GSM2244548","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: C7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422405","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964744","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244548/suppl/GSM2244548_1001000187.C7.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"C7"},{"1":"GBM cell 1105","2":"GSM2244549","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: F5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422406","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964745","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244549/suppl/GSM2244549_1001000187.F5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"F5"},{"1":"GBM cell 1106","2":"GSM2244550","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: A7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422407","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964746","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244550/suppl/GSM2244550_1001000187.A7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A7"},{"1":"GBM cell 1107","2":"GSM2244551","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: F11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422408","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964747","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244551/suppl/GSM2244551_1001000187.F11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"F11"},{"1":"GBM cell 1108","2":"GSM2244552","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: F8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422409","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964748","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244552/suppl/GSM2244552_1001000187.F8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"F8"},{"1":"GBM cell 1109","2":"GSM2244553","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: B2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422410","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964749","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244553/suppl/GSM2244553_1001000187.B2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B2"},{"1":"GBM cell 1110","2":"GSM2244554","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: D8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422411","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964750","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244554/suppl/GSM2244554_1001000187.D8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"D8"},{"1":"GBM cell 1111","2":"GSM2244555","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: E6","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422412","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964751","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244555/suppl/GSM2244555_1001000187.E6.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"E6"},{"1":"GBM cell 1112","2":"GSM2244556","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: H8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422413","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964752","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244556/suppl/GSM2244556_1001000187.H8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H8"},{"1":"GBM cell 1113","2":"GSM2244557","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: E11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422414","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964753","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244557/suppl/GSM2244557_1001000187.E11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"E11"},{"1":"GBM cell 1114","2":"GSM2244558","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: C2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422415","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964754","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244558/suppl/GSM2244558_1001000187.C2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C2"},{"1":"GBM cell 1115","2":"GSM2244559","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: H1","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422416","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964755","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244559/suppl/GSM2244559_1001000187.H1.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"10","58":"H1"},{"1":"GBM cell 1116","2":"GSM2244560","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: C5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422417","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964756","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244560/suppl/GSM2244560_1001000187.C5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"C5"},{"1":"GBM cell 1118","2":"GSM2244562","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: H3","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422419","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964758","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244562/suppl/GSM2244562_1001000187.H3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H3"},{"1":"GBM cell 1119","2":"GSM2244563","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: F2","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422420","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964759","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244563/suppl/GSM2244563_1001000187.F2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"F2"},{"1":"GBM cell 1120","2":"GSM2244564","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: B7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422421","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964760","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244564/suppl/GSM2244564_1001000187.B7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B7"},{"1":"GBM cell 1121","2":"GSM2244565","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: A12","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422422","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964761","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244565/suppl/GSM2244565_1001000187.A12.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A12"},{"1":"GBM cell 1122","2":"GSM2244566","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: A9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422423","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964762","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244566/suppl/GSM2244566_1001000187.A9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"A9"},{"1":"GBM cell 1123","2":"GSM2244567","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: D7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422424","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964763","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244567/suppl/GSM2244567_1001000187.D7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"D7"},{"1":"GBM cell 1124","2":"GSM2244568","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: H7","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422425","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964764","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244568/suppl/GSM2244568_1001000187.H7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"H7"},{"1":"GBM cell 1125","2":"GSM2244569","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: B11","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422426","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964765","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244569/suppl/GSM2244569_1001000187.B11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"9","58":"B11"},{"1":"GBM cell 1127","2":"GSM2244571","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: G5","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 5","16":"cell type: Neuron","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422361","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964767","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244571/suppl/GSM2244571_1001000187.G5.csv.gz","50":"Neuron","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"5","58":"G5"},{"1":"GBM cell 1148","2":"GSM2244592","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000187","12":"well: F9","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Neurons(Thy1)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000187.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423034","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964788","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244592/suppl/GSM2244592_1001000187.F9.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Neurons(Thy1)","56":"Periphery","57":"10","58":"F9"},{"1":"GBM cell 1169","2":"GSM2244613","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000194","12":"well: C8","13":"tissue: Periphery","14":"patient id: BT_S2","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000194.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422972","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964809","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244613/suppl/GSM2244613_1001000194.C8.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S2","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Periphery","57":"2","58":"C8"},{"1":"GBM cell 1215","2":"GSM2244659","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422710","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964855","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244659/suppl/GSM2244659_1001000012.F9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F9"},{"1":"GBM cell 1219","2":"GSM2244663","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422715","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964859","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244663/suppl/GSM2244663_1001000012.F8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F8"},{"1":"GBM cell 1223","2":"GSM2244667","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422725","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964863","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244667/suppl/GSM2244667_1001000012.D8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D8"},{"1":"GBM cell 1224","2":"GSM2244668","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: G10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422727","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964864","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244668/suppl/GSM2244668_1001000012.G10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G10"},{"1":"GBM cell 1225","2":"GSM2244669","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422728","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964865","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244669/suppl/GSM2244669_1001000012.B3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B3"},{"1":"GBM cell 1226","2":"GSM2244670","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422730","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964866","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244670/suppl/GSM2244670_1001000012.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C1"},{"1":"GBM cell 1227","2":"GSM2244671","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422732","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964867","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244671/suppl/GSM2244671_1001000012.C9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C9"},{"1":"GBM cell 1228","2":"GSM2244672","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422733","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964868","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244672/suppl/GSM2244672_1001000012.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B5"},{"1":"GBM cell 1229","2":"GSM2244673","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422735","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964869","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244673/suppl/GSM2244673_1001000012.C10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C10"},{"1":"GBM cell 1230","2":"GSM2244674","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: A6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422737","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964870","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244674/suppl/GSM2244674_1001000012.A6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A6"},{"1":"GBM cell 1231","2":"GSM2244675","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: C2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422740","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964871","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244675/suppl/GSM2244675_1001000012.C2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C2"},{"1":"GBM cell 1232","2":"GSM2244676","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: G6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.G6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422742","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964872","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244676/suppl/GSM2244676_1001000012.G6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G6"},{"1":"GBM cell 1233","2":"GSM2244677","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422745","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964873","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244677/suppl/GSM2244677_1001000012.D4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D4"},{"1":"GBM cell 1234","2":"GSM2244678","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422748","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964874","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244678/suppl/GSM2244678_1001000012.B7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B7"},{"1":"GBM cell 1235","2":"GSM2244679","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422749","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964875","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244679/suppl/GSM2244679_1001000012.B10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"B10"},{"1":"GBM cell 1236","2":"GSM2244680","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422752","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964876","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244680/suppl/GSM2244680_1001000012.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B11"},{"1":"GBM cell 1237","2":"GSM2244681","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422754","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964877","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244681/suppl/GSM2244681_1001000012.A10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A10"},{"1":"GBM cell 1238","2":"GSM2244682","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422755","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964878","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244682/suppl/GSM2244682_1001000012.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"11","58":"D2"},{"1":"GBM cell 1239","2":"GSM2244683","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422757","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964879","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244683/suppl/GSM2244683_1001000012.D3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D3"},{"1":"GBM cell 1240","2":"GSM2244684","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422759","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964880","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244684/suppl/GSM2244684_1001000012.H2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H2"},{"1":"GBM cell 1241","2":"GSM2244685","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422761","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964881","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244685/suppl/GSM2244685_1001000012.A3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A3"},{"1":"GBM cell 1242","2":"GSM2244686","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422763","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964882","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244686/suppl/GSM2244686_1001000012.B8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B8"},{"1":"GBM cell 1243","2":"GSM2244687","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422765","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964883","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244687/suppl/GSM2244687_1001000012.D11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D11"},{"1":"GBM cell 1244","2":"GSM2244688","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: H10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422767","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964884","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244688/suppl/GSM2244688_1001000012.H10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H10"},{"1":"GBM cell 1245","2":"GSM2244689","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422768","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964885","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244689/suppl/GSM2244689_1001000012.B6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B6"},{"1":"GBM cell 1246","2":"GSM2244690","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423176","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964886","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244690/suppl/GSM2244690_1001000012.B4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B4"},{"1":"GBM cell 1247","2":"GSM2244691","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422770","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964887","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244691/suppl/GSM2244691_1001000012.C8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C8"},{"1":"GBM cell 1248","2":"GSM2244692","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423171","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964888","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244692/suppl/GSM2244692_1001000012.D1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D1"},{"1":"GBM cell 1249","2":"GSM2244693","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423172","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964889","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244693/suppl/GSM2244693_1001000012.H7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H7"},{"1":"GBM cell 1250","2":"GSM2244694","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423173","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964890","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244694/suppl/GSM2244694_1001000012.G3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G3"},{"1":"GBM cell 1251","2":"GSM2244695","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423174","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964891","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244695/suppl/GSM2244695_1001000012.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H1"},{"1":"GBM cell 1253","2":"GSM2244697","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423177","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964893","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244697/suppl/GSM2244697_1001000012.F5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F5"},{"1":"GBM cell 1254","2":"GSM2244698","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423178","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964894","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244698/suppl/GSM2244698_1001000012.D9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D9"},{"1":"GBM cell 1257","2":"GSM2244701","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423181","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964897","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244701/suppl/GSM2244701_1001000012.E7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"E7"},{"1":"GBM cell 1258","2":"GSM2244702","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423182","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964898","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244702/suppl/GSM2244702_1001000012.H8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H8"},{"1":"GBM cell 1259","2":"GSM2244703","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: H3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423183","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964899","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244703/suppl/GSM2244703_1001000012.H3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H3"},{"1":"GBM cell 1260","2":"GSM2244704","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000012","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000012.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423184","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964900","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244704/suppl/GSM2244704_1001000012.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G2"},{"1":"GBM cell 1279","2":"GSM2244723","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000014","12":"well: F6","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000014.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423201","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964919","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244723/suppl/GSM2244723_1001000014.F6.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Periphery","57":"2","58":"F6"},{"1":"GBM cell 1294","2":"GSM2244738","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425219","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964934","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244738/suppl/GSM2244738_1001000017.A11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"A11"},{"1":"GBM cell 1295","2":"GSM2244739","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425220","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964935","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244739/suppl/GSM2244739_1001000017.H7.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"H7"},{"1":"GBM cell 1297","2":"GSM2244741","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425222","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964937","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244741/suppl/GSM2244741_1001000017.A8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"A8"},{"1":"GBM cell 1302","2":"GSM2244746","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425227","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964942","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244746/suppl/GSM2244746_1001000017.D10.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"D10"},{"1":"GBM cell 1303","2":"GSM2244747","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425228","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964943","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244747/suppl/GSM2244747_1001000017.C9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"C9"},{"1":"GBM cell 1304","2":"GSM2244748","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: E12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425290","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964944","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244748/suppl/GSM2244748_1001000017.E12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"1","58":"E12"},{"1":"GBM cell 1305","2":"GSM2244749","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425291","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964945","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244749/suppl/GSM2244749_1001000017.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"11","58":"E10"},{"1":"GBM cell 1306","2":"GSM2244750","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425292","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964946","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244750/suppl/GSM2244750_1001000017.B1.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"B1"},{"1":"GBM cell 1307","2":"GSM2244751","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: G1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425229","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964947","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244751/suppl/GSM2244751_1001000017.G1.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"G1"},{"1":"GBM cell 1308","2":"GSM2244752","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425230","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964948","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244752/suppl/GSM2244752_1001000017.D9.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"D9"},{"1":"GBM cell 1309","2":"GSM2244753","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425231","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964949","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244753/suppl/GSM2244753_1001000017.B3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"B3"},{"1":"GBM cell 1310","2":"GSM2244754","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425289","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964950","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244754/suppl/GSM2244754_1001000017.F10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"8","58":"F10"},{"1":"GBM cell 1311","2":"GSM2244755","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425293","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964951","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244755/suppl/GSM2244755_1001000017.E4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"8","58":"E4"},{"1":"GBM cell 1312","2":"GSM2244756","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425294","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964952","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244756/suppl/GSM2244756_1001000017.B5.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"B5"},{"1":"GBM cell 1314","2":"GSM2244758","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425296","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964954","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244758/suppl/GSM2244758_1001000017.E5.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"E5"},{"1":"GBM cell 1315","2":"GSM2244759","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: G5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425297","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964955","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244759/suppl/GSM2244759_1001000017.G5.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"G5"},{"1":"GBM cell 1316","2":"GSM2244760","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: D8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425298","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964956","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244760/suppl/GSM2244760_1001000017.D8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"D8"},{"1":"GBM cell 1317","2":"GSM2244761","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425299","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964957","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244761/suppl/GSM2244761_1001000017.F5.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"F5"},{"1":"GBM cell 1318","2":"GSM2244762","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425300","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964958","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244762/suppl/GSM2244762_1001000017.G9.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"G9"},{"1":"GBM cell 1319","2":"GSM2244763","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425301","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964959","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244763/suppl/GSM2244763_1001000017.D1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"D1"},{"1":"GBM cell 1320","2":"GSM2244764","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425302","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964960","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244764/suppl/GSM2244764_1001000017.F7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"11","58":"F7"},{"1":"GBM cell 1322","2":"GSM2244766","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000017","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000017.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425304","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964962","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244766/suppl/GSM2244766_1001000017.H4.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"H4"},{"1":"GBM cell 1323","2":"GSM2244767","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000018","12":"well: C1","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000018.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425305","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964963","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244767/suppl/GSM2244767_1001000018.C1.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"2","58":"C1"},{"1":"GBM cell 1324","2":"GSM2244768","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000018","12":"well: D10","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000018.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425306","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964964","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244768/suppl/GSM2244768_1001000018.D10.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"D10"},{"1":"GBM cell 1325","2":"GSM2244769","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000018","12":"well: F11","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000018.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425307","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964965","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244769/suppl/GSM2244769_1001000018.F11.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"F11"},{"1":"GBM cell 1326","2":"GSM2244770","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: B5","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425308","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964966","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244770/suppl/GSM2244770_1001000019.B5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"B5"},{"1":"GBM cell 1327","2":"GSM2244771","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: C6","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425309","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964967","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244771/suppl/GSM2244771_1001000019.C6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"C6"},{"1":"GBM cell 1328","2":"GSM2244772","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: C2","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425310","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964968","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244772/suppl/GSM2244772_1001000019.C2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"C2"},{"1":"GBM cell 1329","2":"GSM2244773","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: B12","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425311","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964969","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244773/suppl/GSM2244773_1001000019.B12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"7","58":"B12"},{"1":"GBM cell 1330","2":"GSM2244774","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: G9","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425312","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964970","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244774/suppl/GSM2244774_1001000019.G9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"G9"},{"1":"GBM cell 1331","2":"GSM2244775","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: F6","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425313","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964971","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244775/suppl/GSM2244775_1001000019.F6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"F6"},{"1":"GBM cell 1332","2":"GSM2244776","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: F9","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425314","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964972","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244776/suppl/GSM2244776_1001000019.F9.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"2","58":"F9"},{"1":"GBM cell 1333","2":"GSM2244777","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: B10","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425315","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964973","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244777/suppl/GSM2244777_1001000019.B10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"B10"},{"1":"GBM cell 1334","2":"GSM2244778","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: C1","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425316","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964974","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244778/suppl/GSM2244778_1001000019.C1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"C1"},{"1":"GBM cell 1335","2":"GSM2244779","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: A3","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425317","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964975","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244779/suppl/GSM2244779_1001000019.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"A3"},{"1":"GBM cell 1336","2":"GSM2244780","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: D5","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425318","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964976","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244780/suppl/GSM2244780_1001000019.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"D5"},{"1":"GBM cell 1337","2":"GSM2244781","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: G12","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424415","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964977","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244781/suppl/GSM2244781_1001000019.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"8","58":"G12"},{"1":"GBM cell 1338","2":"GSM2244782","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000019","12":"well: E3","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000019.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424417","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964978","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244782/suppl/GSM2244782_1001000019.E3.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Periphery","57":"2","58":"E3"},{"1":"GBM cell 1339","2":"GSM2244783","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000028","12":"well: D10","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000028.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424419","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964979","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244783/suppl/GSM2244783_1001000028.D10.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"10","58":"D10"},{"1":"GBM cell 1340","2":"GSM2244784","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000028","12":"well: F6","13":"tissue: Periphery","14":"patient id: BT_S1","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000028.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424421","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964980","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244784/suppl/GSM2244784_1001000028.F6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Periphery","57":"11","58":"F6"},{"1":"GBM cell 1342","2":"GSM2244786","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424425","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964982","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244786/suppl/GSM2244786_1001000031.B4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B4"},{"1":"GBM cell 1344","2":"GSM2244788","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424427","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964984","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244788/suppl/GSM2244788_1001000031.F12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F12"},{"1":"GBM cell 1345","2":"GSM2244789","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424428","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964985","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244789/suppl/GSM2244789_1001000031.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B11"},{"1":"GBM cell 1348","2":"GSM2244792","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424436","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964988","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244792/suppl/GSM2244792_1001000031.C4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"9","58":"C4"},{"1":"GBM cell 1349","2":"GSM2244793","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424438","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964989","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244793/suppl/GSM2244793_1001000031.F6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F6"},{"1":"GBM cell 1350","2":"GSM2244794","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424439","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964990","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244794/suppl/GSM2244794_1001000031.B7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B7"},{"1":"GBM cell 1356","2":"GSM2244800","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423227","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1964996","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244800/suppl/GSM2244800_1001000031.G3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"G3"},{"1":"GBM cell 1370","2":"GSM2244814","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 12","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423426","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965010","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244814/suppl/GSM2244814_1001000031.E10.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"12","58":"E10"},{"1":"GBM cell 1372","2":"GSM2244816","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423434","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965012","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244816/suppl/GSM2244816_1001000031.D6.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"D6"},{"1":"GBM cell 1373","2":"GSM2244817","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423436","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965013","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244817/suppl/GSM2244817_1001000031.D1.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"D1"},{"1":"GBM cell 1374","2":"GSM2244818","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: H5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423438","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965014","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244818/suppl/GSM2244818_1001000031.H5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"H5"},{"1":"GBM cell 1375","2":"GSM2244819","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423440","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965015","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244819/suppl/GSM2244819_1001000031.A11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"A11"},{"1":"GBM cell 1377","2":"GSM2244821","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 6","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423444","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965017","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244821/suppl/GSM2244821_1001000031.B10.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"6","58":"B10"},{"1":"GBM cell 1378","2":"GSM2244822","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423446","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965018","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244822/suppl/GSM2244822_1001000031.E3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"E3"},{"1":"GBM cell 1379","2":"GSM2244823","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: C7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423448","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965019","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244823/suppl/GSM2244823_1001000031.C7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C7"},{"1":"GBM cell 1380","2":"GSM2244824","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423450","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965020","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244824/suppl/GSM2244824_1001000031.H7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"H7"},{"1":"GBM cell 1382","2":"GSM2244826","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423454","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965022","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244826/suppl/GSM2244826_1001000031.G7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"G7"},{"1":"GBM cell 1383","2":"GSM2244827","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423456","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965023","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244827/suppl/GSM2244827_1001000031.F11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F11"},{"1":"GBM cell 1385","2":"GSM2244829","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423460","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965025","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244829/suppl/GSM2244829_1001000031.F1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F1"},{"1":"GBM cell 1386","2":"GSM2244830","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423462","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965026","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244830/suppl/GSM2244830_1001000031.G12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"G12"},{"1":"GBM cell 1387","2":"GSM2244831","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423464","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965027","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244831/suppl/GSM2244831_1001000031.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"E2"},{"1":"GBM cell 1388","2":"GSM2244832","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 6","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423466","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965028","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244832/suppl/GSM2244832_1001000031.E5.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"6","58":"E5"},{"1":"GBM cell 1389","2":"GSM2244833","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423468","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965029","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244833/suppl/GSM2244833_1001000031.E12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"E12"},{"1":"GBM cell 1390","2":"GSM2244834","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423470","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965030","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244834/suppl/GSM2244834_1001000031.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C11"},{"1":"GBM cell 1391","2":"GSM2244835","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423471","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965031","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244835/suppl/GSM2244835_1001000031.A9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"A9"},{"1":"GBM cell 1392","2":"GSM2244836","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423473","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965032","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244836/suppl/GSM2244836_1001000031.C10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"9","58":"C10"},{"1":"GBM cell 1393","2":"GSM2244837","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424358","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965033","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244837/suppl/GSM2244837_1001000031.B9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B9"},{"1":"GBM cell 1394","2":"GSM2244838","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: H10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424360","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965034","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244838/suppl/GSM2244838_1001000031.H10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"H10"},{"1":"GBM cell 1395","2":"GSM2244839","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424362","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965035","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244839/suppl/GSM2244839_1001000031.F7.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"9","58":"F7"},{"1":"GBM cell 1396","2":"GSM2244840","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: G1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 6","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424364","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965036","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244840/suppl/GSM2244840_1001000031.G1.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"6","58":"G1"},{"1":"GBM cell 1397","2":"GSM2244841","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423475","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965037","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244841/suppl/GSM2244841_1001000031.D3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"9","58":"D3"},{"1":"GBM cell 1398","2":"GSM2244842","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: C2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423477","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965038","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244842/suppl/GSM2244842_1001000031.C2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C2"},{"1":"GBM cell 1399","2":"GSM2244843","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: G10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423479","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965039","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244843/suppl/GSM2244843_1001000031.G10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"G10"},{"1":"GBM cell 1400","2":"GSM2244844","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423481","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965040","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244844/suppl/GSM2244844_1001000031.A10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"A10"},{"1":"GBM cell 1401","2":"GSM2244845","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424366","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965041","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244845/suppl/GSM2244845_1001000031.E1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"E1"},{"1":"GBM cell 1402","2":"GSM2244846","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: A7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424368","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965042","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244846/suppl/GSM2244846_1001000031.A7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"A7"},{"1":"GBM cell 1403","2":"GSM2244847","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424370","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965043","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244847/suppl/GSM2244847_1001000031.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C1"},{"1":"GBM cell 1404","2":"GSM2244848","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424372","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965044","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244848/suppl/GSM2244848_1001000031.F9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F9"},{"1":"GBM cell 1408","2":"GSM2244852","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424378","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965048","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244852/suppl/GSM2244852_1001000031.E4.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"9","58":"E4"},{"1":"GBM cell 1409","2":"GSM2244853","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424380","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965049","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244853/suppl/GSM2244853_1001000031.B12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B12"},{"1":"GBM cell 1412","2":"GSM2244856","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424386","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965052","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244856/suppl/GSM2244856_1001000031.D12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"D12"},{"1":"GBM cell 1413","2":"GSM2244857","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: E6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424388","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965053","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244857/suppl/GSM2244857_1001000031.E6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"E6"},{"1":"GBM cell 1414","2":"GSM2244858","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: F4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424391","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965054","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244858/suppl/GSM2244858_1001000031.F4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F4"},{"1":"GBM cell 1415","2":"GSM2244859","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000031","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000031.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424392","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965055","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244859/suppl/GSM2244859_1001000031.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"H1"},{"1":"GBM cell 1418","2":"GSM2244862","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 12","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424398","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965058","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244862/suppl/GSM2244862_1001000032.F7.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"12","58":"F7"},{"1":"GBM cell 1419","2":"GSM2244863","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424400","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965059","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244863/suppl/GSM2244863_1001000032.C4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C4"},{"1":"GBM cell 1421","2":"GSM2244865","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424403","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965061","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244865/suppl/GSM2244865_1001000032.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"11","58":"D2"},{"1":"GBM cell 1422","2":"GSM2244866","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424407","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965062","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244866/suppl/GSM2244866_1001000032.F8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"F8"},{"1":"GBM cell 1426","2":"GSM2244870","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423365","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965066","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244870/suppl/GSM2244870_1001000032.A10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"A10"},{"1":"GBM cell 1428","2":"GSM2244872","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423367","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965068","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244872/suppl/GSM2244872_1001000032.A5.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"A5"},{"1":"GBM cell 1429","2":"GSM2244873","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423369","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965069","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244873/suppl/GSM2244873_1001000032.F1.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"F1"},{"1":"GBM cell 1431","2":"GSM2244875","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423374","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965071","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244875/suppl/GSM2244875_1001000032.E2.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"E2"},{"1":"GBM cell 1432","2":"GSM2244876","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423376","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965072","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244876/suppl/GSM2244876_1001000032.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B5"},{"1":"GBM cell 1433","2":"GSM2244877","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423378","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965073","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244877/suppl/GSM2244877_1001000032.A9.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"A9"},{"1":"GBM cell 1434","2":"GSM2244878","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423380","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965074","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244878/suppl/GSM2244878_1001000032.D6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"D6"},{"1":"GBM cell 1435","2":"GSM2244879","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423381","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965075","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244879/suppl/GSM2244879_1001000032.D12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"D12"},{"1":"GBM cell 1442","2":"GSM2244886","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 6","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423395","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965082","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244886/suppl/GSM2244886_1001000032.D11.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"6","58":"D11"},{"1":"GBM cell 1446","2":"GSM2244890","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423402","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965086","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244890/suppl/GSM2244890_1001000032.C8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C8"},{"1":"GBM cell 1458","2":"GSM2244902","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423419","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965098","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244902/suppl/GSM2244902_1001000032.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"C11"},{"1":"GBM cell 1459","2":"GSM2244903","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 12","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423421","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965099","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244903/suppl/GSM2244903_1001000032.F10.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"12","58":"F10"},{"1":"GBM cell 1460","2":"GSM2244904","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423312","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965100","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244904/suppl/GSM2244904_1001000032.E5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"E5"},{"1":"GBM cell 1461","2":"GSM2244905","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: B2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423314","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965101","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244905/suppl/GSM2244905_1001000032.B2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"1","58":"B2"},{"1":"GBM cell 1462","2":"GSM2244906","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000032","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 3","16":"cell type: Vascular","17":"neoplastic: Regular","18":"selection: Endothelial(BSC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000032.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423316","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965102","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244906/suppl/GSM2244906_1001000032.E7.csv.gz","50":"Vascular","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Endothelial(BSC)","56":"Tumor","57":"3","58":"E7"},{"1":"GBM cell 1463","2":"GSM2244907","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423318","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965103","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244907/suppl/GSM2244907_1001000033.B11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B11"},{"1":"GBM cell 1464","2":"GSM2244908","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423320","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965104","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244908/suppl/GSM2244908_1001000033.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A11"},{"1":"GBM cell 1465","2":"GSM2244909","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423321","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965105","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244909/suppl/GSM2244909_1001000033.A5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A5"},{"1":"GBM cell 1467","2":"GSM2244911","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423324","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965107","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244911/suppl/GSM2244911_1001000033.D12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D12"},{"1":"GBM cell 1468","2":"GSM2244912","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423326","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965108","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244912/suppl/GSM2244912_1001000033.E4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E4"},{"1":"GBM cell 1469","2":"GSM2244913","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423328","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965109","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244913/suppl/GSM2244913_1001000033.B12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B12"},{"1":"GBM cell 1470","2":"GSM2244914","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423330","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965110","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244914/suppl/GSM2244914_1001000033.E10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E10"},{"1":"GBM cell 1510","2":"GSM2244954","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: G6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.G6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423293","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965150","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244954/suppl/GSM2244954_1001000033.G6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G6"},{"1":"GBM cell 1513","2":"GSM2244957","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423301","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965153","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244957/suppl/GSM2244957_1001000033.E7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E7"},{"1":"GBM cell 1515","2":"GSM2244959","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423305","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965155","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244959/suppl/GSM2244959_1001000033.E1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E1"},{"1":"GBM cell 1516","2":"GSM2244960","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: G10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424297","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965156","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244960/suppl/GSM2244960_1001000033.G10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"G10"},{"1":"GBM cell 1517","2":"GSM2244961","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423295","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965157","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244961/suppl/GSM2244961_1001000033.B9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B9"},{"1":"GBM cell 1541","2":"GSM2244985","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000033","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000033.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424344","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965181","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2244nnn/GSM2244985/suppl/GSM2244985_1001000033.E3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E3"},{"1":"GBM cell 1556","2":"GSM2245000","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424236","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965196","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245000/suppl/GSM2245000_1001000036.D11.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"D11"},{"1":"GBM cell 1557","2":"GSM2245001","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424237","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965197","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245001/suppl/GSM2245001_1001000036.C1.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"C1"},{"1":"GBM cell 1558","2":"GSM2245002","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424238","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965198","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245002/suppl/GSM2245002_1001000036.B9.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"B9"},{"1":"GBM cell 1559","2":"GSM2245003","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424239","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965199","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245003/suppl/GSM2245003_1001000036.D6.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"D6"},{"1":"GBM cell 1561","2":"GSM2245005","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: H5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424241","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965201","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245005/suppl/GSM2245005_1001000036.H5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"7","58":"H5"},{"1":"GBM cell 1562","2":"GSM2245006","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424242","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965202","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245006/suppl/GSM2245006_1001000036.G3.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"G3"},{"1":"GBM cell 1563","2":"GSM2245007","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: F4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424243","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965203","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245007/suppl/GSM2245007_1001000036.F4.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"F4"},{"1":"GBM cell 1564","2":"GSM2245008","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424244","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965204","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245008/suppl/GSM2245008_1001000036.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"7","58":"A11"},{"1":"GBM cell 1565","2":"GSM2245009","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424245","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965205","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245009/suppl/GSM2245009_1001000036.D10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"8","58":"D10"},{"1":"GBM cell 1566","2":"GSM2245010","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424246","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965206","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245010/suppl/GSM2245010_1001000036.B7.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"B7"},{"1":"GBM cell 1568","2":"GSM2245012","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424248","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965208","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245012/suppl/GSM2245012_1001000036.C11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"C11"},{"1":"GBM cell 1569","2":"GSM2245013","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424249","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965209","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245013/suppl/GSM2245013_1001000036.B5.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"B5"},{"1":"GBM cell 1570","2":"GSM2245014","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: E9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.E9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424250","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965210","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245014/suppl/GSM2245014_1001000036.E9.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"9","58":"E9"},{"1":"GBM cell 1571","2":"GSM2245015","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: E11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424251","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965211","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245015/suppl/GSM2245015_1001000036.E11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"8","58":"E11"},{"1":"GBM cell 1572","2":"GSM2245016","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424252","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965212","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245016/suppl/GSM2245016_1001000036.E5.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"E5"},{"1":"GBM cell 1573","2":"GSM2245017","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424253","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965213","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245017/suppl/GSM2245017_1001000036.D2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"8","58":"D2"},{"1":"GBM cell 1574","2":"GSM2245018","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424254","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965214","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245018/suppl/GSM2245018_1001000036.F9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"1","58":"F9"},{"1":"GBM cell 1575","2":"GSM2245019","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424255","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965215","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245019/suppl/GSM2245019_1001000036.A8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"8","58":"A8"},{"1":"GBM cell 1576","2":"GSM2245020","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000036","12":"well: E12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Oligodendrocytes(GC)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000036.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424256","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965216","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245020/suppl/GSM2245020_1001000036.E12.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Oligodendrocytes(GC)","56":"Tumor","57":"2","58":"E12"},{"1":"GBM cell 1577","2":"GSM2245021","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424257","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965217","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245021/suppl/GSM2245021_1001000037.B8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B8"},{"1":"GBM cell 1578","2":"GSM2245022","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424258","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965218","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245022/suppl/GSM2245022_1001000037.G9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G9"},{"1":"GBM cell 1579","2":"GSM2245023","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424259","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965219","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245023/suppl/GSM2245023_1001000037.C9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C9"},{"1":"GBM cell 1580","2":"GSM2245024","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424260","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965220","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245024/suppl/GSM2245024_1001000037.F8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F8"},{"1":"GBM cell 1581","2":"GSM2245025","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424261","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965221","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245025/suppl/GSM2245025_1001000037.C8.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"9","58":"C8"},{"1":"GBM cell 1582","2":"GSM2245026","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424262","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965222","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245026/suppl/GSM2245026_1001000037.F1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F1"},{"1":"GBM cell 1583","2":"GSM2245027","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424263","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965223","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245027/suppl/GSM2245027_1001000037.C11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C11"},{"1":"GBM cell 1584","2":"GSM2245028","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424264","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965224","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245028/suppl/GSM2245028_1001000037.B10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B10"},{"1":"GBM cell 1585","2":"GSM2245029","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424265","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965225","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245029/suppl/GSM2245029_1001000037.G1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G1"},{"1":"GBM cell 1586","2":"GSM2245030","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424266","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965226","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245030/suppl/GSM2245030_1001000037.A8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A8"},{"1":"GBM cell 1587","2":"GSM2245031","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424267","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965227","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245031/suppl/GSM2245031_1001000037.C6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"8","58":"C6"},{"1":"GBM cell 1588","2":"GSM2245032","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424268","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965228","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245032/suppl/GSM2245032_1001000037.B12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B12"},{"1":"GBM cell 1589","2":"GSM2245033","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424269","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965229","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245033/suppl/GSM2245033_1001000037.C7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C7"},{"1":"GBM cell 1590","2":"GSM2245034","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424270","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965230","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245034/suppl/GSM2245034_1001000037.E10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"E10"},{"1":"GBM cell 1591","2":"GSM2245035","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424271","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965231","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245035/suppl/GSM2245035_1001000037.D7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D7"},{"1":"GBM cell 1592","2":"GSM2245036","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424272","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965232","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245036/suppl/GSM2245036_1001000037.F12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F12"},{"1":"GBM cell 1593","2":"GSM2245037","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424273","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965233","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245037/suppl/GSM2245037_1001000037.B11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B11"},{"1":"GBM cell 1594","2":"GSM2245038","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424274","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965234","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245038/suppl/GSM2245038_1001000037.E8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"E8"},{"1":"GBM cell 1595","2":"GSM2245039","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424275","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965235","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245039/suppl/GSM2245039_1001000037.A5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A5"},{"1":"GBM cell 1596","2":"GSM2245040","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424276","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965236","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245040/suppl/GSM2245040_1001000037.G11.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G11"},{"1":"GBM cell 1597","2":"GSM2245041","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424278","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965237","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245041/suppl/GSM2245041_1001000037.E7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"E7"},{"1":"GBM cell 1598","2":"GSM2245042","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424279","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965238","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245042/suppl/GSM2245042_1001000037.G7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G7"},{"1":"GBM cell 1599","2":"GSM2245043","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424281","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965239","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245043/suppl/GSM2245043_1001000037.B7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B7"},{"1":"GBM cell 1600","2":"GSM2245044","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424283","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965240","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245044/suppl/GSM2245044_1001000037.B1.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"9","58":"B1"},{"1":"GBM cell 1601","2":"GSM2245045","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424285","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965241","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245045/suppl/GSM2245045_1001000037.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G2"},{"1":"GBM cell 1603","2":"GSM2245047","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424289","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965243","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245047/suppl/GSM2245047_1001000037.D10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D10"},{"1":"GBM cell 1604","2":"GSM2245048","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424291","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965244","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245048/suppl/GSM2245048_1001000037.D3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D3"},{"1":"GBM cell 1606","2":"GSM2245050","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424295","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965246","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245050/suppl/GSM2245050_1001000037.F10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F10"},{"1":"GBM cell 1607","2":"GSM2245051","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425319","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965247","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245051/suppl/GSM2245051_1001000037.D2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D2"},{"1":"GBM cell 1608","2":"GSM2245052","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425320","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965248","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245052/suppl/GSM2245052_1001000037.C12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C12"},{"1":"GBM cell 1609","2":"GSM2245053","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425321","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965249","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245053/suppl/GSM2245053_1001000037.B5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B5"},{"1":"GBM cell 1610","2":"GSM2245054","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425322","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965250","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245054/suppl/GSM2245054_1001000037.A10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A10"},{"1":"GBM cell 1611","2":"GSM2245055","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: E2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.E2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425323","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965251","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245055/suppl/GSM2245055_1001000037.E2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"E2"},{"1":"GBM cell 1612","2":"GSM2245056","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: H10","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425324","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965252","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245056/suppl/GSM2245056_1001000037.H10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H10"},{"1":"GBM cell 1613","2":"GSM2245057","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425325","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965253","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245057/suppl/GSM2245057_1001000037.G4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G4"},{"1":"GBM cell 1614","2":"GSM2245058","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425326","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965254","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245058/suppl/GSM2245058_1001000037.D4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D4"},{"1":"GBM cell 1615","2":"GSM2245059","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425327","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965255","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245059/suppl/GSM2245059_1001000037.D5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D5"},{"1":"GBM cell 1616","2":"GSM2245060","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425328","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965256","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245060/suppl/GSM2245060_1001000037.F5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F5"},{"1":"GBM cell 1617","2":"GSM2245061","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425329","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965257","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245061/suppl/GSM2245061_1001000037.F7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F7"},{"1":"GBM cell 1618","2":"GSM2245062","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C3","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425330","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965258","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245062/suppl/GSM2245062_1001000037.C3.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C3"},{"1":"GBM cell 1619","2":"GSM2245063","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425331","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965259","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245063/suppl/GSM2245063_1001000037.G12.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"G12"},{"1":"GBM cell 1620","2":"GSM2245064","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425332","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965260","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245064/suppl/GSM2245064_1001000037.D1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D1"},{"1":"GBM cell 1621","2":"GSM2245065","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425333","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965261","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245065/suppl/GSM2245065_1001000037.F2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F2"},{"1":"GBM cell 1623","2":"GSM2245067","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425335","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965263","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245067/suppl/GSM2245067_1001000037.H8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H8"},{"1":"GBM cell 1624","2":"GSM2245068","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: H6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425336","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965264","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245068/suppl/GSM2245068_1001000037.H6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H6"},{"1":"GBM cell 1638","2":"GSM2245082","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C5","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425350","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965278","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245082/suppl/GSM2245082_1001000037.C5.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C5"},{"1":"GBM cell 1642","2":"GSM2245086","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425354","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965282","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245086/suppl/GSM2245086_1001000037.E4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"E4"},{"1":"GBM cell 1645","2":"GSM2245089","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425357","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965285","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245089/suppl/GSM2245089_1001000037.H1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H1"},{"1":"GBM cell 1646","2":"GSM2245090","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C2","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425358","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965286","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245090/suppl/GSM2245090_1001000037.C2.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"9","58":"C2"},{"1":"GBM cell 1650","2":"GSM2245094","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425362","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965290","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245094/suppl/GSM2245094_1001000037.D9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D9"},{"1":"GBM cell 1651","2":"GSM2245095","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F4","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425363","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965291","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245095/suppl/GSM2245095_1001000037.F4.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F4"},{"1":"GBM cell 1652","2":"GSM2245096","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425364","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965292","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245096/suppl/GSM2245096_1001000037.A9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"A9"},{"1":"GBM cell 1653","2":"GSM2245097","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425365","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965293","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245097/suppl/GSM2245097_1001000037.F9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"F9"},{"1":"GBM cell 1654","2":"GSM2245098","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425366","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965294","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245098/suppl/GSM2245098_1001000037.B9.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"B9"},{"1":"GBM cell 1655","2":"GSM2245099","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424215","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965295","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245099/suppl/GSM2245099_1001000037.H7.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"H7"},{"1":"GBM cell 1656","2":"GSM2245100","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424216","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965296","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245100/suppl/GSM2245100_1001000037.D6.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"D6"},{"1":"GBM cell 1657","2":"GSM2245101","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000037","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S1","15":"tsne cluster: 1","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Astrocytes(HEPACAM)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000037.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424217","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965297","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245101/suppl/GSM2245101_1001000037.C1.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S1","54":"1.001e+09","55":"Astrocytes(HEPACAM)","56":"Tumor","57":"1","58":"C1"},{"1":"GBM cell 1660","2":"GSM2245104","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: A12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424220","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965300","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245104/suppl/GSM2245104_1001000229.A12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A12"},{"1":"GBM cell 1662","2":"GSM2245106","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424222","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965302","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245106/suppl/GSM2245106_1001000229.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B3"},{"1":"GBM cell 1664","2":"GSM2245108","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424224","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965304","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245108/suppl/GSM2245108_1001000229.H7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H7"},{"1":"GBM cell 1665","2":"GSM2245109","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424225","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965305","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245109/suppl/GSM2245109_1001000229.C9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C9"},{"1":"GBM cell 1667","2":"GSM2245111","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423141","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965307","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245111/suppl/GSM2245111_1001000229.G9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G9"},{"1":"GBM cell 1668","2":"GSM2245112","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423142","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965308","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245112/suppl/GSM2245112_1001000229.D6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D6"},{"1":"GBM cell 1669","2":"GSM2245113","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423143","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965309","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245113/suppl/GSM2245113_1001000229.C4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C4"},{"1":"GBM cell 1670","2":"GSM2245114","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423144","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965310","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245114/suppl/GSM2245114_1001000229.G2.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G2"},{"1":"GBM cell 1671","2":"GSM2245115","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423145","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965311","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245115/suppl/GSM2245115_1001000229.E7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E7"},{"1":"GBM cell 1672","2":"GSM2245116","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423146","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965312","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245116/suppl/GSM2245116_1001000229.E5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E5"},{"1":"GBM cell 1674","2":"GSM2245118","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: C2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423148","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965314","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245118/suppl/GSM2245118_1001000229.C2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C2"},{"1":"GBM cell 1675","2":"GSM2245119","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: E11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423149","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965315","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245119/suppl/GSM2245119_1001000229.E11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E11"},{"1":"GBM cell 1678","2":"GSM2245122","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: F2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423152","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965318","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245122/suppl/GSM2245122_1001000229.F2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F2"},{"1":"GBM cell 1679","2":"GSM2245123","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423153","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965319","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245123/suppl/GSM2245123_1001000229.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G12"},{"1":"GBM cell 1680","2":"GSM2245124","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423154","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965320","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245124/suppl/GSM2245124_1001000229.F9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F9"},{"1":"GBM cell 1681","2":"GSM2245125","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423155","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965321","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245125/suppl/GSM2245125_1001000229.H9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H9"},{"1":"GBM cell 1682","2":"GSM2245126","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: A7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423156","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965322","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245126/suppl/GSM2245126_1001000229.A7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A7"},{"1":"GBM cell 1683","2":"GSM2245127","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: A2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423157","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965323","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245127/suppl/GSM2245127_1001000229.A2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A2"},{"1":"GBM cell 1684","2":"GSM2245128","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423158","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965324","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245128/suppl/GSM2245128_1001000229.B5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B5"},{"1":"GBM cell 1685","2":"GSM2245129","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: C11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.C11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423159","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965325","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245129/suppl/GSM2245129_1001000229.C11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C11"},{"1":"GBM cell 1686","2":"GSM2245130","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423160","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965326","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245130/suppl/GSM2245130_1001000229.H4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H4"},{"1":"GBM cell 1738","2":"GSM2245182","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: G6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.G6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423072","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965378","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245182/suppl/GSM2245182_1001000229.G6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G6"},{"1":"GBM cell 1739","2":"GSM2245183","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423073","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965379","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245183/suppl/GSM2245183_1001000229.F10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F10"},{"1":"GBM cell 1740","2":"GSM2245184","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423075","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965380","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245184/suppl/GSM2245184_1001000229.C10.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"C10"},{"1":"GBM cell 1744","2":"GSM2245188","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423083","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965384","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245188/suppl/GSM2245188_1001000229.B1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B1"},{"1":"GBM cell 1745","2":"GSM2245189","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423085","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965385","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245189/suppl/GSM2245189_1001000229.E8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E8"},{"1":"GBM cell 1746","2":"GSM2245190","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423087","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965386","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245190/suppl/GSM2245190_1001000229.G11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G11"},{"1":"GBM cell 1747","2":"GSM2245191","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423089","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965387","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245191/suppl/GSM2245191_1001000229.C12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C12"},{"1":"GBM cell 1749","2":"GSM2245193","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423092","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965389","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245193/suppl/GSM2245193_1001000229.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A10"},{"1":"GBM cell 1750","2":"GSM2245194","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000229","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000229.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423094","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965390","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245194/suppl/GSM2245194_1001000229.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F12"},{"1":"GBM cell 1757","2":"GSM2245201","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422893","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965397","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245201/suppl/GSM2245201_1001000230.E8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E8"},{"1":"GBM cell 1758","2":"GSM2245202","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422895","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965398","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245202/suppl/GSM2245202_1001000230.D2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D2"},{"1":"GBM cell 1765","2":"GSM2245209","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422908","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965405","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245209/suppl/GSM2245209_1001000230.F8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F8"},{"1":"GBM cell 1768","2":"GSM2245212","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422914","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965408","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245212/suppl/GSM2245212_1001000230.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A11"},{"1":"GBM cell 1771","2":"GSM2245215","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422920","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965411","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245215/suppl/GSM2245215_1001000230.B9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B9"},{"1":"GBM cell 1779","2":"GSM2245223","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422935","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965419","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245223/suppl/GSM2245223_1001000230.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H1"},{"1":"GBM cell 1783","2":"GSM2245227","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422945","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965423","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245227/suppl/GSM2245227_1001000230.G9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G9"},{"1":"GBM cell 1816","2":"GSM2245260","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: C5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.C5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05422831","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965456","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245260/suppl/GSM2245260_1001000230.C5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C5"},{"1":"GBM cell 1817","2":"GSM2245261","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425259","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965457","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245261/suppl/GSM2245261_1001000230.G11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G11"},{"1":"GBM cell 1818","2":"GSM2245262","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425260","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965458","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245262/suppl/GSM2245262_1001000230.D11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D11"},{"1":"GBM cell 1819","2":"GSM2245263","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: B2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425261","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965459","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245263/suppl/GSM2245263_1001000230.B2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B2"},{"1":"GBM cell 1820","2":"GSM2245264","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425262","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965460","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245264/suppl/GSM2245264_1001000230.F1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F1"},{"1":"GBM cell 1821","2":"GSM2245265","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 11","16":"cell type: Neoplastic","17":"neoplastic: Neoplastic","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425263","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965461","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245265/suppl/GSM2245265_1001000230.G8.csv.gz","50":"Neoplastic","51":"glioblastoma","52":"Neoplastic","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"11","58":"G8"},{"1":"GBM cell 1824","2":"GSM2245268","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425266","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965464","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245268/suppl/GSM2245268_1001000230.C12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C12"},{"1":"GBM cell 1825","2":"GSM2245269","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: C3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425267","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965465","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245269/suppl/GSM2245269_1001000230.C3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C3"},{"1":"GBM cell 1826","2":"GSM2245270","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425268","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965466","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245270/suppl/GSM2245270_1001000230.A8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A8"},{"1":"GBM cell 1827","2":"GSM2245271","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425269","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965467","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245271/suppl/GSM2245271_1001000230.E3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"E3"},{"1":"GBM cell 1828","2":"GSM2245272","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425270","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965468","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245272/suppl/GSM2245272_1001000230.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A10"},{"1":"GBM cell 1829","2":"GSM2245273","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: F3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425271","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965469","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245273/suppl/GSM2245273_1001000230.F3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F3"},{"1":"GBM cell 1830","2":"GSM2245274","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: G1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425272","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965470","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245274/suppl/GSM2245274_1001000230.G1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G1"},{"1":"GBM cell 1831","2":"GSM2245275","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425273","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965471","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245275/suppl/GSM2245275_1001000230.H9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H9"},{"1":"GBM cell 1832","2":"GSM2245276","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425274","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965472","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245276/suppl/GSM2245276_1001000230.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F12"},{"1":"GBM cell 1833","2":"GSM2245277","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425275","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965473","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245277/suppl/GSM2245277_1001000230.H2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H2"},{"1":"GBM cell 1834","2":"GSM2245278","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000230","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000230.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425276","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965474","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245278/suppl/GSM2245278_1001000230.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D5"},{"1":"GBM cell 1835","2":"GSM2245279","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425277","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965475","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245279/suppl/GSM2245279_1001000231.F8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F8"},{"1":"GBM cell 1836","2":"GSM2245280","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425278","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965476","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245280/suppl/GSM2245280_1001000231.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A3"},{"1":"GBM cell 1837","2":"GSM2245281","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425279","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965477","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245281/suppl/GSM2245281_1001000231.F1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F1"},{"1":"GBM cell 1838","2":"GSM2245282","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425280","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965478","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245282/suppl/GSM2245282_1001000231.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H11"},{"1":"GBM cell 1840","2":"GSM2245284","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: F10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.F10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425282","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965480","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245284/suppl/GSM2245284_1001000231.F10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F10"},{"1":"GBM cell 1842","2":"GSM2245286","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425284","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965482","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245286/suppl/GSM2245286_1001000231.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A11"},{"1":"GBM cell 1843","2":"GSM2245287","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425285","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965483","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245287/suppl/GSM2245287_1001000231.B5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"B5"},{"1":"GBM cell 1854","2":"GSM2245298","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425239","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965494","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245298/suppl/GSM2245298_1001000231.D3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D3"},{"1":"GBM cell 1856","2":"GSM2245300","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425241","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965496","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245300/suppl/GSM2245300_1001000231.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G12"},{"1":"GBM cell 1859","2":"GSM2245303","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425244","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965499","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245303/suppl/GSM2245303_1001000231.G8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"G8"},{"1":"GBM cell 1862","2":"GSM2245306","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: C2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.C2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425247","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965502","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245306/suppl/GSM2245306_1001000231.C2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C2"},{"1":"GBM cell 1869","2":"GSM2245313","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: F2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425254","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965509","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245313/suppl/GSM2245313_1001000231.F2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"F2"},{"1":"GBM cell 1870","2":"GSM2245314","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05425255","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965510","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245314/suppl/GSM2245314_1001000231.C8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"C8"},{"1":"GBM cell 1913","2":"GSM2245357","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423590","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965553","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245357/suppl/GSM2245357_1001000231.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A10"},{"1":"GBM cell 1915","2":"GSM2245359","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423593","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965555","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245359/suppl/GSM2245359_1001000231.A5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"A5"},{"1":"GBM cell 1921","2":"GSM2245365","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423599","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965561","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245365/suppl/GSM2245365_1001000231.D2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"D2"},{"1":"GBM cell 1923","2":"GSM2245367","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000231","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000231.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423601","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965563","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245367/suppl/GSM2245367_1001000231.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Tumor","57":"8","58":"H1"},{"1":"GBM cell 1924","2":"GSM2245368","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000232","12":"well: E4","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 10","16":"cell type: Astocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000232.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423602","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965564","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245368/suppl/GSM2245368_1001000232.E4.csv.gz","50":"Astocyte","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"10","58":"E4"},{"1":"GBM cell 1926","2":"GSM2245370","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000232","12":"well: B3","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000232.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423604","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965566","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245370/suppl/GSM2245370_1001000232.B3.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"9","58":"B3"},{"1":"GBM cell 1930","2":"GSM2245374","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000233","12":"well: C6","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000233.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423608","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965570","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245374/suppl/GSM2245374_1001000233.C6.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"2","58":"C6"},{"1":"GBM cell 1931","2":"GSM2245375","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000233","12":"well: C4","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000233.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423609","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965571","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245375/suppl/GSM2245375_1001000233.C4.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"2","58":"C4"},{"1":"GBM cell 1932","2":"GSM2245376","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000233","12":"well: G11","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000233.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423610","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965572","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245376/suppl/GSM2245376_1001000233.G11.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"9","58":"G11"},{"1":"GBM cell 1933","2":"GSM2245377","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000234","12":"well: E10","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 9","16":"cell type: OPC","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000234.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423611","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965573","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245377/suppl/GSM2245377_1001000234.E10.csv.gz","50":"OPC","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"9","58":"E10"},{"1":"GBM cell 1934","2":"GSM2245378","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000234","12":"well: C3","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000234.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423555","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965574","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245378/suppl/GSM2245378_1001000234.C3.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"2","58":"C3"},{"1":"GBM cell 1935","2":"GSM2245379","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000234","12":"well: C9","13":"tissue: Periphery","14":"patient id: BT_S4","15":"tsne cluster: 2","16":"cell type: Oligodendrocyte","17":"neoplastic: Regular","18":"selection: Unpanned","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Periphery","24":"1001000234.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423556","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965575","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245379/suppl/GSM2245379_1001000234.C9.csv.gz","50":"Oligodendrocyte","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Unpanned","56":"Periphery","57":"2","58":"C9"},{"1":"GBM cell 1936","2":"GSM2245380","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423557","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965576","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245380/suppl/GSM2245380_1001000235.G4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G4"},{"1":"GBM cell 1938","2":"GSM2245382","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423613","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965578","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245382/suppl/GSM2245382_1001000235.A9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A9"},{"1":"GBM cell 1940","2":"GSM2245384","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423558","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965580","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245384/suppl/GSM2245384_1001000235.D4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D4"},{"1":"GBM cell 1941","2":"GSM2245385","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423559","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965581","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245385/suppl/GSM2245385_1001000235.A11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A11"},{"1":"GBM cell 1942","2":"GSM2245386","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423560","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965582","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245386/suppl/GSM2245386_1001000235.G8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G8"},{"1":"GBM cell 1943","2":"GSM2245387","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423561","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965583","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245387/suppl/GSM2245387_1001000235.G2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G2"},{"1":"GBM cell 1944","2":"GSM2245388","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423562","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965584","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245388/suppl/GSM2245388_1001000235.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B3"},{"1":"GBM cell 1946","2":"GSM2245390","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423564","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965586","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245390/suppl/GSM2245390_1001000235.C12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C12"},{"1":"GBM cell 1947","2":"GSM2245391","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423565","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965587","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245391/suppl/GSM2245391_1001000235.D6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D6"},{"1":"GBM cell 1948","2":"GSM2245392","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423566","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965588","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245392/suppl/GSM2245392_1001000235.F2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F2"},{"1":"GBM cell 1949","2":"GSM2245393","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423567","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965589","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245393/suppl/GSM2245393_1001000235.B8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B8"},{"1":"GBM cell 1950","2":"GSM2245394","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423568","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965590","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245394/suppl/GSM2245394_1001000235.F6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F6"},{"1":"GBM cell 1951","2":"GSM2245395","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423569","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965591","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245395/suppl/GSM2245395_1001000235.A2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A2"},{"1":"GBM cell 1952","2":"GSM2245396","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423570","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965592","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245396/suppl/GSM2245396_1001000235.B5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B5"},{"1":"GBM cell 1953","2":"GSM2245397","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423571","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965593","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245397/suppl/GSM2245397_1001000235.B10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B10"},{"1":"GBM cell 1954","2":"GSM2245398","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: E6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423572","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965594","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245398/suppl/GSM2245398_1001000235.E6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E6"},{"1":"GBM cell 1955","2":"GSM2245399","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423573","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965595","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245399/suppl/GSM2245399_1001000235.F8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"F8"},{"1":"GBM cell 1956","2":"GSM2245400","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423574","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965596","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245400/suppl/GSM2245400_1001000235.G11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G11"},{"1":"GBM cell 1957","2":"GSM2245401","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423575","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965597","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245401/suppl/GSM2245401_1001000235.A5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A5"},{"1":"GBM cell 1958","2":"GSM2245402","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423576","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965598","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245402/suppl/GSM2245402_1001000235.B11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B11"},{"1":"GBM cell 1959","2":"GSM2245403","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423577","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965599","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245403/suppl/GSM2245403_1001000235.B7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B7"},{"1":"GBM cell 1960","2":"GSM2245404","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423578","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965600","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245404/suppl/GSM2245404_1001000235.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F12"},{"1":"GBM cell 1961","2":"GSM2245405","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423579","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965601","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245405/suppl/GSM2245405_1001000235.G6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G6"},{"1":"GBM cell 1962","2":"GSM2245406","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423580","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965602","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245406/suppl/GSM2245406_1001000235.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H11"},{"1":"GBM cell 1963","2":"GSM2245407","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423581","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965603","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245407/suppl/GSM2245407_1001000235.D11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D11"},{"1":"GBM cell 1964","2":"GSM2245408","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: E12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.E12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423582","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965604","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245408/suppl/GSM2245408_1001000235.E12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E12"},{"1":"GBM cell 1966","2":"GSM2245410","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423584","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965606","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245410/suppl/GSM2245410_1001000235.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A3"},{"1":"GBM cell 1967","2":"GSM2245411","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423708","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965607","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245411/suppl/GSM2245411_1001000235.H8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H8"},{"1":"GBM cell 1968","2":"GSM2245412","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423709","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965608","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245412/suppl/GSM2245412_1001000235.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G12"},{"1":"GBM cell 1969","2":"GSM2245413","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423710","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965609","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245413/suppl/GSM2245413_1001000235.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D1"},{"1":"GBM cell 1971","2":"GSM2245415","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423712","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965611","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245415/suppl/GSM2245415_1001000235.B2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B2"},{"1":"GBM cell 1972","2":"GSM2245416","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423713","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965612","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245416/suppl/GSM2245416_1001000235.D8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D8"},{"1":"GBM cell 1973","2":"GSM2245417","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423714","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965613","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245417/suppl/GSM2245417_1001000235.H3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H3"},{"1":"GBM cell 1974","2":"GSM2245418","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423715","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965614","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245418/suppl/GSM2245418_1001000235.B9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B9"},{"1":"GBM cell 1975","2":"GSM2245419","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423716","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965615","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245419/suppl/GSM2245419_1001000235.E8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E8"},{"1":"GBM cell 1977","2":"GSM2245421","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423718","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965617","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245421/suppl/GSM2245421_1001000235.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H1"},{"1":"GBM cell 1978","2":"GSM2245422","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423719","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965618","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245422/suppl/GSM2245422_1001000235.G3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G3"},{"1":"GBM cell 1979","2":"GSM2245423","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423720","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965619","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245423/suppl/GSM2245423_1001000235.A1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A1"},{"1":"GBM cell 1980","2":"GSM2245424","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423721","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965620","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245424/suppl/GSM2245424_1001000235.B6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B6"},{"1":"GBM cell 1983","2":"GSM2245427","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: C7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423724","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965623","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245427/suppl/GSM2245427_1001000235.C7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C7"},{"1":"GBM cell 1984","2":"GSM2245428","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423725","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965624","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245428/suppl/GSM2245428_1001000235.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A10"},{"1":"GBM cell 1985","2":"GSM2245429","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: E11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.E11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423726","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965625","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245429/suppl/GSM2245429_1001000235.E11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E11"},{"1":"GBM cell 1986","2":"GSM2245430","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423727","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965626","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245430/suppl/GSM2245430_1001000235.G1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G1"},{"1":"GBM cell 1987","2":"GSM2245431","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423728","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965627","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245431/suppl/GSM2245431_1001000235.G7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G7"},{"1":"GBM cell 1988","2":"GSM2245432","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423729","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965628","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245432/suppl/GSM2245432_1001000235.F5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F5"},{"1":"GBM cell 1989","2":"GSM2245433","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423730","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965629","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245433/suppl/GSM2245433_1001000235.C9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C9"},{"1":"GBM cell 1990","2":"GSM2245434","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423731","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965630","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245434/suppl/GSM2245434_1001000235.D3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D3"},{"1":"GBM cell 1991","2":"GSM2245435","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423733","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965631","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245435/suppl/GSM2245435_1001000235.F1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F1"},{"1":"GBM cell 1992","2":"GSM2245436","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423734","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965632","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245436/suppl/GSM2245436_1001000235.C4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C4"},{"1":"GBM cell 1994","2":"GSM2245438","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423736","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965634","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245438/suppl/GSM2245438_1001000235.B12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B12"},{"1":"GBM cell 1995","2":"GSM2245439","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423737","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965635","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245439/suppl/GSM2245439_1001000235.H6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H6"},{"1":"GBM cell 1996","2":"GSM2245440","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423387","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965636","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245440/suppl/GSM2245440_1001000235.H9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H9"},{"1":"GBM cell 1997","2":"GSM2245441","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423732","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965637","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245441/suppl/GSM2245441_1001000235.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D5"},{"1":"GBM cell 1998","2":"GSM2245442","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423389","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965638","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245442/suppl/GSM2245442_1001000235.H10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H10"},{"1":"GBM cell 1999","2":"GSM2245443","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423391","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965639","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245443/suppl/GSM2245443_1001000235.H4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H4"},{"1":"GBM cell 2000","2":"GSM2245444","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423392","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965640","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245444/suppl/GSM2245444_1001000235.E1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E1"},{"1":"GBM cell 2005","2":"GSM2245449","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423403","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965645","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245449/suppl/GSM2245449_1001000235.F11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F11"},{"1":"GBM cell 2006","2":"GSM2245450","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423405","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965646","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245450/suppl/GSM2245450_1001000235.B1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B1"},{"1":"GBM cell 2008","2":"GSM2245452","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: A6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.A6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423409","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965648","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245452/suppl/GSM2245452_1001000235.A6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A6"},{"1":"GBM cell 2012","2":"GSM2245456","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: E7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.E7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423418","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965652","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245456/suppl/GSM2245456_1001000235.E7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E7"},{"1":"GBM cell 2013","2":"GSM2245457","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000235","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000235.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423420","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965653","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245457/suppl/GSM2245457_1001000235.D12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D12"},{"1":"GBM cell 2022","2":"GSM2245466","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423437","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965662","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245466/suppl/GSM2245466_1001000236.H3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H3"},{"1":"GBM cell 2023","2":"GSM2245467","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423439","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965663","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245467/suppl/GSM2245467_1001000236.B4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B4"},{"1":"GBM cell 2024","2":"GSM2245468","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423441","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965664","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245468/suppl/GSM2245468_1001000236.H4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H4"},{"1":"GBM cell 2025","2":"GSM2245469","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423443","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965665","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245469/suppl/GSM2245469_1001000236.C6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C6"},{"1":"GBM cell 2026","2":"GSM2245470","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423445","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965666","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245470/suppl/GSM2245470_1001000236.A4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A4"},{"1":"GBM cell 2027","2":"GSM2245471","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423263","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965667","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245471/suppl/GSM2245471_1001000236.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D5"},{"1":"GBM cell 2028","2":"GSM2245472","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423265","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965668","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245472/suppl/GSM2245472_1001000236.E1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E1"},{"1":"GBM cell 2029","2":"GSM2245473","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423267","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965669","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245473/suppl/GSM2245473_1001000236.A9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A9"},{"1":"GBM cell 2030","2":"GSM2245474","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423269","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965670","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245474/suppl/GSM2245474_1001000236.H11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H11"},{"1":"GBM cell 2031","2":"GSM2245475","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423272","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965671","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245475/suppl/GSM2245475_1001000236.D3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D3"},{"1":"GBM cell 2032","2":"GSM2245476","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423274","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965672","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245476/suppl/GSM2245476_1001000236.A2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A2"},{"1":"GBM cell 2033","2":"GSM2245477","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: E10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.E10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423276","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965673","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245477/suppl/GSM2245477_1001000236.E10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E10"},{"1":"GBM cell 2035","2":"GSM2245479","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423280","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965675","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245479/suppl/GSM2245479_1001000236.B2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B2"},{"1":"GBM cell 2036","2":"GSM2245480","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423282","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965676","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245480/suppl/GSM2245480_1001000236.D7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D7"},{"1":"GBM cell 2037","2":"GSM2245481","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423284","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965677","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245481/suppl/GSM2245481_1001000236.C1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C1"},{"1":"GBM cell 2039","2":"GSM2245483","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423288","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965679","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245483/suppl/GSM2245483_1001000236.H1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H1"},{"1":"GBM cell 2040","2":"GSM2245484","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423290","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965680","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245484/suppl/GSM2245484_1001000236.C8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C8"},{"1":"GBM cell 2041","2":"GSM2245485","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: F6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.F6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423292","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965681","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245485/suppl/GSM2245485_1001000236.F6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F6"},{"1":"GBM cell 2042","2":"GSM2245486","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423294","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965682","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245486/suppl/GSM2245486_1001000236.E3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E3"},{"1":"GBM cell 2043","2":"GSM2245487","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423296","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965683","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245487/suppl/GSM2245487_1001000236.A7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A7"},{"1":"GBM cell 2044","2":"GSM2245488","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423297","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965684","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245488/suppl/GSM2245488_1001000236.C3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C3"},{"1":"GBM cell 2045","2":"GSM2245489","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423300","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965685","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245489/suppl/GSM2245489_1001000236.C4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C4"},{"1":"GBM cell 2046","2":"GSM2245490","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423302","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965686","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245490/suppl/GSM2245490_1001000236.B1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B1"},{"1":"GBM cell 2047","2":"GSM2245491","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423304","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965687","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245491/suppl/GSM2245491_1001000236.G5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G5"},{"1":"GBM cell 2048","2":"GSM2245492","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423306","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965688","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245492/suppl/GSM2245492_1001000236.H7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H7"},{"1":"GBM cell 2050","2":"GSM2245494","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423310","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965690","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245494/suppl/GSM2245494_1001000236.D10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D10"},{"1":"GBM cell 2051","2":"GSM2245495","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423313","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965691","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245495/suppl/GSM2245495_1001000236.G9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G9"},{"1":"GBM cell 2052","2":"GSM2245496","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: E6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.E6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423315","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965692","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245496/suppl/GSM2245496_1001000236.E6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E6"},{"1":"GBM cell 2053","2":"GSM2245497","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423317","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965693","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245497/suppl/GSM2245497_1001000236.D2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D2"},{"1":"GBM cell 2054","2":"GSM2245498","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423616","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965694","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245498/suppl/GSM2245498_1001000236.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A10"},{"1":"GBM cell 2055","2":"GSM2245499","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423617","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965695","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245499/suppl/GSM2245499_1001000236.G11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G11"},{"1":"GBM cell 2056","2":"GSM2245500","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423618","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965696","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245500/suppl/GSM2245500_1001000236.D9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D9"},{"1":"GBM cell 2058","2":"GSM2245502","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 7","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423323","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965698","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245502/suppl/GSM2245502_1001000236.B6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"7","58":"B6"},{"1":"GBM cell 2059","2":"GSM2245503","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423325","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965699","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245503/suppl/GSM2245503_1001000236.B11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B11"},{"1":"GBM cell 2060","2":"GSM2245504","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423615","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965700","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245504/suppl/GSM2245504_1001000236.G3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G3"},{"1":"GBM cell 2061","2":"GSM2245505","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: F2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.F2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423619","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965701","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245505/suppl/GSM2245505_1001000236.F2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F2"},{"1":"GBM cell 2062","2":"GSM2245506","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423620","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965702","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245506/suppl/GSM2245506_1001000236.G7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G7"},{"1":"GBM cell 2063","2":"GSM2245507","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: E8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.E8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423621","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965703","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245507/suppl/GSM2245507_1001000236.E8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E8"},{"1":"GBM cell 2064","2":"GSM2245508","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: F9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.F9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423622","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965704","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245508/suppl/GSM2245508_1001000236.F9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F9"},{"1":"GBM cell 2065","2":"GSM2245509","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423623","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965705","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245509/suppl/GSM2245509_1001000236.C10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C10"},{"1":"GBM cell 2066","2":"GSM2245510","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423624","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965706","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245510/suppl/GSM2245510_1001000236.D12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D12"},{"1":"GBM cell 2069","2":"GSM2245513","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423627","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965709","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245513/suppl/GSM2245513_1001000236.C12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C12"},{"1":"GBM cell 2070","2":"GSM2245514","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423628","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965710","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245514/suppl/GSM2245514_1001000236.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B3"},{"1":"GBM cell 2071","2":"GSM2245515","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423629","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965711","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245515/suppl/GSM2245515_1001000236.H5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H5"},{"1":"GBM cell 2073","2":"GSM2245517","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423631","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965713","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245517/suppl/GSM2245517_1001000236.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A3"},{"1":"GBM cell 2074","2":"GSM2245518","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: C7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.C7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423632","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965714","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245518/suppl/GSM2245518_1001000236.C7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C7"},{"1":"GBM cell 2075","2":"GSM2245519","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423633","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965715","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245519/suppl/GSM2245519_1001000236.B8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B8"},{"1":"GBM cell 2076","2":"GSM2245520","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: F3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423634","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965716","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245520/suppl/GSM2245520_1001000236.F3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F3"},{"1":"GBM cell 2077","2":"GSM2245521","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423635","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965717","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245521/suppl/GSM2245521_1001000236.D4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D4"},{"1":"GBM cell 2079","2":"GSM2245523","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423637","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965719","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245523/suppl/GSM2245523_1001000236.B12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B12"},{"1":"GBM cell 2080","2":"GSM2245524","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423638","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965720","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245524/suppl/GSM2245524_1001000236.A1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A1"},{"1":"GBM cell 2081","2":"GSM2245525","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: F7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.F7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423639","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965721","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245525/suppl/GSM2245525_1001000236.F7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F7"},{"1":"GBM cell 2082","2":"GSM2245526","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: E4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.E4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423640","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965722","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245526/suppl/GSM2245526_1001000236.E4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E4"},{"1":"GBM cell 2084","2":"GSM2245528","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: A8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.A8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423642","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965724","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245528/suppl/GSM2245528_1001000236.A8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A8"},{"1":"GBM cell 2085","2":"GSM2245529","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: H2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.H2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423643","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965725","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245529/suppl/GSM2245529_1001000236.H2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H2"},{"1":"GBM cell 2087","2":"GSM2245531","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: D6","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.D6","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424429","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965727","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245531/suppl/GSM2245531_1001000236.D6.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D6"},{"1":"GBM cell 2091","2":"GSM2245535","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G8","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G8","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424435","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965731","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245535/suppl/GSM2245535_1001000236.G8.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G8"},{"1":"GBM cell 2105","2":"GSM2245549","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: G10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423236","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965745","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245549/suppl/GSM2245549_1001000236.G10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G10"},{"1":"GBM cell 2106","2":"GSM2245550","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423238","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965746","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245550/suppl/GSM2245550_1001000236.B10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B10"},{"1":"GBM cell 2108","2":"GSM2245552","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: F11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.F11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423242","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965748","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245552/suppl/GSM2245552_1001000236.F11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F11"},{"1":"GBM cell 2109","2":"GSM2245553","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000236","12":"well: B5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000236.B5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423244","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965749","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245553/suppl/GSM2245553_1001000236.B5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B5"},{"1":"GBM cell 2111","2":"GSM2245555","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: E5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.E5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423250","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965751","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245555/suppl/GSM2245555_1001000237.E5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E5"},{"1":"GBM cell 2113","2":"GSM2245557","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: A3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.A3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423253","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965753","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245557/suppl/GSM2245557_1001000237.A3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A3"},{"1":"GBM cell 2114","2":"GSM2245558","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: E3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.E3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423255","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965754","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245558/suppl/GSM2245558_1001000237.E3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E3"},{"1":"GBM cell 2115","2":"GSM2245559","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: B11","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.B11","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423257","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965755","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245559/suppl/GSM2245559_1001000237.B11.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B11"},{"1":"GBM cell 2116","2":"GSM2245560","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: G12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.G12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424277","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965756","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245560/suppl/GSM2245560_1001000237.G12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G12"},{"1":"GBM cell 2117","2":"GSM2245561","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: D3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.D3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05423248","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965757","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245561/suppl/GSM2245561_1001000237.D3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D3"},{"1":"GBM cell 2118","2":"GSM2245562","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: G2","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.G2","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424280","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965758","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245562/suppl/GSM2245562_1001000237.G2.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G2"},{"1":"GBM cell 2119","2":"GSM2245563","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: C9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.C9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424282","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965759","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245563/suppl/GSM2245563_1001000237.C9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C9"},{"1":"GBM cell 2121","2":"GSM2245565","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: A5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.A5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424286","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965761","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245565/suppl/GSM2245565_1001000237.A5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A5"},{"1":"GBM cell 2122","2":"GSM2245566","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: C10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.C10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424288","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965762","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245566/suppl/GSM2245566_1001000237.C10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"C10"},{"1":"GBM cell 2123","2":"GSM2245567","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: A12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.A12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424290","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965763","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245567/suppl/GSM2245567_1001000237.A12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A12"},{"1":"GBM cell 2125","2":"GSM2245569","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: F3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.F3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424294","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965765","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245569/suppl/GSM2245569_1001000237.F3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F3"},{"1":"GBM cell 2126","2":"GSM2245570","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: D10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.D10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424296","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965766","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245570/suppl/GSM2245570_1001000237.D10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D10"},{"1":"GBM cell 2127","2":"GSM2245571","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: H7","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.H7","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424299","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965767","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245571/suppl/GSM2245571_1001000237.H7.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"H7"},{"1":"GBM cell 2128","2":"GSM2245572","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: E1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.E1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424302","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965768","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245572/suppl/GSM2245572_1001000237.E1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"E1"},{"1":"GBM cell 2129","2":"GSM2245573","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: D1","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.D1","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424304","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965769","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245573/suppl/GSM2245573_1001000237.D1.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D1"},{"1":"GBM cell 2130","2":"GSM2245574","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: B3","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.B3","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424306","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965770","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245574/suppl/GSM2245574_1001000237.B3.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"B3"},{"1":"GBM cell 2131","2":"GSM2245575","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: D5","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.D5","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424307","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965771","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245575/suppl/GSM2245575_1001000237.D5.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"D5"},{"1":"GBM cell 2132","2":"GSM2245576","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: G4","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.G4","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424309","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965772","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245576/suppl/GSM2245576_1001000237.G4.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G4"},{"1":"GBM cell 2133","2":"GSM2245577","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: G10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.G10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424311","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965773","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245577/suppl/GSM2245577_1001000237.G10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"G10"},{"1":"GBM cell 2134","2":"GSM2245578","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: A10","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.A10","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424313","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965774","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245578/suppl/GSM2245578_1001000237.A10.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A10"},{"1":"GBM cell 2135","2":"GSM2245579","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: F12","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.F12","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424315","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965775","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245579/suppl/GSM2245579_1001000237.F12.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"F12"},{"1":"GBM cell 2136","2":"GSM2245580","3":"Public on Nov 01 2017","4":"Jul 20 2016","5":"Nov 01 2017","6":"SRA","7":"1","8":"Brain","9":"Homo sapiens","10":"diagnosis: glioblastoma","11":"plate id: 1001000237","12":"well: A9","13":"tissue: Tumor","14":"patient id: BT_S4","15":"tsne cluster: 8","16":"cell type: Immune cell","17":"neoplastic: Regular","18":"selection: Microglia(CD45)","19":"polyA RNA","20":"single-cell sorting in 96-well plates followed by cDNA preparation using Smart-seq2","21":"Smart-Seq2 for preparation of cDNA followed by library preparation using Nextera XT. Nextera tagmentation according to the standard protocol for single cell RNA-seq following Smart-Seq2","22":"9606","23":"Single cell from Tumor","24":"1001000237.A9","25":"Short read trimming: Prinseq to remove short reads (-min_len 30) trim the first 10 bp on the 5’-end (-trim_left 10), trim reads with low quality on the 3’-end (-trim_qual_right 25) and filter low complexity reads (-lc_method entropy \\\\-lc_threshold 65).  We used FASTQC to determine overrepresented sequences and removed those using cutadapt (-e 0.15 –m 30). We then used Prinseq to remove orphan pairs less than 30bp in length followed by removal of nextera adapters using Trim Galore  (--stringency 1).","26":"Read alignment: reads were aligned to the hg19 genome with STAR using the following options (-outFilterType BySJout \\\\--outFilterMultimapNmax 20 \\\\--alignSJoverhangMin 8 \\\\--alignSJDBoverhangMin 1 \\\\--outFilterMismatchNmax 999 \\\\--outFilterMismatchNoverLmax 0.04 \\\\--alignIntronMin 20 \\\\--alignIntronMax 1000000 \\\\--alignMatesGapMax 1000000 \\\\--outSAMstrandField intronMotif ).","27":"Per-gene read assignement: aligned reads were converted to counts for every gene using HTSeq (-m intersection-nonempty \\\\-s no).","28":"Genome_build: hg19","29":"Supplementary_files_format_and_content: tab-delimited text files with raw read values for each sample","30":"GPL18573","31":"Spyros,,Darmanis","32":"spyros.darmanis@gmail.com","33":"6506960861","34":"Stephen Quake","35":"Bioengineering","36":"Stanford University","37":"Clark Center, E300","38":"Stanford","39":"California","40":"94305","41":"USA","42":"0","43":"Illumina NextSeq 500","44":"cDNA","45":"transcriptomic","46":"RNA-Seq","47":"BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05424317","48":"SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1965776","49":"ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2245nnn/GSM2245580/suppl/GSM2245580_1001000237.A9.csv.gz","50":"Immune cell","51":"glioblastoma","52":"Regular","53":"BT_S4","54":"1.001e+09","55":"Microglia(CD45)","56":"Tumor","57":"8","58":"A9"}],"options":{"columns":{"min":{},"max":[10],"total":[58]},"rows":{"min":[10],"max":[10],"total":[1698]},"pages":{}}}
  </script>
</div>
<!-- rnb-frame-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
<div id="normalization-step-3-calculate-pooled-cpms-and-normalize-using-sum-factors" class="section level4">
<h4>Normalization step 3: Calculate pooled CPMs and normalize using sum factors:</h4>
<p>For more details on the normalization <a href="https://rdrr.io/bioc/scran/man/computeSumFactors.html">read here</a></p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
</div>
</div>
<div id="compare-normalized-data-to-count-data" class="section level2">
<h2>Compare normalized data to count data</h2>
<p>One way to determine whether our normalization is yielding biolgically relevant results, is to plot it and see if similarly labeled samples and cells end up together. Because plotting 1000’s of genes together isn’t practical, we will reduce the dimensions of our data using Principal Components Analysis. Note that the <code>t()</code> function is being used here to transpose the <code>matrix</code>, so it is samples x genes. Because, in this instance, we would like to have PCA scores for each cell. We could however, obtain PCA scores for each gene if we didn’t transpose our data.frame first.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBVc2UgUENBIGZvciBkaW1lbnNpb24gcmVkdWN0aW9uIG9mIGNlbGxzJyByYXcgZGF0YVxuY291bnRfcGNhIDwtIHByY29tcCh0KGNvdW50cyhzY2VfcmF3KSkpXG4jIFVzZSBQQ0EgZm9yIGRpbWVuc2lvbiByZWR1Y3Rpb24gb2YgY2VsbHMnIHNjcmFuIG5vcm1hbGl6ZWQgZGF0YVxubm9ybV9wY2EgPC0gcHJjb21wKHQobG9nY291bnRzKHNjZV9ub3JtKSkpXG5gYGAifQ== -->
<pre class="r"><code># Use PCA for dimension reduction of cells&#39; raw data
count_pca &lt;- prcomp(t(counts(sce_raw)))
# Use PCA for dimension reduction of cells&#39; scran normalized data
norm_pca &lt;- prcomp(t(logcounts(sce_norm)))</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>There’s a lot of data we can pull from these <code>prcomp</code> objects, but for now, we need the PCA scores for the samples. In order to retrieve the PCA scores, we will use <code>$x</code> at the end of our <code>prcomp</code> objects and because we only really want</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-plot-begin eyJjb25kaXRpb25zIjpbXSwiaGVpZ2h0Ijo0MzIuNjMyOSwic2l6ZV9iZWhhdmlvciI6MCwid2lkdGgiOjcwMH0= -->
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAArwAAAGwCAYAAABLkLalAAAEGWlDQ1BrQ0dDb2xvclNwYWNlR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19zn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7J0HfBTV9sd/u0k2vZLQQu8dBCkiAgI27GIDu8+K+uRvefbeHqjPLqKCvaKo2BsgINJ7h4SQENJ732R3/+fcZTazJY2UTTmXz7Izd+7cO/Odze5vzpx7jsFGBVKEgBAQAkJACAgBISAEhEArJWBspeclpyUEhIAQEAJCQAgIASEgBBQBEbzyQRACQkAICAEhIASEgBBo1QRE8LbqyysnJwSEgBAQAkJACAgBISCCVz4DQkAICAEhIASEgBAQAq2agAjeVn155eSEgBAQAkJACAgBISAEfNs6grKyMlgslraOQc5fCHiVgK+vL0wmk1eOwWw2o6Kiwitjy6BCQAjYCfj4+MDf319wCIFGI9DmBW9qaipyc3MbDbB0LASEQM0EoqKi0LVr15obNkKLjIwMZGZmNkLP0qUQEAK1JRAeHo4ePXrUtrm0EwJ1JiAuDXVGJjsIASEgBISAEBACQkAItCQCInhb0tWSYxUCQkAICAEhIASEgBCoMwERvHVGJjsIASEgBISAEBACQkAItCQCInhb0tWSYxUCQkAICAEhIASEgBCoMwERvHVGJjsIASEgBISAEBACQkAItCQCInhb0tWSYxUCQkAICAEhIASEgBCoMwERvHVGJju0VAIcc1mKEBACQkAICAEh0PYIiOBte9e8TZ7xG2+8gfXr17fJc5eTFgJCQAgIASHQ1gmI4G3rn4Bmcv7FxcVIS0tzO5ry8nIkJiY6ZcLKy8uD1Wp1tNUSh3DGvMLCQpU57/DhwygqKlJt2LLL6zwGZ9XiUlBQgPj4eFWnKuQ/ISAEmgWBqv6O9QeXk5OD9PR0RxVnyuO/d/5u4Je+D/33B//dZ2dnO/aTBSEgBNoOgTafaa3tXOrme6YLFy7Ezz//jJ49eyrB+vzzzyMkJAQLFizA33//jYiICJUJ65lnnlFtrr/+esyfPx/t27dXP2wXX3wxfv31V+zduxevvfaaqgsLC0NcXBweeeQR9UPIy/n5+ejQoQOysrLw3nvvoVevXti3bx9uuukmTJ06tfkCkiMTAm2IQFV/x6NGjVLfA/z9oAlb/s546qmnsGfPHvBTHK7nNNX3338/Xn31VbVsMBjA2fQuuugi/P777+Cb6MmTJ+O6665rQ1TlVIWAEBDBK58BrxLYsWOHErsff/wxAgICwO9r165FYGAg/vrrLyxatEjVf/3111iyZAnuvvvuao/34MGDePfdd1WKyi+++AI//fSTEr0siM8880yMGDEC9957rxK5EydOxP79+9WPZbWdykYhIASalICnv2MWvHwTHBsbi7lz56ob23vuuQdbtmyBv78/EhIS1PcHp6lmAcxPdT788EN1Y8zfG5s2bVLfDSx+b731VhG8TXpFZTAh4H0C4tLg/WvQpo+ArTn8Q8Zil8uVV16JadOmYdeuXRg3bpyjftKkSVi1apWTa4MncGzB1fKx8w8juzG4ljPOOAPz5s3Dfffdp34YxbrrSkjWhYB3CVT1d8xClm9Uufj4+OCUU07BsmXL1HqnTp0QHR0No9H+s9axY0cldnkjbxs9erRq165dO7AbFLs9SBECQqDtEBDB23audbM8U3Y9YP87rbAPbmZmJmJiYtSjR62efW/5UST/mPEjSu3HqqSkRGui3kNDQx3r3M5TYUH96aefYsqUKVi9ejXmzJnjqZnUCQEh4CUCVf0ds6DVf1/w94L2XcBPhfRFu4nW6kwmk1qs6ntBayfvQkAItE4CInhb53VtMWc1fvx4sFsD+95xYXeE5cuXK8vNhg0blCWG69n3bsCAAUrw8o8hP77ksm7dOvVe03/8Y6eFJXv00UeVZZctvfyoMzU11elHtKa+ZLsQEALeIcBPev78808lcvlml5/6DBo0yDsHI6MKASHQogiID2+Lulyt72BZvF5wwQW4+uqr1WNHfpQ5e/ZssEBln1uuZ5+84OBgPPHEEwrArFmz8N///lc9ruzbty/YSlxTGTp0KF555RUVpeHSSy9Vk9vYv4+tyezPxxNdpAgBIdC8CWhPZa666iolevmG+dxzz1UuUM37yOXohIAQ8DYBg42Ktw/Cm+PzxAYtrJU3j6Otj83uCvwKCgpyQsF1bJnlqA36UlW9vo3rMj/+ZGGr+fhx1AbuV1t3bS/rTUeAb2q6du3adAPqRkpOTlY3ProqWWzmBNj1iSeq+fn5NfMjlcOrLYHw8HDH/Iva7iPthEBdCIhZqy60pG2jEeAfLk8/XnWtr+4ANR8+rU1tLMNaW3kXAkKg+RBwvQFuPkcmRyIEhEBzJSA+vM31yshxCQEhIASEgBAQAkJACDQIARG8DYJROhECQkAICAEhIASEgBBorgRE8DbXKyPHJQSEgBAQAkJACAgBIdAgBETwNghG6UQICAEhIASEgBAQAkKguRIQwdtcr4wclxAQAkJACAgBISAEhECDEGjzYcmKioqcMnpVRZXTWHIYHE+paqvapyHreWwtcUJD9lubvjhUGI+tZTSqzT4N1YYjK3AIMm9Ez+NMTVarVcXubajzqW0//HnjjFD6rFK13be+7TgyBodvc81iV99+q9ufr7NrSLrq2jfkNj7PhvzbYn78t8KfHW8Ub35u+Xz5s8N/r974vuDx+buSS0NeU9VhLf/jv10ubfn8j+e7iz83En2jlh8yaXZcBNp8WDKOzVpaWlojPP4R4SQJOTk5Xvkh4/GbUoBoQPiLKyIiAnxjUBtO2n4N9c7iwVtim5NdsGjxBne+3szeG2PzDw//aDfl2Hyu3ir8HdCQ58pxnblPvlHzRvHm55bPl29cWOx5S3ByimFv/d3y+fP4LPi98X3J43vzu4PH5xuO4/n+4OOWIgQak4C4NDQmXelbCAgBISAEhIAQEAJCwOsERPB6/RLIAQgBISAEhIAQEAJCQAg0JgERvI1JV/oWAkJACAgBISAEhIAQ8DoBEbxevwRyAEJACAgBISAEhIAQEAKNSUAEb2PSlb6FgBAQAkJACAgBISAEvE5ABK/XL4EcgBAQAkJACAgBISAEhEBjEhDB25h0pW8hIASEgBAQAkJACAgBrxMQwev1SyAHIASEgBAQAkJACAgBIdCYBETwNiZd6VsICAEhIASEgBAQAkLA6wRE8Hr9EsgBCAEhIASEgBAQAkJACDQmARG8jUlX+hYCQkAICAEhIASEgBDwOgFfrx+BHIAQEAKtikDGvg+Qn7wCfoHRiD3xcfj4Bbeq85OTEQK1JWC1lCJ1+6soyzuIgMgB6DhsDgwGsTPVlp+0EwINSUAEb0PSlL6EQBsnsP/XGSjJ2g6btRww+CA7bjEGX7QOfkEd2jgZOf22RsBmtWD7ZwNgMJro78GMvOQ/kbbjdQybuRdGH1NbwyHnKwS8TkBuNb1+CeQAhEDrIJCfvAzFGZvsYpdPyWZRojdp3UOt4wTlLIRAHQik7niNPv++Suyq3ejvwUBCN2PPwjr0Ik2FgBBoKAIieBuKpPQjBNo4gfLidBh9g5wp0I+8uSjJuU7WhEAbIGAuOkI3fRVOZ2ojFwf5e3BCIitCoMkIiOBtMtQykBBo3QT8w/u4n6DRD4GRg9zrpUYItHICQVFDyKIb4HaWgVFD3eqkQggIgcYnIIK38RnLCEKgTRAIaX8iovpc5jhXg08gfE3h6HbS8446WRACbYVAdP9raOJmezpdgzplfvoREN4P0X1nthUEcp5CoFkRkElrzepyyMEIgZZNoMuJjyGs0ykoTFsP34AoRPe7mibtyNdMy76qcvTHQ8BgMGDQBSuRFfclzAWJ8A/rhaheFx1PV7KPEBACDUBAfokaAKJ0IQSEQCWBsNgp4JcUISAEgHa9LxUMQkAINAMC4tLQDC6CHIIQEAJCQAgIASEgBIRA4xEQwdt4bKVnISAEhIAQEAJCQAgIgWZAQARvM7gIcghCQAg0bwJllhIkFGxHesnh5n2gcnRCQAgIASHgkYD48HrEIpVCQAgIATuB5KL9mLdjJipsZSiqyMPYmPNx64DXBY8QEAJCQAi0IAIieFvQxZJDFQJCoGkJFJbn4KHNU50GXZfxHXqGDMWZXW52qpeV1kugoiwXRzc9jTJKJhEcMwqdht8DjsIgRQgIgZZDQFwaWs61kiMVAkKgiQnszFmJQJ9Qt1H/Sv3crU4qWicBK2VH27l4BLIPfYuitLVI3zkfO74cRimDKXW2FCEgBFoMARG8LeZSyYEKASHQ1ASMBh9KG+D+NWkwuNc19bHJeE1DIHnDE5Q7wkeXJthKyxZkHfikaQ5ARhECQqBBCMi3doNglE6EgBBojQRGtJuGAJ9gJ9HrazBhVq/HWuPpyjl5IGAmNwYWuPpirShGeUmGvkqWhYAQaOYERPA28wskhycEhID3CJiMAXhq1G8I9A0lO68PgnzCcE2fZzEkcqL3DkpGblICwe3HULZAk9uYge2GuNUVksvD/mW34OCK21GSs9ttu1QIASHgPQIyac177GVkISAEWgCBYN9wvHnSzhZwpHKIjUGgw+DZyNz3ASrKclT3LH7Du0xFRNcznIbLjluCxH/uctRlHvwKPSYuQEQ353aOBrIgBIRAkxIQwdukuGUwISAEhIAQaEkEDEYfDLl4I7LivkRFcToCIgcqwas/B0t5oZPY1bYlrb0XYZ0nwugbqFXJuxAQAl4iIILXS+BlWCEgBFofASv5eh7M24wycxm6Bg6Cj0G+YlvLVW7X+9IqT8VizoePKRwWc55TG5vNBo7yIILXCYusCAGvEJBvY69gl0GFgBBobQRKLUV4ZttFyDWnwmKrQHFFPl4ftx0hfpGt7VTlfFwI+PpHefTztZYXkNgNdmktq0JACHiDgExa8wZ1GVMICIFWR2D2msE4UrQXBeXZSuzyCXKGNquNwlhJadUEjL4B6Dlpgf0cyapv4JdPAPpN/xFGH/cJb3WBkX1oKfb+cCZ2f3cq0nYdG6MuHUhbISAEFAGx8MoHQQgIASFQTwJszeXMW+zSoC/ppYeRXXYU0QFd9NWy3AoJBMeMxKAL/0Fh8k90dgYEdz4N/iFd63WmGTRZ7uimZynJRZnqJ3X7SxQOLQ1dTny0Xv3KzkKgLRIQwdsWr7qcsxAQAg1GYEnCC/g9eZFyY3DttNRSCA5tJqVtEDAFd0Ls8NvBvrulpaX1Omn2/U3e4Bzv2UZ1WQc+Q/uBN4LHkiIEhEDtCYjgrT0raSkEhIAQcCLwwYEHsTr9S5Qfs8A5baSVEVHTEGaKdq2W9UYmYLWYkbbzTZiLUxAcPQJRvWY08oiN0z2HQLNZzU6dG41+sFJUCClCQAjUjYAI3rrxktZCQAgIAUWgwlqO5akfeaQR4huJSR1n4eIe93ncLpWNR8BGPtMr57dXPrTKIrr/I7KKfo6+ZyxuvEEboWeD0R+BkYNQnLWNerc5RrCU58MU2s2xLgtCQAjUjoBMWqsdJ2klBISAEHAjwGmGXUuATwgeG/U9Lul5v/Lrdd0u641LIG7N4+RC60PZgCtdCoqzdyL38I9qYHNRMjL3f4Ls+G/IelreuAdTj97ZJ7zXqQupBxuMfiH0ClMifuD5K2ginH89epZdhUDbJNDmLbx+fn4IDKw5KLjRaL83CA8P98onhY8zMtJ74Y2Cg4Nrxamh4fj6+oJf7BPX1MXHx0cJFm9w588b/+AFBDS9/yfz5vGb8rwrKiqa+vLWezxferQ8LOpUbMn6nSRJZSQG9tv19wmqd//SwfERKMykrHgukwdZ/JYVJKAg9R/E/XkFCUb+zrchcc3/Ycgl2+Dr753v9ZrO0DegHYbN3I/8o8vpcK0I6TgevhTvV4oQEAJ1J9DmBW95eTkKCgpqJMfCw2QyIS8vD1Zr5Y9bjTs2UAMWHzk59tSWDdRlrbph0dWxY0cUFRXVexJGrQZ0aRQWFqbGtlicZ7+7NGuU1ejoaPDng695Uxf+vDH7kpKSph4aoaGhSmg35ectKKhlCcT4gq1Yn/EDOgX2wR7jGlhQrsKPcfzdYLLE/XvNSNzc/zWc1P6CJr9+bX3AoIi+yEla6eb7yokh4v6YqfBYK4qOYTLi8Oo70Hvqhw5sJTl7SAjfi/LSdIqy0J22fUSxdJv+xlM7IA5r5prGWNsm70JACNSeQJsXvLVHJS2FgBBorQS0Jzi1Ob/VqYvx9r45FHjKACM9OmeRO63z9fjj6CK1e2F5rnpfsO8OdA0ZgG4hg2rTbYO24ZulupxTQw7OY3tz/N7jH8WRbfMpEQQ9GbJWkBuAvwoPFhjem7KhRVA2NPv1sZ+zFUXpGxysykvSse/Hsxw4KkqzKf7tRAy5cHWtRS+fOxdv8dcO3lvj87jevP7a+cu7EHAlIILXlYisCwEh0KYI8A80W7VrU3LLMpTY5bY2+sdil8uqtM/Vu/4/9u/dW/Q3Bncaq6+u1/Le7LVYmbwYfjSh6fze/0ZUQEe3/jRXnNqek1sH9axglxh+Cubv7x0/Ux536p152L/qcZQWJCIosj9yk1chgSy5rql/7adqc1z/HStvcj57ur42sgYXJn2D2GEu25xbOtaYPxd+IuiNwvxZcHrr+mufPz6OuhRvPMWry/FJ25ZPoG6fyJZ/vnIGQkAICAEnAizOCgtrF+bpUMFuhPhGobAi26mPcos9MYC+0mAwwmo2KJcYTjv8cdyjiC/YgnC/GNwx6B0E+Ybpm6vlgtQ1qCjNUrPzA8giqS+rUr/EwgN3qyqy32Lpodfx+Ak/oUfIUH0zsL+/t1xx+EDYPYXFS1mZOxOnA22klaioKBLcNrQbdCcs5UXY8cVgGonnYHhyRfOBf1hv/PP+UPgGdaAm7q5T1gozivIzau3axHNCGiIO7/HiYfc3FrzecMXiY+YbDha9xcXFdToFb8xXqNMBSuMWT0AEb4u/hHICQkAINBWBCFMHVNgq46JG5o9HTN5UVPjm4XDHt2Ax2LexIC23luLUTleo7Gu3rBkAH0o3yxbhFMThjrXD8cLoNYj0r0weELfsWhRlbFJixWLOR+zopxDT/yp1apyuWBO7XKFNkpu/5zbMHb2yqU6/xY2TtvN14ulHAlQfjYGuDvlZ+4d2R0n2dpTk7FZEzcVH6Z2ss+Smop/0xvuGdp7Y4s5dDlgICAFnAvbQA851siYEhIAQEAIeCESRQL2o+z1qS+/kezB+15/om/wghiQ9j+lrixGGLgj2jcDgyAl4dexW5Xrwx9H3we4NmvsDi1W2AH5x6FnHCOm730HB0RWUUKCAHrvnq/rkjY9TDNbtajm3LE1Zlh07HFvIMae6Vsm6joCFEjQ4i13eaENY51MQHDPCLm5pXRWKgsCWUb8g+00IT3Iz+gah69j/IqjdMHsb+V8ICIEWS0AsvC320smBCwEh4A0Cp8f+C+1KRyHhn2OTm0gvcRQsg8GGK2iG/5SZ5TCbzcqtgI+vsDyHxK7ewsgP1y3IL890HH5u4k+OZccCWRoLUlYpsRXh34Esxe6h26rK8Oboo40vhHeZhqz9HxMF57CGEd3PRm7iL06WXEbFWc3CyJob2eMCci3JJNeSgeTy0LNRKLIYz9j7HirKcpUAD+s8uVHGkU6FgBCwExALr3wShIAQEAJ1JBBTNg5+/s4+oTabAdlH/dx66h8+jqI5uNsWOO2wVnz8I7RFxzv7APv42SfThfpF4creT1VuI5cJX3pU/+QJJNqkVEkgrPMkdBhym307uyocK6V5BxHe5bRjFl6t1v4eFjsNIR3GIKL79EYTu1by+d79zQRKf/w6MvcuRDy5syRv/q/zgciaEBACDUpABG+D4pTOhIAQaAsE/IPYLcH1TG0kgp0rk4v2I6loNwaE2SM1+BuDwZnYWASzpVgrsSMfchZfJM44WUK7vrO0Jhjf4SI8MnwpJne8AqdRGLRnRy2jsGcDHdtlwTOBjsPutG/QJaNgockpeqN6zVDbfMin1+gbgvaDZ5MQnuK5owasTVg5m9xXyN1CN9kxa/+HKM7kNMJShIAQaAwC7maHxhhF+hQCQkAItCICHXqY0S7WjLRDAcfOykYuDcCESzh6A2fxgkpMMX/vbOXHS2pW1U3tfA36hI7CyOjT1br2X0B4Hww6/y/KAnYVrBUl8Gs3GB3HPK5iyWpt+L132Anqpa+T5eoJcHY1I1nK2T9aKyw003e9hUEXrER0v6tgLjoCU0g3BEVxRIfGKRVlOchPXkafEx+U5MXRDZOLiwpZ9Evz4xEUPbxxDkB6FQJtnIAI3jb+AZDTFwJC4PgITLgkBwc2BuHofsrCSBbfoZMLEBBid3PILkvBm3tvVR2bKVqDVn45skD57740ZgNFaOioVat3U0gXdJ2+BM/vnIWU4lUwbxqP8e1n4Kb+Lzu1k5U6EiBhye4hrsVcmIjy4nTykR6qXq7bG3KdXSj2/nC6mgRnpWQYIOu9a2GLrx+HRpMiBIRAoxBw/xZolGGkUyEgBIRA6yPQ98RiTJqVjZMuyEVIBM1cO1YSC3apaA3auvbOk9W4PL7lbJRZSrRq9c5RHG5fOwyHC3fCbLVvW5P+NZYkvODUTlbqRiC4/VhlNXffy0CTxha5VzdwDfvr7v2e/LUpCgSLWk9iVw1JolyiQTQwfOlOCOgIiODVwZBFISAEhEBDEAgzRTvCkHnqj6M2HMjf4LQpLn8LrdtdH/Qbfjrypn5VlutIwMcvmKIgnOphLwoQV1G35AgeOqmxqrzoqIr7W1NDnqBYkLK6pmayXQgIgeMkIIL3OMHJbkJACAiBqgj0ChuB4VFTq9pMySvKKRGFc0QHu8XXedIbd8Bt88wZVfYlG2om0K7fTI+NIrqf47G+ISuNplAy7tacdc5GFmApQkAINB4BEbyNx1Z6FgJCoA0TuHXA65jV63G08+9CdttKyy1nYSu1FFKkBnvkBg0RT0jTt9Pq2erLWdqkHD8BjnHbYdj/qQ58/COVxbXruLkq/Njx91q7Pf0CotF+0C2UwM0+mbGqvXhSHYdRkyIEhEDjEJBv0cbhKr0KASEgBFToMQ4/9kPS6/gqYa6KnduPhO4dA9+m2LzO9oYg31Cc1P5CrElf4kSuZ8gwhPhFOtXJSt0JdKLwZFG9LgK7GJhCusIU3LnunRznHp2G/x+lMu6JzH2LVAqMkqwd1JOzNZ8FcVlhEgIj+h3nKLKbEBAC1REQwVsdHdkmBISAEGgAAud0vR38qqnc0O8lysyWi505f6lwZmNizsW1fSUhQU3carvdn4Quv7xRonpdQIL7AjX0riUnUYSIFKfDMBr9UJZ3oErBa6XIDpwdjsPWscXaFGxPgezUiawIASFQJQERvFWikQ1CQAgIgaYlwFbfu4Z8QGmELWT/s4krQ9Pib7LR/II6ugleToThGxDj8Rgs5UUU1uw0WMwU5YF8fY+sewB9zliCkJiRHttLpRAQAu4EnJ+puW+XGiEgBISAEGhiAkZKTiB+u00MvQmH63aSFmrumG83+WiHktWWUxp7Kvt+PIsEchqFNcsnCy+JXiqHVvwLFWW5nppLnRAQAh4IiOD1AEWqhIAQEAJCQAg0FoGA8N4YcskWRHSbjpCOJ6PL6MfRe8r7VQ5XXpJOlt3KOM/ckOP7pmx9HoXpzuHtquxENgiBNk5AXBra+AdATl8ICIHmS6AoYxMqSrMQGDWkSSdZNV8irefIfClaRI+Jb9TqhDhGb4VLdjYbxRDOivsSWQc+QfvBt6LzCffVqi9pJATaKgGx8LbVKy/nLQSEQJMTKLMUo4gmpdlszjP0PR3Iob9uQdyya5D4z3+w+5vxyD38o6dmUtcGCHQdRxMXPYWms5ars8/Ys1AsvW3gcyCnWD8CInjrx0/2FgJCQAjUisD3ia+p1MFz1p2I61Z3o2gMOVXul7ZzPvKPLlepaC1mu59mwqo7UEqz+KV4n0BpfjyKMjbDwqmCm6CEd5mKfmd9R5b+oTAY/T2OWJzJmfqkCAEhUBUBcWmoiozUCwEhIAQ8ENiU+Qs2ZP6AIN9wzOjxHwTTe01lRcqnWJr0KsodGbcMeHTzGfjviX/B5CEhQe7h78ll0zk7l8HHX6WeDQjvW9Nwsr0OBPKSlyPx7zl0c1FMlvcKDDj3dwSE96myh8S19yM3YSkJTz+KmpCH0E4T7dZV8rHlZBaxg2dVuW99NgRFDUb/6d8jfsWNyE/+U0VrqOzPRhEeoitXZUkICAE3AmLhdUMiFUJACAgBzwQ+jXscr++5GWszvsOKlE9w2z9DkF5y2HNjXe3PR94isVuqq7GhxFKAvXlrdXWViz4mTyLaBqNvUGUjWao3gULykT60/DolXG2UwpmTQez9cTrMlJzCU0nd/ipyDn1LkRKK1T7cpiBlJd2clFC0MDOS1t6HjINfe9q1wep4ghuHJgNl7OPCwtvoG6iSaqgK+U8ICAGPBETwesQilUJACAgBZwJHivbit6MLSRKx2KBZ8rDPmn973xznhh7WDIbK1MLa5hJKL7w+w7NfbueRD5GS8dGaqmUbTVqK6jWjsk6W6kXAZrXg4K8eeJJwzdz/sce+cw59Q+JWf+Pi3IxFb+L6Z5wrG3jNFByLoZduRxi5OQRGDUPMwBsw5OJNDTyKdCcEWh8BcWlofddUzkgICIFGIJBVdhRBPuEotuQ59X6ocJta35+3Hr8lL1Q69YzYG9En5ERHuymdrsan8Y871rWFdWQpPqPLv9A1eKByd1ia+AoSi3ajY2AvTD/zayQuv0GFowppPwYcu9VglK9sjV193y0cz5bTOytrqXNvVcW3rY2FvSn8en1MYeg1+R3ng5Y1ISAEqiUg357V4pGNQkAICAE7gdTiQ25il7dYye/zy/jnyPr7LipsZtV4Q/pPCPaJoPb5ar1r8AD4G4NQZi1W69p/vkYTkov2o0vQANz4dx/4Gkyqjx1YgV+T34F/p2Dapwh3DLwQPU2h2m7y3gAEjDT5y2DwI79dZ19p7jqy5/keR+g4/G4krLxFuS94amCg6xne+WRPm6ROCAgBLxMQlwYvXwAZXggIgeZPgCeqLU541uOBGsiX8qfkNx1iV2tUZKHwY+T4wK8kcofwIXHlWkpIEEeY2oPdIrgfTTBr7hIsdrnM3zsb+/MkwYArv/qsG30DEHviI86uI9RhRPfzENphrMeuw7tMQbeT/6e2se+sj79uohiFDeMkEv2mLfS4r1QKASHgXQJi4fUufxldCAiBFkDgnf1zHGLU9XBZnGqWWddt2jqLXouaFEX6iv7Z6B+nDh4YfjK+OfwiDhRsUsJYa+/6XkH7fhL/GIZETET3kMEYE3OuaxNZPw4C0f2upIgM/XB061yaiFaK6H5XIbrv5dX2FNn9HPArfvn1FC1hma6tgSIltCOrsbFWcZZ1O8qiEBACTUBABG8TQJYhhIAQaOkE3Ced8Rmx1TY2qB8KyrORY06p9iSt5Ct6z+CP8Vfa5yiuyMOodmfB3ycILKZrU5IKd+Nw4Q7VdDlFiLhv2Oe12U3a1EAgpMMY9DujbpEV2E/XWezSIHRTkpf4E8pLc+DrH1HDqLJZCAiBpiYgLg1NTVzGEwJCoMUROKHd6eRwoIuacOwMLuv5IJ4c+QtuGzi/xnMqt5Xitd03qxi+M3rciymdr0J8wdYa99MaaG4OvB5XsJkiPPygbZL3JiZgo5i7niaw2eimxnYs+1kTH5IMJwSEQA0ERPDWAEg2CwEhIASu6zsXfjTJiV/BvhEUrSEMz4z8A6fH3qDg9AkbhVv6v6aW2WWhqlJms/vkPrvtYqQWxyPUN7Kqpo569u11LWZrCZKL97tWy3oTEfClOMlB7YbTaPprTa4q1gqYgto30VHIMEJACNSFgLg01IWWtBUCQqBNEjAZA7Dg5H3YlbNKhQ/rG3Yigv2cH1uPa38BhRcbhC15v+K7+FdQ7mH2vwbPQpEdVqcvRv+wcVpVle8nt5+hEl1oE9q4IVubo/w7VrmPbGgYAmytTd70DPKP/KEsuj0nLYB/WE/Vec/J72LHl0NhpEx5nAXPP6Qbek/7tGEGll6EgBBocAIieBscqXQoBIRAayUwOPKUak8tNrgfFh28hya4cdau6oqNJrFVIMzUDiZDEMw253Bl2p6PjfhBxeRlccyT3HgfP4M/AnyDMalj46Sw1caWd2Dn12MpoxqFliPuXPYsnYq+Zy5BcPQI+PgFY8QV8SjNj6ftNpr81luQCQEh0IwJiOBtxhdHDk0ICIGWRWBL1m+Iy99S40FzlIaJHS5Hp6DeCDdFI6Ms0WkfdmOY0ukq9Azlx+bAuyfH44tDTyOzNAm9wk7A2V1mO7WXlYYnkJf0m5PYtY9gxZH1D6P/9Er/6YCwXg0/uPQoBIRAgxMQwdvgSKVDISAE2iqBXHO6xwQTUaZYZJuTCQv7fNrQPXgIOgT2UJhY1LoKXg5jVlJRoLbzf74U8/WK3k841mWh8QmwZdfoGwhreeV14FErSrMbf3AZQQgIgQYn4D4bosGHkA6FgBAQAm2DQOegvs7zmOi0OUZvpH8H+MJEazYFgiecvbzrOrU8kiJAuO1ENdtzVsBsKVFt5L+mJVCQshqp2192E7sGmrQYUkVSiqY9QhlNCAiBuhIQwVtXYtJeCAgBIVAFgf7hY3FqbKVvrckYiBC/SBV+rAL2tMO8K09AO5C/keLq7gJPdvMU8ozdHuLqELasikOS6joSKMrcirhl18JcdES3J/9UUhQGCkdmLkqB1VKq21b/xfLiNJQVJknCivqjlB6EQJUExKWhSjSyQQgIASFQdwL/GjQPQyNPxe6sNUrsTu18De74ZzjKrM4T0zgjV7nVLpzCTFFgdwh94W1+RrYKS2lKAodX3e6YpFY5rtW+SJPXijI2Ys+3kzDw/L/I5SGgsslxLHHc3sQ1dyMv6Ve1t7WiGEMu2UaJK8KPozfZRQgIgeoIiIW3OjqyTQgIASFwHARGtJuKS3rej7O63AwOacZhzFwLZ1uz0ex+Lud0uUPF+NXa8KQ1TkXM8X2lNC0Bq7XSEu9xZBK9nGkt62D9M90lkLjOpexsLHT5xeXgb5eoeL4ex5ZKISAEjpuACN7jRic7CgEhIARqJsBph3fmrnRr6GcIwDPbL8QLO67EiHbTcGbsTaoN+/wOj5qKN0/a7baPVDQ+gYhuZ5H3gntWPf3IVouZ/HsL9VXHtVxwdDlsljKnfctLM1CSs9epTlaEgBCoPwFxaag/Q+lBCAiBNkqALbTfJr6Ev9O+UpEUrus7DyODpzjR2Jz1i8fIDZxqmMvO3L/wwMbJKlFF37DReGDYYhhJcCUX7Scf350UqzcaQyInOvUpK41HIHbUw8hL/AXlpVkq1i5bXm1WCw14zK2Bhybre2jn+l8TAyWtQIXzxEQbiWkYOJqHFCEgBBqSgAjehqQpfQkBIdCmCMzdcRn25a2j6WV2MTRvx0zcaViAUe2mOzjwhDRtu6PSZUHLysYT2T6Jewwc7eGLQ89Qsgk/lFjy0St0BB4d8b3LXrLaGAQMFAJu8Ix1yD+6ktwMihAcMwqHV9+JwrR/YPQLU/69saMepdTCw5yGz9jzHnKTfoZ/UDR6TnietgU7bfe0Et3vKqTvfktn5TUo14agqMGemteqzkoW45K8eJiLLTAFx9ZqH2kkBNoCARG8beEqyzkKASHQ4ASSCvdgb94/Tv2y3+2iPfdj1IRKwZtnzoD52OQ0p8YeV2xYTdbiMmvRsa126x9ben9Kmo/pXW/1uJdWWWopwi+HvkNJeRH6B56M9oHdtU3yXkcCYToLbp/TPiPBu45i8GYhIHIAXJNNHPx9FgrT1ysxXETRHLITfsKAc/+g7Gt9qh210/D/Q3lxKrLjvlDpiYNjRoPTF9dUirN2ojh7O01ui0JEtzMdzcsKErH221nkY1yAirI8RPQ4Hz0mvOLYLgtCoC0TEMHblq++nLsQEALHTaCoIhfBvhHgd33Rr7P/7leH5+o317jM0RnYssviWSt+FRWwbl2A/bt+Q25gMH4PL4a/TxBm9bZbg6002z+jJBFPbTsfFpTDYqUgaNYS/GfoFxgUMV7rRt7rQaCq+LuFaWtJDK/R9WyfiJj4z73od+Y3unqQm0QmeUZY4BfUwVHf7aS56DKGkoqQewwnuqippO1+G6nbXiSvB1/qqkRZcTliBLte7PnO2c0iN+E7pEUOQofBN9fUrWwXAq2egAjeVn+J5QSFQOMSyNz3IVJ3vKYe/4Z0nEAWqrfox7j1z4ftFjKYfG2dz5PdF2ICujmAF5V7FsWOBh4WogO6Ir88AxaLXfD6kLfELUd8yGkiF8XYQgksbDg7yYBXu1nw4KYpuKn/K/j5yAIkFblPcntjz82Ye+JKFR7Nw1BS1QAEykvSydc3jKyq+U69seVWK+wDfPjvOchPXqZi+dqs5Rh66XblI8xtjD61C29WnLkNKZufVd3aYJ/sxuNwkoxAcoMw+oW6JcvIOvCJCF7tQsh7mybg/G3dplHIyQsBIVBXAmm7FiB58zP0qDdDWZgKjv6FhJXVP3av6xjNrX1KcRwe3Hgq7lw3CmzB5RLoE4Yg33DlQjDv5L8chxzp31FNQHNUuC24T05KL02gUGaBysrLzSfmGMhqS9a/Y/sa6ZG52WDD8AL7vm/vu9Oj2OXmBvrHWd2kNB6BwIiBsJKA1Re2vvqH9XZUxf15pYq1yz7BNk5aQRb5uD+vcmyv7UJhxgZwtjd9YfHMQjov6Q8Su5orjL6F+2dMv1WWhUBbIaB9h7aV85XzFAJCoAEJpGx5Tjfhhn/Hy1CQ+jdK8w404CjNp6vC8hw8sGkyjpYcdCSN4KMrtRQoN4KHhn1DrgaVj6XZ7eCOge+oE2DxqS9sDeYYvZ5K/7CxGBxxCsL8ohFWwS2d9/Wjp+ZBHDighlJYkaP6qKGZbK4HgYCIvuh0wn/sPVB0DaNvMHwDotDr1HcdvRamb6C/DX18X7LP5seDfW7rUnxMYSR4XR/MGsBW5pz4xdSVffKk1ieL404j7tVW5V0ItGkCInjb9OWXkxcC9SNg8HG2Ntl7M5K11x5yq369N7+9/0r9DL7kX+taOA0w+8w+vvVsmF3SzvYLH43nR6+hRBQPYHzMRWQNDlW+vxM7Xo5+FIbMU9mduwZX9H6cXBsykRhgRQX1ry8snlP8nev027Xlk9tfjE5BlZZGrb6tvtvItzlj3wdIIR/Y/OTlDYah/YDr0PeMr9Fx6L/RbfQDOOGydU5uCp5cFjiaQl3Dj0X1nKH8fw2ODHx8I2SjyGbOWfq0E2OxG9njHG1V3o+DQFlZGZ544gkkJSWpvT/66CP8+uuvte7JbDajtLR1fh/WGkIzaSiCt5lcCDkMIdASCYR24kkyzl8jVvJlDIjo1xJPp8ZjriArHU8Q81Q49FhJRQE2Zbj/GHJc3SByexhIE8heGbcZb5y0A9f2/S8iTJWTl5z7tGFj5s/KrWFLmA15pLFZVLPwLSd3hj3BVhwJCVRuFP7GIKddOUsbW4ZP73wDbuz/ktO2trzCaXy3fz4QKVvmIm3H64hffh2S1j5QaySF6RtxZMMTOEr7cwQEfeGYyXOTHsLTRYvwXM7bKLA6J6WI6j3DxRWBrhKFP/MP6arvpsZlg9GHoj/8icieF5DLRC/wRLoek98FW37dCn3movtf7VYtFXUjwGL18ccfdxK8v/zyS606ycnJwbBhw5CYWDdLfq06l0Z1JuD6bKTOHcgOQkAItF0CPSa8hh1fUjxSmrxlpEf5Rl9Ko0sz040eLb8Ny2nLli1g64tW+vXrh6ioKLWamZmJTZs2oUePHujfv7/WpN7vo2POxjeJL1bZD4tSi4s/58H8zXiaoif4G+1xWRceuAevjN2McFMMtmb/4bGvDoE9KZGFiQSvLyyGcnzU2Yq+RQYEkxtDNvkzJJLXhIHcRx4c9rVKTHHX+jEI8AlGBY3NyTA4DNofR9/DpqyfadLaKpUUw+NAbagyedPTKoEE+7xqJfvQNxS661yEdqw+kkXG/o+QvP4R+250TdJ3zcfA81fAP7SH8uN+aPNUrUuwG8mdq+1W/VA/++cx9sTHYC5MUr62/HcSTEK158S31D4VZbmwmPPIckv+3lX83bBlmm95OEawgZJSdDtpnmM8Kz1RYDcKi1k3aU79PfpTfyZHO1loegK5ubnYt29f0w8sI3ok4Gya8dhEKoWAEBACngmwwB0+az/6TPuUojOQCDhvOUz0w93YpYLCdN1777345ptvHK/k5GQ1LAvh6667Dvv378d//vMftb2hjocTQvzf4A9UdyZKDexaOEnE8OhK8cMWXxa7XDi2rj2+LkVY2H2jqmPLr2thC+153e7Aye1nOE14OxBsw1ay9rLY5cLi+scjbyDKvxMWTjiE+4Z+iQqbWYUlK7UUkr3Zgqyyo/gs/kn7Dm38/+KMzUTAxTpPVt+SHPfoFnpUZoqC4BC7vMHG4hNIWHW7el986DlyMNH7WJMlnkT1b8kL1Xb+j6OW9Dp1EYZdvheDL9mE3lPeVzeHqWRp3rVkLPb9dA62f9affN8PkjA+goO/X47d307Egd8uReLa++imcji2fTZAtdMLdu6b3SU4TrBapigNvv4RCAiJxeCL1qo6+c9OYMOGDbjrrrtw3nnn4bXXXkNaWpoDTX5+Ph5++GGceeaZuOqqq/Dnn386th3vQlFRER54wP4Egftmq/Btt92GlStXOnW5ZMkSPP/88ygvL8eNN96IHTt24Pbbb8e5556Ll19+GYWFzk8LPvzwQ1x66aXqPF566SXwd6GU2hFocsH7+++/N8gFYsvO33//jTVr1qgPina6/Ohg7dq1jteBA61z8ox2vvIuBJoDgeCYkfR4dZz6EW+K40lISECXLl0wd+5cx2vo0KFqaP6RePrpp3HHHXfg7bffxqJFi8B+dA1VhkdNIZeEnXj0hO8xImqa6pbj8Yb7tcczI/9EqClS1ZkpRur+vPXkyhDuMrQNCYXbaNJbGcbFXEBSyflrmF0j+oePU6HEnhu1TEVs4Glr7sWG4gq7VY8twWvSl7g3IVH8T/o3HurbXpUfZx1zucFgy6lfYPtqYZQXJcPH335N9Q3LChLUanFFHrqX2HB5ihFXHTViZJ5B3XhwEhB94aQVHDfXx9fugpJL6YvTdr6uJn1aKVEEl73fTyOhO0EluTAXJqKIkllkH/xChfyjhijN3YdDK2fru1XLbGnmMGds+R0w7U2MvWqTZzcHtz3bRsW6deswdepUxMfH48ILLwSLzBkzZqiTZ2E6atQo/Pzzzzj//PPh6+uLs88+G+yrW5/C/QwfPlx1wd9NsbGxSE1Nxf/+9z+nbtldgovFYsG7776L0047DSUlJUrQvv7667jmmmvUUxtuc+edd+Luu+9G3759MX78eMybNw8XX3wxb5JSCwJN6tLw9ddfqzuWiRMnqg9VLY7PYxP+MFx//fUYPHgwjh49isWLF6sPET/q4Q8M37lFRESofdl/hj8cUoSAEGg9BPhGlgUvW0345pd/JIKCgtTN9JEjR5TfHJ9thw4dVD1bf3v27KkAHDp0SIlkjQZbfKZMmaKt1uo9EpHoAhIZXb5FSmEcJZ/IR5eQfgigR8s+Pj6IK9yM5zbOJFFrRrHF2d+TB7CQlfDGv/vgnWl7sL9wLRLyd1Kt3U7Illu/ECMiA3mUSCw+JwMb037FU+vsP9D6A5zSfSYiI+1i7HDxNv0mx7KFPH+1No7KRlzg82e3Cr4e3igsNLi4nnPwaS/j74X0W8Cil8QjT/zyD+mMXidcU+1hBvr2R/wxq66+If/e8BiT/fqiNP1Xio9st/KeRJc7nIxuk0++RG230ATOHT/MRH7aRjWZ09c/HL3GPYzsfR/RYbhMZuIJkXRsbpboYwOzdbcofS2CTGb4B7v6f0ciukN3+PlRH1Qiq3CPONZVo70ZjRQ4j9j4+/vXaYzGtFTec889uPrqq8ECkgtbT9lKunfvXvUEKCUlBRs3bkR4eDhuvfVWsHsUP0G68sor63QO+sZ8/pdffjkefPBBXHbZZapPfvLEQjs7O1u5X23duhW7d+92GoetzAsX2p8OsBDn119//YXOnTur4//4448xc+ZMNRSLXdY3vH3SpEn64WXZA4EmEbz8QX700UeRlZXl4RDoUR/9YPEPEv84BQc75x//7bffwD9QN998s2PfL774AmPHjsWcOXNUHW/jO7hx48aBfwjZ6tOtWzdHe1kQAkKgdRFgdwX2jWMLCn8/vPPOO3j//feVJZe/Q/gHVyv8I8Y/MJrgtVqtTr6/bFnRt9f2q+1759A+Tk3TKePZ/aunOdVVtXLrsuEY2m4ijhj2kVWQ/G+PNfzPyslYMG0H/MkaGJ+3Dc9tmAl2oTDb7AIpwCcEE7tcgtN6VIq1ziF9sT93o9tQp8ReXK/zc+uwFhXMsz5MazFElU14XBbcruOzQJx0awr2rbgbZYVHEdl1MrqPmuPWzrXjwLBu6DX2IRxYdb99E7kncJzdEy9brvatWPeGQ+xyAz+bAcNK/NHbt7Pa/s/7gylsGP/22a9ueUkGHcNdx4StvcvK/9nlwsXtonKjWlL+vCTAXc9P34y3Vbdd37ahl7VxtfeG7r+u/fFngYUli16tREdHY9myZWp1/fr16NixI5577jlts9IjbDjjm+ewsDBHfX0XWMzyPIMvv/wSt9xyC9g94YwzzkCnTp0ckRzOOussxzAnnHACYmJisHnzZmXc43Nh14xt2ypvbkNCQpRYF8HrwFblQpMIXv5BYTE6ffp0nHrqqU4HwxNLOORHr1691A8Y+67w4wStsBjmRw76cvDgQZx++umOqpEjR6q7JLbm8g9bRkaG8pOZPHmysgI5GtICf4BXrFjhqOJ9unfv7livakGzGgQGBjoeL1TVtjHq2WriDYuJ9qXFd6t8597UhbkzcxYpTV34fHl8b3DXrDQa/6Y8dx6bz70pz7uuny32deOXdoz8PcHWXrb08veNvvANd0BApb9t79698d577zma5OXlqe8NR8VxLvCP0Z68NVibtYScFDgzmvNxeOqWoz5syfjDbVNxeSGWx32NoZETMecf1wlVBnQO7INZ3Z5EUlq8igkc6tcOZ3e6HauSFyvrsb7DS7o81CDnp++zumW+JnwN+Jp4o7Cg4O8LnjDkqXQc9ayjOifHcxtHg2MLwd0vRy+/zsiOX6Kyo7UfdDPKbNEoo98bnkjm6lfrS/FvE3ctQVD0cIrowO4KdrHr6FdZcR1rlQtcf8wCXVmpXzKQW0QZisuDUExjeypsdebvDf4t9Ebh3wr+vSouLq7T8Pq/0TrtWENj1g/sB+tqTNN240gK/JnVfwd17dpV+d/q67T29Xnn3xP2EWYrLX9/ffrpp8qfWN+nXo/wdeSn1Xz8/Hnm/Zmv/neBXbf4abeUmgk0ieDlC8SPDT0V9pNh6++JJ56o/kBZ8LIwZh8btuBw7Dt2KH/22WfVhZ89e7byg9HfdfEyC9m4uDj1Jct3QCyS2ALMjxD0ApotwE899ZTjUJ555hnH409HZTUL+nGradYom9hS5a2iCQtvjM+fH28V/uI2mdrmTOem/LzV9ceRnwixlUb7XPITHX4s2a5dO3WDzGJL+9zwDz8/DmzMwsL16W0XIKM0ieLwlngUu74Gk/LtrM1xlFJYqzf33oILut2lYvYWVeiFGU1cozTCB/I24pXd16vuODIAx9y9qPu9+D7pVSWveoeOUGmH9YkwajN2W2xTRhEUUrf9DxUULSGy+zmI6nWRG4awzhPBL9cSFD2C3AzYsl4pai3kk5u6/VXKfEb+DY6Yua57uq4b0O/sH5Gx+23kHPpW+eDaozOQzdfCE5Mo255vIDoOJ+uwlFoTYAto+/btlZ6YNs3+5IVviNilgcVinz59wE+S2e9fE7isJXiOEFtX2YXyeIsmTPlmWCvXXnstXnjhBWXl5bkFrtqILc9sIOTCc5JYs7BRj3+HeGIbu2Ow/y4XvrH84IMPlLuEqpD/qiXQJIK3qiPgO6vt27cry4wWyLmgoAC7du1Sfil8p8rWJnb05g+AdgfIIkRvxWELDgvcgQMHKn8czXeLP8g8YUUveNnCrA8TwsfAP5Q1FR6b++XHHN6wNvLYfKxNXfgPlh/38NjeCJ7NNxh8h66/3k3FgAUVf8GwBbCpC3/emH19vmyP95hDQ0PV3xo/KWmqognX2o7HM53T09OVFYbFMv9I8I8XW0DY3Wnp0qW45JJL1JMe/tvRvhNq239d2z22ZTpSS+LcrKv6fk7teCV+T1mkr3Ism8iw50u/icUu38jfJv5PTVpzNDy2wO4Pz2y/0Kn67/Sv7GHM6HE3J8fYnfu3SnLh1EhW3AiYi1Kwd+kUenJHF4GiNhRQml4WsF3HVVqC3XbSVfScuAA7vxpJYjRIWXo1a68Su9yOboZI9fICr1Vd6JrlH/kT3U9+meLnXovy4hSYQrqpSXW7lpykjo1jXB/d+DiKKXNbj4lvVN2XbHEiwNZUFpns7zqZnvzOnz8fq1evVi4F/N3DOoGfNLORjH/nrrjiCuUWyX6/9fkO1sIk8pNs/h1lI8KgQYMwZswYFTGCfXy1G3PtgNnNgYU5W3ofeeQRJch53hNrHA6xyAbCV155Bfykig12b731Fvbs2aPtLu/VEGj6Z9QuB8M/UHzHcsEFF6jXf//7X+WGwK4GPJFkwIABanYjL2t3NSxE9I9rNAsOm/z1opCtPt4SqC6nKatCQAg0IAEWs/yYj5/gsKWGLSD84sJPgb766ivMmjVLRWnQQgM14PBuXaWVHPIodjkpBEdw+PfAhdic/avbflqFmeZRFdPLtZBTDToG9oKfLgUxi9lJHS4nIew+KYwnw3FhQcztvk6Y59qlrLsQiPtjlt0lgcSuVnISlqI4s9JPUqvn9/LidHJt+Bq5h3+k/SwqjfCwmfvRfsht9n70jY8tGylcmF30etioVdE1s5bb3feCyWoc0e0sBEUNRvyya0jssntM5fHlH11BkRzWa3vKew0EHnroIeUGyU+PWXSyDy27NbHYHT16ND755BMlHNmXVtMcHLqsvoUNNuy3ywL6ySefdHTH31tsyGNrr2vh42FRzoKXjX8//vij8iNm4993332nXEU46gPrIA6fxk/JeVlKzQRc7Ak179CQLdjqwr4nfOHZcZuDxXPYjVdffdUxDM/EZlGsL6eccooKIcLv/KPHoclYKLMljuPsff7558pC9cMPP6iZi9pjCn0fsiwEhEDLJcA/JGzdYOsuP/HRW0n4h4IntvINsBatpbHPlJM+FFawJc+53Dn4PfQOPQHLUj6gmLjJzht5jZ90avPrtHddKx/y5+R4vAXksrA+Yyn8fYJwcY/7kWtOw/rM73Ut3RdZ9B4p3ue+QWqcCFgr3B9ZG4y+NNGsMk6rtkMhxfKN+/0ycrMln3ASodZVt1E4sB3kfhCKvKQqbmjoGsYMuBYdBt+K3KRfkEXRGdh1oiw/jrqtfNTNy2GxznNceFyL2YMPMGVc83R82nHKuzMBto4uWLBAaQv+XuAJ8vrCUQ/4xa5S7Magd2Njgax3SWD3h7oUDnfGT671T7FYk7BoZUuva+EIVBylgfUMH4u+sIWX9Q5v4yfb7MIlpfYEnJVk7fdrsJZ8cR9//HHlvM2uAhy+Q38ReZaia2FzP/vX8AeUPzj8WECbgX3RRRfhpptuUh8G/lHU++u69iPrQkAItGwC+h8R1zNpKrHL417W8yF8ePAhlNvK1GFwbN3Y4P4YRKmEuXA8Xo/Fg8jV2tE8e9Xf5E5XktANJKH7H20TOMYrW29LKMFEVYUnzsUG9atqs9QfIxAQOYDEYzqtVVpQLeZclbpXD4nTCR/81e7bq7ktcIbBhL/vRG9KKmGj8GMeCwnjDoNvUTGqoyglML9619m+AABAAElEQVS4HF5zD3LiyQ3FFKEsw51PuF+lCnbtIzBqEMzFR5XA1rZZKataQHgfbVXea0mAb4xdxa5+V46VW9vCbnZ6Iey6H2sTzdjGbmJcOKIM++Wyge6+++5z3cWxzoLbVew6NtJCU86v0I/b0pebXPCuWrXKiRnf5XB8XnZFqK2fHVt8WcjyXRPfuektwBykmWdBsuWXBa8UISAEhEBjEzil42UwkfX147hHlO/1+A4X4dLuDzqGbedPT6pqO2mNjH6+FPaqV9hIzB74phK7jo6OLbBFOdAnFPnlma6b1LqJXCA4vfDlvR7xuF0qKwl0G/8Cdn11oorLqwnZjiPudROUJTm7SJyGqzTAjr3Z5zdlDYU5S0JY17MpU9p+xyZtoe9ZS9VkM+4768BnyrobHDMK3WncmAHXkYtEKgLCepHA7qnt4vTeddw85C0e7nR8vgHRSN74JLpSogn/kK5O7WWlaQhwfFzOA1BV4QQXbFXWF84ZwC5WbNi74YYb9JvU94arddmpgazUm4CB7lD0z1Tq3WFL66C2k7Fk0ppMWmvKz3ZbnLTmLasFPx6sa5SI6j4LHP6IZ1+XmcuwOetXssIWoGfICDy8eSrZfe3hytgCzBnVnFwauNNjLg4Lx8eRq4apymGKKLvXnWtPUL66+ka+Bn+c0mUGYgK6YlrMv0iEH8tDrG/UyMtsdWfrV3MNS+bp9K2UACJ99zvkQ0uGktgpHi2txdm7cPC3yyh5hLtVnX10tWxp+v47j3oU7QdeT5ZAK/Z8NwkVpdkUcYEswWT1bT94NjqfUGm11+/numy1mOn4FiBt19tkSWYXB60Y0Of0xQhpf6JWoQxHPOFVP8/FsbEJFtiKerxhyWpr9GqC02i0IXhSHH+/S2l6Ak1u4W36U5QRhYAQEAJNS6CCrHmPbj6d/HY5AoxNid7r+szDN4kvKv9bJXb5kNil4ZjJgRd7WSMwZ8wv1Ypd3o0UFE1k80eFpVytav9xpIExHc9ClCnWY3QHrZ28OxMwkk9ux6F3OFe6rPEEsrAuU2iy2i/E3+y01ZPY5QbZcV8owXtk/aNkyU0j14XK/TL2LgKn5M4+uBjmoiSK2TsSXcY8SZY+97nk7FOcuu1FpzHtKzYkrf0PBp63zMM2z1Vl+YdQSCmL2Vod0e1Mz42kttEIiNhtNLQ1diyCt0ZE0kAICAEhUDcC87bNojBlh5zi7r53sAprHindQGMI5p9cfWgh9ttlv16euBbsF4G+YWOwI2c56eXKh3QWSnbx0qYblTsDt+0XNhb3DfscRk5mIKXeBDqP+A8Kjq4ktwa+0ajkXlXHPOGME0UUpq11Ervcnp+tHlpBj7WPJZooydlLLg+fUtSHaPScNJ/E8ChHtylbWeyyEK70M9Y2VpRmaYs1vuck/IDENXfRkCYyMpchiZ4ADL54I4zVPE2osVNpIARaCAH3W8kWcuBymEJACAiB5krgcOEuJ7Fb03GWu1gM9e051Nj8vbeTC8MozP5nCB7cOAVsQb590AJEmDqokGUcvsxebErs8jIL4QP5G/DKruuPbZO3+hCwUMiw3d9OILHLSUBqFrsGH38lWvd8NxllBYfchyYfayViySpvLyxmreT2kI4D5DqRT8JaK2V5B9Q2bb3y3Qj/0B6Vq9UssWX38OrblfBm1w32KbbQe/Kmp6rZSzYJgdZDQARv67mWciZCQAg0EwIhFHu3toUns3UPGVJl8ye3nod1Gd+hzEoJWCjU2NGSA7jx7974LP5JvDR2A87vOof29SzAOL3xfhK9yUXuk6mqHFA2eCSQn7ycJp+FeNzmqTIwcii5P3yvEkjY4+hWtuJ0xCq0mQeLrWpFNznJGx5z7BAQOZAmrblnnPQxhaHPaV842lW3UJS5WU18c25jpaxu3zlXyZoQaKUERPC20gsrpyUEhID3CFzffy49gK7ZjYATUwyk0GUPDv/a48GWWYqRRGmEXQtbb1elfonP45/ChswfSDZpVkLXlvb1imosyJ73kFo9AY7CkLbzNY8T1vTtnJetx0Stcy07btv4xZPXKBpHVaVCWZLtWzsMma0sswatPb37BnbA4BnrVLizqvpwqqd99D7E2jb2P2ZLrxQh0NoJVP3X1trPXM5PCAgBIdBIBAZHnYInTvgZb+27HWWWEpwYPR1ryUqbX54BP4qkYKHH2FM7XYNJnS5H56C+1R6FyRhIcXfdBUkFxfxdlvIRugYPrHZ/jhLRJXhAtW1kY9UELBTzds+3p1TdoIotnLKYZqB52ErWeG3ymsOdwbmZwWiiyAujHZVGco8YfkU80ne9RRPcjlAGtqFo1/dyx/baLIS0H0PN2Mbl4gdMY3HyDR+/2luvazNec2rDUVg4UUNDFo6EocXXbch+pa/GIyCCt/HYSs9CQAi0YQJdQwbimVF/Oghc3uthssb+hMLybPQKHVGtG4O2E09QGxxxCjZlUWQAD24LFSSczu82By/vus7NyhvgE4JQv3Z4ePg38NEsg1rH8l5rAkeUa4EunEYt9wxqNwSFqWs8XDVdBxSRIbzL6fALiUXmnncpcgLFjic97E9JJXpMfEvXkLWzAR2G3OpUV5cVv8AYFevXnuFNt6e1DL7+UbqK1rfISa341ZCFr4eUlkVABG/Lul5ytEJACLQgAhzmfGniK1id/hUlk/DDdX3nYjRZe+tSbh3wOh7cNBXppYdpt0pfXXaZCPGLRJgp2qme+w43xWD2gPkqrbEvWfCkHD+BitJM2rmSe2176jT8bhhGPoC9S6eQ20GwI/6u0/70+eCEExyPt8Ogm1GUsZEsraEI7TTBqVlDrHBos95TP8Lub8aD4wZz+DMOY8chzQyUqliKEGjtBMSHt7VfYTk/ISAEvEbg+Z2z8G3iS8ggsZpSchBzd1yO9Rk/1Ol4WLDOG70Kc0etVPtxhrVgmhTXK/QEvDDmH7x/4D6y7jpbr9jym1WWDBG7dULtsXFYlzOovg6CkMKM9TvtAwRSymLOoDZ81gF0HTcXMQNv8NC/FVG97OmK2QIb0e2sRhG72sDFWTvg4x+pXBg4AUa/6T/CL7C9tlnehUCrJiAW3lZ9eeXkhIAQ8BaBzVm/YXfuaqfhOcoCpx8eE3OOU31tVjoE9cCiCYeRXLxPTYiLDe6ndiunR9KuhQVwuQp75bpF1utKIKb/lchLJFeUtDXV7EpRj32DlN/tgNPINYH8YTmjFpf45f8iy+1mnqvmKCw6uaL7yS+7pTB2NDrOhYQNLyBl94dkUbag08j7Ednd/lnjOL8JK2929GqzVODAz+dj0AUr7a4Uji2yIARaJwERvK3zuspZCQEh4CUCLECf2nwREgp2OI7AZI4heeODMlMqiivyHfU1Lfx65B1szv4Nob5Ryh2CE064TlKb0OESfHnoWXroXvnYvYTSzw6LPLWm7mV7LQn0Oe1TFKT8DU7ckB3/JXk4OEfF4Elmg8g1wC+oA3xNgeQqQLn0KLFDwuo7aT+7Zd4xFPlTR/e7miafHUV+8jIleE3BnR2ba1pQqYnplsdTsohDK25E3pHfHV0k/n0XpSRmAU7prouSHfXaAsfizTn8A6L7ztKq5F0ItFoCInhb7aWVExMCQsAbBP69cjRySlMpUNixWeE8Cam8IybsIGuvwYYtE8bV6rCe2noBEgq3q9i7nDVtY9ZPmHfiKrQP7KH235+3XoUlK6YoDDEB3cnHNwEhvpEwGo14aPSXiDR2qtU40qh2BEI7nUyCtiNyEykMHIXy0hcb3eSkbPsf+cL6ouuI2TAFd8HOxaM8hzEjsZy245VjuxuRue999DtrKYLaDdN36bbMQjdh1e1KeNso8kdE93PRfcKrajIbN+bQaXqxy3Uchqwkaysveiw2m5W0e2W6Y4+NpFIItBIC4sPbSi6knIYQEAJNSyCNUgc/ueVczF4zGHesHY70ksOUAc2M7LKUSrHLh0SPskv8E1HmnwIfaxDOPLShxgPdmfMX4go2KbHLje3WWwMWHbCnJ44v2EL+wJchvnArpTCOQ445laI5TMAdg97B26ftQL/IypBWNQ4mDWpNICC8t3Jb4MQRjkKTEblkx31BqYE/wdbFJ2P9h308i13VstISr4UIS1g5W22p7r9dX49F/pE/SaCWqGZ5Sb/h6OZnHLsoEV7HCYrcV3jX0x19tPmF3BwgM6PBMWRmZuLtt99GXl5erfsuKbFf51rvUIuGjdFnLYZtNk1E8DabSyEHIgSEQEshUFSei/s2TlSCs9iSjwIKNfbgpinILD1CqX7dM2JV+BTAYKXsWvQoOi/NHyWF9q9eTizx0s5rcMPqPrh2VVd8c/hFhSDPnAmenOZcbGoiGte9uPNqEsOVcUXZXze+YBtFgqD4rSb2D5XSGATYTcFSXkwuC7pJguSX7VbIVaAupbw0q9rmZQUJykVCHy2CrcpZBz937FectbMyvq+j1nXBPvnOxxRBk9ei0OeMJWSNrr07hWtvrWadQpYZP1gEn2efgs/c5+B7520Axe5tqPLuu+/i4YcfxnvvvVerLufMmYOff/65Vm1r26i8vBxDhgypbfNW2U4Eb6u8rHJSQkAINCaB75NeVxPH9GNY6VH170cX4Yxu15PoDajcZDMgrHgogszdVZ3VYsD27GX4K/Uz3LJmALbnrAAnkeDy05H5+DX5XZUoQi9oeZsPWRI7BvamiXB/O4ld3sbFSNEBCiqy7Svyf6MQ2LN0qgod5uTDS9zrWzxlQHPqk3yCjT66z9SxjdaKUnJjWEYT425wsvY67eu0YkG3Ca9ReLKP1WS1kJiRTlvb6opxwZswbN8GQ1kZDOYy5Q3v8+pLdAOhu7GpB5xFixbhxRdfxFtvOcdW5i5zcnKwfft2FBTY3WTYCrt7927k5+ejjI5HK+np6UhKStJWHe8Wmpy4f/9+8DsX7s9srnRT4eXc3FywlTkhIQEZGZUWbJ5YuXPnTjWWo8NWvCA+vK344sqpCQEh0DgEyqzFbokeOL0vTxa7pf9LKCrLx4qUz2C0+qNTxiUYGjf/2IHYUGI6gvePzFai1e6qUPmImye8/ZT0Js6IvQHndf03vjpsT1HMVmN+xeVvxpt7bj2WeY2n/VfuW1SRix4hQxvnhKVXZWEtL04h5M4T1shRtn50SDD3p/Bg1RVTaA+YQrqgJDvX0YwnynF64UMrrnfUVbfA7aN6X4qoHudW16xNbjMcPACDLhOb+svKI9YpdL1jY+vFZNWqVfD19cVVV12Fp59+GsuWLcOUKVNUn59++qmy/A4bNgwbN27EvHnzYDKZsHXrViVQu3fvjv79++O6665T6yxWhw4diqVLl8LHxwerV6/GhRdeiBEjRigx/NJLL+H7779HdHQ0nnzySTXG888/rwQvC2lOvnH11VerNitXrsSsWbOU1XfTpk1KkF9/fe0+S/UC4sWdjV4cW4YWAkJACLRIAmOiPYuGkzpcqM7n+gHzsHBCPN6esAenVZA1mESNj58VFbEbsWLEUHC6X7PVs48ei2ku53S7HQ8O+xrndbuTklWcg8KKHBRb8tS7akBil5NZcEY1TkJxz5BPEGGSmKp2No3zP4vG6gtJJWoT2V1LLsLSiV/VFPps5Cb9Uk0D6oGyevWZ9plqY/QNVIksQjudgvIid4ufp47Y57jb+BfRdezTKMrcQtbg59QkuwpzzT6lFnMBhTObjZ1fjcbubyehrCDR0xAtu45EplthAUyisr6FrbvXXHON6ubaa6/F/PnazS/A21jkfvvtt/juu++Udfbiiy/GmDFjlBA+9dRTcc8996Bv375gURofH4/CwkL8+ac9g+Mll1yCDz/8EL///js++ugjfPbZZ0ocf/LJJ47D5noel4Uvi2R2lWAB/uyzz4Lb/fHHH9i7dy/mzp2roos4dmyFC2LhbYUXVU5JCAiBxiUwMOIkXN7zEXx+6CmKjBClJpdd2P0uDI2c7DQwJbPCKZdlk3WQDINkjH1465UwlxQ6tdGvsBV3SMQkR1W/8DFgf95vE//nqNMvtPPvguldb6EJaxMRHdBFv0mWG5iA0ccf7frORNb+T8ioW/mouXIYZRdUGcxyDv9EAvNllBcnq/Bj2fFLHJPNKtsfW6KJjum75qPD4FvJbcHfbbNW4WMKxYgrE2AuPELRIPzIlzif3CtoYqO50uqrtXV95/BjoZ0nks/vF0haex9tZmHuS9EiXsXA81fCP7Sb6y5q3UYf3B1f0lMDdts4Ztnes/RUDDj7FwRE9PW4T0ustJ58CowrKOMc+blysXHaYP6D7dixXqfDbgqLFy/Grbfeiv/973/IyspSwjY1NZW67qiEKIvRd955B+effz6uuOIKt/HWrl2rhDFvYKE6Y8YMJWwHDhyo3CCmTp2q9hk9erQSv7wSGBiINWvWKIEbGhqKwYMHQz9hjS3FbB1mscyCmAu7QvBYJ510klpvjf+J4G2NV1XOSQgIgUYncGaXm8jyejZlUUsksdm1WsH5d8YX2EGRF0osVYtdPmB2aZjS6SrHsfOktjf23uxYd11gy3HfsNHVju26j36d+9+bt5bkjwEDSMSb9L7H+oayrAjEjnoEVnM+sg99S8LW55jwJWGkiv3dRuHDuCRvfFSJSQ4Vlh3/taqr6j+blScgav1U1cpez64NXIzlwSREPT+kNZBwttEEO60Etx8DW0XZMbHLtTTWsYl1CatuQ9cxT6O8NIOyww2mSWydtN1IIH9KWjeA+rKfk9pAwjd589PoPeUDR7uWvmA75zzYyGfWsO4f2Pz8YOvTF9brb6z3aX355ZfKOtuhQwflTsAil90XtElsV155JaZPn65cDFh4svV3x47K+N18ALHkUqH3yWXhWkHWZxaybLHlmM9aYd9fFsIsotldgkMUatZlrY32zq4TN9xwA/zofLnccsst6NOnj7a5Vb6L4G2Vl1VOSggIgaYg0C4gFvyqrjy/Yxb25a2jiWk8kYStgPrC65U/WLxl3s6ZuG/ol2ArMrs+BPmGU7IKz4+esygqBEeHmD3gTcre5tnNQj+afjnXnI6HNk1VR2QlP1SONvHK2M0IN8Xom8myjgC7FnQb/4J6ZR74lETtU1VablmIcJvchB+dBaOuP7VIwpknpHmalObaVL/O2dx6TJyPuN8vo48VRQAh8Wsw+mD0zFU4uPphZCf8TtnfAsiyO1lZjznSA2d4s5RR6C1dKcnZjYN/XkH7+yprcfdTXndkZ+OIFJ58lC1lNVuVdUO0iEXrrCthvZRY8p/jMRFY3wNfuHAhOOKCXnSyX+7dd9+NBx54AOy+cOONN6rtkyZNUv60HE0hICDAYZG99NJLlesB+/3yJLOvv/5aCdqIiAiMHz9eiWXuh32DH3roIfzzzz/KX3js2LFKzD722GPqNFjgcmHxHBMTA95++PBhsOg+evQo2FK8fPly1aa1/ieCt7VeWTkvISAEvE5gU+Yv2JW7SncczuLWVexqDT84+AD+e+IKBPmEwY9CjVVVzDa75W3h/nvQM3QEJaDoWlVTp3qOKDFn3SinOl55YecVePKEX0n8uApzt6ZtvsIvsKPdylsFCSsldEjdOq+KrVzN6YhDYQqMRq8p71fTrupNoR3GYuAFq5B7+Ed1LJE9z0dQRB8MP5diAmemqbTGBUf/Ar84Tq+R0h+7FfosWMsrnzwcXnUHZX/rh8CIfuBkGylb57rtEuHwUXbb1LIrfO3WzoY4CfaL3bZtGy666CKn7th1gV0cfvzxR+Wfe/vtt+OJJ55AcnIyeNIZW1wnTpyo2rA1lyeWffPNN+jdu7ey7PL+bI3lwqHOLrvsMjUZjkXyK6+8ourZosyWZBa57dq1U3VsDZ4wYQI6deqkokDwmDNnzlQ+xDyZ7f/+7/+Um4Vq3Er/E8HbSi+snJYQEALeJcCZ0N7ZP+e4DiKzNAnPbJuBaZ2uUckknt52gT0MGglRqy7+rta5mVwh4vO31Frw5pjT4G8MgjZBTusnrSRBxRQOM9l/JLV6eXcnEN5lCoKih6Mw9W/3jVzj0c9X39RAbgaFNAmsAHu+m6ji4g4i8Wohlwl2g2CXiYju58DXP1y/k9uyf0hXsuDaBZB+4/5fLkRp7j4y0Nr9UnmbwScQqNDiy2o3NS43YTRuwdEVSvAGRQ1BlzHP4Mi6B2D0C6MerCrxRvtBVbvZ8DhSgAEDBqCoqMgNBYtQDhGmlS1btiA7OxtssWUXBC7//ve/cdNNNynBynVsveXQYkFBQapO2/fkk0/GkSNHlG+wJmy1bXzTyi4L+rJixQoKL1ys+mFRzBZe9udli29bKCJ428JVlnMUAkKgSQlw1rVnt8847jHZ/eFA/nr1OpfCk70w+h+sSbf7ga5JX0LZ1eKd+rZRSLTfKAbw2PbnOdVXteJvDCTp4h5OiyNHmDzEe62qn7Ze32faJ8ptoSBlNU1QS0ExRUDw8QslS2ogytl1QCc23Vk587eUZWPXkpNoF/IlJWHKcvTI+ofID3gFTSrr4bZ7RVkeWXaXkrtEucqW5hvYno7hKMwBFvgHxaAkZy89QGDfYF0ha27siU+gMH09Cdgg2v8nauIiymiffDqf9oNuUjtG00S90I7jKSTaTvgGtENIh3GODstLMpC6/WVUlGYivMtpFPbsYsc2Wag9gaioKLfGbLHVFxbEVRW92N2wYQPeeOMN5aZwxhlnuO3Collf2orY5XMWwau/8rIsBISAEGgAAmvTlpJF1pckpYvgOI6+f6NEFJM7zlLhyXj3AeHjSUw7Pyblek5FnFIcj2j6V1Jhw+HCgxSvNxOdg+jRtJ/zD2qIXyRO7/wvSpSxkMKj2d0i/Az+OIMm4gX40GQoKdUS4ExrnNY3O+4rSiFcSpbY6ehLWcvYbcDPp5wiKJRg21cTqu3D00YWu1w45a9md41fdh0GnEsWX4qqoJXy4nTs/eE0Zb3lY0ne9CSJ0Wjaz6yiN3Qb+W8lulW6YW0nereSuGU3hZgB16CsMAk5cYt1WysXi0gQ8xh+QfYwd/6h3Ul0d1cNSshqzC4U7KOcvmsB96qEdV7Sr2Dh329qZdityh5lqakIdO7cWUVa4HBn4prkTL3yL8i5XtaEgBAQAkLgOAn4kDgx0iQiq6ZadP1wRAROAexLArPEahc4vJlj6dpIPNiTUVTuUEFhq9JKExyT43qRry6HL+OIDq5lwb7/w9FDPyIl+wFE2BZT1AUThUzLwd0Uo3do5ESn5pf0vB9hpmj8efQDFc93Wuy1FCHiaqc2suKZwKEV/0J+8nLHRraUGskq23Xcs/CnkFAsBtkFQBOwjobHsVBenEpCtRg+JnYpsJc9SyerOm2d39nKqpXEza8iLHYaClJXkQjWPid0C0aiOCC8j2pmI6FuJB9iKyVLcS1G/tyU58EPznGd8478QYkubiDfCHr0rqIDOH/AeXt+yj+I7FJ3se96DLJ+fAQ4qsPNN4vLiSd6nmOaeGopdUJACAgBIVArAid1IJ9b8oX0VFjQsrA9ucMMPHHCz2S9vUJZcAdFTFDpg133sZCVeN6Oy8l6G6c2+VIM1sHhp7g2U+txBfsRn/kG2uF96qtIiV3e8CJNRksrOeS2D2d0mzd6FZ49kbI/idh14+OpgpM16MUut+G4vDkJ30CfyKHTiLs97V7nOitZeznhBVtks8gim5v4C41X85MDdrGI7kc3MBy9gcR4RI9zyD3iL8f4/mE9SAA7W/61jRzj10S+wfpSQZEZlNjlSrIq03/6zfZl8hs1k0CXIgSaIwERvM3xqsgxCQEh0KIJRPl3wuMn/KTOgSeHuZYKWzn+SvsMYX7tcGXvJ7Ei9VMVzcEeusy1tX39/QOcMMBepneZrS06vRsNxYg0fOdUxysm8tndlr3MrV4qakeAxV4cuRbsWDwSe5dOs0/+ctmVXRsy9ix01Mb0v8ax7LZAYcTYl5bt+vZiJItwKNr1u8qpKQvd7hNeI0vtGhp3KoVBexKHV99JepND3NVQSHzGjnoII66Ix/CZe9Bjwqtkha6M+MHJK/qe7urSQMdDN2oDzv3DLUza0az1KDa6i1z9Ewl2oQggId3cCkc+4MliDfnS4tc2t3OV46magLg0VM1GtggBISAEjptA56A+SvTO330b0srcravl5Dv7dcLzqDg2sYmtvtWVvPIMx+a1md86lvULFkTQ9LXKR9/6bfVNKsGxepOKdqsuuwUPbjP+gRxebOfiEXTebLG36JDytDK9ACSf1t0LYClNRZ9Jr6I074CurfMix8xlv9thl++iKA2HyR0hi5I+DFQWV578lbHnXSU82w/8F/zDemP3NyepDhxCV7kUVP154YxtHYbOcR7Uwxr76A6buY8E9VqalLaNkk50IVeIUykyRKRb6+VZ32Kg/nSPtWAXHVVIKEeTYA+JOcFtX29X+Pv7e/sQZPxmQEAEbzO4CHIIQkAItD4CeeYMPL5lerUntjrd1cLmuTn7/PYOHUk+wRYs2PtvbMz62WPDMpyFFNulCMUMEqTaY296zEzRF8aTC8XxlhLy83x2+8XILUtFOUWQKKWMcW+etIuSYngW18c7TnPcL+vAJ2T0dM5cZj9Od/XH/rI5CT+haMjNJGKzKWJDmJpE5npeKjUxCcTDa+5CT0oeoS9hnSeCX1rhCWLsv8vhyhyFbj44iURQ1FBKW12GiG5nq8xunBWNJyr1Hv8kgrtOczSvboHFcXjsJPX6f/auAzCKcuueLUk2m94bpEEg9CZNOgj6wIIFFPUhdlGfPnvF/mNXeHbsDQs2QEQEQZr0XkMN6b3XTXb3v/fbzGZnSwqEFJyry85887W5s9k9c+d85zZUL58oMhv9zRhXaAG4FmKOCic79cAw/WCR4IKl2tqjsRQXZydrSWM/c7YzxTqOBxTA23GulTJTxQOKBzqQB35NfZtmax8FdDwBFT3WdhXdZeUEDT169nMPxS3d3sCr+68F6/saiRLhzO7rfhFmHxqIVNV36IRrKCapQTfffrirx/unlTY4tfwQPjryACk+7LMbToU39t+AJ/v/bFd+7u2aasqJQmAb2bWco1fIYFQU7LVZFGYpNxG/lsGuWBzGkVhXRjcv9lxgZ1WZ6mCbPlaqw5nZukz4QtpFcHfKFEYJJAICQ6B19xLartaDp7lhqq2kj7AKJZSFb28hUWLo/qbQzYg+pSoY6aO93c+Eh4a+izDPuNMcoXWacWIFfrWkKQoILenN1ulLAbyt42dlFMUDigf+YR6wqCg4RgHt3UDsTXpQLv8x9lR746G+Cykl8VZ4aymFKEVn+Qf2iEhR7Bzsemq8sSDp33go7gb8K24ejCRRFtqMJ7mbcn7G3oK/UFCdgXB9PMJ18fgu+QX76dbtmwW9oaymECxxdi6bjqgGDnq2dMK84Is1cO3NbGT1Ax0dDxL82xOr5bxc2/pNWXzG0V7PgF5C41dEhrkDig5zCmBbY5oER4I1pK97plZLGsIn1txENIcDgi98MrIbpdu13LylUO6KFE/L53pM2Ix2D3bP1BdK+3PHAwrgPXeupXImigcUD7QjDwwNuYwWo33tMCNWcyLsarUwz3jkVCWjfsGaCnMGLCX93K6I95HzId3UOtQ6AVncWSXRDNhWnfoCPfyHYZDvxWK/Kf+8dfBW7M7/k4C3BcAllWxutBlnaXOlRNFo4w5SoST9LySvu4MuGC32IipHvVHUk44JDdr6QrGl1noTzcCibawP6m13VL4bGN80mknCpG+RuvkxlGT8BQajrL17bMWVCOt9DyL63y/v9Az3ODPb/kX8ubMAXO4uOvMoetJ9zUGbJ/j8ZMLfI/wMR1OaKx5oPQ808Lyl9SahjKR4QPGA4oFzzQM9/IfjuvjnZKdlNvNXLuvweiHcswue6b8M18Q/Sbq6OoIPGvhog/BEv58E2JU1rNsZFTadAm2NhW3NWJO20Flzp2WHiv7GjvzfrWDXaSUnhYODLz6nObyceOHEmlkU4aSbABnYZWcQg9WJDjIfMVG6YE+/BN4UqX017v5i2+EfAtHeYUMdil0VRA1+RmRz4wgy6lQacg59gLwjX7lqclrlnDyCaRR8jpJp6C7tvBKbuzRx1IR+geOkKsq74oF27wElwtvuL5EyQcUDigc6qgcmRt2IwSGTsTztG2wrOAStJhLXx92LvnVpQnfm/YF3Dt9h5eRWmcqwNnMhEnzPE6fMlIGV6Z+Ao6kDgiZiRvzTRDnIpEVrFskzV37REb2hqZZdmQyWTuMxmmJ6ja/Q7L0q7pGmVO+wdcrzd0Ht7kfYsrjRc1BRIhAzp/ElXm6XsW+T2kE4qqoImFICEquygn0vBKJrKrLtS13ul2VvEUDUNnsaL5LLPfwpqSNc77JdUw9IUWnmIKuIMmFveiGvVwYvotgY6VxZTs/+CYR9G2Vf8UB78oACeNvT1VDmonhA8cA55wF/9zACqv+ll+OpvXXoFo4VWg8w75cVGC4qv00sVHtw63ABLpju8Hv6AszsMhfT4x5vFPDe2vtV1LETrH272gjRRRPAafhhn5fGjygXtXQOczA24jpXXZ1T5VoCuyqRYKHx0+IosNYzGHFjPkRw52FikVn+8Z+Qtu0JpzxfqcesffMEcyCs12ypqIF3M10nx59sl4DaRU+cGKKWgLabVyTcPENErfxji0jl4WmKTjMVgz6PBOBtjRNXxPaYjWfjxoPVRyKIbhOi62xbRdlWPNDuPeD419Pup6xMUPGA4gHFA+eGB7SUWMA+RXC1sQLzD94sQBPLidkC4kXJc8W+ngBohdF55DFMH4tQfTSKi50ft/dcr4CRlHZ4LLbl/So7xDQLP7dgzE58F0WGbHTy6v6PWqDkHTYMHn7daLHYDplfnO+YYCRubdbe+QLwFmdsROqmxrm1HKHlNiGJN1GiBznItB/HK3QwFckXN7IMmR9Jkp3aeD/Jn5UiIPZSBATMtG9q3c89/Dkyd79Ka94odXB1AaKHvwp3n1ikbnmMcK6NbBdpRLMJhQiK+AZ1vRrhfe6x9nOubtSQugYvJHQnfeKWsKysLKxZswYzZsxoie7abR8bN26Er68v+vTpgy+//BJXXHEFvLy82t18FcDb7i6JMiHFA4oH/ike6OY7BAeLNhKIrQcyvJ1bleLUBWq4o6iwRujxOq1AhdkVyfg9+RMMD6hfEFVJvFKO3HlqbVYd2XRwV4/38HfOROwrWIu8qlQE6SJJ93cQLoicRVFFOXfTptk5v8lKC001jvKW525DVUkyDv12ldNmgvpgx/0VHGGnteWFGq0eiRevxIGfhhBTwktQKALiLkfe4Y8JpLFsmgklaSthKj+KhFEvCOCWuvVJsI4vR4H9Ol+EouTFlk7rsG3Kpoeg80+Ug11pWNZ+Hv+5iAS768Ol0nPynRcBnlh3P/JOLCa/migqX4UhNxwn1YumU4OcOebo0aN4/PHHz3nAu3DhQnTp0kUA3gceeABjx45VAK+zD4RSpnhA8YDigX+qB+6k6Oldm/vQIjYPogxUN+qGcmMBTuzTwJ0WRVV573JZ/8vDT2H48CtFFrcPku6htMJ/UvKJKmjokfiFkbdiejxF9Ozs/NArwC/FbDxAnFyXxo/97cArIUvUVOYRTYFuEliOw85EAgvbNsSVVROQbSy6K3XDmdH6XXsMhopMaAj0Ji2bTEPKpdEy9n+CiB4zkLT8JlQVHqamlnkUJS+VurF5V5FmMM3XianUNDc3b4p2nttgl0/96J+3oyBlBQFd6W9QhQPLLkefqX+02A0fJ74oLy+Ht7c3GAgzQOT0xIWFhaiurkZ4uMXPZWVlopzLcnNzRT2e48mTJxEREQGdTse74HoajQaenqQTR1ZSUgLOKMdl0jhHjhxBVFSUiL6KSvQPazqfOHFCpFnu3Nk1LcVoNOL48eNifO5TMuamHzt2DNHR0bJ+pePt+b1h4lZ7nrkyN8UDigcUD3RwD3i5+ePjkScxgnR2XakvWFO3mhlEqZAU9hoqdMcaPPMqY7k4/vyey7Azb4XItMbAhxNW/Jb+Lm7dmICHto3Adyf+r8F+/ukHXS0G63HpX/DrdAG5R/4Taqotp4hpN6goWYgzixuzQBRzBjZWb9BR2uBeV25xVtVlmYoWwnl4dxY6wCIxhF1NpiuUZO+UgV1LlfqnCNYmBLg9KVubMzMbK1FdctLZIWsZP/6vLEoS6ZGthR1wozBlpQ3Y5RMw0zmloVLcMLTMCW3btk1EPs8//3zMmjULcXFxeOWVVzB69GjxmjNnjhiI36+66iqMHz8eEyZMwDXXXCP2Z86cia5du4LpA2yPPPII3nuvPksf9/nTTz+Bxxk3bhwGDhyIO++8E7GxsVi1apVow+B62LBhmDZtGsaMGYOpU6cKACwO2vyzYcMGAcDvuusu9OrVC8uXLxdHV69eLfr773//i5iYGHzyySc2rdr/pvyvtf3PV5mh4gHFA4oHzikPcNT1/JAr6YG0HJAwh3Y0CfuPCL0KkaVTKGpIkTp61WqL6VXKv8lEcOBHrnLKAQNkH7dASv9bjtSyg07lxmoo2su0iVWZn+LtQ7efU/5syZPxjRqHyEFPiS41tIhN6xmKbv9aCg/fWMSMeIP2QwjcuhN49aPkED3Re/oeuHn4IqTbNQ6gV0+Z2XzCh6HvjMOIpXTCcWPeR/cpv1GkVi+itM6yuTV0Lsnr/wMjpXy2N87ypnUn6kojCxFFO+Ltxox4E/6xU+27EfunNt7rUkmipjIHh5dOxLE/puHQ4jEUUb6UzqOBiLjTEdpHoVrryJ82iUi8/G/rTGd78OBB/PDDD9i8eTO6desmgOi+ffvAQPLDDz+0dl9aWiqAK4PX77//HhdeeCHWr18Ppgt8++231nquNnbt2gWmGTB/+NFHH7X2zX0NGDAAO3fuFFHmmpoabNq0yaEbBsRffPEFVq5cKTi533zzjagzd+5cfP3112Lehw8fxssvv+wUMDt02E4KFA5vO7kQyjQUDygeOLc9wE+4K4rpMbGGuLQ+FnCbTCl7OUVvSU0unbzlx5UlwjihQ6S+G27q9opwyle7DiDDZ5ncQVTdAObmWsxD7UWUBQ30lG1r3uhNKC+tIAjdMADhBXMHCtfjZOkexPn0k7qiDG9bxGI6lkdbmfEpVqR/CF5M18t/FP7T80OanzxWoio1QZNcA7M3lfeydtNhN/ixb2XhQbqpMCKk+w0IjL8CRpInc/MME1nU+MSYhtCborPVpckC6Hn4xtHjb4tf4s5/iRJElKPw1K+iHieYiBz0pPAHpwT2iRghtjlCemrjfShOXUHJKqqpfCTix31q7UdUcvJP2rZnqM0fND/5TRJXjT//eexfPtNJK7UA7LxYjaPE7t4xBLzfgdbDH7Ej52FP+mqYa0pk7ZjLWpi8BKE9b5WVm4wGHPhxiKysMn8fKT08hy6jXpKVd4SdkG4zkLmvPlrKf4ucUlofmNii0+doq0QjiI+PR48ePUT/kZGRyM7OBtMe2EaOHCmoFCEhIfDx8RGAl8u53rp163izQeNxODLLlpCQIIAvby9atIjfRISZ33NycgQA56izZKmpqWDAzdFltsGDBwvwy/QKjvwyEOaFaWwcMWbw3lFMAbwd5Uop81Q8oHigw3qgplqF9d8FoqxQixqDCvqgcmzvPxHHy7fZnBMhYjLWw7054XWMCp8u9o30G1gW/RvAv4UWTCzKbf9xV3viWtLo1RGvc2K3a6EyabC/dBNVV1Eg2NKvbX3bbQbX5bVFoqiW+KAv7Z2OzMpjtG+m8mIZv3h/4Vq8f/hu3NnjXWsXmuMGeL9cKPbVZbTkrmc5jM+FWo93tA2mCRxdeQ0MpacoekVJn0n9oPe03fDwiXF6Kh4+sdZyjq7uX3UnynJ2iKhtLFEY/DtPsh6vLDxEGNpANIKeIgKc9NsUSk5xhI5brlFZ1ibKqPYoqSdYbnSsDe02ilN/p/4lvqnlIEeaQ3vdCf/IYSLFsNEgB68c8U28eAUBXNIWZv3eQx+Thu9n8DFFIsSQCP+KYBS62bWhrvOPf+8AeCsL9lEKY19ShrCtbyZw/AvQAQFvzNCnabHhSbpB+Z1uaDzhHTIAiRd+Zef1M99l/q6tSXxc+4WhrurZtuU2EkDmcub0ShYYGChtCuBs3aENjhaPGjXKWhQQEGDd5g0G2MzZ5Zs+yTgyHRwcLHi/t9xyi+AY87E77rhD0Cykeu39XQG87f0KKfNTPKB4oMN7YMl8XpDCPyAWxFqep4fXXoqadbEFvJbT1NLqeBb2ZyuniPDahYEoDvIHwrlt/Y+QqFD3D3NzWbLMSACtCOmY3vVRsUhNr/Wzglnb+rbbDHZjvC08Tub8ppUfFlxfqY7tYroaWli3t3A1cipPIdQzBqoCI3znFEhVxbvqcBW03xejeqplMY3sYAfY2f/jUNKjJQ60zYK1YytnEP1gWYORV4567v9hIJ1h/XVKXnsbpf+9F2F97sKJ1TegsuAAXUG6KSAQ3eWC70QmNluXsJpCwfEfCMwa0GnI/1F0WIc00sctOrkEJioL7XErpRJ+gECZHDhxH2biE7vpglFTVQg1cYgdYvt0PvsX9UPipX/i8JIJJE2mo1OsQj61LSwLQ0hxJxSHa2DSyFvWL+Sqn6nQA7aL8vNRjoILWgMBpo5kDB4TJ31O8y8TNzl8U9DeLSgoCAcOHBDTZPkzpinceOONDU776quvxooVK8AcXF4wd+WVV+Lyyy9H9+7dre38KSkOR3yXLl0quMNMt3jiiScE9WHo0KE4deoUrr/+emRkZIgoMNMmOoopgLejXCllnooHFA+cFQ+o1eoWXW3MPyT8Mpksj5uLclXE6zSDo7ySqQicROVNw94ut0lF1ncGr3paea3X+eLHV7xgUhmgc08AIpyDXW7IALmsLkr745E3RHRmSsxsqDh46MQ48utJoKmaAM/TQ39BVFCsqJVRcUQGdp00FaDPQ+8GXx9fqLYRwNFTbxX1c1PRabuvr4BmZpiz5me9TKul7GYUnWJd0OaasYZoIHZgl/uoqUgnvnQhPIm24Mqyk74nVQO6XvQo3Nay9/8PhSd/IPWGHBH1lY4dX3W1tGn3bqJI6VJq8ws9Uu+FCgLJkmXvf4uEIbLQqe8tOPH3HJv+iEpBvOzMPa8ijUCn1kMetbO0t1yjpF8nEyanCB5de8lKPfMQWBJGayLrryMf46ixd1AitMZs6APoM1hnPj4jkaoPQaXB8mRAKud++Xy7DH8SfB2aY9LfS3PatHTdM5Uha+n5NNTfDTfcgEmTJqFnz55C+WH48OENVRfHpk+fjh9//BFMeeCILdMenGkEP/nkk2Bw/MILLwhViPnz54v2zz77rKjPi+34et13331WdYlGB28HFVT0xSD/hLeDSbXmFJiDIlJANjIoP3rg0D/zbNriD5PH5rm2tvGdL8ulNNVPLT0//tFiiRWWSGlt4y8EJvU3VcC/JefHnzf2fWVlZUt226S++JEWj8+crdYyvV4PP7+2iarw9WUJoJYyPhf+3PCLrbRAg1WfB1DkzcLvlMZhILtsmF7alb1H6hPQu/pWqHZOw7ru41CrKRUvs9ogq+dqx50WvL034qBYtPbMrosdqvm6BWFa3KPoGzgOgaS5K9mdG/ugtIZjfg3b52NSBc9Yu7kSug8oomgDeLmlqbMbyl5rG1oDR6j4O5plmppr/Kh/9ze9ZWBQ6qPzkOcpwjpL2nV4zzv6HSVwmEN0gZb6m+XPiyNHlwdWaWihG3GqGWAC9KJMfE02ztZW9wTB2sakQlxOb2hr3XG00w5RzGCXKR0a4oRzUo2A2EsQP6aeypKxZx4lsXjd2oW04RM2BH2nLmvS76rUht/d3d3B1+5sGD/ut3383xJj8PdzW31n2c6fv6eZ69sc478N9rdEqXDVNj8/HxxJtrfTGdO+j7bYl38Dt8UMlDEVDygeUDzQxh7gG6qWejHYsu1L70ei/yE1FC2rjy0YVZVIC/7a5VlnVBzFlqJFWNEvDpW6ZNS45UOA3fouXLblAxzxrSXyr07jCy+tI4jgyO6Q4Msok1qYbK5XxT7iUh6N++XI8HMD/qDIIo1BPqse4Aazr5rmxkctZib8ZbgrSNavrT/O9jbHcPh1OuOYoaWsZzcSjtRJp2N9T906B6c2z3HZr3f4aAK79VFTa8PT3mCwW/9UwLYbAXb5KNEKOg99Xujl2h7nbU4ooXWmocsxLvsUxRTZda/xRGhRDGKyeomuRMpiArwMdtkKk39F5v4F1vPXeoQQtcJLHLP+Q/PhlMWn4/9/eOzN6sLmbjQX7HL/HEhqDOxyPWdgl8tPZ0xu19Zm8zXV1lNRxlc8oHhA8cC56YGR0wrgoTcRQCGmpcaEooB1yPb/jSJqrqPaxd7E77UHuM7xj53TiEJBSRE8NJ5CnowzR9lbtalcKDrYl48Jn4FZCS/BU+Njf0js6zUUPSXKhdW0KpS8FAxTJNEI3CkeGahG7aNhMCU4yjxZ27TzjcgBjyC4279plo7OLji+CAVEN+DsZWU5dH1sjJNCxF/wlU2JZZMjpW7e0XYg2rFvh4bEw+W2DRkDc9aKVYlIr11NAp+R/R+W9yHxbjnCWwd6VSY19NUkpUYcXjX9lxp6mDpSy9uJrs3EL15kHSSQ0g2z2oPVTzwH+qzFj3SM+lobKRuKB9rQAwrgbUPnK0MrHlA88M/wgIZwwZQ7czDlbuJC+poQXnEBBp78BP/alg+vynpupMwbjImagItkbcSOGa+MWCO2vGjR2qyEl61VOELLi+Ke7LcYWhdgakTYVXjv/IPo7NXT2k7aKDcWUtrhTtKu5d2dQO8rwSj6LBzFb4fCNNgu6iev3SH2ogY9QYkdgh3myo/4Uzbej+T1dwv92T0LEwQ3V6roGz6CsnOtELucpYzpAF4hg9Dz0jWkdHCbAIgcFfWOGElRWec3FdxYRTcrWo9AdJ30rdS1i3czKX70pn7d5McJ2PpGjhVyap2GvCDALYNjv06T0PeaA5St7ShFsm+CX/QUdNZPRdf0gcjzzcTmxKUwu2lpfHd6Od60qLX1kW9+pM/qFaE9b4c+eCCpUVxESTS2knqDc5qOfILKnuKB1veAwuFVOLwNfur4S03h8BY36KOzcVDh8J4Nrzrvkzm8FRXEh2wh8/LygsFgsHJ4bbv99Z1QSgtLFADOmlZnlW7pWDMgEUZNS3E/STszbDLu6vaBNARY7/evzIUi8js2/FpEeXWzHnO1kVOZjIe3jxIA2URAjzV9/9NjAQYF/8tVE1HOHGZBd2hBXnSDA9odZEkmppUUFdktqLKr19jusVXXgWXCXPFopfYMDHtdsUXo2XIZp3qtqS5GYfoWlOfvRXXRUU6QR1JXg0n7ltK8EhBl/d3Dy/6F6uLjTvon2brgAeg6caFQaShK+QPJ6wgsE2C1XWgmjd93RhItlCsjhYhBUpH1Pbz/QwjvfZd139lGaeYGlGRuJAWJvWKxGyfQCO42k5JKTCDeea6I2lraacAawT4Rwx0Btk3HUnrb5v5N8XeevUSWTbdntHkuc3jPyDH/sMb8PEIxxQOKBxQPKB5oBQ8YKuVgl4fUmIh6UNEbRT7bGp1BF+8BOF62i+oxYLbnO9Q3L6zKqt+hrViSHZuV8KKsrLGdUM9YvDOMskBlfolaWhR1XvBkivpahPKdtWU5M05k0V13Hs2u4z885AxkIrkCP/q3X+Rl4wCzsQb5x75DWK/braVaiuwWp61GXtLnVGahlBSR6gKbzr87SZwtR3fK2MYgVahCiCPSP0x78RBgl0v8o2kl/uV/oyJvF8rzdpN+7ocCcHoG9ELcuI/q6jpGY7ltzv53KMp7JdydcXnpePrOl5B36COxOI2jxMzb7Tz0RYpMeyPxkpU4tPQC4vDSjUOdD5LXzxbz7XHZWpe6xDxuezNWjODgTUtaS/fXknNT+nLugX884OUPbVPkU1i6iI0FmaVt5y49O6VNnWdLjy79UfN5N8VPLT0++5rHlubR0v031B+P2VZ+5/PmV1v5vK3Ou6HrcS4c07rJ5cn4nNyNgZQquGFFAcqLRXknDHii/8+kolCA31LfpQQVlcTDdcOfmZ85uGZg6ESHstMp8HLzxyXR/7E2TS8/IlIWd/JKFBxhPlBLIOm1/f9GSjnJZ9GCqApjCd4ffQA6NF8WzDpQO9gwUxIOVkNgwGomFQNKQ0bbzm40WFe3nIDoJyIRQ1D0WLGYKy/pU6dnUVWURNJdbyO8zz2IPv91JG+4l4aQq4SUZW/CkeVTkXDhD4IG4U4LwfjlHzOFdHgfpO8l+j0iEGr7vShUFeha2BpHn2sr85wC3oq8Pcg9+L61ulikRn2mbZtD6YbnCTpGb6IolGRQJr61twgwLOpQC04l3HvaLhfyZ9Yu28UGL4ZrygKtdjFZZRJn1QP/eMDLoIIfQTVmDLrYuG5brCbl8Zsyz8bO43SPs4QJ+6q1jQEff1m1hc8l0NcWfpdAvu0PWmv5nn3O47bmebfF9W0tf9qO03dcCXb+4WcBUHUHqtwyUeUmj8jatuFoaYLfYNzX+zMhBebnHoIZXZ62ViE9AhGFZfDrRrzcQF0EZnR/oklyeidKd2FvwRpSc/DGhMgbBOXB2rHdxgdJ92BX/h80By0qKAPbS4PWIlwfj6d3TUZW5QmZfu+z2y/FCwNXESDvmD8xDHYP/mxJ/2vnBie7ZgFgLWDYiKy985zUkRfl0+IvBrxu+ggC0s4lFyvyd2M/pe/laC8nq1CT1Jy7bzy9axCccC3Kc3cQ5SAPPsQbZskwD78uqKJMbrbGCgvu3naca6rA55e6rf4zZG1DkewKomHYWjZFie2TTzAPuThtFYK6TLOt2i63+buM5R3PhiyZfUa0dukAZVJWD3TMbyPr9M98g7lmrPPamDHoYgDAXKC20OFlEML5rVvb+MuC/6j5C6MpesUtPT8ev610eJmLxlqqbeF3/rxJX9Qt7dPG+mMdXr65ac3zZt7nP8Fi+1YSvxXYvZLVGSyPWBnwmlW0at6FUfwQUaTLy5QBH7dAhOiiZTVndp2LEaHTcLJsD3zdgjG5xyyn/GFZI9pZnfkFvjz2JH3ONAKYfnvyebw9bC+83RyTFnx5bA625y5HjbnK2s0zuyfj9cGbkVlxTPB7+YCKyKpDT9JiqdpQ5AYeQ3hcorV+R9owlKcLqS/mxsqMAbytSoX1oKMShvWQkw3OVFZTmYujv1/m5Gh9kbG6QJYxzVCWLA6WpK0UEV6mWhQc+xbFqSsRR2mMD/0ySnB9GUQzqI0b95mVWyz1yjeXexYyNcXZZ46eeNJiOVujr2CnxgsgO4pJEnUtOV/+flasY3ngHw94O9blUmareEDxQEf3QJcBFG0yqLF/reWRv29FP8Rl/gfHOr3k8tQ2ZP+ATbmLKU1wISZE3IB/d31BVreL7wDwq6nGqYG/OPaEqG4m0GSiF5GHsCDpv7i/9+cO3ezIl4NdrsAR5cPFm+BOigJVxjIBdl//4QuElUQRFFLB609vlD5QhdpBOof+2nuBWkuJHWhhmYMR2GUFBfMZJpcwlJ6kCOlKUjSgpA41DdNZHOYgFdA1k6w47Q8EkUxYn+l7UZFBSgsEdt2DRkFHUV97s/CK69vKj5sRN/YjWVFw91koy94sK2PesW/nlqHNyDpWdhQPnEUPtP4z6rN4MkrXigcUDyge6AgeSOn0Fnb0uArpQYuQGvIZMoK/E9MeHXYN/N3DxDbLh0lWZSoTYJf3/8z8HOuzvheH8qvS8cyuKbh1Q1fcuD4GR0u2S01k74XVmViSMh8/Jb+GDIrIplUcdkhIwSoMyWXyx9lSJzqNo9QYc3e1pPc7udNsokLo8PCKl9C5IA7eBh94GbxFU++3iqDOdf7IXuq7Pb67NxY6dAAAQABJREFUeYaSUsH1AtxK82M+rG/UBGn3jN8zdr0iqApn3BF1wOoNJ9feTvzbp9G5/52IPe9+p2A3lxbRZe5+1fWQFO0/ufZW2XH/6IsQMeAxUabSehJFojN6kMya1p2fUji3mqp8AvJ20XHnVZVSxQOt5gElwttqrlYGUjygeEDxgMUDHLHN8N9HL8vKfS51V3sKFYSbur1KEmJf4bNjFpDhzGdrs78RqgkPbBsmO/zS3ml4PWAtYkiVQbJ9hWvx5v4biHZAiS/ovyWp8zEj7hmRjU2qI71zwgpbM1Ak89Ojj6CwOsu2mHpR06K5CvQJGIt+geOFOkNidl+4m+TtzZSYQnPUAFNI4+skZAO0g52oQU+SFm8oKTB8LRQTQhJvgXfYUFqwNbp5s2O9Y+bp2nF1TQZSP2BjDV2KyJ6pmWrLREKMbQuHY8i1m2TdGQ2lSPp9Kgwlx2XlDjs0x/LcnUSRWAW/zheIw7XEAy44/h0tYqOkI5R6uaYimzK4WW7K7NvXkhzb0RWzKBkGSaXVViCQOL7RwxsA2PYdKPuKB86iBxTAexadq3SteEDxgOIBZx6wjZiyuBizAVO1Q7G+JAUXRJqJW/uUs2bWMg+1HuuyvqE0wDoZr5ZTCi88PFfo7K5N/U5o6OZUJ1vb8eIntqUEevsGjMPWvKV1xxjCqkln90PkVqUgwD2CordumL2pFy0YZahcH6Ul7RD09B+Bu3q8RwvYLA8Jr4x9CD5BeaDVbHX9Wd5UVTSivuM+SAzrdRvJjd0mO6f4CV/hxJ/Xo0bjBi1JkrFPa+gCupNr+YbCIu9FAJZ4urztQ7q7ZZlrBQCUdSR2SP2GaA0mipaz0oMkYeZYr2klrKJQVZaO7COL4B46STTi67fv+/oboEZ7ItCbvuMFAXi57f5F9lQZFY4suxgxI+dT0ov6fs0mI9XtR93zp9nyOePMbO6UZS68T73SR6PjKxUUD5wlDyiA9yw5VulW8YDiAcUDrjxQzrxNMym/qIwCHnC9wJrtWFY1HLu234kgNBzxy6w4iqTizUIf136M3TmrsdW0zL5Ytm8iIHNZ9L1I9D8fW3OXEL3BT4Dc/9t7BYFkN1QaS3Fj15cFoK2xW6QVro/Dg32+kvXHOxU3+8Ln2QLi8loO8emZ/CnVcH951NehYQcr8I0YKTKMlRcdxotJlK2ssgp6ko3L9nTDvYOXIIw4vhl730Pu0e8oqFtNygkHSQt3GvKOfk3XXH5DALqRcPOKokQT/WjxGdW3kxUTriGagVbfCW66AHiShm/BiZ+pH7n8mK0LGXhypFUixBSd+pUi1J4UnW16YhNeUMcJKdxICo0VGeRawWZUl54g2bRL0GnIXKJ+XCuGZ1UJzh5nqpEvrs478pUCeG0vkLLdZh7ouLfebeYyZWDFA4oHFA+cvgeqK1XILD8pwK5tLyqKrOmN6cjU9kY1XPEjOXoGFBgynYJdPmYgfd7GrMJYTGoMgRgfcT0e7fs9BgVdhHVEk6gxVQmwy+0/P/a4iBDb91VR63yRlbGbu0gxbOrsBlOoFtWT9CiZF2Lf/JzY13r4wy9sGF4afRBXDPsGE4YswJzzNyHMMw5ZBz+pA7usaGECKz5UFR9BUMJ1tG/3k0s3F54BiShJ+9M52KUWrLnbY8pSSlTxC9EDXiZubnyDPmQqgV/k+dY6phrKIkiguTmmIukzIy1MUxMdgxfAubL0Hc/TuR2zHnZIcUxH7CXNrJXb4Ua1sQKVtXLAfibT3L9/PxYtWuTQxcKFC1FQUOBQfqYFGzduxL59+067G1ZjYsvJycFPP/102v2014Z2f33tdZrKvBQPKB5QPHBueGDNl8HQ1jomZdAQzDURABIgSRPk4mTrwqd2R5n/66H2QhfvQXZHnO9eEfMQWM9XsuXpHxBQrpcc43IPigral7mpPNA7YIzUzOHdFEVyW291RtV7Uaj8N52jxgLQHSqeQwXd/YaST0bDU2tZqJdNgFeWApiiupUU5Q2hRXC+nSy8WD59FfGlOV1wWO+7KYrqWpaPAWd16SnhsdLMjQQwnfFwiZJC4/PCum7j5sEnpJ5q4EMRaRMtamuOmegJhLs+Csnr7yLAax+Vru9JTZQNPjc2fVA/Bwk0Pr/ALlfWN2inWyzV9unhx3DLmu64eU0CblwdDwa/Z2qrVq3C9OnTsXTpUllXjzzyCNLT02VlLbHDQHrlypWn1dXq1asxe/Zs0ZYB788/05OEc8wUwHuOXVDldBQPKB5o3x4wVKmRmPJ/UNss8DITsaFGRXrXmk4Eej2hbuCRtf3ZcXKHnn4jMDJsGqUd3mF/2GF/Sqc7cWn0PaKc1RtOle1Havkhh3rMTb2ks4V7qSfKg6fGB90I3N2c8JpDXaUAqCo6iiMrriSAWM93lvxiNBTj8K8TEdz9Bsqu9iaBwOkU8Z0BN88gJC2bjOqS+iip1Mb2nekDRSm/I337cxQylUdc1aSWENrzNkSdN4eiwL8iqveNtk1F4om4Me/Lypqyc2T5xZSEYg9Vda0xzJJqWl2w6E6l1iLhIgtIYgoFg++grtcgatCcpgzXpnU+OPhf/Jn2pXg6wjz4KopuP7/9ihZJeBQdHY3bb78dhYWFLs+RNe45GlxS4vj0hMFnamqqrG1+fr6Y27Fjx0RuANlBmx3WsOeIb0ZGhk0phE63bTnnFjh48KBIVsNzSExMxP/+9z9Zm5SUlCYls5E1amc7jXJ4q6ursWHDBpFsYfjw4SIJge05bN68WSQlGDdunG2xsq14QPGA4gHFA048oPMyIjp3FnFdtdgXfzcMmhoUa3shTTeZn//Ct5Ykw0xpTlo6L6o0BWBX4V90kJINmCkjoso5v3Nc+L8xKnw64n36i44Wn3oTv6W9LxZdOeu5yliOizrdRrSHmThRtpsArxe6+w2TVS0x5GF/4Tpa4OaOvoHjsCrjM5QZ89HVbyDOC7hYVvdc3jmauxrlK24SS7Uaimmf+PM68KK3ToOfxd5vezTLJambH4VGF+jYhvjYBlJO4Kxr1SUnUJL8LXF4C6ELHY3AuKmivtloEBFgVnJoSWNQ6xVS/1SBpcqG3JQBkyEfVdVGh4hvS47dkn1tzPxR0HmkPnmRZmbFcaSXH0Un725S8Wm9MzbixBf33HMPvvzyS4c+OLJ67bXXonfv3tixYwdef/113HTTTQKk3njjjcjLy0Nubi769OmDJUuWgLNw9uvXDz179hT9Hjp0CI8//jjuvvtuWd8LFizAiy++KNpt2bIFU6dOxQcffIC0tDRMmDABXbp0EUC6V69emD9/Prg+j/Xmm2/iwgsvFNHeXbt2CRDO+xUVFWLs0aNHY968ebKxOspOg4A3Ozsb48ePF8ifTygyMhKffPKJcIZ0ghxCz8rKggJ4JY8o74oHFA8oHnDtgcFTCrH6i1B0zruWXtejWlOFP8/7Cmr9PnRzq0VO2Q+81l8AUbHqn3iftioJUs9eWn9UmnxJCixFKiK+J632p9VialoMZ2ucVKK0Jt8Kdrfl/oafU96wreKwHePVl1IOe4lX/8AJDsdPlR3A87svFRq8RnrsbjBXkrSazkKDoIBUn4Dv8EBvxx94h446eMHmnF+Que5exNB5qK1LEF2fVPK62YgZ8SYt8CJ1BrukEx6+XQWorcjZ6tgBcXl9oyYir+QD2TFeJFaUspwCsdWychz9CRm7XqbFbiFEgzhK91KNc7vlHTS+x9HsI79dgu5TlhNFQyMacAYyd5ItoyTFjXfQTmqwjrQ9fYeTsZjoBrQljKOlDCwXL16Myy67TNbl3Llz8fXXXwsQypiLASUD3QcffBAJCQlYsWKFSIvMIPXPP//EpEkW9Y2RI0fiqaeeEnxbjiJffvnl1n6ZosHUBq4fHx+P5ORkAXDfe+89/PbbbwIwf//99zAYDAIse3l5ifGWL1+Op59+GhzIlOzJJ58U4Prjjz8WkeFLLrlEgHHGgx3NGqQ0PPvssyKdLoe+k5KSxB0Cnyw7RTHFA4oHFA8oHmieBzhZw/zMC7FhaE9kB/+CbP/l6DPhFD4eMwVf93oU98Q9iZfO200au1NI+mskrox5lMCrBUhIIzGPdmDQhSQL9j50utmkp2vhjkrHGezS753MGDCfKN1lLVuV8al129XGsNBLXR2ihT1leHrXRWLhXKWxRIBdrmwLGo4Ub8XO/BUu+zgXDpTXFuP9pP9AT0/9HcGu859XVjwwkooCL0ZzMAJYXcd/7pTTy2mGw3vPhn+MJXLOurhqN772FFO2B7t1HddWZKKyYO9ZAbtiCAKFNZXZKMlY63AqHalgVMRVdtNVkcJeKTp7J9qVn95uQECAiK7ecccdssVqHLnlJ+hffPEFZs2aBeb2MvWBASe/rrrKMi+tVosrr7wS33zzjXUCEsANDQ3FgAEDsH37dusxvun48MMPsWbNGhFZnjlzpnhKzwCXATNHfPv27QvGeAyuvb3l3yHWjmhj69atuPRSy3eBm5sbfv/9dxH8tK3TUbYbjPBu2rQJDz30kAi18wm988474o6DLwLfPZx/fv1K0I5ywso8FQ8oHlA80FYeeHb3xSLTmVFdg60J08U0DleF4JKcv/HCUZ14JF5cG4R7uzyFA1U7cajKhPOinsS2tKeIUuBL9c3opO8u9HILDVkoJ/qAvZnN0kN1Oer11PpYq9onmJAOMJhm/V1OXPGvTrdLxQ7vUqa28tq65AkONYgnSDzk7MpkJ0fOnaLymkJ4awNw3LMIwcQk0cgivC64r0RB8A4/XySzsI0f8gKvEOLiJhFf11TrGI2NHv66oAjEjnob1f0fBvOCC0/+jNzDn7SxQ4nt3cxFcW08YYfhb0ici5SyQzhYuFFw1aO8EvDEIHrSQsCxpYyDhRMnTsR//mPhxUv9uru745ZbbgGDSTYGxV27dkVUVJSIwEr1WEGhttb5AsKysjLo9fULH5l+MHDgQPCYF110ER5++GF07txZ8H5jY2Nx/PhxrF27Fj/++COYqrp+/XppGIf34OBgEdmVDpw6dQpBQUENgmSpbnt7bxDwxsXFobS0VDbn//73v8jMzBSInwGxYooHFA8oHlA80DQPZFWcoIisfNFRqTEGDx7S1XegzsdbRDlQ8WIh+sHlZUNTIt/AJC+z0Mtlvuyiky9ideaXqDRPoUfHA+ALyw8W0xk44meCjtJD1PM1OUr8YO+vxBj8uDNSn4D9RetoXw6Kr+3yLIJ1USKDmqjs4h+mU9Q2IFfFzfhxcKBHhIsezo1iH7dgcSJb/czoXqEiLWUzDISROHYbPfBpVOTuQHGKXBM5KOF6EcH18OtO0dEs0Z7lvFitwStkINK2PEFl8uvCWri2agcePtGiXWXRIRqMPjt2ChviYDP/UbEEGQ/bjAWTPITRUCIAfDOHa1fVOYHK04MXgznpNfQUJkh3dh7XM1eWubpMXWALCQnB0KFDwSDy+uuvF1QBpi5wZJbVHZjqMH78ePCiNganHAWW7LvvvhP8XF7sxu3PO+88/PLLL+IwK0DwwrbXXntNAOmvvrL87TNg/vzzz8HyZczZ5bE4MsyUB51OJ9ZjSf1L7xxZZsUGjihz+7FjxwqqRENRYalte3tvEPByJJdD3szV4LsE6Q7kpZdeEuRmvhDMMeE7AMUUDygeUDygeKBhD+hIOqqmRs61TK2dQo0YaViiSWrvZYRz6/e5x+UlOTiS+TEC1RZ5KU4MwabHd0gzP4Ni8wXwVy0j8BtMi+Gq4K9eLY5L//TxH4uAOvD52I5xyK9Oo0P1oCpKn4j7en1KYLeT1KTB90h9VwwOmYItlLSCtXslY64w0yc4ghymi8PQENe0CKlNR35nKbJbu8/DmwduwFeUIS+2Ug1vkxrTBr6HsNALgR43IXNPAvKSPoOREzIQosw/+pV42Z43S4+VZW+BT8QISuHrI6K3tsc5pS9bWc52kig7CQ/KXqYP6kupjoe7pDPYtm/KtkV+jG6ynBndMAlAzNQJilDbml/0xR1mcZrtvJ1t+7qfXSwjURs48ioZY6wZM2bglVdeEbSD++67D+Hh4WIhGwNNXlzGQJO5vxz9lYxBa48ePYRKw7fffgvuWzLGZVdccQUGDRoEf39/wcFlLi+rOjCQZmoEL3xjIN2/f3+xLotVGLh/Huexxx6TuhL9MPDmqDODXI5Qc18d0VR0t1//rWd3BixpwYTljz76CMuWLRNkaqkKrzq88847xV3CtGnTwATojmjMl+GL3pjx3Q9/oPjOjCU8Wtt47IZkTc7WfPiRDv/xNdVPLT0PX19f8OeQP2+tbdKjnOLi4tYeWtxts+8lIfDWnICPj48Yn/llrWX8OM7Pz6+1hpONw9eXHwG2lPECEObK1dTII7nc/8bsH/Dp0UesSSM4nW+h5lUk10zneK4wte8XBHjp+biNaQloJJa/QrWlWjYHabPKHAedWw94qzNQYdgnQKe8BvBs/+VIq0jCR0fuI6hb/7WvVbljYuSNuDr+Sfsmje7/dOo1bMlZTJq9ekyNfgBHSraiqDaL1ByGYmzo9S36SLjRydRVCAwMFN/RRUWu6RZN7aup9dLLj2DH8XfgWZaPXpH/QnzizdbflZKMdTixemYTulKRZNkbyNz9CmqIe2s1iv56eMcINQTOmsYmMp+RHJ24aaFI+tk2/5hLBMDO2Pl/TofqfvEqygLXVRzz8PAQq/mb+zcl/cY6HeAMC/mRvys6wOl2zd/PLfmdxd+3HPG1N/4c8/cjUx8k69Spk6AkMLWA5+CKesHnzfxf9q298fVh+MffV5IxtuHvLmf1eR7828AqER3VGozwsiNYooJX7bHTbI1PmiUumAxtr/FmW0/ZVjygeEDxgOIBiwdGhF1FSgZ6fHH8cVEwLOQyDAu/EtfsoAVfEgatpSiTlnQzbeiDtRQxVXNswqbM1qc61Un09OqE67q8hrn7rkJ5jSPYe27Ppbgo6jYZ2OU+aukRdnrFEdvumrx9RcyD4JdkA4MniR9nvkFlSctz0Sry9+HUhv8QHSFXaM12n7wUqqNLEHNoBZ2uCjknNyF/66sUqfWlSG0pvVzrr8r9Y0ZF3i4kTFqEg7+MpPZ8A6iCh28c6ffeiJSN99hVd87nlFdqmT3fqLGkEDEBrgBvEmkMc1rhHpethodHVMsM+g/rxRnYZRdwhNaVNXSM2zREO7Dl/Er9q9Vqp2CXjzc2ltRHe36Xo1gnM+Uvrvfff1+E122dN3nyZLHKj2UxRowY4aSlUqR4QPGA4gHFA/YeGBwymegApLlrYx/1A2buBtxVVTBV9oLZNwNao1aAU6PaiMv3XIlTMW+gVitfU2HTBS6LuRdRXt3R1X8A9uSusT0ktpk7nFN1ijKy6VFtqo9oMw0h2pvGpEfVJemrSSqrXDwqd9OHOvTxTyxgSS9DWSolcOgMDUmJcRIIq5HiwsHF4yjkKo/IGw1FBHQdbzqs7VxsaD1DRaKIvtccQjmBX3Wdzm3KpkdctHBeHBgzEQWnVjo/eBqlNVUFAuBb7rikOzPbjsz0uSnF4SUXoP/VW6DxrH+8bltL2W4ZD3CwkZ+8KtY8DzQIeFmgmHkgBw4cEORm1nqTjB+xM+eEydW82s9ZCFyqq7wrHlA8oHhA8YDcA0U5WhRm0eNqTxP6JVRjbT81ti2JRGFOOGVhG4p9Ub8hK/gdeBmID1s5Fu5pT+JQDAEfF1HeWO++YoBnz1+Myxf7OURy+SBnZYvQd0Fy2T5Rl/VHmYN7eef7SE/1YlSXpYly1ofNHHIr/i5bh7KaAnT1GYRr4ufQYp4zi96piikZwYkamLzUMHarf0QrBm2H/2Ttews5BxfQI2ONALDuPnE0S74ANqCPpLksy9Sc002ac1pewf1FdbXWEz6k5CCZlhNOELWBUKVU5PLdO6Q/4oY80qKAN2v3a8jc+SKNaXPeDjMgogzNL//4T4jsfbPDUaWg5TwwZQrz/hVrrgcaBLxz5swRHEJeBRgXx3/o9cYk5nvvvVes2GMxYyZaK6Z4QPGA4gHFA4174Mg2LxxY7wO12oxaWtbvHVCLskKWJTJbpa16Z0xEYdBTKPDfg7/9l6Lv8ffhXhMCg7sjt7qLz0Di0XqKgVlWLDFwGA4VbHKYSIx3b9yR+BbWZn0jQG+QOy1IpmxqJ1Zdj0qKZEqA6sdQI9Kz3qc0x5YutuUvA7+e7LcYXX0HOvTblALt/mp4vUlRT5IwUJfTeXdxQ+lzBOSIC9kejaPdWXtel03NQAvGHIwi4yqVm4iQOxxrZgFHdT0DeqKy8CA0HgHQB/YSPYQk3oScA+81qbeKgsPY++s1Tarb1EpmFzq/9u2ZW1xdlm5frOwrHmgXHmD1FJfGGT5uvfVWB7ArNRgyZAguvvhikQlEKlPeFQ8oHlA8oHjAtQcKMtywbw1l2apVEdjlr2AVyoq0BJo4elYP/lRmNySkPWHt6EDsAwgppCxLQnrMWiw2OCOard3b/wPqyfHrfWLkTaLamPAZuKHrXFwcfbdIC1xVeFiA3R0+JnwVYUQqYWcJ7Nr2+8Hhu213m7ytzjXCZ24h1JVmAXa5oeZkDXQ/ljW5j9auWJTye5OH1HryYqP6a9dYQ41HkNMqRal/4NCS8UhefzdF3Kfg+OpZoh4rKKhIEaIpZqIoay0ltji7Rp8tWqjozDL3vovygoPODrVZGa854nVILf1qsxNSBj4tDzh+I9p0w6vjOX9zQzZmzBi05mrYhuaiHFM8oHhA8UB790BuqjtxM+0eDVOyCHu9HE4rrDXWA1m1yQPRufSo2C5tMJ/vhVG3Irl0L949dCde2DwdWeUnKVYsf8TOSSV+T1/g1D28wGphuBEbA8zIY6aB3fSkRsU1edJms945umvSyQGhiqbn/ndVs/ppzcoiixlRGVwZR2CZ0xs9Yj56Xr5RcHxd1bUv9wo9z75I7FcV7BPUCYn/W5q5DtkHFgipMqHL7LSVXSGpNlgkxuzKW3SXotquPiT0uTu15fkWHe1MOmMlAk9PT7GAi9chtdTLVt3gTOantG09DzRIaeBMaszRHTdunMsZcYo6ZdGaS/coBxQPKB5QPCDzgLvOJACvyWgLAOXRXalBnt8a4vO6w6QmmTO3AviXD8CovVuxvu8QeKr8KAprwNSYB0SU9vk9U61JLTZnLgXLjbECg2Q15mr8kf4hLu58l1RkfS/qfSVyst6w7rsKVmrp0b2tHSrahA8otW4lpWE1mCoxd9AadNE7BknMHnSuTsIroty2w3a0HdbrDuQf+YpuRJzIfhEXOn7sRwLkutFCM7bOw17G8VUzmnQGAZQeuCxrMzFIGonEEl0ic/fLyDvyOUJ73UkUi9ea1L/LO5Ymtm5KNbOx0mW12urmL9hz2dkZHpDkHVta2pL7VUDvGV6cVm7u5Cuofgbjx48HZ+hgsWJnxovVFi9eLHi8zo4rZYoHFA8oHlA8IPdATJ9KWMCubRhVhX4XWrIvVWvzKEZWQ+kbqpGQ/gimbKmArroT+hz9EVqTD0XvYjEs6Shu9l6B14dswQWRs/DKvmusYFcazRbsSmVlJFf286k3aKFatVQk3mt8wuGhctTqtK3krvLE84NWWotyKpPx8r7pKDJkC9UH1vZ9etdFyK+ycDhVZSa47aiCdm81aoZ4wOyjlrExzPTrU35X22gvW0+igQ0Gshy55ZS/tsb7wd1nCl1cCezycc+ARIe6tu3qt1XQ+SXA3SuCovU2EWQC0fZjiTYEuGvK05G19w10Hv4q/DpfSGMPpqgv8Z+dmu2NlNMKZ70wpOuVZ32M5gzAYJd1eFv61Zw5KHXb3gMNRnh5Udrff/8tcjI/9NBDGDBgAMLCwnDkyBGRju7tt98W6fAuvJAyyiimeEDxgOIBxQONeuDkbn0d4K2vqiKKg87DHREzvsCu5YEILppAqYE9oTF6iErn79uHT4d+itDSb3Eq6BQF8FT4yFyM76ovwp8lhwlqEliiPGuNGWdBW5ryFhanvIm3h+2Ft5tFPqqMtGJr7FIeE+sRQaSpGqzrjATfwZgQORO2magWp8x3GM5IXNM1aQtxhfZu+D6WK6K6Kkq3y1Mr+iAUPi8TjzejFmZvNSpuIR5zJ3nE2KHDNi5wI25un+l7BK+Wk0GoGewm3ojI/g85zEzLFAeihtRWNkbTMJO2bjy6/esXIeNlqMiiVMOeiOj/IC1M+4Dkv+jGx1lUmaK9Bcd/QNR5c0ga7TLndRxm1XoFGnd/QaUIiLkQEb1vadFkLq13FspI57IHGgS8nNlj0aJFePDBB/HZZ5+BNXfZWJy4W7dueOONN0SauXPZQcq5KR5QPKB4oCU9kJPiKMdlJnrDjqQj6D8hkrRyRxMslddRkXZDUHkwjoccJ3qDCYGG7Ygo+wPP7JpPRxoDWPLZmwh9svbu58cew1093seCpHuxNXepLDsba/XqtX54cfBfQspM3oNlzz5KzKUm+k9FC9M852TKmnA01+vDYlJlcL5YS1a5ne2wFm4vivSy5BabiuXBXJiHTywB3hwXRy3FrOer1liub8/LN1jrlufupOQO41F4iq4FaSFDyJ1ZD4sNo6EYyWtnNwJ2bZ8cyNs3d4+j9swlV1Hik06Dn6M0ya+htopuZGSmppuAWSSjNgJuumB4+XdG7tEfUGvSUjR6kqymsqN4oC090CDg5YnxqsZ58+aJV05ODpKTk0VeZtskFG15AsrYigcUDyge6Ege0OmNQo7MZCODYFJV42DVMvy8ay4u0+TT6cgBr5bq6k1JcDMXQltbgU7Vy+pOuQ6ENdMBHOlNK0/CkeKt+DvnJ1lrBjhhnvGY0/8Xl2CXG7DSw9a8pbK2vFBudMWlMJPOrqq8ftEcL1Bz21PPJ5Y16iA7DQFd6RS8ggehPHcXAVLLdZHKbd85Q1tFwX7ofLuIyC4fyz/2PVI3P2ypRtQGC9hlxmG9D/mgZ2Af8EK21jIenUkXKgLoPpGjaAHdO06GJlCsJnDbaQJKs7dg96LhBOg96CmGgQLV1eAkGmqtnBbipBOlSPHAWfcA/0U1apwTnsFuaGgoWIpMAbuNukypoHhA8YDiAQcPHNuuR/I+ojQIsGuJxBkJ7KrNHkiKeEnU3xP+OHFd6+kJDE7djZ4we76G7hXvIqJquUO/zY3psWSZn3swsquSSb/XS9YfR/U4K5sbRfUaMk5aYW9qAmu7K/4iwOY4I6Oxhoody+37aHTfSBHkL0rgNzsHfnflgBUg2ouF97uPbhfoHBm0ujCzsYoWt12Hvd/2QGXREZHBzAp2uU1dZJezrtlb4cmfBXfYAkPtj7b8vobOho2zqJVmrqeMcylOBjHDn3jFtVX5OL7yagFyjYYSeqcnDyo1UiQg76SlUqR4oDU90CDgZaB71113wdfXV3B3u3fvjo0bN7bm/JSxFA8oHlA8cE54IOWgDvvWEm/VWP+1W+S1AymhH+O3Ib4EDizRvOyIhfAeYHnUbVTVINdvFR33p2irBdh5mi2L22ydwrCkOVCS1Rbu6fkxgj0610Ga+t6Y7hDkEVlfYLdloMxsx0p2YEf+cqEEYXvYRGBtje5HGHvqQDLCVjNoDJg3/hk8sn0Ugel6MG+t0IwN3wfz4PFHBdTFpHZRaII38YLdtjWP1tGM4ZpVlakKQ29OQ2ji9Q22Y2oCG2vtsgayxsNxAZqxmiL9dqoYDIZL0v+ilvLIL/d1Nq37lN9QlPKb0yF8oi4QILwsZ5ugPsgqERe5NGOdrEjZUTzQVh6o/+Z1MgPO17xgwQI88MAD4GxqrGV3zTXXKGR0J75SihQPKB5QPNCQBzizmlyKTIUD8fdgP72Mmgq41fpjyKHFGPN3Mir3DcOoa/Kxe3x/bO05hY7XJ2ggmCeYlfZjSaC3MeCr1/iRusNm4uj6oof/cPQNGG/tSkMAiyO7zO11ZvnVGXhs+1i8eWAWSZx97KAMwXSIAF0YDI+G4ED3fSjWFSLDNwVvjp+D9V1WoNiQh3VZ3zrrukllmiMGcBKLunsD0YZliT0/L2lS+9aoZKwpQ86hz5o0FHNjqyh7m9nkSPcwm+jEnFAjzMYK6puvsvznmykXKrWcCmM/CTXpBjcUfbavz/vchrO/seYwNZZV4fG8SVOY55q87g56dxJtpyhvRzEDnQe/Wso4Sy2vg7K3hQsXoqCgwL5Y2T/LHnD93IUGXrJkCW6//Xa88MILYhojR44UiSh27NiBUaNGneWpKd0rHlA8oHjgHPKAEyQ6IOVd/NlzEAE4LS7alkcwxrJIyFRtxvpvdfDqRtEz4vwa1RWo8sgQznAjXd6QwguQHfSrU+fIIYljlc5eiTK1hTt7vIt+2ROwt3AN/NyCcWn0vUgpP4jPjj6K8tpCdPU5D3f2eI8WIVXjga1DHTpkegRzdxns8vw91F5IIm7w11Pex9GSrbL6rAVcWsMc5dMzXhBn1tNIZXJnqivk+6fXe8u0MpSlgRUbaqsLG+2QU/Fq3LxJoeFhpG97qq4+XUGWK2s0Ei6P8vKiOp1/D1QVHXI5bmDcZShOWwVWnGiqsYIEW0T/ByxRXquCBF1vAuoh3Wche/87gsfrmPBCjbgxC5o6VJvWezPlb3yZuUcA3jB3bywf8G+4q21k405jdqtWrcJ9990nsNQll1xi7eGRRx4RWCow0DGyb62kbLS4Bxq89crLyxNqDNKovXv3RkhICE6dOiUVKe+KBxQPKB5QPNAED8T158ic3PTF/fD6wF0YWvKKAIsMGi1mee9+5E2M2rcZE3em4LzDPxAwdiOVhir0zngJOko8YW9Sa/tyaZ+lxq6IdZTUGhF2JWYnvo1ruzyDEsqmNnfPlUivSCKN3RzsLlhFahCTcbx0t1BukPqS3jUEzsJ18TR/NhU2ZP2AZ7ZfAh9tgFPKQ1df51nGpP4aejd2IZ6EHbY1U9imNrrB2E1DXbboMaOhVCxEMzmJ2DofiFIta/UEGmeiywULERB3BYISZkDTxDTC9n1awK7rTwH3nTBJijg2zWfB3WaihGgJlYVHyfe20U8zOg2ZSwvuvqXFapsI/DpSVbzDzxcRYPt5trf9uSfX4fOM3aikc6A8dcgxlOPmQ7+0yDSjo6NF4LCw0PUNUFVVFTgaXFJS/6SirKwMlZX1yT34WHW1JYKen58PppyeOHFCNsejR4869MFtSktLcfDgQaFDLGvwD9tp8BNvMBjA0mS2FhQUpFAabB2ibCseUDygeKAJHkgcVo6ibDekJ3nCzYOSMlDGtdFXF1C2pmCMCboRB532oYKb0V8ciSicioTUJ3Ay8i0UjHoQ3VWDsKdwtdNWrgrdNZ7YkrMEu/JXUpQ3CIl+wxDvM0BW/e2Dt9NPfn30kBNY5Fal4AQBXmdQSkMLtHKrU6gPORI9WPQ3fCliXGjIIpqE5XdkfMRM9PQfIRvPdocXtR0oInBlLEd336FijrLjpN9b9ngAfJ8ogIkivTwhU4gGZXPaNlJmJBmxk3/divK8XYQJK6EP7C2UGGzn7mxbTcDWUJ6BwpOLoSXN35gRb4hqRackFQ5nrRork18HkSKZijTufji2aqbwGYNsEykoqEh+zkKRcN1n1p5XqTEtbKRrYm9pW5+gYDQlFaG+nJl/9L+cFbe7su+y98NgA+Zr6fN/tDwfxyoK0FV/Zp8tzlTLiS/uuecefPnllw7nvnr1alx77bXggCI/PX/99ddx0003gaPAXbp0wf333y/azJo1C9OmTcOMGTNE3b59+4p8CJz8i+kRjz/+uAhI7ty5Ey+99BJmz56NOXPm4OTJkzhw4AD8/f3BIJqz4/K6rH+iNQh4/4kOUc5Z8YDiAcUDZ8sDwy4rQgVFamoNatIrraVInmWksDiSJdvg7TBsfcTXcig690YciXkGfYJGEQDd41C/sYJKYylWZ30hqrG2L+kx4Nr4ZzAp6mZrU16U5swYIEfpEynSu9PK3XVTeWBa3OP4/uRch8VoWuKU3tvrY2RXniTubi5ivPugm99gZ12LMhMBjud3TyXliJMCO1cYi/F0/2WI8+kra2OMc0fR+6HQHibZK0pZXNuXwDSleW1L2/dd77rhLWCzsvgo0Qt6Er3A+W2MNFdTbRmlDn6FZOrcYawpgVfoEIrAfg8371gYC5p/faV+bd9NxClm47HsraH0wLK6TsCu5ThRTFiNQRhfAz5/IrkQoHb3iUFwt+ssh9r5v26UW8BAoNTWONJbawOCbY81d/t///sfevXqJTLTXnbZZbLmc+fOxddff40JEyYgOzsbo0ePxo033iir42yH+/n9999FADIqKgq//PKLyHqbkpKCwYMH45ZbbhHNOFMuJwvjVMissvXHH3/gqquuctblOV/WKODlu4fU1FSrI3Jzc4VjbWkNffr0EYvZrJWUDcUDigcUDygecOoBvS9HT+sjqFwpMKIG8UR5OLGbImkNmBpuImo6PPQKLEt9r4GajR9isMu2KPlFivQOR7R3T7E/KOhCrMr8TAZgGShzxHV4yOWC3pBVeYLkzDwxNfoBkdp4dcYXyKikR942VlZbgAD3CAK6Ehi0OVi3WVFbAjVRInQUQXx133VIJe6wbUrkV/fNwCuDN1gzwkk9mH3VlK7YURZNOt6a71VFRwXAYz6uZBzxrCLQ26DV6e0yYDTWgUZOPpGx6xVUtRDYbXB8cdAC0Buv15QaZgK5sQjtdg08/WLhFXlRUxq1izr/CkrAjzkHZc8oykhHuJs+uEXmFxAQABYBuO2222TrnxhPbdiwAV988YU1+svUh82bNzc6LgNjBrGHDh2CTqfDmDFjRBumUHBiMOYPs02ePFnU422OGDO94Z9qDQLeTp06ISkpSbwkB3Eo/PDhw+IllRUXFyuAV3KG8q54QPGA4oHT8MCASSWI6FqFgxt9qLUZVWUaVNKL0wizMXdXlbgG84Zux12b+qDKZdRNVK//R8I0LoKgrMyQXLbXCninxz9BCSV+FVxezrjGC81mJ76LUM8Y0ecLg1bW9123NbvHO5izcxItX9MQlLcA6ZsTXnegJEgNOYrM1Imk4i3UfxVivfsis+KYDOxyXRWt8D9GEeX+gROkpu3uXdA/KPGCg9GiM1ZhcKpcwJWdLUqjssKTxB0Vi9bkEUeH/ttdgRqe/omI6n8vNBpNh6I+Ph0/DvvLcnCkIh9eJC0X4eGNz3teQTdjLv5oTsP3vGht4sSJDtlpmTbK0Vg3N4uO3x133IGuXbsKkFpbW8+LZjqCrUn5ECIjIwVlwmQyCb9znYqKCitf13ZhHGfJbREtbNuJdKBtJ3+l9bPncLliigcUDygeUDzQOh4IjzcgPN6iYlBrUGHJ/8LA2rYmtQEnol7DYe/n4JP6CKpJmsqWZ3sms6sm4MxcW8mYk/vm0G3YU7AalbWl6Oo7CMG6TuJwUvFmfHD4HpSSegPTLeYOWi2Odfbqgf8N3YXlaR8Q39OE4WFTEa3rI3Xp8P7g1vNJrcGiSsEHT5TuInUHvUO9SmMZdE7KHSq2YYHOL4Gwq+OCRAatLsGuy/la6ADtHfCqtV6CE1xbRddQLNDj9e8mRPS7X6hTaFooMurSTS18QEs3Vj/3m4GM6lJU08K1GJ1/i4Jdabrz588X/FumLrCxCMDQoUOFEMD111+PjIwMQW1Ys2YNeL0Uc2/ZsrKywNxcZ1QHBryJiYlYunQppk6dKha/8QK4QYMGgfnBitV7oEHAW19N2VI8oHhA8YDigdb0wN8/B+BgzENIDvwcJk0lvSxcycUp80TKX+a8NskaCFKxpFiEviv6B13g0FW/wPGyspzKU3hx7zRZ2cOUSOKp/ksoQttHSJ1dTdFhvV4vIk7SinJZA9oprM6iRWmUiUv2AJnhkpHOy83KD+ZocaB7OBJJK7g9G0eh1W4+MFYXyKbJurhmY32ETnbQ5Y4ZsaPexpHfp9p5x2UDEsolaocL3nUDrc7okIdfNwF4aypzRD98/lpdIM37cuIKlyO81y0IH/DkGY3RFo0jPfjpytkzidpgK1H27LPPioVor7zyCmVgNAkZs/DwcNxwww2YNGkSevbsKbLbDh/u+u/g+eefx3XXXScWqTEl4ptvvgEDYcXkHlBReFt64CU/8g/Z4w8HS4I0ZsyR4Q8r35nxh7K1jcduSNbkbM2HOUL8x9dUP7X0PJhCU15eLn5AW7rvxvoLDg4W0i9M2Wlt488b+95Wlqa15uDj4yM4Ycwvay1jkOTn5yiz1Rrj8/XlR4AtZV5eXmCFG5YNaswqStTY/ps/Sgu0cCfVhlGk2qDzMiHtsA5blgTgr74DUOq1T9YNLxTj1L8m+u9MLdA9CtfEPYEQz2haHNavwe7eP3w3Nucudlrnqf5LSe2hvzjWGOAtMmTj8e3jUUGg19YY7N7X6zN8dOR+imqbMDh4CsmkPS3AvW29xrb5ES5/RxcVFTVWtcWOH1pyAapLjp1xfz6dJqHL2AXU10kcWjKeAr3uLhUQzniwM+hAReoSZsFZdg0fos57BiGJs5o8ivQb2+QGzajIdABbekAzmrqsyt/PLfmdxd+3HPG1N1fl9vV4vzl1nbU/18uUCO+5foWV81M8oHigXXqgplqF5e+H0dwYNKiIs6um/VD86/YcFOVYvpp1NeEoNe+nwxZg4VYTCK3JCzXu6dzEapx9rUA7AAG1eyku2jjQ5oZMSSgm8PnZsccIfBZjVNjVuLnba9Y+7TdcqTdwvf/bczneG34QLHvmzLbnLafIbiaivLoLWTIG1weLNlijvFqVO2JJjaF3wGjBUXbWR2uXlWauh4ESNHh4x8I7bIjL4WsqsgmgHnd5vDkHStP+IA7vUtLjvQT9rrUseqssPIgjyy+lC8YaxE27ts0Zs/l16YMnMqq5BrvcZ96RL5sFeJs/j3OrhTOwy2foqtzZ2TenrrP253qZAnjP9SusnJ/igX+gBzhpDmtaxsbGonv37u3SA4f+9qZFTfRg3yQhV8pcRQ/cDm/2Qpk6Hev6/Bsl+jqwS9giMeX/EJ95D61hqyXQ64MVgyJhcLc8Uj6qvwN6YwoBXqrfRGNKgRG1Auxyk405P6BPwFgMCbnYaQ8MiHfmr3B6zEg84ydp0Rpzev9I+QQ/nXhD8H97+Y8iDd9UITVmJG4kL4AbFXYNyZV9goe2jqBMbkXQkiRXP0pvfFvifKd9t0Xhib9uQ1nWRhqamNLEzw2IvxIx57/udConKaWu5abF6WGHQo1HKNEfLNfN4SAVpG1/Gn7RF9ITHg0KTvxAnNgixI37QiR4KEn9zVmT1i1j4M2L6hq5sWKdX8UUD7QnDyiAtz1dDWUuigcUD5yxB3bt2oWnnnpK8N/effddsGD75Zdffsb9tnQHNdW0YtoKdi298355VSU+8usFSopmjeLGZ9wvwK7GTBFUAr8MVi/YeQInQt/FwbhH4WVMRok2kfR9N0BNiSI4yW9zjTnBPya/TKB3DDy1jlzGAUETcXGnu/Fr2ttOuy6rKcAnRx7EtvxlqBERQGBH/h80EznXeFPOTxgUfBHmD9uB8poiIUvmbDyng7RCYf6x71BCkVZbK0peAr9OE+Ef7Si1ZTQ0j/JkonMOiL+K0vQur6MF2I5E15YkyhjknvjzOkpKkUn0DANFVOnVXow+X54Bg1CRt5Nm5PxzptboENrz1vYyY2UeigeEB3hppWKKBxQPKB4QHqitKsCeXy7GqvkB2P1VLP0oO4/otWd3zZs3Dy+88IKQ/1mwYAE++eQTwaltb3OO6sZrBxwBQ07nr0htlxQLpMAv1YrNvgMC7NadBNMRajXlyA/4k0QR3BBRvQI9y99Asbqr6NOx17qGjbxlVyVj9qaeyK9Kd1rzqrhHMLPLXKfHmFP8d+5PVrDLlezBLpfVEng7VrCdN+Hl5u8UXIuDbfRPScZfDiObTTUoFRFfh0PwiRhtoRs4HnJawqoGhRS5tXBgHatwRDln/zuoLk2xJItoJ2CXk0l4+MShx2V/oesFX4kUypxljRUbfIl7zKYioKuiiH1E39mULvkGx5NTShQPtKEHlAhvGzpfGVrxQHvygImE1vf/MJCmxEjLAplObbgb6rEfwTdyTHuaqsu58MKUtLQ0cNpNtrCwMKEakJ6ejri4OFHG6gFcRzLWweQFMy1lrHXJOqSNrQfu1M2IxGGVRGGgrFS0YI3XwiYMrkR2WA1UZeR/mzVpJpVjhE9lpniFygSzppp4uxYLNVqA5JmcC4Ppj48+iMcHLBLd/J31E9ZlfUeJJvS4ritlZYu+EWmVh7E640s6bvmccMUqkhBj1Qd7uTQV6Qib6zjIXE9r0iJ0lQ7abjRrtQ2q54MtYLyYiK+BVnt6P2/u+lCaFz22J5ArGSsuuHsGO+0zeshTxLv9icApecPE0WybCyc6qP97kvqrf7fIedXvA+H9/ouiU79TX+2EEkAqFANmHKDraqbMgHoCtBa/Drz+CGoq8wT1QqsLQE7S10jd+owAvOm73oR/zKWky5tge2oNbvN1O1vGf48tvdicP2OKdSwPnN43Qsc6R2W2igcUDzTBA8Upv4lojSxjFP3oZ+6hBAIdBPDm5OSAVRJsfzx5JTXnmpcALyfOmT59utUjDz74IG699VbrfmtuXHA1MIAChCX5avgEAsGRXsgqn47Pjz4um8bJ8LfRO+V1qE0eopzBh7sxAPm+a2X1WmKH++asafMP3YTU0iSazwlrtzvyfhcR2Xi/fugdPAL78zbAxz2QgK4K/x30Pp7bJJct44YC7HLyDAa99H+tphb6XE/4PpwH9cRAuF0fbu2/JTdOdwGP3wUvYs2x7+mGhW4miObBAI8jvH0nPAc1qSY4s0n35iLz8A+oqSpEQdoGFGfugIdXKHqMfx07fpqKGqIo8A0ldWTXvH5fTdFSchLyk76kfvLt6rXdrqdPZ4RHxbuYgEVVoDB9E1I2PSLqSLdAB34Zh/GzT9Giv6Zd35ZUSrGdLN94enp6ipdtubL9z/OAAnj/eddcOWPFA049YOYQlZMoi7m2cdk+px22QSFHcoxGOWeUo762EVxOr/nZZ59ZZ8fyc/n5LQcweCwes8kySIShfCMs0+FpuMEHzwz6Fc/suFik1GU+7IDhOhRrl8L72FWoVZejXHcEm3tOFgkprCfSghtF1dnYlrXcaY+cjOJQ/t/wo8VXz5/3O2nqliLGpzd+OP6K0/r+FUEo0tf5ty6I9/bY5xG/qBu6fJwIw195qHq5zgFOe2heIUsZcjTPPjNVc3oZcO0+HF8zG9Xl6dBT9rCY4f+HwqKGU7K6h4yDiihBZXs+QVVpGioo5fDJPV+i39U7UVFA2UlXzEBtpSXhgLO5SIu8THVphp3VaYuysL73Iic7lQLe5dCQ3q5a6/g0JGnd8w5TU2s9cXzPNwhLnOlwzFnB6UbknfVlW8Y3vyw92uS/R9vGDWxzv3xzrVjH8YACeDvOtVJmqnjgrHrAJ3I0PZatlI3BHL3ALo5RO1mldrTD2YlYt5lpCx4elmgoR3dtRdg5JaetiHtL6/ByilDW4LXV4S0r0mDTTwEoK9KSj1UYOS0fYXGONAXJlbH6fiTzdQg5VcnwcQtCoEcE/tR8jm8Dg4nFoIVBSwDShiYgtWutd+brcgIJBuXvn38YbmYPrEz/1OnwVTr5Z4oraYwabI/ZiC55iVBn1MK8qQQ1gxyBlNMOGylksMsv1kI+fdMidsyH1uYch3XVX211MZj6U5q9ldgMchpC1oGP6GR9kbnrRWtfTd5oJzJkabvewKnNc8RTE1NNGRIu/AleIUx9qjdn9B1W+DfSjZ8rv9W3tmydTYpAs25A7SfmYt/2KZKLKkpxO/OAQkJpZxdEmY7igbbygJtnKLpdZEksoPUIoEiOXoDdjrTamqNEnKpzyZIlwo3r1q0TCWM4cUtbGevtrlgQipI8C9jlefz9YyDy04kn2oB5krh/jHdvAXa52oTIGxDhGwWDW16zwC4Dj7NlRnrkzwkpfkh2Ht3lcSN8iMdpNweNWQuPGgvA5fmpSusf7Z+tuZ6NfvmpyP5F/VCaudEB7PJ4ZpLmyto77/SGdqA/NNYN00YsN3mN1WzO8dryNLHAjsEu29EVV9CCumQYDSUoTF5Kr18R2PVqhy7NxkpStZjsUK4UKB5oKw8oEd628rwyruKBdugBfVBvDL+ZfszKk1FKgTkPn9h2OMuGp3TnnXfi4Ycfxs8//ywWL7FEWVvaqQOetNiHEufW1scXTCQ/lrTFG+dfUdisqT038HfsoCQOGRVHsST1LVJEsKGbMKisoww0q9Mzqmwm6bHfAX65sH93fR4v7KWUs3WpkHkRm0FbjckHrhIt1FWk5RDbMPh30XWbFxedWiZuDFlZwZUx//f0jPnDlGmtiSoNPS/fiMqC/Ti59nYazu4O4/Qm4LJVbhLpAh8hpQZSZeBFigyGw/s9iKw9rwnag8bdl7SD/5+9qwCM4tra32o2GxeIkABBg7uVQvG2lEL9tbSvLXV3l/e/+nt1e3V3qNIWaEspxd0lBIeEuPv6f87dzO7OSowkBDKnJTNz/Z6Z2fnuuUc+EuGGAzaiZCgcaGUOKIC3lRmudKdwoK1zQKsPQ2j4CJhbMbRwc/KkS5cumDt3rggtGxkZ2ZxNN6ktu40DSvgiURupNjSFhsWejWE4G6kRp+EZApI6ayR5urXBrvXVMQ0k3W0INuboZ1byuXo89MCAb9AtfAieHroYz267UIQM7nGkJ+5Z9jQ4RLKDHDVUXxJ60gJeO6swkBeDQMRuugwRvQmIbgtUpM50NpoTQR5qFwt1Fd794+mUzXe2CWCX59AIiXLBno/EUGweYLzowLfod9EW6AlVBIV0QI3JiorcdeTJIY+8NaTC0AiPDXXNU8lTONBUDiiAt6mcU+opHFA40KY50BbALjMosUcNdiwN9+FVj6GVPmn1JWSkkfFavhbBoXb0HDocr4/agqUbt+Ov0tdRGrLJR8LrxwYR1Y6uqHIMQYz6x4Dd/WfYcryy6yoRIS1goQZkdAsdLEolGnvizTHbnTVGk4eJjtWorrTD2kcPW0//ng8a0PwJL8I+eL313nlQ7MaMgWpMz8uQOOQhbP+6dxPHqiYvB8kixDGrCNRNx6EWIsAuL8AaCpZ9y5orjmDXd0PIhdk24c3i4N/XEOBdTZsOatgsZUgY+ijilGAUdd9CJbdFORB4adqi3SqNKxxQOKBwoH1wIDTKhvGXkd4tkd5goy1fO/qNJwDQQ27gxPlHdxsw/7U4/PBiPH56JQ6mKvdP9LKvo7F+YQjS14Zhy5IQ/PRaLEK1sZh12iSEs4pyAwXGVsTDrvIF4Ny/RCvzvhUhf6Vr+bFhHbGrspvWpKLKWiavTr53zZONMM0kye5JDHZ5UnpjPHpMc/or1ugjhVu/2B4XYdDFq5A64w/S67Xj4F9XI6b31VS6YXzjdl3ksEBnTCBdYDfYtdmjUFJxF4rLH4DZ2stV9PhPGiL3l3oJDIx3/DgJmVvfFNHqWNWBwS4TG+5V5tOiTCGFAyeIA+5f0xM0AKVbhQMKBxQOnOoc6JBswcw7cwj4FuGs6/Mp4ISvdLcoS4dNCyMprKwz5LDNosLij2LJh6sKLNktyCCDJJtTGqoioy+z2Yrlf+cL1o1Nmk5yNNIPaAAZsR2x+LTOkr+SfrCNdE817CnAiy7ofJ9IYUBbF4hjf75MrMpgc5DLu1OUQsljQf9LtiHljHfR86wf0HPi/2hLPxFp88ejYN8XJOVcQ751P6PZOxCbei1COo6i84aBX5VKS/XJIK6WrLYEHMndi6Kyf6Go/HFk5O2gUNSTpezjP1J/QoWigc+Svw5ZZzl/79d+slQoy1ruJ/3UT9q3bx/eeecd/Pe//8W6detkrhOzs7Mxf77TWHjt2rXYunVrszCEPWdwpHJnCTkAAEAASURBVEl/HjSa0sHixYtx4MCBplRtVJ3qavfirlEVG1C41QAvx7fnmyn9Y1dBx0PsdmjVqlVYvXq1zP2P1GZ5eTmWLl0qXSpHhQMKBxQOnFAO6IIciOhAPoFJHcEfbV8aRgJBTyCkog+jCln7DDhyzCkh9qzHoYb3HsqGmaR/2VX7oa0Fp8JbWWABHLmXqvbnbtmzaXGeU3OQgCqBXvIMrCEgxO0PjZ6GHykAhhZ6gmz8+eCOPMfs0wxKzHk4UrHDN+MUStHqIxAaN4p0VZ2qC9u+n+icnUv31n3PnSF3fW8QS4gFkSqEIFKLcNBCQa2LcF7T34y8bZQWSlxn/6/OxU9e8Re0OIl2lTmuE7rfKtLn1YUmot+FG50R5xrZIAfsUOt4fPLnggN4aMg+oK3SWwcsGPpnNfr8Xo1Zq2rIINT3HjVl7G+99RZOP/107N27V7hou/vuu3HGGWdAAnac/txzz4mmGQw3F+Bl13w33nhjswHet99+Gxs2bGgKCxpc56677sKiRYsaXL6xBVtFh5d94N1///0YNmyYa3xXXHEFoqOb9pLyg3LNNdegX79+yMrKwrfffouXX35Z+AmUOuDrgwcPYuLE2h8eKUM5KhxQOHBcHKjI24Ajq+6CtYbCihLoSZ35J/QhnY6rzfZeuaZCjfIiPz/H9M2l7xYcwSUi4ITWzkDCSXaVBeVBe3HD6nOEJPZfq09ne3lU6i1Y2HUfdkfne2MOUZGD/5oQTnCpzAuSSC27jyzFtcGCczrdguSQPngn/XaRSd5V3YUE6PW49DplSXHQ/EqE5BfDMtwA87hgrxKnzqW5Mht7F16PmtJ9fidVkP4JyrP+9ptnjBmIFArjnbHuEZjKDpCRVyqiu11EKhFXuso7yN8xoVDXtfPEDou1G4HJ4xMiSY2yhNZmKkYhSae7nk664Zl/iqATxphBFPL4F1QX75aK+j2yGkP301/E9p/OpAe3NggMgXh20eYE+36rndDEV/Za8NFhK0y165L9FNr7+k1mfDKC+d102rFjBx555BGwwC8lJUU09Nhjj+HSSy8VGObrr7+WNX7dddfJcEw+GQ5z0IyEhAQRSMXTLoGjSrLgLzk5WdYGpx05csTVn5TJkl7GRBxK3bMOB91h/+UZGRkCHHfu3FmqIo4sne7USf77znU4wAvX6datm6s8l+Vw7pznSUePHgVHvOR/DMS5vmckRBZQMk7cvXs3Bg8e7PKjHmjMnm035tzPL2xjqjes7OHDh5GUlCTE+f5q8A3iWPfMKO/IJX/88QcOHTokVipSXbbAZl+bvBpg4lUMr4xGjx4trrkO33CFFA4oHGheDpgqMrD/D3cgCpaBpP08BX3PWw5dcIfm7aydtGaqVmHRux0FsPWestWsRniMFQN7G5C5vooM6Q0EQSmanMoEDYGfYz1I2mp1elPoXB6BcIvzA83nd43/jeAv3SG5oE3A0yACu0x8/7yyRbr0R1JLWJr9BWKD5B9WqUxdRwbM1fZy9PmNABl5NNBvMEG7y4Sqm2qlmXVVPsny+N3Y88sUAezqGrqp/JBvNgFCfVgXYewV1+8mVORREAu6OeU5q6Aif9iwOlVg1OoCkuYmyerbHbHQqANHcJMVbuAFg1ZP/8ExPWcjrv/NiOxyjlDVcILuWnTo0aaaJN39Zy5AaHQvsJu0A39eQUZ9lWCw3HnM805jPo/ybeX0vUNW2s1wj8ZC5ztK7dhH/qF7hnkvMNzl6jvjiI6zZ8+WgU8OWPGf//xHCOy8o7+xC0UGnwySr732Wvz5558CPDI45HpLliwRQr45c+agoKAADIgHDBgg/I5zlMmffvpJ1GNhoGfbxcXFOOuss8RueElJCQYOHCjcNnKbgwYNwtixYwVmYgB78cUX49VXX0Vubi7Gjx8v/Jjn5OS4AvnwnPv37y/a2Lhxo1DH4B17HjOPc/PmzWJ+N998M8rKynDmmWeCw0bz+Li9F198UdRlnMZjZxo3bhz+8Y9/COk2z4s97TDwDTRmUakJf1oF8DLqZ8D722+/CeQ+depUGI30EhNt2rQJTzzxhFglpKen47bbbsM555zjmgqDYY6c5En79+/HtGnTXElDhw4VKwMGvHyTvvrqK9HOG2+84SojnezZswf8EEo0c+ZMwVjpOtCRbxZTWNiJ2ZJhh/q8OjpRxPdLilzVmmPgqFUcgae59JAaM3a+5/yDcCL4Lj1vvBpvbZJ47m/eaevv9x0ObWFWZs5Hl+H3+uY1MIVX/e2VVs6Lhl0Iwjyhp/Prq9E68PeXsRh7sQpD5izHd4s2IaZyNKoNR1HY5UuMjb4ci/JeEawr15PcthbwdqoMxztLZ+DJkcuQGeYEtxJ/uRd2V5ZQGYrBBfFYE38UpQZPia1U0n2sspXhaFXdkj0uzTq/o+JmYHXOjwhSG9GlqAf+9cMrBHbdoEFHoFczyQxbr8Y921lpO/D2kdtRrM+HjsLb/mvUr4iiCHQngjj8b8aah0gndalgZtfxb4ldD5Zi1kmSxNO7EKUnDSe93EM/I2P1PYQnyf+uzde3b1zUNcgqJGM4D4oMeZ74keGR0vynhfu+Rlj86QR4p6PfBetxdO1DJKn+izoST5MLyPa7YB2CQ5zfKTbo60O7PycD6Wka1c5XTjZc83H+LLG6wpQpU2Rt8gVLe/l3dufOnT55nMDAdc2aNWDsxOVYIsxAkOm+++5Dz5498fvvvwtQO3nyZAGEeTf7yiuvFOcjRozAJ4RzuA2mefPmYciQIUKPmEOvM+7hvNNOO03k9+nTR7hyZOEkt8075LwrP2vWLDz//PNCIssYzpM4jzEdg1mWAPOYJ0yYAJbmcv8srWZpdt++ffHhhx8KsH3uueeCJdO8w//FF18IIej27duFqsfDDz8s1FR5957n8u6779Y5Zs+xNPS8VQAv33QGs7ySYGnt+++/L24Gx7D//PPPwaua4cOHg1cJDHinT5+OH374QdThFQevEp599lmwOJ+dyvNqw1NkzueZmZlCVP7MM88IyW8gcMbx1XksEvE1P1D1EQMfpoaUra+tpuQz6DtRffN4GYC1ZOjHQDzhPlsqxnqgPqV0vucniu/S89bSPLdZa1BRsJMCIwQjNLafmDr3yf37e978fdBZzxDkr9VfeYmX9R0bGn60vnZOxnxLDYNB5++LfPykTlDrq3f9z1GYeu1puOn8JOwqWQHTtnGo3HATisPWEnB8WQjc3hmwEf9eO4Hkv05wGWEy4I6to/DAuMXyZrk36k5HIPTswz3xW9f9Pvn+E/wgAq+CrOd716APcF3qS6iqqUDiA1ZoTLXb2lJZ6ltdyJ6DG06Vu7PwSOF0CNVVruYow4Nrx+P5MasQqe/Y8IaaqeSOeYNI2s7PvXMWB5fOEdLypjZvjB0Jc+UxHF11h7MJG7Xth4KDlqFzx74oLHuWfBoHI9z4KUKDv/dTsrmTHCjN+B1hiWdg72+zyPNCBdm2BdP0q4Wf4fBOEyjwxD0koXbuMDR37y3d3pnxGvycRb6sPToqp1vQO8zfe+lRqJ5TVttklQRv4gU+C/MYA3Foc29iMMqCP0ngceGFFwoAyOXYDuqjjz4SVfjbyHmsGhEfH4/g4GCBpThzxowZ4necz1ntk+nqq68WRwad3333nQvwclmmrl27CuESg9j169e7+mSpswSORUH6w9Ja/k6kpaXBYDAIvWTOY5WIXr16Cek0t8FAlom/DwyQmVhCzZJf1l3+7LPPXOMSmbV/6huzZ9mGnrcK4L3++uvB/ySpLt9onjjfUEb3fM6rFSbW5di1a5dYZXA4UGYSA1yW3jJTmRh88SpFIhbd843mm56amioktnwT/BEDa47AJBGL+qWVk5Tm78h983gYlJ8IaRT3zWNtbeIHml8kvi/+XtyWHg8vZljC73m/W7pPqX3+MbJYLH5/kKQyLXXk5415Lxk2tEQ/5opM7CP1BPYjymFC2TF87+kLEB4RJd41f+9FaPJM5B/4WT4c+ugHdZzcoPdIXtF9Jf02uFPaz1kYqSxUltEOkiw4hfxDy54aFr3dkdxedUC/03th5y4ycKPIbRHFp6F3xhPY0+URpMcU4/ppC3DXphGIMgWT5DYD83rt8stIDRnHJVSFYV9kIYwWHap0Fr/lPBPVhHfVBGitpNPrj1i6e3vf90UWh0VW02+3LakY6nyKGOaBJNRVDlg7qlFsyoFOHYRQHftUq5vmr3gOqlQK4MGDYCL22EksvmTvh7iwv/OD6sxo+b9lx5Y6O/GQ1jY4mhr75xXifA+GUGtVhZuRn/YR8Yzuq6W8zknotAcQH/2POsu0RKYmKJL0k2eIQBIS0Od+LNW56Nj3hpMW7PIcnu6vw4YiO3JNZA5IqCiaRL5fjwoi13zy95DLNoZYVYCB5YMPPiirxgb3HTt2FDvf/jwfsM4u6/9K5AmKWZrqKSDgbwRjIAbH/L1ifMIYyVtIxQCTVQck8gy37nnOAg/eUeX2PL/53gKN0NBQ0VRiYqL4Pkv9ciIDZh6T9A2V+mRVUwbPLPXlesuWLROSZVZJ9Ud1jdlf+frS3PtM9ZU8jnzWz2WQKxGvANgVBxOvUFjMfd5554l/rNvConPWMZk0aZIAsHyD+VxaYTATPb088Dkzb+HChUKszuoOd9xxh5Ams9hdIYUDCgfkHGCQu/un02GpyiawW0KZdtSU7EPmxifkBb2uIpKmkCTHqbrAEh7eek0c8ojLOt2ruHLZAA4MO5skPAR2VRKY8ysrdEZrY1dlO/4OF2BXajrI4pRw2giAlVHI3heGrUZecCVmHO6Nz3+/AL2LYqSi4qi10W6RXYO7toxGsC3A7hbjylpsyZXY8wNpV+Ai9UhnGxSFbVDUJEyIvwIxQUlIMqbi7n4fo3/UeJEv/am6PkKAXfKiJppz0Bcn95xqPFZ6Lh7eNAG3rR2ID/c6nyepjr9jZVCFG+zWFrCqLTCbqv0Vb5a0KgrTm75wJnb9cBoOLb+ZJLpOIQu/O02WZJJBmKHWk4NskLRLEsjQTVbuBF4kUOhgS3UO3Ui3sEkaTnWxfwGTlN/Wj3oCtksnGPDL2CB8PjIIv9KRQe/x0uWXXy4iPvLOsyQo4x3mm266CU8//XTA5s8//3whCGT1TTbw+vLLL11lL7nkEnHNgJJ3qL///nuMGTMGvXv3FoIKycsBA21JUMT6sQwq2XEACw9feOGFej0uTJgwAd98840YN+M1Bqf+iLEXCxp/+eUXkc1qGvyP+2LpMwsYeRyMAblNli4zsZT30UcfFfrA3AYTC3okIU9TxiwaqeNPqwDe5cuXC90RHgcj/7/++kvoaPCqgpWrWYLLRzZaY9UF6SZxeQa/rF/iSbxK4ZvKqw+WQrFrMtZP4YeCFaH53+uvvy70ZCT/dp71lXOFA+2dA5UFW8iqO8KLDXaUHPaS3nqV4MvYXv8kSRSt7vnDR9u6WVueRdHBH/yUVJIawgFDiB2z7s5GrxEV6Da4Ev3HlzsRpt/KDHzdGTW6HGzrcZ0zgVCpivSpP//jAgzOj4fRqhOA9sm1JDgoinVVOiOzC97961wE2bUYkZOImJpgaMn9mScNYLVfStKTIJJiZSCO5BXXZ6rRT9sD74/dj3dO24O7+3+KoTHTYNSGw2SrxO6S1Z5NiHNHOElyP4lDzbkhME0zouTeUNzQeTwyKtNQQ3WYVuTOw4KMt8S5vz+qCjsmppM6gzfR+IYnuu09vLOP59pclSOkmdVF22lRmEVBFJZgz69nCtAbEjeSjv6l3J59BscMJR46bT8809mAS0ULRU9if7vBUX3FP1WbUgsgtS6SOqfOXOp0KaZ27rJ6jp0XzFpDtGfSSXvePZS8zpCRmu44JbsSA9gIf8GCBWInm70ZdO/eHQxYWT+W9W0DERttsa4uC/pYf5eNxCQJKxvBsaont8Ugl3etGUDzjiCDS1YLZTzFaqHSzhn3yeC4a9euQrWUVT4vu+yyQN2LdFY3YGEl98Fu1RhjBaKnnnoKt956qzBCY0Mz3m1nEHvBBRcIu6oePXqIcd5+++0urw48L/ZecXWtmgW3zWoSbOz2wQcfCD41dsyBxielq0h07fHzKSU375F1cNnhMrsQY6tCNlrjibPonMX2//73v8HicV4BsYUgK1TXRbyyYUM3XkVwG8w4rudJrNLA0uJPP/3UM9nnnNUEPMX2PgVqEySVBjaKk1Zqgcq2RPqJVmloKJ+ae+6KSkPLSLA44tHBpdfWSnfdd40/bmOvOyxW2vyu+qO0+RNhKj9CWe5tWQbAqef8Bn1okr8q9abxD7M/I7l6KzZDAd4u5IV4cxF/5HjLkbcXm0plBRosnxsDUyXLJAjZeZCKgK3zV1uF4tC1WJ96Hsw6p0HLeAKzN+4cRiBVLrlNjyzAI2OXiFaG5iXg0Q1uSaxFZcM95NEhK7TC1UscqR1elKcmibETe0eRPiODsthBd2NfTKTwzxukDsEn+x8S51xRpwrC8A7TcfeQD1wSHQt5ZhB5pLrAlEag+I3d14ON4DyJPUC8ONIXMHMZ41sl0JNf1K+Gv4N5wz9EeDX1r7biyoIHMerSqz2bEeesN8m/0WyN3lTa9/tFFBVso6w6vxtJI54kV2Hno7JgK/b9dp5YNNpYBYEWGj5E/ALrt3uRMXYI7absJc0GYrKQljIotmHgpXto/ViF/Uv+SW7JDlJWy7z7XsMJfMnqF2JeND7S0Q/vNBllx5zPkGel8E6T0G3iR64kBlO8nd7Yd0r6xroaasYTBk6MG5qTGGA25jeLn0n+rfFUHwg0HlZz4O1/BrxMbETGWOnjjz92VeHnm383JT1fVwad8K63P7evjMW4PPO6ocT98HeYsVZ9xN8MT3djUnlugw3++bmQiNPYrottvDxtrhiP8Ril/poyZqkP7yO9kS1PzCwW6fMLwBP2nBy7pWCRPAOqhjwIPFpWg+AVBeuVsu6uP6MmlgrXB3ZbfuZKDwoH2iYHjLFDERTelXQHyUq49qOsIlCSOPThegdsNRVRGfkHnsFQDX2kmwp46+20HRVgMPvnJ+yGiCctB7uc4iD1hwTTFoyr+gCqYhN6qlLwcX8n4NWTqoL8znANIJikvRJt7piNO8YvxPW7hiHIqsVfyYdkYJfL5dL3cHWEHeNK1CATRlInIG8pZNj4bPk7sJdaqQ87zHY5ILM4TNhSuBhHy3cjMbgXPky/D6vyvhNlI3Ud8EItoFX7kXpy3UCkItdQrFJx+YabMDl9BgpD6KNaloCwSd3gaw4UqJXGpVvNvoZErHsr6deGxA5G/4u3oIqAL+uwZpDXAh/yA3a5DPuvTp3xOw6TL2sT+etVa0MISD9BIN2EXT+OEVJk6Z30abM1E1xSbBLxE/kDuyyp7jousHS+NYfb1vtiANdQjMN2M5LXBZ4X716zxNaTPH3yeqbzuT+wy+mMxRpLdfXj3ZY/sMtlvNtgrw0M3lky7YkHuaw3GG/KmLkdf9QqgFfqWBKvS9eex4Y+CJ51TpSLMM8xKOcKB05GDrB0osfUeUhfcLaQ1mooMlIC6ebGkr/N+kgXHEeSYTkg4G1NnTGuvqpKfgM4cGibUYBdu6eaAaFgPSrJYExPPlftyA4agh2OczGq6nOcebQ7QsmA7Y3B67ArKo+ku24pCndnVtuwJvGYrOdjYeX49+i/ZWneF1vo22imvhLIYK7MoMLmEKdzeO9yntd2kli+vv1GZFcegsVRC4gJrJZY8pD2xNcYVT0NQRcEowLFrmp62iafmjgHK3LmYU3+jzBqwnFFjyfJ+4LzebKMNEC30wwSRCO+LEn848pPam7EzfbPyLDIDeZdjR7nSWTymcgtO+RaDHJzdnITFkLR1CTSBkWR1HOiuMzb/QFJZfdLWYGPZNjHHg7Sfp1KmNbJH36XDi27nqKbdSGpLu8K+FuyBG7yhOZQVDanekfDJYYndLwnSee8S7Rt2zbh6YB3i1auXOkT/OEkmYrfYbKKA+/ys6pFa1KrqDS05oQa21dDt+ql7RZFpaGxHD6+8ry6U7w0HB8PG1ubF5L8vAdSaagq2kX6jaw7ydJHQjN0jO7xD3Qe/Z/GduUqr6g0uFiBtDWhSFtF4WNlYYbd+dKZisDlpIqXkWwhKSPJYSt1NuhJqpgdXI7u5dEo13FwCjW2xuZg3eg4bLQtRo3drbYgtdOQo14dDCtJIFmy21QKrQnHW19/B3WQDlfMnkh6kgySHJiWeJ0IP7yapMHOQBfO5+qJIb+hS2g/0Z3luT3ouCMSFfoympMW84Z+iJ+HfYNzkm/B+V3kRm/NodLABmppP08iV2EZQgLrsJuRNPJZxHS/0O/02eBzD4HYukl6X+oudbLlDpp9gHSS3YssltgpKg0n211sH+NtVQlv+2CpMkuFA6c2B4zR/dDvwvU4tuEJ8hNbgaiu5wm9xlN71q03u5gEkmYSNuKlhItYz8Gp4+BOItWAQ7rRAvBqCYhGkgcH0qRF54oI5AdVYnnSEaxJyEQueWxIJuM0ewfn1rSrgUacWOw1FFRCT75fA6se1NecmsB3sZGiKFX0xFelaSiYZhJuyQpqMvD4lmke1Z0zfy/9DjwzzKkzuvDy35C2bjGiK2NxLPIIjsQcEAzaX7bJo17znTKA63veMgqr+xftZhSD9W4N4d0CdlBDoYDrI50xEawO1BTdXOey0reHQOm+JZs3xabWQqcxkmrGIhnYbd5elNYUDjQvB+rXQm7e/pTWFA4oHDgFOKAL7oiu4/+H7pM+VcBuM9/Pjl3NSBno9GDATWsIZOocVaTH6mV0QyD4oGEcCjVdhKxdGobOoUGsyYjzD/TBvZvHoFpnJtWEUh+dW6l8Q44sebUeB9jlPsqCSxBZHQOHxQ51jQPh+hiwv95icw5CtVE+wygxO90XcQYHlziUuB+ru//lBLuUpqYQG00Jd+zTUR0JoR2HkwpCJXlp+JPchgUGtVVFO+pohT6zpOOeMuF9ETq4joLyLFrQcEjfoPDusvvrWYiXOM1H9bdmpedgn5F0vFNGiXDi+pBOzdd9C7bEKlysQ9vc/1pwyErTLcABRcLbAkxVmlQ4oHBA4cDxcKD7sCoc3mEkCboaXUzrMKT6W/wY+bJc6lsr8V0eQtv6ZQ/KuiOvvgIkxVYb8cKyC3DPGQtIOhtKELGaBMU2WdkWvfAQQd6w/AECvE73VUd7H0NHpIquE409CUzLPVoQPEEEGbpJND7uMvxw+EUC7TXEA3Ksz7qwVId1fVuKrDVFQq2B1Rk4qlrW5mfRbdJnCE8c79MlLwDVWqPwsuCTSeONG3AnjNF9yQdvH1TmsZN9pxTbt6wzJZz8XUd3vxisS8zu0FqeGOz6jslOaVWUxY9aJWktbCa97j2hNrw78gNo2Q/3SUDsiIp1YhVSOKAAXuUZUDigcEDhQBvjgEbnBh+HgsaiQNcDRls+gY44ilHh1pfkYVerIwOOnqW9BpIOq1UmkhBbCLgQhKGmvbQjAtY/ngwGrQ5yr5CaPQCzN9yIlMJeFNGtAu+Pfxl94s9xAd5YQxL+2f1pvL/3LiG11ZLlP/97dPCPru5ZQvfa6M2Yf+RV7CvbiFhDMi5OeUioRLgKNfPJHjLodBpnuu/FkZW3o8/Mv8jvbIyst9ieVxAgfo4YS9JcPy7K8ne/h9gelyGkwzACvGtldX0vVIihshFJk4UXlboky751m5rCm72sn+2eK8vQowhw9xz5L7yx9zbSs85FgrEHXuv5AoJOErDL3OBnh11defr35/TjJW63LkP8421fqd/8HFAAb/PzVGlR4YDCAYUDx8UBY5gdyX2qScobIgBumYYiEQkg5bXtTIZrBofcp61nx1YCuLui84RbLym+L4Pd5ge9PC5PsARc0f0pbCxYiDSswodjXyb92w7IjDqM/LAc6MsTMKIDGz46aWzchQSmumN70V8EpoyYuv98RD1JG+jaQlRfEQ5bN6cnhlld7pKqtPjRG+w6O1STasN+hHoBXtb5HXhpGrK3Po+83e/W3iv3EDmYxJHV96AiZ6U7MeCZQ0R2Sx79PDJWt9Z8/Uv9g0hvOSw4AY8M+j7gaE+GDPbB2xJ+eE+GuStjdHOAl3UKKRxQOKBwQOFAG+OANkgOIIX0kNEqEyFWNQUD0DtqcE7Zv0WS9Mfp6YD8vZLETkuGYh/23+IFRZtbwusLdnksy3K+wllJN4hhsZHZls5rBdjlhFV5vgCqW9hgnNflHlzw52x0eM8B7X4LdHssCPu/QuhWO114icZa6Q+7HfMmNmCzVOejPHslSX/lCw2W+CUOedBvmG0bGas1DOzW9khqFE6w2/wu17znFPjajhDSYT7w1xzs/G4U+Qg+HWVZy+nRa7qnjsB9KTkKB1qeAwrgbXkeKz0oHGjzHCg+9BNJpt4XH3JpsPxBz9z4JPkIvRF5aR9KycqxlThQeIzDz3qBXqlvAld28powrPJLMmhzG7gxFGG9y7SofKxOPIqrp/4Is8a/9E40HaB5qRvfI4PbWtBNZ2w4xqEpQjS+4DCv5jCCNWEI0fqqXMTlxiP8vnxEzslB5JU5UGc6DfJUhSSx/qNK1i2rHBs/loNLWYEWuuh82ks+LeuMCchY9wgOr7gVO+YNpIhp+3zK9J5S+65wpDXiDVOnEU8J92biolF/5LrNjarayMJhiZNFDbWe7hf5NtYERePQ0jkoz1pKwTJyYanMxMG/rsS2L7v5gP1GdqUUVzhwQjigqDScELYrnSocaDsc4FDBluo82jG30D8zKrKuQP+pb4gPuoo+fJxemvE7ivZ/g9RzF7edgZ/iIwmJtKI4O7CET0WStt6WZQRwnRJWxq589t9hK7ExLtuNS6UMb35xYfb1y2HMApXxqGPQhGJywlXoHTEKRyt2Y0fx38J7wrSk6/DCDt+AJRyEomvYABg0Iai0liCEMK2e+tFaOuKluZ9Qy24gHv5IAcpeiIWKvDfYQ0j7t5IH5CZOb20Kiz8NfWYtQ+aG/xPGaDZLlYiM5qgNl8zj2fv7heStYAW0QRFieGzcVnjwJ4QljIGlpgzcRlTXWQgmV378/lQX72rkNJz31nkzW5YH3Sa8S9ESD+Dw8lspGE0ZWCodiA4uvQY9z/wuULaSrnCgTXJAkfC2yduiDErhQOtwIHfn/+jjdog+6JUC7HKveXu/xdaf/0HfWI0Au9JIzJVZYEmwQq3Dgb5jOUgEAx7/lGr6gyCjlmSITiDEJfnfPZtPFydGiw5hJj1ia4yIMAX5b4SM2uIKZ2JW5/p1RWtsFViY+Tbig1Mwo/OteHjQt7i5D7mmCxuCC7rc72qf/UNoyYvC/w3+FRxF7bFBP+KMIhWuztLg0mwtLs8vRJUxy1VenBD2DfqtCrY4Msjz2jEXs/NKk1duuaugsC7keu8T9Jw2T0g4PcEu96oiIzWn1wXnGHZ+PwLHtrxKOyVrSPq7B/lp71O47WRhONVt8qeNHqhaF44+BKhje/1TuCgLiTut0W14V+DQ1OXVF6O08gaYLX1c2du+6kGS3EJnxLh61BaqKfiM1VTiqqucKBw4GTigSHhPhrukjFHhQAtxoDxntU/LDpsZlUXpJPVzS+C4kN1WA3NVrk95JaFlOBAWbcOEywvw95ex1AHDPjn4DbKXE9j18s1LpfSktzt3wcVIi8kXA0stjsXd435DaZCfoBEaK2aNmYLT4s4nn7aJ2LzpAQQRuFzrq6Eg2mKXYB/texAPDZwrrqU/0zpdQ0Zn3bAy91vydarCiI7nIjqok8g27/sRgyqDSeeY+3eC83XjZmPckgXQWmvdRVGyykx/DGrnUWqYjjxrOwVl0xyywJYSWOLtUaVFTjX6MNrKl4M8u62aVBWCRX+F+75xbvVL7w0fSa2B3Zl1HvNfmgdJrsm7AdeRE81QeHeQv29cptPQhxBEgDlp5JNCd5bVCY6H2FgxM38DrLZkao8WtAhDh8hrEW78Qowhe9tL0JBKg/c8vfvkOago+IRCCgdOJg4oT+zJdLeUsSocaGYO6EknkSW5nuCW1RiCI1Jgqsh2SX1Ft/QBN4SnNPMIlObq4kBMJwvOuiEHv70X71OsUCu/FwwltxouwIGgcQSt7BhT+hE6WncIGXClllySCVmwXFTaPfwi9I0+H5WWMizZ9SgywkjRgPb9yLkDCDf7pb2l61BlLYNRS05ZPWhA1ARsLvgdq/N/xPbCZUKN4Ykhi1CU/iXUHmoAXKVaV4O0zisx4OCZogWLxoRfhv+CiqOVGJnYF30zBnq0TKcEok+EWoPnIBKGPISjq+52vxPCPZyK1BfGOedQQwsMb8koBQsxk+4rqwzZaSFp9+KDs326cxJI9uyQzq2mYldK9tYXfd5VV2YDT/JKPoLZ2pNKc1hnJxWUvAmDfj302r2oJp1kNYPvOog9TkR2ng6NLrSOUkqWJwf27duHJUuWoLS0FBMmTMDw4cNF+GUuU11djc8//9xVnEMzp6SkYNw4eo8lI1XKzczMxNKlS1FQUIDp06ejd+/erjrKScM4UPeT3bA2lFIKBxQOnKQcSCAJkvjY1n7kVOT/1EEf38GzvhMfdga/TCqNAWGJExCRPO0knenJN+yMPQb8/kEs/v6qA4LDWZLLkNZNWbqBlOL+Cf897GFsCz4fFZqOKNfEY3H4/VgU110YrpUZTFTWDXbtDi3S7T/ih5IncQYJ+Wes/h2H9SRBrW2Odr0DEo9iY/4in/yFGW9jZd53MNkqBdjlAv/ecg6KbKU+ZdXUyOJeG0S6RW3BDVeej3nVL+Hno6/h4RnXYv6gr2R1WKfXmnJi5TNRXc5Bl9PfEFJatTYEkV1mYMAlW13jdEYdk98jzjSVHRQBLNJ+IlUTb0Dsqu3/pCJ3HfYvno3d5CEhP51AUQBg7L+2b2pkKIFmMjT0JJXKjBrzaJHkoIVPeJJzEeJZRjpnFYuOfW9Al7GvSEmnzDFtrR7fvhCOr5+JwJ+fhdDipHmm9tZbb+H000/H3r17YTabcffdd+OMM84QQJd7YBB84403YtOmTdiyZQuWL1+O6667DmeffTaNwTmIb775BiNGjBB5RUVFOP/883HLLbc0zwDbUSsqikLi+4a2IwYUFxcLp9T1TdlgMCAqKgq5ubmuh7C+Os2Zz33zWFubeIUZHx8v+mbn3a1N4eHhqKysbHan4Q2ZR2xsLCwWMtiiH6TWJn7emPe8+m9pslurcXTNA0ISxY7xe417EsaQCOTl5ZIO4gfCDZMxZgAZ38xssaGwA/eICKfhT4t1EqBhvr9VVXLPAAGKNiiZozrxh42fnabS0d0GbFwYSfhIQp7Sz7R07Wx5VCXpl5r+IoCbgPkR//XpzqTNxx8jSIpfSxyhzGp3YKvjEKVwm+72EvAfxKv/JxV1HuVFZHkvj1xPagvuth/dNBnHqvbKyvBF74ooTC0sJUmzsy92m8bb+wtUc/Dighvx5PQ7sbXzethUbvWMIIsB//3lY3SpJGkkVSt/Mgb2uIYB3ujoaPEbXVIiVz/wGVgzJvBndNuXKQFa5HlL9y9AkVZKdtBCx2QZjGMFK6lH5/0ALIiPvhQhhl/rHcWAf+wiyW6tGkqA0iyh1Gg0jX6npG9sgGaPK7mioqJOP7w7VwRhx3IDRTZ08kStcSC+mxWTZrs9oHgPgH+f6/vN2rFjh5DUMpBNSXE+H/ysXHrppSLM8ddff42cnBwkJCSI3wut1vmM83g7duyIlStXkgDCgcmTJ2Pt2rVITXVGJ2TQ27dvX8ybNw/jx4/3HppyHYADDfsFCVBZSVY4oHDg5OcA6yB2HfeGayJq2rJkYoMcluY0lKymUgLHudAFdyCr9QBKoA1trJ2X2/CrN/+8QBN9BMdWvoeu5vXEKRXMKiP09gqY1fJtZp1NrnbAPlQrMIJkfKVk8CZfYBTgCsTDDXhZChtPardZ7t1v2V15YssMPD9ipYi6VWEpRoXFP8BMDy1GsE2FCSUq6tOBfD3wU0cb7vzNqQqQQcEoPMEud6Ix6JF+ZTZi1SNg7UW7DMFuSbZsEG3kwkaSUbE7Ql5OfKkVwa6a9IntgRfJKlpU6LQHSIXhb5LqTqShmimqXU6DwC7rI7O0mhe/pxptXUoPuce2hp2e1/yjGpTkqRHZsemi3k8++QSzZ892gV3mGwPl//znP+jXr19AEM4glxcNLPBhUHvJJZe4wC63wYu69PT0egE3l1XIzQEF8Lp5oZwpHFA4UA8HLGS0dnDZDagp3k3SRwtSJnxMIVAnouTIIvJP+jD9mquFO6POY15AdPeL62lNyQ7MAX+iVaf0iesMrv4Wh/SjkWY4EwmWXXT9g48Bm4r0R42OIlcXWuiRUG5AB2sUjoTbCGS6ssSJg/Il0tA3PpHAbkYAsMvl2NXYwfKtKLcU4v30u2ARRmlSC/LjjtAw9KiqQLyZdowIE168ZxKGZXeHLUGDyNhE5JmzZBWqbGUI7ZoAa5Rz8SXLbIMXQvWHgMwJpzrArjQ2taoEQbptsNnjERL0C6LD/0/KqvtIz1Ph/q9PScBL2JKku17Tp9spSXy9chp8yWoMU6ZM8SmfQtJenU6HnTt3ih1ULvDee+8JkMu7yAsWLBB6vj169MDGjRuFSoR3I/VJl73LK9eSV2yFEwoHFA4oHKiHA6z6sOuHUagu3C7ALhc/vPwGFO77mhzx3ywsuyXfnUfX3I+KPKeOZj3NKtl+OFAfdtpqvARZugEo0nbFHsMU7AiajjPLnxMt6eyVQtobZs/DhcUPIa4ilozGDLhx62C8svwsvLImEp2qbOR+1y25IuUZxOJT4ZKXJbuh5DBAgN06MJzVYUF+zVG8tefmOsGuhrwB3J5ZiUQCu5KEJSL0byy/438oe6kDrhngDPDAag5MGpIk9gwfgX5RpPN6kpBGa0T8gDuErrvnkLUG9rDhqTPree5ZsvXOVeR3OTbiQXTuOBgxEY+TxNH9HPAotIYOAQfDfoZPRerUi4w62R+1B1lqSLob5+s5w6NIvacsifWnCsi6uSaTCaw2JxGrP2zfvh2szsA6vAsXLhRZHTp0EIZqUjnl2HQOSL8/TW9BqalwQOFAu+BA0aH5JMGlD7aH4QxLeXN2vOZ3/kUHv0doxxF+85TEujmQ3If0qncbAxdisFpraBhuy8Fg0w8CLl5afCOydf2gITCaZNlGaTactvsH2rL+HyYdczYXRHV/Wv0aBk99WrgKc9CHviPeR4L6ddIXdO7slpIWQf1qpw7sKV0deIyUw54hEkw22DVGqG0kMq6lnaEO7Cj7DnM3rMbZFH74zdHb8fG+B0haXIShMWe6QhJL5U+GY1z/WwksxiJ/97vi3kT3vByxPS7Fzu9Hkp9rp5qBIbw7GYVNpkXiV/W6/vKcM9+X+hZBnuWP59xaU0Cd8Ra/r81GbK8rj6fpNlt37PlV+P5gOKkYEODXOaCjsN5nXlNBEtfjG/LYsWPx3Xff4cEHH5Q1tGrVKqGjm5SUJHR4OfONN96ApMPrWXjYsGFgozVvuuqqqzBp0iTwUaGGcUABvA3jk1JK4UC75wB/pD3BrsQQu52kIF5AmBKETqNURjk2jgPDzi5F7pEgmKrU9BEkDwUWln56iFtrwS63elb5My5fDQZHBVLM60RnLK/iGl0oYMjwAvI3hnKRzn/CrTXYv+he/JVQijcHL4dGXav36dGFZ3euil4nq/MYaHPoC7mUUCo2kFyVXRZ+CfKLHyPDUyfg/TvKhq1CtZiM+mqO4OuDT6HcXITb+74vVZMdVSUUXmMfRQEMVsHav22rOMT0+AeSBlwtDI0kyd6AS7aBgzqwjm9Nabr4pyaJsPydYR1l/zxsTbDrZDyverzBrgrJY14kdYb+sntzqlxoCAld8mAZirI1QrUhOsEGTjteuvzyy/HBBx/gmWeewcMPPywM1Vj39qabbsLTTz/doOYvu+wyvPrqq0Lv95577hFtsF7v77//jtde8y9saFDD7bBQM9zSdsg1ZcoKB9ohB0zlB/3OOqLTZIrA9qNwZ+Yu4EBcvxvdl8pZozigJsnSjFvzUHhMJ/QIN/0WgarSQD/XDG3dRGZJsKr0sKiCsdp4LaZUUDABClAhAWCpJGPbCJjcYFfKaOTRH9gN0UbiuWFLkU4+exfm/4GBBKB49DWkL+EEu+5OLPYa/JLxBqZ1ug7BWrnRnXaXCaEvlYC0IqCqpp46aFD2Im0DazyRubuttniWve1lGpaK1opuEMnGg2py9ccRDp3kH+y2lfmoyeeuWsNi/1ObGOg2J7HHFtbHve2229CtWzehoxsaGor7778fV155ZYO6CgsLw/z583HHHXe4AO6gQYPw1VdfITIyskFtKIWcHAj0C6rwR+GAwoF2xIHqknRUZK+CWh+K0EFzXDPP2f4a8na/VxsdymkpvzHMjjTakibBI6YWadE9ZRY6pF6F9AXThQ6jzhCNLuPehNMvqasp5aQJHODAE0yDJpVh7U9RtKjwAnok/ivQdEOCdZeQs34X/hKCHaUMr1Co7SZA7lrjVRhb9aEQ2Eqg105XKxIP470h22WjCqFwxPdsHoO+RR1JLYKsyYetxOa4bFkZ3wuW8ZIkmiSYBk0oLux+L85IuBTv7boP6/J/go2MnbaQS7FrM8kIiCSbOvImYfFyusDBDhj4BsMNeFWFNoQ943TFKM1aXUDBT74pR83lQkTsO5Q2mFJNBp7eOyMMfuXLlMADby1VhsAj4BxyJkfqSwo1ngOsgzt37lzhKo9dILKLUU9it5/slaEu6tmzJxYtWiS8OrALRfbeoFDjOaAA3sbzTKmhcOCU4kDRwR/ID+/9tOWqo++yCRnkkxfkr5X97pYc/VWkOSdMrnpIv21VFH38ahHIl/FWdNSbMDCqDwZfceiU4ktbmkxiTxN6DK/Avg0MCCX4x6cqLA+9DZeV3IQyVSyqNDGoUMXLhn4o6DSMqfqYICnDXN44V+GHLhX4pt8mgjFuIyQ1+fz97I8LSOvXThJhJyJ9YNNYPDdiBbZ1yJW16XmhsQXjwi4PYXrKHJHMPpXTCtdiNQWhkKicQhi/2VmDkcZBiLUfQ7bpsJQljmYCu6G6aFmabjdFJiM1BjVJdiVSkQBOv67mpAK8hrCuqOB3q5kAY+urONAzY6lAeOJE6TYoxyZwQK1W+4DdxjbDOr4K2G0s19zlvdbZ7gzlTOGAwoFTnwOm8qM4uvoeIYFybrnWggsyeio+9L0H2GVekMSNAEe0h6CHge8nZGykUMtzIDzW/3arSR2Gz6I+xdbgC/wOwkYqDhzqgYk0E5Gt7YOv+9NCxiPQA+eNyUlCtcbiArucpiNdgiv2DCSDMwOMVT1Y0OdDNnUNOuyXq6/kVR8haS/rDbvJSmj1IHLwwKBvRaJREw5WfQjTxeCN0dt8Qtqyzq4ntne1pHPOxXXdxk/iB93rBLvkfYKJQ/OyioD/yYkidf5pHYmv01pLo48QXht6z/idjvIFSZ2DVDIVDrRBDigS3jZ4U5QhKRxoLQ5UF+2gj28ESXBKG9Qle+6hIEQyYn+sCrUsB8qLNMig6GuByE5g6qDhDAKkdHM8RYDkUSPClkXyWvK7S1DWTpq0dw6dDruDDN0opKwn6W2U7+WaifMNVh06512H3pmPY2v3a5EbtYCwmvMhUNl16FAyFSYzR99yt9cxuDNdy/VS1dQ/R2Yz2aqQHNIPedUHSRVXj0cGfkeg1xdMWYYGCZ1d1TErJGzuIBFN5R0nl94iRyYbNHs/sre9CHNFJoyxQyhM9xlI/2UKs7dNUqcR/yaJ7gRwQA1DeDdwcBqFFA6c7BxQJLwn+x1Uxq9w4Dg4oBER0fxLDr2bZYgTRCJdjpQlkRPEdJIulWMLcKCyVIPFH3ZAHnltqFcqWCv+46ATDHzjrOk4p4x8rVJNsj8nWa8J72x9B9X20ZQt//nf0iEHBptcBmJSW0nX9whSM56A3hqD4Xu/RVhVf6eklx6IznlzMGLvd6g2HsDR3GwKW+wEvb0iR2Jk7EwXN7SkIqNTB+GaHi/goU1nIKNyF0wUJKHKWorHN5+FIpMfPWG1CuXPxMDaVw97GDlYoyAVFQ9Hwdal7RtP1ZQdwYEl/8SeX89E5sYnWayLxCEPUUTDN9Gxz7Xky3obpcl57WLWiT4hferI5GkICkuGMbqfAnZP9P1Q+m82DrTRN67Z5qc0pHBA4UAdHAiLPw0hHUagImcVbbu6JXRSFfYrajWR4RBJChk0ZevtQn9XR8I7vg4NjsOjqZ+iIP0zAlA2RHaeDp0xTqquHJuBAyvmRguhbb1gt7YvMv8SWrh8ObX8eZLpeuigUJrBZkYEBaSwxpKRoS0EGjutYEjdYOjuDfgs8U/MObYc5Try3mBXY118JvbhQWT0ughF4WtIAmzG8PR5MOlykVhwGfnx1eBAwivYG/oU1LvJO4S2HC+OWI9OEV1xTa/nRfCI7UVLST83Cucm346fj75O0maWNLsXWXYC54sy38Hl3Z+onYH7oN1lhjqLJLw0BQd5ZrB2b/tg12oqwdZ5o2gS/IY4KBzvYXq/VqP3OQthqcomqWklheDOJ/VrcufmFJS7J9wWzmhRm7X5WXQ5/bW2MJpmGQOH81VI4YACeJVnQOFAO+dA90kfU/CINyls6DewVGZCoycLYHKbFNPrKpJK3Y+asoPIXPcYRU7biASzGXMyHTgaokVs0tkY1/1OHPz1bPpw8/a1Hcc2PoFeZ/9ySoYfPVGPiTO8qfcHm5GSd5pzhHZSE1CRDraDpIrsmkxL554UbLPh1S0LcEe/bIws2Uog1oTo8vHQ2cIp3HAnPDB0OfRBK2EPCkZQ2a3YnnAlPQ8kDa6NyLWx98UwmBNRGL4cccUzsDfpadhJj5f/Md23fhSmd74JmwsWk9rCIQFub0t9T4DeGlulDOxyeQa/1bYKPpWROtuK0P8WS91ClWVG6H05qHg5gSzX/M9d1sAJutj31y3UM4/PiWYddhPM9F7tX3wpWIWIpb1sBMaGoW2TbLDUFLbNoTVhVOwBgd2DKaRwQAG8yjOgcEDhAIVFvU38s9KHTm3NRWhkJ9pujhCcYR2+7lO+wKFlNwlJcAzpJPbrfRties4WDvW92Xfw72vRd9YyZSvUmzFNvI5OMCN7v4EWFZ4gz/Pct2FHrZeFdcYrMa7ybaHOwKVyNT2wOuQ6VKmjcM4RNpyaKquscRjQPeM1jJ2mRZcB1fhs61fQlITAppH8xVJx0t+tCTqGrA5zkRXzPQWod3t64Ma0tlAsPjgXFl2Rq+0399yAoflnYnLCVVie+7UrXToZ3WGWdOo6Gt8l92oeasBqkjhXV5Qj78+t6Dh9iKucqoIK0f+OcLmKhqvAcZ5wlLTK/M00bzVFDhxJh7rDb5mr2aOFXHTraoNVTWqJ/drarfLFiJR3oo8cDe5UIZbuchhfGy30mpvYI4lCJw8HFMB78twrZaQKB1qcA1pDDMLIjZLBYEBVfr6rP95+7TbhPdc1n9hISqXSBJO2Q22Urtpcdm1mqcpBUHiKrLxy0TQOcNS1X14PpgUEqZOQ6zDeBjeSu4yqMgZeDHz9SXud4O9w0BjEW9PQ27SEtHdDEW3LwIyy/xO6vN9GvIoKTSy1IAfPQbZYHNhihjG1EJ+ZlyOJJMABSbIm8ynggVRr87YU/oFScz4ZqmmFb16fKl4Jqho5aORsBr0rjnyNyeZOiNR2gPHtUug30vjsVJZUHkre7ujVyvFdsupB+sJzhJcFh91Kktly9KfIaVryXhCIIpMmoro4XdRxlfEIxy2lseQxcdjj5OPahJytz0vJfo58L335ydHa7NYqP+WbnqTWhiA4ZgA6ps5peiNtsKbFYhE+bJtzaIqaRHNys3XaapklceuMXelF4YDCgRPIAZVa61faZTOXQhN0clnSn0A21tu13uDA+fdmY8CEcvQZW4G+Y8sJ7LKsQgKq0tGjKQ+dxXxtT/K2q6f/qgTQZcM1plllDyOIgkCQV2VnxVqF0ghrpmh6WekRmHW9SZvBLZV0FvT4y1174VLW47XofD13cD8Hyjf7Bbuv7Loa+8o2ejRMPh9ON5CKhVwqF2wNwYbk5Xhnz+1C3YF98qpM5HSNBaV0DP2vW6osa6wJFwxwd30/AtbqPNhIj53BLjPm0N/X1dla5+EPkVoQAWIC9rwgVJGxnlond9HGDfBCkfXj4/vfgtDESb5tkspDSMdRwqNDVLd/UFAY8mRBaZqgaCQMfQRdx78NbUiSb70mpsT1v42M6t5Az6nfNLEFpZrCgbbNAQXwtu37o4xO4UCb5QCHRk0e9ZzP+PhjrBXeH3yylIQmcoB30XsMrUKfMQRQZaoN9Td4UD9WFOLAEzIioNrZvFl4GGOvDl3M63Fh8Z04t+wxTN9/G/Q2kihronE0yL9/X1dbtXhbQwZwQWYKekFS2MaSjfSMF2T8T1bNNCMUacmk80pUFlSCCn0ZHp9xC3IijuFw6TZodptc7sq4DHtK02RQCOVsX+NLzm8smSsyCKiy2ocnOVBVQHrPVqe+smeOdM4LweFX7ESXsa8gcehD6DltHiK7nitly45lmUvENQcl8CZWnTCVHUBl7loUk4qI03UgLRtoYVKQ9iHtoHSD1md83q3Ud60mI9NE9DlvBRIG34fwTn6Ad31NKPkKB04SDigqDSfJjVKGqXCgpTlQSh/frM3P0VZsNeJ6zERM/wfr7TKKPuRBYSlk9Paq2MKN7fVPRCRNqbeeUqDpHAgmdQYNucmwyeLzSmJWX2mvg9RRcrR9kWzd6tUpAWBCiRyG+Kriq4SgVqptN1lx/h8b8Mpgp9qEnXSCVWSY6CE4lrWlN3dE5+JLsT/udVl6Yy4sZNzlSXbq79Fzrkf3/FSE1oTjcMw+lBqLRREOZmzT2igghvwTprISH6RJeDbWhHNWGSB9A5+aImJaA/rgd0Oi8MpjKDrwHbUnB+Ns0MbUoffVKM9aTu+QO58DwVjpn4tq1SLsZpI2U2La/IkkJq5D+u6qGPhER15W+l2wOnABJUfhwCnEAd9l5Sk0OWUqCgcUDjSMAyVHf8fh5TeRRGk/Occ/hmM7PsbhFbc3qLIxpj+6TfgA3Sd9qoDdBnHs+Ap1JWOyoGCGPE6Qq9bwOSMw/icBXzr1oCXh96NMLddv1ZNqw1HdcApMcYxCRtDWu0d5cpgF3Z49mG2MRHLNzwQvnWC3VuvBo6Tz1EGqB6kHX0B0yXifPE5ggBqh60DGc4E9E0xJnCOr+9WBf1MtcnvWYQ+2Ja93gV0uVI1yFI6qks2WBd8O8t2rivdwFC1rsXEX7F4vKuV8UktwB/zgKGlxA+4A7240hkLjxwo3ZJ51uF1WT9j53TAcoWiHHVKvE9ksVdaTqgL3XScR2FUdj6cH8hZhiE6tswslU+HAqcQB+fL4VJqZMheFAwoHGsyBIytvkxnZ2EmyVHZsKRnfpCE4qk+D2zkZC2o0GkRGNp/OMce7DwqiKGF2BqItQ5c9ZMGOFXbkZ6oRFu3AkIkmMqQC9qzXYM2vbP3vhq9aRzV61yzBEd0IDDAtgEllFO7KFoc9CLM6lIKJlAsd30oyYMvS9idIWo0epmXkbcGKNUf/RcZybk8CLOFl0Osp6VXZNQg2J2PxsC4wawv8TjhYG4oPpu3Buuxf8eLmqwjG6skZmZlCC0fASuoMF/e8HxO7Xyyrm1mzm2C2r4RVTTB4TMIsxJ0zAijNgGMbGW4ZaGB9CLQ/ngTmP1Nz3NPIs97C/hURyKWAGxqS+CYNvg1Jg26QjdP7gp8nJjb8dFMkRly2Guu/GCr0e52SXBWqST3CUSulzd/zHvqd/TkiEk+jexmOrJ0f0fR+o3spNwp1tUkI3yG5nJNuN693pHNXQd8TlVpHusVaDJ7xBe0WNK/sVWInAABAAElEQVTLLlbPYIMuvb5xC4+W8KLgO3MlpT1zQAG87fnuK3NXOCBxgKQ98ApQwB9EEXRCKnOKHvlDW17OBknNQ+zz00z+itkyvCUpeQDA/6xmFXKOqaEnqW98Lwdm3tgBiz4m/c8qFXSOKlxUchcFiLAQTHRuf28zzEKutg8irRnQ0pZ6oTYF86LepKGyHNYs8NLqkOsxpOob8uH7F6XIyYEgKkNb8bXCZJbuxhfOxL6kZwMauA2JnobKsir0D5mE98buI6cKFOqYjMKKzdnCP2+kPg4lJXJDtyhtIo1HS6BXvm0/OfFqEaSipKwUeJB8RleHCv1dh5E2LGvKEG2kOrTY8G5PPouGX8UOeBD8T6L62g0OJs8ltCqoqfFQRxCVozHwsnTaQTkqvDgcXXWXC+xyNhvJ7VvxCPqSPi1MlQhJvgiqdc/QbWFe+1k8sdKyN8CtA+waIlOFf2wG0BxQJn7gXSiv5GdUzncx1OP4w4s9Bv1VVY3zICFfIBzHANpI1YKCAsyfPx/XXHONWAB4DuuHH37A0KFD0bVrV8/kFjn//PPPccEFFyi+iIm7CuBtkUdMaVThwInnAPvUrS7eQx+3GJLS1r11GUZbrmVZS2UfVpu5hEKLEqJSqM1yIO+IHut/iRR4yFyjwdAzS9BnmIOkgs4hn1X2NIHeagKObsA0uPpHLAwfhCN6kpBSjlNc60RKdgKzgghgbTFeStHUViM/5iNoCMCVBRHwIlJLbso8wNW+JAJm3JYfYhg9OHqKK0evrpV80horRBfYvRdHXluT/yP1RwCWpKBaCqgRF5yCK22PQv8JgV2dCjUzQuCI0EjY29VHWz1Rk0qEIaInTOVHySCO/BuTRxNPspnLXJdstDbgku3I3vYy8tM+onvKANJ9H0VBj3vgqhjgpNOwxxCWcHqAXCVZ80cZtPMI/FfbYR8SDMs9pAJEKjJNpYiICDzyyCPo2bMnxo8f72omLy8PV155JY4cOeJKa8mTe++9FxMmTFAALzHZ/y9US3JfaVvhgMKBFudAefZKpP08ifRwb0X6grNwaPnNAftk9YXY3lcJsKvWBostV9ZVTJnwYS0YClhVyTiBHKgo1mDF3BiS5GrAYJdp8++RKCtUYfBkp3TZaC+mH3k5SLKTND/UTj6WWarvqZvgORcydCPxI67dNRjvLzkX7/w1A98v+Ae05jh0CednSQ4EWMrrEL7BPBtxnhs0Roqk1ngJOocjZmnwhPjZAjBfnPIwni+bj/B/FyHoD9JjXlSFyJvzoc5oWUm674yOPyUkdgixV66uoVLrSQI7SNY4RzDM3fE6gV2ORCe/j66CLOWth8ISzlDAbh080vxaCu0HRVAVUgj1Kgrbsr4auhfy6qhRf5ZOpxPA9ssvv5QV/uqrrzBjxgzExMSIdN5h2r9/P/bu3SvzFcw7RDt27EBWVpasPl/wjpQnYOaynjsPHGijrMy9ePJsIDc3F9u3b5ftalVUVIjgHAcPHmzxnSnPsbT2uQJ4W5vjSn8KB1qYA6ayQziw5AohPWIpLVPp0d+Qn/6ZT89mChCR9vNkHFl5h/AXSo6dEN/nSuHu6Ojqe7Fjbn8UH1noU48T8na/j53fDsP2b/ri4N830Ac8wAfZb20l8Xg5cHQ36azytrYX7V6rxoDTrejcrwrVal8JqsFRSelRXrV8L8dUfYhepuXkq1dN/1i1AHhzy6XoYmBft179Mv6VY2BXgxw2uE/kWNe17IS8Khg/KEXEjbmIuCEX2m1yNQCWBl/Z4xnc1e9jnBU6B6HvO4Gz6K72cQt9XS4llbXfRi+0hmh0n/yFGB377FXrwmkXpi+6TfxYNmJ2S6Ym/efjpeqSPTId/eNt71Srr/2QwK7F/UzzuXpbNVQZ3go9jZs5qzN8++23QsVJqvnpp5/iuuuuE5e7du3CgAEDcOutt+Kiiy5Cnz59UFRUhMzMTPTv3x8PPvggzjzzTFx66aVSdTz66KPo0aMHrr32WgwfPhylpaVYvXo1zjrrLFcZVpm4+uqrXdd8YrVacd555+Hss88GS32Tk5Px66+/ijIsiT733HMxcuRI3H///bJ6p9KFotJwKt1NZS4KB4gDZVnLhGU5uzVykwNZm55CTA9yYE/SWyaO8LT7h9HuInTGxjTHtr0hSzuy4hayGv8JIbGDRbq1pgh7f5tFuogZrnLl2ctImnwbUsa/5UpTTlqWA2o1faAZ+bm/06JDjdaB3z4KQtYBDcpDbhEBJhgbsnSDfC2Q8dpIobcrCpMUl6W8BnspJpa/iljbQSpnwx9hDyHFtIZ03tzSU64fW1KAy9clIzciBunRhaIJnz/SeGoBMHtayKk+gA6GZJ+iEXfmQ1VKo5LA60slqLw1ApZRwT5l1YW01RxGChLlUgfOIuoCuY6vT8VWSLCSakIF+ctlrwmG7mcST50S97q65vep/0WbUJG3gUCtEeGJ42XFHeQSrZBcmdkpQEWdJD0Dtfz2V9ZK4Y7zdn+IuP43+ctW0kg9Bh6AVzCE1Rmq5c9aYxnFAJb/LVy4UIBNlqwyQJ08ebJo6u+//xbgk8Er04gRI7BkyRIUFxdj0KBBmDdvngDLDEhZCrt161Z88MEHOHToEDis8dNPPy1Aa1JSkqhf15+0tDSwmsXmzRQmm+iFF17AZ599JqTNfM2AOJ+ia7L++alKioT3VL2zyrzaLQdYHYENzryJ/YfuX3yZ6weNPTBoSLIkJ/8/doeX3yrqMUje+d1QGdjl+hxOmP2ImsoPy5tTrlqMA8l9a+AnXgGKclQC7HLHxdrO+CbyLeF+LEvbD6uN12JZ2B3OMdWCXY3DjEtLbkFH216CpuTqihD0lPLn6a+fzwN9FGN2Z+DZNVMwPrOL/7kx8OJ/NqeVPntaeGnnlaixVcrKa/eQgVy5G+xyJkcqDv7Cv/qDPYbGI9cCcLbnL03WU8tesD7u7h/HImPNAziy6g6s/bATGXs2zBCM9esjO5/lA3Z5xLt+GIWCPZ/wy1X/BJjf9VDOzjfahRFqPWzwm20fbCCDS3mWqpICr3QN7EZPXjrwFYPZL75wSvM/+eQTIZmVwhLffPPNiI+Px+OPPy4krPv27RPGjtOmTcO6deswcOBAPPHEE5gzh3Y3QkNF2tSpUwXY5R4fe+wxXH755YE798hhSfIDDzyAl156Cf/85z/x7rvvorravZgaN24cLdhoQenvR8WjnZP51OsWn8xTUcaucEDhAHMgOuU8ITHyxw1T2UGKFLVFZLGkVzjR91fQK81SdQzZW19A4f65Mr+ksmL0YxnQhZKsoHLRHBwIibBh4j+dbsC0ejt5abAhOtEkAK9n+zWk1vB32N34I/wR7DdMdGfR/WLqZlpFclz2vOAmDSFLi0ouZeWlEJeRyl27ayiiq+Vl3C3QmZq2g13rJwfe2H2DLJulZ44gqTV3FutQ+iM2Tqu+PMyVJfzu0rqu7BmnLqQroxVPWP89bf54ioJWIVSI+MgcSl98daNHUXxkAXZ+PxLbvu6DXT+eDt5JkQJTNLoxPxXUGj0sFCZZIV8OWO4lAzUifh4doeSFOlKNmveTAf3xQ6RLLrkEy5YtQ2FhIebOnStTNbjzzjuFpJUN21599VWcdtppQrDQtWtXHDhwAK+88oqQ9o4ZMwbbtm1DbGysTMeW9XaPHTsmxu7p1o2lwd60dOlSMFhmUHv77bcL8OtZhgH1qU7HfzdPdQ4p81M4cJJxgA3P+py7hEbtCybo19QFctm/robcEzWUig7MFUEpHDb/em12SzlZoPdoaHNKuWbgQEQHioh2bzbOuSUP02/OQ0UR+eC1+bnvdfSlEWoLviCzWh2OA0H9RU16anyeJhvpIXSsrvXh6ltdjo6pld0lq1BldRvS2HqS9MyrnoMArK17YKmaeaIR5f+KhmlCMExTg1H2QizsyYHL1zHtZslifXnWv5WTA+W5G+RJ9VyxkSm7KbMSIHWQCoOlkg2VavU86qnrma1SByF5yB1keBrpmSzObaZi6ILjfdKVBOJAkBqm71Ng/lc8LPd1hOltArux9DA2AzGQnDVrFu677z7hiqxTp06uVleuXClUGthrA7tz27Bhg1AtYD3fW265Rag+vPXWW0hNTcXhw4cxc+ZMrFixQoBnbuThhx8WIJoN4DhfArqLFi1y9SGdrFmzBlOmTME999wjdH9ZzaKlXSdKfbeVowJ428qdUMahcKAZOaDRh4HD/Ho7YrFZSmGs1cXl7mJ7+5bxC5SprN1qIvdI7/IZV5URf2j7XbSRVClOHPiQDagdXZD3KgpU4KDACOSpy+B7b+pjRaZuCOnqyhcxVgo/cVA/FvtjluPvRPqQauX53GaExYBCQ5VoXmut3wjOQVLjW9b0w8Fy5w4DS9IqCLwy2UNod4D+WbvpUPFQ3W1ZU/WouiEC1VdHwB7XPKBEDKIJfzgqmt/wvqSGsHfRLFpfkpeFnW+TKtFsUne4BzbhWsy3o6Nr7nMtRJ25jb+PXI/f+x6nP4XeE1+q7cS9+Oky9jVog3yNGGsLKgfigKOvAfZBtGthaF5oxEZqrM4gGatJzGYDMQbCDERZbWHixInCYwNLhY8ePSr0eHv37o2UlBRhvBYVFYXbbrtNGK2xH1/2uMDAmA3cuG6vXr3Qr18/oSYh9SEdWY2BjeTYRRpLjLldliK3J1KRgrLXGrs9TZ903Eg53NdBuC8P2Ck2P2z8gLVkBCXfnp0p3DePtbWJtz9Yx6ihfGru8YWHh6OyshKe2zXN3Ueg9qTtIzYyaG3i541576lj1dgxsNFL+sIZqCkhXV2yBNdQ9KYe076F3uiW8gjDtZ/GwUY6h87oT2xsY0NITF9UFu2hL4Dzw8thUOVGcPLRcH6fmX+RcVuiPKOBV2yAwQYVJ4L4/jbWSX5d42ytwBP+xpC9Pwhr50c1Wsobb96JsyqeQ40qlHzuUgAE/TisD70alxVdS8oONSQDplDDJOPlp0GCAt/03IFve+0mtKpGl7zrcST+XTI+Y5+4VIoN6gIQ+9N9cujvSDT2ECVYj1e73yy2k619nQaVAao2KDk6OrpZA0/U12nG2kfIuGyeD/Dld4KN0VjNQbxbbMhGQLjfhetJ0urcQpfa3vndCFJhIFdxMmJONxT4Ukhl0ttPPXcxyg/PRfbuz4VMXuy6RPVD51HP0kJ3kKz1lro4nsAT/J1rCWLJJxtlNSfx73Nz/GZJQVL4ufUm/l1iiMa/KZ7EgW3Y9VhYmFvFh/NZxYHLsku0QMSqFdwXj7+90YldHrc3bivzVTjQihxgx/WpMxahpnQ/SWdrYIjs6fLQIA2D9Xj7nb8GmRseR1XhDgKsSUid9AqMoVFY+lZCbTEVGdVMQHnOKvp4+zcoUtPHXTjRbyLglcajHI+PAwk9TDj7GhsWvM8Ll4Z/0Iq1XbAh+BLk6foS6A1DuYYWRQR8F4Q/iTDy2RttO4KB1d/CRkA2PTQO3/faiB0dDlL4YfLb7AhCasZTGKCajXTzH7Boi3Gsw1d09G+4ZaOob0uzPxfR0ni2jjA1LENqg1H4mf6G/AX44sDjqLSWIjooAU8NXYwgTR26w37aaMmk5NHPojJ/E71n6bJueIFo8/SUIozP1PSu/dvHm0lU11nkNvBjAYjdjdgJMIchpudsen8rUbjvC3eW60yL4Jh+MISniMhp1UW7kLntbVcun/CC12oK4FFDVlK5OBEcYCMxf2CXx8KCAH/EYZv9hW5uSDhtyf+vv3ZP9TQF8J7qd1iZX7vnQH16tQyMk0kCxFR08Hvk7vkSJZlL6eMrSZccKM34rU4+sr9ffViXOssoma3DAaHiQAIeq3Cz1DDQa1KHoUzdCQWabhRAgsEyEUU4K9MkokzbCVmOAdgRPAvjTDfhotNuhsYeTEoQGpyX/QP6ZM1GdGoG5kVMJTu1cALBJHXSmqAhF10cQlhIfJ0tir+sD2wl93cNoX1lG/FO+u1gkMxUYDqGxzZPxXPDlkLbhtRnIpKnCA8lMiOz2uAd9CJ5TNXu4+GEMxOHPowSescslZkeZfkVNJGOcDA6DXsYRYd+ojWI3BhJozciru8NiOxyjqiXve0VWX1xQe9xacZihHea5JunpCgcaEcckHan2tGUlakqHFA44I8Du386A5nr/4XD655GybGV/or4SXMCql5n/0o6pP6lEX4qKUktxIHKEg1+eUdLYJfvS8PALg/FaMvHMf0gN9iVxle77ckgWOMwYVfQBbCR2oNZ4wTFP8dfgLf6m/G6fjf0ZgemZkbhrEwD4stJAkVBI67v9bLUkux4RsJs2XWgi4/3PuACu1yGQwxXWkqwufD3QFVOSHpc/9sEOJX873LUNOeC0RPsOodm87NLwovOWJLkkhaubPysCsHu/phiul9Mf+WfbA5FHBI3SuTzH6GjSwsVT2K9ek2QrxGbZxnlXOFAe+CA/O1pDzNW5qhwQOGADwdyd71LkqcjYutULpHyKUqeHTpQogSmnB/0zPWP+RZUUlqdAxl7SPe7Dv1ZaUDDK7+kwBIr0ZWCS0wue15EXrOTRLYusqmCsC7iNFkRUt9FKRnKVdmH482l/8Bl6QNwyd5+eHX5WRiSGweNpjPm9HRusYdqo2DURuDm1LfQNXSArJ1AFzYCuN7EEmOLvcY7+YRes2eUQbP3o0PqHEQmTUJC/xuQMvFTuhcEfL2IPTFIxK7HzJXHCBvbCJRGkQqDr2pHDXmCOLj0GgoNPA760CR69WjxIaKzhaHnWT+ToaLb0woDb7XwwV37fpKUmV0Pxg24XepSOSocaLcckC8F2y0blIkrHGjfHKjMb7gbJX/GazVl+1Ga+Scikqa0b0aegNlXFGtgqlYjLNoqAlGwUNZXrigfWJphGi4svZcSbWSSFg6yXibprryM0xWZPLF7aQh6F5kpyppJVvjHNa8i1GIl+aQbNF+3aTRmhvdGpnYEBoeNw8OphYjURSNE13Bp49iOF+LHoy/L1CKqbeUYEDVB1n9buGCjsU7DHkNwcLAwNCo6toGwKfu6lqtvMLhlOrbpaRTs/ZzOyPcr6fr2vWAd8ne/B3NVFl27+Ws3F6Ps2F/iX+Lwf8EY1Z8WphVkhEbeNQgkexIbw42Zsxvb5p8PU1WBcBPYefQLyu6LJ5OU83bLAQXwtttbr0y8PXGgInc9cra/TB/KajKCuZxCDF8im77e2Em4FGtIIAqHML6RQyo2iis5skgBvDKutvzF1j/DcHiHkSSpDphrNBg1q0iA3lpMFXAAlZpYzI18E5PKX8Gf4XeTzFS+lc4Vye6fgCbfZ6F1S5vpevqnwtmHIwjwuqWUXDbcYkKa4WzSA45DlC0TfUyLSfVBh06leciICceW8gh8mBmBf/Xi0g2nmV3uxJ6yteTDdyVCtJFQk8Tyvv5fIlzvlmo2vLXWLWmMGYBQUjcoz1pBoNcJYFni23nsK/Quvo6C9E+F9FUaVRp5S+l/8TYc2/gEig58IyXLjjlbX0KfWX+TlwfeZfFP+uAYjLxsBYqKivwXaIep7SGoQju8rY2ecrsHvGwhySvy+khy88Huok6EJzcN6cw1ZJz1zaOx+ZLrErYIlc4b28bxlNdqtWCenwhXcDzfE8V3ft6aym97bWAIjqzEVHpsOQ4sudz1ca0q3AZLxX6kjHnKdWtSxjxG0qZPxXYpu04SkimbBYmD70TW1lcoBHEo3QMyQCLJk4PArQ/R1nPxoe9RXbQDQy5Z4ZNdX0JT51pfu6dy/qFtwTi0PYQWMSrQrRK0jlySzbjBil/fY0krg1W5hNZZiv+qYKLAEosi/o9KUXhfL91QLsHpZF4mgK5ZU4Igm9OVlsE7sAV181P4/6gN8rRLag85kT/j/Rgr+lStQIGe/Tw76TfCyI/2JC3VQEOSCtLRxMEXSG3BoAnFAwO+RmblHvLSUILkkL6kFuEd6MGjYhs7TTnjfQoocTfKs5dDTW4BEwY/gKgu0yk89wjX+ygNmSXB2VufR1j8aBQfJgM1Tw8PtYVU9E5bRQCJwIBXak85KhxQOCDnQLsHvMyOhgBYqQwfpXM5K/+/vfOAj6rK/vhvJjOT3itJIKF3kN4RXRQVsAu2dZcV+dvWsru23VV3Xcva1rZ213Wt6ypiQVFUUFQQAekdAiEhCem9zGRm/ufcybRkJo1M2pzLZ3jv3Xfr973MnHfeuef4/qir+rXPrCv6t/Puir67et7cf1vmzVrWg2uvJdOCb8kPK2lyB16IQXOexZ7PeLGLe8rf9yaSRvyG3BmlqxMB+nBMXnIMRzf8mYThY4jsMxEJo24iN2YGpI67lX5kS2mV+EpkbvwL/VA7X9GyF29Xd461ZQeQve2fSBl7g3uHctThBI5sJx+vJOy6pgC9FaX5Wlx5dzXe/BsvIvQk9LrnNRZ2WZ/L2l3O15BWl5Nd2GWfC0VB7na103NDUU/aXC69u9/tyEgmTwEaK46RT95w7ekwWL4njw5pqLM7/VAtev9vQ/6H+M+hO1WBWnMVeWT4Bqmhw7xX6MZn+EEubeaTjhFWF+/CwdWXeHUTVkmu/8L7zPAo7HIjKlqasqF3NCk7QkAItJKA3wu8rDlsTeAJO0929twV2kbW7rZlnPbxnuzWrnljR9dd0T9rlpl5VwSe4Ndg3G9XzJuvG7NvS987/3eKzReuEnLIxdiRTylaVqxaOOMqpKq26dVqVVkOxYpPcrtFkieS31VyZs5a9YKCAljrqlBXnkGLw8lOtLacBHC34m7Crv1M9s+PImbIEjrX+jWx3vxN2tuUbVMCLNx6SloybzDQ2qdzb8rDx0+7X19beXchuXEbLOpayLZX29jMgdxbBWmP4NWRLEg7U0K1Tgm7FcG7kZHi9Mpg1ZJ4TEMcrjmVWqMFa6Gvk3a3+cVqRyq240VyQ+aa/rJtPh6e+C2iDImu2T1uv5b+jg6sOpeeQbxL/joKChOR8guaG18jT9dXQ9rhcRhz6T6PC9x6HBQZsBDoRAKt/0XqxEFJV0JACLSNQB2t5LaoV6DOH0m2Gyw69F8Ex4yixtyFHPabGxgxsNlO6uvKKFLbOaSRuhj7V85DPi2oaU1is4e87U7BpzV1pEzbCYyYWdnIIwOZFJi0GDzedg+U5pM7Kp0n4cp5j3jslaRUj44e6AFmYsUXuOygBX/YnIA//pSEuzYlocxghok0ujWBWdCb3KNFsfZfqzFDrykkbwTn4IOjj3ns0p75SdYz9l3Hln35bi5c5TjuqTsZa5Y0K+zyvNj7wr6V5M9Y781sg7TvFOSltX+LPZWVjFsI+IKACLy+oCptCgEfE7CSlqgibz2t3F5LQk4VadIoKhOtCG+cLPXV6H/qi5RNQkxACP1Y2uzVB5z+us1nJ53h8MKZ39+CHf8djm1vDcKe1UtRXZKBXe+NpShN+5XWmBezqdepoUNVFxYLvU6n4AONNb62/um1euZntl3532cE4vsaMePiYtU+a3VjUkyYf8MJ6BocJQTo+Jp7696L0MsXlKTUUHMxUut+RppxIwItFF2P8+mzIex6DChNIb+9AdCRWweWpyedCFWPU4G16eTpwd3cwbV3Fn4/znoKrMX1ljy5IeN++f7u6YnNjDwmbSD0YWnqVHnWKtRxZEST93DmbNtrqjnhsSnJFAJCwDsBvzdp8I5GzgiB7kmABdSDX1zcELGJtHrGMgw79xvlo7Om2PlDySvCQ2LHqhXd/Aq0+MgKlFPEpfK878mv5xIShoIw8uJN2PPBNBKaWaixCRVFh1fgB/o0TbSEqXo/kiauxM6vMkgdFUuvqI8hOvRPJCO5CyRaQ1jT6pLT4QQS04246PZcj+3GJJsQnWRCYZaBHk5sGn4WjEfNKseObyLplbiFbIAb6TxIKj297BGkmHeSXS49HJGwGYhqWpT2MEp15AO2UbLZ+lqxum859NZkBJfSAq24v6pSLFLbenWvtLPkG/QPH+ue2XD0iz6/wvbir93OmSjgxaT4BW55PfEgPGkmLUb7mJi6PxQMX7haLWo7vvlvbvbxzc1RqwtFXWUWAsP6NldMzgkBIeBCoNG3ncsZ2RUCQqBbEtj/6dmoKdlLgm5pg80ucPjLxRgw51U1Xi1FPOMfRA4lOuiMdxrygmihTDEqTqwnlS4tOqPXxGwCsX/l2cpVmV3YbWnCbAucMHgwTl16HsaecxrGnXcZ+frsT++tnf5XWcxJn/FUS03J+U4gMPOSYtL8UhBgvQWBIWaMO6MMgydX4/xbc8EaYroR3EdBDz1xlqN0BUnQtVYpYZcLzKt4gATaaveyDUda1vRSjQ3JVVgXsxQHLW+h0HJRk6a5uJZKsucFb2lMzGm4YsB96nRIQKSy2/3LuM9oa/MQ4a1eT8jvO+UBJeyyBxQtv2mhiGj96O8kkEJy86LQVmmx2S6eAk8UH/4f2I1Z0aF3e8LUZYxCoFsQEA1vt7gMMggh0HoCRtLsNNYSsdbXWJWNU648irqKY/SbSKFdaQGMayrY87L76m8Seutri1TZxovaXOs13i/KWI54CoOakGbz1hC14HNs/+9It2IHSQAfeeGPbVq45taAHHQIAZaPTr3MZvbg2mAAPZ9woIoTR2xmMDprLWZWvoAU0w4SSeuaaGZ5GVts/RHk6d2vM7epJ4F3bwy9rifZOba2DqWB01ClmYHj1gcwWjuWbHhtPmi5rIXcnJ3ex+mq7KPMp7Am9z/kwaEGE+POwdWDH8MZKUswPfFCVNeXIVIfD0ODGQ7X72nJZlfPaz6D6CGUorFdcYTc931IJgsV5KN3GoKiBqsphSVMob8Vesxo9PzhPl+6mA1vYfhhl1PWj3cgOJpctZHPX0lCQAg0T0AE3ub5yFkh0O0IsAaXzRhcE/8Acj6nwPB+rqec+x69JtBLadIYtSXlbLpXRYQKjZ+IftMeRcGBNxsJ4GxmUY4CcqyfQKFWJXVPAuljapCxjf34WnFlydUkjGrpn7tpin3kQdZKGDWh9kO6Z6oRH7kUIUEUYEJrQV/L8zDUTkEheYewNoQotiIYNYZ1SLKeDTM9XA2OnIhlQ56EriHc7r8P3oH1+cvJ365NIN5Y8LEawW+GPEpBJiLVx9FhN9+pyP2BIqfdp+776L6/wIBZjyF3+xPq78RC82O3fyMpkhqHDo4ZcEGT2YQlTkbCyOtwYufT6u2MhWzm1ZsYt5Ierg1pidmOXwReN1ByIAQ8EhCB1yMWyRQC3ZdAn3F3UTQmChhAWl1bouAp0cPVx9uoORiFmUwaGidLfQUM4en0Q23TGDU+7+mYF7AZK46qT3n2l4jse5ZD82QvzwtreMGbpO5LIDK+Xi16y3gji/SuBtLs2jT2jUfMgnBxQJr6aMj+NNBSieRUdhFGnnfJdpsfl24uuBbL+m4nYdf1J0WLw3VheGryj0huFNyHA0t8m/e2W1ccaOLHgo9wUfrtiDT0nMAKlfmbcXjNrxwPfQWH3kMZRVczVuc5/kbNpkoc/HIRhpz1kdLkuk284aDP2N9RpMIzKHjLbuiCYlF67DPSBq8gzw0ciplMTMIHorroZ/eqdD3YREKSEBACLRNw/XZqubSUEAJCoMsJxA2+VL0ezdlss3WM7n8+ksf/sdlxFez7l0d/vFyJhVcKA4Ug+kG10mvVutI9qn0OQ9xSYk1zec639KNLEQiVW7SGGiT4BIYPaKm6nO8iAvxmvCiHo/mRn9xJRTB/oyevC+4CL8dfq9FE43DgDNSSR46rSq6kSGpa/Gn8WMTXBZGIXOsYvVZjhIHMIuo07va5Vqse5aZiEnhTHGV5h12N6ck7gV27az/Ji+CMZN7Qk9KRb5c5hF0eNz+I1lVkuk+B5mskUyOORBgSO8b9nMsRa2rt2trIvmcgceT1MNUWIjhqiPLKkvl9I4GX6tbx368kISAEWiQgAm+LiKSAEOh+BGL6nwf+tDaZyadus3a6tBjNWJ2NAXNeQWL6TFhqc7DhrRkeXqs27ZEXw6kAJWwaQT/sLPzywrWEEdc0LSw5XU6g3qjBN2/FoqZSS/cExVOrmYxF2jcpjJdzaBx2YmWfMfgs+hrccegNjDRuoisKHAhLQm5wCExGshEnAdeedDBhTM1qbArl1/VckhPphjUVSAuJtB3S/3vJGUhOTRm2Fz5ImmE91bK/pbAVqbNUIy6wh3keaLCrdUzS2w6Vs1rcPTR4K2rPZxvfINjsfNlMCKzNdbzZsZWqLmwqBNvry1YICAEnARF4nSxkTwj0GAIVud8hZ+ujZH9ZjWiyCUwa1Xwo37DEKeSs/kWaX7OrYpQnB7YzDIkbQUELQsnUwV3r5wmQlrTD6bOfU69iObRwUNRQJAxf6vXVrac2JK/zCHz6bALMFJLYSovNVNKGYU3y3Tgj6266OziP3M/p9bjrlMuw9qsHkGDKcYiwgypPYPHhaOji3O8LszUQF+SvxK7+A1GDkSQu1yAMm3BBJqmSx59JK+QsuHMvsK7Iilozx3F7GIOQQaGHf1RDCNNFI1wfiz+PXdHj7pvw5Nnkd/pT9bBnA+r5f7Op3KG99Vyi+Vw2c9CSfbTF7SGBPGQExTVfUc4KASGgCIjAKzeCEOhhBMqyv8bRddeStogWtlDihS78urTftIe9ziRr45/oXPPCLq8cd33dqnzzem3RecJSX0l2hhGIpwVqLCxL6r4EzKRgdBN21VA1yK9PR+7NDyKyKhNWCittSe2La/eTttV0wiHsctFA0uDPyT+ExUPewJ1Fl6BSGwUN5dUZp6O28CMsq9yOfSn30Z1mxaATVyLZNAuHC9fjnh3rkFF/OeXy/RGtej2EtzHSOhOh2mLMTrpU2e4GuNkAq2Ld/r9+Ux9B6dFP1JsN9n3NEQ6HzXsHxzY9hKqCTeSxLxx68pgy+Iz/klkRucdoZ4rqdxbydjxJtvH7bC2oRajkApAeNiUJASHQMgEReFtmJCWEQLcicOSbpTQep/DKNoOlmSvJhGAp+cS1vf50HXBl/iaKzJTvmuV1vzhjBWKT7lLn2Y0SC8EtJw0Of3UFaZur1KKckDjPQQVabkdKdAYBDj7BQq9rMtWRh4awIJj7OO+f36RSOQrbFlDvYutAlQLptXymYQyuT96G/sad0Jkice7eeaq5iOqxmHLwZUfTrAd+6ugFyNZ+0CDsOk4pLXA5TiUh+l2y5w2iICY98+eIH/JOufIIyiioi5n+BuLTTiMBNxGD571HrgJzlE2vIawfCbtt84biJOXcG0YuAPN2PqM8M+iDE5Ay4c/QUThiSUJACLRMgBz7SRICQqAnEWAfu40Ta47qyU7XU7LQCnFSP3k61STvxM6nKBLbJpXPbsdal6xK2OWyBz4/jyLAZbeumpTqdAIBJFMOnVpJmnjnAxM/PCX2r0UImR24JmtoKLTp/eisUzg2k+VtGZkf1GuCoNFHwVA+Dwv2kslCQ+JFZ6zdNWts7edOfAImXSnVqrIXcdkGKKGX7h4MDB/nkt8zd3mRWQwtIDWEOv1fG0KTERjRv0OEXTuVpNG/pQfLDyhk+AsquqI9X7ZCQAg0T0AE3ub5yFkh0O0I2ARR9z9ddivGK7k9JS1rgBp8nXo675rHi2r2fbUMefvfR9qMJ11PtWpfQyvvy7JWt6qsFOoaAsOnV2H4dHJHF2SLvjZ0SiW5JyvxOJiaXy2BlVyKWShShVETgjzDSKyOfQR/3pGA27ckYmE2RfVzEYi5ERZ11yVX4Nkx+TAl2O6FZM0DlGszweEyvELOglAkar/ErMTF4AhrkoSAEBACviTQM98h+ZKItC0EujmB9FnPYdd7FMGKA02QHR+7Dxs8b7myo/U0dCu9ZmU7wtaZJ1hVpLadn/2KHOHfgDGXHUAhBZbI3/0iRWU74an5Rnkk7nTAq9tGjcphBxMYNrUK/Gkx0eK1qnvvg6aiAhZSAH/5Ij1Umega2xe8NRJ2yQ+B0u5uSj2C1dMTsL/0Qmwv+RLhmh/J18ClOGx9nbq0YFBwIS5PXIvBES9hVPTsFochBYSAEBACJ0tABN6TJSj1hUAnE9AFRmLs5QfJbvczEkLqENHnVGUz6G0YhrA0OsV6t7Yl9t1bmvkJuSsj+18KCtCaxK7PgiKcdqCtqSNluj8Ba3g4MncEk+cO8rBQ7/p2wWbuYNKQ8Ey+eOsDLFg34WasO/VBss0FJscvwJaSTHyadxShZDs8RTMJEfoYFNfl4KNjJtwy4t/df/IyQiEgBHoFAddvrl4xIZmEEPAHAmyzG82+eAdcgsKDb2PX+5Ow493RyN78tybT51DDyRSdzZEc9rzN//lzIAlj5bFWC7v29osOvmXflW0vIKApLkLwaxS45Ot1ZIngvoCNp8c2uwHWEOgt0QghP7Hn7foP9q5OUDNfWwi8kncDijQPI8P8V3JJVoH82mOot9rMG/65dxn2lP7QCyjJFISAEOjuBETD292vkIxPCDRDIPP7m5QW1l6EhU3WsvKn4vi3KmJafwomETfkClo8MxCHv/4lFbVre10XKbGmzp5vb619Wwv1LamXEKipQeijf1eT6aspxebIBc64EpTLwi4vVLPdPXT/1Iap+GuHt2pxZFciKi1W3Ewnlw8qRUXku2S5G0LeGKodcFjw/eTY0xgRNcORJztCQAgIAV8QaF7F44sepU0hIAQcBDh6Ersz4vC81tZGbGqoXU9hfdnkwDWxVrbowBsoPvQeuSLLI3vcI9j3yRmoLtpJoUm/IwmFNHRWT9GeOkbY5bGwcC2pdxAIfudNNRGN1YpISy5Or/yHOtZp6aGqQdi1z9Qm9jqPLCYNQswBCK0PwEUZtehXFmo/6bY1ttJcxq2SHAgBISAE2khANLxtBCbFhUBHEagrP4L9qxZSZKkAFUSCo6aNXrSTFp+Fqy7qKrJw9Mf7KJJoCSL7nk2C5OVuXXPgCV64xvWaJlftrQXZP92ttL28YMi3SYPijA+Qt/0Jirg2DH2nPEDr6uRrxrfMfde6prycAks4H4b6mbZiUeXNODhwMX4smEo+eW02vJ5HYHNRlhv9AbYPuoaihJlIRK7BccudiNF8iGCNLYBCmbEA5cYiRBhiPTcjuUJACAiBDiAgGt4OgChNCIG2EmDPCns/Po08J1RS+N4ym9BKHheOfn+jaspYfQJb/zeVNLgrwWGEs3/6I46su96tGz2FFA30EGjCrVDDgbE6Bzpyhg+fO/e30pg/JY3ydhJ8l2PX8snKi4SnMUle9ydQP2gwrAHuPpxDjIXoPzMA+2NqlHOx5mbBWl+L1oR6XTm5NquhohpEalZjn/VLlFps/nsL67Jx60+TUEQL2SQJASEgBHxFQAReX5GVdoVAMwRqSvcjQB/hXoJMGirzNqi8fZ9fRlpfMj1wMXOoyF2HyhMb3eqkzXzG7djbQX1tIUqPfNhgztCcVs5bC23IZ7MJTmQ6Ya4rRi5peyX1TALGs85RbuZY6LXqdCT86lA390xoBvbD4vPLURLYdBFb45kWR5ApTUPSUECKIByGAUdxDI/R4jXyEc3GEXSfv3/EZitsLytbISAEhEBHEpB3jR1JU9oSAq0kEKAPpR/5psICLzbjVE+2vU2TtkmI4PLsL8lkIJDk4rqmxV1zXARnXUgy6quPu5716X7B3peRNPb35NIq0Kf9SOM+IEBCbuXfHoRuyyZoqqpgTkuHhT6chkUBh/QamLzcemzjWx6yE5lJL6ry9v+0qKMzJDzTv3rE0x694SBdcbmpyF5EtkSgqnArjq3/Pfm/LiK3g33I1/b79JAcJmyEgBBoJwHR8LYTnFQTAidDIIhMESJT51LEX6cQqNEa0HfqI6rZ0Lgx9PbX/VWyxVQOrueWlNDcBrtc6iOiz0y3Jnx/YKVFdO/4vhvpwWcE6idMgmn2HCXscgCKzZ9F4pOnEmGqdL9H1QAabH41dG+eSHmewgo773ErBawwI5xirqWSkBtDwi75eG5IE2LPsu/6/dZYlYODn1+AuvIMZfJUW3YQez+aQz6QPdnr+z0uASAEWkVABN5WYZJCQqDjCaTNfEr50dUFxSMwvD/6TnsMsYMWqY4GzHiU3vS6aoC1pCX9HYKjh7kNJLLvPLXgzS2z2QN6pRw5CDGDOteTgidtdrPDlJPdlsDaN2ORuSsY9Sb++bA5JLMPVkNmLDpyTJZevxGTh27HneffTaYQc9RpizWIdLlnYLd1A9WqwUDNZdBpbG8yQnVRmNPHfVGmvU1/3GZt/KP7tOm7gBenFh9+zz1fjoSAEGg1ATFpaDUqKSgEOp5A3yn3kyeD+5s0vHcVC74uwgQtaNOHpDQpFxiehkFnvodDqy9pcs5jBnl2iE4/V0Vmi0w9nX5A30dZ9tckXNtMKTzW6YDM6LSFHdCKNNHVBOqqNSg7oadhuNqB2+7T0CgzUobUYtI8MxkrDEZdXR0qyQx9i/EVKs33Fxsx2LS9ozWjyR9vmWM6VfWleO3gXbii76M48FMYjDVaJA+uRZ+BXuwlHDV7547ZVNFkYmzT79kjS5OikiEEhIAHAqLh9QBFsoRAVxKoLt4N/rCA4EikOcv5+QHHoetOWMIkDDj9dcpyFUJcS9C6I1ogpw9JwpCzP1HCLp+NTP0F+s14HLkUAzYr0ILjhjaYRrg37/WI7YsHzfuQ+kzwWkZO9BwCVnJDFqB3uS8bhk7PY5j760KMnlMBLVk5lOQFYOc34di7LgwJpAm20jI1u7Crx3GwHS87LXNNtTt3oPahN5D21T9RtyUD65fHYPf3/mmzGp1+Hv05u/88s31/RPJprshkXwgIgTYQEA1vG2BJUSHQGQQspiroAqNRT/53XZPFg9bHfl5DP46agCCygmDXT+4picIKx/Y/HwGBkSSMBLmdXPnJKUiuM1JoWJuwXE2r6INJDnEPIuBWpdUH/WY8QT/Qp9NceCW+pN5AICjMgsgEE4qyDTQd2z2j0ZIoG2yBzmATYI8f1GHNWxxkQt1IuNYajn+NKMTxcFMDAnJVRgKwlswa7GnW8TTcuG0smUNsU1kppp3YGPxLHNx8JlKH1iIynlTFfpTih15FHlt+oKA0X5Bf7iha4Fqv7PsbmzT5ERKZqhA4aQIi8J40QmlACHQsgeCYkaTc4dfGLon85xrC010y3HdD4k4hYTYY5kYCLy+ESxr5f7BQBLYTO59VkdeCo0cgcdT1+HTTUqRV8atmp2Y4hGQUfvF8skmjjwSbMUjQiZMl2f3qz7y4BB/RgjWdzqLWVUaRADzj4mI10HqjpkHY5UO6rxpupYsOReE/4wuUG+hgbTL54h2JKuMGKkO6X4rGdsu2qbTvarMOjK95F8ciZqCyJMDvBF6m1//UF8mf9S7y0lCggrgYQvtwtiQhIATaSUAE3naCk2pCwFcE2GXZiAUrsP29GaTdsWlHDaGpGHzWCo9d8gpuDk8cN/RKEmqfJiEkWPnwDYxIx9D5n9O+GTveGeZwX1Z67DPkk6uwWlLl8r/G2tzGxx47bSHTaiqjKHLnYdj8T1so2f7TW7duVXai9haGDBmCmJgYdVhYWIgtW7YgPZ0YDB1qLyLbDiDAmtyLbstDVSnZLpBMGxrpFFTrqrUwkLaXbXDtqZoE41Aq+vhQDUpJG/ynvUARXsNQTFJa3iBLMIw6EnzrnQ9eXNei0UNbW03tO9uyt+kv25DYUf4yVZmnEPA5ARF4fY5YOhACbScQHDkQk351EIWZ60hza0BY0gwKQdz0h78s6ysc+XYpGekaHOfTpj8OQ1gqQmLJtRml3G2Pk2BCfk/tvnrJJ6+lnl4T1xtgIhmj4U102wfZQo3aErZF3ouQmOEtlGz76fr6etx2222YMGGCo/KVV16pBF4WhO+55x6ceeaZeO655/DrX/8aF1xwgaOc7HQMAV6k1jgFVeVhcOU21NUHYl/gTPx3aCmOh5lgIkF34DGy7SWrhjql9Q3GTuwil2UUNltHN2HEC0BxoVtzgdYqxI8NRFSi0/TBrYAcCAEhIATaQEAE3jbAkqJCoDMJVBftITveIhjI6bwnYZcd0ithlwdFC1psb481pL19BUPO+sAx1LqKo/Tm2N0Gkm19ByUuRNHhdz1qeR2VT3LHbCw5yRY8Vz969ChSU1Px8MMPNynw5JNP4v7778fYsWOxaNEiLF26FPPnz4fBwHanknxFIGDXToS99TomBOjJXyxw89RklHCghAbF7aHqpm8TzIhGFQ1o28VXY/JLD8Oq15P5DS2Mo/s549w7MW66CLu+ul7SrhDwNwIi8PrbFZf59ggCGd/9AYWH2YTBqlwRhfeZjYG/eN1t7LXklD7AEE2O6V2FSitqS+mdsUsKjh5OrsdW04I2dxdPGjMtVqPXxnZh2G672xEmDfbuq/K3IDxpuv2ww7YHDx5UAu/nn3+uzBrOOOMMhISEgDW/2dnZGDPGpt1OTExU+cePH0f//v1V/1lZWXjmGWdIZq47Y8aMDhubjqKTsXBt4QgNXZACKAywRqNBZGRkp/VuLS2F9c3/qP409UYUGcJRQ28m7MIun2BxV8eutbTuPzus8Q1JT4fmsSeh2boFWrqGOGU8BkVRKLd2JOZvpeAXnTl/12Ha+w8MtLlgcz3XGfvcf2dff9d52e8/PT28tCWZzU3fGLSlvpQVAi0RcP/maam0nBcCQsDnBEqPfY78/W+69VN5YgOKDr1LgSkWO/L1wfFkptDUfy7763RNCSP+D/l7XoCZ88mBPdv4sv/eqsKfHcIul+9IQdfe/4ld/0REyhwyrxhtz+qQ7YEDB7B//36lxT1y5AhefvllvPbaazAajQgNDVU/+PaOWPApLi52CLy1tbXIyMiwn0ZFRQUtwOq4r0L7D75W29QExdGpj3dY4OnIObU0XMvxbNQTd1D4YXvSk+14LdnuuiZa5obFmRvwbto0R7aeMMWG6KEPJQ8ic0535HvbKa7NRVldIeKCUxBusNlsu5bluXf2/F37t193HkNXJHv/nXn9XefZ3v75IUWSEPAlgY77lvflKKVtIeBHBNgdUeNkpYAR5TnfuAm8geS1IW7oL5G/m+wf7YlsdQed+T/7kdpqyDHq6EU7ULD/dZiqjiOQ7INjBy7C3o/nupXzxQHpp2ml+baTFniff/55pb3lMbKZwjXXXKM+rNXlxEEOWNvL2trGmiLW+gYFOd2xDR48GO+//76qx/+VlZWhqKjIcXyyOyxws+BtMtndcJ1si22rHxcXp/rmeXVW0pJ2LqSeHqYaOow3VmBcaSa+ixsKa4PgZzCbMLYsC9cfWeMm8JpIER5UU4qi2pZH+0X2K1hx7DGK0BZIQS2KcfOIVzEu9gy3irxwkbXrpaR17ooUHEyLRkl44werrkjR0dFK4OeHvK5IrNnmh77q6raFQea/Uf7bkSQEfEWg61QQvpqRtCsEejiBgKBY0sI2eh1KgqyeQhA3Tsnj7kS/6f9AWOI00qSeTra7KxAaN65xMXXMvj2Tx5NPXhJ2OfGxzxMJ6tCcvO0sa2ntH/4xZRMFFnLtqV+/fsjNzUVsbCwpGavczvEPf3Jysr2obH1AwJKWjvrBQyiMsFOH8sLPryGltgR6erMQSoskz8rbibc3Po8QMqUJI81vJBWNoM+XU0ECbMuD2lXyLd458lfUmquUsMs1ntrzGxyv2t9yZSkhBISA3xNwfjv5PQrvAMy04KfSVEoaDKeWyHtpOSMETo5AwvBrULjv32SCQDZtarEZSQO0rS0/RD45i6Ajgdg1xQy4EPxhLXBdRSaMVblojc/OOBJ4T+x5ibS+2a7Ndfh+UNTgk27z8ssvd2vjo48+Qn5+Pu666y6lSVqzZg1++9vfqtf4U6ZMwccff4xLLrkE69atA2u8+CPJtwRqr7wK+rVfI5AiplmDg2CeOw/fvP0MaqrroCWNZzDdn7woLWTCKXhkBN3S9AZ7PJnphjQye/A2yjW5bzQ5pSU/D5sKP0NKqLieawJHMoSAEHAjIAKvG46mB3k1R/CPXVeh3FSgNAvLhj6F6QkXNi0oOUKggwiwH94JV+zE4W9vRNHRVeSBwWaTW5m3Hrven4hhC1aTI3p3IdJIpgqHvrwM5rpSmE3liOx3DvrPfk6NqKpgC2pK90MfnEDhhOeSQJyDY+v/gBpyG2Y2dsJrb16y38GJhVn20LBkyRIUFBQoU4bx48erXq6//nrcfvvtWLFiBdiekF2USfI9AbJYwMaKi5BnvgKWcmACmSmk/fYWhD34N1gDKQogRfMzjT0F9eddgPYsEdRrmyoctJoA6BoHafH9VKUHISAEeiABEXibuWhVpNW9c/NstxKvHrgNEfpYjIo+1S1fDoRARxJg37txAy9E0ZGV1KzrYg4rjm28C0PmOW1Q2afunhXuIkQZBZfI2/lPEoBLaLHb22qFvLW+GoFRw1HXyItDR47bU1uBkemesk8qLyIiAg888IDS7rKJg+uK+LS0NLz77rvKhjOqnSv9T2pwflr5k2cSlUsxFmw5bVlF6tuzgX4PPAwt2UhbDXpYo9qvaT8n9VpsJm2u2UqSdUOqtxpxWtKV9kPZCgEhIAS8EhAbXq9ogM1Fn8GgDXYrwV+wXxx/xS1PDoSALwiYaotVuODGbdfXFrplsQcHrT7CLY8PCva9Rp9/kYK4hiwibAtIOlvYTRx9i9IsNxlcB2XwojVXYde1WRF2XWn4dj8/06A0uHZhl3tjf7q714WDVjDBkpDgJuwerATW0G28j7atTWlho3DH6HdVcdb2poQMwcMT1yFU3z73Za3tV8oJASHQOwiIhreZ62ihiFRujiQbylrItZMkIeBrAmHxp5AQ4X6vaSiiWmi8M7oYj0FDfk09BaawUHCJrk06cn/Wr2uHIL13CgEVLEJPPjlIyHVNprqmOpVnjwJvkdl4ABUtJ2uXi/sAdw9xreV9f0jkJLw2K8t7ATkjBISAEPBCoOm3kZeC/pjN7m6MlqauVc5KXeaPOGTOnUwgNHYkksfdpXrVBASRFjeMQgb3Q79pj7qNJDRhCgICm2q5dIYI5XPXrXCnHphhqs7r1B6ls64hEJtidLgks49AG2BFfD+nJw3O/568v72USe566TmOhV1OH+SSp4YC2778LwSEgBDwFQEReJshG2VIxH3jvlAlQnSRCAwIwWUD7sXo6DnN1JJTQqBjCJQc+xJsrhCefBo4eETqpL/SgrUvUF9TgBO7nkPujqfIc0MGmT0YHK7GXHs2kUcHPsdaYVty1765lvXNvhW64ETfNC2tdisC+kAr5lxpM7UxBFmgD7QgIa0OU88vdRvnWg/ujvk92tci8LpxkgMhIAQ6noCYNLTAtF/YCLw04yBqUIrkuDRa7W7qspChLQxVTvciAse3PY2szQ86ZlSRs5YCSrxPHhaysW/lWTZTBzJ3OLHjCQw4/XWKmrbVUdaxQ1HYogdepTw31BTvpGwt6uuKSGDOdxTx6Q6toNcHtX+Rkk/HJo13OIHwGDPOuyWPviPD6M2CGWGxTd+OhdMvjp6eu0wu6zDZK1mEvsOHIw0KASEgBNwIiIbXDYfnAwMtkEgMSUdEYIznApIrBDqQgKn6hJuwa2/62A+3UHS00yg6MNnmckjhBvvezO9vRoChqUkD16su3IaYgRcp37ysDTbXVdib8/2WxudtXL7vXHroCgI6gxWJ6WbEJrvbntvH8uu+tiAT9h8efufAJW/qby8hWyEgBISAbwjYv3d807q02uMIFNQew6HyLSg3enj32ONm0zMHbKotgM7Dw5WJvDM4zRNc5kZSQ23ZAZcM5y4LvIfJP6+FfPPCUkfBKTp3IVto/HjnYGTP7wlEkSZ3NUVWGxYG9KFgghMigVVTKDS0OQdP7l6Ce38+G+9k3EdvMFxUwH5PTQAIASHQEQTEpKEjKPaSNj7MfAKfH3+RwnwaKHRnCX47/GVMjCdHmm1IAtr0cAAAJsJJREFUHPbzq5x/o6q+DCOjZoq/4jawsxc1hNCydbBlo3tiTwxaXSjqzbVuJ9jXrtGbgKAitbkVlwMh0KUE2HzhHRdHIzX1FbhuwxRa9KYlf9EW5FQfxI7iNXhgwtfgwBKShIAQEAIdQUA0vB1BsRe0weE5Pzz2j4Y49SVqRs/svQZZVXtbNbtKUwl2l6zD73+ago+OPYVV2S/gsV1X4v2jD7eqvhRyEuDQwQNmP2XLICEXtOhMExCMobRgLW3Gk86CDXvJ4/9Mi8Pim+R3dIbWHAD+GIwU8aoVCriIvmd19BCkvV5I4F8Hfu8Qdnl6JmsdSury8N0Jm8/dXjhlmZIQEAJdQEA0vF0AvTt2uS7vnSbDCtDo8HPhakwaeFqTc64Z+8s24uk9S1FbX0n2eA2+hhoKfHn8VUyIPQv9w8e6VvG6vyF/BX4s+Bih5BXjsgH3IIL++WOKSZuHMRetxYn9y+n1rgXhfWYhgNyShfeZgeHnriUPDU+QCW8dYgdfhojkOQiJOwWHVl9CqFohiTqAkjDNAnVLWmBqctre81AeXASDOQiZ8btRbCBfUi0kfVAMeZnYiLBEemctSQh4IVBiPKE0u66nTRTgp5IiXUoSAkJACHQUARF4O4pkD2+ncUQ5no4WtMpeS4Z2zaTC2mw8tONiryU09Eoyt/pwqwTefx34A74/8Z768dOQzmd9/nI8feomRGtSvLZvP2G01OJ41X4E6yKQFNw7VsCExoxAUNQuZP/0ZwoP/I4KE9xv+hOIGXAB0mc+bZ+62oYlTCRBeA2OfP9bGCsyKbpZPOpokZq3pAkIRVTaORQooAbl2V/bFsJ5KRxWHQ2tVYuYqiRbX7XRKAknIUXb1OzCtYmig2+DPxGpZ2DAnJddT8m+EHAQGBU9G5mVu8BRLO2JwwcPDB9nP5StEBACQuCkCYhJw0kj7B0NnNuPVvqTRtc18avF0/pc6ZrVZH978dcI1IY0ybdn1JorEBWYiBM1R/DMnmtw04/jcfummWB7Ydd0rHKPeoXJNnycrA2ayqe2XeNazON+fk0m7tw0m0worsCdm2fj4R2Le8Wil8qCbcgkAdZsLFPCLk/+2PpbUVWwxY2D2VSF8pxvYaQgD0PP/gRjFu9E0uibyd7X+3Wxkq11Scb7KMtc2aywyx2F1UaRwOu0pexbMIweR2ilnJU+rUgVx9ei6PB7rSgpRfyRwHn9bkG4Psb2gK0JUg/Z56Rch2FR0/wRh8xZCAgBHxFwl3B81Ik02/0JpIWNxJ/GrsCD2y+kMLUBSA0ZihuHv0gaU1pO3UxiDXBzC0v6h41FjKEP/rjlF2CtDady2rC98Mqsf+LRST8gOjAJxcZcMmOIosVu7q8xi2pzmumdvBOQ4Hb75pluZfaVbcDyzEdwcfodbvk97SB/f1MzE55DccYKR3jhkmOrcOyHW0n4tKrrZjFXY/TiXbDU11DJlp5nW2f+YNLXwaIxoyykENlxB0jOtWBo1mTsT/0JFvK32lKykslEefZXFByDTS4kCQF3ArqcPDxjeBG7A3Ygs48Z6WFjwCGEJQkBISAEOpJApwm8dXV12Lx5M/0oazBp0iTo9Sfnaby59vbv34/MzEyMHz8ecXFxHcmrV7c1IPwUvDLT+2twT5OfEn8ePsh8TAmedu2sa7ljVbtx79azHcKu6zl+hcmL3B4hoTcpeECTMrxqO8LQ/PVj++EgTRhqrZWOplk7zKYRngReC/mG5VXgWtJmJ4cMctTpjjuaADYnYaHVxXSAPTWofKBg/xs4vulux9Bt4qsGGd8sVSGJLfVNHf87CrdhpygiB4cTt+NEzFGHGUNZWFtCY2k9ullrwxCkaC8lYPjqCxjWfUu3uRYTa2txyuChqLlahN1eerllWkKgSwm0pALqkMHV1NTg17/+NdauXYs333wTt99++0m9cm6uvSeeeAKPPvootm7diquvvhrHjh3rkDlII54JBJL3gAcnrFFa2sYmEVzDTNo9tq/1liwknC7PfFTZ3V6cfmdDMY16rRlEdqb3T/vMW1WVz8Krq7BrL1xpKsb/Mh6EhRZ82RN7krjn53lkc3wJ7t5yJq5bPxItaZDtddkkg+vdtmkGntp9NeotNm21/bwvtkkjltCaMuczaeyJWehzbD76RFwDE4UXdhV2nf1bUXXiRxxcvZiynHN3nm/fXl5chkPYbXsLFiRP+FPbq0mNXk0g4NBBBH71JTRGIzQk7HIKOHwQ+o0bevW8ZXJCQAh0DYGAv1DydddvvfWW0rSyoDt//nysWLEC8fHxSE1NVV2ztpYFU4PBoD6u41m9ejW++uorTJw40ZHtrb36+nq8/PLLeOONNzBr1iylTV63bp3ad1RutFNLX7Rcr6Wk0+kQHByMqqqqkxLWW+rH23num8fa2Yk18mFhpEFthhObNcxLuUaZLuwrW++2+ITHy359LSqekufRxxiSMC3hArCGeXjUdAozGks+fGfh2qFPIzYsCSaTySPzCmMxHt11ucdGLdTjoYot5CLtCcxIuFiZXdzw4ygypyiCiQRw1kbXk43y6pxXyBo1ALFBycqGcAMtlHt27/X4OOtpbM3/EjOSL8LBoq34889zUVSXjWryL8zBOdirxenJVzWxe/Y4GJfMg+WbwW6Y1uS+ofpjUxJObMN8vPoAggPCYaCHCL7fOCxvcOJstfBr4vdvIjn7XMSVnIbQrzUwGSqQr/lIeWpwad6525LnBWfJTtljrXRY4lSvffEbn6AgcnfWBYm/f/ge66jE32Nms7nLQpCHhISovnleXZH4WnLgCGbQXNL/tBEBR4+wNbgjadiftIX+Nse5OOp1nG3dDn9Xcv9d8X3JI7S/vWzN70rrZtS2Ujx//t5mxVBXJP7u0pLGvq1/U/bf2K4Ys/TpHwSc6iMfzvfQoUM488wzHT2wqcGePXswdepUbNmyBX/9618xYMAAsCnCjTfeqIRie2H+0mYh0zV5a6+6uhpjxoxRf2xcnvv59NNPXati06ZN+Nvf/ubIu+GGG3D66ac7jr3t8BcIp5iYrgkvHBAQ0KXmGeHh4Urw9caH82eGnIvXDt3lVoTDMs9N+xU+O/KiW77rwWnplzrmFhe3ADMGLnCc5nkHBgZ6FHi/2vuKo5ynHfvCt9s2z1DCpacynLfi2GPqQz666Mhp17q94Fss+qypSQXbIpcY87Cm8FVcMfxurM58DTvy12Fb4dfQwYDUiKEUHMK24nxS0tmYlXoxEkP7YWfBOvx9xyKH6cbhip9RaDmKMmMBfsxZSQ8GOgr4UYpHZq3ByPDp6kcrNyEC36XMwpenPYLT9y3A3P3nqmlErghDxKzBKA3ZrI67+38Fe1/CqNOdf3eNx2skLZ8kPyNAghlJh+R41/mwYaXvWQs9YEsSAkJACHQ0gU4RePPy8hAR4fSnyvvZ2dlqLqyNveeee5QGt7i4WAm855xzDj744AMlAGdlZaG8vBwPPvggoqKicP3118NbeywYRUZSrMqGxP0UFbmHyOU8FortiY9b8yTKghc/gbamrL3tjtyywN1VfbPGgrU1LWlsovRJuHfKh7jnx/kI00crc4Zx8b/ANSMfxRVD7sbHGc/i2+PvIq86A0EBYUq8nNrnXMxJuazZuZXVFGF/ySaqE4rhMU4tYXOL5Rqzb07D7CzrFHZteY2PnSV577/7H8IHB58kkw13TUpRwXGb3Ezy8+7i7/HaHu+v8z86/Ix7o3R0+3en47Ihf0ZK+CD84+erYdGRpiyZ2kreiqOxh7B0/e9g0htpfzQiazYRR1cdGSnIyIXY6MNzEWYMQUloEQ70+w61WivqyIAplJrSNSrfZAA+yDDTIrrm7l8JJesD6N28SeO06dB/+w3ZPZmhIa0u/7Wxhrdu4fndfOQyPCEgBHoigU4ReFlYdBWW+FUPv3YpKSnBjh078Pnnn+OLL75Q/CoqKrB7924MHjwY0dHR6vUQC7isDba/8vTWnrd81wszdOhQ3H///Y4sHkNZWZnj2NsO980CNY/PQl/OnZ2YRWvG2dHjYkGbX5Gy9rw1rwj76cfgmanblV9NDh7BASfs4z476Xrw53jVAeTVHEakIQGDIiY4znsae7k2D39cP480vBa1MI7b/MfkjdBR9LHxEfPxOu7xVK3T8hoLu46O3WVQR3Zrd9454LxHXet8NfwjnHbgHDx01m2oCCrEb8jKxdBILg+g4zATmQdZdIiujENU3nTcN/k76CjfSN7FFudoEGUmO+mG2zixtC8iq+Ng1NUgO36/TdHt2mkH7BvC+jV7nfkek+RnBAyBqLrrzwh+4zVoCwthIYVG7cWLQD8OfgZCpisEhEBnEOgUgZc9JbD21p54v2/fvuqQtaYLFy5U2lPOOP/885GSkqK0uXzM5gwsyLqaHXhrj7W7LEDbE/fTp08f+6FsO4kA+9RkZ/LeUkroEPCnpVRdX46bNriv2OaFZ/859CdcPeRRRJHA/MfRy/HgzotaaqpnnlcqL/eh68x6PHD272E0FCGGNGM5gRqkkdDr0PJSnQDymcsR0TgZaH9AWRxSqiKQFV6O0ccnoN/ui8iC2YIPxr+ORUcSMKQgiYThAOV6rH/+aPwwYgUsWpv9pcaiPYnFavaxazBswWr7gWyFgJMAvT2q+U3LvradFWRPCAgBIdA+Ap3ipYEXkK1atUppCAvpSX79+vUYN26c0uCOHDlSmSjwNjExUZkuuGqDeWHb8OHD3WbnrT12d7Zr1y6wGQRrkT/55BNMnjzZra4c9BwCbOMaqo9yGzCbJmwpWuXIGxI1Gc9N3Y0lgx7GsiFP4ukp2xCk9W4DSA6yaIkaqTl7QvKgJa4ILkMMCnHFCSsWFGiRXquFkcrV0wvhOo0VEVXxmEy2vo1ToFmnbIDv//gFzDx8BmYfnofHVt6PyfkpStjl8hxcggXh/nmjHdUN9YGYtPl2pB38NeJzT3Pkqx0WyBuSwUgLZSzeuFpxYNV59qKyFQJCQAgIASHQ6QQ6RcM7d+5c/PDDD7jsssvUgrJLL70U/fv3V5P9zW9+g7/85S94++23lanA4sWLERsb6wDBgnHj1Fx7y5Ytw9KlS9XisrS0NFx++eWNq8txDyGgIrjxqu3GqVFeiD4Cp/ZxXuc/jf0Ad291LpJ0rZ4aaguokVW1B8uPPo6cGnqF3yixfTEHtHBdwNaoSJcdRtH6nvlFTkn4cLAVa2Ns9o9ssnB1If1dNYqAFlpvQF6QEY+vvRf7Endg7eDPUGSYhlNzBmNStZ4eAUyo0CZgY8gvUauhxYnGw8gvrqWFfnVYuC0WBlMI+ucHkhZZi+JjP2H7lJts83cOA0Z9DQLIhMLMfZPgnZW/if6eo9Anbj4MugOoLd2LyhObyFODu8a+y0BKx0JACAgBIeBXBDpF4GWzBfaMwPavbLvLx/Y0evRoLF++XNnzsp1qa1Jz7S1YsADz5s0De3dgd1qSei6BIZGTkRYxCgdowRoHqeDELs6uHvJ4s5PqGzYcN414hUIZL1Pux1wLT4o7BwnBaeozIe5slBsL8VXOa+RyLAd9Q4dhbvISsg/W44YNoyjqm7ttN3ucYCG8ot5pnuPadmfsj6ykVew0Ky2Jn8cDrfg0nnSyDYInrUvDM2M3YmxhIpk0BKCWoqDV6Ez44/SvEWyKwoej38K/Zz6Jgdm3YWj2/6FGSxHZSNtdpQ3H8khmysKqhvTHgymEHTD6xAmEUKQ7F7kWoaWjEZM7F8V9vlLTNdNYKunPWWex4kSICSYSdlNrNIiPWobjhRtI8N2K1PgpCNTvQkX+TyLwdsZNIn0IASEgBIRAEwJOybPJqY7PYNdW3lJrhV3X+t7aY68Cdl+IruVlv+cR+OvUT3Dv+oXIqNiOEPJRu6j/nzAh7qwWJzI+dh4W9L0Bn2Q9o4RUDUUoiwtMxcJ+DdrJhhY4ktuF6X9o0t68lGUU+vgZR9AM1m4GaPS4ddR/cN+2hU3Ku2awazIloLOkyApqV4nRtWA79uupLV5rxrZIX8Q6hV1uigVfHZ1dMvdDpFfQgk+zFhmRJTAFWKDXFSthl8sNyrmN7HyDccwwCSW6T7Em7FYaY1PrpuTKiiZDDzQHI7h8KEACbxmplJcn0GJCqsqL4bQk9OppvnW0au7SnJ3QFNXQYsNgFJU9jGTS9NYUbefuJQkBISAEhIAQ6HQCnSrwdvrspMMeTyBAE4C7TnnPzctHayd1UfrtGBd7Bo5W7kKkPo4E5bNbWxXnkmBs1Zuw4tCT0GsCMTpmDq4d9k+wlvcfk3/COxn3kdlDJabEn4v8mmMU5GIz+gQPwlmpyxAfZFuQyVrj7UVf4Y3D95Lc23JwE6+DcxGad4ZbMaFco/wSN7JcUNVZtg4iiZgFXddk0hlJICUbXVqMZtU4x7Iq4i8klDe4a3CpEECeSPrSGxn2ZexYEEfn2cZ3e2w+wkmT++8U93q0vo2MIGzpw8Q6zMkuI6E7BOWaGPaqRiGRDQ1nZSMEhIAQEAJCoHMJiMDbubylt04mMCB8HEVwG9euXq8e/RCuGnZfE3daMYF9cMPw51tsMzYwWUVj44hsRnMNlq0fQsIjeT1QOtoWq9sKsLBrT7RfTZrU15OAy0+QppbMjLdFuJvs1nlbN0Zt2JvKin8NA3NvcwqyHrS7qeT7ulZrIeHZXfPLwm9oZT+8NtZCZgy0WM79tH2kJN7rURq2G/FlfVAUTb6XyXfakBHXOs7LjhAQAkJACAiBziTg5eeqM4cgfQmB3k+AwwW/OvMo5qdejylx5+HCfre1btKssuUPp4ZtpcGCl/pasLWRsOuQaG2lbf/bpVw6skee25v2RxRGrHGUsuc7MmiHdMgwB5Ahb6PEAu/pBxbQ4rRAr8IuV6nXmki7GwBN8CpsSn8cZSSIh8SOatSaHAqB9hHYUbwWr5N7wvePPow6epiUJASEgBBoiYBoeFsiJOeFQAcR4MhwF/e/w9Hagn434ovsl/Hu0fsdebxzbt+bMC35XNy1ca5bfuMDXpgWaSTvCSTU1gTUUyAK28I+t3J2YZkzSWNLNg3q9I8jzkJwXSpCqgdh0oHlZHtL0rNLygkLJ4HV89eDliK5sbsyo95uwOBSkXbZiUZofSiq4+/Dd0nfqZO70oZhsXsxORIC7SLw34z78cXxl+mRjJZckl39yqx/4skpW5Rf7nY1KJWEgBDwCwKi4fWLyyyT7I4EtGRKcHbf/8NTU37GqUmXY2r8+fj9qDdoEd1t6B8xFs/P3omZiZdgYuw5uDDtNpzRZ4nbNIwUcrgwpA75odWehV230g0HLPTSx1I3E6MyXkBx1AZ8O2aqOmkhQwSbgzNahEaeVB6a+4HKd9UAsw0vux+r1VNQi9oE23lXLTLtk6MHVOursCWRhF06DqpOx+KB3sMrN4xMNkKgRQLHKvfg8+MvKmGXC5utJnrnoMWrB5ouPG2xMSkgBISAXxHwrMLxKwQyWSHQtQQiDfFYMvjhJoOICeqDpUP+4chffvQRx759hwwPaLHcQORSqGZP9sG2BWdsoECCbkMK1IRh8aizMfvUsRi0Yj+OF+TjaMKLOJjyEA5q/4bU4oWYfjwU0UX34Mpf9cOb//kKVboqMnEwoSTYiBvOJ+8V5otRinNwfsnF2BscjfKgEtLscm8k4ZJ3CD1pgeNMZgw+sAkV0x9oNvKefVyyFQItESiozSRvLZGoNjtdBvK9nVN9sKWqcl4ICAE/JyACr5/fADL9nkMgltyqsR/gOku1Y9BajY48UczDrUmXoaA2C3FBqcitPoyfCj9BMAXQmN/3RvAiOw7JXGbMRySFYw7TRzfUt2LeYi0Olefg/u03KG1sXyxBUcIleC3mfhKlgxFfvRMXLZmP6YeWIKj2FGSEzMJZ2ydi5di/whiYjQ/192B6jg5TchfBYElGfORvER7yJvIxENU5u6FPOYzfjH/aMV7ZEQInQyCGFoJytEXXxI9ZIbpI1yzZFwJCQAg0ISACbxMkkiEEuieBU0mo/STrKRjryL8t/WN3aSZrHS4m92tsH5wQnK4GnhjcH6fEznWbBAu5TkHX7RQGRYzHs1N3YiMJyRarGbP7XYiU6DQUFBSgIMuAnz5dT/1QiDdyyTaG4hiHJujw96134pTtJ2Am+4Ud8fHYnpgAaxBQUvmy+ujIIe+QyZUYdWqIe2dyJAROgkD/8LGYnnAh1uS+rloh79jq3v/DqLdOolWpKgSEgD8QEIHXH66yzLFXENCQcPn45I1Ylf0SMit3IiEoDbzwjYXdk02h+iic3ueXqpnwYGeAmPi+Rsy/1r4YjsMt25MGJcZEEoKBdJ0GkXmlMFNUjOhEEwLIvleSEPAVgasGPYAhEZPxc9EXCKcHuYX9bkaEwRmO3lf9SrtCQAj0bAIi8Pbs6yej90MCZ1Nwi26RDE4XENFJzmAW3WJsMoheTWBqwnngjyQhIASEQGsJiJeG1pKSckJACAgBISAEhIAQEAI9koAIvD3yssmghYAQEAJCQAgIASEgBFpLQATe1pKSckJACAgBISAEhIAQEAI9koAIvD3yssmghYAQEAJCQAgIASEgBFpLQATe1pKSckJACAgBISAEhIAQEAI9koAIvD3yssmghYAQEAJCQAgIASEgBFpLQATe1pKSckJACAgBISAEhIAQEAI9koAIvD3yssmghYAQEAJCQAgIASEgBFpLQATe1pKSckJACAgBISAEhIAQEAI9koAIvD3yssmghYAQEAJCQAgIASEgBFpLQATe1pKSckJACAgBISAEhIAQEAI9koDGSqlHjryDBm0ymWA2m1tsrbq6GgUFBUhNTUVAQECL5Tu6APfZmnF2dL8WiwVZWVmIi4tDaGhoRzffYntdNW8eWG5uLgIDAxETE9PiODu6gEajAX+Yf2enkpIS8P2ekpLSaV3zddbr9Z3Wn2tHrf0OcK3T3L5WqwV/rXbVV2teXp5iGRsb29wwfXauq+efn5+v/nbi4+N9NsfmGu4O8+fxJSQkNDdMn51r73cXczMYDD4blzQsBPxe4G3tLfD555/j5ptvxg8//KCEv9bW6+nlSktLMWXKFDz22GNYuHBhT59Om8Z/3nnnYdSoUXjggQfaVK+nF/773/+O1atXY82aNT19Kn45/gsuuADDhg3DQw895Jfz/+Uvf4moqCg888wzfjn/6667DrW1tfj3v//tl/OXSQsBbwTEpMEbGckXAkJACAgBISAEhIAQ6BUERODtFZdRJiEEhIAQEAJCQAgIASHgjUDAXyh5Oyn5TgJsl8R2rJMmTeoyW0PnaDpvj+fNdlVs1tBVNoGdN1v3ntiudMyYMUhPT3c/0cuP2JaO58xzl9TzCPB9O3r0aPTv37/nDb4DRszzHzFiBAYNGtQBrfW8Jvjvl01ahg4d2vMGLyMWAj4kIDa8PoQrTQsBISAEhIAQEAJCQAh0PQExaej6ayAjEAJCQAgIASEgBISAEPAhAb80afjyyy+RlpYGfvVjT4cOHcL27duRlJTkZrJQV1eHjRs3Ijs7W53j12X2VFhYiO+++065C2NzB3tqTx173c7YZmRkYOfOnWo+Op3O0WVz496/fz9+/vlnREZGIiQkpFV1vPFxVO6inbVr1yrzFHY5Zk/e5tfcHNpTx95fd902dw901zHLuNwJNL6//ema+tNc3a860Pi683lv31GN68qxEPAHAn4n8C5fvhwPP/ww2HWNXdj73e9+p4Q59nn65JNPKmGYfZDW1NRgyZIlqKqqUu7I+Atl3rx5ysfj1q1bccsttyj3Ny+++CKCgoIwfPjwdtXpzBuNXRXxPNinL7samzVrFsLDw5sd9xNPPIEPP/wQRqMRzz77LGbMmKEE3/bw6cy5eupr3bp1uPvuu3H22Wera8dlvM3P2zVubx1P4+lOec1dz+40ThmLdwKN729/uqb+NNfGd0Dj687nvX2vNa4rx0LAbwhw4Al/SORc3nrXXXdZly1bZp05c6aV/BSqaZOm03rllVc6EJDvUeutt96qjsmPoZW+NBznuO6GDRvU8VVXXWXdtm2b2idH79YFCxZYSbtgbU8dRwc+3qGAAlYS2B1zJ0Hd+vzzz6tevY37yJEjVvLraSUBWZV75513rA8++GCzdfikNz6qYhf9R4FDrL/61a/UtTp69KgaRXPz8zaH9tTpoim3qVtv90CbGpHCXUbA0/3tT9fUn+bqepN5uu7NfUe51pV9IeBPBJzv9Hu5iM8azalTpyoNpetUeTXvSy+95MgqKytTTrs5g80cxo8f7zjH+3v27EF9fb0ycbCvYk9MTFSv+Y8fP96uOo4OfLwTFhamzDU2b94MnueuXbsc0Xi8zZXNH3iedvMPOwMeqrc6zfHx8RS9Nk9/1OCACjfeeKPSxtsLeptfc3NoTx17f9156+16ducxy9hsBLzd3/50Tf1prvb73tt19/YdZa8nWyHgjwT8RuBle81zzz3XYcZgv9gsyAUHB6tDDkn5xhtvgDR76phDdEZERNiLqv2ioiJwOQ6zyy677IltW4uLi9GeOvY2fL1lE47bb78dd955Jy688EJwSFVmwsnbuDm8Ls/NnpgHM+DkrU5zfOztdPb2/fffR9++fTFx4kS3rr3Nr7k5tKeOW6fd9MDb9eymw5VhuRDwdn/70zX1p7naL7236+7tO8peT7ZCwB8JOFcs9bLZ06t6pYnlaS1atAishW0u0Ssg3HHHHcpmlzXBnHiBGmuG7Ym1fiwcN87n83yO7Xgbn2tNHXv7Hb3lOa1cuVI1y4Lq9OnT8cgjj6iQk8nJyUrbzTa9bNPa2nHb58ONtrYOl7Xz4X1fp8bznj17NlatWgW+Jxqn9syhPXUa99sdj73NqzuO1Z/H1BH3d2/k52/3L98Hbf1e643XXeYkBFpLoNdqeFkraf/wF2Fzae/evSC7Xdxwww0gW1xHUfa8wFpbe+J9FhQ5AAMvZOMVwfZkP9eeOvY2Onqr1+sdDHhhGtkcY9q0acohO3taWLx4sfJAwf16G3d8fHwTBn369FFD9VanOT4dPUdP7TWe99dff43MzEwsXLgQZ555ptJML126FD/99BO8za+5ObSnjqdxdrc8b9ezu43T38fTlvvbn66pP82V/wba873m7387Mn//JtBrBd7LL79cmSaweQJ/EXpL7HaKX/Pfe++9OPXUU92KsQcDfoKmBW7gcuvXr8e4ceOUWQRHHvv4449VeV4hGx0drT7tqePWaQcepKamOhiwCQNrrlm459XMnNjdml2b7W3cHFmObX2zsrKUlvaTTz7B5MmTVX1vddh0whsfVdHH/zWeNwu3/OOwevVq9WHXc6+88oqah7f5NTeH9tTx8ZQ7pHlv17NDGpdGOoxAW+5vf7qm/jRXvpna873WYTehNCQEeiCBXmvS0Npr8b///Q+lpaW4+eabHVViYmKUG665c+cqd2SXXXaZWrR16aWXOsJ1Xn/99UpQXrFihTp3zz33qPrtqePo2Mc7bMM6f/58/P73v1cuxhISEnDddde1OG7yTqG+XJkL+y/mhwlO3XmuaoCt+I9NPbzNz9s1bk+dVgyly4s0dz27fHAygHYR8Kdr6k9zbelmaO47qqW6cl4I9FYCElq4FVe2oqJC2e7a/fa6VmFhOSoqyjVL7benTpNGfJTBK3tZy+saQMLelbdx8wI3NuFgTw+Nk7c6XM4bn8ZtdPVxc/PzNof21Onqebam/+auZ2vqS5nuR8Cfrqk/zbWlO62576iW6sp5IdDbCIjA29uuqMxHCAgBISAEhIAQEAJCwI1Ar7XhdZulHAgBISAEhIAQEAJCQAj4LQEReP320svEhYAQEAJCQAgIASHgHwRE4PWP6yyzFAJCQAgIASEgBISA3xIQgddvL71MXAgIASEgBISAEBAC/kFABF7/uM4ySyEgBISAEBACQkAI+C0Bv/fD67dXXiYuBLohgdzcXLz00ktuI2NXeP3798c555yjwne7niwvLwcHfvnmm2+UC7xTTjlFhQcPDQ11LebYZ7/ZHCRmzpw5jjzZEQJCQAgIgd5PQNyS9f5rLDMUAj2GwJYtWzBx4kQVBc/u85nDeO/evVsJqhwd0B7a+uDBg5g9e7YKojJv3jyYzWZ88cUXKrIih9G217dPngVjDk5w//33q6Ax9nzZCgEhIASEQO8nICYNvf8aywyFQI8jwFpeDgfNnx9//FGFt2bB98EHH1Rzyc7Oxi9+8QtwONn8/Hy8/fbbePfdd3H06FFwuTvvvNMxZ3a+f9999+HMM8+ERqNx5MuOEBACQkAI+A8BEXj951rLTIVAjyXAIa2nTZuG7du3qzm88847SrP78ssvIyAgwDEvjnr4yiuvKO2u0WhU+a+99hpeffVVFS58yJAhjrKyIwSEgBAQAv5DQARe/7nWMlMh0GMJsNb2u+++Q3p6uprDhg0blAAcGRnZZE7z58/H3//+dxgMBnVu4cKFYPOHs846q0lZyRACQkAICAH/ICCL1vzjOssshUCPIvDcc88hKSkJVqsV+/fvx5o1a1BfX4877rhDzePnn3/GokWLWjUnbkeSEBACQkAI+DcBEXj9+/rL7IVAtyTApgsZGRlqbHFxcVi2bBmuu+46pKamqryUlBTk5eV1y7HLoISAEBACQqD7ERCBt/tdExmREPB7Ai+++CLGjh3rlcP48eOxfv16j+f37duHf/3rX7jlllvAgrEkISAEhIAQEAJiwyv3gBAQAj2OALshY9djK1eubDJ2Nnt4/vnnER4e3uScZAgBISAEhIB/EhANr39ed5m1EOjRBBYsWICbbroJV1xxBR577DGwAJyTk4O33noLH3/8MR5//HFERET06DnK4IWAEBACQqDjCIjA23EspSUhIAQ6kQALusHBwXjhhRdw7bXXwmKxIDExEbzgje19JQkBISAEhIAQsBOQSGt2ErIVAkKgxxIoLS1FdXU1kpOTe+wcZOBCQAgIASHgOwIi8PqOrbQsBISAEBACQkAICAEh0A0IyKK1bnARZAhCQAgIASEgBISAEBACviMgAq/v2ErLQkAICAEhIASEgBAQAt2AgAi83eAiyBCEgBAQAkJACAgBISAEfEdABF7fsZWWhYAQEAJCQAgIASEgBLoBARF4u8FFkCEIASEgBISAEBACQkAI+I6ACLy+YystCwEhIASEgBAQAkJACHQDAiLwdoOLIEMQAkJACAgBISAEhIAQ8B0BEXh9x1ZaFgJCQAgIASEgBISAEOgGBP4fWalN0fZeelwAAAAASUVORK5CYII=" /></p>
<!-- rnb-plot-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>The normalization in particular has appeared to help separate the rest of the brain cells from the immune cells. But overall, data seems moderately clustered by cell type, and for PC1 in particular, this appears to have been improved by the <code>scran</code> normalization.</p>
</div>
<div id="explore-gene-variance" class="section level2">
<h2>Explore gene variance</h2>
<p>Even after normalization, our transcript data is being dominated by a few genes that are highly expressed. Some of these high expression genes are probably “housekeeping-like” in that their expression patterns are stable across different circumstances. These “housekeeping-like” genes are unlikely to help us determine biologically meaningful gene expression patterns. So another way to determine potential genes of interest is to calculate the variance.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBDcmVhdGUgYSBnZW5lIHZhcmlhbmNlIHZlY3RvclxuZ2VuZV92YXIgPC0gYXBwbHkoc2NlX25vcm0sIDEgLCBzZClcblxuIyBDcmVhdGUgYSBnZW5lIG1lYW4gdmVjdG9yXG5nZW5lX21lYW4gPC0gYXBwbHkoc2NlX25vcm0sIDEgLCBtZWFuKVxuXG4jIENvbWJpbmUgdGhlc2UgaW50byBhIGRhdGEgZnJhbWUgdGhhdCBhbHNvIGNvbnRhaW5zIGxvZyh2YXJpYW5jZS9tZWFuKVxuZ2VuZV9zdGF0cyA8LSBkYXRhLmZyYW1lKGxvZ192YXIgPSBsb2cyKDEgKyBnZW5lX3ZhciksIFxuICAgICAgICAgICAgICAgICAgICAgICAgIGxvZ19tZWFuID0gbG9nMigxICsgZ2VuZV9tZWFuKSkgJT4lIFxuICBkcGx5cjo6bXV0YXRlKHZhcl9tZWFuX3JhdGlvID0gKGxvZ192YXIgLSBsb2dfbWVhbikpXG5gYGAifQ== -->
<pre class="r"><code># Create a gene variance vector
gene_var &lt;- apply(sce_norm, 1 , sd)

# Create a gene mean vector
gene_mean &lt;- apply(sce_norm, 1 , mean)

# Combine these into a data frame that also contains log(variance/mean)
gene_stats &lt;- data.frame(log_var = log2(1 + gene_var), 
                         log_mean = log2(1 + gene_mean)) %&gt;% 
  dplyr::mutate(var_mean_ratio = (log_var - log_mean))</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Let’s plot the relationship of gene variance and gene means so we can get a better idea of what kinds of genes we are working with. Here we will plot the <code>log(variance/mean)</code> against the <code>log(mean)</code> expression for each gene. On your own, it’s good practice to see how this graph changes with changes of normalization parameters, as well as high variance gene cutoffs.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBMZXQncyBwbG90IHRoaXMgd2l0aCBnZ3Bsb3QyXG5nZ3Bsb3QoZ2VuZV9zdGF0cywgYWVzKHggPSBsb2dfbWVhbiwgeSA9IHZhcl9tZWFuX3JhdGlvKSkgK1xuICAjIFdlIGFyZSB1c2luZyBhbHBoYSBzbyB0aGF0IHRoZSBwb2ludHMgYXJlIG1vcmUgc2VlLXRocm91Z2hcbiAgZ2VvbV9wb2ludChhbHBoYSA9IDAuMTUpICtcbiAgeGxhYihcIkdlbmUgbG9nKE1lYW4pXCIpICtcbiAgeWxhYihcIkdlbmUgbG9nKFZhcmlhbmNlL01lYW4pXCIpICsgXG4gIGNvd3Bsb3Q6OnRoZW1lX2Nvd3Bsb3QoKSBcbmBgYCJ9 -->
<pre class="r"><code># Let&#39;s plot this with ggplot2
ggplot(gene_stats, aes(x = log_mean, y = var_mean_ratio)) +
  # We are using alpha so that the points are more see-through
  geom_point(alpha = 0.15) +
  xlab(&quot;Gene log(Mean)&quot;) +
  ylab(&quot;Gene log(Variance/Mean)&quot;) + 
  cowplot::theme_cowplot() </code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<p>Because our x axis has the same data that we plotted in the above section, you’ll notice that we continue to see zero inflation that our single-cell is known for.</p>
</div>
<div id="save-the-normalized-data-to-tsv-file" class="section level2">
<h2>Save the normalized data to tsv file</h2>
<p>In case we wanted to return to this data later, let’s save the normalized data to a tsv file. <code>readr</code> requires that it be a <code>data.frame</code> so we will convert our <code>matrix</code> first.</p>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuIyBTYXZlIHRoaXMgZ2VuZSBtYXRyaXggdG8gYSB0c3YgZmlsZVxubG9nY291bnRzKHNjZV9ub3JtKSAlPiUgYXMuZGF0YS5mcmFtZSgpICU+JVxuICByZWFkcjo6d3JpdGVfdHN2KGZpbGUucGF0aChcImRhdGFcIiwgXCJzY3Jhbl9ub3JtX2dibV9nZW5lX21hdHJpeC50c3ZcIikpXG5gYGAifQ== -->
<pre class="r"><code># Save this gene matrix to a tsv file
logcounts(sce_norm) %&gt;% as.data.frame() %&gt;%
  readr::write_tsv(file.path(&quot;data&quot;, &quot;scran_norm_gbm_gene_matrix.tsv&quot;))</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
<!-- rnb-text-begin -->
<div id="print-session-info" class="section level3">
<h3>Print session info</h3>
<!-- rnb-text-end -->
<!-- rnb-chunk-begin -->
<!-- rnb-source-begin eyJkYXRhIjoiYGBgclxuc2Vzc2lvbkluZm8oKVxuYGBgIn0= -->
<pre class="r"><code>sessionInfo()</code></pre>
<!-- rnb-source-end -->
<!-- rnb-chunk-end -->
</div>
</div>

<div id="rmd-source-code">---
title: "scRNA-seq Normalization"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2019**

In this notebook, we'll perform quality control analyses and normalization of 
scRNA-seq count data. 

As opposed to bulk RNA-seq, there are there are a few main things to look out for
in single-cell RNA-seq:

**Single-cell RNA-seq...**

- Requires more PCR amplification (and therefore more PCR-associated biases and
error).  
- Has more zeroes in the gene expression data (most genes aren't expressed 
across cell types).  
People sometimes refer to this as "sparsity" or "dropout."  
  
For this tutorial, we will be using a pair of single-cell analysis specific 
R packages: `scater` and `scran`. 
This tutorial is in part based on the [scran
tutorial.](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html)
  
## Set Up 

For these analyses, we will need `scater` and `scran` packages, which have been 
installed already on your Docker container. 

```{r Set Up}
# Read in a function for filtering the gene matrix from a source script
source(file.path("scripts", "GeneMatrixFilter.R"))

# Magrittr pipe
`%>%` <- dplyr::`%>%`

# Load the already installed packages
library(scater)
library(scran)
```

## Import single-cell RNA-seq counts matrix

This [data set](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) 
we are using is glioblastoma data that was Fluorescence-Activated Cell sorted 
and then was processed by paired end sequencing using Smart-seq2 protocol 
[(Darmanis et al. _Cell Reports._ 2017.).](https://www.ncbi.nlm.nih.gov/pubmed/29091775).
We're using a subset of this dataset (n = 786 cells).
You have been provided with a counts matrix from single-cell RNA-seq data that 
has already been processed using `Salmon` and `tximport` and saved the counts to 
a tsv file. 
You already know how to process this type of non-tag-based data, because you 
can follow essentially the same steps as we did in the bulk RNA-seq modules.
As far as *tag-based* scRNA-seq data, we will explain how to process raw data, 
like fastq files, in the next section, so you can also get it to a count matrix
and could use these same steps to filter and normalize the data.
  
```{r Import Data}
# Read in the data
sc_data <- readr::read_tsv(file.path("data", "unfiltered_darmanis_counts.tsv"), 
                           progress = FALSE)

# Read in the metadata
sc_metadata <- readr::read_tsv(file.path("data", "unfiltered_darmanis_metadata.tsv"), 
                               progress = FALSE)
```

## Filter the counts matrix
  
Single-cell RNA-seq experiments are generally much larger dimensions than bulk 
RNA-seq, since every individual cell is being measured. 
However, due to the high risk of technical artifacts, we don't necessarily want 
to keep the data for all the cells and genes. 

There are different approaches to filtering your data. 
In an ideal world, all RNA-seq data would come with the percent mapped reads 
statistics associated with it. 
This should be one of the first things you filter your data by.
Side note: even the highest quality single-cell RNA-seq data still has much 
lower percent mapped reads than bulk RNA-seq, so don't expect the same range of
percent mapped reads of bulk RNA-seq.
However, because a lot of publicly available data does not have percent mapped
read statistics associated with it, we are going to attempt to get around this
by trying other strategies of filtering first.

#### Total counts as a quality measure

One basic of method of exploring if a particular cell's data is trustworthy is
seeing how many counts were measured total, over all the genes. 
Let's take a look at this data from how many counts are there per sample:
Note that for `apply` the second argument indicates whether the function you 
specify is being applied across all the rows or the columns (1 = rows, 2 = columns).
Also note that in this case we are excluding the `genes` column for the calculation. 

```{r}
# Make a vector of the total number of counts per sample
total_counts <- apply(sc_data[, -1], 2, sum)

# Take a look at the summary statistics for the total counts
summary(total_counts)
```

Yikes, one of the samples has only 48 counts. 
It's highly likely that this 'cell' is either an empty well or the well it was
in didn't get sequenced properly. 
Regardless of what may have happened, we can't trust that 'cell'.
These kinds of things are to be expected; it's highly unlikely that when obtain 
a single-cell RNA-seq dataset that all of cells within the set have reliable, 
high quality data.

This is why exploring your data and filtering it is important before we attempt to
look for any biologically meaningful information. 
  
Let's visualize the distribution of total counts to see if the 48 count sample 
is the only one we should get rid of.

```{r Explore total counts distribution}
# Let's take a look at what this looks like:
hist(total_counts, col = "lightblue", breaks = 40);
  # let's plot what we might want to be our cutoff:
  abline(v = 50000, col = "red")
```

```{r Filter by total counts}
# Filter our data based on this cutoff 
sc_data_filtered <- sc_data %>% 
  tibble::column_to_rownames("genes") %>%
  dplyr::select(which(total_counts > 50000))

# So we can be sure that our data matches or metadata, we will use a filter
sc_metadata_filtered <- sc_metadata %>% 
  dplyr::filter(geo_accession %in% colnames(sc_data_filtered))

# Always good to double check that your data and your metadata matches
all.equal(colnames(sc_data_filtered ), sc_metadata_filtered$geo_accession)
```
We got rid of some of the ridiculously low total count samples. 

### Number of genes a cell expresses as a quality measure

What if a particular cell has all its counts belonging to one gene?
This cell would not have helpful data for us, so we should look to remove any 
cells we suspect might not have a useful amount of its transcriptome measured.

But before we can determine how many genes we consider a particular cell to be 
expressing we need to determine a numeric cutoff for what we consider to be a 
detected gene.
How many counts must there be for you to consider a gene expressed? 
For single-cell data, it is probably best to be a bit lenient due to the nature
of the data. 
Here let's go for a simple detection cutoff of > 0 

```{r Create detection matrix}
detection_mat <- as.matrix(sc_data_filtered) > 0 
```

Now that we have turned our data into a matrix of `TRUE/FALSE` for detection, we
can sum this data by column to effectively get a vector of how many genes were 
measured in each cell. 

```{r}
# Make a vector that contains the number of genes expressed by a particular cell
num_genes_exp <- apply(detection_mat, 2, sum)

# Let's take a look at what this looks like:
qplot(num_genes_exp, geom = "density") + 
  geom_density(fill = "lightblue")
```

This distribution makes it seem like maybe any cells expressing less than 750
genes might be a good cutoff?

But as you may have noticed in our `sc_metadata` we have cell-type information 
for these data. 
Let's use the power of our single-cell resolution data to our advantage and look 
at this data with cell-type labels. 

```{r}
# Let's match this to the metadata so we can plot samples by their cell type
num_genes_exp_df <- data.frame('geo_accession' = colnames(detection_mat), 
                               num_genes_exp) %>% 
  dplyr::inner_join(sc_metadata, by = 'geo_accession')

# Plot these data
ggplot(num_genes_exp_df, aes(x = num_genes_exp)) + 
  geom_density(aes(color = cell.type.ch1)) +
  geom_vline(xintercept = 200, color = "red")
```

### Number of samples that express a gene as a quality measure

Now we have an idea of what samples we probably want to get rid of.
But what if our data contains genes that we can't reliably measure in these cells?
Some gene types, like many long non-coding RNAs for example, are very tissue 
specific and wouldn't be useful for us to look at it. 
For example if only one cell is supposedly expressing a particular gene at a low
level, do we think that is biologically relevant, or is it more likely a 
technical artifact? 
Because our single-cell data is already riddled with zeroes, it's best just to 
get rid of these genes that we suspect are not being reliably measured by many 
cells. 
To do this, we will use our detection matrix to add up how many samples express
each of these genes. 

```{r}
# Make a vector that contains the number of cells that express a particular gene
num_samples_that_exp <- apply(detection_mat, 1, sum)

# Let's take a look at what this looks like:
qplot(num_samples_that_exp, geom = "density") + 
  geom_density(fill = "lightblue") +
  geom_vline(xintercept = 5, col = "red")
```

### Apply a filter based on our criteria

Now that we've explored our data, let's used the information we've learned to 
actually filter our data.

```{r Apply filter}
sc_data_filtered <- GeneMatrixFilter(sc_data_filtered, 
                                     min_counts = 0, 
                                     num_samples = 5, 
                                     num_genes = 200)
```

Let's see how our filter did in regards to getting rid of samples that are 
likely bad quality according to percent mapped reads data.

```{r Check percent mapped reads}
# Let's read in the percent mapped reads data
perc_map_reads <- readr::read_tsv(file.path("data",
                                            "darmanis_percent_mapped_reads.tsv")) %>%
  # Let's make a new variable that says whether or not the cell was filtered out
  dplyr::mutate("kept" = geo_accession %in% colnames(sc_data_filtered)) 

# Plot the densities of these data
ggplot(perc_map_reads, aes(x = percent_mapped_reads, fill = kept)) +
  geom_density(alpha = 0.15)
```

The samples we filtered out do appear to include the lower percent mapped reads 
so this is decent support for our filtering methods.

## Set up a SingleCellExperiment object from count data

For the rest of this module and some of the upcoming modules, we are going to 
use some single-cell RNA-seq R packages called `scater` and `scran` to do some 
processing for us.
The functions in these packages require the data to be in their own special 
object type (not an uncommon thing for R packages to do) called `SingleCellExperiment`.
So first, we are going to set up our data in this format.

```{r Set up SCE object}
# Prep a matrix with just the rounded counts so we can create an sce object
counts_mat <- round(as.matrix(sc_data_filtered))

# Set up the Single Cell Experiment object 
sce_raw <- SingleCellExperiment::SingleCellExperiment(list(counts = counts_mat))
```

Special object type are difficult to use at times if we don't know what they 
look like. 
Based on things we've learned in the previous modules, use this space to explore
the `SingleCellExperiment` object you just made.

```{r Explore SCE structure}
# The SingleCellExperiment is a special type of object used by scater and scran
# R packages. 
# Use what we've learned so far to identify how many genes and cells passed
# the filtering step
```

Let's examine what our filtered counts data looks like with a gene mean density
plot.

```{r}
# Let's calculate the gene means and make a density plot
gene_means <- apply(counts(sce_raw), 1, mean)

# Plot the means
qplot(gene_means, geom = "density", xlab = "counts") + 
  geom_density(fill = "lightblue")
```

The unnormalized, counts data has so many zeroes, which we should expect. 
For this dataset, 79.5% of the data points are zeroes, which is not 
out of the ordinary for single-cell RNA-seq data. 
This means our data consists of a few genes that are expressed a lot, and the 
rest of the genes are barely detected. 

## Normalization of count data 

In whatever data we are working with, we are always looking to maximize biological 
variance and minimize technical variance. 
This is where normalization methods usually come into the workflow.
With this high amount of zeroes in our data, this makes normalization particularly
tricky.
So in order to normalize with less of these zeroes in the way, we normalize cells
in groups with other cells like them; a method determine by [Lun et al 2016](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0947-7).
  
*Overall strategy for `scran::computeSumFactors` based normalization:*
1) Group cells with other like cells by clustering.  
2) Compute sum factors for each cell cluster grouping.  
3) Normalize using these pooled sum factors to create log counts per million reads (CPMs).  
  
Because of differences in sequencing depth, it's pretty common to try to correct
for this by using total per million reads.
You have probably seen data reported that was, in part, normalized based the 
total per million reads for a sample such as counts per million reads (CPMs),
(or TPMs, FPKMs, and RPKMs, which you can [read about more here](https://haroldpimentel.wordpress.com/2014/05/08/what-the-fpkm-a-review-rna-seq-expression-units/) on your own).

#### Normalization step 1: Estimate cell clusters:

Using a quick clustering, let's determine groupiungs for our cells that we will
normalize by.

```{r}
# Calculate clusters (can adjust min.size depending on clusters sizes you may 
# expect based on prior knowledge)
qclust <- scran::quickCluster(sce_raw)
```

#### Normalization step 2: Compute sum factors:

Compute factors for each sample using the clusters we just made.

```{r}
# Calculate the size factors based on the clusters
sce_raw <- scran::computeSumFactors(sce_raw, clusters = qclust)
```

The authors of `scran` recommend removing cells that have negative size factors, 
so we will do that here. 
Of course, if you have a lot of cells that turn up has negative size factors, 
you should probably reconsider your filter and/or whether something has affected
the quality of your data. 

```{r Remove negative size factor cells}
# Extract the negative factor data
neg.fact <- which(sce_raw@int_colData@listData$size_factor <= 0)

# Let's see how many cells this is
length(neg.fact)

# Since it's only three, let's go ahead and remove them from the data
sce_raw <- sce_raw[, -neg.fact]

# Take these out the metadata too
sc_metadata_filtered <- sc_metadata_filtered %>%
  dplyr::filter(geo_accession %in% sce_raw@colData@rownames)
```

#### Normalization step 3: Calculate pooled CPMs and normalize using sum factors:

For more details on the normalization [read here](https://rdrr.io/bioc/scran/man/computeSumFactors.html)

```{r Normalize the Data}
# Normalize the data
sce_norm <- scater::normalize(sce_raw)
```

## Compare normalized data to count data

One way to determine whether our normalization is yielding biolgically relevant 
results, is to plot it and see if similarly labeled samples and cells end up 
together.
Because plotting 1000's of genes together isn't practical, we will reduce the 
dimensions of our data using Principal Components Analysis. 
Note that the `t()` function is being used here to transpose the `matrix`, so 
it is samples x genes. 
Because, in this instance, we would like to have PCA scores for each cell.
We could however, obtain PCA scores for each gene if we didn't transpose our
data.frame first. 

```{r}
# Use PCA for dimension reduction of cells' raw data
count_pca <- prcomp(t(counts(sce_raw)))

# Use PCA for dimension reduction of cells' scran normalized data
norm_pca <- prcomp(t(logcounts(sce_norm)))
```

There's a lot of data we can pull from these `prcomp` objects, but for now, 
we need the PCA scores for the samples.
In order to retrieve the PCA scores, we will use `$x` at the end of our `prcomp`
objects and because we only really want

```{r Graph PCA}
# Make this into a dataframe for easy graphing 
pca_scores <- rbind.data.frame(count_pca$x[, 1:2], 
                               norm_pca$x[, 1:2]) %>% 
  # Make data and cell type labels
  dplyr::mutate("data_label" = c(rep("counts", nrow(count_pca$x)),
                                 rep("norm", nrow(norm_pca$x))),
                "cell_type" = rep(sc_metadata_filtered$cell.type.ch1, 2)) %>%
  # Change these into factors
  dplyr::mutate("data_label" = as.factor(data_label),
                "cell_type" = as.factor(cell_type))

# Now plot it
ggplot(pca_scores, aes(x = PC1, y = PC2, color = cell_type)) +
  geom_point() +
  facet_wrap(~data_label, scales = "free")
```

The normalization in particular has appeared to help separate the rest of the 
brain cells from the immune cells. 
But overall, data seems moderately clustered by cell type, and for PC1 in 
particular, this appears to have been improved by the `scran` normalization.

## Explore gene variance

Even after normalization, our transcript data is being dominated by a few genes 
that are highly expressed. 
Some of these high expression genes are probably "housekeeping-like" in that 
their expression patterns are stable across different circumstances. 
These "housekeeping-like" genes are unlikely to help us determine biologically 
meaningful gene expression patterns. 
So another way to determine potential genes of interest is to calculate the 
variance.

```{r Calculate Gene Variance}
# Create a gene variance vector
gene_var <- apply(sce_norm, 1 , sd)

# Create a gene mean vector
gene_mean <- apply(sce_norm, 1 , mean)

# Combine these into a data frame that also contains log(variance/mean)
gene_stats <- data.frame(log_var = log2(1 + gene_var), 
                         log_mean = log2(1 + gene_mean)) %>% 
  dplyr::mutate(var_mean_ratio = (log_var - log_mean))
```

Let's plot the relationship of gene variance and gene means so we can get a 
better idea of what kinds of genes we are working with.
Here we will plot the `log(variance/mean)` against the `log(mean)` expression 
for each gene.
On your own, it's good practice to see how this graph changes 
with changes of normalization parameters, as well as high variance gene cutoffs.

```{r Plot Variance vs Means}
# Let's plot this with ggplot2
ggplot(gene_stats, aes(x = log_mean, y = var_mean_ratio)) +
  # We are using alpha so that the points are more see-through
  geom_point(alpha = 0.15) +
  xlab("Gene log(Mean)") +
  ylab("Gene log(Variance/Mean)") + 
  cowplot::theme_cowplot() 
```

Because our x axis has the same data that we plotted in the above section, 
you'll notice that we continue to see zero inflation that our single-cell is
known for.

## Save the normalized data to tsv file

In case we wanted to return to this data later, let's save the normalized data
to a tsv file. 
`readr` requires that it be a `data.frame` so we will convert our `matrix` first.

```{r Save Data to .tsv}
# Save this gene matrix to a tsv file
logcounts(sce_norm) %>% as.data.frame() %>%
  readr::write_tsv(file.path("data", "scran_norm_gbm_gene_matrix.tsv"))
```

### Print session info

```{r}
sessionInfo()
```</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});

$(document).ready(function () {
  $('.knitsql-table').addClass('kable-table');
  var container = $('.kable-table');
  container.each(function() {

    // move the caption out of the table
    var table = $(this).children('table');
    var caption = table.children('caption').detach();
    caption.insertBefore($(this)).css('display', 'inherit');
  });
});

</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
