---
title: "scRNA-seq Dimension Reduction"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2019**

In this notebook, we'll perform dimension reduction and clustering analyses of 
single-cell RNA-seq data. 

## Set Up 
For these analyses, we will need Seurat.
```{r}
library(Seurat)

# Magrittr pipe
`%>%` <- dplyr::`%>%`
```

Put the path to the Seurat object we created previously here:
```{r Path to Seurat}
# Load in Seurat object
seurat_norm <- readRDS(file.path("data", "seurat_object.RDS"))
```

Remember that previously, we filtered our dataset, so as we import our metadata
we need to filter it so that it only contains the samples we are working with, 
in the order we are working with them in the data. 
```{r Filter metadata}
# Get list of samples after we previously filtered the data
samples <- colnames(seurat_norm@raw.data)

# Read in metadata, but filter out samples that were previously filtered out of 
# the full dataset
sc_meta <- readr::read_tsv(file.path("data", "darmanis_metadata.tsv")) %>% 
  dplyr::filter(geo_accession %in% c(samples))
```

## Principal Components Analysis
Just like we did with bulk RNA-seq, we will also run PCA with our single-cell data. 
However, because single-cell data is more skewed as far as gene expression
(remember there are more genes that have zeroes), we will do our PCA based on the 
high variance genes, which should give us a more informative analysis.
This will also print for us the top five genes contributing to the variance for 
each component. 
```{r PCA}
# Run PCA
seurat_norm <- RunPCA(object = seurat_norm, pc.genes = seurat_norm@var.genes,
                      genes.print = 5)
```

Let's plot our first two principal components and label our cells with what cell
type they are (remember this data was FACS sorted).
```{r Plot PCA}
# Make a dataframe with the metdata and pca data
pca <- data.frame(seurat_norm@dr$pca@cell.embeddings)

# Plot this with ggplot2 and label the points with cell types
ggplot(pca, aes(x = PC1, y = PC2, color = sc_meta$cell.type.ch1)) + 
  geom_point() +
  colorblindr::scale_color_OkabeIto()  # Add this so our color palette is colorblindness friendly
```

## t-SNE experiment
t-SNE is a popular dimension reduction technique that has become even more popular
with single-cell RNA-seq data analysis. 
It would be a good use of our time to explore t-SNE parameters and how they 
affect the outcomes.
For our experiment, let's evaluate how various parameters, like perplexity,
affect t-SNE outcomes.
For us to determine a good number range to use for testing perplexity, let's 
first look up the documentation for `Seurat::RunTSNE` and find out what the 
default for the `perplexity` argument is. 
Pick a good range of numbers smaller and larger than that default number and 
run t-SNE at different parameters and plot it with `ggplot2`. 
ggplot2 is another tidyverse tool and is great for making data visualizations.
It's worth checking out [ggplot2 tutorials](https://ggplot2.tidyverse.org/index.html). 
```{r Run tSNE and plot it}
# Run tSNE 
seurat_norm <- RunTSNE(object = seurat_norm, dims.use = 1:10, seed.use = 1234, 
                       perplexity = 30, do.fast = TRUE)

# Make this a data.frame so ggplot will like it
tsne <- data.frame(seurat_norm@dr$tsne@cell.embeddings)
  
# Plot this with ggplot2
ggplot(tsne, aes(x = tSNE_1, y = tSNE_2)) + geom_point() 
```
You'll notice that for some settings of tSNE parameters, it's hard to distinguish
distinct clusters of cells.
  
#### Apply metadata labels to our tSNE
Let's re-try this experiment but add labels using some of our metadata to see 
how the cells are clustering (or not clustering) by different variables. 
Remember that previously we set up `sc_meta` to have the corresponding
metadata for each of our cells. 
Below we have set up the cells to be colored by the `cell.type` variable. 
Run this, but also feel free to look at what other information we have for these
cells in `sc_meta` by using `str()` and change this to a different variable to 
graph it by.
```{r tSNE}
# Do tsne plots for each perplexity parameter, but this time use color argument
seurat_norm <- RunTSNE(object = seurat_norm, dims.use = 1:10, seed.use = 1234, 
                       perplexity = 30, do.fast = TRUE)

# Make this a data.frame so ggplot will like it
tsne <- data.frame(seurat_norm@dr$tsne@cell.embeddings)
  
# Plot this with ggplot2
ggplot(tsne, aes(x = tSNE_1, y = tSNE_2, color = sc_meta$cell.type.ch1)) + 
  geom_point() + 
  colorblindr::scale_color_OkabeIto() 
```
You should notice that although the coordinates of tSNE output for any given 
cell might change drastically, the cells are still largely grouped together by
their cell type. 
This probably means that you shouldn't rely too heavily on the exact values of
t-SNE's output. 
However, using the cell-type labeling, we are finding that for this dataset t-SNE
does appear to give some biologically relevant output information. 
Read [this](https://distill.pub/2016/misread-tsne/) to learn more about using 
t-SNE properly. 
  
However, also note that PCA, [UMAP](https://github.com/lmcinnes/umap), and 
other single-cell RNA-seq specific methods like 
[VASC](https://www.sciencedirect.com/science/article/pii/S167202291830439X) 
and [scVI](https://scvi.readthedocs.io/en/master/readme.html) are options for 
dimension reduction techniques.

#### Some 'big picture' thoughts to take from this experiment: 
1) Different analyses (such as t-SNE) have various strengths and limitations 
2) It's important to know these strengths and limitations when you are 
   interpreting the results of an analysis.
3) Playing with parameters so you can fine-tune them is a good way to give you more
   information about a particular analysis as well as the data itself. 
   ie. if the results of an analysis can be completely changed by changing it's
   parameters, you should be more cautious when it comes to the conclusions you
   draw from it as well as having good rationale for he parameters you choose. 

## Clustering 
Now that we've ran two different types of dimension reduction techniques on our 
data, let's see what types of clusters each type of dimension reduction makes. 
```{r Run}
# Cluster analysis for PCA
seurat_norm <- FindClusters(object = seurat_norm, reduction.type = "pca",
                            dims.use = 1:10, print.output = 0, save.SNN = TRUE)
```

#### Print session info:
```{r}
sessionInfo()
```
