---
title: "Exercise: Gene markers in single-cell data"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

In this exercise, we will be using the [Darmanis data](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) 
we normalized and filtered in `01-normalizing_scRNA`. 

This data comes from Fluorescence-Activated Cell Sorted (FACS) glioblastoma cells
and then was processed by paired-end sequencing using Smart-seq2
[(Darmanis et al. _Cell Reports._ 2017.).](https://www.ncbi.nlm.nih.gov/pubmed/29091775).

Our objective for this exercise is to investigate the expression of genes of
interest across cell-types. TODO: look at cell-type/cell type
We will extract the gene's expression levels and make density plots.
Then, we will calculate the mean expression of the marker genes by cell-type and 
make a heatmap.
Lastly, we will project cells onto principal components and color them by the 
expression level of a marker gene. 

## Setup 

We'll be using the `%>%` operator (pipe) and we will need to set the seed so our 
results are reproducible.

```{r}
# Set the seed
set.seed(1234)

# Load the pipe
`%>%` <- dplyr::`%>%`
```

Use the template code chunk below to make a new directory where you can save plots.

```{r eval=FALSE}
plots_dir <- <DIRECTORY_NAME>
if (!dir.exists(<FILL_IN_THE_BLANK>)) {
  dir.create(<FILL_IN_THE_BLANK>)
}
```

## Import and set up the data and metadata

Use `readr`'s function for reading RDS files. 
Import the `SingleCellExperiment` object that we stored in "sce_norm.RDS" in the
`data` folder in `01-normalizing_scRNA-seq`. 
Remember to use `file.path` and `::` to call any packages that you might use that
you haven't imported. 

```{r eval=FALSE}
sce_norm <- 
```

Use the empty code chunk below to explore `sce_norm`.

```{r}

```

Here we are extracting the gene names from `sce_norm`. 
We've done this step for you. 
Run the code chunk below and see what `genes` object gives you.

```{r}
# Extract the gene names 
genes <- rowData(sce_norm)$value
```

Complete the following code chunk to extract the normalized gene matrix and save 
it as a data.frame called `sce_norm_df`, where the first column is called 
"genes" and has the genes we extracted above.

```{r}
# Extract the normalized data as a data.frame and have a genes column
sce_norm_df <- 
```

Load the metadata that we saved in a TSV file called "gbm_filtered_metadata.tsv" 
in the "data" folder in `01-normalizing_scRNA-seq`.

```{r}
metadata <- 
```

Complete the code chunk below to investigate the variables in `metadata`.
Then extract the cell-type labels as their own object, made into a `factor`,
named `cell_types`.

```{r}
cell_types <- 
```

## Set up marker gene data

For this exercise, we want to analyze specific genes and their expression among 
the cell-types in our dataset. 

Select a gene of your own liking and get its information from
https://www.genecards.org/.
Add your chosen gene in the same format we have started for you but replacing 
our <FILL_IN_THE_BLANKS>. 

In the machine-learning module, in `01-medulloblastoma_data_prep` we will show 
you how to convert Ensembl IDs to other kinds of gene IDs in a bulk fashion. 
However, for this exercise, we will only need a few genes and we've already set 
some up for you. 

```{r eval=FALSE}
# Make a data.frame that contains gene symbols and their associated Ensembl IDs
markers <- data.frame(symbols = c("CD45", # Marker for macrophages and microglia
                                  "MOG", # Myelination gene
                                  "SOX9", # Gene expressed in neurons
                                  <GENE_SYMBOL>),
                      ensembl = c("ENSG00000081237", 
                                  "ENSG00000204655", 
                                  "ENSG00000125398",
                                  <ENSEMBL_GENE_ID>)
                      )
```

Depending on the gene you chose, it may not be in our filtered matrix, so you 
may need to try a few. 
You can use an `%in%` operator to check if the gene you are looking for is in 
our filtered gene matrix (Remember it has to be an Ensembl gene ID e.g.`ENSG`,
and it needs to be in quotes).

```{r eval=FALSE}
# Use the phrase below to find out if a gene you are interested in is in sce_norm_df
<ENSEMBL_GENE_ID> %in% sce_norm_df$genes
```

Use this code chunk to look at what `markers` looks like.

```{r}

```

Let's build a `dplyr` pipe to create a `data.frame` with the data for our marker 
genes and add on `cell_type` vector as its own column.
Remember that each step in your pipe needs to be followed by the `%>%` operator
and that we haven't loaded the `dplyr` library so use the `::` if needed. 

```{r}
# Make a data.frame with only the genes in `markers`
marker_data <- sce_norm_df %>%
  # Use this line to filter the data to only the marker genes we care about.
  # Use this line for tibble::column_to_rownames function to hide the gene names
  # Use this line to transpose this data
  as.data.frame() %>% # We've done this step of turning the matrix back into a 
                      # data.frame after we transposed
  # Use this line to add a column with the information in `cell_types`
```

## Plot the marker genes' expression as a density plot by cell-type

Use the following `ggplot` template to make a density plot by cell-type for one
of your genes of interest (hint: use the `marker_data` we just made).
Remember that for each step in making a `ggplot`, you need to add a `+`.

```{r eval=FALSE}
# Plot the gene data by cell-type using `facet_wrap`
ggplot2::ggplot(<DATA_FRAME>, ggplot2::aes(x = <GENE_DATA_COLUMN>)) +
  ggplot2::geom_density(fill = <COLOR_NAME>) + # Choose a color 
  ggplot2::facet_wrap(~ <CELL_TYPE_INFO>) # Use cell-types in this function to 
                                          # separate the density plots
```

Feel free to play around with aspects of this plot and see how the plot changes.
Look at [ggplot2's guide](https://ggplot2.tidyverse.org/) for a list of the 
options. 

## Make a heatmap of mean expression of marker genes by cell-type

Let's build a `dplyr` pipe to calculate the means of these marker genes based on 
the cell-type labels. 

```{r}
# Build a pipe to calculate the mean expressions within each cell-type
marker_means <- marker_data %>%
  # Use this line for a dplyr function to group the data by their cell-types 
  # Use this line for a dplyr function to summarize to a mean
  # Use the tibble::column_to_rownames hide the cell-type column
```

Let's change the Ensembl gene IDs to symbols since they are easier to read in 
the heatmap.
 
```{r eval=FALSE}
# Change the column names to gene symbols
colnames(marker_means) <- 
```

Use `pheatmap::pheatmap` to make a heatmap of cell-type mean expression of these
marker genes. 
Use the argument `scale = "row"`. 

```{r eval=FALSE}
# Make a heatmap of gene expression means and cell-types

```

What do you think? Do these results make sense?

If you'd like to make a more customizable heatmap, we recommend using the 
package `ComplexHeatmap`. 
They provide [excellent vignettes](https://jokergoo.github.io/ComplexHeatmap-reference/book/).

## Plot the cells' PCA scores and color code by marker gene expression

Another way to examine the relationship of genes and cell-types is to make a 
PCA plot, like we did previously, but color the scatterplot points according 
to one of the marker gene's expression. 

Perform PCA on `sce_norm_df` by using the `prcomp` function (hints: you will need 
to transpose the data and tell R to ignore the gene column - try a negative index). 

```{r}
# Perform PCA on the samples of sce_norm_df
pca_res <-
```

Use the following code chunk to inspect your `pca_res` object with your PCA 
results. 
Remember that `x` labeled entry in this object contains the cells' PCA scores. 
Do the number of scores you have correspond to the number of cells you have?
You can use this chunk to check. 

```{r}

```

Make the PCA results into a data.frame that is more easily used by `ggplot2`. 
This data.frame should include the cell-type information, the first two PC scores, 
and `marker_data`. 
Call this data.frame `pca_res_df`.

```{r}
# Make a dataframe that includes the cell-type information, the first two PC scores, 
# and `marker_data`
pca_res_df <- 
```

Use your `pca_res_df` to make a scatterplot of the PCA results, coloring cells
by the expression of a marker gene, of your choosing. 
Mark your cell-types by making the points different shapes.
We've given you a template below with <FILL_IN_THE_BLANKS>. 

```{r eval=FALSE}
# Plot expression of select genes on PCA
# Fill in the blanks on this template ggplot
ggplot2::ggplot(<DATA_FRAME>, # What data.frame should be here?
                ggplot2::aes(x = <X_DATA>, # What do we want plotted on the X-axis? 
                             y = <Y_DATA>, # What do we want plotted on the Y-axis?
                             color = <GENE_ID>, # What do we want to color the points by?
                             shape = <CELL_TYPE>)) + # What do we want to designate
                                                     # the shapes of the points by?
  # Use this line to specify that we want a scatterplot
  # Use this line to add a title for this graph
  ggplot2::scale_shape_manual(values = c(4, 8, 15, 16, 17, 18, 21)) # Since we 
  # have more than 6 cell-types we have to specify the shapes with this step
```

After you have successfully made this plot, repeat this `ggplot` template for 
the other genes in your marker list. 
Make more code chunks as you need them.

```{r}

```

### Advanced topic assignment: 
Make the ggplot2 code chunk you've used above into its own function so you could
apply it more easily. 

### Session info

```{r}
sessionInfo()
```
