---
title: "Exercise: Gene markers in single-cell data"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

In this exercise, we will be using the This [Darmanis data](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) we normalized and filtered 
in `01-normalizing_scRNA`. 

This is glioblastoma data that was Fluorescence-Activated Cell sorted 
and then was processed by paired-end sequencing using Smart-seq2 protocol 
[(Darmanis et al. _Cell Reports._ 2017.).](https://www.ncbi.nlm.nih.gov/pubmed/29091775).

Our objective for this exercise is to investigate the expression of data by 
cell-types. 
We will do this first by extract the gene data and making density plots.
Then, we will calculate the marker gene means by cell type and make a heatmap.
Lastly, we will make a PCA plot but color the points by the gene data. 

## Setup 

We'll be using the `%>%` operator and we will need to set the seed so our results
are reproducible.

```{r}
# Set the seed
set.seed(1234)

# Load the pipe
`%>%` <- dplyr::`%>%`
```

Use the template code chunk below to make a new directory where you can save plots.

```{r eval=FALSE}
plots_dir <- <DIRECTORY_NAME>
if (!dir.exists(<FILL_IN_THE_BLANK>)) {
  dir.create(<FILL_IN_THE_BLANK>)
}
```

## Import and set up the data and metadata

Use `readr`'s function for reading RDS files. 
Import the `SingleCellExperiment` object that we stored in "sce_norm.RDS" in the
`data` folder. 
Remember to use `file.path` and `::` to call any packages that you might use that
you haven't imported. 

```{r}
sce_norm <- 
```

Use the empty code chunk below to explore `sce_norm`.

```{r}

```

Here we are extracting the gene names from `sce_norm`.

```{r}
# Extract the gene names 
genes <- rowData(sce_norm)$value
```

Use the following code chunk to extract the normalized gene matrix and save it
as a data.frame called `sce_norm_df`, where the first column is called "genes"
and has the genes we extracted above.

```{r}
# Extract the normalized data as a data.frame and have a genes column
sce_norm_df <- 
```

Load in the metadata that we saved in a TSV file called 
"gbm_filtered_metadata.tsv" in the "data" folder. 

```{r}
metadata <- 
```

Use the empty code chunk below to get investigate the variables in `metadata`.
Then extract the cell-type labels as their own object, made into a `factor`,
named `cell_types`.

```{r}
cell_types <-
```

## Set up gene marker data

For this exercise, we want analyze specific genes and how they are expressed
amongst the cell types in our dataset. 

Use https://www.genecards.org/ and pick another more gene of your own liking to add
to this analysis. Add your chosen gene in the same format we have started for 
you but replacing our <FILL_IN_THE_BLANKS>. 

In a later module, we will show you how to convert Ensembl IDs to other kinds of 
gene IDs, however, for this exercise, we will only need a few genes and we've 
already set some up for you. 

```{r eval=FALSE}
# Make a data.frame that contains gene symbols and their associated Ensembl IDs
markers <- data.frame(symbols = c("CD45", # Marker for macrophages and microglia
                                  "MOG", # Myelination gene
                                  "SOX9", # Gene expressed in neurons
                                  <GENE_SYMBOL>),
                      ensembl = c("ENSG00000081237", 
                                  "ENSG00000204655", 
                                  "ENSG00000125398",
                                  <ENSEMBL_GENE_ID>)
                      )
```

Depending on the gene you chose, it may not be in our filtered matrix, so you 
may need to try a few. 
You can use a `%in%` operator to check if the gene you are looking for is in 
our filtered gene matrix (Remember it has to be an Ensembl Gene ID e.g.`ENSG`,
and it needs to be in quotes).

```{r eval=FALSE}
# Use the phrase below to find out if a gene you are interested in is in sce_norm_df
<ENSEMBL_GENE_ID> %in% sce_norm_df$genes
```

Use this code chunk to look at what `markers` looks like.

```{r}

```

Let's build a `dplyr` pipe to create a `data.frame` with the data for our marker 
genes and add on `cell_type` vector as it's own column.
Remember that each step in your pipe needs to be followed by the `%>%` operator.
Also remember that we haven't loaded the `dplyr` library so remember to use the
`::` if needed. 

```{r}
# Make a data.frame with the genes in `markers` only
marker_data <- sce_norm_df %>%
  # Use this line to filter the data to only the marker genes we care about.
  # Use this line for tibble::column_to_rownames function to hide the gene names
  # Use this line to transpose this data
  as.data.frame() %>% # Need to turn this back into a data.frame after we transposed
  # Use this line to add a column with the information in `cell_types`
```

## Plot the genes as a density plot by cell type

Use the following `ggplot` template to make a density plot by cell-type for one
of your genes of interest (hint: use the `marker_data` we just made).
Remember that for each step in making a ggplot, you need to add a `+`.

```{r eval=FALSE}
# Plot the gene data by cell type using `facet_wrap`
ggplot2::ggplot(<DATA_FRAME>, ggplot2::aes(x = <GENE_DATA_COLUMN>)) + # Choose a gene you'd like to plot
  ggplot2::geom_density(fill = <COLOR_NAME>) + # Choose a color you'd like to use
  ggplot2::facet_wrap(~ <CELL_TYPE_INFO>) # Use cell types in this function to separate the 
                                          # density plots by their cell-types
```

## Make a heatmap of gene means by cell-type

Here we will calculate the means of these marker genes based on the cell-type 
labels we have. If you don't have cell-type labels, you could also calculate 
your means based on a clustering analysis - we will be doing more clustering 
analyses in the machine learning module. 

Let's build a `dplyr` pipe to calculate means of our marker genes by cell-type.

```{r}
# Build a pipe to calculate the means based on the cell types
marker_means <- marker_data %>%
  # Use this line to use a dplyr function to group the data by their cell types 
  # Use this next line to use a dplyr function to summarize to a mean
  # Use the tibble::column_to_rownames hide the cell-type column
```

Let's change the names to the symbols since they are easier to read when we make
the heatmap
 
```{r eval=FALSE}
# Change the column names to the gene symbols
colnames(marker_means) <- 
```

Use `pheatmap::pheatmap` to make a heatmap of cell-type mean expression of these
genes of interest. Use the argument `scale = "row"`. 

```{r eval=FALSE}
# Make a heatmap of gene means and cell types

```

What do you think? Do these results make sense? 

## Plot the PCA scores with the marker expression color coding

Another way to examine the relationship of genes and cell-types is to make a 
PCA plot, like we did previously, but color the scatter plot points according 
to 

Perform PCA on `sce_norm_df` by using the `prcomp` function (hints: you will need 
to _t_ranspose the data, and tell R to ignore the gene column). 

```{r}
# Perform PCA on the samples of sce_norm_df
pca_res <-
```

Use the following code chunk to inspect your `pca_res` object with your PCA 
results. 
Remember that `x` labeled entry in this object contains the PCA scores. 
Do the number of scores you have correspond to the number of samples you have? 

```{r}

```

Make the PCA results into a data.frame that is more easily used by `ggplot2`. 
This data.frame should include the cell_type information, the first two PC scores, 
and the `marker_data`. Call this data.frame `pca_res_df`.

```{r}
# Make a dataframe that includes the cell_type information, the first two PC scores, 
# and the marker_data
pca_res_df <- 
```

Use your `pca_res_df` to make a scatterplot of the PCA results, using data for 
the genes in your marker list to color them by. 
Mark your cell types by making the points different shapes
We've given you a template below with <FILL_IN_THE_BLANKS>. 

```{r eval=FALSE}
# Plot expression of select genes on PCA
# Fill in the blanks on this template ggplot
ggplot2::ggplot(<DATA_FRAME>, # What data.frame should be here?
                ggplot2::aes(x = <X_DATA>, # What do we want plotted on the X-axis? 
                             y = <Y_DATA>, # What do we want plotted on the Y-axis?
                             color = <GENE_ID>, # What do we want to color the points by?
                             shape = <CELL_TYPE>)) + # What do we want to designate
                                                     # the shapes of the points by?
  # Use this line to specify that we want a scatterplot
  # Use this line to add a title for this graph (if you want)
  ggplot2::scale_shape_manual(values = c(4, 8, 15, 16, 17, 18, 21)) # Specifying shapes
```

After you have successfully used this 
Repeat this `ggplot` template for the other genes in your marker list. 
Make more code chunks as you need them.

```{r}

```

### Advanced topic assignment: 
Make the ggplot2 code chunk you've used above into it's own function so you could
apply it more easily. 

### Session Info

```{r}
sessionInfo()
```
