---
title: "scRNA-seq Dimension Reduction"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2020**

In this notebook, we'll try out some dimension reduction techniques on single-cell RNA-seq data. 

Visualizing highly dimensional data is a common challenge in genomics, and especially with RNA-seq data.
The expression of every gene we look at is another dimension describing a sample.
When we also have hundreds or thousands of individual samples, as in the case of single-cell analysis, figuring out how to clearly display all of the data in a meaningful way is difficult. 

A common practice is to common to use dimension reduction techniques so all of the data is in a more easily manageable form for plotting, clustering, and other downstream analyses. 

## Set Up 

```{r}
# Load libraries
library(ggplot2)
library(scater)
library(scran)

# Magrittr pipe
library(magrittr)

# Setting the seed for reproducibility
set.seed(12345)
```

For this module, we are going to use the dataset that we processed part of in 
`04-tag-based_scRNA-seq_processing.Rmd`

## Directories and files

```{r filepaths}

# Path for the alevin results
alevin_file <- file.path("data", "tabula-muris", "alevin-quant", 
                          "10X_P4_3_full", "alevin", "quants_mat.gz")

# Path to the metadata file 
metadata_file <- file.path("data", "tabula-muris", 
                           "TM_droplet_metadata.csv")

# Mitochondrial gene table
mito_file <- file.path("data", "tabula-muris", 
                       "mm_mitochondrial_genes.tsv")
```


```{r tximport_sce}
tm_txi <- tximport::tximport(alevin_file, type = "alevin")
tm_sce <- SingleCellExperiment::SingleCellExperiment(list(counts = tm_txi$counts))
```

```{r read_metadata}
tm_metadata <- readr::read_csv(metadata_file, guess_max = 10000) %>%
  dplyr::filter(channel == "10X_P4_3")
```

As we did before with the Smart-seq2 data, we will want to filter and normalize this data with `scran/scater`.
While we did the filtering manually before, we can actually do most of the same analyses with `scran`.

First we will want to add a bit of extra information to the `SingleCellExperiment` object.

```{r mitogenes}
mito_genes <- readr::read_tsv(mito_file) %>%
  dplyr::filter(gene_id %in% rownames(tm_sce)) %>%
  dplyr::pull(gene_id)


```


```{r calculateQC}
tm_sce <- scater::addPerCellQC(tm_sce, 
                               subsets = list(mito = mito_genes)) 

cell_qc_df <- as.data.frame(colData(tm_sce)) 
```

Make some plots of the QC data to decide on cutoffs for your cells.

```{r qc_plots, live = TRUE}
ggplot(cell_qc_df, 
       aes(x = sum, y = detected, color = subsets_mito_percent)) +
  geom_point() +
  scale_color_viridis_c() +
  scale_x_log10() +
  scale_y_log10() +
  geom_hline(yintercept = 500, color = "red")

ggplot(cell_qc_df, 
       aes(x = subsets_mito_percent)) +
  geom_histogram() + 
  geom_vline(xintercept = 20, color = "red")
```

Looking at the data, it seems like a reasonable cutoff might be 500 detected genes, and a mitochondiral percentage less than 20%

Using these QC data, we can now create a subset of the `SingleCellExperiment` object, keeping only the columns (samples) that pass our QC thresholds:

```{r qcfilter, live = TRUE}
qc_pass <- (tm_sce$detected > 500) & (tm_sce$subsets_mito_percent < 20)
tm_sce_filtered <- tm_sce[, qc_pass]
```

We will also filter the genes to only the ones detected in at least 5 cells, and with a mean count > 1

```{r gene_qc}
tm_sce_filtered <- scater::addPerFeatureQC(tm_sce_filtered)
detected <- rowData(tm_sce_filtered)$detected > 5
expressed <- rowData(tm_sce_filtered)$mean > 1

# filter the genes (rows) this time
tm_sce_filtered <- tm_sce_filtered[detected & expressed, ]
```

### Normalize

Now we will perform the same normalization steps we did with the Smart-seq2 data:

```{r normalize}
# Cluster similar cells
qclust <- scran::quickCluster(tm_sce_filtered)

# Compute sum factors for each cell cluster grouping.  
tm_sce_filtered <- scran::computeSumFactors(tm_sce_filtered, clusters = qclust)

# Normalize and log transform. 
tm_sce_filtered <- scater::logNormCounts(tm_sce_filtered)
```


![**Status of the current dataset**](diagrams/tag-based_3.png)

## Principal Components Analysis

The first dimensionality reduction algorithm we will use is PCA, which we have already used!
The idea of PCA is that it finds the axes which explain the greatest amount of variance in the data.


```{r PCA}
# Run PCA
pca_tab_mur <- calculatePCA(tm_sce_filtered)

# Make a dataframe with these PCA scores
pca_tab_mur <- data.frame(pca_tab_mur)
```

Let's plot our first two principal components and label our cells with what 
tissue they are from.

```{r Plot PCA}
# Plot this with ggplot2 and label the points with cell types
ggplot(pca_tab_mur, 
       aes(x = PC1, y = PC2)) + 
  geom_point() +
  colorblindr::scale_color_OkabeIto()
```

## t-SNE experiment

t-SNE is a popular dimension reduction technique that has become even more 
popular with single-cell RNA-seq data analysis. 
It would be a good use of our time to explore t-SNE parameters and how they 
affect the outcomes.
First, let's run our data through t-SNE on default parameters and see what our 
data looks like with a `ggplot2` scatterplot.
Here we will be using the R package, `Rtsne` to perform t-SNE. 
Although packages like `scater` have built in methods for performing dimension
reduction (`scater::runTSNE`), we want to show you a method that can be applied 
to most any data so long as it is in a matrix.

```{r t-SNE}
# Run t-SNE using the Rtsne package
tsne_tab_mur <- Rtsne::Rtsne(transposed_data)

# Make this a data.frame so ggplot will like it
# Note that Y is the output coordinates from Rtsne
tsne_data <- data.frame(tsne_tab_mur$Y)
  
# Plot this with ggplot2
ggplot(tsne_data, aes(x = X1, y = X2)) + 
  geom_point() + 
  colorblindr::scale_color_OkabeIto()
```

#### Apply metadata labels to our t-SNE

Let's re-plot this with labels using some of our metadata to see 
how the cells are clustering (or not clustering) by different variables. 
Remember that `tab_mur_meta` has the corresponding information for each of our
cells. 
Below we have set up the cells to be colored by the `tissue` variable. 
Run this, but also feel free to look at what other information we have for these
cells in `tab_mur_meta` by using `str()` and change this to a different variable to 
graph it by.

```{r}
# Plot this with ggplot2, but add the color argument
ggplot(tsne_data, aes(x = X1, y = X2, color = tab_mur_meta$tissue)) + 
  geom_point() +
  colorblindr::scale_color_OkabeIto()
```

Now that we have an idea of what the default parameters look like, let's try
experimenting with the perplexity parameter. 
For us to determine a good number range to use for testing perplexity, let's 
first look up the documentation for `Rtsne::Rtsne` and find out what the 
default for the `perplexity` argument is. 

In order to prepare for this experiment, we will create a *function* that allows
us to rerun this same code chunk easily, but create an argument that allows us
to change one variable: our perplexity variable. 

```{r Make t-SNE plot function}
TsnePlotWrapper <- function(perplexity_param = 30) {
  # Purpose: Run t-SNE and plot the output
  # Args: perplexity_param: a single numeric argument that will change the 
  #                         perplexity variable in the Rtsne function. 
  # Output: a ggplot scatterplot with the two t-SNE coordinates plotted and 
  #         cell-types labeled with data point colors. 

  # Run t-SNE but set the perplexity parameter
  tsne_tab_mur <- Rtsne::Rtsne(transposed_data, perplexity = perplexity_param)

  # Make this a data.frame so ggplot will like it
  tsne <- data.frame(tsne_tab_mur$Y)
  
  # Plot this with ggplot2
  ggplot(tsne, aes(x = X1, y = X2, color = tab_mur_meta$tissue)) + 
    geom_point() + 
    colorblindr::scale_color_OkabeIto()
}
```

Fill in the next five code chunks with the function and the perplexity argument 
you would like to use for each. 
Then run the chunks and compare your output graphs.

```{r Run tSNE 1}
# We've set up the first example for you
TsnePlotWrapper(perplexity_param = 3)
```

```{r Run t-SNE 2}
# Run tSNE_plot_wrapper function for a second time, but use a different perplexity
# argument
```

```{r Run t-SNE 3}
# Run tSNE_plot_wrapper function for a third time, but use a different perplexity
# argument
```

```{r Run t-SNE 4}
# Run tSNE_plot_wrapper function for a fourth time, but use a different perplexity
# argument
```

```{r Run t-SNE 5}
# Run tSNE_plot_wrapper function for a fifth time, but use a different perplexity
# argument
```

#### Some 'big picture' thoughts to take from this experiment: 

1) Different analyses (such as t-SNE) have various limitations for interpretability.
   We found in our experiment that the coordinates of t-SNE output for any given 
   cell might change drastically. 
   This probably means that you shouldn't rely too heavily on the exact values of
   t-SNE's output. 
2) Different analyses also have their strengths.   
   Using cell-type labeling, our experiment illustrated that t-SNE does 
   appear to give some biologically relevant output information for this dataset. 
3) Playing with parameters so you can fine-tune them is a good way to give you more
   information about a particular analysis as well as the data itself. 
   
In summary, if the results of an analysis can be completely changed by changing its
parameters, you should be more cautious when it comes to the conclusions you
draw from it as well as having good rationale for the parameters you choose. 

### For further reading on dimension reduction:  

- [Wattenberg et al](https://distill.pub/2016/misread-tsne/) discuss how to 
use t-SNE properly with great visuals.  
- [Nyguen & Holmes _PLoS Comput Biol_ 2019.](https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1006907&type=printable)
lay out guidelines on choosing dimensions reduction methods.  
- [Becht et al _Nature Biotechnology_ 2018.](https://www.nature.com/articles/nbt.4314) discusses using [UMAP](https://github.com/lmcinnes/umap) for single-cell data.  
- This website explains [PCA visually](http://setosa.io/ev/principal-component-analysis/).  
  
**Single-cell specific dimension reduction methods to consider:**  
- [VASC](https://www.sciencedirect.com/science/article/pii/S167202291830439X)  
- [scVI](https://scvi.readthedocs.io/en/master/readme.html)  

#### Print session info:

```{r}
sessionInfo()
```
