---
title: "scRNA-seq_post_processing"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2019**

In this notebook, we'll perform dimension reduction and clustering analyses of 
single cell RNA-seq data. 

#### Set Up 
For these analyses, we will need Seurat a lot, so we will go ahead and attach it.  

Put the path to the Alevin output here:
```{r Path to seurat}
# Load in seurat object
seurat_norm <- readRDS(file.path("data", "seurat_object.RDS"))

# Read in metadata
sc_meta <- readr::read_tsv(file.path("data", "darmanis_metadata.tsv"))
```

#### Execute PCA with high variance genes
Just like we did with bulk RNA-seq, we will also run PCA with our single cell data. 
However, because single cell data is more skewed as far as gene expression, we 
will do our PCA based on the high variance genes, which should give us a more 
informative analysis.
```{r PCA}
# Run PCA
seurat_norm <- RunPCA(object = seurat_norm, pc.genes = seurat_norm@var.genes,
                      do.print = TRUE, pcs.print = 1:5, genes.print = 5)

# Cluster analyses
seurat_norm <- FindClusters(object = seurat_norm, reduction.type = "pca", 
                            dims.use = 1:10, resolution = 0.6, print.output = 0, 
                            save.SNN = TRUE)
```

```{r Plot PCA}
# Make a dataframe with the metdata and pca data
pca <- data.frame(seurat_norm@dr$pca@cell.embeddings) %>% 
       tibble::rownames_to_column('geo_accession') %>%
       dplyr::inner_join(sc_meta, by = 'geo_accession')

# Plot this with ggplot2
ggplot(pca, aes(x = PC1, y = PC2, color = cell.type.ch1)) + 
  geom_point() +
  colorblindr::scale_fill_OkabeIto()
```

# tSNE experiment
tsne can be a lot more wild than people like to admit. 
It would be a good use of our time to explor tSNE parameters and how they 
affect the outcomes.
For more reading about tSNE parameters see 
[this article](https://distill.pub/2016/misread-tsne/)
```{r}
# Let's see what happens if we try this with a lot of different perplexity 
# parameters
perplexity_params <- c(2, 5, 10, 30, 50, 100)

for (param in perplexity.params) {
  # Run tSNE 
  tsne <- RunTSNE(object = seurat_norm, dims.use = 1:10, do.fast = TRUE, 
                  perplexity = param, k.seed = 10)

  # Make a dataframe with the metdata and pca data
  tsne <- data.frame(tsne@dr$tsne@cell.embeddings) %>% 
          tibble::rownames_to_column('geo_accession') %>%
          dplyr::inner_join(sc_meta, by = 'geo_accession')

  # Plot this with ggplot2
  tsne_plot <- ggplot(tsne, aes(x = tSNE_1, y = tSNE_2, color = cell.type.ch1)) + 
               geom_point() + 
               colorblindr::scale_fill_OkabeIto()
  
  # Print the plot
  print(tsne_plot)
}
```


#### Print session info:
```{r}
sessionInfo()
```
