---
title: "Exercise: Pathway analysis of scRNA-seq marker genes"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

## Introduction to the data set

In this exercise, we'll be working with a dataset that assays liver cirrhosis from [Ramachandran *et al.* (2019)](https://doi.org/10.1038/s41586-019-1631-3).
Specifically, we're going to use the murine data, which consists of hepatic macrophages from mice that were treated with an agent that causes fibrosis and from healthy mice.

You can see how we downloaded, processed, and clustered the data in [this notebook](https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/setup/mouse-liver/prepare_mouse_liver_markers.Rmd).

Running the chunk below will bring up a UMAP plot that includes cluster labels.

```{r include_umap}
knitr::include_graphics(file.path("figures", "mouse_liver_umap.png"))
```

We've prepared the tables of marker genes for you.
If there's one cluster in this plot that piques your interest, use it to practice over-representation analysis and gene set enrichment analysis below!

## Setup

### Libraries

```{r libraries}
library(org.Mm.eg.db)
library(clusterProfiler)
library(magrittr)
```

## What cluster looks interesting to you?

Take a look at the UMAP plot above, where samples are colored and labeled by cluster identity. 

Does any cluster look particularly interesting to you?

We've identified marker genes for you (`scran::findMarkers(..., pval.type = "all")`) and saved the results in `analysis/mouse-liver/markers`.
(You can see how we did that and how we generated that plot [here](https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/setup/mouse-liver/prepare_mouse_liver_markers.Rmd)!)

Take a look at the files available in `analysis/mouse-liver/markers`, read in the marker genes for the cluster you're interested in analyzing in the chunk below and assign them to `markers_df`. 

*We'll use cluster 7 in the solved version, but you don't have to!*

```{r read_in_markers, solution = TRUE}
# Read in the marker genes for your favorite cluster and assign to `markers_df`

```

Use the next chunk to explore what the markers table looks like and what gene identifiers are already in the table.

```{r peek_at_markers, solution = TRUE}
# Make sure you can see what kind of identifier is in the gene column

```

The values in the `gene` column are gene symbols.

## What gene sets do you want to use?

Our choice of gene sets will be influenced by the experimental design of the data assays, our upstream analytical steps, and what we hope to learn about our data.

We're going to be analyzing marker genes (e.g., genes that distinguish clusters) from an experiment examining hepatic macrophages in the context of liver fibrosis. 

### Molecular Signatures Database (MSigDB)

MSigDB contains multiple collections of gene sets that we can access via the `msigdbr` package. 
To see what collections are available, we can run the following chunk:

```{r show_collections}
msigdbr::msigdbr_collections()
```

This table could definitely be more helpful!
You can read more about individual collections [here](https://www.gsea-msigdb.org/gsea/msigdb/collection_details.jsp).

We'll briefly summarize some information about collections you may want to use below.

##### C2 Sub-collection: Canonical pathways

The canonical pathways sub-collection of C2 (prefixed with `CP:` in `gs_subcat`) are generally popular for pathway analysis.
These are curated gene sets assembled by experts.
You may already be familiar with [KEGG (Kyoto Encyclopedia of Genes and Genomes)](https://www.genome.jp/kegg/) or [Reactome](https://reactome.org/) pathways.

These gene sets often capture things like signaling pathways, which may be of interest here!

##### C5 Sub-collection: Gene Ontology 

During instruction, we used `enrichGO()` to perform ORA with biological process (BP) terms from the Gene Ontology, but we could have used `enricher()` with just the `GO:BP` subcategory gene sets from `msigdbr` just as well.

GO terms have a wide range in terms of their specificity.
Some of the BP terms were specific enough to give us an idea about the cell type identity of clusters we examined during instruction.

In this context, it might be interesting to understand if a particular cluster expresses genes that are indicative of a biological process that is considered pro-fibrotic.

##### C7 

During instruction, we used the C7 immunologic signature gene sets to analyze a dataset that was comprised in part of immune cells with GSEA.
Because of the experimental design, we might expect that gene sets that tell us about myeloid cell phenotypes could be useful in this context.

##### H

Hallmark gene sets, which were assembled for specific use with GSEA, are derived by identifying overlaps between existing, often redundant gene sets.

From the [collection description](https://www.gsea-msigdb.org/gsea/msigdb/collection_details.jsp#H):

> Hallmark gene sets summarize and represent specific well-defined biological states or processes and display coherent expression.

From the table above, we can see that there are only 50 Hallmark gene sets.
The more gene sets we test, the higher our multiple hypothesis testing burden will be.

#### `msigdbr()` supported species

Beyond multiple collections, the `msigdbr()` package also supports multiple organisms. 
(Run the next chunk to see which ones!)

```{r show_species}
msigdbr::msigdbr_species()
```

We're working with a mouse dataset, so we would need to set `species = "Mus musculus"` in our call to `msigdbr()` to return only the mouse gene sets.

### CellMarker

[CellMarker](http://bio-bigdata.hrbmu.edu.cn/CellMarker/) is a manually curated database of cell marker data for human and mouse ([Zhang *et al.* 2018](https://doi.org/10.1093/nar/gky900)).
Studies/evidence of the following nature were included in the manual curation process ([Zhang *et al.* 2018](https://doi.org/10.1093/nar/gky900)):

* scRNA-seq
* Flow cytometry
* Immunostaining

There are both gene and protein markers.
If there are CellMarker gene sets that distinguish different macrophage phenotypes, that information may be really useful here!

### Obtain your gene sets data frame

Depending on what you think is most interesting, you'll either need to obtain relevant gene sets from `msigdbr` or to read in the TSV of CellMarker data. 
(The CellMarker _mouse_ data is available at `gene-sets/CellMarker_cleaned_mouse_markers.tsv`.)

As a reminder, `msigdbr()` will return _lots_ of gene sets; we recommend only working with the collection or subcategory that you are interested in.
To obtain the KEGG gene sets, for example, you would use the arguments `category = "C2"` and `subcategory = "CP:KEGG"` when calling `msigdbr()` or filter (e.g., use `dplyr::filter()`) your data frame from `msigdbr()` to `gs_subcat == "CP:KEGG"` after the fact.

*We're going to use KEGG gene sets from `msigdbr` in the solved version, but you don't have to!*

```{r setup_gene_sets, solution = TRUE}
# Call msigdbr() or read in the TSV to obtain the gene sets you want to use!
# Make sure you're using mouse gene sets

# Do you need to use `dplyr::filter()` here to subset to specific MSigDB 
# collections?
```

Take a look at the first few rows of `gene_sets_df`.

```{r view_gene_sets_df}
head(gene_sets_df)
```

Does this data frame have two columns, where the first one contains a gene set identifier or name and the second column contains gene symbols (the gene identifier we have in `markers_df`)?

If not, use the next chunk to subset `gene_sets_df` to only the gene set name and gene symbol columns.

```{r select_gene_set_columns, solution = TRUE}
# dplyr::select() will select columns based on their names
# Call the updated data frame `gene_sets_df`

```

## ORA

Let's perform over-representation analysis on the marker genes from the cluster you're interested in!

### Genes of interest

First, we need to identify the genes we want to use as the genes of interest.
If we'd like to use ORA to help interpret our clusters, we generally want to find genes with high expression in our cluster of interest and with small FDR values.
As a reminder, these FDR values shouldn't be used to determine statistical significance because of the circularity involved in calculating them.
Nonetheless, we can use them for ranking!

Use the next chunk to get top ranked genes to use as input to ORA and call that vector `genes_of_interest`.

```{r goi_list, solution = TRUE}
# Get a vector of genes called `genes_of_interest`
# You'll probably want to filter by summary log fold change, sort by FDR, and 
# pick some number of genes

```

### Background set

The background set we use for ORA should include all genes that we could reliably measure.
The marker gene tables we created include all of the genes that passed the pre-normalization filtering in our single cell analysis, so we will want to include all of these genes as our background set. 

Use the next chunk to get a vector of genes to use as a background set called `background_set`.

```{r background_set, solution = TRUE}
# Create a vector of all gene symbols in the marker genes table called 
# `background_set`

```

### Perform ORA

Now we're ready to perform ORA using the `clusterProfiler` package.
The `enricher()` function will take any generic set of gene sets, provided they are in the correct format!
Earlier, we checked to make sure `gene_sets_df` contained 2 columns: gene set name/ID and gene symbol, so we should be all set on the formatting front.
You will pass this data frame to the `TERM2GENE` argument of `enricher()`.

*In the solved version, we'll use the data frame of KEGG gene sets from `msigdbr` we extracted above!*

In the next chunk, run ORA with your function and gene sets of choice.
Assign the results to `ora_results`.

```{r enrich, solution = TRUE}
# Run ORA using enricher() from the clusterProfiler package
# Pass your gene sets data frame to the TERM2GENE argument
# Save as ora_results

```

### Visualizing ORA results

Let's use the next chunk to visualize the results.
We can use `enrichplot::dotplot()`, `barplot()`, or check out [the `clusterProfiler` chapter on visualizations](http://yulab-smu.top/clusterProfiler-book/chapter12.html).

Note that these visualizations will only work if you have significant results!
Run the next chunk to view the results table.

```{r view_ora, eval = FALSE}
View(ora_results@result)
```

Are there significant results?
If so, use the next chunk for the visualization itself!

```{r viz_ora, solution = TRUE}
# Visualize `ora_results`

```

## GSEA

Let's run Gene Set Enrichment Analysis (GSEA) next!

### Prep ranked statistics

First, let's use the chunk below to check if we have any duplicated gene identifiers in the data frame of marker genes.

```{r check_dups}
any(duplicated(markers_df$gene))
```

Since there are no duplicate identifiers, we can go ahead and use the next chunk to prep the sorted vector of statistics (`summary.logFC`).

We'll note here that summary log fold change values have some _limitations_.
Summary log fold change is collapsing the effect sizes into a single (summarized!) value.
When we use `pval.type = "all"` like we did upstream, this value will reflect the fold change value of the pairwise comparison with the largest p-value (e.g., weakest differential expression) ([ref](https://rdrr.io/bioc/scran/man/combineMarkers.html)).
In practice, these values may reflect the fold change for a gene relative to the cluster that is _most similar_ to the cluster we are analyzing. 
This also means the summary log fold change values are sensitive to the cluster assignments and are likely to change between runs if clustering results are unstable.

```{r lfc_vector, solution = TRUE}
# Grab a named vector of summary log fold changes

# Order the vector

```

### Run GSEA

Now we're ready to perform GSEA using the `clusterProfiler` package with `GSEA()`.
`GSEA()` will take any generic set of gene sets, provided they are in the correct format (the same format used for `enricher()`).

Use the next chunk to run GSEA and assign the results to `gsea_results`.

```{r run_gsea, solution = TRUE}
# Run GSEA using from the clusterProfiler package
# Pass your gene sets data frame to the TERM2GENE argument
# Save as `gsea_results`

```

### Visualizing GSEA results

First, check and see if you got any significant results by running the chunk below and looking at the results table.

```{r view_gsea, eval = FALSE}
View(gsea_results@result)
```

Next, choose a gene set to plot using `enrichplot::gseaplot()`.
Even if there were no significant results, you can still create a plot by passing the gene set `ID` to `geneSetID` argument (don't forget to use quotes!) and you'll probably want to pass something descriptive to the `title` argument of `enrichplot::gseaplot()`.

*In the solved version, we'll use `geneSetID = "KEGG_CELL_ADHESION_MOLECULES_CAMS"`.*

```{r gseaplot, solution = TRUE}
# Use enrichplot::gseaplot() to visualize gsea_results

```

## Wrapping up & exploring more!

* Did the pathway analysis you performed give you any insight into the potential cell type or phenotype of the cluster you analyzed?
* Did you get similar results for ORA and GSEA? Why do you think that may or may not be?

If you're interested in working with this dataset more, we've made the `SingleCellExperiment`-compatible object, complete with PCA and UMAP available in `data/mouse-liver/normalized/healthy_cirrhotic_mouse_liver_normalized.rds`.
You'll need to use `readr::read_rds()` to read in this file!

## Session Info

```{r sessionInfo}
sessionInfo()
```
