---
title: "Single Cell Exercise: Cell type assignment"
output:
  html_notebook:
    toc: true
    toc_float: true
---

In this exercise notebook we will work with a glioblastoma dataset from a 71 year old male patient that is available as an example dataset from 10x Genomics.
This sample was processed by 10x Genomics using a 3' RNA kit (v3.1), sequenced, and quantified with Cell Ranger 6.0.
Further details about the sample and processing can be found on the [10x website](https://www.10xgenomics.com/resources/datasets/2-k-sorted-cells-from-human-glioblastoma-multiforme-3-v-3-1-3-1-standard-6-0-0).

Previous bulk RNA-seq and scRNA-seq studies of glioblastoma samples revealed that glioblastoma cells exist in 4 main cellular states that resemble normal neurodevelopment ([Patel _et al._ 2014](https://doi.org/10.1126/science.1254257), [Couturier _et al._ 2020](https://doi.org/10.1038/s41467-020-17186-5), [Neftel _et al._ 2019](https://doi.org/10.1016/j.cell.2019.06.024)).
Additionally, these tumors are highly heterogeneous and cells in these different cellular states can all be found within the same tumor, however the frequency of each state is variable across tumors.

From [Neftel _et al._ (2019)](https://doi.org/10.1016/j.cell.2019.06.024):

> We find that malignant cells in glioblastoma exist in a limited set of cellular states that recapitulate (i) neural progenitor-like (NPC-like), (ii) oligodendrocyte-progenitor-like (OPC-like), (iii) astrocyte-like (AC-like), and (iv) mesenchymal like (MES-like) states.

In addition to malignant cells, scRNA-seq studies of glioblastoma have also captured presence of normal brain cells (e.g., endothelial cells, oligodendrocytes, microglia, neurons).
Let's see what cell types or cell states we are able to identify in this particular glioblastoma dataset.

We will start with a version of this data that has already undergone filtering, normalization, dimension reduction, and clustering.
This exercise notebook will give you the opportunity to practice different approaches to cell type assignment and to compare assignments for a given dataset.

---

In this exercise notebook you will,

- Part A: Annotate cell types using `SingleR`
- Part B: Annotate cell types with `SingleR` using a `Metacell`-like approach
- Part C: Explore expression of cell-type specific marker genes

## Part A: Annotate cell types using `SingleR`

In this first part, we will assign cell types for the glioblastoma sample using the [`SingleR` package](https://bioconductor.org/books/3.16/SingleRBook/).
As a reminder, `SingleR` requires a reference dataset containing known cell types that are expected to be present in the glioblastoma sample.
Selecting the reference dataset is the most critical step, because selecting a dataset without cell types we expect to be present means we won't be able to identify those cell types in our sample of interest.
Worse, we might assign cells inaccurately to the "closest match" cell type in our reference dataset, even if it is not a particularly good match!
Here you will assign cell types using two different reference datasets and then compare your findings.

### Set up

Before we can assign cell types, we will need to set up our analysis by loading libraries and defining relevant directories and file paths.

```{r setup}
# set the seed
set.seed(12345)

# load libraries
library(SingleCellExperiment)
library(SingleR)
library(ggplot2)
```

We are going to be reading in a file containing the filtered and normalized glioblastoma data that we have normalized and saved as an SCE object.
For details on the processing that we performed, you can refer to the [notebook we used for processing](https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd)

To do that we will need to define the file path, which we will do in two steps with two variables:

1. `data_dir`: This directory holds the normalized output from the intro instruction notebook, `data/glioblastoma-10x/normalized`.
2. `sce_file`: This is the path to the `glioblastoma_normalized_sce.rds` file containing the filtered and normalized glioblastoma data

```{r input paths}
# data directory
data_dir <- file.path("data", "glioblastoma-10x", "normalized")

# input SCE file
sce_file <- file.path(data_dir, "glioblastoma_normalized_sce.rds")
```

Our output from this notebook will be a table of per-cell information and include the cell type assignments we have made throughout this notebook.
This file should be named `glioblastoma_cellinfo.tsv` and saved to the `analysis/glioblastoma` directory.
In addition to defining the path to the file (`cellinfo_file`), you will need to define the path to the output directory (`output_dir`) and check to make sure that directory exists.
If it doesn't exist, create it!

```{r output paths, solution = TRUE}
# define the output directory

# create the output directory

# define the output file

```


Next you will need to read in the input `rds` file and save it to a variable called `sce`.

```{r read sce, solution = TRUE}
# read in sce object

```

Print out the `sce` object and take a look at the output.
How many cells are present in this dataset?

```{r print sce, solution = TRUE}
# print out the sce object

```

Before we saved this SCE object, we also performed graph-based clustering and assigned each cell in our sample to a cluster.
In the next chunk, take a quick peek at the cluster assignments by using `scater::plotReducedDim()` to plot UMAP results and color each cell by cluster assignment.
(Hint: Cluster assignments can be found in the `nn_cluster` column of the SCE's `colData` table.)

```{r cluster umap, solution = TRUE}

```

Sometimes, but definitely not always, cluster assignments could be related to cell types.
Based on this plot, it looks like we have some clear populations that clustered separately.

### `SingleR` with Human Primary Cell Atlas Data

Now we are ready to annotate the cell types present in our sample!

Before we can use `SingleR`, we will first need to pick a reference dataset.
As discussed during instruction, `SingleR` will first identify a set of marker genes that differentiates cell types from each other in the reference dataset.

Here we are examining a glioblastoma sample, so as discussed earlier we expect it to include cell types typically found in the brain (e.g., endothelial cells, oligodendrocytes, microglia, neurons).
We also expect to cells that may resemble the different states present in glioblastoma, astrocytes, oligoprogenitors, mesenchymal cells, and neural progenitors.
Ideally we would like to use a reference dataset that contains these cell types, so that we can identify them.

However, our sample is a tumor sample, so there are likely to be both malignant and non-malignant cells in this sample.
Malignant cells tend to have distinct gene expression patterns from normal cells, so it is likely that the marker genes that differentiate malignant cells are not the same as those that differentiate normal cells.
Therefore, using a reference dataset with only normal cell types may pose a challenge for annotating tumor samples, which means `SingleR` may not be as effective at assigning cell types in this context.
This is why it is very important to consider your reference dataset when performing cell type annotation with tools like `SingleR`.

The first dataset that we will use comes from the `celldex` package, the Human Primary Cell Atlas (HPCA).
The HPCA reference consists of publicly available microarray datasets derived from human primary cells ([Mabbott et al. 2013](https://doi.org/10.1186/1471-2164-14-632)).
This dataset is mostly comprised of blood cell types, but also contains some other tissue types, such as the brain.

Go ahead and load the dataset, saving it to the variable, `primary_atlas`.
We will also use the `ensembl=TRUE` option here to save the Ensembl ids as the rownames of the SCE object, matching the format of the glioblastoma SCE object.
Of course, if you already know the Ensembl ids by heart, you can perform this manually as well.

```{r celldex}
# load HPCA reference data
primary_atlas <- celldex::HumanPrimaryCellAtlasData(ensembl = TRUE)
```

After loading in the dataset, let's first take a look at the types of cells that are present.
As a reminder, the `celldex` datasets contain three columns: `label.main`, `label.fine`, and `label.ont`.
For our purposes here, we are interested in the `label.main` annotations.

```{r count hpca celltypes, solution = TRUE}
# what types of cells are found in the atlas?

```

In looking at this output you should notice that most of the cell types are immune cells, but we also see cell types that would be present in the brain â€“ neurons, astrocytes, and neuroepithelial cells.
Given we are using a glioblastoma dataset, we expect these cell types may be present in our dataset.
However, this doesn't contain the entire set of expected cell types as we mentioned earlier, so we still may miss some cell type assignments or incorrectly assign cells.
It's important to think about what type of cells we expect to see when thinking about selecting our references!

In this exercise, we will use the two step process to run `SingleR`.
First, train the model with the `SingleR::trainSingleR()` function, saving the output to the variable `hpca_model`.
Be sure to use the HPCA dataset as the reference and the labels present in the `label.main` column.
(Hint: If you want this to go faster, don't forget to use the `BBPARAM` option we used during instruction!)

```{r hpca build model, solution = TRUE}
# build model with hpca dataset as reference

```

In the next chunk, use the model you trained to classify cell types using `SingleR::classifySingleR()`.
Save the result to a variable named `hpca_result`.

```{r hpca classify cells, solution = TRUE}
# classify cell types with hpca model

```

Let's explore our results!
Use the `SingleR::plotScoreHeatmap` function to take a look at the cell type assignments identified by `SingleR`.

```{r hpca singleR heatmap, solution = TRUE}
# plot singleR heatmap

```

What cell types do you see?
Are these cell types what you would expect from a glioblastoma sample?

In the below chunk add the cell types identified by `SingleR` (the pruned labels stored in `pruned.labels`) to the SCE object, saving them to a column in the SCE `colData` named `celltype_hpca`.

```{r hpca store cell types, solution = TRUE}
# add the labels to the sce object

```

Then, create a UMAP plot, coloring the cells by the identified cell types now present in the `celltype_hpca` column of the `colData`.

```{r hpca plot cell types, solution = TRUE}
# plot UMAP with HPCA cell types

```

It looks like most of the cells belong to only a few groups and we see a lot of astrocytes, macrophages, and monocytes.
We may also have a small cluster of T cells.
Can you make any conclusions about which of the 4 cell states found in glioblastoma (OPC-like, NPC-like, MES-like, and AC-like) may be dominant in this particular sample?

This plot is a bit messy though, probably because we have a lot of cell types, some of which may only be present because a few cells were assigned to that cell type.
Let's look at just the top 5 represented cell types.
To do this we can use the `forcats` package again and utilize the [`fct_lump_n()`](https://forcats.tidyverse.org/reference/fct_lump.html) function, which will lump all groups except those that are the most frequent.
Here we use the argument `n = 5` to specify that we want to keep the top 5 most common categories (here, cell types).

```{r top cell types}
# make a new vector of collapsed labels
hpca_collapsed_labels <- hpca_result$pruned.labels |>
  # only keep top 5 groups
  forcats::fct_lump_n(n = 5)

# make a table of the new collapsed labels
table(hpca_collapsed_labels)
```

As expected from the earlier plot, using the HPCA dataset as a reference, most of our cells appear to be astrocytes, macrophages, and monocytes.
We also have a very small group of T cells that may be present.

### `SingleR` with a brain specific dataset

As we hinted at earlier, one major caveat of using the HPCA data as a reference dataset for the glioblastoma sample is that it mostly contains immune cells with limited types of brain cells.
It also includes only normal cells and no malignant cell types.

It would probably be more appropriate to use a reference dataset that is both specific to brain tissue and contains a larger group of brain cell types that could be present in our sample.
Unfortunately, none of the other available datasets in the `celldex` package are brain-specific, but that's ok!

One of the benefits of using `SingleR` is that we can actually use pretty much any gene expression data with labeled cell types as a reference dataset, including other single-cell expression data.
You could find another publicly available dataset with cell type annotations and use that as your reference dataset.
Or you could even use another SCE object that you have already annotated for a similar project that contains overlapping cell types with your target dataset (in this case the glioblastoma sample) and use it as the reference dataset to train your model.

In both scenarios, you are using a pre-labeled dataset to annotate another unlabeled dataset.

**A word of caution.**
Although you can use almost any gene expression data as your reference dataset, it is still important to remember that there are multiple ways to annotate cell types, and annotations are often error-prone.
Be sure that you trust the cell type annotations you use as a reference, and that you understand how cell types were annotated in that reference.

Here, we will use a single-cell RNA-seq reference dataset that is available as part of the `scRNAseq` package.
The full list of datasets available in the package can be found in [this `scRNAseq` vignette](https://bioconductor.org/packages/3.16/data/experiment/vignettes/scRNAseq/inst/doc/scRNAseq.html#available-data-sets).
The dataset we will use is from a single-cell RNA-seq study of brain tissue from [Darmanis _et al._ (2015)](https://doi.org/10.1073/pnas.1507125112).
In this study, they were able to identify and label all major cell types present in the brain and further characterize neurons based on their differentiation stage.

Let's start by reading in the dataset, again using the `ensembl=TRUE` option to use the Ensembl ids as the rownames of the SCE object.

```{r brain ref}
# load in brain ref
brain_sce_ref <- scRNAseq::DarmanisBrainData(ensembl = TRUE)
```

Go ahead and explore the reference that we grabbed.
Print it out, and then take a look at the `colData` to figure out which column contains the cell type assignments.

```{r explore brain ref, solution = TRUE}
# print it out

# find which column has the cell types

```

How many cells are present in this data?
Which column contains the cell type annotations for this reference dataset?

Let's take a look at the distribution of cell types that are found in this dataset.
How do these compare to the cell types found in the HPCA data?
Do we expect to find these cell types in our dataset, based on what we know about cell states in glioblastoma tumors?

```{r brain cell types}
# what types of cells are found in this dataset?
colData(brain_sce_ref) |>
  as.data.frame() |>
  dplyr::count(cell.type)
```

This dataset has a higher number of brain cell types compared to the HPCA data, which is expected considering it is brain-specific.
Specifically, it contains astrocytes, neurons, and OPCs allowing us to identify cells that may be AC-like, NPC-like, or OPC-like, while the HPCA dataset only contained astrocytes and neurons.

The data in the `scRNAseq` package are compiled from publications and large studies, so annotations found in each dataset will be specific to the given study.
In this study, [Darmanis _et al._ (2015)](https://doi.org/10.1073/pnas.1507125112) labeled neurons according to their differentiation stage as either `fetal_quiescent` and `fetal_replicating`.
If we were to look at a different brain study, we may see slightly different annotations for cell types.

Something else that we might want to do here that we did not have to do with the `celldex` data is ensure that our data is normalized and log-transformed.
Although this is not always required, `SingleR` expects the reference datasets to be normalized with a `logcounts` assay by default.
However, the SCE objects available from `scRNAseq` only contain the raw counts, so here we will apply the same normalization that we'd normally apply to an SCE object before building our model.
We will save the normalized SCE object with the added `logcounts` assay matrix to the same variable, `brain_sce_ref`.

```{r normalize}
# Perform rough clustering
qclust <- scran::quickCluster(brain_sce_ref)

# use clusters to compute scaling factors and add to SCE object
brain_sce_ref <- scran::computeSumFactors(brain_sce_ref, clusters = qclust)

# perform normalization using scaling factors
# and save as a new SCE object
brain_sce_ref <- scuttle::logNormCounts(brain_sce_ref)
```

Now we can train our new model, using the brain-specific reference dataset.
We will use the same approach that we used with the HPCA dataset, with one alteration.

We'll specify the argument `de.method="wilcox"` so that the model uses the Wilcoxon rank sum test to identify top markers for each pairwise comparison.
This method is more appropriate for [annotation with a single-cell reference dataset](https://bioconductor.org/books/3.16/SingleRBook/more-markers.html#annotation-with-test-based-marker-detection) because it accommodates the low coverage and sparsity commonly seen in single-cell data.

**A little tip:** When using the Wilcoxon rank sum test, normalized counts are not required.
If you would like to use raw counts instead, be sure to use the `assay.type=counts` option.
See the section on [choosing assay data from the `SingleR` book](https://bioconductor.org/books/release/SingleRBook/using-the-classic-mode.html#choices-of-assay-data) for more details.

```{r brain train model}
# build model
brain_model <- SingleR::trainSingleR(
  brain_sce_ref, # reference dataset
  labels = brain_sce_ref$cell.type, # labels for training dataset
  # use DE to select genes (default)
  genes = "de",
  de.method = "wilcox", # recommended for single cell reference
  # only use genes in the sce object
  restrict = rownames(sce),
  # parallel processing
  BPPARAM = BiocParallel::MulticoreParam(4)
)
```

In the next chunk use `classifySingleR` to annotate cell types using the model trained with the brain specific reference.
Save the results to a variable named `brain_result`.

```{r brain singleR, solution = TRUE}
# classify cell types with brain model

```

Look at the results using both a table and then a heatmap.

```{r brain singleR table, solution = TRUE}
# make a table of the brain reference annotations

```

```{r brain singleR heatmap, solution = TRUE}
# make a heatmap of the brain reference annotations

```

Now, add in the annotations to the SCE object and create a UMAP plot, coloring the cells by the identified cell type.
When adding in the cell types to the SCE object, save them to a column in the `colData` named `celltype_brain`.

```{r brain singleR umap, solution = TRUE}
# add annotations to SCE object

# plot cell type assignments on UMAP

```

Interestingly, in these results, cells that the HPCA reference classified as macrophages and monocytes are now being classified as microglia, which are a special population of macrophages residing in the brain and central nervous system.
We also see that the cluster of astrocytes appear to be a mix of various cells: astrocytes, endothelial cells, and neurons.
Does this change our answer to the previous question of which of the 4 glioblastoma cell states are present in our sample?
Using the HPCA reference we had seen this sample was comprised mostly of AC-like cells, but when using this reference dataset we also see cells that may be NPC-like.

In this particular example, we are using a dataset that is more appropriately tailored to our tissue of interest and therefore we obtain cell types that are a better representation of our expected cell types.

Conversely, we will not be able to assess the presence of any T cells here, because those cells are not labeled in our brain-specific reference.

### Comparing annotations between reference datasets

Now that we have classified our glioblastoma data using two different references, let's compare the classifications.
To do this, we will make a heatmap with annotations from the HPCA dataset on one axis and the annotations from the brain dataset on the other axis.
The color of the heatmap should indicate how many cells are classified as both the specified HPCA and brain reference cell type.

We will be using the `pheatmap` package which requires the input to be a matrix.
Before we can create our plot, we need to create the matrix that will be used to make our plot.
This matrix will be a contingency table of the cell types from each reference dataset that each cell was classified as.
To do this, use the `table()` function and input two vectors with the cell type annotations from both the HPCA and brain references.
Name this variable, `preds_mtx`.
Don't forget to use the `useNA="ifany"` option to capture `NA` assignments.

```{r preds mtx, solution = TRUE}
# combine predictions in one table

```

Now we can use the `pheatmap::pheatmap()` function to create the heatmap showing the number of cells categorized as each cell type with the reference dataset.
Before you make the heatmap, you might want to use the `log1p()` function (which is a shortcut for `log(x+1)`) to log-transform the input matrix.
This will highlight some of the groups with smaller cell numbers and make interpretation a bit easier.

```{r preds heatmap, solution = TRUE}
# first make a log-transformed matrix

# make a heatmap of the log-transformed prediction matrix

```
Are there cells assigned to the same cell type in both reference datasets?
For cells labeled as microglia, which are only present in the brain reference dataset, what cell type are they categorized as when using the HPCA reference?

You also see that cells that were assigned to astrocytes in the HPCA dataset are assigned to various other brain cell types in the brain dataset and appear to be composed of more than just astrocytes.

### Comparing annotations to cluster assignments

We can also look at how the original cluster assignments relate to the cell type assignments.
To do this, make a UMAP plot, coloring the cells by cluster assignment and facet the plot by the cell type assignments using the brain reference.
(Hint: Remember to use the `other_fields` option in conjunction with `facet_wrap()`.)

```{r celltype cluster umap, solution = TRUE}
# UMAP colored by cluster assignment
# and faceted by cell type

```

## Part B: Annotate cell types with `SingleR` using a `Metacell`-like approach

In addition to looking at cell type annotation using different reference datasets, let's also try another method of cell type annotation that uses clustering to inform cell type annotations.
We will use the approach inspired by [Baran _et al._ (2019)](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1812-2) using metacells that we tried during instruction.

First we need to perform fine-scale clustering, which we can do using the `bluster::clusterRows()` function and running K-means clustering with `bluster::KmeansParam()`.
With K-means clustering we have to pick how many clusters should be present in our dataset, e.g., the number of centers that will be set before performing clustering.
During instruction we were working with a sample that contained about 8,000 cells, and we set the number of clusters (which corresponds to k-mean centers) to be 100.
However, here we only have around 1,200 cells.
So if we set our number of centers to be 100, then we might end up with very small clusters.
Remember our goal here is not to find differences among clusters, just to get homogeneous groups of cells so we don't need to get the exact right number in our clustering assignment.
One rule of thumb is to use the square root of the total number of cells, so here we will use `centers = 35`.
This is not a perfect rule by any means, so feel free to try different numbers and see how that affects your results.

Go ahead and identify cluster assignments for the glioblastoma sample, naming the results `kclusters`.
Be sure to use the `PCA` as your input.

```{r kmeans, solution = TRUE}
# perform k-means clustering

```

Now we can use `SingleR` as we did before, but this time we will classify each cluster ("metacell"), rather than each individual cell.
Use the `DelayedArray::colsum()` function to create a new matrix named `metacell_mat` where counts are summed for each gene across all groups.
In this case cells should be grouped by cluster assignment.
This is the matrix we will use as input to `SingleR`.

What are the dimensions of this new matrix?

```{r metacell matrix, solution = TRUE}
# create a "metacell" matrix by summing fine-scale clusters

# print dimensions

```

Let's perform this classification twice.
Use `classifySingleR()` to classify each cluster with the `metacell_mat` as input, first with with the model trained from the HPCA dataset (`hpca_model`) and then with the model trained from the brain specific dataset (`brain_model`).

First, classify metacells using the HPCA reference dataset and save results to the variable `hpca_metacell`.

Then, create a new column in the `colData` of the SCE object named `hpca_metacell` to store the identified labels from `SingleR`.
Remember that you will need to make sure the dimensions match up!

```{r hpca metacell, solution = TRUE}
# classify metacells with SingleR using HPCA model

# apply metacell cell type assignments to individual cells

```

Now, perform classification again but this time using the brain-specific model as the reference for `SingleR`.
Similar to before, save the `SingleR` results to a variable called `brain_metacell`, and then create a new SCE  `colData` column also called `brain_metacell` to store the annotations.

```{r brain metacell, solution = TRUE}
# and now classify using brain model

# apply metacell cell type assignments to individual cells

```

Now we can compare these results with UMAPs colored by cell type annotations from each reference.
However, instead of printing out each plot individually, we can actually save our plots to variables and then use the [`patchwork::wrap_plots()` function](https://patchwork.data-imaginist.com/index.html) to combine plots into a grid.
`patchwork` is a handy package for when we want to combine multiple panels or plots that we've made separately into one figure.

```{r metacell umap}
# create hpca umap
hpca_umap <- scater::plotUMAP(sce,
                              colour_by = "hpca_metacell")

# create brain umap
brain_umap <- scater::plotUMAP(sce,
                               colour_by = "brain_metacell")

# combine into one plot
patchwork::wrap_plots(hpca_umap,
                      brain_umap,
                      # stack umaps with only one column
                      ncol = 1)
```

When looking at these results do you see any similarities or differences between using the two references?
As a reminder, each of our references have a different set of cell types represented so the colors used for annotation may not be the same in each panel.

Before we move on to our last section, go ahead and save all of the cell type annotations to our output file, `celltype_file`.
Remember, we stored all of the cell type annotations to the `colData` of the SCE object, so we can grab the `colData` and save that to a TSV file without having to save the whole SCE object.

```{r save cell info, solution = TRUE}
# save coldata as dataframe and write to output

```


## Part C: Explore expression of cell type specific marker genes

The last thing we will look at is the expression of individual marker genes.
Marker genes can be used to identify cell types, setting thresholds as we showed in the instruction notebook.
Alternatively, we can use marker genes that have previously been identified to validate the cell type annotations from `SingleR` or other methods we may have used.

From the [reference dataset for the motor cortex on `Azimuth`](https://azimuth.hubmapconsortium.org/references/#Human%20-%20Motor%20Cortex) we can pull out some expected marker genes that should be found in the cell types identified in our glioblastoma sample.
For this example we will focus on microglia cells.
The microglia gene set can be found in the `subclass` drop down menu of "Annotation Details".

Using `SingleR` with the brain-specific reference dataset we identified a group of cells that are likely to be microglia.
If they are indeed microglia then they should express some of the marker genes that are specific to microglia.
We can check this by plotting the expression of microglia marker genes across cell types.
Keep in mind that we are working with a tumor sample and there will be a mix of tumor cells and non-tumor cells present that we have not yet annotated.
Therefore marker genes for normal cells may not always be equivalent to marker genes of tumor cells of the same cell type.

The marker genes found on `Azimuth` are stored as gene symbols so we will first need to map those symbols to the Ensembl ids that we have stored in our SCE object.

```{r map ensemlb id}
# define microglia geneset
microglia_genes <- c("DOCK8", "P2RY12", "APBB1IP", "FYB", "PTPRC",
                     "TBXAS1", "CX3CR1", "BLNK", "SLCO2B1", "CSF1R")

# print corresponding ids for reference
microglia_info <- as.data.frame(rowData(sce)) |>
  dplyr::filter(Symbol %in% microglia_genes) |>
  dplyr::select(ID, Symbol)

microglia_info
```

Now we can create UMAP plots and color the cells by expression of each of the microglia specific marker genes.
To do this we will use the [`scuttle::makePerCellDF()` function](https://rdrr.io/bioc/scuttle/man/makePerCellDF.html) which conveniently takes all the cell metadata from a SCE object, including the dimension reduction results and creates a data frame we can use for plotting.
We can also provide a list of features to include in the data frame using `features = ensembl_ids` so that all of the counts for our genes of interest are included in the returned data frame.
After a little bit of wrangling, we can use `ggplot2` to create faceted UMAP plots.
For these UMAP plots we want one plot per marker gene where each cell is colored by the expression of the specified gene.

```{r marker gene umaps}
# grab corresponding ensembl ids for plotting
ensembl_ids <- microglia_info |>
  dplyr::pull(ID, name = Symbol)

# create umap + logcounts data frame
umap_expression_df <- scuttle::makePerCellDF(sce, features = ensembl_ids) |>
  # select only UMAP embeddings and counts for ensembl genes
  dplyr::select(UMAP1="UMAP.1",
                UMAP2="UMAP.2",
                all_of(ensembl_ids)) |>
  # pivot to have one column with gene names
  #  and one column with logcounts for a given gene
  tidyr::pivot_longer(!starts_with("UMAP"),
                      names_to = "gene",
                      values_to = "logcounts")

# create faceted umaps colored by gene expression
ggplot(umap_expression_df,
       aes(x=UMAP1, y=UMAP2, color = logcounts)) +
  geom_point(size = 0.5) +
  scale_colour_viridis_c() +
  # facet by gene name
  facet_wrap(vars(gene)) +
  theme_bw()
```

These plots show us the expression of all of our genes of interest.
We can also see that all the microglia genes we explored are indeed expressed in the same group of cells that `SingleR` annotated as microglia (using the brain reference).

We could also plot the expression of all marker genes across the identified cell types, confirming that the marker genes for microglia are specifically expressed in cells labeled as microglia and not other cell types.
Here we can use the `scater::plotExpression()` function to plot the expression of all marker genes and facet by any of the cell type annotations we have added to our SCE throughout this exercise.

```{r marker expression}
# plot expression of all marker genes
scater::plotExpression(sce,
                       # plot all marker genes using ensembl id
                       features = ensembl_ids,
                       other_fields = "celltype_brain",
                       point_size = 0.1) +
  # facet by cell type label of choice
  facet_grid(cols = vars(celltype_brain)) +
  # label by symbol instead of ensembl id on x axis
  scale_x_discrete(labels = names(ensembl_ids))
```

Does this plot look how we expect given that we are looking at a list of genes that are specific markers of microglia?

Because we are looking at samples from glioblastoma tumors, we can also look at markers that may be specific to tumor cells.
[Yu _et al._ (2020)](https://doi.org/10.1093/nsr/nwaa099) identified markers of tumor cells and non-tumor cells in their single-cell RNA-seq study of brain tumors.
Figure 1 identifies genes like `OLIG2`, `PTPRZ1`, and `AQP4` to be highly expressed in tumor cells.
Are there cells or groups of cells in this glioblastoma sample that follow this pattern?
Go ahead and explore on your own!

```{r marker genes diy}
# explore some genes and gene sets on your own!
```

## Session Info

```{r session info}
sessionInfo()
```

