---
title: "scRNA-seq_post_processing"
output: html_notebook
---

**CCDL 2019**

In this notebook, we'll perform normalization and quality control of scRNA-seq
count data. 

We'll use Seurat for post-processing of scRNA-seq Alevin processed data
Adapted from the Seurat pbmc tutorial found [here](https://satijalab.org/seurat/pbmc3k_tutorial.html) 
```{r Install Packages}
# Attach library
library(Seurat)
```

#### Import gene expression matrix data
```{r Import Data}
counts <- readr::read_tsv(file.path("data", "counts_data.tsv"))
```

#### Mitochondrial genes by their ensembl IDs
This list is from [here](http://jdblischak.github.io/singleCellSeq/analysis/qc-filter-ipsc.html)
```{r Set up mitochondrial gene list}
# Identify mitochondrial genes by their ensembl IDs
mtgene <- c("ENSG00000198899", "ENSG00000198727", "ENSG00000198888", 
            "ENSG00000198886", "ENSG00000212907", "ENSG00000198786",
            "ENSG00000198695", "ENSG00000198712", "ENSG00000198804",
            "ENSG00000198763","ENSG00000228253", "ENSG00000198938", 
            "ENSG00000198840")

# Get the gene symbol names for these genes add regex for exact matches
mtgene <- data$gene[match(mtgene, counts$ensembl)]
mtgene <- paste0("^", mtgene, "$")
```

#### Set up data in Seurat format
```{r Set Up Data into seurat format}
# Get gene factor vector to sort by
genes <- as.factor(counts$gene)

# Change data to a matrix with only one gene entry per 
counts <- as.list(counts[,-c(1:2)])
counts <- do.call("cbind", 
              lapply(data, function(sample) tapply(sample, genes, sum))
              )

# Make our data into a Seurat Object
seurat.data <- CreateSeuratObject(raw.data = counts, min.cells = 3,
                                  min.genes = 200, project = "4kpbmc")
```

#### Get mitochondrial gene percentages
```{r Calculate Mito Percentages}
# Get those mito genes
mito.genes <- grep(pattern = paste0(mtgene, collapse = "|"), x = rownames(seurat.data@data), value = TRUE)

# Calculate the percentage of mitochonrial transcripts to total transcripts
percent.mito <- Matrix::colSums(seurat.data@raw.data[mito.genes, ])/
  Matrix::colSums(seurat.data@raw.data)

# AddMetaData adds columns to object@meta.data, and is a great place to
# stash QC stats
seurat.data <- AddMetaData(object = seurat.data, metadata = percent.mito,
                             col.name = "percent_mito")
```

#### Make plots of the samples' statistics 
```{r Make Sample Set Plots}
# Make a violin plot
VlnPlot(object = salmon.seurat, features.plot = c("nGene", "nUMI", "percent_mito"),
        nCol = 3)

# Examine relationships of nUMI, percent_mito and nGene
par(mfrow = c(1, 2))
GenePlot(object = seurat.data, gene1 = "nUMI", gene2 = "percent_mito")
GenePlot(object = seurat.data, gene1 = "nUMI", gene2 = "nGene")
```

#### Normalize and filter data
From seurat tutorial: we employ a global-scaling normalization method “LogNormalize” 
that normalizes the gene expression measurements for each cell by the total expression,
multiplies this by a scale factor (10,000 by default), and log-transforms the result.
```{r Filter Normalize}
# Filter out cells
filtered.seurat <- FilterCells(object = seurat.data, 
                             subset.names = c("nGene", "percent_mito"), 
                             low.thresholds = c(200, -Inf), 
                             high.thresholds = c(2500, 0.05))

# Normalizing the data
filtered.seurat <- NormalizeData(object = seurat.data, 
                                 normalization.method = "LogNormalize", 
                                 scale.factor = 10000)
```

#### Explore variance to average expression 
```{r Plot Dispersion vs Avg Exp}
# Make a variance to average expression plot
filtered.seurat <- FindVariableGenes(object = filtered.seurat, 
                                     mean.function = ExpMean, 
                                     dispersion.function = LogVMR,
                                     x.low.cutoff = 0.0125,
                                     x.high.cutoff = 3,
                                     y.cutoff = 0.5)
```

#### Save the normalized data
```{r Save Data to RDS}
# Save this seurat object to an RDS file
readr::write_tsv(filtered.seurat, file = file.path("data", "seurat"))
```

```{r}
sessionInfo()
```