---
title: "scRNA-seq_post_processing"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2019**

In this notebook, we'll perform quality control analyses and normalization of 
scRNA-seq count data. 

As opposed to bulk rna-seq, there are there are a few main things to look out for
in single cell RNA-seq:

**Single-cell rna-seq has...**  
- More amplification (and more of it's associated biases and error)  
- More 0's in the gene expression data (most genes aren't expressed across cell types)  

This script is adapted from 
[Seurat pbmc tutorial](https://satijalab.org/seurat/pbmc3k_tutorial.html)
  
#### Set Up 
For these analyses, we will need Seurat a lot, so we will go ahead and attach it.  
```{r Set Up}
# Attach library
library(Seurat)
```

#### Import single cell RNA-seq counts matrix
Here we are importing a counts matrix from single cell rna-seq data that has 
already been processed using Salmon and tximport and saved the counts to a tsv 
file. 
This [data set](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) 
is glioblastoma data that was Fluorescent Activated Cell sorted 
and then was processed by paired end sequencing using smart-seq2 protocol 
[(Darmanis et al, 2017).](https://www.ncbi.nlm.nih.gov/pubmed/29091775) 
```{r Import Data}
# Read in the data
sc_data <- readr::read_tsv(file.path("data", "darmanis_counts.tsv")) %>% 
           tibble::column_to_rownames("gene") %>% 
           as.data.frame()
```

#### Set up the Seurat object and apply a filter to the data. 
In this example, we are applying a gene filter cutoff of at least 3 cells 
expressing each gene. 
For cell filtering, each cell must express at least 200 genes.
```{r Set Up Seurat Object}
# Set up the seurat object 
seurat_obj <- CreateSeuratObject(raw.data = sc_data, min.cells = 3,
                                 min.genes = 200, 
                                 project = "darmanis_gbm")
```

#### Normalize the data using log normalization
These are the default settings for normalization and should work in most 
instances. 
```{r Normalize the Data}
seurat_norm <- NormalizeData(object = seurat_obj, 
                             normalization.method = "LogNormalize",
                             scale.factor = 10000)
```

#### Identify variable genes and scale the data
In this step, we will identify genes with higher variance. 
These are probably the genes that are of more interest to us.
```{r Find Variable Genes}
seurat_norm <- FindVariableGenes(object = seurat_norm, mean.function = ExpMean,
                                x.low.cutoff = 0.0125, x.high.cutoff = 3,
                                y.cutoff = 0.5, do.plot = FALSE)

# Scale the data
seurat_norm <- ScaleData(object = seurat_norm)
```

#### Save the normalized data to tsv file
```{r Save Data to TSV}
# Take out the data and make genes a column
gene_matrix <- data.frame("genes" = rownames(seurat_norm@raw.data),
                          seurat_norm@raw.data)

# Save this gene matrix to a tsv file
readr::write_tsv(gene_matrix, 
                 file.path("data", "normalized_gbm_gene_matrix.tsv"))
```

#### Save the seurat object to RDS file
```{r Save Seurat to RDS}
# Save this gene matrix to a tsv file
saveRDS(seurat_norm, file.path("data", "seurat_object.RDS"))
```

#### Print session info:
```{r}
sessionInfo()
```
