---
title: "Set up murine leukemia DGE results for over-representation analysis"
output: 
  html_notebook: 
    toc: true
    toc_float: true
author: CCDL for ALSF
date: 2021
---

Here, we're performing differential gene expression (DGE) analysis using data from [Sachs _et al._ (2014)](https://dx.doi.org/10.1182%2Fblood-2013-08-521708) for the purpose of using the results in the over-representation analysis (ORA) instruction notebook.
The premise is to generate two sets of DGE results, where we can then take the set difference between genes that meet certain criteria for each comparison and use that as input to ORA.
This showcases one of the advantages of ORA over GSEA; we can easily use it for something beyond a simple 2 group comparison.

#### Libraries

```{r}
library(magrittr)
library(org.Mm.eg.db)
```

#### Input

```{r}
root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))

# We're going to use data that we distribute for an RNA-seq exercise
input_dir <- file.path(root_dir, "RNA-seq", "data", "leukemia", "txi")
txi_file <- file.path(input_dir, "leukemia_stem_cell_txi.RDS")
```

#### Output

```{r}
output_dir <- file.path(root_dir, "pathway-analysis", "data", "leukemia")
dir.create(output_dir, recursive = TRUE, showWarnings = FALSE)

# We'll save the full list of genes for both comparisons as TSV files
vs_low_file <- file.path(output_dir, 
                         "high_capacity_vs_low_capacity_results.tsv")
vs_unsorted_file <- file.path(output_dir,
                              "high_capacity_vs_unsorted_results.tsv")
```

#### Set up data & DGE

```{r}
# Read in tximeta output and make a DESeqDataSet
txi <- readr::read_rds(txi_file)
ddset <- DESeq2::DESeqDataSet(txi, design = ~ cell_sorting)
# Uses cell_sorting by default
deseq_object <- DESeq2::DESeq(ddset)
```
#### Relevant comparisons

First, compare the population that were identified as having high stem cell capacity by the authors to the population identified as low stem cell capacity.

```{r}
# Comparing the high stem cell capacity ("MacLowKitposSca1pos") to low stem cell
# capacity ("MacHigh")
vs_low_df <- DESeq2::results(deseq_object, 
                                 contrast = c("cell_sorting", 
                                              "MacLowKitposSca1pos", 
                                              "MacHigh")) %>%
  as.data.frame() %>%
  tibble::rownames_to_column(var = "Gene") %>%
  # Also write to file
  readr::write_tsv(vs_low_file)
```

Then compare the high stem cell capacity population to the unsorted cells.
We'll use to "narrow" the gene set produced in the first comparison to only what shows up in the comparison for the two cell populations.

```{r}
# Comparing the high stem cell capacity ("MacLowKitposSca1pos") to the unsorted
# cells 
vs_unsorted_df <- DESeq2::results(deseq_object, 
                                  contrast = c("cell_sorting", 
                                               "MacLowKitposSca1pos", 
                                               "unsorted")) %>%
  as.data.frame() %>%
  tibble::rownames_to_column(var = "Gene")
```

So we don't have to do gene identifier conversion _twice_ during instruction, we're going to include gene symbols here.

```{r}
vs_unsorted_df <- vs_unsorted_df %>%
  # Add a gene symbol column
  dplyr::mutate(gene_symbol = mapIds(org.Mm.eg.db, 
                                     keys = vs_unsorted_df$Gene, 
                                     column = "SYMBOL", 
                                     keytype = "ENSEMBL", 
                                     multiVals = "first")) %>%
  # Also drop zero expression, low expression (e.g., padj = NA), and no
  # gene symbol
  tidyr::drop_na() %>%
  # Write to file
  readr::write_tsv(vs_unsorted_file)
```

#### Check results

```{r}
# Genes with higher expression in high capacity cells vs. low capacity cells
vs_low_genes <- vs_low_df %>%
  dplyr::filter(stat > 0, padj < 0.05) %>%
  dplyr::pull(Gene)

# Genes with higher expression in high capacity cells vs. unsorted cells
vs_unsorted_genes <- vs_unsorted_df %>%
  dplyr::filter(stat > 0, padj < 0.05) %>%
  dplyr::pull(Gene)
```

How many genes would we plan to use as the genes of interest for ORA?
Note that we may remove `NA`, etc. in the notebook itself, but this can give us a good idea of what we're working with to start. 

```{r}
length(setdiff(vs_low_genes, vs_unsorted_genes))
```

#### Session info

```{r}
sessionInfo()
```

