# Advanced scRNA-seq Training Module

This Childhood Cancer Data Lab-designed module covers more advanced topics in the analysis of single-cell RNA-seq data.

The module builds on material in the [scRNA-seq module](https://github.com/AlexsLemonade/training-modules/tree/master/scRNA-seq), and analyses are designed to be performed within a [Docker container](https://github.com/AlexsLemonade/training-modules/tree/master/docker-install) or on the Data Lab RStudio server.
It covers cell-type identification, integration of multiple single-cell RNA-seq libraries, and differential expression analyses, among other related topics.

The notebooks that comprise this module are:

* [Introduction and review of processing scRNA-seq data](./01-read_filter_normalize_scRNA.Rmd)
* [Cell type annotation](./02-celltype_assignment.Rmd)
* [Integration of multiple scRNA-seq libraries](./03-dataset_integration.Rmd)
* [Differential expression analysis](./04-differential_expression.Rmd)


Additional exercise notebooks:

* [Working with CITE-seq data](./exercise_01-citeseq.Rmd)
* [Cell type annotation](./exercise_02-celltype.Rmd)
* [Integration of multiple scRNA-seq libraries](./exercise_03-integration.Rmd)
* [Differential expression analysis](./exercise_04-diffexp.Rmd)
