---
title: "Exercise: Exploring drug exposure in the NCI-60 cell lines"
output: 
  html_notebook: 
    toc: true
    toc_float: true
---

## NCI-60 Cell Line Data

In this exercise, we'll work with data from GEO accession [`GSE116436`](https://www.refine.bio/experiments/GSE116436/drug-induced-change-in-gene-expression-across-nci-60-cell-lines-after-exposure-to-15-anticancer-agents-for-2-6-and-24h) that was processed with refine.bio. 

Here's an excerpt of the description from the authors:

> To identify patterns of drug-induced gene modulation that occur across different cell types, we measured gene expression changes across NCI-60 cell lines after exposure to 15 anticancer agents.

**We've supplied the following to you:**

* A PLIER model that was trained on the expression data from this experiment (`models/NCI60_PLIER_model.RDS`)
* The sample metadata (`data/GSE116436/processed/metadata_GSE116436_cleaned.tsv`)

We're going to look specifically at **pathways that change with treatment**, by using the PLIER model.
Specifically, we can find PLIER latent variables (LVs) associated with pathways of interest and look at how the LV estimates or expression values change during the course of treatment.

### Notes on processing and training for your reference

#### Expression data and metadata

When we initially obtained these data from [refine.bio](https://www.refine.bio), the complete experiment was not available. 
Any given cell line, treatment, and dose combination is complete if there are 3 instances in the metadata, so we filtered only to the complete combinations that were available.

#### PLIER model

This PLIER model was trained using some earlier recommendations from the PLIER author where the number of latent variables (_k_) was more restrictive.
We supply some reference code for training the model that is not intended to be run below.

```
# Set seed for reproducibility of results
set.seed(12345)
# Determine the number of significant PCs via the elbow method
set_k <- PLIER::num.pcs(zscored_matrix[common_genes, ])
# Run PLIER
plier_results <- PLIER::PLIER(zscored_matrix[common_genes, ], 
                              pathway_matrix[common_genes, ], 
                              k = round(set_k * 1.3, 0))
```

## Set up

### Libraries

We'll use the Tidyverse quite a bit in this notebook, so use the chunk below to load it.

```{r libraries, solution = TRUE}

```

### Directories and files

#### Directories

The metadata is available at `data/GSE116436/processed/metadata_GSE116436_cleaned.tsv`.
The PLIER model is available at `models/NCI60_PLIER_model.RDS`.
You'll want to save a plot later on.
Set up all the directories you need in the next chunk.

```{r directories, solution = TRUE}
# Directory that holds the data

# Directory with the PLIER model

# Plot directory, create it if it doesn't exist yet

```

#### Input files

Use the next chunk to set up your input files.

```{r input_files, solution = TRUE}
# Metadata file

# PLIER file

```

#### Output files

Use the next chunk to set up the plot output file name.
You may need to change this file name once you know more about what you're plotting!
It's good practice to start a new session and run all chunks (use the `Run` dropdown menu above) once you've completed the notebook.

```{r output_files, solution = TRUE}
# Plot output file name

```

## Read in files

### Metadata

Read in the metadata file with `readr` and save it to a variable named `metadata_df` in the next chunk.

```{r read_in_metadata, solution = TRUE}

```

What metadata do we have for these samples? 
Use this chunk to explore `metadata_df`.

```{r explore_metadata, eval = FALSE, solution = TRUE}

```

### PLIER model

Read in the PLIER model with `readr` and save it as `plier_results`.

```{r read_in_plier, solution = TRUE}
# Save as plier_results

```

## What pathways are perturbed by the drugs in this experiment?

The `summary` element of `plier_results` can tell us about associations between pathways and LVs, you can access it with `$` because `plier_results` is a named list. 
It is a data frame (`data.frame`).

Use this chunk to view the PLIER model `summary`.

```{r view_summary, solution = TRUE}

```

Save the summary information as its own data.frame called `summary_df`. 
We'll use this object to find out more about our model. 

```{r save_summary_df, solution = TRUE}
# Save the summary data.frame as summary_df

```

The latent variables from this model will be associated with a pathway if the transcriptomic data supports that there is some shared variation in the expression levels of genes in that pathway.
The variation could be due to treatment with a drug or it could arise from tissue- or cell line-specific differences.

### Is there a latent variable associated with the proteasome pathway?

We can use a chain of `dplyr` functions to filter `summary_df`.
Let's call `FDR < 0.05` associated; this is the same cutoff we used in the PLIER notebook earlier.

```{r detect_proteasome}
summary_df %>%
  filter(FDR < 0.05,  # significant associations only
         # str_detect() returns TRUE when the pattern "proteasome" is found
         # in the pathway column - using regex() the way we do here means we'll 
         # ignore case
         str_detect(pathway, regex("proteasome", ignore_case = TRUE)))
```

The LV index column of `summary_df` tells us _which **row**_ in the LV by sample matrix `plier_results$B` we should look at.
To make plotting easier downstream, let's make a data frame that contains two columns, `geo_accession` and `proteasome_lv`, which contain the sample identifiers and the values for the latent variable associated with the proteasome gene set, respectively.
Recall that the column names of `plier_results$B` are the sample identifiers.
(Hint: `?colnames`)

Call this data frame `proteasome_df`. 
Here's an example of what `proteasome_df` should look like in table form (you can use `Preview` to view a rendered version):

| geo_accession | proteasome_lv |
|---------------|---------------|
| GSMXXXXXXX    |  0.8923723    |
| GSMXXXXXXX    | -0.2342353    |


```{r proteasome_df, solution = TRUE}
# Create the proteasome_df data frame

```

Now that you have `proteasome_df`, let's join it with our metadata using an `inner_join` from `dplyr` to form a new data frame called `proteasome_meta_df`.
We'll join by the sample names, which is what `by = "geo_accession"` specifies.

```{r join_meta}
proteasome_meta_df <- inner_join(x = proteasome_df, 
                                 y = metadata_df, 
                                 by = "geo_accession")
```

Let's find out what samples have the highest values for the proteasome latent variable.
We can sort `proteasome_meta_df` with `arrange`.
Use the `Help` panel or `?arrange` to remind ourselves about what to expect from this function.

```{r sort_proteasome_meta, solution = TRUE}

```

> What do you notice about the samples with the highest values?
> Look up the mechanism of action for the drugs in `treatment`.
> Do these results make sense?

### Line graph

Let's make a plot that shows the change of the proteasome latent variable expression level over time, when treated with the drug that comes to the top of the list.
We'll look at a single tissue of your choosing.
This requires us to do the following:

1. Filter `proteasome_meta_df` to samples from your tissue of interest and the
   drug that we are interested in.
   Create a new data frame, `df_to_plot`, with only the samples for plotting.
   
```{r df_to_plot, solution = TRUE}

  # Use these lines to filter to only the samples that are treated with the drug
  # of interest and to the tissue of your choice

```

2. Make a line graph that tracks the expression value of the proteasome 
   latent variable during different time points of the experiment.
   
Save as `treatment_plot` - we'll continue to alter this object as we customize our plot.

```{r initial_plot}
treatment_plot <- df_to_plot %>%
  ggplot(aes(x = time_hours,  # Time on the x-axis
             y = proteasome_lv,  # LV estimate on the y-axis
             color = cell_line)) + # Color by individual cell line
  # These two geoms make a line graph connecting individual points
  geom_point() +
  geom_line()

treatment_plot
```

Uh oh, all of the dosages are mixed together which makes this more difficult to interpret.
We can continue add to `treatment_plot` or tweak things by using the `+` operator without repeating the code that set it up.
Add `facet_wrap()` to `treatment_plot` (saving `treatment_plot`) such that each dosage is its own facet and use `colorblindr::scale_color_OkabeIto()`.

```{r use_facets, solution = TRUE}
# Update the treatment_plot object by assigning your edits to treatment_plot
# with <-

  # Use facets to separate dosages - what column do we need to use?

  # Use this scale for accessibility

```

Let's see what the plot looks like with this chunk below.

```{r view_treatment_facet, solution = TRUE}

```

> Based on what you know about the mechanism of action of this drug, do these results look how you expect?
> What did the PLIER model uncover and what does the plot mean?

#### Customizing the plot

We can also use the `+` operator, without assigning anything, to continue to tweak the plot.
In the chunk below, we're changing the theme of the plot to `theme_bw()` without saving this to an object in our environment.

```{r theme_bw}
treatment_plot + theme_bw()
```

Using the new theme, label the x-axis "Time (Hours)" and the y-axis "Proteasome LV Expression Level."
Make the title the name of the drug you are plotting data for.

```{r relabel_plot, solution = TRUE}

  # Add new theme

  # Relabel the axes and add a title

```

If we wanted to change the spacing of the x-axis, such that the time points are evenly spaced, we'd have to make `time_hours` into a factor.

_This is not always an appropriate visualization choice, it depends on your experiment.
If the interval is important, the points should not be evenly spaced._

Let's check what class `time_hours` currently is with the function `class`.

```{r time_class}
class(proteasome_meta_df$time_hours)
```

Use `mutate()` and `as.factor()` to make the `time_hours` column into a factor. 

```{r time_factor, solution = TRUE}

```

Now make a new plot, `treatment_plot_factor`, using the same theme, color scale,
labels, etc.

```{r factor_plot, solution = TRUE}

```

Display `treatment_plot_factor`.

```{r}
treatment_plot_factor
```

> Did it display in the order that you wanted?
If not, you can use the `levels` argument of the `factor()` function to specify the order.
See [this post](https://rstudio-pubs-static.s3.amazonaws.com/7433_4537ea5073dc4162950abb715f513469.html).

#### Save to file

Save the plot of your choosing to the plots directory.

```{r save_plot, solution = TRUE}

```

## Additional analyses

Now that you've made and customized a line graph, explore the data in another way. 
You can try another pathway/latent variable, tissue, drug combination!

We've also provided you with the following:

* The gene expression matrix (`data/GSE116436/processed/GSE116436_filtered.tsv`)
* Sample-sample Pearson correlation matrix (`data/GSE116436/processed/GSE116436_sample_correlation.RDS`)
* PCA (`prcomp` output) (`models/NCI60_prcomp_results.RDS`) - no filtering to high variance genes was performed.

With these data and results, you could make a boxplot of PC1 by tissue or treatment or make a heatmap of the sample-sample correlation matrix using annotation bars to help identify patterns.

## Session Info

```{r session_info}
sessionInfo()
```

