---
title: "NB cell line: gene-level summarization with `tximport`"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2018**

This is the solved notebook version as a result of the prompt in `04-nb_cell_line_tximport.md`.

In this notebook, we'll import transcript level output from `salmon quant` and
summarize it to the gene level using [`tximport`](https://bioconductor.org/packages/release/bioc/html/tximport.html).

![](../diagrams/rna-seq_5.png)

For more information about `tximport`, see [this excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/tximport/inst/doc/tximport.html) from Love, Soneson, and Robinson.

We'll need the `quant.sf` files for all the samples in an experiment and a file
that maps between Ensembl transcript ids and Ensembl gene ids.
In refine.bio, we generate these along side the transcriptome indices for an
organism.

## Libraries and functions

```{r library, live = TRUE}
# Load the tximport package
library(tximport)
```

## Directories and files

```{r directories, live = TRUE}
# directory where the quant files are located, each sample is its own
# directory
quant_dir <- file.path("..", "data", "NB-cell", "salmon_quant")

# create a directory to hold the tximport results if it doesn't exist yet
txi_dir <- file.path("..", "data", "NB-cell", "tximport")
if (!dir.exists(txi_dir)) {
  dir.create(txi_dir, recursive = TRUE)
}
```

```{r input-names}
# the quant files themselves
sf_files <- list.files(quant_dir, recursive = TRUE, full.names = TRUE,
                       pattern = "quant.sf")

# file that contains the gene to transcript mapping from refine.bio
tx2gene_file <- file.path("..", "index", "Homo_sapiens",
                          "Homo_sapiens.GRCh38.95_tx2gene.tsv")
```

**Output**

```{r output-names, live = TRUE}
# Name the output NB-cell_tximport.RDS and use the directory created 
# above as the rest of the path
txi_out_file <- file.path(txi_dir, "NB-cell_tximport.RDS")
```

### Naming

All output files from `salmon quant` we'll use with `tximport` are named 
`quant.sf`.
The file names do not have any information about the sample name.

```{r sf_files, live = TRUE}
# Let's look at the full path for the quant.sf files
sf_files
```

Let's extract the _sample_ names from the **file paths** using the `stringr` 
package.

Notice how the file path is separated by `/`.
If we were to split up this character by `/`, the fourth item is the sample 
name.
This is exactly what `stringr::word` allows us to do: split up the file paths
by `/` and extract the sample names.

```{r sample_names}
sample_names <- stringr::word(sf_files, 4, sep = "/")
sample_names
```

We'll add these names to the file paths vector, that way they are reflected in
what we do next with `tximport`.

```{r names_sf_files}
names(sf_files) <- sample_names
```

## tximport

### Read in tx to gene mapping

```{r read_tsv, live = TRUE}
# Read in the TSV file
tx2gene_df <- readr::read_tsv(tx2gene_file)
```

Let's take a look at the top of this file with `head`.

```{r head, live = TRUE}
head(tx2gene_df)
```

### tximport main function

We'll summarize to the gene level.
Notice that `tx2gene_df` uses identifiers that don't include [Ensembl version information](https://useast.ensembl.org/Help/Faq?id=488).
Version numbers follow a period (`.`) at the end of the identifier. 
For example, the second version of a human Ensembl transcript ID follows this pattern: `ENSTXXXXXXXXXX.2`.

We need to use the `ignoreTxVersion = TRUE` argument to `tximport` here or else we'll get an error.

```{r make-txi}
txi <- tximport(sf_files, type = "salmon", 
                tx2gene = tx2gene_df,
                countsFromAbundance = "no", 
                ignoreTxVersion = TRUE)
```

We can use the `class` function to see what type of object the `tximport`
function returns.

```{r class-txi, live = TRUE}
# Check what type of object txi is
class(txi)
```

A list is a type of R data structure that is essentially a vector that can be
made up of different object types, similar to how data frames can have different
types of columns.
Lists can be made up of vectors, data frames, or even other lists.

If we try to combine different object types into a vector, what happens?

```{r mixed-vector, live = TRUE}
# Add a vector that is a mix of types -- number, character, logical -- using c()
mixed_vector <- c(1, "apple", TRUE)

# Check what kind of vector is created
class(mixed_vector)
```

Using the `list` function to create a list allows us to mix different types in,
as the name suggests, a list!

```{r mixed-list, live = TRUE}
# Create a list that includes a numeric vector, a character vector of letters,
# a logical vector, and a data frame
mixed_list <- list(
  number = 1:3,
  letter = c("a", "b", "c"),
  logical = c(TRUE, FALSE),
  tx2gene = tx2gene_df
)

# Check the class of the new mixed type list
class(mixed_list)
```

We can use `$` to access different elements of the list by their names.

```{r use-$, live = TRUE}
# Get the letters portion of the mixed type list
mixed_list$letter
```

```{r head_mixed_list, live = TRUE}
# Let's look at the first few rows of the data frame we added
head(mixed_list$tx2gene)
```

Now that we've seen what we can do with lists, let's look at the structure of 
`txi` -- the object returned by `tximport`.

```{r txi-str, live = TRUE}
# Look at the structure of txi
str(txi)
```

Now, here's a breakdown of the `txi` object.
![](../diagrams/tximport.png)

Save to file -- we'll import this with the `DESeq2` package.

```{r write-txi, live = TRUE}
# Write txi to RDS object
readr::write_rds(txi, file = txi_out_file)
```

## Session Info

Record session info for reproducibility & provenance purposes.

```{r sessioninfo}
sessionInfo()
```
