---
title: "Setup: Introduction Module"
author: "Stephanie Spielman for the Data Lab"
date: "`r Sys.Date()`"
output: 
  html_notebook:
    toc: true
    toc_depth: 3
    toc_float: true
    number_sections: true
---

```{r setup, include=FALSE}
set.seed(2022)

library(SingleCellExperiment)
library(magrittr)
library(SingleR) # annotation
library(bluster) # clustering to give the people their UMAP colors
```

To get the raw data:
```{sh eval = FALSE}
NAME=10k_PBMCs_TotalSeq_B_3p_raw_feature_bc_matrix.tar.gz
wget https://cf.10xgenomics.com/samples/cell-exp/6.0.0/10k_PBMCs_TotalSeq_B_3p/$NAME
tar -xf $NAME
# This gets unpacked into `raw_feature_bc_matrix`


# For posterity, first attempt was with MALT, but it had wildly low ADT counts
#NAME=malt_10k_protein_v3_raw_feature_bc_matrix.tar.gz
#wget https://cf.10xgenomics.com/samples/cell-exp/3.0.0/malt_10k_protein_v3/$NAME
```

## Filtering and normalization

```{r}
pbmc_10x_dir <- "raw_feature_bc_matrix"
# for now..
mito_file <- "https://raw.githubusercontent.com/AlexsLemonade/scpca-downstream-analyses/main/reference-files/Homo_sapiens.GRCh38.104.mitogenes.txt"



pbmc_sce_raw <- DropletUtils::read10xCounts(pbmc_10x_dir)
mito_genes <- readr::read_tsv(mito_file, 
                              col_names=FALSE) %>%
  # only keep relevant mitochondrial genes
  dplyr::filter(X1 %in% rownames(pbmc_sce_raw)) %>%
  dplyr::pull(X1)
```


### Initial Mitochondria QC

```{r}
pbmc_sce_percellqc <- scater::addPerCellQC(
  pbmc_sce_raw, 
  subsets = list(mito = mito_genes))

# Filter out 0s, and see before/after dims:
dim(pbmc_sce_percellqc) 
pbmc_sce_percellqc <- pbmc_sce_percellqc[, pbmc_sce_percellqc$total > 0]
dim(pbmc_sce_percellqc) 
```


Before proceeding with further filtering, we'll want to re-arrange the SCE object to handle the RNA and ADT separately.
Currently, the ADTs and genes are both part of the primary experiment, so let's move the ADT counts into an altExp slot.

```{r}
pbmc_sce_percellqc <- splitAltExps(pbmc_sce_percellqc, 
                                   #"Gene Expression" or "Antibody Capture"
                                   rowData(pbmc_sce_percellqc)$Type, 
                                   # The Type which should be the main experiment
                                   "Gene Expression")
pbmc_sce_percellqc
```



### Filter RNA

#### First, with `DropletUtils`

```{r}
# Filter RNA
droplet_df <- DropletUtils::emptyDrops(
  counts(pbmc_sce_percellqc), 
  BPPARAM = BiocParallel::MulticoreParam(4)
) 

# Filter down to low FDR cells and check out dimensions
# Values here are mostly NA.
# NOTE: this filtering does propagate to altExp!! :) 
cells_to_retain <- which(droplet_df$FDR <= 0.01)
filtered_sce <- pbmc_sce_percellqc[, cells_to_retain]
dim(filtered_sce) 
```


#### Now, with `miQC`

Mito QC stats are already present in `filtered_sce` from previously running `scater::addPerCellQC()`.
```{r}
miqc_model <- miQC::mixtureModel(filtered_sce) 
filtered_sce<- miQC::filterCells(filtered_sce,
                                 model = miqc_model)

dim(filtered_sce)
```


### Filter ADT

This data does have a control antibody (IgG1), so we can leverage this ADT for QC and filtering.
```{r}
# Filter ADT
# http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html#applying-custom-qc-filters

controls <- c("IgG1")
adt_qc_df <- DropletUtils::cleanTagCounts(altExp(filtered_sce), controls=controls) # I feel like this control might be crap!

summary(adt_qc_df$high.controls) # these are all NA????????
summary(adt_qc_df$discard) # these are MOSTLY NA????????

# Let's do this - remove cells that have zero ambient concentrations:
# This removes 11 cells
cells_to_retain <- which(adt_qc_df$zero.ambient == FALSE)
filtered_sce <- filtered_sce[, cells_to_retain]
dim(filtered_sce)
```



### Normalize RNA and ADT:


```{r}
# Normalize RNA
qclust <- scran::quickCluster(filtered_sce)
filtered_sce <- scran::computeSumFactors(filtered_sce, clusters = qclust)
norm_sce <- scater::logNormCounts(filtered_sce)

# Normalize ADT

# Calculate baseline:
# Try out control-based normalization here 
#  http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html#control-based-normalization
control_sf <- scater::librarySizeFactors(altExp(norm_sce), subset_row=controls)  # `controls` was previously defined during filtering
sum(control_sf == 0)/length(control_sf) # _HOLY WOW_ - virtually everything is 0.......


# Go for median instead, BUT TODO: would we want to exclude control here during baseline calculation? 
baseline <- DropletUtils::ambientProfileBimodal(altExp(norm_sce)) # !! currently this does not exclude controls
median_sf <- scater::medianSizeFactors(altExp(norm_sce), reference = baseline)
sum(median_sf == 0) # none are 0 here, soooo let's do this?

# Normalize with the median size factors
altExp(norm_sce) <- scater::logNormCounts(altExp(norm_sce), 
                                          size.factors = median_sf)

```

### Behold, the filtered and normalized SCE:

```{r}
norm_sce

altExp(norm_sce)
```

## Dimension reduction

HVGs:
```{r}
num_genes <- 5000 # 2000? 

gene_variance <- scran::modelGeneVar(norm_sce)
hvg_list <- scran::getTopHVGs(gene_variance,
                               n = num_genes)
```

PCA:
```{r}
# Do we want to set the number of PCs?
norm_sce <- scater::runPCA(norm_sce,
                           subset_row = hvg_list)
```

UMAP:
```{r I map you map we all map for MapMap}
# Perform graph clustering first
nn_k <- 15
nn_clusters <- clusterRows(
  reducedDim(norm_sce, "PCA"), 
  NNGraphParam(k = nn_k, 
               cluster.fun = "walktrap")
)

# pop back into sce
norm_sce$nn_clusters <- factor(nn_clusters)

# Add in UMAPs and plot with colors
norm_sce <- scater::runUMAP(norm_sce,
                            dimred = "PCA")
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "nn_clusters")
```

## Cell type annotation

We can introduce annotation with `SingleR`, and it might be a good idea to show how the results are highly influenced by the reference. 
For now, I'm just using the `HumanPrimaryCellAtlasData` reference as a starting place.


```{r}
ref_data <- celldex::HumanPrimaryCellAtlasData(ensembl = TRUE) # pick one for now to start
preds <- SingleR::SingleR(
  test = norm_sce, 
  ref = ref_data,
  labels = ref_data$label.main # "coarse-grained" labels
)
preds

# We can place these into the sce
norm_sce$celltype_predictions <- preds$pruned.labels

# **heavily** biased towards monocytes and T-cells
celltype_counts <- table(preds$pruned.labels) %>%
  as.data.frame() %>%
  dplyr::rename(
    celltype = Var1, 
    n = Freq
  ) %>%
  dplyr::arrange(-n)

celltype_counts

```



Visualize the predictions in a couple ways:
```{r}
SingleR::plotScoreHeatmap(preds)
```


```{r UMAP for the people}
# Not too shabby!
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "celltype_predictions", 
                       point_alpha = 0.4, point_size = 0.25)
```


It might be helpful to hone in on the most prevalent celltypes only for ease of visualization.

```{r}
# The top 5 cell types account for over 98% of the cells!
celltype_counts %>%
  dplyr::slice(1:5) %>%
  dplyr::summarize(sum(n) / sum(celltype_counts$n))

top_celltypes <- as.character(celltype_counts$celltype[1:5]) 

# Recode a new celltype vector with top 5 separated from "other" using some forcats magic
norm_sce$celltype_predictions_top <- forcats::fct_lump_n(norm_sce$celltype_predictions, 5)

scater::plotReducedDim(norm_sce, "UMAP", colour_by = "celltype_predictions_top", 
                       point_alpha = 0.6, point_size = 0.25)
```

## ADT "celltyping"

We can leverage the ADTs as protein markers to identify likely celltypes, which can be compared back to the `SingleR` annotations as a form of validation.


```{r}
# What are the proteins?
rownames(altExp(norm_sce))

# Which ADT is the most prevalent for each cell? 
max_adt_indices <- apply(
  logcounts(altExp(norm_sce)),
  2,
  which.max
)
max_adt <- rownames(altExp(norm_sce))[max_adt_indices]

unique(max_adt)

# Add this into the main SCE experiment for UMAPing
norm_sce$max_adt <- max_adt


# Misc thinking...how can we quantify this?
## ratio of biggest:\sum(rest)?
## difference between highest and next highest?
```

Happily, none of these is the control!
What are these proteins, and what celltypes might they indicate?
In the source quotes below, I have added the bold to the celltypes.

- `CD3`
  - "CD3 (cluster of differentiation 3) is a protein complex and T cell co-receptor that is involved in activating both the **cytotoxic T cell (CD8+ naive T cells) and T helper cells (CD4+ naive T cells).** ... This high specificity, combined with the presence of CD3 at all stages of T-cell development, makes it a useful immunohistochemical marker for T-cells in tissue sections."  [source](https://en.wikipedia.org/wiki/CD3_(immunology))
- `CD4`
  - TCR co-receptor that we can probably assume is mostly on CD4+ T-cells
- `CD8`
  - TCR co-receptor that we can probably assume is mostly on CD8+ T-cells
- `CD11c`
  - "CD11c is a type I transmembrane protein that is expressed on **monocytes**, **granulocytes**, a subset of **B cells**, **dendritic cells**, and **macrophages** and may be demonstrated on B-cell chronic lymphocytic leukemia, marginal zone lymphomas, and hairy cell leukemia." [source](https://www.sciencedirect.com/topics/neuroscience/cd11c)
- `CD14`
  - "CD14 is a glycolipid-anchored membrane glycoprotein expressed on cells of the myelomonocyte lineage including **monocytes**, **macrophages**, and some **granulocytes**." [source](https://www.sciencedirect.com/topics/neuroscience/cd14)
- `CD16`
  - "CD16 is involved in antibody-dependent cell-mediated cytotoxicity and is expressed on large granular lymphocytes (LGLs) of both **NK- and T-cell types**." [source](https://www.sciencedirect.com/topics/neuroscience/cd16)
- `CD19`
  - "CD19 is an IgSF surface glycoprotein of 95 kDa that is expressed from the earliest stages of **B cell** development until plasma cell terminal differentiation, when its expression is lost." [source](https://www.sciencedirect.com/topics/neuroscience/cd19)
- `CD56`
  - "CD56 expression is also found in, among others, the hematopoietic system. Here, the expression of CD56 is mostly associated with, but not limited to, **natural killer cells**. CD56 has been detected on other lymphoid cells, including **gamma delta Τ cells and activated CD8+ T cells**, as well as on **dendritic cells**." [source](https://en.wikipedia.org/wiki/Neural_cell_adhesion_molecule)



Let's color cells by their most prevalent ADT:

```{r does anybody have a UMAP}
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "max_adt", 
                       point_alpha = 0.6, point_size = 0.25)

# let's also run this to see them side-by-side
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "celltype_predictions_top", 
                       point_alpha = 0.6, point_size = 0.25)
```



## sessionInfo

```{r}
sessionInfo()
```
