---
title: "Setup: Introduction Module"
author: "Stephanie Spielman for the Data Lab"
date: "`r Sys.Date()`"
output: 
  html_notebook:
    toc: true
    toc_depth: 3
    toc_float: true
    number_sections: false
---

## Setup

```{r setup, include=FALSE}
set.seed(2022)

library(magrittr)

library(SingleCellExperiment)
library(SingleR)
library(bluster) 
```


### Download Data

To get the raw data:
```{sh eval = FALSE}
NAME=10k_PBMCs_TotalSeq_B_3p_raw_feature_bc_matrix.tar.gz
wget https://cf.10xgenomics.com/samples/cell-exp/6.0.0/10k_PBMCs_TotalSeq_B_3p/$NAME
tar -xf $NAME
# This gets unpacked into `raw_feature_bc_matrix`


# For posterity, first attempt was with MALT, but it had wildly low ADT counts
#NAME=malt_10k_protein_v3_raw_feature_bc_matrix.tar.gz
#wget https://cf.10xgenomics.com/samples/cell-exp/3.0.0/malt_10k_protein_v3/$NAME
```

## Loading Data

```{r}
pbmc_10x_dir <- "raw_feature_bc_matrix"
# for now..
mito_file <- "https://raw.githubusercontent.com/AlexsLemonade/scpca-downstream-analyses/main/reference-files/Homo_sapiens.GRCh38.104.mitogenes.txt"



pbmc_sce_raw <- DropletUtils::read10xCounts(pbmc_10x_dir)
mito_genes <- readr::read_tsv(mito_file, 
                              col_names="gene") %>%
  # only keep relevant mitochondrial genes
  dplyr::filter(gene %in% rownames(pbmc_sce_raw)) %>%
  dplyr::pull(gene)

# let's have a quick look at the raw SCE and genes:
pbmc_sce_raw
mito_genes
```

### Split antibody tag data
Before proceeding with further filtering, we'll want to re-arrange the SCE object to handle the RNA and ADT separately.
Currently, the ADTs and genes are both part of the primary experiment, so let's move the ADT counts into an `altExp` slot.

```{r}
# We can see this in the `Type` column in the rowData() where the creators of this dataset
#  stored experiment information
unique(rowData(pbmc_sce_raw)$Type)

# Split out the ADT into an altExp
pbmc_sce_split <- splitAltExps(pbmc_sce_raw, 
                               #"Gene Expression" or "Antibody Capture"
                               rowData(pbmc_sce_raw)$Type, 
                               # The Type which should be the main experiment
                               "Gene Expression")


# The names for the experiments were directly derived from `Type`:
mainExpName(pbmc_sce_split)
altExpNames(pbmc_sce_split)

# But here on the computer, we don't like spaces, so let's rename it to something informative
#  and space-free, like "CITEseq" or "ADT". We'll use "ADT" since typing less > typing more
altExpNames(pbmc_sce_split) <- "ADT"

# And confirm:
altExpNames(pbmc_sce_split)

# We can do this for main if we want too!
#mainExpName(pbmc_sce_split) <- "RNA"
```

### Quick ADT exploration

Let's look at our ADTs:

```{r}
# Here's how we can see the altExp itself - it's its own SCE object
altExp(pbmc_sce_split)

# What are the tags?
rownames(altExp(pbmc_sce_split))
```

These are all human surface proteins except `IgG1`, which is a mouse isotype that was used as a control. 
If there's no contamination, there should be 0 counts for this protein since the data is human:

```{r}
# Here's the counts assay:
counts(altExp(pbmc_sce_split))

# Use rowSums to see totals:
# Wow, almost no IgG1! Hurray. 
rowSums(counts(altExp(pbmc_sce_split)))
```


> At this point we could consider removing cells with IgG1 > 0 and remove the antibody entirely.

### Filter RNA

#### First, with `DropletUtils`

```{r}
# Filter RNA
droplet_df <- DropletUtils::emptyDropsCellRanger(
  counts(pbmc_sce_split), 
  BPPARAM = BiocParallel::MulticoreParam(4)
) 

# Filter down to low FDR cells and check out dimensions
# Values here are mostly NA.
# NOTE: this filtering does propagate to altExp!! :) 
cells_to_retain <- which(droplet_df$FDR <= 0.01)
filtered_sce <- pbmc_sce_split[, cells_to_retain]
dim(pbmc_sce_split)  # before filtering
dim(filtered_sce)  # after filtering
```


#### Now, with `miQC`

We'll need some mitochondria stats for `miQC`:

```{r}
filtered_sce <- scater::addPerCellQC(
  filtered_sce, 
  subsets = list(mito = mito_genes))


# See that `sum` is RNA only and `total` is both RNA and ADT?
head(filtered_sce$sum)
head(filtered_sce$total)

all(filtered_sce$total >= filtered_sce$sum) # total is never going to be LOWER than sum
```

```{r}
miqc_model <- miQC::mixtureModel(filtered_sce) 
filtered_sce <- miQC::filterCells(filtered_sce,
                                  model = miqc_model)

dim(filtered_sce)
```


### Filter ADT

We're not going to talk about this in depth, since it's still an active area of study! 
But, be aware that RNA and ADTs are treated differently. 
As always we recommend [Orchestrating](ttp://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html).

We will focus on only removing 0 count cells.

```{r}
# Run some QC on the tags
adt_qc_df <- DropletUtils::cleanTagCounts(altExp(filtered_sce))
adt_qc_df

# Keep only cells where discard is TRUE, indicating that ambient concentrations are neither too high nor too low.
cells_to_retain <- which(adt_qc_df$discard == FALSE)
filtered_sce <- filtered_sce[, cells_to_retain]
dim(filtered_sce)
```


### Normalize RNA and ADT:


```{r}
# Normalize RNA
qclust <- scran::quickCluster(filtered_sce)
filtered_sce <- scran::computeSumFactors(filtered_sce, clusters = qclust)
norm_sce <- scater::logNormCounts(filtered_sce)

# Normalize ADT using median counts
baseline <- DropletUtils::ambientProfileBimodal(altExp(norm_sce)) 
median_sf <- scater::medianSizeFactors(altExp(norm_sce), reference = baseline)
sum(median_sf == 0) # make sure nothing is 0!

# Normalize with the median size factors
altExp(norm_sce) <- scater::logNormCounts(altExp(norm_sce), 
                                          size.factors = median_sf)

```

### Behold, the filtered and normalized SCE:

```{r}
norm_sce

altExp(norm_sce)
```

## Dimension reduction

HVGs:
```{r}
num_genes <- 5000 # 2000? 

gene_variance <- scran::modelGeneVar(norm_sce)
hvg_list <- scran::getTopHVGs(gene_variance,
                               n = num_genes)
```

PCA:
```{r}
# Do we want to set the number of PCs?
norm_sce <- scater::runPCA(norm_sce,
                           subset_row = hvg_list)
```

UMAP:
```{r I map you map we all map for MapMap}
# Perform graph clustering first
nn_k <- 20
nn_clusters <- clusterRows(
  reducedDim(norm_sce, "PCA"), 
  NNGraphParam(k = nn_k, 
               cluster.fun = "louvain",
               BPPARAM = BiocParallel::MulticoreParam(4)
  )
)

# pop back into sce
norm_sce$nn_clusters <- factor(nn_clusters)

# Add in UMAPs and plot with colors
norm_sce <- scater::runUMAP(norm_sce,
                            dimred = "PCA")
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "nn_clusters")
```


## Cell type annotation from ADTs


We can leverage the ADTs as protein markers to identify likely cell types.

```{r}
# Again, what are the proteins?
rownames(altExp(norm_sce))
```


Let's explore via UMAPs, for example by coloring ADT counts:


```{r does anybody have a UMAP anybody maybe happen to know how to heck to plot this}

# For example CD8
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "CD8", 
                       point_alpha = 0.6, point_size = 0.25)
```

```{r}
# CD4, another T cell marker
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "CD4", 
                       point_alpha = 0.6, point_size = 0.25)
```

```{r}
# CD3 which lives on both CD4+ and CD8+ cells
# notice the greens coalescing!
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "CD3", 
                       point_alpha = 0.6, point_size = 0.25)
```


We can get a bit more sophisticated and look at the the most prevalent ADT across cells:


```{r}

# Which ADT is the most prevalent for each cell? 
max_adt_indices <- apply(
  logcounts(altExp(norm_sce)),
  2,
  which.max
)
max_adt <- rownames(altExp(norm_sce))[max_adt_indices]

unique(max_adt)

# Add this into the main SCE experiment for UMAPing
norm_sce$max_adt <- max_adt
```


What are these proteins, and what cell types might they indicate?
In the source quotes below, I have added the bold to the cell types.

- `CD3`
  - "CD3 (cluster of differentiation 3) is a protein complex and T cell co-receptor that is involved in activating both the **cytotoxic T cell (CD8+ naive T cells) and T helper cells (CD4+ naive T cells).** ... This high specificity, combined with the presence of CD3 at all stages of T-cell development, makes it a useful immunohistochemical marker for T-cells in tissue sections."  [source](https://en.wikipedia.org/wiki/CD3_(immunology))
- `CD4`
  - TCR co-receptor that we can probably assume is mostly on CD4+ T-cells
- `CD8`
  - TCR co-receptor that we can probably assume is mostly on CD8+ T-cells
- `CD11c`
  - "CD11c is a type I transmembrane protein that is expressed on **monocytes**, **granulocytes**, a subset of **B cells**, **dendritic cells**, and **macrophages** and may be demonstrated on B-cell chronic lymphocytic leukemia, marginal zone lymphomas, and hairy cell leukemia." [source](https://www.sciencedirect.com/topics/neuroscience/cd11c)
- `CD14`
  - "CD14 is a glycolipid-anchored membrane glycoprotein expressed on cells of the myelomonocyte lineage including **monocytes**, **macrophages**, and some **granulocytes**." [source](https://www.sciencedirect.com/topics/neuroscience/cd14)
- `CD16`
  - "CD16 is involved in antibody-dependent cell-mediated cytotoxicity and is expressed on large granular lymphocytes (LGLs) of both **NK- and T-cell types**." [source](https://www.sciencedirect.com/topics/neuroscience/cd16)
- `CD19`
  - "CD19 is an IgSF surface glycoprotein of 95 kDa that is expressed from the earliest stages of **B cell** development until plasma cell terminal differentiation, when its expression is lost." [source](https://www.sciencedirect.com/topics/neuroscience/cd19)
- `CD56`
  - "CD56 expression is also found in, among others, the hematopoietic system. Here, the expression of CD56 is mostly associated with, but not limited to, **natural killer cells**. CD56 has been detected on other lymphoid cells, including **gamma delta Τ cells and activated CD8+ T cells**, as well as on **dendritic cells**." [source](https://en.wikipedia.org/wiki/Neural_cell_adhesion_molecule)



Let's color cells by their most prevalent ADT:

```{r does anybody have a UMAP}
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "max_adt", 
                       point_alpha = 0.6, point_size = 0.25)
```




## Cell type annotation from RNA

We can introduce annotation with `SingleR`, and it might be a good idea to show how the results are highly influenced by the reference. 
For now, I'm just using the `HumanPrimaryCellAtlasData` reference as a starting place.
Note that all references here involve some caching, and we can expect some interactive prompting to go by for setting up these caches that we'll have to walk everyone through.


```{r}
ref_data <- celldex::HumanPrimaryCellAtlasData(ensembl = TRUE) # pick one for now to start
singleR_predictions <- SingleR::SingleR(
  test = norm_sce, 
  ref = ref_data,
  labels = ref_data$label.main, # "coarse-grained" labels
  # faster!
  BPPARAM = BiocParallel::MulticoreParam(4)
)
preds

# We can place these into the sce
norm_sce$celltype_predictions <- preds$pruned.labels

# **heavily** biased towards monocytes and T-cells
celltype_counts <- preds %>%
  as.data.frame() %>%
  dplyr::count(pruned.labels) %>%
  dplyr::arrange(desc(n))

celltype_counts

```



Visualize the predictions in a couple ways:
```{r}
SingleR::plotScoreHeatmap(preds)
```


```{r UMAP for the people}
# Not too shabby!
scater::plotReducedDim(norm_sce, "UMAP", colour_by = "celltype_predictions", 
                       point_alpha = 0.4, point_size = 0.25)
```


It might be helpful to hone in on the most prevalent cell types only for ease of visualization, but this is a bit more advanced to dive into.

```{r}
# The top 5 cell types account for over 98% of the cells!
celltype_counts %>%
  dplyr::slice(1:5) %>%
  dplyr::summarize(sum(n) / sum(celltype_counts$n))

top_celltypes <- as.character(celltype_counts$celltype[1:5]) 

# Recode a new celltype vector with top 5 separated from "other" using some forcats magic
norm_sce$celltype_predictions_top <- forcats::fct_lump_n(norm_sce$celltype_predictions, 5)

scater::plotReducedDim(norm_sce, "UMAP", colour_by = "celltype_predictions_top", 
                       point_alpha = 0.6, point_size = 0.25)
```

## Session Info

```{r}
sessionInfo()
```
