---
title: "Mitochondrial Gene Lists"
author: Joshua Shapiro for the CCDL
date: 2022-11
output: 
  html_notebook: 
    toc: true
    toc_float: true
---

This notebook generates a list of mitochondrial genes for use in scRNA-seq quality control.

Most of the functionality of this notebook is accomplished via Bioconductor annotations. For more information about using Bioconductor annotation packages, see the
[AnnotationDbi package vignette](http://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf).

To get current annotations, we will use [AnnotationHub](https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html).

## Setup

```{r setup}
# magrittr for pipes
library(magrittr)
```


First some locations for the gene lists we will be making. These will be in the /shared directory

```{r paths}
data_dir = "/shared/data/training-modules/scRNA-seq-advanced/data"

# the human gene list will be in the reference subfolder.
hs_mito_file <- file.path(data_dir, "reference", "hs_mitochondrial_genes.tsv")

```


## Human Ensembl Annotation

### Retrieving the db

To get the annotations, we will first create an `AnnotationHub` object, which is used as the interface to the annotation files that the Bioconductor project makes available.
We then query it for to find out what Ensdb packages (containing Ensembl annotations) are available for *Homo sapiens*.

```{r}
hub <- AnnotationHub::AnnotationHub()
```

```{r}
AnnotationHub::query(hub, c("Ensdb", "Homo sapiens"))
```
We are interested in the annotation package for Ensembl 95, which was what we used for the the annotations we built this time around.
Lets add that to the query.

```{r}
AnnotationHub::query(hub, c("Ensdb", "Homo sapiens", "95"))
```


The AnnotationHub ID for that is then `AH67950`.
The first time we select that item, AnnotationHub will download and save it to a new object that uses a standard Bioconductor annotation interface.
This will be cached locally in case we need it again, so this step may be slow the first time, but should be quick when repeated.

```{r}
ensdb_hs <- hub[['AH67950']] 
```

You may also see the `ensembldb` library load automatically at this point. 
We will need it to access the the elements of the database

### Finding mitochondrial genes

Now we can query the `ensdb_hs` with the AnnotationDbi functions that are part of Bioconductor.

First we will get the genes and store them, using the `genes()` function that was loaded along with the annotations.
This will create a `GenomicRanges` object, which is a way that Bioconductor packages store information about the coordinates of features like genes along a reference genome, (along with other information about the feature).

```{r}
ensg_hs <- genes(ensdb_hs)
```

Now we will select only those genes that are on the mitochondria (`MT`).
Here we are using the `GenomicRanges` function `seqnames()` to get the chromosome that each feature is on. 
Just to check, we can print the results out as a data frame for easy examination.

```{r}
mitogenes_hs <- ensg_hs[seqnames(ensg_hs) == 'MT']
as.data.frame(mitogenes_hs)
```

Encouragingly, there are 37 mitochondrial genes, which is correct.

Now some quick reformatting and we'll save the output table.

```{r}
mitogenes_hs %>%
  tibble::as_tibble() %>%
  dplyr::select(gene_id, gene_name, dplyr::everything()) %>%
  readr::write_tsv(hs_mito_file)
```

## Session Info

```{r}
sessionInfo()
```
